--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

SET search_path = public, pg_catalog;

--
-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY users (id, email, name, url, username, created_at, updated_at, orcid, area_of_expertise, deleted, deleted_at, role) FROM stdin;
\.


--
-- Data for Name: audits; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY audits (id, auditable_id, auditable_type, associated_id, associated_type, user_id, user_type, username, action, audited_changes, version, comment, remote_address, request_uuid, created_at) FROM stdin;
1	1	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 238\nname: ALK\ndescription: ALK amplifications, fusions and mutations have been shown to be driving\n  events in non-small cell lung cancer. While crizontinib has demonstrated efficacy\n  in treating the amplification, mutations in ALK have been shown to confer resistance\n  to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied\n  success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been\n  shown to be cytostatic in ALK-altered cell lines.\nofficial_name: anaplastic lymphoma receptor tyrosine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	c03b8fa9-9740-4b94-b4d2-c7efe2f4d366	2015-06-21 16:49:19.347573
2	2	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 207\nname: AKT1\ndescription: AKT1, also referred to as protein kinase B, is a known oncogene. AKT\n  activation relies on the PI3K pathway, and is recognized as a critical node in the\n  pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been\n  shown to result in activation of the protein. Mutations in AKT1 have also been shown\n  to confer resistance to allosteric kinase inhibitors in vitro.\nofficial_name: v-akt murine thymoma viral oncogene homolog 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	3b821bcb-a7bc-40b1-9e87-230069d72305	2015-06-21 16:49:19.637422
3	3	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 369\nname: ARAF\ndescription: ARAF has recently become increasingly considered for its oncogenic potential.\n  Its potential as a target for informing clinical action was demonstrated by a single\n  case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated\n  with sorafenib, acheived near-complete clinical remission. This finding has brought\n  new focus on ARAF as a marker that should be assayed for in cancer treatment.\nofficial_name: A-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	d9c06581-2b19-4628-9b3b-279e381870d6	2015-06-21 16:49:19.900461
4	4	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 25\nname: ABL1\ndescription: ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein\n  that has become a signature of chronic myeloid leukemia (CML). Cells harboring this\n  fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook\n  of the disease. However, additional mutations in ABL1 have been shown to confer\n  resistance to imatinib. In these resistance cases, second-generation tyrosine kinase\n  inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are\n  currently undergoing clinical trials for treating acquired resistance in CML.\nofficial_name: ABL proto-oncogene 1, non-receptor tyrosine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	6708ed78-acb7-468a-87bd-e14f3ea01730	2015-06-21 16:49:20.166317
5	5	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 673\nname: BRAF\ndescription: BRAF mutations are found to be recurrent in many cancer types. Of these,\n  the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E\n  has been determined to be an activating mutation, and cells that harbor it, along\n  with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is\n  also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK\n  inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations\n  have also been correlated with poor prognosis in many cancer types, although there\n  is at least one study that questions this conclusion in papillary thyroid cancer.\nofficial_name: B-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	fc01f6d6-a208-4528-a703-53213f584cf9	2015-06-21 16:49:20.445607
6	6	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 672\nname: BRCA1\ndescription: BRCA1 mutations in the germline have become a hallmark for hereditary\n  breast and ovarian cancers. Variants that have been demonstrated to reduce the function\n  of the protein have been shown to increase risk for these cancers, as well as prostate\n  and pancreatic cancer. These findings have been the impetus for the increased popularity\n  of genetic testing of healthy indivudals to assess risk.\nofficial_name: breast cancer 1, early onset\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	99a283ff-ef45-4573-ba05-21ba79d9f5f4	2015-06-21 16:49:20.74189
7	7	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 675\nname: BRCA2\ndescription: BRCA2 mutations in the germline have become a hallmark for hereditary\n  breast and ovarian cancers. Variants that have been demonstrated to reduce the function\n  of the protein have been shown to increase risk for these cancers, as well as prostate\n  and pancreatic cancer. These findings have been the impetus for the increased popularity\n  of genetic testing of healthy indivudals to assess risk.\nofficial_name: breast cancer 2, early onset\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	6ccd6187-56ab-4ecf-a4e8-ce2a13adde2e	2015-06-21 16:49:21.05197
8	8	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 595\nname: CCND1\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	7824b55b-e395-40c7-9de8-593114b5606b	2015-06-21 16:49:21.306763
9	9	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 894\nname: CCND2\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	67537e51-a728-486b-8bf3-79219e62b110	2015-06-21 16:49:21.5485
10	10	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 896\nname: CCND3\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared\n  to the amplifcations and overexpressions of the other cyclin D's. In a mouse study,\n  the targeted therapeutic palbociclib significantly increased the median survival\n  of the cyclin D3 knockouts.\nofficial_name: cyclin D3\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	370ce980-b80b-409e-8fd6-e7609631c092	2015-06-21 16:49:21.762601
11	11	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 898\nname: CCNE1\ndescription: Cyclin E, while currenly not as widely implicated as its cyclin D counterparts,\n  has been implicated in various carcinomas, including breast, gastric, stomach and\n  colorectal. High levels of cyclin E, either by gene amplification or overexpression,\n  have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements\n  are correlated with later stage disease. In lung cancer, neoplastic cells with higher\n  levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly\n  expressed counterparts.\nofficial_name: cyclin E1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b25e2e5c-66a7-4641-8e0c-6f81fe05bba1	2015-06-21 16:49:21.989071
12	12	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1021\nname: CDK6\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 6\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	7eb1d8f3-bfce-4ad4-b2c6-5965553a9d1b	2015-06-21 16:49:22.219549
13	13	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1019\nname: CDK4\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 4\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	ef3cb189-6d5f-4ea9-b684-5ff9d95ef4f5	2015-06-21 16:49:22.45564
14	14	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1029\nname: CDKN2A\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. While no targeted therapeutic has been engaged in clinical trials, the prognostic\n  impact has been studied by a number of meta-analyses. One mechanism by which loss\n  of CDKN2A can occur is by hypermethylation of the promoter region for the gene.\n  However, the prognostic impact of promoter hypermethylation has been relatively\n  ambiguous. Many studies have suggesting poorer prognostic outcome for patients with\n  hypermethylation in colorectal, liver, and younger lung cancer patients. This being\n  said, there is still research to be done before this becomes a widely-accepted prognostic\n  indicator.\nofficial_name: cyclin-dependent kinase inhibitor 2A\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	3f7bb55d-7d11-4c42-a869-e3a52e07e4c9	2015-06-21 16:49:22.782823
15	15	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1050\nname: CEBPA\ndescription: AML with mutated CEBPA' is a provisional entity in the WHO classification\n  of acute myeloid leukemia (AML) and is recommended to be tested for in patients\n  with AML. CEBPA mutations are particularly associated with cytogenetically normal\n  AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation\n  in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha\n  protein while C-terminal mutations reduce the DNA-binding potential of this transcription\n  factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1\n  and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations\n  may have prognostic implications.\nofficial_name: CCAAT/enhancer binding protein (C/EBP), alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	6eb1d286-c905-43bd-9237-318f37a07a4f	2015-06-21 16:49:23.065241
16	16	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3337\nname: DNAJB1\ndescription: DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating\n  p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1\n  to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In\n  one study, this fusion was observed in 15/15 FL-HCC cases examined and functional\n  studies found that the fusion retained kinase activity.\nofficial_name: DnaJ (Hsp40) homolog, subfamily B, member 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	07414dee-5291-4b2d-afe6-58e3f4ee059f	2015-06-21 16:49:23.317228
17	17	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5566\nname: PRKACA\ndescription: PRKACA has been studied in breast cancer and has been found to mediate\n  resistance to HER2 targeted therapies. It has also been found to contain a mutation\n  hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular\n  fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be\n  a diagnostic marker for this rare subtype of HCC. In one study, this fusion was\n  observed in 15/15 FL-HCC cases examined and functional studies found that the fusion\n  retained kinase activity.\nofficial_name: protein kinase, cAMP-dependent, catalytic, alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	74baeff3-ecf9-41c8-b856-a21b112ad211	2015-06-21 16:49:23.551875
18	18	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1788\nname: DNMT3A\ndescription: DNMT3A is one of several epigenetic modifiers identified as recurrently\n  mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically\n  normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant\n  negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers.\n  AML patient bone marrow harboring R882 mutations were similarly demonstrated to\n  be hypomethylated compared to patients with wildtype DNMT3A. These studies also\n  indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner\n  from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes\n  and treatment protocols may need to be considered for these two classes of DNMT3A\n  mutations.\nofficial_name: DNA (cytosine-5-)-methyltransferase 3 alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	23034c5a-1043-400a-a551-830047dcc4bb	2015-06-21 16:49:23.785273
19	19	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1956\nname: EGFR\ndescription: EGFR is widely recognized for its importance in cancer. Amplification\n  and mutations have been shown to be driving events in many cancer types. Its role\n  in non-small cell lung cancer has spurred many research and drug development efforts.\n  Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably\n  gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance\n  to these drugs, particularly the variant T790M, which has been functionally characterized\n  as a resistance marker for both of these drugs. The later generation TKI's have\n  seen some success in treating these resistant cases, and targeted sequencing of\n  the EGFR locus has become a common practice in non-small cell lung cancer.\nofficial_name: epidermal growth factor receptor\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	53486888-790a-48a3-bcba-0190c0165919	2015-06-21 16:49:24.044767
57	57	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5966\nname: REL\ndescription: ''\nofficial_name: v-rel avian reticuloendotheliosis viral oncogene homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	73fe6edd-4c3c-401b-a01e-8c0672a146af	2015-06-21 16:49:33.644569
59	59	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 596\nname: BCL2\ndescription: ''\nofficial_name: B-cell CLL/lymphoma 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	a123ccd8-2281-4d01-a650-f49cbcc703c4	2015-06-21 16:49:34.109838
20	20	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2064\nname: ERBB2\ndescription: ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast\n  cancer, and is treated in a separate manner than the other subtypes of breast cancer.\n  Apart from being amplified, ERBB2 activating mutations have been shown to have clinical\n  importance in HER2-negative breast cancer. These mutations have shown sensitivity\n  to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical\n  sequencing efforts in treating breast cancer.\nofficial_name: erb-b2 receptor tyrosine kinase 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	108542d1-42b9-45ac-8c3d-8ee0d66df04b	2015-06-21 16:49:24.331627
21	21	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2099\nname: ESR1\ndescription: ESR1 has been a focus in breast cancer for quite some time, but has also\n  shown significance in endometrial, ovarian and other cancer types. ER-positive breast\n  cancer that is resistant to hormone therapy has instigated clinical sequencing efforts\n  to shed light on the mechanisms of this resistance. A number of mutations in the\n  ligand binding domain of the protein have been implicated in hormone resistance\n  and anti-estrogen therapies. This has spurred efforts to develop therapeutics that\n  act to degrade the protein, rather than act as an antagonist. These agents are currently\n  in clinical trials and have seen some success, highlighting the importance of sequencing\n  efforts in treating breast cancer.\nofficial_name: estrogen receptor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	41dc4ad3-1538-4dbf-8e1b-65626ec4d121	2015-06-21 16:49:24.612589
22	22	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2263\nname: FGFR2\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions that lead to pathway\n  actiation have been demonstrated to have oncogenic potential across multiple cancer\n  types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success\n  in treating over-active FGFR signalling, prompting use of diagnostic sequencing\n  targeting the FGFR genes, especially in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	a554284b-53f1-4f3d-8815-56484af55fba	2015-06-21 16:49:24.921199
23	23	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2261\nname: FGFR3\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions that lead to pathway\n  actiation have been demonstrated to have oncogenic potential across multiple cancer\n  types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success\n  in treating over-active FGFR signalling, prompting use of diagnostic sequencing\n  targeting the FGFR genes, especially in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 3\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	49b122ce-a46c-4b23-bfed-0d3b0a0aa866	2015-06-21 16:49:25.200508
24	24	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2322\nname: FLT3\ndescription: FLT3 is an important cytokine receptor involved in normal hematopoiesis.\n  Mutations in this gene are common in acute myeloid leukemia (AML) and screening\n  for mutations in this gene has been recommended by the World Health Organization\n  in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML).\n  FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated\n  with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have\n  been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD\n  mutations are still up for debate.\nofficial_name: fms-related tyrosine kinase 3\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	fac059ee-056e-4b2b-b208-8912e287d109	2015-06-21 16:49:25.455101
25	25	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2624\nname: GATA2\ndescription: GATA2 is a transcription factor involved in stem cell maintenance with\n  key roles in hematopoietic development. GATA2 mutations are associated with a variety\n  of inherited and acquired immune disorders including myelodysplastic syndrome and\n  acute myeloid leukemia. In addition to a role in hematopoiesis, the maintenance\n  GATA2 expression has been implicated as a requirement in KRAS-driven non-small cell\n  lung cancer. Preclinical models have indicated therapeutic benefit from targeting\n  GATA2-mediated pathways in the context of KRAS-driven NSCLC.\nofficial_name: GATA binding protein 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	353261ea-5102-43ff-842f-d7f272566682	2015-06-21 16:49:25.705621
26	26	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3417\nname: IDH1\ndescription: IDH1 mutations have been observed in a number of cancer types, including\n  sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in\n  the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial\n  (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described\n  as mutually exclusive in many of these cancer types. The most frequent mutations\n  involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic\n  enzyme activity. The implications of mutations in this gene vary greatly by cancer\n  type. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations\n  have been associated with worse outcome, shorter overall survival, and normal karyotype.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Unlike the association\n  with cytogenetically normal AML, in glioblastoma, IDH1 mutations have been associated\n  with specific cytogenetic abnormalities, 1p and 19q deletions.\nofficial_name: isocitrate dehydrogenase 1 (NADP+), soluble\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	7ce11f7b-7a4b-4a29-a19e-19f00a8e3def	2015-06-21 16:49:25.965164
27	27	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3418\nname: IDH2\ndescription: IDH2 mutations have been observed in a number of cancer types, including\n  sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in\n  the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial\n  (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described\n  as mutually exclusive in many of these cancer types. The most frequent mutations\n  involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic\n  enzyme activity. Although IDH2 (R172) mutations are associated with poorer overall\n  prognosis in AML patients, its utility as a prognostic marker in MDS is still under\n  debate. Additionally, IDH2 (R140) has been associated with improved overall survival\n  in AML. IDH2 mutations have been associated with improved prognosis in gliomas.\nofficial_name: isocitrate dehydrogenase 2 (NADP+), mitochondrial\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	5ca00cb7-79d7-48ff-9836-48e96821dacb	2015-06-21 16:49:26.262031
28	28	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3717\nname: JAK2\ndescription: JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative\n  diseases and cancers. The mutation V617F is the most clinically relevant variant,\n  and is seen in around half of myeloproliferative disorders. The variant is a known\n  activating mutation, and activated JAK2 is sufficient to drive myeloproliferative\n  disorders in mouse models. V617F, while most recurrent, is not the only mechanism\n  by which JAK2 can be activated in patients. JAK2 is now one of the first diagnostic\n  markers tested upon diagnosis with a myeloproliferative disorder.\nofficial_name: Janus kinase 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	50d5a025-ca0f-4341-ac5e-d20264c7302a	2015-06-21 16:49:26.530283
29	29	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3815\nname: KIT\ndescription: c-KIT activation has been shown to have oncogenic activity in gastrointestinal\n  stromal tumors (GISTs), melanomas, lung cancer, and other tumor types. The targeted\n  therapeutics nilotinib and sunitinib have shown efficacy in treating KIT overactive\n  patients, and are in late-stage trials in melanoma and GIST. KIT overactivity can\n  be the result of many genomic events from genomic amplification to overexpression\n  to missense mutations. Missense mutations have been shown to be key players in mediating\n  clinical response and acquired resistance in patients being treated with these targeted\n  therapeutics.\nofficial_name: v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	09baa298-dafc-4731-934f-0bab2ac7d425	2015-06-21 16:49:26.773203
30	30	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3845\nname: KRAS\ndescription: Mutations in the RAS family of proteins have frequently observed across\n  cancer types. The amino acid positions account for the overwhelming majority of\n  these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also\n  behave very differently when expressed in non-native tissue types, likely due to\n  differences in the C-terminal hyper-variable regions. Mis-regulation of isoform\n  expression has been shown to be a driving event in cancer, as well as missense mutations\n  at the three hotspots previously mentioned. While highly recurrent in cancer, targeted\n  these RAS mutants has also been very elusive, and has not yet become common practice\n  in the clinic.\nofficial_name: Kirsten rat sarcoma viral oncogene homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	fe100c10-ad32-46a0-88ba-21205d3c752b	2015-06-21 16:49:27.052065
31	31	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5604\nname: MAP2K1\ndescription: ''\nofficial_name: mitogen-activated protein kinase kinase 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	08b39456-7c60-4fcf-8235-64b8858208bd	2015-06-21 16:49:27.308143
32	32	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4209\nname: MEF2D\ndescription: ''\nofficial_name: myocyte enhancer factor 2D\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	0f0825ef-8529-4c00-a145-39f4d6d26e63	2015-06-21 16:49:27.541583
33	33	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1436\nname: CSF1R\ndescription: ''\nofficial_name: colony stimulating factor 1 receptor\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	c2c0b787-9d21-45a4-9254-cc2e3ffb62fa	2015-06-21 16:49:27.783591
34	34	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4255\nname: MGMT\ndescription: ''\nofficial_name: O-6-methylguanine-DNA methyltransferase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	baa853a4-cc68-4861-95bc-c63770192274	2015-06-21 16:49:28.008196
35	35	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4869\nname: NPM1\ndescription: AML with mutated NPM1 is provisional entity in the WHO classification\n  of AML and is recommended to be tested for in patients with cytogenetically normal\n  AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they\n  have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently\n  W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations\n  in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing\n  the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia,\n  indicating it may require additional mutations to promote leukemic development.\nofficial_name: nucleophosmin (nucleolar phosphoprotein B23, numatrin)\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	1528fba3-c44d-494c-b839-cd4a6579e903	2015-06-21 16:49:28.247952
36	36	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4893\nname: NRAS\ndescription: Mutations in the RAS family of proteins have frequently observed across\n  cancer types. The amino acid positions account for the overwhelming majority of\n  these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also\n  behave very differently when expressed in non-native tissue types, likely due to\n  differences in the C-terminal hyper-variable regions. Mis-regulation of isoform\n  expression has been shown to be a driving event in cancer, as well as missense mutations\n  at the three hotspots previously mentioned. While highly recurrent in cancer, targeted\n  these RAS mutants has also been very elusive, and has not yet become common practice\n  in the clinic.\nofficial_name: neuroblastoma RAS viral (v-ras) oncogene homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	6a4b2822-8fea-45c1-ac47-e1ba107bfda7	2015-06-21 16:49:28.493883
37	37	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5290\nname: PIK3CA\ndescription: PIK3CA is the most recurrently mutated gene in breast cancer, and has\n  been found to important in a number of cancer types. An integral part of the PI3K\n  pathway, PIK3CA has long been described as an oncogene, with two main hotspots for\n  activating mutations, the 542/545 region of the helical domain, and the 1047 region\n  of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways,\n  is the subject of an immense amount of research and development, and PI3K inhibition\n  has seen some limited success in recent clinical trials. While monotherapies seem\n  to be limited in their potential, there is a recent interest in pursuing PI3K inhibition\n  as part of a combination therapy regiment with inhibition partners including TKI's,\n  MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.\nofficial_name: phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	f70dec4e-f77b-4447-99c3-b0b05f520576	2015-06-21 16:49:28.74398
38	38	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5156\nname: PDGFRA\ndescription: ''\nofficial_name: platelet-derived growth factor receptor, alpha polypeptide\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	41eaa997-2160-4e3d-bfe7-46ded227b6e2	2015-06-21 16:49:28.988651
39	39	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5371\nname: PML\ndescription: The PML-RARA fusion is the result of a recurrent, balanced translocation\n  between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event\n  in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown\n  sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA\n  fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for\n  treating these patients, and early results seem promising.\nofficial_name: promyelocytic leukemia\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	579dbed9-6f97-4ef2-96b0-3361f019a64c	2015-06-21 16:49:29.221859
58	58	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7428\nname: VHL\ndescription: ''\nofficial_name: von Hippel-Lindau tumor suppressor, E3 ubiquitin protein ligase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	d4e0ef09-fb86-40a1-8409-e8eadf1861a7	2015-06-21 16:49:33.87318
40	40	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5914\nname: RARA\ndescription: The PML-RARA fusion is the result of a recurrent, balanced translocation\n  between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event\n  in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown\n  sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA\n  fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for\n  treating these patients, and early results seem promising.\nofficial_name: retinoic acid receptor, alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	4eb06a28-ac81-4738-a07c-539b84e83efb	2015-06-21 16:49:29.450433
41	41	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5728\nname: PTEN\ndescription: ''\nofficial_name: phosphatase and tensin homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	9d2f7fa6-f727-4ea4-9796-8cdf7f1301c8	2015-06-21 16:49:29.713195
42	42	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5979\nname: RET\ndescription: RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine\n  kinase receptor and are associated with thyroid cancers. RET point mutations are\n  the most common mutations identified in medullary thyroid cancer (MTC) with germline\n  and somatic mutations in RET associated with hereditary and sporadic forms, respectively.\n  The most common somatic form mutation is M918T (exon 16) and a variety of other\n  mutations effecting exons 10, 11 and 15 have been described. The prognostic significance\n  of these mutations have been hotly debated in the field, however, data suggests\n  that some RET mutation may confer drug resistence. No RET-specific agents are currently\n  clinically available but several promiscuous kinase inhibitors that target RET,\n  among others, have been approved for MTC treatment.\nofficial_name: ret proto-oncogene\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	e5f02bec-5da5-436c-a695-1babca91e904	2015-06-21 16:49:29.996572
43	43	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 861\nname: RUNX1\ndescription: ''\nofficial_name: runt-related transcription factor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	a49d9c2a-7ade-4449-a94a-c18a468562aa	2015-06-21 16:49:30.259693
44	44	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 23451\nname: SF3B1\ndescription: SF3B1 mutations have been described in several myeloid malignancies,\n  predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies\n  and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has\n  been identified as recurrently mutated in MDS and other malignanices. The mutations\n  affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally\n  deleterious with K700E described as a major hotspot mutation. MDS patients with\n  SF3B1 mutations have been reported to have better overall and event-free survival\n  than their wildtype counterparts. Additionally, these mutations are highly associated\n  with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with\n  ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring\n  sideroblasts).\nofficial_name: splicing factor 3b, subunit 1, 155kDa\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	d23dd8bf-eb79-497e-a1d6-0a2fb5d8d584	2015-06-21 16:49:30.518728
45	45	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7157\nname: TP53\ndescription: TP53 mutations are universal across cancer types. Loss of tumor suppressors\n  is most recognized by large deleterious events, such as frameshift mutations, or\n  premature stop codons. In TP53 however, many of the observed mutations in cancer\n  are found to be single nucleotide variants, or missense mutations. These variants\n  are also very broadly distributed throughout the gene, not localizing in any particular\n  hotspot. While a large proportion of cancer genomics research is focused on somatic\n  variants, TP53 is also of note in the germline. Germline TP53 mutations are the\n  hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been\n  found to have prognostic impact on patient outcomes.\nofficial_name: tumor protein p53\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	303a96be-f479-4b48-9c70-6bc2f5c98667	2015-06-21 16:49:30.777785
46	46	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7248\nname: TSC1\ndescription: ''\nofficial_name: tuberous sclerosis 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	92d77536-81d4-4345-8895-b18907657dc3	2015-06-21 16:49:31.004563
47	47	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7249\nname: TSC2\ndescription: ''\nofficial_name: tuberous sclerosis 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	af423409-d280-4a2f-948c-344b2e02e7a5	2015-06-21 16:49:31.233804
48	48	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7307\nname: U2AF1\ndescription: U2AF1 is one of several spliceosome complex genes frequently mutated\n  in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes\n  (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot\n  mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein.\n  These mutations have been associated with altered splicing patterns in vitro and\n  in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation\n  to secondary acute myeloid leukemia, however, the impact of these mutations on overall\n  survival has been an area of debate.\nofficial_name: U2 small nuclear RNA auxiliary factor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	3d3359c9-15cb-494f-aa25-4ed3bbc958b8	2015-06-21 16:49:31.48441
49	49	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7490\nname: WT1\ndescription: WT1 is a tumor suppressor gene associated with the development of Wilms'\n  Tumor, from which it was named. Mutations in exon 7 and 9 of WT1 have been recurrently\n  identified in acute myeloid leukemia and associated with poorer prognosis and chemotherapy\n  resistance.\nofficial_name: Wilms tumor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	d7688c38-6d1b-49ff-b8de-33864d499136	2015-06-21 16:49:31.748178
50	50	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4851\nname: NOTCH1\ndescription: ''\nofficial_name: notch 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	53d2d4d5-fe6b-4153-8e0e-23ec1c2ffebb	2015-06-21 16:49:32.008199
51	51	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4921\nname: DDR2\ndescription: ''\nofficial_name: discoidin domain receptor tyrosine kinase 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	476a2dc6-1407-4e48-99c9-22f18d0eea94	2015-06-21 16:49:32.262402
52	52	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4233\nname: MET\ndescription: ''\nofficial_name: MET proto-oncogene, receptor tyrosine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	7603ad54-7934-4107-80ba-f4b59ccf72a9	2015-06-21 16:49:32.503727
53	53	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2313\nname: FLI1\ndescription: ''\nofficial_name: Fli-1 proto-oncogene, ETS transcription factor\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	bbf8df34-5b02-47b5-95e1-2df145442871	2015-06-21 16:49:32.731429
54	54	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2130\nname: EWSR1\ndescription: ''\nofficial_name: EWS RNA-binding protein 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	d4b78c27-8d79-41b1-a6cc-b8bfe9cb94a5	2015-06-21 16:49:32.972323
55	55	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 54790\nname: TET2\ndescription: ''\nofficial_name: tet methylcytosine dioxygenase 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	54ffbab3-68f1-4b28-bdb1-4d6fe787f879	2015-06-21 16:49:33.182236
56	56	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 27086\nname: FOXP1\ndescription: ''\nofficial_name: forkhead box P1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	80734891-ead0-4157-9a68-7236f1a01369	2015-06-21 16:49:33.423309
60	60	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3481\nname: IGF2\ndescription: ''\nofficial_name: insulin-like growth factor 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	0f3467a2-ecb2-478f-b2f7-986a97673b90	2015-06-21 16:49:34.349691
61	61	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 6790\nname: AURKA\ndescription: ''\nofficial_name: aurora kinase A\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	29def105-b218-45c1-b50e-b182c8171b32	2015-06-21 16:49:34.610751
62	62	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 55193\nname: PBRM1\ndescription: ''\nofficial_name: polybromo 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	60a4dd44-33b4-49d9-af07-e976c2d758a7	2015-06-21 16:49:34.842293
63	63	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2146\nname: EZH2\ndescription: ''\nofficial_name: enhancer of zeste 2 polycomb repressive complex 2 subunit\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	978c1514-efe7-4a53-bff1-3e1501f37805	2015-06-21 16:49:35.111211
64	1	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL\ndescription: The BCR-ABL fusion protein, commonly referred to as the philadelphia\n  chromosome, is the most well-studied fusion gene in cancer. It has widely been considered\n  the initiating event in chronic myelogenous leukemia (CML), but despite its ability\n  initiate disease in mice, its status an initiating mutation is in dispute. In what\n  is commonly used as the poster-child for targeted therapeutics, the development\n  and use of imatinib in the clinic has led to profound improvements in the prognosis\n  of the disease. However, imatinib resistance is still seen in patients with mutations\n  in the ABL kinase domain of the fusion, most notably the T315I variant. In patients\n  resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's\n  (dasatinib and nilotinib) have seen some success in delivering a tumor response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	20af177a-3b5e-434e-a39e-302cf7abfc64	2015-06-21 16:49:35.20028
65	2	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL T315I\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation T315I has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	1089799d-8c66-4f08-b082-3028c7570667	2015-06-21 16:49:35.218366
66	3	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL E255K\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation E255K has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	7bb22095-b59a-4803-84dc-fb2af53272b5	2015-06-21 16:49:35.235748
67	4	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 2\nname: E17K\ndescription: AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal,\n  lung, and ovarian cancer. It has been convincingly shown to be an activating mutation\n  resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation\n  decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and\n  other AKT1 mutations, are the subject of much research and development for therapeutics.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	b6c59a14-2f87-4659-aa1f-054843fb28ac	2015-06-21 16:49:35.25367
68	5	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK\ndescription: The EML4-ALK fusion has been seen in non-small cell lung cancer, and\n  appears to be an alternative mechanism for ALK activation. Cells with this fusion\n  have been shown to be sensitive to the ALK inhibitor crizotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	a51dda41-31a8-4b16-834d-9f5ddacc7836	2015-06-21 16:49:35.272207
69	6	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK C1156Y\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	ebd92c19-d425-4ebd-844c-d6631b98ef10	2015-06-21 16:49:35.289251
70	7	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK L1196M\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	c4a1d3bf-4c04-478a-a6c6-bc5670f6c88e	2015-06-21 16:49:35.306731
71	8	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: F1174L\ndescription: ALK F1174L has been observed as recurrent in neuroblastoma, non-small\n  cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing\n  this mutation have shown resistance to low doses of criztonib. However, increased\n  dosage can overcome this resistance in cell lines studies. TAE684 has also proven\n  effective in both NSCLC and neuroblastoma F1174L containing cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	6dc0a28d-4fd4-4478-9792-cc47273c689e	2015-06-21 16:49:35.324914
72	9	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: R1275Q\ndescription: ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma,\n  non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma\n  cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This,\n  and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC\n  cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	c293e83b-224b-48e7-859c-3db545a60ddf	2015-06-21 16:49:35.342154
93	30	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	06b15d92-7efc-4e70-a7a8-125302eb3203	2015-06-21 16:49:35.772676
73	10	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 3\nname: S214C\ndescription: ARAF S214C has been found to be a recurrent oncogenic mutation in non-small\n  cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib\n  in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this\n  mutation, sorafenib also acheived near-complete clinical remission. This case has\n  brought more interest to the variant from a research and clinical perspective.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	81c3d400-57b3-4c97-938b-b5046f7fa22c	2015-06-21 16:49:35.365215
74	11	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600D\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	ba44e0d7-037b-4522-92fa-a1d2f10a4c23	2015-06-21 16:49:35.395495
75	12	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E\ndescription: BRAF V600E has been shown to be recurrent in many cancer types. It is\n  one of the most widely studied variants in cancer. This variant is correlated with\n  poor prognosis in certain cancer types, including colorectal cancer and papillary\n  thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective\n  in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib\n  has also shown to be effective when combined with the MEK inhibitor trametinib in\n  colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA\n  mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib,\n  and a number of combination therapies have been successful in treating V600E mutations.\n  While the drugs cetuximab and panitumumab have been largely shown to be ineffective\n  without supplementary treatment.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	84f994c0-5bff-4be0-86c0-06ab9b7327f4	2015-06-21 16:49:35.414165
76	13	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E/V600M\ndescription: A case study of a single patient harboring both a V600E and a V600M mutation,\n  dabrafenib was shown to acheive clinical response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	d4206c08-2fd8-48db-ad15-3fcd61d58d19	2015-06-21 16:49:35.432081
77	14	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E AMPLIFICATION\ndescription: Amplification of BRAF V600E has been shown to confer resistance to MEK\n  inhibitors. For more information on the V600 locus, see the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	8a491922-a01f-4446-87cc-d11fbadb2bf2	2015-06-21 16:49:35.450278
78	15	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600M\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	5888ba30-bc14-40b7-840e-d0ecded34a78	2015-06-21 16:49:35.469247
79	16	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600R\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	4ba25d7b-fb39-4807-84b6-73abd8c6ac78	2015-06-21 16:49:35.491918
80	17	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600\ndescription: BRAF mutations of the valine 600 residue have been shown to be recurrent\n  in many cancer types. Of the V600 mutations, V600E is the most widely researched.\n  V600 mutations as a whole have been correlated to poorer prognosis in colorectal\n  and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity\n  to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual\n  mutation pages on the left sidebar.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	3ef85414-3484-4bd9-aa23-348bb5449a24	2015-06-21 16:49:35.509162
81	18	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: AMPLIFICATION\ndescription: CCND1 amplification has been implicated in poorer prognosis in non-small\n  cell lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	827d9c08-bac3-4711-b7c4-dbe935eef04c	2015-06-21 16:49:35.527707
82	19	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: EXPRESSION\ndescription: CCND1 expression, and its prognositc impact, is still in dispute. Three\n  experiments in non-small cell lung cancer have shown it to have no impact on survival,\n  but three additional studies have shown it results in poorer prognosis. There is\n  also some ambiguity in how the boundaries between expression and overexpression\n  are defined.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	793317da-5387-4a6e-8210-5b6baa2bcc1c	2015-06-21 16:49:35.550341
83	20	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	0af4641b-565b-48b9-a365-03927a207c13	2015-06-21 16:49:35.56978
94	31	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in the rare form of adolescent\n  liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion\n  was observed in 15/15 FL-HCC cases examined and functional studies found that the\n  fusion retained kinase activity. The presence of this fusion may be used as a diagnostic\n  marker for this rare tumor type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	9603b743-6c15-4e57-862c-aac3da818831	2015-06-21 16:49:35.794727
84	21	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	46d07b93-8bea-4c77-8c1c-12a190cc167b	2015-06-21 16:49:35.589169
85	22	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: PROMOTER DEMETHYLATION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	785300aa-e0b8-46de-99b3-7f66584921be	2015-06-21 16:49:35.610738
86	23	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 10\nname: LOSS\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL),\n  a seemingly unique trend when compared to the amplifcations and overexpressions\n  of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic\n  palbociclib significantly increased the median survival of a Notch-driven model\n  of T-ALL.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	213e8c99-ab50-450d-817b-9e3599eacca1	2015-06-21 16:49:35.631302
87	24	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 11\nname: OVEREXPRESSION\ndescription: Cyclin E, while currenly not as widely implicated as its counterpart,\n  cyclin D, has been implicated in various carcinomas, including breast, gastric,\n  stomach and colorectal. High levels of cyclin E, either by gene amplification or\n  overexpression, are correlated with later stage disease and have been shown to lead\n  to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer\n  specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex\n  are more radiosensitive than their lowly expressed counterparts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	8fd1d333-6b7f-49d8-8137-db9d0b64ff12	2015-06-21 16:49:35.651836
88	25	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: EXPRESSION\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	60ce3ce4-638b-4498-8b24-e31e83bb7492	2015-06-21 16:49:35.672264
89	26	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 12\nname: EXPRESSION\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	6f1269df-9d13-466c-be49-d72b797e637b	2015-06-21 16:49:35.695521
90	27	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 14\nname: PROMOTER HYPERMETHYLATION\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the CDKN2A\n  promoter region. While no targeted therapeutic has been engaged in clinical trials,\n  the prognostic impact has been studied by a number of meta-analyses. The prognostic\n  impact of promoter hypermethylation has been relatively ambiguous. Many studies\n  have shown significant p-values suggesting poorer prognostic outcomes for patients\n  with hypermethylation in colorectal, liver, and younger lung cancer patients. This\n  being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	275df56a-cdd7-4259-ab4c-f26fe623810f	2015-06-21 16:49:35.714945
91	28	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: N-TERMINAL FRAME SHIFT\ndescription: CEBPA N-terminal frame shift mutations that result in a premature stop\n  codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML).\n  CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3\n  mutations should also be assessed in CN-AML patients as concurrent mutations may\n  have prognostic implications.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	ab5a06b3-7884-4388-89c2-592a08e2f168	2015-06-21 16:49:35.733207
92	29	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: MUTATION\ndescription: CEBPA mutations are associated with cytogenetically normal acute myeloid\n  leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should\n  also be assessed in CN-AML patients as concurrent mutations may have prognostic\n  implications.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	4e2cf347-56fd-4317-ab65-039a64fc0a3d	2015-06-21 16:49:35.752002
708	60	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '209113112'\nstop:\n- \n- '209113112'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	9b00fb81-e049-4af9-8d68-60a0d18605fa	2015-06-21 16:52:22.966958
95	32	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R882\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	2252aeb0-a9e5-40e5-b077-9ec488fc65fb	2015-06-21 16:49:35.819893
96	33	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: L858R\ndescription: EGFR L858R has long been recognized as a functionally significant mutation\n  in cancer, and is one of the most prevalent single mutations in lung cancer. Best\n  described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity\n  to first and second generation TKI's like gefitinib and neratinib. NSCLC patients\n  with this mutation treated with TKI's show increased overall and progression-free\n  survival, as compared to chemotherapy alone. Third generation TKI's are currently\n  in clinical trials that specifically focus on mutant forms of EGFR, a few of which\n  have shown efficacy in treating patients that failed to respond to earlier generation\n  TKI therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	a00e6b95-23d9-419a-9511-0e4f5df8ee92	2015-06-21 16:49:35.858413
97	34	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: T790M\ndescription: EGFR T790M was one of the very first mutations recognized to confer resistance\n  to targeted therapies in non-small cell lung cancer. While successful in amplified\n  EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib,\n  neratinib) in treating patients harboring this mutation before treatment is notably\n  lower. This lack of efficacy can likely be to blame for the poorer prognosis for\n  patients with this mutation as compared to patients with wildtype EGFR or other\n  types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired\n  resistance to TKI inhibition have been shown to harbor this mutation, implicating\n  it as a mechanism of acquired therapy resistence. The third generation TKI's are\n  being developed with this resistance problem in mind, and early stage experiments\n  have shown some efficacy of these drugs in previously resistant tumors with mutant\n  EGFR.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	c61907f8-846c-4ff4-ae42-d7ece1eb7d15	2015-06-21 16:49:35.883674
98	35	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: D769H\ndescription: ERBB2 D769H was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	a11b73f9-4392-478a-b87b-3fdc1495f916	2015-06-21 16:49:35.906976
99	36	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: D769Y\ndescription: ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	bd4f9170-1e9c-4fc4-814f-257f8f359e05	2015-06-21 16:49:35.930085
100	37	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: DEL 755-759\ndescription: ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants\n  to be functionally classified (Bose et al. 2012). This mutation was shown to be\n  an activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	25ebede1-074c-4f16-92d2-c947d3aa0b7a	2015-06-21 16:49:35.949086
101	38	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: G309A\ndescription: ERBB2 G309A was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	1b404163-99e7-4b5a-9044-1dbd017ae9a7	2015-06-21 16:49:35.973219
102	39	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: L755S\ndescription: ERBB2 L755S was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike\n  many of the other variants queried. This mutation was also shown to confer resistance\n  to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	4696da51-9baf-4de7-acb3-a2fc5781f8e6	2015-06-21 16:49:35.990818
132	69	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 14 MUTATIONS\ndescription: c-KIT exon 14 mutations lie within the tyrosine kinase domain of the\n  protein. While relatively rare in primary gastrointestinal tumors, they are notably\n  more prevalent in refractory disease, suggesting a role in imatinib resistance.\n  Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	6f6c18e3-5294-448e-b326-37ba7cfaf0a0	2015-06-21 16:49:36.643899
103	40	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: L755W\ndescription: ERBB2 L755W was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	b10e598d-9c80-422b-ab22-6fe0e5a1175d	2015-06-21 16:49:36.010135
104	41	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: P780INS\ndescription: ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally\n  classified (Bose et al. 2012). This mutation was shown to be an activating mutation\n  in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	7a132f9c-de45-48b8-aedd-6136b2e1ee42	2015-06-21 16:49:36.028889
105	42	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: R678Q\ndescription: ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	30720464-0bd2-4cef-bd30-7779cb81e8b1	2015-06-21 16:49:36.050198
106	43	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: R896C\ndescription: ERBB2 R896C was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	9c71f431-2653-4c58-90f3-69392227e6f7	2015-06-21 16:49:36.067651
107	44	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: V777L\ndescription: ERBB2 V777L was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	721104a7-1198-4e74-a2ed-60332d1992f6	2015-06-21 16:49:36.08585
108	45	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: V842I\ndescription: ERBB2 V842I was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	6b18b5a0-cabb-4820-8b5c-fd0e1466a8e3	2015-06-21 16:49:36.10517
109	46	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: L536Q\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of\n  these ligand binding domain mutations, and is commonly implicated in this hormone\n  resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant,\n  may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	c0cc6cf5-eabf-4168-a7df-03cdcd9b0ee8	2015-06-21 16:49:36.124673
110	47	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: N538G\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	ed997981-850e-45ce-9bd1-0fe4c0396dcb	2015-06-21 16:49:36.142445
111	48	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537C\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	6e9a53c4-3204-4a02-b597-a21856ffdb44	2015-06-21 16:49:36.1609
112	49	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537N\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	6382981a-c757-452a-93f4-9a6a73d38c6f	2015-06-21 16:49:36.178806
113	50	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537S\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	65ad3d6c-b9a8-42c1-9e48-8cb8ccf314c1	2015-06-21 16:49:36.198395
114	51	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: P235R\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	dae67681-6961-4d04-89df-b4570e319cc9	2015-06-21 16:49:36.221068
115	52	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: FGFR2-MGEA5\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. Additionally, tumor size reduction was\n  achieved by both ponatinib and pazopanib treatments administered separately in a\n  single patient with intrahepatic cholangiocarcinoma and this fusion. The authors\n  use these cases to highlight the need for enhanced clinical sequencing efforts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	005dd054-77d8-4122-b2ab-3cf5fafa031a	2015-06-21 16:49:36.239211
116	53	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: FGFR3-BAIAP2L1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	922dfd4b-79b5-4c51-b14d-d086f7798e5a	2015-06-21 16:49:36.257591
117	54	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: FGFR3-TACC3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	ce3ce3e6-0052-4c45-9242-a16a64472636	2015-06-21 16:49:36.276333
118	55	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: ITD MUTATIONS\ndescription: FLT3-ITD (internal tandem duplications) frequently occur in patients\n  with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid\n  leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically\n  normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of\n  FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated\n  sequence resulting in an in-frame duplication event. The length of these duplications\n  can vary widely which may have prognostic consequences, but this has not been conclusively\n  determined. FLT3-ITD mutations overall have generally been associated with poor\n  prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the\n  prognosis associated with this variant.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	396b86cb-a96f-4960-8467-473aefdc18b1	2015-06-21 16:49:36.310693
119	56	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: TKD MUTATIONS\ndescription: FLT3 tyrosine kinase domain mutations (FLT3-TKD) are much less common\n  than FLT3-ITD (internal tandem duplication) mutations and may not confer the same\n  prognostic impact. Although the majority of mutations are point mutations effecting\n  D835 (most frequently D835Y), a small proportion involve an in-frame deletion of\n  I836. These mutations are within the activation loop of the second tyrosine kinase\n  domain of FLT3 and thought to result in constitutive activation of the receptor.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	2e747a25-8b14-4c23-8ec1-f8cc9670f45b	2015-06-21 16:49:36.329843
120	57	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: EXPRESSION\ndescription: GATA2 misregulation has been observed in a number of hematologic malignancies,\n  as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using\n  the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung\n  cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	fa002b05-1428-4043-82c4-0eb3f7d306da	2015-06-21 16:49:36.353552
131	68	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 13 MUTATIONS\ndescription: c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations.\n  These mutations lie within the tyrosine kinase 1 domain, and are found primarily\n  in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations\n  show sensitivity to imatinib and sunitinib treatment. However, only imatinib has\n  seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant\n  cases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	3710aa95-5321-4c2b-9b42-2f36ea2c0f30	2015-06-21 16:49:36.608043
121	58	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	f94e5dd1-5905-4996-98a0-b26a90a78023	2015-06-21 16:49:36.373956
122	59	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132C\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	bedf680b-e567-4f61-885a-908db34e361d	2015-06-21 16:49:36.393049
123	60	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132H\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	8591fddf-122b-49e7-835e-410dfdb8130e	2015-06-21 16:49:36.417087
124	61	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132L\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	b143cd86-e01d-4d67-be51-700eefb2bff6	2015-06-21 16:49:36.438329
125	62	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: R140Q/L\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival\n  as compared to their wild-type counterparts. In myelodysplastic syndromes, however,\n  no prognostic link was discovered between IDH2 mutation status and overall survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	442eaacc-7c16-4035-9d87-11bccfd48b72	2015-06-21 16:49:36.458262
126	63	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: R172K\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis\n  and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic\n  syndromes, studies did not find a prognostic association between this variant and\n  patient outcomes.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	382d2498-4674-4c7e-a43a-e66d28a0044b	2015-06-21 16:49:36.479263
127	64	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: V617F\ndescription: JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases,\n  occuring in around half of all MPD's. While less associated with cancer, when it\n  is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The\n  V617F mutation is an activating mutation, resulting in increased kinase activity.\n  The mutation seems to be restricted to hematologic malignancies. Treatment of JAK\n  mutant diseases with ruxolitinib has seen some clinical success.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	ededada1-958c-4250-bbe7-484674b8b7bb	2015-06-21 16:49:36.498893
128	65	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: D816V\ndescription: KIT D816V is a mutation observed in acute myeloid leukemia (AML). This\n  variant has been linked to poorer prognosis and worse outcome in AML patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	15c63986-fc83-4ace-90ba-a2b4dee8c8b5	2015-06-21 16:49:36.527919
129	66	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 11 MUTATIONS\ndescription: c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are\n  very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis\n  than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity\n  to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients\n  treated with the drug in the first year.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	d822401d-b1d4-42b2-83e7-0cc81cbd9e68	2015-06-21 16:49:36.551133
130	67	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: INTERNAL DUPLICATION\ndescription: c-KIT internal duplications have been observed in exon 11, within the\n  juxtamembrane domain. In a case study of an anal melanoma patient harboring this\n  event, imatinib confered marked response. Also, cells harboring exon 11 mutations\n  have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better\n  prognosis to patients treated with the drug in the first year.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	add295bc-7710-4af2-ae90-32a810d202d5	2015-06-21 16:49:36.5797
133	70	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 17 MUTATIONS\ndescription: c-KIT exon 17 mutations lie within the TK2 domain, containing the activation\n  loop of the protein. In cell lines, mutations within this domain have been shown\n  to be sensitive to imatinib. However, in double KIT mutants in which the exon 17\n  mutation is a secondary mutation, cell lines have shown resistance to both imatinib\n  and sunitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	aadb526f-aee7-4c19-b0f2-11dfcf7413c0	2015-06-21 16:49:36.679682
134	71	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 9 MUTATIONS\ndescription: c-KIT exon 9 mutations lie within the dimerization motif of the protein.\n  Relative to other KIT mutations, exon 9 mutations have been associated with better\n  overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and\n  in vivo.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	1b0ebc40-7f2b-4351-8939-c650a6609434	2015-06-21 16:49:36.708597
135	72	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: L576P\ndescription: KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain.\n  It is one of the most recurrent KIT mutations in melanoma, and both in vitro and\n  in vivo studies have shown sensitivity to imatinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	659c40a6-298c-4f88-95e5-9055d1fcb5b7	2015-06-21 16:49:36.736898
136	73	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: V654A\ndescription: KIT V654A is an exon 13 mutation that lies within the tyrosine kinase\n  1 domain of the protein. It has been shown to be an activating mutation by in vitro\n  studies. This mutation is associated with imatinib resistance in melanoma patients.\n  However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have\n  seen success in acheiving tumor response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	4a23b008-8385-47f2-9baf-0c663947eb1e	2015-06-21 16:49:36.766584
137	74	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: EXON 1 MUTATIONS\ndescription: A study by Lièvre et al in 2006 showed that colorectal cancer patients\n  with KRAS exon 1 mutations had low cetuximab response rates.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	c51ce3ec-a88f-412d-badd-a005e23afd63	2015-06-21 16:49:36.79613
138	75	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: EXON 2 MUTATIONS\ndescription: In a study by Pao et al in 2005, non-small cell lung cancer patients\n  harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors\n  gefinitib and erlotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	75c75ed7-6998-4038-8224-370401db474e	2015-06-21 16:49:36.817602
139	76	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	c423ea22-7e18-4fff-a17c-a3981f7f9a23	2015-06-21 16:49:36.853896
140	77	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12/G13\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation\n  may result in a less responsive tumor when treated with first-generation TKI's like\n  gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	4fde50f7-edf5-4642-92d8-5de2cba6cd5b	2015-06-21 16:49:36.871854
141	78	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12C\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	39179814-0136-40ec-8ee5-03c4f60d502d	2015-06-21 16:49:36.890404
142	79	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12D\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	60d15c4c-a994-4504-bb34-4bfa133f8464	2015-06-21 16:49:36.90946
143	80	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G13\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	829edb04-da83-42c4-b12d-b18b90cb9d03	2015-06-21 16:49:36.928899
156	93	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: G13D\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	c27fd745-c0d8-4011-84d0-afa254c73075	2015-06-21 16:49:37.22658
144	81	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G13D\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	22d18821-6765-484e-a32a-96d866d84ec0	2015-06-21 16:49:36.952259
145	82	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: P124S\ndescription: MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder\n  and colon cancer to a lesser degree. The P124S mutation has been shown to contribute\n  to AZD6244 resistance in melanoma cell lines, but considerably less so than its\n  Q56P counterpart.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	aaaca301-de0a-41f7-817c-f434c2167c57	2015-06-21 16:49:36.971105
146	83	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: Q56P\ndescription: MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This\n  mutation has been shown to confer considerable resistance to AZD6244 treatment of\n  melanoma cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	d7420df3-41e4-43f2-90e7-3fea0cb43579	2015-06-21 16:49:36.989112
147	84	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 32\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	715a2164-716c-4cd0-acef-27200d02a6f9	2015-06-21 16:49:37.008522
148	85	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: PROMOTER METHYLATION\ndescription: MGMT promoter methylation has been observed to impact tumor progression\n  in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating\n  agent carmustine has shown efficacy. In patients lacking methylation, combining\n  carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more\n  experiments are required. Clinical trials have also shown selective sensitivity\n  of promoter methylation-positive patients to temozolomide, making a case for wider\n  methylation screening in GBM patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	c7d046a2-47d4-4da2-a4c6-943f72f61465	2015-06-21 16:49:37.028841
149	86	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 12 MUTATIONS\ndescription: NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	a902ab8a-9af6-4bf9-98b7-04e356d56b8c	2015-06-21 16:49:37.055451
150	87	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: W288FS\ndescription: NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most\n  common NPM1 mutation identified in acute myeloid leukemia. This mutation results\n  in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response\n  to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation,\n  it is generally grouped with other exon 12 mutations for patient analysis (see NPM1\n  Exon 12 variants for more information).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	16c23f15-6606-4c5b-bfa0-90897dabf180	2015-06-21 16:49:37.079215
151	88	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 1 MUTATIONS\ndescription: NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown\n  to be correlated with poorer overall survival relative to wild-type NRAS in melanoma\n  patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	1d97a033-da68-42d9-9c1a-02cfe3e78ca8	2015-06-21 16:49:37.10596
152	89	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 2 MUTATIONS\ndescription: NRAS exon 2 mutations have been shown to be correlated with poorer overall\n  survival in melanoma patients and colorectal cancer patients, however no prognostic\n  impact was seen in acute myeloid leukemia patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	ebdb2cab-8295-419e-b503-758b20cced4c	2015-06-21 16:49:37.131841
153	90	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 3 MUTATIONS\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	b3ded43d-35e9-424a-b553-b7deea4d9380	2015-06-21 16:49:37.158081
154	91	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 4 MUTATIONS\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	0ced0282-bd96-4141-adda-ae0530576451	2015-06-21 16:49:37.18115
155	92	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: G12\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	33a57cb2-53aa-4494-95fa-ef911fbfe1ab	2015-06-21 16:49:37.206305
221	158	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 61\nname: EXPRESSION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	e7d1a9ea-e825-4828-bb29-21ded9ebedd6	2015-06-21 16:49:38.562797
157	94	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations\n  at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide.\n  However, in colorectal cancer patients, mutations at this locus have been shown\n  to confer resistance to cetuximab. The prognostic impact of mutations at this locus\n  is currently under study.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	f27d10f9-c94f-44a8-88b5-7b9d661fb15b	2015-06-21 16:49:37.245773
158	95	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61L\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	17cd47e6-2609-4a2d-9a7b-832f5897fe6c	2015-06-21 16:49:37.26409
159	96	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61R\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	86d922c0-4677-4127-b700-ae084185714b	2015-06-21 16:49:37.282549
160	97	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: V536E\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	af583d44-dfff-4654-887b-bedf9506e538	2015-06-21 16:49:37.301035
161	98	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842I\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	fc73a9ec-7fde-46da-ac66-e9de459ea436	2015-06-21 16:49:37.319749
162	99	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842V\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	a9e04ad9-9a6c-47d8-8328-aa76c19570dd	2015-06-21 16:49:37.338535
163	100	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842Y\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	f553aaf2-c082-4d20-bb9d-3ba550ef46e6	2015-06-21 16:49:37.370196
164	101	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: DEL I843\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	90a947ff-6202-4dfb-950a-b8694f45e506	2015-06-21 16:49:37.388122
165	102	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: DI842-843IM\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	4511f36f-fd0a-4237-90ce-ce5aec65638b	2015-06-21 16:49:37.406347
166	103	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: E542K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	b6275ed7-a504-4b12-95b0-9c9da8c88711	2015-06-21 16:49:37.42511
167	104	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: E545K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	0c9fd47b-36b0-4258-97f0-6b8cfe818092	2015-06-21 16:49:37.442807
168	105	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: EXON 20 MUTATIONS\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	7806011b-c577-45dc-805f-38f8b020c519	2015-06-21 16:49:37.466468
169	106	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: EXON 9 MUTATIONS\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	1be783a7-c21d-459d-9920-dd7e5593c051	2015-06-21 16:49:37.48464
170	107	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: H1047R\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	4d22dd8f-6b66-436d-b0bc-1aa3073c860e	2015-06-21 16:49:37.507886
171	108	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 39\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and arsenic trioxide for treating these patients, and early results\n  seem promising.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	04eb7249-7dae-45fa-9ba7-8923c04b4ff4	2015-06-21 16:49:37.526537
172	109	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in a rare form of adolescent\n  liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found\n  Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined,\n  and functional studies found that the fusion retained kinase activity. The presence\n  of this fusion may be used as a diagnostic marker for this rare tumor type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	78a3b62e-b3e0-4676-9ef9-28e507ad6fb3	2015-06-21 16:49:37.545596
173	110	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 41\nname: R233*\ndescription: PTEN R233* has been shown to be a loss of function mutation, and PTEN\n  loss has been the subject of considerable research in breast cancer. PTEN loss may\n  sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data\n  to support that PTEN loss is both associated with poorer prognosis, and no change\n  in prognosis.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	3a1f7031-f205-4d93-812f-7d3541700bf2	2015-06-21 16:49:37.56835
174	111	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and Arsenic Trioxide for treating these patients, and early results\n  seem promising.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	5f3778ac-1179-41d3-b3e3-bff0af2134eb	2015-06-21 16:49:37.586191
175	112	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: C634W\ndescription: RET C639W has been implicated as an alternate mechanism of activating\n  RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting\n  agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the C639W mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	4d5d5f6f-4af4-4518-a005-191af952cab2	2015-06-21 16:49:37.604663
176	113	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: M918T\ndescription: RET M819T is the most common somatically acquired mutation in medullary\n  thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents,\n  promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the M918T mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T\n  leads to more aggressive MTC with a poorer prognosis.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	d575da3e-ad34-4dc9-b2f9-de6cf2251522	2015-06-21 16:49:37.629142
177	114	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: K666N\ndescription: SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	0bd7c052-a017-445c-b272-00ef2a3af0b9	2015-06-21 16:49:37.652743
178	115	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: K700E\ndescription: SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients. Additionally,\n  these mutations are the most common SF3B1 mutation observed in MDS and highly associated\n  with subtypes of MDS that are defined by ringed sideroblasts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	a93430ed-9978-4e03-8fa3-9565fdaaa605	2015-06-21 16:49:37.671418
179	116	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R175H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R175H variant seems not only to result in loss of tumor-suppression, but also acts\n  as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell\n  lines harboring this mutant is also more responsive to treatment with doxorubicin\n  than its wild-type counterparts. While the prognostic impact of individual TP53\n  mutations is influenced by the cohort being studied, it has been shown that the\n  R175H mutation is correlated with worse overall survival than wild-type TP53, but\n  is not as detrimental as the R248W variant.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	147a937b-313f-4201-a28d-17588361b82a	2015-06-21 16:49:37.690087
180	117	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R248Q\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival. The R248Q mutation has also shown an increased invasive behavior in cell\n  lines. This is specific to the 248Q variant.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	5855bdbf-221d-4926-aa7f-2d909bc9be88	2015-06-21 16:49:37.715083
181	118	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R248W\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	e642683b-5e74-4d2d-954e-1e4accd29ed3	2015-06-21 16:49:37.735316
182	119	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R249T\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	f7c82fe1-a735-4f04-9b2a-4cf12c930512	2015-06-21 16:49:37.757227
183	120	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R249W\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	66bebf81-c8d3-47f2-a50c-0e5e63d98e3a	2015-06-21 16:49:37.775617
184	121	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R273C\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	19a2374d-c0e0-481b-9120-baed48f8377f	2015-06-21 16:49:37.796052
185	122	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R273H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	4fad4336-cb1d-4659-a451-edbdf7e86a45	2015-06-21 16:49:37.815829
186	123	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: V173G/A\ndescription: While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R249 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	adf12eec-73f2-4304-96f1-abae896ed7cc	2015-06-21 16:49:37.837915
187	124	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: FRAMESHIFT TRUNCATION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	077cc752-ff2a-4ccc-be2f-135e0788c4c4	2015-06-21 16:49:37.877901
188	125	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	87fb03e8-c694-4999-836f-d649aecaed38	2015-06-21 16:49:37.906026
709	61	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '209113112'\nstop:\n- \n- '209113112'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- A\n	2	\N	\N	46e7c5f5-e28e-4231-909f-7b55d8a9d09e	2015-06-21 16:52:22.983989
189	126	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	be470497-dc24-4942-afae-aa912aa6a255	2015-06-21 16:49:37.929361
190	127	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 48\nname: Q157P/R\ndescription: U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is less common than the S34F mutation, occurs in the second zinc finger domain of\n  U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations\n  on overall survival in MDS has been debated, however, patients with U2AF1 mutations\n  were shown to be at an increased risk of transformation to secondary AML. The presence\n  of this mutation was not associated with a specific prognostic outcome in AML when\n  compared to U2AF1 wildtype patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	7a6583f9-6c27-4f90-8e31-07e07331f12e	2015-06-21 16:49:37.956011
191	128	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 48\nname: S34Y/F\ndescription: U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is the most commonly identified variant in MDS, occurs in the first zinc finger\n  domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1\n  mutations on overall survival in MDS has been debated, however, patients with U2AF1\n  mutations were shown to be at an increased risk of transformation to secondary AML.\n  The presence of this mutation was not associated with a specific prognostic outcome\n  in AML when compared to U2AF1 wildtype patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	39046429-fb0f-44cf-863f-cbf33132c536	2015-06-21 16:49:37.97808
192	129	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 49\nname: EXON 7 MUTATIONS\ndescription: WT1 exon 7 mutations have been shown to be recurrent in acute myeloid\n  leukemia. Many sources have examined the prognostic impact of these, agreeing that\n  the mutant exon 7 cohort is correlated with worse overall survival and a number\n  of poor prognistic outcomes. This may be the result of an overall poor response\n  to chemotherapy from WT1 mutant tumors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	76d72ac5-bd4e-4f39-a53b-e8afd2a1e5fd	2015-06-21 16:49:38.003132
193	130	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 49\nname: EXON 9 MUTATIONS\ndescription: WT1 exon 9 mutations have been shown to be recurrent in acute myeloid\n  leukemia, although at a less frequent rate than their exon 7 counterparts. Many\n  sources have examined the prognostic impact of these, agreeing that the mutant exon\n  9 cohort is correlated with worse overall survival and a number of poor prognistic\n  outcomes. This may be the result of an overall poor response to chemotherapy from\n  WT1 mutant tumors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	5653ab3a-db01-4186-86ea-8e6d35a5a8be	2015-06-21 16:49:38.021724
194	131	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: LOSS-OF-FUNCTION\ndescription: BRCA1 loss of function mutations have been shown to increase risk of\n  breast and ovarian cancer in those carrying the allele in their germline. Treating\n  BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant\n  response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	c65d7571-1cef-4b45-8e82-6dc19edc307f	2015-06-21 16:49:38.044726
195	132	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: LOSS-OF-FUNCTION\ndescription: BRCA2 loss of function mutations have been shown to increase risk of\n  breast and ovarian cancer in those carrying the allele in their germline. Treating\n  BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant\n  response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	3c6227a4-9106-4e8b-ac1a-f6ac8caf14f0	2015-06-21 16:49:38.062884
196	133	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: EXON 19 DELETION\ndescription: Deletions within exon 19 of EGFR are most common in lung cancer. These\n  deletions, in non-small cell lung cancer, have been shown to be sensitive to the\n  EGFR tyrosine kinase inhibitors gefitinib and erlotinib. There is also data to suggest\n  that this event is a good prognostic marker in lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	84f307d0-0bd4-4209-945b-4c508ba55cb0	2015-06-21 16:49:38.080657
197	134	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: G719S\ndescription: While not as recurrent as the L858R mutation, EGFR G719S has also been\n  shown to be an activating mutation. In lung cancer cell lines, it also confers sensitivity\n  to the tyrosine kinase inhibitors gefitinib and erlotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	80e619f2-8bf6-403c-b828-719cc8dc7298	2015-06-21 16:49:38.102124
198	135	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: D1643H\ndescription: Activating mutations in NOTCH1, including D1643H, have been shown to\n  be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	7ec798b2-a53d-426a-9a0a-a81bafb63f77	2015-06-21 16:49:38.121457
199	136	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: R2328W\ndescription: Activating mutations in NOTCH1, including R2328W, have been shown to\n  be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	93f5ee7c-3150-4415-be0f-2ff3f3c2b508	2015-06-21 16:49:38.138847
200	137	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: V2444FS\ndescription: Activating mutations in NOTCH1, including a frameshift insertion at V2444,\n  have been shown to be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	8ed76b2f-a613-409b-968b-1c8852f22db7	2015-06-21 16:49:38.157737
222	159	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 58\nname: LOSS OF FUNCTION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	39a25746-4431-44c9-806e-92f294474815	2015-06-21 16:49:38.58056
201	138	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: S2275FS\ndescription: Activating mutations in NOTCH1, including a frameshift insertion at S2275,\n  have been shown to be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	7a48f209-ba4b-4044-bb77-0198a5c4163b	2015-06-21 16:49:38.175495
202	139	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: L63V\ndescription: Activating mutations in DDR2, including L63V, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	cdfb8413-9d47-45bd-add4-0ea3188abc02	2015-06-21 16:49:38.19425
203	140	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: L239R\ndescription: Activating mutations in DDR2, including L239R, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	9df0b2b2-6f53-4112-ad93-e3ab6f7643c1	2015-06-21 16:49:38.214279
204	141	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: G253C\ndescription: Activating mutations in DDR2, including G253C, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	224761c7-d1a6-4299-9713-f05c59f8aab5	2015-06-21 16:49:38.231646
205	142	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: G505S\ndescription: Activating mutations in DDR2, including G505S, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	80f6e547-70ec-4ce4-94a7-39c7e083d4ba	2015-06-21 16:49:38.248806
206	143	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: I638F\ndescription: DDR2 I638F has been shown to be a loss of function mutation, but also\n  confers sensitivity to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	f851e1f0-c22b-4a0c-8496-f88903510e95	2015-06-21 16:49:38.271699
207	144	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: G774V\ndescription: Activating mutations in DDR2, including G774V, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	5eb321cc-7a7b-4d83-a722-c287e3a769a9	2015-06-21 16:49:38.289302
208	145	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: S768R\ndescription: Activating mutations in DDR2, including S768R, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	45c96d74-3d1a-45e6-999b-6440b65cf31c	2015-06-21 16:49:38.307868
209	146	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 52\nname: AMPLIFICATION\ndescription: MET amplification, like EGFR T790M, has been shown to be capable of driving\n  acquired resistance to dacomitinib in patients with lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	4c86a4ce-0eff-41d8-b139-826b2cc849b4	2015-06-21 16:49:38.340221
210	147	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12V\ndescription: KRAS G12V, like EGFR T790M, has been shown to be capable of driving acquired\n  resistance to tyrosine kinase inhibitors in lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	68fc71bd-c1fd-49ee-972c-a1a77dd1a245	2015-06-21 16:49:38.363042
211	148	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12A\ndescription: KRAS G12A, like EGFR T790M, has been shown to be capable of driving acquired\n  resistance to tyrosine kinase inhibitors in lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	405515d0-30df-4c3c-a891-e8ee7a519b8a	2015-06-21 16:49:38.380245
212	149	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 54\nname: EWS-FLI1\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	95fc3b30-d311-4f91-bfae-ea444c676976	2015-06-21 16:49:38.39818
213	150	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 53\nname: EWS-FLI1\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	a01b0395-3aa5-4538-b742-fa139b9fdc2f	2015-06-21 16:49:38.41597
214	151	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 59\nname: T(14;18)\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	1933a8f4-db0a-4ddd-84a7-6bd2fcb48bd9	2015-06-21 16:49:38.434063
215	152	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 59\nname: EXPRESSION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	3ebc673a-35ff-414f-a873-53e80db4689c	2015-06-21 16:49:38.454779
216	153	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 56\nname: AMPLIFICATION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	9ff8e46b-2be8-46bc-90f8-cbf3c62376b6	2015-06-21 16:49:38.472502
217	154	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 57\nname: AMPLIFICATION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	947d8ffd-520c-497b-b55d-ffe79a07eeb0	2015-06-21 16:49:38.491915
218	155	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: MUTATION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	dcf50458-d28f-4452-b50a-c8f6c05ac14f	2015-06-21 16:49:38.509496
219	156	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 60\nname: OVEREXPRESSION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	ea361a59-aa1f-4485-ad90-7730c401b3a7	2015-06-21 16:49:38.527261
220	157	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 55\nname: MUTATION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	c0c65076-5e48-43e9-9a33-968d455ba166	2015-06-21 16:49:38.545209
223	160	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 58\nname: MUTATION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	5fec259d-32bf-43b3-abf4-86b3d425247f	2015-06-21 16:49:38.598399
224	161	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 62\nname: MUTATION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	1d5ccfb0-5769-4fec-98ec-d173c8cdb620	2015-06-21 16:49:38.620653
225	162	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 57\nname: EXPRESSION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	7847a413-693e-4917-bb7f-4d1abc0d24fd	2015-06-21 16:49:38.638018
226	163	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 63\nname: MUTATION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	51ff6c76-3e4e-428d-9b54-66b4e8c6d195	2015-06-21 16:49:38.656309
227	164	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 53\nname: TYPE 1 EWS-FLI1\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \n	1	\N	\N	259880ef-dedb-4382-82da-ac55def9f08c	2015-06-21 16:49:38.674376
228	1	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL\n  or CLL).\ndisease_id: 1\nsource_id: 51\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \n	1	\N	\N	54449a12-96c7-4d7b-85b1-05cfd46fd55a	2015-06-21 16:49:38.827
229	2	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: GIST tumors harboring PDGFRA D842V mutation are more likely to be benign\n  than malignant.\ndisease_id: 2\nsource_id: 52\nvariant_id: 99\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152093-55152093 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \n	1	\N	\N	fc2b8fbc-19db-436b-8056-671004b9ab85	2015-06-21 16:49:38.884845
230	1	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Imatinib Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	e4b7d250-5df6-4db4-b5a0-3395c1239382	2015-06-21 16:49:38.943473
231	3	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutations occur most often in de novo AML patients with intermediate\n  cytogenic risk.\ndisease_id: 3\nsource_id: 53\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	6f61728c-fcb6-4ea4-b6a5-1f09c40aa341	2015-06-21 16:49:38.98731
232	4	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young AML patients (<60 years old) with DNMT3A mutations (60% of which\n  were R882) were older in age, had higher white blood cell counts and had higher\n  platelet counts than patients wildtype for DNMT3A\ndisease_id: 3\nsource_id: 54\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	5bdd3054-7d91-49c8-9ce6-87642678add6	2015-06-21 16:49:39.023907
233	5	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical\n  CML).\ndisease_id: 4\nsource_id: 51\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	a1e767c5-cebe-44b1-8d91-d34b858a7c14	2015-06-21 16:49:39.061439
234	6	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: JAK2 V617F is associated with myeloid neoplasms (AML, MDS, CMML/aCML)\ndisease_id: 4\nsource_id: 51\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	77890dea-8ceb-4aff-b3bf-8185da505496	2015-06-21 16:49:39.096276
235	7	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The SNP rs10974944 in the Jak2 locus is associated with increased predisposition\n  for JAK2 V617F mutation and its associated cancer\ndisease_id: 5\nsource_id: 55\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	f2c63125-9f2e-488c-a9c2-92df45ce46a3	2015-06-21 16:49:39.13527
236	8	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Children with early age leukaemia who had second hand smoke exposure\n  are more likely to harbor KRAS mutation.\ndisease_id: 6\nsource_id: 56\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	09f1e204-1094-48ce-9f1b-1a6dede1bbbb	2015-06-21 16:49:39.168891
237	9	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Children with early age leukaemia who had second hand smoke exposure\n  are more likely to harbor KRAS mutation.\ndisease_id: 6\nsource_id: 56\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	a2d4205a-ece2-4986-9900-854e350c52fc	2015-06-21 16:49:39.197432
238	10	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma associated with NRAS Q61 mutation was more often associated\n  with those at the extremity than those at the trunk\ndisease_id: 7\nsource_id: 57\nvariant_id: 94\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	652c76f9-86ec-41dc-962f-d49cf96e0e88	2015-06-21 16:49:39.229719
239	11	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Daunorubicin treatment resulted in similar overall survival and disease\n  free survival in de novo AML patients with DNMT3A R882 mutation compared to those\n  who do not harbor this mutation.\ndisease_id: 3\nsource_id: 53\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	dd446478-725a-4b27-af7b-6ef3d7a836fc	2015-06-21 16:49:39.263587
710	63	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '15'\nstart:\n- \n- '90631838'\nstop:\n- \n- '90631838'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	9cb299ab-37a2-4c7c-81e6-4405d6d53ffa	2015-06-21 16:52:23.001603
240	12	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition\n  by about 5 fold.\ndisease_id: 7\nsource_id: 58\nvariant_id: 82\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:66729162-66729162 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	b06c49a5-7264-467c-b304-7da30f09d2dc	2015-06-21 16:49:39.31705
241	13	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by\n  100 fold.\ndisease_id: 7\nsource_id: 59\nvariant_id: 83\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:66727451-66727451 (A->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	56a49913-3a79-4b2b-8ceb-e12f19ce1954	2015-06-21 16:49:39.35312
242	14	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.\ndisease_id: 7\nsource_id: 60\nvariant_id: 94\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	6de9f5d7-7083-4f62-99a5-5b2fdf8a0bb8	2015-06-21 16:49:39.388031
243	15	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: GIST cancer with D842V mutation is resistant to imatinib.\ndisease_id: 2\nsource_id: 61\nvariant_id: 99\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152093-55152093 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	e09dcf6c-b2cf-4036-9c8e-5f37e9d8650e	2015-06-21 16:49:39.424664
244	16	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib,\n  double mutation of V561D and D842V mutants are resistant to imatinib.\ndisease_id: 2\nsource_id: 62\nvariant_id: 99\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152093-55152093 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	a7edd8cb-e634-4a78-b639-06fdb80d762f	2015-06-21 16:49:39.466685
245	17	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In one patient with S214C mutation, the use of sorafenib has led to more\n  than 5 years of survival and near remission.\ndisease_id: 8\nsource_id: 63\nvariant_id: 10\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: X:47426121-47426121 (C->G)\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	4509d0c7-24b7-46fc-80c9-3482ec349897	2015-06-21 16:49:39.508861
246	18	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Idarubicin increases the overall survival and disease free survival in\n  de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor\n  this mutation.\ndisease_id: 3\nsource_id: 53\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	6a8ff0ca-c9b5-449a-8022-d5f5819e64a3	2015-06-21 16:49:39.550444
247	19	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads\n  to reduction in the percentage of cells harboring JAK2 V617F.\ndisease_id: 9\nsource_id: 64\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	754a7ac5-0baf-44e1-b5b0-1dbaafea13e9	2015-06-21 16:49:39.614059
248	20	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells\ndisease_id: 9\nsource_id: 65\nvariant_id: 64\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	5c324a7b-85de-4e43-b161-e907d56b0019	2015-06-21 16:49:39.676255
249	21	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Likely due to increased reliance of mutant NRAS on HSP90 for stabilization,\n  inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic\n  malignant melanoma with an NRAS G13D mutation\ndisease_id: 7\nsource_id: 66\nvariant_id: 93\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 1:115258744-115258744 (C->T)\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	a578c97c-b6b4-474a-8dc2-da59d8c735ef	2015-06-21 16:49:39.740941
250	22	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide\n  resulted in disease free survival of 14 months\ndisease_id: 7\nsource_id: 67\nvariant_id: 95\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 1:115256528-115256529 (TT->CA)\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	b91fb0b8-ae7b-4376-ab57-77af8efa8f60	2015-06-21 16:49:39.796324
251	23	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a melanoma patient with Q61R mutation, treatment with temozolomide\n  resulted in overall survival of 16 months\ndisease_id: 7\nsource_id: 67\nvariant_id: 96\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 1:115256528-115256529 (TT->CC)\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	4292d12d-800e-4966-862a-9530575a68ac	2015-06-21 16:49:39.839021
252	24	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no difference in the complete remission rate of de novo AML\n  patients with DNMT3A mutation compared to those who are wild type for DNMT3A.\ndisease_id: 3\nsource_id: 53\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	e07052ac-982f-4127-b8fc-02f66acee29a	2015-06-21 16:49:39.893004
253	25	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: De novo AML patients with DNMT3A D882 mutation showed worse survival\n  (event-free and overall) outcome than those without DNMT3A mutation.\ndisease_id: 3\nsource_id: 68\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	69edb1be-f35c-4a70-8127-0f082035cc7c	2015-06-21 16:49:39.923723
711	64	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '9'\nstart:\n- \n- '5073770'\nstop:\n- \n- '5073770'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	0fdeadf0-f496-4c52-83aa-feba518c905d	2015-06-21 16:52:23.018962
254	26	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In acute myloid leukemia patients, D816 mutation is associated with earlier\n  relapse and poorer prognosis than wildtype KIT\ndisease_id: 10\nsource_id: 69\nvariant_id: 65\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55599321-55599321 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	29fefdf4-8424-426f-8d0e-2c9565476e56	2015-06-21 16:49:39.955412
255	2	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: KIT Exon 17\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	77df0f1c-7347-4d6e-bf4b-af087d615e0c	2015-06-21 16:49:39.979144
256	27	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\ndisease_id: 11\nsource_id: 70\nvariant_id: 89\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	c3303119-8708-4122-9feb-b453c34b8c86	2015-06-21 16:49:40.01257
257	28	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\ndisease_id: 11\nsource_id: 70\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	790816c3-d511-4f4d-8d36-7a52b46fc1d0	2015-06-21 16:49:40.048121
258	29	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\ndisease_id: 11\nsource_id: 70\nvariant_id: 91\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	438c7dbe-01cb-4f86-853c-66e5d9fdf1dd	2015-06-21 16:49:40.082423
259	30	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF mutations are associated with melanoma arising in non-chronic sun\n  damaged skin and with superficial spreading melanoma\ndisease_id: 7\nsource_id: 71\nvariant_id: 17\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	e7a231b8-60eb-40f5-bd73-d01e12928436	2015-06-21 16:49:40.115427
260	31	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutations were associated with older age, higher white blood\n  cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of\n  cytogenetically normal AML patients\ndisease_id: 3\nsource_id: 72\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	205065d2-8801-407c-9872-e9c12c881c8e	2015-06-21 16:49:40.151334
261	32	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation\n  were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK\n  without this mutation.\ndisease_id: 12\nsource_id: 73\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	62c6e16e-5164-4bac-99b8-ff24b4b6ada0	2015-06-21 16:49:40.191508
262	33	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation\n  were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK\n  without this mutation.\ndisease_id: 8\nsource_id: 73\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	4ac3ad6a-9b1e-42d7-b732-24db48308a99	2015-06-21 16:49:40.230503
263	3	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Crizotinib Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	1fb34127-8d98-490e-a2d8-eb49e2b47479	2015-06-21 16:49:40.256466
264	34	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The melanoma cell line WM3211, which harbors the L576P mutation, is not\n  sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the\n  L576P mutation alters the conformation of KIT and reduces the binding affinity of\n  imatinib.\ndisease_id: 7\nsource_id: 74\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	5d4bb16b-8d04-407b-9a3f-2fb27ac91ab8	2015-06-21 16:49:40.300925
265	4	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: KIT Exon 11\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	0dbfae7f-8fed-431f-b640-e064e337bd7f	2015-06-21 16:49:40.337653
266	35	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC, Exon 2 KRAS mutations were associated with resistance to the\n  EGFR kinase inhibitors gefitinib and erlotinib\ndisease_id: 8\nsource_id: 75\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	5788b4f4-0691-47e3-96bb-91e3d7e22458	2015-06-21 16:49:40.377359
267	36	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring NRAS\n  mutation (primarily Q61) have a significantly lower response rate to cetuximab than\n  patients wildtype for NRAS.\ndisease_id: 11\nsource_id: 76\nvariant_id: 94\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	de33bc29-6250-4550-8707-1f1cfcd5ca29	2015-06-21 16:49:40.434505
268	37	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant\n  in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.\ndisease_id: 13\nsource_id: 77\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	5ef8a623-ffcd-4743-b47c-057ba36faeb3	2015-06-21 16:49:40.499696
353	119	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with M5a and M5b FAB subtypes of acute\n  myeloid leukemia\ndisease_id: 3\nsource_id: 113\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	7bf0bbc9-ee89-4538-ae64-402313b1c07c	2015-06-21 16:49:43.663753
269	38	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: High levels of crizotinib can overcome drug resistance of Ba/F3 cells\n  expressing the EML4-ALK fusion containing the F1174L mutation.\ndisease_id: 13\nsource_id: 78\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	f4295671-4871-4410-922f-82cda61cf99d	2015-06-21 16:49:40.537085
270	39	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however,\n  cells with the R1275Q mutation were inhibited at a higher IC50 than those expressing\n  F1174L.\ndisease_id: 13\nsource_id: 79\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29432664-29432664 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	1b330ec8-2c92-420c-b7b3-d07ece54d898	2015-06-21 16:49:40.575308
271	40	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar\n  colony formation of AALE cells expressing ARAF S214C in vitro\ndisease_id: 8\nsource_id: 63\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: X:47426121-47426121 (C->G)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	9dd87ae1-8edd-43f8-9474-370abb7156df	2015-06-21 16:49:40.613087
272	41	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar\n  colony formation of AALE cells expressing ARAF S214C in vitro\ndisease_id: 8\nsource_id: 63\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: X:47426121-47426121 (C->G)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	799cb0f6-e8dd-4957-8d2c-b48cac21c6e2	2015-06-21 16:49:40.649889
273	42	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive\n  to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation\n  of KIT but did not alter the binding affinity of dasatinib.\ndisease_id: 7\nsource_id: 74\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	2f989edd-6a9f-467d-8347-17d932cb54f9	2015-06-21 16:49:40.695285
274	43	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\ndisease_id: 2\nsource_id: 80\nvariant_id: 98\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152092-55152093 (GA->AT)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	5102fd70-1e50-4b43-9815-f494db379fdd	2015-06-21 16:49:40.74864
275	44	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\ndisease_id: 2\nsource_id: 80\nvariant_id: 99\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152093-55152093 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	1168e410-5583-46df-807b-c499e7e9bbc8	2015-06-21 16:49:40.798203
276	45	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\ndisease_id: 2\nsource_id: 80\nvariant_id: 100\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152092-55152092 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	1f1cbe25-24c9-43d7-9fe5-ed0c119eade9	2015-06-21 16:49:40.843325
277	46	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib\n  resistance, crenolanib was significantly more potent at inhibiting kinase activity\n  than imatinib.\ndisease_id: 2\nsource_id: 80\nvariant_id: 101\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	a075fa56-4b57-4ad7-995b-b69e29d0ce2a	2015-06-21 16:49:40.879899
278	47	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib\n  resistance, crenolanib was significantly more potent at inhibiting kinase activity\n  than imatinib.\ndisease_id: 2\nsource_id: 80\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	81641f74-f4b9-43bf-97d0-b51a6d16199d	2015-06-21 16:49:40.91481
279	48	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The SMS-KCNR cell line harboring the R1275Q mutation was resistant to\n  TAE684.\ndisease_id: 13\nsource_id: 79\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29432664-29432664 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	8ce8d075-e7a5-431c-b9b4-3b6d4f812b10	2015-06-21 16:49:40.952234
280	49	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a large cohort of AML patients (mean = 48 years), DNMT3A mutation\n  (64.5% of which were R882) had no prognostic value on overall, relapse free and\n  event free survival.\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	de907d07-f384-4cfc-b481-f9668cfb5e46	2015-06-21 16:49:40.986018
281	50	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a large cohort of cytogenetically normal AML patients (18-60 years\n  old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value\n  for overall, relapse free and event free survival\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	b457d3e1-3229-4c34-a402-6348a13fd29e	2015-06-21 16:49:41.01738
712	65	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55599321'\nstop:\n- \n- '55599321'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- T\n	2	\N	\N	a4b6fec4-b48e-48c9-905c-36acd79cbc10	2015-06-21 16:52:23.033926
713	73	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55594258'\nstop:\n- \n- '55594258'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	7cb81a77-07c3-40cc-a67c-2b63be37bd42	2015-06-21 16:52:23.050903
282	51	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutation status (60% of\n  which were R882) was not predictive of overall and relapse free survival in patients\n  with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\ndisease_id: 3\nsource_id: 54\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	e54372cc-a6e3-4028-aefc-27d13c368ecd	2015-06-21 16:49:41.062777
283	52	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutation status was not\n  predictive of overall and relapse free survival in patients with NPM1 mutations\n  and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\ndisease_id: 3\nsource_id: 54\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	99b69f6c-ecd8-4f56-b1db-5aa6eef2a57c	2015-06-21 16:49:41.092526
284	53	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no significant association between wildtype KIT or KIT mutations\n  in exon 9 or 11 in survival among GIST patients.\ndisease_id: 14\nsource_id: 82\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	736f4cf3-c607-4642-a356-61cd98d83a9d	2015-06-21 16:49:41.129119
285	54	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no significant association between wildtype KIT or KIT mutations\n  in exon 9 or 11 in survival among GIST patients.\ndisease_id: 14\nsource_id: 82\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	b8e76b6b-f419-4742-96a0-b24786cdac59	2015-06-21 16:49:41.176361
286	55	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutation status was not\n  predictive of overall and relapse free survival in patients with NPM1 mutations\n  and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\ndisease_id: 3\nsource_id: 54\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	08001ed1-2ae5-4ccd-b209-033a0463ac15	2015-06-21 16:49:41.215925
287	56	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of NRAS mutation in AML patients does not impact diease\n  prognosis (resistant disease, disease-free survival, complete remission rate, relapse\n  rate, induction death)\ndisease_id: 3\nsource_id: 83\nvariant_id: 89\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	0c4a368a-53bc-437f-84e9-ec6783cddb2e	2015-06-21 16:49:41.258417
288	57	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NRAS mutation status does not impact overall, event-free or disease-free\n  survival in patients with AML\ndisease_id: 3\nsource_id: 84\nvariant_id: 92\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	8a6466bf-d2c2-41af-af5a-3c417d899604	2015-06-21 16:49:41.296579
289	58	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with KIT mutation but not KIT amplification showed\n  response to imatinib treatment in a small cohort of patients.\ndisease_id: 7\nsource_id: 85\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	a5911b52-bf06-4787-9825-9ef4a9bb0b4f	2015-06-21 16:49:41.340695
290	59	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with KIT mutation but not KIT amplification showed\n  response to imatinib treatment in a small cohort of patients.\ndisease_id: 7\nsource_id: 85\nvariant_id: 68\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	9b0c1382-3624-41b8-a89e-5ce89bf29a32	2015-06-21 16:49:41.381965
291	60	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with KIT mutation but not KIT amplification showed\n  response to imatinib treatment in a small cohort of patients.\ndisease_id: 7\nsource_id: 85\nvariant_id: 70\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	63e1e1f0-d303-4bfc-b767-85043294895e	2015-06-21 16:49:41.418993
292	61	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a large cohort of young AML patients (18-60 years old), DNMT3A R882\n  mutations were associated with reduced relapse free survival in the entire cohort\n  as well as the subset of patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	c23cde42-122b-472d-87e3-70da285cf418	2015-06-21 16:49:41.457052
293	62	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with DNMT3A mutations (59% of which were R882) showed worse\n  survival (event-free and overall) outcome than those without DNMT3A mutation.\ndisease_id: 3\nsource_id: 68\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	3c474eb7-d523-4bcd-83b4-9e265d76c1f5	2015-06-21 16:49:41.486429
294	63	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutation was associated with reduced relapse free and overall\n  survival in ELN-unfavorable, cytogenetically normal AMLs\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	271d2d7f-3c4f-4a08-b744-45ffabbf20ab	2015-06-21 16:49:41.518533
295	64	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations\n  (including R882) were associated with worse overall survival compared to those without\n  DNMT3A mutation.\ndisease_id: 3\nsource_id: 68\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	e21ad633-347a-4f22-8e70-98a9c8437d6e	2015-06-21 16:49:41.570083
296	65	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In cytogenetically normal AML patients, DNMT3A R882 mutations are associated\n  with lower overall and disease free survival as compared to patients with wild type\n  DNMT3A.\ndisease_id: 3\nsource_id: 72\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	6097ba55-89ca-4e98-943a-5cb12edc0b86	2015-06-21 16:49:41.604054
297	66	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In older cytogenetically normal AML patients (>59 years), DNMT3A R882\n  mutation is prognostic for shorter disease free survival and overall survival compared\n  to patients without the mutation.\ndisease_id: 3\nsource_id: 86\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	b43d63da-24bb-4eca-90d9-5a187dfb21d4	2015-06-21 16:49:41.635939
298	67	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutations were associated\n  with significantly reduced overall survival and relapse free survival in patients\n  wildtype for NPM1 and FLT3\ndisease_id: 3\nsource_id: 54\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	3da687a0-255d-4df6-8fb0-fc7e280d660e	2015-06-21 16:49:41.669085
299	68	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations\n  other than R882 are prognostic for shorter disease free survival and overall survival\n  compared to patients without the mutation.\ndisease_id: 3\nsource_id: 86\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	dd5ebad2-0da2-4325-b4da-4f60b04bade6	2015-06-21 16:49:41.704413
300	69	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations\n  were associated with worse overall survival compared to those without DNMT3A mutation.\ndisease_id: 3\nsource_id: 68\nvariant_id: 55\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	74f801c1-b4ba-47c0-888d-2499aeb3a5e1	2015-06-21 16:49:41.748377
301	70	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: GIST patients with exon 14 KIT mutations had reduced overall survival\n  compared to patients wiltype for KIT\ndisease_id: 14\nsource_id: 82\nvariant_id: 69\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	b436bcf1-6bdb-475c-bb4a-c79666e4e86a	2015-06-21 16:49:41.784693
302	71	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with stage IV melanoma, NRAS mutation was associated with\n  reduced median survival compared to patients with wildtype NRAS\ndisease_id: 7\nsource_id: 87\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	dda19100-cc93-4f7e-8bdc-a51152dc387c	2015-06-21 16:49:41.821689
303	72	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with stage IV melanoma, NRAS mutation was associated with\n  reduced median survival compared to patients with wildtype NRAS\ndisease_id: 7\nsource_id: 87\nvariant_id: 89\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	d373bb27-39a7-4d91-ae56-c5116a25aeb4	2015-06-21 16:49:41.859512
304	73	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A single patient with BRAF V600E/V600M bi-allelic mutation responded\n  to the V600E drug dabrafenib.\ndisease_id: 7\nsource_id: 88\nvariant_id: 13\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	3b185a80-391c-43f4-a26d-8c90400f4fcd	2015-06-21 16:49:41.89349
305	5	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Other V600's\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	80ce574a-0945-4c1a-9b38-494e6eb8b1aa	2015-06-21 16:49:41.921318
306	74	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with medullary carcinoma, the presence of RET M918T mutation\n  is associated with increased probability of lymph node metastases.\ndisease_id: 15\nsource_id: 89\nvariant_id: 113\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43617416-43617416 (T->C)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	9e4abfcf-3fd5-4676-819d-95627a006205	2015-06-21 16:49:41.960093
307	75	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Medullary thyroid cancer cells with RET C634W mutation are insensitive\n  to motesanib, compared to wild-type RET.\ndisease_id: 15\nsource_id: 90\nvariant_id: 112\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43609950-43609950 (C->G)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	8da72cf6-9abd-4005-a42c-b49d8179ad43	2015-06-21 16:49:41.994971
308	6	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Motesanib Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	543cd318-4cee-4dbb-b56c-692e1d1029e6	2015-06-21 16:49:42.018975
309	76	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Medullary thyroid cancer cells with RET M918T mutation are insensitive\n  to motesanib, compared to wild-type RET.\ndisease_id: 15\nsource_id: 90\nvariant_id: 113\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43617416-43617416 (T->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	a06d57bd-2dfa-43ad-8aa4-8da117558301	2015-06-21 16:49:42.050598
310	77	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led\n  to strong repression of tyroid cancer cell growth.\ndisease_id: 15\nsource_id: 91\nvariant_id: 113\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43617416-43617416 (T->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	f2c4a67f-7cfc-4e4f-9966-1d3a9bae671c	2015-06-21 16:49:42.107022
714	78	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '12'\nstart:\n- \n- '25398285'\nstop:\n- \n- '25398285'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- A\n	2	\N	\N	9a4404c2-5c39-4d78-b825-911493bb9c34	2015-06-21 16:52:23.064608
311	78	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Compared to those who harbor a wild type RET, patients with RET M918T\n  mutation develop larger and more aggressive medullary thyroid cancer.\ndisease_id: 15\nsource_id: 92\nvariant_id: 113\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43617416-43617416 (T->C)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	d760ef8d-f15f-4356-be7b-ee978e2b75b4	2015-06-21 16:49:42.141658
312	79	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF V600E is shown to be associated with the tall-cell variant of papillary\n  thyroid cancer\ndisease_id: 16\nsource_id: 93\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	9a51fb7d-bea6-4f7e-be2d-81e279975dbe	2015-06-21 16:49:42.179161
313	80	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\ndisease_id: 16\nsource_id: 94\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	ab7618cc-af98-4ff4-9999-890c7ad12e67	2015-06-21 16:49:42.209976
314	81	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\ndisease_id: 16\nsource_id: 95\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	88e1e631-7f86-414d-87ac-b01a559dc60e	2015-06-21 16:49:42.244259
315	82	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\ndisease_id: 7\nsource_id: 96\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	0b396a0d-6bc0-4296-9a1d-db3c9d6b9603	2015-06-21 16:49:42.279082
316	83	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation status was not predictive of response to regorafenib\n  treatment in patients that had received standard therapy and progressed within 3\n  months of their last treatment\ndisease_id: 11\nsource_id: 97\nvariant_id: 103\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	eb0106ca-0c5b-497c-af47-d9fef702e969	2015-06-21 16:49:42.32066
317	84	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation status was not predictive of response to regorafenib\n  treatment in patients that had received standard therapy and progressed within 3\n  months of their last treatment\ndisease_id: 11\nsource_id: 97\nvariant_id: 104\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	25656270-027d-4614-ab00-f843bc4b9af1	2015-06-21 16:49:42.354053
318	85	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation status was not predictive of response to regorafenib\n  treatment in patients that had received standard therapy and progressed within 3\n  months of their last treatment\ndisease_id: 11\nsource_id: 97\nvariant_id: 107\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	c09463a9-5ebd-4187-82e1-b655bd408dfa	2015-06-21 16:49:42.382835
319	86	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation\n  of RAS-GTP but does not show resistance to MEK inhibitors.\ndisease_id: 7\nsource_id: 98\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	d97cc490-66aa-4e43-837f-81abcf0f941d	2015-06-21 16:49:42.411479
320	87	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\ndisease_id: 11\nsource_id: 97\nvariant_id: 75\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	e07541ed-77fe-4c3e-8489-6896e49c06c8	2015-06-21 16:49:42.441398
321	88	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations\n  were associated with poor progression free survival regardless of treatment (panitumumab\n  with best supportive care or best supportive care alone)\ndisease_id: 11\nsource_id: 99\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	768c0130-6a8d-407e-8054-fc33d49e7a54	2015-06-21 16:49:42.478322
322	89	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation\n  unless BRAF inhibitor such as Sorafenib is introduced.\ndisease_id: 17\nsource_id: 100\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	c8b1b7e2-0f2e-4765-8eb8-d3acf4a28242	2015-06-21 16:49:42.514984
323	90	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation\n  of RAS-GTP and resistance to RAF inhibitors.\ndisease_id: 7\nsource_id: 98\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	a59d804e-0be9-4e80-9a5c-9670371a1e4c	2015-06-21 16:49:42.561513
324	91	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\ndisease_id: 8\nsource_id: 101\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	8cc77c92-20e8-4267-ad85-d1e6c67cac52	2015-06-21 16:49:42.594744
325	92	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: COLO201 and COLO206F cells harboring BRAF V600E mutations were cloned\n  to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown\n  to be amplification of the BRAF V600E gene.\ndisease_id: 11\nsource_id: 102\nvariant_id: 14\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	38e3e521-65ab-42ca-a4dc-f4785ca78835	2015-06-21 16:49:42.627148
326	93	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with BRAF V600E/K mutations had longer progression-free\n  survival, increased tumor regression,  and increased duration of response to combined\n  dabrafenib and trametinib treatment compared to dabrafenib alone\ndisease_id: 7\nsource_id: 103\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	2fe5e1db-90f1-4f41-b337-6d28b96dbbd9	2015-06-21 16:49:42.659471
327	94	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with other BRAF V600 mutations also respond well to the V600E\n  drug dabrafenib.\ndisease_id: 7\nsource_id: 104\nvariant_id: 11\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453135-140453136 (CA->AT)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	7476e4eb-3929-4d4a-bdbf-d381cf120eb0	2015-06-21 16:49:42.695185
328	95	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\ndisease_id: 7\nsource_id: 105\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	9f665712-beae-42c0-8bdd-89db944b206a	2015-06-21 16:49:42.736733
329	96	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely\n  injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited\n  tumor growth and reduced cellular proliferation\ndisease_id: 17\nsource_id: 106\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	3fb130ca-1710-44cf-8f82-9ac89f991d4d	2015-06-21 16:49:42.783207
330	97	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\ndisease_id: 17\nsource_id: 107\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	244eafac-36e4-4784-b560-fc6b633799cd	2015-06-21 16:49:42.837942
331	98	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\ndisease_id: 11\nsource_id: 108\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	fbb5954f-02b5-4544-9780-37611c1bb7a0	2015-06-21 16:49:42.887924
332	99	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cell lines expressing the BRAF V600E mutation responded better to vemurafenib\n  treatment than cells wildtype for BRAF as measured by reduced cellular proliferation\n  and inhibition of MET and ERK phosphorylation\ndisease_id: 11\nsource_id: 108\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	9e6d1cb3-8d38-4a2e-acd3-054d139ce5f0	2015-06-21 16:49:42.944165
333	100	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with other BRAF V600 mutations also respond well to the V600E\n  drug dabrafenib.\ndisease_id: 7\nsource_id: 104\nvariant_id: 15\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453137-140453137 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	325d120a-18d8-4e34-a849-12dae579d665	2015-06-21 16:49:42.981105
334	101	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with other BRAF V600 mutations also respond well to the V600E\n  drug dabrafenib.\ndisease_id: 7\nsource_id: 104\nvariant_id: 16\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453137 (AC->CT)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	9c4a1598-b08f-41f9-928d-8a19fee20859	2015-06-21 16:49:43.04418
335	102	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Unlike other studies that suggest a poorer outcome, BRAF mutation in\n  this study was not correlated with poorer prognosis in papillary thyroid cancer.\ndisease_id: 16\nsource_id: 109\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	217e765a-b9d1-4c92-b484-a4f15061cf01	2015-06-21 16:49:43.079156
336	103	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: V600E is associated with adverse pathological features of colorectal\n  cancer. This can be concluded as a marker of poor prognosis.\ndisease_id: 11\nsource_id: 110\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	81584587-17b8-4fbe-8831-7fe06190d6ab	2015-06-21 16:49:43.111562
337	104	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF V600E is correlated with shorter disease-free and overall Survival\n  in a Spanish cohort of melanoma patients.\ndisease_id: 7\nsource_id: 111\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	dc73ca14-1045-46cc-8cd5-8da524aef1eb	2015-06-21 16:49:43.14378
715	79	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '12'\nstart:\n- \n- '25398284'\nstop:\n- \n- '25398284'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	66de37a2-50f7-47da-a114-1f84d2681848	2015-06-21 16:52:23.076814
338	105	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF mutation correlated with poor prognosis in papillary thyroid cancer\n  in both older (>65 yo) and younger (<65 yo) cohorts.\ndisease_id: 16\nsource_id: 93\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	03256825-c684-4675-ba6b-beb0802ab7d8	2015-06-21 16:49:43.173965
339	106	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF V600E is correlated with poor prognosis in papillary thyroid cancer\n  in a study of 187 patients with PTC and other thyroid diseases.\ndisease_id: 16\nsource_id: 112\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	86195ef5-e636-47cc-aa2a-e6224f5c46ec	2015-06-21 16:49:43.20503
340	107	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: V600E is correlated with disease recurrence in both age cohorts (>65\n  and <65 yo).\ndisease_id: 16\nsource_id: 93\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	9dddd0d8-2f4e-472e-be64-d854656d8ebb	2015-06-21 16:49:43.234265
341	108	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were not associated with the M3 acute myeloid leukemia\n  FAB subtype (0/55).\ndisease_id: 3\nsource_id: 113\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	5af08176-bb69-4c79-8f6a-d4d65947e920	2015-06-21 16:49:43.267435
342	109	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: No NPM1 mutations were identified in patients with favorable risk cytogenetics\n  (79/215 patients)\ndisease_id: 3\nsource_id: 68\nvariant_id: 87\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	45a1d966-da9d-451d-90e1-ce345a9c06cf	2015-06-21 16:49:43.297691
343	7	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: NPM1 Exon 12\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b6cc8671-a33c-4703-bbdb-b1cb97f2f0e5	2015-06-21 16:49:43.317292
344	110	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Therapy-related AML was less common in patients with DNMT3A mutations\n  (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of\n  younger (18-60) AML patients\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \n	1	\N	\N	40907424-9bba-4a69-bc11-0ce66fc58dbd	2015-06-21 16:49:43.34668
345	111	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid\n  leukemia.\ndisease_id: 3\nsource_id: 114\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \n	1	\N	\N	4d776d00-4abc-4286-a0f4-ad0500ede6b0	2015-06-21 16:49:43.375752
346	112	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (59% of which were R882) were associated with an intermediate\n  risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes\ndisease_id: 3\nsource_id: 68\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	f349f4fb-4005-458c-9d12-4c4bae52eab2	2015-06-21 16:49:43.404953
347	113	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (59% of which were R882) were associated with intermediate\n  risk cytogenetics (including normal karyotype)\ndisease_id: 3\nsource_id: 68\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	9b425793-a0dc-4ba6-b156-4b1f386af330	2015-06-21 16:49:43.432007
348	114	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (64.5% R882) were associated with older age, higher\n  white blood cell count and cytogenetically normal AML in a large cohort of younger\n  (18-60) AML patients\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	59e2eb7c-6cac-412f-9b05-27d1a5837efd	2015-06-21 16:49:43.459673
349	115	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutations were associated with cytogenetically normal AML\n  in a large cohort of younger (18-60) AML patients\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	246b75af-6a1f-4a55-9080-bb9a2abdae84	2015-06-21 16:49:43.487524
350	116	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML with mutated NPM1 is a provisional entity in WHO classification of\n  acute myeloid leukemia (AML). This mutation should be tested for in clinical trials\n  and is recommended for testing in patients with cytogentically normal AML.\ndisease_id: 3\nsource_id: 115\nvariant_id: 86\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	9652d712-b793-4fbe-b504-bb7d65aca33e	2015-06-21 16:49:43.524287
351	117	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with de novo AML as well as M4 and M5\n  FAB subtypes\ndisease_id: 3\nsource_id: 116\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	ceda059d-4fd0-4ebd-8c92-5aaabe193c91	2015-06-21 16:49:43.560658
352	118	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute\n  myeloid leukemia.\ndisease_id: 3\nsource_id: 114\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	1a425803-0655-42e8-bfb2-f4fb59ef2656	2015-06-21 16:49:43.61324
354	120	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with normal karyotype in older (>60) patients\ndisease_id: 3\nsource_id: 117\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	13d80166-190e-44af-9400-fd18f7ea9183	2015-06-21 16:49:43.701344
355	121	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with intermediate risk cytogenetics (including\n  normal karyotype)\ndisease_id: 3\nsource_id: 68\nvariant_id: 87\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	0e40f285-ca27-4ee3-a200-1a72153820a4	2015-06-21 16:49:43.738134
356	122	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEBPA mutation status had no impact on ATRA treatment response in older\n  (>60) patients with AML\ndisease_id: 3\nsource_id: 118\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	260ce404-c471-4786-90fa-dce234993c1b	2015-06-21 16:49:43.786612
357	123	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA\n  mutations (Exon 9) had no significant impact on response rate, disease control rate,\n  progression free surivival or overall survival following cetuximab plus chemotherapy\n  than those with wildtype PIK3CA\ndisease_id: 11\nsource_id: 76\nvariant_id: 106\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	19729113-2889-45b3-9b34-34ae368df8c7	2015-06-21 16:49:43.827211
358	124	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In chemotherapy-refractory patients with colorectal cancer, NRAS mutation\n  status (primarily Q61) was not informative for disease control rate, progression\n  free survival or overall survival following cetuximab plus chemotherapy.\ndisease_id: 11\nsource_id: 76\nvariant_id: 94\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	394fb9b7-be83-4cc6-b934-1b11c05d9401	2015-06-21 16:49:43.857511
359	125	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation.\ndisease_id: 13\nsource_id: 119\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	1cab1d4c-82b5-4594-83de-e8d1a68b02a1	2015-06-21 16:49:43.89438
360	126	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring BRAF\n  mutations had lower response and disease control rates as well as shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype BRAF\ndisease_id: 11\nsource_id: 76\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	a1f0bedc-a6bb-41e6-9dad-59aec3b5acbe	2015-06-21 16:49:43.923705
361	127	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: ATRA treatment did not effect overall survival in patients <60 years\n  old with FLT3-ITD mutation\ndisease_id: 3\nsource_id: 120\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	2d90869f-e13e-4e89-ace8-7efd20aa7e44	2015-06-21 16:49:43.958119
362	128	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: FLT3-ITD mutation without NPM1 was associated with increased relapse\n  risk and reduced overall survival in young adult AML patients (median age 43)\ndisease_id: 3\nsource_id: 116\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	2f759ddd-3008-4073-a488-819732dd5881	2015-06-21 16:49:43.992511
363	129	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Increasing level of FLT3-ITD mutant correlated with reduced disease-free\n  and overall survival as well as increased relapse rate\ndisease_id: 3\nsource_id: 116\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	1ea664da-a02f-4d90-8587-5c0803ce0692	2015-06-21 16:49:44.025143
364	130	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to daunorubicin than cells with FLT3-ITD alone.\ndisease_id: 3\nsource_id: 121\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	ac7e5627-9e3c-492b-9d87-195f44407653	2015-06-21 16:49:44.055499
365	131	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells\n  with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\ndisease_id: 3\nsource_id: 121\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	1dc03f5c-f3ff-4314-81e9-a3837a14e0e0	2015-06-21 16:49:44.085272
366	132	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to daunorubicin than cells with FLT3-ITD alone.\ndisease_id: 3\nsource_id: 121\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	3d39f047-a9cd-4d97-a0f9-4ee40fcaa0ac	2015-06-21 16:49:44.121351
380	145	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment\n  in isogenic SW48 cells and in a mouse xenograft model\ndisease_id: 17\nsource_id: 123\nvariant_id: 81\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398281-25398281 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	a335d64e-1a69-485d-b87b-a44b4941b4be	2015-06-21 16:49:44.701258
367	133	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells\n  with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\ndisease_id: 3\nsource_id: 121\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	05855188-cb77-4deb-adca-2b791d150e7b	2015-06-21 16:49:44.159395
368	134	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring KRAS\n  mutations (primarily G12/G13) had lower response and disease control rates and shorter\n  progression free and overall survival following cetuximab plus chemotherapy than\n  those with wildtype KRAS\ndisease_id: 11\nsource_id: 76\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	6e52603f-bc1b-414b-aefe-42599f79c5ab	2015-06-21 16:49:44.19992
369	135	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations were significantly associated with lack of response to\n  cetuximab in patients with advanced colorectal cancer\ndisease_id: 11\nsource_id: 122\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	644f102d-6eea-499e-a6d9-d666b4c5bb5a	2015-06-21 16:49:44.241203
370	136	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment\n  in isogenic SW48 cells and in a mouse xenograft model\ndisease_id: 17\nsource_id: 123\nvariant_id: 81\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398281-25398281 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	9dda265c-a4e0-47f1-90f7-af482669a59e	2015-06-21 16:49:44.289802
371	137	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: ATRA treatment did not effect overall survival in patients <60 years\n  old with NPM1 mutation regardless of FLT3-ITD status\ndisease_id: 3\nsource_id: 120\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	417f6c6f-52f8-4c01-a129-8c9ba3b669a1	2015-06-21 16:49:44.326708
372	138	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring Exon\n  20 PIK3CA mutations had lower response and disease control rates as well as shorter\n  progression free and overall survival following cetuximab plus chemotherapy than\n  those with wildtype PIK3CA\ndisease_id: 11\nsource_id: 76\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	399f03ff-51c8-46c6-b74d-4e0b814b710c	2015-06-21 16:49:44.370519
373	139	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with an inferior response to induction\n  chemotherapy with a higher rate of resistant disease in young (15-60+, median 45)\n  patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 124\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	20b81a2e-b4b2-4bf0-ab8f-a56253b6aea5	2015-06-21 16:49:44.409826
374	140	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with an inferior response to induction\n  chemotherapy with a higher rate of resistant disease in young (15-60+, median 45)\n  patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 124\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	1207c2a1-7734-49e2-943d-a501e9e26dab	2015-06-21 16:49:44.446033
375	141	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CH5424802 treatment resulted in significant tumor regression in xenograft\n  models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M\n  mutation\ndisease_id: 8\nsource_id: 77\nvariant_id: 7\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	a4cf9118-9393-4bfd-bad1-b6cf0ae38799	2015-06-21 16:49:44.485606
376	142	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing\n  ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition\ndisease_id: 13\nsource_id: 79\nvariant_id: 8\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	cae97326-15bd-49fe-9282-a8efcde93827	2015-06-21 16:49:44.522698
377	143	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation\ndisease_id: 13\nsource_id: 119\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29432664-29432664 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	9c14edaf-d4b3-4db4-94f9-33796f6e647c	2015-06-21 16:49:44.55811
378	144	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Following identification of this fusion in a patient, this fusion was\n  expressed in HEK 293T cells leading to oligomerization in the absence of ligand,\n  induced morphologic changes in the cell and increased cellular proliferation consistent\n  with FGFR activation. Mouse xenografts using the bladder cell line SW780 which harbors\n  an FGFR3-BAIAP2L1 fusion showed reduced tumor burden when treated with the FGFR\n  inhibitor PD173074.\ndisease_id: 18\nsource_id: 125\nvariant_id: 53\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	a9be52e3-e273-40d6-a1f0-1c9710d02f52	2015-06-21 16:49:44.624953
379	8	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: FGFR fusions\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b6b3df8f-d4d6-4706-8a40-e3324f18b35c	2015-06-21 16:49:44.66625
694	16	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453136'\nstop:\n- \n- '140453137'\nreference_bases:\n- \n- AC\nvariant_bases:\n- \n- CT\n	2	\N	\N	2fff0306-d9a8-406e-8340-4238074b7c1f	2015-06-21 16:52:22.750589
381	146	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: ATRA treatment improved overall and relapse-free survival in older (>60)\n  patients with mutant NPM1 and without FLT3-ITD\ndisease_id: 3\nsource_id: 117\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	54f54397-f685-41d4-88fb-fd2c56f37481	2015-06-21 16:49:44.746029
382	147	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young patients (18-60) with NPM1 mutations had improved overall survival\n  following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis.\n  However, the adjusted p-value for multiple testing is 0.11.\ndisease_id: 3\nsource_id: 126\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	078f8dc0-0453-452e-b520-b8323d24941f	2015-06-21 16:49:44.801701
383	148	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: For patients with NPM1 mutation that achieved CR following induction\n  therapy, relapse-free survival was improved following treatment with valproic acid\ndisease_id: 3\nsource_id: 127\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	6107b392-8a3a-4e16-9966-826d3d736d42	2015-06-21 16:49:44.839103
384	149	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NSC348884 combined with ATRA treatment induces apoptosis in primary AML\n  cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1\n  and FLT3-ITD\ndisease_id: 3\nsource_id: 128\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	add00717-e916-436c-be46-0243866ed745	2015-06-21 16:49:44.872586
385	150	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CD33 and CD123 expression was significantly increased in patients with\n  NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33\n  and anti-CD123 therapy\ndisease_id: 3\nsource_id: 129\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	cb9b1ce0-5e86-4f52-91bd-e5f915efa8a8	2015-06-21 16:49:44.910804
386	151	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CD33 expression was significantly increased in patients with NPM1 mutation\n  with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy\ndisease_id: 3\nsource_id: 129\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	275179d6-bff5-40cc-898c-efac403e8505	2015-06-21 16:49:44.942439
387	152	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.\ndisease_id: 3\nsource_id: 128\nvariant_id: 87\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	ebf8649d-4b19-47be-ad5d-14e7835d08e1	2015-06-21 16:49:44.984371
388	153	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM\n  when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization\n  of NPM in AML patients was associated with good response to induction therapy.\ndisease_id: 3\nsource_id: 130\nvariant_id: 87\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	07228c4d-86ab-4ecc-8b79-8aa16a156dc9	2015-06-21 16:49:45.03533
389	154	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Following rapamycin treatment of mice with KrasG12D-induced lung tumors,\n  mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung\n  exhibit increased overall survival, increased apoptosis, reduced proliferation and\n  reduced tumor burden compared to Tsc1 wildtype mice.\ndisease_id: 8\nsource_id: 131\nvariant_id: 124\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	958cc4cd-b90a-45d0-9893-c0edd15bc9f5	2015-06-21 16:49:45.09202
390	9	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: TSC Loss\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	3e8b7989-d622-44e2-a99a-8e360534fb45	2015-06-21 16:49:45.119882
391	155	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity\n  to everolimus than their wild-type counterparts.\ndisease_id: 18\nsource_id: 132\nvariant_id: 125\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	3d7650aa-5418-4b9b-ba09-b69846e70b5d	2015-06-21 16:49:45.163712
392	156	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\ndisease_id: 3\nsource_id: 133\nvariant_id: 28\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	43e3264d-8b59-4dc7-afce-cabbbcfa9a83	2015-06-21 16:49:45.212049
393	157	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rates did not differ between patients with wildtype\n  or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML\ndisease_id: 3\nsource_id: 86\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	41b14890-7040-44f9-b753-1986d9a1c58d	2015-06-21 16:49:45.240972
394	158	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: FLT3-ITD duplication length did not effect prognosis in patients with\n  an NPM1 mutation\ndisease_id: 3\nsource_id: 134\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	0df2fc92-c493-49f6-a26e-12546df060b1	2015-06-21 16:49:45.271506
695	33	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '55259514'\nstop:\n- \n- '55259515'\nreference_bases:\n- \n- CT\nvariant_bases:\n- \n- AG\n	2	\N	\N	a14ee269-6e15-45fc-ac98-d8cfc2fb7e04	2015-06-21 16:52:22.768547
395	159	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Size of the FLT3-ITD mutant duplication had no impact on overall survival\n  or relapse rate\ndisease_id: 3\nsource_id: 116\nvariant_id: 55\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	0c7ed785-87d3-48ff-94d1-2b60ca14ccfa	2015-06-21 16:49:45.30141
396	160	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutation status of IDH1 did not change event-free survival of patients\n  with an NPM1 mutation\ndisease_id: 3\nsource_id: 135\nvariant_id: 58\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	05d43e6a-9867-4dbc-98c1-aad1652b14f7	2015-06-21 16:49:45.334104
397	161	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutation status of IDH1 did not change event-free survival of patients\n  with an NPM1 mutation\ndisease_id: 3\nsource_id: 135\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	827be2f8-212d-4077-a643-37db185e3d80	2015-06-21 16:49:45.362536
398	162	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: No differences in relapse-free or overall survival were identified between\n  young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in\n  cytogenetically normal AML\ndisease_id: 3\nsource_id: 136\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	e67ec5fa-fc82-4dd7-a212-e7e6c99a4cde	2015-06-21 16:49:45.393828
399	163	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rates of complete remission and refractory disease are not different\n  in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young\n  (16-60) patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 136\nvariant_id: 129\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	b636ac2f-8192-496a-a2d1-54234fa14c89	2015-06-21 16:49:45.426733
400	164	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: No differences in relapse-free or overall survival were identified between\n  young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in\n  cytogenetically normal AML\ndisease_id: 3\nsource_id: 136\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	048abef7-fd29-414f-9817-3de46375d46a	2015-06-21 16:49:45.456379
401	165	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rates of complete remission and refractory disease are not different\n  in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young\n  (16-60) patients with cytogenetically normal AML.\ndisease_id: 3\nsource_id: 136\nvariant_id: 130\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	3855ff30-ff4d-4d0c-9e3e-4e01f3e92934	2015-06-21 16:49:45.486145
402	166	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEBPA mutation was associated with improved overall survival in older\n  (>60) patients with AML\ndisease_id: 3\nsource_id: 118\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	f603d6f5-8846-410a-b05f-afec6f79fc60	2015-06-21 16:49:45.51672
403	167	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEBPA mutation was significantly associated with complete remission\ndisease_id: 3\nsource_id: 137\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	08741fb3-7296-42c9-93ee-ae68b59eb58d	2015-06-21 16:49:45.547479
404	168	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Overall survival was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA mutations\ndisease_id: 3\nsource_id: 133\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	974ad3af-a753-41bf-98c7-a054339a1c42	2015-06-21 16:49:45.578665
405	169	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Remission duration was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\ndisease_id: 3\nsource_id: 133\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	4c9a7dd7-0774-4b78-9343-96fbeddcfe7d	2015-06-21 16:49:45.61438
406	170	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations were associated with achievement of complete remission\n  in a large cohort of younger (18-60) AML patients\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	c79a3387-4184-4541-9b2e-bd4038f5a543	2015-06-21 16:49:45.655386
407	171	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Event-free survival was significantly improved in patients with both\n  NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone\n  2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis\n  3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal\n  karyotype patients were analyzed\ndisease_id: 3\nsource_id: 138\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	aa70363d-edcd-42c2-ba82-85cab88b22e0	2015-06-21 16:49:45.699124
408	172	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MGMT promoter methylation was associated with increased overall and disease-free\n  survival as well as tumor regression in patients with gliomas.\ndisease_id: 19\nsource_id: 139\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	8ac3be73-63ba-450a-83e8-76b86d89e950	2015-06-21 16:49:45.744671
409	173	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation without FLT3-ITD was associated with reduced relapse risk\n  and increased overall survival in young adult acute myeloid leukemia patients (median\n  age 43)\ndisease_id: 3\nsource_id: 116\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	a533286d-80ed-4c32-a07f-8292c478f39a	2015-06-21 16:49:45.786909
410	174	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutations were associated with improved complete remission\n  rates as well as overall and disease free survival when all ages and karyotype were\n  analyzed; however, when only normal karyotype patients were analyzed, only complete\n  remission rates were improved\ndisease_id: 3\nsource_id: 117\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	57e17d17-ec2f-4cab-b9a8-1d453e5fa154	2015-06-21 16:49:45.832836
411	175	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rates were higher and both disease-free and overall\n  survival were longer for patients with Exon 12 NPM1 mutations.\ndisease_id: 3\nsource_id: 113\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	7a26bf63-db03-4a71-81de-6aa02d6be3c6	2015-06-21 16:49:45.868669
412	176	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rates were higher and event-free survival was longer\n  for normal karyotype AML patients with Exon 12 NPM1 mutations.\ndisease_id: 3\nsource_id: 114\nvariant_id: 86\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	3f2a42e3-300e-4da5-b1a4-845088a71161	2015-06-21 16:49:45.908125
413	177	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cytoplasmic localization of NPM in AML patients was an independent prognositic\n  factor associated with complete remission in response to induction therapy in a\n  multivariate analysis (p=0.019) but not in a univariate analysis (p=0.07).\ndisease_id: 3\nsource_id: 130\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	9ffd71c5-d7d6-440f-98c5-219a43ca4c19	2015-06-21 16:49:45.948647
414	178	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Event-free survival was significantly improved in patients with both\n  NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone\n  2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis\n  3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal\n  karyotype patients were analyzed\ndisease_id: 3\nsource_id: 138\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	30e0aecc-1a1c-4c35-b241-eba1af97bf62	2015-06-21 16:49:45.980895
415	179	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Intermediate risk patients with mutant NPM1 had improved overall survival\n  with the presence of either IDH1 or IDH2 mutations than those wildtype for both\n  IDH1 and IDH2\ndisease_id: 3\nsource_id: 126\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	4d3c8d7c-32f8-47ca-a1b0-8cf9bcc2e98c	2015-06-21 16:49:46.011238
416	180	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Normal karyotype patients with NPM1 mutation and without FLT3-ITD had\n  improved overall and relapse-free survival, complete remission rates and remission\n  duration compared to all other combinations of mutants\ndisease_id: 3\nsource_id: 140\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	8f27825f-a65a-457f-8967-ef521cebe774	2015-06-21 16:49:46.046746
417	181	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation was associated with higher complete remission rates, lower\n  cumulative incidence of relapse and higher overall survival in intermediate risk\n  AML patients\ndisease_id: 3\nsource_id: 141\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	41955a3b-435f-4906-903f-8c3c6cc6edc0	2015-06-21 16:49:46.078672
418	182	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation was associated with increased overall, event-free and relapse-free\n  survival and remission after induction therapy in the entire cohort and in the cytogenetically\n  normal subset of patients\ndisease_id: 3\nsource_id: 142\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	b95eb364-0040-47b6-b1ee-1f0a9cc062a6	2015-06-21 16:49:46.11657
419	183	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation without FLT3-ITD was significantly associated with complete\n  remission\ndisease_id: 3\nsource_id: 137\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	bdc438fc-399d-47c4-b3e8-7aeee042de53	2015-06-21 16:49:46.154467
420	184	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with increased complete remission rates\n  as well as longer overall disease free survival in normal karyotype AML patients\n  >59 years old, particularly in those >69 years old\ndisease_id: 3\nsource_id: 143\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	ee157a91-c31f-4f60-8585-67c7ddcf125b	2015-06-21 16:49:46.196768
421	185	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation\n  and without FLT3-ITD have increased overall survival, relapse-free survival and\n  response to induction chemotherapy\ndisease_id: 3\nsource_id: 144\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	6ee960d1-5e7f-45dc-a81e-2553688ee657	2015-06-21 16:49:46.238574
422	186	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rate was not different between young AML patients\n  (<60 years old) with or without DNMT3A mutations (60% of which were R882)\ndisease_id: 3\nsource_id: 54\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	b71c8f8c-92a7-4f11-a1c9-666a2fdbd64a	2015-06-21 16:49:46.27562
423	187	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (62% of which were R882) were associated with reduced\n  disease-free survival in patients with cytogenetically normal AML.\ndisease_id: 3\nsource_id: 86\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	e92e3aab-f64b-4aec-88a0-a43599799d8d	2015-06-21 16:49:46.314266
424	188	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young AML patients (<60 years old) with DNMT3A mutation have shorter\n  overall survival and relapse-free survival than patients with wildtype DNMT3A\ndisease_id: 3\nsource_id: 54\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	d8ca9447-a23b-4e57-9d72-4291caadaeee	2015-06-21 16:49:46.351935
425	189	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter\n  overall survival and relapse-free survival than patients with wildtype DNMT3A\ndisease_id: 3\nsource_id: 54\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	94d968d1-f23d-472b-8eed-fbe794edee39	2015-06-21 16:49:46.386704
426	190	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Meta-analysis of studies involving cytogentically normal younger (<60)\n  patients showed reduced overall and relapse-free survival for patients with FLT3-ITD\ndisease_id: 3\nsource_id: 145\nvariant_id: 55\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	98ff3f2e-57f1-460b-8ef9-63649b95b12a	2015-06-21 16:49:46.425092
427	191	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced\n  event-free and overall survival in old (>59) and younger patients (<60) with an\n  NPM1 mutation\ndisease_id: 3\nsource_id: 134\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	81b5a549-36c4-4195-872a-813869ed6acf	2015-06-21 16:49:46.460161
428	192	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival\n  compared to patients with wildtype FLT3\ndisease_id: 3\nsource_id: 134\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	31847f4c-a47f-4fab-8495-2ac826f2b2e2	2015-06-21 16:49:46.527222
429	193	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable,\n  a higher ratio was associated with reduced event-free survival in patients with\n  an NPM1 mutation\ndisease_id: 3\nsource_id: 134\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	c3897d8a-3fc1-4d8b-85cf-1d93941c3862	2015-06-21 16:49:46.563463
430	194	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations were significantly associated with reduced progression-free\n  and overall survival in patients with advanced colorectal cancer\ndisease_id: 11\nsource_id: 122\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	d9bb8d19-f8fa-4194-aee1-9f22121820f9	2015-06-21 16:49:46.599813
431	195	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Coocurrence of FLT3-ITD mutations significantly reduced overall survival\n  of normal karyotype AML patients\ndisease_id: 3\nsource_id: 113\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	12cc031b-ff4f-4535-9911-c9913bf5add6	2015-06-21 16:49:46.638697
432	196	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse\n  in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction\n  in NPM1 levels following first line therapy (N=15)\ndisease_id: 3\nsource_id: 146\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	977b36ca-e295-4d25-b193-dbc9c48620a8	2015-06-21 16:49:46.676688
433	197	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival\n  compared to patients with wildtype FLT3\ndisease_id: 3\nsource_id: 134\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	ecf46c1f-d03d-45f3-9329-53e04eff88e6	2015-06-21 16:49:46.722851
434	198	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall\n  survival and cumulative incidence of relapse than those with NPM1 mutations alone\n  in intermediate risk AML patients\ndisease_id: 3\nsource_id: 141\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	8f00decd-f81c-4b34-ba6f-d2176589630c	2015-06-21 16:49:46.761853
435	199	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: TET2 mutation reduces overall survival of normal karyotype patients with\n  either an NPM1 mutation or NPM1 mutation without FLT3-ITD\ndisease_id: 3\nsource_id: 147\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	6f4255cf-84bc-47a5-8254-59163b6c7cbc	2015-06-21 16:49:46.802326
436	200	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable,\n  a higher ratio was associated with reduced event-free survival in patients with\n  an NPM1 mutation\ndisease_id: 3\nsource_id: 134\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	52c4a840-5349-4a81-9313-bdf714914742	2015-06-21 16:49:46.844315
437	201	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutations  in WT1 were associated with increased risk of recurrence in\n  young patients (15-50) with cytogenetically normal AML\ndisease_id: 3\nsource_id: 148\nvariant_id: 129\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	4484667f-b84d-423e-bd00-f0c639586f79	2015-06-21 16:49:46.882198
438	202	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutations in WT1 were associated with a worse overall prognosis than\n  patients wildtype for WT1 in young patients (15-50), primarily because of increased\n  risk of disease recurrence\ndisease_id: 3\nsource_id: 148\nvariant_id: 129\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	c1159e52-b682-48ef-80a5-8dae1521e5b6	2015-06-21 16:49:46.919951
439	203	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had\n  significantly lower complete remission and higher refractory disease rates than\n  those with wildtype WT1 and without FLT3-ITD following induction therapy in young\n  (16-60) patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 136\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	c060908f-9dcb-42e3-825c-9dd4ba860e16	2015-06-21 16:49:46.961822
440	204	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\ndisease_id: 3\nsource_id: 149\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	29d48ddc-e2e2-4c5b-857f-6b7e992496f4	2015-06-21 16:49:46.998541
441	205	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were a negative prognostic factor for overall survival\n  in young (15-60+, median 45) patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 124\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	ad83ca9c-363e-4000-a392-77d2dda2180b	2015-06-21 16:49:47.034973
442	206	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\ndisease_id: 3\nsource_id: 124\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	2da0ceb2-2539-4a93-b9bc-547953ee948c	2015-06-21 16:49:47.070359
443	207	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had\n  significantly lower complete remission and higher refractory disease rates than\n  those with wildtype WT1 and without FLT3-ITD following induction therapy in young\n  (16-60) patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 136\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	3ffa91c2-6933-486d-8cf9-b882eff7fb64	2015-06-21 16:49:47.108267
444	208	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were a negative prognostic factor for overall survival\n  in young (15-60+, median 45) patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 124\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	5743c498-a58f-42ba-a295-5efcedaed1ef	2015-06-21 16:49:47.144885
445	209	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\ndisease_id: 3\nsource_id: 124\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	fbe92f05-eea5-4877-a4c7-7e8863455225	2015-06-21 16:49:47.189781
446	210	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\ndisease_id: 3\nsource_id: 149\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	0a7ffe89-76ab-42d3-a8f4-150e7552db42	2015-06-21 16:49:47.227042
447	211	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The combination of PARP inhibition (Olaparib) with BRCA mutation results\n  in irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation\n  positive women with high grade serous or undifferentiated ovarian cancer.\ndisease_id: 20\nsource_id: 150\nvariant_id: 131\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 1\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	a61f1a06-e0fc-42ef-92ab-270f20b521d1	2015-06-21 16:49:47.271129
448	10	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: BRCA Germline Variants\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	234f55ae-650e-4c20-b93d-cf158dfa6892	2015-06-21 16:49:47.296104
449	212	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The combination of PARP inhibition (Olaparib) with BRCA mutation results\n  in irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation\n  positive women with high grade serous or undifferentiated ovarian cancer.\ndisease_id: 20\nsource_id: 150\nvariant_id: 132\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 1\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	7e9f06bc-fe03-477d-9348-e6b90caa9ca7	2015-06-21 16:49:47.336355
450	213	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Data from deep sequencing of a Ph-negative clone of a Ph-positive CML\n  patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion,\n  and may imply the possibility that BCR-ABL is not universally the initiating event\n  in CML.\ndisease_id: 4\nsource_id: 151\nvariant_id: 1\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	f5c99a1d-dda7-4e72-859a-3b98fd7079b7	2015-06-21 16:49:47.384048
451	214	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WBC counts were not significantly different in AML patients with FLT3\n  D835 mutations compared to wild type\ndisease_id: 3\nsource_id: 152\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	3a30f68f-0929-489a-acf1-7d74f44afebc	2015-06-21 16:49:47.427304
452	215	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations in lung cancer patients are not associated with smoking\n  history, age or gender\ndisease_id: 8\nsource_id: 153\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	62533cef-c4df-49cd-a397-2abd753b57dd	2015-06-21 16:49:47.463858
453	216	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations in lung cancer patients are not associated with smoking\n  history, age or gender\ndisease_id: 8\nsource_id: 153\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	44f55b27-6913-4afd-8038-0774ca74f8d8	2015-06-21 16:49:47.494173
454	217	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Age, sex, FAB subtype and karyotypes were not statistically significant\n  between AML patients with U2AF Q157P/R mutations and those who harbor wild type\n  U2AF.\ndisease_id: 3\nsource_id: 154\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	11d2f5c2-1420-4671-a4b8-fea4af83582c	2015-06-21 16:49:47.531837
455	218	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Age, sex, FAB subtype and karyotypes were not statistically significant\n  between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.\ndisease_id: 3\nsource_id: 154\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	77037251-6fc1-410d-bfb6-8a3ed60e01ea	2015-06-21 16:49:47.563675
456	219	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CCND2 hypomethylation is seen to be more common in stage III and IV gastric\n  tumors than stage I and II.\ndisease_id: 21\nsource_id: 155\nvariant_id: 22\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \n	1	\N	\N	2e284c61-cf72-45c2-818b-4aab2f42dac4	2015-06-21 16:49:47.594925
457	220	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of BCR-ABL fusion is considered the characterizing feature\n  of chronic myeloid leukemia, and has been widely thought of as the initiating event\n  in the disease.\ndisease_id: 4\nsource_id: 156\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	078c4093-0672-42fc-b9c5-5791b5e7ecfd	2015-06-21 16:49:47.627608
458	221	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D1 overexpression is associated with the ER-positive subtype of\n  breast cancer.\ndisease_id: 22\nsource_id: 157\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	8a2d8153-9fa8-449c-b901-58850a649e3f	2015-06-21 16:49:47.658572
459	222	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WBC counts were higher in AML patients with internal tandem repeat of\n  Flt3 compared to wild type\ndisease_id: 3\nsource_id: 152\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	c69e80d1-87f4-4dea-9b43-1b88f96da0ae	2015-06-21 16:49:47.688901
460	223	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In leukemia patients, IDH1 R132 mutation is associated with adults and\n  normal karyotype.\ndisease_id: 3\nsource_id: 158\nvariant_id: 58\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	6bbe1a1b-8b48-4d11-9852-44b056ef23b3	2015-06-21 16:49:47.718427
461	224	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation is associated with patients of older age, high platelet\n  count during diagnosis, cytogenic normalcy and NPM1 mutation\ndisease_id: 3\nsource_id: 159\nvariant_id: 59\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:209113113-209113113 (G->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	e8f540e8-e75d-43d9-8636-837c880a7143	2015-06-21 16:49:47.749248
462	225	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutation is associated with larger, more invasive tumors, greater\n  pathologic histology and older patients compared to tumors with wildtype KIT\ndisease_id: 14\nsource_id: 160\nvariant_id: 66\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	ffa582a3-2991-4baa-bc27-391568dc5358	2015-06-21 16:49:47.78535
463	226	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutation is associated with larger, more invasive tumors, greater\n  pathologic histology and older patients compared to tumors with wildtype KIT\ndisease_id: 14\nsource_id: 160\nvariant_id: 70\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	43fbc223-e314-46a9-84ee-87acfadc2221	2015-06-21 16:49:47.818107
696	34	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '55249071'\nstop:\n- \n- '55249071'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	d62e0ff8-a4c4-4dfa-9eca-6388c450a019	2015-06-21 16:52:22.782298
464	227	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12C occur more frequently in women than men\ndisease_id: 23\nsource_id: 161\nvariant_id: 78\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398285-25398285 (C->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	9a21f3e4-4c0e-4824-8da7-ae9de08c8838	2015-06-21 16:49:47.859307
465	228	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12D mutation occurs in never smokers significantly more often than\n  in smokers\ndisease_id: 23\nsource_id: 161\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398284-25398284 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	80457573-2ca3-4f87-a660-e9bdc3546ee6	2015-06-21 16:49:47.893129
466	229	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no statistical difference in progression free survival between\n  lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.\ndisease_id: 8\nsource_id: 162\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55259514-55259515 (CT->AG)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	42bff280-25e4-4857-ae5b-ffed70cbd64c	2015-06-21 16:49:47.92954
467	230	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no statistical difference in progression free survival between\n  lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.\ndisease_id: 8\nsource_id: 162\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55259514-55259515 (CT->AG)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	93bdaa6d-8d92-40cf-89a6-a27ff77a1d6b	2015-06-21 16:49:47.96253
468	231	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity\n  to the AKT inhibitor MK-2206 in breast cancer cell lines.\ndisease_id: 24\nsource_id: 163\nvariant_id: 4\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 14:105246551-105246551 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	6a07d4b0-6537-4dcb-b2d2-2a85f7134a60	2015-06-21 16:49:48.002032
469	232	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BCR-ABL fusions have been found to contribute to imatinib resistance\n  in AML cell lines.\ndisease_id: 3\nsource_id: 164\nvariant_id: 1\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	835d4c9a-e56c-4a08-9ac3-8a58dbd17e40	2015-06-21 16:49:48.040899
470	233	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\ndisease_id: 4\nsource_id: 165\nvariant_id: 3\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	de9992e1-80c5-45cf-be3f-f879c4e70e32	2015-06-21 16:49:48.070695
471	234	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In chronic myeloid leukemia patients with the ABL T315I mutation, tumors\n  have shown to be resistant to imatinib treatment.\ndisease_id: 4\nsource_id: 156\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	ee1dc3f5-c483-40e0-a849-35a151d6db7a	2015-06-21 16:49:48.103659
472	235	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\ndisease_id: 4\nsource_id: 165\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	bc323af4-d6b6-4ef3-9a66-9240cce12b56	2015-06-21 16:49:48.170721
473	236	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with non-small cell lung cancer harboring EML4-ALK fusion,\n  the C1156Y variant has been shown to confer resistance to crizotinib.\ndisease_id: 8\nsource_id: 166\nvariant_id: 6\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	f946b5c6-ff75-471f-b1ea-16150b93856d	2015-06-21 16:49:48.210079
474	11	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: ALK Fusions\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	36d64409-cea8-4958-9696-3328f937ffa7	2015-06-21 16:49:48.231005
475	237	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with non-small cell lung cancer harboring EML4-ALK fusion,\n  the L1196M variant has been shown to confer resistance to crizotinib.\ndisease_id: 8\nsource_id: 166\nvariant_id: 7\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	28549114-7d39-4d04-83df-f0f664c921d6	2015-06-21 16:49:48.260275
476	238	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The T790M mutation in EGFR has been shown to confer resistance to the\n  tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are\n  placed on the drug are likely to relapse.\ndisease_id: 25\nsource_id: 167\nvariant_id: 34\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	13e612cd-3b68-482f-a614-20df460b0caa	2015-06-21 16:49:48.302901
477	12	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: EGFR TKI Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	4f1724c8-6b85-4617-b86b-6cd1e9444c5b	2015-06-21 16:49:48.329086
478	239	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In non-small cell lung cancer, the appearance of T790M mutation leads\n  to resistance to gefitinib.\ndisease_id: 8\nsource_id: 168\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	d11d07d8-fd69-4aee-86a9-2f516637efa5	2015-06-21 16:49:48.364741
697	35	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37880261'\nstop:\n- \n- '37880261'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- C\n	2	\N	\N	718e329e-eaa2-414c-b556-5b0596f10851	2015-06-21 16:52:22.795174
698	36	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37880261'\nstop:\n- \n- '37880261'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	39e173e8-ed60-4f7d-aba5-9b65d71bd712	2015-06-21 16:52:22.808089
479	240	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804),\n  MET amplification and EGFR T790M were found to be the primary drivers of resistance.\ndisease_id: 8\nsource_id: 169\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	c3eed3dc-5517-41d6-9ac6-0cb433fbb7d5	2015-06-21 16:49:48.396457
480	241	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The L755S mutation was shown to confer resistance to lapatinib in MCF10A\n  cell lines retrovirally transduced with wildtype or mutant ERBB2.\ndisease_id: 22\nsource_id: 170\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37880220-37880220 (T->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	0b06a308-7b0d-462a-8346-e4e2c6e72328	2015-06-21 16:49:48.441331
481	242	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 22\nsource_id: 171\nvariant_id: 46\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	382d6ee5-c07c-4ee4-9514-2c1e46fd800b	2015-06-21 16:49:48.479294
482	13	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: ESR1 Ligand-Binding Domain\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	d9e69a71-4b6e-4d1a-a289-5494fd5092e0	2015-06-21 16:49:48.503138
483	243	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the N538G mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 22\nsource_id: 171\nvariant_id: 47\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	2589136a-3b45-46d5-927e-3ee29d016d6a	2015-06-21 16:49:48.534971
484	244	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 22\nsource_id: 171\nvariant_id: 48\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419923-152419923 (A->G)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	f0e6a47a-0d32-4f05-ac28-e554e3f3dfc6	2015-06-21 16:49:48.573413
485	245	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 22\nsource_id: 171\nvariant_id: 49\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419922-152419922 (T->A)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	ee1c48d2-6e3d-4b10-93b8-32bcff07f5e0	2015-06-21 16:49:48.615183
486	246	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the L537S mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 22\nsource_id: 171\nvariant_id: 50\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419923-152419923 (A->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	940c28de-19e7-4522-abaa-00f0809f5a88	2015-06-21 16:49:48.662309
487	247	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Emergence of D835 mutation in AML patients with FLT3 tandem repeat is\n  associated with sorafenib resistance\ndisease_id: 3\nsource_id: 172\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	6585f391-17e5-4014-aef7-ee7a1a0d3040	2015-06-21 16:49:48.699014
488	248	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Sorafenib is effective in patients with FLT3 internal tandem repeat mutations,\n  but not in patients with wild type or D835 mutation.\ndisease_id: 3\nsource_id: 173\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	bf0adb08-756c-45fc-b8af-1396cfe99017	2015-06-21 16:49:48.735034
489	249	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT V654A results in imatinib resistance in GIST patient-derived cell\n  lines.\ndisease_id: 14\nsource_id: 62\nvariant_id: 73\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55594258-55594258 (T->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	8e985276-0839-4a63-bcb4-fe69723b5989	2015-06-21 16:49:48.767714
490	250	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer who harbor KRAS mutation have low response\n  rate to cetuximab\ndisease_id: 11\nsource_id: 174\nvariant_id: 74\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	7fa5c04b-0226-41f8-b3e9-0ad8fc706bf7	2015-06-21 16:49:48.811409
491	251	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 26\nsource_id: 175\nvariant_id: 148\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	af9cab1c-df31-4d4b-9157-cefc6d978eef	2015-06-21 16:49:48.853695
492	252	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 26\nsource_id: 175\nvariant_id: 78\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	fa438184-83c9-4dad-8130-7ff3e53fa11c	2015-06-21 16:49:48.894793
493	253	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 26\nsource_id: 175\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	a54ab6b8-658a-48da-a1de-6c7b93ac10c1	2015-06-21 16:49:48.933358
699	38	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37868205'\nstop:\n- \n- '37868205'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- C\n	2	\N	\N	15f9c67c-8803-4133-91d6-27ac08a4c584	2015-06-21 16:52:22.822799
494	254	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 26\nsource_id: 175\nvariant_id: 147\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	8f55d890-59df-4458-93bc-cdb58d8f6721	2015-06-21 16:49:48.971181
495	255	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804),\n  MET amplification and EGFR T790M were found to be the primary drivers of resistance.\ndisease_id: 8\nsource_id: 169\nvariant_id: 146\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	7304081e-dccb-4b2d-af77-070534a08801	2015-06-21 16:49:49.015398
496	256	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20\n  were associated with lack of panitumumab or cetuximab response.\ndisease_id: 11\nsource_id: 176\nvariant_id: 103\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	014919dc-20ae-492b-91a1-700c63f8d1dd	2015-06-21 16:49:49.061634
497	257	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20\n  were associated with lack of panitumumab or cetuximab response.\ndisease_id: 11\nsource_id: 176\nvariant_id: 104\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	622126ba-0284-4c5a-8212-fb4a9ac4d8da	2015-06-21 16:49:49.092781
498	258	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R)\n  were associated with lack of panitumumab or cetuximab response.\ndisease_id: 11\nsource_id: 176\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	01d52f6a-16d7-44f5-b011-6353b1ca50f0	2015-06-21 16:49:49.130499
499	259	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Treatment of Philadelphia Chromosome positive leukemias with Arsenic\n  Trioxide results in high rates of complete remission in patients.\ndisease_id: 3\nsource_id: 164\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	7123167f-b91f-4de1-904c-c8e7e55af7f9	2015-06-21 16:49:49.162957
500	260	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The clinical use of imatinib in patients with BCR-ABL fusion has resulted\n  in drastic sensitivity to the drug.\ndisease_id: 4\nsource_id: 156\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	f24d4a78-7d51-4a79-8583-738b09409014	2015-06-21 16:49:49.192848
501	261	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The use of second-generation BCR-ABL targeted therapies has been effective\n  in patients resistant to imatinib.\ndisease_id: 4\nsource_id: 156\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	9ed1324c-fe94-4dc6-8ea5-2314e01ad0f4	2015-06-21 16:49:49.220702
502	262	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with non-small cell lung cancer harboring the EML4-ALK fusion,\n  treatment with crizotinib has shown to be effective in acheiving stable disease.\ndisease_id: 8\nsource_id: 166\nvariant_id: 5\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	67571bf0-7151-4025-8592-38bbb5cbdb3d	2015-06-21 16:49:49.25714
503	263	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with\n  Palbociclib (PD-0332991) significantly increased median survival.\ndisease_id: 27\nsource_id: 177\nvariant_id: 23\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	5fed172f-a34c-47d7-aeb8-5d330b32c94e	2015-06-21 16:49:49.295461
504	264	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\ndisease_id: 28\nsource_id: 178\nvariant_id: 25\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	fc1fb127-9c01-4495-a7fd-c39a20681349	2015-06-21 16:49:49.330691
505	265	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\ndisease_id: 28\nsource_id: 178\nvariant_id: 26\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	e91bc3e4-e95a-487b-ae9f-450ccfb87754	2015-06-21 16:49:49.364696
506	266	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the G253C mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 8\nsource_id: 179\nvariant_id: 141\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	6a1123c4-93de-4bd5-b537-0349e3f70f48	2015-06-21 16:49:49.395961
507	267	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the G505S mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 8\nsource_id: 179\nvariant_id: 142\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	27989b9b-74d1-4a36-8636-76bcbae8fb16	2015-06-21 16:49:49.424524
700	39	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37880220'\nstop:\n- \n- '37880220'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	723a096d-3ec8-445b-95cf-9a00d6c9ca45	2015-06-21 16:52:22.835731
508	268	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the G774V mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 8\nsource_id: 179\nvariant_id: 144\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	bbea078c-b58d-4e33-adc7-d55ac361846e	2015-06-21 16:49:49.452464
509	269	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the I638F mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 8\nsource_id: 179\nvariant_id: 143\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	77a50f5c-344d-482c-ab95-89f005d6c4d8	2015-06-21 16:49:49.483372
510	270	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the L239R mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 8\nsource_id: 179\nvariant_id: 140\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	cd5137b7-8dfd-40a6-ad87-3c6013d0cbca	2015-06-21 16:49:49.519
511	271	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the L63V mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 8\nsource_id: 179\nvariant_id: 139\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	fd240c54-5fdf-4373-a386-9404ba37261c	2015-06-21 16:49:49.55745
512	272	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a patient expressing a DDR2 S768R mutation, treatment with dasatinib\n  plus erlotinib acheived a partial response.\ndisease_id: 8\nsource_id: 179\nvariant_id: 145\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	215832d4-e5e1-4649-9ce2-3e844914dd8e	2015-06-21 16:49:49.627812
513	273	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Exon 19 deletion has been shown to be correlated with gefitinib response.\ndisease_id: 8\nsource_id: 180\nvariant_id: 133\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	e9239b53-7f1e-43f0-9f10-d179eeeae9ea	2015-06-21 16:49:49.680259
514	274	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Gefinitib has been shown to be effective in treating cell lines with\n  G719S missense mutations.\ndisease_id: 8\nsource_id: 181\nvariant_id: 134\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	460f52ac-ee71-4bcc-818e-ee042529bd33	2015-06-21 16:49:49.71204
515	275	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence\n  of L858R mutation is prognostic for better progression free survival.\ndisease_id: 8\nsource_id: 182\nvariant_id: 33\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55259514-55259515 (CT->AG)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	b2f87c16-7aa9-4ffe-a4e5-52f67af52ac5	2015-06-21 16:49:49.74298
516	276	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Gefinitib has been shown to be effective in treating cell lines with\n  L858R missense mutations.\ndisease_id: 8\nsource_id: 181\nvariant_id: 33\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	8e7075a6-32d3-40bd-97dc-df2ca99f9e36	2015-06-21 16:49:49.778385
517	277	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a NSCLC patient with T790M and L858R mutation, combination treatment\n  with erlotinib and premetrexed has shown to be effective, reducing tumor size and\n  resulting in stable disease.\ndisease_id: 8\nsource_id: 183\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	36be0c36-415d-4eae-b786-9216592f2310	2015-06-21 16:49:49.809432
518	278	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC containing a T790M mutation, staurosporine may be inhibitive\n  of EGFR, especially when an L858R mutation is also present.\ndisease_id: 8\nsource_id: 184\nvariant_id: 34\nrating: 1\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	e2b203cc-8894-47bd-9d21-d5bdd008d4b9	2015-06-21 16:49:49.842552
519	279	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the D769H mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 35\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37880261-37880261 (G->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	36476ef3-d04c-45ac-9f77-5c9e5a4db2cd	2015-06-21 16:49:49.87826
520	14	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: HER2 Activating\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	34cdbb9e-7fa7-49bd-856c-c15cac65a221	2015-06-21 16:49:49.901979
521	280	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the D769Y mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 36\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37880261-37880261 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	0608b577-12d4-45bb-b333-cee66034fc97	2015-06-21 16:49:49.933814
522	281	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines retrovirally transduced with wildtype or mutant\n  ERBB2, cells expressing the in-frame deletion of amino acids 755-759 were shown\n  to be sensitive to neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 37\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	e9f51f0e-81fd-423d-9482-cdde81748baa	2015-06-21 16:49:49.968782
701	43	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37881616'\nstop:\n- \n- '37881616'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	882708c4-45fd-4194-b754-6d00a7b6d3ab	2015-06-21 16:52:22.849342
523	282	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the G309A mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 38\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37868205-37868205 (G->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	1687ab70-03b1-4ac4-853e-d4577598c823	2015-06-21 16:49:50.005781
524	283	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The L755S mutation showed mild response to neratinib in MCF10A cell lines\n  retrovirally transduced with wildtype or mutant ERBB2.\ndisease_id: 22\nsource_id: 170\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37880220-37880220 (T->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	02070507-63cd-4a93-a3fa-d3c5716ee4e8	2015-06-21 16:49:50.044996
525	284	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive\n  to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.\ndisease_id: 22\nsource_id: 170\nvariant_id: 40\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	3014cbf8-02cf-4017-9763-e2796a87254a	2015-06-21 16:49:50.074141
526	285	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the in-frame insertion of a single amino acid at\n  position 780 was shown to be sensitive to neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 41\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	7770c445-7f94-4497-80c8-9079af67e0ab	2015-06-21 16:49:50.102466
527	286	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\ndisease_id: 22\nsource_id: 170\nvariant_id: 42\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	8662d78a-ba9a-469c-9fd4-9c889a43dcaf	2015-06-21 16:49:50.137643
528	287	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the R896C mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 43\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37881616-37881616 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	9d5fb406-95e2-4a31-a1f2-f8de2dce0c1c	2015-06-21 16:49:50.170339
529	288	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the V777L mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 44\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37881000-37881000 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	5c320d26-377e-48bb-b6a8-426651a124ab	2015-06-21 16:49:50.214547
530	289	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the V842I mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 45\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37881332-37881332 (G->A)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	7aaef5ef-22f8-4a24-9ed5-c27b15a3be06	2015-06-21 16:49:50.24976
531	290	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The L536Q ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  MCF7 cell lines.\ndisease_id: 22\nsource_id: 185\nvariant_id: 46\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	2148b7dc-33c0-449b-8655-e55bd936f27f	2015-06-21 16:49:50.285412
532	291	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The N538G ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 22\nsource_id: 185\nvariant_id: 47\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	04205d92-fb9a-407e-8ec0-4118d46aa8a0	2015-06-21 16:49:50.323442
533	292	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The Y537C ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 22\nsource_id: 185\nvariant_id: 48\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419923-152419923 (A->G)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	c263c57f-1b57-4f38-9ff3-d46316ed28a7	2015-06-21 16:49:50.365076
534	293	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The Y537N ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 22\nsource_id: 185\nvariant_id: 49\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419922-152419922 (T->A)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	d5831c9b-0fd1-4d67-9d64-8e631013d637	2015-06-21 16:49:50.40269
535	294	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The Y537S ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 22\nsource_id: 185\nvariant_id: 50\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419923-152419923 (A->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	74707705-ac28-4eca-98c3-6998bea1a157	2015-06-21 16:49:50.443977
536	295	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a single patient with this fusion in intrahepatic cholangiocarcinoma.\n  Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic\n  sites and overall stable disease. However, this response did not meet criteria for\n  RECIST partial response.\ndisease_id: 29\nsource_id: 186\nvariant_id: 52\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	4505d713-9228-4457-89a2-ea816551ef4b	2015-06-21 16:49:50.48525
537	296	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This\n  fusion is also shown to be recurrent in multiple other cancer types.\ndisease_id: 29\nsource_id: 186\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	96e06110-fcc3-496a-8ae4-2857687fbac4	2015-06-21 16:49:50.521563
538	297	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated\n  AML patients with FLT3 mutations\ndisease_id: 3\nsource_id: 187\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	3f2a0038-6f08-472a-a3c4-3ccf97972b30	2015-06-21 16:49:50.569133
539	298	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Sorafenib is effective in patients with FLT3 internal tandem repeats,\n  but not in wild type or D835 mutation.\ndisease_id: 3\nsource_id: 173\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	36358bd3-0135-4578-b451-12a024aff45d	2015-06-21 16:49:50.602595
540	299	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML\n  cells with FLT3 internal tandem repeat\ndisease_id: 3\nsource_id: 188\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	07de01ee-c98b-4f0b-92b0-d8b78a39d8b8	2015-06-21 16:49:50.631975
541	300	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated\n  AML patients with FLT3 mutations\ndisease_id: 3\nsource_id: 187\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	b692c62f-c9ff-4c58-ac96-18cee2b10911	2015-06-21 16:49:50.661055
542	301	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib\n  and fasudil led to dramatic tumor regression.\ndisease_id: 30\nsource_id: 189\nvariant_id: 57\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	b5e13882-239f-464e-8e98-3b2d6d8848f8	2015-06-21 16:49:50.696997
543	302	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a case study, a patient with anal melanoma harboring a 7 codon duplication\n  in the juxtamambrane region (exon 11) of KIT showed marked response 4 months after\n  imatinib treatment.\ndisease_id: 31\nsource_id: 190\nvariant_id: 67\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	772f0d05-c071-46b3-891f-42a381e15708	2015-06-21 16:49:50.733864
544	303	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib,\n  nilotinib and imatinib.\ndisease_id: 8\nsource_id: 191\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	5af3513d-05e0-4ed9-b6b4-27ea16070350	2015-06-21 16:49:50.777516
545	304	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In GIST tumor cells and patients harboring KIT V654A mutation, SU11248\n  is effective for those that are refractory to imatinib.\ndisease_id: 14\nsource_id: 192\nvariant_id: 73\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55594258-55594258 (T->C)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	671ab97f-2c44-4a4a-a34a-54fcb9b3a9df	2015-06-21 16:49:50.817543
546	305	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\ndisease_id: 8\nsource_id: 193\nvariant_id: 79\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398284-25398284 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	66e49576-94f7-4cac-98a2-5b6bfe261946	2015-06-21 16:49:50.863551
547	306	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G13D mutation is associated with better response to Cetuximab with\n  longer progression-free and overall survival in colorectal patients compared to\n  other KRAS\ndisease_id: 17\nsource_id: 123\nvariant_id: 81\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398281-25398281 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	89adf373-a448-42d1-b04a-454a419e99e8	2015-06-21 16:49:50.896979
548	307	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a randomized clinical trial, patients with MGMT promoter methyaltion\n  benefitted from temozolomide. This benefit was also methylation status dependent,\n  as those without methylation did not see increased survival.\ndisease_id: 19\nsource_id: 194\nvariant_id: 85\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	d5eb4a44-7672-4221-a4f1-b7d3a70cf321	2015-06-21 16:49:50.934775
549	308	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MGMT promoter methylation has been shown to confer sensitivity to alkylating\n  agents, such as carmustine, in glioblastoma cells.\ndisease_id: 19\nsource_id: 139\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	9e05e525-b93f-4ae5-a5f3-200be6043bfc	2015-06-21 16:49:50.966019
550	309	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor,\n  such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may\n  be useful in treating patients with initial resistance to alkylating agents.\ndisease_id: 19\nsource_id: 139\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	4a1fda87-72e3-49cd-b25c-a3010b9725a0	2015-06-21 16:49:51.026818
551	310	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer cell lines with E542K/E545K mutations showed increased\n  sensitivity to CH5132799 than cells with wild-type PIK3CA gene.\ndisease_id: 22\nsource_id: 195\nvariant_id: 103\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	7e43655b-e62b-4651-abbb-0e10e84948d3	2015-06-21 16:49:51.05881
552	311	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rapamycin inhibits transformation induced by mutation in PIK3CA\ndisease_id: 22\nsource_id: 196\nvariant_id: 103\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	38fb37b8-3a20-4a4c-9964-c8abb811b623	2015-06-21 16:49:51.099106
553	312	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer cell lines with E542K/E545K mutations showed increased\n  sensitivity to CH5132799 than cells with wild-type PIK3CA gene.\ndisease_id: 22\nsource_id: 195\nvariant_id: 104\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	49779025-6e5d-4edb-9390-a5bbcdf788bc	2015-06-21 16:49:51.135155
554	313	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rapamycin inhibits transformation induced by mutation in PIK3CA\ndisease_id: 22\nsource_id: 196\nvariant_id: 104\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	5596eb65-77f2-45cd-a757-a2accdcc67c2	2015-06-21 16:49:51.166949
555	314	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer cell lines with H1047R mutation showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\ndisease_id: 22\nsource_id: 195\nvariant_id: 107\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	c3703ad8-58be-4670-b950-f823bf3c509f	2015-06-21 16:49:51.199189
556	315	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rapamycin inhibits transformation induced by mutation in PIK3CA\ndisease_id: 22\nsource_id: 196\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	511feb80-2b52-495d-b383-980306bddee9	2015-06-21 16:49:51.226431
557	316	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Fusion protein degradation is accelerated upon ATRA treatment of acute\n  promyelocytic cells at pharmalogical concentrations.\ndisease_id: 32\nsource_id: 197\nvariant_id: 108\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	7b78546e-6eb2-4f8b-98de-d0c09059b55e	2015-06-21 16:49:51.258663
558	317	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells with PTEN deficiency have been shown to exhibit slowed growth in\n  reponse to PI3K-mTOR inhibitors.\ndisease_id: 24\nsource_id: 198\nvariant_id: 110\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:89717672-89717672 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	2590f40f-f1df-4ff1-b551-572330f5a659	2015-06-21 16:49:51.289233
559	15	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: PTEN Loss-of-Function\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	85a12b32-3975-4555-8032-724414b4d578	2015-06-21 16:49:51.315677
560	318	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Fusion protein degradation is accelerated upon ATRA treatment of acute\n  promyelocytic cells at pharmalogical concentrations.\ndisease_id: 32\nsource_id: 197\nvariant_id: 111\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	a1db55da-fbfe-49cb-95e0-967f5d4da5ef	2015-06-21 16:49:51.346867
561	319	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast tumors in a mouse model with R175H mutation are more responsive\n  to doxorubicin than breast tumors with wild type TP53.\ndisease_id: 22\nsource_id: 199\nvariant_id: 116\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7578406-7578406 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	7a0e6dba-2316-4027-9c32-e6951d58283c	2015-06-21 16:49:51.384484
562	320	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A patient with metastatic bladder cancer that responded well to the mTOR\n  inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del,\n  p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations\n  also had mild responses to everolimus.\ndisease_id: 33\nsource_id: 200\nvariant_id: 124\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	90e137ab-a449-4c8a-8531-52d6d72f9404	2015-06-21 16:49:51.418095
563	321	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity\n  to everolimus than their wild-type counterparts.\ndisease_id: 18\nsource_id: 200\nvariant_id: 126\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	17e82c1b-9b05-406a-a5e0-606797f16a6f	2015-06-21 16:49:51.452755
564	322	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with D835 mutation of FLT3 did not have worse overall survival\n  than patients with wild type FLT3\ndisease_id: 3\nsource_id: 152\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	4e06a059-82f9-44ae-a00b-d3c087c2e04e	2015-06-21 16:49:51.486346
565	323	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The complete remission and overall survival rates in patients with IDH1\n  R132 mutation is not significantly different from those who do not have this mutation\ndisease_id: 3\nsource_id: 201\nvariant_id: 58\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	aa37535a-a378-48fe-88de-fcd4308263b4	2015-06-21 16:49:51.516295
566	324	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\ndisease_id: 3\nsource_id: 159\nvariant_id: 59\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:209113113-209113113 (G->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	a77c71aa-f7c2-4566-95cc-a4765e72396e	2015-06-21 16:49:51.545008
567	325	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\ndisease_id: 3\nsource_id: 159\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:209113112-209113112 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	905f4057-e479-451c-bc4c-f4a4f2687464	2015-06-21 16:49:51.574633
568	326	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\ndisease_id: 3\nsource_id: 159\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:209113112-209113112 (C->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	e60a87b6-bdfa-46cd-bacb-de0336ff289d	2015-06-21 16:49:51.603136
569	327	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with IDH2 mutations such as R140Q/L have event free survival\n  and overall survival similar to those with wild-type IDH2\ndisease_id: 3\nsource_id: 202\nvariant_id: 62\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	8c27509f-e846-49e5-a6ca-9791b5906bbb	2015-06-21 16:49:51.634593
570	328	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with IDH2 R140Q/L mutation, the presence of mutation does\n  not impact overall survival or disease free survival.\ndisease_id: 3\nsource_id: 203\nvariant_id: 62\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	04648171-d49c-4324-8b0a-daa3832b36e4	2015-06-21 16:49:51.664111
571	329	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients suffering from MDS, the presence of IDH2 mutation such as\n  R140Q/L do not confer prognostic value (overall survival)\ndisease_id: 34\nsource_id: 204\nvariant_id: 62\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	b98bffaf-31fd-4e3b-b64a-53f089f78a46	2015-06-21 16:49:51.69463
572	330	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: R140Q mutation in IDH2 does not have prognostic value in patients with\n  MDS.\ndisease_id: 34\nsource_id: 205\nvariant_id: 62\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	7d806f4c-2adf-42b2-a5ba-f31235d8b721	2015-06-21 16:49:51.728209
573	331	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with IDH2 mutations such as R172K have event free survival\n  and overall survival similar to those with wild-type IDH2.\ndisease_id: 3\nsource_id: 202\nvariant_id: 63\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:90631838-90631838 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	f1e29c04-4996-4a76-8728-438e13b3898a	2015-06-21 16:49:51.755346
574	332	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients suffering from MDS, the presence of IDH2 mutations such as\n  R172K do not confer prognostic value (overall survival)\ndisease_id: 34\nsource_id: 204\nvariant_id: 63\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:90631838-90631838 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	0273d172-7b20-4b5d-be14-70b467301a18	2015-06-21 16:49:51.784186
575	333	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutations were identified in morphologically benign, incidentally\n  discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients\n  indicating is not a prognostic marker for GIST\ndisease_id: 14\nsource_id: 206\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	4d31c515-d509-4eaf-9db9-a89774880e5e	2015-06-21 16:49:51.812253
576	334	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutations detected in 5/60 patients showed no prognostic significance\n  in patients with small cell lung cancer\ndisease_id: 23\nsource_id: 207\nvariant_id: 66\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	30e78486-6371-4c76-9bf5-49a5f812c109	2015-06-21 16:49:51.852013
577	335	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutations detected in 5/60 patients showed no prognostic significance\n  in patients with small cell lung cancer\ndisease_id: 23\nsource_id: 207\nvariant_id: 71\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	221f463a-d41f-488b-ba01-1f315b224757	2015-06-21 16:49:51.88612
578	336	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with stage III colorectal cancer undergoing chemotherapy,\n  KRAS G12 mutation did not impact overall or disease free survival.\ndisease_id: 17\nsource_id: 208\nvariant_id: 76\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	e2b4e9c9-c49c-48d5-a7fa-50c86c93edd4	2015-06-21 16:49:51.919857
579	337	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no strong association between KRAS mutation status and overall\n  or progression free survival in patients with NSCLC\ndisease_id: 8\nsource_id: 209\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	2a120266-5155-421c-8335-006741b5a7d5	2015-06-21 16:49:51.954655
702	44	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37881000'\nstop:\n- \n- '37881000'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	eb198ac9-bac0-482a-821b-eb9785960cce	2015-06-21 16:52:22.863294
580	338	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with AML, those who harbor Q157P/R mutation of U2AF do not\n  show statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\ndisease_id: 3\nsource_id: 154\nvariant_id: 127\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	660e83b6-b175-4903-a196-7b61a497aeec	2015-06-21 16:49:51.984424
581	339	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with MDS, those who harbor Q157P/R mutation of U2AF do not\n  show statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\ndisease_id: 34\nsource_id: 154\nvariant_id: 127\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	9555b171-bec8-4e55-887f-fed1f649db28	2015-06-21 16:49:52.016221
582	340	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with AML, complete remission rates are not different between\n  patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\ndisease_id: 3\nsource_id: 154\nvariant_id: 128\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	409d1d53-95cf-4ba8-b973-546655dc5c79	2015-06-21 16:49:52.044193
583	341	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with AML, complete remission rates are not different between\n  patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\ndisease_id: 34\nsource_id: 154\nvariant_id: 128\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	a85ac9d6-0c09-4ef6-811e-f87b6342df35	2015-06-21 16:49:52.084858
584	342	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Three studies have found no significant survival impact for CCND1 in\n  lung cancer.\ndisease_id: 8\nsource_id: 210\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	5e4493b7-a25a-4dac-b301-a196f638484e	2015-06-21 16:49:52.122364
585	343	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PTEN nonsense mutations, including R233*, have been shown to be inactivating\n  and loss-of-function, but do not have prognostic value in glioblastoma multiforme\n  patients.\ndisease_id: 19\nsource_id: 211\nvariant_id: 110\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:89717672-89717672 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	a2f7a052-6e53-4cb6-8d33-15ab6c031dd6	2015-06-21 16:49:52.155635
586	344	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BCR-ABL inhibitors such as imatinib have lead to significantly improved\n  prognosis, response rate, overall survival, and patient outcome in CML patients\n  compared to previous therapeutic regimens.\ndisease_id: 4\nsource_id: 156\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	a18ed877-bafa-4646-9b09-5898312ec31b	2015-06-21 16:49:52.188797
587	345	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\ndisease_id: 28\nsource_id: 212\nvariant_id: 25\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	5f96eef4-b6b5-4ee3-8e04-02f344e2cb0e	2015-06-21 16:49:52.220714
588	346	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\ndisease_id: 28\nsource_id: 212\nvariant_id: 26\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	b78c9b27-dfcb-4e91-80c6-17b6ce907527	2015-06-21 16:49:52.254694
589	347	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Median survival of patients with EGFR L858R mutation is better than those\n  with wild type EGFR.\ndisease_id: 8\nsource_id: 213\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55259514-55259515 (CT->AG)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	7f46d752-c93c-447e-b7b1-e76bb4b511ff	2015-06-21 16:49:52.317583
590	348	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic\n  for better survival compared to patients who harbor wild type mutation.\ndisease_id: 35\nsource_id: 214\nvariant_id: 58\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	0f12ab80-4bfb-4e0d-a387-5b6ede2a6a95	2015-06-21 16:49:52.348276
591	349	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with glioblastoma, those harboring IDH1 R132 mutation has\n  higher overall survival compared to those who do not have IDH1 mutation\ndisease_id: 36\nsource_id: 215\nvariant_id: 58\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	5969f99d-3887-4443-98d1-9dadb4da443c	2015-06-21 16:49:52.377823
592	350	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML, patients with IDH2 R140K mutation have improved overall survival\n  compared to those with wild-type IDH2\ndisease_id: 3\nsource_id: 216\nvariant_id: 62\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	02e271fe-bb5e-4f7e-8c2f-7d3890d2665b	2015-06-21 16:49:52.40692
593	351	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G13D mutation is associated with better response to Cetuximab with\n  longer progression-free and overall survival in colorectal patients compared to\n  other KRAS\ndisease_id: 17\nsource_id: 123\nvariant_id: 81\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398281-25398281 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	4c41eb73-9f7a-4e6d-a89a-bc9fdf396688	2015-06-21 16:49:52.44326
594	352	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with SF3B1 mutations had a statistically significant longer\n  overall survival as well as event free-survival. Both before and after adjustment\n  for age, karyotype and sex.\ndisease_id: 34\nsource_id: 217\nvariant_id: 114\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:198267359-198267359 (C->A)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	235dbf2d-7df9-4b5c-aff7-5203ee94a41a	2015-06-21 16:49:52.473536
595	353	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with SF3B1 mutations had a statistically significant longer\n  overall survival as well as event free-survival. Both before and after adjustment\n  for age, karyotype and sex.\ndisease_id: 34\nsource_id: 217\nvariant_id: 115\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:198266834-198266834 (T->C)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	4054d001-ddf2-4770-8129-0f479a63e271	2015-06-21 16:49:52.511554
596	354	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Inceased copy number of CCND1 is associated with poorer overall survival.\ndisease_id: 8\nsource_id: 210\nvariant_id: 18\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	e6df0394-7b88-4990-aa4e-967c8bd71e96	2015-06-21 16:49:52.541777
597	355	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Three studies have associated CCND1 expression with poorer survival.\ndisease_id: 8\nsource_id: 210\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	a2aab54f-9591-4504-acb3-5887768e6dec	2015-06-21 16:49:52.570016
598	356	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In ER-positive breast cancer patients, Cyclin D1 overexpression is associated\n  with shorter overall survival and increased metastasis.\ndisease_id: 22\nsource_id: 157\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	6e4285bf-b3a4-4832-8878-74ae63c824c7	2015-06-21 16:49:52.604107
599	357	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D1 overexpression is associated with shorter overall survival\n  and increased metastasis in head and neck squamous cell carcinoma.\ndisease_id: 37\nsource_id: 218\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	d892775f-7721-452e-80fd-496b1b6bc2bc	2015-06-21 16:49:52.639472
600	358	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D1 overexpression is associated with shorter overall survival\n  and increased metastasis in mantle cell lymphoma.\ndisease_id: 38\nsource_id: 219\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	3d556a05-f467-4ed3-8fb2-a3c5a471cea8	2015-06-21 16:49:52.675396
601	359	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with poor prognosis in gastric\n  cancers.\ndisease_id: 21\nsource_id: 220\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	fb6bc409-8293-443c-bec0-a2c91bed536f	2015-06-21 16:49:52.706301
602	360	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with the increased depth of cancer\n  invasion in gastric cancers.\ndisease_id: 21\nsource_id: 220\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	4df0c86e-4010-4e88-8014-e53ec13379f2	2015-06-21 16:49:52.733994
603	361	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with the presence pf lymph node\n  metastasis in gastric cancers.\ndisease_id: 21\nsource_id: 220\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	1a2fe4af-45d5-4ed7-81d5-06f7263a434d	2015-06-21 16:49:52.762341
604	362	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with vascular invasion by cancer\n  cells in gastric cancers.\ndisease_id: 21\nsource_id: 220\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	95dee29a-4840-4717-a11a-59f255951ea3	2015-06-21 16:49:52.796915
605	363	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with high expression of total cyclin E and low-molecular weight\n  cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard\n  ratio for these patients in 13.3 times higher than those with normal cyclin E levels.\ndisease_id: 22\nsource_id: 221\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	f8881783-701b-4e21-af1e-35924072db5f	2015-06-21 16:49:52.825822
606	364	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced\n  stage gastric carcinoma.\ndisease_id: 39\nsource_id: 222\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	4ee72b8f-651a-48a7-b44c-3f937296fc09	2015-06-21 16:49:52.856452
607	365	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\ndisease_id: 39\nsource_id: 222\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	12539f31-78a9-49b9-8182-6591757def4c	2015-06-21 16:49:52.883902
703	45	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37881332'\nstop:\n- \n- '37881332'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	3f096b38-6284-4115-bc1f-bdf4d94dedfa	2015-06-21 16:52:22.879594
704	48	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '6'\nstart:\n- \n- '152419923'\nstop:\n- \n- '152419923'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- G\n	2	\N	\N	ca5dbaaf-2ce9-45b1-a949-0b18b02539c5	2015-06-21 16:52:22.896637
608	366	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\ndisease_id: 23\nsource_id: 222\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	2ed1c54e-4447-4e33-a146-54f62747e2e7	2015-06-21 16:49:52.91283
609	367	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A)\n  promoter hypermethylation is associated with shorter overall survival.\ndisease_id: 8\nsource_id: 223\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	6ad39d69-91d4-46f8-b66b-6992206a1ca5	2015-06-21 16:49:52.942829
610	368	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A)\n  promoter hypermethylation is associated with shorter time to recurrence.\ndisease_id: 8\nsource_id: 223\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	88bfe069-c3d8-4f75-b185-2ed2626b39bd	2015-06-21 16:49:52.973929
611	369	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients with T790M and another activating mutations, their\n  progression free survival is shorter compared to those who do not possess T790M\n  mutation.\ndisease_id: 8\nsource_id: 224\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	1acfdeed-c88d-454c-8ad0-d7726e50e0ef	2015-06-21 16:49:53.004407
612	370	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NSCLC harboring EGFR T790M mutation have statistically\n  worse overall survival compared to patients with L858R or other exon 19 activating\n  mutations.\ndisease_id: 8\nsource_id: 225\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	8096a9a6-99a6-4357-8432-30d820930638	2015-06-21 16:49:53.037928
613	371	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with internal tandem repeat of FLT3 has worse overall survival\n  than patients with wild type FLT3\ndisease_id: 3\nsource_id: 152\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	fb4a2267-0094-41c6-8a71-5978dc67ea41	2015-06-21 16:49:53.072395
614	372	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads\n  to poor survival and inferior disease free survival in patients with AML.\ndisease_id: 3\nsource_id: 226\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	1ce9ddfe-a083-476c-9844-7c561f5f8380	2015-06-21 16:49:53.101241
615	373	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML patients, FLT3-TKD mutation was associated with poorer disease\n  free survival compared to patients with wild type FLT3\ndisease_id: 3\nsource_id: 227\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	93ffc879-fd18-4d8d-bb05-37e4c08bcc99	2015-06-21 16:49:53.13372
616	374	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients suffering from myelodysplastic syndromes\n  is associated with worse overall survival\ndisease_id: 40\nsource_id: 228\nvariant_id: 58\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	7628b0d4-0fde-4ca7-83fc-3684de86b995	2015-06-21 16:49:53.170812
617	375	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML, patients with an IDH2 R172K mutation have worse overall survival\n  compared to those with wild-type IDH2\ndisease_id: 3\nsource_id: 216\nvariant_id: 63\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:90631838-90631838 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	1e29e96c-3881-4601-b288-2904bd5edf65	2015-06-21 16:49:53.198439
618	376	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutation is associated with worse overall and cause-specific prognosis\n  in patients with GIST compared to patients with wildtype KIT\ndisease_id: 14\nsource_id: 160\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	2c46ca9b-81b8-4b37-a9a1-c588aead7304	2015-06-21 16:49:53.225976
619	377	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of KRAS mutations in MM patients is prognostic for shorter\n  overall and progression free survival\ndisease_id: 41\nsource_id: 229\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	c3d38c85-3e0a-4517-a67c-7325a3ddcc38	2015-06-21 16:49:53.261234
620	378	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A meta-analysis showed KRAS mutation is associated with worse outcome\n  in patients with NSCLC\ndisease_id: 8\nsource_id: 230\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	09929318-0df8-48d4-b13f-d3bc8d1aad43	2015-06-21 16:49:53.289669
621	379	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of KRAS mutations in MM patients is prognostic for shorter\n  overall and progression free survival\ndisease_id: 42\nsource_id: 229\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	c3086855-4c54-440c-91c0-3390f5539a3b	2015-06-21 16:49:53.318966
705	49	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '6'\nstart:\n- \n- '152419922'\nstop:\n- \n- '152419922'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- A\n	2	\N	\N	50630570-726c-42d4-a411-7d35ed473424	2015-06-21 16:52:22.913778
622	380	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including D1643H, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 8\nsource_id: 231\nvariant_id: 135\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	4bd6f66d-2cbd-4068-8a62-09b0cdf658db	2015-06-21 16:49:53.349415
623	381	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including R2328W, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 8\nsource_id: 231\nvariant_id: 136\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	2579f339-6c5a-42c9-9f38-433ec128ece9	2015-06-21 16:49:53.379728
624	382	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including S2275fs, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 8\nsource_id: 231\nvariant_id: 138\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	dfa0e785-499b-40f8-bc61-d85a55940fb7	2015-06-21 16:49:53.411829
625	383	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including V2444fs, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 8\nsource_id: 231\nvariant_id: 137\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	5d5d96a9-eb6c-4610-8425-b7acc089bbdf	2015-06-21 16:49:53.445419
626	384	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not\n  either alone, is prognostic for worse overall survival and cancer specific survival.\ndisease_id: 11\nsource_id: 232\nvariant_id: 103\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	ec98c220-11e0-40ce-b659-2e7d59daeaa0	2015-06-21 16:49:53.476833
627	385	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of exon 9 (such as E542K) or exon 20 mutations were associated\n  with poorer survival compared to those with wild-type PIK3CA mutation.\ndisease_id: 11\nsource_id: 176\nvariant_id: 103\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	cd7789fc-44c1-4ea4-b2cd-01b453977606	2015-06-21 16:49:53.532346
628	386	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not\n  either alone, is prognostic for worse overall survival and cancer specific survival.\ndisease_id: 11\nsource_id: 232\nvariant_id: 104\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	3c749807-a057-401f-ac44-5fe304f72a98	2015-06-21 16:49:53.562858
629	387	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of exon 9 (such as E545K) or exon 20 mutation may result\n  in poorer survival compared to those with wild-type PIK3CA mutation.\ndisease_id: 11\nsource_id: 176\nvariant_id: 104\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	b3ee0769-49a2-4f91-8e62-bbd81dba264b	2015-06-21 16:49:53.592623
630	388	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of exon 20 (such as H1047R) or exon 9 mutation may result\n  in poorer survival compared to those with wild-type PIK3CA mutation.\ndisease_id: 11\nsource_id: 176\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	51c13694-cc56-40df-b8da-646f94b915e4	2015-06-21 16:49:53.622466
631	389	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R175H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 22\nsource_id: 233\nvariant_id: 116\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7578406-7578406 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	d93a55ff-9189-4e73-a066-52691cbb6941	2015-06-21 16:49:53.653649
632	390	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R248Q mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 22\nsource_id: 233\nvariant_id: 117\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577538-7577538 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	0f44e288-f72e-40f4-8feb-15d2928f3132	2015-06-21 16:49:53.685321
633	391	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R248 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\ndisease_id: 22\nsource_id: 234\nvariant_id: 117\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577538-7577538 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	98a849a3-8a14-4fc5-80d9-02b20802a4e2	2015-06-21 16:49:53.719146
634	392	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring R248W mutation, the prognosis is\n  worse than any other hotspot TP53 mutation, as well as worse than patients with\n  wild type TP53.\ndisease_id: 22\nsource_id: 233\nvariant_id: 118\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577539-7577539 (G->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	36653824-3749-4e02-9765-32f998b52ad1	2015-06-21 16:49:53.747878
706	50	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '6'\nstart:\n- \n- '152419923'\nstop:\n- \n- '152419923'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- C\n	2	\N	\N	0d7aed71-bc1a-45c9-a8d9-8a7bb3dde052	2015-06-21 16:52:22.931194
635	393	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in conserved\n  regions such as R249 are prognostic for a worse overall survival compared to those\n  harboring wild-type TP53.\ndisease_id: 22\nsource_id: 234\nvariant_id: 119\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577535-7577535 (C->G)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	900a9f78-b55e-4bf2-b70e-f8dad932ab16	2015-06-21 16:49:53.781841
636	394	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in conserved\n  regions such as R249 are prognostic for a worse overall survival compared to those\n  harboring wild-type TP53.\ndisease_id: 22\nsource_id: 234\nvariant_id: 120\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577536-7577536 (T->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	ac570347-a65b-4eac-a147-70b45f63ed5e	2015-06-21 16:49:53.809661
637	395	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R273C mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 22\nsource_id: 233\nvariant_id: 121\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577121-7577121 (G->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	6b8fe3dd-59a3-4a8a-b54f-2689966fcb43	2015-06-21 16:49:53.837522
638	396	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients  harboring TP53 mutation, mutations in DNA\n  contact regions such as R273 are prognostic for a worse relapse-free survival compared\n  to other non-silent mutations in TP53.\ndisease_id: 22\nsource_id: 234\nvariant_id: 121\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577121-7577121 (G->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	01bef1d9-1c2e-437a-a31e-11c164624eaf	2015-06-21 16:49:53.868261
639	397	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R273H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 22\nsource_id: 233\nvariant_id: 122\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577120-7577120 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	1768fc35-2ff7-4d04-847d-f9722ea67d52	2015-06-21 16:49:53.895446
640	398	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\ndisease_id: 22\nsource_id: 234\nvariant_id: 122\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577120-7577120 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	56679c32-7baf-426b-8d48-a3f4b04f0cc8	2015-06-21 16:49:53.932513
641	399	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast tumors with R175H mutations are more responsive to doxorubicin\n  than breast tumors with wild type TP53.\ndisease_id: 22\nsource_id: 234\nvariant_id: 123\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	e4d34019-8cc5-48e0-80e5-0550adbf9707	2015-06-21 16:49:53.96545
642	400	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: After adjusting for age and cancer stage, presence of U2AF mutation such\n  as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.\ndisease_id: 34\nsource_id: 235\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	bd868042-38d6-482f-96ac-169d5fbedce7	2015-06-21 16:49:53.99625
643	401	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: After adjusting for age and cancer stage, the presence of U2AF mutations\n  such as S34Y/F are prognostic for poorer survival outcomes in patients with MDS.\ndisease_id: 34\nsource_id: 235\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	6d91b098-12a7-42a9-9efe-556ad90a3dd2	2015-06-21 16:49:54.02841
644	402	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\ndisease_id: 8\nsource_id: 236\nvariant_id: 79\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398284-25398284 (C->T)\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \n	1	\N	\N	601c672b-c22c-4859-9bb2-073f848d2ef4	2015-06-21 16:49:54.060191
645	403	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase\n  inhibitors Imatinib and GW-2580\ndisease_id: 43\nsource_id: 237\nvariant_id: 30\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	c79fce0a-2dbc-4aba-8505-48a7d737927a	2015-06-21 16:49:54.098357
646	404	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase\n  inhibitors Imatinib and GW-2580\ndisease_id: 43\nsource_id: 237\nvariant_id: 84\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	0cbf4b4f-40b0-4d2b-8e82-140a36f1baa1	2015-06-21 16:49:54.135689
647	405	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA\n  resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\ndisease_id: 44\nsource_id: 238\nvariant_id: 31\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	469da3a7-1491-4fda-8886-7790629cba8f	2015-06-21 16:49:54.172849
648	406	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA\n  resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\ndisease_id: 44\nsource_id: 238\nvariant_id: 109\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	163140c4-9c58-4528-943d-e1a4676a6673	2015-06-21 16:49:54.204765
649	407	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The t(14;18) transloaction of BCL2 is shown to be associated with germinal-center\n  B-cell–like diffuse large-B-cell lymphomas.\ndisease_id: 45\nsource_id: 239\nvariant_id: 151\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	9021c84e-68bb-4d04-ad7f-81f4cd15ea6d	2015-06-21 16:49:54.236552
650	408	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Amplification of FOXP1 is shown to be associated with ABC DLBCL\ndisease_id: 45\nsource_id: 240\nvariant_id: 153\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	e3cc727a-f0ee-45c8-a82b-dea7a2c7c037	2015-06-21 16:49:54.28023
651	409	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Amplification of REL is shown to be associated with germinal-center B-cell–like\n  diffuse large-B-cell lymphomas.\ndisease_id: 45\nsource_id: 239\nvariant_id: 154\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	243ad426-6938-41f0-93da-a1e89ab951b8	2015-06-21 16:49:54.309286
652	410	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Amplification of REL is shown to be associated with germinal-center B-cell–like\n  diffuse large-B-cell lymphomas.\ndisease_id: 45\nsource_id: 239\nvariant_id: 154\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	802508ee-62d3-45f7-be18-0dbeca4b57b8	2015-06-21 16:49:54.33694
653	411	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In non-APL patients with AML, those who had mutations in RUNX1 and were\n  treated with allogeneic hematopoietic stem-cell transplantation had longer relapse-free\n  survival than those treated with conventional post remmision treatment.\ndisease_id: 3\nsource_id: 241\nvariant_id: 155\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	6dbf6de1-a2e4-4ae9-83f6-2b6127fcf56a	2015-06-21 16:49:54.367097
654	412	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: OSI-906, an IGF1R/INSR inhibitor, does not have a significant effect\n  on chemotherapy resistant castration-resistant prostate cancer cell lines.\ndisease_id: 46\nsource_id: 242\nvariant_id: 156\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	8b721391-87fb-4351-8eab-c983618c1893	2015-06-21 16:49:54.406751
655	413	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NSCLC patients with exon 19 deletions in EGFR had longer overall survival\n  than patients with an L858R mutation when treated with gefitinib and erlotinib (38\n  versus 17 months; P = 0.04).\ndisease_id: 8\nsource_id: 243\nvariant_id: 133\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	7e8ba836-df58-4d9a-9ef9-f88279483fce	2015-06-21 16:49:54.437246
656	414	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: OSI-906, an IGF1R/INSR inhibitor, sensitizes cell lines to docetaxel\n  and cabazitaxel, resulting in better overall survival in chemotherapy resistant\n  CRPC cell lines.\ndisease_id: 46\nsource_id: 244\nvariant_id: 156\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \n	1	\N	\N	eebac746-91a5-4650-a0c8-fe8e4996759d	2015-06-21 16:49:54.476508
657	415	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a cohort of 783 young adult patients with acute myeloid leukemia,\n  those with TET2 mutations did not have significantly different rates of complete\n  recovery, refractory disease, or hypoplastic death.\ndisease_id: 3\nsource_id: 245\nvariant_id: 157\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	dbef216b-4468-4237-adbc-61013b6743c2	2015-06-21 16:49:54.516277
658	416	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with ovarian carcinomas, those with Aurora kinase A expression\n  (as measured by immunohistochemistry) had increased progression-free and overall\n  survival compared to those without expression.\ndisease_id: 47\nsource_id: 246\nvariant_id: 158\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	7f2bd5fe-d13f-4760-b474-c37ddda11d16	2015-06-21 16:49:54.560231
659	417	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with myelodysplastic syndromes, those with TET2 mutations\n  have been shown to have better overall survival than patients without TET2 mutations.\ndisease_id: 34\nsource_id: 247\nvariant_id: 157\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	a8def43c-ddd4-46ac-b9d4-c67de9a2e29c	2015-06-21 16:49:54.595891
660	418	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with renal clear cell carcinoma, patients with VHL mutations\n  had longer cancer-specific survival.\ndisease_id: 48\nsource_id: 248\nvariant_id: 160\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	d40cbc44-3774-4be8-8e6b-05e79dbfa7cf	2015-06-21 16:49:54.628402
707	59	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '209113113'\nstop:\n- \n- '209113113'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	e9a9b75b-3320-417e-8242-39872f08ff34	2015-06-21 16:52:22.950053
661	419	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with cytogenetically normal acute myeloid leukemia, those\n  in the intermediate-I risk group with TET2 mutations had higher rates of complete\n  remission.\ndisease_id: 3\nsource_id: 245\nvariant_id: 157\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \n	1	\N	\N	515e98d8-b5a9-475a-adcf-6ca7ff59e449	2015-06-21 16:49:54.663301
662	420	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In renal clear cell carcinoma, patients with PBRM1 mutations did not\n  have a significantly different cancer specific survival.\ndisease_id: 48\nsource_id: 249\nvariant_id: 161\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	8ca66ae2-37fd-46e0-a0a9-69a6ea690450	2015-06-21 16:49:54.694133
663	421	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with cytogenetically normal acute myeloid leukemia, those\n  in the intermediate-I risk group with TET2 mutations did not have significantly\n  different rates of event-free survival, complete remission, disease-free survival,\n  or overall survival.\ndisease_id: 3\nsource_id: 250\nvariant_id: 157\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	a32fc364-348e-499f-b0c5-be2dbff42ecb	2015-06-21 16:49:54.726784
664	422	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In renal clear cell carcinoma, patients with VHL mutations did not have\n  a significant impact on cancer specific survival.\ndisease_id: 48\nsource_id: 249\nvariant_id: 160\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 6\ndeleted: false\ndeleted_at: \n	1	\N	\N	e16345f1-db8d-4adf-9907-24d88a84fb53	2015-06-21 16:49:54.781468
665	423	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with Diffuse Large B-Cell Lymphoma, those with increased\n  levels of BCL2 were shown to have decreased overall, disease-free, and relapse-free\n  survival.\ndisease_id: 45\nsource_id: 251\nvariant_id: 152\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	ee1ec959-7a6c-4da2-9653-96c866dd044f	2015-06-21 16:49:54.81937
666	424	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with Diffuse Large B-Cell Lymphoma, those with increased\n  levels of BCL2 were shown to have decreased OS, DFS, and RFS.\ndisease_id: 45\nsource_id: 251\nvariant_id: 152\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	357dd958-5d24-465a-926c-50a6b6c8aa69	2015-06-21 16:49:54.851806
667	425	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In myelodysplastic syndrome, patients with point mutations in EZH2 have\n  worse overall survival that their wild-type counterparts.\ndisease_id: 49\nsource_id: 252\nvariant_id: 163\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	3c2cb430-3e0f-46a1-bb74-7466de8200b4	2015-06-21 16:49:54.888609
668	426	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients classified as intermediate-risk via cytogenetics, TET2 mutations\n  have been shown to be correlated with poor prognosis.\ndisease_id: 3\nsource_id: 253\nvariant_id: 157\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	8a3e9e91-f2bd-4042-97ed-9fd807929b71	2015-06-21 16:49:54.922345
669	427	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with cytogenetically normal acute myeloid leukemia, those\n  in the ELN favorable-risk group with TET2 mutations had shorter event-free survival,\n  lower rates of complete remission, and shorter disease-free survival.\ndisease_id: 3\nsource_id: 250\nvariant_id: 157\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	93cca18e-5f8d-4c89-94e6-4a1c95218a99	2015-06-21 16:49:54.951062
670	428	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with renal clear cell carcinoma, patients with loss of function\n  VHL mutations had shorter tumor-specific survival.\ndisease_id: 48\nsource_id: 254\nvariant_id: 159\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	184684ad-3560-4561-990a-af14dadbacb9	2015-06-21 16:49:54.981123
671	429	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In non-APL patients with AML, those who had mutations in RUNX1 had higher\n  rates of refractory disease than those without.\ndisease_id: 3\nsource_id: 241\nvariant_id: 155\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	98d82b83-7475-4c15-9f9f-12f78a294a49	2015-06-21 16:49:55.016256
672	430	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In non-APL patients with AML, those who had mutations in RUNX1 had shorter\n  event-free survival and relapse-free survival relative to wild-type.\ndisease_id: 3\nsource_id: 241\nvariant_id: 155\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \n	1	\N	\N	1d663132-08c6-4368-ac88-ac8eeb258c6d	2015-06-21 16:49:55.043922
673	431	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with Ewing's sarcoma, those with type 1 EWS-FLI1 fusions\n  had longer overall survival than those with other types of EWS-FLI1 fusions.\ndisease_id: 50\nsource_id: 255\nvariant_id: 164\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \n	1	\N	\N	31d14a4a-a52e-48e1-8dcd-29ca98095bc3	2015-06-21 16:49:55.076816
674	1	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'While imatinib has shown to be incredibly successful in treating philadelphia chromosome\n  positive CML, patients that have shown primary or secondary resistance to the drug\n  have been observed to harbor T315I and E255K ABL kinase domain mutations. These\n  mutations, among others, have been observed both in primary refractory disease and\n  acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations\n  have also been shown to confer resistance to imatinib. '\n	2	\N	\N	efc87d20-19d8-4ffa-82f3-96abde5991d1	2015-06-21 16:49:55.148063
675	11	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset\n  of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase\n  the activity of the ALK oncogene relative to normal cells. While EML4 is the most\n  common fusion partner, other 5'' partners have been observed. The EML4-ALK fusion\n  has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. '\n	2	\N	\N	8ab00a4e-01bd-43d2-99dc-88193bf44329	2015-06-21 16:49:55.163373
676	12	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine\n  kinase inhibitors have been developed to target EGFR pathway activity. One such\n  inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting.\n  However, the T790M missense mutation has shown to confer resistance to this inhibitor\n  in cell lines and case studies.  '\n	2	\N	\N	bdc7920d-881c-434a-bf6c-2b8be5eb35d8	2015-06-21 16:49:55.175759
677	3	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'The ALK oncogene has long been considered a driving factor in non-small cell lung\n  cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be\n  effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance\n  to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive\n  this resistance. '\n	2	\N	\N	d64fa083-7855-4481-bea7-9625ced1edbf	2015-06-21 16:49:55.18592
678	5	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants\n  have also been observed and studied to a lesser degree. At first approximation,\n  many of these variants seem to behave similarly to V600E, and treatment with dabrafenib\n  has been shown to be effective. '\n	2	\N	\N	8373d61d-7942-43a1-aab0-71a47d456a08	2015-06-21 16:49:55.196491
679	14	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'HER2-positive breast cancer is widely recognized as a molecular subtype of breast\n  cancer. While amplification of ERBB2 is a common mechanism for activation of the\n  pathway, missense mutations have also demonstrated activation potential. These activating\n  missense mutations also confer senstivity to the targeted therapeutic neratinib. '\n	2	\N	\N	77dde3ae-4d89-497d-8b23-1583ca59dca9	2015-06-21 16:49:55.206537
680	13	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone\n  therapy has been widely effective in treating the disease, however sequencing of\n  resistant patients has uncovered a number of missense mutations in mediating this\n  resistance. Many of these mutations lie within the ligand-binding domain, and contribute\n  to constitutive activity of the receptor. This has lead to the development of estrogen\n  receptor degrading agents such as fulvestrant, which have shown early efficacy in\n  clinical trials.  '\n	2	\N	\N	6466dc56-3394-4958-91b7-e94087f36359	2015-06-21 16:49:55.216569
681	8	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'FGFR fusions have been demonstrated across many cancer types to have oncogenic\n  potential. FGFR2 and FGFR3 have been shown to be fused to many 3'' partners, which\n  may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2\n  fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib\n  and pazopanib have shown efficacy in case studies.  '\n	2	\N	\N	7d591a54-efc6-40b5-9965-0c0a13fb0297	2015-06-21 16:49:55.228578
682	7	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\n	2	\N	\N	9e30514c-50b7-4abf-9e31-07616f64cf2b	2015-06-21 16:49:55.238574
683	15	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'PTEN loss is a common event in breast cancer, as well as others. The most common\n  single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell\n  lines harboring this mutation behave similarly to cell lines harboring larger insertion\n  or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded\n  growth. '\n	2	\N	\N	fdeeba45-656a-4c2d-a2a6-138f84950515	2015-06-21 16:49:55.24978
684	6	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment\n  of these patients with the targeted therapeutic motesanib has shown to be effective.\n  However, the missense mutations C634W and M918T have shown to confer motesanib resistance\n  in cell lines. '\n	2	\N	\N	3d7d85bb-4998-4325-b5a8-3c9d593930ca	2015-06-21 16:49:55.259773
685	9	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types.\n  Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor\n  everolimus has seen significant anti-tumorigenic activity. '\n	2	\N	\N	5cbf419a-452a-46c0-9e89-fd637e1610ec	2015-06-21 16:49:55.271006
686	10	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'BRCA germline variants have long been recognized as important potential predictors\n  of breast and ovarian cancer risk. '\n	2	\N	\N	db139772-8966-4f24-8d68-ad44f880b4ef	2015-06-21 16:49:55.282381
687	4	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '14'\nstart:\n- \n- '105246551'\nstop:\n- \n- '105246551'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	f0fb8b4d-bea6-4295-b4b7-012ec9ac2b0f	2015-06-21 16:52:22.649994
688	8	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '29443695'\nstop:\n- \n- '29443695'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	6a29b271-6b05-45c3-a911-8786f4b35a2f	2015-06-21 16:52:22.670009
689	9	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '29432664'\nstop:\n- \n- '29432664'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	af165f02-bfa6-43a6-88b6-d97e01c04107	2015-06-21 16:52:22.682244
690	10	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- X\nstart:\n- \n- '47426121'\nstop:\n- \n- '47426121'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- G\n	2	\N	\N	83ae58b5-21aa-465d-a035-a7c42740a8a3	2015-06-21 16:52:22.693854
691	11	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453135'\nstop:\n- \n- '140453136'\nreference_bases:\n- \n- CA\nvariant_bases:\n- \n- AT\n	2	\N	\N	6f19f047-a2d4-43c5-890e-9f825e31bf18	2015-06-21 16:52:22.707136
692	12	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453136'\nstop:\n- \n- '140453136'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- T\n	2	\N	\N	f00537be-7b09-488d-b51b-8c5818ed7a9c	2015-06-21 16:52:22.719592
693	15	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453137'\nstop:\n- \n- '140453137'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	923d663d-6c0c-40ed-9a28-0329e801ae2d	2015-06-21 16:52:22.734151
716	81	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '12'\nstart:\n- \n- '25398281'\nstop:\n- \n- '25398281'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	5998efd4-540f-4f39-a466-998eea8b9851	2015-06-21 16:52:23.09119
717	82	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '15'\nstart:\n- \n- '66729162'\nstop:\n- \n- '66729162'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	1d139ad7-e0ce-4e2f-bbe4-8b0245da4d14	2015-06-21 16:52:23.103687
718	83	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '15'\nstart:\n- \n- '66727451'\nstop:\n- \n- '66727451'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- C\n	2	\N	\N	afa7aa05-b4fc-40e4-8d35-3caee079b8f1	2015-06-21 16:52:23.120021
719	87	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '5'\nstart:\n- \n- '170837546'\nstop:\n- \n- '170837547'\nreference_bases:\n- \n- '0'\nvariant_bases:\n- \n- TCAG\n	2	\N	\N	e84a200a-ea47-4271-a4d7-35381de48933	2015-06-21 16:52:23.135839
720	93	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '1'\nstart:\n- \n- '115258744'\nstop:\n- \n- '115258744'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	a504383b-1c7b-4a21-904f-6a4481f42ad0	2015-06-21 16:52:23.151447
721	95	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '1'\nstart:\n- \n- '115256528'\nstop:\n- \n- '115256529'\nreference_bases:\n- \n- TT\nvariant_bases:\n- \n- CA\n	2	\N	\N	e32185ad-61ea-4bef-b9a4-56b2757b383d	2015-06-21 16:52:23.170057
722	96	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '1'\nstart:\n- \n- '115256528'\nstop:\n- \n- '115256529'\nreference_bases:\n- \n- TT\nvariant_bases:\n- \n- CC\n	2	\N	\N	8f628f3a-0bf2-4936-b046-f6b0376ea1ac	2015-06-21 16:52:23.18178
723	98	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55152092'\nstop:\n- \n- '55152093'\nreference_bases:\n- \n- GA\nvariant_bases:\n- \n- AT\n	2	\N	\N	05a61ee5-377d-4100-bdc5-06a6e3d60aa8	2015-06-21 16:52:23.195276
724	99	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55152093'\nstop:\n- \n- '55152093'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- T\n	2	\N	\N	c06bae64-a3dc-4ca3-b543-4a4a6986ff5c	2015-06-21 16:52:23.208473
725	100	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55152092'\nstop:\n- \n- '55152092'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	19fde84f-d098-482a-b5b5-0e16d8bf7230	2015-06-21 16:52:23.221747
726	103	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '3'\nstart:\n- \n- '178936082'\nstop:\n- \n- '178936082'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	567b7a5f-677a-415b-b247-5bf9c3a2f4fa	2015-06-21 16:52:23.238657
727	104	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '3'\nstart:\n- \n- '178936091'\nstop:\n- \n- '178936091'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	16584975-401d-4899-afe7-c1f2cb67a29b	2015-06-21 16:52:23.251267
728	107	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '3'\nstart:\n- \n- '178952085'\nstop:\n- \n- '178952085'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- G\n	2	\N	\N	83703785-592e-458f-8e4c-522a4bfa383a	2015-06-21 16:52:23.270308
729	110	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '10'\nstart:\n- \n- '89717672'\nstop:\n- \n- '89717672'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	cbe64e34-c42a-477a-87fd-cb7c37abb68e	2015-06-21 16:52:23.28424
730	112	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '10'\nstart:\n- \n- '43609950'\nstop:\n- \n- '43609950'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- G\n	2	\N	\N	bbc25363-08ae-4010-89c5-96c53553b9bd	2015-06-21 16:52:23.297543
731	113	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '10'\nstart:\n- \n- '43617416'\nstop:\n- \n- '43617416'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	6cfaf8db-8ff3-4a75-b135-44fc1191e1d6	2015-06-21 16:52:23.311256
732	114	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '198267359'\nstop:\n- \n- '198267359'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- A\n	2	\N	\N	3b341f73-00c0-4ddc-a07a-23c88c4e9929	2015-06-21 16:52:23.326353
733	115	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '198266834'\nstop:\n- \n- '198266834'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	0863fe2f-4ebd-44ab-957a-8a22c7fe8226	2015-06-21 16:52:23.342083
734	116	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7578406'\nstop:\n- \n- '7578406'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	b34702f4-a967-4deb-a3f3-ee4491b79859	2015-06-21 16:52:23.355708
735	117	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577538'\nstop:\n- \n- '7577538'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	a987a137-e4e8-454d-8008-da183c893621	2015-06-21 16:52:23.368787
736	118	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577539'\nstop:\n- \n- '7577539'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	ab5098b4-c91f-4148-b45b-802cf4724fc7	2015-06-21 16:52:23.38508
737	119	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577535'\nstop:\n- \n- '7577535'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- G\n	2	\N	\N	c714a890-4403-481d-96ea-df5ff56888db	2015-06-21 16:52:23.398977
738	120	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577536'\nstop:\n- \n- '7577536'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- A\n	2	\N	\N	3b24cf87-3e51-4e3d-a144-ce2c08cb63a7	2015-06-21 16:52:23.411337
739	121	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577121'\nstop:\n- \n- '7577121'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	4db18463-c1a7-4dab-bb8f-c9e4ae4a2d9c	2015-06-21 16:52:23.423292
740	122	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577120'\nstop:\n- \n- '7577120'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	0f97b35b-3399-4200-9430-6a70e20c9094	2015-06-21 16:52:23.437367
\.


--
-- Name: audits_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('audits_id_seq', 740, true);


--
-- Data for Name: authorizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY authorizations (id, user_id, provider, uid, created_at, updated_at) FROM stdin;
\.


--
-- Name: authorizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('authorizations_id_seq', 1, false);


--
-- Data for Name: comments; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY comments (id, title, comment, commentable_id, commentable_type, user_id, role, created_at, updated_at) FROM stdin;
\.


--
-- Name: comments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('comments_id_seq', 1, false);


--
-- Data for Name: data_versions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY data_versions (id, version) FROM stdin;
\.


--
-- Name: data_versions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('data_versions_id_seq', 1, false);


--
-- Data for Name: definitions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY definitions (id, term, text, created_at, updated_at) FROM stdin;
\.


--
-- Name: definitions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('definitions_id_seq', 1, false);


--
-- Data for Name: delayed_jobs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY delayed_jobs (id, priority, attempts, handler, last_error, run_at, locked_at, failed_at, locked_by, queue, created_at, updated_at) FROM stdin;
\.


--
-- Name: delayed_jobs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('delayed_jobs_id_seq', 1, false);


--
-- Data for Name: disease_ontology_mirrors; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY disease_ontology_mirrors (id, doid, name) FROM stdin;
\.


--
-- Name: disease_ontology_mirrors_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('disease_ontology_mirrors_id_seq', 1, false);


--
-- Data for Name: diseases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY diseases (id, doid, name, created_at, updated_at) FROM stdin;
6	12603	Acute Leukemia	2015-06-21 16:49:39.158472	2015-06-21 16:49:39.158472
7	1909	Melanoma	2015-06-21 16:49:39.220457	2015-06-21 16:49:39.220457
13	769	Neuroblastoma	2015-06-21 16:49:40.484031	2015-06-21 16:49:40.484031
24	1612	Breast Cancer	2015-06-21 16:49:47.992093	2015-06-21 16:49:47.992093
29	4947	Cholangiocarcinoma	2015-06-21 16:49:50.476097	2015-06-21 16:49:50.476097
35	3069	Astrocytoma	2015-06-21 16:49:52.339581	2015-06-21 16:49:52.339581
1	10747	Lymphoid Leukemia	2015-06-21 16:49:38.785887	2015-06-21 16:52:09.023495
2	9253	Gastrointestinal Stromal Tumor	2015-06-21 16:49:38.874659	2015-06-21 16:52:09.214657
3	9119	Acute Myeloid Leukemia	2015-06-21 16:49:38.975652	2015-06-21 16:52:09.425103
4	8552	Chronic Myeloid Leukemia	2015-06-21 16:49:39.051471	2015-06-21 16:52:09.626434
5	4960	Bone Marrow Cancer	2015-06-21 16:49:39.123188	2015-06-21 16:52:09.847263
8	3908	Non-small Cell Lung Carcinoma	2015-06-21 16:49:39.497306	2015-06-21 16:52:11.248246
9	8997	Polycythemia Vera	2015-06-21 16:49:39.595091	2015-06-21 16:52:11.435402
10	9119	Acute Myeloid Leukemia	2015-06-21 16:49:39.94571	2015-06-21 16:52:11.640954
11	9256	Colorectal Cancer	2015-06-21 16:49:40.001357	2015-06-21 16:52:11.864856
12	0050905	Inflammatory Myofibroblastic Tumor	2015-06-21 16:49:40.179823	2015-06-21 16:52:12.050945
14	9253	Gastrointestinal Stromal Tumor	2015-06-21 16:49:41.117881	2015-06-21 16:52:12.507184
15	3973	Thyroid Medullary Carcinoma	2015-06-21 16:49:41.945972	2015-06-21 16:52:12.70871
16	1781	Thyroid Cancer	2015-06-21 16:49:42.169106	2015-06-21 16:52:12.962093
17	9256	Colorectal Cancer	2015-06-21 16:49:42.505213	2015-06-21 16:52:13.185313
18	4007	Bladder Carcinoma	2015-06-21 16:49:44.60579	2015-06-21 16:52:13.452388
19	3068	Glioblastoma Multiforme	2015-06-21 16:49:45.730816	2015-06-21 16:52:13.719114
20	2394	Ovarian Cancer	2015-06-21 16:49:47.25873	2015-06-21 16:52:14.178878
21	10534	Stomach Cancer	2015-06-21 16:49:47.585905	2015-06-21 16:52:14.793226
22	1612	Breast Cancer	2015-06-21 16:49:47.649756	2015-06-21 16:52:15.305939
23	1324	Lung Cancer	2015-06-21 16:49:47.847323	2015-06-21 16:52:15.617341
25	3908	Non-small Cell Lung Carcinoma	2015-06-21 16:49:48.291213	2015-06-21 16:52:16.437365
26	3910	Lung Adenocarcinoma	2015-06-21 16:49:48.84016	2015-06-21 16:52:16.709507
27	715	T-cell Leukemia	2015-06-21 16:49:49.286559	2015-06-21 16:52:16.926816
28	0060075	Estrogen-receptor Positive Breast Cancer	2015-06-21 16:49:49.320871	2015-06-21 16:52:17.092426
30	3909	Bronchogenic Lung Adenocarcinoma	2015-06-21 16:49:50.686452	2015-06-21 16:52:17.472673
31	14145	Malignant Anus Melanoma	2015-06-21 16:49:50.724533	2015-06-21 16:52:17.637155
32	9119	Acute Myeloid Leukemia	2015-06-21 16:49:51.249704	2015-06-21 16:52:17.834968
33	6477	Invasive Bladder Transitional Cell Carcinoma	2015-06-21 16:49:51.408451	2015-06-21 16:52:18.479653
34	0050908	Myelodysplastic Syndrome	2015-06-21 16:49:51.685656	2015-06-21 16:52:18.650085
36	3068	Glioblastoma Multiforme	2015-06-21 16:49:52.369039	2015-06-21 16:52:19.346928
37	5520	Head And Neck Squamous Cell Carcinoma	2015-06-21 16:49:52.630475	2015-06-21 16:52:19.534623
38	0050746	Mantle Cell Lymphoma	2015-06-21 16:49:52.664795	2015-06-21 16:52:19.710241
39	5635	Gastric Adenosquamous Carcinoma	2015-06-21 16:49:52.847314	2015-06-21 16:52:19.882259
40	0050908	Myelodysplastic Syndrome	2015-06-21 16:49:53.159522	2015-06-21 16:52:20.070034
41	9538	Multiple Myeloma	2015-06-21 16:49:53.251866	2015-06-21 16:52:20.33538
42	9119	Acute Myeloid Leukemia	2015-06-21 16:49:53.309625	2015-06-21 16:52:20.770043
43	9952	Acute Lymphocytic Leukemia	2015-06-21 16:49:54.088762	2015-06-21 16:52:21.009731
44	5015	Hepatocellular Fibrolamellar Carcinoma	2015-06-21 16:49:54.163625	2015-06-21 16:52:21.182735
45	0050745	Diffuse Large B-cell Lymphoma	2015-06-21 16:49:54.227498	2015-06-21 16:52:21.359582
46	10283	Prostate Cancer	2015-06-21 16:49:54.397584	2015-06-21 16:52:21.618839
47	4001	Ovarian Carcinoma	2015-06-21 16:49:54.54839	2015-06-21 16:52:21.860271
48	4467	Renal Clear Cell Carcinoma	2015-06-21 16:49:54.619413	2015-06-21 16:52:22.083147
49	0050908	Myelodysplastic Syndrome	2015-06-21 16:49:54.879207	2015-06-21 16:52:22.257446
50	3369	Peripheral Primitive Neuroectodermal Tumor	2015-06-21 16:49:55.067774	2015-06-21 16:52:22.485262
\.


--
-- Name: diseases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('diseases_id_seq', 50, true);


--
-- Data for Name: drugs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs (id, name, pubchem_id, created_at, updated_at) FROM stdin;
1	N/A	\N	2015-06-21 16:49:38.842477	2015-06-21 16:49:38.842477
2	Daunorubicin	\N	2015-06-21 16:49:39.272105	2015-06-21 16:49:39.272105
3	AZD6244	\N	2015-06-21 16:49:39.325175	2015-06-21 16:49:39.325175
4	Vemurafenib	\N	2015-06-21 16:49:39.398805	2015-06-21 16:49:39.398805
5	Imatinib	\N	2015-06-21 16:49:39.432754	2015-06-21 16:49:39.432754
6	Sorafenib	\N	2015-06-21 16:49:39.518954	2015-06-21 16:49:39.518954
7	Idarubicin	\N	2015-06-21 16:49:39.563505	2015-06-21 16:49:39.563505
8	Pegylated IFN-α–2a	\N	2015-06-21 16:49:39.629799	2015-06-21 16:49:39.629799
9	TG101348	\N	2015-06-21 16:49:39.689459	2015-06-21 16:49:39.689459
10	17-AAG	\N	2015-06-21 16:49:39.752457	2015-06-21 16:49:39.752457
11	Temozolomide 	\N	2015-06-21 16:49:39.806725	2015-06-21 16:49:39.806725
12	Crizotinib	\N	2015-06-21 16:49:40.201849	2015-06-21 16:49:40.201849
13	Nilotinib	\N	2015-06-21 16:49:40.317059	2015-06-21 16:49:40.317059
14	Gefitinib	\N	2015-06-21 16:49:40.389462	2015-06-21 16:49:40.389462
15	Erlotinib	\N	2015-06-21 16:49:40.395234	2015-06-21 16:49:40.395234
16	Cetuximab	\N	2015-06-21 16:49:40.447485	2015-06-21 16:49:40.447485
17	CH5424802	\N	2015-06-21 16:49:40.510119	2015-06-21 16:49:40.510119
18	TAE684	\N	2015-06-21 16:49:40.585871	2015-06-21 16:49:40.585871
19	Trametinib	\N	2015-06-21 16:49:40.661317	2015-06-21 16:49:40.661317
20	Dasatinib	\N	2015-06-21 16:49:40.70667	2015-06-21 16:49:40.70667
21	Crenolanib	\N	2015-06-21 16:49:40.759107	2015-06-21 16:49:40.759107
22	Dabrafenib	\N	2015-06-21 16:49:41.905049	2015-06-21 16:49:41.905049
23	Motesanib	\N	2015-06-21 16:49:42.004782	2015-06-21 16:49:42.004782
24	AZD1480	\N	2015-06-21 16:49:42.114969	2015-06-21 16:49:42.114969
25	Dacarbazine	\N	2015-06-21 16:49:42.287044	2015-06-21 16:49:42.287044
26	Temozolomide	\N	2015-06-21 16:49:42.291486	2015-06-21 16:49:42.291486
27	Regorafenib	\N	2015-06-21 16:49:42.330448	2015-06-21 16:49:42.330448
28	Panitumumab	\N	2015-06-21 16:49:42.486457	2015-06-21 16:49:42.486457
29	PD0325901	\N	2015-06-21 16:49:42.800468	2015-06-21 16:49:42.800468
30	PLX4720	\N	2015-06-21 16:49:42.806697	2015-06-21 16:49:42.806697
31	Nutlin-3	\N	2015-06-21 16:49:42.852373	2015-06-21 16:49:42.852373
32	Capecitabine	\N	2015-06-21 16:49:42.903271	2015-06-21 16:49:42.903271
33	Bevacizumab	\N	2015-06-21 16:49:42.912675	2015-06-21 16:49:42.912675
34	ATRA	\N	2015-06-21 16:49:43.969848	2015-06-21 16:49:43.969848
35	SU5614	\N	2015-06-21 16:49:44.095501	2015-06-21 16:49:44.095501
36	PD173074	\N	2015-06-21 16:49:44.641876	2015-06-21 16:49:44.641876
37	Valproic Acid	\N	2015-06-21 16:49:44.848027	2015-06-21 16:49:44.848027
38	NSC348884	\N	2015-06-21 16:49:44.882788	2015-06-21 16:49:44.882788
39	Induction Therapy	\N	2015-06-21 16:49:45.050121	2015-06-21 16:49:45.050121
40	Rapamycin	\N	2015-06-21 16:49:45.103225	2015-06-21 16:49:45.103225
41	Everolimus	\N	2015-06-21 16:49:45.174919	2015-06-21 16:49:45.174919
42	Olaparib	\N	2015-06-21 16:49:47.27979	2015-06-21 16:49:47.27979
43	MK-2206	\N	2015-06-21 16:49:48.010543	2015-06-21 16:49:48.010543
44	Dacomitinib	\N	2015-06-21 16:49:48.406886	2015-06-21 16:49:48.406886
45	Lapatinib	\N	2015-06-21 16:49:48.454	2015-06-21 16:49:48.454
46	Hormone Therapy	\N	2015-06-21 16:49:48.487382	2015-06-21 16:49:48.487382
47	Arsenic Trioxide	\N	2015-06-21 16:49:49.170852	2015-06-21 16:49:49.170852
48	Palbociclib (PD-0332991)	\N	2015-06-21 16:49:49.303233	2015-06-21 16:49:49.303233
49	BYL719	\N	2015-06-21 16:49:49.338217	2015-06-21 16:49:49.338217
50	Premetrexed	\N	2015-06-21 16:49:49.820592	2015-06-21 16:49:49.820592
51	Stauroporine	\N	2015-06-21 16:49:49.852944	2015-06-21 16:49:49.852944
52	Neratinib	\N	2015-06-21 16:49:49.889859	2015-06-21 16:49:49.889859
53	Tamoxifen	\N	2015-06-21 16:49:50.292649	2015-06-21 16:49:50.292649
54	Fulvestrant	\N	2015-06-21 16:49:50.29697	2015-06-21 16:49:50.29697
55	Ponatinib	\N	2015-06-21 16:49:50.492568	2015-06-21 16:49:50.492568
56	Pazopanib	\N	2015-06-21 16:49:50.529553	2015-06-21 16:49:50.529553
57	CEP701	\N	2015-06-21 16:49:50.576795	2015-06-21 16:49:50.576795
58	AG1296	\N	2015-06-21 16:49:50.640095	2015-06-21 16:49:50.640095
59	Bortezomib	\N	2015-06-21 16:49:50.704935	2015-06-21 16:49:50.704935
60	Fasudil	\N	2015-06-21 16:49:50.709537	2015-06-21 16:49:50.709537
61	SU11248	\N	2015-06-21 16:49:50.82992	2015-06-21 16:49:50.82992
62	NVP-BEZ235	\N	2015-06-21 16:49:50.871102	2015-06-21 16:49:50.871102
63	ARRY-142886	\N	2015-06-21 16:49:50.876071	2015-06-21 16:49:50.876071
64	Carmustine	\N	2015-06-21 16:49:50.976535	2015-06-21 16:49:50.976535
65	O(6)-benzylguanine	\N	2015-06-21 16:49:51.034784	2015-06-21 16:49:51.034784
66	CH5132799	\N	2015-06-21 16:49:51.06894	2015-06-21 16:49:51.06894
67	MTOR Inhibitors	\N	2015-06-21 16:49:51.300949	2015-06-21 16:49:51.300949
68	Doxorubicin	\N	2015-06-21 16:49:51.392457	2015-06-21 16:49:51.392457
69	Palbociclib	\N	2015-06-21 16:49:52.230469	2015-06-21 16:49:52.230469
70	GW-2580	\N	2015-06-21 16:49:54.110125	2015-06-21 16:49:54.110125
71	Allogeneic Hemopoietic Stem-cell Transplantation	\N	2015-06-21 16:49:54.375269	2015-06-21 16:49:54.375269
72	OSI-906 (Linsitinib)	\N	2015-06-21 16:49:54.41406	2015-06-21 16:49:54.41406
73	Docetaxel	\N	2015-06-21 16:49:54.487568	2015-06-21 16:49:54.487568
74	Cabazitaxel	\N	2015-06-21 16:49:54.49291	2015-06-21 16:49:54.49291
\.


--
-- Data for Name: genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes (id, entrez_id, name, description, official_name, created_at, updated_at, clinical_description, deleted, deleted_at) FROM stdin;
1	238	ALK	ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines.	anaplastic lymphoma receptor tyrosine kinase	2015-06-21 16:49:19.305128	2015-06-21 16:49:19.305128	\N	f	\N
2	207	AKT1	AKT1, also referred to as protein kinase B, is a known oncogene. AKT activation relies on the PI3K pathway, and is recognized as a critical node in the pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been shown to result in activation of the protein. Mutations in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors in vitro.	v-akt murine thymoma viral oncogene homolog 1	2015-06-21 16:49:19.630622	2015-06-21 16:49:19.630622	\N	f	\N
3	369	ARAF	ARAF has recently become increasingly considered for its oncogenic potential. Its potential as a target for informing clinical action was demonstrated by a single case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated with sorafenib, acheived near-complete clinical remission. This finding has brought new focus on ARAF as a marker that should be assayed for in cancer treatment.	A-Raf proto-oncogene, serine/threonine kinase	2015-06-21 16:49:19.894275	2015-06-21 16:49:19.894275	\N	f	\N
4	25	ABL1	ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein that has become a signature of chronic myeloid leukemia (CML). Cells harboring this fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook of the disease. However, additional mutations in ABL1 have been shown to confer resistance to imatinib. In these resistance cases, second-generation tyrosine kinase inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are currently undergoing clinical trials for treating acquired resistance in CML.	ABL proto-oncogene 1, non-receptor tyrosine kinase	2015-06-21 16:49:20.159323	2015-06-21 16:49:20.159323	\N	f	\N
5	673	BRAF	BRAF mutations are found to be recurrent in many cancer types. Of these, the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E has been determined to be an activating mutation, and cells that harbor it, along with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations have also been correlated with poor prognosis in many cancer types, although there is at least one study that questions this conclusion in papillary thyroid cancer.	B-Raf proto-oncogene, serine/threonine kinase	2015-06-21 16:49:20.439745	2015-06-21 16:49:20.439745	\N	f	\N
6	672	BRCA1	BRCA1 mutations in the germline have become a hallmark for hereditary breast and ovarian cancers. Variants that have been demonstrated to reduce the function of the protein have been shown to increase risk for these cancers, as well as prostate and pancreatic cancer. These findings have been the impetus for the increased popularity of genetic testing of healthy indivudals to assess risk.	breast cancer 1, early onset	2015-06-21 16:49:20.734529	2015-06-21 16:49:20.734529	\N	f	\N
7	675	BRCA2	BRCA2 mutations in the germline have become a hallmark for hereditary breast and ovarian cancers. Variants that have been demonstrated to reduce the function of the protein have been shown to increase risk for these cancers, as well as prostate and pancreatic cancer. These findings have been the impetus for the increased popularity of genetic testing of healthy indivudals to assess risk.	breast cancer 2, early onset	2015-06-21 16:49:21.046256	2015-06-21 16:49:21.046256	\N	f	\N
8	595	CCND1	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D1	2015-06-21 16:49:21.300918	2015-06-21 16:49:21.300918	\N	f	\N
9	894	CCND2	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D2	2015-06-21 16:49:21.54153	2015-06-21 16:49:21.54153	\N	f	\N
10	896	CCND3	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. In a mouse study, the targeted therapeutic palbociclib significantly increased the median survival of the cyclin D3 knockouts.	cyclin D3	2015-06-21 16:49:21.753867	2015-06-21 16:49:21.753867	\N	f	\N
11	898	CCNE1	Cyclin E, while currenly not as widely implicated as its cyclin D counterparts, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements are correlated with later stage disease. In lung cancer, neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly expressed counterparts.	cyclin E1	2015-06-21 16:49:21.982453	2015-06-21 16:49:21.982453	\N	f	\N
12	1021	CDK6	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 6	2015-06-21 16:49:22.213061	2015-06-21 16:49:22.213061	\N	f	\N
13	1019	CDK4	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 4	2015-06-21 16:49:22.446034	2015-06-21 16:49:22.446034	\N	f	\N
14	1029	CDKN2A	CDKN2A loss has been shown to be a significant event in a number of cancer types. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. One mechanism by which loss of CDKN2A can occur is by hypermethylation of the promoter region for the gene. However, the prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have suggesting poorer prognostic outcome for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	cyclin-dependent kinase inhibitor 2A	2015-06-21 16:49:22.776707	2015-06-21 16:49:22.776707	\N	f	\N
15	1050	CEBPA	AML with mutated CEBPA' is a provisional entity in the WHO classification of acute myeloid leukemia (AML) and is recommended to be tested for in patients with AML. CEBPA mutations are particularly associated with cytogenetically normal AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha protein while C-terminal mutations reduce the DNA-binding potential of this transcription factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	CCAAT/enhancer binding protein (C/EBP), alpha	2015-06-21 16:49:23.055349	2015-06-21 16:49:23.055349	\N	f	\N
16	3337	DNAJB1	DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	DnaJ (Hsp40) homolog, subfamily B, member 1	2015-06-21 16:49:23.311708	2015-06-21 16:49:23.311708	\N	f	\N
17	5566	PRKACA	PRKACA has been studied in breast cancer and has been found to mediate resistance to HER2 targeted therapies. It has also been found to contain a mutation hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	protein kinase, cAMP-dependent, catalytic, alpha	2015-06-21 16:49:23.544501	2015-06-21 16:49:23.544501	\N	f	\N
18	1788	DNMT3A	DNMT3A is one of several epigenetic modifiers identified as recurrently mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers. AML patient bone marrow harboring R882 mutations were similarly demonstrated to be hypomethylated compared to patients with wildtype DNMT3A. These studies also indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes and treatment protocols may need to be considered for these two classes of DNMT3A mutations.	DNA (cytosine-5-)-methyltransferase 3 alpha	2015-06-21 16:49:23.779392	2015-06-21 16:49:23.779392	\N	f	\N
19	1956	EGFR	EGFR is widely recognized for its importance in cancer. Amplification and mutations have been shown to be driving events in many cancer types. Its role in non-small cell lung cancer has spurred many research and drug development efforts. Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance to these drugs, particularly the variant T790M, which has been functionally characterized as a resistance marker for both of these drugs. The later generation TKI's have seen some success in treating these resistant cases, and targeted sequencing of the EGFR locus has become a common practice in non-small cell lung cancer.	epidermal growth factor receptor	2015-06-21 16:49:24.038682	2015-06-21 16:49:24.038682	\N	f	\N
20	2064	ERBB2	ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast cancer, and is treated in a separate manner than the other subtypes of breast cancer. Apart from being amplified, ERBB2 activating mutations have been shown to have clinical importance in HER2-negative breast cancer. These mutations have shown sensitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical sequencing efforts in treating breast cancer.	erb-b2 receptor tyrosine kinase 2	2015-06-21 16:49:24.325594	2015-06-21 16:49:24.325594	\N	f	\N
21	2099	ESR1	ESR1 has been a focus in breast cancer for quite some time, but has also shown significance in endometrial, ovarian and other cancer types. ER-positive breast cancer that is resistant to hormone therapy has instigated clinical sequencing efforts to shed light on the mechanisms of this resistance. A number of mutations in the ligand binding domain of the protein have been implicated in hormone resistance and anti-estrogen therapies. This has spurred efforts to develop therapeutics that act to degrade the protein, rather than act as an antagonist. These agents are currently in clinical trials and have seen some success, highlighting the importance of sequencing efforts in treating breast cancer.	estrogen receptor 1	2015-06-21 16:49:24.606817	2015-06-21 16:49:24.606817	\N	f	\N
22	2263	FGFR2	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions that lead to pathway actiation have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 2	2015-06-21 16:49:24.914214	2015-06-21 16:49:24.914214	\N	f	\N
23	2261	FGFR3	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions that lead to pathway actiation have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 3	2015-06-21 16:49:25.194308	2015-06-21 16:49:25.194308	\N	f	\N
24	2322	FLT3	FLT3 is an important cytokine receptor involved in normal hematopoiesis. Mutations in this gene are common in acute myeloid leukemia (AML) and screening for mutations in this gene has been recommended by the World Health Organization in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML). FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD mutations are still up for debate.	fms-related tyrosine kinase 3	2015-06-21 16:49:25.447875	2015-06-21 16:49:25.447875	\N	f	\N
25	2624	GATA2	GATA2 is a transcription factor involved in stem cell maintenance with key roles in hematopoietic development. GATA2 mutations are associated with a variety of inherited and acquired immune disorders including myelodysplastic syndrome and acute myeloid leukemia. In addition to a role in hematopoiesis, the maintenance GATA2 expression has been implicated as a requirement in KRAS-driven non-small cell lung cancer. Preclinical models have indicated therapeutic benefit from targeting GATA2-mediated pathways in the context of KRAS-driven NSCLC.	GATA binding protein 2	2015-06-21 16:49:25.698932	2015-06-21 16:49:25.698932	\N	f	\N
26	3417	IDH1	IDH1 mutations have been observed in a number of cancer types, including sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described as mutually exclusive in many of these cancer types. The most frequent mutations involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic enzyme activity. The implications of mutations in this gene vary greatly by cancer type. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations have been associated with worse outcome, shorter overall survival, and normal karyotype. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Unlike the association with cytogenetically normal AML, in glioblastoma, IDH1 mutations have been associated with specific cytogenetic abnormalities, 1p and 19q deletions.	isocitrate dehydrogenase 1 (NADP+), soluble	2015-06-21 16:49:25.958723	2015-06-21 16:49:25.958723	\N	f	\N
27	3418	IDH2	IDH2 mutations have been observed in a number of cancer types, including sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described as mutually exclusive in many of these cancer types. The most frequent mutations involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic enzyme activity. Although IDH2 (R172) mutations are associated with poorer overall prognosis in AML patients, its utility as a prognostic marker in MDS is still under debate. Additionally, IDH2 (R140) has been associated with improved overall survival in AML. IDH2 mutations have been associated with improved prognosis in gliomas.	isocitrate dehydrogenase 2 (NADP+), mitochondrial	2015-06-21 16:49:26.254526	2015-06-21 16:49:26.254526	\N	f	\N
28	3717	JAK2	JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative diseases and cancers. The mutation V617F is the most clinically relevant variant, and is seen in around half of myeloproliferative disorders. The variant is a known activating mutation, and activated JAK2 is sufficient to drive myeloproliferative disorders in mouse models. V617F, while most recurrent, is not the only mechanism by which JAK2 can be activated in patients. JAK2 is now one of the first diagnostic markers tested upon diagnosis with a myeloproliferative disorder.	Janus kinase 2	2015-06-21 16:49:26.523774	2015-06-21 16:49:26.523774	\N	f	\N
29	3815	KIT	c-KIT activation has been shown to have oncogenic activity in gastrointestinal stromal tumors (GISTs), melanomas, lung cancer, and other tumor types. The targeted therapeutics nilotinib and sunitinib have shown efficacy in treating KIT overactive patients, and are in late-stage trials in melanoma and GIST. KIT overactivity can be the result of many genomic events from genomic amplification to overexpression to missense mutations. Missense mutations have been shown to be key players in mediating clinical response and acquired resistance in patients being treated with these targeted therapeutics.	v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog	2015-06-21 16:49:26.76424	2015-06-21 16:49:26.76424	\N	f	\N
30	3845	KRAS	Mutations in the RAS family of proteins have frequently observed across cancer types. The amino acid positions account for the overwhelming majority of these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also behave very differently when expressed in non-native tissue types, likely due to differences in the C-terminal hyper-variable regions. Mis-regulation of isoform expression has been shown to be a driving event in cancer, as well as missense mutations at the three hotspots previously mentioned. While highly recurrent in cancer, targeted these RAS mutants has also been very elusive, and has not yet become common practice in the clinic.	Kirsten rat sarcoma viral oncogene homolog	2015-06-21 16:49:27.045947	2015-06-21 16:49:27.045947	\N	f	\N
31	5604	MAP2K1		mitogen-activated protein kinase kinase 1	2015-06-21 16:49:27.302485	2015-06-21 16:49:27.302485	\N	f	\N
32	4209	MEF2D		myocyte enhancer factor 2D	2015-06-21 16:49:27.533097	2015-06-21 16:49:27.533097	\N	f	\N
33	1436	CSF1R		colony stimulating factor 1 receptor	2015-06-21 16:49:27.777632	2015-06-21 16:49:27.777632	\N	f	\N
34	4255	MGMT		O-6-methylguanine-DNA methyltransferase	2015-06-21 16:49:27.999712	2015-06-21 16:49:27.999712	\N	f	\N
35	4869	NPM1	AML with mutated NPM1 is provisional entity in the WHO classification of AML and is recommended to be tested for in patients with cytogenetically normal AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia, indicating it may require additional mutations to promote leukemic development.	nucleophosmin (nucleolar phosphoprotein B23, numatrin)	2015-06-21 16:49:28.241486	2015-06-21 16:49:28.241486	\N	f	\N
36	4893	NRAS	Mutations in the RAS family of proteins have frequently observed across cancer types. The amino acid positions account for the overwhelming majority of these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also behave very differently when expressed in non-native tissue types, likely due to differences in the C-terminal hyper-variable regions. Mis-regulation of isoform expression has been shown to be a driving event in cancer, as well as missense mutations at the three hotspots previously mentioned. While highly recurrent in cancer, targeted these RAS mutants has also been very elusive, and has not yet become common practice in the clinic.	neuroblastoma RAS viral (v-ras) oncogene homolog	2015-06-21 16:49:28.487777	2015-06-21 16:49:28.487777	\N	f	\N
37	5290	PIK3CA	PIK3CA is the most recurrently mutated gene in breast cancer, and has been found to important in a number of cancer types. An integral part of the PI3K pathway, PIK3CA has long been described as an oncogene, with two main hotspots for activating mutations, the 542/545 region of the helical domain, and the 1047 region of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the subject of an immense amount of research and development, and PI3K inhibition has seen some limited success in recent clinical trials. While monotherapies seem to be limited in their potential, there is a recent interest in pursuing PI3K inhibition as part of a combination therapy regiment with inhibition partners including TKI's, MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.	phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha	2015-06-21 16:49:28.737504	2015-06-21 16:49:28.737504	\N	f	\N
38	5156	PDGFRA		platelet-derived growth factor receptor, alpha polypeptide	2015-06-21 16:49:28.982072	2015-06-21 16:49:28.982072	\N	f	\N
39	5371	PML	The PML-RARA fusion is the result of a recurrent, balanced translocation between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	promyelocytic leukemia	2015-06-21 16:49:29.216151	2015-06-21 16:49:29.216151	\N	f	\N
40	5914	RARA	The PML-RARA fusion is the result of a recurrent, balanced translocation between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	retinoic acid receptor, alpha	2015-06-21 16:49:29.443108	2015-06-21 16:49:29.443108	\N	f	\N
41	5728	PTEN		phosphatase and tensin homolog	2015-06-21 16:49:29.705927	2015-06-21 16:49:29.705927	\N	f	\N
42	5979	RET	RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine kinase receptor and are associated with thyroid cancers. RET point mutations are the most common mutations identified in medullary thyroid cancer (MTC) with germline and somatic mutations in RET associated with hereditary and sporadic forms, respectively. The most common somatic form mutation is M918T (exon 16) and a variety of other mutations effecting exons 10, 11 and 15 have been described. The prognostic significance of these mutations have been hotly debated in the field, however, data suggests that some RET mutation may confer drug resistence. No RET-specific agents are currently clinically available but several promiscuous kinase inhibitors that target RET, among others, have been approved for MTC treatment.	ret proto-oncogene	2015-06-21 16:49:29.989537	2015-06-21 16:49:29.989537	\N	f	\N
43	861	RUNX1		runt-related transcription factor 1	2015-06-21 16:49:30.254262	2015-06-21 16:49:30.254262	\N	f	\N
44	23451	SF3B1	SF3B1 mutations have been described in several myeloid malignancies, predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has been identified as recurrently mutated in MDS and other malignanices. The mutations affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally deleterious with K700E described as a major hotspot mutation. MDS patients with SF3B1 mutations have been reported to have better overall and event-free survival than their wildtype counterparts. Additionally, these mutations are highly associated with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring sideroblasts).	splicing factor 3b, subunit 1, 155kDa	2015-06-21 16:49:30.512706	2015-06-21 16:49:30.512706	\N	f	\N
45	7157	TP53	TP53 mutations are universal across cancer types. Loss of tumor suppressors is most recognized by large deleterious events, such as frameshift mutations, or premature stop codons. In TP53 however, many of the observed mutations in cancer are found to be single nucleotide variants, or missense mutations. These variants are also very broadly distributed throughout the gene, not localizing in any particular hotspot. While a large proportion of cancer genomics research is focused on somatic variants, TP53 is also of note in the germline. Germline TP53 mutations are the hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been found to have prognostic impact on patient outcomes.	tumor protein p53	2015-06-21 16:49:30.771644	2015-06-21 16:49:30.771644	\N	f	\N
46	7248	TSC1		tuberous sclerosis 1	2015-06-21 16:49:30.998141	2015-06-21 16:49:30.998141	\N	f	\N
47	7249	TSC2		tuberous sclerosis 2	2015-06-21 16:49:31.227551	2015-06-21 16:49:31.227551	\N	f	\N
48	7307	U2AF1	U2AF1 is one of several spliceosome complex genes frequently mutated in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein. These mutations have been associated with altered splicing patterns in vitro and in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation to secondary acute myeloid leukemia, however, the impact of these mutations on overall survival has been an area of debate.	U2 small nuclear RNA auxiliary factor 1	2015-06-21 16:49:31.477117	2015-06-21 16:49:31.477117	\N	f	\N
49	7490	WT1	WT1 is a tumor suppressor gene associated with the development of Wilms' Tumor, from which it was named. Mutations in exon 7 and 9 of WT1 have been recurrently identified in acute myeloid leukemia and associated with poorer prognosis and chemotherapy resistance.	Wilms tumor 1	2015-06-21 16:49:31.742247	2015-06-21 16:49:31.742247	\N	f	\N
50	4851	NOTCH1		notch 1	2015-06-21 16:49:32.002668	2015-06-21 16:49:32.002668	\N	f	\N
51	4921	DDR2		discoidin domain receptor tyrosine kinase 2	2015-06-21 16:49:32.257053	2015-06-21 16:49:32.257053	\N	f	\N
52	4233	MET		MET proto-oncogene, receptor tyrosine kinase	2015-06-21 16:49:32.498123	2015-06-21 16:49:32.498123	\N	f	\N
53	2313	FLI1		Fli-1 proto-oncogene, ETS transcription factor	2015-06-21 16:49:32.725095	2015-06-21 16:49:32.725095	\N	f	\N
54	2130	EWSR1		EWS RNA-binding protein 1	2015-06-21 16:49:32.966448	2015-06-21 16:49:32.966448	\N	f	\N
55	54790	TET2		tet methylcytosine dioxygenase 2	2015-06-21 16:49:33.176374	2015-06-21 16:49:33.176374	\N	f	\N
56	27086	FOXP1		forkhead box P1	2015-06-21 16:49:33.417525	2015-06-21 16:49:33.417525	\N	f	\N
57	5966	REL		v-rel avian reticuloendotheliosis viral oncogene homolog	2015-06-21 16:49:33.638204	2015-06-21 16:49:33.638204	\N	f	\N
58	7428	VHL		von Hippel-Lindau tumor suppressor, E3 ubiquitin protein ligase	2015-06-21 16:49:33.867403	2015-06-21 16:49:33.867403	\N	f	\N
59	596	BCL2		B-cell CLL/lymphoma 2	2015-06-21 16:49:34.093763	2015-06-21 16:49:34.093763	\N	f	\N
60	3481	IGF2		insulin-like growth factor 2	2015-06-21 16:49:34.343982	2015-06-21 16:49:34.343982	\N	f	\N
61	6790	AURKA		aurora kinase A	2015-06-21 16:49:34.605185	2015-06-21 16:49:34.605185	\N	f	\N
62	55193	PBRM1		polybromo 1	2015-06-21 16:49:34.836461	2015-06-21 16:49:34.836461	\N	f	\N
63	2146	EZH2		enhancer of zeste 2 polycomb repressive complex 2 subunit	2015-06-21 16:49:35.105652	2015-06-21 16:49:35.105652	\N	f	\N
\.


--
-- Data for Name: sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY sources (id, pubmed_id, study_type, description, created_at, updated_at) FROM stdin;
1	24889366	\N	Rossi et al., 2014, Int. J. Oncol.	2015-06-21 16:49:19.371262	2015-06-21 16:49:55.808516
2	23401436	\N	Shaw et al., 2013, J. Clin. Oncol.	2015-06-21 16:49:19.379307	2015-06-21 16:49:56.248285
3	16095999	\N	Bellacosa et al., 2005, Adv. Cancer Res.	2015-06-21 16:49:19.643676	2015-06-21 16:49:56.663026
4	15023437	\N	Fresno Vara et al., 2004, Cancer Treat. Rev.	2015-06-21 16:49:19.647565	2015-06-21 16:49:57.167861
5	15676015	\N	Lee et al., 2005, APMIS	2015-06-21 16:49:19.906057	2015-06-21 16:49:57.78328
6	24569458	\N	Imielinski et al., 2014, J. Clin. Invest.	2015-06-21 16:49:19.910976	2015-06-21 16:49:58.201806
7	15719031	\N	Ren, 2005, Nat. Rev. Cancer	2015-06-21 16:49:20.17221	2015-06-21 16:49:58.593886
8	17457302	\N	Weisberg et al., 2007, Nat. Rev. Cancer	2015-06-21 16:49:20.176571	2015-06-21 16:49:59.318854
9	19724843	\N	Li et al., 2009, Oncol. Rep.	2015-06-21 16:49:20.451516	2015-06-21 16:49:59.93388
10	23594689	\N	Pakneshan et al., 2013, Pathology	2015-06-21 16:49:20.455191	2015-06-21 16:50:00.344185
11	24366442	\N	Nelson et al., 2014, Ann. Intern. Med.	2015-06-21 16:49:20.748825	2015-06-21 16:50:00.855551
12	12432268	\N	Diehl, Cancer Biol. Ther.	2015-06-21 16:49:21.315588	2015-06-21 16:50:01.225136
13	24387133	\N	Casimiro et al., 2014, Expert Opin Investig Drugs	2015-06-21 16:49:21.319973	2015-06-21 16:50:01.777508
14	14965268	\N	Mazumder et al., 2004, Curr Cancer Drug Targets	2015-06-21 16:49:21.99593	2015-06-21 16:50:02.210922
15	20370706	\N	Graf et al., 2010, Mini Rev Med Chem	2015-06-21 16:49:22.225642	2015-06-21 16:50:02.564132
16	24089445	\N	Sheppard et al., 2013, Clin. Cancer Res.	2015-06-21 16:49:22.229455	2015-06-21 16:50:03.006191
17	20473920	\N	Shima et al., 2011, Int. J. Cancer	2015-06-21 16:49:22.791691	2015-06-21 16:50:03.387003
18	23111194	\N	Bradly et al., 2012, Diagn. Mol. Pathol.	2015-06-21 16:49:22.796831	2015-06-21 16:50:03.939644
19	19357394	\N	Vardiman et al., 2009, Blood	2015-06-21 16:49:23.071673	2015-06-21 16:50:04.29627
20	11242107	\N	Pabst et al., 2001, Nat. Genet.	2015-06-21 16:49:23.076161	2015-06-21 16:50:04.671162
21	9012825	\N	Zhang et al., 1997, Proc. Natl. Acad. Sci. U.S.A.	2015-06-21 16:49:23.080271	2015-06-21 16:50:05.207913
22	24578576	\N	Honeyman et al., 2014, Science	2015-06-21 16:49:23.324481	2015-06-21 16:50:05.772223
23	24909179	\N	Moody et al., 2015, Oncogene	2015-06-21 16:49:23.560604	2015-06-21 16:50:06.139793
24	24167195	\N	Kim et al., 2013, Blood	2015-06-21 16:49:23.792907	2015-06-21 16:50:06.692112
25	24656771	\N	Russler-Germain et al., 2014, Cancer Cell	2015-06-21 16:49:23.798601	2015-06-21 16:50:07.207407
26	23953842	\N	Yewale et al., 2013, Biomaterials	2015-06-21 16:49:24.050416	2015-06-21 16:50:07.922564
27	18712184	\N	Charpidou et al., In Vivo	2015-06-21 16:49:24.054544	2015-06-21 16:50:08.536262
28	16947083	\N	Badache et al., 2006, J Mammary Gland Biol Neoplasia	2015-06-21 16:49:24.340373	2015-06-21 16:50:09.148989
29	10878580	\N	Yu et al., 2000, Bioessays	2015-06-21 16:49:24.344763	2015-06-21 16:50:09.765625
30	24185510	\N	Robinson et al., 2013, Nat. Genet.	2015-06-21 16:49:24.61841	2015-06-21 16:50:10.372605
31	22279420	\N	Mohibi et al., 2011, J Carcinog	2015-06-21 16:49:24.623027	2015-06-21 16:50:10.770223
32	18636142	\N	Katoh, 2008, Int. J. Oncol.	2015-06-21 16:49:24.927145	2015-06-21 16:50:11.153319
33	23558953	\N	Wu et al., 2013, Cancer Discov	2015-06-21 16:49:24.930972	2015-06-21 16:50:11.528113
34	12951584	\N	Stirewalt et al., 2003, Nat. Rev. Cancer	2015-06-21 16:49:25.462599	2015-06-21 16:50:12.221795
35	22541434	\N	Kumar et al., 2012, Cell	2015-06-21 16:49:25.711139	2015-06-21 16:50:12.836208
36	25619630	\N	Mir et al., 2015, Cancer Med	2015-06-21 16:49:25.716093	2015-06-21 16:50:13.274928
37	24065766	\N	Lu et al., 2013, Genes Dev.	2015-06-21 16:49:25.973143	2015-06-21 16:50:13.760201
38	19228619	\N	Yan et al., 2009, N. Engl. J. Med.	2015-06-21 16:49:25.977725	2015-06-21 16:50:14.374574
39	22417203	\N	Patel et al., 2012, N. Engl. J. Med.	2015-06-21 16:49:26.276204	2015-06-21 16:50:14.989115
40	17133099	\N	Levine et al., 2007, Curr. Opin. Hematol.	2015-06-21 16:49:26.536042	2015-06-21 16:50:15.389948
41	23944364	\N	Stankov et al., 2014, Curr. Pharm. Des.	2015-06-21 16:49:26.78016	2015-06-21 16:50:15.88366
42	22589270	\N	Prior et al., 2012, Cancer Res.	2015-06-21 16:49:27.058217	2015-06-21 16:50:16.576134
43	23226219	\N	Chou et al., 2012, PLoS ONE	2015-06-21 16:49:28.253704	2015-06-21 16:50:17.160788
44	18073307	\N	Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-06-21 16:49:30.004462	2015-06-21 16:50:17.577258
45	25465739	\N	Perri et al., 2015, Crit. Rev. Oncol. Hematol.	2015-06-21 16:49:30.008703	2015-06-21 16:50:18.058077
46	21995386	\N	Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-06-21 16:49:30.526054	2015-06-21 16:50:18.675359
47	24136165	\N	Cazzola et al., 2013, Blood	2015-06-21 16:49:30.529946	2015-06-21 16:50:19.290602
48	20182602	\N	Olivier et al., 2010, Cold Spring Harb Perspect Biol	2015-06-21 16:49:30.783669	2015-06-21 16:50:19.671447
49	25311244	\N	Okeyo-Owuor et al., 2015, Leukemia	2015-06-21 16:49:31.490476	2015-06-21 16:50:20.256115
50	22158538	\N	Graubert et al., 2012, Nat. Genet.	2015-06-21 16:49:31.495445	2015-06-21 16:50:20.648883
51	16081687	Retrospective study	Levine et al., 2005, Blood	2015-06-21 16:49:38.783508	2015-06-21 16:50:21.037132
52	15146165		Lasota et al., 2004, Lab. Invest.	2015-06-21 16:49:38.872701	2015-06-21 16:50:21.438156
53	22081665	Retrospective study	LaRochelle et al., 2011, Oncotarget	2015-06-21 16:49:38.973517	2015-06-21 16:50:22.361722
54	22490330	Retrospective study	Ribeiro et al., 2012, Blood	2015-06-21 16:49:39.010999	2015-06-21 16:50:22.974729
55	19287384	Retrospective study	Kilpivaara et al., 2009, Nat. Genet.	2015-06-21 16:49:39.121401	2015-06-21 16:50:23.391275
56	24571676	Retrospective study	Andrade et al., 2014, BMC Cancer	2015-06-21 16:49:39.156764	2015-06-21 16:50:23.945059
57	23861977	Retrospective study	Tschandl et al., 2013, PLoS ONE	2015-06-21 16:49:39.218627	2015-06-21 16:50:24.513098
58	19915144		Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-06-21 16:49:39.305568	2015-06-21 16:50:25.124923
59	19915144	In vitro study	Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-06-21 16:49:39.341943	2015-06-21 16:50:25.740652
60	23569304	Clinical trial	Trunzer et al., 2013, J. Clin. Oncol.	2015-06-21 16:49:39.374669	2015-06-21 16:50:26.353876
61	15928335		Corless et al., 2005, J. Clin. Oncol.	2015-06-21 16:49:39.414279	2015-06-21 16:50:26.733616
62	16954519		Heinrich et al., 2006, J. Clin. Oncol.	2015-06-21 16:49:39.455231	2015-06-21 16:50:27.277098
63	24569458		Imielinski et al., 2014, J. Clin. Invest.	2015-06-21 16:49:39.495298	2015-06-21 16:50:27.891243
64	16709929	Clinical trial	Kiladjian et al., 2006, Blood	2015-06-21 16:49:39.590744	2015-06-21 16:50:28.330143
65	18394554	In vitro study	Wernig et al., 2008, Cancer Cell	2015-06-21 16:49:39.654632	2015-06-21 16:50:28.812071
66	18375819	Case report	Banerji et al., 2008, Mol. Cancer Ther.	2015-06-21 16:49:39.718257	2015-06-21 16:50:29.222534
67	21576590	Case report	Soon et al., 2011, Arch Dermatol	2015-06-21 16:49:39.777858	2015-06-21 16:50:29.733149
68	21067377	Retrospective study	Ley et al., 2010, N. Engl. J. Med.	2015-06-21 16:49:39.913879	2015-06-21 16:50:30.349908
69	16384925	Retrospective study	Cairoli et al., 2006, Blood	2015-06-21 16:49:39.943924	2015-06-21 16:50:30.959745
70	24666267	Meta analysis	Therkildsen et al., 2014, Acta Oncol	2015-06-21 16:49:39.999447	2015-06-21 16:50:31.334117
71	21166657	Meta analysis	Lee et al., 2011, Br. J. Dermatol.	2015-06-21 16:49:40.104314	2015-06-21 16:50:32.188441
72	24512939	Retrospective study	El Ghannam et al., Blood Cells Mol. Dis.	2015-06-21 16:49:40.141196	2015-06-21 16:50:32.570756
73	21030459		Sasaki et al., 2010, Cancer Res.	2015-06-21 16:49:40.177682	2015-06-21 16:50:32.928156
74	19671763	In vitro study	Woodman et al., 2009, Mol. Cancer Ther.	2015-06-21 16:49:40.286731	2015-06-21 16:50:33.727208
75	15696205	Retrospective study	Pao et al., 2005, PLoS Med.	2015-06-21 16:49:40.36322	2015-06-21 16:50:34.340682
76	20619739	Retrospective study	De Roock et al., 2010, Lancet Oncol.	2015-06-21 16:49:40.41784	2015-06-21 16:50:34.957112
77	21575866		Sakamoto et al., 2011, Cancer Cell	2015-06-21 16:49:40.481547	2015-06-21 16:50:35.570926
78	21948233		Heuckmann et al., 2011, Clin. Cancer Res.	2015-06-21 16:49:40.525503	2015-06-21 16:50:35.996933
79	18923525		George et al., 2008, Nature	2015-06-21 16:49:40.563185	2015-06-21 16:50:36.412327
80	22745105		Heinrich et al., 2012, Clin. Cancer Res.	2015-06-21 16:49:40.73453	2015-06-21 16:50:36.799215
81	23632886	Retrospective study	Gaidzik et al., 2013, Blood	2015-06-21 16:49:40.97622	2015-06-21 16:50:37.412159
82	16551858	Retrospective study	Wardelmann et al., 2006, Clin. Cancer Res.	2015-06-21 16:49:41.116135	2015-06-21 16:50:38.027419
83	15951308	Retrospective study	Bowen et al., 2005, Blood	2015-06-21 16:49:41.24535	2015-06-21 16:50:39.255086
84	16434492	Clinical trial	Bacher et al., 2006, Blood	2015-06-21 16:49:41.28365	2015-06-21 16:50:39.870695
85	23775962	Clinical trial	Hodi et al., 2013, J. Clin. Oncol.	2015-06-21 16:49:41.326142	2015-06-21 16:50:40.426689
86	22291079	Retrospective study	Marcucci et al., 2012, J. Clin. Oncol.	2015-06-21 16:49:41.625143	2015-06-21 16:50:41.148927
87	22180178	Clinical trial	Jakob et al., 2012, Cancer	2015-06-21 16:49:41.811288	2015-06-21 16:50:41.580771
88	23031422	Case report	Ponti et al., 2012, J Hematol Oncol	2015-06-21 16:49:41.882559	2015-06-21 16:50:42.027846
89	18073307		Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-06-21 16:49:41.944037	2015-06-21 16:50:42.634629
90	21422803		Coxon et al., 2012, J. Endocrinol. Invest.	2015-06-21 16:49:41.985135	2015-06-21 16:50:43.248063
91	23056499		Couto et al., 2012, PLoS ONE	2015-06-21 16:49:42.096483	2015-06-21 16:50:43.867609
92	9839497		Egawa et al., 1998, Jpn. J. Clin. Oncol.	2015-06-21 16:49:42.130791	2015-06-21 16:50:44.480332
93	21594703	Retrospective study	Howell et al., 2011, Ann. Surg. Oncol.	2015-06-21 16:49:42.167425	2015-06-21 16:50:44.887634
94	24570209	Retrospective study	Crescenzi et al., 2014, Horm. Metab. Res.	2015-06-21 16:49:42.200112	2015-06-21 16:50:45.400627
95	24588959	Perspective study	Zhang et al., 2014, Diagn Pathol	2015-06-21 16:49:42.234069	2015-06-21 16:50:45.799699
96	24586605	Retrospective study	Meckbach et al., 2014, PLoS ONE	2015-06-21 16:49:42.269139	2015-06-21 16:50:46.323395
97	24559322	Clinical trial	Sartore-Bianchi et al., 2014, Expert Rev Anticancer Ther	2015-06-21 16:49:42.306037	2015-06-21 16:50:46.65996
98	24576830	In vitro study	Nissan et al., 2014, Cancer Res.	2015-06-21 16:49:42.401805	2015-06-21 16:50:47.084831
99	23325582	Clinical trial	Peeters et al., 2013, Clin. Cancer Res.	2015-06-21 16:49:42.468324	2015-06-21 16:50:47.459348
100	19001320	In vitro study	Di Nicolantonio et al., 2008, J. Clin. Oncol.	2015-06-21 16:49:42.503495	2015-06-21 16:50:48.167176
101	23524406	Case report	Rudin et al., 2013, J Thorac Oncol	2015-06-21 16:49:42.582704	2015-06-21 16:50:48.78134
102	21098728	In vitro study	Corcoran et al., 2010, Sci Signal	2015-06-21 16:49:42.616351	2015-06-21 16:50:49.393679
103	23020132	Clinical trial	Flaherty et al., 2012, N. Engl. J. Med.	2015-06-21 16:49:42.649245	2015-06-21 16:50:50.067173
104	23463675	Clinical trial	Ponti et al., 2013, J. Clin. Pathol.	2015-06-21 16:49:42.684412	2015-06-21 16:50:50.489029
105	24583796	Clinical trial	Menzies et al., 2014, Clin. Cancer Res.	2015-06-21 16:49:42.725897	2015-06-21 16:50:51.238705
106	23845441	In vitro study	Rad et al., 2013, Cancer Cell	2015-06-21 16:49:42.769443	2015-06-21 16:50:51.852248
107	23812671	In vitro study	Ji et al., 2013, Clin. Cancer Res.	2015-06-21 16:49:42.823651	2015-06-21 16:50:52.467312
108	22180495	In vivo study	Yang et al., 2012, Cancer Res.	2015-06-21 16:49:42.873702	2015-06-21 16:50:53.080081
109	24354346	Retrospective study	Walczyk et al., 2014, Clin. Endocrinol. (Oxf)	2015-06-21 16:49:43.069968	2015-06-21 16:50:53.482427
110	24594804	Metaanalysis	Chen et al., 2014, PLoS ONE	2015-06-21 16:49:43.102435	2015-06-21 16:50:54.004007
111	24388723	Retrospective study	Nagore et al., 2014, J. Am. Acad. Dermatol.	2015-06-21 16:49:43.133778	2015-06-21 16:50:54.616052
112	24396464	Retrospective study	He et al., 2014, Oncol Lett	2015-06-21 16:49:43.194327	2015-06-21 16:50:55.538518
113	16455956	Retrospective study	Thiede et al., 2006, Blood	2015-06-21 16:49:43.256287	2015-06-21 16:50:56.154943
114	16076867	Retrospective study	Schnittger et al., 2005, Blood	2015-06-21 16:49:43.366447	2015-06-21 16:50:56.766209
115	19357394		Vardiman et al., 2009, Blood	2015-06-21 16:49:43.507941	2015-06-21 16:50:57.379828
116	17957027	Retrospective study	Gale et al., 2008, Blood	2015-06-21 16:49:43.545984	2015-06-21 16:50:58.044721
117	19059939	Retrospective study	Schlenk et al., 2009, Haematologica	2015-06-21 16:49:43.688791	2015-06-21 16:50:58.43625
118	19965647	Retrospective study	Burnett et al., 2010, Blood	2015-06-21 16:49:43.772233	2015-06-21 16:50:58.812497
119	22072639		Bresler et al., 2011, Sci Transl Med	2015-06-21 16:49:43.87892	2015-06-21 16:50:59.232534
120	19965647		Burnett et al., 2010, Blood	2015-06-21 16:49:43.944169	2015-06-21 16:50:59.838545
121	15626738	In vitro study	Bagrintseva et al., 2005, Blood	2015-06-21 16:49:44.045777	2015-06-21 16:51:00.225733
122	18202412	In vitro study	Lièvre et al., 2008, J. Clin. Oncol.	2015-06-21 16:49:44.228066	2015-06-21 16:51:00.760229
123	20978259	Clinical trial	De Roock et al., 2010, JAMA	2015-06-21 16:49:44.274643	2015-06-21 16:51:01.175262
124	18591546	Retrospective study	Virappane et al., 2008, J. Clin. Oncol.	2015-06-21 16:49:44.396988	2015-06-21 16:51:01.5466
125	23558953		Wu et al., 2013, Cancer Discov	2015-06-21 16:49:44.601181	2015-06-21 16:51:01.990756
126	22417203	Retrospective study	Patel et al., 2012, N. Engl. J. Med.	2015-06-21 16:49:44.769119	2015-06-21 16:51:02.604803
127	24797300		Tassara et al., 2014, Blood	2015-06-21 16:49:44.82802	2015-06-21 16:51:03.217907
128	21719597	In vitro study	Balusu et al., 2011, Blood	2015-06-21 16:49:44.862206	2015-06-21 16:51:03.832394
129	24927407		Ehninger et al., 2014, Blood Cancer J	2015-06-21 16:49:44.899317	2015-06-21 16:51:04.223891
130	15659725		Falini et al., 2005, N. Engl. J. Med.	2015-06-21 16:49:45.022511	2015-06-21 16:51:04.754124
131	19966866	Mouse model	Liang et al., 2010, Oncogene	2015-06-21 16:49:45.078878	2015-06-21 16:51:05.283962
132	22923433	Retrospective study	Iyer et al., 2012, Science	2015-06-21 16:49:45.144257	2015-06-21 16:51:05.674847
133	14726504	Retrospective study	Fröhling et al., 2004, J. Clin. Oncol.	2015-06-21 16:49:45.20101	2015-06-21 16:51:06.068022
134	21537333		Schnittger et al., 2011, Leukemia	2015-06-21 16:49:45.260682	2015-06-21 16:51:06.59587
135	20805365	Retrospective study	Schnittger et al., 2010, Blood	2015-06-21 16:49:45.323026	2015-06-21 16:51:07.210863
136	19221039	Retrospective study	Gaidzik et al., 2009, Blood	2015-06-21 16:49:45.383299	2015-06-21 16:51:07.82786
137	18450602	Retrospective study	Schlenk et al., 2008, N. Engl. J. Med.	2015-06-21 16:49:45.536526	2015-06-21 16:51:08.442021
138	17965322	Retrospective study	Bacher et al., 2008, Blood	2015-06-21 16:49:45.683932	2015-06-21 16:51:09.054845
139	11070098		Esteller et al., 2000, N. Engl. J. Med.	2015-06-21 16:49:45.727893	2015-06-21 16:51:09.408987
140	19047294	Retrospective study	Büchner et al., 2009, J. Clin. Oncol.	2015-06-21 16:49:46.036567	2015-06-21 16:51:09.97625
141	24855211		Linch et al., 2014, Blood	2015-06-21 16:49:46.067497	2015-06-21 16:51:10.386949
142	22430270	Retrospective study	Gaidzik et al., 2012, J. Clin. Oncol.	2015-06-21 16:49:46.102965	2015-06-21 16:51:10.898078
143	20026798	Retrospective study	Becker et al., 2010, J. Clin. Oncol.	2015-06-21 16:49:46.180419	2015-06-21 16:51:11.316515
144	16051734	Retrospective study	Döhner et al., 2005, Blood	2015-06-21 16:49:46.223984	2015-06-21 16:51:11.818785
145	24801015		Port et al., 2014, Ann. Hematol.	2015-06-21 16:49:46.41109	2015-06-21 16:51:12.43403
146	19587375	Retrospective study	Schnittger et al., 2009, Blood	2015-06-21 16:49:46.662714	2015-06-21 16:51:12.79391
147	24859829		Tian et al., 2014, Int. J. Hematol.	2015-06-21 16:49:46.787677	2015-06-21 16:51:13.252993
148	19536888	Retrospective study	Renneville et al., 2009, Cancer	2015-06-21 16:49:46.869199	2015-06-21 16:51:13.663618
149	18559874	Retrospective study	Paschka et al., 2008, J. Clin. Oncol.	2015-06-21 16:49:46.986157	2015-06-21 16:51:14.029247
150	23346317	Review	Goff, 2013, J Gynecol Oncol	2015-06-21 16:49:47.25683	2015-06-21 16:51:14.430118
151	25212276		Schmidt et al., 2014, Leukemia	2015-06-21 16:49:47.37154	2015-06-21 16:51:15.197996
152	11290608	Retrospective study	Yamamoto et al., 2001, Blood	2015-06-21 16:49:47.412952	2015-06-21 16:51:15.814331
153	18794081	Retrospective study	Riely et al., 2008, Clin. Cancer Res.	2015-06-21 16:49:47.452583	2015-06-21 16:51:16.187421
154	23029227		Qian et al., 2012, PLoS ONE	2015-06-21 16:49:47.522058	2015-06-21 16:51:16.735392
155	14612939		Oshimo et al., 2003, Int. J. Oncol.	2015-06-21 16:49:47.584151	2015-06-21 16:51:17.348089
156	20537386		An et al., 2010, Leuk. Res.	2015-06-21 16:49:47.617344	2015-06-21 16:51:18.271218
157	15961768		Arnold et al., 2005, J. Clin. Oncol.	2015-06-21 16:49:47.647939	2015-06-21 16:51:18.884112
158	20376086	Retrospective study	Ho et al., 2010, Leukemia	2015-06-21 16:49:47.708394	2015-06-21 16:51:19.249836
159	20538800	Retrospective study	Abbas et al., 2010, Blood	2015-06-21 16:49:47.739317	2015-06-21 16:51:19.677349
160	10485475	Retrospective study	Taniguchi et al., 1999, Cancer Res.	2015-06-21 16:49:47.77508	2015-06-21 16:51:20.113639
161	23014527	Retrospective study	Dogan et al., 2012, Clin. Cancer Res.	2015-06-21 16:49:47.844703	2015-06-21 16:51:20.533915
162	24736073		Lim et al., 2014, J Thorac Oncol	2015-06-21 16:49:47.91783	2015-06-21 16:51:21.035518
163	23888070		Beaver et al., 2013, Clin. Cancer Res.	2015-06-21 16:49:47.990264	2015-06-21 16:51:21.466095
164	12476305		Nimmanapalli et al., 2002, Oncogene	2015-06-21 16:49:48.029271	2015-06-21 16:51:21.957211
165	15194504	Cell Line	Yamamoto et al., 2004, Biochem. Biophys. Res. Commun.	2015-06-21 16:49:48.060618	2015-06-21 16:51:22.570716
166	20979473	Clinical Trial	Choi et al., 2010, N. Engl. J. Med.	2015-06-21 16:49:48.197983	2015-06-21 16:51:23.195467
167	25668228	Review	Denis et al., 2015, Clin. Chim. Acta	2015-06-21 16:49:48.289464	2015-06-21 16:51:23.799138
168	15728811		Kobayashi et al., 2005, N. Engl. J. Med.	2015-06-21 16:49:48.35348	2015-06-21 16:51:24.415568
169	21430269		Sequist et al., 2011, Sci Transl Med	2015-06-21 16:49:48.386719	2015-06-21 16:51:25.09034
170	23220880		Bose et al., 2013, Cancer Discov	2015-06-21 16:49:48.43127	2015-06-21 16:51:25.470829
171	24185512		Toy et al., 2013, Nat. Genet.	2015-06-21 16:49:48.469719	2015-06-21 16:51:25.950997
172	22368270	Clinical trial	Man et al., 2012, Blood	2015-06-21 16:49:48.688534	2015-06-21 16:51:26.353394
173	18230792	In vitro study	Zhang et al., 2008, J. Natl. Cancer Inst.	2015-06-21 16:49:48.724393	2015-06-21 16:51:26.872773
174	16618717	Retrospective study	Lièvre et al., 2006, Cancer Res.	2015-06-21 16:49:48.801515	2015-06-21 16:51:27.224155
175	19794967		Marchetti et al., 2009, Neoplasia	2015-06-21 16:49:48.838093	2015-06-21 16:51:28.103884
176	19223544		Sartore-Bianchi et al., 2009, Cancer Res.	2015-06-21 16:49:49.050964	2015-06-21 16:51:28.495587
177	23079656		Sawai et al., 2012, Cancer Cell	2015-06-21 16:49:49.284702	2015-06-21 16:51:29.024012
178	25002028		Vora et al., 2014, Cancer Cell	2015-06-21 16:49:49.319182	2015-06-21 16:51:29.440713
179	22328973		Hammerman et al., 2011, Cancer Discov	2015-06-21 16:49:49.386596	2015-06-21 16:51:29.995022
180	16730237		Ji et al., 2006, Cancer Cell	2015-06-21 16:49:49.668471	2015-06-21 16:51:30.394195
181	15118125		Paez et al., 2004, Science	2015-06-21 16:49:49.702333	2015-06-21 16:51:30.866305
182	24457318		Fukihara et al., 2014, Oncology	2015-06-21 16:49:49.733586	2015-06-21 16:51:31.2688
183	24636847		Li et al., 2014, Lung Cancer	2015-06-21 16:49:49.799284	2015-06-21 16:51:31.644053
184	24658966		Ai et al., 2014, Amino Acids	2015-06-21 16:49:49.832897	2015-06-21 16:51:32.095523
185	24185510		Robinson et al., 2013, Nat. Genet.	2015-06-21 16:49:50.275966	2015-06-21 16:51:32.759522
186	24550739	Case study	Borad et al., 2014, PLoS Genet.	2015-06-21 16:49:50.474219	2015-06-21 16:51:33.32959
187	14726387	Clinical trial	Smith et al., 2004, Blood	2015-06-21 16:49:50.556628	2015-06-21 16:51:33.939247
188	12357354	In vitro study	Tse et al., 2002, Leukemia	2015-06-21 16:49:50.622406	2015-06-21 16:51:34.551084
189	22624710		Barbacid, 2012, Cancer Cell	2015-06-21 16:49:50.684682	2015-06-21 16:51:35.164621
190	18421059	Observational study	Hodi et al., 2008, J. Clin. Oncol.	2015-06-21 16:49:50.722894	2015-06-21 16:51:35.782533
191	17372901	In vitro study	Antonescu et al., 2007, Int. J. Cancer	2015-06-21 16:49:50.768042	2015-06-21 16:51:36.41529
192	16638875		Prenen et al., 2006, Clin. Cancer Res.	2015-06-21 16:49:50.808249	2015-06-21 16:51:37.010568
193	19029981	In vivo study	Engelman et al., 2008, Nat. Med.	2015-06-21 16:49:50.854046	2015-06-21 16:51:37.354844
194	15758010	Clinical Trial	Hegi et al., 2005, N. Engl. J. Med.	2015-06-21 16:49:50.923525	2015-06-21 16:51:37.713084
195	21558396		Tanaka et al., 2011, Clin. Cancer Res.	2015-06-21 16:49:51.048872	2015-06-21 16:51:38.249499
196	15647370		Kang et al., 2005, Proc. Natl. Acad. Sci. U.S.A.	2015-06-21 16:49:51.08455	2015-06-21 16:51:38.854338
197	8674046		Yoshida et al., 1996, Cancer Res.	2015-06-21 16:49:51.247897	2015-06-21 16:51:39.312124
198	20085938		Courtney et al., 2010, J. Clin. Oncol.	2015-06-21 16:49:51.27922	2015-06-21 16:51:39.685587
199	22698404		Jackson et al., 2012, Cancer Cell	2015-06-21 16:49:51.36807	2015-06-21 16:51:40.187844
200	22923433		Iyer et al., 2012, Science	2015-06-21 16:49:51.406668	2015-06-21 16:51:40.565348
201	20368538	Retrospective study	Wagner et al., 2010, J. Clin. Oncol.	2015-06-21 16:49:51.50625	2015-06-21 16:51:41.004105
202	22616558		Zhou et al., 2012, Leuk. Lymphoma	2015-06-21 16:49:51.624331	2015-06-21 16:51:41.430741
203	20421455	Retrospective study	Thol et al., 2010, Blood	2015-06-21 16:49:51.654693	2015-06-21 16:51:42.232295
204	21997850		Lin et al., 2012, Ann. Hematol.	2015-06-21 16:49:51.683964	2015-06-21 16:51:42.846535
205	22033490		Patnaik et al., 2012, Leukemia	2015-06-21 16:49:51.718563	2015-06-21 16:51:43.18097
206	12000708	Retrospective study	Corless et al., 2002, Am. J. Pathol.	2015-06-21 16:49:51.802874	2015-06-21 16:51:43.770453
207	15217946	Retrospective study	Boldrini et al., 2004, Clin. Cancer Res.	2015-06-21 16:49:51.842046	2015-06-21 16:51:44.382762
208	19934290	Retrospective study	Ogino et al., 2009, Clin. Cancer Res.	2015-06-21 16:49:51.910557	2015-06-21 16:51:44.99911
209	11208838	Retrospective study	Schiller et al., 2001, J. Clin. Oncol.	2015-06-21 16:49:51.941562	2015-06-21 16:51:45.357451
210	17070615		Gautschi et al., 2007, Lung Cancer	2015-06-21 16:49:52.112191	2015-06-21 16:51:46.106909
211	22479427		Carico et al., 2012, PLoS ONE	2015-06-21 16:49:52.144753	2015-06-21 16:51:46.588736
212	23898052		Logan et al., 2013, Anticancer Res.	2015-06-21 16:49:52.211156	2015-06-21 16:51:47.148629
213	24662454		Douillard et al., 2014, J Thorac Oncol	2015-06-21 16:49:52.281083	2015-06-21 16:51:47.580842
214	19933982	Retrospective study	Dubbink et al., 2009, Neurology	2015-06-21 16:49:52.33788	2015-06-21 16:51:48.069492
215	20127344	Retrospective study	Bleeker et al., 2010, Acta Neuropathol.	2015-06-21 16:49:52.367418	2015-06-21 16:51:48.494381
216	21596855		Green et al., 2011, Blood	2015-06-21 16:49:52.39784	2015-06-21 16:51:48.9907
217	21995386		Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-06-21 16:49:52.463675	2015-06-21 16:51:49.391181
218	16309541		Thomas et al., 2005, Int J Exp Pathol	2015-06-21 16:49:52.628688	2015-06-21 16:51:49.916325
219	17891190		Jares et al., 2007, Nat. Rev. Cancer	2015-06-21 16:49:52.662683	2015-06-21 16:51:50.374514
220	10547574		Takano et al., 1999, J. Pathol.	2015-06-21 16:49:52.695617	2015-06-21 16:51:50.833703
221	12432043		Keyomarsi et al., 2002, N. Engl. J. Med.	2015-06-21 16:49:52.816366	2015-06-21 16:51:51.237145
222	10224221		Donnellan et al., 1999, FASEB J.	2015-06-21 16:49:52.845575	2015-06-21 16:51:52.051031
223	23111194		Bradly et al., 2012, Diagn. Mol. Pathol.	2015-06-21 16:49:52.933653	2015-06-21 16:51:52.575639
224	24623981		Ding et al., 2014, Onco Targets Ther	2015-06-21 16:49:52.994395	2015-06-21 16:51:52.987395
225	24729716		Li et al., 2014, Onco Targets Ther	2015-06-21 16:49:53.024419	2015-06-21 16:51:53.352009
226	11585760	Retrospective study	Whitman et al., 2001, Cancer Res.	2015-06-21 16:49:53.091999	2015-06-21 16:51:53.776678
227	17940205	Retrospective study	Whitman et al., 2008, Blood	2015-06-21 16:49:53.123377	2015-06-21 16:51:54.523484
228	20494930	Retrospective study	Thol et al., 2010, Haematologica	2015-06-21 16:49:53.157611	2015-06-21 16:51:55.135332
229	18528420	Retrospective study	Chng et al., 2008, Leukemia	2015-06-21 16:49:53.250083	2015-06-21 16:51:55.491816
230	15597105	Metaanalysis	Mascaux et al., 2005, Br. J. Cancer	2015-06-21 16:49:53.280369	2015-06-21 16:51:56.059576
231	20007775		Westhoff et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-06-21 16:49:53.339273	2015-06-21 16:51:56.67128
232	22357840		Liao et al., 2012, Clin. Cancer Res.	2015-06-21 16:49:53.466469	2015-06-21 16:51:57.59442
233	16489069		Olivier et al., 2006, Clin. Cancer Res.	2015-06-21 16:49:53.643744	2015-06-21 16:51:58.002708
234	9569050		Berns et al., 1998, Br. J. Cancer	2015-06-21 16:49:53.708722	2015-06-21 16:51:58.414736
235	23861105		Wu et al., 2013, Am. J. Hematol.	2015-06-21 16:49:53.985086	2015-06-21 16:51:58.822188
236	23524406	case report	Rudin et al., 2013, J Thorac Oncol	2015-06-21 16:49:54.050565	2015-06-21 16:51:59.176214
237	24186003		Lilljebjörn et al., 2014, Leukemia	2015-06-21 16:49:54.086981	2015-06-21 16:51:59.743641
238	24578576	Retrospective study	Honeyman et al., 2014, Science	2015-06-21 16:49:54.16115	2015-06-21 16:52:00.358161
239	12075054	Retrospective Study	Rosenwald et al., 2002, N. Engl. J. Med.	2015-06-21 16:49:54.225651	2015-06-21 16:52:00.973389
240	18765795	Retrospective Study	Lenz et al., 2008, Proc. Natl. Acad. Sci. U.S.A.	2015-06-21 16:49:54.25669	2015-06-21 16:52:01.386778
241	21343560	Retrospective study	Gaidzik et al., 2011, J. Clin. Oncol.	2015-06-21 16:49:54.35774	2015-06-21 16:52:01.894937
242	25670080		Vidal et al., 2015, Cancer Cell	2015-06-21 16:49:54.395789	2015-06-21 16:52:02.288577
243	16818686	Retrospective study	Jackman et al., 2006, Clin. Cancer Res.	2015-06-21 16:49:54.427461	2015-06-21 16:52:02.81491
244	25670080	PDX	Vidal et al., 2015, Cancer Cell	2015-06-21 16:49:54.466156	2015-06-21 16:52:03.430413
245	22430270	Retrospective Study	Gaidzik et al., 2012, J. Clin. Oncol.	2015-06-21 16:49:54.506509	2015-06-21 16:52:04.043999
246	19157502	Retrospective Study	Mendiola et al., 2009, Hum. Pathol.	2015-06-21 16:49:54.546271	2015-06-21 16:52:04.509381
247	19666869	Retrospective study	Kosmider et al., 2009, Blood	2015-06-21 16:49:54.582596	2015-06-21 16:52:04.965897
248	18464292	Retrospective Study	Patard et al., 2008, Int. J. Cancer	2015-06-21 16:49:54.617628	2015-06-21 16:52:05.409024
249	23620406	Retrospective Study	Hakimi et al., 2013, Clin. Cancer Res.	2015-06-21 16:49:54.684737	2015-06-21 16:52:05.886816
250	21343549	Retrospective Study	Metzeler et al., 2011, J. Clin. Oncol.	2015-06-21 16:49:54.71597	2015-06-21 16:52:06.296041
251	9207459	Retrospective Study	Gascoyne et al., 1997, Blood	2015-06-21 16:49:54.808408	2015-06-21 16:52:06.726342
252	20601954	Retrospective Study	Nikoloski et al., 2010, Nat. Genet.	2015-06-21 16:49:54.877254	2015-06-21 16:52:07.118747
253	21828143	Retrospective study	Chou et al., 2011, Blood	2015-06-21 16:49:54.911719	2015-06-21 16:52:07.493975
254	11793370	Retrospective Study	Schraml et al., 2002, J. Pathol.	2015-06-21 16:49:54.971544	2015-06-21 16:52:08.036404
255	9552022	Retrospective study	de Alava et al., 1998, J. Clin. Oncol.	2015-06-21 16:49:55.066015	2015-06-21 16:52:08.651795
\.


--
-- Data for Name: variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variants (id, gene_id, name, description, created_at, updated_at, deleted, deleted_at, genome_build, chromosome, start, stop, reference_bases, variant_bases, representative_transcript, chromosome2, start2, stop2) FROM stdin;
1	4	BCR-ABL	The BCR-ABL fusion protein, commonly referred to as the philadelphia chromosome, is the most well-studied fusion gene in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML), but despite its ability initiate disease in mice, its status an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and nilotinib) have seen some success in delivering a tumor response.	2015-06-21 16:49:35.188256	2015-06-21 16:49:35.188256	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
2	4	BCR-ABL T315I	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-06-21 16:49:35.211187	2015-06-21 16:49:35.211187	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
3	4	BCR-ABL E255K	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-06-21 16:49:35.228538	2015-06-21 16:49:35.228538	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
5	1	EML4-ALK	The EML4-ALK fusion has been seen in non-small cell lung cancer, and appears to be an alternative mechanism for ALK activation. Cells with this fusion have been shown to be sensitive to the ALK inhibitor crizotinib.	2015-06-21 16:49:35.26486	2015-06-21 16:49:35.26486	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
6	1	EML4-ALK C1156Y	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-06-21 16:49:35.282437	2015-06-21 16:49:35.282437	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
7	1	EML4-ALK L1196M	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-06-21 16:49:35.299524	2015-06-21 16:49:35.299524	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
13	5	V600E/V600M	A case study of a single patient harboring both a V600E and a V600M mutation, dabrafenib was shown to acheive clinical response.	2015-06-21 16:49:35.42524	2015-06-21 16:49:35.42524	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
14	5	V600E AMPLIFICATION	Amplification of BRAF V600E has been shown to confer resistance to MEK inhibitors. For more information on the V600 locus, see the V600E entry.	2015-06-21 16:49:35.443384	2015-06-21 16:49:35.443384	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
17	5	V600	BRAF mutations of the valine 600 residue have been shown to be recurrent in many cancer types. Of the V600 mutations, V600E is the most widely researched. V600 mutations as a whole have been correlated to poorer prognosis in colorectal and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual mutation pages on the left sidebar.	2015-06-21 16:49:35.50209	2015-06-21 16:49:35.50209	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
18	8	AMPLIFICATION	CCND1 amplification has been implicated in poorer prognosis in non-small cell lung cancer.	2015-06-21 16:49:35.520602	2015-06-21 16:49:35.520602	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
19	8	EXPRESSION	CCND1 expression, and its prognositc impact, is still in dispute. Three experiments in non-small cell lung cancer have shown it to have no impact on survival, but three additional studies have shown it results in poorer prognosis. There is also some ambiguity in how the boundaries between expression and overexpression are defined.	2015-06-21 16:49:35.542226	2015-06-21 16:49:35.542226	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
20	8	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-06-21 16:49:35.561572	2015-06-21 16:49:35.561572	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
21	9	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-06-21 16:49:35.582202	2015-06-21 16:49:35.582202	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
22	9	PROMOTER DEMETHYLATION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-06-21 16:49:35.601716	2015-06-21 16:49:35.601716	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
23	10	LOSS	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL), a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic palbociclib significantly increased the median survival of a Notch-driven model of T-ALL.	2015-06-21 16:49:35.621802	2015-06-21 16:49:35.621802	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
24	11	OVEREXPRESSION	Cyclin E, while currenly not as widely implicated as its counterpart, cyclin D, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, are correlated with later stage disease and have been shown to lead to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	2015-06-21 16:49:35.644205	2015-06-21 16:49:35.644205	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
25	13	EXPRESSION	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-06-21 16:49:35.66506	2015-06-21 16:49:35.66506	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
26	12	EXPRESSION	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-06-21 16:49:35.686985	2015-06-21 16:49:35.686985	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
27	14	PROMOTER HYPERMETHYLATION	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the CDKN2A promoter region. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcomes for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	2015-06-21 16:49:35.706867	2015-06-21 16:49:35.706867	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
28	15	N-TERMINAL FRAME SHIFT	CEBPA N-terminal frame shift mutations that result in a premature stop codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML). CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-06-21 16:49:35.726193	2015-06-21 16:49:35.726193	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
29	15	MUTATION	CEBPA mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-06-21 16:49:35.744576	2015-06-21 16:49:35.744576	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
30	33	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-06-21 16:49:35.765157	2015-06-21 16:49:35.765157	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
31	16	DNAJB1-PRKACA	This fusion has been found to be very recurrent in the rare form of adolescent liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-06-21 16:49:35.78572	2015-06-21 16:49:35.78572	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
32	18	R882	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-06-21 16:49:35.808944	2015-06-21 16:49:35.808944	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
74	30	EXON 1 MUTATIONS	A study by Lièvre et al in 2006 showed that colorectal cancer patients with KRAS exon 1 mutations had low cetuximab response rates.	2015-06-21 16:49:36.780526	2015-06-21 16:49:36.780526	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
37	20	DEL 755-759	ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-06-21 16:49:35.941913	2015-06-21 16:49:35.941913	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
40	20	L755W	ERBB2 L755W was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-06-21 16:49:36.002514	2015-06-21 16:49:36.002514	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
41	20	P780INS	ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-06-21 16:49:36.021845	2015-06-21 16:49:36.021845	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
42	20	R678Q	ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-06-21 16:49:36.042574	2015-06-21 16:49:36.042574	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
75	30	EXON 2 MUTATIONS	In a study by Pao et al in 2005, non-small cell lung cancer patients harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors gefinitib and erlotinib.	2015-06-21 16:49:36.808061	2015-06-21 16:49:36.808061	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
46	21	L536Q	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of these ligand binding domain mutations, and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-06-21 16:49:36.11656	2015-06-21 16:49:36.11656	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
47	21	N538G	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-06-21 16:49:36.135445	2015-06-21 16:49:36.135445	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
51	22	P235R		2015-06-21 16:49:36.212889	2015-06-21 16:49:36.212889	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
52	22	FGFR2-MGEA5	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. Additionally, tumor size reduction was achieved by both ponatinib and pazopanib treatments administered separately in a single patient with intrahepatic cholangiocarcinoma and this fusion. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-06-21 16:49:36.231483	2015-06-21 16:49:36.231483	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
53	23	FGFR3-BAIAP2L1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-06-21 16:49:36.250363	2015-06-21 16:49:36.250363	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
54	23	FGFR3-TACC3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-06-21 16:49:36.268728	2015-06-21 16:49:36.268728	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
55	24	ITD MUTATIONS	FLT3-ITD (internal tandem duplications) frequently occur in patients with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated sequence resulting in an in-frame duplication event. The length of these duplications can vary widely which may have prognostic consequences, but this has not been conclusively determined. FLT3-ITD mutations overall have generally been associated with poor prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the prognosis associated with this variant.	2015-06-21 16:49:36.302883	2015-06-21 16:49:36.302883	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
56	24	TKD MUTATIONS	FLT3 tyrosine kinase domain mutations (FLT3-TKD) are much less common than FLT3-ITD (internal tandem duplication) mutations and may not confer the same prognostic impact. Although the majority of mutations are point mutations effecting D835 (most frequently D835Y), a small proportion involve an in-frame deletion of I836. These mutations are within the activation loop of the second tyrosine kinase domain of FLT3 and thought to result in constitutive activation of the receptor.	2015-06-21 16:49:36.322373	2015-06-21 16:49:36.322373	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
57	25	EXPRESSION	GATA2 misregulation has been observed in a number of hematologic malignancies, as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung cancer.	2015-06-21 16:49:36.342462	2015-06-21 16:49:36.342462	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
76	30	G12	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-06-21 16:49:36.846551	2015-06-21 16:49:36.846551	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
58	26	R132	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-06-21 16:49:36.366199	2015-06-21 16:49:36.366199	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
62	27	R140Q/L	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival as compared to their wild-type counterparts. In myelodysplastic syndromes, however, no prognostic link was discovered between IDH2 mutation status and overall survival.	2015-06-21 16:49:36.450632	2015-06-21 16:49:36.450632	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
66	29	EXON 11 MUTATIONS	c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-06-21 16:49:36.541863	2015-06-21 16:49:36.541863	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
67	29	INTERNAL DUPLICATION	c-KIT internal duplications have been observed in exon 11, within the juxtamembrane domain. In a case study of an anal melanoma patient harboring this event, imatinib confered marked response. Also, cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-06-21 16:49:36.567356	2015-06-21 16:49:36.567356	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
68	29	EXON 13 MUTATIONS	c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations. These mutations lie within the tyrosine kinase 1 domain, and are found primarily in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations show sensitivity to imatinib and sunitinib treatment. However, only imatinib has seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant cases.	2015-06-21 16:49:36.596036	2015-06-21 16:49:36.596036	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
69	29	EXON 14 MUTATIONS	c-KIT exon 14 mutations lie within the tyrosine kinase domain of the protein. While relatively rare in primary gastrointestinal tumors, they are notably more prevalent in refractory disease, suggesting a role in imatinib resistance. Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.	2015-06-21 16:49:36.627031	2015-06-21 16:49:36.627031	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
70	29	EXON 17 MUTATIONS	c-KIT exon 17 mutations lie within the TK2 domain, containing the activation loop of the protein. In cell lines, mutations within this domain have been shown to be sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.	2015-06-21 16:49:36.666903	2015-06-21 16:49:36.666903	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
71	29	EXON 9 MUTATIONS	c-KIT exon 9 mutations lie within the dimerization motif of the protein. Relative to other KIT mutations, exon 9 mutations have been associated with better overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and in vivo.	2015-06-21 16:49:36.694357	2015-06-21 16:49:36.694357	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
72	29	L576P	KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain. It is one of the most recurrent KIT mutations in melanoma, and both in vitro and in vivo studies have shown sensitivity to imatinib.	2015-06-21 16:49:36.723668	2015-06-21 16:49:36.723668	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
77	30	G12/G13	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-06-21 16:49:36.864674	2015-06-21 16:49:36.864674	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
80	30	G13	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-06-21 16:49:36.921746	2015-06-21 16:49:36.921746	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
84	32	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-06-21 16:49:37.00067	2015-06-21 16:49:37.00067	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
85	34	PROMOTER METHYLATION	MGMT promoter methylation has been observed to impact tumor progression in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating agent carmustine has shown efficacy. In patients lacking methylation, combining carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more experiments are required. Clinical trials have also shown selective sensitivity of promoter methylation-positive patients to temozolomide, making a case for wider methylation screening in GBM patients.	2015-06-21 16:49:37.021277	2015-06-21 16:49:37.021277	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
86	35	EXON 12 MUTATIONS	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-06-21 16:49:37.044383	2015-06-21 16:49:37.044383	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
88	36	EXON 1 MUTATIONS	NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown to be correlated with poorer overall survival relative to wild-type NRAS in melanoma patients.	2015-06-21 16:49:37.095276	2015-06-21 16:49:37.095276	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
89	36	EXON 2 MUTATIONS	NRAS exon 2 mutations have been shown to be correlated with poorer overall survival in melanoma patients and colorectal cancer patients, however no prognostic impact was seen in acute myeloid leukemia patients.	2015-06-21 16:49:37.122358	2015-06-21 16:49:37.122358	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
90	36	EXON 3 MUTATIONS	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-06-21 16:49:37.146931	2015-06-21 16:49:37.146931	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
91	36	EXON 4 MUTATIONS	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-06-21 16:49:37.172145	2015-06-21 16:49:37.172145	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
92	36	G12	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-06-21 16:49:37.196403	2015-06-21 16:49:37.196403	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
94	36	Q61	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-06-21 16:49:37.238263	2015-06-21 16:49:37.238263	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
111	40	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and Arsenic Trioxide for treating these patients, and early results seem promising.	2015-06-21 16:49:37.579249	2015-06-21 16:49:37.579249	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
97	38	V536E		2015-06-21 16:49:37.294065	2015-06-21 16:49:37.294065	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
101	38	DEL I843	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-06-21 16:49:37.380649	2015-06-21 16:49:37.380649	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
102	38	DI842-843IM	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-06-21 16:49:37.398986	2015-06-21 16:49:37.398986	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
105	37	EXON 20 MUTATIONS	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-06-21 16:49:37.45735	2015-06-21 16:49:37.45735	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
106	37	EXON 9 MUTATIONS	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-06-21 16:49:37.477559	2015-06-21 16:49:37.477559	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
108	39	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	2015-06-21 16:49:37.519761	2015-06-21 16:49:37.519761	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
109	17	DNAJB1-PRKACA	This fusion has been found to be very recurrent in a rare form of adolescent liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined, and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-06-21 16:49:37.538549	2015-06-21 16:49:37.538549	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
123	45	V173G/A	While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-06-21 16:49:37.827717	2015-06-21 16:49:37.827717	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
124	46	FRAMESHIFT TRUNCATION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-06-21 16:49:37.870046	2015-06-21 16:49:37.870046	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
125	46	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-06-21 16:49:37.896331	2015-06-21 16:49:37.896331	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
126	47	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-06-21 16:49:37.921438	2015-06-21 16:49:37.921438	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
4	2	E17K	AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal, lung, and ovarian cancer. It has been convincingly shown to be an activating mutation resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and other AKT1 mutations, are the subject of much research and development for therapeutics.	2015-06-21 16:49:35.246704	2015-06-21 16:52:22.645522	f	\N	\N	14	105246551	105246551	C	T	\N	\N	\N	\N
127	48	Q157P/R	U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is less common than the S34F mutation, occurs in the second zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-06-21 16:49:37.94332	2015-06-21 16:49:37.94332	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
128	48	S34Y/F	U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is the most commonly identified variant in MDS, occurs in the first zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-06-21 16:49:37.970728	2015-06-21 16:49:37.970728	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
129	49	EXON 7 MUTATIONS	WT1 exon 7 mutations have been shown to be recurrent in acute myeloid leukemia. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 7 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-06-21 16:49:37.993948	2015-06-21 16:49:37.993948	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
130	49	EXON 9 MUTATIONS	WT1 exon 9 mutations have been shown to be recurrent in acute myeloid leukemia, although at a less frequent rate than their exon 7 counterparts. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 9 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-06-21 16:49:38.014284	2015-06-21 16:49:38.014284	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
131	6	LOSS-OF-FUNCTION	BRCA1 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.	2015-06-21 16:49:38.037296	2015-06-21 16:49:38.037296	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
132	7	LOSS-OF-FUNCTION	BRCA2 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.	2015-06-21 16:49:38.055713	2015-06-21 16:49:38.055713	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
133	19	EXON 19 DELETION	Deletions within exon 19 of EGFR are most common in lung cancer. These deletions, in non-small cell lung cancer, have been shown to be sensitive to the EGFR tyrosine kinase inhibitors gefitinib and erlotinib. There is also data to suggest that this event is a good prognostic marker in lung adenocarcinoma.	2015-06-21 16:49:38.073628	2015-06-21 16:49:38.073628	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
134	19	G719S	While not as recurrent as the L858R mutation, EGFR G719S has also been shown to be an activating mutation. In lung cancer cell lines, it also confers sensitivity to the tyrosine kinase inhibitors gefitinib and erlotinib.	2015-06-21 16:49:38.09419	2015-06-21 16:49:38.09419	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
135	50	D1643H	Activating mutations in NOTCH1, including D1643H, have been shown to be poor prognostic markers in lung cancer.	2015-06-21 16:49:38.113459	2015-06-21 16:49:38.113459	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
136	50	R2328W	Activating mutations in NOTCH1, including R2328W, have been shown to be poor prognostic markers in lung cancer.	2015-06-21 16:49:38.132187	2015-06-21 16:49:38.132187	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
137	50	V2444FS	Activating mutations in NOTCH1, including a frameshift insertion at V2444, have been shown to be poor prognostic markers in lung cancer.	2015-06-21 16:49:38.150779	2015-06-21 16:49:38.150779	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
138	50	S2275FS	Activating mutations in NOTCH1, including a frameshift insertion at S2275, have been shown to be poor prognostic markers in lung cancer.	2015-06-21 16:49:38.168733	2015-06-21 16:49:38.168733	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
139	51	L63V	Activating mutations in DDR2, including L63V, has been shown to be sensitive to dasatinib in cell lines.	2015-06-21 16:49:38.186862	2015-06-21 16:49:38.186862	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
140	51	L239R	Activating mutations in DDR2, including L239R, has been shown to be sensitive to dasatinib in cell lines.	2015-06-21 16:49:38.207225	2015-06-21 16:49:38.207225	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
141	51	G253C	Activating mutations in DDR2, including G253C, has been shown to be sensitive to dasatinib in cell lines.	2015-06-21 16:49:38.224703	2015-06-21 16:49:38.224703	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
142	51	G505S	Activating mutations in DDR2, including G505S, has been shown to be sensitive to dasatinib in cell lines.	2015-06-21 16:49:38.242044	2015-06-21 16:49:38.242044	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
143	51	I638F	DDR2 I638F has been shown to be a loss of function mutation, but also confers sensitivity to dasatinib in cell lines.	2015-06-21 16:49:38.263452	2015-06-21 16:49:38.263452	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
144	51	G774V	Activating mutations in DDR2, including G774V, has been shown to be sensitive to dasatinib in cell lines.	2015-06-21 16:49:38.282281	2015-06-21 16:49:38.282281	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
145	51	S768R	Activating mutations in DDR2, including S768R, has been shown to be sensitive to dasatinib in cell lines.	2015-06-21 16:49:38.300555	2015-06-21 16:49:38.300555	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
146	52	AMPLIFICATION	MET amplification, like EGFR T790M, has been shown to be capable of driving acquired resistance to dacomitinib in patients with lung adenocarcinoma.	2015-06-21 16:49:38.319059	2015-06-21 16:49:38.319059	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
147	30	G12V	KRAS G12V, like EGFR T790M, has been shown to be capable of driving acquired resistance to tyrosine kinase inhibitors in lung adenocarcinoma.	2015-06-21 16:49:38.354822	2015-06-21 16:49:38.354822	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
148	30	G12A	KRAS G12A, like EGFR T790M, has been shown to be capable of driving acquired resistance to tyrosine kinase inhibitors in lung adenocarcinoma.	2015-06-21 16:49:38.37339	2015-06-21 16:49:38.37339	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
149	54	EWS-FLI1		2015-06-21 16:49:38.39102	2015-06-21 16:49:38.39102	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
150	53	EWS-FLI1		2015-06-21 16:49:38.408783	2015-06-21 16:49:38.408783	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
151	59	T(14;18)		2015-06-21 16:49:38.427239	2015-06-21 16:49:38.427239	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
152	59	EXPRESSION		2015-06-21 16:49:38.447142	2015-06-21 16:49:38.447142	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
153	56	AMPLIFICATION		2015-06-21 16:49:38.465479	2015-06-21 16:49:38.465479	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
154	57	AMPLIFICATION		2015-06-21 16:49:38.484974	2015-06-21 16:49:38.484974	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
155	43	MUTATION		2015-06-21 16:49:38.502648	2015-06-21 16:49:38.502648	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
156	60	OVEREXPRESSION		2015-06-21 16:49:38.520481	2015-06-21 16:49:38.520481	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
157	55	MUTATION		2015-06-21 16:49:38.538366	2015-06-21 16:49:38.538366	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
158	61	EXPRESSION		2015-06-21 16:49:38.555971	2015-06-21 16:49:38.555971	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
159	58	LOSS OF FUNCTION		2015-06-21 16:49:38.573536	2015-06-21 16:49:38.573536	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
160	58	MUTATION		2015-06-21 16:49:38.591589	2015-06-21 16:49:38.591589	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
161	62	MUTATION		2015-06-21 16:49:38.613595	2015-06-21 16:49:38.613595	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
162	57	EXPRESSION		2015-06-21 16:49:38.631208	2015-06-21 16:49:38.631208	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
163	63	MUTATION		2015-06-21 16:49:38.64887	2015-06-21 16:49:38.64887	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
164	53	TYPE 1 EWS-FLI1		2015-06-21 16:49:38.66683	2015-06-21 16:49:38.66683	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
8	1	F1174L	ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells.	2015-06-21 16:49:35.317639	2015-06-21 16:52:22.665204	f	\N	\N	2	29443695	29443695	G	T	\N	\N	\N	\N
9	1	R1275Q	ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma, non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This, and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC cell lines.	2015-06-21 16:49:35.335371	2015-06-21 16:52:22.677684	f	\N	\N	2	29432664	29432664	C	T	\N	\N	\N	\N
10	3	S214C	ARAF S214C has been found to be a recurrent oncogenic mutation in non-small cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this mutation, sorafenib also acheived near-complete clinical remission. This case has brought more interest to the variant from a research and clinical perspective.	2015-06-21 16:49:35.356664	2015-06-21 16:52:22.689892	f	\N	\N	X	47426121	47426121	C	G	\N	\N	\N	\N
11	5	V600D	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-06-21 16:49:35.375983	2015-06-21 16:52:22.701896	f	\N	\N	7	140453135	140453136	CA	AT	\N	\N	\N	\N
12	5	V600E	BRAF V600E has been shown to be recurrent in many cancer types. It is one of the most widely studied variants in cancer. This variant is correlated with poor prognosis in certain cancer types, including colorectal cancer and papillary thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib has also shown to be effective when combined with the MEK inhibitor trametinib in colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib, and a number of combination therapies have been successful in treating V600E mutations. While the drugs cetuximab and panitumumab have been largely shown to be ineffective without supplementary treatment.	2015-06-21 16:49:35.406709	2015-06-21 16:52:22.715367	f	\N	\N	7	140453136	140453136	A	T	\N	\N	\N	\N
15	5	V600M	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-06-21 16:49:35.462465	2015-06-21 16:52:22.729528	f	\N	\N	7	140453137	140453137	C	T	\N	\N	\N	\N
16	5	V600R	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-06-21 16:49:35.483587	2015-06-21 16:52:22.744993	f	\N	\N	7	140453136	140453137	AC	CT	\N	\N	\N	\N
33	19	L858R	EGFR L858R has long been recognized as a functionally significant mutation in cancer, and is one of the most prevalent single mutations in lung cancer. Best described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity to first and second generation TKI's like gefitinib and neratinib. NSCLC patients with this mutation treated with TKI's show increased overall and progression-free survival, as compared to chemotherapy alone. Third generation TKI's are currently in clinical trials that specifically focus on mutant forms of EGFR, a few of which have shown efficacy in treating patients that failed to respond to earlier generation TKI therapies.	2015-06-21 16:49:35.835095	2015-06-21 16:52:22.764307	f	\N	\N	7	55259514	55259515	CT	AG	\N	\N	\N	\N
34	19	T790M	EGFR T790M was one of the very first mutations recognized to confer resistance to targeted therapies in non-small cell lung cancer. While successful in amplified EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients harboring this mutation before treatment is notably lower. This lack of efficacy can likely be to blame for the poorer prognosis for patients with this mutation as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired resistance to TKI inhibition have been shown to harbor this mutation, implicating it as a mechanism of acquired therapy resistence. The third generation TKI's are being developed with this resistance problem in mind, and early stage experiments have shown some efficacy of these drugs in previously resistant tumors with mutant EGFR.	2015-06-21 16:49:35.873431	2015-06-21 16:52:22.778149	f	\N	\N	7	55249071	55249071	C	T	\N	\N	\N	\N
35	20	D769H	ERBB2 D769H was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-06-21 16:49:35.897862	2015-06-21 16:52:22.790491	f	\N	\N	17	37880261	37880261	G	C	\N	\N	\N	\N
36	20	D769Y	ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-06-21 16:49:35.921516	2015-06-21 16:52:22.80407	f	\N	\N	17	37880261	37880261	G	T	\N	\N	\N	\N
38	20	G309A	ERBB2 G309A was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-06-21 16:49:35.964669	2015-06-21 16:52:22.817916	f	\N	\N	17	37868205	37868205	G	C	\N	\N	\N	\N
39	20	L755S	ERBB2 L755S was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike many of the other variants queried. This mutation was also shown to confer resistance to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.	2015-06-21 16:49:35.983784	2015-06-21 16:52:22.830975	f	\N	\N	17	37880220	37880220	T	C	\N	\N	\N	\N
43	20	R896C	ERBB2 R896C was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-06-21 16:49:36.060373	2015-06-21 16:52:22.845316	f	\N	\N	17	37881616	37881616	C	T	\N	\N	\N	\N
44	20	V777L	ERBB2 V777L was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-06-21 16:49:36.077968	2015-06-21 16:52:22.858832	f	\N	\N	17	37881000	37881000	G	T	\N	\N	\N	\N
45	20	V842I	ERBB2 V842I was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-06-21 16:49:36.096035	2015-06-21 16:52:22.875012	f	\N	\N	17	37881332	37881332	G	A	\N	\N	\N	\N
48	21	Y537C	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-06-21 16:49:36.153789	2015-06-21 16:52:22.891022	f	\N	\N	6	152419923	152419923	A	G	\N	\N	\N	\N
49	21	Y537N	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-06-21 16:49:36.171778	2015-06-21 16:52:22.908988	f	\N	\N	6	152419922	152419922	T	A	\N	\N	\N	\N
50	21	Y537S	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-06-21 16:49:36.189579	2015-06-21 16:52:22.925958	f	\N	\N	6	152419923	152419923	A	C	\N	\N	\N	\N
59	26	R132C	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-06-21 16:49:36.385194	2015-06-21 16:52:22.945182	f	\N	\N	2	209113113	209113113	G	A	\N	\N	\N	\N
60	26	R132H	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-06-21 16:49:36.406788	2015-06-21 16:52:22.96221	f	\N	\N	2	209113112	209113112	C	T	\N	\N	\N	\N
61	26	R132L	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-06-21 16:49:36.429221	2015-06-21 16:52:22.97992	f	\N	\N	2	209113112	209113112	C	A	\N	\N	\N	\N
63	27	R172K	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic syndromes, studies did not find a prognostic association between this variant and patient outcomes.	2015-06-21 16:49:36.471494	2015-06-21 16:52:22.997474	f	\N	\N	15	90631838	90631838	C	T	\N	\N	\N	\N
64	28	V617F	JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases, occuring in around half of all MPD's. While less associated with cancer, when it is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The V617F mutation is an activating mutation, resulting in increased kinase activity. The mutation seems to be restricted to hematologic malignancies. Treatment of JAK mutant diseases with ruxolitinib has seen some clinical success.	2015-06-21 16:49:36.491169	2015-06-21 16:52:23.014434	f	\N	\N	9	5073770	5073770	G	T	\N	\N	\N	\N
65	29	D816V	KIT D816V is a mutation observed in acute myeloid leukemia (AML). This variant has been linked to poorer prognosis and worse outcome in AML patients.	2015-06-21 16:49:36.516358	2015-06-21 16:52:23.029588	f	\N	\N	4	55599321	55599321	A	T	\N	\N	\N	\N
73	29	V654A	KIT V654A is an exon 13 mutation that lies within the tyrosine kinase 1 domain of the protein. It has been shown to be an activating mutation by in vitro studies. This mutation is associated with imatinib resistance in melanoma patients. However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have seen success in acheiving tumor response.	2015-06-21 16:49:36.75555	2015-06-21 16:52:23.046594	f	\N	\N	4	55594258	55594258	T	C	\N	\N	\N	\N
78	30	G12C	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-06-21 16:49:36.88316	2015-06-21 16:52:23.060538	f	\N	\N	12	25398285	25398285	C	A	\N	\N	\N	\N
79	30	G12D	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-06-21 16:49:36.902451	2015-06-21 16:52:23.072342	f	\N	\N	12	25398284	25398284	C	T	\N	\N	\N	\N
81	30	G13D	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-06-21 16:49:36.943551	2015-06-21 16:52:23.085804	f	\N	\N	12	25398281	25398281	C	T	\N	\N	\N	\N
82	31	P124S	MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder and colon cancer to a lesser degree. The P124S mutation has been shown to contribute to AZD6244 resistance in melanoma cell lines, but considerably less so than its Q56P counterpart.	2015-06-21 16:49:36.964062	2015-06-21 16:52:23.099369	f	\N	\N	15	66729162	66729162	C	T	\N	\N	\N	\N
83	31	Q56P	MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This mutation has been shown to confer considerable resistance to AZD6244 treatment of melanoma cell lines.	2015-06-21 16:49:36.98197	2015-06-21 16:52:23.115809	f	\N	\N	15	66727451	66727451	A	C	\N	\N	\N	\N
87	35	W288FS	NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most common NPM1 mutation identified in acute myeloid leukemia. This mutation results in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation, it is generally grouped with other exon 12 mutations for patient analysis (see NPM1 Exon 12 variants for more information).	2015-06-21 16:49:37.069585	2015-06-21 16:52:23.131382	f	\N	\N	5	170837546	170837547	0	TCAG	\N	\N	\N	\N
93	36	G13D	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-06-21 16:49:37.217906	2015-06-21 16:52:23.147443	f	\N	\N	1	115258744	115258744	C	T	\N	\N	\N	\N
95	36	Q61L	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-06-21 16:49:37.256647	2015-06-21 16:52:23.165264	f	\N	\N	1	115256528	115256529	TT	CA	\N	\N	\N	\N
96	36	Q61R	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-06-21 16:49:37.274805	2015-06-21 16:52:23.177828	f	\N	\N	1	115256528	115256529	TT	CC	\N	\N	\N	\N
98	38	D842I	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-06-21 16:49:37.312239	2015-06-21 16:52:23.191105	f	\N	\N	4	55152092	55152093	GA	AT	\N	\N	\N	\N
99	38	D842V	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-06-21 16:49:37.330804	2015-06-21 16:52:23.203276	f	\N	\N	4	55152093	55152093	A	T	\N	\N	\N	\N
100	38	D842Y	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-06-21 16:49:37.363046	2015-06-21 16:52:23.217212	f	\N	\N	4	55152092	55152092	G	T	\N	\N	\N	\N
103	37	E542K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-06-21 16:49:37.418007	2015-06-21 16:52:23.233844	f	\N	\N	3	178936082	178936082	G	A	\N	\N	\N	\N
104	37	E545K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-06-21 16:49:37.435678	2015-06-21 16:52:23.246652	f	\N	\N	3	178936091	178936091	G	A	\N	\N	\N	\N
107	37	H1047R	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-06-21 16:49:37.500104	2015-06-21 16:52:23.261939	f	\N	\N	3	178952085	178952085	A	G	\N	\N	\N	\N
110	41	R233*	PTEN R233* has been shown to be a loss of function mutation, and PTEN loss has been the subject of considerable research in breast cancer. PTEN loss may sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data to support that PTEN loss is both associated with poorer prognosis, and no change in prognosis.	2015-06-21 16:49:37.560358	2015-06-21 16:52:23.280149	f	\N	\N	10	89717672	89717672	C	T	\N	\N	\N	\N
112	42	C634W	RET C639W has been implicated as an alternate mechanism of activating RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the C639W mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib.	2015-06-21 16:49:37.597495	2015-06-21 16:52:23.292774	f	\N	\N	10	43609950	43609950	C	G	\N	\N	\N	\N
113	42	M918T	RET M819T is the most common somatically acquired mutation in medullary thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the M918T mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T leads to more aggressive MTC with a poorer prognosis.	2015-06-21 16:49:37.619665	2015-06-21 16:52:23.307154	f	\N	\N	10	43617416	43617416	T	C	\N	\N	\N	\N
114	44	K666N	SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients.	2015-06-21 16:49:37.644932	2015-06-21 16:52:23.321372	f	\N	\N	2	198267359	198267359	C	A	\N	\N	\N	\N
115	44	K700E	SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients. Additionally, these mutations are the most common SF3B1 mutation observed in MDS and highly associated with subtypes of MDS that are defined by ringed sideroblasts.	2015-06-21 16:49:37.663305	2015-06-21 16:52:23.3357	f	\N	\N	2	198266834	198266834	T	C	\N	\N	\N	\N
116	45	R175H	While loss-of-function events in TP53 are very common in cancer, the R175H variant seems not only to result in loss of tumor-suppression, but also acts as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell lines harboring this mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been shown that the R175H mutation is correlated with worse overall survival than wild-type TP53, but is not as detrimental as the R248W variant.	2015-06-21 16:49:37.682648	2015-06-21 16:52:23.351302	f	\N	\N	17	7578406	7578406	C	T	\N	\N	\N	\N
117	45	R248Q	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival. The R248Q mutation has also shown an increased invasive behavior in cell lines. This is specific to the 248Q variant.	2015-06-21 16:49:37.706459	2015-06-21 16:52:23.364844	f	\N	\N	17	7577538	7577538	C	T	\N	\N	\N	\N
118	45	R248W	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival.	2015-06-21 16:49:37.725703	2015-06-21 16:52:23.378935	f	\N	\N	17	7577539	7577539	G	A	\N	\N	\N	\N
119	45	R249T	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-06-21 16:49:37.749929	2015-06-21 16:52:23.39406	f	\N	\N	17	7577535	7577535	C	G	\N	\N	\N	\N
120	45	R249W	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-06-21 16:49:37.767882	2015-06-21 16:52:23.406942	f	\N	\N	17	7577536	7577536	T	A	\N	\N	\N	\N
121	45	R273C	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-06-21 16:49:37.787364	2015-06-21 16:52:23.418972	f	\N	\N	17	7577121	7577121	G	A	\N	\N	\N	\N
122	45	R273H	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-06-21 16:49:37.807323	2015-06-21 16:52:23.431544	f	\N	\N	17	7577120	7577120	C	T	\N	\N	\N	\N
\.


--
-- Data for Name: evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_items (id, description, disease_id, source_id, variant_id, created_at, updated_at, rating, status, remote_errors, remote_ids, variant_hgvs, drug_interaction_description, evidence_level, evidence_type, variant_origin, evidence_direction, clinical_significance, deleted, deleted_at) FROM stdin;
1	JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).	1	51	64	2015-06-21 16:49:38.790604	2015-06-21 16:49:38.790604	4	accepted	\N	\N	9:5073770-5073770 (G->T)	\N	1	0	0	0	5	f	\N
2	GIST tumors harboring PDGFRA D842V mutation are more likely to be benign than malignant.	2	52	99	2015-06-21 16:49:38.876383	2015-06-21 16:49:38.876383	3	accepted	\N	\N	4:55152093-55152093 (A->T)	\N	1	0	0	0	5	f	\N
3	DNMT3A R882 mutations occur most often in de novo AML patients with intermediate cytogenic risk.	3	53	32	2015-06-21 16:49:38.977647	2015-06-21 16:49:38.977647	3	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
4	Young AML patients (<60 years old) with DNMT3A mutations (60% of which were R882) were older in age, had higher white blood cell counts and had higher platelet counts than patients wildtype for DNMT3A	3	54	32	2015-06-21 16:49:39.013038	2015-06-21 16:49:39.013038	3	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
5	JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical CML).	4	51	64	2015-06-21 16:49:39.053245	2015-06-21 16:49:39.053245	4	accepted	\N	\N	9:5073770-5073770 (G->T)	\N	1	0	0	0	4	f	\N
6	JAK2 V617F is associated with myeloid neoplasms (AML, MDS, CMML/aCML)	4	51	64	2015-06-21 16:49:39.087652	2015-06-21 16:49:39.087652	4	accepted	\N	\N	9:5073770-5073770 (G->T)	\N	1	0	0	0	4	f	\N
7	The SNP rs10974944 in the Jak2 locus is associated with increased predisposition for JAK2 V617F mutation and its associated cancer	5	55	64	2015-06-21 16:49:39.124766	2015-06-21 16:49:39.124766	4	accepted	\N	\N	9:5073770-5073770 (G->T)	\N	1	0	0	0	4	f	\N
8	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	6	56	76	2015-06-21 16:49:39.160226	2015-06-21 16:49:39.160226	3	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
9	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	6	56	80	2015-06-21 16:49:39.189402	2015-06-21 16:49:39.189402	3	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
10	Melanoma associated with NRAS Q61 mutation was more often associated with those at the extremity than those at the trunk	7	57	94	2015-06-21 16:49:39.221908	2015-06-21 16:49:39.221908	3	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
11	Daunorubicin treatment resulted in similar overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	3	53	32	2015-06-21 16:49:39.255157	2015-06-21 16:49:39.255157	4	accepted	\N	\N	N/A	\N	1	2	0	1	1	f	\N
12	The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition by about 5 fold.	7	58	82	2015-06-21 16:49:39.308103	2015-06-21 16:49:39.308103	3	accepted	\N	\N	15:66729162-66729162 (C->T)	\N	2	2	0	0	1	f	\N
13	MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.	7	59	83	2015-06-21 16:49:39.343844	2015-06-21 16:49:39.343844	3	accepted	\N	\N	15:66727451-66727451 (A->C)	\N	2	2	0	0	1	f	\N
14	Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.	7	60	94	2015-06-21 16:49:39.376707	2015-06-21 16:49:39.376707	4	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
15	GIST cancer with D842V mutation is resistant to imatinib.	2	61	99	2015-06-21 16:49:39.416176	2015-06-21 16:49:39.416176	4	accepted	\N	\N	4:55152093-55152093 (A->T)	\N	1	2	0	0	1	f	\N
16	While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib, double mutation of V561D and D842V mutants are resistant to imatinib.	2	62	99	2015-06-21 16:49:39.45734	2015-06-21 16:49:39.45734	4	accepted	\N	\N	4:55152093-55152093 (A->T)	\N	1	2	0	0	1	f	\N
17	In one patient with S214C mutation, the use of sorafenib has led to more than 5 years of survival and near remission.	8	63	10	2015-06-21 16:49:39.498956	2015-06-21 16:49:39.498956	2	accepted	\N	\N	X:47426121-47426121 (C->G)	\N	3	2	0	0	0	f	\N
18	Idarubicin increases the overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	3	53	32	2015-06-21 16:49:39.539772	2015-06-21 16:49:39.539772	4	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
19	In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction in the percentage of cells harboring JAK2 V617F.	9	64	64	2015-06-21 16:49:39.597589	2015-06-21 16:49:39.597589	4	accepted	\N	\N	9:5073770-5073770 (G->T)	\N	1	2	0	0	0	f	\N
20	TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells	9	65	64	2015-06-21 16:49:39.657837	2015-06-21 16:49:39.657837	3	accepted	\N	\N	9:5073770-5073770 (G->T)	\N	2	2	0	0	0	f	\N
21	Likely due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic malignant melanoma with an NRAS G13D mutation	7	66	93	2015-06-21 16:49:39.722188	2015-06-21 16:49:39.722188	2	accepted	\N	\N	1:115258744-115258744 (C->T)	\N	3	2	0	0	0	f	\N
22	In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted in disease free survival of 14 months	7	67	95	2015-06-21 16:49:39.781229	2015-06-21 16:49:39.781229	2	accepted	\N	\N	1:115256528-115256529 (TT->CA)	\N	3	2	0	0	0	f	\N
23	In a melanoma patient with Q61R mutation, treatment with temozolomide resulted in overall survival of 16 months	7	67	96	2015-06-21 16:49:39.827985	2015-06-21 16:49:39.827985	2	accepted	\N	\N	1:115256528-115256529 (TT->CC)	\N	3	2	0	0	0	f	\N
24	There is no difference in the complete remission rate of de novo AML patients with DNMT3A mutation compared to those who are wild type for DNMT3A.	3	53	32	2015-06-21 16:49:39.883945	2015-06-21 16:49:39.883945	4	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
25	De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	3	68	32	2015-06-21 16:49:39.91573	2015-06-21 16:49:39.91573	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
26	In acute myloid leukemia patients, D816 mutation is associated with earlier relapse and poorer prognosis than wildtype KIT	10	69	65	2015-06-21 16:49:39.947242	2015-06-21 16:49:39.947242	4	accepted	\N	\N	4:55599321-55599321 (A->T)	\N	1	1	0	0	3	f	\N
27	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	11	70	89	2015-06-21 16:49:40.003339	2015-06-21 16:49:40.003339	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
28	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	11	70	90	2015-06-21 16:49:40.040023	2015-06-21 16:49:40.040023	3	accepted	\N	\N	N/A	\N	0	1	0	0	3	f	\N
29	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	11	70	91	2015-06-21 16:49:40.073478	2015-06-21 16:49:40.073478	3	accepted	\N	\N	N/A	\N	0	1	0	0	3	f	\N
30	BRAF mutations are associated with melanoma arising in non-chronic sun damaged skin and with superficial spreading melanoma	7	71	17	2015-06-21 16:49:40.106091	2015-06-21 16:49:40.106091	4	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
31	DNMT3A R882 mutations were associated with older age, higher white blood cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically normal AML patients	3	72	32	2015-06-21 16:49:40.143028	2015-06-21 16:49:40.143028	3	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
32	Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without this mutation.	12	73	8	2015-06-21 16:49:40.181992	2015-06-21 16:49:40.181992	3	accepted	\N	\N	2:29443695-29443695 (G->T)	\N	2	2	0	0	1	f	\N
33	Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without this mutation.	8	73	8	2015-06-21 16:49:40.220644	2015-06-21 16:49:40.220644	3	accepted	\N	\N	2:29443695-29443695 (G->T)	\N	2	2	0	0	1	f	\N
34	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	7	74	72	2015-06-21 16:49:40.289741	2015-06-21 16:49:40.289741	3	accepted	\N	\N	N/A	\N	2	2	0	0	1	f	\N
35	In NSCLC, Exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	8	75	75	2015-06-21 16:49:40.365265	2015-06-21 16:49:40.365265	3	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
36	Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation (primarily Q61) have a significantly lower response rate to cetuximab than patients wildtype for NRAS.	11	76	94	2015-06-21 16:49:40.421915	2015-06-21 16:49:40.421915	3	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
37	CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.	13	77	8	2015-06-21 16:49:40.487098	2015-06-21 16:49:40.487098	3	accepted	\N	\N	2:29443695-29443695 (G->T)	\N	2	2	0	0	0	f	\N
38	High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing the EML4-ALK fusion containing the F1174L mutation.	13	78	8	2015-06-21 16:49:40.528043	2015-06-21 16:49:40.528043	3	accepted	\N	\N	2:29443695-29443695 (G->T)	\N	2	2	0	0	0	f	\N
39	TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.	13	79	9	2015-06-21 16:49:40.565079	2015-06-21 16:49:40.565079	3	accepted	\N	\N	2:29432664-29432664 (C->T)	\N	2	2	0	0	0	f	\N
40	Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	8	63	10	2015-06-21 16:49:40.603792	2015-06-21 16:49:40.603792	3	accepted	\N	\N	X:47426121-47426121 (C->G)	\N	2	2	0	0	0	f	\N
41	Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	8	63	10	2015-06-21 16:49:40.63777	2015-06-21 16:49:40.63777	3	accepted	\N	\N	X:47426121-47426121 (C->G)	\N	2	2	0	0	0	f	\N
42	The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation of KIT but did not alter the binding affinity of dasatinib.	7	74	72	2015-06-21 16:49:40.68347	2015-06-21 16:49:40.68347	3	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
43	In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	2	80	98	2015-06-21 16:49:40.736452	2015-06-21 16:49:40.736452	4	accepted	\N	\N	4:55152092-55152093 (GA->AT)	\N	2	2	0	0	0	f	\N
44	In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	2	80	99	2015-06-21 16:49:40.785959	2015-06-21 16:49:40.785959	4	accepted	\N	\N	4:55152093-55152093 (A->T)	\N	2	2	0	0	0	f	\N
45	In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	2	80	100	2015-06-21 16:49:40.833545	2015-06-21 16:49:40.833545	4	accepted	\N	\N	4:55152092-55152092 (G->T)	\N	2	2	0	0	0	f	\N
46	In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	2	80	101	2015-06-21 16:49:40.8712	2015-06-21 16:49:40.8712	4	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
47	In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	2	80	102	2015-06-21 16:49:40.906728	2015-06-21 16:49:40.906728	4	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
48	The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.	13	79	9	2015-06-21 16:49:40.942716	2015-06-21 16:49:40.942716	3	accepted	\N	\N	2:29432664-29432664 (C->T)	\N	2	2	0	0	1	f	\N
49	In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5% of which were R882) had no prognostic value on overall, relapse free and event free survival.	3	81	32	2015-06-21 16:49:40.97812	2015-06-21 16:49:40.97812	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
50	In a large cohort of cytogenetically normal AML patients (18-60 years old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall, relapse free and event free survival	3	81	32	2015-06-21 16:49:41.008063	2015-06-21 16:49:41.008063	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
51	In young AML patients (<60 years old), DNMT3A mutation status (60% of which were R882) was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	3	54	32	2015-06-21 16:49:41.055028	2015-06-21 16:49:41.055028	4	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
52	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	3	54	55	2015-06-21 16:49:41.084332	2015-06-21 16:49:41.084332	4	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
53	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	14	82	66	2015-06-21 16:49:41.119861	2015-06-21 16:49:41.119861	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
54	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	14	82	71	2015-06-21 16:49:41.165681	2015-06-21 16:49:41.165681	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
55	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	3	54	86	2015-06-21 16:49:41.206338	2015-06-21 16:49:41.206338	4	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
56	The presence of NRAS mutation in AML patients does not impact diease prognosis (resistant disease, disease-free survival, complete remission rate, relapse rate, induction death)	3	83	89	2015-06-21 16:49:41.248147	2015-06-21 16:49:41.248147	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
57	NRAS mutation status does not impact overall, event-free or disease-free survival in patients with AML	3	84	92	2015-06-21 16:49:41.286073	2015-06-21 16:49:41.286073	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
58	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	7	85	66	2015-06-21 16:49:41.328264	2015-06-21 16:49:41.328264	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
59	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	7	85	68	2015-06-21 16:49:41.372328	2015-06-21 16:49:41.372328	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
60	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	7	85	70	2015-06-21 16:49:41.408558	2015-06-21 16:49:41.408558	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
61	In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations were associated with reduced relapse free survival in the entire cohort as well as the subset of patients with cytogenetically normal AML	3	81	32	2015-06-21 16:49:41.448813	2015-06-21 16:49:41.448813	3	accepted	\N	\N	N/A	\N	0	1	0	0	3	f	\N
62	AML patients with DNMT3A mutations (59% of which were R882) showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	3	68	32	2015-06-21 16:49:41.478002	2015-06-21 16:49:41.478002	5	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
63	DNMT3A R882 mutation was associated with reduced relapse free and overall survival in ELN-unfavorable, cytogenetically normal AMLs	3	81	32	2015-06-21 16:49:41.508585	2015-06-21 16:49:41.508585	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
64	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including R882) were associated with worse overall survival compared to those without DNMT3A mutation.	3	68	32	2015-06-21 16:49:41.545032	2015-06-21 16:49:41.545032	5	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
65	In cytogenetically normal AML patients, DNMT3A R882 mutations are associated with lower overall and disease free survival as compared to patients with wild type DNMT3A.	3	72	32	2015-06-21 16:49:41.595469	2015-06-21 16:49:41.595469	5	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
66	In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation is prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	3	86	32	2015-06-21 16:49:41.627204	2015-06-21 16:49:41.627204	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
67	In young AML patients (<60 years old), DNMT3A mutations were associated with significantly reduced overall survival and relapse free survival in patients wildtype for NPM1 and FLT3	3	54	32	2015-06-21 16:49:41.660069	2015-06-21 16:49:41.660069	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
68	In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations other than R882 are prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	3	86	32	2015-06-21 16:49:41.695181	2015-06-21 16:49:41.695181	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
69	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated with worse overall survival compared to those without DNMT3A mutation.	3	68	55	2015-06-21 16:49:41.737094	2015-06-21 16:49:41.737094	5	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
70	GIST patients with exon 14 KIT mutations had reduced overall survival compared to patients wiltype for KIT	14	82	69	2015-06-21 16:49:41.772345	2015-06-21 16:49:41.772345	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
71	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	7	87	88	2015-06-21 16:49:41.813358	2015-06-21 16:49:41.813358	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
72	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	7	87	89	2015-06-21 16:49:41.848366	2015-06-21 16:49:41.848366	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
73	A single patient with BRAF V600E/V600M bi-allelic mutation responded to the V600E drug dabrafenib.	7	88	13	2015-06-21 16:49:41.884736	2015-06-21 16:49:41.884736	3	accepted	\N	\N	N/A	\N	3	2	0	0	0	f	\N
74	In patients with medullary carcinoma, the presence of RET M918T mutation is associated with increased probability of lymph node metastases.	15	89	113	2015-06-21 16:49:41.948818	2015-06-21 16:49:41.948818	5	accepted	\N	\N	10:43617416-43617416 (T->C)	\N	1	0	0	0	4	f	\N
75	Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib, compared to wild-type RET.	15	90	112	2015-06-21 16:49:41.986948	2015-06-21 16:49:41.986948	3	accepted	\N	\N	10:43609950-43609950 (C->G)	\N	2	2	0	0	1	f	\N
76	Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib, compared to wild-type RET.	15	90	113	2015-06-21 16:49:42.039982	2015-06-21 16:49:42.039982	3	accepted	\N	\N	10:43617416-43617416 (T->C)	\N	2	2	0	0	1	f	\N
77	The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong repression of tyroid cancer cell growth.	15	91	113	2015-06-21 16:49:42.098535	2015-06-21 16:49:42.098535	3	accepted	\N	\N	10:43617416-43617416 (T->C)	\N	2	2	0	0	0	f	\N
78	Compared to those who harbor a wild type RET, patients with RET M918T mutation develop larger and more aggressive medullary thyroid cancer.	15	92	113	2015-06-21 16:49:42.132856	2015-06-21 16:49:42.132856	4	accepted	\N	\N	10:43617416-43617416 (T->C)	\N	1	1	0	0	3	f	\N
79	BRAF V600E is shown to be associated with the tall-cell variant of papillary thyroid cancer	16	93	12	2015-06-21 16:49:42.170535	2015-06-21 16:49:42.170535	3	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	1	0	0	0	4	f	\N
80	Thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	16	94	12	2015-06-21 16:49:42.201904	2015-06-21 16:49:42.201904	5	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	1	0	0	0	4	f	\N
81	Thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	16	95	12	2015-06-21 16:49:42.235936	2015-06-21 16:49:42.235936	5	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	1	0	0	0	4	f	\N
82	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	7	96	12	2015-06-21 16:49:42.270989	2015-06-21 16:49:42.270989	2	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	1	2	0	1	6	f	\N
83	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	11	97	103	2015-06-21 16:49:42.307842	2015-06-21 16:49:42.307842	4	accepted	\N	\N	3:178936082-178936082 (G->A)	\N	1	2	0	1	6	f	\N
84	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	11	97	104	2015-06-21 16:49:42.345725	2015-06-21 16:49:42.345725	4	accepted	\N	\N	3:178936091-178936091 (G->A)	\N	1	2	0	1	6	f	\N
85	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	11	97	107	2015-06-21 16:49:42.374862	2015-06-21 16:49:42.374862	4	accepted	\N	\N	3:178952085-178952085 (A->G)	\N	1	2	0	1	6	f	\N
86	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP but does not show resistance to MEK inhibitors.	7	98	12	2015-06-21 16:49:42.403607	2015-06-21 16:49:42.403607	3	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	2	2	0	1	1	f	\N
87	KRAS mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	11	97	75	2015-06-21 16:49:42.433013	2015-06-21 16:49:42.433013	4	accepted	\N	\N	N/A	\N	1	2	0	1	0	f	\N
88	In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations were associated with poor progression free survival regardless of treatment (panitumumab with best supportive care or best supportive care alone)	11	99	17	2015-06-21 16:49:42.47016	2015-06-21 16:49:42.47016	3	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
89	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	17	100	12	2015-06-21 16:49:42.506702	2015-06-21 16:49:42.506702	3	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	2	2	0	0	1	f	\N
90	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP and resistance to RAF inhibitors.	7	98	12	2015-06-21 16:49:42.551992	2015-06-21 16:49:42.551992	3	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	2	2	0	0	1	f	\N
91	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	8	101	12	2015-06-21 16:49:42.585487	2015-06-21 16:49:42.585487	2	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	3	2	0	0	1	f	\N
92	COLO201 and COLO206F cells harboring BRAF V600E mutations were cloned to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification of the BRAF V600E gene.	11	102	14	2015-06-21 16:49:42.618283	2015-06-21 16:49:42.618283	4	accepted	\N	\N	N/A	\N	4	2	0	0	1	f	\N
93	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	7	103	17	2015-06-21 16:49:42.65107	2015-06-21 16:49:42.65107	3	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
94	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	7	104	11	2015-06-21 16:49:42.686507	2015-06-21 16:49:42.686507	5	accepted	\N	\N	7:140453135-140453136 (CA->AT)	\N	1	2	0	0	0	f	\N
95	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	7	105	12	2015-06-21 16:49:42.727813	2015-06-21 16:49:42.727813	5	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	1	2	0	0	0	f	\N
96	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	17	106	12	2015-06-21 16:49:42.771521	2015-06-21 16:49:42.771521	3	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	2	2	0	0	0	f	\N
97	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	17	107	12	2015-06-21 16:49:42.825886	2015-06-21 16:49:42.825886	2	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	2	2	0	0	0	f	\N
98	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	11	108	12	2015-06-21 16:49:42.876012	2015-06-21 16:49:42.876012	2	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	2	2	0	0	0	f	\N
99	Cell lines expressing the BRAF V600E mutation responded better to vemurafenib treatment than cells wildtype for BRAF as measured by reduced cellular proliferation and inhibition of MET and ERK phosphorylation	11	108	12	2015-06-21 16:49:42.93158	2015-06-21 16:49:42.93158	2	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	2	2	0	0	0	f	\N
100	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	7	104	15	2015-06-21 16:49:42.969976	2015-06-21 16:49:42.969976	5	accepted	\N	\N	7:140453137-140453137 (C->T)	\N	1	2	0	0	0	f	\N
101	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	7	104	16	2015-06-21 16:49:43.014082	2015-06-21 16:49:43.014082	5	accepted	\N	\N	7:140453136-140453137 (AC->CT)	\N	1	2	0	0	0	f	\N
102	Unlike other studies that suggest a poorer outcome, BRAF mutation in this study was not correlated with poorer prognosis in papillary thyroid cancer.	16	109	12	2015-06-21 16:49:43.071737	2015-06-21 16:49:43.071737	5	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	1	1	0	1	3	f	\N
103	V600E is associated with adverse pathological features of colorectal cancer. This can be concluded as a marker of poor prognosis.	11	110	12	2015-06-21 16:49:43.104154	2015-06-21 16:49:43.104154	5	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	0	1	0	0	3	f	\N
104	BRAF V600E is correlated with shorter disease-free and overall Survival in a Spanish cohort of melanoma patients.	7	111	12	2015-06-21 16:49:43.135736	2015-06-21 16:49:43.135736	3	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	1	1	0	0	3	f	\N
105	BRAF mutation correlated with poor prognosis in papillary thyroid cancer in both older (>65 yo) and younger (<65 yo) cohorts.	16	93	12	2015-06-21 16:49:43.165592	2015-06-21 16:49:43.165592	3	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	1	1	0	0	3	f	\N
106	BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in a study of 187 patients with PTC and other thyroid diseases.	16	112	12	2015-06-21 16:49:43.196342	2015-06-21 16:49:43.196342	3	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	1	1	0	0	3	f	\N
107	V600E is correlated with disease recurrence in both age cohorts (>65 and <65 yo).	16	93	12	2015-06-21 16:49:43.224855	2015-06-21 16:49:43.224855	3	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	1	1	0	0	3	f	\N
108	NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype (0/55).	3	113	86	2015-06-21 16:49:43.258866	2015-06-21 16:49:43.258866	3	accepted	\N	\N	N/A	\N	1	0	0	1	4	f	\N
109	No NPM1 mutations were identified in patients with favorable risk cytogenetics (79/215 patients)	3	68	87	2015-06-21 16:49:43.288191	2015-06-21 16:49:43.288191	4	accepted	\N	\N	5:170837546-170837547 (0->TCAG)	\N	1	0	0	1	4	f	\N
110	Therapy-related AML was less common in patients with DNMT3A mutations (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of younger (18-60) AML patients	3	81	32	2015-06-21 16:49:43.339006	2015-06-21 16:49:43.339006	3	accepted	\N	\N	N/A	\N	1	0	0	0	5	f	\N
111	NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid leukemia.	3	114	86	2015-06-21 16:49:43.368357	2015-06-21 16:49:43.368357	4	accepted	\N	\N	N/A	\N	1	0	0	0	5	f	\N
112	DNMT3A mutations (59% of which were R882) were associated with an intermediate risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes	3	68	32	2015-06-21 16:49:43.397293	2015-06-21 16:49:43.397293	5	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
113	DNMT3A mutations (59% of which were R882) were associated with intermediate risk cytogenetics (including normal karyotype)	3	68	32	2015-06-21 16:49:43.423685	2015-06-21 16:49:43.423685	4	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
114	DNMT3A mutations (64.5% R882) were associated with older age, higher white blood cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML patients	3	81	32	2015-06-21 16:49:43.451647	2015-06-21 16:49:43.451647	3	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
115	DNMT3A R882 mutations were associated with cytogenetically normal AML in a large cohort of younger (18-60) AML patients	3	81	32	2015-06-21 16:49:43.479029	2015-06-21 16:49:43.479029	3	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
116	AML with mutated NPM1 is a provisional entity in WHO classification of acute myeloid leukemia (AML). This mutation should be tested for in clinical trials and is recommended for testing in patients with cytogentically normal AML.	3	115	86	2015-06-21 16:49:43.509728	2015-06-21 16:49:43.509728	5	accepted	\N	\N	N/A	\N	0	0	0	0	4	f	\N
117	NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes	3	116	86	2015-06-21 16:49:43.548212	2015-06-21 16:49:43.548212	2	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
118	NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid leukemia.	3	114	86	2015-06-21 16:49:43.597024	2015-06-21 16:49:43.597024	4	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
119	NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid leukemia	3	113	86	2015-06-21 16:49:43.650573	2015-06-21 16:49:43.650573	3	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
120	NPM1 mutations were associated with normal karyotype in older (>60) patients	3	117	86	2015-06-21 16:49:43.691695	2015-06-21 16:49:43.691695	3	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
121	NPM1 mutations were associated with intermediate risk cytogenetics (including normal karyotype)	3	68	87	2015-06-21 16:49:43.72956	2015-06-21 16:49:43.72956	4	accepted	\N	\N	5:170837546-170837547 (0->TCAG)	\N	1	0	0	0	4	f	\N
122	CEBPA mutation status had no impact on ATRA treatment response in older (>60) patients with AML	3	118	29	2015-06-21 16:49:43.775298	2015-06-21 16:49:43.775298	4	accepted	\N	\N	N/A	\N	1	2	0	1	6	f	\N
123	In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations (Exon 9) had no significant impact on response rate, disease control rate, progression free surivival or overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	11	76	106	2015-06-21 16:49:43.818099	2015-06-21 16:49:43.818099	3	accepted	\N	\N	N/A	\N	1	2	0	1	6	f	\N
124	In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status (primarily Q61) was not informative for disease control rate, progression free survival or overall survival following cetuximab plus chemotherapy.	11	76	94	2015-06-21 16:49:43.848567	2015-06-21 16:49:43.848567	3	accepted	\N	\N	N/A	\N	1	2	0	1	1	f	\N
125	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation.	13	119	8	2015-06-21 16:49:43.885639	2015-06-21 16:49:43.885639	3	accepted	\N	\N	2:29443695-29443695 (G->T)	\N	2	2	0	0	1	f	\N
126	Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype BRAF	11	76	12	2015-06-21 16:49:43.915243	2015-06-21 16:49:43.915243	3	accepted	\N	\N	7:140453136-140453136 (A->T)	\N	1	2	0	0	1	f	\N
127	ATRA treatment did not effect overall survival in patients <60 years old with FLT3-ITD mutation	3	120	55	2015-06-21 16:49:43.945972	2015-06-21 16:49:43.945972	3	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
128	FLT3-ITD mutation without NPM1 was associated with increased relapse risk and reduced overall survival in young adult AML patients (median age 43)	3	116	55	2015-06-21 16:49:43.984975	2015-06-21 16:49:43.984975	4	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
129	Increasing level of FLT3-ITD mutant correlated with reduced disease-free and overall survival as well as increased relapse rate	3	116	55	2015-06-21 16:49:44.013519	2015-06-21 16:49:44.013519	3	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
130	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	3	121	55	2015-06-21 16:49:44.047785	2015-06-21 16:49:44.047785	3	accepted	\N	\N	N/A	\N	2	2	0	0	1	f	\N
131	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	3	121	55	2015-06-21 16:49:44.076446	2015-06-21 16:49:44.076446	3	accepted	\N	\N	N/A	\N	2	2	0	0	1	f	\N
132	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	3	121	56	2015-06-21 16:49:44.111197	2015-06-21 16:49:44.111197	3	accepted	\N	\N	N/A	\N	2	2	0	0	1	f	\N
133	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	3	121	56	2015-06-21 16:49:44.148202	2015-06-21 16:49:44.148202	3	accepted	\N	\N	N/A	\N	2	2	0	0	1	f	\N
134	Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations (primarily G12/G13) had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype KRAS	11	76	77	2015-06-21 16:49:44.189401	2015-06-21 16:49:44.189401	3	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
135	KRAS mutations were significantly associated with lack of response to cetuximab in patients with advanced colorectal cancer	11	122	77	2015-06-21 16:49:44.230132	2015-06-21 16:49:44.230132	3	accepted	\N	\N	N/A	\N	2	2	0	0	1	f	\N
136	Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	17	123	81	2015-06-21 16:49:44.277757	2015-06-21 16:49:44.277757	4	accepted	\N	\N	12:25398281-25398281 (C->T)	\N	2	2	0	0	1	f	\N
137	ATRA treatment did not effect overall survival in patients <60 years old with NPM1 mutation regardless of FLT3-ITD status	3	120	86	2015-06-21 16:49:44.31687	2015-06-21 16:49:44.31687	3	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
138	Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	11	76	105	2015-06-21 16:49:44.35874	2015-06-21 16:49:44.35874	3	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
139	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	3	124	129	2015-06-21 16:49:44.398767	2015-06-21 16:49:44.398767	3	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
140	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	3	124	130	2015-06-21 16:49:44.4352	2015-06-21 16:49:44.4352	3	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
141	CH5424802 treatment resulted in significant tumor regression in xenograft models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation	8	77	7	2015-06-21 16:49:44.475688	2015-06-21 16:49:44.475688	3	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
142	SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition	13	79	8	2015-06-21 16:49:44.513661	2015-06-21 16:49:44.513661	4	accepted	\N	\N	2:29443695-29443695 (G->T)	\N	2	2	0	0	0	f	\N
143	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation	13	119	9	2015-06-21 16:49:44.546896	2015-06-21 16:49:44.546896	3	accepted	\N	\N	2:29432664-29432664 (C->T)	\N	2	2	0	0	0	f	\N
144	Following identification of this fusion in a patient, this fusion was expressed in HEK 293T cells leading to oligomerization in the absence of ligand, induced morphologic changes in the cell and increased cellular proliferation consistent with FGFR activation. Mouse xenografts using the bladder cell line SW780 which harbors an FGFR3-BAIAP2L1 fusion showed reduced tumor burden when treated with the FGFR inhibitor PD173074.	18	125	53	2015-06-21 16:49:44.609585	2015-06-21 16:49:44.609585	3	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
145	Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	17	123	81	2015-06-21 16:49:44.69269	2015-06-21 16:49:44.69269	4	accepted	\N	\N	12:25398281-25398281 (C->T)	\N	2	2	0	0	0	f	\N
146	ATRA treatment improved overall and relapse-free survival in older (>60) patients with mutant NPM1 and without FLT3-ITD	3	117	86	2015-06-21 16:49:44.729902	2015-06-21 16:49:44.729902	2	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
147	Young patients (18-60) with NPM1 mutations had improved overall survival following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However, the adjusted p-value for multiple testing is 0.11.	3	126	86	2015-06-21 16:49:44.771531	2015-06-21 16:49:44.771531	2	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
148	For patients with NPM1 mutation that achieved CR following induction therapy, relapse-free survival was improved following treatment with valproic acid	3	127	86	2015-06-21 16:49:44.830103	2015-06-21 16:49:44.830103	2	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
149	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	3	128	86	2015-06-21 16:49:44.864401	2015-06-21 16:49:44.864401	3	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
150	CD33 and CD123 expression was significantly increased in patients with NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33 and anti-CD123 therapy	3	129	86	2015-06-21 16:49:44.90156	2015-06-21 16:49:44.90156	2	accepted	\N	\N	N/A	\N	4	2	0	0	0	f	\N
151	CD33 expression was significantly increased in patients with NPM1 mutation with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy	3	129	86	2015-06-21 16:49:44.933632	2015-06-21 16:49:44.933632	2	accepted	\N	\N	N/A	\N	4	2	0	0	0	f	\N
152	NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.	3	128	87	2015-06-21 16:49:44.972133	2015-06-21 16:49:44.972133	3	accepted	\N	\N	5:170837546-170837547 (0->TCAG)	\N	2	2	0	0	0	f	\N
153	NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM in AML patients was associated with good response to induction therapy.	3	130	87	2015-06-21 16:49:45.024786	2015-06-21 16:49:45.024786	3	accepted	\N	\N	5:170837546-170837547 (0->TCAG)	\N	4	2	0	0	0	f	\N
154	Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit increased overall survival, increased apoptosis, reduced proliferation and reduced tumor burden compared to Tsc1 wildtype mice.	8	131	124	2015-06-21 16:49:45.082152	2015-06-21 16:49:45.082152	3	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
155	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	18	132	125	2015-06-21 16:49:45.150413	2015-06-21 16:49:45.150413	3	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
156	FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically normal AML patients with CEBPA	3	133	28	2015-06-21 16:49:45.202836	2015-06-21 16:49:45.202836	2	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
157	Complete remission rates did not differ between patients with wildtype or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML	3	86	32	2015-06-21 16:49:45.233123	2015-06-21 16:49:45.233123	4	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
158	FLT3-ITD duplication length did not effect prognosis in patients with an NPM1 mutation	3	134	55	2015-06-21 16:49:45.263326	2015-06-21 16:49:45.263326	4	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
159	Size of the FLT3-ITD mutant duplication had no impact on overall survival or relapse rate	3	116	55	2015-06-21 16:49:45.292614	2015-06-21 16:49:45.292614	2	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
160	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	3	135	58	2015-06-21 16:49:45.325156	2015-06-21 16:49:45.325156	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
161	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	3	135	86	2015-06-21 16:49:45.354451	2015-06-21 16:49:45.354451	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
162	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	3	136	129	2015-06-21 16:49:45.385398	2015-06-21 16:49:45.385398	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
163	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML	3	136	129	2015-06-21 16:49:45.417718	2015-06-21 16:49:45.417718	4	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
164	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	3	136	130	2015-06-21 16:49:45.447961	2015-06-21 16:49:45.447961	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
165	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML.	3	136	130	2015-06-21 16:49:45.477719	2015-06-21 16:49:45.477719	4	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
166	CEBPA mutation was associated with improved overall survival in older (>60) patients with AML	3	118	29	2015-06-21 16:49:45.508699	2015-06-21 16:49:45.508699	4	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
167	CEBPA mutation was significantly associated with complete remission	3	137	29	2015-06-21 16:49:45.538945	2015-06-21 16:49:45.538945	4	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
168	Overall survival was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA mutations	3	133	28	2015-06-21 16:49:45.56935	2015-06-21 16:49:45.56935	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
169	Remission duration was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA	3	133	28	2015-06-21 16:49:45.603064	2015-06-21 16:49:45.603064	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
170	DNMT3A mutations were associated with achievement of complete remission in a large cohort of younger (18-60) AML patients	3	81	32	2015-06-21 16:49:45.644601	2015-06-21 16:49:45.644601	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
171	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	3	138	56	2015-06-21 16:49:45.686383	2015-06-21 16:49:45.686383	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
172	MGMT promoter methylation was associated with increased overall and disease-free survival as well as tumor regression in patients with gliomas.	19	139	85	2015-06-21 16:49:45.73266	2015-06-21 16:49:45.73266	4	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
173	NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and increased overall survival in young adult acute myeloid leukemia patients (median age 43)	3	116	86	2015-06-21 16:49:45.77791	2015-06-21 16:49:45.77791	4	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
174	Patients with NPM1 mutations were associated with improved complete remission rates as well as overall and disease free survival when all ages and karyotype were analyzed; however, when only normal karyotype patients were analyzed, only complete remission rates were improved	3	117	86	2015-06-21 16:49:45.820849	2015-06-21 16:49:45.820849	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
175	Complete remission rates were higher and both disease-free and overall survival were longer for patients with Exon 12 NPM1 mutations.	3	113	86	2015-06-21 16:49:45.858524	2015-06-21 16:49:45.858524	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
176	Complete remission rates were higher and event-free survival was longer for normal karyotype AML patients with Exon 12 NPM1 mutations.	3	114	86	2015-06-21 16:49:45.898896	2015-06-21 16:49:45.898896	5	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
177	Cytoplasmic localization of NPM in AML patients was an independent prognositic factor associated with complete remission in response to induction therapy in a multivariate analysis (p=0.019) but not in a univariate analysis (p=0.07).	3	130	86	2015-06-21 16:49:45.936082	2015-06-21 16:49:45.936082	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
178	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	3	138	86	2015-06-21 16:49:45.971667	2015-06-21 16:49:45.971667	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
179	Intermediate risk patients with mutant NPM1 had improved overall survival with the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1 and IDH2	3	126	86	2015-06-21 16:49:46.002499	2015-06-21 16:49:46.002499	2	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
180	Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved overall and relapse-free survival, complete remission rates and remission duration compared to all other combinations of mutants	3	140	86	2015-06-21 16:49:46.038627	2015-06-21 16:49:46.038627	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
181	NPM1 mutation was associated with higher complete remission rates, lower cumulative incidence of relapse and higher overall survival in intermediate risk AML patients	3	141	86	2015-06-21 16:49:46.069909	2015-06-21 16:49:46.069909	4	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
182	NPM1 mutation was associated with increased overall, event-free and relapse-free survival and remission after induction therapy in the entire cohort and in the cytogenetically normal subset of patients	3	142	86	2015-06-21 16:49:46.105276	2015-06-21 16:49:46.105276	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
183	NPM1 mutation without FLT3-ITD was significantly associated with complete remission	3	137	86	2015-06-21 16:49:46.142934	2015-06-21 16:49:46.142934	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
184	NPM1 mutations were associated with increased complete remission rates as well as longer overall disease free survival in normal karyotype AML patients >59 years old, particularly in those >69 years old	3	143	86	2015-06-21 16:49:46.18549	2015-06-21 16:49:46.18549	4	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
185	Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and without FLT3-ITD have increased overall survival, relapse-free survival and response to induction chemotherapy	3	144	86	2015-06-21 16:49:46.226074	2015-06-21 16:49:46.226074	4	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
186	Complete remission rate was not different between young AML patients (<60 years old) with or without DNMT3A mutations (60% of which were R882)	3	54	32	2015-06-21 16:49:46.265543	2015-06-21 16:49:46.265543	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
187	DNMT3A mutations (62% of which were R882) were associated with reduced disease-free survival in patients with cytogenetically normal AML.	3	86	32	2015-06-21 16:49:46.30485	2015-06-21 16:49:46.30485	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
188	Young AML patients (<60 years old) with DNMT3A mutation have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	3	54	32	2015-06-21 16:49:46.340758	2015-06-21 16:49:46.340758	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
189	Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	3	54	32	2015-06-21 16:49:46.376706	2015-06-21 16:49:46.376706	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
190	Meta-analysis of studies involving cytogentically normal younger (<60) patients showed reduced overall and relapse-free survival for patients with FLT3-ITD	3	145	55	2015-06-21 16:49:46.412985	2015-06-21 16:49:46.412985	2	accepted	\N	\N	N/A	\N	0	1	0	0	3	f	\N
191	A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation	3	134	55	2015-06-21 16:49:46.448384	2015-06-21 16:49:46.448384	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
192	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	3	134	55	2015-06-21 16:49:46.514973	2015-06-21 16:49:46.514973	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
193	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	3	134	55	2015-06-21 16:49:46.552191	2015-06-21 16:49:46.552191	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
194	KRAS mutations were significantly associated with reduced progression-free and overall survival in patients with advanced colorectal cancer	11	122	77	2015-06-21 16:49:46.588534	2015-06-21 16:49:46.588534	3	accepted	\N	\N	N/A	\N	2	1	0	0	3	f	\N
195	Coocurrence of FLT3-ITD mutations significantly reduced overall survival of normal karyotype AML patients	3	113	86	2015-06-21 16:49:46.62771	2015-06-21 16:49:46.62771	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
196	Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction in NPM1 levels following first line therapy (N=15)	3	146	86	2015-06-21 16:49:46.664925	2015-06-21 16:49:46.664925	2	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
197	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	3	134	86	2015-06-21 16:49:46.711419	2015-06-21 16:49:46.711419	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
198	Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate risk AML patients	3	141	86	2015-06-21 16:49:46.749382	2015-06-21 16:49:46.749382	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
199	TET2 mutation reduces overall survival of normal karyotype patients with either an NPM1 mutation or NPM1 mutation without FLT3-ITD	3	147	86	2015-06-21 16:49:46.790366	2015-06-21 16:49:46.790366	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
200	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	3	134	86	2015-06-21 16:49:46.832246	2015-06-21 16:49:46.832246	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
201	Mutations  in WT1 were associated with increased risk of recurrence in young patients (15-50) with cytogenetically normal AML	3	148	129	2015-06-21 16:49:46.87165	2015-06-21 16:49:46.87165	2	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
202	Mutations in WT1 were associated with a worse overall prognosis than patients wildtype for WT1 in young patients (15-50), primarily because of increased risk of disease recurrence	3	148	129	2015-06-21 16:49:46.907773	2015-06-21 16:49:46.907773	2	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
203	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	3	136	129	2015-06-21 16:49:46.947369	2015-06-21 16:49:46.947369	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
204	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	3	149	129	2015-06-21 16:49:46.988139	2015-06-21 16:49:46.988139	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
205	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	3	124	129	2015-06-21 16:49:47.023914	2015-06-21 16:49:47.023914	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
206	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	3	124	129	2015-06-21 16:49:47.058447	2015-06-21 16:49:47.058447	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
207	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	3	136	130	2015-06-21 16:49:47.095969	2015-06-21 16:49:47.095969	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
208	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	3	124	130	2015-06-21 16:49:47.136287	2015-06-21 16:49:47.136287	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
209	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	3	124	130	2015-06-21 16:49:47.177015	2015-06-21 16:49:47.177015	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
210	WT1 mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	3	149	130	2015-06-21 16:49:47.216591	2015-06-21 16:49:47.216591	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
211	The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation positive women with high grade serous or undifferentiated ovarian cancer.	20	150	131	2015-06-21 16:49:47.260246	2015-06-21 16:49:47.260246	4	accepted	\N	\N	N/A	\N	1	2	1	0	0	f	\N
212	The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation positive women with high grade serous or undifferentiated ovarian cancer.	20	150	132	2015-06-21 16:49:47.324959	2015-06-21 16:49:47.324959	4	accepted	\N	\N	N/A	\N	1	2	1	0	0	f	\N
213	Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML.	4	151	1	2015-06-21 16:49:47.374332	2015-06-21 16:49:47.374332	2	accepted	\N	\N	N/A	\N	2	0	0	1	4	f	\N
214	WBC counts were not significantly different in AML patients with FLT3 D835 mutations compared to wild type	3	152	56	2015-06-21 16:49:47.41495	2015-06-21 16:49:47.41495	3	accepted	\N	\N	N/A	\N	1	0	0	1	4	f	\N
215	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	8	153	76	2015-06-21 16:49:47.45507	2015-06-21 16:49:47.45507	3	accepted	\N	\N	N/A	\N	1	0	0	1	4	f	\N
216	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	8	153	80	2015-06-21 16:49:47.48436	2015-06-21 16:49:47.48436	3	accepted	\N	\N	N/A	\N	1	0	0	1	4	f	\N
217	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.	3	154	127	2015-06-21 16:49:47.524145	2015-06-21 16:49:47.524145	3	accepted	\N	\N	N/A	\N	4	0	0	1	4	f	\N
218	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.	3	154	128	2015-06-21 16:49:47.55562	2015-06-21 16:49:47.55562	3	accepted	\N	\N	N/A	\N	4	0	0	1	4	f	\N
219	CCND2 hypomethylation is seen to be more common in stage III and IV gastric tumors than stage I and II.	21	155	22	2015-06-21 16:49:47.587434	2015-06-21 16:49:47.587434	3	accepted	\N	\N	N/A	\N	2	0	0	0	5	f	\N
220	The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease.	4	156	1	2015-06-21 16:49:47.619306	2015-06-21 16:49:47.619306	4	accepted	\N	\N	N/A	\N	0	0	0	0	4	f	\N
221	Cyclin D1 overexpression is associated with the ER-positive subtype of breast cancer.	22	157	20	2015-06-21 16:49:47.651128	2015-06-21 16:49:47.651128	4	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
222	WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared to wild type	3	152	55	2015-06-21 16:49:47.680411	2015-06-21 16:49:47.680411	3	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
223	In leukemia patients, IDH1 R132 mutation is associated with adults and normal karyotype.	3	158	58	2015-06-21 16:49:47.710197	2015-06-21 16:49:47.710197	2	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
224	IDH1 R132 mutation is associated with patients of older age, high platelet count during diagnosis, cytogenic normalcy and NPM1 mutation	3	159	59	2015-06-21 16:49:47.741307	2015-06-21 16:49:47.741307	3	accepted	\N	\N	2:209113113-209113113 (G->A)	\N	1	0	0	0	4	f	\N
225	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	14	160	66	2015-06-21 16:49:47.777074	2015-06-21 16:49:47.777074	2	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
226	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	14	160	70	2015-06-21 16:49:47.810258	2015-06-21 16:49:47.810258	3	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
227	KRAS G12C occur more frequently in women than men	23	161	78	2015-06-21 16:49:47.849594	2015-06-21 16:49:47.849594	2	accepted	\N	\N	12:25398285-25398285 (C->A)	\N	1	0	0	0	4	f	\N
228	KRAS G12D mutation occurs in never smokers significantly more often than in smokers	23	161	79	2015-06-21 16:49:47.884702	2015-06-21 16:49:47.884702	3	accepted	\N	\N	12:25398284-25398284 (C->T)	\N	1	0	0	0	4	f	\N
229	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.	8	162	33	2015-06-21 16:49:47.921571	2015-06-21 16:49:47.921571	3	accepted	\N	\N	7:55259514-55259515 (CT->AG)	\N	1	2	0	1	6	f	\N
230	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.	8	162	33	2015-06-21 16:49:47.954249	2015-06-21 16:49:47.954249	3	accepted	\N	\N	7:55259514-55259515 (CT->AG)	\N	1	2	0	1	6	f	\N
231	PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the AKT inhibitor MK-2206 in breast cancer cell lines.	24	163	4	2015-06-21 16:49:47.99388	2015-06-21 16:49:47.99388	3	accepted	\N	\N	14:105246551-105246551 (C->T)	\N	2	2	0	1	0	f	\N
232	BCR-ABL fusions have been found to contribute to imatinib resistance in AML cell lines.	3	164	1	2015-06-21 16:49:48.031748	2015-06-21 16:49:48.031748	3	accepted	\N	\N	N/A	\N	2	2	0	0	1	f	\N
233	COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	4	165	3	2015-06-21 16:49:48.062431	2015-06-21 16:49:48.062431	4	accepted	\N	\N	N/A	\N	2	2	0	0	1	f	\N
234	In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have shown to be resistant to imatinib treatment.	4	156	2	2015-06-21 16:49:48.095942	2015-06-21 16:49:48.095942	4	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
235	COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	4	165	2	2015-06-21 16:49:48.162998	2015-06-21 16:49:48.162998	4	accepted	\N	\N	N/A	\N	2	2	0	0	1	f	\N
236	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y variant has been shown to confer resistance to crizotinib.	8	166	6	2015-06-21 16:49:48.19987	2015-06-21 16:49:48.19987	4	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
237	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M variant has been shown to confer resistance to crizotinib.	8	166	7	2015-06-21 16:49:48.252362	2015-06-21 16:49:48.252362	4	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
238	The T790M mutation in EGFR has been shown to confer resistance to the tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are placed on the drug are likely to relapse.	25	167	34	2015-06-21 16:49:48.292678	2015-06-21 16:49:48.292678	5	accepted	\N	\N	7:55249071-55249071 (C->T)	\N	0	2	0	0	1	f	\N
239	In non-small cell lung cancer, the appearance of T790M mutation leads to resistance to gefitinib.	8	168	34	2015-06-21 16:49:48.356324	2015-06-21 16:49:48.356324	3	accepted	\N	\N	7:55249071-55249071 (C->T)	\N	1	2	0	0	1	f	\N
240	In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804), MET amplification and EGFR T790M were found to be the primary drivers of resistance.	8	169	34	2015-06-21 16:49:48.388722	2015-06-21 16:49:48.388722	4	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
241	The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	22	170	39	2015-06-21 16:49:48.433066	2015-06-21 16:49:48.433066	5	accepted	\N	\N	17:37880220-37880220 (T->C)	\N	2	2	0	0	1	f	\N
242	MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	22	171	46	2015-06-21 16:49:48.471871	2015-06-21 16:49:48.471871	3	accepted	\N	\N	N/A	\N	2	2	0	0	1	f	\N
243	MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	22	171	47	2015-06-21 16:49:48.52682	2015-06-21 16:49:48.52682	3	accepted	\N	\N	N/A	\N	2	2	0	0	1	f	\N
244	MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	22	171	48	2015-06-21 16:49:48.564898	2015-06-21 16:49:48.564898	3	accepted	\N	\N	6:152419923-152419923 (A->G)	\N	2	2	0	0	1	f	\N
245	MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	22	171	49	2015-06-21 16:49:48.601145	2015-06-21 16:49:48.601145	3	accepted	\N	\N	6:152419922-152419922 (T->A)	\N	2	2	0	0	1	f	\N
246	MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	22	171	50	2015-06-21 16:49:48.654061	2015-06-21 16:49:48.654061	3	accepted	\N	\N	6:152419923-152419923 (A->C)	\N	2	2	0	0	1	f	\N
247	Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated with sorafenib resistance	3	172	56	2015-06-21 16:49:48.690755	2015-06-21 16:49:48.690755	4	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
248	Sorafenib is effective in patients with FLT3 internal tandem repeat mutations, but not in patients with wild type or D835 mutation.	3	173	56	2015-06-21 16:49:48.726457	2015-06-21 16:49:48.726457	4	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
249	KIT V654A results in imatinib resistance in GIST patient-derived cell lines.	14	62	73	2015-06-21 16:49:48.755598	2015-06-21 16:49:48.755598	3	accepted	\N	\N	4:55594258-55594258 (T->C)	\N	2	2	0	0	1	f	\N
250	Patients with colorectal cancer who harbor KRAS mutation have low response rate to cetuximab	11	174	74	2015-06-21 16:49:48.803378	2015-06-21 16:49:48.803378	4	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
251	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	26	175	148	2015-06-21 16:49:48.841911	2015-06-21 16:49:48.841911	3	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
252	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	26	175	78	2015-06-21 16:49:48.886304	2015-06-21 16:49:48.886304	3	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
253	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	26	175	79	2015-06-21 16:49:48.92515	2015-06-21 16:49:48.92515	3	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
254	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	26	175	147	2015-06-21 16:49:48.963207	2015-06-21 16:49:48.963207	3	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
255	In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804), MET amplification and EGFR T790M were found to be the primary drivers of resistance.	8	169	146	2015-06-21 16:49:49.006681	2015-06-21 16:49:49.006681	4	accepted	\N	\N	N/A	\N	1	2	0	0	1	f	\N
256	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	11	176	103	2015-06-21 16:49:49.052983	2015-06-21 16:49:49.052983	3	accepted	\N	\N	3:178936082-178936082 (G->A)	\N	4	2	0	0	1	f	\N
257	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	11	176	104	2015-06-21 16:49:49.085076	2015-06-21 16:49:49.085076	3	accepted	\N	\N	3:178936091-178936091 (G->A)	\N	4	2	0	0	1	f	\N
258	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.	11	176	107	2015-06-21 16:49:49.118169	2015-06-21 16:49:49.118169	3	accepted	\N	\N	3:178952085-178952085 (A->G)	\N	4	2	0	0	1	f	\N
259	Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide results in high rates of complete remission in patients.	3	164	1	2015-06-21 16:49:49.154665	2015-06-21 16:49:49.154665	5	accepted	\N	\N	N/A	\N	0	2	0	0	0	f	\N
260	The clinical use of imatinib in patients with BCR-ABL fusion has resulted in drastic sensitivity to the drug.	4	156	1	2015-06-21 16:49:49.185298	2015-06-21 16:49:49.185298	5	accepted	\N	\N	N/A	\N	0	2	0	0	0	f	\N
261	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.	4	156	1	2015-06-21 16:49:49.21297	2015-06-21 16:49:49.21297	4	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
262	In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment with crizotinib has shown to be effective in acheiving stable disease.	8	166	5	2015-06-21 16:49:49.248906	2015-06-21 16:49:49.248906	4	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
263	Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib (PD-0332991) significantly increased median survival.	27	177	23	2015-06-21 16:49:49.287979	2015-06-21 16:49:49.287979	3	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
264	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	28	178	25	2015-06-21 16:49:49.322334	2015-06-21 16:49:49.322334	4	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
265	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	28	178	26	2015-06-21 16:49:49.353729	2015-06-21 16:49:49.353729	4	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
266	In 3T3 fibroblasts expressing the G253C mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	8	179	141	2015-06-21 16:49:49.388535	2015-06-21 16:49:49.388535	4	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
267	In 3T3 fibroblasts expressing the G505S mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	8	179	142	2015-06-21 16:49:49.417049	2015-06-21 16:49:49.417049	4	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
268	In 3T3 fibroblasts expressing the G774V mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	8	179	144	2015-06-21 16:49:49.444445	2015-06-21 16:49:49.444445	4	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
269	In 3T3 fibroblasts expressing the I638F mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	8	179	143	2015-06-21 16:49:49.475	2015-06-21 16:49:49.475	4	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
270	In 3T3 fibroblasts expressing the L239R mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	8	179	140	2015-06-21 16:49:49.507948	2015-06-21 16:49:49.507948	4	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
271	In 3T3 fibroblasts expressing the L63V mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	8	179	139	2015-06-21 16:49:49.546136	2015-06-21 16:49:49.546136	4	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
272	In a patient expressing a DDR2 S768R mutation, treatment with dasatinib plus erlotinib acheived a partial response.	8	179	145	2015-06-21 16:49:49.583661	2015-06-21 16:49:49.583661	3	accepted	\N	\N	N/A	\N	3	2	0	0	0	f	\N
273	Exon 19 deletion has been shown to be correlated with gefitinib response.	8	180	133	2015-06-21 16:49:49.670918	2015-06-21 16:49:49.670918	4	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
274	Gefinitib has been shown to be effective in treating cell lines with G719S missense mutations.	8	181	134	2015-06-21 16:49:49.70423	2015-06-21 16:49:49.70423	4	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
275	In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence of L858R mutation is prognostic for better progression free survival.	8	182	33	2015-06-21 16:49:49.735509	2015-06-21 16:49:49.735509	4	accepted	\N	\N	7:55259514-55259515 (CT->AG)	\N	1	2	0	0	0	f	\N
276	Gefinitib has been shown to be effective in treating cell lines with L858R missense mutations.	8	181	33	2015-06-21 16:49:49.768153	2015-06-21 16:49:49.768153	4	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
277	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.	8	183	34	2015-06-21 16:49:49.801136	2015-06-21 16:49:49.801136	3	accepted	\N	\N	7:55249071-55249071 (C->T)	\N	3	2	0	0	0	f	\N
278	In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR, especially when an L858R mutation is also present.	8	184	34	2015-06-21 16:49:49.834698	2015-06-21 16:49:49.834698	1	accepted	\N	\N	7:55249071-55249071 (C->T)	\N	4	2	0	0	0	f	\N
279	In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.	22	170	35	2015-06-21 16:49:49.869973	2015-06-21 16:49:49.869973	4	accepted	\N	\N	17:37880261-37880261 (G->C)	\N	2	2	0	0	0	f	\N
280	In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.	22	170	36	2015-06-21 16:49:49.919079	2015-06-21 16:49:49.919079	4	accepted	\N	\N	17:37880261-37880261 (G->T)	\N	2	2	0	0	0	f	\N
281	In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2, cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive to neratinib.	22	170	37	2015-06-21 16:49:49.961024	2015-06-21 16:49:49.961024	5	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
282	In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.	22	170	38	2015-06-21 16:49:49.993536	2015-06-21 16:49:49.993536	5	accepted	\N	\N	17:37868205-37868205 (G->C)	\N	2	2	0	0	0	f	\N
283	The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	22	170	39	2015-06-21 16:49:50.035146	2015-06-21 16:49:50.035146	5	accepted	\N	\N	17:37880220-37880220 (T->C)	\N	2	2	0	0	0	f	\N
284	MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	22	170	40	2015-06-21 16:49:50.066135	2015-06-21 16:49:50.066135	5	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
285	In MCF10A cell lines, the in-frame insertion of a single amino acid at position 780 was shown to be sensitive to neratinib.	22	170	41	2015-06-21 16:49:50.093644	2015-06-21 16:49:50.093644	5	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
286	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	22	170	42	2015-06-21 16:49:50.129488	2015-06-21 16:49:50.129488	5	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
287	In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.	22	170	43	2015-06-21 16:49:50.162657	2015-06-21 16:49:50.162657	5	accepted	\N	\N	17:37881616-37881616 (C->T)	\N	2	2	0	0	0	f	\N
288	In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.	22	170	44	2015-06-21 16:49:50.200705	2015-06-21 16:49:50.200705	5	accepted	\N	\N	17:37881000-37881000 (G->T)	\N	2	2	0	0	0	f	\N
289	In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.	22	170	45	2015-06-21 16:49:50.241731	2015-06-21 16:49:50.241731	5	accepted	\N	\N	17:37881332-37881332 (G->A)	\N	2	2	0	0	0	f	\N
290	The L536Q ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.	22	185	46	2015-06-21 16:49:50.2778	2015-06-21 16:49:50.2778	5	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
291	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	22	185	47	2015-06-21 16:49:50.315664	2015-06-21 16:49:50.315664	5	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
292	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	22	185	48	2015-06-21 16:49:50.353193	2015-06-21 16:49:50.353193	5	accepted	\N	\N	6:152419923-152419923 (A->G)	\N	2	2	0	0	0	f	\N
293	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	22	185	49	2015-06-21 16:49:50.393127	2015-06-21 16:49:50.393127	5	accepted	\N	\N	6:152419922-152419922 (T->A)	\N	2	2	0	0	0	f	\N
294	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	22	185	50	2015-06-21 16:49:50.435197	2015-06-21 16:49:50.435197	5	accepted	\N	\N	6:152419923-152419923 (A->C)	\N	2	2	0	0	0	f	\N
295	In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic sites and overall stable disease. However, this response did not meet criteria for RECIST partial response.	29	186	52	2015-06-21 16:49:50.477617	2015-06-21 16:49:50.477617	2	accepted	\N	\N	N/A	\N	3	2	0	0	0	f	\N
296	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This fusion is also shown to be recurrent in multiple other cancer types.	29	186	54	2015-06-21 16:49:50.514084	2015-06-21 16:49:50.514084	3	accepted	\N	\N	N/A	\N	3	2	0	0	0	f	\N
297	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	3	187	55	2015-06-21 16:49:50.559402	2015-06-21 16:49:50.559402	4	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
298	Sorafenib is effective in patients with FLT3 internal tandem repeats, but not in wild type or D835 mutation.	3	173	55	2015-06-21 16:49:50.592116	2015-06-21 16:49:50.592116	4	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
299	AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with FLT3 internal tandem repeat	3	188	55	2015-06-21 16:49:50.624289	2015-06-21 16:49:50.624289	4	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
300	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	3	187	56	2015-06-21 16:49:50.653364	2015-06-21 16:49:50.653364	4	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
301	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	30	189	57	2015-06-21 16:49:50.687952	2015-06-21 16:49:50.687952	4	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
302	In a case study, a patient with anal melanoma harboring a 7 codon duplication in the juxtamambrane region (exon 11) of KIT showed marked response 4 months after imatinib treatment.	31	190	67	2015-06-21 16:49:50.725987	2015-06-21 16:49:50.725987	3	accepted	\N	\N	N/A	\N	3	2	0	0	0	f	\N
303	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	8	191	72	2015-06-21 16:49:50.769889	2015-06-21 16:49:50.769889	3	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
304	In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective for those that are refractory to imatinib.	14	192	73	2015-06-21 16:49:50.809986	2015-06-21 16:49:50.809986	3	accepted	\N	\N	4:55594258-55594258 (T->C)	\N	1	2	0	0	0	f	\N
305	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.	8	193	79	2015-06-21 16:49:50.85579	2015-06-21 16:49:50.85579	4	accepted	\N	\N	12:25398284-25398284 (C->T)	\N	2	2	0	0	0	f	\N
306	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	17	123	81	2015-06-21 16:49:50.888989	2015-06-21 16:49:50.888989	4	accepted	\N	\N	12:25398281-25398281 (C->T)	\N	1	2	0	0	0	f	\N
307	In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted from temozolomide. This benefit was also methylation status dependent, as those without methylation did not see increased survival.	19	194	85	2015-06-21 16:49:50.925623	2015-06-21 16:49:50.925623	5	accepted	\N	\N	N/A	\N	0	2	0	0	0	f	\N
308	MGMT promoter methylation has been shown to confer sensitivity to alkylating agents, such as carmustine, in glioblastoma cells.	19	139	85	2015-06-21 16:49:50.957767	2015-06-21 16:49:50.957767	4	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
309	In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor, such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may be useful in treating patients with initial resistance to alkylating agents.	19	139	85	2015-06-21 16:49:51.018691	2015-06-21 16:49:51.018691	4	accepted	\N	\N	N/A	\N	4	2	0	0	0	f	\N
310	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	22	195	103	2015-06-21 16:49:51.050866	2015-06-21 16:49:51.050866	4	accepted	\N	\N	3:178936082-178936082 (G->A)	\N	2	2	0	0	0	f	\N
311	Rapamycin inhibits transformation induced by mutation in PIK3CA	22	196	103	2015-06-21 16:49:51.086567	2015-06-21 16:49:51.086567	3	accepted	\N	\N	3:178936082-178936082 (G->A)	\N	2	2	0	0	0	f	\N
312	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	22	195	104	2015-06-21 16:49:51.121677	2015-06-21 16:49:51.121677	4	accepted	\N	\N	3:178936091-178936091 (G->A)	\N	2	2	0	0	0	f	\N
313	Rapamycin inhibits transformation induced by mutation in PIK3CA	22	196	104	2015-06-21 16:49:51.158859	2015-06-21 16:49:51.158859	3	accepted	\N	\N	3:178936091-178936091 (G->A)	\N	2	2	0	0	0	f	\N
314	Breast cancer cell lines with H1047R mutation showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	22	195	107	2015-06-21 16:49:51.190672	2015-06-21 16:49:51.190672	4	accepted	\N	\N	3:178952085-178952085 (A->G)	\N	2	2	0	0	0	f	\N
315	Rapamycin inhibits transformation induced by mutation in PIK3CA	22	196	107	2015-06-21 16:49:51.218593	2015-06-21 16:49:51.218593	3	accepted	\N	\N	3:178952085-178952085 (A->G)	\N	2	2	0	0	0	f	\N
316	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	32	197	108	2015-06-21 16:49:51.251215	2015-06-21 16:49:51.251215	5	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
317	Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse to PI3K-mTOR inhibitors.	24	198	110	2015-06-21 16:49:51.281085	2015-06-21 16:49:51.281085	4	accepted	\N	\N	10:89717672-89717672 (C->T)	\N	2	2	0	0	0	f	\N
318	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	32	197	111	2015-06-21 16:49:51.33864	2015-06-21 16:49:51.33864	5	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
319	Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.	22	199	116	2015-06-21 16:49:51.369905	2015-06-21 16:49:51.369905	3	accepted	\N	\N	17:7578406-7578406 (C->T)	\N	2	2	0	0	0	f	\N
320	A patient with metastatic bladder cancer that responded well to the mTOR inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del, p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations also had mild responses to everolimus.	33	200	124	2015-06-21 16:49:51.409913	2015-06-21 16:49:51.409913	3	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
321	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	18	200	126	2015-06-21 16:49:51.444375	2015-06-21 16:49:51.444375	3	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
322	AML patients with D835 mutation of FLT3 did not have worse overall survival than patients with wild type FLT3	3	152	56	2015-06-21 16:49:51.478052	2015-06-21 16:49:51.478052	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
323	The complete remission and overall survival rates in patients with IDH1 R132 mutation is not significantly different from those who do not have this mutation	3	201	58	2015-06-21 16:49:51.508012	2015-06-21 16:49:51.508012	2	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
324	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	3	159	59	2015-06-21 16:49:51.536744	2015-06-21 16:49:51.536744	2	accepted	\N	\N	2:209113113-209113113 (G->A)	\N	1	1	0	1	6	f	\N
325	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	3	159	60	2015-06-21 16:49:51.566226	2015-06-21 16:49:51.566226	3	accepted	\N	\N	2:209113112-209113112 (C->T)	\N	1	1	0	1	6	f	\N
326	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	3	159	61	2015-06-21 16:49:51.594821	2015-06-21 16:49:51.594821	3	accepted	\N	\N	2:209113112-209113112 (C->A)	\N	1	1	0	1	6	f	\N
327	AML patients with IDH2 mutations such as R140Q/L have event free survival and overall survival similar to those with wild-type IDH2	3	202	62	2015-06-21 16:49:51.626533	2015-06-21 16:49:51.626533	2	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
328	In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact overall survival or disease free survival.	3	203	62	2015-06-21 16:49:51.656452	2015-06-21 16:49:51.656452	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
329	In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L do not confer prognostic value (overall survival)	34	204	62	2015-06-21 16:49:51.687042	2015-06-21 16:49:51.687042	2	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
330	R140Q mutation in IDH2 does not have prognostic value in patients with MDS.	34	205	62	2015-06-21 16:49:51.720596	2015-06-21 16:49:51.720596	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
331	AML patients with IDH2 mutations such as R172K have event free survival and overall survival similar to those with wild-type IDH2.	3	202	63	2015-06-21 16:49:51.747679	2015-06-21 16:49:51.747679	2	accepted	\N	\N	15:90631838-90631838 (C->T)	\N	1	1	0	1	6	f	\N
332	In patients suffering from MDS, the presence of IDH2 mutations such as R172K do not confer prognostic value (overall survival)	34	204	63	2015-06-21 16:49:51.776434	2015-06-21 16:49:51.776434	2	accepted	\N	\N	15:90631838-90631838 (C->T)	\N	1	1	0	1	6	f	\N
333	KIT mutations were identified in morphologically benign, incidentally discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating is not a prognostic marker for GIST	14	206	66	2015-06-21 16:49:51.804682	2015-06-21 16:49:51.804682	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
334	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	23	207	66	2015-06-21 16:49:51.844088	2015-06-21 16:49:51.844088	2	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
335	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	23	207	71	2015-06-21 16:49:51.877248	2015-06-21 16:49:51.877248	2	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
336	In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12 mutation did not impact overall or disease free survival.	17	208	76	2015-06-21 16:49:51.912383	2015-06-21 16:49:51.912383	4	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
337	There is no strong association between KRAS mutation status and overall or progression free survival in patients with NSCLC	8	209	76	2015-06-21 16:49:51.943813	2015-06-21 16:49:51.943813	3	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
338	In patients with AML, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	3	154	127	2015-06-21 16:49:51.976239	2015-06-21 16:49:51.976239	2	accepted	\N	\N	N/A	\N	4	1	0	1	6	f	\N
339	In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	34	154	127	2015-06-21 16:49:52.00685	2015-06-21 16:49:52.00685	2	accepted	\N	\N	N/A	\N	4	1	0	1	6	f	\N
340	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	3	154	128	2015-06-21 16:49:52.03651	2015-06-21 16:49:52.03651	2	accepted	\N	\N	N/A	\N	4	1	0	1	6	f	\N
341	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	34	154	128	2015-06-21 16:49:52.071333	2015-06-21 16:49:52.071333	2	accepted	\N	\N	N/A	\N	4	1	0	1	6	f	\N
342	Three studies have found no significant survival impact for CCND1 in lung cancer.	8	210	19	2015-06-21 16:49:52.114496	2015-06-21 16:49:52.114496	3	accepted	\N	\N	N/A	\N	1	1	0	1	3	f	\N
343	PTEN nonsense mutations, including R233*, have been shown to be inactivating and loss-of-function, but do not have prognostic value in glioblastoma multiforme patients.	19	211	110	2015-06-21 16:49:52.147196	2015-06-21 16:49:52.147196	3	accepted	\N	\N	10:89717672-89717672 (C->T)	\N	1	1	0	1	3	f	\N
344	BCR-ABL inhibitors such as imatinib have lead to significantly improved prognosis, response rate, overall survival, and patient outcome in CML patients compared to previous therapeutic regimens.	4	156	1	2015-06-21 16:49:52.181277	2015-06-21 16:49:52.181277	5	accepted	\N	\N	N/A	\N	0	1	0	0	2	f	\N
345	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	28	212	25	2015-06-21 16:49:52.213102	2015-06-21 16:49:52.213102	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
346	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	28	212	26	2015-06-21 16:49:52.246284	2015-06-21 16:49:52.246284	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
347	Median survival of patients with EGFR L858R mutation is better than those with wild type EGFR.	8	213	33	2015-06-21 16:49:52.283159	2015-06-21 16:49:52.283159	3	accepted	\N	\N	7:55259514-55259515 (CT->AG)	\N	1	1	0	0	2	f	\N
348	In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic for better survival compared to patients who harbor wild type mutation.	35	214	58	2015-06-21 16:49:52.340965	2015-06-21 16:49:52.340965	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
349	In patients with glioblastoma, those harboring IDH1 R132 mutation has higher overall survival compared to those who do not have IDH1 mutation	36	215	58	2015-06-21 16:49:52.370453	2015-06-21 16:49:52.370453	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
350	In AML, patients with IDH2 R140K mutation have improved overall survival compared to those with wild-type IDH2	3	216	62	2015-06-21 16:49:52.399561	2015-06-21 16:49:52.399561	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
351	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	17	123	81	2015-06-21 16:49:52.432525	2015-06-21 16:49:52.432525	4	accepted	\N	\N	12:25398281-25398281 (C->T)	\N	1	1	0	0	2	f	\N
352	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	34	217	114	2015-06-21 16:49:52.465587	2015-06-21 16:49:52.465587	3	accepted	\N	\N	2:198267359-198267359 (C->A)	\N	4	1	0	0	2	f	\N
353	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	34	217	115	2015-06-21 16:49:52.49885	2015-06-21 16:49:52.49885	3	accepted	\N	\N	2:198266834-198266834 (T->C)	\N	4	1	0	0	2	f	\N
354	Inceased copy number of CCND1 is associated with poorer overall survival.	8	210	18	2015-06-21 16:49:52.533769	2015-06-21 16:49:52.533769	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
355	Three studies have associated CCND1 expression with poorer survival.	8	210	19	2015-06-21 16:49:52.562002	2015-06-21 16:49:52.562002	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
356	In ER-positive breast cancer patients, Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis.	22	157	20	2015-06-21 16:49:52.596138	2015-06-21 16:49:52.596138	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
357	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in head and neck squamous cell carcinoma.	37	218	20	2015-06-21 16:49:52.631926	2015-06-21 16:49:52.631926	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
358	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in mantle cell lymphoma.	38	219	20	2015-06-21 16:49:52.667302	2015-06-21 16:49:52.667302	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
359	Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.	21	220	21	2015-06-21 16:49:52.697553	2015-06-21 16:49:52.697553	4	accepted	\N	\N	N/A	\N	2	1	0	0	3	f	\N
360	Cyclin D2 overexpression is associated with the increased depth of cancer invasion in gastric cancers.	21	220	21	2015-06-21 16:49:52.726276	2015-06-21 16:49:52.726276	4	accepted	\N	\N	N/A	\N	2	1	0	0	3	f	\N
361	Cyclin D2 overexpression is associated with the presence pf lymph node metastasis in gastric cancers.	21	220	21	2015-06-21 16:49:52.754005	2015-06-21 16:49:52.754005	4	accepted	\N	\N	N/A	\N	2	1	0	0	3	f	\N
362	Cyclin D2 overexpression is associated with vascular invasion by cancer cells in gastric cancers.	21	220	21	2015-06-21 16:49:52.789032	2015-06-21 16:49:52.789032	4	accepted	\N	\N	N/A	\N	2	1	0	0	3	f	\N
363	Patients with high expression of total cyclin E and low-molecular weight cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard ratio for these patients in 13.3 times higher than those with normal cyclin E levels.	22	221	24	2015-06-21 16:49:52.818211	2015-06-21 16:49:52.818211	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
364	Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced stage gastric carcinoma.	39	222	24	2015-06-21 16:49:52.848899	2015-06-21 16:49:52.848899	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
365	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	39	222	24	2015-06-21 16:49:52.876361	2015-06-21 16:49:52.876361	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
366	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	23	222	24	2015-06-21 16:49:52.904769	2015-06-21 16:49:52.904769	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
367	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter overall survival.	8	223	27	2015-06-21 16:49:52.935389	2015-06-21 16:49:52.935389	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
368	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter time to recurrence.	8	223	27	2015-06-21 16:49:52.963827	2015-06-21 16:49:52.963827	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
369	In NSCLC patients with T790M and another activating mutations, their progression free survival is shorter compared to those who do not possess T790M mutation.	8	224	34	2015-06-21 16:49:52.996331	2015-06-21 16:49:52.996331	4	accepted	\N	\N	7:55249071-55249071 (C->T)	\N	1	1	0	0	3	f	\N
370	Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall survival compared to patients with L858R or other exon 19 activating mutations.	8	225	34	2015-06-21 16:49:53.026384	2015-06-21 16:49:53.026384	4	accepted	\N	\N	7:55249071-55249071 (C->T)	\N	1	1	0	0	3	f	\N
371	AML patients with internal tandem repeat of FLT3 has worse overall survival than patients with wild type FLT3	3	152	55	2015-06-21 16:49:53.064221	2015-06-21 16:49:53.064221	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
372	Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor survival and inferior disease free survival in patients with AML.	3	226	55	2015-06-21 16:49:53.09371	2015-06-21 16:49:53.09371	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
373	In AML patients, FLT3-TKD mutation was associated with poorer disease free survival compared to patients with wild type FLT3	3	227	56	2015-06-21 16:49:53.125175	2015-06-21 16:49:53.125175	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
374	IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated with worse overall survival	40	228	58	2015-06-21 16:49:53.16148	2015-06-21 16:49:53.16148	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
375	In AML, patients with an IDH2 R172K mutation have worse overall survival compared to those with wild-type IDH2	3	216	63	2015-06-21 16:49:53.190819	2015-06-21 16:49:53.190819	3	accepted	\N	\N	15:90631838-90631838 (C->T)	\N	1	1	0	0	3	f	\N
376	KIT mutation is associated with worse overall and cause-specific prognosis in patients with GIST compared to patients with wildtype KIT	14	160	66	2015-06-21 16:49:53.218486	2015-06-21 16:49:53.218486	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
377	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	41	229	76	2015-06-21 16:49:53.253366	2015-06-21 16:49:53.253366	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
378	A meta-analysis showed KRAS mutation is associated with worse outcome in patients with NSCLC	8	230	76	2015-06-21 16:49:53.282116	2015-06-21 16:49:53.282116	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
379	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	42	229	80	2015-06-21 16:49:53.311174	2015-06-21 16:49:53.311174	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
380	NOTCH1 gain-of-function mutations, including D1643H, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	8	231	135	2015-06-21 16:49:53.34111	2015-06-21 16:49:53.34111	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
381	NOTCH1 gain-of-function mutations, including R2328W, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	8	231	136	2015-06-21 16:49:53.371498	2015-06-21 16:49:53.371498	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
382	NOTCH1 gain-of-function mutations, including S2275fs, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	8	231	138	2015-06-21 16:49:53.404146	2015-06-21 16:49:53.404146	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
383	NOTCH1 gain-of-function mutations, including V2444fs, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	8	231	137	2015-06-21 16:49:53.435448	2015-06-21 16:49:53.435448	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
384	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	11	232	103	2015-06-21 16:49:53.468582	2015-06-21 16:49:53.468582	2	accepted	\N	\N	3:178936082-178936082 (G->A)	\N	4	1	0	0	3	f	\N
385	The presence of exon 9 (such as E542K) or exon 20 mutations were associated with poorer survival compared to those with wild-type PIK3CA mutation.	11	176	103	2015-06-21 16:49:53.497844	2015-06-21 16:49:53.497844	3	accepted	\N	\N	3:178936082-178936082 (G->A)	\N	4	1	0	0	3	f	\N
386	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	11	232	104	2015-06-21 16:49:53.552402	2015-06-21 16:49:53.552402	2	accepted	\N	\N	3:178936091-178936091 (G->A)	\N	4	1	0	0	3	f	\N
387	The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	11	176	104	2015-06-21 16:49:53.584462	2015-06-21 16:49:53.584462	3	accepted	\N	\N	3:178936091-178936091 (G->A)	\N	4	1	0	0	3	f	\N
388	The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	11	176	107	2015-06-21 16:49:53.614881	2015-06-21 16:49:53.614881	3	accepted	\N	\N	3:178952085-178952085 (A->G)	\N	4	1	0	0	3	f	\N
389	Breast cancer patients who harbor R175H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	22	233	116	2015-06-21 16:49:53.645663	2015-06-21 16:49:53.645663	3	accepted	\N	\N	17:7578406-7578406 (C->T)	\N	1	1	0	0	3	f	\N
390	Breast cancer patients who harbor R248Q mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	22	233	117	2015-06-21 16:49:53.676988	2015-06-21 16:49:53.676988	3	accepted	\N	\N	17:7577538-7577538 (C->T)	\N	1	1	0	0	3	f	\N
391	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R248 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	22	234	117	2015-06-21 16:49:53.710804	2015-06-21 16:49:53.710804	3	accepted	\N	\N	17:7577538-7577538 (C->T)	\N	1	1	0	0	3	f	\N
392	In breast cancer patients harboring R248W mutation, the prognosis is worse than any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.	22	233	118	2015-06-21 16:49:53.740172	2015-06-21 16:49:53.740172	3	accepted	\N	\N	17:7577539-7577539 (G->A)	\N	1	1	0	0	3	f	\N
393	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	22	234	119	2015-06-21 16:49:53.773115	2015-06-21 16:49:53.773115	3	accepted	\N	\N	17:7577535-7577535 (C->G)	\N	1	1	0	0	3	f	\N
394	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	22	234	120	2015-06-21 16:49:53.801878	2015-06-21 16:49:53.801878	3	accepted	\N	\N	17:7577536-7577536 (T->A)	\N	1	1	0	0	3	f	\N
395	Breast cancer patients who harbor R273C mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	22	233	121	2015-06-21 16:49:53.829726	2015-06-21 16:49:53.829726	3	accepted	\N	\N	17:7577121-7577121 (G->A)	\N	1	1	0	0	3	f	\N
396	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	22	234	121	2015-06-21 16:49:53.860329	2015-06-21 16:49:53.860329	3	accepted	\N	\N	17:7577121-7577121 (G->A)	\N	1	1	0	0	3	f	\N
397	Breast cancer patients who harbor R273H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	22	233	122	2015-06-21 16:49:53.887706	2015-06-21 16:49:53.887706	3	accepted	\N	\N	17:7577120-7577120 (C->T)	\N	1	1	0	0	3	f	\N
398	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	22	234	122	2015-06-21 16:49:53.922743	2015-06-21 16:49:53.922743	3	accepted	\N	\N	17:7577120-7577120 (C->T)	\N	1	1	0	0	3	f	\N
399	Breast tumors with R175H mutations are more responsive to doxorubicin than breast tumors with wild type TP53.	22	234	123	2015-06-21 16:49:53.956194	2015-06-21 16:49:53.956194	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
400	After adjusting for age and cancer stage, presence of U2AF mutation such as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.	34	235	127	2015-06-21 16:49:53.987851	2015-06-21 16:49:53.987851	3	accepted	\N	\N	N/A	\N	4	1	0	0	3	f	\N
401	After adjusting for age and cancer stage, the presence of U2AF mutations such as S34Y/F are prognostic for poorer survival outcomes in patients with MDS.	34	235	128	2015-06-21 16:49:54.018052	2015-06-21 16:49:54.018052	3	accepted	\N	\N	N/A	\N	4	1	0	0	3	f	\N
402	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	8	236	79	2015-06-21 16:49:54.052498	2015-06-21 16:49:54.052498	2	accepted	\N	\N	12:25398284-25398284 (C->T)	\N	3	2	0	0	1	f	\N
403	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	43	237	30	2015-06-21 16:49:54.090483	2015-06-21 16:49:54.090483	3	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
404	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	43	237	84	2015-06-21 16:49:54.127943	2015-06-21 16:49:54.127943	3	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
405	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	44	238	31	2015-06-21 16:49:54.165191	2015-06-21 16:49:54.165191	4	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
406	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	44	238	109	2015-06-21 16:49:54.196281	2015-06-21 16:49:54.196281	4	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
407	The t(14;18) transloaction of BCL2 is shown to be associated with germinal-center B-cell–like diffuse large-B-cell lymphomas.	45	239	151	2015-06-21 16:49:54.229006	2015-06-21 16:49:54.229006	4	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
408	Amplification of FOXP1 is shown to be associated with ABC DLBCL	45	240	153	2015-06-21 16:49:54.258392	2015-06-21 16:49:54.258392	3	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
409	Amplification of REL is shown to be associated with germinal-center B-cell–like diffuse large-B-cell lymphomas.	45	239	154	2015-06-21 16:49:54.301544	2015-06-21 16:49:54.301544	4	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
410	Amplification of REL is shown to be associated with germinal-center B-cell–like diffuse large-B-cell lymphomas.	45	239	154	2015-06-21 16:49:54.32927	2015-06-21 16:49:54.32927	4	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
411	In non-APL patients with AML, those who had mutations in RUNX1 and were treated with allogeneic hematopoietic stem-cell transplantation had longer relapse-free survival than those treated with conventional post remmision treatment.	3	241	155	2015-06-21 16:49:54.359511	2015-06-21 16:49:54.359511	4	accepted	\N	\N	N/A	\N	1	0	0	0	4	f	\N
412	OSI-906, an IGF1R/INSR inhibitor, does not have a significant effect on chemotherapy resistant castration-resistant prostate cancer cell lines.	46	242	156	2015-06-21 16:49:54.399172	2015-06-21 16:49:54.399172	3	accepted	\N	\N	N/A	\N	2	2	0	0	6	f	\N
413	NSCLC patients with exon 19 deletions in EGFR had longer overall survival than patients with an L858R mutation when treated with gefitinib and erlotinib (38 versus 17 months; P = 0.04).	8	243	133	2015-06-21 16:49:54.429268	2015-06-21 16:49:54.429268	4	accepted	\N	\N	N/A	\N	1	2	0	0	0	f	\N
414	OSI-906, an IGF1R/INSR inhibitor, sensitizes cell lines to docetaxel and cabazitaxel, resulting in better overall survival in chemotherapy resistant CRPC cell lines.	46	244	156	2015-06-21 16:49:54.468258	2015-06-21 16:49:54.468258	3	accepted	\N	\N	N/A	\N	2	2	0	0	0	f	\N
415	In a cohort of 783 young adult patients with acute myeloid leukemia, those with TET2 mutations did not have significantly different rates of complete recovery, refractory disease, or hypoplastic death.	3	245	157	2015-06-21 16:49:54.508327	2015-06-21 16:49:54.508327	4	accepted	\N	\N	N/A	\N	1	1	0	1	6	f	\N
416	In patients with ovarian carcinomas, those with Aurora kinase A expression (as measured by immunohistochemistry) had increased progression-free and overall survival compared to those without expression.	47	246	158	2015-06-21 16:49:54.550337	2015-06-21 16:49:54.550337	3	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
417	In patients with myelodysplastic syndromes, those with TET2 mutations have been shown to have better overall survival than patients without TET2 mutations.	34	247	157	2015-06-21 16:49:54.584613	2015-06-21 16:49:54.584613	4	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
418	In patients with renal clear cell carcinoma, patients with VHL mutations had longer cancer-specific survival.	48	248	160	2015-06-21 16:49:54.620861	2015-06-21 16:49:54.620861	4	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
419	In patients with cytogenetically normal acute myeloid leukemia, those in the intermediate-I risk group with TET2 mutations had higher rates of complete remission.	3	245	157	2015-06-21 16:49:54.654239	2015-06-21 16:49:54.654239	4	accepted	\N	\N	N/A	\N	1	1	0	0	2	f	\N
420	In renal clear cell carcinoma, patients with PBRM1 mutations did not have a significantly different cancer specific survival.	48	249	161	2015-06-21 16:49:54.686584	2015-06-21 16:49:54.686584	3	accepted	\N	\N	N/A	\N	1	1	0	0	6	f	\N
421	In patients with cytogenetically normal acute myeloid leukemia, those in the intermediate-I risk group with TET2 mutations did not have significantly different rates of event-free survival, complete remission, disease-free survival, or overall survival.	3	250	157	2015-06-21 16:49:54.71818	2015-06-21 16:49:54.71818	4	accepted	\N	\N	N/A	\N	1	1	0	0	6	f	\N
422	In renal clear cell carcinoma, patients with VHL mutations did not have a significant impact on cancer specific survival.	48	249	160	2015-06-21 16:49:54.773894	2015-06-21 16:49:54.773894	3	accepted	\N	\N	N/A	\N	1	1	0	0	6	f	\N
423	In patients with Diffuse Large B-Cell Lymphoma, those with increased levels of BCL2 were shown to have decreased overall, disease-free, and relapse-free survival.	45	251	152	2015-06-21 16:49:54.810582	2015-06-21 16:49:54.810582	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
424	In patients with Diffuse Large B-Cell Lymphoma, those with increased levels of BCL2 were shown to have decreased OS, DFS, and RFS.	45	251	152	2015-06-21 16:49:54.84319	2015-06-21 16:49:54.84319	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
425	In myelodysplastic syndrome, patients with point mutations in EZH2 have worse overall survival that their wild-type counterparts.	49	252	163	2015-06-21 16:49:54.880857	2015-06-21 16:49:54.880857	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
426	In patients classified as intermediate-risk via cytogenetics, TET2 mutations have been shown to be correlated with poor prognosis.	3	253	157	2015-06-21 16:49:54.914026	2015-06-21 16:49:54.914026	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
427	In patients with cytogenetically normal acute myeloid leukemia, those in the ELN favorable-risk group with TET2 mutations had shorter event-free survival, lower rates of complete remission, and shorter disease-free survival.	3	250	157	2015-06-21 16:49:54.943172	2015-06-21 16:49:54.943172	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
428	In patients with renal clear cell carcinoma, patients with loss of function VHL mutations had shorter tumor-specific survival.	48	254	159	2015-06-21 16:49:54.973357	2015-06-21 16:49:54.973357	3	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
429	In non-APL patients with AML, those who had mutations in RUNX1 had higher rates of refractory disease than those without.	3	241	155	2015-06-21 16:49:55.007845	2015-06-21 16:49:55.007845	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
430	In non-APL patients with AML, those who had mutations in RUNX1 had shorter event-free survival and relapse-free survival relative to wild-type.	3	241	155	2015-06-21 16:49:55.036245	2015-06-21 16:49:55.036245	4	accepted	\N	\N	N/A	\N	1	1	0	0	3	f	\N
431	In patients with Ewing's sarcoma, those with type 1 EWS-FLI1 fusions had longer overall survival than those with other types of EWS-FLI1 fusions.	50	255	164	2015-06-21 16:49:55.069308	2015-06-21 16:49:55.069308	4	accepted	\N	\N	N/A	\N	1	1	0	0	4	f	\N
\.


--
-- Data for Name: drugs_evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs_evidence_items (drug_id, evidence_item_id) FROM stdin;
1	1
1	2
1	3
1	4
1	5
1	6
1	7
1	8
1	9
1	10
2	11
3	12
3	13
4	14
5	15
5	16
6	17
7	18
8	19
9	20
10	21
11	22
11	23
1	24
1	25
1	26
1	27
1	28
1	29
1	30
1	31
12	32
12	33
5	34
13	34
6	34
14	35
15	35
16	36
17	37
12	38
18	39
6	40
19	41
20	42
21	43
21	44
21	45
21	46
21	47
18	48
1	49
1	50
1	51
1	52
1	53
1	54
1	55
1	56
1	57
5	58
5	59
5	60
1	61
1	62
1	63
1	64
1	65
1	66
1	67
1	68
1	69
1	70
1	71
1	72
22	73
1	74
23	75
23	76
24	77
1	78
1	79
1	80
1	81
25	82
26	82
27	83
27	84
27	85
1	86
27	87
28	88
6	89
1	90
22	91
3	92
22	93
19	93
22	94
22	95
19	95
29	96
30	96
30	97
31	97
32	98
4	98
33	98
4	99
22	100
22	101
1	102
1	103
1	104
1	105
1	106
1	107
1	108
1	109
1	110
1	111
1	112
1	113
1	114
1	115
1	116
1	117
1	118
1	119
1	120
1	121
1	122
16	123
16	124
12	125
16	126
34	127
1	128
1	129
2	130
35	131
2	132
35	133
16	134
16	135
16	136
34	137
16	138
1	139
1	140
17	141
18	142
12	143
36	144
16	145
34	146
2	147
37	148
38	149
34	149
1	150
1	151
38	152
39	153
40	154
41	155
1	156
1	157
1	158
1	159
1	160
1	161
1	162
1	163
1	164
1	165
1	166
1	167
1	168
1	169
1	170
1	171
1	172
1	173
1	174
1	175
1	176
1	177
1	178
1	179
1	180
1	181
1	182
1	183
1	184
1	185
1	186
1	187
1	188
1	189
1	190
1	191
1	192
1	193
1	194
1	195
1	196
1	197
1	198
1	199
1	200
1	201
1	202
1	203
1	204
1	205
1	206
1	207
1	208
1	209
1	210
42	211
42	212
1	213
1	214
1	215
1	216
1	217
1	218
1	219
1	220
1	221
1	222
1	223
1	224
1	225
1	226
1	227
1	228
14	229
15	229
14	230
15	230
43	231
5	232
5	233
5	234
5	235
12	236
12	237
15	238
14	239
44	240
45	241
46	242
46	243
46	244
46	245
46	246
6	247
6	248
5	249
16	250
14	251
15	251
14	252
15	252
14	253
15	253
14	254
15	254
44	255
16	256
28	256
16	257
28	257
16	258
28	258
47	259
5	260
13	261
20	261
12	262
48	263
49	264
49	265
20	266
20	267
20	268
20	269
20	270
20	271
20	272
15	272
14	273
14	274
1	275
14	276
15	277
50	277
51	278
52	279
52	280
52	281
52	282
52	283
52	284
52	285
52	286
45	286
52	287
52	288
52	289
53	290
54	290
53	291
54	291
53	292
54	292
53	293
54	293
53	294
54	294
55	295
56	296
55	296
57	297
6	298
58	299
57	300
59	301
60	301
5	302
20	303
5	303
13	303
61	304
62	305
63	305
16	306
26	307
64	308
65	309
66	310
40	311
66	312
40	313
66	314
40	315
34	316
67	317
34	318
68	319
41	320
41	321
1	322
1	323
1	324
1	325
1	326
1	327
1	328
1	329
1	330
1	331
1	332
1	333
1	334
1	335
1	336
1	337
1	338
1	339
1	340
1	341
1	342
1	343
5	344
69	345
69	346
1	347
1	348
1	349
1	350
16	351
1	352
1	353
1	354
1	355
1	356
1	357
1	358
1	359
1	360
1	361
1	362
1	363
1	364
1	365
1	366
1	367
1	368
1	369
1	370
1	371
1	372
1	373
1	374
1	375
1	376
1	377
1	378
1	379
1	380
1	381
1	382
1	383
1	384
1	385
1	386
1	387
1	388
1	389
1	390
1	391
1	392
1	393
1	394
1	395
1	396
1	397
1	398
1	399
1	400
1	401
22	402
5	403
70	403
5	404
70	404
1	405
1	406
1	407
1	408
1	409
1	410
71	411
72	412
14	413
15	413
72	414
73	414
74	414
1	415
1	416
1	417
1	418
1	419
1	420
1	421
1	422
1	423
1	424
1	425
1	426
1	427
1	428
1	429
1	430
1	431
\.


--
-- Name: drugs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('drugs_id_seq', 74, true);


--
-- Data for Name: events; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY events (id, created_at, updated_at, action, description, originating_user_id, subject_id, subject_type) FROM stdin;
\.


--
-- Name: events_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('events_id_seq', 1, false);


--
-- Name: evidence_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_items_id_seq', 431, true);


--
-- Data for Name: feeds; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY feeds (id, created_at, updated_at, owner_id, event_id, acknowledged) FROM stdin;
\.


--
-- Name: feeds_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('feeds_id_seq', 1, false);


--
-- Data for Name: gene_aliases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY gene_aliases (id, name) FROM stdin;
1	CD246
2	NBLST3
3	AKT
4	CWS6
5	PKB
6	PKB-ALPHA
7	PRKBA
8	RAC
9	RAC-ALPHA
10	A-RAF
11	ARAF1
12	PKS2
13	RAFA1
14	ABL
15	JTK7
16	bcr/abl
17	c-ABL
18	c-ABL1
19	p150
20	v-abl
21	B-RAF1
22	BRAF1
23	NS7
24	RAFB1
25	BRCAI
26	BRCC1
27	BROVCA1
28	FANCS
29	IRIS
30	PNCA4
31	PPP1R53
32	PSCP
33	RNF53
34	BRCC2
35	BROVCA2
36	FACD
37	FAD
38	FAD1
39	FANCD
40	FANCD1
41	GLM3
42	PNCA2
43	XRCC11
44	BCL1
45	D11S287E
46	PRAD1
47	U21B31
48	KIAK0002
49	MPPH3
50	CCNE
51	MCPH12
52	PLSTIRE
53	CMM3
54	PSK-J3
55	ARF
56	CDK4I
57	CDKN2
58	CMM2
59	INK4
60	INK4A
61	MLM
62	MTS-1
63	MTS1
64	P14
65	P14ARF
66	P16
67	P16-INK4A
68	P16INK4
69	P16INK4A
70	P19
71	P19ARF
72	TP16
73	C/EBP-alpha
74	CEBP
75	HSPF1
76	Hdj1
77	Hsp40
78	RSPH16B
79	Sis1
80	PKACA
81	PPNAD4
82	DNMT3A2
83	M.HsaIIIA
84	TBRS
85	ERBB
86	ERBB1
87	HER1
88	NISBD2
89	PIG61
90	mENA
91	CD340
92	HER-2
93	HER-2/neu
94	HER2
95	MLN 19
96	NEU
97	NGL
98	TKR1
99	ER
100	ESR
101	ESRA
102	ESTRR
103	Era
104	NR3A1
105	BBDS
106	BEK
107	BFR-1
108	CD332
109	CEK3
110	CFD1
111	ECT1
112	JWS
113	K-SAM
114	KGFR
115	TK14
116	TK25
117	ACH
118	CD333
119	CEK2
120	HSFGFR3EX
121	JTK4
122	CD135
123	FLK-2
124	FLK2
125	STK1
126	DCML
127	IMD21
128	MONOMAC
129	NFE1B
130	HEL-216
131	HEL-S-26
132	IDCD
133	IDH
134	IDP
135	IDPC
136	PICD
137	D2HGA2
138	ICD-M
139	IDHM
140	IDPM
141	mNADP-IDH
142	JTK10
143	THCYT3
144	C-Kit
145	CD117
146	PBT
147	SCFR
148	C-K-RAS
149	CFC2
150	K-RAS2A
151	K-RAS2B
152	K-RAS4A
153	K-RAS4B
154	KI-RAS
155	KRAS1
156	KRAS2
157	NS
158	NS3
159	RASK2
160	CFC3
161	MAPKK1
162	MEK1
163	MKK1
164	PRKMK1
165	C-FMS
166	CD115
167	CSF-1R
168	CSFR
169	FIM2
170	FMS
171	HDLS
172	M-CSF-R
173	B23
174	NPM
175	ALPS4
176	CMNS
177	N-ras
178	NCMS
179	NRAS1
180	NS6
181	CLOVE
182	CWS5
183	MCAP
184	MCM
185	MCMTC
186	PI3K
187	p110-alpha
188	CD140A
189	PDGFR-2
190	PDGFR2
191	RHEPDGFRA
192	MYL
193	PP8675
194	RNF71
195	TRIM19
196	NR1B1
197	RAR
198	10q23del
199	BZS
200	CWS1
201	DEC
202	GLM2
203	MHAM
204	MMAC1
205	PTEN1
206	TEP1
207	CDHF12
208	CDHR16
209	HSCR1
210	MEN2A
211	MEN2B
212	MTC1
213	PTC
214	RET-ELE1
215	RET51
216	AML1
217	AML1-EVI-1
218	AMLCR1
219	CBFA2
220	EVI-1
221	PEBP2aB
222	Hsh155
223	MDS
224	PRP10
225	PRPF10
226	SAP155
227	SF3b155
228	BCC7
229	LFS1
230	P53
231	TRP53
232	LAM
233	TSC
234	PPP1R160
235	TSC4
236	FP793
237	RN
238	RNU2AF1
239	U2AF35
240	U2AFBP
241	AWT1
242	EWS-WT1
243	GUD
244	NPHS4
245	WAGR
246	WIT-2
247	WT33
248	AOS5
249	AOVD1
250	TAN1
251	hN1
252	MIG20a
253	NTRKR3
254	TKT
255	TYRO10
256	AUTS9
257	DFNB97
258	HGFR
259	RCCP2
260	c-Met
261	EWSR2
262	SIC-1
263	EWS
264	EWS-FLI1
265	bK984G1.4
266	KIAA1546
267	12CC4
268	HSPC215
269	MFH
270	QRF1
271	hFKH1B
272	C-Rel
273	HRCA1
274	RCA1
275	VHL1
276	pVHL
277	Bcl-2
278	PPP1R50
279	C11orf43
280	IGF-II
281	PP9974
282	AIK
283	ARK1
284	AURA
285	AURORA2
286	BTAK
287	PPP1R47
288	STK15
289	STK6
290	STK7
291	BAF180
292	PB1
293	ENX-1
294	ENX1
295	EZH1
296	EZH2b
297	KMT6
298	KMT6A
299	WVS
300	WVS2
\.


--
-- Data for Name: gene_aliases_genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY gene_aliases_genes (gene_alias_id, gene_id) FROM stdin;
1	1
2	1
3	2
4	2
5	2
6	2
7	2
8	2
9	2
10	3
11	3
12	3
13	3
14	4
15	4
16	4
17	4
18	4
19	4
20	4
21	5
22	5
23	5
24	5
25	6
26	6
27	6
28	6
29	6
30	6
31	6
32	6
33	6
34	7
35	7
36	7
37	7
38	7
39	7
40	7
41	7
42	7
43	7
44	8
45	8
46	8
47	8
48	9
49	9
50	11
51	12
52	12
53	13
54	13
55	14
56	14
57	14
58	14
59	14
60	14
61	14
62	14
63	14
64	14
65	14
66	14
67	14
68	14
69	14
70	14
71	14
72	14
73	15
74	15
75	16
76	16
77	16
78	16
79	16
80	17
81	17
82	18
83	18
84	18
85	19
86	19
87	19
88	19
89	19
90	19
91	20
92	20
93	20
94	20
95	20
96	20
97	20
98	20
99	21
100	21
101	21
102	21
103	21
104	21
105	22
106	22
107	22
108	22
109	22
110	22
111	22
112	22
113	22
114	22
115	22
116	22
117	23
118	23
119	23
120	23
121	23
122	24
123	24
124	24
125	24
126	25
127	25
128	25
129	25
130	26
131	26
132	26
133	26
134	26
135	26
136	26
137	27
138	27
133	27
139	27
134	27
140	27
141	27
142	28
143	28
144	29
145	29
146	29
147	29
148	30
149	30
150	30
151	30
152	30
153	30
154	30
155	30
156	30
157	30
158	30
159	30
160	31
161	31
162	31
163	31
164	31
165	33
166	33
167	33
168	33
169	33
170	33
171	33
172	33
173	35
174	35
175	36
176	36
177	36
178	36
179	36
180	36
181	37
182	37
183	37
184	37
185	37
186	37
187	37
188	38
189	38
190	38
191	38
192	39
193	39
194	39
195	39
196	40
197	40
198	41
199	41
200	41
201	41
202	41
203	41
204	41
205	41
206	41
207	42
208	42
209	42
210	42
211	42
212	42
213	42
214	42
215	42
216	43
217	43
218	43
219	43
220	43
221	43
222	44
223	44
224	44
225	44
226	44
227	44
228	45
229	45
230	45
231	45
232	46
233	46
232	47
234	47
235	47
236	48
237	48
238	48
239	48
240	48
241	49
242	49
243	49
244	49
245	49
246	49
247	49
248	50
249	50
250	50
251	50
252	51
253	51
254	51
255	51
256	52
257	52
258	52
259	52
260	52
261	53
262	53
263	54
264	54
265	54
266	55
223	55
267	56
268	56
269	56
270	56
271	56
272	57
273	58
274	58
275	58
276	58
277	59
278	59
279	60
280	60
281	60
282	61
283	61
284	61
285	61
286	61
287	61
288	61
289	61
290	61
291	62
292	62
293	63
294	63
295	63
296	63
297	63
298	63
299	63
300	63
\.


--
-- Name: gene_aliases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('gene_aliases_id_seq', 300, true);


--
-- Name: genes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('genes_id_seq', 63, true);


--
-- Data for Name: genes_sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes_sources (gene_id, source_id, created_at, updated_at) FROM stdin;
1	1	2015-06-21 16:49:19.373737	2015-06-21 16:49:19.373737
1	2	2015-06-21 16:49:19.381085	2015-06-21 16:49:19.381085
2	3	2015-06-21 16:49:19.645219	2015-06-21 16:49:19.645219
2	4	2015-06-21 16:49:19.64891	2015-06-21 16:49:19.64891
3	5	2015-06-21 16:49:19.908171	2015-06-21 16:49:19.908171
3	6	2015-06-21 16:49:19.912339	2015-06-21 16:49:19.912339
4	7	2015-06-21 16:49:20.173897	2015-06-21 16:49:20.173897
4	8	2015-06-21 16:49:20.178179	2015-06-21 16:49:20.178179
5	9	2015-06-21 16:49:20.453003	2015-06-21 16:49:20.453003
5	10	2015-06-21 16:49:20.456481	2015-06-21 16:49:20.456481
6	11	2015-06-21 16:49:20.751176	2015-06-21 16:49:20.751176
7	11	2015-06-21 16:49:21.059904	2015-06-21 16:49:21.059904
8	12	2015-06-21 16:49:21.317438	2015-06-21 16:49:21.317438
8	13	2015-06-21 16:49:21.321586	2015-06-21 16:49:21.321586
9	12	2015-06-21 16:49:21.554031	2015-06-21 16:49:21.554031
9	13	2015-06-21 16:49:21.556284	2015-06-21 16:49:21.556284
10	12	2015-06-21 16:49:21.769256	2015-06-21 16:49:21.769256
10	13	2015-06-21 16:49:21.771714	2015-06-21 16:49:21.771714
11	14	2015-06-21 16:49:21.997423	2015-06-21 16:49:21.997423
12	15	2015-06-21 16:49:22.227134	2015-06-21 16:49:22.227134
12	16	2015-06-21 16:49:22.23084	2015-06-21 16:49:22.23084
13	15	2015-06-21 16:49:22.461396	2015-06-21 16:49:22.461396
13	16	2015-06-21 16:49:22.465812	2015-06-21 16:49:22.465812
14	17	2015-06-21 16:49:22.793838	2015-06-21 16:49:22.793838
14	18	2015-06-21 16:49:22.798404	2015-06-21 16:49:22.798404
15	19	2015-06-21 16:49:23.073464	2015-06-21 16:49:23.073464
15	20	2015-06-21 16:49:23.077928	2015-06-21 16:49:23.077928
15	21	2015-06-21 16:49:23.08159	2015-06-21 16:49:23.08159
16	22	2015-06-21 16:49:23.326882	2015-06-21 16:49:23.326882
17	22	2015-06-21 16:49:23.558037	2015-06-21 16:49:23.558037
17	23	2015-06-21 16:49:23.562541	2015-06-21 16:49:23.562541
18	24	2015-06-21 16:49:23.795171	2015-06-21 16:49:23.795171
18	25	2015-06-21 16:49:23.800512	2015-06-21 16:49:23.800512
19	26	2015-06-21 16:49:24.051919	2015-06-21 16:49:24.051919
19	27	2015-06-21 16:49:24.056721	2015-06-21 16:49:24.056721
20	28	2015-06-21 16:49:24.342261	2015-06-21 16:49:24.342261
20	29	2015-06-21 16:49:24.346397	2015-06-21 16:49:24.346397
21	30	2015-06-21 16:49:24.620164	2015-06-21 16:49:24.620164
21	31	2015-06-21 16:49:24.624748	2015-06-21 16:49:24.624748
22	32	2015-06-21 16:49:24.928688	2015-06-21 16:49:24.928688
22	33	2015-06-21 16:49:24.93226	2015-06-21 16:49:24.93226
23	32	2015-06-21 16:49:25.205914	2015-06-21 16:49:25.205914
23	33	2015-06-21 16:49:25.208403	2015-06-21 16:49:25.208403
24	34	2015-06-21 16:49:25.464734	2015-06-21 16:49:25.464734
24	19	2015-06-21 16:49:25.46805	2015-06-21 16:49:25.46805
25	35	2015-06-21 16:49:25.713137	2015-06-21 16:49:25.713137
25	36	2015-06-21 16:49:25.718042	2015-06-21 16:49:25.718042
26	37	2015-06-21 16:49:25.974959	2015-06-21 16:49:25.974959
26	38	2015-06-21 16:49:25.97913	2015-06-21 16:49:25.97913
27	37	2015-06-21 16:49:26.268601	2015-06-21 16:49:26.268601
27	38	2015-06-21 16:49:26.272366	2015-06-21 16:49:26.272366
27	39	2015-06-21 16:49:26.278377	2015-06-21 16:49:26.278377
28	40	2015-06-21 16:49:26.537623	2015-06-21 16:49:26.537623
29	41	2015-06-21 16:49:26.781909	2015-06-21 16:49:26.781909
30	42	2015-06-21 16:49:27.059836	2015-06-21 16:49:27.059836
35	43	2015-06-21 16:49:28.255948	2015-06-21 16:49:28.255948
35	19	2015-06-21 16:49:28.25864	2015-06-21 16:49:28.25864
42	44	2015-06-21 16:49:30.006295	2015-06-21 16:49:30.006295
42	45	2015-06-21 16:49:30.010003	2015-06-21 16:49:30.010003
44	46	2015-06-21 16:49:30.527586	2015-06-21 16:49:30.527586
44	47	2015-06-21 16:49:30.531493	2015-06-21 16:49:30.531493
45	48	2015-06-21 16:49:30.78568	2015-06-21 16:49:30.78568
48	49	2015-06-21 16:49:31.492132	2015-06-21 16:49:31.492132
48	50	2015-06-21 16:49:31.498021	2015-06-21 16:49:31.498021
\.


--
-- Data for Name: ratings; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY ratings (id, value, evidence_item_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Name: ratings_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('ratings_id_seq', 1, false);


--
-- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY schema_migrations (version) FROM stdin;
20140805184308
20141021192035
20141022180523
20141022195329
20141108234203
20141108234935
20141109000136
20141114222425
20150107205621
20150115231117
20150116220720
20150116230539
20150116230632
20150204192653
20150205000010
20150205205734
20150205210725
20150210191015
20150210204445
20150219202227
20150220160805
20150220182109
20150223170106
20150227204142
20150227212146
20150303202505
20150305200429
20150319181853
20150416182633
20150501151624
20150511191452
20150512184905
20150528170506
20150528211748
20150602202928
20150609210448
20150611153328
20150611154621
20150611200713
\.


--
-- Name: sources_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('sources_id_seq', 255, true);


--
-- Data for Name: subscriptions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY subscriptions (id, user_id, subscribable_id, subscribable_type, type, created_at, updated_at, action_type, action_class) FROM stdin;
\.


--
-- Name: subscriptions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('subscriptions_id_seq', 1, false);


--
-- Data for Name: suggested_changes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY suggested_changes (id, suggested_changes, moderated_id, moderated_type, user_id, status, created_at, updated_at) FROM stdin;
\.


--
-- Name: suggested_changes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('suggested_changes_id_seq', 1, false);


--
-- Data for Name: tsv_releases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY tsv_releases (id, path, created_at, updated_at) FROM stdin;
\.


--
-- Name: tsv_releases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('tsv_releases_id_seq', 1, false);


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('users_id_seq', 1, false);


--
-- Data for Name: variant_groups; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_groups (id, name, description, created_at, updated_at, deleted, deleted_at) FROM stdin;
2	KIT Exon 17	\N	2015-06-21 16:49:39.971377	2015-06-21 16:49:39.971377	f	\N
4	KIT Exon 11	\N	2015-06-21 16:49:40.329405	2015-06-21 16:49:40.329405	f	\N
1	Imatinib Resistance	While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. 	2015-06-21 16:49:38.936157	2015-06-21 16:49:55.143921	f	\N
11	ALK Fusions	ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase the activity of the ALK oncogene relative to normal cells. While EML4 is the most common fusion partner, other 5' partners have been observed. The EML4-ALK fusion has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. 	2015-06-21 16:49:48.226633	2015-06-21 16:49:55.156899	f	\N
12	EGFR TKI Resistance	EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine kinase inhibitors have been developed to target EGFR pathway activity. One such inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting. However, the T790M missense mutation has shown to confer resistance to this inhibitor in cell lines and case studies.  	2015-06-21 16:49:48.324086	2015-06-21 16:49:55.172566	f	\N
3	Crizotinib Resistance	The ALK oncogene has long been considered a driving factor in non-small cell lung cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive this resistance. 	2015-06-21 16:49:40.249044	2015-06-21 16:49:55.183002	f	\N
5	Other V600's	While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants have also been observed and studied to a lesser degree. At first approximation, many of these variants seem to behave similarly to V600E, and treatment with dabrafenib has been shown to be effective. 	2015-06-21 16:49:41.914315	2015-06-21 16:49:55.193495	f	\N
14	HER2 Activating	HER2-positive breast cancer is widely recognized as a molecular subtype of breast cancer. While amplification of ERBB2 is a common mechanism for activation of the pathway, missense mutations have also demonstrated activation potential. These activating missense mutations also confer senstivity to the targeted therapeutic neratinib. 	2015-06-21 16:49:49.897387	2015-06-21 16:49:55.203675	f	\N
13	ESR1 Ligand-Binding Domain	ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone therapy has been widely effective in treating the disease, however sequencing of resistant patients has uncovered a number of missense mutations in mediating this resistance. Many of these mutations lie within the ligand-binding domain, and contribute to constitutive activity of the receptor. This has lead to the development of estrogen receptor degrading agents such as fulvestrant, which have shown early efficacy in clinical trials.  	2015-06-21 16:49:48.497237	2015-06-21 16:49:55.213541	f	\N
8	FGFR fusions	FGFR fusions have been demonstrated across many cancer types to have oncogenic potential. FGFR2 and FGFR3 have been shown to be fused to many 3' partners, which may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2 fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib and pazopanib have shown efficacy in case studies.  	2015-06-21 16:49:44.652677	2015-06-21 16:49:55.225584	f	\N
7	NPM1 Exon 12	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-06-21 16:49:43.312395	2015-06-21 16:49:55.235525	f	\N
15	PTEN Loss-of-Function	PTEN loss is a common event in breast cancer, as well as others. The most common single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell lines harboring this mutation behave similarly to cell lines harboring larger insertion or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded growth. 	2015-06-21 16:49:51.308362	2015-06-21 16:49:55.246788	f	\N
6	Motesanib Resistance	RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment of these patients with the targeted therapeutic motesanib has shown to be effective. However, the missense mutations C634W and M918T have shown to confer motesanib resistance in cell lines. 	2015-06-21 16:49:42.01241	2015-06-21 16:49:55.256642	f	\N
9	TSC Loss	Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types. Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor everolimus has seen significant anti-tumorigenic activity. 	2015-06-21 16:49:45.114015	2015-06-21 16:49:55.268076	f	\N
10	BRCA Germline Variants	BRCA germline variants have long been recognized as important potential predictors of breast and ovarian cancer risk. 	2015-06-21 16:49:47.291606	2015-06-21 16:49:55.279433	f	\N
\.


--
-- Data for Name: variant_group_variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_group_variants (variant_id, variant_group_id, created_at, updated_at) FROM stdin;
99	1	2015-06-21 16:49:38.955685	2015-06-21 16:49:38.955685
99	1	2015-06-21 16:49:39.441262	2015-06-21 16:49:39.441262
99	1	2015-06-21 16:49:39.482035	2015-06-21 16:49:39.482035
65	2	2015-06-21 16:49:39.986593	2015-06-21 16:49:39.986593
8	3	2015-06-21 16:49:40.264255	2015-06-21 16:49:40.264255
72	4	2015-06-21 16:49:40.345141	2015-06-21 16:49:40.345141
72	4	2015-06-21 16:49:40.71752	2015-06-21 16:49:40.71752
98	1	2015-06-21 16:49:40.769049	2015-06-21 16:49:40.769049
99	1	2015-06-21 16:49:40.818074	2015-06-21 16:49:40.818074
100	1	2015-06-21 16:49:40.857691	2015-06-21 16:49:40.857691
101	1	2015-06-21 16:49:40.895257	2015-06-21 16:49:40.895257
102	1	2015-06-21 16:49:40.930009	2015-06-21 16:49:40.930009
66	4	2015-06-21 16:49:41.149692	2015-06-21 16:49:41.149692
66	4	2015-06-21 16:49:41.35566	2015-06-21 16:49:41.35566
70	2	2015-06-21 16:49:41.436538	2015-06-21 16:49:41.436538
13	5	2015-06-21 16:49:41.928289	2015-06-21 16:49:41.928289
112	6	2015-06-21 16:49:42.025447	2015-06-21 16:49:42.025447
113	6	2015-06-21 16:49:42.065795	2015-06-21 16:49:42.065795
11	5	2015-06-21 16:49:42.710693	2015-06-21 16:49:42.710693
15	5	2015-06-21 16:49:42.998142	2015-06-21 16:49:42.998142
16	5	2015-06-21 16:49:43.058554	2015-06-21 16:49:43.058554
87	7	2015-06-21 16:49:43.323533	2015-06-21 16:49:43.323533
87	7	2015-06-21 16:49:43.755593	2015-06-21 16:49:43.755593
53	8	2015-06-21 16:49:44.675252	2015-06-21 16:49:44.675252
87	7	2015-06-21 16:49:45.003992	2015-06-21 16:49:45.003992
87	7	2015-06-21 16:49:45.061224	2015-06-21 16:49:45.061224
124	9	2015-06-21 16:49:45.127063	2015-06-21 16:49:45.127063
125	9	2015-06-21 16:49:45.186081	2015-06-21 16:49:45.186081
131	10	2015-06-21 16:49:47.307509	2015-06-21 16:49:47.307509
132	10	2015-06-21 16:49:47.354236	2015-06-21 16:49:47.354236
66	4	2015-06-21 16:49:47.798768	2015-06-21 16:49:47.798768
70	2	2015-06-21 16:49:47.831514	2015-06-21 16:49:47.831514
3	1	2015-06-21 16:49:48.08458	2015-06-21 16:49:48.08458
2	1	2015-06-21 16:49:48.150581	2015-06-21 16:49:48.150581
2	1	2015-06-21 16:49:48.185556	2015-06-21 16:49:48.185556
6	11	2015-06-21 16:49:48.236795	2015-06-21 16:49:48.236795
6	3	2015-06-21 16:49:48.240487	2015-06-21 16:49:48.240487
7	11	2015-06-21 16:49:48.274474	2015-06-21 16:49:48.274474
7	3	2015-06-21 16:49:48.277887	2015-06-21 16:49:48.277887
34	12	2015-06-21 16:49:48.339312	2015-06-21 16:49:48.339312
34	12	2015-06-21 16:49:48.418433	2015-06-21 16:49:48.418433
46	13	2015-06-21 16:49:48.511706	2015-06-21 16:49:48.511706
47	13	2015-06-21 16:49:48.550008	2015-06-21 16:49:48.550008
48	13	2015-06-21 16:49:48.588446	2015-06-21 16:49:48.588446
49	13	2015-06-21 16:49:48.638589	2015-06-21 16:49:48.638589
50	13	2015-06-21 16:49:48.676553	2015-06-21 16:49:48.676553
73	4	2015-06-21 16:49:48.788233	2015-06-21 16:49:48.788233
148	12	2015-06-21 16:49:48.873734	2015-06-21 16:49:48.873734
78	12	2015-06-21 16:49:48.912547	2015-06-21 16:49:48.912547
79	12	2015-06-21 16:49:48.95025	2015-06-21 16:49:48.95025
147	12	2015-06-21 16:49:48.992519	2015-06-21 16:49:48.992519
146	12	2015-06-21 16:49:49.0333	2015-06-21 16:49:49.0333
5	11	2015-06-21 16:49:49.271204	2015-06-21 16:49:49.271204
35	14	2015-06-21 16:49:49.907601	2015-06-21 16:49:49.907601
36	14	2015-06-21 16:49:49.949507	2015-06-21 16:49:49.949507
38	14	2015-06-21 16:49:50.021046	2015-06-21 16:49:50.021046
41	14	2015-06-21 16:49:50.11648	2015-06-21 16:49:50.11648
43	14	2015-06-21 16:49:50.186896	2015-06-21 16:49:50.186896
44	14	2015-06-21 16:49:50.230489	2015-06-21 16:49:50.230489
45	14	2015-06-21 16:49:50.264156	2015-06-21 16:49:50.264156
46	13	2015-06-21 16:49:50.303725	2015-06-21 16:49:50.303725
47	13	2015-06-21 16:49:50.339277	2015-06-21 16:49:50.339277
48	13	2015-06-21 16:49:50.380927	2015-06-21 16:49:50.380927
49	13	2015-06-21 16:49:50.421711	2015-06-21 16:49:50.421711
50	13	2015-06-21 16:49:50.461368	2015-06-21 16:49:50.461368
52	8	2015-06-21 16:49:50.501768	2015-06-21 16:49:50.501768
54	8	2015-06-21 16:49:50.541896	2015-06-21 16:49:50.541896
67	4	2015-06-21 16:49:50.753547	2015-06-21 16:49:50.753547
72	4	2015-06-21 16:49:50.796752	2015-06-21 16:49:50.796752
73	4	2015-06-21 16:49:50.838372	2015-06-21 16:49:50.838372
110	15	2015-06-21 16:49:51.324391	2015-06-21 16:49:51.324391
124	9	2015-06-21 16:49:51.432347	2015-06-21 16:49:51.432347
126	9	2015-06-21 16:49:51.466759	2015-06-21 16:49:51.466759
66	4	2015-06-21 16:49:51.826486	2015-06-21 16:49:51.826486
66	4	2015-06-21 16:49:51.865103	2015-06-21 16:49:51.865103
110	15	2015-06-21 16:49:52.170143	2015-06-21 16:49:52.170143
66	4	2015-06-21 16:49:53.239304	2015-06-21 16:49:53.239304
\.


--
-- Name: variant_groups_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_groups_id_seq', 15, true);


--
-- Name: variants_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variants_id_seq', 164, true);


--
-- PostgreSQL database dump complete
--

