--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

SET search_path = public, pg_catalog;

--
-- Data for Name: audits; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY audits (id, auditable_id, auditable_type, associated_id, associated_type, user_id, user_type, username, action, audited_changes, version, comment, remote_address, request_uuid, created_at) FROM stdin;
1	1	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 238\nname: ALK\ndescription: ALK amplifications, fusions and mutations have been shown to be driving\n  events in non-small cell lung cancer. While crizontinib has demonstrated efficacy\n  in treating the amplification, mutations in ALK have been shown to confer resistance\n  to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied\n  success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been\n  shown to be cytostatic in ALK-altered cell lines.\nofficial_name: anaplastic lymphoma receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	53e2f12e-aac1-4b1f-a62d-3025fbfa4b8b	2015-02-20 17:01:48.571658
2	2	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 207\nname: AKT1\ndescription: AKT1, also referred to as protein kinase B, is a known oncogene. AKT\n  activation relies on the PI3K pathway, and is recognized as a critical node in the\n  pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been\n  shown to result in activation of the protein. Mutations in AKT1 have also been shown\n  to confer resistance to allosteric kinase inhibitors in vitro.\nofficial_name: v-akt murine thymoma viral oncogene homolog 1\nclinical_description: \n	1	\N	\N	88ea7494-6a35-4daf-a3e0-898d9254722f	2015-02-20 17:01:49.108046
3	3	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 369\nname: ARAF\ndescription: ARAF has recently become increasingly considered for its oncogenic potential.\n  Its potential as a target for informing clinical action was demonstrated by a single\n  case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated\n  with sorafenib, acheived near-complete clinical remission. This finding has brought\n  new focus on ARAF as a marker that should be assayed for in cancer treatment.\nofficial_name: A-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	81434904-fc7a-4bbc-bfd4-5d8653d5fbdf	2015-02-20 17:01:49.31039
4	4	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 25\nname: ABL1\ndescription: ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein\n  that has become a signature of chronic myeloid leukemia (CML). Cells harboring this\n  fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook\n  of the disease. However, additional mutations in ABL1 have been shown to confer\n  resistance to imatinib. In these resistance cases, second-generation tyrosine kinase\n  inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are\n  currently undergoing clinical trials for treating acquired resistance in CML.\nofficial_name: ABL proto-oncogene 1, non-receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	1ee848ff-9517-4e9b-8f25-7a2b6e1bd0f6	2015-02-20 17:01:49.517454
5	5	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 673\nname: BRAF\ndescription: BRAF mutations are found to be recurrent in many cancer types. Of these,\n  the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E\n  has been determined to be an activating mutation, and cells that harbor it, along\n  with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is\n  also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK\n  inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations\n  have also been correlated with poor prognosis in many cancer types, although there\n  is at least one study that questions this conclusion in papillary thyroid cancer.\nofficial_name: B-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	e8c8466d-5bde-40fc-bb34-034b359548c6	2015-02-20 17:01:49.731617
6	6	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 595\nname: CCND1\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D1\nclinical_description: \n	1	\N	\N	c422a6ea-9a36-4554-b30c-7d2638593ba4	2015-02-20 17:01:49.948193
7	7	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 894\nname: CCND2\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D2\nclinical_description: \n	1	\N	\N	3d41f8b7-90ea-48b2-b48c-d178c2ef9828	2015-02-20 17:01:50.149064
8	8	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 896\nname: CCND3\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared\n  to the amplifcations and overexpressions of the other cyclin D's. In a mouse study,\n  the targeted therapeutic palbociclib significantly increased the median survival\n  of the cyclin D3 knockouts.\nofficial_name: cyclin D3\nclinical_description: \n	1	\N	\N	6717de3e-fa70-4836-a460-5842737e891d	2015-02-20 17:01:50.35025
9	9	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 898\nname: CCNE1\ndescription: Cyclin E, while currenly not as widely implicated as its cyclin D counterparts,\n  has been implicated in various carcinomas, including breast, gastric, stomach and\n  colorectal. High levels of cyclin E, either by gene amplification or overexpression,\n  have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements\n  are correlated with later stage disease. In lung cancer, neoplastic cells with higher\n  levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly\n  expressed counterparts.\nofficial_name: cyclin E1\nclinical_description: \n	1	\N	\N	05b0283a-3c7c-4b27-8393-bb28b3d7ed24	2015-02-20 17:01:50.547994
10	10	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1021\nname: CDK6\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 6\nclinical_description: \n	1	\N	\N	0690ed5a-f8d1-4dcd-b03a-63f6e2fd1687	2015-02-20 17:01:50.753486
11	11	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1019\nname: CDK4\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 4\nclinical_description: \n	1	\N	\N	26182f11-e34d-4f22-98c2-b64df909dba7	2015-02-20 17:01:50.956378
12	12	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1029\nname: CDKN2A\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. While no targeted therapeutic has been engaged in clinical trials, the prognostic\n  impact has been studied by a number of meta-analyses. One mechanism by which loss\n  of CDKN2A can occur is by hypermethylation of the promoter region for the gene.\n  However, the prognostic impact of promoter hypermethylation has been relatively\n  ambiguous. Many studies have suggesting poorer prognostic outcome for patients with\n  hypermethylation in colorectal, liver, and younger lung cancer patients. This being\n  said, there is still research to be done before this becomes a widely-accepted prognostic\n  indicator.\nofficial_name: cyclin-dependent kinase inhibitor 2A\nclinical_description: \n	1	\N	\N	4fb1a30d-6f51-4ceb-9819-e12a6a880adf	2015-02-20 17:01:51.172061
13	13	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1050\nname: CEBPA\ndescription: AML with mutated CEBPA' is a provisional entity in the WHO classification\n  of acute myeloid leukemia (AML) and is recommended to be tested for in patients\n  with AML. CEBPA mutations are particularly associated with cytogenetically normal\n  AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation\n  in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha\n  protein while C-terminal mutations reduce the DNA-binding potential of this transcription\n  factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1\n  and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations\n  may have prognostic implications.\nofficial_name: CCAAT/enhancer binding protein (C/EBP), alpha\nclinical_description: \n	1	\N	\N	536f6b0e-2ae6-41b6-b43f-7fa1d26c0ad0	2015-02-20 17:01:51.385654
14	14	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3337\nname: DNAJB1\ndescription: DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating\n  p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1\n  to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In\n  one study, this fusion was observed in 15/15 FL-HCC cases examined and functional\n  studies found that the fusion retained kinase activity.\nofficial_name: DnaJ (Hsp40) homolog, subfamily B, member 1\nclinical_description: \n	1	\N	\N	7cd1468c-97d9-4844-8f52-d075e612e7bd	2015-02-20 17:01:51.588966
15	15	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5566\nname: PRKACA\ndescription: '"PRKACA has been studied in breast cancer and has been found to mediate\n  resistance to HER2 targeted therapies. It has also been found to contain a mutation\n  hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular\n  fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be\n  a diagnostic marker for this rare subtype of HCC. In one study, this fusion was\n  observed in 15/15 FL-HCC cases examined and functional studies found that the fusion\n  retained kinase activity."'\nofficial_name: protein kinase, cAMP-dependent, catalytic, alpha\nclinical_description: \n	1	\N	\N	5f1937a3-71ec-438b-984f-ee15a1e30781	2015-02-20 17:01:51.81042
16	16	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1788\nname: DNMT3A\ndescription: DNMT3A is one of several epigenetic modifiers identified as recurrently\n  mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically\n  normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant\n  negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers.\n  AML patient bone marrow harboring R882 mutations were similarly demonstrated to\n  be hypomethylated compared to patients with wildtype DNMT3A. These studies also\n  indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner\n  from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes\n  and treatment protocols may need to be considered for these two classes of DNMT3A\n  mutations.\nofficial_name: DNA (cytosine-5-)-methyltransferase 3 alpha\nclinical_description: \n	1	\N	\N	699233c6-8771-4009-ad05-7f114eb574df	2015-02-20 17:01:52.023801
17	17	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1956\nname: EGFR\ndescription: EGFR is widely recognized for its importance in cancer. Amplification\n  and mutations have been shown to be driving events in many cancer types. Its role\n  in non-small cell lung cancer has spurred many research and drug development efforts.\n  Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably\n  gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance\n  to these drugs, particularly the variant T790M, which has been functionally characterized\n  as a resistance marker for both of these drugs. The later generation TKI's have\n  seen some success in treating these resistant cases, and targeted sequencing of\n  the EGFR locus has become a common practice in non-small cell lung cancer.\nofficial_name: epidermal growth factor receptor\nclinical_description: \n	1	\N	\N	90e7765b-0fe8-4c64-99d3-bf461e63e78d	2015-02-20 17:01:52.254964
18	18	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2064\nname: ERBB2\ndescription: ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast\n  cancer, and is treated in a separate manner than the other subtypes of breast cancer.\n  Apart from being amplified, ERBB2 activating mutations have been shown to have clinical\n  importance in HER2-negative breast cancer. These mutations have shown senitivity\n  to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical\n  sequencing efforts in treating breast cancer.\nofficial_name: erb-b2 receptor tyrosine kinase 2\nclinical_description: \n	1	\N	\N	d67c3377-235e-4ccf-8b6c-a6407caa03d1	2015-02-20 17:01:52.474125
19	19	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2099\nname: ESR1\ndescription: ESR1 has been a focus in breast cancer for quite some time, but has also\n  shown significance in endometrial, ovarian and other cancer types. ER-positive breast\n  cancer that is resistant to hormone therapy has instigated clinical sequencing efforts\n  to shed light on the mechanisms of this resistance. A number of mutations in the\n  ligand binding domain of the protein have been implicated in hormone resistance\n  and anti-estrogen therapies. This has spurred efforts to develop therapeutics that\n  act to degrade the protein, rather than act as an antagonist. These agents are currently\n  in clinical trials and have seen some success, highlighting the importance of sequencing\n  efforts in treating breast cancer.\nofficial_name: estrogen receptor 1\nclinical_description: \n	1	\N	\N	0d5897dd-0f56-4aac-ab61-d5c041003a0f	2015-02-20 17:01:52.686888
158	111	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and arsenic trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	67efc888-7762-4751-a8aa-83148470b989	2015-02-20 17:02:01.890671
20	20	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2263\nname: FGFR2\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions have been demonstrated\n  to have oncogenic potential across multiple cancer types. The targeted therapeutics\n  ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR\n  signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially\n  in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 2\nclinical_description: \n	1	\N	\N	038d9b6b-7d15-4439-b454-b65a540b6f1b	2015-02-20 17:01:52.905477
21	21	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2261\nname: FGFR3\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to the activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFRs can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions have been demonstrated\n  to have oncogenic potential across multiple cancer types. The targeted therapeutics\n  ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR\n  signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially\n  in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 3\nclinical_description: \n	1	\N	\N	dddba24b-bf1e-4a3b-ba64-0f952765ded8	2015-02-20 17:01:53.116489
22	22	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2322\nname: FLT3\ndescription: FLT3 is an important cytokine receptor involved in normal hematopoiesis.\n  Mutations in this gene are common in acute myeloid leukemia (AML) and screening\n  for mutations in this gene has been recommended by the World Health Organization\n  in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML).\n  FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated\n  with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have\n  been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD\n  mutations are still up for debate.\nofficial_name: fms-related tyrosine kinase 3\nclinical_description: \n	1	\N	\N	32256bb8-2872-4477-8f41-ffbd8252d8b0	2015-02-20 17:01:53.389103
23	23	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2624\nname: GATA2\ndescription: N/A\nofficial_name: GATA binding protein 2\nclinical_description: \n	1	\N	\N	9333e637-f8ea-4bde-93e8-3cb251230edb	2015-02-20 17:01:53.582536
24	24	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3417\nname: IDH1\ndescription: IDH1 mutations have been observed in a number of cancer types, and appear\n  to behave quite differently in different environments. In myelodysplastic syndromes\n  and acute myeloid leukemia (AML), IDH1 mutations have been shown to be associated\n  with worse outcome and shorter overall survival. They have also been linked to cytogenetically\n  normal AML, an intermediate risk subtype of the disease. However, in glioblastoma\n  and astrocytoma, patients with IDH1 mutations have shown better overall survival\n  than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1\n  mutations have been associated with specific cytogenetic abnormalities, 1p and 19q\n  deletions.\nofficial_name: isocitrate dehydrogenase 1 (NADP+), soluble\nclinical_description: \n	1	\N	\N	3c86273c-7b91-4f43-bcd6-cac16cbe6779	2015-02-20 17:01:53.771523
25	25	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3418\nname: IDH2\ndescription: N/A\nofficial_name: isocitrate dehydrogenase 2 (NADP+), mitochondrial\nclinical_description: \n	1	\N	\N	dd6cbb47-0680-470d-a001-5b5a149fafa6	2015-02-20 17:01:53.962347
26	26	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3717\nname: JAK2\ndescription: JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative\n  diseases and cancers. The mutation V617F is the most clinically relevant variant,\n  and is seen in around half of myeloproliferative disorders.\nofficial_name: Janus kinase 2\nclinical_description: \n	1	\N	\N	93a80424-90a0-4ad0-ae2e-2efe2f66950d	2015-02-20 17:01:54.167501
27	27	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3815\nname: KIT\ndescription: N/A\nofficial_name: v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	eb85b138-395e-4fdb-ab4b-aeb3a8f26f54	2015-02-20 17:01:54.355676
28	28	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3845\nname: KRAS\ndescription: N/A\nofficial_name: Kirsten rat sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	287e69e3-7bd3-4ea0-909f-4b791ecb33c0	2015-02-20 17:01:54.556992
29	29	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5604\nname: MAP2K1\ndescription: N/A\nofficial_name: mitogen-activated protein kinase kinase 1\nclinical_description: \n	1	\N	\N	547cd732-e11b-4fc5-a87a-e76f6b7def52	2015-02-20 17:01:54.746828
30	30	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4209\nname: MEF2D\ndescription: N/A\nofficial_name: myocyte enhancer factor 2D\nclinical_description: \n	1	\N	\N	cf5c6b81-3991-4321-8bce-97ca2d3101c7	2015-02-20 17:01:54.964284
31	31	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1436\nname: CSF1R\ndescription: N/A\nofficial_name: colony stimulating factor 1 receptor\nclinical_description: \n	1	\N	\N	808cf9a6-b853-4bb2-b4cc-18ce1ba98d05	2015-02-20 17:01:55.155792
32	32	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4255\nname: MGMT\ndescription: N/A\nofficial_name: O-6-methylguanine-DNA methyltransferase\nclinical_description: \n	1	\N	\N	a7f36271-ed31-4d06-a2e3-f41f982a9a5f	2015-02-20 17:01:55.402111
33	33	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4869\nname: NPM1\ndescription: AML with mutated NPM1' is provisional entity in the WHO classification\n  of AML and is recommended to be tested for in patients with cytogenetically normal\n  AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they\n  have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently\n  W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations\n  in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing\n  the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia,\n  indicating it may require additional mutations to promote leukemic development.\nofficial_name: nucleophosmin (nucleolar phosphoprotein B23, numatrin)\nclinical_description: \n	1	\N	\N	937955c1-4856-4d82-a18e-00e6f96b7eee	2015-02-20 17:01:55.623826
34	34	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4893\nname: NRAS\ndescription: N/A\nofficial_name: neuroblastoma RAS viral (v-ras) oncogene homolog\nclinical_description: \n	1	\N	\N	a2c77eb7-51d7-4c32-b9a4-d95f198ad641	2015-02-20 17:01:55.82244
50	3	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL E255K\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation E255K has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	99d29ac7-df5a-48bf-8c5b-8733864fbdd2	2015-02-20 17:01:58.66514
35	35	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5290\nname: PIK3CA\ndescription: PIK3CA is the most recurrently mutated gene in breast cancer, and has\n  been found to important in a number of cancer types. An integral part of the PI3K\n  pathway, PIK3CA has long been described as an oncogene, with two main hotspots for\n  activating mutations, the 542/545 region of the helical domain, and the 1047 region\n  of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways,\n  is the subject of an immense amount of research and development, and PI3K inhibition\n  has seen some limited success in recent clinical trials. While monotherapies seem\n  to be limited in their potential, there is a recent interest in pursuing PI3K inhibition\n  as part of a combination therapy regiment with inhibition partners including TKI's,\n  MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.\nofficial_name: phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha\nclinical_description: \n	1	\N	\N	76d8a2da-a726-40b7-a9a6-a2077bc6918a	2015-02-20 17:01:56.019647
36	36	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5156\nname: PDGFRA\ndescription: N/A\nofficial_name: platelet-derived growth factor receptor, alpha polypeptide\nclinical_description: \n	1	\N	\N	0c6bbcd7-2ed6-4cb0-b3e5-afa988cb9341	2015-02-20 17:01:56.217414
37	37	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5371\nname: PML\ndescription: N/A\nofficial_name: promyelocytic leukemia\nclinical_description: \n	1	\N	\N	169d2384-a7a3-4969-be22-f050ebf48335	2015-02-20 17:01:56.410819
38	38	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5914\nname: RARa\ndescription: N/A\nofficial_name: retinoic acid receptor, alpha\nclinical_description: \n	1	\N	\N	26ce7ef5-a03e-4e56-b029-cd6fce7f4e14	2015-02-20 17:01:56.620541
39	39	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5728\nname: PTEN\ndescription: N/A\nofficial_name: phosphatase and tensin homolog\nclinical_description: \n	1	\N	\N	7dcad2cd-4df1-49b5-8854-937c639f7a0d	2015-02-20 17:01:56.813332
40	40	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5979\nname: RET\ndescription: RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine\n  kinase receptor and are associated with thyroid cancers. RET point mutations are\n  the most common mutations identified in medullary thyroid cancer (MTC) with germline\n  and somatic mutations in RET associated with hereditary and sporadic forms, respectively.\n  The most common somatic form mutation is M918T (exon 16) and a variety of other\n  mutations effecting exons 10, 11 and 15 have been described. The prognostic significance\n  of these mutations have been hotly debated in the field, however, data suggests\n  that some RET mutation may confer drug resistence. No RET-specific agents are currently\n  clinically available but several promiscuous kinase inhibitors that target RET,\n  among others, have been approved for MTC treatment.\nofficial_name: ret proto-oncogene\nclinical_description: \n	1	\N	\N	df4835f6-72ed-4a62-a764-bbb9a1bf1588	2015-02-20 17:01:57.021365
41	41	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 861\nname: RUNX1\ndescription: N/A\nofficial_name: runt-related transcription factor 1\nclinical_description: \n	1	\N	\N	0538a319-ef78-4488-a9dd-060f94df8728	2015-02-20 17:01:57.212238
42	42	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 23451\nname: SF3B1\ndescription: SF3B1 mutations have been described in several myeloid malignancies,\n  predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies\n  and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has\n  been identified as recurrently mutated in MDS and other malignanices. The mutations\n  affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally\n  deleterious with K700E described as a major hotspot mutation. MDS patients with\n  SF3B1 mutations have been reported to have better overall and event-free survival\n  than their wildtype counterparts. Additionally, these mutations are highly associated\n  with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with\n  ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring\n  sideroblasts).\nofficial_name: splicing factor 3b, subunit 1, 155kDa\nclinical_description: \n	1	\N	\N	d8136266-1123-4350-8e6f-f8ced4ecf5c8	2015-02-20 17:01:57.49036
43	43	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7157\nname: TP53\ndescription: N/A\nofficial_name: tumor protein p53\nclinical_description: \n	1	\N	\N	72802b02-93f5-40e0-bc6c-443b64399a90	2015-02-20 17:01:57.734853
44	44	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7248\nname: TSC1\ndescription: N/A\nofficial_name: tuberous sclerosis 1\nclinical_description: \n	1	\N	\N	ea27cfcd-839f-4a91-9ce9-064dec6d0a11	2015-02-20 17:01:57.922761
45	45	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7249\nname: TSC2\ndescription: N/A\nofficial_name: tuberous sclerosis 2\nclinical_description: \n	1	\N	\N	8a88bde7-b5b4-4a55-9981-ab98a1e00eaf	2015-02-20 17:01:58.117992
46	46	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7307\nname: U2AF1\ndescription: U2AF1 is one of several spliceosome complex genes frequently mutated\n  in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes\n  (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot\n  mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein.\n  These mutations have been associated with altered splicing patterns in vitro and\n  in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation\n  to secondary acute myeloid leukemia, however, the impact of these mutations on overall\n  survival has been an area of debate.\nofficial_name: U2 small nuclear RNA auxiliary factor 1\nclinical_description: \n	1	\N	\N	541d90ef-945c-4de5-a0a8-a476c90868ae	2015-02-20 17:01:58.34894
47	47	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7490\nname: WT1\ndescription: N/A\nofficial_name: Wilms tumor 1\nclinical_description: \n	1	\N	\N	f76c0ff3-fcc9-481d-998c-07c7ca219f04	2015-02-20 17:01:58.540278
48	1	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL\ndescription: The BCR-ABL fusion protein, commonly referred to as the philadelphia\n  chromosome, is the most well-studied fusion gene in cancer. It has widely been considered\n  the initiating event in chronic myelogenous leukemia (CML), but despite its ability\n  initiate disease in mice, its status an initiating mutation is in dispute. In what\n  is commonly used as the poster-child for targeted therapeutics, the development\n  and use of imatinib in the clinic has led to profound improvements in the prognosis\n  of the disease. However, imatinib resistance is still seen in patients with mutations\n  in the ABL kinase domain of the fusion, most notably the T315I variant. In patients\n  resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's\n  (dasatinib and nilotinib) have seen some success in delivering a tumor response.\n	1	\N	\N	f58ed16f-7fcb-4825-b307-be0b3e4c5761	2015-02-20 17:01:58.60257
49	2	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL T315I\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation T315I has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	a1ecc605-6d23-4f30-8e26-aaec85f4e680	2015-02-20 17:01:58.636099
51	4	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 2\nname: E17K\ndescription: AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal,\n  lung, and ovarian cancer. It has been convincingly shown to be an activating mutation\n  resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation\n  decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and\n  other AKT1 mutations, are the subject of much research and development for therapeutics.\n	1	\N	\N	b76ec2a8-d049-46f9-8d3f-adcd2aa6bbe9	2015-02-20 17:01:58.691222
52	5	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK\ndescription: The EML4-ALK fusion has been seen in non-small cell lung cancer, and\n  appears to be an alternative mechanism for ALK activation. Cells with this fusion\n  have been shown to be sensitive to the ALK inhibitor crizotinib.\n	1	\N	\N	df81bc33-cf2d-480a-874d-35697111ade7	2015-02-20 17:01:58.739552
53	6	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: C1156Y\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	1b5b74d1-4deb-452d-b7b5-14c08e290a5e	2015-02-20 17:01:58.768915
54	7	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: L1196M\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	8a1e8966-b2ac-42c5-925e-1cb23b59db2c	2015-02-20 17:01:58.794249
55	8	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: F1174L\ndescription: ALK F1174L has been observed as recurrent in neuroblastoma, non-small\n  cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing\n  this mutation have shown resistance to low doses of criztonib. However, increased\n  dosage can overcome this resistance in cell lines studies. TAE684 has also proven\n  effective in both NSCLC and neuroblastoma F1174L containing cells.\n	1	\N	\N	995507c4-b42e-4104-83d7-00ec25981f0d	2015-02-20 17:01:58.817752
56	9	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: R1275Q\ndescription: ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma,\n  non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma\n  cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This,\n  and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC\n  cell lines.\n	1	\N	\N	23450824-90fb-4a86-bf4f-3f4291113d29	2015-02-20 17:01:58.841407
57	10	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 3\nname: S214C\ndescription: ARAF S214C has been found to be a recurrent oncogenic mutation in non-small\n  cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib\n  in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this\n  mutation, sorafenib also acheived near-complete clinical remission. This case has\n  brought more interest to the variant from a research and clinical perspective.\n	1	\N	\N	87c3dae0-4dcc-4491-a046-be36f5e7b0ea	2015-02-20 17:01:58.868884
58	11	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600D\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	e358b0a5-a863-41c2-91f3-1ba18d578fef	2015-02-20 17:01:58.906358
59	12	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E\ndescription: BRAF V600E has been shown to be recurrent in many cancer types. It is\n  one of the most widely studied variants in cancer. This variant is correlated with\n  poor prognosis in certain cancer types, including colorectal cancer and papillary\n  thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective\n  in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib\n  has also shown to be effective when combined with the MEK inhibitor trametinib in\n  colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA\n  mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib,\n  and a number of combination therapies have been successful in treating V600E mutations.\n  While the drugs cetuximab and panitumumab have been largely shown to be ineffective\n  without supplementary treatment.\n	1	\N	\N	cee60e82-7b6c-4020-868b-3cc1f0b1a709	2015-02-20 17:01:58.936126
60	13	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E/V600M\ndescription: A case study of a single patient harboring both a V600E and a V600M mutation,\n  dabrafenib was shown to acheive clinical response.\n	1	\N	\N	97e13c51-0811-449c-bb24-1f3e09555a65	2015-02-20 17:01:58.964387
61	14	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E AMPLIFICATION\ndescription: Amplification of BRAF V600E has been shown to confer resistance to MEK\n  inhibitors. For more information on the V600 locus, see the V600E entry.\n	1	\N	\N	946b1acd-4b5d-4e27-8d27-c5dcde1095df	2015-02-20 17:01:58.991921
62	15	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600M\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	140fbecf-d57c-4f17-80ab-5a31c0cc181e	2015-02-20 17:01:59.017439
63	16	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600R\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	16eced79-2579-4788-a6ca-a9d2db6e3932	2015-02-20 17:01:59.048021
64	17	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600\ndescription: BRAF mutations of the valine 600 residue have been shown to be recurrent\n  in many cancer types. Of the V600 mutations, V600E is the most widely researched.\n  V600 mutations as a whole have been correlated to poorer prognosis in colorectal\n  and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity\n  to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual\n  mutation pages on the left sidebar.\n	1	\N	\N	777e1f54-967b-4ab3-a2cb-6a7a3660f54f	2015-02-20 17:01:59.076184
65	18	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: AMPLIFICATION\ndescription: CCND1 amplification has been implicated in poorer prognosis in non-small\n  cell lung cancer.\n	1	\N	\N	528504e3-2ef3-4624-9c2c-c644ca060e23	2015-02-20 17:01:59.10847
66	19	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: EXPRESSION\ndescription: CCND1 expression, and its prognositc impact, is still in dispute. Three\n  experiments in non-small cell lung cancer have shown it to have no impact on survival,\n  but three additional studies have shown it results in poorer prognosis. There is\n  also some ambiguity in how the boundaries between expression and overexpression\n  are defined.\n	1	\N	\N	244a5644-21be-49c4-ae00-5bc383842c9d	2015-02-20 17:01:59.132858
67	20	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	ac447154-2026-4bcb-a7a3-d0f5715d6769	2015-02-20 17:01:59.15865
68	21	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	a43fdd5c-0942-4c26-9c71-d6f635516726	2015-02-20 17:01:59.185935
69	22	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: PROMOTER DEMETHYLATION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	265bc9f5-7c9a-46c9-bf6f-a1542998a41d	2015-02-20 17:01:59.239405
70	23	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: LOSS\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL),\n  a seemingly unique trend when compared to the amplifcations and overexpressions\n  of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic\n  palbociclib significantly increased the median survival of a Notch-driven model\n  of T-ALL.\n	1	\N	\N	b5ed8619-20b0-448a-a286-7a784d22b7cf	2015-02-20 17:01:59.27646
71	24	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: OVEREXPRESSION\ndescription: Cyclin E, while currenly not as widely implicated as its counterpart,\n  cyclin D, has been implicated in various carcinomas, including breast, gastric,\n  stomach and colorectal. High levels of cyclin E, either by gene amplification or\n  overexpression, are correlated with later stage disease and have been shown to lead\n  to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer\n  specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex\n  are more radiosensitive than their lowly expressed counterparts.\n	1	\N	\N	ed74c16c-31fa-4f5a-bf7a-5a4d9dbaab39	2015-02-20 17:01:59.31549
72	25	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 11\nname: EXPRESSION\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	2a13a2ef-5563-467b-ab55-2fb5b90a03bc	2015-02-20 17:01:59.349788
73	26	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 10\nname: EXPRESSION\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	618c8e8e-a528-4fd6-98b8-ee8011404f7f	2015-02-20 17:01:59.380195
74	27	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 12\nname: PROMOTER HYPERMETHYLATION\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the CDKN2A\n  promoter region. While no targeted therapeutic has been engaged in clinical trials,\n  the prognostic impact has been studied by a number of meta-analyses. The prognostic\n  impact of promoter hypermethylation has been relatively ambiguous. Many studies\n  have shown significant p-values suggesting poorer prognostic outcomes for patients\n  with hypermethylation in colorectal, liver, and younger lung cancer patients. This\n  being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\n	1	\N	\N	c45234b5-5630-489d-ba39-df268047e07c	2015-02-20 17:01:59.405608
75	28	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: N-TERMINAL FRAME SHIFT\ndescription: CEBPA N-terminal frame shift mutations that result in a premature stop\n  codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML).\n  CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3\n  mutations should also be assessed in CN-AML patients as concurrent mutations may\n  have prognostic implications.\n	1	\N	\N	e667411d-631d-4095-9553-c3eb8ea7717f	2015-02-20 17:01:59.433219
76	29	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: MUTATION\ndescription: CEBPA mutations are associated with cytogenetically normal acute myeloid\n  leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should\n  also be assessed in CN-AML patients as concurrent mutations may have prognostic\n  implications.\n	1	\N	\N	5d5c32f7-aad7-425e-bbf7-cfbc8db041d6	2015-02-20 17:01:59.46255
77	30	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	3bf049d1-acea-4860-b581-3765fdb93a4b	2015-02-20 17:01:59.507478
78	31	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 14\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in the rare form of adolescent\n  liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion\n  was observed in 15/15 FL-HCC cases examined and functional studies found that the\n  fusion retained kinase activity. The presence of this fusion may be used as a diagnostic\n  marker for this rare tumor type.\n	1	\N	\N	8a3a7d0f-9cd9-415f-923e-4ffba44db0e2	2015-02-20 17:01:59.536225
89	42	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R678Q\ndescription: ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	d5562831-b1a0-4a79-b607-abf11fdcd3eb	2015-02-20 17:01:59.840522
79	32	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: R882\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\n	1	\N	\N	4e3a22e5-3872-4042-b427-a6cae789375a	2015-02-20 17:01:59.565835
80	33	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: L858R\ndescription: EGFR L858R has long been recognized as a functionally significant mutation\n  in cancer, and is one of the most prevalent single mutations in lung cancer. Best\n  described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity\n  to first and second generation TKI's like gefitinib and neratinib. NSCLC patients\n  with this mutation treated with TKI's show increased overall and progression-free\n  survival, as compared to chemotherapy alone. Third generation TKI's are currently\n  in clinical trials that specifically focus on mutant forms of EGFR, a few of which\n  have shown efficacy in treating patients that failed to respond to earlier generation\n  TKI therapies.\n	1	\N	\N	29722b36-9f75-4718-befc-f631a4af81ef	2015-02-20 17:01:59.592184
81	34	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: T790M\ndescription: EGFR T790M was one of the very first mutations recognized to confer resistance\n  to targeted therapies. While successful in amplified EGFR, the efficacy of the first\n  and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients\n  harboring this mutation before treatment is notably lower. This lack of efficacy\n  can likely be to blame for the poorer prognosis for patients with this mutation\n  as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately\n  half of EGFR mutant tumors with acquired resistance to TKI inhibition have been\n  shown to harbor this mutation, implicating it as a mechanism of acquired therapy\n  resistence. The third generation TKI's are being developed with this resistance\n  problem in mind, and early stage experiments have shown some efficacy of these drugs\n  in previously resistant tumors with mutant EGFR.\n	1	\N	\N	075c918c-6531-4f77-9ba0-2d0f9426bfba	2015-02-20 17:01:59.620736
82	35	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: D769H\ndescription: ERBB2 D769H was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	3ac2ff97-1cab-4050-8be9-85811c08c374	2015-02-20 17:01:59.654122
83	36	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: D769Y\ndescription: ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	faa983d1-1866-4e12-b8b3-a16cd9c4906b	2015-02-20 17:01:59.679521
84	37	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: DEL 755-759\ndescription: ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants\n  to be functionally classified (Bose et al. 2012). This mutation was shown to be\n  an activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	65a19abf-dfc8-4265-b8b8-79f583c402b4	2015-02-20 17:01:59.710216
85	38	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: G309A\ndescription: ERBB2 G309A was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	25233642-dc22-4efd-86cf-7093b083da5d	2015-02-20 17:01:59.739429
86	39	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: L755S\ndescription: ERBB2 L755S was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike\n  many of the other variants queried. This mutation was also shown to confer resistance\n  to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.\n	1	\N	\N	bb0dcc0e-f7ad-447d-96de-2a4e046db8ea	2015-02-20 17:01:59.765891
87	40	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: L755W\ndescription: ERBB2 L755W was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	2982e4cf-9fc1-44a6-b6c1-e8b0f2fcd4ea	2015-02-20 17:01:59.791835
88	41	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: P780INS\ndescription: ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally\n  classified (Bose et al. 2012). This mutation was shown to be an activating mutation\n  in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	1d16971e-26a8-4eaa-bd4b-932b34a55370	2015-02-20 17:01:59.815916
141	94	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 3\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	e7719582-5e7a-4abe-a4c2-237f788ffd8a	2015-02-20 17:02:01.411964
142	95	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 4\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	df726df4-e70c-43d2-ab6c-7fd89b773327	2015-02-20 17:02:01.435878
90	43	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R896C\ndescription: ERBB2 R896C was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	ed516b5c-270c-4660-b59f-9974ff0bbfb2	2015-02-20 17:01:59.875155
91	44	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: V777L\ndescription: ERBB2 V777L was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	27e8977b-b9c2-4a96-bc68-5df081034248	2015-02-20 17:01:59.90035
92	45	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: V842I\ndescription: ERBB2 V842I was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	3f2632d7-04be-4fa5-b4da-2178929e9f93	2015-02-20 17:01:59.9288
93	46	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: L536Q\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of\n  these ligand binding domain mutations, and is commonly implicated in this hormone\n  resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant,\n  may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	9fd00571-9039-4c28-858b-da3cdf15cd26	2015-02-20 17:01:59.957826
94	47	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: N538G\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	8be93008-835d-44a1-b74b-4ac4361e42dd	2015-02-20 17:01:59.983568
95	48	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537C\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	8ae83cc8-1735-4382-9e3d-6f8ab8aa0c7b	2015-02-20 17:02:00.019237
96	49	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537N\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	cae98edc-563c-45a4-94c2-33f1058740f4	2015-02-20 17:02:00.046563
97	50	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537S\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	b7846d12-8c90-4529-ae8a-d12597c10de9	2015-02-20 17:02:00.075039
98	51	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-AFF3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	c347c2ee-6db7-4533-9556-e037f2e2be4f	2015-02-20 17:02:00.102843
99	52	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-BICC1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	803dce24-5b03-4afd-968f-7e6f00b39a19	2015-02-20 17:02:00.132182
100	53	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-CASP7\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	74770bd9-19cf-4a57-9217-f54e75378e6d	2015-02-20 17:02:00.161314
143	96	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: G12\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	70b5fc8b-0924-4e0a-a2cb-6b45591709cb	2015-02-20 17:02:01.465046
101	54	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-CCDC6\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	8455aacf-a44c-400f-8766-b006aa7a8bed	2015-02-20 17:02:00.208038
102	55	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-KIAA1967\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	c31f1842-c2cc-459e-89d0-c0c688620063	2015-02-20 17:02:00.233817
103	56	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-MGEA5\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. Additionally, tumor size reduction was\n  achieved by both ponatinib and pazopanib treatments administered separately in a\n  single patient with intrahepatic cholangiocarcinoma and this fusion. The authors\n  use these cases to highlight the need for enhanced clinical sequencing efforts.\n	1	\N	\N	fb29824a-104b-4ab3-ab5b-d005edbba604	2015-02-20 17:02:00.266434
104	57	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-OFD1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	3d89cff5-3805-4636-96e0-2a63882ab522	2015-02-20 17:02:00.295995
105	58	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-SLC45A3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	7c775d1a-7a44-4f02-9a4d-69099dc72f4f	2015-02-20 17:02:00.319538
106	59	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: FGFR3-TACC3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\n	1	\N	\N	a2b0b54d-369e-4a40-a7a2-94006d4827ca	2015-02-20 17:02:00.351456
107	60	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: ITD\ndescription: FLT3-ITD (internal tandem duplications) frequently occur in patients\n  with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid\n  leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically\n  normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of\n  FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated\n  sequence resulting in an in-frame duplication event. The length of these duplications\n  can vary widely which may have prognostic consequences, but this has not been conclusively\n  determined. FLT3-ITD mutations overall have generally been associated with poor\n  prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the\n  prognosis associated with this variant.\n	1	\N	\N	609ff436-0deb-47ac-a153-cdec53231eba	2015-02-20 17:02:00.381358
108	61	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: TKD\ndescription: FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common\n  than FLT3-ITD (internal tandem duplication) mutations and may not confer the same\n  prognostic impact. Although the majority of mutations are point mutations effecting\n  D835 (most frequently D835Y), a small proportion involve an in-frame deletion of\n  I836. These mutations are within the activation loop of the second tyrosine kinase\n  domain of FLT3 and thought to result in constitutive activation of the receptor.\n	1	\N	\N	6c36eba7-4b84-4979-b6ee-adcdedf38a08	2015-02-20 17:02:00.413016
109	62	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: EXPRESSION\ndescription: GATA2 misregulation has been observed in a number of hematologic malignancies,\n  as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using\n  the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung\n  cancer.\n	1	\N	\N	0dd1f85c-a17f-4b0d-a67e-4be8417ea846	2015-02-20 17:02:00.445902
110	63	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	0df9a0b0-cf45-4717-8865-385aafffde52	2015-02-20 17:02:00.474363
111	64	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132C\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	6adcf6e1-009b-4568-b95c-ec98d573bf7f	2015-02-20 17:02:00.510623
144	97	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: G13D\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	1860f823-d67b-49b4-bd7f-f33a846f33f7	2015-02-20 17:02:01.489498
112	65	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132H\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	d5c458bf-aa1e-4033-ab62-ff8ee8b2a269	2015-02-20 17:02:00.544561
113	66	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132L\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	a62d619f-915e-4b49-a308-15e57646e858	2015-02-20 17:02:00.582442
114	67	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: R140Q/L\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival\n  as compared to their wild-type counterparts. In myelodysplastic syndromes, however,\n  no prognostic link was discovered between IDH2 mutation status and overall survival.\n	1	\N	\N	0192ddf6-6e2c-4239-b0b8-1c2a4ee57944	2015-02-20 17:02:00.620549
115	68	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: R172K\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis\n  and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic\n  syndromes, studies did not find a prognostic association between this variant and\n  patient outcomes.\n	1	\N	\N	d1ca07ab-1061-4a2f-b4cb-bd569215ea6f	2015-02-20 17:02:00.656824
116	69	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: V617F\ndescription: JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases,\n  occuring in around half of all MPD's. While less associated with cancer, when it\n  is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The\n  V617F mutation is an activating mutation, resulting in increased kinase activity.\n  The mutation seems to be restricted to hematologic malignancies. Treatment of JAK\n  mutant diseases with ruxolitinib has seen some clinical success.\n	1	\N	\N	66622934-e1d7-45a9-af29-3d2799947a4d	2015-02-20 17:02:00.68845
117	70	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: D816V\ndescription: KIT D816V is a mutation observed in acute myeloid leukemia (AML). This\n  variant has been linked to poorer prognosis and worse outcome in AML patients.\n	1	\N	\N	817b4238-76f1-4f61-b92d-3b2ba04c4ee8	2015-02-20 17:02:00.720628
118	71	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 11\ndescription: c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are\n  very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis\n  than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity\n  to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients\n  treated with the drug in the first year.\n	1	\N	\N	28653816-c007-4d31-af6c-75cc4a988207	2015-02-20 17:02:00.748484
119	72	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 13\ndescription: c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations.\n  These mutations lie within the tyrosine kinase 1 domain, and are found primarily\n  in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations\n  show sensitivity to imatinib and sunitinib treatment. However, only imatinib has\n  seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant\n  cases.\n	1	\N	\N	84c8ec3f-6673-4214-8fb0-aceac9c1794d	2015-02-20 17:02:00.776044
120	73	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 14\ndescription: c-KIT exon 14 mutations lie within the tyrosine kinase domain of the\n  protein. While relatively rare in primary gastrointestinal tumors, they are notably\n  more prevalent in refractory disease, suggesting a role in imatinib resistance.\n  Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.\n	1	\N	\N	eb34077d-95cc-4d11-92f6-0953294f2fb2	2015-02-20 17:02:00.801518
121	74	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 17\ndescription: c-KIT exon 17 mutations lie within the TK2 domain, containing the activation\n  loop of the protein. In cell lines, mutations within this domain have been shown\n  to be sensitive to imatinib. However, in double KIT mutants in which the exon 17\n  mutation is a secondary mutation, cell lines have shown resistance to both imatinib\n  and sunitinib.\n	1	\N	\N	6cdc685e-67b2-4695-8609-d5e1b74e5146	2015-02-20 17:02:00.834032
122	75	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 9\ndescription: c-KIT exon 9 mutations lie within the dimerization motif of the protein.\n  Relative to other KIT mutations, exon 9 mutations have been associated with better\n  overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and\n  in vivo.\n	1	\N	\N	a8ab442e-3a0e-4558-ac69-6894eb8fbe16	2015-02-20 17:02:00.864484
123	76	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: L576P\ndescription: KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain.\n  It is one of the most recurrent KIT mutations in melanoma, and both in vitro and\n  in vivo studies have shown sensitivity to imatinib.\n	1	\N	\N	7e25e197-7ec7-4dce-862d-b43b3f503999	2015-02-20 17:02:00.891732
124	77	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: V654A\ndescription: KIT V654A is an exon 13 mutation that lies within the tyrosine kinase\n  1 domain of the protein. It has been shown to be an activating mutation by in vitro\n  studies. This mutation is associated with imatinib resistance in melanoma patients.\n  However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have\n  seen success in acheiving tumor response.\n	1	\N	\N	de6a62a3-74f8-472c-ab65-97373a2e0dd7	2015-02-20 17:02:00.91971
125	78	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: EXON 1\ndescription: A study by Lièvre et al in 2006 showed that colorectal cancer patients\n  with KRAS exon 1 mutations had low cetuximab response rates.\n	1	\N	\N	10a7e777-2cc2-4d00-ab7a-c5b04d8e412d	2015-02-20 17:02:00.971936
126	79	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: EXON 2\ndescription: In a study by Pan et al in 2005, non-small cell lung cancer patients\n  harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors\n  gefinitib and erlotinib.\n	1	\N	\N	09e0092d-7555-45dd-8c61-5e2e88665be2	2015-02-20 17:02:00.998013
127	80	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	2cf085a5-a21b-4cae-864f-c955f86558a0	2015-02-20 17:02:01.025896
128	81	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12/G13\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation\n  may result in a less responsive tumor when treated with first-generation TKI's like\n  gefitinib.\n	1	\N	\N	faa17de7-b8f3-4d5e-ae66-3920d34549a9	2015-02-20 17:02:01.055291
129	82	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12C\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	f44d7c0f-618a-471e-af4d-fe98ac5637bc	2015-02-20 17:02:01.082637
130	83	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12D\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	7c040e9c-c5fa-41c3-bc79-31b30c73c0e7	2015-02-20 17:02:01.113128
131	84	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G13\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	54ff54b7-f9f6-4fc9-9410-a3e8294347e9	2015-02-20 17:02:01.143003
132	85	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G13D\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	1629acd5-0810-44b9-9c4b-f7368e35f1ca	2015-02-20 17:02:01.171211
133	86	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: P124S\ndescription: MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder\n  and colon cancer to a lesser degree. The P124S mutation has been shown to contribute\n  to AZD6244 resistance in melanoma cell lines, but considerably less so than its\n  Q56P counterpart.\n	1	\N	\N	4148c7ee-cf7e-47b9-a27d-8f4871fcd080	2015-02-20 17:02:01.195887
134	87	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: Q56P\ndescription: MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This\n  mutation has been shown to confer considerable resistance to AZD6244 treatment of\n  melanoma cell lines.\n	1	\N	\N	d7ef7591-c19c-4fbb-91c0-5b1a53105f66	2015-02-20 17:02:01.221233
135	88	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	df9c24ca-f672-450b-a37e-957ed86e265d	2015-02-20 17:02:01.244523
136	89	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 32\nname: PROMOTER METHYLATION\ndescription: MGMT promoter methylation has been observed to impact tumor progression\n  in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating\n  agent carmustine has shown efficacy. In patients lacking methylation, combining\n  carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more\n  experiments are required. Clinical trials have also shown selective sensitivity\n  of promoter methylation-positive patients to temozolomide, making a case for wider\n  methylation screening in GBM patients.\n	1	\N	\N	f6553bbd-22bd-4dd0-bd8e-b7351ff2e291	2015-02-20 17:02:01.273573
137	90	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: EXON 12\ndescription: NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\n	1	\N	\N	c3622e45-c755-4fd1-83a3-b15428011f01	2015-02-20 17:02:01.305292
138	91	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: W288FS\ndescription: NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most\n  common NPM1 mutation identified in acute myeloid leukemia. This mutation results\n  in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response\n  to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation,\n  it is generally grouped with other exon 12 mutations for patient analysis (see NPM1\n  Exon 12 variants for more information).\n	1	\N	\N	0034e2d2-628b-42f9-81f5-5b0029dc652a	2015-02-20 17:02:01.330776
139	92	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 1\ndescription: NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown\n  to be correlated with poorer overall survival relative to wild-type NRAS in melanoma\n  patients.\n	1	\N	\N	e511ac73-0c4e-451e-89ae-b50b501b2043	2015-02-20 17:02:01.358752
140	93	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 2\ndescription: NRAS exon 2 mutations have been shown to be correlated with poorer overall\n  survival in melanoma patients and colorectal cancer patients, however no prognostic\n  impact was seen in acute myeloid leukemia patients.\n	1	\N	\N	f3178413-2b1a-42ed-aba4-8de985ec73e2	2015-02-20 17:02:01.381628
145	98	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations\n  at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide.\n  However, in colorectal cancer patients, mutations at this locus have been shown\n  to confer resistance to cetuximab. The prognostic impact of mutations at this locus\n  is currently under study.\n	1	\N	\N	51195822-a540-4a11-a29e-28bf83607ec6	2015-02-20 17:02:01.513362
146	99	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61L\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	23021fec-6c5c-425b-ad20-61cea95bc917	2015-02-20 17:02:01.536992
147	100	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61R\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	1a115c0f-e3eb-439d-8608-ffdf1a643264	2015-02-20 17:02:01.569116
148	101	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842I\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	2aab0ca1-7e08-493c-aaff-6c059c1850e5	2015-02-20 17:02:01.59371
149	102	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842V\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	c4bccb3a-8aac-4ee7-9f84-b823564de748	2015-02-20 17:02:01.642782
150	103	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842Y\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	51c344fc-0e5c-4caa-8b5e-b28610897517	2015-02-20 17:02:01.669507
151	104	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: DEL I843\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	9b5c75bf-0d2f-4c19-9394-f9655c7f9c37	2015-02-20 17:02:01.694614
152	105	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: DI842-843IM\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	aa863dfe-14e5-4a29-b308-7f008166d8f2	2015-02-20 17:02:01.721863
153	106	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: E542K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	45ab1f34-1953-47df-a9a0-590c015ee14c	2015-02-20 17:02:01.752365
154	107	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: E545K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	7acccd51-adef-48d3-9102-c94870d3b16d	2015-02-20 17:02:01.77847
155	108	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 20\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	e8008b1f-959f-49f7-884e-acf0e9f04035	2015-02-20 17:02:01.804129
156	109	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 9\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	b3acc7f2-8315-4480-9fbf-e6bfb6fd5a90	2015-02-20 17:02:01.833037
157	110	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: H1047R\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	1fce3248-a2a7-4d19-8493-412d961240a2	2015-02-20 17:02:01.861862
159	112	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in a rare form of adolescent\n  liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found\n  Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined,\n  and functional studies found that the fusion retained kinase activity. The presence\n  of this fusion may be used as a diagnostic marker for this rare tumor type.\n	1	\N	\N	57a774d1-cce9-4232-8008-d5c9f7cd1095	2015-02-20 17:02:01.922469
160	113	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 39\nname: R233*\ndescription: PTEN R233* has been shown to be a loss of function mutation, and PTEN\n  loss has been the subject of considerable research in breast cancer. PTEN loss may\n  sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data\n  to support that PTEN loss is both associated with poorer prognosis, and no change\n  in prognosis.\n	1	\N	\N	25988b36-c4e3-405b-8b69-f13df982f897	2015-02-20 17:02:01.952748
161	114	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and Arsenic Trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	60a4955e-f205-4ef7-a2be-5056412c753c	2015-02-20 17:02:01.97906
162	115	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: C634W\ndescription: RET C639W has been implicated as an alternate mechanism of activating\n  RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting\n  agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the C639W mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib.\n	1	\N	\N	170c326f-f751-456d-9895-e96f29a4ee86	2015-02-20 17:02:02.014109
163	116	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: M918T\ndescription: RET M819T is the most common somatically acquired mutation in medullary\n  thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents,\n  promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the M918T mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T\n  leads to more aggressive MTC with a poorer prognosis.\n	1	\N	\N	b163ed55-274b-424b-9353-cda94160121c	2015-02-20 17:02:02.042913
164	117	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: K666N\ndescription: SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients.\n	1	\N	\N	12220bdb-10fa-458b-8f32-278be18b6b8b	2015-02-20 17:02:02.06928
165	118	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: K700E\ndescription: SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients. Additionally,\n  these mutations are the most common SF3B1 mutation observed in MDS and highly associated\n  with subtypes of MDS that are defined by ringed sideroblasts.\n	1	\N	\N	895d5c91-8b5d-45e2-a11e-bacf77801691	2015-02-20 17:02:02.09228
166	119	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R175H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R175H variant seems not only to result in loss of tumor-suppression, but also acts\n  as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell\n  lines harboring this mutant is also more responsive to treatment with doxorubicin\n  than its wild-type counterparts. While the prognostic impact of individual TP53\n  mutations is influenced by the cohort being studied, it has been shown that the\n  R175H mutation is correlated with worse overall survival than wild-type TP53, but\n  is not as detrimental as the R248W variant.\n	1	\N	\N	3c9bf59e-c680-4ec4-8269-e290f2a82185	2015-02-20 17:02:02.125479
167	120	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R248Q\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival. The R248Q mutation has also shown an increased invasive behavior in cell\n  lines. This is specific to the 248Q variant.\n	1	\N	\N	9f58a052-040b-4435-be17-6d0be8ed1bce	2015-02-20 17:02:02.152954
168	121	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R248W\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival.\n	1	\N	\N	5ed8286a-1e33-4cef-a08b-ceb4a56bbe77	2015-02-20 17:02:02.183284
169	122	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R249T\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	e6e61c9c-3672-4227-bb56-d483129a3fbe	2015-02-20 17:02:02.211866
170	123	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R249W\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	ffd1ad32-de2e-42c2-820f-d3a1eb0e741c	2015-02-20 17:02:02.239888
171	124	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R273C\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	131e5a9e-8d53-4963-9303-dd2d051aa377	2015-02-20 17:02:02.265261
248	68	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with the ER-positive subtype of breast\n  cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 76\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	62be177b-d736-4df4-8f7e-828c46534a93	2015-02-20 17:02:07.271999
172	125	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R273H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	95c8589d-59d1-4b03-94dd-c9172f33d606	2015-02-20 17:02:02.305997
173	126	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: V173G/A\ndescription: While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R249 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	b7bf33bc-bb18-4bad-bcec-4f746f0873fd	2015-02-20 17:02:02.335096
174	127	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: FRAMESHIFT TRUNCATION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	c2f89916-7e0d-4bae-95cf-26bc5a731bb5	2015-02-20 17:02:02.35944
175	128	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	e606e5e4-1e9c-4dde-8de3-8c584a1a5b85	2015-02-20 17:02:02.38697
176	129	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	2b00fbe7-a2ee-424e-9d44-5a52c69d9113	2015-02-20 17:02:02.411851
177	130	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: Q157P/R\ndescription: U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is less common than the S34F mutation, occurs in the second zinc finger domain of\n  U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations\n  on overall survival in MDS has been debated, however, patients with U2AF1 mutations\n  were shown to be at an increased risk of transformation to secondary AML. The presence\n  of this mutation was not associated with a specific prognostic outcome in AML when\n  compared to U2AF1 wildtype patients.\n	1	\N	\N	bfc00d8b-ceb6-4f19-8688-43e837e933a9	2015-02-20 17:02:02.437794
178	131	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: S34Y/F\ndescription: U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is the most commonly identified variant in MDS, occurs in the first zinc finger\n  domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1\n  mutations on overall survival in MDS has been debated, however, patients with U2AF1\n  mutations were shown to be at an increased risk of transformation to secondary AML.\n  The presence of this mutation was not associated with a specific prognostic outcome\n  in AML when compared to U2AF1 wildtype patients.\n	1	\N	\N	96c58484-c61e-4be6-b7b3-2943019c143a	2015-02-20 17:02:02.469898
179	132	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: EXON 7\ndescription: WT1 exon 7 mutations have been shown to be recurrent in acute myeloid\n  leukemia. Many sources have examined the prognostic impact of these, agreeing that\n  the mutant exon 7 cohort is correlated with worse overall survival and a number\n  of poor prognistic outcomes. This may be the result of an overall poor response\n  to chemotherapy from WT1 mutant tumors.\n	1	\N	\N	ce253900-7e94-4c2b-bdef-82b12e04c60a	2015-02-20 17:02:02.500918
180	133	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: EXON 9\ndescription: WT1 exon 9 mutations have been shown to be recurrent in acute myeloid\n  leukemia, although at a less frequent rate than their exon 7 counterparts. Many\n  sources have examined the prognostic impact of these, agreeing that the mutant exon\n  9 cohort is correlated with worse overall survival and a number of poor prognistic\n  outcomes. This may be the result of an overall poor response to chemotherapy from\n  WT1 mutant tumors.\n	1	\N	\N	bac158a1-acb5-4231-9632-7878766e97c3	2015-02-20 17:02:02.527671
181	1	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL fusions have been found to contribute to imatinib resistance in AML\n  cell lines.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 1\nsource_id: 41\nvariant_id: 1\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	dc86f7cf-3efc-4670-8fff-97a5e20c24ad	2015-02-20 17:02:02.68297
182	2	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide\n  results in high rates of complete remission in patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 1\nsource_id: 41\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e5c0419d-0376-47fc-92a7-706214adc885	2015-02-20 17:02:02.743025
183	3	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of BCR-ABL fusion is considered the characterizing feature of chronic\n  myeloid leukemia, and has been widely thought of as the initiating event in the\n  disease.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	eeed71ff-e532-4b6e-bd71-fb1c8630ec93	2015-02-20 17:02:02.807212
184	4	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient\n  has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may\n  imply the possibility that BCR-ABL is not universally the initiating event in CML.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 2\nsource_id: 43\nvariant_id: 1\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d604244d-1b97-461b-97cb-e9f4845a1827	2015-02-20 17:02:02.875015
185	5	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The clinical use of imatinib in patients with BCR-ABL fusion has resulted in\n  drastic sensitivity to the drug.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	679d3fde-2b4a-4137-b5df-5fa8c6bbd53a	2015-02-20 17:02:02.964951
186	6	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has been effective in\n  patients resistant to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4cea464c-aa33-4dc3-b5b8-4a0e717bca16	2015-02-20 17:02:03.035798
187	7	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has been effective in\n  patients resistant to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 5\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0eaf17e5-61fe-4a6d-b9f2-e2d1cb2775fe	2015-02-20 17:02:03.092931
188	8	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL inhibitors such as imatinib have lead to "significantly improved prognosis,\n  response rate, overall survival, and patient outcome in CML patients compared to\n  previous therapeutic regimens."\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	26f4ac20-e538-4f2d-96e6-8764ff05dbd3	2015-02-20 17:02:03.156366
189	9	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 2\nsource_id: 44\nvariant_id: 3\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	acaa7a8a-5b02-4675-b185-08f8e17f3d9b	2015-02-20 17:02:03.223721
190	10	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have\n  shown to be resistant to imatinib treatment.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 2\nsource_id: 42\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	08e17af4-7c17-4791-802a-f3ad4bdbb2ba	2015-02-20 17:02:03.320122
191	11	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 2\nsource_id: 44\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	286ac4e0-d4da-48d6-91c6-007c58f1c6f0	2015-02-20 17:02:03.386389
192	12	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the\n  AKT inhibitor MK-2206 in breast cancer cell lines.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 3\nsource_id: 45\nvariant_id: 4\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	afb9fbb6-da2a-4b8f-8e0f-8a8228ee08cd	2015-02-20 17:02:03.465171
193	13	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 4\nsource_id: 46\nvariant_id: 6\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	505e4a90-9c4e-4da9-9dbe-36b47bf0c34a	2015-02-20 17:02:03.553757
194	14	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment\n  with crizotinib has shown to be effective in acheiving stable disease.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 4\nsource_id: 46\nvariant_id: 5\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e9439648-bcc4-4657-92b5-2f23b068ec6f	2015-02-20 17:02:03.638622
195	15	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without\n  this mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 5\nsource_id: 47\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0fe4880f-0c86-4b9d-bce0-cac4e708ce45	2015-02-20 17:02:03.710154
196	16	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in\n  a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 8\ndisease_id: 6\nsource_id: 48\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e70a41f3-4dc6-42c2-831f-71fbe89d137a	2015-02-20 17:02:03.778237
197	17	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing\n  the EML4-ALK fusion containing the F1174L mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 6\nsource_id: 49\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e483dd74-544a-45ad-8151-c91a372d3015	2015-02-20 17:02:03.839302
198	18	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 6\nsource_id: 50\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1c60309f-f573-412d-8016-6ec0a08c9704	2015-02-20 17:02:03.910624
199	19	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing\n  ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 9\ndisease_id: 6\nsource_id: 51\nvariant_id: 8\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	003222cb-48fc-4d50-a8b0-33fa9eb1d6d2	2015-02-20 17:02:03.983926
200	20	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without\n  this mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 4\nsource_id: 47\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	06ccaf51-8493-49a5-a96c-e8730d214fa0	2015-02-20 17:02:04.046007
358	178	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8dea598e-72b7-4d74-9d55-3887e1d5a0db	2015-02-20 17:02:14.878123
201	21	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 treatment resulted in significant tumor regression in xenograft models\n  produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 8\ndisease_id: 4\nsource_id: 48\nvariant_id: 7\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	284a40dc-77b2-4748-b426-757f5ae68d70	2015-02-20 17:02:04.139582
202	22	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 4\nsource_id: 46\nvariant_id: 7\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d5f05770-0736-4d36-8043-60016d2db667	2015-02-20 17:02:04.203234
203	23	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells\n  with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 9\ndisease_id: 6\nsource_id: 51\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9d3d2f04-c3af-42be-9f24-9d1c25ce1269	2015-02-20 17:02:04.26688
204	24	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 9\ndisease_id: 6\nsource_id: 51\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	44fc23c8-cd22-444c-b0e2-7401dd0760e7	2015-02-20 17:02:04.326914
205	25	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In one patient with S214C mutation, the use of sorafenib has led to more than\n  5 years of survival and near remission.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 4\nsource_id: 52\nvariant_id: 10\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c44fa37a-1759-4a37-97cb-5f94be25241f	2015-02-20 17:02:04.402908
206	26	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony\n  formation of AALE cells expressing ARAF S214C in vitro\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 11\ndisease_id: 4\nsource_id: 52\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	78afe367-5a16-4e4b-8029-f91abae7eef3	2015-02-20 17:02:04.479335
207	27	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony\n  formation of AALE cells expressing ARAF S214C in vitro\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 4\nsource_id: 52\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b0574260-6a1c-474f-9edf-06392b504df2	2015-02-20 17:02:04.543847
208	28	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations\n  were associated with poor progression free survival regardless of treatment (panitumumab\n  with best supportive care or best supportive care alone)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 7\nsource_id: 53\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2d550fce-be25-4e8a-95b9-430c2e470902	2015-02-20 17:02:04.607629
209	29	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutations are associated with melanoma arising in non-chronic sun damaged\n  skin and with superficial spreading melanoma\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 54\nvariant_id: 17\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f39f7da5-4f58-4360-9bb6-b21010fdfdfb	2015-02-20 17:02:04.673957
210	30	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 55\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4b67fdc4-d9e0-4d38-ab1e-f8446ea307f4	2015-02-20 17:02:04.777019
211	31	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 8\nsource_id: 55\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	89b219ba-c03e-4029-b7bc-99bc3b509fd7	2015-02-20 17:02:04.828375
212	32	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 56\nvariant_id: 11\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	39b399d5-243b-482f-b9dd-610991131414	2015-02-20 17:02:04.888588
213	33	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 9\nsource_id: 57\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	502af469-c596-450c-bd65-022035f982a5	2015-02-20 17:02:04.966379
214	34	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 12\ndisease_id: 9\nsource_id: 57\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	536d5e8c-1924-4e0f-93ac-5610d7e89243	2015-02-20 17:02:05.020468
215	35	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 9\nsource_id: 57\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8e812128-3cd7-4229-8727-3b650947316b	2015-02-20 17:02:05.071948
230	50	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 65\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4a0b6f84-39d6-4191-befb-3857e9c84532	2015-02-20 17:02:06.052933
216	36	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 15\ndisease_id: 9\nsource_id: 58\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e3f44a79-3b5c-4428-afa4-d48c46f9ec47	2015-02-20 17:02:05.148209
217	37	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 9\nsource_id: 58\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6d9d1991-e472-4263-b5ce-8e1f9f777d73	2015-02-20 17:02:05.20102
218	38	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 9\nsource_id: 59\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a0dea4eb-1b00-4432-a3bb-eef3a9aa7e51	2015-02-20 17:02:05.294753
219	39	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 17\ndisease_id: 9\nsource_id: 59\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4c0bf919-5691-43b4-8879-39a1c394a911	2015-02-20 17:02:05.347514
220	40	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 18\ndisease_id: 7\nsource_id: 60\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d8f8c0eb-0ebe-4219-9c4d-52ac700658f7	2015-02-20 17:02:05.414206
221	41	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 19\ndisease_id: 7\nsource_id: 60\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a59cb045-043b-47c1-844c-58f9037f2278	2015-02-20 17:02:05.468403
222	42	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 20\ndisease_id: 7\nsource_id: 60\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	31771db1-86c0-4c5a-a1b7-73dcc66d08d4	2015-02-20 17:02:05.522196
223	43	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines expressing the BRAF V600E mutation responded better to vemurafenib\n  treatment than cells wildtype for BRAF as measured by reduced cellular proliferation\n  and inhibition of MET and ERK phosphorylation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 19\ndisease_id: 7\nsource_id: 60\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8eb64745-8bb0-4446-87a0-fe1ae5be3c84	2015-02-20 17:02:05.586654
224	44	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is associated with adverse pathological features of colorectal cancer.\n  This can be concluded as a marker of poor prognosis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 7\nsource_id: 61\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6fdf0bb8-8932-46d1-bde9-ed59df629f50	2015-02-20 17:02:05.646803
225	45	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations\n  had lower response and disease control rates as well as shorter progression free\n  and overall survival following cetuximab plus chemotherapy than those with wildtype\n  BRAF\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1cddba86-4142-49e5-8e4f-b4732bc64159	2015-02-20 17:02:05.706598
226	46	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  but does not show resistance to MEK inhibitors.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 8\nsource_id: 63\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	46b58dc7-65e3-45da-897d-9f39a1fca019	2015-02-20 17:02:05.799791
227	47	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  and resistance to RAF inhibitors.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 8\nsource_id: 63\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3ea77d93-3646-479d-a9c0-5913177a62bd	2015-02-20 17:02:05.862125
228	48	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 21\ndisease_id: 8\nsource_id: 64\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	340542bb-37cb-44b5-8278-3e5eab0af355	2015-02-20 17:02:05.927756
229	49	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 22\ndisease_id: 8\nsource_id: 64\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	10d0d79b-91b4-4945-8f90-3808e64ba296	2015-02-20 17:02:05.986627
438	258	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12C occur more frequently in women than men\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 162\nvariant_id: 82\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b8460678-3318-4179-b4a4-679ab4805e71	2015-02-20 17:02:20.282569
231	51	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 8\nsource_id: 65\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b7a194df-3fd5-4d8b-8137-9ea521cbed8f	2015-02-20 17:02:06.105917
232	52	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with shorter disease-free and overall Survival in a\n  Spanish cohort of melanoma patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 66\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	32385ab1-300f-4363-adbb-81684158535d	2015-02-20 17:02:06.169035
233	53	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 4\nsource_id: 67\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b61bb83a-f5d2-4554-9d29-8abe4c9c181e	2015-02-20 17:02:06.237623
234	54	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 68\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7ff4dff9-287a-47f0-a0ee-ead310ad45e5	2015-02-20 17:02:06.299596
235	55	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 69\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	50794e84-9ad2-4369-a970-87c4e4585ea2	2015-02-20 17:02:06.382664
236	56	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is shown to be associated with the tall-cell variant of papillary\n  thyroid cancer\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 70\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c65088be-f06a-445f-a13b-59571cb87317	2015-02-20 17:02:06.450291
237	57	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Unlike other studies that suggest a poorer outcome, BRAF mutation in this study\n  was not correlated with poorer prognosis in papillary thyroid cancer.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 71\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d96aaf87-d498-4bc5-b1fe-f7c8733e4d3d	2015-02-20 17:02:06.514965
238	58	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is correlated with disease recurrence in both age cohorts (>65 and <65\n  yo).\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 70\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ef079a79-0989-487e-85b4-73aa1e1efe03	2015-02-20 17:02:06.581524
239	59	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in\n  a study of 187 patients with PTC and other thyroid diseases.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 72\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	737d476a-c10d-4172-b95c-b215bfa3c69e	2015-02-20 17:02:06.653194
240	60	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutation correlated with poor prognosis in papillary thyroid cancer in\n  both older (>65 yo) and younger (<65 yo) cohorts.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 70\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	990d6b9a-5aa0-4904-8357-2d41eb51a174	2015-02-20 17:02:06.709609
241	61	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COLO201 and COLO206F cell harboring BRAF V600E mutations were cloned to be MEK\n  inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification\n  of the BRAF V600E gene.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 23\ndisease_id: 7\nsource_id: 73\nvariant_id: 14\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	244d31a1-1422-4045-8737-84b040227eb5	2015-02-20 17:02:06.778199
242	62	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A single patient with BRAF V600E/V600M bi-allelic mutation responded to the\n  V600E drug dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 74\nvariant_id: 13\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	087374dc-f5c1-4bfa-a88e-ffd27ccae7ef	2015-02-20 17:02:06.851566
243	63	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 56\nvariant_id: 15\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	78db23dc-40d5-4017-8fd5-e7f51c689a4a	2015-02-20 17:02:06.946732
244	64	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 56\nvariant_id: 16\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	64659f48-bc89-4974-ad07-8420e53aa10b	2015-02-20 17:02:07.009
245	65	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Inceased copy number of CCND1 is associated with poorer overall survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 75\nvariant_id: 18\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4e601bcc-43fe-4c5f-bc3e-3b8432bf9785	2015-02-20 17:02:07.079068
246	66	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have associated CCND1 expression with poorer survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 75\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ce0118fd-32bc-448c-afe1-3a3962bb66b1	2015-02-20 17:02:07.140801
247	67	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have found no significant survival impact for CCND1 in lung cancer.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 75\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	65367025-2987-40ef-b94f-6376de0089a7	2015-02-20 17:02:07.204634
249	69	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In ER-positive breast cancer patients, Cyclin D1 overexpression is associated\n  with shorter overall survival and increased metastasis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 76\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e5ce6305-b299-482e-b669-44bec084084d	2015-02-20 17:02:07.349168
250	70	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in head and neck squamous cell carcinoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 12\nsource_id: 77\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	03e46d1d-bad4-40ac-bfbd-92449c5813ce	2015-02-20 17:02:07.445937
251	71	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in mantle cell lymphoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 13\nsource_id: 78\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	55708a26-2d3c-4c2f-94dd-955040a19c10	2015-02-20 17:02:07.506499
252	72	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the presence pf lymph node metastasis\n  in gastric cancers.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 79\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	aa9265da-9f77-4ad5-9815-bbf9ed494d19	2015-02-20 17:02:07.572094
253	73	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the increased depth of cancer invasion\n  in gastric cancers.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 79\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ecebd9ec-cce1-4f6a-a273-105e71e01fbc	2015-02-20 17:02:07.64026
254	74	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 79\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3a29237b-57fe-41a3-a1e7-1ec09e50de56	2015-02-20 17:02:07.697962
255	75	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with vascular invasion by cancer cells\n  in gastric cancers.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 79\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6b183220-c86f-402a-924a-51a774c97529	2015-02-20 17:02:07.757828
256	76	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CCND2 hypomethylation is seen to be more common in stage III and IV gastric\n  tumors than stage I and II.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 80\nvariant_id: 22\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5c7250fe-875f-4354-a37b-68ebbb6f25d9	2015-02-20 17:02:07.821094
257	77	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib\n  (PD-0332991) significantly increased median survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 24\ndisease_id: 15\nsource_id: 81\nvariant_id: 23\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1fc7d21f-e084-4144-94a5-0fc44b4bca0f	2015-02-20 17:02:07.890427
258	78	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with high expression of total cyclin E and low-molecular weight cyclin\n  E is strongly associated with poor prognosis in breast cancer, and the hazard ratio\n  for these patients in 13.3 times higher than those with normal cyclin E levels.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 82\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	de17c7b2-071c-4481-a5a7-be57bfae508b	2015-02-20 17:02:07.973998
259	79	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced\n  stage gastric carcinoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 16\nsource_id: 83\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	154afacb-e624-403d-a5e9-5c6a7c7d6de5	2015-02-20 17:02:08.049794
260	80	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 16\nsource_id: 83\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6e7b850a-c760-48f1-befa-4df8dc95c0e7	2015-02-20 17:02:08.119053
261	81	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 83\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	656645e4-048e-4afa-ad87-3e5ae819a72c	2015-02-20 17:02:08.192054
262	82	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 25\ndisease_id: 18\nsource_id: 84\nvariant_id: 25\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3a2df3c8-0f6f-48c3-a821-092859ffdcca	2015-02-20 17:02:08.259997
263	83	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 18\nsource_id: 85\nvariant_id: 25\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f3f20e5e-86a7-4358-a4ca-c7ba9a5b78fb	2015-02-20 17:02:08.332674
264	84	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 25\ndisease_id: 18\nsource_id: 84\nvariant_id: 26\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e629c095-7d55-4732-8ca3-db9152f432bd	2015-02-20 17:02:08.404052
265	85	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 18\nsource_id: 85\nvariant_id: 26\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e68961f6-235e-4c34-ace9-2eaa19a21850	2015-02-20 17:02:08.499667
266	86	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter time to recurrence.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 86\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	53537026-4d57-425c-a33c-05216736cd4f	2015-02-20 17:02:08.565036
267	87	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter overall survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 86\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	25ee3d61-55ee-4946-b7ab-ccaa6ba27922	2015-02-20 17:02:08.625183
268	88	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation status had no impact on ATRA treatment response in older (>60)\n  patients with AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 87\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1d0e4fd1-935e-4131-8ff9-c34a7b87be79	2015-02-20 17:02:08.696794
269	89	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was significantly associated with complete remission\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 88\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	dde6ef86-3179-4e42-8c69-3abf496054ab	2015-02-20 17:02:08.757582
270	90	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was associated with improved overall survival in older (>60)\n  patients with AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 87\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2c684b7b-752b-4aba-82c7-2fef28b6e8ac	2015-02-20 17:02:08.813435
271	91	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Overall survival was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA mutations\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 89\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2b303745-c552-489d-81d8-ab1986164e13	2015-02-20 17:02:08.875816
272	92	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Remission duration was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 89\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6e70283d-a1eb-4c88-adee-5d8f1ece1c83	2015-02-20 17:02:08.962526
273	93	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 89\nvariant_id: 28\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4edf5c2b-8011-45f7-98b2-4bca01860aeb	2015-02-20 17:02:09.027318
274	94	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 90\nvariant_id: 30\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8588de07-6e9c-4e63-a331-239afc7a250f	2015-02-20 17:02:09.097262
275	95	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 27\ndisease_id: 19\nsource_id: 90\nvariant_id: 30\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1a986acd-65d5-4c7f-a4ea-043b1741dfb9	2015-02-20 17:02:09.153541
276	96	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 20\nsource_id: 91\nvariant_id: 31\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	af0c994c-82e4-4b86-a2c2-a629b2144e7c	2015-02-20 17:02:09.227943
277	97	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutations (60% of which were\n  R882) were older in age, had higher white blood cell counts and had higher platelet\n  counts than patients wildtype for DNMT3A\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d0f1884e-2438-48c0-bd8b-882ae74d164e	2015-02-20 17:02:09.29652
278	98	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations occur most often in de novo AML patients with intermediate\n  cytogenic risk.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 93\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1940bc1d-0cb6-4610-8d00-025f8ca269ac	2015-02-20 17:02:09.353331
279	99	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Daunorubicin treatment resulted in similar overall survival and disease free\n  survival in de novo AML patients with DNMT3A R882 mutation compared to those who\n  do not harbor this mutation.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 28\ndisease_id: 1\nsource_id: 93\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	19b28c22-6abb-4208-9455-64c58ebb7bdb	2015-02-20 17:02:09.422486
439	259	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12D mutation occurs in never smokers significantly more often than in\n  smokers\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 162\nvariant_id: 83\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	964b41ee-6dd9-4bbe-984b-7395a181b783	2015-02-20 17:02:20.340668
280	100	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Idarubicin increases the overall survival and disease free survival in de novo\n  AML patients with DNMT3A R882 mutation compared to those who do not harbor this\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 1\nsource_id: 93\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	23f20e26-daf2-4d58-a954-fc26f4375fbc	2015-02-20 17:02:09.502623
281	101	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no difference in the complete remission rate of de novo AML patients\n  with DNMT3A mutation compared to those who are wild type for DNMT3A.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 93\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4fa9d4b0-d791-4537-a4af-4de087bbe52c	2015-02-20 17:02:09.565526
282	102	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free\n  and overall) outcome than those without DNMT3A mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	672d5b2f-26b8-4773-a3d6-61185d02e7d7	2015-02-20 17:02:09.6351
283	103	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with older age, higher white blood cell\n  count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically\n  normal AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 95\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b648a13d-6a37-4b57-abe9-a8b2c775f9da	2015-02-20 17:02:09.706864
284	104	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status (60% of which\n  were R882) was not predictive of overall and relapse free survival in patients with\n  NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ff4c1aad-ff16-4803-8a2e-d35f47792494	2015-02-20 17:02:09.770912
285	105	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5%\n  of which were R882) had no prognostic value on overall, relapse free and event free\n  survival.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	009af995-1621-458f-a134-bc1824dcde42	2015-02-20 17:02:09.832879
286	106	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of cytogenetically normal AML patients (18-60 years old),\n  DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall,\n  relapse free and event free survival\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3fb0709e-d4cb-4c4b-8b88-e88c72c9e657	2015-02-20 17:02:09.896811
287	107	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with DNMT3A mutations (59% of which were R882) showed worse survival\n  (event-free and overall) outcome than those without DNMT3A mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2956b388-9f12-4ac0-bb8d-0e2049038167	2015-02-20 17:02:09.984406
288	108	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including\n  R882) were associated with worse overall survival compared to those without DNMT3A\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bf1a274a-4108-4e5e-8c51-a2eac005327d	2015-02-20 17:02:10.047962
289	109	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cytogenetically normal AML patients, DNMT3A R882 mutations are associated\n  with lower overall and disease free survival as compared to patients with wild type\n  DNMT3A.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 95\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	87a860b2-69bb-4420-90c6-7399a5359d6a	2015-02-20 17:02:10.106539
290	110	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutations were associated with\n  significantly reduced overall survival and relapse free survival in patients wildtype\n  for NPM1 and FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4ce8d61d-f67e-456e-8c2c-ca29451e28c4	2015-02-20 17:02:10.16772
291	111	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation\n  is prognostic for shorter disease free survival and overall survival compared to\n  patients without the mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 97\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	33042b49-c2d7-44b6-aeb6-180668f5dfe6	2015-02-20 17:02:10.232498
292	112	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations\n  other than R882 are prognostic for shorter disease free survival and overall survival\n  compared to patients without the mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 97\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	07252c3a-1b9b-429a-87a4-27a20e3909a5	2015-02-20 17:02:10.298574
293	113	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutation was associated with reduced relapse free and overall survival\n  in ELN-unfavorable, cytogenetically normal AMLs\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	29f01bed-bf39-4fe8-8df0-68a319bfdcf9	2015-02-20 17:02:10.363944
294	114	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations\n  were associated with reduced relapse free survival in the entire cohort as well\n  as the subset of patients with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e9ff46df-21f2-4eea-b761-a9056e4c4c70	2015-02-20 17:02:10.454346
295	115	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Therapy-related AML was less common in patients with DNMT3A mutations (64.5%\n  of which were R882) than patients wildtype for DNMT3A in a large cohort of younger\n  (18-60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	13b32c41-eb5d-4a3c-a99d-1f2064d53953	2015-02-20 17:02:10.518934
296	116	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with an intermediate\n  risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b6ef6bef-5fb5-41c0-b2dc-1422328f3bcf	2015-02-20 17:02:10.582191
297	117	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with intermediate\n  risk cytogenetics (including normal karyotype)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b3e141d3-0925-457b-93a7-5c0f2cec06a0	2015-02-20 17:02:10.644512
298	118	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (64.5% R882) were associated with older age, higher white blood\n  cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML\n  patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6ae2523c-9924-4bce-91f3-75b9ec95ba91	2015-02-20 17:02:10.700591
299	119	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with cytogenetically normal AML in a large\n  cohort of younger (18-60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2c0233ae-6c7b-4d11-a681-e2b634d7b395	2015-02-20 17:02:10.76766
300	120	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations were associated with achievement of complete remission in a\n  large cohort of younger (18-60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	622b45ab-8d77-4ddc-a535-1225f396bb4d	2015-02-20 17:02:10.840734
301	121	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates did not differ between patients with wildtype or mutant\n  DNMT3A (62% of which affected R882) and cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 97\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	da8e74f4-417d-45d3-b916-d47bff7af372	2015-02-20 17:02:10.982341
302	122	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (62% of which were R882) were associated with reduced disease-free\n  survival in patients with cytogenetically normal AML.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 97\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2c3a4d13-2d44-4c1d-aca9-af8936a1bc66	2015-02-20 17:02:11.052239
303	123	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutation have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a6c5d7b0-fd8e-4037-8d88-e26ced8bbd19	2015-02-20 17:02:11.111016
304	124	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rate was not different between young AML patients (<60 years\n  old) with or without DNMT3A mutations (60% of which were R882)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1fdedcfb-7846-40bd-ad15-36c37e7c9db0	2015-02-20 17:02:11.17417
305	125	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	fbdd92e7-0b40-4e87-a484-d8c825d71df4	2015-02-20 17:02:11.236277
306	126	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 30\ndisease_id: 4\nsource_id: 98\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9efeb43d-4194-49b9-9077-3e71d9875f7a	2015-02-20 17:02:11.302474
307	127	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 4\nsource_id: 98\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5068830f-c2f7-425a-83d4-02ef65836d5c	2015-02-20 17:02:11.354509
308	128	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence\n  of L858R mutation is prognostic for better progression free survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 99\nvariant_id: 33\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	45d174e6-18b4-490b-a8b4-6c24c58a4b5a	2015-02-20 17:02:11.413253
309	129	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Geftinib added to pemetrexed and carboplatin showed greater effectiveness than\n  premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 30\ndisease_id: 4\nsource_id: 100\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f01be1d8-63b3-4b79-bfb6-8b5db85e6137	2015-02-20 17:02:11.481293
310	130	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Median survival of patients with EGFR L858R mutation is better than those with\n  wild type EGFR.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 101\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	84c2284f-e355-4b01-b6db-38c54dc9b4d9	2015-02-20 17:02:11.559746
311	131	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The T790M mutation in EGFR has been shown to confer resistance to the tyrosine\n  kinase inhibitor erlotinib, and patients harboring this mutation that are placed\n  on the drug are likely to relapse.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 31\ndisease_id: 21\nsource_id: 102\nvariant_id: 34\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a806c6cc-39e0-46cf-b84a-e7d16040ec92	2015-02-20 17:02:11.626453
312	132	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In non-small cell lung cancer, the appearance of T790M mutation leads to resistance\n  to gefitinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 30\ndisease_id: 4\nsource_id: 103\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e6dd17ed-d596-43cb-b408-3ecba88eecdc	2015-02-20 17:02:11.705089
313	133	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting\n  in stable disease.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 4\nsource_id: 104\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	40c531ba-2e7d-4baf-a4d4-965cd2f76507	2015-02-20 17:02:11.762937
314	134	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting\n  in stable disease.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 32\ndisease_id: 4\nsource_id: 104\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	88306a20-a8d8-45b5-aac7-f4f31cbbabe4	2015-02-20 17:02:11.821386
315	135	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR,\n  especially when an L858R mutation is also present.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 33\ndisease_id: 4\nsource_id: 105\nvariant_id: 34\nrating: 1\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	28f3a983-261a-48ec-9e88-cb0b4053cc21	2015-02-20 17:02:11.898257
316	136	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall\n  survival compared to patients with L858R or other exon 19 activating mutations.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 106\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	64bfc55c-15f1-4273-82f5-97e3bafc692b	2015-02-20 17:02:11.963692
317	137	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients with T790M and another activating mutations, their progression\n  free survival is shorter compared to those who do not possess T790M mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 107\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ce452248-0bd6-4637-9e0e-1063780d7fa1	2015-02-20 17:02:12.03029
318	138	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 35\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	47a50f33-49d6-4240-81af-4314c7c9db2a	2015-02-20 17:02:12.11723
319	139	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 36\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	30aa532d-2e46-4353-a81a-758cf344c5f9	2015-02-20 17:02:12.194954
320	140	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2,\n  cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive\n  to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 37\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	fe266277-adea-4fa4-b0a3-4d0a979a6b1f	2015-02-20 17:02:12.266117
321	141	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 38\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	99363b45-0179-41c4-b5e5-1ca0133036a9	2015-02-20 17:02:12.334106
322	142	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally\n  transduced with wildtype or mutant ERBB2.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	40fe33d8-323b-4ad3-872d-7ea26790b341	2015-02-20 17:02:12.399772
323	143	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell\n  lines retrovirally transduced with wildtype or mutant ERBB2.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 11\nsource_id: 108\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	af94b1e2-0bce-4ad9-bf8b-499a0d2f7dc2	2015-02-20 17:02:12.466704
324	144	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive\n  to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 40\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	83c5d8ce-b7c6-4695-ae65-b730cf7eef6e	2015-02-20 17:02:12.524233
325	145	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the in-frame insertion of a single amino acid at position\n  780 was shown to be sensitive to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 41\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2080a5b1-a2c6-4ef2-af71-8268a89843bb	2015-02-20 17:02:12.581203
326	146	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 42\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cac9e5c3-b45c-4301-a7e8-952e765ccf0f	2015-02-20 17:02:12.677742
327	147	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 11\nsource_id: 108\nvariant_id: 42\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	59feb3fe-3cd5-4489-a340-6991f20198c2	2015-02-20 17:02:12.739793
328	148	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 43\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5323f8af-9108-40fb-8629-a2b8526dbc7e	2015-02-20 17:02:12.796991
329	149	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 44\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	216f0a7e-d42a-4701-9b98-a6c0e5df42e3	2015-02-20 17:02:12.863523
330	150	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 45\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cf48a6f8-b28c-4898-be8f-0c009b1c52cf	2015-02-20 17:02:12.930684
331	151	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L536Q ligand binding domain mutation have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 36\ndisease_id: 11\nsource_id: 109\nvariant_id: 46\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bbc6ca1e-5ae4-47dd-8c9e-110df0d6da0d	2015-02-20 17:02:13.010363
332	152	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L536Q ligand binding domain mutation have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 46\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f0c55317-4757-4255-b39b-5274279e3389	2015-02-20 17:02:13.074468
333	153	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 46\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ad249078-af9a-4f10-8867-e4562166555b	2015-02-20 17:02:13.150433
334	154	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The N538G ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 36\ndisease_id: 11\nsource_id: 109\nvariant_id: 47\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7b51e494-c31d-4d00-84d5-b6d52bdcc3e5	2015-02-20 17:02:13.244478
335	155	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The N538G ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 47\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ca146493-d6a3-4e28-99cc-b86a508f138a	2015-02-20 17:02:13.299652
336	156	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 47\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cc382257-f2a1-421a-baf5-f504c3fff540	2015-02-20 17:02:13.371744
337	157	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537C ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 36\ndisease_id: 11\nsource_id: 109\nvariant_id: 48\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ea1eb6d6-becd-4095-a9e0-a9d09de3bca4	2015-02-20 17:02:13.437762
338	158	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537C ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 48\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7da9c178-12ab-494d-9670-b1f494434758	2015-02-20 17:02:13.495348
339	159	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 48\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	18d23355-ae86-4151-ad24-e7a2d6161b7b	2015-02-20 17:02:13.559836
340	160	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537N ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 36\ndisease_id: 11\nsource_id: 109\nvariant_id: 49\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	efb4eef7-11bf-41b8-9691-14d50da4796d	2015-02-20 17:02:13.629696
341	161	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537N ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 49\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b4dbd515-dd2e-4445-b7ba-cde800cadf28	2015-02-20 17:02:13.683012
342	162	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 49\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	654649f5-327d-42f6-a906-144c376f5633	2015-02-20 17:02:13.781723
343	163	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537S ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 36\ndisease_id: 11\nsource_id: 109\nvariant_id: 50\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	09c02a21-a2ba-4928-b66e-3ec29c73e4ca	2015-02-20 17:02:13.846321
344	164	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537S ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 50\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8a3ff83b-8c10-4537-bdc6-ff373de198a0	2015-02-20 17:02:13.908153
345	165	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 50\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	96c4e52d-8fbe-43d8-a533-c3786549b258	2015-02-20 17:02:13.976338
346	166	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib\n  treatment resulted in necrosis and shrinkage of the primary and metastatic sites\n  and overall stable disease. However, this response did not meet criteria for RECIST\n  partial response.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 22\nsource_id: 111\nvariant_id: 56\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	04b580d6-d23e-46c7-9b01-467510df60ef	2015-02-20 17:02:14.054025
347	167	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be recurrent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 39\ndisease_id: 22\nsource_id: 111\nvariant_id: 59\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9dc6b2ca-ae49-4e91-8d22-45a803200389	2015-02-20 17:02:14.12815
348	168	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be recurrent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 22\nsource_id: 111\nvariant_id: 59\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	07a3da38-5065-4cf4-98b2-cd2695908f73	2015-02-20 17:02:14.181004
349	169	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6843f709-cf94-43e6-899b-45e1991a230a	2015-02-20 17:02:14.270776
350	170	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated\n  with worse overall survival compared to those without DNMT3A mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 60\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a7829a05-d3c8-4098-8d84-69774cdddf2e	2015-02-20 17:02:14.33617
351	171	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD mutation without NPM1 was associated with increased relapse risk and\n  reduced overall survival in young adult AML patients (median age 43)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0d3b478a-9d8d-4769-8c30-a068fc6331db	2015-02-20 17:02:14.402687
352	172	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 40\ndisease_id: 1\nsource_id: 113\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c04c57bb-1ed0-441b-9db9-19a4047a41d2	2015-02-20 17:02:14.475633
353	173	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  FLT3-ITD mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 41\ndisease_id: 1\nsource_id: 114\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1dc54fb8-6aff-4bfa-9b6c-64ddbfa73042	2015-02-20 17:02:14.541248
354	174	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 1\nsource_id: 113\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d2225f4a-367e-4d90-b18d-9fe1c826c449	2015-02-20 17:02:14.599967
355	175	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Increasing level of FLT3-ITD mutant correlated with reduced disease-free and\n  overall survival as well as increased relapse rate\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8d39b9ac-dc8e-464d-a3ef-8a1775b33a21	2015-02-20 17:02:14.665878
356	176	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD duplication length did not effect prognosis in patients with an NPM1\n  mutation\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d5a0a028-38a9-487b-83d8-90f7e358f645	2015-02-20 17:02:14.728189
357	177	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Size of the FLT3-ITD mutant duplication had no impact on overall survival or\n  relapse rate\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 60\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2f4a6910-1e4f-4af5-902c-c513d7eb5e0f	2015-02-20 17:02:14.812914
359	179	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	138b08cf-0e23-4a7f-8623-d0cc726ab13b	2015-02-20 17:02:14.932975
360	180	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free\n  and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a6806fee-7e06-4591-8be1-76bb169b7857	2015-02-20 17:02:14.990912
361	181	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Meta-analysis of studies involving cytogentically normal younger (<60) patients\n  showed reduced overall and relapse-free survival for patients with FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 116\nvariant_id: 60\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1996018d-57bd-48cd-952a-5109cea51f26	2015-02-20 17:02:15.053715
362	182	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared\n  to wild type\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 117\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8e233818-82c7-45bb-86dd-f63a0255c0df	2015-02-20 17:02:15.122328
363	183	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeats, but not\n  in wild type or D835 mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 1\nsource_id: 118\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c30212de-6044-4a12-a4da-acc17398819b	2015-02-20 17:02:15.196329
364	184	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML\n  patients with FLT3 mutations\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 42\ndisease_id: 1\nsource_id: 119\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4c1c5d9c-a033-4b7a-a5da-66a7a4b7d12a	2015-02-20 17:02:15.287448
365	185	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with\n  FLT3 internal tandem repeat\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 43\ndisease_id: 1\nsource_id: 120\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a3186069-8d72-44dd-85c9-6aae0e0160f9	2015-02-20 17:02:15.348084
366	186	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor\n  survival and inferior disease free survival in patients with AML.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 121\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1e2fc7aa-6698-4fb9-b808-ed5b08acd919	2015-02-20 17:02:15.411667
367	187	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with internal tandem repeat of FLT3 has worse overall survival\n  than patients with wild type FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 117\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3324cd0b-9cff-4b6e-b668-c7750266219b	2015-02-20 17:02:15.472965
368	188	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 40\ndisease_id: 1\nsource_id: 113\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	470d3b4e-ac74-46aa-9666-6221e6bce712	2015-02-20 17:02:15.532333
369	189	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 1\nsource_id: 113\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4bd99c03-4d6f-4c0b-9a9e-10c109b1a8f5	2015-02-20 17:02:15.586905
370	190	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 122\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1f6ddd58-e26d-4b54-a62f-c84542341684	2015-02-20 17:02:15.659966
371	191	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were not significantly different in AML patients with FLT3 D835 mutations\n  compared to wild type\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 117\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4c9b427f-1098-4a94-834c-d485281af7dc	2015-02-20 17:02:15.72289
372	192	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated\n  with sorafenib resistance\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 1\nsource_id: 123\nvariant_id: 61\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c835ab6f-3ea9-4c1d-94b0-c02e809b9bd5	2015-02-20 17:02:15.812102
373	193	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeat mutations,\n  but not in patients with wild type or D835 mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 1\nsource_id: 118\nvariant_id: 61\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e4d115a9-3603-4f96-aa09-e003f2724cea	2015-02-20 17:02:15.880171
374	194	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML\n  patients with FLT3 mutations\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 42\ndisease_id: 1\nsource_id: 119\nvariant_id: 61\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4e121d0f-2fed-427a-a53c-a0bff9cb4397	2015-02-20 17:02:15.937545
375	195	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with D835 mutation of FLT3 did not have worse overall survival\n  than patients with wild type FLT3\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 117\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	dbaac649-bfdf-40e8-b6f5-10dde0ebbb54	2015-02-20 17:02:15.99766
376	196	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients, FLT3-TKD mutation was associated with poorer disease free survival\n  compared to patients with wild type FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 124\nvariant_id: 61\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	de23dfe1-f4ac-45cf-a13d-d4f79999ffb7	2015-02-20 17:02:16.05603
377	197	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 44\ndisease_id: 23\nsource_id: 125\nvariant_id: 62\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1bd3936d-5ee7-4344-83cc-e16497c2101b	2015-02-20 17:02:16.125023
378	198	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 45\ndisease_id: 23\nsource_id: 125\nvariant_id: 62\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8971441c-bfb6-4ea9-983b-039d460d8776	2015-02-20 17:02:16.188496
379	199	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 126\nvariant_id: 63\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d49b0024-7596-44eb-b82a-2a7ef2b19118	2015-02-20 17:02:16.271758
380	200	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In leukemia patients, IDH1 R132 mutation is associated with adults and normal\n  karyotype.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 127\nvariant_id: 63\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1641774d-8bf4-444a-aead-79114d3ea644	2015-02-20 17:02:16.331004
381	201	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The complete remission and overall survival rates in patients with IDH1 R132\n  mutation is not significantly different from those who do not have this mutation\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 128\nvariant_id: 63\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4ea7764a-33fd-488d-be64-f5c2860c803c	2015-02-20 17:02:16.389735
382	202	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic\n  for better survival compared to patients who harbor wild type mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 24\nsource_id: 129\nvariant_id: 63\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0c8bb032-6eca-4fba-b014-6eef54987296	2015-02-20 17:02:16.463966
383	203	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with glioblastoma, those harboring IDH1 R132 mutation has higher\n  overall survival compared to those who do not have IDH1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 25\nsource_id: 130\nvariant_id: 63\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4c23a015-b4b4-4119-9485-4240010b1880	2015-02-20 17:02:16.523934
384	204	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated\n  with worse overall survival\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 26\nsource_id: 131\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7576f085-af5f-4b1c-8fa5-d150f9f50665	2015-02-20 17:02:16.58842
385	205	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation is associated with patients of older age, high platelet count\n  during diagnosis, cytogenic normalcy and NPM1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 132\nvariant_id: 64\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5420a8f7-3454-4856-9d37-f6adc4675225	2015-02-20 17:02:16.658259
386	206	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 132\nvariant_id: 64\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a4c3b62b-3e55-428d-bc3b-8987ba85e6e0	2015-02-20 17:02:16.74643
387	207	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 132\nvariant_id: 65\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	19a0e4f8-1fc2-404a-a5e6-874cafb02dc9	2015-02-20 17:02:16.823397
388	208	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 132\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1fe659fd-fa36-4a7f-8c0c-e2c9000d9db3	2015-02-20 17:02:16.885816
389	209	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with IDH2 R140K mutation have improved overall survival compared\n  to those with wild-type IDH2\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 133\nvariant_id: 67\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9177753b-eab2-4925-8f94-7ffe11bd7079	2015-02-20 17:02:16.950555
390	210	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact\n  overall survival or disease free survival.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 134\nvariant_id: 67\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1445909b-d135-4016-ad13-03677069fbd6	2015-02-20 17:02:17.012679
391	211	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R140Q/L have event free survival and\n  overall survival similar to those with wild-type IDH2\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 135\nvariant_id: 67\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6b1107f7-9d0d-42f6-a8c0-00deed326354	2015-02-20 17:02:17.080039
392	212	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: R140Q mutation in IDH2 does not have prognostic value in patients with MDS.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 27\nsource_id: 136\nvariant_id: 67\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d93c018c-9d32-42c2-811c-d60bdab826c9	2015-02-20 17:02:17.149621
393	213	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L\n  do not confer prognostic value (overall survival)\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 137\nvariant_id: 67\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a6df5b2d-40be-42b1-9f69-cc0f818a6d75	2015-02-20 17:02:17.240348
394	214	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R172K have event free survival and\n  overall survival similar to those with wild-type IDH2.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 135\nvariant_id: 68\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bfb6aa66-4536-4ec8-92a3-9667d9f90e21	2015-02-20 17:02:17.30784
395	215	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with an IDH2 R172K mutation have worse overall survival compared\n  to those with wild-type IDH2\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 133\nvariant_id: 68\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	668396e8-24ca-4f29-b3d6-716b6cbdc5ac	2015-02-20 17:02:17.372435
396	216	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutations such as R172K\n  do not confer prognostic value (overall survival)\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 137\nvariant_id: 68\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8359fbc6-4f59-47a7-8760-aa894d5a7766	2015-02-20 17:02:17.443338
397	217	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F not associatted wth  lymphoid leukemia.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 2\nsource_id: 138\nvariant_id: 69\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4f5ce49e-bc37-454f-a514-afedfe4783e4	2015-02-20 17:02:17.510411
398	218	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical\n  CML).\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 2\nsource_id: 138\nvariant_id: 69\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bd9729d5-0dba-4d46-b0ba-5c2ca5da1581	2015-02-20 17:02:17.570944
399	219	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 46\ndisease_id: 28\nsource_id: 139\nvariant_id: 69\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ce6ddd51-bbde-4ce1-8fb4-bba24499f5b3	2015-02-20 17:02:17.637821
400	220	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or\n  CLL).\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 29\nsource_id: 138\nvariant_id: 69\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	af1db81b-5b00-4f0a-b1e2-fc77308d559c	2015-02-20 17:02:17.787425
401	221	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: rs10974944 SNP in the Jak2 locus is associated with increased predisposition\n  for JAK2 V617F mutation and its associated cancer\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 30\nsource_id: 140\nvariant_id: 69\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	15f9ce3d-5c08-4735-82b5-f79a09a906b8	2015-02-20 17:02:17.852678
402	222	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction\n  in the percentage of cells harboring JAK2 V617F.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 47\ndisease_id: 31\nsource_id: 141\nvariant_id: 69\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b5be5f1b-0acb-43c8-af3b-23a4c0f4f09e	2015-02-20 17:02:17.923959
403	223	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In acute myloid leukemia patients, D816 mutation is associated with earlier\n  relapse and poorer prognosis than wildtype KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 32\nsource_id: 142\nvariant_id: 70\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8810e69a-432e-44da-b28b-26bdee20c5aa	2015-02-20 17:02:17.986359
404	224	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between wildtype KIT or KIT mutations in\n  exon 9 or 11 in survival among GIST patients.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 143\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ff89b7b7-5fcf-4860-b965-5586bb53f549	2015-02-20 17:02:18.060392
405	225	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 144\nvariant_id: 71\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c23c5712-fb3b-4afe-8662-9649c0e9b471	2015-02-20 17:02:18.142933
406	226	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations were identified in morphologically benign, incidentally discovered\n  GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating\n  is not a prognostic marker for GIST\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 145\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	fe0c32f1-05cc-42b2-8c16-e0f03ea3437a	2015-02-20 17:02:18.210465
407	227	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with worse overall and cause-specific prognosis in\n  patients with GIST compared to patients with wildtype KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 144\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1f9c2f32-3311-4585-9d53-f6e035e03271	2015-02-20 17:02:18.276273
408	228	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 146\nvariant_id: 71\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ceb55ff0-359c-4564-8448-a3728fa16092	2015-02-20 17:02:18.332337
409	229	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 8\nsource_id: 147\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	93b6d7c2-a36f-4921-bbf3-4243349bd3b3	2015-02-20 17:02:18.395913
410	230	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 8\nsource_id: 147\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ee714dde-ff05-48aa-b925-e1c8d9c68612	2015-02-20 17:02:18.460708
411	231	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST patients with exon 14 KIT mutations had reduced overall survival compared\n  to patients wiltype for KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 143\nvariant_id: 73\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	75d66d67-140d-4694-a264-b976ec45ddc0	2015-02-20 17:02:18.524654
412	232	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 144\nvariant_id: 74\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7610e7f9-092c-483e-9f7d-d0c01fc8a9b6	2015-02-20 17:02:18.5775
413	233	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 8\nsource_id: 147\nvariant_id: 74\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	928ec95c-45d5-4de4-9306-5c42dd4ab14d	2015-02-20 17:02:18.65
414	234	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between wildtype KIT or KIT mutations in\n  exon 9 or 11 in survival among GIST patients.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 143\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	df878d45-f657-41dd-a9c2-e1ff714bc04a	2015-02-20 17:02:18.709256
415	235	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 146\nvariant_id: 75\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	64e7c2e5-b5fc-43e3-8cf8-e923f9417c22	2015-02-20 17:02:18.77457
416	236	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 8\nsource_id: 148\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ed5a049b-75a3-4c47-b3b4-240d2d7e1ec3	2015-02-20 17:02:18.845364
417	237	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 8\nsource_id: 148\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e8bd0d8b-53d2-487d-ac61-48b4a8a88e25	2015-02-20 17:02:18.903008
418	238	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 8\nsource_id: 148\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	dd6e4040-f416-419a-a262-77d250eaa759	2015-02-20 17:02:18.95747
419	239	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive\n  to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation\n  of KIT but did not alter the binding affinity of dasatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 5\ndisease_id: 8\nsource_id: 148\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a007d4f2-768f-4efe-b7e3-a0bb5f602d72	2015-02-20 17:02:19.023167
420	240	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 5\ndisease_id: 4\nsource_id: 149\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	04bf57e2-6e86-4cbb-9e22-4258b4f1e2e9	2015-02-20 17:02:19.090483
421	241	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 4\nsource_id: 149\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	607523e9-b642-414b-9833-8ae237f6101d	2015-02-20 17:02:19.135389
422	242	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 4\nsource_id: 149\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7ae021b5-be96-47f7-94fe-326a8081fe45	2015-02-20 17:02:19.186679
423	243	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT V654A results in imatinib resistance in GIST patient-derived cell lines.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 33\nsource_id: 150\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3e490973-8acf-4534-ac07-0fbe1641781d	2015-02-20 17:02:19.254052
424	244	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective\n  for those that are refractory to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 48\ndisease_id: 33\nsource_id: 151\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	94a40672-bfb6-46fd-b5c3-81a0a17d92b4	2015-02-20 17:02:19.341044
425	245	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer who harbor KRAS mutation have low response rate\n  to cetuximab\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 152\nvariant_id: 78\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3ee3471e-6aa7-4382-bda1-693fd2db38e8	2015-02-20 17:02:19.419572
426	246	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 49\ndisease_id: 7\nsource_id: 153\nvariant_id: 79\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8a17625b-cacb-4439-a6ba-3e410a679990	2015-02-20 17:02:19.493716
427	247	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 30\ndisease_id: 4\nsource_id: 154\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5cee9c3b-4eda-4f62-ac62-9b00738d1473	2015-02-20 17:02:19.564083
428	248	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 4\nsource_id: 154\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	80466f98-a62e-470d-8c02-cdf6fd418ea4	2015-02-20 17:02:19.620031
429	249	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12\n  mutation did not impact overall or disease free survival.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 9\nsource_id: 155\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	97aeb6ae-e94c-4fe1-b759-31b6d0622b65	2015-02-20 17:02:19.684822
430	250	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 34\nsource_id: 156\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3ee0953f-e9a0-4476-9d69-eeb5a2c37450	2015-02-20 17:02:19.755448
431	251	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 35\nsource_id: 157\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9e965311-fb14-4cdf-8003-0f4211184819	2015-02-20 17:02:19.820999
432	252	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung cancer patients are not associated with smoking history,\n  age or gender\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 158\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ca8d1040-36a2-4a0a-8c52-be83ec3ab2cf	2015-02-20 17:02:19.887232
433	253	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no strong association between KRAS mutation status and overall or progression\n  free survival in patients with NSCLC\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 159\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	22317d30-11c4-4918-aba7-0929c5e79541	2015-02-20 17:02:19.956029
434	254	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A meta-analysis showed KRAS mutation is associated with worse outcome in patients\n  with NSCLC\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 160\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e8c9da6a-86a3-44bb-a307-667c7321ead8	2015-02-20 17:02:20.021944
435	255	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with lack of response to cetuximab\n  in patients with advanced colorectal cancer\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 7\nsource_id: 161\nvariant_id: 81\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5165fa72-f29c-4f43-afbb-e431bf8efaef	2015-02-20 17:02:20.086082
436	256	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations\n  (primarily G12/G13) had lower response and disease control rates and shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype KRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 81\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9efa43b6-587b-4a11-9b23-4c3b9636c975	2015-02-20 17:02:20.144978
437	257	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with reduced progression-free and\n  overall survival in patients with advanced colorectal cancer\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 161\nvariant_id: 81\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	457f1f2e-85f7-4b9d-8335-fa67de0140b3	2015-02-20 17:02:20.207773
440	260	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 4\nsource_id: 163\nvariant_id: 83\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c87fbac4-49ea-452f-8283-c1df17708b80	2015-02-20 17:02:20.415207
441	261	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 50\ndisease_id: 4\nsource_id: 164\nvariant_id: 83\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	043d22e8-495a-4cdb-97ef-d53396cbc297	2015-02-20 17:02:20.486751
442	262	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 51\ndisease_id: 4\nsource_id: 164\nvariant_id: 83\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e8f8a78b-7abe-42c6-92f7-64e83d58f5bc	2015-02-20 17:02:20.548636
443	263	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 34\nsource_id: 156\nvariant_id: 84\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2ff387d4-82fd-4d83-a253-d04ef50a34e6	2015-02-20 17:02:20.597336
444	264	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 36\nsource_id: 157\nvariant_id: 84\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c2e4d1c9-0911-4164-85ce-0d5e739f6e34	2015-02-20 17:02:20.652753
445	265	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung cancer patients are not associated with smoking history,\n  age or gender\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 158\nvariant_id: 84\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6003af62-7a97-436c-a4ea-de44b1a81e15	2015-02-20 17:02:20.714215
446	266	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft model\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 9\nsource_id: 165\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ca2f58be-69a3-4986-869d-165b364e6e08	2015-02-20 17:02:20.772925
447	267	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft model\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 9\nsource_id: 165\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	16bf4245-fadb-475a-8576-125e9e31efd2	2015-02-20 17:02:20.824565
448	268	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 9\nsource_id: 165\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2ca01972-be94-49b0-8659-a50964a71793	2015-02-20 17:02:20.886619
449	269	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 9\nsource_id: 165\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a2948cd8-fd8f-42f5-aa79-c7dc14dd797f	2015-02-20 17:02:20.990901
450	270	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition\n  by about 5 fold.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 23\ndisease_id: 8\nsource_id: 166\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	609a0055-2c20-428f-b963-8d505b097e35	2015-02-20 17:02:21.055769
451	271	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 23\ndisease_id: 8\nsource_id: 167\nvariant_id: 87\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d1bfe204-4511-4903-8e4e-2a92e0d4a0a3	2015-02-20 17:02:21.125908
452	272	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 90\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	520df45c-24bd-457a-a9d4-1e4a07b7f49e	2015-02-20 17:02:21.184046
453	273	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 27\ndisease_id: 19\nsource_id: 90\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	79533747-6520-4e33-8d4e-a3d98e8aacf4	2015-02-20 17:02:21.236105
454	274	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted\n  from temozolomide. This benefit was also methylation status dependent, as those\n  without methylation did not see increased survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 22\ndisease_id: 37\nsource_id: 168\nvariant_id: 89\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0658d6ce-e292-4f34-8f12-217021388f64	2015-02-20 17:02:21.311845
455	275	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation was associated with increased overall and disease-free\n  survival as well as tumor regression in patients with gliomas.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 37\nsource_id: 169\nvariant_id: 89\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5b646104-7433-466d-8fa2-0f04b556a2bb	2015-02-20 17:02:21.369929
456	276	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor,\n  such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may\n  be useful in treating patients with initial resistance to alkylating agents.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 52\ndisease_id: 37\nsource_id: 169\nvariant_id: 89\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	077dc3ab-b5cc-4657-8daa-07f9ae07ff82	2015-02-20 17:02:21.430966
457	277	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation has been shown to confer sensitivity to alkylating\n  agents, such as carmustine, in glioblastoma cells.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 53\ndisease_id: 37\nsource_id: 169\nvariant_id: 89\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5a920ae4-69bd-4e50-ad5e-e8d119cdd7c0	2015-02-20 17:02:21.499959
458	278	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	08cdc33f-c254-4801-8409-e1189b54b1ea	2015-02-20 17:02:21.558664
459	279	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid\n  leukemia.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 170\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	61d998c4-b0e8-4f37-b828-a90f4121d0f4	2015-02-20 17:02:21.622631
460	280	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype\n  (0/55).\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 171\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9323e040-a8a5-4db5-99a8-66f8c12df98a	2015-02-20 17:02:21.676944
461	281	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid\n  leukemia.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 170\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1155f7fc-f615-4b0a-9d2b-50fd5ca67050	2015-02-20 17:02:21.738246
462	282	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid\n  leukemia\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 171\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	be9486b6-1758-417c-8589-68f3b744d8d9	2015-02-20 17:02:21.804882
463	283	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with normal karyotype in older (>60) patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 172\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	69d75b9f-842b-43f0-b471-430919251ec4	2015-02-20 17:02:21.861702
464	284	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cf24c45a-2329-444b-902b-a34d062dfe8c	2015-02-20 17:02:21.929852
465	285	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  NPM1 mutation regardless of FLT3-ITD status\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 41\ndisease_id: 1\nsource_id: 114\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0a954a9b-9c94-4103-81d0-ff0266f0be07	2015-02-20 17:02:21.985828
466	286	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse\n  in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction\n  in NPM1 levels following first line therapy (N=15)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 173\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	71c00a7a-20ad-4a16-a90d-b694b85f1f73	2015-02-20 17:02:22.054685
467	287	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and\n  increased overall survival in young adult acute myeloid leukemia patients (median\n  age 43)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a2554091-3ae4-42aa-b32b-b9360bbd5e5e	2015-02-20 17:02:22.124548
468	288	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and\n  FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 54\ndisease_id: 1\nsource_id: 174\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1052ec93-1dde-4f0e-bb96-c7e9978d0ea5	2015-02-20 17:02:22.189955
469	289	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and\n  FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 41\ndisease_id: 1\nsource_id: 174\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	48dc9adf-930a-4aea-86f5-1cd52399c954	2015-02-20 17:02:22.245063
470	290	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations were associated with improved complete remission\n  rates as well as overall and disease free survival when all ages and karyotype were\n  analyzed; however, when only normal karyotype patients were analyzed, only complete\n  remission rates were improved\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 172\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0dbc271e-e5b1-4abb-9dac-2d8a3d87138a	2015-02-20 17:02:22.303544
471	291	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment improved overall and relapse-free survival in older (>60) patients\n  with mutant NPM1 and without FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 41\ndisease_id: 1\nsource_id: 172\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6b408dce-15c7-4987-8fee-3bc8f7ce05f9	2015-02-20 17:02:22.368415
472	292	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young patients (18-60) with NPM1 mutations had improved overall survival following\n  high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However,\n  the adjusted p-value for multiple testing is 0.11.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 28\ndisease_id: 1\nsource_id: 175\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	32416bb0-783c-42ed-b567-4b4be8f7a287	2015-02-20 17:02:22.429186
473	293	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: For patients with NPM1 mutation that achieved CR following induction therapy,\n  relapse-free survival was improved following treatment with valproic acid\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 55\ndisease_id: 1\nsource_id: 176\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	fafca4fa-5cea-4c9f-97a2-f196e9e14115	2015-02-20 17:02:22.486368
474	294	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 expression was significantly increased in patients with NPM1 mutation with\n  or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 177\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e70809f6-4029-4824-9a18-0d52b2f68d5c	2015-02-20 17:02:22.597316
475	295	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 and CD123 expression was significantly increased in patients with NPM1\n  mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33\n  and anti-CD123 therapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 177\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cef938c2-bf44-4c13-a1d0-457e36555afd	2015-02-20 17:02:22.653957
476	296	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and event-free survival was longer for\n  normal karyotype AML patients with Exon 12 NPM1 mutations.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 170\nvariant_id: 90\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	68037b04-a959-45a5-87bf-440d934cced4	2015-02-20 17:02:22.712989
477	297	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and\n  without FLT3-ITD have increased overall survival, relapse-free survival and response\n  to induction chemotherapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 178\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e25d236f-ef1f-44e3-98bc-4c071b447a52	2015-02-20 17:02:22.774985
478	298	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with increased complete remission rates as well\n  as longer overall disease free survival in normal karyotype AML patients >59 years\n  old, particularly in those >69 years old\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 179\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	959becbe-df7f-4455-bc45-971759fc4430	2015-02-20 17:02:22.845745
479	299	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with higher complete remission rates, lower cumulative\n  incidence of relapse and higher overall survival in intermediate risk AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 180\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	be35a749-9028-4383-a233-d47b5a69325f	2015-02-20 17:02:22.907778
480	300	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and both disease-free and overall survival\n  were longer for patients with Exon 12 NPM1 mutations.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 171\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7f3fe366-769c-4fb7-a782-25de4e179aec	2015-02-20 17:02:22.964554
481	301	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 122\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4716a913-1a0e-459a-95be-df98207fe296	2015-02-20 17:02:23.024951
482	302	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved\n  overall and relapse-free survival, complete remission rates and remission duration\n  compared to all other combinations of mutants\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 181\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	84f65bee-5eea-406a-838e-96a29274c12c	2015-02-20 17:02:23.098995
483	303	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with increased overall, event-free and relapse-free\n  survival and remission after induction therapy in the entire cohort and in the cytogenetically\n  normal subset of patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 182\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	21e51dee-e4ef-4fb3-ac50-3ffc39d334f4	2015-02-20 17:02:23.163984
484	304	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Intermediate risk patients with mutant NPM1 had improved overall survival with\n  the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1\n  and IDH2\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 175\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e0083858-d5f2-4bd5-8092-27f441add3e6	2015-02-20 17:02:23.232321
485	305	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was significantly associated with complete remission\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 88\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b80bb015-8b40-4236-b4ea-8cd9941206c3	2015-02-20 17:02:23.291518
486	306	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 126\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	aa248541-3f6f-4262-8b89-f9c31b753d41	2015-02-20 17:02:23.348225
487	307	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coocurrence of FLT3-ITD mutations significantly reduced overall survival of\n  normal karyotype AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 171\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3e54c768-bfcd-4d9a-b9b9-3f4ed309e6d0	2015-02-20 17:02:23.411492
488	308	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	062b3cfa-71a0-493c-a3f7-1ecf3da386cb	2015-02-20 17:02:23.473742
489	309	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5b64c4c4-e0a5-4873-b298-0fcfacdf1bc6	2015-02-20 17:02:23.525631
490	310	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TET2 mutation reduces overall survival of normal karyotype patients with either\n  an NPM1 mutation or NPM1 mutation without FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 183\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0080e72c-6b77-46de-b3bd-335622121bfb	2015-02-20 17:02:23.587418
491	311	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival\n  and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate\n  risk AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 180\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f6e4067e-c62b-417f-90f9-f833c0b1c0d2	2015-02-20 17:02:23.648441
492	312	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: '"AML with mutated NPM1" is a provisional entity in WHO classification of acute\n  myeloid leukemia (AML). This mutation should be tested for in clinical trials and\n  is recommended for testing in patients with cytogentically normal AML.'\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 184\nvariant_id: 90\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bee0043f-a1b1-4606-ab73-35242f6a36ea	2015-02-20 17:02:23.71479
493	313	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No NPM1 mutations were identified in patients with favorable risk cytogenetics\n  (79/215 patients)\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 91\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a837b13a-2d05-41c5-8c23-e32680bada9f	2015-02-20 17:02:23.782121
494	314	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with intermediate risk cytogenetics (including\n  normal karyotype)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 91\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b3e7f338-f449-4ced-81b6-b57b85c3db79	2015-02-20 17:02:23.853714
495	315	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 54\ndisease_id: 1\nsource_id: 174\nvariant_id: 91\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	943be13c-c00c-4b93-9062-d57fc56691df	2015-02-20 17:02:23.924794
496	316	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected\n  into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM\n  in AML patients was associated with good response to induction therapy.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 185\nvariant_id: 91\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4e00627e-7e52-49f5-8dca-3db2c219dcde	2015-02-20 17:02:24.006672
497	317	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 186\nvariant_id: 92\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	928bf557-3749-4bc9-a0f3-1062bb2c284a	2015-02-20 17:02:24.080129
498	318	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of NRAS mutation in AML patients does not impact diease prognosis\n  (resistant disease, disease-free survival, complete remission rate, relapse rate,\n  induction death)\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 187\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4bebc4ba-700f-461e-9739-bf54e26298ce	2015-02-20 17:02:24.190241
499	319	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 188\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bb85a780-4559-4f82-8316-0899b5084d37	2015-02-20 17:02:24.259258
500	320	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 186\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1ee2e15b-1ea1-474f-b72c-c0553f4c19c2	2015-02-20 17:02:24.322731
501	321	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 7\nsource_id: 188\nvariant_id: 94\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	897f3dbb-e63f-4bec-9e16-913dd55d2c8b	2015-02-20 17:02:24.386983
502	322	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 7\nsource_id: 188\nvariant_id: 95\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6e95b8f1-b311-49f9-ba3f-3cf3650e9f77	2015-02-20 17:02:24.443244
503	323	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NRAS mutation status does not impact overall, event-free or disease-free survival\n  in patients with AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 189\nvariant_id: 96\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	61cff5c9-58a2-4b3b-881c-f362ad050a43	2015-02-20 17:02:24.513048
504	324	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Likely due to increased reliance of mutant NRAS on HSP90 for stabilization,\n  inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic\n  malignant melanoma with an NRAS G13D mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 56\ndisease_id: 8\nsource_id: 190\nvariant_id: 97\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a6db65c8-eb42-4cd0-ac49-0724111a768a	2015-02-20 17:02:24.580135
505	325	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation\n  (primarily Q61) have a significantly lower response rate to cetuximab than patients\n  wildtype for NRAS.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 98\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e6de7c78-6710-487e-a33a-893edb6e3a1c	2015-02-20 17:02:24.647521
506	326	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status\n  (primarily Q61) was not informative for disease control rate, progression free survival\n  or overall survival following cetuximab plus chemotherapy.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 98\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3d71df65-ea03-4afa-a319-80948d0c061a	2015-02-20 17:02:24.713856
507	327	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma associated with NRAS Q61 mutation was more often associated with those\n  at the extremity than those at the trunk\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 191\nvariant_id: 98\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a672efa3-9e9a-4a86-92ee-2d89bde383a4	2015-02-20 17:02:24.780587
508	328	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 19\ndisease_id: 8\nsource_id: 192\nvariant_id: 98\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	517d93d3-425b-4529-9a74-1431701cc244	2015-02-20 17:02:24.842048
509	329	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted\n  in disease free survival of 14 months\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 57\ndisease_id: 8\nsource_id: 193\nvariant_id: 99\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	13c203c9-6ed9-4932-bfed-e9048fa09f30	2015-02-20 17:02:24.908114
510	330	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with Q61R mutation, treatment with temozolomide resulted\n  in overall survival of 16 months\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 57\ndisease_id: 8\nsource_id: 193\nvariant_id: 100\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0741240b-10ef-4b3b-97c1-4f84c2584a5a	2015-02-20 17:02:24.966793
511	331	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 58\ndisease_id: 38\nsource_id: 194\nvariant_id: 101\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4fb12b69-787d-4aae-9c4a-7f0b1d2451ed	2015-02-20 17:02:25.037555
512	332	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST tumors harboring PDGFRA D842V mutation are likely to be benign\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 38\nsource_id: 195\nvariant_id: 102\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c91766d4-c7c4-4c1d-9b47-19efc1cac6c7	2015-02-20 17:02:25.11484
513	333	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib,\n  double mutation of V561D and D842V mutants are resistant to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 38\nsource_id: 150\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9ad9175f-1f10-4c0f-85e5-f94fc46ed9b8	2015-02-20 17:02:25.177654
514	334	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST cancer with D842V mutation is resistant to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 38\nsource_id: 196\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0b82f25d-6e45-4d0f-9c7f-a8a78a408fc3	2015-02-20 17:02:25.244006
515	335	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 58\ndisease_id: 38\nsource_id: 194\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c54ecb75-8f4a-4b5f-ae55-31b0fd65ae84	2015-02-20 17:02:25.315542
516	336	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 58\ndisease_id: 38\nsource_id: 194\nvariant_id: 103\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c553c8bb-2a87-4013-b0c7-062fafd9ab71	2015-02-20 17:02:25.385205
517	337	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib\n  resistance, crenolanib was significantly more potent at inhibiting kinase activity\n  than imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 58\ndisease_id: 38\nsource_id: 194\nvariant_id: 104\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c32cb01a-8984-440e-8871-7311931e543b	2015-02-20 17:02:25.462462
518	338	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 58\ndisease_id: 38\nsource_id: 194\nvariant_id: 105\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9690c02d-26a7-41eb-9036-d21427daa886	2015-02-20 17:02:25.529985
519	339	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 59\ndisease_id: 11\nsource_id: 197\nvariant_id: 106\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a45b2620-dc33-4ee1-848e-bf4cc9544904	2015-02-20 17:02:25.59582
520	340	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 60\ndisease_id: 11\nsource_id: 198\nvariant_id: 106\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bd9625fb-48f6-4ad5-a0e2-65441b5f9c4e	2015-02-20 17:02:25.654004
521	341	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 49\ndisease_id: 7\nsource_id: 153\nvariant_id: 106\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e9bbb5e1-2096-4d5d-82be-e7524b9dc6a2	2015-02-20 17:02:25.75793
522	342	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 7\nsource_id: 199\nvariant_id: 106\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c39b1e01-d31e-4904-bcb8-54ec1e8e6296	2015-02-20 17:02:25.823466
523	343	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 12\ndisease_id: 7\nsource_id: 199\nvariant_id: 106\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3cb9f038-9366-42ac-9ed4-3c79597050ed	2015-02-20 17:02:25.874705
524	344	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E542K) or exon 20 mutations were associated\n  with poorer survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 199\nvariant_id: 106\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c2fa1d35-4daa-494a-aa11-011928b2dd09	2015-02-20 17:02:25.943592
525	345	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 200\nvariant_id: 106\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	52042e64-bff3-40e9-986d-94a27d5ca5e8	2015-02-20 17:02:26.015318
526	346	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 59\ndisease_id: 11\nsource_id: 197\nvariant_id: 107\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	56cb39f4-c303-43f2-b6e5-f49e7a053314	2015-02-20 17:02:26.069878
527	347	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 60\ndisease_id: 11\nsource_id: 198\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	20e3e09e-f46f-4375-8f59-b2c5d627a2a3	2015-02-20 17:02:26.130221
528	348	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 49\ndisease_id: 7\nsource_id: 153\nvariant_id: 107\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6f8c0251-68bf-43ff-8c4f-a1b138b89e9d	2015-02-20 17:02:26.186948
529	349	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 7\nsource_id: 199\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ff8b3d14-3000-4aff-baf8-394898a882cc	2015-02-20 17:02:26.247793
530	350	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 12\ndisease_id: 7\nsource_id: 199\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	dd2fdd69-9b0d-49f2-93f5-92b33178e65e	2015-02-20 17:02:26.30782
531	351	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 199\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c10194b4-4d4a-4278-ba20-c3cb9c3d7cf1	2015-02-20 17:02:26.367237
532	352	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 200\nvariant_id: 107\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a999d19f-4b4e-4c03-a4d5-ea5913e80794	2015-02-20 17:02:26.431284
533	353	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA\n  mutations had lower response and disease control rates as well as shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype PIK3CA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 108\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e6d6638e-6b7c-47cb-89fb-693855af35ed	2015-02-20 17:02:26.489983
549	369	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong\n  repression of tyroid cancer cell growth.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 62\ndisease_id: 40\nsource_id: 206\nvariant_id: 116\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8c3639c6-c2e8-4106-9235-11ede0224e7e	2015-02-20 17:02:27.570253
534	354	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations\n  (Exon 9) had no significant impact on response rate, disease control rate, progression\n  free surivival or overall survival following cetuximab plus chemotherapy than those\n  with wildtype PIK3CA\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 109\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cc491fd8-6d27-4c9e-a722-4c16507698e5	2015-02-20 17:02:26.547135
535	355	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with H1047R mutation showed increased sensitivity to\n  CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 59\ndisease_id: 11\nsource_id: 197\nvariant_id: 110\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	dd81c5d8-e7fe-4ae1-bed0-af7483b63235	2015-02-20 17:02:26.604235
536	356	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 60\ndisease_id: 11\nsource_id: 198\nvariant_id: 110\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	549d091d-abad-4a78-b85a-5ffc4fe341a3	2015-02-20 17:02:26.679057
537	357	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 49\ndisease_id: 7\nsource_id: 153\nvariant_id: 110\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b9aa7f37-bf2c-4762-899a-c73dee7e5bfa	2015-02-20 17:02:26.745936
538	358	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 7\nsource_id: 199\nvariant_id: 110\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cd351996-cc24-48e0-9950-d047e0632ace	2015-02-20 17:02:26.804549
539	359	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 12\ndisease_id: 7\nsource_id: 199\nvariant_id: 110\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f9779759-73ff-43b0-9d06-310f54c74e22	2015-02-20 17:02:26.862512
540	360	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 199\nvariant_id: 110\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1a2487a1-e97c-4370-a24b-4e3031655279	2015-02-20 17:02:26.917113
541	361	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic\n  cells at pharmalogical concentrations.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 41\ndisease_id: 39\nsource_id: 201\nvariant_id: 111\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7888ea0f-ea6d-4d0a-beb3-2ed1a6be1062	2015-02-20 17:02:26.987519
542	362	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 20\nsource_id: 91\nvariant_id: 112\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	919db7c8-a774-4647-8299-6af1fdd70b6f	2015-02-20 17:02:27.049325
543	363	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse\n  to PI3K-mTOR inhibitors.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 3\nsource_id: 202\nvariant_id: 113\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1ec7f5f8-c76a-4958-8144-509ac273768d	2015-02-20 17:02:27.112981
544	364	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PTEN nonsense mutations, including R233*, have been shown to be inactivating\n  and loss-of-function, but do not have prognostic value in glioblastoma multiforme\n  patients.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 37\nsource_id: 203\nvariant_id: 113\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	63884306-3d75-4e49-bf7d-340e19d50518	2015-02-20 17:02:27.186192
545	365	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic\n  cells at pharmalogical concentrations.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 41\ndisease_id: 39\nsource_id: 201\nvariant_id: 114\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	73800fdd-8a61-4d10-9dc7-b53d2ca6001f	2015-02-20 17:02:27.311916
546	366	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 61\ndisease_id: 40\nsource_id: 204\nvariant_id: 115\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ca273db0-9302-4723-8ddd-8ed3c6a23e4b	2015-02-20 17:02:27.375685
547	367	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 61\ndisease_id: 40\nsource_id: 204\nvariant_id: 116\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a33ed4de-abf4-491f-b6e9-1ef465e716d5	2015-02-20 17:02:27.442728
548	368	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with medullary carcinoma, the presence of RET M918T mutation is\n  associated with increased probability of lymph node metastases.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 40\nsource_id: 205\nvariant_id: 116\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	02728481-e163-4ba4-ab5c-7256e1992377	2015-02-20 17:02:27.510958
550	370	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Compared to those who harbor a wild type RET, patients with RET M918T mutation\n  develop larger and more aggressive medullary thyroid cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 40\nsource_id: 207\nvariant_id: 116\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	705611e4-8337-4740-8f9c-870c550da1e5	2015-02-20 17:02:27.636765
551	371	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 208\nvariant_id: 117\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d77a3d14-f1bc-4a68-add7-f2efb43da91c	2015-02-20 17:02:27.701374
552	372	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 208\nvariant_id: 118\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5db186c0-89fb-4dbe-b2e9-393a1aaecc52	2015-02-20 17:02:27.762476
553	373	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin\n  than breast tumors with wild type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 63\ndisease_id: 11\nsource_id: 209\nvariant_id: 119\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	10298bbb-ab4c-458b-803f-a181e29c48fd	2015-02-20 17:02:27.832051
554	374	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R175H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 119\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cbcc7c60-028d-41e1-8881-f65736b2224e	2015-02-20 17:02:27.899028
555	375	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R248 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 120\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	22c0e39d-2da0-4f38-9b27-9e843ba411ea	2015-02-20 17:02:27.963923
556	376	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R248Q mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 120\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	691167c6-8598-4782-83f2-1e0dfdec1274	2015-02-20 17:02:28.026999
557	377	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring R248W mutation, the prognosis is worse than\n  any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 121\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	368bd902-d648-4916-856a-a449f1ef80fd	2015-02-20 17:02:28.086426
558	378	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved regions\n  such as R249 are prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 122\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e73244b6-5805-41ea-9d12-a4a2eac2f5b0	2015-02-20 17:02:28.14204
559	379	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved regions\n  such as R249 are prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 123\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7cae3cdf-7b78-46bb-8036-e7f82b276a5b	2015-02-20 17:02:28.213741
560	380	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 124\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	43a8ec07-5ed6-4d0f-a45c-2a8a11cb9bac	2015-02-20 17:02:28.284099
561	381	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273C mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 124\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d90d7c1a-c06c-44a8-bf02-41dba993d354	2015-02-20 17:02:28.342627
562	382	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 125\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	689b83a1-d6a8-47cd-970f-c625319ac865	2015-02-20 17:02:28.407032
563	383	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 125\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bca734e4-abf8-41e7-9abe-b6d279ca9292	2015-02-20 17:02:28.465922
564	384	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumor with R175H mutation are more responsive to doxorubicin than breast\n  tumors with wild type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 126\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f1391dd8-15b4-4e48-a9b7-7b6aa04c9c5c	2015-02-20 17:02:28.524991
565	385	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A patient with metastatic bladder cancer that responded well to the mTOR inhibitor\n  everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del,\n  p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations\n  also had mild responses to everolimus.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 64\ndisease_id: 41\nsource_id: 212\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3f7b14b2-51da-41f8-adb4-81592c74ae3b	2015-02-20 17:02:28.594183
566	386	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice\n  with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit\n  increased overall survival, increased apoptosis, reduced proliferation and reduced\n  tumor burden compared to Tsc1 wildtype mice.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 60\ndisease_id: 4\nsource_id: 213\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9392d5f5-ab79-4225-a737-6d699f830fe6	2015-02-20 17:02:28.676473
567	387	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 64\ndisease_id: 42\nsource_id: 214\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cd6a5b0a-5672-4bc3-9f17-d308f569f385	2015-02-20 17:02:28.796328
568	388	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 64\ndisease_id: 42\nsource_id: 212\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	becde31e-804a-4ced-92d3-41a677848120	2015-02-20 17:02:28.860914
569	389	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 215\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9d9695e4-0695-42d2-afbc-0a013c288c34	2015-02-20 17:02:28.93056
570	390	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 215\nvariant_id: 130\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9796418b-24e8-4a19-8244-71e095ab1848	2015-02-20 17:02:28.991955
571	391	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 215\nvariant_id: 130\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	dcd4384c-c458-4550-afa8-e56182eb2bf1	2015-02-20 17:02:29.063192
572	392	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R\n  is prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 216\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3e14ed13-6b74-4e0a-9824-0e0d0f92e97e	2015-02-20 17:02:29.123158
573	393	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 215\nvariant_id: 131\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	59596905-dfc9-4032-a70b-3be217d4666d	2015-02-20 17:02:29.179502
574	394	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 215\nvariant_id: 131\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d4d913ff-36b6-486b-a3ab-bb3507c5e285	2015-02-20 17:02:29.240907
575	395	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 215\nvariant_id: 131\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	66d94800-a11e-4f6f-84cd-65f33e9569e3	2015-02-20 17:02:29.305729
576	396	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, the presence of U2AF mutations such as\n  S34Y/F are prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 216\nvariant_id: 131\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	56a63b43-a925-41a6-b79d-ba5c00dfec6c	2015-02-20 17:02:29.375806
577	397	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 132\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	16d2bbff-2a83-4193-bb0e-2f2559663a7b	2015-02-20 17:02:29.442776
578	398	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 132\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f4c79515-4cff-449e-af62-1fc90baf13c6	2015-02-20 17:02:29.511743
579	399	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No differences in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 132\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b6b24ff1-0e2e-4395-ae86-0fb436575ccb	2015-02-20 17:02:29.576164
580	400	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 132\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a0a972f8-cd87-4f34-acd7-db802cc91c9e	2015-02-20 17:02:29.641375
581	401	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 132\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4c6ce05b-925a-4129-beaf-6dcc76beeba3	2015-02-20 17:02:29.713244
582	402	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 219\nvariant_id: 132\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	94b8eded-1d72-415b-9d6c-eab3d0759f7a	2015-02-20 17:02:29.774808
583	403	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 132\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2d422c0a-aa9a-429d-8011-c0f53ff9ac52	2015-02-20 17:02:29.834565
584	404	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutations in WT1 were associated with a worse overall prognosis than patients\n  wildtype for WT1 in young patients (15-50), primarily because of increased risk\n  of disease recurrence\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 220\nvariant_id: 132\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ff28dddc-29af-4b41-bc2c-bb1baf5edfd8	2015-02-20 17:02:29.903909
585	405	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutations  in WT1 were associated with increased risk of recurrence in young\n  patients (15-50) with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 220\nvariant_id: 132\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d93b1163-8bbc-41f2-8501-d5610013b0ee	2015-02-20 17:02:29.969059
586	406	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 133\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	429a63e2-d8aa-4d4b-93e5-d97893746f36	2015-02-20 17:02:30.045864
587	407	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 133\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6bae6fe8-b73d-4df3-a81b-da5222f2e1f0	2015-02-20 17:02:30.115323
588	408	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No differences in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 133\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4522d604-74aa-48e5-bbf1-8b445e25bfce	2015-02-20 17:02:30.173372
589	409	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 133\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b429685a-5dd8-4995-bf16-4fae4d4a60ce	2015-02-20 17:02:30.236749
590	410	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 133\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	26dedf2f-38ad-43dd-b076-3982a6c06e51	2015-02-20 17:02:30.337634
591	411	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 219\nvariant_id: 133\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	fe04507c-cf57-4d51-8ee2-f71b4753b1ad	2015-02-20 17:02:30.402785
592	412	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 133\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a53ebc4b-9c70-4633-9243-487c0c447e9c	2015-02-20 17:02:30.467387
\.


--
-- Name: audits_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('audits_id_seq', 592, true);


--
-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY users (id, email, name, url, nickname, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: authorizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY authorizations (id, user_id, provider, uid, created_at, updated_at) FROM stdin;
\.


--
-- Name: authorizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('authorizations_id_seq', 1, false);


--
-- Data for Name: comments; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY comments (id, title, comment, commentable_id, commentable_type, user_id, role, created_at, updated_at) FROM stdin;
\.


--
-- Name: comments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('comments_id_seq', 1, false);


--
-- Data for Name: data_versions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY data_versions (id, version) FROM stdin;
\.


--
-- Name: data_versions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('data_versions_id_seq', 1, false);


--
-- Data for Name: definitions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY definitions (id, term, text, created_at, updated_at) FROM stdin;
\.


--
-- Name: definitions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('definitions_id_seq', 1, false);


--
-- Data for Name: delayed_jobs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY delayed_jobs (id, priority, attempts, handler, last_error, run_at, locked_at, failed_at, locked_by, queue, created_at, updated_at) FROM stdin;
\.


--
-- Name: delayed_jobs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('delayed_jobs_id_seq', 1, false);


--
-- Data for Name: diseases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY diseases (id, doid, name, created_at, updated_at) FROM stdin;
3	1612	Breast Cancer	2015-02-20 17:02:03.445958	2015-02-20 17:02:03.445958
6	769	Neuroblastoma	2015-02-20 17:02:03.762061	2015-02-20 17:02:03.762061
8	1909	Melanoma	2015-02-20 17:02:04.654045	2015-02-20 17:02:04.654045
22	4947	Cholangiocarcinoma	2015-02-20 17:02:14.035444	2015-02-20 17:02:14.035444
24	3069	Astrocytoma	2015-02-20 17:02:16.441482	2015-02-20 17:02:16.441482
34	1240	Leukemia	2015-02-20 17:02:19.736455	2015-02-20 17:02:19.736455
1	9119	Acute Myeloid Leukemia	2015-02-20 17:02:02.65813	2015-02-20 17:04:45.530091
2	8552	Chronic Myeloid Leukemia	2015-02-20 17:02:02.789403	2015-02-20 17:04:45.691329
4	3908	Non-small Cell Lung Carcinoma	2015-02-20 17:02:03.537054	2015-02-20 17:04:46.298914
5	1115	Sarcoma	2015-02-20 17:02:03.69347	2015-02-20 17:04:46.480695
7	9256	Colorectal Cancer	2015-02-20 17:02:04.591179	2015-02-20 17:04:46.815197
9	9256	Colorectal Cancer	2015-02-20 17:02:04.948714	2015-02-20 17:04:47.161808
10	1781	Thyroid Cancer	2015-02-20 17:02:06.282424	2015-02-20 17:04:47.32727
11	1612	Breast Cancer	2015-02-20 17:02:07.248288	2015-02-20 17:04:47.618626
12	8618	Oral Cavity Cancer	2015-02-20 17:02:07.423775	2015-02-20 17:04:47.857805
13	0050746	Mantle Cell Lymphoma	2015-02-20 17:02:07.489584	2015-02-20 17:04:47.990421
14	10534	Stomach Cancer	2015-02-20 17:02:07.554689	2015-02-20 17:04:48.23637
15	715	T-cell Leukemia	2015-02-20 17:02:07.873251	2015-02-20 17:04:48.467573
16	5635	Gastric Adenosquamous Carcinoma	2015-02-20 17:02:08.032805	2015-02-20 17:04:48.626496
17	1324	Lung Cancer	2015-02-20 17:02:08.169097	2015-02-20 17:04:48.859113
18	0060075	Estrogen-receptor Positive Breast Cancer	2015-02-20 17:02:08.240192	2015-02-20 17:04:48.987587
19	9952	Acute Lymphocytic Leukemia	2015-02-20 17:02:09.075217	2015-02-20 17:04:49.194768
20	5015	Hepatocellular Fibrolamellar Carcinoma	2015-02-20 17:02:09.208361	2015-02-20 17:04:49.3492
21	3908	Non-small Cell Lung Carcinoma	2015-02-20 17:02:11.608127	2015-02-20 17:04:49.588517
23	3909	Bronchogenic Lung Adenocarcinoma	2015-02-20 17:02:16.10522	2015-02-20 17:04:49.929638
25	3068	Glioblastoma Multiforme	2015-02-20 17:02:16.508285	2015-02-20 17:04:50.551411
26	3069	Astrocytoma	2015-02-20 17:02:16.570629	2015-02-20 17:04:50.810405
27	0050908	Myelodysplastic Syndrome	2015-02-20 17:02:17.132227	2015-02-20 17:04:51.014294
28	1240	Leukemia	2015-02-20 17:02:17.620917	2015-02-20 17:04:51.240206
29	10747	Lymphoid Leukemia	2015-02-20 17:02:17.770505	2015-02-20 17:04:51.383403
30	4960	Bone Marrow Cancer	2015-02-20 17:02:17.836591	2015-02-20 17:04:51.563819
31	8997	Polycythemia Vera	2015-02-20 17:02:17.904913	2015-02-20 17:04:51.706897
32	9119	Acute Myeloid Leukemia	2015-02-20 17:02:17.970392	2015-02-20 17:04:51.879365
33	9253	Gastrointestinal Stromal Tumor	2015-02-20 17:02:18.04342	2015-02-20 17:04:52.014221
35	9538	Multiple Myeloma	2015-02-20 17:02:19.802008	2015-02-20 17:04:52.481542
36	9119	Acute Myeloid Leukemia	2015-02-20 17:02:20.637688	2015-02-20 17:04:52.621157
37	3068	Glioblastoma Multiforme	2015-02-20 17:02:21.294116	2015-02-20 17:04:52.816152
38	9253	Gastrointestinal Stromal Tumor	2015-02-20 17:02:25.020676	2015-02-20 17:04:52.955311
39	9119	Acute Myeloid Leukemia	2015-02-20 17:02:26.963762	2015-02-20 17:04:53.115734
40	3973	Thyroid Medullary Carcinoma	2015-02-20 17:02:27.359568	2015-02-20 17:04:53.290265
41	6477	Invasive Bladder Transitional Cell Carcinoma	2015-02-20 17:02:28.57557	2015-02-20 17:04:53.569316
42	4007	Bladder Carcinoma	2015-02-20 17:02:28.727584	2015-02-20 17:04:53.82041
\.


--
-- Name: diseases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('diseases_id_seq', 42, true);


--
-- Data for Name: drugs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs (id, name, pubchem_id, created_at, updated_at) FROM stdin;
1	Imatinib	\N	2015-02-20 17:02:02.652508	2015-02-20 17:02:02.652508
2	Arsenic Trioxide	\N	2015-02-20 17:02:02.724175	2015-02-20 17:02:02.724175
3	N/A	\N	2015-02-20 17:02:02.785743	2015-02-20 17:02:02.785743
4	Nilotinib	\N	2015-02-20 17:02:03.011559	2015-02-20 17:02:03.011559
5	Dasatinib	\N	2015-02-20 17:02:03.072126	2015-02-20 17:02:03.072126
6	MK-2206	\N	2015-02-20 17:02:03.442216	2015-02-20 17:02:03.442216
7	Crizotinib	\N	2015-02-20 17:02:03.531868	2015-02-20 17:02:03.531868
8	CH5424802	\N	2015-02-20 17:02:03.758326	2015-02-20 17:02:03.758326
9	TAE684	\N	2015-02-20 17:02:03.957896	2015-02-20 17:02:03.957896
10	Sorafenib	\N	2015-02-20 17:02:04.382036	2015-02-20 17:02:04.382036
11	Trametinib	\N	2015-02-20 17:02:04.456797	2015-02-20 17:02:04.456797
12	Panitumumab	\N	2015-02-20 17:02:04.587552	2015-02-20 17:02:04.587552
13	Dabrafenib	\N	2015-02-20 17:02:04.753124	2015-02-20 17:02:04.753124
14	Cetuximab	\N	2015-02-20 17:02:04.944726	2015-02-20 17:02:04.944726
15	PD0325901	\N	2015-02-20 17:02:05.125839	2015-02-20 17:02:05.125839
16	PLX4720	\N	2015-02-20 17:02:05.184009	2015-02-20 17:02:05.184009
17	Nutlin-3	\N	2015-02-20 17:02:05.326382	2015-02-20 17:02:05.326382
18	Capecitabine	\N	2015-02-20 17:02:05.392026	2015-02-20 17:02:05.392026
19	Vemurafenib	\N	2015-02-20 17:02:05.44688	2015-02-20 17:02:05.44688
20	Bevacizumab	\N	2015-02-20 17:02:05.500991	2015-02-20 17:02:05.500991
21	Dacarbazine	\N	2015-02-20 17:02:05.910285	2015-02-20 17:02:05.910285
22	Temozolomide	\N	2015-02-20 17:02:05.963879	2015-02-20 17:02:05.963879
23	AZD6244	\N	2015-02-20 17:02:06.757043	2015-02-20 17:02:06.757043
24	Palbociclib (PD-0332991)	\N	2015-02-20 17:02:07.868707	2015-02-20 17:02:07.868707
25	BYL719	\N	2015-02-20 17:02:08.236082	2015-02-20 17:02:08.236082
26	Palbociclib	\N	2015-02-20 17:02:08.310624	2015-02-20 17:02:08.310624
27	GW-2580	\N	2015-02-20 17:02:09.132137	2015-02-20 17:02:09.132137
28	Daunorubicin	\N	2015-02-20 17:02:09.403513	2015-02-20 17:02:09.403513
29	Idarubicin	\N	2015-02-20 17:02:09.484752	2015-02-20 17:02:09.484752
30	Gefitinib	\N	2015-02-20 17:02:11.283828	2015-02-20 17:02:11.283828
31	Erlotinib	\N	2015-02-20 17:02:11.337831	2015-02-20 17:02:11.337831
32	Premetrexed	\N	2015-02-20 17:02:11.803626	2015-02-20 17:02:11.803626
33	Stauroporine	\N	2015-02-20 17:02:11.876769	2015-02-20 17:02:11.876769
34	Neratinib	\N	2015-02-20 17:02:12.097528	2015-02-20 17:02:12.097528
35	Lapatinib	\N	2015-02-20 17:02:12.445072	2015-02-20 17:02:12.445072
36	Tamoxifen	\N	2015-02-20 17:02:12.980455	2015-02-20 17:02:12.980455
37	Fulvestrant	\N	2015-02-20 17:02:13.048712	2015-02-20 17:02:13.048712
38	Ponatinib	\N	2015-02-20 17:02:14.029934	2015-02-20 17:02:14.029934
39	Pazopanib	\N	2015-02-20 17:02:14.1098	2015-02-20 17:02:14.1098
40	SU5614	\N	2015-02-20 17:02:14.454548	2015-02-20 17:02:14.454548
41	ATRA	\N	2015-02-20 17:02:14.519769	2015-02-20 17:02:14.519769
42	CEP701	\N	2015-02-20 17:02:15.24486	2015-02-20 17:02:15.24486
43	AG1296	\N	2015-02-20 17:02:15.330346	2015-02-20 17:02:15.330346
44	Bortezomib	\N	2015-02-20 17:02:16.101594	2015-02-20 17:02:16.101594
45	Fasudil	\N	2015-02-20 17:02:16.164325	2015-02-20 17:02:16.164325
46	TG101348	\N	2015-02-20 17:02:17.615521	2015-02-20 17:02:17.615521
47	Pegylated IFN-α–2a	\N	2015-02-20 17:02:17.899539	2015-02-20 17:02:17.899539
48	SU11248	\N	2015-02-20 17:02:19.297481	2015-02-20 17:02:19.297481
49	Regorafenib	\N	2015-02-20 17:02:19.470005	2015-02-20 17:02:19.470005
50	NVP-BEZ235	\N	2015-02-20 17:02:20.467722	2015-02-20 17:02:20.467722
51	ARRY-142886	\N	2015-02-20 17:02:20.52648	2015-02-20 17:02:20.52648
52	O(6)-benzylguanine	\N	2015-02-20 17:02:21.412863	2015-02-20 17:02:21.412863
53	Carmustine	\N	2015-02-20 17:02:21.476633	2015-02-20 17:02:21.476633
54	NSC348884	\N	2015-02-20 17:02:22.167801	2015-02-20 17:02:22.167801
55	Valproic Acid	\N	2015-02-20 17:02:22.46989	2015-02-20 17:02:22.46989
56	17-AAG	\N	2015-02-20 17:02:24.556697	2015-02-20 17:02:24.556697
57	Temozolomide 	\N	2015-02-20 17:02:24.885884	2015-02-20 17:02:24.885884
58	Crenolanib	\N	2015-02-20 17:02:25.016569	2015-02-20 17:02:25.016569
59	CH5132799	\N	2015-02-20 17:02:25.577014	2015-02-20 17:02:25.577014
60	Rapamycin	\N	2015-02-20 17:02:25.634851	2015-02-20 17:02:25.634851
61	Motesanib	\N	2015-02-20 17:02:27.355728	2015-02-20 17:02:27.355728
62	AZD1480	\N	2015-02-20 17:02:27.551801	2015-02-20 17:02:27.551801
63	Doxorubicin	\N	2015-02-20 17:02:27.813961	2015-02-20 17:02:27.813961
64	Everolimus	\N	2015-02-20 17:02:28.571779	2015-02-20 17:02:28.571779
\.


--
-- Name: drugs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('drugs_id_seq', 64, true);


--
-- Data for Name: evidence_levels; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_levels (id, level, description, created_at, updated_at) FROM stdin;
1	B	\N	2015-02-20 17:02:02.663087	2015-02-20 17:02:02.663087
2	A	\N	2015-02-20 17:02:02.726827	2015-02-20 17:02:02.726827
3	C	\N	2015-02-20 17:02:02.857705	2015-02-20 17:02:02.857705
4	D	\N	2015-02-20 17:02:06.761435	2015-02-20 17:02:06.761435
\.


--
-- Data for Name: evidence_types; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_types (id, evidence_type, created_at, updated_at) FROM stdin;
1	Predictive	2015-02-20 17:02:02.660729	2015-02-20 17:02:02.660729
2	Diagnostic	2015-02-20 17:02:02.791196	2015-02-20 17:02:02.791196
3	Prognostic	2015-02-20 17:02:03.1346	2015-02-20 17:02:03.1346
\.


--
-- Data for Name: genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes (id, entrez_id, name, description, official_name, created_at, updated_at, clinical_description) FROM stdin;
1	238	ALK	ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines.	anaplastic lymphoma receptor tyrosine kinase	2015-02-20 17:01:48.51614	2015-02-20 17:01:48.51614	\N
2	207	AKT1	AKT1, also referred to as protein kinase B, is a known oncogene. AKT activation relies on the PI3K pathway, and is recognized as a critical node in the pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been shown to result in activation of the protein. Mutations in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors in vitro.	v-akt murine thymoma viral oncogene homolog 1	2015-02-20 17:01:49.086282	2015-02-20 17:01:49.086282	\N
3	369	ARAF	ARAF has recently become increasingly considered for its oncogenic potential. Its potential as a target for informing clinical action was demonstrated by a single case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated with sorafenib, acheived near-complete clinical remission. This finding has brought new focus on ARAF as a marker that should be assayed for in cancer treatment.	A-Raf proto-oncogene, serine/threonine kinase	2015-02-20 17:01:49.287471	2015-02-20 17:01:49.287471	\N
4	25	ABL1	ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein that has become a signature of chronic myeloid leukemia (CML). Cells harboring this fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook of the disease. However, additional mutations in ABL1 have been shown to confer resistance to imatinib. In these resistance cases, second-generation tyrosine kinase inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are currently undergoing clinical trials for treating acquired resistance in CML.	ABL proto-oncogene 1, non-receptor tyrosine kinase	2015-02-20 17:01:49.495228	2015-02-20 17:01:49.495228	\N
5	673	BRAF	BRAF mutations are found to be recurrent in many cancer types. Of these, the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E has been determined to be an activating mutation, and cells that harbor it, along with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations have also been correlated with poor prognosis in many cancer types, although there is at least one study that questions this conclusion in papillary thyroid cancer.	B-Raf proto-oncogene, serine/threonine kinase	2015-02-20 17:01:49.707263	2015-02-20 17:01:49.707263	\N
6	595	CCND1	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D1	2015-02-20 17:01:49.922488	2015-02-20 17:01:49.922488	\N
7	894	CCND2	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D2	2015-02-20 17:01:50.130176	2015-02-20 17:01:50.130176	\N
8	896	CCND3	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. In a mouse study, the targeted therapeutic palbociclib significantly increased the median survival of the cyclin D3 knockouts.	cyclin D3	2015-02-20 17:01:50.332539	2015-02-20 17:01:50.332539	\N
9	898	CCNE1	Cyclin E, while currenly not as widely implicated as its cyclin D counterparts, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements are correlated with later stage disease. In lung cancer, neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly expressed counterparts.	cyclin E1	2015-02-20 17:01:50.528443	2015-02-20 17:01:50.528443	\N
10	1021	CDK6	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 6	2015-02-20 17:01:50.730251	2015-02-20 17:01:50.730251	\N
11	1019	CDK4	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 4	2015-02-20 17:01:50.938003	2015-02-20 17:01:50.938003	\N
12	1029	CDKN2A	CDKN2A loss has been shown to be a significant event in a number of cancer types. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. One mechanism by which loss of CDKN2A can occur is by hypermethylation of the promoter region for the gene. However, the prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have suggesting poorer prognostic outcome for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	cyclin-dependent kinase inhibitor 2A	2015-02-20 17:01:51.148021	2015-02-20 17:01:51.148021	\N
13	1050	CEBPA	AML with mutated CEBPA' is a provisional entity in the WHO classification of acute myeloid leukemia (AML) and is recommended to be tested for in patients with AML. CEBPA mutations are particularly associated with cytogenetically normal AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha protein while C-terminal mutations reduce the DNA-binding potential of this transcription factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	CCAAT/enhancer binding protein (C/EBP), alpha	2015-02-20 17:01:51.356518	2015-02-20 17:01:51.356518	\N
14	3337	DNAJB1	DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	DnaJ (Hsp40) homolog, subfamily B, member 1	2015-02-20 17:01:51.571916	2015-02-20 17:01:51.571916	\N
15	5566	PRKACA	"PRKACA has been studied in breast cancer and has been found to mediate resistance to HER2 targeted therapies. It has also been found to contain a mutation hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity."	protein kinase, cAMP-dependent, catalytic, alpha	2015-02-20 17:01:51.790286	2015-02-20 17:01:51.790286	\N
16	1788	DNMT3A	DNMT3A is one of several epigenetic modifiers identified as recurrently mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers. AML patient bone marrow harboring R882 mutations were similarly demonstrated to be hypomethylated compared to patients with wildtype DNMT3A. These studies also indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes and treatment protocols may need to be considered for these two classes of DNMT3A mutations.	DNA (cytosine-5-)-methyltransferase 3 alpha	2015-02-20 17:01:52	2015-02-20 17:01:52	\N
17	1956	EGFR	EGFR is widely recognized for its importance in cancer. Amplification and mutations have been shown to be driving events in many cancer types. Its role in non-small cell lung cancer has spurred many research and drug development efforts. Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance to these drugs, particularly the variant T790M, which has been functionally characterized as a resistance marker for both of these drugs. The later generation TKI's have seen some success in treating these resistant cases, and targeted sequencing of the EGFR locus has become a common practice in non-small cell lung cancer.	epidermal growth factor receptor	2015-02-20 17:01:52.230738	2015-02-20 17:01:52.230738	\N
18	2064	ERBB2	ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast cancer, and is treated in a separate manner than the other subtypes of breast cancer. Apart from being amplified, ERBB2 activating mutations have been shown to have clinical importance in HER2-negative breast cancer. These mutations have shown senitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical sequencing efforts in treating breast cancer.	erb-b2 receptor tyrosine kinase 2	2015-02-20 17:01:52.452792	2015-02-20 17:01:52.452792	\N
19	2099	ESR1	ESR1 has been a focus in breast cancer for quite some time, but has also shown significance in endometrial, ovarian and other cancer types. ER-positive breast cancer that is resistant to hormone therapy has instigated clinical sequencing efforts to shed light on the mechanisms of this resistance. A number of mutations in the ligand binding domain of the protein have been implicated in hormone resistance and anti-estrogen therapies. This has spurred efforts to develop therapeutics that act to degrade the protein, rather than act as an antagonist. These agents are currently in clinical trials and have seen some success, highlighting the importance of sequencing efforts in treating breast cancer.	estrogen receptor 1	2015-02-20 17:01:52.665649	2015-02-20 17:01:52.665649	\N
20	2263	FGFR2	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 2	2015-02-20 17:01:52.880611	2015-02-20 17:01:52.880611	\N
21	2261	FGFR3	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to the activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFRs can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 3	2015-02-20 17:01:53.095974	2015-02-20 17:01:53.095974	\N
22	2322	FLT3	FLT3 is an important cytokine receptor involved in normal hematopoiesis. Mutations in this gene are common in acute myeloid leukemia (AML) and screening for mutations in this gene has been recommended by the World Health Organization in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML). FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD mutations are still up for debate.	fms-related tyrosine kinase 3	2015-02-20 17:01:53.368766	2015-02-20 17:01:53.368766	\N
23	2624	GATA2	N/A	GATA binding protein 2	2015-02-20 17:01:53.569376	2015-02-20 17:01:53.569376	\N
24	3417	IDH1	IDH1 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with specific cytogenetic abnormalities, 1p and 19q deletions.	isocitrate dehydrogenase 1 (NADP+), soluble	2015-02-20 17:01:53.757605	2015-02-20 17:01:53.757605	\N
25	3418	IDH2	N/A	isocitrate dehydrogenase 2 (NADP+), mitochondrial	2015-02-20 17:01:53.949936	2015-02-20 17:01:53.949936	\N
26	3717	JAK2	JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative diseases and cancers. The mutation V617F is the most clinically relevant variant, and is seen in around half of myeloproliferative disorders.	Janus kinase 2	2015-02-20 17:01:54.156038	2015-02-20 17:01:54.156038	\N
27	3815	KIT	N/A	v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog	2015-02-20 17:01:54.343813	2015-02-20 17:01:54.343813	\N
28	3845	KRAS	N/A	Kirsten rat sarcoma viral oncogene homolog	2015-02-20 17:01:54.545108	2015-02-20 17:01:54.545108	\N
29	5604	MAP2K1	N/A	mitogen-activated protein kinase kinase 1	2015-02-20 17:01:54.734896	2015-02-20 17:01:54.734896	\N
30	4209	MEF2D	N/A	myocyte enhancer factor 2D	2015-02-20 17:01:54.952408	2015-02-20 17:01:54.952408	\N
31	1436	CSF1R	N/A	colony stimulating factor 1 receptor	2015-02-20 17:01:55.143481	2015-02-20 17:01:55.143481	\N
32	4255	MGMT	N/A	O-6-methylguanine-DNA methyltransferase	2015-02-20 17:01:55.390701	2015-02-20 17:01:55.390701	\N
33	4869	NPM1	AML with mutated NPM1' is provisional entity in the WHO classification of AML and is recommended to be tested for in patients with cytogenetically normal AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia, indicating it may require additional mutations to promote leukemic development.	nucleophosmin (nucleolar phosphoprotein B23, numatrin)	2015-02-20 17:01:55.602444	2015-02-20 17:01:55.602444	\N
34	4893	NRAS	N/A	neuroblastoma RAS viral (v-ras) oncogene homolog	2015-02-20 17:01:55.810509	2015-02-20 17:01:55.810509	\N
35	5290	PIK3CA	PIK3CA is the most recurrently mutated gene in breast cancer, and has been found to important in a number of cancer types. An integral part of the PI3K pathway, PIK3CA has long been described as an oncogene, with two main hotspots for activating mutations, the 542/545 region of the helical domain, and the 1047 region of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the subject of an immense amount of research and development, and PI3K inhibition has seen some limited success in recent clinical trials. While monotherapies seem to be limited in their potential, there is a recent interest in pursuing PI3K inhibition as part of a combination therapy regiment with inhibition partners including TKI's, MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.	phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha	2015-02-20 17:01:56.005688	2015-02-20 17:01:56.005688	\N
36	5156	PDGFRA	N/A	platelet-derived growth factor receptor, alpha polypeptide	2015-02-20 17:01:56.20562	2015-02-20 17:01:56.20562	\N
37	5371	PML	N/A	promyelocytic leukemia	2015-02-20 17:01:56.397683	2015-02-20 17:01:56.397683	\N
38	5914	RARa	N/A	retinoic acid receptor, alpha	2015-02-20 17:01:56.608289	2015-02-20 17:01:56.608289	\N
39	5728	PTEN	N/A	phosphatase and tensin homolog	2015-02-20 17:01:56.800896	2015-02-20 17:01:56.800896	\N
40	5979	RET	RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine kinase receptor and are associated with thyroid cancers. RET point mutations are the most common mutations identified in medullary thyroid cancer (MTC) with germline and somatic mutations in RET associated with hereditary and sporadic forms, respectively. The most common somatic form mutation is M918T (exon 16) and a variety of other mutations effecting exons 10, 11 and 15 have been described. The prognostic significance of these mutations have been hotly debated in the field, however, data suggests that some RET mutation may confer drug resistence. No RET-specific agents are currently clinically available but several promiscuous kinase inhibitors that target RET, among others, have been approved for MTC treatment.	ret proto-oncogene	2015-02-20 17:01:56.993931	2015-02-20 17:01:56.993931	\N
41	861	RUNX1	N/A	runt-related transcription factor 1	2015-02-20 17:01:57.198646	2015-02-20 17:01:57.198646	\N
42	23451	SF3B1	SF3B1 mutations have been described in several myeloid malignancies, predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has been identified as recurrently mutated in MDS and other malignanices. The mutations affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally deleterious with K700E described as a major hotspot mutation. MDS patients with SF3B1 mutations have been reported to have better overall and event-free survival than their wildtype counterparts. Additionally, these mutations are highly associated with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring sideroblasts).	splicing factor 3b, subunit 1, 155kDa	2015-02-20 17:01:57.467416	2015-02-20 17:01:57.467416	\N
43	7157	TP53	N/A	tumor protein p53	2015-02-20 17:01:57.721301	2015-02-20 17:01:57.721301	\N
44	7248	TSC1	N/A	tuberous sclerosis 1	2015-02-20 17:01:57.910111	2015-02-20 17:01:57.910111	\N
45	7249	TSC2	N/A	tuberous sclerosis 2	2015-02-20 17:01:58.1056	2015-02-20 17:01:58.1056	\N
46	7307	U2AF1	U2AF1 is one of several spliceosome complex genes frequently mutated in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein. These mutations have been associated with altered splicing patterns in vitro and in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation to secondary acute myeloid leukemia, however, the impact of these mutations on overall survival has been an area of debate.	U2 small nuclear RNA auxiliary factor 1	2015-02-20 17:01:58.327151	2015-02-20 17:01:58.327151	\N
47	7490	WT1	N/A	Wilms tumor 1	2015-02-20 17:01:58.526791	2015-02-20 17:01:58.526791	\N
\.


--
-- Data for Name: sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY sources (id, pubmed_id, study_type, description, created_at, updated_at) FROM stdin;
1	24889366	\N	Rossi et al., 2014, Int. J. Oncol.	2015-02-20 17:01:48.52699	2015-02-20 17:02:31.196619
2	23401436	\N	Shaw et al., 2013, J. Clin. Oncol.	2015-02-20 17:01:48.541384	2015-02-20 17:02:32.119614
3	16095999	\N	Bellacosa et al., 2005, Adv. Cancer Res.	2015-02-20 17:01:49.089435	2015-02-20 17:02:32.539428
4	15023437	\N	Fresno Vara et al., 2004, Cancer Treat. Rev.	2015-02-20 17:01:49.095451	2015-02-20 17:02:32.957582
5	15676015	\N	Lee et al., 2005, APMIS	2015-02-20 17:01:49.290861	2015-02-20 17:02:33.374987
6	24569458	\N	Imielinski et al., 2014, J. Clin. Invest.	2015-02-20 17:01:49.296624	2015-02-20 17:02:33.79378
7	15719031	\N	Ren, 2005, Nat. Rev. Cancer	2015-02-20 17:01:49.498488	2015-02-20 17:02:34.750455
8	17457302	\N	Weisberg et al., 2007, Nat. Rev. Cancer	2015-02-20 17:01:49.50361	2015-02-20 17:02:35.672079
9	19724843	\N	Li et al., 2009, Oncol. Rep.	2015-02-20 17:01:49.710448	2015-02-20 17:02:36.090412
10	23594689	\N	Pakneshan et al., 2013, Pathology	2015-02-20 17:01:49.715514	2015-02-20 17:02:36.504901
11	12432268	\N	Diehl, Cancer Biol. Ther.	2015-02-20 17:01:49.925701	2015-02-20 17:02:36.924314
12	24387133	\N	Casimiro et al., 2014, Expert Opin Investig Drugs	2015-02-20 17:01:49.931198	2015-02-20 17:02:37.3441
13	14965268	\N	Mazumder et al., 2004, Curr Cancer Drug Targets	2015-02-20 17:01:50.531451	2015-02-20 17:02:37.807711
14	20370706	\N	Graf et al., 2010, Mini Rev Med Chem	2015-02-20 17:01:50.733764	2015-02-20 17:02:38.169592
15	24089445	\N	Sheppard et al., 2013, Clin. Cancer Res.	2015-02-20 17:01:50.739792	2015-02-20 17:02:38.599636
16	20473920	\N	Shima et al., 2011, Int. J. Cancer	2015-02-20 17:01:51.151169	2015-02-20 17:02:39.015517
17	23111194	\N	Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-20 17:01:51.156316	2015-02-20 17:02:39.432128
18	19357394	\N	Vardiman et al., 2009, Blood	2015-02-20 17:01:51.362362	2015-02-20 17:02:39.842858
19	11242107	\N	Pabst et al., 2001, Nat. Genet.	2015-02-20 17:01:51.367463	2015-02-20 17:02:40.203638
20	9012825	\N	Zhang et al., 1997, Proc. Natl. Acad. Sci. U.S.A.	2015-02-20 17:01:51.371787	2015-02-20 17:02:40.687955
21	24578576	\N	Honeyman et al., 2014, Science	2015-02-20 17:01:51.57513	2015-02-20 17:02:41.090988
22	24909179	\N	Moody et al., 2014, Oncogene	2015-02-20 17:01:51.796994	2015-02-20 17:02:41.525575
23	24167195	\N	Kim et al., 2013, Blood	2015-02-20 17:01:52.004508	2015-02-20 17:02:41.93939
24	24656771	\N	Russler-Germain et al., 2014, Cancer Cell	2015-02-20 17:01:52.0106	2015-02-20 17:02:42.364866
25	23953842	\N	Yewale et al., 2013, Biomaterials	2015-02-20 17:01:52.234034	2015-02-20 17:02:42.925344
26	18712184	\N	Charpidou et al., In Vivo	2015-02-20 17:01:52.239223	2015-02-20 17:02:43.278849
27	16947083	\N	Badache et al., 2006, J Mammary Gland Biol Neoplasia	2015-02-20 17:01:52.455858	2015-02-20 17:02:43.675869
28	10878580	\N	Yu et al., 2000, Bioessays	2015-02-20 17:01:52.460803	2015-02-20 17:02:44.070207
29	24185510	\N	Robinson et al., 2013, Nat. Genet.	2015-02-20 17:01:52.668865	2015-02-20 17:02:44.452516
30	22279420	\N	Mohibi et al., 2011, J Carcinog	2015-02-20 17:01:52.673983	2015-02-20 17:02:44.812175
31	18636142	\N	Katoh, 2008, Int. J. Oncol.	2015-02-20 17:01:52.884135	2015-02-20 17:02:45.218449
32	23558953	\N	Wu et al., 2013, Cancer Discov	2015-02-20 17:01:52.889749	2015-02-20 17:02:45.621298
33	12951584	\N	Stirewalt et al., 2003, Nat. Rev. Cancer	2015-02-20 17:01:53.372069	2015-02-20 17:02:45.97199
34	23226219	\N	Chou et al., 2012, PLoS ONE	2015-02-20 17:01:55.605951	2015-02-20 17:02:46.361894
35	18073307	\N	Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-20 17:01:56.997113	2015-02-20 17:02:46.743317
36	25465739	\N	Perri et al., 2014, Crit. Rev. Oncol. Hematol.	2015-02-20 17:01:57.002551	2015-02-20 17:02:47.171809
37	21995386	\N	Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-20 17:01:57.471032	2015-02-20 17:02:47.677199
38	24136165	\N	Cazzola et al., 2013, Blood	2015-02-20 17:01:57.476493	2015-02-20 17:02:48.0996
39	25311244	\N	Okeyo-Owuor et al., 2014, Leukemia	2015-02-20 17:01:58.330297	2015-02-20 17:02:48.500368
40	22158538	\N	Graubert et al., 2012, Nat. Genet.	2015-02-20 17:01:58.335544	2015-02-20 17:02:49.419672
41	12476305		Nimmanapalli et al., 2002, Oncogene	2015-02-20 17:02:02.655864	2015-02-20 17:02:49.783656
42	20537386		An et al., 2010, Leuk. Res.	2015-02-20 17:02:02.787761	2015-02-20 17:02:50.152868
43	25212276		Schmidt et al., 2014, Leukemia	2015-02-20 17:02:02.855209	2015-02-20 17:02:50.599954
44	15194504	Cell Line	Yamamoto et al., 2004, Biochem. Biophys. Res. Commun.	2015-02-20 17:02:03.200806	2015-02-20 17:02:52.056236
45	23888070		Beaver et al., 2013, Clin. Cancer Res.	2015-02-20 17:02:03.444189	2015-02-20 17:02:52.600567
46	20979473	Clinical Trial	Choi et al., 2010, N. Engl. J. Med.	2015-02-20 17:02:03.534733	2015-02-20 17:02:52.971619
47	21030459		Sasaki et al., 2010, Cancer Res.	2015-02-20 17:02:03.69163	2015-02-20 17:02:53.355139
48	21575866		Sakamoto et al., 2011, Cancer Cell	2015-02-20 17:02:03.760253	2015-02-20 17:02:53.743172
49	21948233		Heuckmann et al., 2011, Clin. Cancer Res.	2015-02-20 17:02:03.822268	2015-02-20 17:02:54.265729
50	22072639		Bresler et al., 2011, Sci Transl Med	2015-02-20 17:02:03.886579	2015-02-20 17:02:54.682805
51	18923525		George et al., 2008, Nature	2015-02-20 17:02:03.959712	2015-02-20 17:02:55.103965
52	24569458		Imielinski et al., 2014, J. Clin. Invest.	2015-02-20 17:02:04.38412	2015-02-20 17:02:56.919039
53	23325582	Clinical trial	Peeters et al., 2013, Clin. Cancer Res.	2015-02-20 17:02:04.58941	2015-02-20 17:02:57.399573
54	21166657	Meta analysis	Lee et al., 2011, Br. J. Dermatol.	2015-02-20 17:02:04.651812	2015-02-20 17:02:57.791362
55	23020132	Clinical trial	Flaherty et al., 2012, N. Engl. J. Med.	2015-02-20 17:02:04.7551	2015-02-20 17:02:58.236608
56	23463675	Clinical trial	Ponti et al., 2013, J. Clin. Pathol.	2015-02-20 17:02:04.871332	2015-02-20 17:02:59.92312
57	19001320	In vitro study	Di Nicolantonio et al., 2008, J. Clin. Oncol.	2015-02-20 17:02:04.946866	2015-02-20 17:03:00.323852
58	23845441	In vitro study	Rad et al., 2013, Cancer Cell	2015-02-20 17:02:05.128612	2015-02-20 17:03:00.744624
59	23812671	In vitro study	Ji et al., 2013, Clin. Cancer Res.	2015-02-20 17:02:05.274678	2015-02-20 17:03:01.290995
60	22180495	In vivo study	Yang et al., 2012, Cancer Res.	2015-02-20 17:02:05.393909	2015-02-20 17:03:01.690219
61	24594804	Metaanalysis	Chen et al., 2014, PLoS ONE	2015-02-20 17:02:05.630345	2015-02-20 17:03:02.204209
62	20619739	Retrospective study	De Roock et al., 2010, Lancet Oncol.	2015-02-20 17:02:05.68981	2015-02-20 17:03:02.623422
63	24576830	In vitro study	Nissan et al., 2014, Cancer Res.	2015-02-20 17:02:05.750401	2015-02-20 17:03:03.146645
64	24586605	Retrospective study	Meckbach et al., 2014, PLoS ONE	2015-02-20 17:02:05.912192	2015-02-20 17:03:03.645197
65	24583796	Clinical trial	Menzies et al., 2014, Clin. Cancer Res.	2015-02-20 17:02:06.035575	2015-02-20 17:03:03.994833
66	24388723	Retrospective study	Nagore et al., 2014, J. Am. Acad. Dermatol.	2015-02-20 17:02:06.143862	2015-02-20 17:03:04.396233
67	23524406	Case report	Rudin et al., 2013, J Thorac Oncol	2015-02-20 17:02:06.217467	2015-02-20 17:03:04.818887
68	24588959	Perspective study	Zhang et al., 2014, Diagn Pathol	2015-02-20 17:02:06.279675	2015-02-20 17:03:05.34295
69	24570209	Retrospective study	Crescenzi et al., 2014, Horm. Metab. Res.	2015-02-20 17:02:06.361486	2015-02-20 17:03:05.759691
70	21594703	Retrospective study	Howell et al., 2011, Ann. Surg. Oncol.	2015-02-20 17:02:06.425681	2015-02-20 17:03:06.172257
71	24354346	Retrospective study	Walczyk et al., 2014, Clin. Endocrinol. (Oxf)	2015-02-20 17:02:06.494845	2015-02-20 17:03:06.590306
72	24396464	Retrospective study	He et al., 2014, Oncol Lett	2015-02-20 17:02:06.628347	2015-02-20 17:03:07.006931
73	21098728	In vitro study	Corcoran et al., 2010, Sci Signal	2015-02-20 17:02:06.759121	2015-02-20 17:03:08.410472
74	23031422	Case report	Ponti et al., 2012, J Hematol Oncol	2015-02-20 17:02:06.832631	2015-02-20 17:03:08.88711
75	17070615		Gautschi et al., 2007, Lung Cancer	2015-02-20 17:02:07.057618	2015-02-20 17:03:09.305378
76	15961768		Arnold et al., 2005, J. Clin. Oncol.	2015-02-20 17:02:07.246371	2015-02-20 17:03:09.715584
77	16309541		Thomas et al., 2005, Int J Exp Pathol	2015-02-20 17:02:07.421451	2015-02-20 17:03:10.165494
78	17891190		Jares et al., 2007, Nat. Rev. Cancer	2015-02-20 17:02:07.487701	2015-02-20 17:03:12.376869
79	10547574		Takano et al., 1999, J. Pathol.	2015-02-20 17:02:07.552752	2015-02-20 17:03:12.752521
80	14612939		Oshimo et al., 2003, Int. J. Oncol.	2015-02-20 17:02:07.801125	2015-02-20 17:03:13.169803
81	23079656		Sawai et al., 2012, Cancer Cell	2015-02-20 17:02:07.871219	2015-02-20 17:03:13.589689
82	12432043		Keyomarsi et al., 2002, N. Engl. J. Med.	2015-02-20 17:02:07.954738	2015-02-20 17:03:14.0058
83	10224221		Donnellan et al., 1999, FASEB J.	2015-02-20 17:02:08.030598	2015-02-20 17:03:14.497219
84	25002028		Vora et al., 2014, Cancer Cell	2015-02-20 17:02:08.238123	2015-02-20 17:03:14.947644
85	23898052		Logan et al., 2013, Anticancer Res.	2015-02-20 17:02:08.312701	2015-02-20 17:03:15.572525
86	23111194		Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-20 17:02:08.541744	2015-02-20 17:03:16.094002
87	19965647	Retrospective study	Burnett et al., 2010, Blood	2015-02-20 17:02:08.677477	2015-02-20 17:03:16.510902
88	18450602	Retrospective study	Schlenk et al., 2008, N. Engl. J. Med.	2015-02-20 17:02:08.739204	2015-02-20 17:03:17.46405
89	14726504	Retrospective study	Fröhling et al., 2004, J. Clin. Oncol.	2015-02-20 17:02:08.854517	2015-02-20 17:03:17.872054
90	24186003		Lilljebjörn et al., 2014, Leukemia	2015-02-20 17:02:09.073299	2015-02-20 17:03:18.297225
91	24578576	Retrospective study	Honeyman et al., 2014, Science	2015-02-20 17:02:09.205467	2015-02-20 17:03:19.290796
92	22490330	Retrospective study	Ribeiro et al., 2012, Blood	2015-02-20 17:02:09.273548	2015-02-20 17:03:19.758821
93	22081665	Retrospective study	LaRochelle et al., 2011, Oncotarget	2015-02-20 17:02:09.337171	2015-02-20 17:03:21.298419
94	21067377	Retrospective study	Ley et al., 2010, N. Engl. J. Med.	2015-02-20 17:02:09.617419	2015-02-20 17:03:21.737175
95	24512939	Retrospective study	El Ghannam et al., Blood Cells Mol. Dis.	2015-02-20 17:02:09.685921	2015-02-20 17:03:22.163605
96	23632886	Retrospective study	Gaidzik et al., 2013, Blood	2015-02-20 17:02:09.814425	2015-02-20 17:03:22.57168
97	22291079	Retrospective study	Marcucci et al., 2012, J. Clin. Oncol.	2015-02-20 17:02:10.213598	2015-02-20 17:03:22.921236
98	24736073		Lim et al., 2014, J Thorac Oncol	2015-02-20 17:02:11.285724	2015-02-20 17:03:24.432989
99	24457318		Fukihara et al., 2014, Oncology	2015-02-20 17:02:11.395234	2015-02-20 17:03:24.86823
100	24585406		Li et al., 2014, Ann. Surg. Oncol.	2015-02-20 17:02:11.464022	2015-02-20 17:03:25.394127
101	24662454		Douillard et al., 2014, J Thorac Oncol	2015-02-20 17:02:11.543198	2015-02-20 17:03:26.01532
102	25668228	Review	Denis et al., 2015, Clin. Chim. Acta	2015-02-20 17:02:11.605785	2015-02-20 17:03:26.438506
103	15728811		Kobayashi et al., 2005, N. Engl. J. Med.	2015-02-20 17:02:11.682143	2015-02-20 17:03:27.95489
104	24636847		Li et al., 2014, Lung Cancer	2015-02-20 17:02:11.744649	2015-02-20 17:03:29.148055
105	24658966		Ai et al., 2014, Amino Acids	2015-02-20 17:02:11.878708	2015-02-20 17:03:29.565952
106	24729716		Li et al., 2014, Onco Targets Ther	2015-02-20 17:02:11.943929	2015-02-20 17:03:29.985987
107	24623981		Ding et al., 2014, Onco Targets Ther	2015-02-20 17:02:12.013264	2015-02-20 17:03:31.490478
108	23220880		Bose et al., 2013, Cancer Discov	2015-02-20 17:02:12.099486	2015-02-20 17:03:31.972842
109	24185510		Robinson et al., 2013, Nat. Genet.	2015-02-20 17:02:12.982244	2015-02-20 17:03:32.701758
110	24185512		Toy et al., 2013, Nat. Genet.	2015-02-20 17:02:13.130005	2015-02-20 17:03:33.118686
111	24550739	Case study	Borad et al., 2014, PLoS Genet.	2015-02-20 17:02:14.032767	2015-02-20 17:03:34.828224
112	17957027	Retrospective study	Gale et al., 2008, Blood	2015-02-20 17:02:14.383448	2015-02-20 17:03:35.208447
113	15626738	In vitro study	Bagrintseva et al., 2005, Blood	2015-02-20 17:02:14.456975	2015-02-20 17:03:35.94387
114	19965647		Burnett et al., 2010, Blood	2015-02-20 17:02:14.521898	2015-02-20 17:03:36.46119
115	21537333		Schnittger et al., 2011, Leukemia	2015-02-20 17:02:14.707156	2015-02-20 17:03:37.088403
116	24801015		Port et al., 2014, Ann. Hematol.	2015-02-20 17:02:15.035972	2015-02-20 17:03:38.908581
117	11290608	Retrospective study	Yamamoto et al., 2001, Blood	2015-02-20 17:02:15.098221	2015-02-20 17:03:39.282852
118	18230792	In vitro study	Zhang et al., 2008, J. Natl. Cancer Inst.	2015-02-20 17:02:15.168688	2015-02-20 17:03:39.702605
119	14726387	Clinical trial	Smith et al., 2004, Blood	2015-02-20 17:02:15.246859	2015-02-20 17:03:40.119649
120	12357354	In vitro study	Tse et al., 2002, Leukemia	2015-02-20 17:02:15.33216	2015-02-20 17:03:40.539926
121	11585760	Retrospective study	Whitman et al., 2001, Cancer Res.	2015-02-20 17:02:15.391776	2015-02-20 17:03:40.994759
122	17965322	Retrospective study	Bacher et al., 2008, Blood	2015-02-20 17:02:15.642669	2015-02-20 17:03:41.79157
123	22368270	Clinical trial	Man et al., 2012, Blood	2015-02-20 17:02:15.786565	2015-02-20 17:03:43.735797
124	17940205	Retrospective study	Whitman et al., 2008, Blood	2015-02-20 17:02:16.03994	2015-02-20 17:03:45.174869
125	22624710		Barbacid, 2012, Cancer Cell	2015-02-20 17:02:16.103474	2015-02-20 17:03:45.550742
126	20805365	Retrospective study	Schnittger et al., 2010, Blood	2015-02-20 17:02:16.251391	2015-02-20 17:03:46.175316
127	20376086	Retrospective study	Ho et al., 2010, Leukemia	2015-02-20 17:02:16.315137	2015-02-20 17:03:46.595339
128	20368538	Retrospective study	Wagner et al., 2010, J. Clin. Oncol.	2015-02-20 17:02:16.373815	2015-02-20 17:03:47.010783
129	19933982	Retrospective study	Dubbink et al., 2009, Neurology	2015-02-20 17:02:16.439141	2015-02-20 17:03:47.43193
130	20127344	Retrospective study	Bleeker et al., 2010, Acta Neuropathol.	2015-02-20 17:02:16.506457	2015-02-20 17:03:50.187637
131	20494930	Retrospective study	Thol et al., 2010, Haematologica	2015-02-20 17:02:16.568841	2015-02-20 17:03:50.670816
132	20538800	Retrospective study	Abbas et al., 2010, Blood	2015-02-20 17:02:16.638464	2015-02-20 17:03:51.08352
133	21596855		Green et al., 2011, Blood	2015-02-20 17:02:16.928757	2015-02-20 17:03:51.535676
134	20421455	Retrospective study	Thol et al., 2010, Blood	2015-02-20 17:02:16.99199	2015-02-20 17:03:52.98816
135	22616558		Zhou et al., 2012, Leuk. Lymphoma	2015-02-20 17:02:17.061348	2015-02-20 17:03:53.382531
136	22033490		Patnaik et al., 2012, Leukemia	2015-02-20 17:02:17.130328	2015-02-20 17:03:53.905391
137	21997850		Lin et al., 2012, Ann. Hematol.	2015-02-20 17:02:17.21734	2015-02-20 17:03:54.254315
138	16081687	Retrospective study	Levine et al., 2005, Blood	2015-02-20 17:02:17.490188	2015-02-20 17:03:55.784022
139	18394554	In vitro study	Wernig et al., 2008, Cancer Cell	2015-02-20 17:02:17.618462	2015-02-20 17:03:56.190391
140	19287384	Retrospective study	Kilpivaara et al., 2009, Nat. Genet.	2015-02-20 17:02:17.834557	2015-02-20 17:03:56.588971
141	16709929	Clinical trial	Kiladjian et al., 2006, Blood	2015-02-20 17:02:17.902294	2015-02-20 17:03:57.039061
142	16384925	Retrospective study	Cairoli et al., 2006, Blood	2015-02-20 17:02:17.968501	2015-02-20 17:03:57.456451
143	16551858	Retrospective study	Wardelmann et al., 2006, Clin. Cancer Res.	2015-02-20 17:02:18.040519	2015-02-20 17:03:58.919172
144	10485475	Retrospective study	Taniguchi et al., 1999, Cancer Res.	2015-02-20 17:02:18.117098	2015-02-20 17:03:59.439849
145	12000708	Retrospective study	Corless et al., 2002, Am. J. Pathol.	2015-02-20 17:02:18.189203	2015-02-20 17:03:59.83985
146	15217946	Retrospective study	Boldrini et al., 2004, Clin. Cancer Res.	2015-02-20 17:02:18.316915	2015-02-20 17:04:00.276312
147	23775962	Clinical trial	Hodi et al., 2013, J. Clin. Oncol.	2015-02-20 17:02:18.381267	2015-02-20 17:04:00.6761
148	19671763	In vitro study	Woodman et al., 2009, Mol. Cancer Ther.	2015-02-20 17:02:18.822422	2015-02-20 17:04:01.008601
149	17372901	In vitro study	Antonescu et al., 2007, Int. J. Cancer	2015-02-20 17:02:19.075691	2015-02-20 17:04:01.398904
150	16954519		Heinrich et al., 2006, J. Clin. Oncol.	2015-02-20 17:02:19.23227	2015-02-20 17:04:01.765553
151	16638875		Prenen et al., 2006, Clin. Cancer Res.	2015-02-20 17:02:19.299218	2015-02-20 17:04:02.138219
152	16618717	Retrospective study	Lièvre et al., 2006, Cancer Res.	2015-02-20 17:02:19.403161	2015-02-20 17:04:02.557084
153	24559322	Clinical trial	Sartore-Bianchi et al., 2014, Expert Rev Anticancer Ther	2015-02-20 17:02:19.47214	2015-02-20 17:04:03.175892
154	15696205	Retrospective study	Pao et al., 2005, PLoS Med.	2015-02-20 17:02:19.54393	2015-02-20 17:04:03.54484
155	19934290	Retrospective study	Ogino et al., 2009, Clin. Cancer Res.	2015-02-20 17:02:19.668064	2015-02-20 17:04:03.934159
156	24571676	Retrospective study	Andrade et al., 2014, BMC Cancer	2015-02-20 17:02:19.734524	2015-02-20 17:04:04.332333
157	18528420	Retrospective study	Chng et al., 2008, Leukemia	2015-02-20 17:02:19.800035	2015-02-20 17:04:05.305605
158	18794081	Retrospective study	Riely et al., 2008, Clin. Cancer Res.	2015-02-20 17:02:19.869937	2015-02-20 17:04:05.755562
159	11208838	Retrospective study	Schiller et al., 2001, J. Clin. Oncol.	2015-02-20 17:02:19.932332	2015-02-20 17:04:06.206082
160	15597105	Metaanalysis	Mascaux et al., 2005, Br. J. Cancer	2015-02-20 17:02:20.003279	2015-02-20 17:04:06.607327
161	18202412	In vitro study	Lièvre et al., 2008, J. Clin. Oncol.	2015-02-20 17:02:20.069147	2015-02-20 17:04:07.986805
162	23014527	Retrospective study	Dogan et al., 2012, Clin. Cancer Res.	2015-02-20 17:02:20.25745	2015-02-20 17:04:08.424526
163	23524406	case report	Rudin et al., 2013, J Thorac Oncol	2015-02-20 17:02:20.38879	2015-02-20 17:04:10.240295
164	19029981	In vivo study	Engelman et al., 2008, Nat. Med.	2015-02-20 17:02:20.46997	2015-02-20 17:04:10.726219
165	20978259	Clinical trial	De Roock et al., 2010, JAMA	2015-02-20 17:02:20.752726	2015-02-20 17:04:11.13885
166	19915144		Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-02-20 17:02:21.035812	2015-02-20 17:04:11.557821
167	19915144	In vitro study	Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-02-20 17:02:21.104953	2015-02-20 17:04:11.977759
168	15758010	Clinical Trial	Hegi et al., 2005, N. Engl. J. Med.	2015-02-20 17:02:21.291742	2015-02-20 17:04:12.394242
169	11070098		Esteller et al., 2000, N. Engl. J. Med.	2015-02-20 17:02:21.352774	2015-02-20 17:04:12.765088
170	16076867	Retrospective study	Schnittger et al., 2005, Blood	2015-02-20 17:02:21.602546	2015-02-20 17:04:13.161656
171	16455956	Retrospective study	Thiede et al., 2006, Blood	2015-02-20 17:02:21.660505	2015-02-20 17:04:14.7898
172	19059939	Retrospective study	Schlenk et al., 2009, Haematologica	2015-02-20 17:02:21.846253	2015-02-20 17:04:15.216181
173	19587375	Retrospective study	Schnittger et al., 2009, Blood	2015-02-20 17:02:22.032135	2015-02-20 17:04:17.013512
174	21719597	In vitro study	Balusu et al., 2011, Blood	2015-02-20 17:02:22.170297	2015-02-20 17:04:17.420042
175	22417203	Retrospective study	Patel et al., 2012, N. Engl. J. Med.	2015-02-20 17:02:22.41293	2015-02-20 17:04:17.826033
176	24797300		Tassara et al., 2014, Blood	2015-02-20 17:02:22.471695	2015-02-20 17:04:18.247787
177	24927407		Ehninger et al., 2014, Blood Cancer J	2015-02-20 17:02:22.576397	2015-02-20 17:04:18.765844
178	16051734	Retrospective study	Döhner et al., 2005, Blood	2015-02-20 17:02:22.757443	2015-02-20 17:04:19.186218
179	20026798	Retrospective study	Becker et al., 2010, J. Clin. Oncol.	2015-02-20 17:02:22.82114	2015-02-20 17:04:19.602587
180	24855211		Linch et al., 2014, Blood	2015-02-20 17:02:22.890694	2015-02-20 17:04:20.12102
181	19047294	Retrospective study	Büchner et al., 2009, J. Clin. Oncol.	2015-02-20 17:02:23.073937	2015-02-20 17:04:20.64391
182	22430270	Retrospective study	Gaidzik et al., 2012, J. Clin. Oncol.	2015-02-20 17:02:23.140157	2015-02-20 17:04:21.004205
183	24859829		Tian et al., 2014, Int. J. Hematol.	2015-02-20 17:02:23.568676	2015-02-20 17:04:21.479504
184	19357394		Vardiman et al., 2009, Blood	2015-02-20 17:02:23.69534	2015-02-20 17:04:23.637276
185	15659725		Falini et al., 2005, N. Engl. J. Med.	2015-02-20 17:02:23.984847	2015-02-20 17:04:25.596835
186	22180178	Clinical trial	Jakob et al., 2012, Cancer	2015-02-20 17:02:24.060672	2015-02-20 17:04:27.505889
187	15951308	Retrospective study	Bowen et al., 2005, Blood	2015-02-20 17:02:24.169826	2015-02-20 17:04:27.899096
188	24666267	Meta analysis	Therkildsen et al., 2014, Acta Oncol	2015-02-20 17:02:24.2391	2015-02-20 17:04:28.375034
189	16434492	Clinical trial	Bacher et al., 2006, Blood	2015-02-20 17:02:24.492914	2015-02-20 17:04:28.895972
190	18375819	Case report	Banerji et al., 2008, Mol. Cancer Ther.	2015-02-20 17:02:24.559071	2015-02-20 17:04:29.31332
191	23861977	Retrospective study	Tschandl et al., 2013, PLoS ONE	2015-02-20 17:02:24.761797	2015-02-20 17:04:29.738063
192	23569304	Clinical trial	Trunzer et al., 2013, J. Clin. Oncol.	2015-02-20 17:02:24.821205	2015-02-20 17:04:30.149851
193	21576590	Case report	Soon et al., 2011, Arch Dermatol	2015-02-20 17:02:24.888009	2015-02-20 17:04:30.670701
194	22745105		Heinrich et al., 2012, Clin. Cancer Res.	2015-02-20 17:02:25.018788	2015-02-20 17:04:31.016742
195	15146165		Lasota et al., 2004, Lab. Invest.	2015-02-20 17:02:25.094542	2015-02-20 17:04:31.406035
196	15928335		Corless et al., 2005, J. Clin. Oncol.	2015-02-20 17:02:25.222893	2015-02-20 17:04:31.87284
197	21558396		Tanaka et al., 2011, Clin. Cancer Res.	2015-02-20 17:02:25.578874	2015-02-20 17:04:32.247784
198	15647370		Kang et al., 2005, Proc. Natl. Acad. Sci. U.S.A.	2015-02-20 17:02:25.636862	2015-02-20 17:04:32.665864
199	19223544		Sartore-Bianchi et al., 2009, Cancer Res.	2015-02-20 17:02:25.807324	2015-02-20 17:04:33.010504
200	22357840		Liao et al., 2012, Clin. Cancer Res.	2015-02-20 17:02:25.990133	2015-02-20 17:04:33.422181
201	8674046		Yoshida et al., 1996, Cancer Res.	2015-02-20 17:02:26.961904	2015-02-20 17:04:33.780699
202	20085938		Courtney et al., 2010, J. Clin. Oncol.	2015-02-20 17:02:27.091944	2015-02-20 17:04:34.129955
203	22479427		Carico et al., 2012, PLoS ONE	2015-02-20 17:02:27.167352	2015-02-20 17:04:35.937805
204	21422803		Coxon et al., 2012, J. Endocrinol. Invest.	2015-02-20 17:02:27.357536	2015-02-20 17:04:36.312397
205	18073307		Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-20 17:02:27.493403	2015-02-20 17:04:36.702492
206	23056499		Couto et al., 2012, PLoS ONE	2015-02-20 17:02:27.553805	2015-02-20 17:04:37.252841
207	9839497		Egawa et al., 1998, Jpn. J. Clin. Oncol.	2015-02-20 17:02:27.620507	2015-02-20 17:04:37.6711
208	21995386		Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-20 17:02:27.683193	2015-02-20 17:04:38.089459
209	22698404		Jackson et al., 2012, Cancer Cell	2015-02-20 17:02:27.815786	2015-02-20 17:04:38.613033
210	16489069		Olivier et al., 2006, Clin. Cancer Res.	2015-02-20 17:02:27.875898	2015-02-20 17:04:40.069049
211	9569050		Berns et al., 1998, Br. J. Cancer	2015-02-20 17:02:27.945066	2015-02-20 17:04:40.489245
212	22923433		Iyer et al., 2012, Science	2015-02-20 17:02:28.573763	2015-02-20 17:04:40.852141
213	19966866	Mouse model	Liang et al., 2010, Oncogene	2015-02-20 17:02:28.658941	2015-02-20 17:04:42.44895
214	22923433	Retrospective study	Iyer et al., 2012, Science	2015-02-20 17:02:28.725346	2015-02-20 17:04:42.95145
215	23029227		Qian et al., 2012, PLoS ONE	2015-02-20 17:02:28.91103	2015-02-20 17:04:43.334741
216	23861105		Wu et al., 2013, Am. J. Hematol.	2015-02-20 17:02:29.106186	2015-02-20 17:04:43.694296
217	18591546	Retrospective study	Virappane et al., 2008, J. Clin. Oncol.	2015-02-20 17:02:29.422905	2015-02-20 17:04:44.071434
218	19221039	Retrospective study	Gaidzik et al., 2009, Blood	2015-02-20 17:02:29.49378	2015-02-20 17:04:44.460374
219	18559874	Retrospective study	Paschka et al., 2008, J. Clin. Oncol.	2015-02-20 17:02:29.753789	2015-02-20 17:04:44.876636
220	19536888	Retrospective study	Renneville et al., 2009, Cancer	2015-02-20 17:02:29.884346	2015-02-20 17:04:45.293096
\.


--
-- Data for Name: variant_origins; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_origins (id, origin) FROM stdin;
1	Somatic
\.


--
-- Data for Name: variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variants (id, gene_id, name, description, created_at, updated_at) FROM stdin;
1	4	BCR-ABL	The BCR-ABL fusion protein, commonly referred to as the philadelphia chromosome, is the most well-studied fusion gene in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML), but despite its ability initiate disease in mice, its status an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and nilotinib) have seen some success in delivering a tumor response.	2015-02-20 17:01:58.584417	2015-02-20 17:01:58.584417
2	4	BCR-ABL T315I	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-02-20 17:01:58.621781	2015-02-20 17:01:58.621781
3	4	BCR-ABL E255K	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-02-20 17:01:58.657083	2015-02-20 17:01:58.657083
4	2	E17K	AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal, lung, and ovarian cancer. It has been convincingly shown to be an activating mutation resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and other AKT1 mutations, are the subject of much research and development for therapeutics.	2015-02-20 17:01:58.68291	2015-02-20 17:01:58.68291
5	1	EML4-ALK	The EML4-ALK fusion has been seen in non-small cell lung cancer, and appears to be an alternative mechanism for ALK activation. Cells with this fusion have been shown to be sensitive to the ALK inhibitor crizotinib.	2015-02-20 17:01:58.732016	2015-02-20 17:01:58.732016
6	1	C1156Y	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-20 17:01:58.759422	2015-02-20 17:01:58.759422
7	1	L1196M	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-20 17:01:58.78668	2015-02-20 17:01:58.78668
8	1	F1174L	ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells.	2015-02-20 17:01:58.80986	2015-02-20 17:01:58.80986
9	1	R1275Q	ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma, non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This, and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC cell lines.	2015-02-20 17:01:58.83397	2015-02-20 17:01:58.83397
10	3	S214C	ARAF S214C has been found to be a recurrent oncogenic mutation in non-small cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this mutation, sorafenib also acheived near-complete clinical remission. This case has brought more interest to the variant from a research and clinical perspective.	2015-02-20 17:01:58.858603	2015-02-20 17:01:58.858603
11	5	V600D	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-20 17:01:58.895954	2015-02-20 17:01:58.895954
12	5	V600E	BRAF V600E has been shown to be recurrent in many cancer types. It is one of the most widely studied variants in cancer. This variant is correlated with poor prognosis in certain cancer types, including colorectal cancer and papillary thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib has also shown to be effective when combined with the MEK inhibitor trametinib in colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib, and a number of combination therapies have been successful in treating V600E mutations. While the drugs cetuximab and panitumumab have been largely shown to be ineffective without supplementary treatment.	2015-02-20 17:01:58.927147	2015-02-20 17:01:58.927147
13	5	V600E/V600M	A case study of a single patient harboring both a V600E and a V600M mutation, dabrafenib was shown to acheive clinical response.	2015-02-20 17:01:58.954656	2015-02-20 17:01:58.954656
14	5	V600E AMPLIFICATION	Amplification of BRAF V600E has been shown to confer resistance to MEK inhibitors. For more information on the V600 locus, see the V600E entry.	2015-02-20 17:01:58.983838	2015-02-20 17:01:58.983838
15	5	V600M	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-20 17:01:59.010189	2015-02-20 17:01:59.010189
16	5	V600R	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-20 17:01:59.036036	2015-02-20 17:01:59.036036
17	5	V600	BRAF mutations of the valine 600 residue have been shown to be recurrent in many cancer types. Of the V600 mutations, V600E is the most widely researched. V600 mutations as a whole have been correlated to poorer prognosis in colorectal and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual mutation pages on the left sidebar.	2015-02-20 17:01:59.067811	2015-02-20 17:01:59.067811
18	6	AMPLIFICATION	CCND1 amplification has been implicated in poorer prognosis in non-small cell lung cancer.	2015-02-20 17:01:59.099411	2015-02-20 17:01:59.099411
19	6	EXPRESSION	CCND1 expression, and its prognositc impact, is still in dispute. Three experiments in non-small cell lung cancer have shown it to have no impact on survival, but three additional studies have shown it results in poorer prognosis. There is also some ambiguity in how the boundaries between expression and overexpression are defined.	2015-02-20 17:01:59.125092	2015-02-20 17:01:59.125092
75	27	EXON 9	c-KIT exon 9 mutations lie within the dimerization motif of the protein. Relative to other KIT mutations, exon 9 mutations have been associated with better overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and in vivo.	2015-02-20 17:02:00.856481	2015-02-20 17:02:00.856481
97	34	G13D	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-20 17:02:01.48148	2015-02-20 17:02:01.48148
20	6	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-20 17:01:59.150446	2015-02-20 17:01:59.150446
21	7	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-20 17:01:59.178128	2015-02-20 17:01:59.178128
22	7	PROMOTER DEMETHYLATION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-20 17:01:59.225167	2015-02-20 17:01:59.225167
23	8	LOSS	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL), a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic palbociclib significantly increased the median survival of a Notch-driven model of T-ALL.	2015-02-20 17:01:59.265491	2015-02-20 17:01:59.265491
24	9	OVEREXPRESSION	Cyclin E, while currenly not as widely implicated as its counterpart, cyclin D, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, are correlated with later stage disease and have been shown to lead to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	2015-02-20 17:01:59.303084	2015-02-20 17:01:59.303084
25	11	EXPRESSION	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-20 17:01:59.338698	2015-02-20 17:01:59.338698
26	10	EXPRESSION	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-20 17:01:59.368728	2015-02-20 17:01:59.368728
27	12	PROMOTER HYPERMETHYLATION	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the CDKN2A promoter region. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcomes for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	2015-02-20 17:01:59.397722	2015-02-20 17:01:59.397722
28	13	N-TERMINAL FRAME SHIFT	CEBPA N-terminal frame shift mutations that result in a premature stop codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML). CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-20 17:01:59.424852	2015-02-20 17:01:59.424852
29	13	MUTATION	CEBPA mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-20 17:01:59.453228	2015-02-20 17:01:59.453228
30	31	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-20 17:01:59.499959	2015-02-20 17:01:59.499959
31	14	DNAJB1-PRKACA	This fusion has been found to be very recurrent in the rare form of adolescent liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-20 17:01:59.527187	2015-02-20 17:01:59.527187
32	16	R882	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-02-20 17:01:59.555508	2015-02-20 17:01:59.555508
76	27	L576P	KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain. It is one of the most recurrent KIT mutations in melanoma, and both in vitro and in vivo studies have shown sensitivity to imatinib.	2015-02-20 17:02:00.882101	2015-02-20 17:02:00.882101
33	17	L858R	EGFR L858R has long been recognized as a functionally significant mutation in cancer, and is one of the most prevalent single mutations in lung cancer. Best described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity to first and second generation TKI's like gefitinib and neratinib. NSCLC patients with this mutation treated with TKI's show increased overall and progression-free survival, as compared to chemotherapy alone. Third generation TKI's are currently in clinical trials that specifically focus on mutant forms of EGFR, a few of which have shown efficacy in treating patients that failed to respond to earlier generation TKI therapies.	2015-02-20 17:01:59.584238	2015-02-20 17:01:59.584238
34	17	T790M	EGFR T790M was one of the very first mutations recognized to confer resistance to targeted therapies. While successful in amplified EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients harboring this mutation before treatment is notably lower. This lack of efficacy can likely be to blame for the poorer prognosis for patients with this mutation as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired resistance to TKI inhibition have been shown to harbor this mutation, implicating it as a mechanism of acquired therapy resistence. The third generation TKI's are being developed with this resistance problem in mind, and early stage experiments have shown some efficacy of these drugs in previously resistant tumors with mutant EGFR.	2015-02-20 17:01:59.611653	2015-02-20 17:01:59.611653
35	18	D769H	ERBB2 D769H was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 17:01:59.645695	2015-02-20 17:01:59.645695
36	18	D769Y	ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 17:01:59.67129	2015-02-20 17:01:59.67129
37	18	DEL 755-759	ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 17:01:59.700901	2015-02-20 17:01:59.700901
38	18	G309A	ERBB2 G309A was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 17:01:59.729839	2015-02-20 17:01:59.729839
39	18	L755S	ERBB2 L755S was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike many of the other variants queried. This mutation was also shown to confer resistance to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.	2015-02-20 17:01:59.757573	2015-02-20 17:01:59.757573
40	18	L755W	ERBB2 L755W was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 17:01:59.783831	2015-02-20 17:01:59.783831
41	18	P780INS	ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 17:01:59.808438	2015-02-20 17:01:59.808438
42	18	R678Q	ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 17:01:59.832599	2015-02-20 17:01:59.832599
43	18	R896C	ERBB2 R896C was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 17:01:59.865609	2015-02-20 17:01:59.865609
44	18	V777L	ERBB2 V777L was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 17:01:59.892055	2015-02-20 17:01:59.892055
45	18	V842I	ERBB2 V842I was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 17:01:59.920411	2015-02-20 17:01:59.920411
46	19	L536Q	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of these ligand binding domain mutations, and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 17:01:59.948477	2015-02-20 17:01:59.948477
47	19	N538G	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 17:01:59.975086	2015-02-20 17:01:59.975086
48	19	Y537C	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 17:02:00.009931	2015-02-20 17:02:00.009931
49	19	Y537N	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 17:02:00.036682	2015-02-20 17:02:00.036682
50	19	Y537S	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 17:02:00.067023	2015-02-20 17:02:00.067023
51	20	FGFR2-AFF3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 17:02:00.095274	2015-02-20 17:02:00.095274
52	20	FGFR2-BICC1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 17:02:00.124592	2015-02-20 17:02:00.124592
53	20	FGFR2-CASP7	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 17:02:00.152648	2015-02-20 17:02:00.152648
54	20	FGFR2-CCDC6	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 17:02:00.197915	2015-02-20 17:02:00.197915
55	20	FGFR2-KIAA1967	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 17:02:00.225347	2015-02-20 17:02:00.225347
56	20	FGFR2-MGEA5	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. Additionally, tumor size reduction was achieved by both ponatinib and pazopanib treatments administered separately in a single patient with intrahepatic cholangiocarcinoma and this fusion. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 17:02:00.255994	2015-02-20 17:02:00.255994
57	20	FGFR2-OFD1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 17:02:00.28691	2015-02-20 17:02:00.28691
58	20	FGFR2-SLC45A3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 17:02:00.312143	2015-02-20 17:02:00.312143
59	21	FGFR3-TACC3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 17:02:00.343352	2015-02-20 17:02:00.343352
60	22	ITD	FLT3-ITD (internal tandem duplications) frequently occur in patients with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated sequence resulting in an in-frame duplication event. The length of these duplications can vary widely which may have prognostic consequences, but this has not been conclusively determined. FLT3-ITD mutations overall have generally been associated with poor prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the prognosis associated with this variant.	2015-02-20 17:02:00.368418	2015-02-20 17:02:00.368418
61	22	TKD	FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common than FLT3-ITD (internal tandem duplication) mutations and may not confer the same prognostic impact. Although the majority of mutations are point mutations effecting D835 (most frequently D835Y), a small proportion involve an in-frame deletion of I836. These mutations are within the activation loop of the second tyrosine kinase domain of FLT3 and thought to result in constitutive activation of the receptor.	2015-02-20 17:02:00.400739	2015-02-20 17:02:00.400739
62	23	EXPRESSION	GATA2 misregulation has been observed in a number of hematologic malignancies, as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung cancer.	2015-02-20 17:02:00.435299	2015-02-20 17:02:00.435299
63	24	R132	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-20 17:02:00.464831	2015-02-20 17:02:00.464831
64	24	R132C	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-20 17:02:00.500067	2015-02-20 17:02:00.500067
65	24	R132H	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-20 17:02:00.534777	2015-02-20 17:02:00.534777
66	24	R132L	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-20 17:02:00.570965	2015-02-20 17:02:00.570965
67	25	R140Q/L	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival as compared to their wild-type counterparts. In myelodysplastic syndromes, however, no prognostic link was discovered between IDH2 mutation status and overall survival.	2015-02-20 17:02:00.608676	2015-02-20 17:02:00.608676
68	25	R172K	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic syndromes, studies did not find a prognostic association between this variant and patient outcomes.	2015-02-20 17:02:00.645236	2015-02-20 17:02:00.645236
69	26	V617F	JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases, occuring in around half of all MPD's. While less associated with cancer, when it is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The V617F mutation is an activating mutation, resulting in increased kinase activity. The mutation seems to be restricted to hematologic malignancies. Treatment of JAK mutant diseases with ruxolitinib has seen some clinical success.	2015-02-20 17:02:00.678727	2015-02-20 17:02:00.678727
70	27	D816V	KIT D816V is a mutation observed in acute myeloid leukemia (AML). This variant has been linked to poorer prognosis and worse outcome in AML patients.	2015-02-20 17:02:00.711004	2015-02-20 17:02:00.711004
71	27	EXON 11	c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-02-20 17:02:00.74016	2015-02-20 17:02:00.74016
72	27	EXON 13	c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations. These mutations lie within the tyrosine kinase 1 domain, and are found primarily in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations show sensitivity to imatinib and sunitinib treatment. However, only imatinib has seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant cases.	2015-02-20 17:02:00.768052	2015-02-20 17:02:00.768052
73	27	EXON 14	c-KIT exon 14 mutations lie within the tyrosine kinase domain of the protein. While relatively rare in primary gastrointestinal tumors, they are notably more prevalent in refractory disease, suggesting a role in imatinib resistance. Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.	2015-02-20 17:02:00.794052	2015-02-20 17:02:00.794052
74	27	EXON 17	c-KIT exon 17 mutations lie within the TK2 domain, containing the activation loop of the protein. In cell lines, mutations within this domain have been shown to be sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.	2015-02-20 17:02:00.8234	2015-02-20 17:02:00.8234
77	27	V654A	KIT V654A is an exon 13 mutation that lies within the tyrosine kinase 1 domain of the protein. It has been shown to be an activating mutation by in vitro studies. This mutation is associated with imatinib resistance in melanoma patients. However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have seen success in acheiving tumor response.	2015-02-20 17:02:00.909947	2015-02-20 17:02:00.909947
78	28	EXON 1	A study by Lièvre et al in 2006 showed that colorectal cancer patients with KRAS exon 1 mutations had low cetuximab response rates.	2015-02-20 17:02:00.958149	2015-02-20 17:02:00.958149
79	28	EXON 2	In a study by Pan et al in 2005, non-small cell lung cancer patients harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors gefinitib and erlotinib.	2015-02-20 17:02:00.990455	2015-02-20 17:02:00.990455
80	28	G12	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 17:02:01.016532	2015-02-20 17:02:01.016532
81	28	G12/G13	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 17:02:01.045192	2015-02-20 17:02:01.045192
82	28	G12C	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 17:02:01.074708	2015-02-20 17:02:01.074708
83	28	G12D	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 17:02:01.100562	2015-02-20 17:02:01.100562
84	28	G13	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 17:02:01.13481	2015-02-20 17:02:01.13481
85	28	G13D	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 17:02:01.162692	2015-02-20 17:02:01.162692
86	29	P124S	MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder and colon cancer to a lesser degree. The P124S mutation has been shown to contribute to AZD6244 resistance in melanoma cell lines, but considerably less so than its Q56P counterpart.	2015-02-20 17:02:01.188502	2015-02-20 17:02:01.188502
87	29	Q56P	MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This mutation has been shown to confer considerable resistance to AZD6244 treatment of melanoma cell lines.	2015-02-20 17:02:01.213172	2015-02-20 17:02:01.213172
88	30	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-20 17:02:01.237316	2015-02-20 17:02:01.237316
89	32	PROMOTER METHYLATION	MGMT promoter methylation has been observed to impact tumor progression in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating agent carmustine has shown efficacy. In patients lacking methylation, combining carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more experiments are required. Clinical trials have also shown selective sensitivity of promoter methylation-positive patients to temozolomide, making a case for wider methylation screening in GBM patients.	2015-02-20 17:02:01.265433	2015-02-20 17:02:01.265433
90	33	EXON 12	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-02-20 17:02:01.296386	2015-02-20 17:02:01.296386
91	33	W288FS	NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most common NPM1 mutation identified in acute myeloid leukemia. This mutation results in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation, it is generally grouped with other exon 12 mutations for patient analysis (see NPM1 Exon 12 variants for more information).	2015-02-20 17:02:01.322936	2015-02-20 17:02:01.322936
92	34	EXON 1	NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown to be correlated with poorer overall survival relative to wild-type NRAS in melanoma patients.	2015-02-20 17:02:01.350654	2015-02-20 17:02:01.350654
93	34	EXON 2	NRAS exon 2 mutations have been shown to be correlated with poorer overall survival in melanoma patients and colorectal cancer patients, however no prognostic impact was seen in acute myeloid leukemia patients.	2015-02-20 17:02:01.374444	2015-02-20 17:02:01.374444
94	34	EXON 3	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-20 17:02:01.402695	2015-02-20 17:02:01.402695
95	34	EXON 4	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-20 17:02:01.428543	2015-02-20 17:02:01.428543
96	34	G12	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-20 17:02:01.455936	2015-02-20 17:02:01.455936
98	34	Q61	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-20 17:02:01.505282	2015-02-20 17:02:01.505282
99	34	Q61L	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-20 17:02:01.529409	2015-02-20 17:02:01.529409
100	34	Q61R	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-20 17:02:01.559516	2015-02-20 17:02:01.559516
101	36	D842I	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 17:02:01.585549	2015-02-20 17:02:01.585549
102	36	D842V	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 17:02:01.633547	2015-02-20 17:02:01.633547
103	36	D842Y	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 17:02:01.661689	2015-02-20 17:02:01.661689
104	36	DEL I843	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 17:02:01.686073	2015-02-20 17:02:01.686073
105	36	DI842-843IM	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 17:02:01.713879	2015-02-20 17:02:01.713879
106	35	E542K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-20 17:02:01.742171	2015-02-20 17:02:01.742171
107	35	E545K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-20 17:02:01.768941	2015-02-20 17:02:01.768941
108	35	EXON 20	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-20 17:02:01.795883	2015-02-20 17:02:01.795883
109	35	EXON 9	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-20 17:02:01.822929	2015-02-20 17:02:01.822929
110	35	H1047R	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-20 17:02:01.854005	2015-02-20 17:02:01.854005
111	37	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	2015-02-20 17:02:01.882143	2015-02-20 17:02:01.882143
112	15	DNAJB1-PRKACA	This fusion has been found to be very recurrent in a rare form of adolescent liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined, and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-20 17:02:01.911217	2015-02-20 17:02:01.911217
113	39	R233*	PTEN R233* has been shown to be a loss of function mutation, and PTEN loss has been the subject of considerable research in breast cancer. PTEN loss may sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data to support that PTEN loss is both associated with poorer prognosis, and no change in prognosis.	2015-02-20 17:02:01.944233	2015-02-20 17:02:01.944233
114	38	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and Arsenic Trioxide for treating these patients, and early results seem promising.	2015-02-20 17:02:01.970647	2015-02-20 17:02:01.970647
115	40	C634W	RET C639W has been implicated as an alternate mechanism of activating RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the C639W mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib.	2015-02-20 17:02:02.004503	2015-02-20 17:02:02.004503
116	40	M918T	RET M819T is the most common somatically acquired mutation in medullary thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the M918T mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T leads to more aggressive MTC with a poorer prognosis.	2015-02-20 17:02:02.034633	2015-02-20 17:02:02.034633
117	42	K666N	SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients.	2015-02-20 17:02:02.061045	2015-02-20 17:02:02.061045
118	42	K700E	SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients. Additionally, these mutations are the most common SF3B1 mutation observed in MDS and highly associated with subtypes of MDS that are defined by ringed sideroblasts.	2015-02-20 17:02:02.08489	2015-02-20 17:02:02.08489
119	43	R175H	While loss-of-function events in TP53 are very common in cancer, the R175H variant seems not only to result in loss of tumor-suppression, but also acts as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell lines harboring this mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been shown that the R175H mutation is correlated with worse overall survival than wild-type TP53, but is not as detrimental as the R248W variant.	2015-02-20 17:02:02.112537	2015-02-20 17:02:02.112537
120	43	R248Q	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival. The R248Q mutation has also shown an increased invasive behavior in cell lines. This is specific to the 248Q variant.	2015-02-20 17:02:02.145069	2015-02-20 17:02:02.145069
121	43	R248W	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival.	2015-02-20 17:02:02.174448	2015-02-20 17:02:02.174448
122	43	R249T	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 17:02:02.20197	2015-02-20 17:02:02.20197
123	43	R249W	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 17:02:02.231717	2015-02-20 17:02:02.231717
124	43	R273C	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 17:02:02.257173	2015-02-20 17:02:02.257173
125	43	R273H	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 17:02:02.298078	2015-02-20 17:02:02.298078
126	43	V173G/A	While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 17:02:02.325889	2015-02-20 17:02:02.325889
127	44	FRAMESHIFT TRUNCATION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-20 17:02:02.351419	2015-02-20 17:02:02.351419
128	44	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-20 17:02:02.375863	2015-02-20 17:02:02.375863
129	45	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-20 17:02:02.404131	2015-02-20 17:02:02.404131
130	46	Q157P/R	U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is less common than the S34F mutation, occurs in the second zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-02-20 17:02:02.42954	2015-02-20 17:02:02.42954
131	46	S34Y/F	U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is the most commonly identified variant in MDS, occurs in the first zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-02-20 17:02:02.458674	2015-02-20 17:02:02.458674
132	47	EXON 7	WT1 exon 7 mutations have been shown to be recurrent in acute myeloid leukemia. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 7 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-20 17:02:02.491167	2015-02-20 17:02:02.491167
133	47	EXON 9	WT1 exon 9 mutations have been shown to be recurrent in acute myeloid leukemia, although at a less frequent rate than their exon 7 counterparts. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 9 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-20 17:02:02.520295	2015-02-20 17:02:02.520295
\.


--
-- Data for Name: evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_items (id, text, clinical_significance, evidence_direction, evidence_type_id, evidence_level_id, drug_id, disease_id, source_id, variant_id, created_at, updated_at, rating, status, remote_errors, remote_ids, variant_origin_id) FROM stdin;
1	BCR-ABL fusions have been found to contribute to imatinib resistance in AML cell lines.		Supports	1	1	1	1	41	1	2015-02-20 17:02:02.666906	2015-02-20 17:02:02.666906	3	accepted	\N	\N	1
2	Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide results in high rates of complete remission in patients.		Supports	1	2	2	1	41	1	2015-02-20 17:02:02.728624	2015-02-20 17:02:02.728624	5	accepted	\N	\N	1
3	The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease.		Supports	2	2	3	2	42	1	2015-02-20 17:02:02.792956	2015-02-20 17:02:02.792956	4	accepted	\N	\N	1
4	Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML.		Does Not Support	2	3	3	2	43	1	2015-02-20 17:02:02.859563	2015-02-20 17:02:02.859563	2	accepted	\N	\N	1
5	The clinical use of imatinib in patients with BCR-ABL fusion has resulted in drastic sensitivity to the drug.		Supports	1	2	1	2	42	1	2015-02-20 17:02:02.944424	2015-02-20 17:02:02.944424	5	accepted	\N	\N	1
6	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.		Supports	1	1	4	2	42	1	2015-02-20 17:02:03.014682	2015-02-20 17:02:03.014682	4	accepted	\N	\N	1
7	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.		Supports	1	1	5	2	42	1	2015-02-20 17:02:03.075004	2015-02-20 17:02:03.075004	4	accepted	\N	\N	1
8	BCR-ABL inhibitors such as imatinib have lead to "significantly improved prognosis, response rate, overall survival, and patient outcome in CML patients compared to previous therapeutic regimens."		Supports	3	2	1	2	42	1	2015-02-20 17:02:03.136969	2015-02-20 17:02:03.136969	5	accepted	\N	\N	1
9	COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations were shown to be resistant to imatinib and exhibited increased kinase activity.		Supports	1	3	1	2	44	3	2015-02-20 17:02:03.205171	2015-02-20 17:02:03.205171	4	accepted	\N	\N	1
10	In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have shown to be resistant to imatinib treatment.		Supports	1	1	1	2	42	2	2015-02-20 17:02:03.303854	2015-02-20 17:02:03.303854	4	accepted	\N	\N	1
11	COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations were shown to be resistant to imatinib and exhibited increased kinase activity.		Supports	1	3	1	2	44	2	2015-02-20 17:02:03.372545	2015-02-20 17:02:03.372545	4	accepted	\N	\N	1
12	PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the AKT inhibitor MK-2206 in breast cancer cell lines.		Does Not Support	1	1	6	3	45	4	2015-02-20 17:02:03.448238	2015-02-20 17:02:03.448238	3	accepted	\N	\N	1
13	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y variant has been shown to confer resistance to crizotinib.		Supports	1	1	7	4	46	6	2015-02-20 17:02:03.539674	2015-02-20 17:02:03.539674	4	accepted	\N	\N	1
14	In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment with crizotinib has shown to be effective in acheiving stable disease.		Supports	1	1	7	4	46	5	2015-02-20 17:02:03.619308	2015-02-20 17:02:03.619308	4	accepted	\N	\N	1
15	Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without this mutation.		Supports	1	3	7	5	47	8	2015-02-20 17:02:03.695745	2015-02-20 17:02:03.695745	3	accepted	\N	\N	1
16	CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.		Supports	1	3	8	6	48	8	2015-02-20 17:02:03.764412	2015-02-20 17:02:03.764412	3	accepted	\N	\N	1
17	High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing the EML4-ALK fusion containing the F1174L mutation.		Supports	1	3	7	6	49	8	2015-02-20 17:02:03.82575	2015-02-20 17:02:03.82575	3	accepted	\N	\N	1
18	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation.		Supports	1	3	7	6	50	8	2015-02-20 17:02:03.889398	2015-02-20 17:02:03.889398	3	accepted	\N	\N	1
19	SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition		Supports	1	3	9	6	51	8	2015-02-20 17:02:03.96219	2015-02-20 17:02:03.96219	4	accepted	\N	\N	1
20	Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without this mutation.		Supports	1	3	7	4	47	8	2015-02-20 17:02:04.028063	2015-02-20 17:02:04.028063	3	accepted	\N	\N	1
21	CH5424802 treatment resulted in significant tumor regression in xenograft models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation		Supports	1	3	8	4	48	7	2015-02-20 17:02:04.097431	2015-02-20 17:02:04.097431	3	accepted	\N	\N	1
22	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M variant has been shown to confer resistance to crizotinib.		Supports	1	1	7	4	46	7	2015-02-20 17:02:04.183535	2015-02-20 17:02:04.183535	4	accepted	\N	\N	1
23	TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.		Supports	1	3	9	6	51	9	2015-02-20 17:02:04.253802	2015-02-20 17:02:04.253802	3	accepted	\N	\N	1
24	The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.		Does Not Support	1	3	9	6	51	9	2015-02-20 17:02:04.310965	2015-02-20 17:02:04.310965	3	accepted	\N	\N	1
25	In one patient with S214C mutation, the use of sorafenib has led to more than 5 years of survival and near remission.		Supports	1	1	10	4	52	10	2015-02-20 17:02:04.386681	2015-02-20 17:02:04.386681	2	accepted	\N	\N	1
26	Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro		Supports	1	3	11	4	52	10	2015-02-20 17:02:04.461067	2015-02-20 17:02:04.461067	3	accepted	\N	\N	1
27	Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro		Supports	1	3	10	4	52	10	2015-02-20 17:02:04.52829	2015-02-20 17:02:04.52829	3	accepted	\N	\N	1
28	In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations were associated with poor progression free survival regardless of treatment (panitumumab with best supportive care or best supportive care alone)		Supports	1	1	12	7	53	17	2015-02-20 17:02:04.593435	2015-02-20 17:02:04.593435	3	accepted	\N	\N	1
29	BRAF mutations are associated with melanoma arising in non-chronic sun damaged skin and with superficial spreading melanoma		Supports	2	1	3	8	54	17	2015-02-20 17:02:04.656566	2015-02-20 17:02:04.656566	4	accepted	\N	\N	1
30	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone		Supports	1	1	13	8	55	17	2015-02-20 17:02:04.757569	2015-02-20 17:02:04.757569	3	accepted	\N	\N	1
31	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone		Supports	1	1	11	8	55	17	2015-02-20 17:02:04.811377	2015-02-20 17:02:04.811377	3	accepted	\N	\N	1
32	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.		Supports	1	1	13	8	56	11	2015-02-20 17:02:04.87419	2015-02-20 17:02:04.87419	5	accepted	\N	\N	1
33	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.		Supports	1	3	14	9	57	12	2015-02-20 17:02:04.950999	2015-02-20 17:02:04.950999	3	accepted	\N	\N	1
34	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.		Supports	1	3	12	9	57	12	2015-02-20 17:02:05.005162	2015-02-20 17:02:05.005162	3	accepted	\N	\N	1
35	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.		Supports	1	3	10	9	57	12	2015-02-20 17:02:05.055322	2015-02-20 17:02:05.055322	3	accepted	\N	\N	1
36	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation		Supports	1	3	15	9	58	12	2015-02-20 17:02:05.132402	2015-02-20 17:02:05.132402	3	accepted	\N	\N	1
37	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation		Supports	1	3	16	9	58	12	2015-02-20 17:02:05.186937	2015-02-20 17:02:05.186937	3	accepted	\N	\N	1
38	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy		Supports	1	3	16	9	59	12	2015-02-20 17:02:05.277753	2015-02-20 17:02:05.277753	2	accepted	\N	\N	1
39	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy		Supports	1	3	17	9	59	12	2015-02-20 17:02:05.329202	2015-02-20 17:02:05.329202	2	accepted	\N	\N	1
40	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.		Supports	1	3	18	7	60	12	2015-02-20 17:02:05.396411	2015-02-20 17:02:05.396411	2	accepted	\N	\N	1
41	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.		Supports	1	3	19	7	60	12	2015-02-20 17:02:05.449553	2015-02-20 17:02:05.449553	2	accepted	\N	\N	1
42	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.		Supports	1	3	20	7	60	12	2015-02-20 17:02:05.50614	2015-02-20 17:02:05.50614	2	accepted	\N	\N	1
43	Cell lines expressing the BRAF V600E mutation responded better to vemurafenib treatment than cells wildtype for BRAF as measured by reduced cellular proliferation and inhibition of MET and ERK phosphorylation		Supports	1	3	19	7	60	12	2015-02-20 17:02:05.569306	2015-02-20 17:02:05.569306	2	accepted	\N	\N	1
44	V600E is associated with adverse pathological features of colorectal cancer. This can be concluded as a marker of poor prognosis.		Supports	3	2	3	7	61	12	2015-02-20 17:02:05.633092	2015-02-20 17:02:05.633092	5	accepted	\N	\N	1
45	Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype BRAF		Supports	1	1	14	7	62	12	2015-02-20 17:02:05.692513	2015-02-20 17:02:05.692513	3	accepted	\N	\N	1
46	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP but does not show resistance to MEK inhibitors.		Does Not Support	1	3	3	8	63	12	2015-02-20 17:02:05.75306	2015-02-20 17:02:05.75306	3	accepted	\N	\N	1
47	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP and resistance to RAF inhibitors.		Supports	1	3	3	8	63	12	2015-02-20 17:02:05.848111	2015-02-20 17:02:05.848111	3	accepted	\N	\N	1
48	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.		Does Not Support	1	1	21	8	64	12	2015-02-20 17:02:05.914789	2015-02-20 17:02:05.914789	2	accepted	\N	\N	1
49	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.		Does Not Support	1	1	22	8	64	12	2015-02-20 17:02:05.969268	2015-02-20 17:02:05.969268	2	accepted	\N	\N	1
50	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.		Supports	1	1	13	8	65	12	2015-02-20 17:02:06.0394	2015-02-20 17:02:06.0394	5	accepted	\N	\N	1
51	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.		Supports	1	1	11	8	65	12	2015-02-20 17:02:06.091427	2015-02-20 17:02:06.091427	5	accepted	\N	\N	1
52	BRAF V600E is correlated with shorter disease-free and overall Survival in a Spanish cohort of melanoma patients.		Supports	3	1	3	8	66	12	2015-02-20 17:02:06.148275	2015-02-20 17:02:06.148275	3	accepted	\N	\N	1
53	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.		Supports	1	1	13	4	67	12	2015-02-20 17:02:06.22005	2015-02-20 17:02:06.22005	2	accepted	\N	\N	1
54	thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.		Supports	2	1	3	10	68	12	2015-02-20 17:02:06.285396	2015-02-20 17:02:06.285396	5	accepted	\N	\N	1
55	thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.		Supports	2	1	3	10	69	12	2015-02-20 17:02:06.364015	2015-02-20 17:02:06.364015	5	accepted	\N	\N	1
56	BRAF V600E is shown to be associated with the tall-cell variant of papillary thyroid cancer		Supports	2	1	3	10	70	12	2015-02-20 17:02:06.428207	2015-02-20 17:02:06.428207	3	accepted	\N	\N	1
57	Unlike other studies that suggest a poorer outcome, BRAF mutation in this study was not correlated with poorer prognosis in papillary thyroid cancer.		Does Not Support	3	1	3	10	71	12	2015-02-20 17:02:06.499133	2015-02-20 17:02:06.499133	5	accepted	\N	\N	1
58	V600E is correlated with disease recurrence in both age cohorts (>65 and <65 yo).		Supports	3	1	3	10	70	12	2015-02-20 17:02:06.567874	2015-02-20 17:02:06.567874	3	accepted	\N	\N	1
59	BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in a study of 187 patients with PTC and other thyroid diseases.		Supports	3	1	3	10	72	12	2015-02-20 17:02:06.631226	2015-02-20 17:02:06.631226	3	accepted	\N	\N	1
60	BRAF mutation correlated with poor prognosis in papillary thyroid cancer in both older (>65 yo) and younger (<65 yo) cohorts.		Supports	3	1	3	10	70	12	2015-02-20 17:02:06.694875	2015-02-20 17:02:06.694875	3	accepted	\N	\N	1
61	COLO201 and COLO206F cell harboring BRAF V600E mutations were cloned to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification of the BRAF V600E gene.		Supports	1	4	23	7	73	14	2015-02-20 17:02:06.763342	2015-02-20 17:02:06.763342	4	accepted	\N	\N	1
62	A single patient with BRAF V600E/V600M bi-allelic mutation responded to the V600E drug dabrafenib.		Supports	1	1	13	8	74	13	2015-02-20 17:02:06.836577	2015-02-20 17:02:06.836577	3	accepted	\N	\N	1
63	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.		Supports	1	1	13	8	56	15	2015-02-20 17:02:06.932348	2015-02-20 17:02:06.932348	5	accepted	\N	\N	1
64	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.		Supports	1	1	13	8	56	16	2015-02-20 17:02:06.99372	2015-02-20 17:02:06.99372	5	accepted	\N	\N	1
65	Inceased copy number of CCND1 is associated with poorer overall survival.		Supports	3	1	3	4	75	18	2015-02-20 17:02:07.060414	2015-02-20 17:02:07.060414	3	accepted	\N	\N	1
66	Three studies have associated CCND1 expression with poorer survival.		Supports	3	1	3	4	75	19	2015-02-20 17:02:07.118291	2015-02-20 17:02:07.118291	3	accepted	\N	\N	1
67	Three studies have found no significant survival impact for CCND1 in lung cancer.		Does Not Support	3	1	3	4	75	19	2015-02-20 17:02:07.186589	2015-02-20 17:02:07.186589	3	accepted	\N	\N	1
68	Cyclin D1 overexpression is associated with the ER-positive subtype of breast cancer.		Supports	2	1	3	11	76	20	2015-02-20 17:02:07.252202	2015-02-20 17:02:07.252202	4	accepted	\N	\N	1
69	In ER-positive breast cancer patients, Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis.		Supports	3	1	3	11	76	20	2015-02-20 17:02:07.328854	2015-02-20 17:02:07.328854	4	accepted	\N	\N	1
70	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in head and neck squamous cell carcinoma.		Supports	3	1	3	12	77	20	2015-02-20 17:02:07.426496	2015-02-20 17:02:07.426496	4	accepted	\N	\N	1
71	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in mantle cell lymphoma.		Supports	3	1	3	13	78	20	2015-02-20 17:02:07.491883	2015-02-20 17:02:07.491883	4	accepted	\N	\N	1
72	Cyclin D2 overexpression is associated with the presence pf lymph node metastasis in gastric cancers.		Supports	3	3	3	14	79	21	2015-02-20 17:02:07.556927	2015-02-20 17:02:07.556927	4	accepted	\N	\N	1
73	Cyclin D2 overexpression is associated with the increased depth of cancer invasion in gastric cancers.		Supports	3	3	3	14	79	21	2015-02-20 17:02:07.622523	2015-02-20 17:02:07.622523	4	accepted	\N	\N	1
74	Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.		Supports	3	3	3	14	79	21	2015-02-20 17:02:07.681541	2015-02-20 17:02:07.681541	4	accepted	\N	\N	1
75	Cyclin D2 overexpression is associated with vascular invasion by cancer cells in gastric cancers.		Supports	3	3	3	14	79	21	2015-02-20 17:02:07.739959	2015-02-20 17:02:07.739959	4	accepted	\N	\N	1
76	CCND2 hypomethylation is seen to be more common in stage III and IV gastric tumors than stage I and II.		Supports	2	3	3	14	80	22	2015-02-20 17:02:07.804126	2015-02-20 17:02:07.804126	3	accepted	\N	\N	1
77	Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib (PD-0332991) significantly increased median survival.		Supports	1	3	24	15	81	23	2015-02-20 17:02:07.87551	2015-02-20 17:02:07.87551	3	accepted	\N	\N	1
78	Patients with high expression of total cyclin E and low-molecular weight cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard ratio for these patients in 13.3 times higher than those with normal cyclin E levels.		Supports	3	1	3	11	82	24	2015-02-20 17:02:07.957934	2015-02-20 17:02:07.957934	4	accepted	\N	\N	1
79	Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced stage gastric carcinoma.		Supports	3	1	3	16	83	24	2015-02-20 17:02:08.035144	2015-02-20 17:02:08.035144	4	accepted	\N	\N	1
80	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.		Supports	3	1	3	16	83	24	2015-02-20 17:02:08.099193	2015-02-20 17:02:08.099193	4	accepted	\N	\N	1
81	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.		Supports	3	1	3	17	83	24	2015-02-20 17:02:08.172357	2015-02-20 17:02:08.172357	4	accepted	\N	\N	1
82	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.		Supports	1	3	25	18	84	25	2015-02-20 17:02:08.242417	2015-02-20 17:02:08.242417	4	accepted	\N	\N	1
83	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.		Supports	3	1	26	18	85	25	2015-02-20 17:02:08.315971	2015-02-20 17:02:08.315971	3	accepted	\N	\N	1
84	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.		Supports	1	3	25	18	84	26	2015-02-20 17:02:08.383964	2015-02-20 17:02:08.383964	4	accepted	\N	\N	1
85	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.		Supports	3	1	26	18	85	26	2015-02-20 17:02:08.475585	2015-02-20 17:02:08.475585	3	accepted	\N	\N	1
86	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter time to recurrence.		Supports	3	1	3	4	86	27	2015-02-20 17:02:08.544462	2015-02-20 17:02:08.544462	3	accepted	\N	\N	1
87	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter overall survival.		Supports	3	1	3	4	86	27	2015-02-20 17:02:08.610723	2015-02-20 17:02:08.610723	3	accepted	\N	\N	1
88	CEBPA mutation status had no impact on ATRA treatment response in older (>60) patients with AML		Does Not Support	1	1	3	1	87	29	2015-02-20 17:02:08.680266	2015-02-20 17:02:08.680266	4	accepted	\N	\N	1
89	CEBPA mutation was significantly associated with complete remission		Supports	3	1	3	1	88	29	2015-02-20 17:02:08.742143	2015-02-20 17:02:08.742143	4	accepted	\N	\N	1
90	CEBPA mutation was associated with improved overall survival in older (>60) patients with AML		Supports	3	1	3	1	87	29	2015-02-20 17:02:08.798731	2015-02-20 17:02:08.798731	4	accepted	\N	\N	1
91	Overall survival was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA mutations		Supports	3	1	3	1	89	28	2015-02-20 17:02:08.857185	2015-02-20 17:02:08.857185	3	accepted	\N	\N	1
92	Remission duration was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA		Supports	3	1	3	1	89	28	2015-02-20 17:02:08.924348	2015-02-20 17:02:08.924348	3	accepted	\N	\N	1
93	FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically normal AML patients with CEBPA		Does Not Support	3	1	3	1	89	28	2015-02-20 17:02:09.010366	2015-02-20 17:02:09.010366	2	accepted	\N	\N	1
94	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580		Supports	1	3	1	19	90	30	2015-02-20 17:02:09.079022	2015-02-20 17:02:09.079022	3	accepted	\N	\N	1
95	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580		Supports	1	3	27	19	90	30	2015-02-20 17:02:09.135033	2015-02-20 17:02:09.135033	3	accepted	\N	\N	1
96	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.		Supports	2	1	3	20	91	31	2015-02-20 17:02:09.211481	2015-02-20 17:02:09.211481	4	accepted	\N	\N	1
97	Young AML patients (<60 years old) with DNMT3A mutations (60% of which were R882) were older in age, had higher white blood cell counts and had higher platelet counts than patients wildtype for DNMT3A		Supports	2	1	3	1	92	32	2015-02-20 17:02:09.2763	2015-02-20 17:02:09.2763	3	accepted	\N	\N	1
98	DNMT3A R882 mutations occur most often in de novo AML patients with intermediate cytogenic risk.		Supports	2	1	3	1	93	32	2015-02-20 17:02:09.339853	2015-02-20 17:02:09.339853	3	accepted	\N	\N	1
99	Daunorubicin treatment resulted in similar overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.		Does Not Support	1	1	28	1	93	32	2015-02-20 17:02:09.407151	2015-02-20 17:02:09.407151	4	accepted	\N	\N	1
100	Idarubicin increases the overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.		Supports	1	1	29	1	93	32	2015-02-20 17:02:09.487652	2015-02-20 17:02:09.487652	4	accepted	\N	\N	1
101	There is no difference in the complete remission rate of de novo AML patients with DNMT3A mutation compared to those who are wild type for DNMT3A.		Does Not Support	3	1	3	1	93	32	2015-02-20 17:02:09.551362	2015-02-20 17:02:09.551362	4	accepted	\N	\N	1
102	De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.		Supports	3	1	3	1	94	32	2015-02-20 17:02:09.620701	2015-02-20 17:02:09.620701	4	accepted	\N	\N	1
103	DNMT3A R882 mutations were associated with older age, higher white blood cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically normal AML patients		Supports	2	1	3	1	95	32	2015-02-20 17:02:09.689032	2015-02-20 17:02:09.689032	3	accepted	\N	\N	1
104	In young AML patients (<60 years old), DNMT3A mutation status (60% of which were R882) was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD		Does Not Support	3	1	3	1	92	32	2015-02-20 17:02:09.753399	2015-02-20 17:02:09.753399	4	accepted	\N	\N	1
105	In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5% of which were R882) had no prognostic value on overall, relapse free and event free survival.		Does Not Support	3	1	3	1	96	32	2015-02-20 17:02:09.81801	2015-02-20 17:02:09.81801	3	accepted	\N	\N	1
106	In a large cohort of cytogenetically normal AML patients (18-60 years old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall, relapse free and event free survival		Does Not Support	3	1	3	1	96	32	2015-02-20 17:02:09.880158	2015-02-20 17:02:09.880158	3	accepted	\N	\N	1
107	AML patients with DNMT3A mutations (59% of which were R882) showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.		Supports	3	1	3	1	94	32	2015-02-20 17:02:09.964765	2015-02-20 17:02:09.964765	5	accepted	\N	\N	1
108	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including R882) were associated with worse overall survival compared to those without DNMT3A mutation.		Supports	3	1	3	1	94	32	2015-02-20 17:02:10.028353	2015-02-20 17:02:10.028353	5	accepted	\N	\N	1
109	In cytogenetically normal AML patients, DNMT3A R882 mutations are associated with lower overall and disease free survival as compared to patients with wild type DNMT3A.		Supports	3	1	3	1	95	32	2015-02-20 17:02:10.09002	2015-02-20 17:02:10.09002	5	accepted	\N	\N	1
110	In young AML patients (<60 years old), DNMT3A mutations were associated with significantly reduced overall survival and relapse free survival in patients wildtype for NPM1 and FLT3		Supports	3	1	3	1	92	32	2015-02-20 17:02:10.154002	2015-02-20 17:02:10.154002	4	accepted	\N	\N	1
111	In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation is prognostic for shorter disease free survival and overall survival compared to patients without the mutation.		Supports	3	1	3	1	97	32	2015-02-20 17:02:10.216345	2015-02-20 17:02:10.216345	4	accepted	\N	\N	1
112	In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations other than R882 are prognostic for shorter disease free survival and overall survival compared to patients without the mutation.		Supports	3	1	3	1	97	32	2015-02-20 17:02:10.282616	2015-02-20 17:02:10.282616	4	accepted	\N	\N	1
113	DNMT3A R882 mutation was associated with reduced relapse free and overall survival in ELN-unfavorable, cytogenetically normal AMLs		Supports	3	1	3	1	96	32	2015-02-20 17:02:10.348621	2015-02-20 17:02:10.348621	3	accepted	\N	\N	1
114	In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations were associated with reduced relapse free survival in the entire cohort as well as the subset of patients with cytogenetically normal AML		Supports	3	2	3	1	96	32	2015-02-20 17:02:10.438257	2015-02-20 17:02:10.438257	3	accepted	\N	\N	1
115	Therapy-related AML was less common in patients with DNMT3A mutations (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of younger (18-60) AML patients		Supports	2	1	3	1	96	32	2015-02-20 17:02:10.503189	2015-02-20 17:02:10.503189	3	accepted	\N	\N	1
116	DNMT3A mutations (59% of which were R882) were associated with an intermediate risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes		Supports	2	1	3	1	94	32	2015-02-20 17:02:10.566302	2015-02-20 17:02:10.566302	5	accepted	\N	\N	1
117	DNMT3A mutations (59% of which were R882) were associated with intermediate risk cytogenetics (including normal karyotype)		Supports	2	1	3	1	94	32	2015-02-20 17:02:10.629939	2015-02-20 17:02:10.629939	4	accepted	\N	\N	1
118	DNMT3A mutations (64.5% R882) were associated with older age, higher white blood cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML patients		Supports	2	1	3	1	96	32	2015-02-20 17:02:10.6867	2015-02-20 17:02:10.6867	3	accepted	\N	\N	1
119	DNMT3A R882 mutations were associated with cytogenetically normal AML in a large cohort of younger (18-60) AML patients		Supports	2	1	3	1	96	32	2015-02-20 17:02:10.753738	2015-02-20 17:02:10.753738	3	accepted	\N	\N	1
120	DNMT3A mutations were associated with achievement of complete remission in a large cohort of younger (18-60) AML patients		Supports	3	1	3	1	96	32	2015-02-20 17:02:10.824736	2015-02-20 17:02:10.824736	3	accepted	\N	\N	1
121	Complete remission rates did not differ between patients with wildtype or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML		Does Not Support	3	1	3	1	97	32	2015-02-20 17:02:10.967422	2015-02-20 17:02:10.967422	4	accepted	\N	\N	1
122	DNMT3A mutations (62% of which were R882) were associated with reduced disease-free survival in patients with cytogenetically normal AML.		Supports	3	1	3	1	97	32	2015-02-20 17:02:11.033634	2015-02-20 17:02:11.033634	4	accepted	\N	\N	1
123	Young AML patients (<60 years old) with DNMT3A mutation have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A		Supports	3	1	3	1	92	32	2015-02-20 17:02:11.095593	2015-02-20 17:02:11.095593	3	accepted	\N	\N	1
124	Complete remission rate was not different between young AML patients (<60 years old) with or without DNMT3A mutations (60% of which were R882)		Supports	3	1	3	1	92	32	2015-02-20 17:02:11.158979	2015-02-20 17:02:11.158979	3	accepted	\N	\N	1
125	Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A		Supports	3	1	3	1	92	32	2015-02-20 17:02:11.220626	2015-02-20 17:02:11.220626	3	accepted	\N	\N	1
126	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib.		Does Not Support	1	1	30	4	98	33	2015-02-20 17:02:11.288225	2015-02-20 17:02:11.288225	3	accepted	\N	\N	1
127	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib.		Does Not Support	1	1	31	4	98	33	2015-02-20 17:02:11.340624	2015-02-20 17:02:11.340624	3	accepted	\N	\N	1
128	In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence of L858R mutation is prognostic for better progression free survival.		Supports	1	1	3	4	99	33	2015-02-20 17:02:11.398105	2015-02-20 17:02:11.398105	4	accepted	\N	\N	1
129	Geftinib added to pemetrexed and carboplatin showed greater effectiveness than premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.		Supports	1	1	30	4	100	33	2015-02-20 17:02:11.466729	2015-02-20 17:02:11.466729	3	accepted	\N	\N	1
130	Median survival of patients with EGFR L858R mutation is better than those with wild type EGFR.		Supports	3	1	3	4	101	33	2015-02-20 17:02:11.545901	2015-02-20 17:02:11.545901	3	accepted	\N	\N	1
131	The T790M mutation in EGFR has been shown to confer resistance to the tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are placed on the drug are likely to relapse.		Supports	1	2	31	21	102	34	2015-02-20 17:02:11.61077	2015-02-20 17:02:11.61077	5	accepted	\N	\N	1
132	In non-small cell lung cancer, the appearance of T790M mutation leads to resistance to gefitinib.		Supports	1	1	30	4	103	34	2015-02-20 17:02:11.686579	2015-02-20 17:02:11.686579	3	accepted	\N	\N	1
133	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.		Supports	1	1	31	4	104	34	2015-02-20 17:02:11.747485	2015-02-20 17:02:11.747485	3	accepted	\N	\N	1
134	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.		Supports	1	1	32	4	104	34	2015-02-20 17:02:11.806429	2015-02-20 17:02:11.806429	3	accepted	\N	\N	1
135	In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR, especially when an L858R mutation is also present.		Supports	1	4	33	4	105	34	2015-02-20 17:02:11.882151	2015-02-20 17:02:11.882151	1	accepted	\N	\N	1
136	Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall survival compared to patients with L858R or other exon 19 activating mutations.		Supports	3	1	3	4	106	34	2015-02-20 17:02:11.946693	2015-02-20 17:02:11.946693	4	accepted	\N	\N	1
137	In NSCLC patients with T790M and another activating mutations, their progression free survival is shorter compared to those who do not possess T790M mutation.		Supports	3	1	3	4	107	34	2015-02-20 17:02:12.01607	2015-02-20 17:02:12.01607	4	accepted	\N	\N	1
138	In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.		Supports	1	3	34	11	108	35	2015-02-20 17:02:12.101903	2015-02-20 17:02:12.101903	4	accepted	\N	\N	1
139	In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.		Supports	1	3	34	11	108	36	2015-02-20 17:02:12.174581	2015-02-20 17:02:12.174581	4	accepted	\N	\N	1
140	In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2, cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive to neratinib.		Supports	1	3	34	11	108	37	2015-02-20 17:02:12.249772	2015-02-20 17:02:12.249772	5	accepted	\N	\N	1
141	In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.		Supports	1	3	34	11	108	38	2015-02-20 17:02:12.313552	2015-02-20 17:02:12.313552	5	accepted	\N	\N	1
142	The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.		Supports	1	3	34	11	108	39	2015-02-20 17:02:12.383384	2015-02-20 17:02:12.383384	5	accepted	\N	\N	1
143	The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.		Supports	1	3	35	11	108	39	2015-02-20 17:02:12.447859	2015-02-20 17:02:12.447859	5	accepted	\N	\N	1
144	MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.		Supports	1	3	34	11	108	40	2015-02-20 17:02:12.508711	2015-02-20 17:02:12.508711	5	accepted	\N	\N	1
145	In MCF10A cell lines, the in-frame insertion of a single amino acid at position 780 was shown to be sensitive to neratinib.		Supports	1	3	34	11	108	41	2015-02-20 17:02:12.56512	2015-02-20 17:02:12.56512	5	accepted	\N	\N	1
146	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.		Supports	1	3	34	11	108	42	2015-02-20 17:02:12.661714	2015-02-20 17:02:12.661714	5	accepted	\N	\N	1
147	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.		Supports	1	3	35	11	108	42	2015-02-20 17:02:12.716422	2015-02-20 17:02:12.716422	5	accepted	\N	\N	1
148	In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.		Supports	1	3	34	11	108	43	2015-02-20 17:02:12.782438	2015-02-20 17:02:12.782438	5	accepted	\N	\N	1
149	In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.		Supports	1	3	34	11	108	44	2015-02-20 17:02:12.847322	2015-02-20 17:02:12.847322	5	accepted	\N	\N	1
150	In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.		Supports	1	3	34	11	108	45	2015-02-20 17:02:12.916055	2015-02-20 17:02:12.916055	5	accepted	\N	\N	1
151	The L536Q ligand binding domain mutation have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.		Supports	1	3	36	11	109	46	2015-02-20 17:02:12.986401	2015-02-20 17:02:12.986401	5	accepted	\N	\N	1
152	The L536Q ligand binding domain mutation have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.		Supports	1	3	37	11	109	46	2015-02-20 17:02:13.0515	2015-02-20 17:02:13.0515	5	accepted	\N	\N	1
153	MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.		Supports	1	3	3	11	110	46	2015-02-20 17:02:13.132802	2015-02-20 17:02:13.132802	3	accepted	\N	\N	1
154	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	36	11	109	47	2015-02-20 17:02:13.230517	2015-02-20 17:02:13.230517	5	accepted	\N	\N	1
155	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	37	11	109	47	2015-02-20 17:02:13.281115	2015-02-20 17:02:13.281115	5	accepted	\N	\N	1
156	MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.		Supports	1	3	3	11	110	47	2015-02-20 17:02:13.355969	2015-02-20 17:02:13.355969	3	accepted	\N	\N	1
157	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	36	11	109	48	2015-02-20 17:02:13.422333	2015-02-20 17:02:13.422333	5	accepted	\N	\N	1
158	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	37	11	109	48	2015-02-20 17:02:13.47817	2015-02-20 17:02:13.47817	5	accepted	\N	\N	1
159	MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.		Supports	1	3	3	11	110	48	2015-02-20 17:02:13.544815	2015-02-20 17:02:13.544815	3	accepted	\N	\N	1
160	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	36	11	109	49	2015-02-20 17:02:13.612809	2015-02-20 17:02:13.612809	5	accepted	\N	\N	1
161	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	37	11	109	49	2015-02-20 17:02:13.668356	2015-02-20 17:02:13.668356	5	accepted	\N	\N	1
162	MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.		Supports	1	3	3	11	110	49	2015-02-20 17:02:13.761298	2015-02-20 17:02:13.761298	3	accepted	\N	\N	1
163	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	36	11	109	50	2015-02-20 17:02:13.831441	2015-02-20 17:02:13.831441	5	accepted	\N	\N	1
164	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	37	11	109	50	2015-02-20 17:02:13.890018	2015-02-20 17:02:13.890018	5	accepted	\N	\N	1
165	MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.		Supports	1	3	3	11	110	50	2015-02-20 17:02:13.958206	2015-02-20 17:02:13.958206	3	accepted	\N	\N	1
166	In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic sites and overall stable disease. However, this response did not meet criteria for RECIST partial response.		Supports	1	1	38	22	111	56	2015-02-20 17:02:14.038197	2015-02-20 17:02:14.038197	2	accepted	\N	\N	1
167	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be recurrent in multiple other cancer types.		Supports	1	1	39	22	111	59	2015-02-20 17:02:14.112563	2015-02-20 17:02:14.112563	3	accepted	\N	\N	1
168	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be recurrent in multiple other cancer types.		Supports	1	1	38	22	111	59	2015-02-20 17:02:14.164384	2015-02-20 17:02:14.164384	3	accepted	\N	\N	1
169	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD		Does Not Support	3	1	3	1	92	60	2015-02-20 17:02:14.254216	2015-02-20 17:02:14.254216	4	accepted	\N	\N	1
170	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated with worse overall survival compared to those without DNMT3A mutation.		Supports	3	1	3	1	94	60	2015-02-20 17:02:14.317055	2015-02-20 17:02:14.317055	5	accepted	\N	\N	1
171	FLT3-ITD mutation without NPM1 was associated with increased relapse risk and reduced overall survival in young adult AML patients (median age 43)		Supports	1	1	3	1	112	60	2015-02-20 17:02:14.386879	2015-02-20 17:02:14.386879	4	accepted	\N	\N	1
172	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.		Supports	1	3	40	1	113	60	2015-02-20 17:02:14.459713	2015-02-20 17:02:14.459713	3	accepted	\N	\N	1
173	ATRA treatment did not effect overall survival in patients <60 years old with FLT3-ITD mutation		Supports	1	1	41	1	114	60	2015-02-20 17:02:14.52443	2015-02-20 17:02:14.52443	3	accepted	\N	\N	1
174	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.		Supports	1	3	28	1	113	60	2015-02-20 17:02:14.583117	2015-02-20 17:02:14.583117	3	accepted	\N	\N	1
175	Increasing level of FLT3-ITD mutant correlated with reduced disease-free and overall survival as well as increased relapse rate		Supports	1	1	3	1	112	60	2015-02-20 17:02:14.648532	2015-02-20 17:02:14.648532	3	accepted	\N	\N	1
176	FLT3-ITD duplication length did not effect prognosis in patients with an NPM1 mutation		Does Not Support	3	1	3	1	115	60	2015-02-20 17:02:14.710149	2015-02-20 17:02:14.710149	4	accepted	\N	\N	1
177	Size of the FLT3-ITD mutant duplication had no impact on overall survival or relapse rate		Does Not Support	3	1	3	1	112	60	2015-02-20 17:02:14.794095	2015-02-20 17:02:14.794095	2	accepted	\N	\N	1
178	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3		Supports	3	1	3	1	115	60	2015-02-20 17:02:14.86098	2015-02-20 17:02:14.86098	4	accepted	\N	\N	1
179	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation		Supports	3	1	3	1	115	60	2015-02-20 17:02:14.918956	2015-02-20 17:02:14.918956	4	accepted	\N	\N	1
180	A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation		Supports	3	1	3	1	115	60	2015-02-20 17:02:14.97358	2015-02-20 17:02:14.97358	4	accepted	\N	\N	1
181	Meta-analysis of studies involving cytogentically normal younger (<60) patients showed reduced overall and relapse-free survival for patients with FLT3-ITD		Supports	3	2	3	1	116	60	2015-02-20 17:02:15.039204	2015-02-20 17:02:15.039204	2	accepted	\N	\N	1
182	WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared to wild type		Supports	2	1	3	1	117	60	2015-02-20 17:02:15.102729	2015-02-20 17:02:15.102729	3	accepted	\N	\N	1
183	Sorafenib is effective in patients with FLT3 internal tandem repeats, but not in wild type or D835 mutation.		Supports	1	3	10	1	118	60	2015-02-20 17:02:15.17321	2015-02-20 17:02:15.17321	4	accepted	\N	\N	1
184	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations		Supports	1	1	42	1	119	60	2015-02-20 17:02:15.268885	2015-02-20 17:02:15.268885	4	accepted	\N	\N	1
185	AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with FLT3 internal tandem repeat		Supports	1	3	43	1	120	60	2015-02-20 17:02:15.334677	2015-02-20 17:02:15.334677	4	accepted	\N	\N	1
186	Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor survival and inferior disease free survival in patients with AML.		Supports	3	1	3	1	121	60	2015-02-20 17:02:15.39443	2015-02-20 17:02:15.39443	4	accepted	\N	\N	1
187	AML patients with internal tandem repeat of FLT3 has worse overall survival than patients with wild type FLT3		Supports	3	1	3	1	117	60	2015-02-20 17:02:15.454406	2015-02-20 17:02:15.454406	3	accepted	\N	\N	1
188	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.		Supports	1	3	40	1	113	61	2015-02-20 17:02:15.518694	2015-02-20 17:02:15.518694	3	accepted	\N	\N	1
189	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.		Supports	1	3	28	1	113	61	2015-02-20 17:02:15.572447	2015-02-20 17:02:15.572447	3	accepted	\N	\N	1
190	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed		Supports	3	1	3	1	122	61	2015-02-20 17:02:15.645737	2015-02-20 17:02:15.645737	3	accepted	\N	\N	1
191	WBC counts were not significantly different in AML patients with FLT3 D835 mutations compared to wild type		Does Not Support	2	1	3	1	117	61	2015-02-20 17:02:15.704109	2015-02-20 17:02:15.704109	3	accepted	\N	\N	1
192	Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated with sorafenib resistance		Supports	1	1	10	1	123	61	2015-02-20 17:02:15.791462	2015-02-20 17:02:15.791462	4	accepted	\N	\N	1
193	Sorafenib is effective in patients with FLT3 internal tandem repeat mutations, but not in patients with wild type or D835 mutation.		Supports	1	3	10	1	118	61	2015-02-20 17:02:15.864726	2015-02-20 17:02:15.864726	4	accepted	\N	\N	1
194	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations		Supports	1	1	42	1	119	61	2015-02-20 17:02:15.918994	2015-02-20 17:02:15.918994	4	accepted	\N	\N	1
195	AML patients with D835 mutation of FLT3 did not have worse overall survival than patients with wild type FLT3		Does Not Support	3	1	3	1	117	61	2015-02-20 17:02:15.982434	2015-02-20 17:02:15.982434	3	accepted	\N	\N	1
196	In AML patients, FLT3-TKD mutation was associated with poorer disease free survival compared to patients with wild type FLT3		Supports	3	1	3	1	124	61	2015-02-20 17:02:16.042686	2015-02-20 17:02:16.042686	4	accepted	\N	\N	1
197	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.		Supports	1	1	44	23	125	62	2015-02-20 17:02:16.107405	2015-02-20 17:02:16.107405	4	accepted	\N	\N	1
198	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.		Supports	1	1	45	23	125	62	2015-02-20 17:02:16.167916	2015-02-20 17:02:16.167916	4	accepted	\N	\N	1
199	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation		Does Not Support	3	1	3	1	126	63	2015-02-20 17:02:16.255047	2015-02-20 17:02:16.255047	3	accepted	\N	\N	1
200	In leukemia patients, IDH1 R132 mutation is associated with adults and normal karyotype.		Supports	2	1	3	1	127	63	2015-02-20 17:02:16.317805	2015-02-20 17:02:16.317805	2	accepted	\N	\N	1
201	The complete remission and overall survival rates in patients with IDH1 R132 mutation is not significantly different from those who do not have this mutation		Does Not Support	3	1	3	1	128	63	2015-02-20 17:02:16.376426	2015-02-20 17:02:16.376426	2	accepted	\N	\N	1
202	In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic for better survival compared to patients who harbor wild type mutation.		Supports	3	1	3	24	129	63	2015-02-20 17:02:16.445411	2015-02-20 17:02:16.445411	3	accepted	\N	\N	1
203	In patients with glioblastoma, those harboring IDH1 R132 mutation has higher overall survival compared to those who do not have IDH1 mutation		Supports	3	1	3	25	130	63	2015-02-20 17:02:16.510512	2015-02-20 17:02:16.510512	3	accepted	\N	\N	1
204	IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated with worse overall survival		Supports	3	1	3	26	131	63	2015-02-20 17:02:16.572873	2015-02-20 17:02:16.572873	4	accepted	\N	\N	1
205	IDH1 R132 mutation is associated with patients of older age, high platelet count during diagnosis, cytogenic normalcy and NPM1 mutation		Supports	2	1	3	1	132	64	2015-02-20 17:02:16.64294	2015-02-20 17:02:16.64294	3	accepted	\N	\N	1
206	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.		Does Not Support	3	1	3	1	132	64	2015-02-20 17:02:16.73081	2015-02-20 17:02:16.73081	2	accepted	\N	\N	1
207	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.		Does Not Support	3	1	3	1	132	65	2015-02-20 17:02:16.804573	2015-02-20 17:02:16.804573	3	accepted	\N	\N	1
208	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.		Does Not Support	3	1	3	1	132	66	2015-02-20 17:02:16.870743	2015-02-20 17:02:16.870743	3	accepted	\N	\N	1
209	In AML, patients with IDH2 R140K mutation have improved overall survival compared to those with wild-type IDH2		Supports	3	1	3	1	133	67	2015-02-20 17:02:16.932151	2015-02-20 17:02:16.932151	3	accepted	\N	\N	1
210	In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact overall survival or disease free survival.		Does Not Support	3	1	3	1	134	67	2015-02-20 17:02:16.995252	2015-02-20 17:02:16.995252	3	accepted	\N	\N	1
211	AML patients with IDH2 mutations such as R140Q/L have event free survival and overall survival similar to those with wild-type IDH2		Does Not Support	3	4	3	1	135	67	2015-02-20 17:02:17.064815	2015-02-20 17:02:17.064815	2	accepted	\N	\N	1
212	R140Q mutation in IDH2 does not have prognostic value in patients with MDS.		Does Not Support	3	1	3	27	136	67	2015-02-20 17:02:17.134902	2015-02-20 17:02:17.134902	3	accepted	\N	\N	1
213	In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L do not confer prognostic value (overall survival)		Does Not Support	3	4	3	27	137	67	2015-02-20 17:02:17.22036	2015-02-20 17:02:17.22036	2	accepted	\N	\N	1
214	AML patients with IDH2 mutations such as R172K have event free survival and overall survival similar to those with wild-type IDH2.		Does Not Support	3	4	3	1	135	68	2015-02-20 17:02:17.292503	2015-02-20 17:02:17.292503	2	accepted	\N	\N	1
215	In AML, patients with an IDH2 R172K mutation have worse overall survival compared to those with wild-type IDH2		Supports	3	1	3	1	133	68	2015-02-20 17:02:17.352791	2015-02-20 17:02:17.352791	3	accepted	\N	\N	1
216	In patients suffering from MDS, the presence of IDH2 mutations such as R172K do not confer prognostic value (overall survival)		Does Not Support	3	4	3	27	137	68	2015-02-20 17:02:17.422155	2015-02-20 17:02:17.422155	2	accepted	\N	\N	1
217	JAK2 V617F not associatted wth  lymphoid leukemia.		Does Not Support	2	1	3	2	138	69	2015-02-20 17:02:17.4943	2015-02-20 17:02:17.4943	4	accepted	\N	\N	1
218	JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical CML).		Supports	2	1	3	2	138	69	2015-02-20 17:02:17.555106	2015-02-20 17:02:17.555106	4	accepted	\N	\N	1
219	TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells		Supports	1	3	46	28	139	69	2015-02-20 17:02:17.623315	2015-02-20 17:02:17.623315	3	accepted	\N	\N	1
220	JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).		Supports	2	1	3	29	138	69	2015-02-20 17:02:17.773016	2015-02-20 17:02:17.773016	4	accepted	\N	\N	1
221	rs10974944 SNP in the Jak2 locus is associated with increased predisposition for JAK2 V617F mutation and its associated cancer		Supports	2	1	3	30	140	69	2015-02-20 17:02:17.838901	2015-02-20 17:02:17.838901	4	accepted	\N	\N	1
222	In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction in the percentage of cells harboring JAK2 V617F.		Supports	1	1	47	31	141	69	2015-02-20 17:02:17.907847	2015-02-20 17:02:17.907847	4	accepted	\N	\N	1
223	In acute myloid leukemia patients, D816 mutation is associated with earlier relapse and poorer prognosis than wildtype KIT		Supports	3	1	3	32	142	70	2015-02-20 17:02:17.972671	2015-02-20 17:02:17.972671	4	accepted	\N	\N	1
224	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.		Does Not Support	3	1	3	33	143	71	2015-02-20 17:02:18.046503	2015-02-20 17:02:18.046503	3	accepted	\N	\N	1
225	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT		Supports	2	1	3	33	144	71	2015-02-20 17:02:18.121968	2015-02-20 17:02:18.121968	2	accepted	\N	\N	1
226	KIT mutations were identified in morphologically benign, incidentally discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating is not a prognostic marker for GIST		Does Not Support	3	1	3	33	145	71	2015-02-20 17:02:18.19339	2015-02-20 17:02:18.19339	3	accepted	\N	\N	1
227	KIT mutation is associated with worse overall and cause-specific prognosis in patients with GIST compared to patients with wildtype KIT		Supports	3	1	3	33	144	71	2015-02-20 17:02:18.261036	2015-02-20 17:02:18.261036	3	accepted	\N	\N	1
228	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer		Does Not Support	3	1	3	17	146	71	2015-02-20 17:02:18.3193	2015-02-20 17:02:18.3193	2	accepted	\N	\N	1
229	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.		Supports	3	1	1	8	147	71	2015-02-20 17:02:18.383715	2015-02-20 17:02:18.383715	3	accepted	\N	\N	1
230	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.		Supports	3	1	1	8	147	72	2015-02-20 17:02:18.443435	2015-02-20 17:02:18.443435	3	accepted	\N	\N	1
231	GIST patients with exon 14 KIT mutations had reduced overall survival compared to patients wiltype for KIT		Supports	3	1	3	33	143	73	2015-02-20 17:02:18.506587	2015-02-20 17:02:18.506587	3	accepted	\N	\N	1
232	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT		Supports	2	1	3	33	144	74	2015-02-20 17:02:18.564769	2015-02-20 17:02:18.564769	3	accepted	\N	\N	1
233	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.		Supports	3	1	1	8	147	74	2015-02-20 17:02:18.6282	2015-02-20 17:02:18.6282	3	accepted	\N	\N	1
234	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.		Does Not Support	3	1	3	33	143	75	2015-02-20 17:02:18.696337	2015-02-20 17:02:18.696337	3	accepted	\N	\N	1
235	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer		Does Not Support	3	1	3	17	146	75	2015-02-20 17:02:18.755291	2015-02-20 17:02:18.755291	2	accepted	\N	\N	1
236	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.		Supports	1	3	1	8	148	76	2015-02-20 17:02:18.826545	2015-02-20 17:02:18.826545	3	accepted	\N	\N	1
237	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.		Supports	1	3	4	8	148	76	2015-02-20 17:02:18.887082	2015-02-20 17:02:18.887082	3	accepted	\N	\N	1
238	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.		Supports	1	3	10	8	148	76	2015-02-20 17:02:18.943953	2015-02-20 17:02:18.943953	3	accepted	\N	\N	1
239	The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation of KIT but did not alter the binding affinity of dasatinib.		Supports	1	3	5	8	148	76	2015-02-20 17:02:19.008773	2015-02-20 17:02:19.008773	3	accepted	\N	\N	1
240	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.		Supports	1	3	5	4	149	76	2015-02-20 17:02:19.078145	2015-02-20 17:02:19.078145	3	accepted	\N	\N	1
241	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.		Supports	1	3	1	4	149	76	2015-02-20 17:02:19.122852	2015-02-20 17:02:19.122852	3	accepted	\N	\N	1
242	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.		Supports	1	3	4	4	149	76	2015-02-20 17:02:19.173522	2015-02-20 17:02:19.173522	3	accepted	\N	\N	1
243	KIT V654A results in imatinib resistance in GIST patient-derived cell lines.		Supports	1	3	1	33	150	77	2015-02-20 17:02:19.236491	2015-02-20 17:02:19.236491	3	accepted	\N	\N	1
244	In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective for those that are refractory to imatinib.		Supports	1	1	48	33	151	77	2015-02-20 17:02:19.301536	2015-02-20 17:02:19.301536	3	accepted	\N	\N	1
245	Patients with colorectal cancer who harbor KRAS mutation have low response rate to cetuximab		Supports	1	1	14	7	152	78	2015-02-20 17:02:19.406068	2015-02-20 17:02:19.406068	4	accepted	\N	\N	1
246	KRAS mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment		Does Not Support	1	1	49	7	153	79	2015-02-20 17:02:19.475224	2015-02-20 17:02:19.475224	4	accepted	\N	\N	1
247	In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib		Supports	1	1	30	4	154	79	2015-02-20 17:02:19.546727	2015-02-20 17:02:19.546727	3	accepted	\N	\N	1
248	In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib		Supports	1	1	31	4	154	79	2015-02-20 17:02:19.605276	2015-02-20 17:02:19.605276	3	accepted	\N	\N	1
249	In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12 mutation did not impact overall or disease free survival.		Does Not Support	3	1	3	9	155	80	2015-02-20 17:02:19.670599	2015-02-20 17:02:19.670599	4	accepted	\N	\N	1
250	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.		Supports	2	1	3	34	156	80	2015-02-20 17:02:19.738815	2015-02-20 17:02:19.738815	3	accepted	\N	\N	1
251	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival		Supports	3	1	3	35	157	80	2015-02-20 17:02:19.804539	2015-02-20 17:02:19.804539	3	accepted	\N	\N	1
252	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender		Does Not Support	2	1	3	4	158	80	2015-02-20 17:02:19.873129	2015-02-20 17:02:19.873129	3	accepted	\N	\N	1
253	There is no strong association between KRAS mutation status and overall or progression free survival in patients with NSCLC		Does Not Support	3	1	3	4	159	80	2015-02-20 17:02:19.934965	2015-02-20 17:02:19.934965	3	accepted	\N	\N	1
254	A meta-analysis showed KRAS mutation is associated with worse outcome in patients with NSCLC		Supports	3	1	3	4	160	80	2015-02-20 17:02:20.00663	2015-02-20 17:02:20.00663	3	accepted	\N	\N	1
255	KRAS mutations were significantly associated with lack of response to cetuximab in patients with advanced colorectal cancer		Supports	1	3	14	7	161	81	2015-02-20 17:02:20.071828	2015-02-20 17:02:20.071828	3	accepted	\N	\N	1
256	Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations (primarily G12/G13) had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype KRAS		Supports	1	1	14	7	62	81	2015-02-20 17:02:20.126877	2015-02-20 17:02:20.126877	3	accepted	\N	\N	1
257	KRAS mutations were significantly associated with reduced progression-free and overall survival in patients with advanced colorectal cancer		Supports	3	3	3	7	161	81	2015-02-20 17:02:20.185043	2015-02-20 17:02:20.185043	3	accepted	\N	\N	1
258	KRAS G12C occur more frequently in women than men		Supports	2	1	3	17	162	82	2015-02-20 17:02:20.260061	2015-02-20 17:02:20.260061	2	accepted	\N	\N	1
259	KRAS G12D mutation occurs in never smokers significantly more often than in smokers		Supports	2	1	3	17	162	83	2015-02-20 17:02:20.326883	2015-02-20 17:02:20.326883	3	accepted	\N	\N	1
260	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.		Supports	1	1	13	4	163	83	2015-02-20 17:02:20.393183	2015-02-20 17:02:20.393183	2	accepted	\N	\N	1
261	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.		Supports	1	3	50	4	164	83	2015-02-20 17:02:20.472822	2015-02-20 17:02:20.472822	4	accepted	\N	\N	1
262	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.		Supports	1	3	51	4	164	83	2015-02-20 17:02:20.531265	2015-02-20 17:02:20.531265	4	accepted	\N	\N	1
263	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.		Supports	2	1	3	34	156	84	2015-02-20 17:02:20.58482	2015-02-20 17:02:20.58482	3	accepted	\N	\N	1
264	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival		Supports	3	1	3	36	157	84	2015-02-20 17:02:20.640682	2015-02-20 17:02:20.640682	3	accepted	\N	\N	1
265	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender		Does Not Support	2	1	3	4	158	84	2015-02-20 17:02:20.699251	2015-02-20 17:02:20.699251	3	accepted	\N	\N	1
266	Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model		Supports	1	3	14	9	165	85	2015-02-20 17:02:20.756845	2015-02-20 17:02:20.756845	4	accepted	\N	\N	1
267	Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model		Supports	1	3	14	9	165	85	2015-02-20 17:02:20.812155	2015-02-20 17:02:20.812155	4	accepted	\N	\N	1
268	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS		Supports	1	1	14	9	165	85	2015-02-20 17:02:20.87311	2015-02-20 17:02:20.87311	4	accepted	\N	\N	1
269	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS		Supports	3	1	14	9	165	85	2015-02-20 17:02:20.933074	2015-02-20 17:02:20.933074	4	accepted	\N	\N	1
270	The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition by about 5 fold.		Supports	1	3	23	8	166	86	2015-02-20 17:02:21.040016	2015-02-20 17:02:21.040016	3	accepted	\N	\N	1
271	MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.		Supports	1	3	23	8	167	87	2015-02-20 17:02:21.109386	2015-02-20 17:02:21.109386	3	accepted	\N	\N	1
272	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580		Supports	1	3	1	19	90	88	2015-02-20 17:02:21.169093	2015-02-20 17:02:21.169093	3	accepted	\N	\N	1
273	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580		Supports	1	3	27	19	90	88	2015-02-20 17:02:21.222261	2015-02-20 17:02:21.222261	3	accepted	\N	\N	1
274	In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted from temozolomide. This benefit was also methylation status dependent, as those without methylation did not see increased survival.		Supports	1	2	22	37	168	89	2015-02-20 17:02:21.29808	2015-02-20 17:02:21.29808	5	accepted	\N	\N	1
275	MGMT promoter methylation was associated with increased overall and disease-free survival as well as tumor regression in patients with gliomas.		Supports	3	1	3	37	169	89	2015-02-20 17:02:21.35632	2015-02-20 17:02:21.35632	4	accepted	\N	\N	1
276	In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor, such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may be useful in treating patients with initial resistance to alkylating agents.		Supports	1	4	52	37	169	89	2015-02-20 17:02:21.415744	2015-02-20 17:02:21.415744	4	accepted	\N	\N	1
277	MGMT promoter methylation has been shown to confer sensitivity to alkylating agents, such as carmustine, in glioblastoma cells.		Supports	1	1	53	37	169	89	2015-02-20 17:02:21.47934	2015-02-20 17:02:21.47934	4	accepted	\N	\N	1
278	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD		Does Not Support	3	1	3	1	92	90	2015-02-20 17:02:21.541477	2015-02-20 17:02:21.541477	4	accepted	\N	\N	1
279	NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid leukemia.		Supports	2	1	3	1	170	90	2015-02-20 17:02:21.605583	2015-02-20 17:02:21.605583	4	accepted	\N	\N	1
280	NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype (0/55).		Does Not Support	2	1	3	1	171	90	2015-02-20 17:02:21.663046	2015-02-20 17:02:21.663046	3	accepted	\N	\N	1
281	NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid leukemia.		Supports	2	1	3	1	170	90	2015-02-20 17:02:21.716378	2015-02-20 17:02:21.716378	4	accepted	\N	\N	1
282	NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid leukemia		Supports	2	1	3	1	171	90	2015-02-20 17:02:21.786483	2015-02-20 17:02:21.786483	3	accepted	\N	\N	1
283	NPM1 mutations were associated with normal karyotype in older (>60) patients		Supports	2	1	3	1	172	90	2015-02-20 17:02:21.848737	2015-02-20 17:02:21.848737	3	accepted	\N	\N	1
284	NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes		Supports	2	1	3	1	112	90	2015-02-20 17:02:21.913626	2015-02-20 17:02:21.913626	2	accepted	\N	\N	1
285	ATRA treatment did not effect overall survival in patients <60 years old with NPM1 mutation regardless of FLT3-ITD status		Supports	1	1	41	1	114	90	2015-02-20 17:02:21.971987	2015-02-20 17:02:21.971987	3	accepted	\N	\N	1
286	Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction in NPM1 levels following first line therapy (N=15)		Supports	1	1	3	1	173	90	2015-02-20 17:02:22.036545	2015-02-20 17:02:22.036545	2	accepted	\N	\N	1
287	NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and increased overall survival in young adult acute myeloid leukemia patients (median age 43)		Supports	1	1	3	1	112	90	2015-02-20 17:02:22.102318	2015-02-20 17:02:22.102318	4	accepted	\N	\N	1
288	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD		Supports	1	3	54	1	174	90	2015-02-20 17:02:22.174164	2015-02-20 17:02:22.174164	3	accepted	\N	\N	1
289	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD		Supports	1	3	41	1	174	90	2015-02-20 17:02:22.228045	2015-02-20 17:02:22.228045	3	accepted	\N	\N	1
290	Patients with NPM1 mutations were associated with improved complete remission rates as well as overall and disease free survival when all ages and karyotype were analyzed; however, when only normal karyotype patients were analyzed, only complete remission rates were improved		Supports	1	1	3	1	172	90	2015-02-20 17:02:22.286698	2015-02-20 17:02:22.286698	3	accepted	\N	\N	1
291	ATRA treatment improved overall and relapse-free survival in older (>60) patients with mutant NPM1 and without FLT3-ITD		Supports	1	1	41	1	172	90	2015-02-20 17:02:22.346257	2015-02-20 17:02:22.346257	2	accepted	\N	\N	1
292	Young patients (18-60) with NPM1 mutations had improved overall survival following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However, the adjusted p-value for multiple testing is 0.11.		Supports	1	1	28	1	175	90	2015-02-20 17:02:22.41536	2015-02-20 17:02:22.41536	2	accepted	\N	\N	1
293	For patients with NPM1 mutation that achieved CR following induction therapy, relapse-free survival was improved following treatment with valproic acid		Supports	1	1	55	1	176	90	2015-02-20 17:02:22.474202	2015-02-20 17:02:22.474202	2	accepted	\N	\N	1
294	CD33 expression was significantly increased in patients with NPM1 mutation with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy		Supports	1	4	3	1	177	90	2015-02-20 17:02:22.57918	2015-02-20 17:02:22.57918	2	accepted	\N	\N	1
295	CD33 and CD123 expression was significantly increased in patients with NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33 and anti-CD123 therapy		Supports	1	4	3	1	177	90	2015-02-20 17:02:22.638943	2015-02-20 17:02:22.638943	2	accepted	\N	\N	1
296	Complete remission rates were higher and event-free survival was longer for normal karyotype AML patients with Exon 12 NPM1 mutations.		Supports	3	1	3	1	170	90	2015-02-20 17:02:22.696186	2015-02-20 17:02:22.696186	5	accepted	\N	\N	1
297	Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and without FLT3-ITD have increased overall survival, relapse-free survival and response to induction chemotherapy		Supports	3	1	3	1	178	90	2015-02-20 17:02:22.760046	2015-02-20 17:02:22.760046	4	accepted	\N	\N	1
298	NPM1 mutations were associated with increased complete remission rates as well as longer overall disease free survival in normal karyotype AML patients >59 years old, particularly in those >69 years old		Supports	3	1	3	1	179	90	2015-02-20 17:02:22.823792	2015-02-20 17:02:22.823792	4	accepted	\N	\N	1
299	NPM1 mutation was associated with higher complete remission rates, lower cumulative incidence of relapse and higher overall survival in intermediate risk AML patients		Supports	3	1	3	1	180	90	2015-02-20 17:02:22.893253	2015-02-20 17:02:22.893253	4	accepted	\N	\N	1
300	Complete remission rates were higher and both disease-free and overall survival were longer for patients with Exon 12 NPM1 mutations.		Supports	3	1	3	1	171	90	2015-02-20 17:02:22.950305	2015-02-20 17:02:22.950305	3	accepted	\N	\N	1
301	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed		Supports	3	1	3	1	122	90	2015-02-20 17:02:23.009893	2015-02-20 17:02:23.009893	3	accepted	\N	\N	1
302	Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved overall and relapse-free survival, complete remission rates and remission duration compared to all other combinations of mutants		Supports	3	1	3	1	181	90	2015-02-20 17:02:23.076464	2015-02-20 17:02:23.076464	3	accepted	\N	\N	1
303	NPM1 mutation was associated with increased overall, event-free and relapse-free survival and remission after induction therapy in the entire cohort and in the cytogenetically normal subset of patients		Supports	3	1	3	1	182	90	2015-02-20 17:02:23.144492	2015-02-20 17:02:23.144492	3	accepted	\N	\N	1
304	Intermediate risk patients with mutant NPM1 had improved overall survival with the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1 and IDH2		Supports	3	1	3	1	175	90	2015-02-20 17:02:23.218589	2015-02-20 17:02:23.218589	2	accepted	\N	\N	1
305	NPM1 mutation without FLT3-ITD was significantly associated with complete remission		Supports	3	1	3	1	88	90	2015-02-20 17:02:23.277549	2015-02-20 17:02:23.277549	3	accepted	\N	\N	1
306	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation		Does Not Support	3	1	3	1	126	90	2015-02-20 17:02:23.329483	2015-02-20 17:02:23.329483	3	accepted	\N	\N	1
307	Coocurrence of FLT3-ITD mutations significantly reduced overall survival of normal karyotype AML patients		Supports	3	1	3	1	171	90	2015-02-20 17:02:23.391731	2015-02-20 17:02:23.391731	4	accepted	\N	\N	1
308	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3		Supports	3	1	3	1	115	90	2015-02-20 17:02:23.460434	2015-02-20 17:02:23.460434	4	accepted	\N	\N	1
309	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation		Supports	3	1	3	1	115	90	2015-02-20 17:02:23.511633	2015-02-20 17:02:23.511633	4	accepted	\N	\N	1
310	TET2 mutation reduces overall survival of normal karyotype patients with either an NPM1 mutation or NPM1 mutation without FLT3-ITD		Supports	3	1	3	1	183	90	2015-02-20 17:02:23.57126	2015-02-20 17:02:23.57126	3	accepted	\N	\N	1
311	Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate risk AML patients		Supports	3	1	3	1	180	90	2015-02-20 17:02:23.632556	2015-02-20 17:02:23.632556	3	accepted	\N	\N	1
312	"AML with mutated NPM1" is a provisional entity in WHO classification of acute myeloid leukemia (AML). This mutation should be tested for in clinical trials and is recommended for testing in patients with cytogentically normal AML.		Supports	2	2	3	1	184	90	2015-02-20 17:02:23.699701	2015-02-20 17:02:23.699701	5	accepted	\N	\N	1
313	No NPM1 mutations were identified in patients with favorable risk cytogenetics (79/215 patients)		Does Not Support	2	1	3	1	94	91	2015-02-20 17:02:23.76221	2015-02-20 17:02:23.76221	4	accepted	\N	\N	1
314	NPM1 mutations were associated with intermediate risk cytogenetics (including normal karyotype)		Supports	2	1	3	1	94	91	2015-02-20 17:02:23.839781	2015-02-20 17:02:23.839781	4	accepted	\N	\N	1
315	NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.		Supports	1	3	54	1	174	91	2015-02-20 17:02:23.905144	2015-02-20 17:02:23.905144	3	accepted	\N	\N	1
316	NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM in AML patients was associated with good response to induction therapy.		Supports	1	4	3	1	185	91	2015-02-20 17:02:23.988361	2015-02-20 17:02:23.988361	3	accepted	\N	\N	1
317	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS		Supports	3	1	3	8	186	92	2015-02-20 17:02:24.06397	2015-02-20 17:02:24.06397	3	accepted	\N	\N	1
318	The presence of NRAS mutation in AML patients does not impact diease prognosis (resistant disease, disease-free survival, complete remission rate, relapse rate, induction death)		Does Not Support	3	1	3	1	187	93	2015-02-20 17:02:24.17324	2015-02-20 17:02:24.17324	3	accepted	\N	\N	1
319	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis.		Supports	3	1	3	7	188	93	2015-02-20 17:02:24.242675	2015-02-20 17:02:24.242675	3	accepted	\N	\N	1
320	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS		Supports	3	1	3	8	186	93	2015-02-20 17:02:24.304798	2015-02-20 17:02:24.304798	3	accepted	\N	\N	1
321	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis.		Supports	3	2	3	7	188	94	2015-02-20 17:02:24.3698	2015-02-20 17:02:24.3698	3	accepted	\N	\N	1
322	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis.		Supports	3	2	3	7	188	95	2015-02-20 17:02:24.42667	2015-02-20 17:02:24.42667	3	accepted	\N	\N	1
323	NRAS mutation status does not impact overall, event-free or disease-free survival in patients with AML		Does Not Support	3	1	3	1	189	96	2015-02-20 17:02:24.497339	2015-02-20 17:02:24.497339	3	accepted	\N	\N	1
324	Likely due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic malignant melanoma with an NRAS G13D mutation		Supports	1	1	56	8	190	97	2015-02-20 17:02:24.561986	2015-02-20 17:02:24.561986	2	accepted	\N	\N	1
325	Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation (primarily Q61) have a significantly lower response rate to cetuximab than patients wildtype for NRAS.		Supports	1	1	14	7	62	98	2015-02-20 17:02:24.629178	2015-02-20 17:02:24.629178	3	accepted	\N	\N	1
326	In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status (primarily Q61) was not informative for disease control rate, progression free survival or overall survival following cetuximab plus chemotherapy.		Does Not Support	1	1	14	7	62	98	2015-02-20 17:02:24.696222	2015-02-20 17:02:24.696222	3	accepted	\N	\N	1
327	Melanoma associated with NRAS Q61 mutation was more often associated with those at the extremity than those at the trunk		Supports	2	1	3	8	191	98	2015-02-20 17:02:24.76502	2015-02-20 17:02:24.76502	3	accepted	\N	\N	1
328	Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.		Supports	1	1	19	8	192	98	2015-02-20 17:02:24.824111	2015-02-20 17:02:24.824111	4	accepted	\N	\N	1
329	In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted in disease free survival of 14 months		Supports	1	1	57	8	193	99	2015-02-20 17:02:24.892213	2015-02-20 17:02:24.892213	2	accepted	\N	\N	1
330	In a melanoma patient with Q61R mutation, treatment with temozolomide resulted in overall survival of 16 months		Supports	1	1	57	8	193	100	2015-02-20 17:02:24.952016	2015-02-20 17:02:24.952016	2	accepted	\N	\N	1
331	In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.		Supports	1	3	58	38	194	101	2015-02-20 17:02:25.022872	2015-02-20 17:02:25.022872	4	accepted	\N	\N	1
332	GIST tumors harboring PDGFRA D842V mutation are likely to be benign		Supports	2	1	3	38	195	102	2015-02-20 17:02:25.099793	2015-02-20 17:02:25.099793	3	accepted	\N	\N	1
333	While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib, double mutation of V561D and D842V mutants are resistant to imatinib.		Supports	1	1	1	38	150	102	2015-02-20 17:02:25.164254	2015-02-20 17:02:25.164254	4	accepted	\N	\N	1
334	GIST cancer with D842V mutation is resistant to imatinib.		Supports	1	1	1	38	196	102	2015-02-20 17:02:25.225428	2015-02-20 17:02:25.225428	4	accepted	\N	\N	1
335	In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.		Supports	1	3	58	38	194	102	2015-02-20 17:02:25.295602	2015-02-20 17:02:25.295602	4	accepted	\N	\N	1
336	In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.		Supports	1	3	58	38	194	103	2015-02-20 17:02:25.367109	2015-02-20 17:02:25.367109	4	accepted	\N	\N	1
366	Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib, compared to wild-type RET.		Supports	1	3	61	40	204	115	2015-02-20 17:02:27.361971	2015-02-20 17:02:27.361971	3	accepted	\N	\N	1
337	In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.		Supports	1	3	58	38	194	104	2015-02-20 17:02:25.443697	2015-02-20 17:02:25.443697	4	accepted	\N	\N	1
338	In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.		Supports	1	3	58	38	194	105	2015-02-20 17:02:25.510252	2015-02-20 17:02:25.510252	4	accepted	\N	\N	1
339	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.		Supports	1	3	59	11	197	106	2015-02-20 17:02:25.581519	2015-02-20 17:02:25.581519	4	accepted	\N	\N	1
340	Rapamycin inhibits transformation induced by mutation in PIK3CA		Supports	1	3	60	11	198	106	2015-02-20 17:02:25.639713	2015-02-20 17:02:25.639713	3	accepted	\N	\N	1
341	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment		Does Not Support	1	1	49	7	153	106	2015-02-20 17:02:25.742348	2015-02-20 17:02:25.742348	4	accepted	\N	\N	1
342	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.		Supports	1	4	14	7	199	106	2015-02-20 17:02:25.809975	2015-02-20 17:02:25.809975	3	accepted	\N	\N	1
343	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.		Supports	1	4	12	7	199	106	2015-02-20 17:02:25.860798	2015-02-20 17:02:25.860798	3	accepted	\N	\N	1
344	The presence of exon 9 (such as E542K) or exon 20 mutations were associated with poorer survival compared to those with wild-type PIK3CA mutation.		Supports	3	4	3	7	199	106	2015-02-20 17:02:25.921176	2015-02-20 17:02:25.921176	3	accepted	\N	\N	1
345	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.		Supports	3	4	3	7	200	106	2015-02-20 17:02:25.992813	2015-02-20 17:02:25.992813	2	accepted	\N	\N	1
346	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.		Supports	1	3	59	11	197	107	2015-02-20 17:02:26.055133	2015-02-20 17:02:26.055133	4	accepted	\N	\N	1
347	Rapamycin inhibits transformation induced by mutation in PIK3CA		Supports	1	3	60	11	198	107	2015-02-20 17:02:26.113361	2015-02-20 17:02:26.113361	3	accepted	\N	\N	1
348	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment		Does Not Support	1	1	49	7	153	107	2015-02-20 17:02:26.171727	2015-02-20 17:02:26.171727	4	accepted	\N	\N	1
349	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.		Supports	1	4	14	7	199	107	2015-02-20 17:02:26.23405	2015-02-20 17:02:26.23405	3	accepted	\N	\N	1
350	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.		Supports	1	4	12	7	199	107	2015-02-20 17:02:26.293453	2015-02-20 17:02:26.293453	3	accepted	\N	\N	1
351	The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.		Supports	3	4	3	7	199	107	2015-02-20 17:02:26.352425	2015-02-20 17:02:26.352425	3	accepted	\N	\N	1
352	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.		Supports	3	4	3	7	200	107	2015-02-20 17:02:26.417234	2015-02-20 17:02:26.417234	2	accepted	\N	\N	1
353	Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA		Supports	1	1	14	7	62	108	2015-02-20 17:02:26.476201	2015-02-20 17:02:26.476201	3	accepted	\N	\N	1
354	In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations (Exon 9) had no significant impact on response rate, disease control rate, progression free surivival or overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA		Does Not Support	1	1	14	7	62	109	2015-02-20 17:02:26.533581	2015-02-20 17:02:26.533581	3	accepted	\N	\N	1
355	Breast cancer cell lines with H1047R mutation showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.		Supports	1	3	59	11	197	110	2015-02-20 17:02:26.589832	2015-02-20 17:02:26.589832	4	accepted	\N	\N	1
356	Rapamycin inhibits transformation induced by mutation in PIK3CA		Supports	1	3	60	11	198	110	2015-02-20 17:02:26.658204	2015-02-20 17:02:26.658204	3	accepted	\N	\N	1
357	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment		Does Not Support	1	1	49	7	153	110	2015-02-20 17:02:26.729672	2015-02-20 17:02:26.729672	4	accepted	\N	\N	1
358	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.		Supports	1	4	14	7	199	110	2015-02-20 17:02:26.790498	2015-02-20 17:02:26.790498	3	accepted	\N	\N	1
359	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.		Supports	1	4	12	7	199	110	2015-02-20 17:02:26.840693	2015-02-20 17:02:26.840693	3	accepted	\N	\N	1
360	The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.		Supports	3	4	3	7	199	110	2015-02-20 17:02:26.903211	2015-02-20 17:02:26.903211	3	accepted	\N	\N	1
361	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.		Supports	1	1	41	39	201	111	2015-02-20 17:02:26.967892	2015-02-20 17:02:26.967892	5	accepted	\N	\N	1
362	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.		Supports	2	1	3	20	91	112	2015-02-20 17:02:27.029955	2015-02-20 17:02:27.029955	4	accepted	\N	\N	1
363	Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse to PI3K-mTOR inhibitors.		Supports	1	3	3	3	202	113	2015-02-20 17:02:27.094528	2015-02-20 17:02:27.094528	4	accepted	\N	\N	1
364	PTEN nonsense mutations, including R233*, have been shown to be inactivating and loss-of-function, but do not have prognostic value in glioblastoma multiforme patients.		Does Not Support	3	1	3	37	203	113	2015-02-20 17:02:27.170056	2015-02-20 17:02:27.170056	3	accepted	\N	\N	1
365	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.		Supports	1	1	41	39	201	114	2015-02-20 17:02:27.288196	2015-02-20 17:02:27.288196	5	accepted	\N	\N	1
367	Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib, compared to wild-type RET.		Supports	1	3	61	40	204	116	2015-02-20 17:02:27.428256	2015-02-20 17:02:27.428256	3	accepted	\N	\N	1
368	In patients with medullary carcinoma, the presence of RET M918T mutation is associated with increased probability of lymph node metastases.		Supports	2	1	3	40	205	116	2015-02-20 17:02:27.496391	2015-02-20 17:02:27.496391	5	accepted	\N	\N	1
369	The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong repression of tyroid cancer cell growth.		Supports	1	3	62	40	206	116	2015-02-20 17:02:27.556389	2015-02-20 17:02:27.556389	3	accepted	\N	\N	1
370	Compared to those who harbor a wild type RET, patients with RET M918T mutation develop larger and more aggressive medullary thyroid cancer.		Supports	3	1	3	40	207	116	2015-02-20 17:02:27.623516	2015-02-20 17:02:27.623516	4	accepted	\N	\N	1
371	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.		Supports	3	4	3	27	208	117	2015-02-20 17:02:27.687674	2015-02-20 17:02:27.687674	3	accepted	\N	\N	1
372	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.		Supports	3	4	3	27	208	118	2015-02-20 17:02:27.746803	2015-02-20 17:02:27.746803	3	accepted	\N	\N	1
373	Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.		Supports	1	3	63	11	209	119	2015-02-20 17:02:27.818175	2015-02-20 17:02:27.818175	3	accepted	\N	\N	1
374	Breast cancer patients who harbor R175H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.		Supports	3	1	3	11	210	119	2015-02-20 17:02:27.880785	2015-02-20 17:02:27.880785	3	accepted	\N	\N	1
375	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R248 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.		Supports	3	1	3	11	211	120	2015-02-20 17:02:27.949447	2015-02-20 17:02:27.949447	3	accepted	\N	\N	1
376	Breast cancer patients who harbor R248Q mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.		Supports	3	1	3	11	210	120	2015-02-20 17:02:28.011053	2015-02-20 17:02:28.011053	3	accepted	\N	\N	1
377	In breast cancer patients harboring R248W mutation, the prognosis is worse than any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.		Supports	3	1	3	11	210	121	2015-02-20 17:02:28.071601	2015-02-20 17:02:28.071601	3	accepted	\N	\N	1
378	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.		Supports	3	1	3	11	211	122	2015-02-20 17:02:28.127991	2015-02-20 17:02:28.127991	3	accepted	\N	\N	1
379	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.		Supports	3	1	3	11	211	123	2015-02-20 17:02:28.199373	2015-02-20 17:02:28.199373	3	accepted	\N	\N	1
380	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.		Supports	3	1	3	11	211	124	2015-02-20 17:02:28.269995	2015-02-20 17:02:28.269995	3	accepted	\N	\N	1
381	Breast cancer patients who harbor R273C mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.		Supports	3	1	3	11	210	124	2015-02-20 17:02:28.328209	2015-02-20 17:02:28.328209	3	accepted	\N	\N	1
382	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.		Supports	3	1	3	11	211	125	2015-02-20 17:02:28.390269	2015-02-20 17:02:28.390269	3	accepted	\N	\N	1
383	Breast cancer patients who harbor R273H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.		Supports	3	1	3	11	210	125	2015-02-20 17:02:28.451444	2015-02-20 17:02:28.451444	3	accepted	\N	\N	1
384	Breast tumor with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.		Supports	3	1	3	11	211	126	2015-02-20 17:02:28.510927	2015-02-20 17:02:28.510927	3	accepted	\N	\N	1
385	A patient with metastatic bladder cancer that responded well to the mTOR inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del, p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations also had mild responses to everolimus.		Supports	1	1	64	41	212	127	2015-02-20 17:02:28.578053	2015-02-20 17:02:28.578053	3	accepted	\N	\N	1
386	Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit increased overall survival, increased apoptosis, reduced proliferation and reduced tumor burden compared to Tsc1 wildtype mice.		Supports	1	3	60	4	213	127	2015-02-20 17:02:28.661742	2015-02-20 17:02:28.661742	3	accepted	\N	\N	1
387	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.		Supports	1	1	64	42	214	128	2015-02-20 17:02:28.729924	2015-02-20 17:02:28.729924	3	accepted	\N	\N	1
388	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.		Supports	1	1	64	42	212	129	2015-02-20 17:02:28.846987	2015-02-20 17:02:28.846987	3	accepted	\N	\N	1
389	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.		Does Not Support	2	4	3	1	215	130	2015-02-20 17:02:28.914565	2015-02-20 17:02:28.914565	3	accepted	\N	\N	1
390	In patients with AML, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.		Does Not Support	3	4	3	1	215	130	2015-02-20 17:02:28.97548	2015-02-20 17:02:28.97548	2	accepted	\N	\N	1
391	In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.		Does Not Support	3	4	3	27	215	130	2015-02-20 17:02:29.04087	2015-02-20 17:02:29.04087	2	accepted	\N	\N	1
392	After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.		Supports	3	4	3	27	216	130	2015-02-20 17:02:29.109022	2015-02-20 17:02:29.109022	3	accepted	\N	\N	1
393	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.		Does Not Support	2	4	3	1	215	131	2015-02-20 17:02:29.164483	2015-02-20 17:02:29.164483	3	accepted	\N	\N	1
394	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.		Does Not Support	3	4	3	1	215	131	2015-02-20 17:02:29.225337	2015-02-20 17:02:29.225337	2	accepted	\N	\N	1
395	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.		Does Not Support	3	4	3	27	215	131	2015-02-20 17:02:29.288322	2015-02-20 17:02:29.288322	2	accepted	\N	\N	1
396	After adjust for age and cancer stage, the presence of U2AF mutations such as S34Y/F are prognostic for poorer survival outcomes in patients with MDS.		Supports	3	4	3	27	216	131	2015-02-20 17:02:29.35747	2015-02-20 17:02:29.35747	3	accepted	\N	\N	1
397	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	1	1	3	1	217	132	2015-02-20 17:02:29.426936	2015-02-20 17:02:29.426936	3	accepted	\N	\N	1
398	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML		Does Not Support	3	1	3	1	218	132	2015-02-20 17:02:29.496759	2015-02-20 17:02:29.496759	4	accepted	\N	\N	1
399	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML		Does Not Support	3	1	3	1	218	132	2015-02-20 17:02:29.557188	2015-02-20 17:02:29.557188	3	accepted	\N	\N	1
400	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	3	1	3	1	217	132	2015-02-20 17:02:29.624853	2015-02-20 17:02:29.624853	3	accepted	\N	\N	1
401	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	3	1	3	1	217	132	2015-02-20 17:02:29.693305	2015-02-20 17:02:29.693305	3	accepted	\N	\N	1
402	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients		Supports	3	1	3	1	219	132	2015-02-20 17:02:29.757593	2015-02-20 17:02:29.757593	3	accepted	\N	\N	1
403	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML		Supports	3	1	3	1	218	132	2015-02-20 17:02:29.817693	2015-02-20 17:02:29.817693	3	accepted	\N	\N	1
404	Mutations in WT1 were associated with a worse overall prognosis than patients wildtype for WT1 in young patients (15-50), primarily because of increased risk of disease recurrence		Supports	3	1	3	1	220	132	2015-02-20 17:02:29.887314	2015-02-20 17:02:29.887314	2	accepted	\N	\N	1
405	Mutations  in WT1 were associated with increased risk of recurrence in young patients (15-50) with cytogenetically normal AML		Supports	3	1	3	1	220	132	2015-02-20 17:02:29.951737	2015-02-20 17:02:29.951737	2	accepted	\N	\N	1
406	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	1	1	3	1	217	133	2015-02-20 17:02:30.025678	2015-02-20 17:02:30.025678	3	accepted	\N	\N	1
407	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML.		Does Not Support	3	1	3	1	218	133	2015-02-20 17:02:30.098691	2015-02-20 17:02:30.098691	4	accepted	\N	\N	1
408	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML		Does Not Support	3	1	3	1	218	133	2015-02-20 17:02:30.158707	2015-02-20 17:02:30.158707	3	accepted	\N	\N	1
409	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	3	1	3	1	217	133	2015-02-20 17:02:30.217557	2015-02-20 17:02:30.217557	3	accepted	\N	\N	1
410	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	3	1	3	1	217	133	2015-02-20 17:02:30.321159	2015-02-20 17:02:30.321159	3	accepted	\N	\N	1
411	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients		Supports	3	1	3	1	219	133	2015-02-20 17:02:30.387643	2015-02-20 17:02:30.387643	3	accepted	\N	\N	1
412	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML		Supports	3	1	3	1	218	133	2015-02-20 17:02:30.450497	2015-02-20 17:02:30.450497	3	accepted	\N	\N	1
\.


--
-- Name: evidence_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_items_id_seq', 412, true);


--
-- Name: evidence_levels_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_levels_id_seq', 4, true);


--
-- Name: evidence_types_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_types_id_seq', 3, true);


--
-- Name: genes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('genes_id_seq', 47, true);


--
-- Data for Name: genes_sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes_sources (gene_id, source_id, created_at, updated_at) FROM stdin;
1	1	2015-02-20 17:01:48.536288	2015-02-20 17:01:48.536288
1	2	2015-02-20 17:01:48.543691	2015-02-20 17:01:48.543691
2	3	2015-02-20 17:01:49.091646	2015-02-20 17:01:49.091646
2	4	2015-02-20 17:01:49.097359	2015-02-20 17:01:49.097359
3	5	2015-02-20 17:01:49.293184	2015-02-20 17:01:49.293184
3	6	2015-02-20 17:01:49.299111	2015-02-20 17:01:49.299111
4	7	2015-02-20 17:01:49.500623	2015-02-20 17:01:49.500623
4	8	2015-02-20 17:01:49.505151	2015-02-20 17:01:49.505151
5	9	2015-02-20 17:01:49.712523	2015-02-20 17:01:49.712523
5	10	2015-02-20 17:01:49.717054	2015-02-20 17:01:49.717054
6	11	2015-02-20 17:01:49.927801	2015-02-20 17:01:49.927801
6	12	2015-02-20 17:01:49.932845	2015-02-20 17:01:49.932845
7	11	2015-02-20 17:01:50.133496	2015-02-20 17:01:50.133496
7	12	2015-02-20 17:01:50.137263	2015-02-20 17:01:50.137263
8	11	2015-02-20 17:01:50.335668	2015-02-20 17:01:50.335668
8	12	2015-02-20 17:01:50.33889	2015-02-20 17:01:50.33889
9	13	2015-02-20 17:01:50.533466	2015-02-20 17:01:50.533466
10	14	2015-02-20 17:01:50.736345	2015-02-20 17:01:50.736345
10	15	2015-02-20 17:01:50.741811	2015-02-20 17:01:50.741811
11	14	2015-02-20 17:01:50.94117	2015-02-20 17:01:50.94117
11	15	2015-02-20 17:01:50.9444	2015-02-20 17:01:50.9444
12	16	2015-02-20 17:01:51.153262	2015-02-20 17:01:51.153262
12	17	2015-02-20 17:01:51.157878	2015-02-20 17:01:51.157878
13	18	2015-02-20 17:01:51.364537	2015-02-20 17:01:51.364537
13	19	2015-02-20 17:01:51.369184	2015-02-20 17:01:51.369184
13	20	2015-02-20 17:01:51.374555	2015-02-20 17:01:51.374555
14	21	2015-02-20 17:01:51.577477	2015-02-20 17:01:51.577477
15	21	2015-02-20 17:01:51.793596	2015-02-20 17:01:51.793596
15	22	2015-02-20 17:01:51.798832	2015-02-20 17:01:51.798832
16	23	2015-02-20 17:01:52.007016	2015-02-20 17:01:52.007016
16	24	2015-02-20 17:01:52.012544	2015-02-20 17:01:52.012544
17	25	2015-02-20 17:01:52.236147	2015-02-20 17:01:52.236147
17	26	2015-02-20 17:01:52.240843	2015-02-20 17:01:52.240843
18	27	2015-02-20 17:01:52.458074	2015-02-20 17:01:52.458074
18	28	2015-02-20 17:01:52.462519	2015-02-20 17:01:52.462519
19	29	2015-02-20 17:01:52.67102	2015-02-20 17:01:52.67102
19	30	2015-02-20 17:01:52.675693	2015-02-20 17:01:52.675693
20	31	2015-02-20 17:01:52.88665	2015-02-20 17:01:52.88665
20	32	2015-02-20 17:01:52.891764	2015-02-20 17:01:52.891764
21	31	2015-02-20 17:01:53.099529	2015-02-20 17:01:53.099529
21	32	2015-02-20 17:01:53.103128	2015-02-20 17:01:53.103128
22	33	2015-02-20 17:01:53.374518	2015-02-20 17:01:53.374518
22	18	2015-02-20 17:01:53.377817	2015-02-20 17:01:53.377817
33	34	2015-02-20 17:01:55.60814	2015-02-20 17:01:55.60814
33	18	2015-02-20 17:01:55.611238	2015-02-20 17:01:55.611238
40	35	2015-02-20 17:01:56.999166	2015-02-20 17:01:56.999166
40	36	2015-02-20 17:01:57.00492	2015-02-20 17:01:57.00492
42	37	2015-02-20 17:01:57.473315	2015-02-20 17:01:57.473315
42	38	2015-02-20 17:01:57.478338	2015-02-20 17:01:57.478338
46	39	2015-02-20 17:01:58.332481	2015-02-20 17:01:58.332481
46	40	2015-02-20 17:01:58.337314	2015-02-20 17:01:58.337314
\.


--
-- Data for Name: notifications; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY notifications (id, subscription_id, user_id, subscribable_id, subscribable_type, content, url, acknowledged, delivered, created_at, updated_at) FROM stdin;
\.


--
-- Name: notifications_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('notifications_id_seq', 1, false);


--
-- Data for Name: ratings; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY ratings (id, value, evidence_item_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Name: ratings_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('ratings_id_seq', 1, false);


--
-- Data for Name: roles; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles (id, name, description, created_at, updated_at) FROM stdin;
\.


--
-- Name: roles_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('roles_id_seq', 1, false);


--
-- Data for Name: roles_users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles_users (role_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY schema_migrations (version) FROM stdin;
20150220160805
20140805184308
20141021192035
20141022180523
20141022195329
20141108234203
20141108234935
20141109000136
20141114222425
20150107205621
20150115231117
20150116220720
20150116230539
20150116230632
20150204192653
20150205000010
20150205205734
20150205210725
20150210191015
20150210204445
20150219202227
\.


--
-- Name: sources_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('sources_id_seq', 220, true);


--
-- Data for Name: subscriptions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY subscriptions (id, user_id, subscribable_id, subscribable_type, type, created_at, updated_at) FROM stdin;
\.


--
-- Name: subscriptions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('subscriptions_id_seq', 1, false);


--
-- Data for Name: suggested_changes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY suggested_changes (id, suggested_changes, moderated_id, moderated_type, user_id, status, created_at, updated_at) FROM stdin;
\.


--
-- Name: suggested_changes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('suggested_changes_id_seq', 1, false);


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('users_id_seq', 1, false);


--
-- Data for Name: variant_groups; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_groups (id, name, description, created_at, updated_at) FROM stdin;
9	KIT Exon 17	\N	2015-02-20 17:02:18.021374	2015-02-20 17:02:18.021374
10	KIT Exon 11	\N	2015-02-20 17:02:18.094794	2015-02-20 17:02:18.094794
1	Imatinib Resistance	While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. 	2015-02-20 17:02:03.275623	2015-02-20 17:02:30.553898
2	ALK Fusions	ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase the activity of the ALK oncogene relative to normal cells. While EML4 is the most common fusion partner, other 5' partners have been observed. The EML4-ALK fusion has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. 	2015-02-20 17:02:03.58915	2015-02-20 17:02:30.558132
5	Erlotinib Resistance	EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine kinase inhibitors have been developed to target EGFR pathway activity. One such inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting. However, the T790M missense mutation has shown to confer resistance to this inhibitor in cell lines and case studies.  	2015-02-20 17:02:11.663502	2015-02-20 17:02:30.560596
3	Crizotinib Resistance	The ALK oncogene has long been considered a driving factor in non-small cell lung cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive this resistance. 	2015-02-20 17:02:03.598293	2015-02-20 17:02:30.56479
4	Other V600's	While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants have also been observed and studied to a lesser degree. At first approximation, many of these variants seem to behave similarly to V600E, and treatment with dabrafenib has been shown to be effective. 	2015-02-20 17:02:04.92421	2015-02-20 17:02:30.568545
6	HER2 Activating	HER2-positive breast cancer is widely recognized as a molecular subtype of breast cancer. While amplification of ERBB2 is a common mechanism for activation of the pathway, missense mutations have also demonstrated activation potential. These activating missense mutations also confer senstivity to the targeted therapeutic neratinib. 	2015-02-20 17:02:12.152371	2015-02-20 17:02:30.571903
7	ESR1 Ligand-Binding Domain	ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone therapy has been widely effective in treating the disease, however sequencing of resistant patients has uncovered a number of missense mutations in mediating this resistance. Many of these mutations lie within the ligand-binding domain, and contribute to constitutive activity of the receptor. This has lead to the development of estrogen receptor degrading agents such as fulvestrant, which have shown early efficacy in clinical trials.  	2015-02-20 17:02:13.112172	2015-02-20 17:02:30.574649
8	FGFR fusions	FGFR fusions have been demonstrated across many cancer types to have oncogenic potential. FGFR2 and FGFR3 have been shown to be fused to many 3' partners, which may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2 fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib and pazopanib have shown efficacy in case studies.  	2015-02-20 17:02:14.086169	2015-02-20 17:02:30.57694
11	NPM1 exon 12	To be filled in by Dr. KK, resident blood expert and knower of all things hematology. :) :) :) 	2015-02-20 17:02:23.818256	2015-02-20 17:02:30.579385
12	PTEN Loss-of-Function	PTEN loss is a common event in breast cancer, as well as others. The most common single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell lines harboring this mutation behave similarly to cell lines harboring larger insertion or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded growth. 	2015-02-20 17:02:27.147505	2015-02-20 17:02:30.582864
13	Motesanib Resistance	RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment of these patients with the targeted therapeutic motesanib has shown to be effective. However, the missense mutations C634W and M918T have shown to confer motesanib resistance in cell lines. 	2015-02-20 17:02:27.4081	2015-02-20 17:02:30.5861
14	TSC Loss	Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types. Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor everolimus has seen significant anti-tumorigenic activity. 	2015-02-20 17:02:28.640135	2015-02-20 17:02:30.588621
\.


--
-- Data for Name: variant_group_variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_group_variants (variant_id, variant_group_id, created_at, updated_at) FROM stdin;
3	1	2015-02-20 17:02:03.285085	2015-02-20 17:02:03.285085
2	1	2015-02-20 17:02:03.353206	2015-02-20 17:02:03.353206
2	1	2015-02-20 17:02:03.421501	2015-02-20 17:02:03.421501
6	2	2015-02-20 17:02:03.593924	2015-02-20 17:02:03.593924
6	3	2015-02-20 17:02:03.602374	2015-02-20 17:02:03.602374
5	2	2015-02-20 17:02:03.675617	2015-02-20 17:02:03.675617
8	3	2015-02-20 17:02:04.07981	2015-02-20 17:02:04.07981
7	2	2015-02-20 17:02:04.233519	2015-02-20 17:02:04.233519
7	3	2015-02-20 17:02:04.237386	2015-02-20 17:02:04.237386
11	4	2015-02-20 17:02:04.927555	2015-02-20 17:02:04.927555
13	4	2015-02-20 17:02:06.909992	2015-02-20 17:02:06.909992
15	4	2015-02-20 17:02:06.977895	2015-02-20 17:02:06.977895
16	4	2015-02-20 17:02:07.041311	2015-02-20 17:02:07.041311
34	5	2015-02-20 17:02:11.666734	2015-02-20 17:02:11.666734
35	6	2015-02-20 17:02:12.15594	2015-02-20 17:02:12.15594
36	6	2015-02-20 17:02:12.230365	2015-02-20 17:02:12.230365
38	6	2015-02-20 17:02:12.367487	2015-02-20 17:02:12.367487
41	6	2015-02-20 17:02:12.643117	2015-02-20 17:02:12.643117
43	6	2015-02-20 17:02:12.830956	2015-02-20 17:02:12.830956
44	6	2015-02-20 17:02:12.900453	2015-02-20 17:02:12.900453
45	6	2015-02-20 17:02:12.96495	2015-02-20 17:02:12.96495
46	7	2015-02-20 17:02:13.115012	2015-02-20 17:02:13.115012
46	7	2015-02-20 17:02:13.213844	2015-02-20 17:02:13.213844
47	7	2015-02-20 17:02:13.337542	2015-02-20 17:02:13.337542
47	7	2015-02-20 17:02:13.403071	2015-02-20 17:02:13.403071
48	7	2015-02-20 17:02:13.526094	2015-02-20 17:02:13.526094
48	7	2015-02-20 17:02:13.594547	2015-02-20 17:02:13.594547
49	7	2015-02-20 17:02:13.745904	2015-02-20 17:02:13.745904
49	7	2015-02-20 17:02:13.813227	2015-02-20 17:02:13.813227
50	7	2015-02-20 17:02:13.941393	2015-02-20 17:02:13.941393
50	7	2015-02-20 17:02:14.009069	2015-02-20 17:02:14.009069
56	8	2015-02-20 17:02:14.090231	2015-02-20 17:02:14.090231
59	8	2015-02-20 17:02:14.212931	2015-02-20 17:02:14.212931
70	9	2015-02-20 17:02:18.024546	2015-02-20 17:02:18.024546
71	10	2015-02-20 17:02:18.099144	2015-02-20 17:02:18.099144
71	10	2015-02-20 17:02:18.173631	2015-02-20 17:02:18.173631
71	10	2015-02-20 17:02:18.244028	2015-02-20 17:02:18.244028
71	10	2015-02-20 17:02:18.302834	2015-02-20 17:02:18.302834
71	10	2015-02-20 17:02:18.36459	2015-02-20 17:02:18.36459
71	10	2015-02-20 17:02:18.427988	2015-02-20 17:02:18.427988
74	9	2015-02-20 17:02:18.610677	2015-02-20 17:02:18.610677
74	9	2015-02-20 17:02:18.679773	2015-02-20 17:02:18.679773
76	10	2015-02-20 17:02:18.991954	2015-02-20 17:02:18.991954
76	10	2015-02-20 17:02:19.059137	2015-02-20 17:02:19.059137
76	10	2015-02-20 17:02:19.218262	2015-02-20 17:02:19.218262
77	10	2015-02-20 17:02:19.282668	2015-02-20 17:02:19.282668
77	10	2015-02-20 17:02:19.383391	2015-02-20 17:02:19.383391
91	11	2015-02-20 17:02:23.822644	2015-02-20 17:02:23.822644
91	11	2015-02-20 17:02:23.887036	2015-02-20 17:02:23.887036
91	11	2015-02-20 17:02:23.966708	2015-02-20 17:02:23.966708
91	11	2015-02-20 17:02:24.043477	2015-02-20 17:02:24.043477
101	1	2015-02-20 17:02:25.074905	2015-02-20 17:02:25.074905
102	1	2015-02-20 17:02:25.147751	2015-02-20 17:02:25.147751
102	1	2015-02-20 17:02:25.208258	2015-02-20 17:02:25.208258
102	1	2015-02-20 17:02:25.279039	2015-02-20 17:02:25.279039
102	1	2015-02-20 17:02:25.349403	2015-02-20 17:02:25.349403
103	1	2015-02-20 17:02:25.426726	2015-02-20 17:02:25.426726
104	1	2015-02-20 17:02:25.493465	2015-02-20 17:02:25.493465
105	1	2015-02-20 17:02:25.560175	2015-02-20 17:02:25.560175
113	12	2015-02-20 17:02:27.150452	2015-02-20 17:02:27.150452
113	12	2015-02-20 17:02:27.271355	2015-02-20 17:02:27.271355
115	13	2015-02-20 17:02:27.410857	2015-02-20 17:02:27.410857
116	13	2015-02-20 17:02:27.471926	2015-02-20 17:02:27.471926
127	14	2015-02-20 17:02:28.643158	2015-02-20 17:02:28.643158
127	14	2015-02-20 17:02:28.706999	2015-02-20 17:02:28.706999
128	14	2015-02-20 17:02:28.826905	2015-02-20 17:02:28.826905
129	14	2015-02-20 17:02:28.895263	2015-02-20 17:02:28.895263
\.


--
-- Name: variant_groups_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_groups_id_seq', 14, true);


--
-- Name: variant_origins_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_origins_id_seq', 1, true);


--
-- Name: variants_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variants_id_seq', 133, true);


--
-- PostgreSQL database dump complete
--

