--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

SET search_path = public, pg_catalog;

--
-- Data for Name: advanced_searches; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY advanced_searches (id, created_at, updated_at, params, search_type, token) FROM stdin;
\.


--
-- Name: advanced_searches_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('advanced_searches_id_seq', 1, false);


--
-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY users (id, email, name, url, username, created_at, updated_at, orcid, area_of_expertise, deleted, deleted_at, role, last_seen_at) FROM stdin;
\.


--
-- Data for Name: audits; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY audits (id, auditable_id, auditable_type, associated_id, associated_type, user_id, user_type, username, action, audited_changes, version, comment, remote_address, request_uuid, created_at) FROM stdin;
1	1	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 238\nname: ALK\ndescription: ALK amplifications, fusions and mutations have been shown to be driving\n  events in non-small cell lung cancer. While crizontinib has demonstrated efficacy\n  in treating the amplification, mutations in ALK have been shown to confer resistance\n  to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied\n  success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been\n  shown to be cytostatic in ALK-altered cell lines.\nofficial_name: anaplastic lymphoma receptor tyrosine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	c0ea4ddf-aacc-4ced-b706-9cc09bb8ea34	2015-12-09 16:58:27.56109
2	2	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 207\nname: AKT1\ndescription: AKT1, also referred to as protein kinase B, is a known oncogene. AKT\n  activation relies on the PI3K pathway, and is recognized as a critical node in the\n  pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been\n  shown to result in activation of the protein. Mutations in AKT1 have also been shown\n  to confer resistance to allosteric kinase inhibitors in vitro.\nofficial_name: v-akt murine thymoma viral oncogene homolog 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	af4015e2-84db-42ef-a972-d01c5d08ee5e	2015-12-09 16:58:27.904088
3	3	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 369\nname: ARAF\ndescription: ARAF has recently become increasingly considered for its oncogenic potential.\n  Its potential as a target for informing clinical action was demonstrated by a single\n  case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated\n  with sorafenib, acheived near-complete clinical remission. This finding has brought\n  new focus on ARAF as a marker that should be assayed for in cancer treatment.\nofficial_name: A-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	105a6d8b-1a60-44d1-a67c-c7ff3f54f4f6	2015-12-09 16:58:28.160466
4	4	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 25\nname: ABL1\ndescription: ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein\n  that has become a signature of chronic myeloid leukemia (CML). Cells harboring this\n  fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook\n  of the disease. However, additional mutations in ABL1 have been shown to confer\n  resistance to imatinib. In these resistance cases, second-generation tyrosine kinase\n  inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are\n  currently undergoing clinical trials for treating acquired resistance in CML.\nofficial_name: ABL proto-oncogene 1, non-receptor tyrosine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b1a377eb-e50a-4a61-8f05-0955584cf2c8	2015-12-09 16:58:28.44671
5	5	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 673\nname: BRAF\ndescription: BRAF mutations are found to be recurrent in many cancer types. Of these,\n  the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E\n  has been determined to be an activating mutation, and cells that harbor it, along\n  with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is\n  also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK\n  inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations\n  have also been correlated with poor prognosis in many cancer types, although there\n  is at least one study that questions this conclusion in papillary thyroid cancer.\nofficial_name: B-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b2247b57-4112-4876-b28a-02b953ab12f8	2015-12-09 16:58:28.695491
6	6	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 672\nname: BRCA1\ndescription: BRCA1 mutations in the germline have become a hallmark for hereditary\n  breast and ovarian cancers. Variants that have been demonstrated to reduce the function\n  of the protein have been shown to increase risk for these cancers, as well as prostate\n  and pancreatic cancer. These findings have been the impetus for the increased popularity\n  of genetic testing of healthy indivudals to assess risk.\nofficial_name: breast cancer 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	3af97817-e272-4bbf-ba67-4cd00c031cf6	2015-12-09 16:58:28.958328
7	7	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 675\nname: BRCA2\ndescription: BRCA2 mutations in the germline have become a hallmark for hereditary\n  breast and ovarian cancers. Variants that have been demonstrated to reduce the function\n  of the protein have been shown to increase risk for these cancers, as well as prostate\n  and pancreatic cancer. These findings have been the impetus for the increased popularity\n  of genetic testing of healthy indivudals to assess risk.\nofficial_name: breast cancer 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	27f266fa-e946-4901-8cf5-76e69f3a576e	2015-12-09 16:58:29.247347
8	8	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 595\nname: CCND1\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	d78bdfc1-4c10-4ba7-985e-484ac3180f60	2015-12-09 16:58:29.498085
9	9	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 894\nname: CCND2\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	ca9d0720-d9d1-440f-a608-ebf68d6bf998	2015-12-09 16:58:29.717707
10	10	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 896\nname: CCND3\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared\n  to the amplifcations and overexpressions of the other cyclin D's. In a mouse study,\n  the targeted therapeutic palbociclib significantly increased the median survival\n  of the cyclin D3 knockouts.\nofficial_name: cyclin D3\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	e3194876-e7b2-4f81-a6a5-b7fdef546953	2015-12-09 16:58:29.908769
11	11	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 898\nname: CCNE1\ndescription: Cyclin E, while currenly not as widely implicated as its cyclin D counterparts,\n  has been implicated in various carcinomas, including breast, gastric, stomach and\n  colorectal. High levels of cyclin E, either by gene amplification or overexpression,\n  have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements\n  are correlated with later stage disease. In lung cancer, neoplastic cells with higher\n  levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly\n  expressed counterparts.\nofficial_name: cyclin E1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	da2c5cb6-e399-4104-b7b6-e6a333899bce	2015-12-09 16:58:30.108633
12	12	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1021\nname: CDK6\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 6\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	eeb3b39d-1d81-44e8-83c5-37fc65052299	2015-12-09 16:58:30.315561
13	13	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1019\nname: CDK4\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 4\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	dc99cd04-4dfb-48a8-bdc5-0a68a155c516	2015-12-09 16:58:30.540309
14	14	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1029\nname: CDKN2A\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. While no targeted therapeutic has been engaged in clinical trials, the prognostic\n  impact has been studied by a number of meta-analyses. One mechanism by which loss\n  of CDKN2A can occur is by hypermethylation of the promoter region for the gene.\n  However, the prognostic impact of promoter hypermethylation has been relatively\n  ambiguous. Many studies have suggesting poorer prognostic outcome for patients with\n  hypermethylation in colorectal, liver, and younger lung cancer patients. This being\n  said, there is still research to be done before this becomes a widely-accepted prognostic\n  indicator.\nofficial_name: cyclin-dependent kinase inhibitor 2A\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	923ab1b0-ca35-40d5-9e0f-e1c56985cf5f	2015-12-09 16:58:30.846897
15	15	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1050\nname: CEBPA\ndescription: AML with mutated CEBPA' is a provisional entity in the WHO classification\n  of acute myeloid leukemia (AML) and is recommended to be tested for in patients\n  with AML. CEBPA mutations are particularly associated with cytogenetically normal\n  AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation\n  in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha\n  protein while C-terminal mutations reduce the DNA-binding potential of this transcription\n  factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1\n  and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations\n  may have prognostic implications.\nofficial_name: CCAAT/enhancer binding protein (C/EBP), alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	dff5b71e-8534-47bf-8a67-c5053ed97dd6	2015-12-09 16:58:31.114004
16	16	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3337\nname: DNAJB1\ndescription: DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating\n  p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1\n  to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In\n  one study, this fusion was observed in 15/15 FL-HCC cases examined and functional\n  studies found that the fusion retained kinase activity.\nofficial_name: DnaJ heat shock protein family (Hsp40) member B1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	0d7218b9-2162-466d-af19-d989d26c7f6a	2015-12-09 16:58:31.355144
17	17	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5566\nname: PRKACA\ndescription: PRKACA has been studied in breast cancer and has been found to mediate\n  resistance to HER2 targeted therapies. It has also been found to contain a mutation\n  hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular\n  fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be\n  a diagnostic marker for this rare subtype of HCC. In one study, this fusion was\n  observed in 15/15 FL-HCC cases examined and functional studies found that the fusion\n  retained kinase activity.\nofficial_name: protein kinase, cAMP-dependent, alpha catalytic subunit\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	f010b294-89cb-4eba-b0bd-f14172007806	2015-12-09 16:58:31.570264
18	18	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1788\nname: DNMT3A\ndescription: DNMT3A is one of several epigenetic modifiers identified as recurrently\n  mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically\n  normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant\n  negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers.\n  AML patient bone marrow harboring R882 mutations were similarly demonstrated to\n  be hypomethylated compared to patients with wildtype DNMT3A. These studies also\n  indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner\n  from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes\n  and treatment protocols may need to be considered for these two classes of DNMT3A\n  mutations.\nofficial_name: DNA (cytosine-5-)-methyltransferase 3 alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	faad4481-e1fd-4cbb-98d7-197f9e47fca8	2015-12-09 16:58:31.782876
19	19	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1956\nname: EGFR\ndescription: EGFR is widely recognized for its importance in cancer. Amplification\n  and mutations have been shown to be driving events in many cancer types. Its role\n  in non-small cell lung cancer has spurred many research and drug development efforts.\n  Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably\n  gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance\n  to these drugs, particularly the variant T790M, which has been functionally characterized\n  as a resistance marker for both of these drugs. The later generation TKI's have\n  seen some success in treating these resistant cases, and targeted sequencing of\n  the EGFR locus has become a common practice in non-small cell lung cancer.\nofficial_name: epidermal growth factor receptor\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	4c4e3ba2-eff0-4291-aebe-649eccd2b5e9	2015-12-09 16:58:32.045062
190	136	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: R2328W\ndescription: Activating mutations in NOTCH1, including R2328W, have been shown to\n  be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	c792a8b9-cb31-4f21-86f1-c1d8d0f51ff9	2015-12-09 16:58:43.778638
20	20	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2064\nname: ERBB2\ndescription: ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast\n  cancer, and is treated in a separate manner than the other subtypes of breast cancer.\n  Apart from being amplified, ERBB2 activating mutations have been shown to have clinical\n  importance in HER2-negative breast cancer. These mutations have shown sensitivity\n  to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical\n  sequencing efforts in treating breast cancer.\nofficial_name: erb-b2 receptor tyrosine kinase 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	c4348b79-ccbf-44c5-8412-7ce42e4ad181	2015-12-09 16:58:32.309487
21	21	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2099\nname: ESR1\ndescription: ESR1 has been a focus in breast cancer for quite some time, but has also\n  shown significance in endometrial, ovarian and other cancer types. ER-positive breast\n  cancer that is resistant to hormone therapy has instigated clinical sequencing efforts\n  to shed light on the mechanisms of this resistance. A number of mutations in the\n  ligand binding domain of the protein have been implicated in hormone resistance\n  and anti-estrogen therapies. This has spurred efforts to develop therapeutics that\n  act to degrade the protein, rather than act as an antagonist. These agents are currently\n  in clinical trials and have seen some success, highlighting the importance of sequencing\n  efforts in treating breast cancer.\nofficial_name: estrogen receptor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	d8562aa8-9c83-4bbb-be6c-11f10bd58110	2015-12-09 16:58:32.592138
22	22	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2263\nname: FGFR2\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions that lead to pathway\n  actiation have been demonstrated to have oncogenic potential across multiple cancer\n  types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success\n  in treating over-active FGFR signalling, prompting use of diagnostic sequencing\n  targeting the FGFR genes, especially in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	c875e7b1-8afc-4e39-be25-84a4e25270dd	2015-12-09 16:58:32.867117
23	23	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2261\nname: FGFR3\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions that lead to pathway\n  actiation have been demonstrated to have oncogenic potential across multiple cancer\n  types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success\n  in treating over-active FGFR signalling, prompting use of diagnostic sequencing\n  targeting the FGFR genes, especially in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 3\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	48fe5acc-d7c7-4b0c-bfa3-2b7e48576e55	2015-12-09 16:58:33.126332
24	24	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2322\nname: FLT3\ndescription: FLT3 is an important cytokine receptor involved in normal hematopoiesis.\n  Mutations in this gene are common in acute myeloid leukemia (AML) and screening\n  for mutations in this gene has been recommended by the World Health Organization\n  in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML).\n  FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated\n  with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have\n  been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD\n  mutations are still up for debate.\nofficial_name: fms-related tyrosine kinase 3\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	801441c6-1b94-444b-86f5-9091a01d04a8	2015-12-09 16:58:33.354061
25	25	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2624\nname: GATA2\ndescription: GATA2 is a transcription factor involved in stem cell maintenance with\n  key roles in hematopoietic development. GATA2 mutations are associated with a variety\n  of inherited and acquired immune disorders including myelodysplastic syndrome and\n  acute myeloid leukemia. In addition to a role in hematopoiesis, the maintenance\n  GATA2 expression has been implicated as a requirement in KRAS-driven non-small cell\n  lung cancer. Preclinical models have indicated therapeutic benefit from targeting\n  GATA2-mediated pathways in the context of KRAS-driven NSCLC.\nofficial_name: GATA binding protein 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	9f8e554b-ba77-467a-9cb3-1d0a55ebddc2	2015-12-09 16:58:33.581715
26	26	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3417\nname: IDH1\ndescription: IDH1 mutations have been observed in a number of cancer types, including\n  sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in\n  the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial\n  (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described\n  as mutually exclusive in many of these cancer types. The most frequent mutations\n  involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic\n  enzyme activity. The implications of mutations in this gene vary greatly by cancer\n  type. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations\n  have been associated with worse outcome, shorter overall survival, and normal karyotype.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Unlike the association\n  with cytogenetically normal AML, in glioblastoma, IDH1 mutations have been associated\n  with specific cytogenetic abnormalities, 1p and 19q deletions.\nofficial_name: isocitrate dehydrogenase 1 (NADP+)\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	06766ee9-fa8b-44a0-808f-cc89aa94aa41	2015-12-09 16:58:33.837769
27	27	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3418\nname: IDH2\ndescription: IDH2 mutations have been observed in a number of cancer types, including\n  sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in\n  the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial\n  (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described\n  as mutually exclusive in many of these cancer types. The most frequent mutations\n  involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic\n  enzyme activity. Although IDH2 (R172) mutations are associated with poorer overall\n  prognosis in AML patients, its utility as a prognostic marker in MDS is still under\n  debate. Additionally, IDH2 (R140) has been associated with improved overall survival\n  in AML. IDH2 mutations have been associated with improved prognosis in gliomas.\nofficial_name: isocitrate dehydrogenase 2 (NADP+), mitochondrial\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	34ad1542-64b3-4fdb-b0c5-938d5b79ab30	2015-12-09 16:58:34.092507
28	28	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3717\nname: JAK2\ndescription: JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative\n  diseases and cancers. The mutation V617F is the most clinically relevant variant,\n  and is seen in around half of myeloproliferative disorders. The variant is a known\n  activating mutation, and activated JAK2 is sufficient to drive myeloproliferative\n  disorders in mouse models. V617F, while most recurrent, is not the only mechanism\n  by which JAK2 can be activated in patients. JAK2 is now one of the first diagnostic\n  markers tested upon diagnosis with a myeloproliferative disorder.\nofficial_name: Janus kinase 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	93447483-235c-43fa-9d65-0ab1e9d4cdb1	2015-12-09 16:58:34.324594
29	29	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3815\nname: KIT\ndescription: c-KIT activation has been shown to have oncogenic activity in gastrointestinal\n  stromal tumors (GISTs), melanomas, lung cancer, and other tumor types. The targeted\n  therapeutics nilotinib and sunitinib have shown efficacy in treating KIT overactive\n  patients, and are in late-stage trials in melanoma and GIST. KIT overactivity can\n  be the result of many genomic events from genomic amplification to overexpression\n  to missense mutations. Missense mutations have been shown to be key players in mediating\n  clinical response and acquired resistance in patients being treated with these targeted\n  therapeutics.\nofficial_name: v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	c559e888-3b60-4007-99a1-a1d995866909	2015-12-09 16:58:34.552343
30	30	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3845\nname: KRAS\ndescription: Mutations in the RAS family of proteins have frequently observed across\n  cancer types. The amino acid positions account for the overwhelming majority of\n  these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also\n  behave very differently when expressed in non-native tissue types, likely due to\n  differences in the C-terminal hyper-variable regions. Mis-regulation of isoform\n  expression has been shown to be a driving event in cancer, as well as missense mutations\n  at the three hotspots previously mentioned. While highly recurrent in cancer, targeted\n  these RAS mutants has also been very elusive, and has not yet become common practice\n  in the clinic.\nofficial_name: Kirsten rat sarcoma viral oncogene homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	52d6b83e-071e-4b19-b2a5-32c2d8c33004	2015-12-09 16:58:34.931109
31	31	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5604\nname: MAP2K1\ndescription: ''\nofficial_name: mitogen-activated protein kinase kinase 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	c9d2d6a6-4609-4566-bbdf-e9e24e8c599b	2015-12-09 16:58:35.187934
32	32	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4209\nname: MEF2D\ndescription: ''\nofficial_name: myocyte enhancer factor 2D\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	bb990f8c-6f71-4903-bbc4-bc48ed263848	2015-12-09 16:58:35.365119
33	33	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1436\nname: CSF1R\ndescription: ''\nofficial_name: colony stimulating factor 1 receptor\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	091caf9a-d467-42f9-bbc3-ec79366dbe7d	2015-12-09 16:58:35.587383
34	34	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4255\nname: MGMT\ndescription: ''\nofficial_name: O-6-methylguanine-DNA methyltransferase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	78340b94-126b-4618-8700-954f82d08083	2015-12-09 16:58:35.788196
35	35	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4869\nname: NPM1\ndescription: AML with mutated NPM1 is provisional entity in the WHO classification\n  of AML and is recommended to be tested for in patients with cytogenetically normal\n  AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they\n  have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently\n  W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations\n  in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing\n  the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia,\n  indicating it may require additional mutations to promote leukemic development.\nofficial_name: nucleophosmin (nucleolar phosphoprotein B23, numatrin)\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	8d0a7629-53fd-4cd6-8497-b6bdfecde553	2015-12-09 16:58:35.980044
36	36	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4893\nname: NRAS\ndescription: Mutations in the RAS family of proteins have frequently observed across\n  cancer types. The amino acid positions account for the overwhelming majority of\n  these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also\n  behave very differently when expressed in non-native tissue types, likely due to\n  differences in the C-terminal hyper-variable regions. Mis-regulation of isoform\n  expression has been shown to be a driving event in cancer, as well as missense mutations\n  at the three hotspots previously mentioned. While highly recurrent in cancer, targeted\n  these RAS mutants has also been very elusive, and has not yet become common practice\n  in the clinic.\nofficial_name: neuroblastoma RAS viral (v-ras) oncogene homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	642a13a6-3312-493f-bffc-267e159334ab	2015-12-09 16:58:36.208574
37	37	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5290\nname: PIK3CA\ndescription: PIK3CA is the most recurrently mutated gene in breast cancer, and has\n  been found to important in a number of cancer types. An integral part of the PI3K\n  pathway, PIK3CA has long been described as an oncogene, with two main hotspots for\n  activating mutations, the 542/545 region of the helical domain, and the 1047 region\n  of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways,\n  is the subject of an immense amount of research and development, and PI3K inhibition\n  has seen some limited success in recent clinical trials. While monotherapies seem\n  to be limited in their potential, there is a recent interest in pursuing PI3K inhibition\n  as part of a combination therapy regiment with inhibition partners including TKI's,\n  MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.\nofficial_name: phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b01445d4-8d2f-44f2-9adf-7cb4321a22a3	2015-12-09 16:58:36.45297
38	38	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5156\nname: PDGFRA\ndescription: ''\nofficial_name: platelet-derived growth factor receptor, alpha polypeptide\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	d1713097-2b3c-4913-9ec3-66e38cd35cc7	2015-12-09 16:58:36.674367
39	39	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5371\nname: PML\ndescription: The PML-RARA fusion is the result of a recurrent, balanced translocation\n  between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event\n  in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown\n  sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA\n  fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for\n  treating these patients, and early results seem promising.\nofficial_name: promyelocytic leukemia\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	3e949b01-a764-418b-b20e-065b06a7b872	2015-12-09 16:58:36.885332
666	82	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '15'\nstart:\n- \n- '66729162'\nstop:\n- \n- '66729162'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	8b8153fc-8e84-4be8-a9bf-33fcaa6a4205	2015-12-09 17:02:08.534649
40	40	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5914\nname: RARA\ndescription: The PML-RARA fusion is the result of a recurrent, balanced translocation\n  between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event\n  in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown\n  sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA\n  fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for\n  treating these patients, and early results seem promising.\nofficial_name: retinoic acid receptor, alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	62ee92a1-0e99-434c-ab9e-14bac6c196b7	2015-12-09 16:58:37.094568
41	41	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5728\nname: PTEN\ndescription: ''\nofficial_name: phosphatase and tensin homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	cb35e3c8-9e4f-4ae4-b4c9-f7c09e6ca2e8	2015-12-09 16:58:37.325274
42	42	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5979\nname: RET\ndescription: RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine\n  kinase receptor and are associated with thyroid cancers. RET point mutations are\n  the most common mutations identified in medullary thyroid cancer (MTC) with germline\n  and somatic mutations in RET associated with hereditary and sporadic forms, respectively.\n  The most common somatic form mutation is M918T (exon 16) and a variety of other\n  mutations effecting exons 10, 11 and 15 have been described. The prognostic significance\n  of these mutations have been hotly debated in the field, however, data suggests\n  that some RET mutation may confer drug resistence. No RET-specific agents are currently\n  clinically available but several promiscuous kinase inhibitors that target RET,\n  among others, have been approved for MTC treatment.\nofficial_name: ret proto-oncogene\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	e7a6385f-f796-4919-a2ba-f51bb1e54aac	2015-12-09 16:58:37.585549
43	43	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 861\nname: RUNX1\ndescription: ''\nofficial_name: runt-related transcription factor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	acf0d307-3435-4784-98fd-b9c51ae7ad77	2015-12-09 16:58:37.852159
44	44	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 23451\nname: SF3B1\ndescription: SF3B1 mutations have been described in several myeloid malignancies,\n  predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies\n  and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has\n  been identified as recurrently mutated in MDS and other malignanices. The mutations\n  affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally\n  deleterious with K700E described as a major hotspot mutation. MDS patients with\n  SF3B1 mutations have been reported to have better overall and event-free survival\n  than their wildtype counterparts. Additionally, these mutations are highly associated\n  with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with\n  ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring\n  sideroblasts).\nofficial_name: splicing factor 3b, subunit 1, 155kDa\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	30aa29d6-c8ce-4d6e-b53d-d369e2a4bf17	2015-12-09 16:58:38.080643
45	45	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7157\nname: TP53\ndescription: TP53 mutations are universal across cancer types. Loss of tumor suppressors\n  is most recognized by large deleterious events, such as frameshift mutations, or\n  premature stop codons. In TP53 however, many of the observed mutations in cancer\n  are found to be single nucleotide variants, or missense mutations. These variants\n  are also very broadly distributed throughout the gene, not localizing in any particular\n  hotspot. While a large proportion of cancer genomics research is focused on somatic\n  variants, TP53 is also of note in the germline. Germline TP53 mutations are the\n  hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been\n  found to have prognostic impact on patient outcomes.\nofficial_name: tumor protein p53\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	c2aa28a6-d61b-46c5-928c-f8e8acf0bd61	2015-12-09 16:58:38.322571
46	46	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7248\nname: TSC1\ndescription: ''\nofficial_name: tuberous sclerosis 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	514bf3fa-7e4c-43d6-8f58-06df18f9cbe5	2015-12-09 16:58:38.526968
47	47	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7249\nname: TSC2\ndescription: ''\nofficial_name: tuberous sclerosis 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	cd571709-6f16-444a-a04a-2aa2ebd5c30a	2015-12-09 16:58:38.730844
48	48	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7307\nname: U2AF1\ndescription: U2AF1 is one of several spliceosome complex genes frequently mutated\n  in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes\n  (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot\n  mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein.\n  These mutations have been associated with altered splicing patterns in vitro and\n  in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation\n  to secondary acute myeloid leukemia, however, the impact of these mutations on overall\n  survival has been an area of debate.\nofficial_name: U2 small nuclear RNA auxiliary factor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	0ef3033e-0ef4-41ea-8c33-4785dc685ed0	2015-12-09 16:58:38.946371
49	49	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7490\nname: WT1\ndescription: WT1 is a tumor suppressor gene associated with the development of Wilms'\n  Tumor, from which it was named. Mutations in exon 7 and 9 of WT1 have been recurrently\n  identified in acute myeloid leukemia and associated with poorer prognosis and chemotherapy\n  resistance.\nofficial_name: Wilms tumor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	77293f8a-9122-4f20-b024-de156ea67c00	2015-12-09 16:58:39.231123
50	50	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4851\nname: NOTCH1\ndescription: ''\nofficial_name: notch 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	aff9fc9a-5f7e-4a9d-b796-0d56b43fdb69	2015-12-09 16:58:39.446471
51	51	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4921\nname: DDR2\ndescription: ''\nofficial_name: discoidin domain receptor tyrosine kinase 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	2f7d2d68-f367-45f5-a62a-04b63d8c1c89	2015-12-09 16:58:39.68332
52	52	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4233\nname: MET\ndescription: ''\nofficial_name: MET proto-oncogene, receptor tyrosine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	dabc2cee-ae18-4b6d-9d61-3818d480f6ee	2015-12-09 16:58:39.898161
53	53	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2313\nname: FLI1\ndescription: ''\nofficial_name: Fli-1 proto-oncogene, ETS transcription factor\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	a4ddc3d8-2a7c-425e-8a50-1fb5ebc5a8b6	2015-12-09 16:58:40.098533
54	54	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2130\nname: EWSR1\ndescription: ''\nofficial_name: EWS RNA binding protein 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	1215a800-98e5-4d14-926a-a2bdf72cda07	2015-12-09 16:58:40.293006
342	130	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Increasing level of FLT3-ITD mutant correlated with reduced disease-free\n  and overall survival as well as increased relapse rate\ndisease_id: 2\nsource_id: 117\nvariant_id: 55\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b4adb827-3beb-4f9a-b6ca-5d06218df63a	2015-12-09 16:58:50.554845
55	1	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL\ndescription: The BCR-ABL fusion protein, commonly referred to as the philadelphia\n  chromosome, is the most well-studied fusion gene in cancer. It has widely been considered\n  the initiating event in chronic myelogenous leukemia (CML), but despite its ability\n  initiate disease in mice, its status an initiating mutation is in dispute. In what\n  is commonly used as the poster-child for targeted therapeutics, the development\n  and use of imatinib in the clinic has led to profound improvements in the prognosis\n  of the disease. However, imatinib resistance is still seen in patients with mutations\n  in the ABL kinase domain of the fusion, most notably the T315I variant. In patients\n  resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's\n  (dasatinib and nilotinib) have seen some success in delivering a tumor response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	fadbefbd-e190-4c08-8733-0b0d050a1ff5	2015-12-09 16:58:40.364607
56	2	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL T315I\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation T315I has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	d3a29268-59b8-4108-9d42-b8b1643ebaaf	2015-12-09 16:58:40.386311
57	3	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL E255K\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation E255K has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	dbc85dc4-7c05-4064-953b-a9ce30daf086	2015-12-09 16:58:40.409257
58	4	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 2\nname: E17K\ndescription: AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal,\n  lung, and ovarian cancer. It has been convincingly shown to be an activating mutation\n  resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation\n  decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and\n  other AKT1 mutations, are the subject of much research and development for therapeutics.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	943ed099-469c-4aa0-8837-ad686b035cb8	2015-12-09 16:58:40.43583
59	5	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK\ndescription: The EML4-ALK fusion has been seen in non-small cell lung cancer, and\n  appears to be an alternative mechanism for ALK activation. Cells with this fusion\n  have been shown to be sensitive to the ALK inhibitor crizotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	32d61574-f9ce-4b67-8f68-c6b4d7b51c0c	2015-12-09 16:58:40.457816
60	6	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK C1156Y\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	7ca44e6b-eb22-43ce-8911-7a8c62a3fa93	2015-12-09 16:58:40.480185
61	7	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK L1196M\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	59485fad-3062-4c02-9cee-0482f46cb076	2015-12-09 16:58:40.508141
62	8	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: F1174L\ndescription: ALK F1174L has been observed as recurrent in neuroblastoma, non-small\n  cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing\n  this mutation have shown resistance to low doses of criztonib. However, increased\n  dosage can overcome this resistance in cell lines studies. TAE684 has also proven\n  effective in both NSCLC and neuroblastoma F1174L containing cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	f741075c-a4b4-4cec-8ab0-89daebbd8211	2015-12-09 16:58:40.529829
63	9	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: R1275Q\ndescription: ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma,\n  non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma\n  cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This,\n  and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC\n  cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	908bbf36-7f01-4ba6-ba45-cd34bde7fb37	2015-12-09 16:58:40.551744
64	10	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 3\nname: S214C\ndescription: ARAF S214C has been found to be a recurrent oncogenic mutation in non-small\n  cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib\n  in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this\n  mutation, sorafenib also acheived near-complete clinical remission. This case has\n  brought more interest to the variant from a research and clinical perspective.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	b8d54adc-d22f-49e5-9697-b3238419d052	2015-12-09 16:58:40.575102
667	83	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '15'\nstart:\n- \n- '66727451'\nstop:\n- \n- '66727451'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- C\n	2	\N	\N	bc19a72d-334b-4ba6-999d-a0c5fb6e5262	2015-12-09 17:02:08.567051
65	11	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600D\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	6196cfc5-18cf-44ee-8e24-34d6d32b4598	2015-12-09 16:58:40.601406
66	12	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E\ndescription: BRAF V600E has been shown to be recurrent in many cancer types. It is\n  one of the most widely studied variants in cancer. This variant is correlated with\n  poor prognosis in certain cancer types, including colorectal cancer and papillary\n  thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective\n  in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib\n  has also shown to be effective when combined with the MEK inhibitor trametinib in\n  colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA\n  mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib,\n  and a number of combination therapies have been successful in treating V600E mutations.\n  While the drugs cetuximab and panitumumab have been largely shown to be ineffective\n  without supplementary treatment.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	3d264ff6-2ab1-4a1d-8c92-379aabbf8103	2015-12-09 16:58:40.6269
67	13	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E/V600M\ndescription: A case study of a single patient harboring both a V600E and a V600M mutation,\n  dabrafenib was shown to acheive clinical response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	d196419a-bf2c-4212-83c7-5ee220cc9a51	2015-12-09 16:58:40.655119
68	14	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E AMPLIFICATION\ndescription: Amplification of BRAF V600E has been shown to confer resistance to MEK\n  inhibitors. For more information on the V600 locus, see the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	a4b25c65-230f-4419-b3a1-014db924d359	2015-12-09 16:58:40.687625
69	15	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600M\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	25b82b75-d097-4338-a513-5535314a7db3	2015-12-09 16:58:40.712802
70	16	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600R\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	3ee560f2-bfe8-45de-9f69-6400c3438919	2015-12-09 16:58:40.734817
71	17	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600\ndescription: BRAF mutations of the valine 600 residue have been shown to be recurrent\n  in many cancer types. Of the V600 mutations, V600E is the most widely researched.\n  V600 mutations as a whole have been correlated to poorer prognosis in colorectal\n  and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity\n  to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual\n  mutation pages on the left sidebar.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	849b035d-cde9-4354-9ce8-44c9db005809	2015-12-09 16:58:40.755595
72	18	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: AMPLIFICATION\ndescription: CCND1 amplification has been implicated in poorer prognosis in non-small\n  cell lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	ad3643ac-e9b8-445b-ae62-9e594cfafb0d	2015-12-09 16:58:40.77909
73	19	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: EXPRESSION\ndescription: CCND1 expression, and its prognositc impact, is still in dispute. Three\n  experiments in non-small cell lung cancer have shown it to have no impact on survival,\n  but three additional studies have shown it results in poorer prognosis. There is\n  also some ambiguity in how the boundaries between expression and overexpression\n  are defined.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	68e3706a-8415-4b8c-ac48-a2959af7b282	2015-12-09 16:58:40.804363
74	20	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	168b9bd5-bc91-4155-a148-0beedb97611a	2015-12-09 16:58:40.828006
83	29	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: MUTATION\ndescription: CEBPA mutations are associated with cytogenetically normal acute myeloid\n  leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should\n  also be assessed in CN-AML patients as concurrent mutations may have prognostic\n  implications.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	a1d122ca-1183-4db2-87b8-032ee5163481	2015-12-09 16:58:41.041048
668	87	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '5'\nstart:\n- \n- '170837546'\nstop:\n- \n- '170837547'\nreference_bases:\n- \n- '0'\nvariant_bases:\n- \n- TCAG\n	2	\N	\N	7fbbf9c2-4391-43c3-aa67-0348bcbe4f52	2015-12-09 17:02:08.595352
75	21	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	d973e793-3ea6-4314-a560-2864bd8f7112	2015-12-09 16:58:40.854082
76	22	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: PROMOTER DEMETHYLATION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	4df84a79-3bee-487d-99b3-d8d6841a6252	2015-12-09 16:58:40.877488
77	23	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 10\nname: LOSS\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL),\n  a seemingly unique trend when compared to the amplifcations and overexpressions\n  of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic\n  palbociclib significantly increased the median survival of a Notch-driven model\n  of T-ALL.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	82d84f7a-6a41-4e33-a41a-b09a4dab5ba5	2015-12-09 16:58:40.900067
78	24	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 11\nname: OVEREXPRESSION\ndescription: Cyclin E, while currenly not as widely implicated as its counterpart,\n  cyclin D, has been implicated in various carcinomas, including breast, gastric,\n  stomach and colorectal. High levels of cyclin E, either by gene amplification or\n  overexpression, are correlated with later stage disease and have been shown to lead\n  to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer\n  specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex\n  are more radiosensitive than their lowly expressed counterparts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	69b8e638-79ff-4788-99e7-68825f4e154f	2015-12-09 16:58:40.921898
79	25	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: EXPRESSION\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	4798ecfa-7b22-4315-98a9-f61ea8eb4078	2015-12-09 16:58:40.943363
80	26	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 12\nname: EXPRESSION\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	8a32e057-aa12-4255-b3f9-027d0b3fa944	2015-12-09 16:58:40.965195
81	27	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 14\nname: PROMOTER HYPERMETHYLATION\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the CDKN2A\n  promoter region. While no targeted therapeutic has been engaged in clinical trials,\n  the prognostic impact has been studied by a number of meta-analyses. The prognostic\n  impact of promoter hypermethylation has been relatively ambiguous. Many studies\n  have shown significant p-values suggesting poorer prognostic outcomes for patients\n  with hypermethylation in colorectal, liver, and younger lung cancer patients. This\n  being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	05c5b3f0-8375-4875-a01a-c0997b28e72d	2015-12-09 16:58:40.992288
82	28	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: N-TERMINAL FRAME SHIFT\ndescription: CEBPA N-terminal frame shift mutations that result in a premature stop\n  codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML).\n  CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3\n  mutations should also be assessed in CN-AML patients as concurrent mutations may\n  have prognostic implications.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	aa268ca2-e11a-4db5-bca7-7223fbf7c5fa	2015-12-09 16:58:41.016308
669	93	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '1'\nstart:\n- \n- '115258744'\nstop:\n- \n- '115258744'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	a8c206a7-950d-43ae-99bc-e51434cd5153	2015-12-09 17:02:08.617864
84	30	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	f9cb8208-7bdd-4226-8148-37c9e9bc56be	2015-12-09 16:58:41.068431
85	31	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in the rare form of adolescent\n  liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion\n  was observed in 15/15 FL-HCC cases examined and functional studies found that the\n  fusion retained kinase activity. The presence of this fusion may be used as a diagnostic\n  marker for this rare tumor type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	643c55c3-042a-4dd3-bd0c-42a49cd148e6	2015-12-09 16:58:41.094599
86	32	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R882\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	4ebc4fe6-d8f6-4c1d-afa1-b1c31f9a7533	2015-12-09 16:58:41.120129
87	33	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: L858R\ndescription: EGFR L858R has long been recognized as a functionally significant mutation\n  in cancer, and is one of the most prevalent single mutations in lung cancer. Best\n  described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity\n  to first and second generation TKI's like gefitinib and neratinib. NSCLC patients\n  with this mutation treated with TKI's show increased overall and progression-free\n  survival, as compared to chemotherapy alone. Third generation TKI's are currently\n  in clinical trials that specifically focus on mutant forms of EGFR, a few of which\n  have shown efficacy in treating patients that failed to respond to earlier generation\n  TKI therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	35cfb4e1-ae74-4c6f-9ac1-0a8470df7545	2015-12-09 16:58:41.143379
88	34	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: T790M\ndescription: EGFR T790M was one of the very first mutations recognized to confer resistance\n  to targeted therapies in non-small cell lung cancer. While successful in amplified\n  EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib,\n  neratinib) in treating patients harboring this mutation before treatment is notably\n  lower. This lack of efficacy can likely be to blame for the poorer prognosis for\n  patients with this mutation as compared to patients with wildtype EGFR or other\n  types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired\n  resistance to TKI inhibition have been shown to harbor this mutation, implicating\n  it as a mechanism of acquired therapy resistence. The third generation TKI's are\n  being developed with this resistance problem in mind, and early stage experiments\n  have shown some efficacy of these drugs in previously resistant tumors with mutant\n  EGFR.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	28f3b783-53eb-4125-8e22-9f64e4ef0282	2015-12-09 16:58:41.171537
89	35	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: D769H\ndescription: ERBB2 D769H was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	867384b8-3fb2-4068-8cdc-e3bb35c41472	2015-12-09 16:58:41.197721
90	36	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: D769Y\ndescription: ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	df7a3809-4602-4c53-96a0-fb1a2a6343ee	2015-12-09 16:58:41.223198
91	37	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: DEL 755-759\ndescription: ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants\n  to be functionally classified (Bose et al. 2012). This mutation was shown to be\n  an activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	7a0f140b-373f-4556-8888-4737e223b45c	2015-12-09 16:58:41.246675
343	131	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: FLT3-ITD duplication length did not effect prognosis in patients with\n  an NPM1 mutation\ndisease_id: 2\nsource_id: 118\nvariant_id: 55\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	612f4a70-3da6-419a-b650-8170f4754d0d	2015-12-09 16:58:50.596152
92	38	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: G309A\ndescription: ERBB2 G309A was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	947a702d-75bd-4b64-8a8e-3ceaabf8bc3a	2015-12-09 16:58:41.269139
93	39	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: L755S\ndescription: ERBB2 L755S was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike\n  many of the other variants queried. This mutation was also shown to confer resistance\n  to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	6d0c2d38-62ca-4525-987d-19e3af559da5	2015-12-09 16:58:41.292686
94	40	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: L755W\ndescription: ERBB2 L755W was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	2d2ae983-1099-4cb1-8a2b-d8a804d88e92	2015-12-09 16:58:41.329937
95	41	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: P780INS\ndescription: ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally\n  classified (Bose et al. 2012). This mutation was shown to be an activating mutation\n  in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	5b08e983-70e0-4eb0-95f0-704bdc614710	2015-12-09 16:58:41.352987
96	42	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: R678Q\ndescription: ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	e29b2b54-7219-4b06-aee3-79ded2a0a8da	2015-12-09 16:58:41.376852
97	43	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: R896C\ndescription: ERBB2 R896C was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	0f9965f2-90bd-4475-aa24-15e5c17a73a0	2015-12-09 16:58:41.401076
98	44	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: V777L\ndescription: ERBB2 V777L was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	fade9837-1ab9-42cb-bd79-e5f7222472d5	2015-12-09 16:58:41.424762
99	45	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: V842I\ndescription: ERBB2 V842I was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	5a040d01-2c9d-4de2-ad23-cf74ca9432d4	2015-12-09 16:58:41.448021
117	63	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: R172K\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis\n  and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic\n  syndromes, studies did not find a prognostic association between this variant and\n  patient outcomes.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	702b29a7-06e9-490d-af79-64ad9afb8d21	2015-12-09 16:58:41.880476
100	46	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: L536Q\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of\n  these ligand binding domain mutations, and is commonly implicated in this hormone\n  resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant,\n  may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	c801fc54-72ad-485b-829a-ba22c9adbc86	2015-12-09 16:58:41.474458
101	47	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: N538G\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	f878ad82-f247-42c1-a16b-b30935dc7b11	2015-12-09 16:58:41.497406
102	48	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537C\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	d4d1bddc-d160-4d29-bc68-8cda169c033e	2015-12-09 16:58:41.519048
103	49	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537N\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	d86d366a-87ea-4d40-b280-5a57a72ff644	2015-12-09 16:58:41.540779
104	50	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537S\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	1991b0f8-c10a-4bf0-98ca-ed6e17ecc7ca	2015-12-09 16:58:41.566448
105	51	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: P235R\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	987ec762-697e-4892-8149-23de5ea20199	2015-12-09 16:58:41.593105
106	52	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: FGFR2-MGEA5\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. Additionally, tumor size reduction was\n  achieved by both ponatinib and pazopanib treatments administered separately in a\n  single patient with intrahepatic cholangiocarcinoma and this fusion. The authors\n  use these cases to highlight the need for enhanced clinical sequencing efforts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	1db23e6e-d1e4-4e4c-8257-c100ce385657	2015-12-09 16:58:41.617722
107	53	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: FGFR3-BAIAP2L1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	27ffafd7-02b7-4004-91c7-b10d70d8e3f5	2015-12-09 16:58:41.640706
108	54	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: FGFR3-TACC3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	07e9dccd-b699-4bb6-9088-9fea870e007f	2015-12-09 16:58:41.662674
670	95	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '1'\nstart:\n- \n- '115256528'\nstop:\n- \n- '115256529'\nreference_bases:\n- \n- TT\nvariant_bases:\n- \n- CA\n	2	\N	\N	92060fd9-1d06-4705-97a5-22623b25bcc4	2015-12-09 17:02:08.64329
109	55	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: ITD MUTATIONS\ndescription: FLT3-ITD (internal tandem duplications) frequently occur in patients\n  with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid\n  leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically\n  normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of\n  FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated\n  sequence resulting in an in-frame duplication event. The length of these duplications\n  can vary widely which may have prognostic consequences, but this has not been conclusively\n  determined. FLT3-ITD mutations overall have generally been associated with poor\n  prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the\n  prognosis associated with this variant.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	6e3eb057-714c-4838-943c-fe3f41e81a8c	2015-12-09 16:58:41.690032
110	56	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: TKD MUTATIONS\ndescription: FLT3 tyrosine kinase domain mutations (FLT3-TKD) are much less common\n  than FLT3-ITD (internal tandem duplication) mutations and may not confer the same\n  prognostic impact. Although the majority of mutations are point mutations effecting\n  D835 (most frequently D835Y), a small proportion involve an in-frame deletion of\n  I836. These mutations are within the activation loop of the second tyrosine kinase\n  domain of FLT3 and thought to result in constitutive activation of the receptor.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	f3287928-51dd-44b2-b24b-c08fac6a2604	2015-12-09 16:58:41.713472
111	57	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: EXPRESSION\ndescription: GATA2 misregulation has been observed in a number of hematologic malignancies,\n  as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using\n  the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung\n  cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	4717ce18-c96f-46d6-ad99-e5d22d696b20	2015-12-09 16:58:41.736691
112	58	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	63eff0eb-6482-4d8a-a0de-acf2e88d67d7	2015-12-09 16:58:41.764778
113	59	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132C\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	2b04b4d9-f72e-4fe9-9085-5997cc752c60	2015-12-09 16:58:41.786852
114	60	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132H\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	edf8da8e-577c-4279-8e6a-32c0a8fc3cf1	2015-12-09 16:58:41.808645
115	61	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132L\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	ee6428ef-1d27-45cf-8917-43d7bf327b71	2015-12-09 16:58:41.830944
116	62	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: R140Q/L\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival\n  as compared to their wild-type counterparts. In myelodysplastic syndromes, however,\n  no prognostic link was discovered between IDH2 mutation status and overall survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	0b516652-8871-40fd-bc28-f6719c6323c8	2015-12-09 16:58:41.858885
671	96	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '1'\nstart:\n- \n- '115256528'\nstop:\n- \n- '115256529'\nreference_bases:\n- \n- TT\nvariant_bases:\n- \n- CC\n	2	\N	\N	07f210a9-6a8c-4b6f-83f2-1e7554fdbc99	2015-12-09 17:02:08.673681
118	64	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: V617F\ndescription: JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases,\n  occuring in around half of all MPD's. While less associated with cancer, when it\n  is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The\n  V617F mutation is an activating mutation, resulting in increased kinase activity.\n  The mutation seems to be restricted to hematologic malignancies. Treatment of JAK\n  mutant diseases with ruxolitinib has seen some clinical success.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	959e0003-a804-4b5c-bf37-285a0f41559f	2015-12-09 16:58:41.903725
119	65	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: D816V\ndescription: KIT D816V is a mutation observed in acute myeloid leukemia (AML). This\n  variant has been linked to poorer prognosis and worse outcome in AML patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	ea6385a8-c5e8-4bc8-9ca0-8086d4ff113e	2015-12-09 16:58:41.927866
120	66	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 11 MUTATIONS\ndescription: c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are\n  very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis\n  than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity\n  to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients\n  treated with the drug in the first year.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	a18f4087-28c1-4d24-8320-7867284040cc	2015-12-09 16:58:41.9695
121	67	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: INTERNAL DUPLICATION\ndescription: c-KIT internal duplications have been observed in exon 11, within the\n  juxtamembrane domain. In a case study of an anal melanoma patient harboring this\n  event, imatinib confered marked response. Also, cells harboring exon 11 mutations\n  have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better\n  prognosis to patients treated with the drug in the first year.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	286e724d-de7a-4c25-a895-dbb51c6cc742	2015-12-09 16:58:41.993052
122	68	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 13 MUTATIONS\ndescription: c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations.\n  These mutations lie within the tyrosine kinase 1 domain, and are found primarily\n  in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations\n  show sensitivity to imatinib and sunitinib treatment. However, only imatinib has\n  seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant\n  cases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	5d73d3e7-3b65-4ca6-a1c0-82be5df08405	2015-12-09 16:58:42.017649
123	69	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 14 MUTATIONS\ndescription: c-KIT exon 14 mutations lie within the tyrosine kinase domain of the\n  protein. While relatively rare in primary gastrointestinal tumors, they are notably\n  more prevalent in refractory disease, suggesting a role in imatinib resistance.\n  Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	90b4113c-1b7e-461a-a6f9-11ab612cd425	2015-12-09 16:58:42.04314
124	70	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 17 MUTATIONS\ndescription: c-KIT exon 17 mutations lie within the TK2 domain, containing the activation\n  loop of the protein. In cell lines, mutations within this domain have been shown\n  to be sensitive to imatinib. However, in double KIT mutants in which the exon 17\n  mutation is a secondary mutation, cell lines have shown resistance to both imatinib\n  and sunitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	e19af33f-3820-4415-b533-20dcce505622	2015-12-09 16:58:42.067754
125	71	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 9 MUTATIONS\ndescription: c-KIT exon 9 mutations lie within the dimerization motif of the protein.\n  Relative to other KIT mutations, exon 9 mutations have been associated with better\n  overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and\n  in vivo.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	9dbbeeba-4994-4e76-9be5-9934d2fb8131	2015-12-09 16:58:42.090825
126	72	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: L576P\ndescription: KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain.\n  It is one of the most recurrent KIT mutations in melanoma, and both in vitro and\n  in vivo studies have shown sensitivity to imatinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	861f486d-858d-4920-8833-86686725f1e1	2015-12-09 16:58:42.112537
127	73	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: V654A\ndescription: KIT V654A is an exon 13 mutation that lies within the tyrosine kinase\n  1 domain of the protein. It has been shown to be an activating mutation by in vitro\n  studies. This mutation is associated with imatinib resistance in melanoma patients.\n  However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have\n  seen success in acheiving tumor response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	4b8d1fb4-cd53-4439-882f-1e105ab7df11	2015-12-09 16:58:42.133816
128	74	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: EXON 1 MUTATIONS\ndescription: A study by Lièvre et al in 2006 showed that colorectal cancer patients\n  with KRAS exon 1 mutations had low cetuximab response rates.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	70d09438-3c54-4731-b25c-d8bf840f9d47	2015-12-09 16:58:42.168422
672	98	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55152092'\nstop:\n- \n- '55152093'\nreference_bases:\n- \n- GA\nvariant_bases:\n- \n- AT\n	2	\N	\N	79c182dd-20e8-4d0f-965f-9c835714b5c9	2015-12-09 17:02:08.699861
129	75	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: EXON 2 MUTATIONS\ndescription: In a study by Pao et al in 2005, non-small cell lung cancer patients\n  harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors\n  gefinitib and erlotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	f5693844-23fb-45d6-881e-352ef83cdcb2	2015-12-09 16:58:42.19103
130	76	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	45b6dc1a-1b12-4957-8aef-5db7cecb1f92	2015-12-09 16:58:42.215552
131	77	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12/G13\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation\n  may result in a less responsive tumor when treated with first-generation TKI's like\n  gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	019f3e43-ae4d-4308-adf6-4d0829dc1834	2015-12-09 16:58:42.238764
132	78	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12C\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	2641e99e-e956-466b-9975-ac5e8b68efc8	2015-12-09 16:58:42.266909
133	79	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12D\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	c72a4226-1886-446e-8695-5331d8c3a35f	2015-12-09 16:58:42.291809
134	80	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G13\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	1580ff98-d662-40b6-982c-449c3f9d85f5	2015-12-09 16:58:42.318084
135	81	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G13D\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	66bc5124-9811-437d-9a75-61f11e564b71	2015-12-09 16:58:42.343627
136	82	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: P124S\ndescription: MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder\n  and colon cancer to a lesser degree. The P124S mutation has been shown to contribute\n  to AZD6244 resistance in melanoma cell lines, but considerably less so than its\n  Q56P counterpart.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	b13e7734-f9ef-492c-b081-ed097087afb8	2015-12-09 16:58:42.373159
137	83	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: Q56P\ndescription: MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This\n  mutation has been shown to confer considerable resistance to AZD6244 treatment of\n  melanoma cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	42ea5f24-e9ec-49a3-b3ef-bc606ebaeb1c	2015-12-09 16:58:42.396108
138	84	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 32\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	9fe47821-338a-40ce-b5a6-7c8390073bbb	2015-12-09 16:58:42.418825
189	135	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: D1643H\ndescription: Activating mutations in NOTCH1, including D1643H, have been shown to\n  be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	488db5eb-41ac-475a-b767-ec06a9ce1d23	2015-12-09 16:58:43.749909
139	85	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: PROMOTER METHYLATION\ndescription: MGMT promoter methylation has been observed to impact tumor progression\n  in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating\n  agent carmustine has shown efficacy. In patients lacking methylation, combining\n  carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more\n  experiments are required. Clinical trials have also shown selective sensitivity\n  of promoter methylation-positive patients to temozolomide, making a case for wider\n  methylation screening in GBM patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	ee14dbb2-0263-49d0-849a-5d20ab9fdec1	2015-12-09 16:58:42.442612
140	86	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 12 MUTATIONS\ndescription: NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	ef8b1c95-0039-45db-80a8-4ca596edce2a	2015-12-09 16:58:42.468946
141	87	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: W288FS\ndescription: NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most\n  common NPM1 mutation identified in acute myeloid leukemia. This mutation results\n  in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response\n  to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation,\n  it is generally grouped with other exon 12 mutations for patient analysis (see NPM1\n  Exon 12 variants for more information).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	58ae0e3f-e632-4f01-97ce-59a8d5e9a3ed	2015-12-09 16:58:42.492909
142	88	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 1 MUTATIONS\ndescription: NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown\n  to be correlated with poorer overall survival relative to wild-type NRAS in melanoma\n  patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	499db133-b0e0-471e-ba27-12ca0dc84347	2015-12-09 16:58:42.514756
143	89	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 2 MUTATIONS\ndescription: NRAS exon 2 mutations have been shown to be correlated with poorer overall\n  survival in melanoma patients and colorectal cancer patients, however no prognostic\n  impact was seen in acute myeloid leukemia patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	dff3d0dd-7108-4685-bff7-6dc671918dd8	2015-12-09 16:58:42.548081
144	90	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 3 MUTATIONS\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	f418d86a-2691-4951-8cb7-6a022ee6f1a6	2015-12-09 16:58:42.573991
145	91	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 4 MUTATIONS\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	39a95db2-3abd-4bdf-903e-ea3fd3acf2bf	2015-12-09 16:58:42.599963
146	92	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: G12\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	99bc3ead-b125-4f06-bfec-9b6363b4169c	2015-12-09 16:58:42.644249
147	93	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: G13D\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	a5c3bd34-3583-4cb2-8b58-c26d9cf331ad	2015-12-09 16:58:42.66831
148	94	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations\n  at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide.\n  However, in colorectal cancer patients, mutations at this locus have been shown\n  to confer resistance to cetuximab. The prognostic impact of mutations at this locus\n  is currently under study.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	129fc1d6-bdf5-48c1-93a7-05175a5fc50c	2015-12-09 16:58:42.693116
149	95	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61L\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	9f9b893f-551c-4386-a1e6-2c70889b57c7	2015-12-09 16:58:42.717797
150	96	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61R\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	270289f2-1be7-4b13-89e0-d40070d81a66	2015-12-09 16:58:42.73996
151	97	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: V536E\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	05a3e78b-b68a-4170-9899-48ae64c89971	2015-12-09 16:58:42.763328
152	98	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842I\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	001cb787-3d66-40ec-ac0d-f5460602f9ed	2015-12-09 16:58:42.784235
153	99	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842V\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	5577a3b6-12c5-403b-9652-1e9a8f67bab9	2015-12-09 16:58:42.808548
154	100	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842Y\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	dc31c99f-5a13-4d57-8d7d-5504f3ce5c05	2015-12-09 16:58:42.833363
155	101	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: DEL I843\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	24683b1f-355a-4757-a4b0-1854eeaf7178	2015-12-09 16:58:42.855475
156	102	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: DI842-843IM\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	6a5f7b22-70bd-4870-8c6e-f0d34752f2d1	2015-12-09 16:58:42.882909
157	103	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: E542K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	52662630-1dfc-40f3-a940-ac24bd397b63	2015-12-09 16:58:42.908102
158	104	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: E545K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	aa6b9d83-7346-4c28-8067-a0b0ca7800b5	2015-12-09 16:58:42.933161
159	105	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: EXON 20 MUTATIONS\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	c4c99246-628c-47ee-bb17-134bc3506391	2015-12-09 16:58:42.957081
160	106	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: EXON 9 MUTATIONS\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	29c4f061-bb37-4d30-9d39-13c69dbd902c	2015-12-09 16:58:42.989836
161	107	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: H1047R\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	07595e2f-8c2f-4dc9-91ec-488e1b5c6cbf	2015-12-09 16:58:43.013827
162	108	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 39\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and arsenic trioxide for treating these patients, and early results\n  seem promising.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	a4633978-2adc-447f-9945-98bce94d67bb	2015-12-09 16:58:43.04047
163	109	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in a rare form of adolescent\n  liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found\n  Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined,\n  and functional studies found that the fusion retained kinase activity. The presence\n  of this fusion may be used as a diagnostic marker for this rare tumor type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	e114cd8c-b2a2-46e9-8aad-dd0af46b3e4e	2015-12-09 16:58:43.065527
164	110	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 41\nname: R233*\ndescription: PTEN R233* has been shown to be a loss of function mutation, and PTEN\n  loss has been the subject of considerable research in breast cancer. PTEN loss may\n  sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data\n  to support that PTEN loss is both associated with poorer prognosis, and no change\n  in prognosis.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	3d469771-eb34-4445-9370-5a23256e61d9	2015-12-09 16:58:43.090755
165	111	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and Arsenic Trioxide for treating these patients, and early results\n  seem promising.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	3e7a8b29-4095-4d25-be7a-4326a4cf259b	2015-12-09 16:58:43.120855
166	112	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: C634W\ndescription: RET C639W has been implicated as an alternate mechanism of activating\n  RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting\n  agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the C639W mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	1a1cf459-0847-46a7-be5e-f98cf00faefc	2015-12-09 16:58:43.145051
167	113	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: M918T\ndescription: RET M819T is the most common somatically acquired mutation in medullary\n  thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents,\n  promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the M918T mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T\n  leads to more aggressive MTC with a poorer prognosis.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	404ee07e-8537-4008-9278-6145866fdf1b	2015-12-09 16:58:43.170161
168	114	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: K666N\ndescription: SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	13dd79b7-7695-4ed0-ab13-bfe1835447d7	2015-12-09 16:58:43.19471
169	115	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: K700E\ndescription: SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients. Additionally,\n  these mutations are the most common SF3B1 mutation observed in MDS and highly associated\n  with subtypes of MDS that are defined by ringed sideroblasts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	36b885c5-9f7b-41be-a926-8220ca459753	2015-12-09 16:58:43.219446
673	99	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55152093'\nstop:\n- \n- '55152093'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- T\n	2	\N	\N	882f649d-acfc-42b7-8e29-433aa54ce5a3	2015-12-09 17:02:08.720274
170	116	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R175H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R175H variant seems not only to result in loss of tumor-suppression, but also acts\n  as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell\n  lines harboring this mutant is also more responsive to treatment with doxorubicin\n  than its wild-type counterparts. While the prognostic impact of individual TP53\n  mutations is influenced by the cohort being studied, it has been shown that the\n  R175H mutation is correlated with worse overall survival than wild-type TP53, but\n  is not as detrimental as the R248W variant.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	8792a346-ffa1-4eec-9f0f-806672e2f2d6	2015-12-09 16:58:43.249862
171	117	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R248Q\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival. The R248Q mutation has also shown an increased invasive behavior in cell\n  lines. This is specific to the 248Q variant.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	987cdc92-8a3d-4631-af68-bf00729e3c95	2015-12-09 16:58:43.298012
172	118	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R248W\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	4ea5a903-7c73-439f-b5e4-41997afbe668	2015-12-09 16:58:43.325139
173	119	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R249T\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	a6e5fc66-f60a-4a68-af85-72929cfefa59	2015-12-09 16:58:43.34812
174	120	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R249W\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	0a196c29-2427-4636-8aab-a0b69a7ec905	2015-12-09 16:58:43.378047
175	121	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R273C\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	c88a5bf1-e3cf-4361-913e-132f6b34d031	2015-12-09 16:58:43.400914
176	122	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R273H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	23287ed8-d63f-4b67-b29b-f217d99bab42	2015-12-09 16:58:43.424683
177	123	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: V173G/A\ndescription: While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R249 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	f6b02b80-c4ff-4b84-9719-c889deb8239d	2015-12-09 16:58:43.450737
178	124	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: FRAMESHIFT TRUNCATION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	52f20015-db87-48ab-ba41-cabc7c5d4c2d	2015-12-09 16:58:43.478682
674	100	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55152092'\nstop:\n- \n- '55152092'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	c7fbd8b5-3b01-4bdf-99fe-b502ba6593e9	2015-12-09 17:02:08.745194
179	125	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	6f0e36f3-e2e1-4d2f-baa7-bf570a6165c7	2015-12-09 16:58:43.504835
180	126	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	8c658440-60b6-45b8-a383-76682536b33d	2015-12-09 16:58:43.528005
181	127	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 48\nname: Q157P/R\ndescription: U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is less common than the S34F mutation, occurs in the second zinc finger domain of\n  U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations\n  on overall survival in MDS has been debated, however, patients with U2AF1 mutations\n  were shown to be at an increased risk of transformation to secondary AML. The presence\n  of this mutation was not associated with a specific prognostic outcome in AML when\n  compared to U2AF1 wildtype patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	2926d91e-aabd-4e7a-876f-2df9d84803b0	2015-12-09 16:58:43.553648
182	128	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 48\nname: S34Y/F\ndescription: U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is the most commonly identified variant in MDS, occurs in the first zinc finger\n  domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1\n  mutations on overall survival in MDS has been debated, however, patients with U2AF1\n  mutations were shown to be at an increased risk of transformation to secondary AML.\n  The presence of this mutation was not associated with a specific prognostic outcome\n  in AML when compared to U2AF1 wildtype patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	47fac3dd-2b72-4a23-b900-6b127e1a8767	2015-12-09 16:58:43.577417
183	129	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 49\nname: EXON 7 MUTATIONS\ndescription: WT1 exon 7 mutations have been shown to be recurrent in acute myeloid\n  leukemia. Many sources have examined the prognostic impact of these, agreeing that\n  the mutant exon 7 cohort is correlated with worse overall survival and a number\n  of poor prognistic outcomes. This may be the result of an overall poor response\n  to chemotherapy from WT1 mutant tumors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	05a40410-6dbd-459f-9496-a6935023d231	2015-12-09 16:58:43.600773
184	130	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 49\nname: EXON 9 MUTATIONS\ndescription: WT1 exon 9 mutations have been shown to be recurrent in acute myeloid\n  leukemia, although at a less frequent rate than their exon 7 counterparts. Many\n  sources have examined the prognostic impact of these, agreeing that the mutant exon\n  9 cohort is correlated with worse overall survival and a number of poor prognistic\n  outcomes. This may be the result of an overall poor response to chemotherapy from\n  WT1 mutant tumors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	9bbd517e-1317-4228-81f2-95f0aff9ea3c	2015-12-09 16:58:43.628604
185	131	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: LOSS-OF-FUNCTION\ndescription: BRCA1 loss of function mutations have been shown to increase risk of\n  breast and ovarian cancer in those carrying the allele in their germline. Treating\n  BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant\n  response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	7f3f1344-a34b-4cac-b396-a6fb8fc3c1a4	2015-12-09 16:58:43.651498
186	132	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: LOSS-OF-FUNCTION\ndescription: BRCA2 loss of function mutations have been shown to increase risk of\n  breast and ovarian cancer in those carrying the allele in their germline. Treating\n  BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant\n  response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	0e4511f9-2068-4686-a0ce-905076f5a558	2015-12-09 16:58:43.674593
187	133	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: EXON 19 DELETION\ndescription: Deletions within exon 19 of EGFR are most common in lung cancer. These\n  deletions, in non-small cell lung cancer, have been shown to be sensitive to the\n  EGFR tyrosine kinase inhibitors gefitinib and erlotinib. There is also data to suggest\n  that this event is a good prognostic marker in lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	9f6fb802-c748-466f-8ff7-c795dbf1a81f	2015-12-09 16:58:43.700396
188	134	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: G719S\ndescription: While not as recurrent as the L858R mutation, EGFR G719S has also been\n  shown to be an activating mutation. In lung cancer cell lines, it also confers sensitivity\n  to the tyrosine kinase inhibitors gefitinib and erlotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	94fa165c-877f-44ee-8220-d136a24eec54	2015-12-09 16:58:43.727181
675	103	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '3'\nstart:\n- \n- '178936082'\nstop:\n- \n- '178936082'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	470a9371-0e35-40de-9742-2d9bebaca367	2015-12-09 17:02:08.768039
191	137	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: V2444FS\ndescription: Activating mutations in NOTCH1, including a frameshift insertion at V2444,\n  have been shown to be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	957eaaac-d72e-46ac-85d2-cdb053b84d28	2015-12-09 16:58:43.808526
192	138	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: S2275FS\ndescription: Activating mutations in NOTCH1, including a frameshift insertion at S2275,\n  have been shown to be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	91e6a6ad-5c57-4669-ae09-e0f59c0cd01f	2015-12-09 16:58:43.835583
193	139	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: L63V\ndescription: Activating mutations in DDR2, including L63V, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	44700116-2634-4251-87ab-63968ba3af40	2015-12-09 16:58:43.86266
194	140	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: L239R\ndescription: Activating mutations in DDR2, including L239R, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	674940d3-0dc8-4ff8-acdc-318d88fe6c91	2015-12-09 16:58:43.887478
195	141	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: G253C\ndescription: Activating mutations in DDR2, including G253C, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	0c147c5d-a576-4e67-9dbd-1d3ecb7e0f87	2015-12-09 16:58:43.91687
196	142	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: G505S\ndescription: Activating mutations in DDR2, including G505S, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	a9bcd1d9-c523-4991-be55-a3414572a9a3	2015-12-09 16:58:43.942163
197	143	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: I638F\ndescription: DDR2 I638F has been shown to be a loss of function mutation, but also\n  confers sensitivity to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	0e001ffd-79ad-4f5e-ac0d-e5a5592360aa	2015-12-09 16:58:43.986957
198	144	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: G774V\ndescription: Activating mutations in DDR2, including G774V, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	d8a22a24-4daf-44fd-b0c0-21ba1b0f2dc4	2015-12-09 16:58:44.011148
199	145	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: S768R\ndescription: Activating mutations in DDR2, including S768R, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	0abb67d8-1f28-4e30-9452-dcfc6116ddac	2015-12-09 16:58:44.038247
200	146	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 52\nname: AMPLIFICATION\ndescription: MET amplification, like EGFR T790M, has been shown to be capable of driving\n  acquired resistance to dacomitinib in patients with lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	c285340b-13ac-4038-b78f-b69b162951f9	2015-12-09 16:58:44.06423
201	147	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12V\ndescription: KRAS G12V, like EGFR T790M, has been shown to be capable of driving acquired\n  resistance to tyrosine kinase inhibitors in lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	007bf437-f56f-4c98-b327-1c2778c27022	2015-12-09 16:58:44.086865
202	148	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12A\ndescription: KRAS G12A, like EGFR T790M, has been shown to be capable of driving acquired\n  resistance to tyrosine kinase inhibitors in lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	bba25f7f-c5cc-400c-9c17-ba808becca27	2015-12-09 16:58:44.111775
203	149	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 54\nname: EWS-FLI1\ndescription: EWS-FLI1 fusions have been found to be extremely common in Ewing's sarcomas.\n  The presence of the fusion is also thought to be a positive prognostic marker of\n  survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	b3e24363-a47b-49e8-bc4b-d0967fdbdb0f	2015-12-09 16:58:44.133489
204	150	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 53\nname: EWS-FLI1\ndescription: EWS-FLI1 fusions have been found to be extremely common in Ewing's sarcomas.\n  The presence of the fusion is also thought to be a positive prognostic marker of\n  survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	1c666c50-eff9-498e-9c6a-af62cb9a22ae	2015-12-09 16:58:44.155137
205	151	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 53\nname: TYPE 1 EWS-FLI1\ndescription: The most common EWS-FLI1 fusion (type 1) links EWS exon 7 to FLI1 exon\n  6. This fusion product is found to be associated with better outcome than other\n  EWS-FLI1 fusions in Ewing's Sarcoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \nrepresentative_transcript2: \nensembl_version: \n	1	\N	\N	2dbe7850-0a3c-45a0-bb36-780deb515a4d	2015-12-09 16:58:44.184375
676	104	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '3'\nstart:\n- \n- '178936091'\nstop:\n- \n- '178936091'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	d023beef-cda0-4078-89ed-64c570ea5636	2015-12-09 17:02:08.787106
206	1	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In one patient with lung adenocarcinoma who had a S214C mutation, the\n  use of sorafenib has led to more than 5 years of survival and near remission.\ndisease_id: 1\nsource_id: 51\nvariant_id: 10\nrating: 2\nstatus: accepted\nvariant_hgvs: X:47426121-47426121 (C->G)\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c084037f-129a-4b12-b747-da34c0fc9403	2015-12-09 16:58:44.288244
207	2	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutations occur most often in de novo AML patients with intermediate\n  cytogenic risk.\ndisease_id: 2\nsource_id: 52\nvariant_id: 32\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9b3a0b38-bc14-407a-9692-84e346fc8b7e	2015-12-09 16:58:44.376502
208	3	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young AML patients (<60 years old) with DNMT3A mutations (60% of which\n  were R882) were older, had higher white blood cell counts and had higher platelet\n  counts than patients wild-type for DNMT3A\ndisease_id: 2\nsource_id: 53\nvariant_id: 32\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3148125c-e4a4-4234-844a-c8242d79b4f3	2015-12-09 16:58:44.422538
209	4	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Daunorubicin treatment resulted in similar overall survival and disease-free\n  survival in de novo AML patients with DNMT3A R882 mutation compared to those who\n  do not harbor this mutation.\ndisease_id: 2\nsource_id: 52\nvariant_id: 32\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5a954778-8bb1-47c4-a3b7-e09151e7f6b2	2015-12-09 16:58:44.468288
210	5	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Idarubicin increases overall and disease free survival in de novo AML\n  patients with DNMT3A R882 mutations compared to those who do not harbor this mutation.\ndisease_id: 2\nsource_id: 52\nvariant_id: 32\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	861f4fb6-a8ad-4192-8e2a-3b15d1aa964b	2015-12-09 16:58:44.509385
211	6	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no difference in the complete remission rate of de novo AML\n  patients with DNMT3A mutation compared to those who are wild type for DNMT3A.\ndisease_id: 2\nsource_id: 52\nvariant_id: 32\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3299480f-0f65-4780-8917-f7099cd7ac67	2015-12-09 16:58:44.543803
212	7	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: De novo AML patients with DNMT3A D882 mutation showed worse survival\n  (event-free and overall) outcome than those without DNMT3A mutation.\ndisease_id: 2\nsource_id: 54\nvariant_id: 32\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	47e1b159-4066-46c0-afe1-589122a1dbb1	2015-12-09 16:58:44.586069
213	8	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL\n  or CLL).\ndisease_id: 3\nsource_id: 55\nvariant_id: 64\nrating: 4\nstatus: accepted\nvariant_hgvs: 9:5073770-5073770 (G->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ac84e071-bee8-47c9-a77c-d0a5f24b9d1c	2015-12-09 16:58:44.656953
214	9	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical\n  CML).\ndisease_id: 4\nsource_id: 55\nvariant_id: 64\nrating: 4\nstatus: accepted\nvariant_hgvs: 9:5073770-5073770 (G->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	54c44a54-6c5b-4d48-b3f0-5896c8a451d7	2015-12-09 16:58:44.694617
215	10	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: JAK2 V617F is associated with myeloid neoplasms (AML, MDS, CMML/aCML)\ndisease_id: 4\nsource_id: 55\nvariant_id: 64\nrating: 4\nstatus: accepted\nvariant_hgvs: 9:5073770-5073770 (G->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a6dc137a-b6d8-4d10-80f0-4a3135401ff2	2015-12-09 16:58:44.734075
216	11	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads\n  to reduction in the percentage of cells harboring JAK2 V617F.\ndisease_id: 5\nsource_id: 56\nvariant_id: 64\nrating: 4\nstatus: accepted\nvariant_hgvs: 9:5073770-5073770 (G->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	701b9e34-9bc4-4a1b-9ef5-9ef8f5f2174c	2015-12-09 16:58:44.778764
217	12	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In acute myloid leukemia patients, D816 mutation is associated with earlier\n  relapse and poorer prognosis than wild-type KIT\ndisease_id: 6\nsource_id: 57\nvariant_id: 65\nrating: 4\nstatus: accepted\nvariant_hgvs: 4:55599321-55599321 (A->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4ceb6549-aa1b-468b-89e2-49978ea4f112	2015-12-09 16:58:44.825433
218	1	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: KIT Exon 17\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	e8056388-c1b5-494d-b0b3-925d543cee4f	2015-12-09 16:58:44.869655
219	13	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition\n  by about 5 fold.\ndisease_id: 7\nsource_id: 58\nvariant_id: 82\nrating: 3\nstatus: accepted\nvariant_hgvs: 15:66729162-66729162 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b5691e7f-4243-4c61-a497-56516033965b	2015-12-09 16:58:44.917534
220	14	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by\n  100 fold.\ndisease_id: 7\nsource_id: 59\nvariant_id: 83\nrating: 3\nstatus: accepted\nvariant_hgvs: 15:66727451-66727451 (A->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ae5c089b-8c6a-4635-b353-0876fda6db50	2015-12-09 16:58:44.97467
221	15	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\ndisease_id: 8\nsource_id: 60\nvariant_id: 89\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	70e4ccee-7ebf-46a6-ba93-0a9c3dcb40ef	2015-12-09 16:58:45.029974
222	16	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\ndisease_id: 8\nsource_id: 60\nvariant_id: 90\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9d85ab56-f720-4fd6-93e6-08b5c1fb4f94	2015-12-09 16:58:45.068686
223	17	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\ndisease_id: 8\nsource_id: 60\nvariant_id: 91\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	54fff505-af05-42f0-928a-c0bfb0f4f17c	2015-12-09 16:58:45.101538
224	18	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Likely due to increased reliance of mutant NRAS on HSP90 for stabilization,\n  inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic\n  malignant melanoma with an NRAS G13D mutation\ndisease_id: 7\nsource_id: 61\nvariant_id: 93\nrating: 2\nstatus: accepted\nvariant_hgvs: 1:115258744-115258744 (C->T)\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	92e410b3-d001-4907-90c9-38e7018c9428	2015-12-09 16:58:45.149912
225	19	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma associated with NRAS Q61 mutation was more often associated\n  with those at the extremity than those at the trunk\ndisease_id: 7\nsource_id: 62\nvariant_id: 94\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	14f30e08-6859-4418-92b9-3d4f3ba5813e	2015-12-09 16:58:45.198371
226	20	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.\ndisease_id: 7\nsource_id: 63\nvariant_id: 94\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d80deead-9e3b-4a88-bfb8-e07122bfdbd3	2015-12-09 16:58:45.237658
227	21	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide\n  resulted in disease free survival of 14 months\ndisease_id: 7\nsource_id: 64\nvariant_id: 95\nrating: 2\nstatus: accepted\nvariant_hgvs: 1:115256528-115256529 (TT->CA)\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8facc3a1-c6b5-4ee6-8e9e-6773b8473ae3	2015-12-09 16:58:45.279803
228	22	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a melanoma patient with Q61R mutation, treatment with temozolomide\n  resulted in overall survival of 16 months\ndisease_id: 7\nsource_id: 64\nvariant_id: 96\nrating: 2\nstatus: accepted\nvariant_hgvs: 1:115256528-115256529 (TT->CC)\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8e579191-f24a-4bcb-bbda-476976ee2ff3	2015-12-09 16:58:45.333892
229	23	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: GIST tumors harboring PDGFRA D842V mutation are more likely to be benign\n  than malignant.\ndisease_id: 9\nsource_id: 65\nvariant_id: 99\nrating: 3\nstatus: accepted\nvariant_hgvs: 4:55152093-55152093 (A->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5db784e7-58b1-45d3-92a3-f5c2e35939c7	2015-12-09 16:58:45.374501
230	2	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Imatinib Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	3e6bc088-87ad-4427-84f3-384c31c82af3	2015-12-09 16:58:45.394999
231	24	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: GIST cancer with D842V mutation is resistant to imatinib.\ndisease_id: 9\nsource_id: 66\nvariant_id: 99\nrating: 4\nstatus: accepted\nvariant_hgvs: 4:55152093-55152093 (A->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ca1a14d6-b947-4ce6-9e83-e4aa986a9656	2015-12-09 16:58:45.436691
232	25	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib,\n  double mutation of V561D and D842V mutants are resistant to imatinib.\ndisease_id: 9\nsource_id: 67\nvariant_id: 99\nrating: 4\nstatus: accepted\nvariant_hgvs: 4:55152093-55152093 (A->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	33cbd7b5-cbf3-4ec9-a283-7238f87e3be8	2015-12-09 16:58:45.488365
233	26	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation\n  were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK\n  without this mutation.\ndisease_id: 10\nsource_id: 68\nvariant_id: 8\nrating: 3\nstatus: accepted\nvariant_hgvs: 2:29443695-29443695 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	51450c92-e76c-49e6-9a9a-73715b2122c4	2015-12-09 16:58:45.554176
234	27	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation\n  were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK\n  without this mutation.\ndisease_id: 1\nsource_id: 68\nvariant_id: 8\nrating: 3\nstatus: accepted\nvariant_hgvs: 2:29443695-29443695 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cb440a11-0565-4974-8c87-ab670a7e81cf	2015-12-09 16:58:45.603737
235	3	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Crizotinib Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	2c9d1d1b-81e6-41c6-87dc-5d2f8b2eb1a5	2015-12-09 16:58:45.62554
236	28	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant\n  in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.\ndisease_id: 11\nsource_id: 69\nvariant_id: 8\nrating: 3\nstatus: accepted\nvariant_hgvs: 2:29443695-29443695 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	37605af9-0a57-4513-82e7-a1b8f7c8e331	2015-12-09 16:58:45.664224
237	29	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: High levels of crizotinib can overcome drug resistance of Ba/F3 cells\n  expressing the EML4-ALK fusion containing the F1174L mutation.\ndisease_id: 11\nsource_id: 70\nvariant_id: 8\nrating: 3\nstatus: accepted\nvariant_hgvs: 2:29443695-29443695 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2eff7995-24b8-4b5b-b3ee-9bf36029561e	2015-12-09 16:58:45.715436
677	107	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '3'\nstart:\n- \n- '178952085'\nstop:\n- \n- '178952085'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- G\n	2	\N	\N	6a7a4e92-799a-4d02-950b-a2c43f8c3ea9	2015-12-09 17:02:08.808568
238	30	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however,\n  cells with the R1275Q mutation were inhibited at a higher IC50 than those expressing\n  F1174L.\ndisease_id: 11\nsource_id: 71\nvariant_id: 9\nrating: 3\nstatus: accepted\nvariant_hgvs: 2:29432664-29432664 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	029e60c1-9fe1-4c9a-ba5b-0e3d827e2bd6	2015-12-09 16:58:45.760154
239	31	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The SMS-KCNR cell line harboring the R1275Q mutation was resistant to\n  TAE684.\ndisease_id: 11\nsource_id: 71\nvariant_id: 9\nrating: 3\nstatus: accepted\nvariant_hgvs: 2:29432664-29432664 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	783b7e7b-5277-4d1c-aba2-715eee4c46e5	2015-12-09 16:58:45.798833
240	32	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar\n  colony formation of AALE cells expressing ARAF S214C in vitro\ndisease_id: 1\nsource_id: 51\nvariant_id: 10\nrating: 3\nstatus: accepted\nvariant_hgvs: X:47426121-47426121 (C->G)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bbd5dc45-cd52-4819-b166-3dfd8c99c5da	2015-12-09 16:58:45.837056
241	33	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar\n  colony formation of AALE cells expressing ARAF S214C in vitro\ndisease_id: 1\nsource_id: 51\nvariant_id: 10\nrating: 3\nstatus: accepted\nvariant_hgvs: X:47426121-47426121 (C->G)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	057bc0fd-0862-41a5-b4a6-8c8d7eed7279	2015-12-09 16:58:45.874371
242	34	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF mutations are associated with melanoma arising in non-chronic sun\n  damaged skin and with superficial spreading melanoma\ndisease_id: 7\nsource_id: 72\nvariant_id: 17\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4351e3bd-bbad-4be7-a2d2-ac1c7a3b78d3	2015-12-09 16:58:45.927483
243	35	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a large cohort of young AML patients (18-60 years old), DNMT3A R882\n  mutations were associated with reduced relapse free survival in the entire cohort\n  as well as the subset of patients with cytogenetically normal AML\ndisease_id: 2\nsource_id: 73\nvariant_id: 32\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c4639568-2898-4070-bb65-82426786a8b0	2015-12-09 16:58:45.990635
244	36	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutations were associated with older age, higher white blood\n  cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of\n  cytogenetically normal AML patients\ndisease_id: 2\nsource_id: 74\nvariant_id: 32\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3dc824da-c8d8-4c8a-9e3d-87320dd88136	2015-12-09 16:58:46.037833
245	37	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a large cohort of AML patients (mean = 48 years), DNMT3A mutation\n  (64.5% of which were R882) had no prognostic value on overall, relapse free and\n  event free survival.\ndisease_id: 2\nsource_id: 73\nvariant_id: 32\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2af0d94f-5611-4d87-8be4-31e74edb889d	2015-12-09 16:58:46.076411
246	38	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a large cohort of cytogenetically normal AML patients (18-60 years\n  old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value\n  for overall, relapse free and event free survival\ndisease_id: 2\nsource_id: 73\nvariant_id: 32\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2aef1921-1a6a-422d-a070-3992f597f6ab	2015-12-09 16:58:46.113697
247	39	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutation status (60% of\n  which were R882) was not predictive of overall and relapse free survival in patients\n  with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\ndisease_id: 2\nsource_id: 53\nvariant_id: 32\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9ea2d3d6-d332-4e11-b0e6-e5b2d9fc7912	2015-12-09 16:58:46.1512
248	40	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with DNMT3A mutations (59% of which were R882) showed worse\n  survival (event-free and overall) outcome than those without DNMT3A mutation.\ndisease_id: 2\nsource_id: 54\nvariant_id: 32\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fa6c922f-2468-45f4-8145-027cb508b6c2	2015-12-09 16:58:46.188599
249	41	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutation was associated with reduced relapse free and overall\n  survival in ELN-unfavorable, cytogenetically normal AMLs\ndisease_id: 2\nsource_id: 73\nvariant_id: 32\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	dd3fbd83-4ad2-4f6d-b582-01936d33cffb	2015-12-09 16:58:46.231386
250	42	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations\n  (including R882) were associated with worse overall survival compared to those without\n  DNMT3A mutation.\ndisease_id: 2\nsource_id: 54\nvariant_id: 32\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	80caa38f-d0a8-4fd0-830b-330d3d095e12	2015-12-09 16:58:46.267721
251	43	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In cytogenetically normal AML patients, DNMT3A R882 mutations are associated\n  with lower overall and disease free survival as compared to patients with wild type\n  DNMT3A.\ndisease_id: 2\nsource_id: 74\nvariant_id: 32\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	53e46bfc-eb5f-4dbc-b852-efd6c0dfec6d	2015-12-09 16:58:46.304605
678	110	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '10'\nstart:\n- \n- '89717672'\nstop:\n- \n- '89717672'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	3be6e3ec-3d31-423d-b56d-d556c78032d2	2015-12-09 17:02:08.830127
252	44	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In older cytogenetically normal AML patients (>59 years), DNMT3A R882\n  mutation is prognostic for shorter disease free survival and overall survival compared\n  to patients without the mutation.\ndisease_id: 2\nsource_id: 75\nvariant_id: 32\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	27378574-f705-4594-9f74-aa7e49e38ca4	2015-12-09 16:58:46.348002
253	45	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutations were associated\n  with significantly reduced overall survival and relapse free survival in patients\n  wildtype for NPM1 and FLT3\ndisease_id: 2\nsource_id: 53\nvariant_id: 32\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5d7f80fa-20ed-4400-8815-a479007d64bc	2015-12-09 16:58:46.37981
254	46	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations\n  other than R882 are prognostic for shorter disease free survival and overall survival\n  compared to patients without the mutation.\ndisease_id: 2\nsource_id: 75\nvariant_id: 32\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	840e318a-1176-479b-96d4-7a9c9a60b8e0	2015-12-09 16:58:46.417545
255	47	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutation status was not\n  predictive of overall and relapse free survival in patients with NPM1 mutations\n  and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\ndisease_id: 2\nsource_id: 53\nvariant_id: 55\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0923d9f9-ddb6-4124-8199-0932f1347e80	2015-12-09 16:58:46.454961
256	48	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations\n  were associated with worse overall survival compared to those without DNMT3A mutation.\ndisease_id: 2\nsource_id: 54\nvariant_id: 55\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ac76edaa-5ec7-4d2e-a97a-3e8e166b8cac	2015-12-09 16:58:46.49009
257	49	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no significant association between wildtype KIT or KIT mutations\n  in exon 9 or 11 in survival among GIST patients.\ndisease_id: 12\nsource_id: 76\nvariant_id: 66\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8bf46aa2-c109-4bd5-8c13-eaf554b0ed07	2015-12-09 16:58:46.533176
258	4	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: KIT Exon 11\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	6fde93a3-952b-432b-8806-4acdb7375d45	2015-12-09 16:58:46.555492
259	50	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with KIT mutation but not KIT amplification showed\n  response to imatinib treatment in a small cohort of patients.\ndisease_id: 7\nsource_id: 77\nvariant_id: 66\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4cc47ae8-5fb3-4584-9684-68f779623182	2015-12-09 16:58:46.597666
260	51	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with KIT mutation but not KIT amplification showed\n  response to imatinib treatment in a small cohort of patients.\ndisease_id: 7\nsource_id: 77\nvariant_id: 68\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d5372304-0743-47cb-8bc2-e85a29309d0c	2015-12-09 16:58:46.659044
261	52	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: GIST patients with exon 14 KIT mutations had reduced overall survival\n  compared to patients wiltype for KIT\ndisease_id: 12\nsource_id: 76\nvariant_id: 69\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9a86e5f1-0bc3-4df5-8af8-ece3828df1f7	2015-12-09 16:58:46.694492
262	53	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with KIT mutation but not KIT amplification showed\n  response to imatinib treatment in a small cohort of patients.\ndisease_id: 7\nsource_id: 77\nvariant_id: 70\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8a730bfa-aa73-4c8c-a4c7-b0b899d77415	2015-12-09 16:58:46.72702
263	54	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no significant association between wildtype KIT or KIT mutations\n  in exon 9 or 11 in survival among GIST patients.\ndisease_id: 12\nsource_id: 76\nvariant_id: 71\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3a2f2ecb-5240-4686-a3e7-1b553a56374a	2015-12-09 16:58:46.767115
264	55	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The melanoma cell line WM3211, which harbors the L576P mutation, is not\n  sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the\n  L576P mutation alters the conformation of KIT and reduces the binding affinity of\n  imatinib.\ndisease_id: 7\nsource_id: 78\nvariant_id: 72\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	e178748b-7fb0-4904-8e13-2893ddfb7487	2015-12-09 16:58:46.808876
265	56	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive\n  to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation\n  of KIT but did not alter the binding affinity of dasatinib.\ndisease_id: 7\nsource_id: 78\nvariant_id: 72\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	51a3b95a-2ff8-4686-8200-8f49b35c1e08	2015-12-09 16:58:46.865889
266	57	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC, Exon 2 KRAS mutations were associated with resistance to the\n  EGFR kinase inhibitors gefitinib and erlotinib\ndisease_id: 1\nsource_id: 79\nvariant_id: 75\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	b75d3ebd-9bf1-4592-90ef-3142dcb9439c	2015-12-09 16:58:46.915587
679	112	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '10'\nstart:\n- \n- '43609950'\nstop:\n- \n- '43609950'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- G\n	2	\N	\N	d8ea7bec-fd8c-4f52-a019-833676fb7c0f	2015-12-09 17:02:08.86022
267	58	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutation status was not\n  predictive of overall and relapse free survival in patients with NPM1 mutations\n  and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\ndisease_id: 2\nsource_id: 53\nvariant_id: 86\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6de540f4-7292-4d2a-b59c-d1170f9b2fbb	2015-12-09 16:58:46.956359
268	59	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with stage IV melanoma, NRAS mutation was associated with\n  reduced median survival compared to patients with wildtype NRAS\ndisease_id: 7\nsource_id: 80\nvariant_id: 88\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a88f26c8-251a-464e-aaaf-67241b7af0e6	2015-12-09 16:58:47.003131
269	60	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of NRAS mutation in AML patients does not impact diease\n  prognosis (resistant disease, disease-free survival, complete remission rate, relapse\n  rate, induction death)\ndisease_id: 2\nsource_id: 81\nvariant_id: 89\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d09dcda2-883c-4147-9f9b-5786df40d726	2015-12-09 16:58:47.052635
270	61	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with stage IV melanoma, NRAS mutation was associated with\n  reduced median survival compared to patients with wildtype NRAS\ndisease_id: 7\nsource_id: 80\nvariant_id: 89\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c8b3185a-cf0b-4086-92b7-202994fdfdd5	2015-12-09 16:58:47.097869
271	62	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NRAS mutation status does not impact overall, event-free or disease-free\n  survival in patients with AML\ndisease_id: 2\nsource_id: 82\nvariant_id: 92\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	27c77d72-f21a-4db8-b7e7-52f5223bd7ba	2015-12-09 16:58:47.147693
272	63	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring NRAS\n  mutation (primarily Q61) have a significantly lower response rate to cetuximab than\n  patients wildtype for NRAS.\ndisease_id: 8\nsource_id: 83\nvariant_id: 94\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	53739c4f-0bea-4049-af41-a7896d4e15f8	2015-12-09 16:58:47.187725
273	64	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\ndisease_id: 9\nsource_id: 84\nvariant_id: 98\nrating: 4\nstatus: accepted\nvariant_hgvs: 4:55152092-55152093 (GA->AT)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ad265114-3847-474d-bbea-f68edcc7d4ad	2015-12-09 16:58:47.228378
274	65	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\ndisease_id: 9\nsource_id: 84\nvariant_id: 99\nrating: 4\nstatus: accepted\nvariant_hgvs: 4:55152093-55152093 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f306891a-27a2-4bf1-91a9-76b335823320	2015-12-09 16:58:47.274158
275	66	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\ndisease_id: 9\nsource_id: 84\nvariant_id: 100\nrating: 4\nstatus: accepted\nvariant_hgvs: 4:55152092-55152092 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cac2e18f-ff4c-4d3f-ae03-4b4e5ae4f76e	2015-12-09 16:58:47.333903
276	67	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib\n  resistance, crenolanib was significantly more potent at inhibiting kinase activity\n  than imatinib.\ndisease_id: 9\nsource_id: 84\nvariant_id: 101\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ab42d8d1-00b1-4915-9024-057b0b712e5a	2015-12-09 16:58:47.377997
277	68	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib\n  resistance, crenolanib was significantly more potent at inhibiting kinase activity\n  than imatinib.\ndisease_id: 9\nsource_id: 84\nvariant_id: 102\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c0ad9202-2ddc-41de-a696-0a3a87a7823b	2015-12-09 16:58:47.423906
278	69	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A single patient with BRAF V600E/V600M bi-allelic mutation responded\n  to the V600E drug dabrafenib.\ndisease_id: 7\nsource_id: 85\nvariant_id: 13\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d70d3d5b-783c-4dcb-901b-51e2716d1831	2015-12-09 16:58:47.473238
279	5	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Other V600's\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	09b9c4a1-b1ba-458b-9a3a-8c59352c646c	2015-12-09 16:58:47.502233
280	70	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Medullary thyroid cancer cells with RET C634W mutation are insensitive\n  to motesanib, compared to wild-type RET.\ndisease_id: 13\nsource_id: 86\nvariant_id: 112\nrating: 3\nstatus: accepted\nvariant_hgvs: 10:43609950-43609950 (C->G)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f4299379-cdb9-4192-87e4-0b1b19eaa0cd	2015-12-09 16:58:47.543287
281	6	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Motesanib Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	76c30466-3885-4495-a322-210f98f0c241	2015-12-09 16:58:47.567425
282	71	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with medullary carcinoma, the presence of RET M918T mutation\n  is associated with increased probability of lymph node metastases.\ndisease_id: 13\nsource_id: 87\nvariant_id: 113\nrating: 5\nstatus: accepted\nvariant_hgvs: 10:43617416-43617416 (T->C)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	eab57311-6aeb-4ff5-b839-b50f55e5887a	2015-12-09 16:58:47.616859
283	72	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Compared to those who harbor a wild type RET, patients with RET M918T\n  mutation develop larger and more aggressive medullary thyroid cancer.\ndisease_id: 13\nsource_id: 88\nvariant_id: 113\nrating: 4\nstatus: accepted\nvariant_hgvs: 10:43617416-43617416 (T->C)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	66a60e6d-3a36-483b-9921-0a52ce8c57b8	2015-12-09 16:58:47.659309
284	73	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Medullary thyroid cancer cells with RET M918T mutation are insensitive\n  to motesanib, compared to wild-type RET.\ndisease_id: 13\nsource_id: 86\nvariant_id: 113\nrating: 3\nstatus: accepted\nvariant_hgvs: 10:43617416-43617416 (T->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	30e1620b-4627-41fa-aabf-a5fd96bd68b8	2015-12-09 16:58:47.691278
285	74	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led\n  to strong repression of tyroid cancer cell growth.\ndisease_id: 13\nsource_id: 89\nvariant_id: 113\nrating: 3\nstatus: accepted\nvariant_hgvs: 10:43617416-43617416 (T->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9bd70021-f9dd-494f-89aa-7aae0baa1156	2015-12-09 16:58:47.749484
286	75	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations\n  were associated with poor progression free survival regardless of treatment (panitumumab\n  with best supportive care or best supportive care alone)\ndisease_id: 8\nsource_id: 90\nvariant_id: 17\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	54138f7c-3ffb-4b34-aff4-bb8dc3450210	2015-12-09 16:58:47.793169
287	76	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with BRAF V600E/K mutations had longer progression-free\n  survival, increased tumor regression,  and increased duration of response to combined\n  dabrafenib and trametinib treatment compared to dabrafenib alone\ndisease_id: 7\nsource_id: 91\nvariant_id: 17\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	7d556de9-1dbd-45d5-b63c-324ca930ccd1	2015-12-09 16:58:47.836139
288	77	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with other BRAF V600 mutations also respond well to the V600E\n  drug dabrafenib.\ndisease_id: 7\nsource_id: 92\nvariant_id: 11\nrating: 5\nstatus: accepted\nvariant_hgvs: 7:140453135-140453136 (CA->AT)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a53c1448-c59b-4ef0-8842-a7d026b80d9e	2015-12-09 16:58:47.886959
289	78	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: V600E is associated with adverse pathological features of colorectal\n  cancer. This can be concluded as a marker of poor prognosis.\ndisease_id: 8\nsource_id: 93\nvariant_id: 12\nrating: 5\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b4e03230-5a46-4f80-9084-efc930fd2151	2015-12-09 16:58:47.950713
290	79	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF V600E is shown to be associated with the tall-cell variant of papillary\n  thyroid cancer\ndisease_id: 14\nsource_id: 94\nvariant_id: 12\nrating: 3\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	279d32ae-fe38-4de5-984c-5c9f8cd874d8	2015-12-09 16:58:48.001405
291	80	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\ndisease_id: 14\nsource_id: 95\nvariant_id: 12\nrating: 5\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	86fc611b-98bf-482b-b796-516adf89231b	2015-12-09 16:58:48.056352
292	81	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\ndisease_id: 14\nsource_id: 96\nvariant_id: 12\nrating: 5\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6812b2ff-8415-466e-8fef-77722fd4d9b9	2015-12-09 16:58:48.09868
293	82	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\ndisease_id: 7\nsource_id: 97\nvariant_id: 12\nrating: 2\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	ad5e4925-1be2-44df-acc5-e15239907fdd	2015-12-09 16:58:48.139526
294	83	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\ndisease_id: 7\nsource_id: 98\nvariant_id: 12\nrating: 5\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	94c1ee24-bd4f-4ab9-809a-9fdf206c1d21	2015-12-09 16:58:48.207513
295	84	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Unlike other studies that suggest a poorer outcome, BRAF mutation in\n  this study was not correlated with poorer prognosis in papillary thyroid cancer.\ndisease_id: 14\nsource_id: 99\nvariant_id: 12\nrating: 5\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f06e629b-4c5f-4f8d-960f-e1f732dfbe32	2015-12-09 16:58:48.269075
296	85	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF V600E is correlated with shorter disease-free and overall Survival\n  in a Spanish cohort of melanoma patients.\ndisease_id: 7\nsource_id: 100\nvariant_id: 12\nrating: 3\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	71fb1885-5892-4f14-9365-50cee2e6e1c0	2015-12-09 16:58:48.31824
297	86	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF mutation correlated with poor prognosis in papillary thyroid cancer\n  in both older (>65 yo) and younger (<65 yo) cohorts.\ndisease_id: 14\nsource_id: 94\nvariant_id: 12\nrating: 3\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	381e1274-39cf-48cb-b510-fe187453dcd9	2015-12-09 16:58:48.355523
298	87	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF V600E is correlated with poor prognosis in papillary thyroid cancer\n  in a study of 187 patients with PTC and other thyroid diseases.\ndisease_id: 14\nsource_id: 101\nvariant_id: 12\nrating: 3\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1908cc50-6b88-4c1e-9a72-495e4098b6ff	2015-12-09 16:58:48.398295
299	88	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: V600E is correlated with disease recurrence in both age cohorts (>65\n  and <65 yo).\ndisease_id: 14\nsource_id: 94\nvariant_id: 12\nrating: 3\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	eb45dbfe-e7c9-454a-a6a7-0b493881e191	2015-12-09 16:58:48.432377
300	89	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation\n  of RAS-GTP but does not show resistance to MEK inhibitors.\ndisease_id: 7\nsource_id: 102\nvariant_id: 12\nrating: 3\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	35592ede-3507-4fcd-8e5d-cdc9e07a077f	2015-12-09 16:58:48.472485
301	90	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation\n  unless BRAF inhibitor such as Sorafenib is introduced.\ndisease_id: 15\nsource_id: 103\nvariant_id: 12\nrating: 3\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0690c5cf-3181-42c6-9550-e852e084c61f	2015-12-09 16:58:48.512971
302	91	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation\n  of RAS-GTP and resistance to RAF inhibitors.\ndisease_id: 7\nsource_id: 102\nvariant_id: 12\nrating: 3\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d5374db4-f9b5-4dd3-9ef1-0b359df95fc2	2015-12-09 16:58:48.544752
303	92	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely\n  injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited\n  tumor growth and reduced cellular proliferation\ndisease_id: 15\nsource_id: 104\nvariant_id: 12\nrating: 3\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	9656d5f7-ba0d-4a89-bad4-fbe6c1463729	2015-12-09 16:58:48.58889
304	93	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\ndisease_id: 15\nsource_id: 105\nvariant_id: 12\nrating: 2\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	baccc360-9c50-47a7-87ad-00a2e738de07	2015-12-09 16:58:48.661397
305	94	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\ndisease_id: 8\nsource_id: 106\nvariant_id: 12\nrating: 2\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	1c211dc4-487e-45c4-ba3b-07aa5f83710f	2015-12-09 16:58:48.70834
306	95	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cell lines expressing the BRAF V600E mutation responded better to vemurafenib\n  treatment than cells wildtype for BRAF as measured by reduced cellular proliferation\n  and inhibition of MET and ERK phosphorylation\ndisease_id: 8\nsource_id: 106\nvariant_id: 12\nrating: 2\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e6912b1d-c43b-4f66-ad75-8bbc6392301c	2015-12-09 16:58:48.752841
307	96	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\ndisease_id: 1\nsource_id: 107\nvariant_id: 12\nrating: 2\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	15f4a93a-802a-42d1-86d3-78ca4d99bb21	2015-12-09 16:58:48.796234
308	97	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: COLO201 and COLO206F cells harboring BRAF V600E mutations were cloned\n  to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown\n  to be amplification of the BRAF V600E gene.\ndisease_id: 8\nsource_id: 108\nvariant_id: 14\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	696bac34-1572-4697-8dd2-1d9627244cf6	2015-12-09 16:58:48.839239
309	98	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with other BRAF V600 mutations also respond well to the V600E\n  drug dabrafenib.\ndisease_id: 7\nsource_id: 92\nvariant_id: 15\nrating: 5\nstatus: accepted\nvariant_hgvs: 7:140453137-140453137 (C->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a793a3bd-11fc-4219-8234-6d2b2ec87408	2015-12-09 16:58:48.881228
310	99	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with other BRAF V600 mutations also respond well to the V600E\n  drug dabrafenib.\ndisease_id: 7\nsource_id: 92\nvariant_id: 16\nrating: 5\nstatus: accepted\nvariant_hgvs: 7:140453136-140453137 (AC->CT)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9fc99152-21df-452a-be80-013900c25b2f	2015-12-09 16:58:48.923601
311	100	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\ndisease_id: 8\nsource_id: 109\nvariant_id: 75\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7f5497b4-e7eb-41b3-8a01-bd424a0244ea	2015-12-09 16:58:48.971222
680	113	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '10'\nstart:\n- \n- '43617416'\nstop:\n- \n- '43617416'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	055560fc-97a7-462b-9185-f86180646f86	2015-12-09 17:02:08.891802
312	101	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation status was not predictive of response to regorafenib\n  treatment in patients that had received standard therapy and progressed within 3\n  months of their last treatment\ndisease_id: 8\nsource_id: 109\nvariant_id: 103\nrating: 4\nstatus: accepted\nvariant_hgvs: 3:178936082-178936082 (G->A)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f17128c2-07ea-4e76-bfed-c0f81a725834	2015-12-09 16:58:49.026735
313	102	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation status was not predictive of response to regorafenib\n  treatment in patients that had received standard therapy and progressed within 3\n  months of their last treatment\ndisease_id: 8\nsource_id: 109\nvariant_id: 104\nrating: 4\nstatus: accepted\nvariant_hgvs: 3:178936091-178936091 (G->A)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cad186a4-6cbe-4a20-b9d2-bc4e92a6d080	2015-12-09 16:58:49.080888
314	103	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation status was not predictive of response to regorafenib\n  treatment in patients that had received standard therapy and progressed within 3\n  months of their last treatment\ndisease_id: 8\nsource_id: 109\nvariant_id: 107\nrating: 4\nstatus: accepted\nvariant_hgvs: 3:178952085-178952085 (A->G)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	75aa2892-89aa-4d76-9741-1076034d6229	2015-12-09 16:58:49.146643
315	104	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CH5424802 treatment resulted in significant tumor regression in xenograft\n  models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M\n  mutation\ndisease_id: 1\nsource_id: 69\nvariant_id: 7\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1d575b1e-9982-4095-bca4-dd68a336349e	2015-12-09 16:58:49.226501
316	105	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation.\ndisease_id: 11\nsource_id: 110\nvariant_id: 8\nrating: 3\nstatus: accepted\nvariant_hgvs: 2:29443695-29443695 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4d24d39b-d36e-45ee-8d9c-4bf973a6ba8c	2015-12-09 16:58:49.309633
317	106	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing\n  ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition\ndisease_id: 11\nsource_id: 71\nvariant_id: 8\nrating: 4\nstatus: accepted\nvariant_hgvs: 2:29443695-29443695 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	23802f40-f7eb-40e7-abb4-36ac99f4ebf6	2015-12-09 16:58:49.382009
318	107	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation\ndisease_id: 11\nsource_id: 110\nvariant_id: 9\nrating: 3\nstatus: accepted\nvariant_hgvs: 2:29432664-29432664 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	772167f7-039b-4749-af89-8427dcb4ae82	2015-12-09 16:58:49.44096
319	108	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring BRAF\n  mutations had lower response and disease control rates as well as shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype BRAF\ndisease_id: 8\nsource_id: 83\nvariant_id: 12\nrating: 3\nstatus: accepted\nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6f87cf12-572d-405a-b34a-b457d5ca6d9d	2015-12-09 16:58:49.519266
320	109	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEBPA mutation status had no impact on ATRA treatment response in older\n  (>60) patients with AML\ndisease_id: 2\nsource_id: 111\nvariant_id: 29\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8640c16e-daf7-4fb1-8df1-96c07ec1bd32	2015-12-09 16:58:49.5805
321	110	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEBPA mutation was associated with improved overall survival in older\n  (>60) patients with AML\ndisease_id: 2\nsource_id: 111\nvariant_id: 29\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	73685d88-3b5a-48ec-ab69-826a09ab6acd	2015-12-09 16:58:49.633039
322	111	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEBPA mutation was significantly associated with complete remission\ndisease_id: 2\nsource_id: 112\nvariant_id: 29\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ac916a47-8a97-4e74-be19-1f7f25b0358c	2015-12-09 16:58:49.695605
323	112	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\ndisease_id: 2\nsource_id: 113\nvariant_id: 28\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2dd1a4bd-b5a9-4ffa-b331-038c0b36c760	2015-12-09 16:58:49.762934
324	113	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Overall survival was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA mutations\ndisease_id: 2\nsource_id: 113\nvariant_id: 28\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	73a2079b-493d-47e2-9761-3a58c7bb7a6b	2015-12-09 16:58:49.80848
325	114	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Remission duration was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\ndisease_id: 2\nsource_id: 113\nvariant_id: 28\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c64793ec-4b18-443e-9c3c-453eaf8d6970	2015-12-09 16:58:49.849179
326	115	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Therapy-related AML was less common in patients with DNMT3A mutations\n  (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of\n  younger (18-60) AML patients\ndisease_id: 2\nsource_id: 73\nvariant_id: 32\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	55abb851-533f-4092-88b5-caca9955bbdf	2015-12-09 16:58:49.887976
327	116	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (59% of which were R882) were associated with an intermediate\n  risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes\ndisease_id: 2\nsource_id: 54\nvariant_id: 32\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9923a7eb-182b-4661-9a62-37458042d0d0	2015-12-09 16:58:49.934169
328	117	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (59% of which were R882) were associated with intermediate\n  risk cytogenetics (including normal karyotype)\ndisease_id: 2\nsource_id: 54\nvariant_id: 32\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b0c181d0-23ab-4c6e-a1fb-f891bb7589c8	2015-12-09 16:58:49.976087
329	118	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (64.5% R882) were associated with older age, higher\n  white blood cell count and cytogenetically normal AML in a large cohort of younger\n  (18-60) AML patients\ndisease_id: 2\nsource_id: 73\nvariant_id: 32\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	180aa5d0-dc5c-4efe-8a53-f74c7c131df0	2015-12-09 16:58:50.014667
330	119	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutations were associated with cytogenetically normal AML\n  in a large cohort of younger (18-60) AML patients\ndisease_id: 2\nsource_id: 73\nvariant_id: 32\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d2604e21-75d5-46b6-ad70-1af3df64a067	2015-12-09 16:58:50.054602
331	120	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rates did not differ between patients with wildtype\n  or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML\ndisease_id: 2\nsource_id: 75\nvariant_id: 32\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	608889c6-7648-42f8-9d70-92709e88a0db	2015-12-09 16:58:50.096899
332	121	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations were associated with achievement of complete remission\n  in a large cohort of younger (18-60) AML patients\ndisease_id: 2\nsource_id: 73\nvariant_id: 32\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	db97862b-9df1-40a7-8adb-99416876b8f6	2015-12-09 16:58:50.136009
333	122	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rate was not different between young AML patients\n  (<60 years old) with or without DNMT3A mutations (60% of which were R882)\ndisease_id: 2\nsource_id: 53\nvariant_id: 32\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8bda72ee-6131-429d-a804-b9d4979d1e0a	2015-12-09 16:58:50.173693
334	123	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (62% of which were R882) were associated with reduced\n  disease-free survival in patients with cytogenetically normal AML.\ndisease_id: 2\nsource_id: 75\nvariant_id: 32\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	63956dc4-9573-4ed9-9bae-2fd1dc823d18	2015-12-09 16:58:50.208709
335	124	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young AML patients (<60 years old) with DNMT3A mutation have shorter\n  overall survival and relapse-free survival than patients with wildtype DNMT3A\ndisease_id: 2\nsource_id: 53\nvariant_id: 32\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2aefb73d-d180-4978-b5e7-c017fe7786a8	2015-12-09 16:58:50.264942
336	125	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter\n  overall survival and relapse-free survival than patients with wildtype DNMT3A\ndisease_id: 2\nsource_id: 53\nvariant_id: 32\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ff42efff-b4f0-48b1-b6f4-a656173887a7	2015-12-09 16:58:50.301646
337	126	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Following identification of this fusion in a patient, this fusion was\n  expressed in HEK 293T cells leading to oligomerization in the absence of ligand,\n  induced morphologic changes in the cell and increased cellular proliferation consistent\n  with FGFR activation. Mouse xenografts using the bladder cell line SW780 which harbors\n  an FGFR3-BAIAP2L1 fusion showed reduced tumor burden when treated with the FGFR\n  inhibitor PD173074.\ndisease_id: 16\nsource_id: 114\nvariant_id: 53\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2dbfbf23-145f-4ef8-9571-fb3bdf698d05	2015-12-09 16:58:50.351786
338	7	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: FGFR fusions\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	80d10a79-abf3-4a1f-ab67-3f7c50d7ca46	2015-12-09 16:58:50.37773
339	127	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Meta-analysis of studies involving cytogentically normal younger (<60)\n  patients showed reduced overall and relapse-free survival for patients with FLT3-ITD\ndisease_id: 2\nsource_id: 115\nvariant_id: 55\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	110efc73-25e3-4084-899a-1412e4029186	2015-12-09 16:58:50.419839
340	128	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: ATRA treatment did not effect overall survival in patients <60 years\n  old with FLT3-ITD mutation\ndisease_id: 2\nsource_id: 116\nvariant_id: 55\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	67c3c531-44f1-49a8-8b92-5c8c53526ae4	2015-12-09 16:58:50.468234
341	129	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: FLT3-ITD mutation without NPM1 was associated with increased relapse\n  risk and reduced overall survival in young adult AML patients (median age 43)\ndisease_id: 2\nsource_id: 117\nvariant_id: 55\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	40c9c741-a2cf-4462-b759-1fecd49b39ba	2015-12-09 16:58:50.51742
681	114	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '198267359'\nstop:\n- \n- '198267359'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- A\n	2	\N	\N	bf320504-0ae0-4b61-8131-4e43fe2b057f	2015-12-09 17:02:08.924552
344	132	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Size of the FLT3-ITD mutant duplication had no impact on overall survival\n  or relapse rate\ndisease_id: 2\nsource_id: 117\nvariant_id: 55\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	aa901b86-32f6-46c7-8e4d-b633c1b25c40	2015-12-09 16:58:50.634256
345	133	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced\n  event-free and overall survival in old (>59) and younger patients (<60) with an\n  NPM1 mutation\ndisease_id: 2\nsource_id: 118\nvariant_id: 55\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ae76bcf1-46e0-4e42-aed9-7f746f17c766	2015-12-09 16:58:50.673515
346	134	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival\n  compared to patients with wildtype FLT3\ndisease_id: 2\nsource_id: 118\nvariant_id: 55\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2509e28c-c39c-4c6c-b461-cd9e653971fd	2015-12-09 16:58:50.710711
347	135	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable,\n  a higher ratio was associated with reduced event-free survival in patients with\n  an NPM1 mutation\ndisease_id: 2\nsource_id: 118\nvariant_id: 55\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a6b421e8-7191-43f7-808b-34ff1ad9bc85	2015-12-09 16:58:50.747309
348	136	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to daunorubicin than cells with FLT3-ITD alone.\ndisease_id: 2\nsource_id: 119\nvariant_id: 55\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	edd4b01a-4708-43eb-9ecf-4095df5e147a	2015-12-09 16:58:50.799422
349	137	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells\n  with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\ndisease_id: 2\nsource_id: 119\nvariant_id: 55\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	22ad25db-ae23-49d1-bdee-9d802c227e69	2015-12-09 16:58:50.837099
350	138	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Event-free survival was significantly improved in patients with both\n  NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone\n  2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis\n  3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal\n  karyotype patients were analyzed\ndisease_id: 2\nsource_id: 120\nvariant_id: 56\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b5f62639-b399-46fa-b73b-d142f95e6571	2015-12-09 16:58:50.887167
351	139	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to daunorubicin than cells with FLT3-ITD alone.\ndisease_id: 2\nsource_id: 119\nvariant_id: 56\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d6f91b9b-ae18-45d2-90e8-a413bb0daf9f	2015-12-09 16:58:50.938981
352	140	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells\n  with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\ndisease_id: 2\nsource_id: 119\nvariant_id: 56\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c8232e6f-3b12-4d43-8e0c-b298e3479b4c	2015-12-09 16:58:50.981776
353	141	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutation status of IDH1 did not change event-free survival of patients\n  with an NPM1 mutation\ndisease_id: 2\nsource_id: 121\nvariant_id: 58\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	97b22559-f40e-4e78-8d89-95e4d700f133	2015-12-09 16:58:51.030506
354	142	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring KRAS\n  mutations (primarily G12/G13) had lower response and disease control rates and shorter\n  progression free and overall survival following cetuximab plus chemotherapy than\n  those with wildtype KRAS\ndisease_id: 8\nsource_id: 83\nvariant_id: 77\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d3526919-58a6-4601-8f31-c6684050e366	2015-12-09 16:58:51.06801
355	143	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations were significantly associated with lack of response to\n  cetuximab in patients with advanced colorectal cancer\ndisease_id: 8\nsource_id: 122\nvariant_id: 77\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7a0e0f61-97a3-4b37-828d-ce171f16e677	2015-12-09 16:58:51.119314
356	144	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations were significantly associated with reduced progression-free\n  and overall survival in patients with advanced colorectal cancer\ndisease_id: 8\nsource_id: 122\nvariant_id: 77\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	99124b96-55ee-4b3f-b86e-d8f16dff1bbb	2015-12-09 16:58:51.156962
357	145	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment\n  in isogenic SW48 cells and in a mouse xenograft model\ndisease_id: 15\nsource_id: 123\nvariant_id: 81\nrating: 4\nstatus: accepted\nvariant_hgvs: 12:25398281-25398281 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1cd51081-457c-42a5-b2db-5283057cb49d	2015-12-09 16:58:51.198581
682	115	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '198266834'\nstop:\n- \n- '198266834'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	8c552fdb-70e5-4e56-8f3e-896a4bcb7551	2015-12-09 17:02:08.962986
358	146	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment\n  in isogenic SW48 cells and in a mouse xenograft model\ndisease_id: 15\nsource_id: 123\nvariant_id: 81\nrating: 4\nstatus: accepted\nvariant_hgvs: 12:25398281-25398281 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d18779be-5cd6-4a56-9b26-3f2a92208ee1	2015-12-09 16:58:51.231398
359	147	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MGMT promoter methylation was associated with increased overall and disease-free\n  survival as well as tumor regression in patients with gliomas.\ndisease_id: 17\nsource_id: 124\nvariant_id: 85\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	276c2c74-7ce5-4035-9013-d4f6143fc2a7	2015-12-09 16:58:51.274525
360	148	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML with mutated NPM1 is a provisional entity in WHO classification of\n  acute myeloid leukemia (AML). This mutation should be tested for in clinical trials\n  and is recommended for testing in patients with cytogentically normal AML.\ndisease_id: 2\nsource_id: 125\nvariant_id: 86\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	09e08f23-1ef8-402b-b3cf-84da9bd39ae2	2015-12-09 16:58:51.32484
361	149	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were not associated with the M3 acute myeloid leukemia\n  FAB subtype (0/55).\ndisease_id: 2\nsource_id: 126\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	93477959-a0ee-4b90-a2a6-1e70a59ebc93	2015-12-09 16:58:51.369603
362	150	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid\n  leukemia.\ndisease_id: 2\nsource_id: 127\nvariant_id: 86\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	20065c6b-7ee7-4bc9-9a65-a3faa162b9ca	2015-12-09 16:58:51.411493
363	151	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with de novo AML as well as M4 and M5\n  FAB subtypes\ndisease_id: 2\nsource_id: 117\nvariant_id: 86\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8f230029-1161-4aeb-8dc7-1180113a5f1b	2015-12-09 16:58:51.450506
364	152	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute\n  myeloid leukemia.\ndisease_id: 2\nsource_id: 127\nvariant_id: 86\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2deb13c4-9826-4d3c-a61e-c97cf0f190cb	2015-12-09 16:58:51.487581
365	153	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with M5a and M5b FAB subtypes of acute\n  myeloid leukemia\ndisease_id: 2\nsource_id: 126\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8b90b767-0d49-478c-ab9f-0f1c7c7f947a	2015-12-09 16:58:51.522566
366	154	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with normal karyotype in older (>60) patients\ndisease_id: 2\nsource_id: 128\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d4411167-247b-42e2-b841-74996a594a98	2015-12-09 16:58:51.583849
367	155	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: ATRA treatment did not effect overall survival in patients <60 years\n  old with NPM1 mutation regardless of FLT3-ITD status\ndisease_id: 2\nsource_id: 116\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	383f1029-eaa2-4eac-979d-7f5ba99d0aed	2015-12-09 16:58:51.618458
368	156	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: ATRA treatment improved overall and relapse-free survival in older (>60)\n  patients with mutant NPM1 and without FLT3-ITD\ndisease_id: 2\nsource_id: 128\nvariant_id: 86\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c2c4acad-e52c-4666-9371-40eeb4191538	2015-12-09 16:58:51.654083
369	157	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young patients (18-60) with NPM1 mutations had improved overall survival\n  following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis.\n  However, the adjusted p-value for multiple testing is 0.11.\ndisease_id: 2\nsource_id: 129\nvariant_id: 86\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f6d22a71-b09a-49fe-831c-4063d27f1d69	2015-12-09 16:58:51.703517
370	158	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: For patients with NPM1 mutation that achieved CR following induction\n  therapy, relapse-free survival was improved following treatment with valproic acid\ndisease_id: 2\nsource_id: 130\nvariant_id: 86\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b11bbe68-6e5b-458d-87e1-dd986a4a9ce6	2015-12-09 16:58:51.747608
371	159	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutation status of IDH1 did not change event-free survival of patients\n  with an NPM1 mutation\ndisease_id: 2\nsource_id: 121\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8034dc48-bba2-48d5-9820-07e8ea245274	2015-12-09 16:58:51.785389
372	160	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation without FLT3-ITD was associated with reduced relapse risk\n  and increased overall survival in young adult acute myeloid leukemia patients (median\n  age 43)\ndisease_id: 2\nsource_id: 117\nvariant_id: 86\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1029253c-d4d5-4b73-94ac-550a039a1dc4	2015-12-09 16:58:51.824226
448	232	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with the presence pf lymph node\n  metastasis in gastric cancers.\ndisease_id: 23\nsource_id: 158\nvariant_id: 21\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a677263b-773c-4c1a-9fd4-829786171d01	2015-12-09 16:58:55.36723
373	161	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutations were associated with improved complete remission\n  rates as well as overall and disease free survival when all ages and karyotype were\n  analyzed; however, when only normal karyotype patients were analyzed, only complete\n  remission rates were improved\ndisease_id: 2\nsource_id: 128\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	acd84a8a-1e23-414a-adb9-197edd5c6c64	2015-12-09 16:58:51.859039
374	162	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rates were higher and both disease-free and overall\n  survival were longer for patients with Exon 12 NPM1 mutations.\ndisease_id: 2\nsource_id: 126\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1d8c5d9a-f8be-450e-b6c3-61d925317257	2015-12-09 16:58:51.894113
375	163	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rates were higher and event-free survival was longer\n  for normal karyotype AML patients with Exon 12 NPM1 mutations.\ndisease_id: 2\nsource_id: 127\nvariant_id: 86\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0db5b594-caf4-423b-8659-15b482b77585	2015-12-09 16:58:51.931243
376	164	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cytoplasmic localization of NPM in AML patients was an independent prognositic\n  factor associated with complete remission in response to induction therapy in a\n  multivariate analysis (p=0.019) but not in a univariate analysis (p=0.07).\ndisease_id: 2\nsource_id: 131\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fdb5cfc5-3dab-40a2-bf3d-fcb43aaf07ed	2015-12-09 16:58:51.98067
377	165	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Event-free survival was significantly improved in patients with both\n  NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone\n  2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis\n  3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal\n  karyotype patients were analyzed\ndisease_id: 2\nsource_id: 120\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b93fdb1c-95d7-4323-9987-9cecd6f4f72d	2015-12-09 16:58:52.021916
378	166	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Intermediate risk patients with mutant NPM1 had improved overall survival\n  with the presence of either IDH1 or IDH2 mutations than those wildtype for both\n  IDH1 and IDH2\ndisease_id: 2\nsource_id: 129\nvariant_id: 86\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1bb61a0e-9cbe-4ae7-bdb4-f5544fb1a2e2	2015-12-09 16:58:52.070986
379	167	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Normal karyotype patients with NPM1 mutation and without FLT3-ITD had\n  improved overall and relapse-free survival, complete remission rates and remission\n  duration compared to all other combinations of mutants\ndisease_id: 2\nsource_id: 132\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1b2d601a-5aa4-4170-ab03-4ba367ff0353	2015-12-09 16:58:52.123644
380	168	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation was associated with higher complete remission rates, lower\n  cumulative incidence of relapse and higher overall survival in intermediate risk\n  AML patients\ndisease_id: 2\nsource_id: 133\nvariant_id: 86\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	71feb929-cdb2-46c1-80e3-94587918e8da	2015-12-09 16:58:52.175679
381	169	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation was associated with increased overall, event-free and relapse-free\n  survival and remission after induction therapy in the entire cohort and in the cytogenetically\n  normal subset of patients\ndisease_id: 2\nsource_id: 134\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	662d4d50-a1f7-4474-80ef-f3d14cb989de	2015-12-09 16:58:52.235142
382	170	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation without FLT3-ITD was significantly associated with complete\n  remission\ndisease_id: 2\nsource_id: 112\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fce4bffb-0be0-482c-a689-8d0f74f5abc2	2015-12-09 16:58:52.296205
383	171	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with increased complete remission rates\n  as well as longer overall disease free survival in normal karyotype AML patients\n  >59 years old, particularly in those >69 years old\ndisease_id: 2\nsource_id: 135\nvariant_id: 86\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	eeef29ca-baf1-4593-9855-ab3ddd04208e	2015-12-09 16:58:52.34682
384	172	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation\n  and without FLT3-ITD have increased overall survival, relapse-free survival and\n  response to induction chemotherapy\ndisease_id: 2\nsource_id: 136\nvariant_id: 86\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1e8c5b7d-a48d-4cb2-82bb-70bade12268f	2015-12-09 16:58:52.393496
385	173	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Coocurrence of FLT3-ITD mutations significantly reduced overall survival\n  of normal karyotype AML patients\ndisease_id: 2\nsource_id: 126\nvariant_id: 86\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4647c1d3-cda8-462b-a089-266c5fa68797	2015-12-09 16:58:52.438949
386	174	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse\n  in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction\n  in NPM1 levels following first line therapy (N=15)\ndisease_id: 2\nsource_id: 137\nvariant_id: 86\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6272e06e-a7d4-4b3f-8f0b-07a41474c820	2015-12-09 16:58:52.500262
387	175	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival\n  compared to patients with wildtype FLT3\ndisease_id: 2\nsource_id: 118\nvariant_id: 86\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	033fcac1-d5b2-4a7e-bb08-2e0e78ac7d2d	2015-12-09 16:58:52.547907
388	176	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall\n  survival and cumulative incidence of relapse than those with NPM1 mutations alone\n  in intermediate risk AML patients\ndisease_id: 2\nsource_id: 133\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9a9eb15a-eb1d-43e2-ac9e-6cf20402ea30	2015-12-09 16:58:52.596149
389	177	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: TET2 mutation reduces overall survival of normal karyotype patients with\n  either an NPM1 mutation or NPM1 mutation without FLT3-ITD\ndisease_id: 2\nsource_id: 138\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	efc292c9-6a2d-4a25-a7ac-e44444c5dad2	2015-12-09 16:58:52.644722
390	178	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable,\n  a higher ratio was associated with reduced event-free survival in patients with\n  an NPM1 mutation\ndisease_id: 2\nsource_id: 118\nvariant_id: 86\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ca0d16c2-74b2-416d-ac73-7ce4e1b53a61	2015-12-09 16:58:52.689456
391	179	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NSC348884 combined with ATRA treatment induces apoptosis in primary AML\n  cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1\n  and FLT3-ITD\ndisease_id: 2\nsource_id: 139\nvariant_id: 86\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	fef42a6c-96b3-4fcb-bd5c-c2d54ea9f44c	2015-12-09 16:58:52.73576
392	180	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CD33 and CD123 expression was significantly increased in patients with\n  NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33\n  and anti-CD123 therapy\ndisease_id: 2\nsource_id: 140\nvariant_id: 86\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a1d771ba-dd95-4919-a1fe-451eed347680	2015-12-09 16:58:52.784144
393	181	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CD33 expression was significantly increased in patients with NPM1 mutation\n  with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy\ndisease_id: 2\nsource_id: 140\nvariant_id: 86\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9b7d7cdd-00d1-4163-baa8-2493555ad9e2	2015-12-09 16:58:52.825759
394	182	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: No NPM1 mutations were identified in patients with favorable risk cytogenetics\n  (79/215 patients)\ndisease_id: 2\nsource_id: 54\nvariant_id: 87\nrating: 4\nstatus: accepted\nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	465690a7-2f5a-4fad-a592-0fea7f1d010b	2015-12-09 16:58:52.862156
395	8	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: NPM1 Exon 12\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	ba15976a-5918-4ab8-8361-d1be77a2a1be	2015-12-09 16:58:52.88909
396	183	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with intermediate risk cytogenetics (including\n  normal karyotype)\ndisease_id: 2\nsource_id: 54\nvariant_id: 87\nrating: 4\nstatus: accepted\nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cc1232e6-b822-490d-bedd-b51f50a6cf3f	2015-12-09 16:58:52.922489
397	184	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.\ndisease_id: 2\nsource_id: 139\nvariant_id: 87\nrating: 3\nstatus: accepted\nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	efde1de6-0401-422b-8384-fc36e9154526	2015-12-09 16:58:52.98822
398	185	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM\n  when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization\n  of NPM in AML patients was associated with good response to induction therapy.\ndisease_id: 2\nsource_id: 131\nvariant_id: 87\nrating: 3\nstatus: accepted\nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6642dfb2-5578-4577-88fb-7239302262cb	2015-12-09 16:58:53.032846
399	186	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In chemotherapy-refractory patients with colorectal cancer, NRAS mutation\n  status (primarily Q61) was not informative for disease control rate, progression\n  free survival or overall survival following cetuximab plus chemotherapy.\ndisease_id: 8\nsource_id: 83\nvariant_id: 94\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a258e739-32f9-4960-a9d5-ded4da78a0f6	2015-12-09 16:58:53.077447
400	187	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring Exon\n  20 PIK3CA mutations had lower response and disease control rates as well as shorter\n  progression free and overall survival following cetuximab plus chemotherapy than\n  those with wildtype PIK3CA\ndisease_id: 8\nsource_id: 83\nvariant_id: 105\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	13cf4e84-eaae-4ac7-b94b-4597098c0c44	2015-12-09 16:58:53.117957
401	188	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA\n  mutations (Exon 9) had no significant impact on response rate, disease control rate,\n  progression free surivival or overall survival following cetuximab plus chemotherapy\n  than those with wildtype PIK3CA\ndisease_id: 8\nsource_id: 83\nvariant_id: 106\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6fff6b22-b8de-4a0b-a7ff-cf3f347fba22	2015-12-09 16:58:53.158784
402	189	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Following rapamycin treatment of mice with KrasG12D-induced lung tumors,\n  mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung\n  exhibit increased overall survival, increased apoptosis, reduced proliferation and\n  reduced tumor burden compared to Tsc1 wildtype mice.\ndisease_id: 1\nsource_id: 141\nvariant_id: 124\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6ef6fe7e-c9b7-471b-be8f-a18dd9d09414	2015-12-09 16:58:53.206272
403	9	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: TSC Loss\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	ae70116a-1998-4478-9831-6a50a07bad20	2015-12-09 16:58:53.232381
404	190	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity\n  to everolimus than their wild-type counterparts.\ndisease_id: 16\nsource_id: 142\nvariant_id: 125\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b027d2c3-5602-46d0-987e-8bc4409ce1d0	2015-12-09 16:58:53.281091
405	191	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with an inferior response to induction\n  chemotherapy with a higher rate of resistant disease in young (15-60+, median 45)\n  patients with cytogenetically normal AML\ndisease_id: 2\nsource_id: 143\nvariant_id: 129\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e1bbb1f3-50cd-4958-93dd-3e3a85f280e6	2015-12-09 16:58:53.340412
406	192	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: No differences in relapse-free or overall survival were identified between\n  young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in\n  cytogenetically normal AML\ndisease_id: 2\nsource_id: 144\nvariant_id: 129\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	98b9982f-702c-4460-9b09-92e6df4ed03f	2015-12-09 16:58:53.389056
407	193	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rates of complete remission and refractory disease are not different\n  in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young\n  (16-60) patients with cytogenetically normal AML\ndisease_id: 2\nsource_id: 144\nvariant_id: 129\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9a94ecc2-060a-401a-a7f9-a80718ac0a5c	2015-12-09 16:58:53.428459
408	194	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutations  in WT1 were associated with increased risk of recurrence in\n  young patients (15-50) with cytogenetically normal AML\ndisease_id: 2\nsource_id: 145\nvariant_id: 129\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c5caf321-7592-4980-9eaa-88635fcebcf6	2015-12-09 16:58:53.476612
409	195	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutations in WT1 were associated with a worse overall prognosis than\n  patients wildtype for WT1 in young patients (15-50), primarily because of increased\n  risk of disease recurrence\ndisease_id: 2\nsource_id: 145\nvariant_id: 129\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	443ae65d-4262-4563-ab60-0696a0c89c38	2015-12-09 16:58:53.517154
410	196	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had\n  significantly lower complete remission and higher refractory disease rates than\n  those with wildtype WT1 and without FLT3-ITD following induction therapy in young\n  (16-60) patients with cytogenetically normal AML\ndisease_id: 2\nsource_id: 144\nvariant_id: 129\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fffe527b-2faf-4439-bb63-503df0165af2	2015-12-09 16:58:53.555469
411	197	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\ndisease_id: 2\nsource_id: 146\nvariant_id: 129\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ab89e15b-b593-4cd7-8a7c-475a0fc49fd1	2015-12-09 16:58:53.600437
412	198	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were a negative prognostic factor for overall survival\n  in young (15-60+, median 45) patients with cytogenetically normal AML\ndisease_id: 2\nsource_id: 143\nvariant_id: 129\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	615fd2c2-50c7-4da7-82c7-c74e14c8c3e8	2015-12-09 16:58:53.645407
413	199	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\ndisease_id: 2\nsource_id: 143\nvariant_id: 129\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	133feff8-df4f-4e23-8ee7-d864ff389c33	2015-12-09 16:58:53.712955
414	200	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with an inferior response to induction\n  chemotherapy with a higher rate of resistant disease in young (15-60+, median 45)\n  patients with cytogenetically normal AML\ndisease_id: 2\nsource_id: 143\nvariant_id: 130\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	56ec1c99-b931-45f6-94a8-e5ba971d88c3	2015-12-09 16:58:53.765889
415	201	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: No differences in relapse-free or overall survival were identified between\n  young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in\n  cytogenetically normal AML\ndisease_id: 2\nsource_id: 144\nvariant_id: 130\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a9c67b1d-564e-4cfc-ac6c-55fa1fe6726c	2015-12-09 16:58:53.808506
431	216	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\ndisease_id: 4\nsource_id: 151\nvariant_id: 3\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	32543c39-dbb5-46f6-8c61-05d66fe69c4b	2015-12-09 16:58:54.581457
416	202	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rates of complete remission and refractory disease are not different\n  in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young\n  (16-60) patients with cytogenetically normal AML.\ndisease_id: 2\nsource_id: 144\nvariant_id: 130\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5087ace4-e3f6-4741-8e1b-da6c17b21ace	2015-12-09 16:58:53.848992
417	203	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had\n  significantly lower complete remission and higher refractory disease rates than\n  those with wildtype WT1 and without FLT3-ITD following induction therapy in young\n  (16-60) patients with cytogenetically normal AML\ndisease_id: 2\nsource_id: 144\nvariant_id: 130\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	27437913-c3db-48af-bad6-0f72bf443ca4	2015-12-09 16:58:53.89225
418	204	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were a negative prognostic factor for overall survival\n  in young (15-60+, median 45) patients with cytogenetically normal AML\ndisease_id: 2\nsource_id: 143\nvariant_id: 130\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	95bd1a33-57e1-4de6-b0ce-bbaa49b54b1a	2015-12-09 16:58:53.9323
419	205	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\ndisease_id: 2\nsource_id: 143\nvariant_id: 130\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c340e35c-aca4-4be0-a68d-6023f1434371	2015-12-09 16:58:53.969581
420	206	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\ndisease_id: 2\nsource_id: 146\nvariant_id: 130\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	36b648c6-c63d-4386-b5f3-ccd354b92005	2015-12-09 16:58:54.012418
421	207	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The combination of PARP inhibition (Olaparib) with BRCA mutation results\n  in irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation\n  positive women with high grade serous or undifferentiated ovarian cancer.\ndisease_id: 18\nsource_id: 147\nvariant_id: 131\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 1\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7971996d-c2c6-4836-8873-533a992b8ee3	2015-12-09 16:58:54.066799
422	10	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: BRCA Germline Variants\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	9cd75435-f204-490a-9cb3-8516d0dd078d	2015-12-09 16:58:54.09224
423	208	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The combination of PARP inhibition (Olaparib) with BRCA mutation results\n  in irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation\n  positive women with high grade serous or undifferentiated ovarian cancer.\ndisease_id: 18\nsource_id: 147\nvariant_id: 132\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 1\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3c18dca9-def5-46ec-af0d-611bdf8e3ce7	2015-12-09 16:58:54.135852
424	209	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of BCR-ABL fusion is considered the characterizing feature\n  of chronic myeloid leukemia, and has been widely thought of as the initiating event\n  in the disease.\ndisease_id: 4\nsource_id: 148\nvariant_id: 1\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4fb6f548-65e0-4114-b70d-7747030d5b56	2015-12-09 16:58:54.196259
425	210	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Treatment of Philadelphia Chromosome positive leukemias with Arsenic\n  Trioxide results in high rates of complete remission in patients.\ndisease_id: 2\nsource_id: 149\nvariant_id: 1\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7ee73385-4623-45bb-be3b-945905ccb2fa	2015-12-09 16:58:54.253663
426	211	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The clinical use of imatinib in patients with BCR-ABL fusion has resulted\n  in drastic sensitivity to the drug.\ndisease_id: 4\nsource_id: 148\nvariant_id: 1\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	73a194da-121a-4740-9c02-d587b2f4a53f	2015-12-09 16:58:54.311782
427	212	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BCR-ABL inhibitors such as imatinib have lead to significantly improved\n  prognosis, response rate, overall survival, and patient outcome in CML patients\n  compared to previous therapeutic regimens.\ndisease_id: 4\nsource_id: 148\nvariant_id: 1\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7edfae41-582f-4e68-a590-4d558dffbb0b	2015-12-09 16:58:54.370119
428	213	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The use of second-generation BCR-ABL targeted therapies has been effective\n  in patients resistant to imatinib.\ndisease_id: 4\nsource_id: 148\nvariant_id: 1\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	da8f28b2-574f-4810-aee3-df9746c6e526	2015-12-09 16:58:54.410297
429	214	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Data from deep sequencing of a Ph-negative clone of a Ph-positive CML\n  patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion,\n  and may imply the possibility that BCR-ABL is not universally the initiating event\n  in CML.\ndisease_id: 4\nsource_id: 150\nvariant_id: 1\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	09683e0d-cebf-4c8c-81bf-8d098fcdebe7	2015-12-09 16:58:54.482181
430	215	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BCR-ABL fusions have been found to contribute to imatinib resistance\n  in AML cell lines.\ndisease_id: 2\nsource_id: 149\nvariant_id: 1\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3948df43-c45c-4493-8eed-ef5ef95e8f70	2015-12-09 16:58:54.531118
432	217	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In chronic myeloid leukemia patients with the ABL T315I mutation, tumors\n  have shown to be resistant to imatinib treatment.\ndisease_id: 4\nsource_id: 148\nvariant_id: 2\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fd0db7a3-9e6f-4577-8971-01c6d1da9b1e	2015-12-09 16:58:54.630092
433	218	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\ndisease_id: 4\nsource_id: 151\nvariant_id: 2\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a550bdff-f6b7-4d36-adbb-d450a0aa1ee3	2015-12-09 16:58:54.67827
434	219	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity\n  to the AKT inhibitor MK-2206 in breast cancer cell lines.\ndisease_id: 19\nsource_id: 152\nvariant_id: 4\nrating: 3\nstatus: accepted\nvariant_hgvs: 14:105246551-105246551 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ef3d9bc1-201f-44b2-a48f-9e69e8ab3d1c	2015-12-09 16:58:54.733173
435	220	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with non-small cell lung cancer harboring the EML4-ALK fusion,\n  treatment with crizotinib has shown to be effective in acheiving stable disease.\ndisease_id: 1\nsource_id: 153\nvariant_id: 5\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	884f8028-f492-4190-9a97-5ca0d9ad6a88	2015-12-09 16:58:54.780937
436	11	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: ALK Fusions\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	be208331-c866-4742-9b7c-b6eb4a4f6309	2015-12-09 16:58:54.804667
437	221	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with non-small cell lung cancer harboring EML4-ALK fusion,\n  the C1156Y variant has been shown to confer resistance to crizotinib.\ndisease_id: 1\nsource_id: 153\nvariant_id: 6\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fb4107c3-b8ae-4ae0-9a35-e0a3cf403162	2015-12-09 16:58:54.838907
438	222	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with non-small cell lung cancer harboring EML4-ALK fusion,\n  the L1196M variant has been shown to confer resistance to crizotinib.\ndisease_id: 1\nsource_id: 153\nvariant_id: 7\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9dfc0c0f-6417-4bd7-b0be-3bb9a9dce530	2015-12-09 16:58:54.887477
439	223	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Inceased copy number of CCND1 is associated with poorer overall survival.\ndisease_id: 1\nsource_id: 154\nvariant_id: 18\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1bb35bdf-8081-4627-8e16-c77a923ca6fa	2015-12-09 16:58:54.944676
440	224	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Three studies have found no significant survival impact for CCND1 in\n  lung cancer.\ndisease_id: 1\nsource_id: 154\nvariant_id: 19\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	183bedea-cb2c-4dee-87a9-f567e00cb739	2015-12-09 16:58:54.983747
441	225	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Three studies have associated CCND1 expression with poorer survival.\ndisease_id: 1\nsource_id: 154\nvariant_id: 19\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7a356e3e-5b39-4ecd-83bd-176ed9378e44	2015-12-09 16:58:55.025222
442	226	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D1 overexpression is associated with the ER-positive subtype of\n  breast cancer.\ndisease_id: 20\nsource_id: 155\nvariant_id: 20\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ee53a12e-484b-402e-aa4e-c02d53e9c503	2015-12-09 16:58:55.071992
443	227	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In ER-positive breast cancer patients, Cyclin D1 overexpression is associated\n  with shorter overall survival and increased metastasis.\ndisease_id: 20\nsource_id: 155\nvariant_id: 20\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	995d6c70-0da1-4ff9-b109-9703f7f32a29	2015-12-09 16:58:55.115883
444	228	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D1 overexpression is associated with shorter overall survival\n  and increased metastasis in head and neck squamous cell carcinoma.\ndisease_id: 21\nsource_id: 156\nvariant_id: 20\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a693397a-f77f-463d-9690-55b40fca72bb	2015-12-09 16:58:55.189648
445	229	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D1 overexpression is associated with shorter overall survival\n  and increased metastasis in mantle cell lymphoma.\ndisease_id: 22\nsource_id: 157\nvariant_id: 20\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9f9a5ac8-ea4c-464d-a0f6-f9f622a41d71	2015-12-09 16:58:55.237275
446	230	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with poor prognosis in gastric\n  cancers.\ndisease_id: 23\nsource_id: 158\nvariant_id: 21\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0caf1971-ca0c-4ef1-8aad-b9bf7a42bbd8	2015-12-09 16:58:55.287235
447	231	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with the increased depth of cancer\n  invasion in gastric cancers.\ndisease_id: 23\nsource_id: 158\nvariant_id: 21\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1ea6fd14-2b15-4954-b813-f9a8439860e5	2015-12-09 16:58:55.329177
658	61	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '209113112'\nstop:\n- \n- '209113112'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- A\n	2	\N	\N	129d868c-c63d-4165-a49e-563fb562be81	2015-12-09 17:02:08.314548
659	63	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '15'\nstart:\n- \n- '90631838'\nstop:\n- \n- '90631838'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	827095ca-ff99-41c9-b5fa-11ed2d0c7a64	2015-12-09 17:02:08.336907
449	233	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with vascular invasion by cancer\n  cells in gastric cancers.\ndisease_id: 23\nsource_id: 158\nvariant_id: 21\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9e7f2540-b67b-4c0e-8d85-00875b3f3454	2015-12-09 16:58:55.400906
450	234	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CCND2 hypomethylation is seen to be more common in stage III and IV gastric\n  tumors than stage I and II.\ndisease_id: 23\nsource_id: 159\nvariant_id: 22\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7e8ec37d-1923-4f43-89ec-38e54dbc7796	2015-12-09 16:58:55.446092
451	235	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with\n  Palbociclib (PD-0332991) significantly increased median survival.\ndisease_id: 24\nsource_id: 160\nvariant_id: 23\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	716b51b2-cd97-4704-b65f-39031ab2bd80	2015-12-09 16:58:55.490956
452	236	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with high expression of total cyclin E and low-molecular weight\n  cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard\n  ratio for these patients in 13.3 times higher than those with normal cyclin E levels.\ndisease_id: 20\nsource_id: 161\nvariant_id: 24\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e6a15951-0b6c-4339-b8b2-cdf23ef0f2ae	2015-12-09 16:58:55.535773
453	237	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced\n  stage gastric carcinoma.\ndisease_id: 25\nsource_id: 162\nvariant_id: 24\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3e51efa8-4475-4364-ad6d-fc697759af62	2015-12-09 16:58:55.579585
454	238	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\ndisease_id: 25\nsource_id: 162\nvariant_id: 24\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6d9a9ddb-51e1-4ca4-bffd-a0ef91ef48b3	2015-12-09 16:58:55.618352
455	239	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\ndisease_id: 26\nsource_id: 162\nvariant_id: 24\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b96f72c9-47bd-4bf0-a2c0-c6f36c0875ed	2015-12-09 16:58:55.659958
456	240	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\ndisease_id: 27\nsource_id: 163\nvariant_id: 25\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	06721fed-6e7e-4c30-87bc-c0d6cc9bb204	2015-12-09 16:58:55.709319
457	241	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\ndisease_id: 27\nsource_id: 164\nvariant_id: 25\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	3770e700-8dae-4c52-a8f0-9f98db7c98b6	2015-12-09 16:58:55.756762
458	242	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\ndisease_id: 27\nsource_id: 163\nvariant_id: 26\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9b682b27-fddb-4e98-b8f6-40f215c9b41a	2015-12-09 16:58:55.800634
459	243	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\ndisease_id: 27\nsource_id: 164\nvariant_id: 26\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	2e5fcb44-7048-43ae-858c-0531ef160b31	2015-12-09 16:58:55.856901
460	244	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A)\n  promoter hypermethylation is associated with shorter overall survival.\ndisease_id: 1\nsource_id: 165\nvariant_id: 27\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bd0dbbf4-6b9e-4a4b-8e25-4dd8f1a84881	2015-12-09 16:58:55.9114
461	245	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A)\n  promoter hypermethylation is associated with shorter time to recurrence.\ndisease_id: 1\nsource_id: 165\nvariant_id: 27\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e567357f-c7d0-4de0-992b-a46ab1c39ba6	2015-12-09 16:58:55.951875
462	246	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the G253C mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 1\nsource_id: 166\nvariant_id: 141\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b4968c5e-aa25-4e76-b2f9-0c2b36634d9a	2015-12-09 16:58:55.997899
463	247	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the G505S mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 1\nsource_id: 166\nvariant_id: 142\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	60a30a00-3eb2-4702-9610-10baf73c1d41	2015-12-09 16:58:56.03587
660	64	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '9'\nstart:\n- \n- '5073770'\nstop:\n- \n- '5073770'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	7b3da984-46ec-44ee-b118-43f3c144fd55	2015-12-09 17:02:08.35951
464	248	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the G774V mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 1\nsource_id: 166\nvariant_id: 144\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0cb98b36-de51-4481-8074-05a60c58485b	2015-12-09 16:58:56.071222
465	249	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the I638F mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 1\nsource_id: 166\nvariant_id: 143\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d01cdf73-67ca-4fa7-82d3-8f404ca35627	2015-12-09 16:58:56.107559
466	250	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the L239R mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 1\nsource_id: 166\nvariant_id: 140\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d4442c4f-546a-4fd3-968b-af68b5ce9af3	2015-12-09 16:58:56.143902
467	251	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the L63V mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 1\nsource_id: 166\nvariant_id: 139\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	17fc1655-e00e-46e6-ad05-878c65b50e8b	2015-12-09 16:58:56.181301
468	252	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a patient expressing a DDR2 S768R mutation, treatment with dasatinib\n  plus erlotinib acheived a partial response.\ndisease_id: 1\nsource_id: 166\nvariant_id: 145\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	caaca9e8-2bfe-427a-b0e5-7af50789fb52	2015-12-09 16:58:56.217858
469	253	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Exon 19 deletion has been shown to be correlated with gefitinib response.\ndisease_id: 1\nsource_id: 167\nvariant_id: 133\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3f67e7cf-5973-4c74-8855-2e6e91db5cc3	2015-12-09 16:58:56.265274
470	254	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Gefinitib has been shown to be effective in treating cell lines with\n  G719S missense mutations.\ndisease_id: 1\nsource_id: 168\nvariant_id: 134\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c8333c73-d4b5-47fa-a8aa-3f2254a997f5	2015-12-09 16:58:56.312376
471	255	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no statistical difference in progression free survival between\n  lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.\ndisease_id: 1\nsource_id: 169\nvariant_id: 33\nrating: 3\nstatus: accepted\nvariant_hgvs: 7:55259514-55259515 (CT->AG)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	5ec87ebd-ecc9-4e49-8ceb-2c6f37fbf9c4	2015-12-09 16:58:56.358589
472	256	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence\n  of L858R mutation is prognostic for better progression free survival.\ndisease_id: 1\nsource_id: 170\nvariant_id: 33\nrating: 4\nstatus: accepted\nvariant_hgvs: 7:55259514-55259515 (CT->AG)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	37752dfd-83f3-430f-99f1-075165c64a54	2015-12-09 16:58:56.428782
473	257	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Median survival of patients with EGFR L858R mutation is better than those\n  with wild type EGFR.\ndisease_id: 1\nsource_id: 171\nvariant_id: 33\nrating: 3\nstatus: accepted\nvariant_hgvs: 7:55259514-55259515 (CT->AG)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	84565153-54a8-4fe8-9496-02d77ff63b52	2015-12-09 16:58:56.485044
474	258	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Gefinitib has been shown to be effective in treating cell lines with\n  L858R missense mutations.\ndisease_id: 1\nsource_id: 168\nvariant_id: 33\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	857ee1aa-3c22-489a-b24d-04848a75c85a	2015-12-09 16:58:56.522929
475	259	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The T790M mutation in EGFR has been shown to confer resistance to the\n  tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are\n  placed on the drug are likely to relapse.\ndisease_id: 28\nsource_id: 172\nvariant_id: 34\nrating: 5\nstatus: accepted\nvariant_hgvs: 7:55249071-55249071 (C->T)\nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8ce49420-72e4-4db4-8cb2-509412ec6e1b	2015-12-09 16:58:56.577047
476	12	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: EGFR TKI Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	bd27eac6-a3b5-47bb-a8ae-e04d2a0afeab	2015-12-09 16:58:56.605773
477	260	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In non-small cell lung cancer, the appearance of T790M mutation leads\n  to resistance to gefitinib.\ndisease_id: 1\nsource_id: 173\nvariant_id: 34\nrating: 3\nstatus: accepted\nvariant_hgvs: 7:55249071-55249071 (C->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c0dd9b85-96fd-4fb3-b29b-802a18804cf7	2015-12-09 16:58:56.64554
478	261	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients with T790M and another activating mutations, their\n  progression free survival is shorter compared to those who do not possess T790M\n  mutation.\ndisease_id: 1\nsource_id: 174\nvariant_id: 34\nrating: 4\nstatus: accepted\nvariant_hgvs: 7:55249071-55249071 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ee8732e8-21cb-43fe-b760-0e269b4ddd96	2015-12-09 16:58:56.689315
479	262	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NSCLC harboring EGFR T790M mutation have statistically\n  worse overall survival compared to patients with L858R or other exon 19 activating\n  mutations.\ndisease_id: 1\nsource_id: 175\nvariant_id: 34\nrating: 4\nstatus: accepted\nvariant_hgvs: 7:55249071-55249071 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	20f59074-0e75-4eb5-a084-89f3402540ce	2015-12-09 16:58:56.732915
480	263	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a NSCLC patient with T790M and L858R mutation, combination treatment\n  with erlotinib and premetrexed has shown to be effective, reducing tumor size and\n  resulting in stable disease.\ndisease_id: 1\nsource_id: 176\nvariant_id: 34\nrating: 3\nstatus: accepted\nvariant_hgvs: 7:55249071-55249071 (C->T)\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	e93d6c71-4cb2-4b82-9130-afbc2b9705c6	2015-12-09 16:58:56.777643
481	264	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC containing a T790M mutation, staurosporine may be inhibitive\n  of EGFR, especially when an L858R mutation is also present.\ndisease_id: 1\nsource_id: 177\nvariant_id: 34\nrating: 1\nstatus: accepted\nvariant_hgvs: 7:55249071-55249071 (C->T)\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cc5eced8-f928-4fe6-b1eb-fdcb1f98d8ed	2015-12-09 16:58:56.827581
482	265	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804),\n  MET amplification and EGFR T790M were found to be the primary drivers of resistance.\ndisease_id: 1\nsource_id: 178\nvariant_id: 34\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e0f68bd6-38f3-45c3-8119-c1387c4a537f	2015-12-09 16:58:56.877478
483	266	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the D769H mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 20\nsource_id: 179\nvariant_id: 35\nrating: 4\nstatus: accepted\nvariant_hgvs: 17:37880261-37880261 (G->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1a7ea478-9ec3-4e53-80ff-fadb3ed2f34f	2015-12-09 16:58:56.930942
484	13	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: HER2 Activating\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	e14a1ec9-b4b2-4737-a5d2-fa3cb990d224	2015-12-09 16:58:56.955433
485	267	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the D769Y mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 20\nsource_id: 179\nvariant_id: 36\nrating: 4\nstatus: accepted\nvariant_hgvs: 17:37880261-37880261 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a10589c7-8e52-4cea-89bb-fb34ee7ecbd4	2015-12-09 16:58:56.989773
486	268	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines retrovirally transduced with wildtype or mutant\n  ERBB2, cells expressing the in-frame deletion of amino acids 755-759 were shown\n  to be sensitive to neratinib.\ndisease_id: 20\nsource_id: 179\nvariant_id: 37\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	03cb1a77-b384-4bfc-ac4b-4ac56b2ad81e	2015-12-09 16:58:57.034526
487	269	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the G309A mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 20\nsource_id: 179\nvariant_id: 38\nrating: 5\nstatus: accepted\nvariant_hgvs: 17:37868205-37868205 (G->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	56b71a1e-d6c9-43e5-b3ec-cc3ac030a145	2015-12-09 16:58:57.070221
488	270	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The L755S mutation was shown to confer resistance to lapatinib in MCF10A\n  cell lines retrovirally transduced with wildtype or mutant ERBB2.\ndisease_id: 20\nsource_id: 179\nvariant_id: 39\nrating: 5\nstatus: accepted\nvariant_hgvs: 17:37880220-37880220 (T->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	519255e8-1c75-4b44-9f0f-8c1a4d0c0979	2015-12-09 16:58:57.128638
489	271	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The L755S mutation showed mild response to neratinib in MCF10A cell lines\n  retrovirally transduced with wildtype or mutant ERBB2.\ndisease_id: 20\nsource_id: 179\nvariant_id: 39\nrating: 5\nstatus: accepted\nvariant_hgvs: 17:37880220-37880220 (T->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d765365d-4614-4220-a39d-db6e7f88e207	2015-12-09 16:58:57.167681
490	272	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive\n  to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.\ndisease_id: 20\nsource_id: 179\nvariant_id: 40\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	65960b5e-819a-4b72-8f29-34fa807e9c55	2015-12-09 16:58:57.203039
491	273	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the in-frame insertion of a single amino acid at\n  position 780 was shown to be sensitive to neratinib.\ndisease_id: 20\nsource_id: 179\nvariant_id: 41\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c3c1cfb2-39de-45b8-a5af-32a1c9bc13b8	2015-12-09 16:58:57.239171
492	274	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\ndisease_id: 20\nsource_id: 179\nvariant_id: 42\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	a4693296-651c-4f8d-bd02-a23f0e28505d	2015-12-09 16:58:57.281212
493	275	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the R896C mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 20\nsource_id: 179\nvariant_id: 43\nrating: 5\nstatus: accepted\nvariant_hgvs: 17:37881616-37881616 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8340584a-4c27-4eef-8c51-b0c2d152ca1d	2015-12-09 16:58:57.321821
494	276	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the V777L mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 20\nsource_id: 179\nvariant_id: 44\nrating: 5\nstatus: accepted\nvariant_hgvs: 17:37881000-37881000 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	feacb0a7-f4df-4cb9-9bd1-62ace5f28e4f	2015-12-09 16:58:57.367948
495	277	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the V842I mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 20\nsource_id: 179\nvariant_id: 45\nrating: 5\nstatus: accepted\nvariant_hgvs: 17:37881332-37881332 (G->A)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7571a6af-7d66-40a7-bf75-9b99dc33492b	2015-12-09 16:58:57.410473
496	278	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 20\nsource_id: 180\nvariant_id: 46\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	813358a3-3c3b-4488-a906-807c8c1671e2	2015-12-09 16:58:57.460022
497	14	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: ESR1 Ligand-Binding Domain\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	f0187f90-5a4d-4628-bf59-9e248f665ebc	2015-12-09 16:58:57.484999
498	279	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The L536Q ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  MCF7 cell lines.\ndisease_id: 20\nsource_id: 181\nvariant_id: 46\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	7cb7878f-6a01-41cd-b9d9-111ce48a6d15	2015-12-09 16:58:57.531535
499	280	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the N538G mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 20\nsource_id: 180\nvariant_id: 47\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	58aaa03f-e257-4d54-8942-cacfe35c7192	2015-12-09 16:58:57.581735
500	281	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The N538G ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 20\nsource_id: 181\nvariant_id: 47\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	ba839e7e-4b93-4703-a47c-f8bddd95251e	2015-12-09 16:58:57.624688
501	282	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 20\nsource_id: 180\nvariant_id: 48\nrating: 3\nstatus: accepted\nvariant_hgvs: 6:152419923-152419923 (A->G)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	94c1d857-d4e5-41af-8792-2730c6e6af4d	2015-12-09 16:58:57.674466
502	283	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The Y537C ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 20\nsource_id: 181\nvariant_id: 48\nrating: 5\nstatus: accepted\nvariant_hgvs: 6:152419923-152419923 (A->G)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	b4f58160-6460-4906-8e43-64dcb1ce959e	2015-12-09 16:58:57.720532
503	284	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 20\nsource_id: 180\nvariant_id: 49\nrating: 3\nstatus: accepted\nvariant_hgvs: 6:152419922-152419922 (T->A)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6de37018-c54b-4841-952b-59938936929f	2015-12-09 16:58:57.779485
504	285	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The Y537N ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 20\nsource_id: 181\nvariant_id: 49\nrating: 5\nstatus: accepted\nvariant_hgvs: 6:152419922-152419922 (T->A)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	5cc0d8a8-7763-4495-8563-bda71e283f87	2015-12-09 16:58:57.824121
505	286	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the L537S mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 20\nsource_id: 180\nvariant_id: 50\nrating: 3\nstatus: accepted\nvariant_hgvs: 6:152419923-152419923 (A->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	75511733-80cd-4036-82e4-858ff547d753	2015-12-09 16:58:57.869844
506	287	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The Y537S ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 20\nsource_id: 181\nvariant_id: 50\nrating: 5\nstatus: accepted\nvariant_hgvs: 6:152419923-152419923 (A->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	5e87311c-3eb1-4ce6-a4c6-c9c7fb5cf51a	2015-12-09 16:58:57.914069
507	288	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a single patient with this fusion in intrahepatic cholangiocarcinoma.\n  Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic\n  sites and overall stable disease. However, this response did not meet criteria for\n  RECIST partial response.\ndisease_id: 29\nsource_id: 182\nvariant_id: 52\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cf9f3d08-2ade-40fa-a30f-97abfbe6dac0	2015-12-09 16:58:57.97773
508	289	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This\n  fusion is also shown to be recurrent in multiple other cancer types.\ndisease_id: 29\nsource_id: 182\nvariant_id: 54\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	9200d4aa-0ef9-4500-be9c-469b919904e5	2015-12-09 16:58:58.025748
509	290	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WBC counts were higher in AML patients with internal tandem repeat of\n  Flt3 compared to wild type\ndisease_id: 2\nsource_id: 183\nvariant_id: 55\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	92c0b4ff-21ce-44e1-8f23-c8c173848eb8	2015-12-09 16:58:58.079962
510	291	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated\n  AML patients with FLT3 mutations\ndisease_id: 2\nsource_id: 184\nvariant_id: 55\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b0b2831d-2862-482a-98bb-1ba4ec5c50b5	2015-12-09 16:58:58.12758
661	65	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55599321'\nstop:\n- \n- '55599321'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- T\n	2	\N	\N	e23b84d1-5c17-4bab-8ced-c39dfb854603	2015-12-09 17:02:08.390598
511	292	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Sorafenib is effective in patients with FLT3 internal tandem repeats,\n  but not in wild type or D835 mutation.\ndisease_id: 2\nsource_id: 185\nvariant_id: 55\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a7385941-3fcf-4bd5-ace1-89aeefb06ca8	2015-12-09 16:58:58.1737
512	293	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with internal tandem repeat of FLT3 has worse overall survival\n  than patients with wild type FLT3\ndisease_id: 2\nsource_id: 183\nvariant_id: 55\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	99e8cd61-d3d3-495f-af93-dc91e3d0fbe9	2015-12-09 16:58:58.20813
513	294	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads\n  to poor survival and inferior disease free survival in patients with AML.\ndisease_id: 2\nsource_id: 186\nvariant_id: 55\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8f6dc896-97cd-49c5-a1cc-62962442291d	2015-12-09 16:58:58.259761
514	295	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML\n  cells with FLT3 internal tandem repeat\ndisease_id: 2\nsource_id: 187\nvariant_id: 55\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1e91660e-79fb-465a-b2b5-9b189872874c	2015-12-09 16:58:58.30152
515	296	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WBC counts were not significantly different in AML patients with FLT3\n  D835 mutations compared to wild type\ndisease_id: 2\nsource_id: 183\nvariant_id: 56\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b43157ba-3c31-400f-b22a-78ab6fa1e3ff	2015-12-09 16:58:58.33795
516	297	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Emergence of D835 mutation in AML patients with FLT3 tandem repeat is\n  associated with sorafenib resistance\ndisease_id: 2\nsource_id: 188\nvariant_id: 56\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	887ff741-3257-4cc4-91db-ea33d6c5b4f6	2015-12-09 16:58:58.381221
517	298	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Sorafenib is effective in patients with FLT3 internal tandem repeat mutations,\n  but not in patients with wild type or D835 mutation.\ndisease_id: 2\nsource_id: 185\nvariant_id: 56\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c0dcb329-14f1-4b2a-901d-0aba250120ea	2015-12-09 16:58:58.436023
518	299	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated\n  AML patients with FLT3 mutations\ndisease_id: 2\nsource_id: 184\nvariant_id: 56\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	165ca43b-9481-4616-8052-130d2634ec08	2015-12-09 16:58:58.474179
519	300	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with D835 mutation of FLT3 did not have worse overall survival\n  than patients with wild type FLT3\ndisease_id: 2\nsource_id: 183\nvariant_id: 56\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	48096502-0987-4b7d-86d1-b97a90df14d5	2015-12-09 16:58:58.510692
520	301	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML patients, FLT3-TKD mutation was associated with poorer disease\n  free survival compared to patients with wild type FLT3\ndisease_id: 2\nsource_id: 189\nvariant_id: 56\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c0133c8c-092e-416a-beed-62023aac26af	2015-12-09 16:58:58.553224
521	302	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib\n  and fasudil led to dramatic tumor regression.\ndisease_id: 30\nsource_id: 190\nvariant_id: 57\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	4126d74a-952c-4beb-999e-697e439f4c5d	2015-12-09 16:58:58.600273
522	303	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In leukemia patients, IDH1 R132 mutation is associated with adults and\n  normal karyotype.\ndisease_id: 2\nsource_id: 191\nvariant_id: 58\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2a61c3e0-6313-46d5-8ff3-73db027fd379	2015-12-09 16:58:58.649187
523	304	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The complete remission and overall survival rates in patients with IDH1\n  R132 mutation is not significantly different from those who do not have this mutation\ndisease_id: 2\nsource_id: 192\nvariant_id: 58\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1e9d21a5-77ae-4914-9759-16bb04900ac1	2015-12-09 16:58:58.691931
524	305	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic\n  for better survival compared to patients who harbor wild type mutation.\ndisease_id: 31\nsource_id: 193\nvariant_id: 58\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7ee714f0-09f9-4b24-9669-a0bab22bd5a7	2015-12-09 16:58:58.741944
525	306	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with glioblastoma, those harboring IDH1 R132 mutation has\n  higher overall survival compared to those who do not have IDH1 mutation\ndisease_id: 32\nsource_id: 194\nvariant_id: 58\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fea96078-8e2b-4b96-b1d3-d49615436fb3	2015-12-09 16:58:58.784367
526	307	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients suffering from myelodysplastic syndromes\n  is associated with worse overall survival\ndisease_id: 33\nsource_id: 195\nvariant_id: 58\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	99ec92de-a8e2-4118-8117-c68124099f31	2015-12-09 16:58:58.835282
527	308	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation is associated with patients of older age, high platelet\n  count during diagnosis, cytogenic normalcy and NPM1 mutation\ndisease_id: 2\nsource_id: 196\nvariant_id: 59\nrating: 3\nstatus: accepted\nvariant_hgvs: 2:209113113-209113113 (G->A)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fadf2a8a-4062-4808-b09c-5d2316c4e8fa	2015-12-09 16:58:58.898237
528	309	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\ndisease_id: 2\nsource_id: 196\nvariant_id: 59\nrating: 2\nstatus: accepted\nvariant_hgvs: 2:209113113-209113113 (G->A)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a1c921c0-75fb-4f53-a009-4bd0f4a67831	2015-12-09 16:58:58.942016
529	310	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\ndisease_id: 2\nsource_id: 196\nvariant_id: 60\nrating: 3\nstatus: accepted\nvariant_hgvs: 2:209113112-209113112 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	69262793-7e1e-4d47-9f44-30a111eaf63e	2015-12-09 16:58:58.988237
530	311	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\ndisease_id: 2\nsource_id: 196\nvariant_id: 61\nrating: 3\nstatus: accepted\nvariant_hgvs: 2:209113112-209113112 (C->A)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9c53a377-3fc1-4215-aa2f-fcf5a153461b	2015-12-09 16:58:59.035832
531	312	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with IDH2 mutations such as R140Q/L have event free survival\n  and overall survival similar to those with wild-type IDH2\ndisease_id: 2\nsource_id: 197\nvariant_id: 62\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	11d44034-b1d3-451f-b4ba-1bcaf0865a3a	2015-12-09 16:58:59.110336
532	313	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with IDH2 R140Q/L mutation, the presence of mutation does\n  not impact overall survival or disease free survival.\ndisease_id: 2\nsource_id: 198\nvariant_id: 62\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	db78a850-1f7b-42ed-9c0f-22abd60dc3c6	2015-12-09 16:58:59.16263
533	314	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients suffering from MDS, the presence of IDH2 mutation such as\n  R140Q/L do not confer prognostic value (overall survival)\ndisease_id: 34\nsource_id: 199\nvariant_id: 62\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ab2b5df3-5383-4617-b9b8-9d595df6dd1c	2015-12-09 16:58:59.212246
534	315	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: R140Q mutation in IDH2 does not have prognostic value in patients with\n  MDS.\ndisease_id: 34\nsource_id: 200\nvariant_id: 62\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3d866771-097c-4fd2-9968-db01f763c499	2015-12-09 16:58:59.273643
535	316	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML, patients with IDH2 R140K mutation have improved overall survival\n  compared to those with wild-type IDH2\ndisease_id: 2\nsource_id: 201\nvariant_id: 62\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	469c7a2b-c745-4329-9d13-9de73304599a	2015-12-09 16:58:59.335479
536	317	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with IDH2 mutations such as R172K have event free survival\n  and overall survival similar to those with wild-type IDH2.\ndisease_id: 2\nsource_id: 197\nvariant_id: 63\nrating: 2\nstatus: accepted\nvariant_hgvs: 15:90631838-90631838 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	51314f34-b7f0-45cf-a9ee-5c9afdf40d5b	2015-12-09 16:58:59.379839
537	318	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients suffering from MDS, the presence of IDH2 mutations such as\n  R172K do not confer prognostic value (overall survival)\ndisease_id: 34\nsource_id: 199\nvariant_id: 63\nrating: 2\nstatus: accepted\nvariant_hgvs: 15:90631838-90631838 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6836806d-cffe-44fa-8f6e-7b366b153744	2015-12-09 16:58:59.435334
538	319	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML, patients with an IDH2 R172K mutation have worse overall survival\n  compared to those with wild-type IDH2\ndisease_id: 2\nsource_id: 201\nvariant_id: 63\nrating: 3\nstatus: accepted\nvariant_hgvs: 15:90631838-90631838 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fb3f107d-d26a-4a88-9d1b-192c75f50886	2015-12-09 16:58:59.48314
539	320	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutation is associated with larger, more invasive tumors, greater\n  pathologic histology and older patients compared to tumors with wildtype KIT\ndisease_id: 12\nsource_id: 202\nvariant_id: 66\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bff40531-bffd-4118-b5c2-2f0980ab0ac2	2015-12-09 16:58:59.536076
540	321	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutations were identified in morphologically benign, incidentally\n  discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients\n  indicating is not a prognostic marker for GIST\ndisease_id: 12\nsource_id: 203\nvariant_id: 66\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ef26e629-ed5a-4e36-afa9-d831523f2a57	2015-12-09 16:58:59.605668
541	322	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutations detected in 5/60 patients showed no prognostic significance\n  in patients with small cell lung cancer\ndisease_id: 26\nsource_id: 204\nvariant_id: 66\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cb37bc50-cfc5-4510-9a88-612c334b35bf	2015-12-09 16:58:59.673118
662	73	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55594258'\nstop:\n- \n- '55594258'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	b0500e1d-106c-45fe-9b36-73fa416b427f	2015-12-09 17:02:08.418665
542	323	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutation is associated with worse overall and cause-specific prognosis\n  in patients with GIST compared to patients with wildtype KIT\ndisease_id: 12\nsource_id: 202\nvariant_id: 66\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	960d8792-ce1e-4a1b-ae8b-693a0569650e	2015-12-09 16:58:59.720037
543	324	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutation is associated with larger, more invasive tumors, greater\n  pathologic histology and older patients compared to tumors with wildtype KIT\ndisease_id: 12\nsource_id: 202\nvariant_id: 70\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fc3a6ab1-170b-42de-9fe2-e01159570260	2015-12-09 16:58:59.769749
544	325	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutations detected in 5/60 patients showed no prognostic significance\n  in patients with small cell lung cancer\ndisease_id: 26\nsource_id: 204\nvariant_id: 71\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2ba28de7-c8f5-458b-bda5-0e26ef20351f	2015-12-09 16:58:59.824352
545	326	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a case study, a patient with anal melanoma harboring a 7 codon duplication\n  in the juxtamambrane region (exon 11) of KIT showed marked response 4 months after\n  imatinib treatment.\ndisease_id: 35\nsource_id: 205\nvariant_id: 67\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f22c4b23-33ba-4d9a-80af-205b2a3ba59b	2015-12-09 16:58:59.904234
546	327	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib,\n  nilotinib and imatinib.\ndisease_id: 1\nsource_id: 206\nvariant_id: 72\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	53239f3b-e898-4386-8923-0e30ca5a41b5	2015-12-09 16:58:59.970764
547	328	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In GIST tumor cells and patients harboring KIT V654A mutation, SU11248\n  is effective for those that are refractory to imatinib.\ndisease_id: 12\nsource_id: 207\nvariant_id: 73\nrating: 3\nstatus: accepted\nvariant_hgvs: 4:55594258-55594258 (T->C)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8d34c2ae-a510-4d28-9a3f-85f82c4dce3b	2015-12-09 16:59:00.035858
548	329	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT V654A results in imatinib resistance in GIST patient-derived cell\n  lines.\ndisease_id: 12\nsource_id: 67\nvariant_id: 73\nrating: 3\nstatus: accepted\nvariant_hgvs: 4:55594258-55594258 (T->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	96e78b7c-eaa4-4e74-9da3-bf331290910e	2015-12-09 16:59:00.093703
549	330	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer who harbor KRAS mutation have low response\n  rate to cetuximab\ndisease_id: 8\nsource_id: 208\nvariant_id: 74\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a2df6b58-66a0-4f16-9485-1d715032b701	2015-12-09 16:59:00.166226
550	331	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations in lung cancer patients are not associated with smoking\n  history, age or gender\ndisease_id: 1\nsource_id: 209\nvariant_id: 76\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8073ffcf-70e6-4aec-9932-b1e5cc5bf587	2015-12-09 16:59:00.230818
551	332	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with stage III colorectal cancer undergoing chemotherapy,\n  KRAS G12 mutation did not impact overall or disease free survival.\ndisease_id: 15\nsource_id: 210\nvariant_id: 76\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	46629829-dbb0-4913-a958-18b2aad892cb	2015-12-09 16:59:00.285803
552	333	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no strong association between KRAS mutation status and overall\n  or progression free survival in patients with NSCLC\ndisease_id: 1\nsource_id: 211\nvariant_id: 76\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f5be9283-3739-4ad5-920e-1e5ea7686c7a	2015-12-09 16:59:00.337151
553	334	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of KRAS mutations in MM patients is prognostic for shorter\n  overall and progression free survival\ndisease_id: 36\nsource_id: 212\nvariant_id: 76\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6083b999-01d4-4bde-beec-5ee81a4463d1	2015-12-09 16:59:00.38719
554	335	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A meta-analysis showed KRAS mutation is associated with worse outcome\n  in patients with NSCLC\ndisease_id: 1\nsource_id: 213\nvariant_id: 76\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4585a405-6164-42a9-abef-7d7d93cba315	2015-12-09 16:59:00.43302
555	336	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 37\nsource_id: 214\nvariant_id: 148\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	09dc6f85-a4cc-4975-93f0-72d4b151a557	2015-12-09 16:59:00.484141
556	337	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12C occur more frequently in women than men\ndisease_id: 26\nsource_id: 215\nvariant_id: 78\nrating: 2\nstatus: accepted\nvariant_hgvs: 12:25398285-25398285 (C->A)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4b1744b3-4d8d-4b85-a706-4f21de245f6d	2015-12-09 16:59:00.543634
557	338	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 37\nsource_id: 214\nvariant_id: 78\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	1a5d0ff5-eef9-42c9-92be-605ad6df6550	2015-12-09 16:59:00.584987
558	339	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12D mutation occurs in never smokers significantly more often than\n  in smokers\ndisease_id: 26\nsource_id: 215\nvariant_id: 79\nrating: 3\nstatus: accepted\nvariant_hgvs: 12:25398284-25398284 (C->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ea150ae7-f0c9-4a44-8401-60039d300abc	2015-12-09 16:59:00.656038
559	340	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\ndisease_id: 1\nsource_id: 216\nvariant_id: 79\nrating: 4\nstatus: accepted\nvariant_hgvs: 12:25398284-25398284 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	186afec3-1458-4074-8523-ff98331844f8	2015-12-09 16:59:00.713491
560	341	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 37\nsource_id: 214\nvariant_id: 79\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	ea737450-9ac4-48ec-970d-c047bb2490db	2015-12-09 16:59:00.766698
561	342	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 37\nsource_id: 214\nvariant_id: 147\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	9e95487a-c316-4dd0-a7d6-b6ff4fe446ff	2015-12-09 16:59:00.831393
562	343	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations in lung cancer patients are not associated with smoking\n  history, age or gender\ndisease_id: 1\nsource_id: 209\nvariant_id: 80\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	30e46e4a-e491-4093-aee2-979f40bd948f	2015-12-09 16:59:00.896578
563	344	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of KRAS mutations in MM patients is prognostic for shorter\n  overall and progression free survival\ndisease_id: 38\nsource_id: 212\nvariant_id: 80\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fe065fbf-3f06-481f-a4f1-d951d722e658	2015-12-09 16:59:00.949603
564	345	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G13D mutation is associated with better response to Cetuximab with\n  longer progression-free and overall survival in colorectal patients compared to\n  other KRAS\ndisease_id: 15\nsource_id: 123\nvariant_id: 81\nrating: 4\nstatus: accepted\nvariant_hgvs: 12:25398281-25398281 (C->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	04a8911d-8cac-42b8-b070-b873eb201129	2015-12-09 16:59:00.991097
565	346	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G13D mutation is associated with better response to Cetuximab with\n  longer progression-free and overall survival in colorectal patients compared to\n  other KRAS\ndisease_id: 15\nsource_id: 123\nvariant_id: 81\nrating: 4\nstatus: accepted\nvariant_hgvs: 12:25398281-25398281 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	33bd2493-faa9-4a5d-9f95-0d9f47ca9b0a	2015-12-09 16:59:01.037894
566	347	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804),\n  MET amplification and EGFR T790M were found to be the primary drivers of resistance.\ndisease_id: 1\nsource_id: 178\nvariant_id: 146\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	555956d9-999a-48c2-aa24-1fde3faf10e6	2015-12-09 16:59:01.080079
567	348	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a randomized clinical trial, patients with MGMT promoter methyaltion\n  benefitted from temozolomide. This benefit was also methylation status dependent,\n  as those without methylation did not see increased survival.\ndisease_id: 17\nsource_id: 217\nvariant_id: 85\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f2d1bdcf-7d4f-4011-8d39-fe56671829f8	2015-12-09 16:59:01.153544
568	349	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MGMT promoter methylation has been shown to confer sensitivity to alkylating\n  agents, such as carmustine, in glioblastoma cells.\ndisease_id: 17\nsource_id: 124\nvariant_id: 85\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	34d8bc17-29ea-4cf1-a9ba-9330246f763a	2015-12-09 16:59:01.205416
569	350	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor,\n  such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may\n  be useful in treating patients with initial resistance to alkylating agents.\ndisease_id: 17\nsource_id: 124\nvariant_id: 85\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	8d52eb7b-a202-499a-84f7-9d678e44586d	2015-12-09 16:59:01.250149
570	351	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including D1643H, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 1\nsource_id: 218\nvariant_id: 135\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	89dec167-0dae-41ad-ac93-d65b269ad689	2015-12-09 16:59:01.313081
571	352	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including R2328W, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 1\nsource_id: 218\nvariant_id: 136\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7cadc6d5-d103-4cea-8a34-9ae64064c570	2015-12-09 16:59:01.362468
572	353	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including S2275fs, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 1\nsource_id: 218\nvariant_id: 138\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	06dace06-1498-4a62-a54e-515aa07829b7	2015-12-09 16:59:01.407977
573	354	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including V2444fs, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 1\nsource_id: 218\nvariant_id: 137\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fa72d5bc-eae5-4309-89fc-f446597f8231	2015-12-09 16:59:01.475387
574	355	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer cell lines with E542K/E545K mutations showed increased\n  sensitivity to CH5132799 than cells with wild-type PIK3CA gene.\ndisease_id: 20\nsource_id: 219\nvariant_id: 103\nrating: 4\nstatus: accepted\nvariant_hgvs: 3:178936082-178936082 (G->A)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8b0bfd09-aa58-42dc-ae66-9e0f312d6991	2015-12-09 16:59:01.533848
575	356	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rapamycin inhibits transformation induced by mutation in PIK3CA\ndisease_id: 20\nsource_id: 220\nvariant_id: 103\nrating: 3\nstatus: accepted\nvariant_hgvs: 3:178936082-178936082 (G->A)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ebbda891-59fc-450c-bb8d-1ce2d6920178	2015-12-09 16:59:01.57901
576	357	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20\n  were associated with lack of panitumumab or cetuximab response.\ndisease_id: 8\nsource_id: 221\nvariant_id: 103\nrating: 3\nstatus: accepted\nvariant_hgvs: 3:178936082-178936082 (G->A)\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	691f8332-4367-47d2-a9b6-bad601a37755	2015-12-09 16:59:01.62371
577	358	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not\n  either alone, is prognostic for worse overall survival and cancer specific survival.\ndisease_id: 8\nsource_id: 222\nvariant_id: 103\nrating: 2\nstatus: accepted\nvariant_hgvs: 3:178936082-178936082 (G->A)\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	257945fe-32ac-40b8-8909-ffb29c1cfbb8	2015-12-09 16:59:01.673688
578	359	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of exon 9 (such as E542K) or exon 20 mutations were associated\n  with poorer survival compared to those with wild-type PIK3CA mutation.\ndisease_id: 8\nsource_id: 221\nvariant_id: 103\nrating: 3\nstatus: accepted\nvariant_hgvs: 3:178936082-178936082 (G->A)\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	93af755f-7e62-4809-a3bb-f001a05bad78	2015-12-09 16:59:01.712409
579	360	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer cell lines with E542K/E545K mutations showed increased\n  sensitivity to CH5132799 than cells with wild-type PIK3CA gene.\ndisease_id: 20\nsource_id: 219\nvariant_id: 104\nrating: 4\nstatus: accepted\nvariant_hgvs: 3:178936091-178936091 (G->A)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0c61013e-85df-42c8-99e6-9f466e69d011	2015-12-09 16:59:01.7587
580	361	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rapamycin inhibits transformation induced by mutation in PIK3CA\ndisease_id: 20\nsource_id: 220\nvariant_id: 104\nrating: 3\nstatus: accepted\nvariant_hgvs: 3:178936091-178936091 (G->A)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1e2e8a2b-4588-4e73-80fc-9d1f9d306512	2015-12-09 16:59:01.800356
581	362	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20\n  were associated with lack of panitumumab or cetuximab response.\ndisease_id: 8\nsource_id: 221\nvariant_id: 104\nrating: 3\nstatus: accepted\nvariant_hgvs: 3:178936091-178936091 (G->A)\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	31a1a363-9fd2-4255-82b3-52ba04a1e8f2	2015-12-09 16:59:01.848176
582	363	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not\n  either alone, is prognostic for worse overall survival and cancer specific survival.\ndisease_id: 8\nsource_id: 222\nvariant_id: 104\nrating: 2\nstatus: accepted\nvariant_hgvs: 3:178936091-178936091 (G->A)\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d8ba0fac-11a4-40b1-ba43-8a44aa98aa87	2015-12-09 16:59:01.896177
583	364	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of exon 9 (such as E545K) or exon 20 mutation may result\n  in poorer survival compared to those with wild-type PIK3CA mutation.\ndisease_id: 8\nsource_id: 221\nvariant_id: 104\nrating: 3\nstatus: accepted\nvariant_hgvs: 3:178936091-178936091 (G->A)\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b2275551-697a-4186-8706-24937112a801	2015-12-09 16:59:01.947599
584	365	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer cell lines with H1047R mutation showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\ndisease_id: 20\nsource_id: 219\nvariant_id: 107\nrating: 4\nstatus: accepted\nvariant_hgvs: 3:178952085-178952085 (A->G)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	14eaf588-6b9b-408f-886f-8842cc32bdb5	2015-12-09 16:59:01.994388
585	366	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rapamycin inhibits transformation induced by mutation in PIK3CA\ndisease_id: 20\nsource_id: 220\nvariant_id: 107\nrating: 3\nstatus: accepted\nvariant_hgvs: 3:178952085-178952085 (A->G)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9e304f91-7411-4b8f-94df-088a87d34a4a	2015-12-09 16:59:02.052089
586	367	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R)\n  were associated with lack of panitumumab or cetuximab response.\ndisease_id: 8\nsource_id: 221\nvariant_id: 107\nrating: 3\nstatus: accepted\nvariant_hgvs: 3:178952085-178952085 (A->G)\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	4d4d66d1-8dd2-421b-80a9-8bf4a251575f	2015-12-09 16:59:02.117013
587	368	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of exon 20 (such as H1047R) or exon 9 mutation may result\n  in poorer survival compared to those with wild-type PIK3CA mutation.\ndisease_id: 8\nsource_id: 221\nvariant_id: 107\nrating: 3\nstatus: accepted\nvariant_hgvs: 3:178952085-178952085 (A->G)\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cb791e0f-dee8-41d0-9411-fa07a5e8f5e7	2015-12-09 16:59:02.182981
588	369	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Fusion protein degradation is accelerated upon ATRA treatment of acute\n  promyelocytic cells at pharmalogical concentrations.\ndisease_id: 39\nsource_id: 223\nvariant_id: 108\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	68783ebc-6adb-4334-adc5-53637aacdd74	2015-12-09 16:59:02.271383
589	370	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PTEN nonsense mutations, including R233*, have been shown to be inactivating\n  and loss-of-function, but do not have prognostic value in glioblastoma multiforme\n  patients.\ndisease_id: 17\nsource_id: 224\nvariant_id: 110\nrating: 3\nstatus: accepted\nvariant_hgvs: 10:89717672-89717672 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	91fb543a-98dd-48b3-be1e-d56bbfd7fb22	2015-12-09 16:59:02.336561
590	15	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: PTEN Loss-of-Function\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	f2422309-c139-4e95-aba8-727d5195a496	2015-12-09 16:59:02.377312
591	371	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells with PTEN deficiency have been shown to exhibit slowed growth in\n  reponse to PI3K-mTOR inhibitors.\ndisease_id: 19\nsource_id: 225\nvariant_id: 110\nrating: 4\nstatus: accepted\nvariant_hgvs: 10:89717672-89717672 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	73a99828-d41c-4aed-99f6-57d614888088	2015-12-09 16:59:02.439671
592	372	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Fusion protein degradation is accelerated upon ATRA treatment of acute\n  promyelocytic cells at pharmalogical concentrations.\ndisease_id: 39\nsource_id: 223\nvariant_id: 111\nrating: 5\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	dc670059-7a38-4e1a-b59e-6d5a6b5b141b	2015-12-09 16:59:02.499437
593	373	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with SF3B1 mutations had a statistically significant longer\n  overall survival as well as event free-survival. Both before and after adjustment\n  for age, karyotype and sex.\ndisease_id: 34\nsource_id: 226\nvariant_id: 114\nrating: 3\nstatus: accepted\nvariant_hgvs: 2:198267359-198267359 (C->A)\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	40fa8099-bd61-44a1-b811-4d43bc68a350	2015-12-09 16:59:02.566646
594	374	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with SF3B1 mutations had a statistically significant longer\n  overall survival as well as event free-survival. Both before and after adjustment\n  for age, karyotype and sex.\ndisease_id: 34\nsource_id: 226\nvariant_id: 115\nrating: 3\nstatus: accepted\nvariant_hgvs: 2:198266834-198266834 (T->C)\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	843b5f70-b682-42af-8cab-4894518d5e7a	2015-12-09 16:59:02.621942
595	375	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R175H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 20\nsource_id: 227\nvariant_id: 116\nrating: 3\nstatus: accepted\nvariant_hgvs: 17:7578406-7578406 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7e697d64-5ed2-40bf-9207-b137bba4b23c	2015-12-09 16:59:02.684928
596	376	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast tumors in a mouse model with R175H mutation are more responsive\n  to doxorubicin than breast tumors with wild type TP53.\ndisease_id: 20\nsource_id: 228\nvariant_id: 116\nrating: 3\nstatus: accepted\nvariant_hgvs: 17:7578406-7578406 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b1738052-6a3a-44e3-98c7-49198d488578	2015-12-09 16:59:02.753941
597	377	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R248Q mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 20\nsource_id: 227\nvariant_id: 117\nrating: 3\nstatus: accepted\nvariant_hgvs: 17:7577538-7577538 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	af3b37b9-2f25-4876-8b53-9a4dea27d607	2015-12-09 16:59:02.804916
598	378	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R248 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\ndisease_id: 20\nsource_id: 229\nvariant_id: 117\nrating: 3\nstatus: accepted\nvariant_hgvs: 17:7577538-7577538 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fb269ed6-acc0-495c-b263-a377c1281128	2015-12-09 16:59:02.859007
599	379	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring R248W mutation, the prognosis is\n  worse than any other hotspot TP53 mutation, as well as worse than patients with\n  wild type TP53.\ndisease_id: 20\nsource_id: 227\nvariant_id: 118\nrating: 3\nstatus: accepted\nvariant_hgvs: 17:7577539-7577539 (G->A)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	91c7aed2-eba2-4881-b0cd-85e3a4890a4c	2015-12-09 16:59:02.904936
600	380	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in conserved\n  regions such as R249 are prognostic for a worse overall survival compared to those\n  harboring wild-type TP53.\ndisease_id: 20\nsource_id: 229\nvariant_id: 119\nrating: 3\nstatus: accepted\nvariant_hgvs: 17:7577535-7577535 (C->G)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0b389b68-3186-4292-8032-516c2d53a2d5	2015-12-09 16:59:02.948162
601	381	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in conserved\n  regions such as R249 are prognostic for a worse overall survival compared to those\n  harboring wild-type TP53.\ndisease_id: 20\nsource_id: 229\nvariant_id: 120\nrating: 3\nstatus: accepted\nvariant_hgvs: 17:7577536-7577536 (T->A)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f6e0e13d-180a-42a7-8b51-010d3e327160	2015-12-09 16:59:02.989923
602	382	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R273C mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 20\nsource_id: 227\nvariant_id: 121\nrating: 3\nstatus: accepted\nvariant_hgvs: 17:7577121-7577121 (G->A)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	06fdab24-c07d-491e-a9a2-6c0f900759de	2015-12-09 16:59:03.030747
603	383	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients  harboring TP53 mutation, mutations in DNA\n  contact regions such as R273 are prognostic for a worse relapse-free survival compared\n  to other non-silent mutations in TP53.\ndisease_id: 20\nsource_id: 229\nvariant_id: 121\nrating: 3\nstatus: accepted\nvariant_hgvs: 17:7577121-7577121 (G->A)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ff42df2f-adf3-45b3-923a-48a2137c2441	2015-12-09 16:59:03.091507
604	384	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R273H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 20\nsource_id: 227\nvariant_id: 122\nrating: 3\nstatus: accepted\nvariant_hgvs: 17:7577120-7577120 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d7ea139f-a025-48f4-af91-8a247e89574d	2015-12-09 16:59:03.12906
605	385	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\ndisease_id: 20\nsource_id: 229\nvariant_id: 122\nrating: 3\nstatus: accepted\nvariant_hgvs: 17:7577120-7577120 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	68bf447b-db6f-4a24-839d-224d592bcb39	2015-12-09 16:59:03.172243
606	386	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast tumors with R175H mutations are more responsive to doxorubicin\n  than breast tumors with wild type TP53.\ndisease_id: 20\nsource_id: 229\nvariant_id: 123\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	dc0c8c06-4a83-4822-87e6-f6c41a709084	2015-12-09 16:59:03.213819
607	387	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A patient with metastatic bladder cancer that responded well to the mTOR\n  inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del,\n  p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations\n  also had mild responses to everolimus.\ndisease_id: 40\nsource_id: 230\nvariant_id: 124\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	219b3995-f359-4fb7-9a10-215a168aa4df	2015-12-09 16:59:03.263075
608	388	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity\n  to everolimus than their wild-type counterparts.\ndisease_id: 16\nsource_id: 230\nvariant_id: 126\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e1fd4864-e179-4b7c-bb31-780f0f641858	2015-12-09 16:59:03.308078
609	389	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Age, sex, FAB subtype and karyotypes were not statistically significant\n  between AML patients with U2AF Q157P/R mutations and those who harbor wild type\n  U2AF.\ndisease_id: 2\nsource_id: 231\nvariant_id: 127\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2d21b185-5261-4733-92dc-33a501a61861	2015-12-09 16:59:03.36233
610	390	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with AML, those who harbor Q157P/R mutation of U2AF do not\n  show statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\ndisease_id: 2\nsource_id: 231\nvariant_id: 127\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	588298d9-6a47-494e-bf0f-3bb09f8fcd03	2015-12-09 16:59:03.400147
611	391	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with MDS, those who harbor Q157P/R mutation of U2AF do not\n  show statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\ndisease_id: 34\nsource_id: 231\nvariant_id: 127\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a28c3b93-21ac-49f8-8a08-6e7a3c04cbe7	2015-12-09 16:59:03.439186
612	392	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: After adjusting for age and cancer stage, presence of U2AF mutation such\n  as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.\ndisease_id: 34\nsource_id: 232\nvariant_id: 127\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a5b3212f-7f92-4809-99e2-6d95c98109d9	2015-12-09 16:59:03.491571
613	393	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Age, sex, FAB subtype and karyotypes were not statistically significant\n  between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.\ndisease_id: 2\nsource_id: 231\nvariant_id: 128\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a0a79a28-71bc-412b-8eb9-ded1fe9f6795	2015-12-09 16:59:03.534354
614	394	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with AML, complete remission rates are not different between\n  patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\ndisease_id: 2\nsource_id: 231\nvariant_id: 128\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	80738456-4799-4daf-9601-2659668afe2e	2015-12-09 16:59:03.57924
615	395	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with AML, complete remission rates are not different between\n  patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\ndisease_id: 34\nsource_id: 231\nvariant_id: 128\nrating: 2\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e8c691a4-58cf-4040-9561-cdc5a94e9815	2015-12-09 16:59:03.61547
616	396	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: After adjusting for age and cancer stage, the presence of U2AF mutations\n  such as S34Y/F are prognostic for poorer survival outcomes in patients with MDS.\ndisease_id: 34\nsource_id: 232\nvariant_id: 128\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	95d205c7-5f04-44b3-9828-521dcd950dc1	2015-12-09 16:59:03.652425
663	78	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '12'\nstart:\n- \n- '25398285'\nstop:\n- \n- '25398285'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- A\n	2	\N	\N	2b9605e8-3972-4fd8-a10d-a124f711f6a2	2015-12-09 17:02:08.442586
617	397	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\ndisease_id: 1\nsource_id: 233\nvariant_id: 79\nrating: 2\nstatus: accepted\nvariant_hgvs: 12:25398284-25398284 (C->T)\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	44cc3f1d-9978-4005-babf-1f0126770017	2015-12-09 16:59:03.700844
618	398	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase\n  inhibitors Imatinib and GW-2580\ndisease_id: 41\nsource_id: 234\nvariant_id: 30\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	f464c673-be50-48a7-ba01-ffe1498c4d78	2015-12-09 16:59:03.771559
619	399	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase\n  inhibitors Imatinib and GW-2580\ndisease_id: 41\nsource_id: 234\nvariant_id: 84\nrating: 3\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	43b060f6-fcd9-4a68-af3c-0b64ce28a465	2015-12-09 16:59:03.816983
620	400	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA\n  resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\ndisease_id: 42\nsource_id: 235\nvariant_id: 31\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5c9b3be4-e0b6-4282-9234-aa47ca81a941	2015-12-09 16:59:03.867205
621	401	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA\n  resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\ndisease_id: 42\nsource_id: 235\nvariant_id: 109\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	22d9f2bd-9bb8-4414-92c7-fe781f828f42	2015-12-09 16:59:03.915932
622	402	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NSCLC patients with exon 19 deletions in EGFR had longer overall survival\n  than patients with an L858R mutation when treated with gefitinib and erlotinib (38\n  versus 17 months; P = 0.04).\ndisease_id: 1\nsource_id: 236\nvariant_id: 133\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	870e5116-3db7-4ae7-847f-5bf040b3647d	2015-12-09 16:59:03.993382
623	403	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with Ewing's sarcoma, those with type 1 EWS-FLI1 fusions\n  had longer overall survival than those with other types of EWS-FLI1 fusions.\ndisease_id: 43\nsource_id: 237\nvariant_id: 151\nrating: 4\nstatus: accepted\nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	60da6b29-3511-4a76-80e7-c49812bd7faa	2015-12-09 16:59:04.060825
624	2	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'While imatinib has shown to be incredibly successful in treating philadelphia chromosome\n  positive CML, patients that have shown primary or secondary resistance to the drug\n  have been observed to harbor T315I and E255K ABL kinase domain mutations. These\n  mutations, among others, have been observed both in primary refractory disease and\n  acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations\n  have also been shown to confer resistance to imatinib. '\n	2	\N	\N	b35da7d2-7b29-454d-957b-c054c854a163	2015-12-09 16:59:04.181935
625	11	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset\n  of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase\n  the activity of the ALK oncogene relative to normal cells. While EML4 is the most\n  common fusion partner, other 5'' partners have been observed. The EML4-ALK fusion\n  has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. '\n	2	\N	\N	36f3271f-5e15-4a60-aadb-d9c10f7d6a94	2015-12-09 16:59:04.206823
626	12	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine\n  kinase inhibitors have been developed to target EGFR pathway activity. One such\n  inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting.\n  However, the T790M missense mutation has shown to confer resistance to this inhibitor\n  in cell lines and case studies.  '\n	2	\N	\N	7deac29b-fd1c-4109-b3f7-479683edebc8	2015-12-09 16:59:04.222301
627	3	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'The ALK oncogene has long been considered a driving factor in non-small cell lung\n  cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be\n  effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance\n  to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive\n  this resistance. '\n	2	\N	\N	724eac80-2bcf-4290-94d7-589944b046f8	2015-12-09 16:59:04.239944
628	5	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants\n  have also been observed and studied to a lesser degree. At first approximation,\n  many of these variants seem to behave similarly to V600E, and treatment with dabrafenib\n  has been shown to be effective. '\n	2	\N	\N	8251b288-dfe9-48f6-b713-8986e5bb502c	2015-12-09 16:59:04.252568
629	13	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'HER2-positive breast cancer is widely recognized as a molecular subtype of breast\n  cancer. While amplification of ERBB2 is a common mechanism for activation of the\n  pathway, missense mutations have also demonstrated activation potential. These activating\n  missense mutations also confer senstivity to the targeted therapeutic neratinib. '\n	2	\N	\N	0fcd9a2e-c9be-4d9c-9077-09ee2f0e0990	2015-12-09 16:59:04.265965
664	79	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '12'\nstart:\n- \n- '25398284'\nstop:\n- \n- '25398284'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	700d436b-9a3e-438d-9e9b-e50ee2c58067	2015-12-09 17:02:08.489387
665	81	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '12'\nstart:\n- \n- '25398281'\nstop:\n- \n- '25398281'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	b6b25d48-156c-4132-a30f-ee54fcb7ed8b	2015-12-09 17:02:08.511219
630	14	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone\n  therapy has been widely effective in treating the disease, however sequencing of\n  resistant patients has uncovered a number of missense mutations in mediating this\n  resistance. Many of these mutations lie within the ligand-binding domain, and contribute\n  to constitutive activity of the receptor. This has lead to the development of estrogen\n  receptor degrading agents such as fulvestrant, which have shown early efficacy in\n  clinical trials.  '\n	2	\N	\N	3b8a1dc1-1b76-4fb3-9c33-ab99f7f7a4bc	2015-12-09 16:59:04.278722
631	7	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'FGFR fusions have been demonstrated across many cancer types to have oncogenic\n  potential. FGFR2 and FGFR3 have been shown to be fused to many 3'' partners, which\n  may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2\n  fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib\n  and pazopanib have shown efficacy in case studies.  '\n	2	\N	\N	af9da7ff-82c0-4602-8e24-40538705c59f	2015-12-09 16:59:04.29153
632	15	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'PTEN loss is a common event in breast cancer, as well as others. The most common\n  single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell\n  lines harboring this mutation behave similarly to cell lines harboring larger insertion\n  or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded\n  growth. '\n	2	\N	\N	51ee566d-968a-4cd6-ba63-3882411d4f80	2015-12-09 16:59:04.306228
633	6	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment\n  of these patients with the targeted therapeutic motesanib has shown to be effective.\n  However, the missense mutations C634W and M918T have shown to confer motesanib resistance\n  in cell lines. '\n	2	\N	\N	fb034057-f84c-4be5-ac3e-9bb8de7f092f	2015-12-09 16:59:04.320432
634	9	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types.\n  Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor\n  everolimus has seen significant anti-tumorigenic activity. '\n	2	\N	\N	83d30ca9-e545-47b3-936d-ddf259c240f7	2015-12-09 16:59:04.337619
635	10	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'BRCA germline variants have long been recognized as important potential predictors\n  of breast and ovarian cancer risk. '\n	2	\N	\N	a5fab5fd-84ce-4e03-8a7c-36e839025b8a	2015-12-09 16:59:04.354995
636	4	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '14'\nstart:\n- \n- '105246551'\nstop:\n- \n- '105246551'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	cb96382d-d613-4ef8-b447-0da0089427fe	2015-12-09 17:02:07.713369
637	8	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '29443695'\nstop:\n- \n- '29443695'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	5348959a-c65a-4012-8de9-66cb1427815a	2015-12-09 17:02:07.737148
638	9	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '29432664'\nstop:\n- \n- '29432664'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	2bcbd728-f1c5-4f6d-8d04-88c788ccbb0e	2015-12-09 17:02:07.758025
639	10	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- X\nstart:\n- \n- '47426121'\nstop:\n- \n- '47426121'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- G\n	2	\N	\N	8e1aa378-077d-4720-a781-3d42b7c0c265	2015-12-09 17:02:07.779186
640	11	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453135'\nstop:\n- \n- '140453136'\nreference_bases:\n- \n- CA\nvariant_bases:\n- \n- AT\n	2	\N	\N	0062a545-15ac-4801-89c1-7f931574d960	2015-12-09 17:02:07.803165
641	12	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453136'\nstop:\n- \n- '140453136'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- T\n	2	\N	\N	85203dd3-44db-4ee6-ba38-47df524e60ca	2015-12-09 17:02:07.831204
642	15	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453137'\nstop:\n- \n- '140453137'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	856540d1-13cf-43f8-a70b-1a8b186d5d24	2015-12-09 17:02:07.865152
643	16	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453136'\nstop:\n- \n- '140453137'\nreference_bases:\n- \n- AC\nvariant_bases:\n- \n- CT\n	2	\N	\N	b968e9b0-9ed5-47fb-ab37-12cf36343458	2015-12-09 17:02:07.890611
644	33	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '55259514'\nstop:\n- \n- '55259515'\nreference_bases:\n- \n- CT\nvariant_bases:\n- \n- AG\n	2	\N	\N	8e17bed4-0fb9-4205-9e03-cb9f35c68458	2015-12-09 17:02:07.936266
645	34	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '55249071'\nstop:\n- \n- '55249071'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	7a0ae4d8-1327-4a2b-ad8e-0d0846b165fc	2015-12-09 17:02:07.969544
646	35	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37880261'\nstop:\n- \n- '37880261'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- C\n	2	\N	\N	53c65c30-6830-40d7-8f0b-c5e383bd70e5	2015-12-09 17:02:08.000589
647	36	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37880261'\nstop:\n- \n- '37880261'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	919a7518-356c-4222-97eb-e45359f4953e	2015-12-09 17:02:08.03912
648	38	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37868205'\nstop:\n- \n- '37868205'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- C\n	2	\N	\N	301b48eb-8cb8-4e36-9f2b-92d4440e3915	2015-12-09 17:02:08.074335
649	39	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37880220'\nstop:\n- \n- '37880220'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	7ddd5125-ab7c-472d-8c8a-c9c592edbe02	2015-12-09 17:02:08.103655
650	43	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37881616'\nstop:\n- \n- '37881616'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	f67269f5-5d63-45bc-abb6-0ea355952857	2015-12-09 17:02:08.127597
651	44	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37881000'\nstop:\n- \n- '37881000'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	6b71f82f-5bb0-4528-af09-9e93b7a0064f	2015-12-09 17:02:08.149921
652	45	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37881332'\nstop:\n- \n- '37881332'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	b55bd046-9354-4ad3-84bb-08d2c4c0b537	2015-12-09 17:02:08.171338
653	48	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '6'\nstart:\n- \n- '152419923'\nstop:\n- \n- '152419923'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- G\n	2	\N	\N	de7a6f21-37af-4be0-8834-65090d23fc9b	2015-12-09 17:02:08.193783
654	49	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '6'\nstart:\n- \n- '152419922'\nstop:\n- \n- '152419922'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- A\n	2	\N	\N	7591726f-be36-4c27-be86-a659a8540bb2	2015-12-09 17:02:08.215508
655	50	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '6'\nstart:\n- \n- '152419923'\nstop:\n- \n- '152419923'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- C\n	2	\N	\N	690bdb0a-f665-43bc-85c6-f6258bcdd503	2015-12-09 17:02:08.244593
656	59	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '209113113'\nstop:\n- \n- '209113113'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	ab17fb5e-efac-46a7-b977-62c8038b819b	2015-12-09 17:02:08.274253
657	60	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '209113112'\nstop:\n- \n- '209113112'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	de292670-30a0-4a33-9491-7fbaf7bffafe	2015-12-09 17:02:08.294249
683	116	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7578406'\nstop:\n- \n- '7578406'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	3394b8a5-68ee-403c-a1b9-6fec498e789a	2015-12-09 17:02:09.020171
684	117	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577538'\nstop:\n- \n- '7577538'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	9d1ac73c-782b-441b-a94f-e225b06b2730	2015-12-09 17:02:09.05425
685	118	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577539'\nstop:\n- \n- '7577539'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	0a1e8c92-9737-439f-ac80-a9c90dd1428a	2015-12-09 17:02:09.083048
686	119	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577535'\nstop:\n- \n- '7577535'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- G\n	2	\N	\N	925e3697-a581-4230-850b-c296ff78aa64	2015-12-09 17:02:09.111287
687	120	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577536'\nstop:\n- \n- '7577536'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- A\n	2	\N	\N	0a029b72-fe79-4cf1-a47e-9b9fcc15d52c	2015-12-09 17:02:09.130136
688	121	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577121'\nstop:\n- \n- '7577121'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	6b728a3c-5f7f-448d-835d-77e3e9bbb685	2015-12-09 17:02:09.14621
689	122	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577120'\nstop:\n- \n- '7577120'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	f9d12c90-51b5-4460-8234-32745d6db128	2015-12-09 17:02:09.163489
\.


--
-- Name: audits_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('audits_id_seq', 689, true);


--
-- Data for Name: authorizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY authorizations (id, user_id, provider, uid, created_at, updated_at) FROM stdin;
\.


--
-- Name: authorizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('authorizations_id_seq', 1, false);


--
-- Data for Name: comments; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY comments (id, title, comment, commentable_id, commentable_type, user_id, role, created_at, updated_at) FROM stdin;
\.


--
-- Name: comments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('comments_id_seq', 1, false);


--
-- Data for Name: data_versions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY data_versions (id, version) FROM stdin;
\.


--
-- Name: data_versions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('data_versions_id_seq', 1, false);


--
-- Data for Name: definitions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY definitions (id, term, text, created_at, updated_at) FROM stdin;
\.


--
-- Name: definitions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('definitions_id_seq', 1, false);


--
-- Data for Name: delayed_jobs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY delayed_jobs (id, priority, attempts, handler, last_error, run_at, locked_at, failed_at, locked_by, queue, created_at, updated_at) FROM stdin;
1	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1a56b65b-fccd-4740-9000-61e4c05b3ca6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/1\n  locale: :en\n	\N	2015-12-09 16:58:27.650622	\N	\N	\N	default	2015-12-09 16:58:27.65157	2015-12-09 16:58:27.65157
2	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1f8aa670-33d5-4e14-a90b-6428e6ad6199\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/2\n  locale: :en\n	\N	2015-12-09 16:58:27.666047	\N	\N	\N	default	2015-12-09 16:58:27.666179	2015-12-09 16:58:27.666179
3	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9ed0d797-c307-4482-b293-eb829308acc9\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/3\n  locale: :en\n	\N	2015-12-09 16:58:27.920616	\N	\N	\N	default	2015-12-09 16:58:27.920818	2015-12-09 16:58:27.920818
4	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7e5e8505-e4c7-423a-a7d7-2925e0055378\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/4\n  locale: :en\n	\N	2015-12-09 16:58:27.933504	\N	\N	\N	default	2015-12-09 16:58:27.933659	2015-12-09 16:58:27.933659
5	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 803e5cfe-94e8-41de-a018-0a6fb5a70577\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/5\n  locale: :en\n	\N	2015-12-09 16:58:28.174726	\N	\N	\N	default	2015-12-09 16:58:28.174869	2015-12-09 16:58:28.174869
6	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 92646534-0a52-4024-9993-07fd0702fb4d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/6\n  locale: :en\n	\N	2015-12-09 16:58:28.18735	\N	\N	\N	default	2015-12-09 16:58:28.187483	2015-12-09 16:58:28.187483
7	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 18dc6d48-c6a9-403c-9d11-e95d56837401\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/7\n  locale: :en\n	\N	2015-12-09 16:58:28.46378	\N	\N	\N	default	2015-12-09 16:58:28.463931	2015-12-09 16:58:28.463931
8	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b20c5578-6a95-42a1-bd7e-42a1a4c1cc02\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/8\n  locale: :en\n	\N	2015-12-09 16:58:28.474007	\N	\N	\N	default	2015-12-09 16:58:28.474175	2015-12-09 16:58:28.474175
9	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a1adaf26-ee57-4554-bc5e-bbabb7c45d21\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/9\n  locale: :en\n	\N	2015-12-09 16:58:28.706979	\N	\N	\N	default	2015-12-09 16:58:28.707127	2015-12-09 16:58:28.707127
10	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3103600e-e72e-4d86-a83b-b1634efe648e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/10\n  locale: :en\n	\N	2015-12-09 16:58:28.717096	\N	\N	\N	default	2015-12-09 16:58:28.717221	2015-12-09 16:58:28.717221
11	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a91e0bd8-4a0d-41d4-a953-f1839c1bcfb0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/11\n  locale: :en\n	\N	2015-12-09 16:58:28.969683	\N	\N	\N	default	2015-12-09 16:58:28.969837	2015-12-09 16:58:28.969837
12	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d9446655-9520-4c83-b9e0-812f5d63f66b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/12\n  locale: :en\n	\N	2015-12-09 16:58:29.510343	\N	\N	\N	default	2015-12-09 16:58:29.510487	2015-12-09 16:58:29.510487
13	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 549e61b0-cb1d-4532-a179-45767730b6ac\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/13\n  locale: :en\n	\N	2015-12-09 16:58:29.521267	\N	\N	\N	default	2015-12-09 16:58:29.521405	2015-12-09 16:58:29.521405
14	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2d6cb17f-7826-4168-bab1-62649d10e7d9\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/14\n  locale: :en\n	\N	2015-12-09 16:58:30.120347	\N	\N	\N	default	2015-12-09 16:58:30.120478	2015-12-09 16:58:30.120478
15	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 35a6af25-4ea5-4ae5-ab01-0c9dd73c0c98\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/15\n  locale: :en\n	\N	2015-12-09 16:58:30.326525	\N	\N	\N	default	2015-12-09 16:58:30.326739	2015-12-09 16:58:30.326739
16	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b89787e4-c50f-4940-b690-a0fb21d127ae\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/16\n  locale: :en\n	\N	2015-12-09 16:58:30.341655	\N	\N	\N	default	2015-12-09 16:58:30.341806	2015-12-09 16:58:30.341806
17	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: aca1bcba-21f6-47a4-af48-88d0751a2822\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/17\n  locale: :en\n	\N	2015-12-09 16:58:30.864417	\N	\N	\N	default	2015-12-09 16:58:30.864615	2015-12-09 16:58:30.864615
18	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 377b09d8-ef34-44db-9433-55aabb38c35f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/18\n  locale: :en\n	\N	2015-12-09 16:58:30.877551	\N	\N	\N	default	2015-12-09 16:58:30.877699	2015-12-09 16:58:30.877699
19	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c25d0ebf-8009-4317-994c-89a73c70229d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/19\n  locale: :en\n	\N	2015-12-09 16:58:31.12494	\N	\N	\N	default	2015-12-09 16:58:31.125068	2015-12-09 16:58:31.125068
20	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0f6e6287-c1a1-420a-815e-19528b76d1d0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/20\n  locale: :en\n	\N	2015-12-09 16:58:31.13442	\N	\N	\N	default	2015-12-09 16:58:31.134549	2015-12-09 16:58:31.134549
21	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f49ab240-73ac-4c8c-8a57-8e8e3deac22d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/21\n  locale: :en\n	\N	2015-12-09 16:58:31.14352	\N	\N	\N	default	2015-12-09 16:58:31.143676	2015-12-09 16:58:31.143676
22	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 17a61a1d-9787-49fc-8835-7519b6f39134\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/22\n  locale: :en\n	\N	2015-12-09 16:58:31.36614	\N	\N	\N	default	2015-12-09 16:58:31.366266	2015-12-09 16:58:31.366266
23	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9e63a3e0-8e79-4af0-8add-75126b090768\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/23\n  locale: :en\n	\N	2015-12-09 16:58:31.583881	\N	\N	\N	default	2015-12-09 16:58:31.584009	2015-12-09 16:58:31.584009
24	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7bc605ba-6d1c-4ee4-a0ea-f7c7492d6899\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/24\n  locale: :en\n	\N	2015-12-09 16:58:31.798783	\N	\N	\N	default	2015-12-09 16:58:31.798935	2015-12-09 16:58:31.798935
25	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 760218d1-8417-4dc9-8986-da679946267b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/25\n  locale: :en\n	\N	2015-12-09 16:58:31.808906	\N	\N	\N	default	2015-12-09 16:58:31.809027	2015-12-09 16:58:31.809027
26	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6c171fd4-9ab2-4234-b6b8-278642923e7d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/26\n  locale: :en\n	\N	2015-12-09 16:58:32.056732	\N	\N	\N	default	2015-12-09 16:58:32.056861	2015-12-09 16:58:32.056861
27	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 85470309-df87-44c5-9eee-23f270ab6396\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/27\n  locale: :en\n	\N	2015-12-09 16:58:32.066447	\N	\N	\N	default	2015-12-09 16:58:32.066609	2015-12-09 16:58:32.066609
28	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: bf35161d-fcaf-48e4-a547-b11427f0d2e1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/28\n  locale: :en\n	\N	2015-12-09 16:58:32.327756	\N	\N	\N	default	2015-12-09 16:58:32.327954	2015-12-09 16:58:32.327954
29	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a241b5a0-51ba-448f-b030-906a952c765b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/29\n  locale: :en\n	\N	2015-12-09 16:58:32.342278	\N	\N	\N	default	2015-12-09 16:58:32.342429	2015-12-09 16:58:32.342429
30	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d8fe9546-811b-41cf-b2c6-21154b02aacf\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/30\n  locale: :en\n	\N	2015-12-09 16:58:32.602942	\N	\N	\N	default	2015-12-09 16:58:32.603074	2015-12-09 16:58:32.603074
31	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 713bdf4d-7b49-43b9-aa9b-f0b0d4a866a2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/31\n  locale: :en\n	\N	2015-12-09 16:58:32.612352	\N	\N	\N	default	2015-12-09 16:58:32.612478	2015-12-09 16:58:32.612478
32	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 379841e5-eb28-4901-8a74-bdbe5092e4a3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/32\n  locale: :en\n	\N	2015-12-09 16:58:32.879181	\N	\N	\N	default	2015-12-09 16:58:32.879357	2015-12-09 16:58:32.879357
33	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4133f48b-a4cb-4fd1-9cf3-546aacf114f0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/33\n  locale: :en\n	\N	2015-12-09 16:58:32.890903	\N	\N	\N	default	2015-12-09 16:58:32.891119	2015-12-09 16:58:32.891119
34	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4a46186a-ee8b-4e35-a86a-808ccd436b04\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/34\n  locale: :en\n	\N	2015-12-09 16:58:33.36493	\N	\N	\N	default	2015-12-09 16:58:33.365058	2015-12-09 16:58:33.365058
35	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2db11096-e0c2-4b44-8c79-71a131b9080c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/35\n  locale: :en\n	\N	2015-12-09 16:58:33.593669	\N	\N	\N	default	2015-12-09 16:58:33.593801	2015-12-09 16:58:33.593801
36	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b23788f1-376d-489b-980e-3445fee75a69\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/36\n  locale: :en\n	\N	2015-12-09 16:58:33.60326	\N	\N	\N	default	2015-12-09 16:58:33.603386	2015-12-09 16:58:33.603386
37	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f7fd6ac3-059b-41b7-a332-925e7a0b27c8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/37\n  locale: :en\n	\N	2015-12-09 16:58:33.85348	\N	\N	\N	default	2015-12-09 16:58:33.853632	2015-12-09 16:58:33.853632
38	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c1aba78b-521f-4928-9bc1-8edb0f891cf3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/38\n  locale: :en\n	\N	2015-12-09 16:58:33.863208	\N	\N	\N	default	2015-12-09 16:58:33.863334	2015-12-09 16:58:33.863334
39	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b61f5df9-c8eb-4968-aa77-7a64082f2909\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/39\n  locale: :en\n	\N	2015-12-09 16:58:34.113366	\N	\N	\N	default	2015-12-09 16:58:34.113529	2015-12-09 16:58:34.113529
40	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 79c5795e-4864-49a6-ad1b-fbecdf6c0538\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/40\n  locale: :en\n	\N	2015-12-09 16:58:34.347177	\N	\N	\N	default	2015-12-09 16:58:34.34732	2015-12-09 16:58:34.34732
41	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4987ef4c-03a3-4241-b826-ae006b8c255a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/41\n  locale: :en\n	\N	2015-12-09 16:58:34.563809	\N	\N	\N	default	2015-12-09 16:58:34.56394	2015-12-09 16:58:34.56394
42	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: bbfa518a-e55f-4cc7-b2de-275222ea134d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/42\n  locale: :en\n	\N	2015-12-09 16:58:34.945045	\N	\N	\N	default	2015-12-09 16:58:34.945196	2015-12-09 16:58:34.945196
43	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ca805cad-39ca-4b23-b150-80a5ea191650\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/43\n  locale: :en\n	\N	2015-12-09 16:58:35.993817	\N	\N	\N	default	2015-12-09 16:58:35.993949	2015-12-09 16:58:35.993949
44	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 04c5d6bc-5dc7-44fc-a6ad-b4078c8810b9\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/44\n  locale: :en\n	\N	2015-12-09 16:58:37.597055	\N	\N	\N	default	2015-12-09 16:58:37.597187	2015-12-09 16:58:37.597187
45	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5c374a96-25a5-45c0-8b4e-3e3628d43e3e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/45\n  locale: :en\n	\N	2015-12-09 16:58:37.60695	\N	\N	\N	default	2015-12-09 16:58:37.607172	2015-12-09 16:58:37.607172
46	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b8688d9a-3e98-47c1-8a8a-c4df457582a1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/46\n  locale: :en\n	\N	2015-12-09 16:58:38.092299	\N	\N	\N	default	2015-12-09 16:58:38.092457	2015-12-09 16:58:38.092457
47	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a9377ac4-8be1-4d15-b407-bef8bf1c31e5\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/47\n  locale: :en\n	\N	2015-12-09 16:58:38.101692	\N	\N	\N	default	2015-12-09 16:58:38.101897	2015-12-09 16:58:38.101897
48	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 78360e39-8a1a-4704-b918-c5fdf445c2f7\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/48\n  locale: :en\n	\N	2015-12-09 16:58:38.333573	\N	\N	\N	default	2015-12-09 16:58:38.333701	2015-12-09 16:58:38.333701
49	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b44d119b-4dd9-4f2e-bb1f-d2d315fa4d56\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/49\n  locale: :en\n	\N	2015-12-09 16:58:38.958559	\N	\N	\N	default	2015-12-09 16:58:38.958755	2015-12-09 16:58:38.958755
50	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f8f8a341-4aa5-44a5-ad7b-cd5028d5d999\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/50\n  locale: :en\n	\N	2015-12-09 16:58:38.976785	\N	\N	\N	default	2015-12-09 16:58:38.977049	2015-12-09 16:58:38.977049
51	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 78e30f3a-2538-4dd4-9899-4260cbc74518\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/51\n  locale: :en\n	\N	2015-12-09 16:58:44.269956	\N	\N	\N	default	2015-12-09 16:58:44.270102	2015-12-09 16:58:44.270102
52	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: aef15845-78d5-4559-9f32-714eb6fc510c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/52\n  locale: :en\n	\N	2015-12-09 16:58:44.362436	\N	\N	\N	default	2015-12-09 16:58:44.362581	2015-12-09 16:58:44.362581
53	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 920ef5c0-623e-4cff-b8df-e4983e6e9fe1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/53\n  locale: :en\n	\N	2015-12-09 16:58:44.407889	\N	\N	\N	default	2015-12-09 16:58:44.40811	2015-12-09 16:58:44.40811
54	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d437c141-333e-4dba-805b-92b0b89d6798\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/54\n  locale: :en\n	\N	2015-12-09 16:58:44.574155	\N	\N	\N	default	2015-12-09 16:58:44.574284	2015-12-09 16:58:44.574284
55	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9f9ba171-c604-4612-8d88-7859b38070b3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/55\n  locale: :en\n	\N	2015-12-09 16:58:44.619371	\N	\N	\N	default	2015-12-09 16:58:44.619584	2015-12-09 16:58:44.619584
56	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 309f277a-4094-4e64-8071-55538bf7ccf3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/56\n  locale: :en\n	\N	2015-12-09 16:58:44.763445	\N	\N	\N	default	2015-12-09 16:58:44.763572	2015-12-09 16:58:44.763572
57	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9f4ab2b8-3d80-49f9-998b-bd3fe19bb34c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/57\n  locale: :en\n	\N	2015-12-09 16:58:44.810946	\N	\N	\N	default	2015-12-09 16:58:44.811075	2015-12-09 16:58:44.811075
58	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c798d760-0466-4ec2-9cb1-c52a5aa464f0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/58\n  locale: :en\n	\N	2015-12-09 16:58:44.903244	\N	\N	\N	default	2015-12-09 16:58:44.903375	2015-12-09 16:58:44.903375
59	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e19c1aec-23cf-45a4-a20e-27b2eb1b8ca7\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/59\n  locale: :en\n	\N	2015-12-09 16:58:44.956626	\N	\N	\N	default	2015-12-09 16:58:44.95679	2015-12-09 16:58:44.95679
60	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 57134c55-79ab-4582-8629-6e4e349adb1e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/60\n  locale: :en\n	\N	2015-12-09 16:58:45.014158	\N	\N	\N	default	2015-12-09 16:58:45.014296	2015-12-09 16:58:45.014296
61	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 09938812-c201-4d53-96ce-bf6c85aeabbd\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/61\n  locale: :en\n	\N	2015-12-09 16:58:45.136084	\N	\N	\N	default	2015-12-09 16:58:45.136228	2015-12-09 16:58:45.136228
62	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d958ca72-53d8-4c0e-b352-35359f40114a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/62\n  locale: :en\n	\N	2015-12-09 16:58:45.183994	\N	\N	\N	default	2015-12-09 16:58:45.184274	2015-12-09 16:58:45.184274
63	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b72308ba-a156-4228-bddd-5bd05de9b688\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/63\n  locale: :en\n	\N	2015-12-09 16:58:45.225346	\N	\N	\N	default	2015-12-09 16:58:45.225476	2015-12-09 16:58:45.225476
64	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 89a31f82-44cb-4c45-a540-5291e8b9e586\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/64\n  locale: :en\n	\N	2015-12-09 16:58:45.267625	\N	\N	\N	default	2015-12-09 16:58:45.267762	2015-12-09 16:58:45.267762
65	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0f7b1a3a-c9cf-40d6-b850-afdf0bdc1ef4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/65\n  locale: :en\n	\N	2015-12-09 16:58:45.361205	\N	\N	\N	default	2015-12-09 16:58:45.361333	2015-12-09 16:58:45.361333
66	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 098a9ce3-ad09-4ddf-8afb-c7184e01ef57\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/66\n  locale: :en\n	\N	2015-12-09 16:58:45.424931	\N	\N	\N	default	2015-12-09 16:58:45.425059	2015-12-09 16:58:45.425059
67	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9a896451-926d-4173-a4c0-39cd7925fded\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/67\n  locale: :en\n	\N	2015-12-09 16:58:45.471791	\N	\N	\N	default	2015-12-09 16:58:45.471922	2015-12-09 16:58:45.471922
68	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d0cb01d7-ad75-424c-991f-9fc243d1cb3d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/68\n  locale: :en\n	\N	2015-12-09 16:58:45.535543	\N	\N	\N	default	2015-12-09 16:58:45.535701	2015-12-09 16:58:45.535701
69	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 54b3a11f-b2aa-4f48-bee4-26739c816f62\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/69\n  locale: :en\n	\N	2015-12-09 16:58:45.650394	\N	\N	\N	default	2015-12-09 16:58:45.650526	2015-12-09 16:58:45.650526
70	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d83377ff-812f-4eb2-922a-6541322e1739\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/70\n  locale: :en\n	\N	2015-12-09 16:58:45.702502	\N	\N	\N	default	2015-12-09 16:58:45.702663	2015-12-09 16:58:45.702663
71	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 65e6d68c-c5f0-4d0d-ab10-19a42eb76ee2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/71\n  locale: :en\n	\N	2015-12-09 16:58:45.746689	\N	\N	\N	default	2015-12-09 16:58:45.746818	2015-12-09 16:58:45.746818
72	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2b5b3d3b-6d7e-44a2-88b6-2fa1b960a7fd\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/72\n  locale: :en\n	\N	2015-12-09 16:58:45.913645	\N	\N	\N	default	2015-12-09 16:58:45.91379	2015-12-09 16:58:45.91379
73	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 408c9676-adcf-441e-907c-85d582995941\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/73\n  locale: :en\n	\N	2015-12-09 16:58:45.960569	\N	\N	\N	default	2015-12-09 16:58:45.960728	2015-12-09 16:58:45.960728
74	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 012e7ee0-1ddf-4af5-87f3-e5d9ba61289a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/74\n  locale: :en\n	\N	2015-12-09 16:58:46.022355	\N	\N	\N	default	2015-12-09 16:58:46.022506	2015-12-09 16:58:46.022506
75	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 624cf164-c5b8-4cfa-9e50-ce8cbf802c82\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/75\n  locale: :en\n	\N	2015-12-09 16:58:46.336097	\N	\N	\N	default	2015-12-09 16:58:46.336227	2015-12-09 16:58:46.336227
76	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 599e5e3b-dc1b-4cc9-bbdc-6dfe189c75f3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/76\n  locale: :en\n	\N	2015-12-09 16:58:46.51754	\N	\N	\N	default	2015-12-09 16:58:46.517668	2015-12-09 16:58:46.517668
77	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: be51f7f2-1e84-472d-ac7d-928e69cb2097\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/77\n  locale: :en\n	\N	2015-12-09 16:58:46.585552	\N	\N	\N	default	2015-12-09 16:58:46.58568	2015-12-09 16:58:46.58568
78	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a51e0083-b0df-44df-bbcf-1b8a079187e6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/78\n  locale: :en\n	\N	2015-12-09 16:58:46.797047	\N	\N	\N	default	2015-12-09 16:58:46.797177	2015-12-09 16:58:46.797177
79	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ec5c5b00-957e-4235-874d-151bf50f691b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/79\n  locale: :en\n	\N	2015-12-09 16:58:46.904058	\N	\N	\N	default	2015-12-09 16:58:46.904185	2015-12-09 16:58:46.904185
80	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: cedfc302-a765-4504-9eb9-2e80a5bae37c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/80\n  locale: :en\n	\N	2015-12-09 16:58:46.987851	\N	\N	\N	default	2015-12-09 16:58:46.988067	2015-12-09 16:58:46.988067
81	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7ccfbf3b-25a2-4d04-9745-aa47e6a215c2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/81\n  locale: :en\n	\N	2015-12-09 16:58:47.03286	\N	\N	\N	default	2015-12-09 16:58:47.033002	2015-12-09 16:58:47.033002
82	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0410afe3-6612-4e2d-8f8a-d06081c44a11\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/82\n  locale: :en\n	\N	2015-12-09 16:58:47.13504	\N	\N	\N	default	2015-12-09 16:58:47.135174	2015-12-09 16:58:47.135174
83	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 72736d69-a0a9-44a0-a55b-5ab1c8850b2c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/83\n  locale: :en\n	\N	2015-12-09 16:58:47.175375	\N	\N	\N	default	2015-12-09 16:58:47.175876	2015-12-09 16:58:47.175876
84	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c973c971-ca7c-4960-b13f-fdc22c3ae5ce\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/84\n  locale: :en\n	\N	2015-12-09 16:58:47.216688	\N	\N	\N	default	2015-12-09 16:58:47.216818	2015-12-09 16:58:47.216818
85	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7bf732f6-934d-4124-8c79-01355d6f0747\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/85\n  locale: :en\n	\N	2015-12-09 16:58:47.461115	\N	\N	\N	default	2015-12-09 16:58:47.461253	2015-12-09 16:58:47.461253
86	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6971cd16-7caf-4b3b-a122-244a85e1e881\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/86\n  locale: :en\n	\N	2015-12-09 16:58:47.529241	\N	\N	\N	default	2015-12-09 16:58:47.529372	2015-12-09 16:58:47.529372
87	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 85e26f31-7a71-4d1d-98e0-f45f8ac4e865\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/87\n  locale: :en\n	\N	2015-12-09 16:58:47.603139	\N	\N	\N	default	2015-12-09 16:58:47.603305	2015-12-09 16:58:47.603305
88	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b7026cad-a5dd-4063-89bd-e89ad1461c60\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/88\n  locale: :en\n	\N	2015-12-09 16:58:47.647464	\N	\N	\N	default	2015-12-09 16:58:47.647594	2015-12-09 16:58:47.647594
89	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: cec3eb55-b371-47f6-8f94-0f41e716910f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/89\n  locale: :en\n	\N	2015-12-09 16:58:47.729438	\N	\N	\N	default	2015-12-09 16:58:47.729649	2015-12-09 16:58:47.729649
90	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 04a4b4de-85e7-4801-a7b6-0e1549c0a998\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/90\n  locale: :en\n	\N	2015-12-09 16:58:47.780877	\N	\N	\N	default	2015-12-09 16:58:47.781008	2015-12-09 16:58:47.781008
91	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0a14beec-ef49-4eb7-a435-577bdd574e8c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/91\n  locale: :en\n	\N	2015-12-09 16:58:47.823926	\N	\N	\N	default	2015-12-09 16:58:47.824058	2015-12-09 16:58:47.824058
92	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 096f4833-09d9-4a0c-ae57-69be1c2c126d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/92\n  locale: :en\n	\N	2015-12-09 16:58:47.868984	\N	\N	\N	default	2015-12-09 16:58:47.869273	2015-12-09 16:58:47.869273
93	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 29783801-b10b-4b04-be84-bf1809c1ce75\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/93\n  locale: :en\n	\N	2015-12-09 16:58:47.921465	\N	\N	\N	default	2015-12-09 16:58:47.921624	2015-12-09 16:58:47.921624
94	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8ea4f318-fff1-4911-829d-30e4abfcfa2e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/94\n  locale: :en\n	\N	2015-12-09 16:58:47.983666	\N	\N	\N	default	2015-12-09 16:58:47.983798	2015-12-09 16:58:47.983798
95	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5b8e66c6-7828-4401-be7c-49bce7ea40d4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/95\n  locale: :en\n	\N	2015-12-09 16:58:48.035278	\N	\N	\N	default	2015-12-09 16:58:48.035437	2015-12-09 16:58:48.035437
96	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 379ba4ba-b70c-46af-8c43-061af4e09039\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/96\n  locale: :en\n	\N	2015-12-09 16:58:48.085659	\N	\N	\N	default	2015-12-09 16:58:48.085789	2015-12-09 16:58:48.085789
97	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f2f33358-343b-42c2-b39b-7831f1d73e68\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/97\n  locale: :en\n	\N	2015-12-09 16:58:48.127229	\N	\N	\N	default	2015-12-09 16:58:48.127355	2015-12-09 16:58:48.127355
98	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 37aa4ba1-24ec-408c-a76a-7a01cf346667\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/98\n  locale: :en\n	\N	2015-12-09 16:58:48.187993	\N	\N	\N	default	2015-12-09 16:58:48.188233	2015-12-09 16:58:48.188233
99	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ffddddda-1c6b-4983-82d8-3e85bebde98b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/99\n  locale: :en\n	\N	2015-12-09 16:58:48.254236	\N	\N	\N	default	2015-12-09 16:58:48.254472	2015-12-09 16:58:48.254472
100	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 46ed30e6-100e-4505-adfb-945e0cb93dad\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/100\n  locale: :en\n	\N	2015-12-09 16:58:48.305337	\N	\N	\N	default	2015-12-09 16:58:48.30549	2015-12-09 16:58:48.30549
101	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c7f79479-41bc-4dfd-bb6d-915bac8d550c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/101\n  locale: :en\n	\N	2015-12-09 16:58:48.383594	\N	\N	\N	default	2015-12-09 16:58:48.383723	2015-12-09 16:58:48.383723
102	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2975ed6d-2584-4a03-8068-84acb0c66bb4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/102\n  locale: :en\n	\N	2015-12-09 16:58:48.459149	\N	\N	\N	default	2015-12-09 16:58:48.459276	2015-12-09 16:58:48.459276
103	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 74a2d97f-2f9a-4cfa-b5bd-07b055cec079\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/103\n  locale: :en\n	\N	2015-12-09 16:58:48.498802	\N	\N	\N	default	2015-12-09 16:58:48.498943	2015-12-09 16:58:48.498943
104	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f8ac0d24-308c-4c50-ba0e-dee112a858b4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/104\n  locale: :en\n	\N	2015-12-09 16:58:48.574287	\N	\N	\N	default	2015-12-09 16:58:48.574426	2015-12-09 16:58:48.574426
105	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 881590ac-6593-4b49-b48e-ef6018bfb3eb\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/105\n  locale: :en\n	\N	2015-12-09 16:58:48.643345	\N	\N	\N	default	2015-12-09 16:58:48.643545	2015-12-09 16:58:48.643545
106	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c5095706-e79d-4276-931e-d57c037b99ee\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/106\n  locale: :en\n	\N	2015-12-09 16:58:48.695087	\N	\N	\N	default	2015-12-09 16:58:48.695215	2015-12-09 16:58:48.695215
107	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 20de1183-718d-4543-98e7-bc6c5802dc52\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/107\n  locale: :en\n	\N	2015-12-09 16:58:48.783479	\N	\N	\N	default	2015-12-09 16:58:48.7837	2015-12-09 16:58:48.7837
108	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7e564bc7-7def-427d-b8e3-2d9efcc5a1f5\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/108\n  locale: :en\n	\N	2015-12-09 16:58:48.823473	\N	\N	\N	default	2015-12-09 16:58:48.823602	2015-12-09 16:58:48.823602
109	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 75b90b2a-acdb-45ae-b54c-e9b8762c9c89\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/109\n  locale: :en\n	\N	2015-12-09 16:58:48.958365	\N	\N	\N	default	2015-12-09 16:58:48.9585	2015-12-09 16:58:48.9585
110	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 391965df-d798-45a4-8437-31e2395ecaf5\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/110\n  locale: :en\n	\N	2015-12-09 16:58:49.275785	\N	\N	\N	default	2015-12-09 16:58:49.276314	2015-12-09 16:58:49.276314
111	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 90759c9d-d651-4fe1-9afd-c7259509ebab\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/111\n  locale: :en\n	\N	2015-12-09 16:58:49.563278	\N	\N	\N	default	2015-12-09 16:58:49.56344	2015-12-09 16:58:49.56344
112	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c25fae56-d6fe-4aa8-a3a1-12d932836d06\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/112\n  locale: :en\n	\N	2015-12-09 16:58:49.674818	\N	\N	\N	default	2015-12-09 16:58:49.675062	2015-12-09 16:58:49.675062
113	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 34683f5c-7948-4813-af7d-44039aa7d09f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/113\n  locale: :en\n	\N	2015-12-09 16:58:49.740319	\N	\N	\N	default	2015-12-09 16:58:49.740504	2015-12-09 16:58:49.740504
114	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3ee62b32-d909-4981-87bb-56ecce8b66b1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/114\n  locale: :en\n	\N	2015-12-09 16:58:50.33826	\N	\N	\N	default	2015-12-09 16:58:50.338404	2015-12-09 16:58:50.338404
115	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ed6b3db3-04be-4646-a4c5-396b952ea038\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/115\n  locale: :en\n	\N	2015-12-09 16:58:50.406074	\N	\N	\N	default	2015-12-09 16:58:50.406203	2015-12-09 16:58:50.406203
116	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 71acac1e-b894-49c2-8219-555ac1184aa9\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/116\n  locale: :en\n	\N	2015-12-09 16:58:50.45399	\N	\N	\N	default	2015-12-09 16:58:50.454128	2015-12-09 16:58:50.454128
117	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 01ca90b9-5320-47ad-9422-a06a4ff551ab\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/117\n  locale: :en\n	\N	2015-12-09 16:58:50.503115	\N	\N	\N	default	2015-12-09 16:58:50.503245	2015-12-09 16:58:50.503245
118	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: adb31eea-b7b3-4225-8665-cbd9f87fb7ad\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/118\n  locale: :en\n	\N	2015-12-09 16:58:50.58138	\N	\N	\N	default	2015-12-09 16:58:50.581536	2015-12-09 16:58:50.581536
119	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b087f1b9-1f9a-425f-bdb0-4ca0232ff19a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/119\n  locale: :en\n	\N	2015-12-09 16:58:50.783887	\N	\N	\N	default	2015-12-09 16:58:50.784099	2015-12-09 16:58:50.784099
120	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 55d75b7f-7089-48f4-a831-34c506a32538\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/120\n  locale: :en\n	\N	2015-12-09 16:58:50.87198	\N	\N	\N	default	2015-12-09 16:58:50.872111	2015-12-09 16:58:50.872111
121	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1567a3ef-ab5d-4803-bb1a-bbd2de9be8a5\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/121\n  locale: :en\n	\N	2015-12-09 16:58:51.016698	\N	\N	\N	default	2015-12-09 16:58:51.016866	2015-12-09 16:58:51.016866
122	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b6c7962d-8e32-4daa-980a-d688b374e19f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/122\n  locale: :en\n	\N	2015-12-09 16:58:51.104725	\N	\N	\N	default	2015-12-09 16:58:51.104883	2015-12-09 16:58:51.104883
123	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: cc274cbb-14e4-45e9-a107-94510592525a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/123\n  locale: :en\n	\N	2015-12-09 16:58:51.186218	\N	\N	\N	default	2015-12-09 16:58:51.186404	2015-12-09 16:58:51.186404
124	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 121f797e-ebb0-4090-907c-75a00015b28d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/124\n  locale: :en\n	\N	2015-12-09 16:58:51.260428	\N	\N	\N	default	2015-12-09 16:58:51.26056	2015-12-09 16:58:51.26056
125	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f00b8892-41fd-4016-93fa-bf3d499b0de3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/125\n  locale: :en\n	\N	2015-12-09 16:58:51.310759	\N	\N	\N	default	2015-12-09 16:58:51.310926	2015-12-09 16:58:51.310926
126	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9d343d42-d733-4bde-ad75-2a589c65f81a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/126\n  locale: :en\n	\N	2015-12-09 16:58:51.356237	\N	\N	\N	default	2015-12-09 16:58:51.356367	2015-12-09 16:58:51.356367
127	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ca5944d9-a14a-4bbd-a2fe-480832a63e1a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/127\n  locale: :en\n	\N	2015-12-09 16:58:51.399283	\N	\N	\N	default	2015-12-09 16:58:51.399437	2015-12-09 16:58:51.399437
128	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 76637e18-1f6f-4800-9afa-0ef38177ed9b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/128\n  locale: :en\n	\N	2015-12-09 16:58:51.569259	\N	\N	\N	default	2015-12-09 16:58:51.569412	2015-12-09 16:58:51.569412
129	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 491fc68a-8a36-4a6c-bbb8-1bf0d0d2f187\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/129\n  locale: :en\n	\N	2015-12-09 16:58:51.688826	\N	\N	\N	default	2015-12-09 16:58:51.688982	2015-12-09 16:58:51.688982
130	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 76c51632-9055-4ff7-afc5-a6ec2334be6b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/130\n  locale: :en\n	\N	2015-12-09 16:58:51.73352	\N	\N	\N	default	2015-12-09 16:58:51.733672	2015-12-09 16:58:51.733672
131	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 46b79028-9d3b-4f8e-9ebf-c0f660762fcc\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/131\n  locale: :en\n	\N	2015-12-09 16:58:51.96364	\N	\N	\N	default	2015-12-09 16:58:51.963772	2015-12-09 16:58:51.963772
132	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 11dea525-49d0-4f35-b5f4-66b3422be4ea\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/132\n  locale: :en\n	\N	2015-12-09 16:58:52.106625	\N	\N	\N	default	2015-12-09 16:58:52.106783	2015-12-09 16:58:52.106783
133	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e5e01810-2512-4f75-b465-001dc43f4b6b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/133\n  locale: :en\n	\N	2015-12-09 16:58:52.159197	\N	\N	\N	default	2015-12-09 16:58:52.159439	2015-12-09 16:58:52.159439
134	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6fc1648d-2de2-46f2-8d6d-313caefed6a2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/134\n  locale: :en\n	\N	2015-12-09 16:58:52.212212	\N	\N	\N	default	2015-12-09 16:58:52.212413	2015-12-09 16:58:52.212413
135	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c9bb4525-fee3-4813-9ac6-1918565f922f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/135\n  locale: :en\n	\N	2015-12-09 16:58:52.332491	\N	\N	\N	default	2015-12-09 16:58:52.332676	2015-12-09 16:58:52.332676
136	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 65ff32bd-cffe-4c0f-846c-d8cc5f4f1bff\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/136\n  locale: :en\n	\N	2015-12-09 16:58:52.379707	\N	\N	\N	default	2015-12-09 16:58:52.379858	2015-12-09 16:58:52.379858
137	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9e44a774-0367-4696-ba19-31fa7e2d1b50\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/137\n  locale: :en\n	\N	2015-12-09 16:58:52.480305	\N	\N	\N	default	2015-12-09 16:58:52.480458	2015-12-09 16:58:52.480458
138	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: df0389d9-5548-401e-9c1b-d5208035d561\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/138\n  locale: :en\n	\N	2015-12-09 16:58:52.629283	\N	\N	\N	default	2015-12-09 16:58:52.629433	2015-12-09 16:58:52.629433
139	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4c03cc14-6c85-4841-8a28-3dd4f63daaa9\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/139\n  locale: :en\n	\N	2015-12-09 16:58:52.722086	\N	\N	\N	default	2015-12-09 16:58:52.722246	2015-12-09 16:58:52.722246
140	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 30097a69-8dd9-4ba2-abe6-cbc1a6693517\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/140\n  locale: :en\n	\N	2015-12-09 16:58:52.771928	\N	\N	\N	default	2015-12-09 16:58:52.772058	2015-12-09 16:58:52.772058
141	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5bdc7601-bf7e-42ff-9c91-4be4ddd84207\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/141\n  locale: :en\n	\N	2015-12-09 16:58:53.190659	\N	\N	\N	default	2015-12-09 16:58:53.190869	2015-12-09 16:58:53.190869
142	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9d1521dd-d685-4d7f-b193-2d3afa9100d1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/142\n  locale: :en\n	\N	2015-12-09 16:58:53.265756	\N	\N	\N	default	2015-12-09 16:58:53.265911	2015-12-09 16:58:53.265911
143	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 56d4bcee-6875-4831-83cb-38f927371f83\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/143\n  locale: :en\n	\N	2015-12-09 16:58:53.326394	\N	\N	\N	default	2015-12-09 16:58:53.326575	2015-12-09 16:58:53.326575
144	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e3134b73-6604-45d7-8297-7c56ff0deb15\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/144\n  locale: :en\n	\N	2015-12-09 16:58:53.373733	\N	\N	\N	default	2015-12-09 16:58:53.373882	2015-12-09 16:58:53.373882
145	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 48125099-bcfc-4e71-8881-52755dbd4133\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/145\n  locale: :en\n	\N	2015-12-09 16:58:53.460934	\N	\N	\N	default	2015-12-09 16:58:53.461088	2015-12-09 16:58:53.461088
146	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5cf0fd28-6e36-454c-83f2-6b99b3e47c49\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/146\n  locale: :en\n	\N	2015-12-09 16:58:53.58594	\N	\N	\N	default	2015-12-09 16:58:53.586093	2015-12-09 16:58:53.586093
147	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 144b3cec-7932-45d9-bc45-ec00ca96d214\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/147\n  locale: :en\n	\N	2015-12-09 16:58:54.047775	\N	\N	\N	default	2015-12-09 16:58:54.04795	2015-12-09 16:58:54.04795
148	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 05062546-0cdb-4317-973a-2ce332932373\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/148\n  locale: :en\n	\N	2015-12-09 16:58:54.178859	\N	\N	\N	default	2015-12-09 16:58:54.179113	2015-12-09 16:58:54.179113
149	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a5a05bab-4b42-445d-9fae-a1e7ac6e8f20\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/149\n  locale: :en\n	\N	2015-12-09 16:58:54.235334	\N	\N	\N	default	2015-12-09 16:58:54.235486	2015-12-09 16:58:54.235486
150	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ef144a87-470c-43d8-945b-27223b2bd4ca\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/150\n  locale: :en\n	\N	2015-12-09 16:58:54.465268	\N	\N	\N	default	2015-12-09 16:58:54.465496	2015-12-09 16:58:54.465496
151	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 89b0ca14-1169-48da-8e21-94a178aa5efd\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/151\n  locale: :en\n	\N	2015-12-09 16:58:54.567006	\N	\N	\N	default	2015-12-09 16:58:54.56721	2015-12-09 16:58:54.56721
152	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 85a25691-2db0-422c-8319-97216ba2b2a8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/152\n  locale: :en\n	\N	2015-12-09 16:58:54.717092	\N	\N	\N	default	2015-12-09 16:58:54.717244	2015-12-09 16:58:54.717244
153	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: fef08061-fdcf-4c86-ac52-d494f1465d8d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/153\n  locale: :en\n	\N	2015-12-09 16:58:54.765519	\N	\N	\N	default	2015-12-09 16:58:54.765667	2015-12-09 16:58:54.765667
154	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4917630d-8134-46d5-ad0a-24632d356ea1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/154\n  locale: :en\n	\N	2015-12-09 16:58:54.931113	\N	\N	\N	default	2015-12-09 16:58:54.931258	2015-12-09 16:58:54.931258
155	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 73cf5b23-1a5d-4515-9d3a-bc4141988456\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/155\n  locale: :en\n	\N	2015-12-09 16:58:55.056095	\N	\N	\N	default	2015-12-09 16:58:55.056238	2015-12-09 16:58:55.056238
156	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ab899a40-972d-49f7-8c55-c801c0c768b9\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/156\n  locale: :en\n	\N	2015-12-09 16:58:55.17402	\N	\N	\N	default	2015-12-09 16:58:55.17417	2015-12-09 16:58:55.17417
157	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e45a8965-1350-4a7e-a2a7-2df66f433600\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/157\n  locale: :en\n	\N	2015-12-09 16:58:55.221507	\N	\N	\N	default	2015-12-09 16:58:55.221648	2015-12-09 16:58:55.221648
158	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: aaf825dc-f033-4eca-93d7-0fe45d9e4fea\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/158\n  locale: :en\n	\N	2015-12-09 16:58:55.268263	\N	\N	\N	default	2015-12-09 16:58:55.268407	2015-12-09 16:58:55.268407
159	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e2c313c0-beb2-44fa-9f83-817db6c230b2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/159\n  locale: :en\n	\N	2015-12-09 16:58:55.433392	\N	\N	\N	default	2015-12-09 16:58:55.433519	2015-12-09 16:58:55.433519
160	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: da721dd0-c7cc-468e-aaf0-ee0b2509de6e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/160\n  locale: :en\n	\N	2015-12-09 16:58:55.477055	\N	\N	\N	default	2015-12-09 16:58:55.477278	2015-12-09 16:58:55.477278
161	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: fe3b2e7f-0635-4bbc-8443-36a07dc0d0c8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/161\n  locale: :en\n	\N	2015-12-09 16:58:55.5242	\N	\N	\N	default	2015-12-09 16:58:55.524333	2015-12-09 16:58:55.524333
162	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3da7becd-1921-42df-b6bc-9319fd427922\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/162\n  locale: :en\n	\N	2015-12-09 16:58:55.565976	\N	\N	\N	default	2015-12-09 16:58:55.566105	2015-12-09 16:58:55.566105
163	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6e93a8f0-eb0d-48dd-975f-dd7ba9475c2f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/163\n  locale: :en\n	\N	2015-12-09 16:58:55.69287	\N	\N	\N	default	2015-12-09 16:58:55.693073	2015-12-09 16:58:55.693073
164	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 13a9e20d-6bfe-4244-9abd-1dc48cacc3c0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/164\n  locale: :en\n	\N	2015-12-09 16:58:55.743056	\N	\N	\N	default	2015-12-09 16:58:55.743189	2015-12-09 16:58:55.743189
165	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a42bcbb9-11f1-4f5a-b3e5-d723056c026c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/165\n  locale: :en\n	\N	2015-12-09 16:58:55.896492	\N	\N	\N	default	2015-12-09 16:58:55.896744	2015-12-09 16:58:55.896744
166	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 207f8905-edca-464e-b8f9-090bed2b1285\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/166\n  locale: :en\n	\N	2015-12-09 16:58:55.984812	\N	\N	\N	default	2015-12-09 16:58:55.984947	2015-12-09 16:58:55.984947
167	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7e5cfc73-df50-4f15-80b7-387b29fe0ba6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/167\n  locale: :en\n	\N	2015-12-09 16:58:56.252112	\N	\N	\N	default	2015-12-09 16:58:56.252299	2015-12-09 16:58:56.252299
168	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 267a171a-f8e2-4c4a-808b-0e4f81cdf1bf\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/168\n  locale: :en\n	\N	2015-12-09 16:58:56.29637	\N	\N	\N	default	2015-12-09 16:58:56.296517	2015-12-09 16:58:56.296517
169	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d92469e7-56a8-48d8-a27d-5ba19e90e007\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/169\n  locale: :en\n	\N	2015-12-09 16:58:56.345976	\N	\N	\N	default	2015-12-09 16:58:56.346203	2015-12-09 16:58:56.346203
170	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a628571d-6531-41a9-9079-5a77f52c04ec\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/170\n  locale: :en\n	\N	2015-12-09 16:58:56.416469	\N	\N	\N	default	2015-12-09 16:58:56.41661	2015-12-09 16:58:56.41661
171	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c5646dba-5f50-4908-978b-7838b63491fe\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/171\n  locale: :en\n	\N	2015-12-09 16:58:56.457289	\N	\N	\N	default	2015-12-09 16:58:56.457423	2015-12-09 16:58:56.457423
172	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 96f91739-e553-4621-b02b-b5c5f219f266\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/172\n  locale: :en\n	\N	2015-12-09 16:58:56.557239	\N	\N	\N	default	2015-12-09 16:58:56.557369	2015-12-09 16:58:56.557369
173	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b8151d89-2fcf-4060-a517-96e7fedb732a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/173\n  locale: :en\n	\N	2015-12-09 16:58:56.631954	\N	\N	\N	default	2015-12-09 16:58:56.63208	2015-12-09 16:58:56.63208
174	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 26f9bb61-c47b-4892-87c2-45048fd368f4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/174\n  locale: :en\n	\N	2015-12-09 16:58:56.675139	\N	\N	\N	default	2015-12-09 16:58:56.675292	2015-12-09 16:58:56.675292
175	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c7685336-50fe-45c3-b49d-3630518e741c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/175\n  locale: :en\n	\N	2015-12-09 16:58:56.720333	\N	\N	\N	default	2015-12-09 16:58:56.720483	2015-12-09 16:58:56.720483
176	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e51f7677-b8f4-4878-8d1c-d64b168603e9\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/176\n  locale: :en\n	\N	2015-12-09 16:58:56.764909	\N	\N	\N	default	2015-12-09 16:58:56.765076	2015-12-09 16:58:56.765076
177	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 30f4dda9-8726-43ca-bc75-0f6f052ae9f2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/177\n  locale: :en\n	\N	2015-12-09 16:58:56.814512	\N	\N	\N	default	2015-12-09 16:58:56.814665	2015-12-09 16:58:56.814665
178	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8ed5657e-be24-43e3-b493-70e146989813\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/178\n  locale: :en\n	\N	2015-12-09 16:58:56.863646	\N	\N	\N	default	2015-12-09 16:58:56.863779	2015-12-09 16:58:56.863779
179	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b6c9a393-f90b-4c02-94fb-73788f882566\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/179\n  locale: :en\n	\N	2015-12-09 16:58:56.917589	\N	\N	\N	default	2015-12-09 16:58:56.917715	2015-12-09 16:58:56.917715
180	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: aa3dcbd6-c99c-4738-9efc-278ec656855f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/180\n  locale: :en\n	\N	2015-12-09 16:58:57.448066	\N	\N	\N	default	2015-12-09 16:58:57.448265	2015-12-09 16:58:57.448265
181	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e2390875-25eb-4698-8b37-e3d0ab6c0e39\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/181\n  locale: :en\n	\N	2015-12-09 16:58:57.514134	\N	\N	\N	default	2015-12-09 16:58:57.514262	2015-12-09 16:58:57.514262
182	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a7b2f1ff-3fa9-46e6-a778-92b5bac3852a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/182\n  locale: :en\n	\N	2015-12-09 16:58:57.956162	\N	\N	\N	default	2015-12-09 16:58:57.956294	2015-12-09 16:58:57.956294
183	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 089f78dd-683e-4596-85a9-19c5650d26d6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/183\n  locale: :en\n	\N	2015-12-09 16:58:58.067642	\N	\N	\N	default	2015-12-09 16:58:58.067776	2015-12-09 16:58:58.067776
184	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 83218060-8962-44b5-90b9-e7e1ed811c79\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/184\n  locale: :en\n	\N	2015-12-09 16:58:58.114341	\N	\N	\N	default	2015-12-09 16:58:58.114494	2015-12-09 16:58:58.114494
185	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b175d268-812b-47f6-8a2c-5a042a16d22f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/185\n  locale: :en\n	\N	2015-12-09 16:58:58.159808	\N	\N	\N	default	2015-12-09 16:58:58.159936	2015-12-09 16:58:58.159936
186	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: af368b5a-a502-4ab8-9158-67ed54c0bc67\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/186\n  locale: :en\n	\N	2015-12-09 16:58:58.24298	\N	\N	\N	default	2015-12-09 16:58:58.243111	2015-12-09 16:58:58.243111
187	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 976eab36-7c35-40f4-8ac8-f48523da66db\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/187\n  locale: :en\n	\N	2015-12-09 16:58:58.287775	\N	\N	\N	default	2015-12-09 16:58:58.287904	2015-12-09 16:58:58.287904
188	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 39f6de93-9d1f-4611-bcc1-118b8b0aeb14\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/188\n  locale: :en\n	\N	2015-12-09 16:58:58.367501	\N	\N	\N	default	2015-12-09 16:58:58.367649	2015-12-09 16:58:58.367649
189	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 51384a73-2b83-4456-9bec-eb5edf8635ce\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/189\n  locale: :en\n	\N	2015-12-09 16:58:58.538684	\N	\N	\N	default	2015-12-09 16:58:58.538813	2015-12-09 16:58:58.538813
190	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6b0c5868-4516-4b31-84f2-f409224849a8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/190\n  locale: :en\n	\N	2015-12-09 16:58:58.584304	\N	\N	\N	default	2015-12-09 16:58:58.584433	2015-12-09 16:58:58.584433
191	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 69f8480b-78b6-4848-bddb-aef49dd231f0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/191\n  locale: :en\n	\N	2015-12-09 16:58:58.63567	\N	\N	\N	default	2015-12-09 16:58:58.635808	2015-12-09 16:58:58.635808
192	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7df2da10-3b09-4fe1-8833-5be1e60bf3f4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/192\n  locale: :en\n	\N	2015-12-09 16:58:58.67828	\N	\N	\N	default	2015-12-09 16:58:58.678463	2015-12-09 16:58:58.678463
193	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1c25ad4e-57a2-4b83-a327-94846db7c3e4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/193\n  locale: :en\n	\N	2015-12-09 16:58:58.727952	\N	\N	\N	default	2015-12-09 16:58:58.728088	2015-12-09 16:58:58.728088
194	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e4f146f8-5d1e-4102-86fa-69a56f8b6e7e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/194\n  locale: :en\n	\N	2015-12-09 16:58:58.770255	\N	\N	\N	default	2015-12-09 16:58:58.770385	2015-12-09 16:58:58.770385
195	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f86b7d81-8e57-4480-a08d-491d97d1386e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/195\n  locale: :en\n	\N	2015-12-09 16:58:58.818186	\N	\N	\N	default	2015-12-09 16:58:58.818333	2015-12-09 16:58:58.818333
196	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0dc1b724-47c7-42e7-b87a-1c7c1bb09e4b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/196\n  locale: :en\n	\N	2015-12-09 16:58:58.874347	\N	\N	\N	default	2015-12-09 16:58:58.874558	2015-12-09 16:58:58.874558
197	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8c89ad39-6b05-4980-815f-aae5a23581ba\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/197\n  locale: :en\n	\N	2015-12-09 16:58:59.093547	\N	\N	\N	default	2015-12-09 16:58:59.093706	2015-12-09 16:58:59.093706
198	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6188b65c-874d-430c-b251-37f8cb115723\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/198\n  locale: :en\n	\N	2015-12-09 16:58:59.148665	\N	\N	\N	default	2015-12-09 16:58:59.148839	2015-12-09 16:58:59.148839
199	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e5301e66-abd9-40a7-b992-0f92aa937f3a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/199\n  locale: :en\n	\N	2015-12-09 16:58:59.195654	\N	\N	\N	default	2015-12-09 16:58:59.195805	2015-12-09 16:58:59.195805
200	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 811c74a9-843a-4357-b987-6be0be04f59e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/200\n  locale: :en\n	\N	2015-12-09 16:58:59.250745	\N	\N	\N	default	2015-12-09 16:58:59.250993	2015-12-09 16:58:59.250993
201	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 20c8f5f5-2967-499f-8d7c-d5fac9e41b53\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/201\n  locale: :en\n	\N	2015-12-09 16:58:59.31836	\N	\N	\N	default	2015-12-09 16:58:59.318706	2015-12-09 16:58:59.318706
202	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8fefa2e7-6d74-4d88-999b-451cba57dda1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/202\n  locale: :en\n	\N	2015-12-09 16:58:59.520664	\N	\N	\N	default	2015-12-09 16:58:59.52088	2015-12-09 16:58:59.52088
203	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c9590c46-e7d0-4e4a-8227-e6cb10df8647\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/203\n  locale: :en\n	\N	2015-12-09 16:58:59.587771	\N	\N	\N	default	2015-12-09 16:58:59.587961	2015-12-09 16:58:59.587961
204	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d78d2957-da5d-42ce-887b-2efcf6ed1c6e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/204\n  locale: :en\n	\N	2015-12-09 16:58:59.657571	\N	\N	\N	default	2015-12-09 16:58:59.657892	2015-12-09 16:58:59.657892
205	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6d3bdbbc-b4f9-4395-9102-3ac8e22af8a0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/205\n  locale: :en\n	\N	2015-12-09 16:58:59.881012	\N	\N	\N	default	2015-12-09 16:58:59.881228	2015-12-09 16:58:59.881228
206	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e7e86ef2-8c36-465e-ba71-ce90fe6e9b43\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/206\n  locale: :en\n	\N	2015-12-09 16:58:59.95677	\N	\N	\N	default	2015-12-09 16:58:59.956974	2015-12-09 16:58:59.956974
207	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ba53a352-e3ae-4b27-b9c4-24f8a4b0d9cc\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/207\n  locale: :en\n	\N	2015-12-09 16:59:00.020651	\N	\N	\N	default	2015-12-09 16:59:00.020792	2015-12-09 16:59:00.020792
208	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1eca6ec6-e855-4911-bbfe-913db79e2bed\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/208\n  locale: :en\n	\N	2015-12-09 16:59:00.143503	\N	\N	\N	default	2015-12-09 16:59:00.14366	2015-12-09 16:59:00.14366
209	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8a66ebfd-3f24-469c-920c-eedb3ce7cc5c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/209\n  locale: :en\n	\N	2015-12-09 16:59:00.209686	\N	\N	\N	default	2015-12-09 16:59:00.209833	2015-12-09 16:59:00.209833
210	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6139e739-0b14-4ffa-8312-103aed24a854\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/210\n  locale: :en\n	\N	2015-12-09 16:59:00.269701	\N	\N	\N	default	2015-12-09 16:59:00.26986	2015-12-09 16:59:00.26986
211	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9b181511-2daf-4ea0-ba4e-fec34b76bbc9\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/211\n  locale: :en\n	\N	2015-12-09 16:59:00.321233	\N	\N	\N	default	2015-12-09 16:59:00.321386	2015-12-09 16:59:00.321386
212	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ee39d45a-d61d-4131-85de-1d4c32465d8f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/212\n  locale: :en\n	\N	2015-12-09 16:59:00.370929	\N	\N	\N	default	2015-12-09 16:59:00.371067	2015-12-09 16:59:00.371067
213	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 41359b20-c239-40da-88fc-d658ed42845a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/213\n  locale: :en\n	\N	2015-12-09 16:59:00.418036	\N	\N	\N	default	2015-12-09 16:59:00.418248	2015-12-09 16:59:00.418248
214	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: eb111989-afc7-49a6-85ee-29eb09ac4e24\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/214\n  locale: :en\n	\N	2015-12-09 16:59:00.466474	\N	\N	\N	default	2015-12-09 16:59:00.466639	2015-12-09 16:59:00.466639
215	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5898d2e2-e7f0-4aaf-83db-97459b12fe10\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/215\n  locale: :en\n	\N	2015-12-09 16:59:00.527045	\N	\N	\N	default	2015-12-09 16:59:00.527297	2015-12-09 16:59:00.527297
216	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 24c2d775-11a6-494c-9d53-a801bcfe7a68\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/216\n  locale: :en\n	\N	2015-12-09 16:59:00.693559	\N	\N	\N	default	2015-12-09 16:59:00.693712	2015-12-09 16:59:00.693712
217	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 76be23d6-ca4a-463f-be93-1519077df3b3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/217\n  locale: :en\n	\N	2015-12-09 16:59:01.131553	\N	\N	\N	default	2015-12-09 16:59:01.131795	2015-12-09 16:59:01.131795
218	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8c603de0-f8e4-43c2-a6ef-93604963a86c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/218\n  locale: :en\n	\N	2015-12-09 16:59:01.292867	\N	\N	\N	default	2015-12-09 16:59:01.293096	2015-12-09 16:59:01.293096
219	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0ab5ee3f-3a71-47c7-8498-3f0507ae3a92\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/219\n  locale: :en\n	\N	2015-12-09 16:59:01.517904	\N	\N	\N	default	2015-12-09 16:59:01.518149	2015-12-09 16:59:01.518149
220	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ee3600f5-7d83-48a7-a751-8933643fbba1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/220\n  locale: :en\n	\N	2015-12-09 16:59:01.565459	\N	\N	\N	default	2015-12-09 16:59:01.565598	2015-12-09 16:59:01.565598
221	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3e25044c-cce5-4366-9278-8bd6411f8a92\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/221\n  locale: :en\n	\N	2015-12-09 16:59:01.609706	\N	\N	\N	default	2015-12-09 16:59:01.6099	2015-12-09 16:59:01.6099
222	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0f129147-b290-47a9-b7da-cbe799f4b23a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/222\n  locale: :en\n	\N	2015-12-09 16:59:01.657772	\N	\N	\N	default	2015-12-09 16:59:01.658019	2015-12-09 16:59:01.658019
223	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d17066ca-2a3c-4d5d-8950-84df2d1e786e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/223\n  locale: :en\n	\N	2015-12-09 16:59:02.247475	\N	\N	\N	default	2015-12-09 16:59:02.247621	2015-12-09 16:59:02.247621
224	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: acf9985d-aa04-4d81-9d78-6c75f9c5a844\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/224\n  locale: :en\n	\N	2015-12-09 16:59:02.312262	\N	\N	\N	default	2015-12-09 16:59:02.312416	2015-12-09 16:59:02.312416
225	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e46e65b8-a61a-4f82-a666-1665e3bd820e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/225\n  locale: :en\n	\N	2015-12-09 16:59:02.425939	\N	\N	\N	default	2015-12-09 16:59:02.426091	2015-12-09 16:59:02.426091
226	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f16d3e55-41bb-4d04-a51e-5cdc745adff5\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/226\n  locale: :en\n	\N	2015-12-09 16:59:02.545951	\N	\N	\N	default	2015-12-09 16:59:02.546187	2015-12-09 16:59:02.546187
227	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6e27041e-82b9-4e6a-8cc5-7fa859f8726d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/227\n  locale: :en\n	\N	2015-12-09 16:59:02.668027	\N	\N	\N	default	2015-12-09 16:59:02.66824	2015-12-09 16:59:02.66824
228	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 30c64545-b842-4958-925a-f776d308e7b2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/228\n  locale: :en\n	\N	2015-12-09 16:59:02.730518	\N	\N	\N	default	2015-12-09 16:59:02.730755	2015-12-09 16:59:02.730755
229	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4be01bda-2500-40b2-9cff-897f5a83e04d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/229\n  locale: :en\n	\N	2015-12-09 16:59:02.842419	\N	\N	\N	default	2015-12-09 16:59:02.842586	2015-12-09 16:59:02.842586
230	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: acb5b584-18ec-47aa-975b-1a608cceef9d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/230\n  locale: :en\n	\N	2015-12-09 16:59:03.248663	\N	\N	\N	default	2015-12-09 16:59:03.248787	2015-12-09 16:59:03.248787
231	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a061c5be-933b-4dcb-9e12-5eda6038d77f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/231\n  locale: :en\n	\N	2015-12-09 16:59:03.347482	\N	\N	\N	default	2015-12-09 16:59:03.347609	2015-12-09 16:59:03.347609
232	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f1f88fc1-8cab-452b-a5af-9b06ed9687f1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/232\n  locale: :en\n	\N	2015-12-09 16:59:03.473395	\N	\N	\N	default	2015-12-09 16:59:03.473543	2015-12-09 16:59:03.473543
233	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f6f23b5f-288b-4e02-94f3-fcf8f88e9456\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/233\n  locale: :en\n	\N	2015-12-09 16:59:03.685496	\N	\N	\N	default	2015-12-09 16:59:03.685673	2015-12-09 16:59:03.685673
234	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2ad2eeb1-6fac-48da-87dd-0f4dee32c3b1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/234\n  locale: :en\n	\N	2015-12-09 16:59:03.754267	\N	\N	\N	default	2015-12-09 16:59:03.754427	2015-12-09 16:59:03.754427
235	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 78b100f5-d02f-4b06-ab33-4566fba15c82\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/235\n  locale: :en\n	\N	2015-12-09 16:59:03.852534	\N	\N	\N	default	2015-12-09 16:59:03.852687	2015-12-09 16:59:03.852687
236	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: edc3ffa0-5610-457d-85fb-692820f7d513\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/236\n  locale: :en\n	\N	2015-12-09 16:59:03.976178	\N	\N	\N	default	2015-12-09 16:59:03.976375	2015-12-09 16:59:03.976375
237	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 21ae17ee-a0c0-4e40-95a2-403cefae0ed1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/237\n  locale: :en\n	\N	2015-12-09 16:59:04.041348	\N	\N	\N	default	2015-12-09 16:59:04.041519	2015-12-09 16:59:04.041519
\.


--
-- Name: delayed_jobs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('delayed_jobs_id_seq', 237, true);


--
-- Data for Name: disease_aliases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY disease_aliases (id, name) FROM stdin;
\.


--
-- Data for Name: diseases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY diseases (id, doid, name, created_at, updated_at) FROM stdin;
7	1909	Melanoma	2015-12-09 16:58:44.906272	2015-12-09 16:58:44.906272
11	769	Neuroblastoma	2015-12-09 16:58:45.653261	2015-12-09 16:58:45.653261
19	1612	Breast Cancer	2015-12-09 16:58:54.720184	2015-12-09 16:58:54.720184
29	4947	Cholangiocarcinoma	2015-12-09 16:58:57.961381	2015-12-09 16:58:57.961381
31	3069	Astrocytoma	2015-12-09 16:58:58.730836	2015-12-09 16:58:58.730836
1	3908	Non-small Cell Lung Carcinoma	2015-12-09 16:58:44.272783	2015-12-09 17:01:43.577188
2	9119	Acute Myeloid Leukemia	2015-12-09 16:58:44.365337	2015-12-09 17:01:43.738343
3	10747	Lymphoid Leukemia	2015-12-09 16:58:44.622977	2015-12-09 17:01:43.884897
4	8552	Chronic Myeloid Leukemia	2015-12-09 16:58:44.682137	2015-12-09 17:01:44.042494
5	8997	Polycythemia Vera	2015-12-09 16:58:44.766567	2015-12-09 17:01:44.17533
6	9119	Acute Myeloid Leukemia	2015-12-09 16:58:44.813908	2015-12-09 17:01:44.334736
8	9256	Colorectal Cancer	2015-12-09 16:58:45.017304	2015-12-09 17:01:50.161531
9	9253	Gastrointestinal Stromal Tumor	2015-12-09 16:58:45.363723	2015-12-09 17:01:50.289866
10	0050905	Inflammatory Myofibroblastic Tumor	2015-12-09 16:58:45.539514	2015-12-09 17:01:50.415535
12	9253	Gastrointestinal Stromal Tumor	2015-12-09 16:58:46.520212	2015-12-09 17:01:50.797439
13	3973	Thyroid Medullary Carcinoma	2015-12-09 16:58:47.532167	2015-12-09 17:01:56.351904
14	1781	Thyroid Cancer	2015-12-09 16:58:47.987558	2015-12-09 17:01:56.517789
15	9256	Colorectal Cancer	2015-12-09 16:58:48.501564	2015-12-09 17:02:02.410869
16	4007	Bladder Carcinoma	2015-12-09 16:58:50.341134	2015-12-09 17:02:02.612358
17	3068	Glioblastoma Multiforme	2015-12-09 16:58:51.263187	2015-12-09 17:02:02.809275
18	2394	Ovarian Cancer	2015-12-09 16:58:54.051316	2015-12-09 17:02:03.048744
20	1612	Breast Cancer	2015-12-09 16:58:55.059276	2015-12-09 17:02:03.784907
21	5520	Head And Neck Squamous Cell Carcinoma	2015-12-09 16:58:55.176905	2015-12-09 17:02:03.922619
22	0050746	Mantle Cell Lymphoma	2015-12-09 16:58:55.224557	2015-12-09 17:02:04.043469
23	10534	Stomach Cancer	2015-12-09 16:58:55.271882	2015-12-09 17:02:04.291421
24	715	T-cell Leukemia	2015-12-09 16:58:55.479765	2015-12-09 17:02:04.436284
25	5635	Gastric Adenosquamous Carcinoma	2015-12-09 16:58:55.568712	2015-12-09 17:02:04.566679
26	1324	Lung Cancer	2015-12-09 16:58:55.648087	2015-12-09 17:02:04.808702
27	0060075	Estrogen-receptor Positive Breast Cancer	2015-12-09 16:58:55.696749	2015-12-09 17:02:04.936119
28	3908	Non-small Cell Lung Carcinoma	2015-12-09 16:58:56.561565	2015-12-09 17:02:05.120273
30	3909	Bronchogenic Lung Adenocarcinoma	2015-12-09 16:58:58.587215	2015-12-09 17:02:05.404698
32	3068	Glioblastoma Multiforme	2015-12-09 16:58:58.772832	2015-12-09 17:02:05.8488
33	0050908	Myelodysplastic Syndrome	2015-12-09 16:58:58.821385	2015-12-09 17:02:05.978267
34	0050908	Myelodysplastic Syndrome	2015-12-09 16:58:59.199025	2015-12-09 17:02:06.112566
35	14145	Malignant Anus Melanoma	2015-12-09 16:58:59.884074	2015-12-09 17:02:06.249479
36	9538	Multiple Myeloma	2015-12-09 16:59:00.373875	2015-12-09 17:02:06.429646
37	3910	Lung Adenocarcinoma	2015-12-09 16:59:00.47014	2015-12-09 17:02:06.626491
38	9119	Acute Myeloid Leukemia	2015-12-09 16:59:00.933248	2015-12-09 17:02:06.772384
39	9119	Acute Myeloid Leukemia	2015-12-09 16:59:02.251543	2015-12-09 17:02:06.91697
40	6477	Invasive Bladder Transitional Cell Carcinoma	2015-12-09 16:59:03.251529	2015-12-09 17:02:07.129123
41	9952	Acute Lymphocytic Leukemia	2015-12-09 16:59:03.757265	2015-12-09 17:02:07.291071
42	5015	Hepatocellular Fibrolamellar Carcinoma	2015-12-09 16:59:03.855664	2015-12-09 17:02:07.416811
43	3369	Peripheral Primitive Neuroectodermal Tumor	2015-12-09 16:59:04.045478	2015-12-09 17:02:07.587026
\.


--
-- Data for Name: disease_aliases_diseases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY disease_aliases_diseases (disease_alias_id, disease_id) FROM stdin;
\.


--
-- Name: disease_aliases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('disease_aliases_id_seq', 1, false);


--
-- Name: diseases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('diseases_id_seq', 43, true);


--
-- Data for Name: drugs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs (id, name, pubchem_id, created_at, updated_at) FROM stdin;
1	Sorafenib	\N	2015-12-09 16:58:44.3094	2015-12-09 16:58:44.3094
2	N/A	\N	2015-12-09 16:58:44.385067	2015-12-09 16:58:44.385067
3	Daunorubicin	\N	2015-12-09 16:58:44.482605	2015-12-09 16:58:44.482605
4	Idarubicin	\N	2015-12-09 16:58:44.518302	2015-12-09 16:58:44.518302
5	Pegylated IFN-α–2a	\N	2015-12-09 16:58:44.788162	2015-12-09 16:58:44.788162
6	AZD6244	\N	2015-12-09 16:58:44.927793	2015-12-09 16:58:44.927793
7	17-AAG	\N	2015-12-09 16:58:45.159562	2015-12-09 16:58:45.159562
8	Vemurafenib	\N	2015-12-09 16:58:45.246981	2015-12-09 16:58:45.246981
9	Temozolomide 	\N	2015-12-09 16:58:45.289491	2015-12-09 16:58:45.289491
10	Imatinib	\N	2015-12-09 16:58:45.445231	2015-12-09 16:58:45.445231
11	Crizotinib	\N	2015-12-09 16:58:45.563783	2015-12-09 16:58:45.563783
12	CH5424802	\N	2015-12-09 16:58:45.67895	2015-12-09 16:58:45.67895
13	TAE684	\N	2015-12-09 16:58:45.770968	2015-12-09 16:58:45.770968
14	Trametinib	\N	2015-12-09 16:58:45.885117	2015-12-09 16:58:45.885117
15	Nilotinib	\N	2015-12-09 16:58:46.828636	2015-12-09 16:58:46.828636
16	Dasatinib	\N	2015-12-09 16:58:46.875523	2015-12-09 16:58:46.875523
17	Gefitinib	\N	2015-12-09 16:58:46.925219	2015-12-09 16:58:46.925219
18	Erlotinib	\N	2015-12-09 16:58:46.931265	2015-12-09 16:58:46.931265
19	Cetuximab	\N	2015-12-09 16:58:47.19666	2015-12-09 16:58:47.19666
20	Crenolanib	\N	2015-12-09 16:58:47.237729	2015-12-09 16:58:47.237729
21	Dabrafenib	\N	2015-12-09 16:58:47.486717	2015-12-09 16:58:47.486717
22	Motesanib	\N	2015-12-09 16:58:47.55291	2015-12-09 16:58:47.55291
23	AZD1480	\N	2015-12-09 16:58:47.759932	2015-12-09 16:58:47.759932
24	Panitumumab	\N	2015-12-09 16:58:47.802639	2015-12-09 16:58:47.802639
25	Dacarbazine	\N	2015-12-09 16:58:48.148731	2015-12-09 16:58:48.148731
26	Temozolomide	\N	2015-12-09 16:58:48.154161	2015-12-09 16:58:48.154161
27	PD0325901	\N	2015-12-09 16:58:48.610862	2015-12-09 16:58:48.610862
28	PLX4720	\N	2015-12-09 16:58:48.618524	2015-12-09 16:58:48.618524
29	Nutlin-3	\N	2015-12-09 16:58:48.674092	2015-12-09 16:58:48.674092
30	Capecitabine	\N	2015-12-09 16:58:48.717453	2015-12-09 16:58:48.717453
31	Bevacizumab	\N	2015-12-09 16:58:48.725311	2015-12-09 16:58:48.725311
32	Regorafenib	\N	2015-12-09 16:58:48.981447	2015-12-09 16:58:48.981447
33	PD173074	\N	2015-12-09 16:58:50.361218	2015-12-09 16:58:50.361218
34	ATRA	\N	2015-12-09 16:58:50.479062	2015-12-09 16:58:50.479062
35	SU5614	\N	2015-12-09 16:58:50.84949	2015-12-09 16:58:50.84949
36	Valproic Acid	\N	2015-12-09 16:58:51.758209	2015-12-09 16:58:51.758209
37	NSC348884	\N	2015-12-09 16:58:52.747063	2015-12-09 16:58:52.747063
38	Induction Therapy	\N	2015-12-09 16:58:53.043495	2015-12-09 16:58:53.043495
39	Rapamycin	\N	2015-12-09 16:58:53.217957	2015-12-09 16:58:53.217957
40	Everolimus	\N	2015-12-09 16:58:53.290187	2015-12-09 16:58:53.290187
41	Olaparib	\N	2015-12-09 16:58:54.076902	2015-12-09 16:58:54.076902
42	Arsenic Trioxide	\N	2015-12-09 16:58:54.269521	2015-12-09 16:58:54.269521
43	MK-2206	\N	2015-12-09 16:58:54.743504	2015-12-09 16:58:54.743504
44	Palbociclib (PD-0332991)	\N	2015-12-09 16:58:55.499581	2015-12-09 16:58:55.499581
45	Palbociclib	\N	2015-12-09 16:58:55.718707	2015-12-09 16:58:55.718707
46	BYL719	\N	2015-12-09 16:58:55.768683	2015-12-09 16:58:55.768683
47	Premetrexed	\N	2015-12-09 16:58:56.791709	2015-12-09 16:58:56.791709
48	Stauroporine	\N	2015-12-09 16:58:56.839212	2015-12-09 16:58:56.839212
49	Dacomitinib	\N	2015-12-09 16:58:56.888894	2015-12-09 16:58:56.888894
50	Neratinib	\N	2015-12-09 16:58:56.940668	2015-12-09 16:58:56.940668
51	Lapatinib	\N	2015-12-09 16:58:57.140811	2015-12-09 16:58:57.140811
52	Hormone Therapy	\N	2015-12-09 16:58:57.469281	2015-12-09 16:58:57.469281
53	Tamoxifen	\N	2015-12-09 16:58:57.540566	2015-12-09 16:58:57.540566
54	Fulvestrant	\N	2015-12-09 16:58:57.545978	2015-12-09 16:58:57.545978
55	Ponatinib	\N	2015-12-09 16:58:57.987849	2015-12-09 16:58:57.987849
56	Pazopanib	\N	2015-12-09 16:58:58.03522	2015-12-09 16:58:58.03522
57	CEP701	\N	2015-12-09 16:58:58.138033	2015-12-09 16:58:58.138033
58	AG1296	\N	2015-12-09 16:58:58.311624	2015-12-09 16:58:58.311624
59	Bortezomib	\N	2015-12-09 16:58:58.609871	2015-12-09 16:58:58.609871
60	Fasudil	\N	2015-12-09 16:58:58.615213	2015-12-09 16:58:58.615213
61	SU11248	\N	2015-12-09 16:59:00.052306	2015-12-09 16:59:00.052306
62	NVP-BEZ235	\N	2015-12-09 16:59:00.725723	2015-12-09 16:59:00.725723
63	ARRY-142886	\N	2015-12-09 16:59:00.734122	2015-12-09 16:59:00.734122
64	Carmustine	\N	2015-12-09 16:59:01.216016	2015-12-09 16:59:01.216016
65	O(6)-benzylguanine	\N	2015-12-09 16:59:01.262005	2015-12-09 16:59:01.262005
66	CH5132799	\N	2015-12-09 16:59:01.544508	2015-12-09 16:59:01.544508
67	MTOR Inhibitors	\N	2015-12-09 16:59:02.451569	2015-12-09 16:59:02.451569
68	Doxorubicin	\N	2015-12-09 16:59:02.770584	2015-12-09 16:59:02.770584
69	GW-2580	\N	2015-12-09 16:59:03.785778	2015-12-09 16:59:03.785778
\.


--
-- Data for Name: genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes (id, entrez_id, name, description, official_name, created_at, updated_at, clinical_description, deleted, deleted_at) FROM stdin;
1	238	ALK	ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines.	anaplastic lymphoma receptor tyrosine kinase	2015-12-09 16:58:27.518572	2015-12-09 16:58:27.518572	\N	f	\N
2	207	AKT1	AKT1, also referred to as protein kinase B, is a known oncogene. AKT activation relies on the PI3K pathway, and is recognized as a critical node in the pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been shown to result in activation of the protein. Mutations in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors in vitro.	v-akt murine thymoma viral oncogene homolog 1	2015-12-09 16:58:27.895761	2015-12-09 16:58:27.895761	\N	f	\N
3	369	ARAF	ARAF has recently become increasingly considered for its oncogenic potential. Its potential as a target for informing clinical action was demonstrated by a single case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated with sorafenib, acheived near-complete clinical remission. This finding has brought new focus on ARAF as a marker that should be assayed for in cancer treatment.	A-Raf proto-oncogene, serine/threonine kinase	2015-12-09 16:58:28.152839	2015-12-09 16:58:28.152839	\N	f	\N
4	25	ABL1	ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein that has become a signature of chronic myeloid leukemia (CML). Cells harboring this fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook of the disease. However, additional mutations in ABL1 have been shown to confer resistance to imatinib. In these resistance cases, second-generation tyrosine kinase inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are currently undergoing clinical trials for treating acquired resistance in CML.	ABL proto-oncogene 1, non-receptor tyrosine kinase	2015-12-09 16:58:28.439668	2015-12-09 16:58:28.439668	\N	f	\N
5	673	BRAF	BRAF mutations are found to be recurrent in many cancer types. Of these, the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E has been determined to be an activating mutation, and cells that harbor it, along with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations have also been correlated with poor prognosis in many cancer types, although there is at least one study that questions this conclusion in papillary thyroid cancer.	B-Raf proto-oncogene, serine/threonine kinase	2015-12-09 16:58:28.687052	2015-12-09 16:58:28.687052	\N	f	\N
6	672	BRCA1	BRCA1 mutations in the germline have become a hallmark for hereditary breast and ovarian cancers. Variants that have been demonstrated to reduce the function of the protein have been shown to increase risk for these cancers, as well as prostate and pancreatic cancer. These findings have been the impetus for the increased popularity of genetic testing of healthy indivudals to assess risk.	breast cancer 1	2015-12-09 16:58:28.95095	2015-12-09 16:58:28.95095	\N	f	\N
7	675	BRCA2	BRCA2 mutations in the germline have become a hallmark for hereditary breast and ovarian cancers. Variants that have been demonstrated to reduce the function of the protein have been shown to increase risk for these cancers, as well as prostate and pancreatic cancer. These findings have been the impetus for the increased popularity of genetic testing of healthy indivudals to assess risk.	breast cancer 2	2015-12-09 16:58:29.234707	2015-12-09 16:58:29.234707	\N	f	\N
8	595	CCND1	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D1	2015-12-09 16:58:29.490606	2015-12-09 16:58:29.490606	\N	f	\N
9	894	CCND2	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D2	2015-12-09 16:58:29.710023	2015-12-09 16:58:29.710023	\N	f	\N
10	896	CCND3	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. In a mouse study, the targeted therapeutic palbociclib significantly increased the median survival of the cyclin D3 knockouts.	cyclin D3	2015-12-09 16:58:29.898572	2015-12-09 16:58:29.898572	\N	f	\N
11	898	CCNE1	Cyclin E, while currenly not as widely implicated as its cyclin D counterparts, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements are correlated with later stage disease. In lung cancer, neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly expressed counterparts.	cyclin E1	2015-12-09 16:58:30.10116	2015-12-09 16:58:30.10116	\N	f	\N
12	1021	CDK6	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 6	2015-12-09 16:58:30.308351	2015-12-09 16:58:30.308351	\N	f	\N
13	1019	CDK4	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 4	2015-12-09 16:58:30.53256	2015-12-09 16:58:30.53256	\N	f	\N
14	1029	CDKN2A	CDKN2A loss has been shown to be a significant event in a number of cancer types. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. One mechanism by which loss of CDKN2A can occur is by hypermethylation of the promoter region for the gene. However, the prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have suggesting poorer prognostic outcome for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	cyclin-dependent kinase inhibitor 2A	2015-12-09 16:58:30.837771	2015-12-09 16:58:30.837771	\N	f	\N
15	1050	CEBPA	AML with mutated CEBPA' is a provisional entity in the WHO classification of acute myeloid leukemia (AML) and is recommended to be tested for in patients with AML. CEBPA mutations are particularly associated with cytogenetically normal AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha protein while C-terminal mutations reduce the DNA-binding potential of this transcription factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	CCAAT/enhancer binding protein (C/EBP), alpha	2015-12-09 16:58:31.106259	2015-12-09 16:58:31.106259	\N	f	\N
16	3337	DNAJB1	DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	DnaJ heat shock protein family (Hsp40) member B1	2015-12-09 16:58:31.348056	2015-12-09 16:58:31.348056	\N	f	\N
17	5566	PRKACA	PRKACA has been studied in breast cancer and has been found to mediate resistance to HER2 targeted therapies. It has also been found to contain a mutation hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	protein kinase, cAMP-dependent, alpha catalytic subunit	2015-12-09 16:58:31.562538	2015-12-09 16:58:31.562538	\N	f	\N
18	1788	DNMT3A	DNMT3A is one of several epigenetic modifiers identified as recurrently mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers. AML patient bone marrow harboring R882 mutations were similarly demonstrated to be hypomethylated compared to patients with wildtype DNMT3A. These studies also indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes and treatment protocols may need to be considered for these two classes of DNMT3A mutations.	DNA (cytosine-5-)-methyltransferase 3 alpha	2015-12-09 16:58:31.775824	2015-12-09 16:58:31.775824	\N	f	\N
19	1956	EGFR	EGFR is widely recognized for its importance in cancer. Amplification and mutations have been shown to be driving events in many cancer types. Its role in non-small cell lung cancer has spurred many research and drug development efforts. Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance to these drugs, particularly the variant T790M, which has been functionally characterized as a resistance marker for both of these drugs. The later generation TKI's have seen some success in treating these resistant cases, and targeted sequencing of the EGFR locus has become a common practice in non-small cell lung cancer.	epidermal growth factor receptor	2015-12-09 16:58:32.037827	2015-12-09 16:58:32.037827	\N	f	\N
20	2064	ERBB2	ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast cancer, and is treated in a separate manner than the other subtypes of breast cancer. Apart from being amplified, ERBB2 activating mutations have been shown to have clinical importance in HER2-negative breast cancer. These mutations have shown sensitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical sequencing efforts in treating breast cancer.	erb-b2 receptor tyrosine kinase 2	2015-12-09 16:58:32.302098	2015-12-09 16:58:32.302098	\N	f	\N
21	2099	ESR1	ESR1 has been a focus in breast cancer for quite some time, but has also shown significance in endometrial, ovarian and other cancer types. ER-positive breast cancer that is resistant to hormone therapy has instigated clinical sequencing efforts to shed light on the mechanisms of this resistance. A number of mutations in the ligand binding domain of the protein have been implicated in hormone resistance and anti-estrogen therapies. This has spurred efforts to develop therapeutics that act to degrade the protein, rather than act as an antagonist. These agents are currently in clinical trials and have seen some success, highlighting the importance of sequencing efforts in treating breast cancer.	estrogen receptor 1	2015-12-09 16:58:32.584488	2015-12-09 16:58:32.584488	\N	f	\N
22	2263	FGFR2	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions that lead to pathway actiation have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 2	2015-12-09 16:58:32.859715	2015-12-09 16:58:32.859715	\N	f	\N
23	2261	FGFR3	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions that lead to pathway actiation have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 3	2015-12-09 16:58:33.119271	2015-12-09 16:58:33.119271	\N	f	\N
24	2322	FLT3	FLT3 is an important cytokine receptor involved in normal hematopoiesis. Mutations in this gene are common in acute myeloid leukemia (AML) and screening for mutations in this gene has been recommended by the World Health Organization in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML). FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD mutations are still up for debate.	fms-related tyrosine kinase 3	2015-12-09 16:58:33.345503	2015-12-09 16:58:33.345503	\N	f	\N
25	2624	GATA2	GATA2 is a transcription factor involved in stem cell maintenance with key roles in hematopoietic development. GATA2 mutations are associated with a variety of inherited and acquired immune disorders including myelodysplastic syndrome and acute myeloid leukemia. In addition to a role in hematopoiesis, the maintenance GATA2 expression has been implicated as a requirement in KRAS-driven non-small cell lung cancer. Preclinical models have indicated therapeutic benefit from targeting GATA2-mediated pathways in the context of KRAS-driven NSCLC.	GATA binding protein 2	2015-12-09 16:58:33.57448	2015-12-09 16:58:33.57448	\N	f	\N
26	3417	IDH1	IDH1 mutations have been observed in a number of cancer types, including sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described as mutually exclusive in many of these cancer types. The most frequent mutations involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic enzyme activity. The implications of mutations in this gene vary greatly by cancer type. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations have been associated with worse outcome, shorter overall survival, and normal karyotype. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Unlike the association with cytogenetically normal AML, in glioblastoma, IDH1 mutations have been associated with specific cytogenetic abnormalities, 1p and 19q deletions.	isocitrate dehydrogenase 1 (NADP+)	2015-12-09 16:58:33.825485	2015-12-09 16:58:33.825485	\N	f	\N
27	3418	IDH2	IDH2 mutations have been observed in a number of cancer types, including sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described as mutually exclusive in many of these cancer types. The most frequent mutations involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic enzyme activity. Although IDH2 (R172) mutations are associated with poorer overall prognosis in AML patients, its utility as a prognostic marker in MDS is still under debate. Additionally, IDH2 (R140) has been associated with improved overall survival in AML. IDH2 mutations have been associated with improved prognosis in gliomas.	isocitrate dehydrogenase 2 (NADP+), mitochondrial	2015-12-09 16:58:34.085352	2015-12-09 16:58:34.085352	\N	f	\N
28	3717	JAK2	JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative diseases and cancers. The mutation V617F is the most clinically relevant variant, and is seen in around half of myeloproliferative disorders. The variant is a known activating mutation, and activated JAK2 is sufficient to drive myeloproliferative disorders in mouse models. V617F, while most recurrent, is not the only mechanism by which JAK2 can be activated in patients. JAK2 is now one of the first diagnostic markers tested upon diagnosis with a myeloproliferative disorder.	Janus kinase 2	2015-12-09 16:58:34.317425	2015-12-09 16:58:34.317425	\N	f	\N
29	3815	KIT	c-KIT activation has been shown to have oncogenic activity in gastrointestinal stromal tumors (GISTs), melanomas, lung cancer, and other tumor types. The targeted therapeutics nilotinib and sunitinib have shown efficacy in treating KIT overactive patients, and are in late-stage trials in melanoma and GIST. KIT overactivity can be the result of many genomic events from genomic amplification to overexpression to missense mutations. Missense mutations have been shown to be key players in mediating clinical response and acquired resistance in patients being treated with these targeted therapeutics.	v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog	2015-12-09 16:58:34.545131	2015-12-09 16:58:34.545131	\N	f	\N
30	3845	KRAS	Mutations in the RAS family of proteins have frequently observed across cancer types. The amino acid positions account for the overwhelming majority of these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also behave very differently when expressed in non-native tissue types, likely due to differences in the C-terminal hyper-variable regions. Mis-regulation of isoform expression has been shown to be a driving event in cancer, as well as missense mutations at the three hotspots previously mentioned. While highly recurrent in cancer, targeted these RAS mutants has also been very elusive, and has not yet become common practice in the clinic.	Kirsten rat sarcoma viral oncogene homolog	2015-12-09 16:58:34.922198	2015-12-09 16:58:34.922198	\N	f	\N
31	5604	MAP2K1		mitogen-activated protein kinase kinase 1	2015-12-09 16:58:35.176782	2015-12-09 16:58:35.176782	\N	f	\N
32	4209	MEF2D		myocyte enhancer factor 2D	2015-12-09 16:58:35.357976	2015-12-09 16:58:35.357976	\N	f	\N
33	1436	CSF1R		colony stimulating factor 1 receptor	2015-12-09 16:58:35.58031	2015-12-09 16:58:35.58031	\N	f	\N
34	4255	MGMT		O-6-methylguanine-DNA methyltransferase	2015-12-09 16:58:35.772456	2015-12-09 16:58:35.772456	\N	f	\N
35	4869	NPM1	AML with mutated NPM1 is provisional entity in the WHO classification of AML and is recommended to be tested for in patients with cytogenetically normal AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia, indicating it may require additional mutations to promote leukemic development.	nucleophosmin (nucleolar phosphoprotein B23, numatrin)	2015-12-09 16:58:35.972801	2015-12-09 16:58:35.972801	\N	f	\N
36	4893	NRAS	Mutations in the RAS family of proteins have frequently observed across cancer types. The amino acid positions account for the overwhelming majority of these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also behave very differently when expressed in non-native tissue types, likely due to differences in the C-terminal hyper-variable regions. Mis-regulation of isoform expression has been shown to be a driving event in cancer, as well as missense mutations at the three hotspots previously mentioned. While highly recurrent in cancer, targeted these RAS mutants has also been very elusive, and has not yet become common practice in the clinic.	neuroblastoma RAS viral (v-ras) oncogene homolog	2015-12-09 16:58:36.201758	2015-12-09 16:58:36.201758	\N	f	\N
37	5290	PIK3CA	PIK3CA is the most recurrently mutated gene in breast cancer, and has been found to important in a number of cancer types. An integral part of the PI3K pathway, PIK3CA has long been described as an oncogene, with two main hotspots for activating mutations, the 542/545 region of the helical domain, and the 1047 region of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the subject of an immense amount of research and development, and PI3K inhibition has seen some limited success in recent clinical trials. While monotherapies seem to be limited in their potential, there is a recent interest in pursuing PI3K inhibition as part of a combination therapy regiment with inhibition partners including TKI's, MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.	phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha	2015-12-09 16:58:36.444944	2015-12-09 16:58:36.444944	\N	f	\N
38	5156	PDGFRA		platelet-derived growth factor receptor, alpha polypeptide	2015-12-09 16:58:36.666454	2015-12-09 16:58:36.666454	\N	f	\N
39	5371	PML	The PML-RARA fusion is the result of a recurrent, balanced translocation between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	promyelocytic leukemia	2015-12-09 16:58:36.878495	2015-12-09 16:58:36.878495	\N	f	\N
40	5914	RARA	The PML-RARA fusion is the result of a recurrent, balanced translocation between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	retinoic acid receptor, alpha	2015-12-09 16:58:37.087413	2015-12-09 16:58:37.087413	\N	f	\N
41	5728	PTEN		phosphatase and tensin homolog	2015-12-09 16:58:37.319135	2015-12-09 16:58:37.319135	\N	f	\N
42	5979	RET	RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine kinase receptor and are associated with thyroid cancers. RET point mutations are the most common mutations identified in medullary thyroid cancer (MTC) with germline and somatic mutations in RET associated with hereditary and sporadic forms, respectively. The most common somatic form mutation is M918T (exon 16) and a variety of other mutations effecting exons 10, 11 and 15 have been described. The prognostic significance of these mutations have been hotly debated in the field, however, data suggests that some RET mutation may confer drug resistence. No RET-specific agents are currently clinically available but several promiscuous kinase inhibitors that target RET, among others, have been approved for MTC treatment.	ret proto-oncogene	2015-12-09 16:58:37.577777	2015-12-09 16:58:37.577777	\N	f	\N
43	861	RUNX1		runt-related transcription factor 1	2015-12-09 16:58:37.844808	2015-12-09 16:58:37.844808	\N	f	\N
44	23451	SF3B1	SF3B1 mutations have been described in several myeloid malignancies, predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has been identified as recurrently mutated in MDS and other malignanices. The mutations affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally deleterious with K700E described as a major hotspot mutation. MDS patients with SF3B1 mutations have been reported to have better overall and event-free survival than their wildtype counterparts. Additionally, these mutations are highly associated with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring sideroblasts).	splicing factor 3b, subunit 1, 155kDa	2015-12-09 16:58:38.073491	2015-12-09 16:58:38.073491	\N	f	\N
45	7157	TP53	TP53 mutations are universal across cancer types. Loss of tumor suppressors is most recognized by large deleterious events, such as frameshift mutations, or premature stop codons. In TP53 however, many of the observed mutations in cancer are found to be single nucleotide variants, or missense mutations. These variants are also very broadly distributed throughout the gene, not localizing in any particular hotspot. While a large proportion of cancer genomics research is focused on somatic variants, TP53 is also of note in the germline. Germline TP53 mutations are the hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been found to have prognostic impact on patient outcomes.	tumor protein p53	2015-12-09 16:58:38.31186	2015-12-09 16:58:38.31186	\N	f	\N
46	7248	TSC1		tuberous sclerosis 1	2015-12-09 16:58:38.51978	2015-12-09 16:58:38.51978	\N	f	\N
47	7249	TSC2		tuberous sclerosis 2	2015-12-09 16:58:38.722601	2015-12-09 16:58:38.722601	\N	f	\N
48	7307	U2AF1	U2AF1 is one of several spliceosome complex genes frequently mutated in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein. These mutations have been associated with altered splicing patterns in vitro and in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation to secondary acute myeloid leukemia, however, the impact of these mutations on overall survival has been an area of debate.	U2 small nuclear RNA auxiliary factor 1	2015-12-09 16:58:38.938691	2015-12-09 16:58:38.938691	\N	f	\N
49	7490	WT1	WT1 is a tumor suppressor gene associated with the development of Wilms' Tumor, from which it was named. Mutations in exon 7 and 9 of WT1 have been recurrently identified in acute myeloid leukemia and associated with poorer prognosis and chemotherapy resistance.	Wilms tumor 1	2015-12-09 16:58:39.224089	2015-12-09 16:58:39.224089	\N	f	\N
50	4851	NOTCH1		notch 1	2015-12-09 16:58:39.437752	2015-12-09 16:58:39.437752	\N	f	\N
51	4921	DDR2		discoidin domain receptor tyrosine kinase 2	2015-12-09 16:58:39.67614	2015-12-09 16:58:39.67614	\N	f	\N
52	4233	MET		MET proto-oncogene, receptor tyrosine kinase	2015-12-09 16:58:39.891872	2015-12-09 16:58:39.891872	\N	f	\N
53	2313	FLI1		Fli-1 proto-oncogene, ETS transcription factor	2015-12-09 16:58:40.091125	2015-12-09 16:58:40.091125	\N	f	\N
54	2130	EWSR1		EWS RNA binding protein 1	2015-12-09 16:58:40.287001	2015-12-09 16:58:40.287001	\N	f	\N
\.


--
-- Data for Name: sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY sources (id, pubmed_id, study_type, description, created_at, updated_at) FROM stdin;
1	24889366	\N	Rossi et al., 2014, Int. J. Oncol.	2015-12-09 16:58:27.57564	2015-12-09 16:59:04.807864
2	23401436	\N	Shaw et al., 2013, J. Clin. Oncol.	2015-12-09 16:58:27.660279	2015-12-09 16:59:05.149605
3	16095999	\N	Bellacosa et al., 2005, Adv. Cancer Res.	2015-12-09 16:58:27.912699	2015-12-09 16:59:05.687258
4	15023437	\N	Fresno Vara et al., 2004, Cancer Treat. Rev.	2015-12-09 16:58:27.928133	2015-12-09 16:59:06.103871
5	15676015	\N	Lee et al., 2005, APMIS	2015-12-09 16:58:28.169007	2015-12-09 16:59:06.625371
6	24569458	\N	Imielinski et al., 2014, J. Clin. Invest.	2015-12-09 16:58:28.181178	2015-12-09 16:59:07.573579
7	15719031	\N	Ren, 2005, Nat. Rev. Cancer	2015-12-09 16:58:28.458545	2015-12-09 16:59:11.972792
8	17457302	\N	Weisberg et al., 2007, Nat. Rev. Cancer	2015-12-09 16:58:28.469385	2015-12-09 16:59:12.698644
9	19724843	\N	Li et al., 2009, Oncol. Rep.	2015-12-09 16:58:28.702496	2015-12-09 16:59:13.178284
10	23594689	\N	Pakneshan et al., 2013, Pathology	2015-12-09 16:58:28.712718	2015-12-09 16:59:14.073258
11	24366442	\N	Nelson et al., 2014, Ann. Intern. Med.	2015-12-09 16:58:28.965083	2015-12-09 16:59:15.996591
12	12432268	\N	Diehl, Cancer Biol. Ther.	2015-12-09 16:58:29.50534	2015-12-09 16:59:16.391314
13	24387133	\N	Casimiro et al., 2014, Expert Opin Investig Drugs	2015-12-09 16:58:29.516639	2015-12-09 16:59:16.882913
14	14965268	\N	Mazumder et al., 2004, Curr Cancer Drug Targets	2015-12-09 16:58:30.115684	2015-12-09 16:59:17.24387
15	20370706	\N	Graf et al., 2010, Mini Rev Med Chem	2015-12-09 16:58:30.321962	2015-12-09 16:59:17.917215
16	24089445	\N	Sheppard et al., 2013, Clin. Cancer Res.	2015-12-09 16:58:30.335625	2015-12-09 16:59:18.534277
17	20473920	\N	Shima et al., 2011, Int. J. Cancer	2015-12-09 16:58:30.856973	2015-12-09 16:59:18.95287
18	23111194	\N	Bradly et al., 2012, Diagn. Mol. Pathol.	2015-12-09 16:58:30.872525	2015-12-09 16:59:19.266755
19	19357394	\N	Vardiman et al., 2009, Blood	2015-12-09 16:58:31.120726	2015-12-09 16:59:19.68191
20	11242107	\N	Pabst et al., 2001, Nat. Genet.	2015-12-09 16:58:31.130211	2015-12-09 16:59:20.623937
21	9012825	\N	Zhang et al., 1997, Proc. Natl. Acad. Sci. U.S.A.	2015-12-09 16:58:31.139557	2015-12-09 16:59:21.040969
22	24578576	\N	Honeyman et al., 2014, Science	2015-12-09 16:58:31.361722	2015-12-09 16:59:21.474041
23	24909179	\N	Moody et al., 2015, Oncogene	2015-12-09 16:58:31.579457	2015-12-09 16:59:23.131052
24	24167195	\N	Kim et al., 2013, Blood	2015-12-09 16:58:31.79353	2015-12-09 16:59:24.071952
25	24656771	\N	Russler-Germain et al., 2014, Cancer Cell	2015-12-09 16:58:31.804945	2015-12-09 16:59:24.843104
26	23953842	\N	Yewale et al., 2013, Biomaterials	2015-12-09 16:58:32.052249	2015-12-09 16:59:26.206335
27	18712184	\N	Charpidou et al., In Vivo	2015-12-09 16:58:32.06212	2015-12-09 16:59:26.701593
28	16947083	\N	Badache et al., 2006, J Mammary Gland Biol Neoplasia	2015-12-09 16:58:32.320181	2015-12-09 16:59:27.832214
29	10878580	\N	Yu et al., 2000, Bioessays	2015-12-09 16:58:32.336695	2015-12-09 16:59:28.7686
30	24185510	\N	Robinson et al., 2013, Nat. Genet.	2015-12-09 16:58:32.598607	2015-12-09 16:59:29.188828
31	22279420	\N	Mohibi et al., 2011, J Carcinog	2015-12-09 16:58:32.607989	2015-12-09 16:59:29.566067
32	18636142	\N	Katoh, 2008, Int. J. Oncol.	2015-12-09 16:58:32.874677	2015-12-09 16:59:30.439413
33	23558953	\N	Wu et al., 2013, Cancer Discov	2015-12-09 16:58:32.884232	2015-12-09 16:59:30.962758
34	12951584	\N	Stirewalt et al., 2003, Nat. Rev. Cancer	2015-12-09 16:58:33.360609	2015-12-09 16:59:31.380909
35	22541434	\N	Kumar et al., 2012, Cell	2015-12-09 16:58:33.589175	2015-12-09 16:59:31.821558
36	25619630	\N	Mir et al., 2015, Cancer Med	2015-12-09 16:58:33.599239	2015-12-09 16:59:32.183641
37	24065766	\N	Lu et al., 2013, Genes Dev.	2015-12-09 16:58:33.848478	2015-12-09 16:59:32.564094
38	19228619	\N	Yan et al., 2009, N. Engl. J. Med.	2015-12-09 16:58:33.859197	2015-12-09 16:59:32.964451
39	22417203	\N	Patel et al., 2012, N. Engl. J. Med.	2015-12-09 16:58:34.107915	2015-12-09 16:59:33.319573
40	17133099	\N	Levine et al., 2007, Curr. Opin. Hematol.	2015-12-09 16:58:34.331463	2015-12-09 16:59:33.829109
41	23944364	\N	Stankov et al., 2014, Curr. Pharm. Des.	2015-12-09 16:58:34.559303	2015-12-09 16:59:34.209082
42	22589270	\N	Prior et al., 2012, Cancer Res.	2015-12-09 16:58:34.939674	2015-12-09 16:59:34.721411
43	23226219	\N	Chou et al., 2012, PLoS ONE	2015-12-09 16:58:35.988911	2015-12-09 16:59:35.117012
44	18073307	\N	Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-12-09 16:58:37.5921	2015-12-09 16:59:35.523245
45	25465739	\N	Perri et al., 2015, Crit. Rev. Oncol. Hematol.	2015-12-09 16:58:37.602309	2015-12-09 16:59:36.489028
46	21995386	\N	Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-12-09 16:58:38.087811	2015-12-09 16:59:36.877559
47	24136165	\N	Cazzola et al., 2013, Blood	2015-12-09 16:58:38.097609	2015-12-09 16:59:37.652342
48	20182602	\N	Olivier et al., 2010, Cold Spring Harb Perspect Biol	2015-12-09 16:58:38.329295	2015-12-09 16:59:38.090295
49	25311244	\N	Okeyo-Owuor et al., 2015, Leukemia	2015-12-09 16:58:38.952908	2015-12-09 16:59:38.568781
50	22158538	\N	Graubert et al., 2012, Nat. Genet.	2015-12-09 16:58:38.968926	2015-12-09 16:59:38.924045
51	24569458		Imielinski et al., 2014, J. Clin. Invest.	2015-12-09 16:58:44.264291	2015-12-09 16:59:39.260025
52	22081665	Retrospective study	LaRochelle et al., 2011, Oncotarget	2015-12-09 16:58:44.35762	2015-12-09 16:59:39.753953
53	22490330	Retrospective study	Ribeiro et al., 2012, Blood	2015-12-09 16:58:44.401554	2015-12-09 16:59:40.138252
54	21067377	Retrospective study	Ley et al., 2010, N. Engl. J. Med.	2015-12-09 16:58:44.570102	2015-12-09 16:59:40.502707
55	16081687	Retrospective study	Levine et al., 2005, Blood	2015-12-09 16:58:44.612098	2015-12-09 16:59:41.409816
56	16709929	Clinical trial	Kiladjian et al., 2006, Blood	2015-12-09 16:58:44.759057	2015-12-09 16:59:41.826438
57	16384925	Retrospective study	Cairoli et al., 2006, Blood	2015-12-09 16:58:44.806367	2015-12-09 16:59:42.330136
58	19915144		Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-12-09 16:58:44.899122	2015-12-09 16:59:42.671627
59	19915144	In vitro study	Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-12-09 16:58:44.948537	2015-12-09 16:59:43.079604
60	24666267	Meta analysis	Therkildsen et al., 2014, Acta Oncol	2015-12-09 16:58:45.009214	2015-12-09 16:59:43.499817
61	18375819	Case report	Banerji et al., 2008, Mol. Cancer Ther.	2015-12-09 16:58:45.131163	2015-12-09 16:59:47.443125
62	23861977	Retrospective study	Tschandl et al., 2013, PLoS ONE	2015-12-09 16:58:45.17671	2015-12-09 16:59:47.8839
63	23569304	Clinical trial	Trunzer et al., 2013, J. Clin. Oncol.	2015-12-09 16:58:45.221275	2015-12-09 16:59:48.268146
64	21576590	Case report	Soon et al., 2011, Arch Dermatol	2015-12-09 16:58:45.263304	2015-12-09 16:59:48.630524
65	15146165		Lasota et al., 2004, Lab. Invest.	2015-12-09 16:58:45.357099	2015-12-09 16:59:49.031078
66	15928335		Corless et al., 2005, J. Clin. Oncol.	2015-12-09 16:58:45.420482	2015-12-09 16:59:50.581534
67	16954519		Heinrich et al., 2006, J. Clin. Oncol.	2015-12-09 16:58:45.467257	2015-12-09 16:59:51.017629
68	21030459		Sasaki et al., 2010, Cancer Res.	2015-12-09 16:58:45.531025	2015-12-09 16:59:51.64498
69	21575866		Sakamoto et al., 2011, Cancer Cell	2015-12-09 16:58:45.645917	2015-12-09 16:59:52.063126
70	21948233		Heuckmann et al., 2011, Clin. Cancer Res.	2015-12-09 16:58:45.698289	2015-12-09 16:59:52.497938
71	18923525		George et al., 2008, Nature	2015-12-09 16:58:45.742324	2015-12-09 16:59:54.050775
72	21166657	Meta analysis	Lee et al., 2011, Br. J. Dermatol.	2015-12-09 16:58:45.908459	2015-12-09 16:59:54.469603
73	23632886	Retrospective study	Gaidzik et al., 2013, Blood	2015-12-09 16:58:45.954638	2015-12-09 16:59:55.027418
74	24512939	Retrospective study	El Ghannam et al., Blood Cells Mol. Dis.	2015-12-09 16:58:46.016244	2015-12-09 16:59:55.775436
75	22291079	Retrospective study	Marcucci et al., 2012, J. Clin. Oncol.	2015-12-09 16:58:46.331561	2015-12-09 16:59:56.139635
76	16551858	Retrospective study	Wardelmann et al., 2006, Clin. Cancer Res.	2015-12-09 16:58:46.513388	2015-12-09 16:59:56.563056
77	23775962	Clinical trial	Hodi et al., 2013, J. Clin. Oncol.	2015-12-09 16:58:46.581419	2015-12-09 16:59:58.835933
78	19671763	In vitro study	Woodman et al., 2009, Mol. Cancer Ther.	2015-12-09 16:58:46.793001	2015-12-09 16:59:59.240987
79	15696205	Retrospective study	Pao et al., 2005, PLoS Med.	2015-12-09 16:58:46.899916	2015-12-09 16:59:59.619257
80	22180178	Clinical trial	Jakob et al., 2012, Cancer	2015-12-09 16:58:46.981258	2015-12-09 17:00:01.285164
81	15951308	Retrospective study	Bowen et al., 2005, Blood	2015-12-09 16:58:47.028241	2015-12-09 17:00:02.216251
82	16434492	Clinical trial	Bacher et al., 2006, Blood	2015-12-09 16:58:47.129973	2015-12-09 17:00:02.610854
83	20619739	Retrospective study	De Roock et al., 2010, Lancet Oncol.	2015-12-09 16:58:47.170797	2015-12-09 17:00:03.34786
84	22745105		Heinrich et al., 2012, Clin. Cancer Res.	2015-12-09 16:58:47.212419	2015-12-09 17:00:03.781272
85	23031422	Case report	Ponti et al., 2012, J Hematol Oncol	2015-12-09 16:58:47.456225	2015-12-09 17:00:04.173521
86	21422803		Coxon et al., 2012, J. Endocrinol. Invest.	2015-12-09 16:58:47.523887	2015-12-09 17:00:04.805586
87	18073307		Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-12-09 16:58:47.597782	2015-12-09 17:00:05.536014
88	9839497		Egawa et al., 1998, Jpn. J. Clin. Oncol.	2015-12-09 16:58:47.643387	2015-12-09 17:00:06.578908
89	23056499		Couto et al., 2012, PLoS ONE	2015-12-09 16:58:47.722826	2015-12-09 17:00:06.942762
90	23325582	Clinical trial	Peeters et al., 2013, Clin. Cancer Res.	2015-12-09 16:58:47.776521	2015-12-09 17:00:07.311132
91	23020132	Clinical trial	Flaherty et al., 2012, N. Engl. J. Med.	2015-12-09 16:58:47.819233	2015-12-09 17:00:07.730757
92	23463675	Clinical trial	Ponti et al., 2013, J. Clin. Pathol.	2015-12-09 16:58:47.861935	2015-12-09 17:00:09.922114
93	24594804	Metaanalysis	Chen et al., 2014, PLoS ONE	2015-12-09 16:58:47.916899	2015-12-09 17:00:10.342577
94	21594703	Retrospective study	Howell et al., 2011, Ann. Surg. Oncol.	2015-12-09 16:58:47.978946	2015-12-09 17:00:11.200598
95	24570209	Retrospective study	Crescenzi et al., 2014, Horm. Metab. Res.	2015-12-09 16:58:48.030055	2015-12-09 17:00:12.117859
96	24588959	Perspective study	Zhang et al., 2014, Diagn Pathol	2015-12-09 16:58:48.081234	2015-12-09 17:00:12.55282
97	24586605	Retrospective study	Meckbach et al., 2014, PLoS ONE	2015-12-09 16:58:48.123119	2015-12-09 17:00:14.417459
98	24583796	Clinical trial	Menzies et al., 2014, Clin. Cancer Res.	2015-12-09 16:58:48.178715	2015-12-09 17:00:14.779782
99	24354346	Retrospective study	Walczyk et al., 2014, Clin. Endocrinol. (Oxf)	2015-12-09 16:58:48.249023	2015-12-09 17:00:15.363977
100	24388723	Retrospective study	Nagore et al., 2014, J. Am. Acad. Dermatol.	2015-12-09 16:58:48.299811	2015-12-09 17:00:16.267754
101	24396464	Retrospective study	He et al., 2014, Oncol Lett	2015-12-09 16:58:48.379459	2015-12-09 17:00:16.811357
102	24576830	In vitro study	Nissan et al., 2014, Cancer Res.	2015-12-09 16:58:48.454812	2015-12-09 17:00:17.125931
103	19001320	In vitro study	Di Nicolantonio et al., 2008, J. Clin. Oncol.	2015-12-09 16:58:48.49425	2015-12-09 17:00:17.861607
104	23845441	In vitro study	Rad et al., 2013, Cancer Cell	2015-12-09 16:58:48.570178	2015-12-09 17:00:18.277574
105	23812671	In vitro study	Ji et al., 2013, Clin. Cancer Res.	2015-12-09 16:58:48.636226	2015-12-09 17:00:20.366416
106	22180495	In vivo study	Yang et al., 2012, Cancer Res.	2015-12-09 16:58:48.690884	2015-12-09 17:00:20.785495
107	23524406	Case report	Rudin et al., 2013, J Thorac Oncol	2015-12-09 16:58:48.777281	2015-12-09 17:00:21.203638
108	21098728	In vitro study	Corcoran et al., 2010, Sci Signal	2015-12-09 16:58:48.819171	2015-12-09 17:00:22.249842
109	24559322	Clinical trial	Sartore-Bianchi et al., 2014, Expert Rev Anticancer Ther	2015-12-09 16:58:48.95397	2015-12-09 17:00:22.666649
110	22072639		Bresler et al., 2011, Sci Transl Med	2015-12-09 16:58:49.265291	2015-12-09 17:00:23.086083
111	19965647	Retrospective study	Burnett et al., 2010, Blood	2015-12-09 16:58:49.557607	2015-12-09 17:00:23.502224
112	18450602	Retrospective study	Schlenk et al., 2008, N. Engl. J. Med.	2015-12-09 16:58:49.667402	2015-12-09 17:00:23.947309
113	14726504	Retrospective study	Fröhling et al., 2004, J. Clin. Oncol.	2015-12-09 16:58:49.733959	2015-12-09 17:00:24.279569
114	23558953		Wu et al., 2013, Cancer Discov	2015-12-09 16:58:50.332523	2015-12-09 17:00:24.695546
115	24801015		Port et al., 2014, Ann. Hematol.	2015-12-09 16:58:50.401668	2015-12-09 17:00:25.05983
116	19965647		Burnett et al., 2010, Blood	2015-12-09 16:58:50.449295	2015-12-09 17:00:25.39818
117	17957027	Retrospective study	Gale et al., 2008, Blood	2015-12-09 16:58:50.498528	2015-12-09 17:00:25.903281
118	21537333		Schnittger et al., 2011, Leukemia	2015-12-09 16:58:50.576546	2015-12-09 17:00:26.260063
119	15626738	In vitro study	Bagrintseva et al., 2005, Blood	2015-12-09 16:58:50.778463	2015-12-09 17:00:26.69915
120	17965322	Retrospective study	Bacher et al., 2008, Blood	2015-12-09 16:58:50.86717	2015-12-09 17:00:27.259237
121	20805365	Retrospective study	Schnittger et al., 2010, Blood	2015-12-09 16:58:51.011646	2015-12-09 17:00:27.589517
122	18202412	In vitro study	Lièvre et al., 2008, J. Clin. Oncol.	2015-12-09 16:58:51.098962	2015-12-09 17:00:27.99242
123	20978259	Clinical trial	De Roock et al., 2010, JAMA	2015-12-09 16:58:51.182103	2015-12-09 17:00:28.41602
124	11070098		Esteller et al., 2000, N. Engl. J. Med.	2015-12-09 16:58:51.255759	2015-12-09 17:00:28.785915
125	19357394		Vardiman et al., 2009, Blood	2015-12-09 16:58:51.305001	2015-12-09 17:00:29.182895
126	16455956	Retrospective study	Thiede et al., 2006, Blood	2015-12-09 16:58:51.351888	2015-12-09 17:00:29.554681
127	16076867	Retrospective study	Schnittger et al., 2005, Blood	2015-12-09 16:58:51.394804	2015-12-09 17:00:30.498489
128	19059939	Retrospective study	Schlenk et al., 2009, Haematologica	2015-12-09 16:58:51.563687	2015-12-09 17:00:30.896376
129	22417203	Retrospective study	Patel et al., 2012, N. Engl. J. Med.	2015-12-09 16:58:51.682492	2015-12-09 17:00:31.403802
130	24797300		Tassara et al., 2014, Blood	2015-12-09 16:58:51.727822	2015-12-09 17:00:31.727787
131	15659725		Falini et al., 2005, N. Engl. J. Med.	2015-12-09 16:58:51.959018	2015-12-09 17:00:33.180891
132	19047294	Retrospective study	Büchner et al., 2009, J. Clin. Oncol.	2015-12-09 16:58:52.100707	2015-12-09 17:00:34.711276
133	24855211		Linch et al., 2014, Blood	2015-12-09 16:58:52.153166	2015-12-09 17:00:35.093494
134	22430270	Retrospective study	Gaidzik et al., 2012, J. Clin. Oncol.	2015-12-09 16:58:52.205609	2015-12-09 17:00:35.619651
135	20026798	Retrospective study	Becker et al., 2010, J. Clin. Oncol.	2015-12-09 16:58:52.32746	2015-12-09 17:00:37.29057
136	16051734	Retrospective study	Döhner et al., 2005, Blood	2015-12-09 16:58:52.37461	2015-12-09 17:00:39.170594
137	19587375	Retrospective study	Schnittger et al., 2009, Blood	2015-12-09 16:58:52.472888	2015-12-09 17:00:39.561903
138	24859829		Tian et al., 2014, Int. J. Hematol.	2015-12-09 16:58:52.624618	2015-12-09 17:00:39.956164
139	21719597	In vitro study	Balusu et al., 2011, Blood	2015-12-09 16:58:52.716513	2015-12-09 17:00:40.52159
140	24927407		Ehninger et al., 2014, Blood Cancer J	2015-12-09 16:58:52.767106	2015-12-09 17:00:40.930191
141	19966866	Mouse model	Liang et al., 2010, Oncogene	2015-12-09 16:58:53.184406	2015-12-09 17:00:43.244629
142	22923433	Retrospective study	Iyer et al., 2012, Science	2015-12-09 16:58:53.260417	2015-12-09 17:00:43.841478
143	18591546	Retrospective study	Virappane et al., 2008, J. Clin. Oncol.	2015-12-09 16:58:53.320903	2015-12-09 17:00:44.285336
144	19221039	Retrospective study	Gaidzik et al., 2009, Blood	2015-12-09 16:58:53.368363	2015-12-09 17:00:44.702873
145	19536888	Retrospective study	Renneville et al., 2009, Cancer	2015-12-09 16:58:53.455243	2015-12-09 17:00:45.08652
146	18559874	Retrospective study	Paschka et al., 2008, J. Clin. Oncol.	2015-12-09 16:58:53.581119	2015-12-09 17:00:46.942123
147	23346317	Review	Goff, 2013, J Gynecol Oncol	2015-12-09 16:58:54.04169	2015-12-09 17:00:47.420344
148	20537386		An et al., 2010, Leuk. Res.	2015-12-09 16:58:54.172707	2015-12-09 17:00:48.56773
149	12476305		Nimmanapalli et al., 2002, Oncogene	2015-12-09 16:58:54.22927	2015-12-09 17:00:48.984967
150	25212276		Schmidt et al., 2014, Leukemia	2015-12-09 16:58:54.444626	2015-12-09 17:00:49.380035
151	15194504	Cell Line	Yamamoto et al., 2004, Biochem. Biophys. Res. Commun.	2015-12-09 16:58:54.56136	2015-12-09 17:00:49.768692
152	23888070		Beaver et al., 2013, Clin. Cancer Res.	2015-12-09 16:58:54.712184	2015-12-09 17:00:52.437247
153	20979473	Clinical Trial	Choi et al., 2010, N. Engl. J. Med.	2015-12-09 16:58:54.760941	2015-12-09 17:00:52.816886
154	17070615		Gautschi et al., 2007, Lung Cancer	2015-12-09 16:58:54.92623	2015-12-09 17:00:53.37142
155	15961768		Arnold et al., 2005, J. Clin. Oncol.	2015-12-09 16:58:55.051171	2015-12-09 17:00:53.896819
156	16309541		Thomas et al., 2005, Int J Exp Pathol	2015-12-09 16:58:55.168617	2015-12-09 17:00:54.422167
157	17891190		Jares et al., 2007, Nat. Rev. Cancer	2015-12-09 16:58:55.216477	2015-12-09 17:00:54.887022
158	10547574		Takano et al., 1999, J. Pathol.	2015-12-09 16:58:55.263711	2015-12-09 17:00:55.344701
159	14612939		Oshimo et al., 2003, Int. J. Oncol.	2015-12-09 16:58:55.429415	2015-12-09 17:00:55.776316
160	23079656		Sawai et al., 2012, Cancer Cell	2015-12-09 16:58:55.472969	2015-12-09 17:00:56.215681
161	12432043		Keyomarsi et al., 2002, N. Engl. J. Med.	2015-12-09 16:58:55.518695	2015-12-09 17:00:56.612549
162	10224221		Donnellan et al., 1999, FASEB J.	2015-12-09 16:58:55.561413	2015-12-09 17:00:57.091448
163	23898052		Logan et al., 2013, Anticancer Res.	2015-12-09 16:58:55.686241	2015-12-09 17:00:57.496405
164	25002028		Vora et al., 2014, Cancer Cell	2015-12-09 16:58:55.738005	2015-12-09 17:00:58.38941
165	23111194		Bradly et al., 2012, Diagn. Mol. Pathol.	2015-12-09 16:58:55.889658	2015-12-09 17:00:58.806261
166	22328973		Hammerman et al., 2011, Cancer Discov	2015-12-09 16:58:55.979955	2015-12-09 17:00:59.537114
167	16730237		Ji et al., 2006, Cancer Cell	2015-12-09 16:58:56.247086	2015-12-09 17:01:00.162186
168	15118125		Paez et al., 2004, Science	2015-12-09 16:58:56.291531	2015-12-09 17:01:00.685383
169	24736073		Lim et al., 2014, J Thorac Oncol	2015-12-09 16:58:56.341894	2015-12-09 17:01:01.310663
170	24457318		Fukihara et al., 2014, Oncology	2015-12-09 16:58:56.411747	2015-12-09 17:01:02.402193
171	24662454		Douillard et al., 2014, J Thorac Oncol	2015-12-09 16:58:56.453071	2015-12-09 17:01:03.08829
172	25668228	Review	Denis et al., 2015, Clin. Chim. Acta	2015-12-09 16:58:56.552802	2015-12-09 17:01:04.941619
173	15728811		Kobayashi et al., 2005, N. Engl. J. Med.	2015-12-09 16:58:56.627553	2015-12-09 17:01:05.356439
174	24623981		Ding et al., 2014, Onco Targets Ther	2015-12-09 16:58:56.670476	2015-12-09 17:01:05.804062
175	24729716		Li et al., 2014, Onco Targets Ther	2015-12-09 16:58:56.714376	2015-12-09 17:01:06.220388
176	24636847		Li et al., 2014, Lung Cancer	2015-12-09 16:58:56.759752	2015-12-09 17:01:06.760049
177	24658966		Ai et al., 2014, Amino Acids	2015-12-09 16:58:56.809507	2015-12-09 17:01:07.106629
178	21430269		Sequist et al., 2011, Sci Transl Med	2015-12-09 16:58:56.857802	2015-12-09 17:01:07.4924
179	23220880		Bose et al., 2013, Cancer Discov	2015-12-09 16:58:56.913435	2015-12-09 17:01:07.998652
180	24185512		Toy et al., 2013, Nat. Genet.	2015-12-09 16:58:57.444018	2015-12-09 17:01:08.414798
181	24185510		Robinson et al., 2013, Nat. Genet.	2015-12-09 16:58:57.509679	2015-12-09 17:01:08.768692
182	24550739	Case study	Borad et al., 2014, PLoS Genet.	2015-12-09 16:58:57.951951	2015-12-09 17:01:09.294435
183	11290608	Retrospective study	Yamamoto et al., 2001, Blood	2015-12-09 16:58:58.062989	2015-12-09 17:01:09.684715
184	14726387	Clinical trial	Smith et al., 2004, Blood	2015-12-09 16:58:58.109582	2015-12-09 17:01:10.267414
185	18230792	In vitro study	Zhang et al., 2008, J. Natl. Cancer Inst.	2015-12-09 16:58:58.155219	2015-12-09 17:01:10.675553
186	11585760	Retrospective study	Whitman et al., 2001, Cancer Res.	2015-12-09 16:58:58.238566	2015-12-09 17:01:11.011383
187	12357354	In vitro study	Tse et al., 2002, Leukemia	2015-12-09 16:58:58.283418	2015-12-09 17:01:11.39738
188	22368270	Clinical trial	Man et al., 2012, Blood	2015-12-09 16:58:58.362853	2015-12-09 17:01:12.174318
189	17940205	Retrospective study	Whitman et al., 2008, Blood	2015-12-09 16:58:58.534218	2015-12-09 17:01:12.698132
190	22624710		Barbacid, 2012, Cancer Cell	2015-12-09 16:58:58.57936	2015-12-09 17:01:13.218697
191	20376086	Retrospective study	Ho et al., 2010, Leukemia	2015-12-09 16:58:58.630765	2015-12-09 17:01:13.640014
192	20368538	Retrospective study	Wagner et al., 2010, J. Clin. Oncol.	2015-12-09 16:58:58.673537	2015-12-09 17:01:14.046136
193	19933982	Retrospective study	Dubbink et al., 2009, Neurology	2015-12-09 16:58:58.723405	2015-12-09 17:01:14.473588
194	20127344	Retrospective study	Bleeker et al., 2010, Acta Neuropathol.	2015-12-09 16:58:58.766296	2015-12-09 17:01:15.418702
195	20494930	Retrospective study	Thol et al., 2010, Haematologica	2015-12-09 16:58:58.813584	2015-12-09 17:01:15.831987
196	20538800	Retrospective study	Abbas et al., 2010, Blood	2015-12-09 16:58:58.865352	2015-12-09 17:01:16.666621
197	22616558		Zhou et al., 2012, Leuk. Lymphoma	2015-12-09 16:58:59.087195	2015-12-09 17:01:17.289222
198	20421455	Retrospective study	Thol et al., 2010, Blood	2015-12-09 16:58:59.143855	2015-12-09 17:01:18.022475
199	21997850		Lin et al., 2012, Ann. Hematol.	2015-12-09 16:58:59.190735	2015-12-09 17:01:18.964166
200	22033490		Patnaik et al., 2012, Leukemia	2015-12-09 16:58:59.243116	2015-12-09 17:01:19.484263
201	21596855		Green et al., 2011, Blood	2015-12-09 16:58:59.311572	2015-12-09 17:01:20.436398
202	10485475	Retrospective study	Taniguchi et al., 1999, Cancer Res.	2015-12-09 16:58:59.514371	2015-12-09 17:01:20.843429
203	12000708	Retrospective study	Corless et al., 2002, Am. J. Pathol.	2015-12-09 16:58:59.582439	2015-12-09 17:01:22.796469
204	15217946	Retrospective study	Boldrini et al., 2004, Clin. Cancer Res.	2015-12-09 16:58:59.651182	2015-12-09 17:01:23.244855
205	18421059	Observational study	Hodi et al., 2008, J. Clin. Oncol.	2015-12-09 16:58:59.872491	2015-12-09 17:01:23.663623
206	17372901	In vitro study	Antonescu et al., 2007, Int. J. Cancer	2015-12-09 16:58:59.950635	2015-12-09 17:01:24.081935
207	16638875		Prenen et al., 2006, Clin. Cancer Res.	2015-12-09 16:59:00.016109	2015-12-09 17:01:24.513799
208	16618717	Retrospective study	Lièvre et al., 2006, Cancer Res.	2015-12-09 16:59:00.138034	2015-12-09 17:01:24.921373
209	18794081	Retrospective study	Riely et al., 2008, Clin. Cancer Res.	2015-12-09 16:59:00.204592	2015-12-09 17:01:25.307827
210	19934290	Retrospective study	Ogino et al., 2009, Clin. Cancer Res.	2015-12-09 16:59:00.264064	2015-12-09 17:01:25.739537
211	11208838	Retrospective study	Schiller et al., 2001, J. Clin. Oncol.	2015-12-09 16:59:00.315369	2015-12-09 17:01:26.171224
212	18528420	Retrospective study	Chng et al., 2008, Leukemia	2015-12-09 16:59:00.365756	2015-12-09 17:01:26.588737
213	15597105	Metaanalysis	Mascaux et al., 2005, Br. J. Cancer	2015-12-09 16:59:00.411977	2015-12-09 17:01:27.008353
214	19794967		Marchetti et al., 2009, Neoplasia	2015-12-09 16:59:00.460933	2015-12-09 17:01:27.435627
215	23014527	Retrospective study	Dogan et al., 2012, Clin. Cancer Res.	2015-12-09 16:59:00.5212	2015-12-09 17:01:28.261286
216	19029981	In vivo study	Engelman et al., 2008, Nat. Med.	2015-12-09 16:59:00.686749	2015-12-09 17:01:28.678102
217	15758010	Clinical Trial	Hegi et al., 2005, N. Engl. J. Med.	2015-12-09 16:59:01.122408	2015-12-09 17:01:29.037726
218	20007775		Westhoff et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-12-09 16:59:01.287485	2015-12-09 17:01:29.398014
219	21558396		Tanaka et al., 2011, Clin. Cancer Res.	2015-12-09 16:59:01.511847	2015-12-09 17:01:29.773046
220	15647370		Kang et al., 2005, Proc. Natl. Acad. Sci. U.S.A.	2015-12-09 16:59:01.561135	2015-12-09 17:01:31.707788
221	19223544		Sartore-Bianchi et al., 2009, Cancer Res.	2015-12-09 16:59:01.605006	2015-12-09 17:01:32.333542
222	22357840		Liao et al., 2012, Clin. Cancer Res.	2015-12-09 16:59:01.652473	2015-12-09 17:01:32.974055
223	8674046		Yoshida et al., 1996, Cancer Res.	2015-12-09 16:59:02.242233	2015-12-09 17:01:33.377498
224	22479427		Carico et al., 2012, PLoS ONE	2015-12-09 16:59:02.305407	2015-12-09 17:01:36.930494
225	20085938		Courtney et al., 2010, J. Clin. Oncol.	2015-12-09 16:59:02.420299	2015-12-09 17:01:37.348793
226	21995386		Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-12-09 16:59:02.538534	2015-12-09 17:01:38.223958
227	16489069		Olivier et al., 2006, Clin. Cancer Res.	2015-12-09 16:59:02.661314	2015-12-09 17:01:38.595021
228	22698404		Jackson et al., 2012, Cancer Cell	2015-12-09 16:59:02.7234	2015-12-09 17:01:38.919454
229	9569050		Berns et al., 1998, Br. J. Cancer	2015-12-09 16:59:02.835566	2015-12-09 17:01:39.640884
230	22923433		Iyer et al., 2012, Science	2015-12-09 16:59:03.244671	2015-12-09 17:01:40.079078
231	23029227		Qian et al., 2012, PLoS ONE	2015-12-09 16:59:03.343405	2015-12-09 17:01:40.587108
232	23861105		Wu et al., 2013, Am. J. Hematol.	2015-12-09 16:59:03.46841	2015-12-09 17:01:41.011338
233	23524406	case report	Rudin et al., 2013, J Thorac Oncol	2015-12-09 16:59:03.679995	2015-12-09 17:01:41.419971
234	24186003		Lilljebjörn et al., 2014, Leukemia	2015-12-09 16:59:03.731574	2015-12-09 17:01:41.803449
235	24578576	Retrospective study	Honeyman et al., 2014, Science	2015-12-09 16:59:03.847544	2015-12-09 17:01:42.361258
236	16818686	Retrospective study	Jackman et al., 2006, Clin. Cancer Res.	2015-12-09 16:59:03.969541	2015-12-09 17:01:42.777392
237	9552022	Retrospective study	de Alava et al., 1998, J. Clin. Oncol.	2015-12-09 16:59:04.034282	2015-12-09 17:01:43.194217
\.


--
-- Data for Name: variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variants (id, gene_id, name, description, created_at, updated_at, deleted, deleted_at, genome_build, chromosome, start, stop, reference_bases, variant_bases, representative_transcript, chromosome2, start2, stop2, reference_build, representative_transcript2, ensembl_version) FROM stdin;
1	4	BCR-ABL	The BCR-ABL fusion protein, commonly referred to as the philadelphia chromosome, is the most well-studied fusion gene in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML), but despite its ability initiate disease in mice, its status an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and nilotinib) have seen some success in delivering a tumor response.	2015-12-09 16:58:40.347359	2015-12-09 16:58:40.347359	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
2	4	BCR-ABL T315I	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-12-09 16:58:40.377418	2015-12-09 16:58:40.377418	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
3	4	BCR-ABL E255K	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-12-09 16:58:40.399859	2015-12-09 16:58:40.399859	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
5	1	EML4-ALK	The EML4-ALK fusion has been seen in non-small cell lung cancer, and appears to be an alternative mechanism for ALK activation. Cells with this fusion have been shown to be sensitive to the ALK inhibitor crizotinib.	2015-12-09 16:58:40.449091	2015-12-09 16:58:40.449091	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
6	1	EML4-ALK C1156Y	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-12-09 16:58:40.471151	2015-12-09 16:58:40.471151	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
7	1	EML4-ALK L1196M	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-12-09 16:58:40.49677	2015-12-09 16:58:40.49677	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
13	5	V600E/V600M	A case study of a single patient harboring both a V600E and a V600M mutation, dabrafenib was shown to acheive clinical response.	2015-12-09 16:58:40.645774	2015-12-09 16:58:40.645774	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
14	5	V600E AMPLIFICATION	Amplification of BRAF V600E has been shown to confer resistance to MEK inhibitors. For more information on the V600 locus, see the V600E entry.	2015-12-09 16:58:40.668051	2015-12-09 16:58:40.668051	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
17	5	V600	BRAF mutations of the valine 600 residue have been shown to be recurrent in many cancer types. Of the V600 mutations, V600E is the most widely researched. V600 mutations as a whole have been correlated to poorer prognosis in colorectal and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual mutation pages on the left sidebar.	2015-12-09 16:58:40.746703	2015-12-09 16:58:40.746703	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
18	8	AMPLIFICATION	CCND1 amplification has been implicated in poorer prognosis in non-small cell lung cancer.	2015-12-09 16:58:40.770302	2015-12-09 16:58:40.770302	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
19	8	EXPRESSION	CCND1 expression, and its prognositc impact, is still in dispute. Three experiments in non-small cell lung cancer have shown it to have no impact on survival, but three additional studies have shown it results in poorer prognosis. There is also some ambiguity in how the boundaries between expression and overexpression are defined.	2015-12-09 16:58:40.793171	2015-12-09 16:58:40.793171	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
20	8	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-12-09 16:58:40.818247	2015-12-09 16:58:40.818247	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
21	9	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-12-09 16:58:40.845221	2015-12-09 16:58:40.845221	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
22	9	PROMOTER DEMETHYLATION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-12-09 16:58:40.866977	2015-12-09 16:58:40.866977	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
23	10	LOSS	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL), a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic palbociclib significantly increased the median survival of a Notch-driven model of T-ALL.	2015-12-09 16:58:40.89141	2015-12-09 16:58:40.89141	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
24	11	OVEREXPRESSION	Cyclin E, while currenly not as widely implicated as its counterpart, cyclin D, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, are correlated with later stage disease and have been shown to lead to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	2015-12-09 16:58:40.91329	2015-12-09 16:58:40.91329	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
25	13	EXPRESSION	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-12-09 16:58:40.934897	2015-12-09 16:58:40.934897	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
26	12	EXPRESSION	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-12-09 16:58:40.956453	2015-12-09 16:58:40.956453	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
27	14	PROMOTER HYPERMETHYLATION	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the CDKN2A promoter region. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcomes for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	2015-12-09 16:58:40.981352	2015-12-09 16:58:40.981352	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
28	15	N-TERMINAL FRAME SHIFT	CEBPA N-terminal frame shift mutations that result in a premature stop codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML). CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-12-09 16:58:41.007515	2015-12-09 16:58:41.007515	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
29	15	MUTATION	CEBPA mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-12-09 16:58:41.029726	2015-12-09 16:58:41.029726	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
30	33	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-12-09 16:58:41.05759	2015-12-09 16:58:41.05759	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
31	16	DNAJB1-PRKACA	This fusion has been found to be very recurrent in the rare form of adolescent liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-12-09 16:58:41.084577	2015-12-09 16:58:41.084577	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
32	18	R882	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-12-09 16:58:41.110841	2015-12-09 16:58:41.110841	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
74	30	EXON 1 MUTATIONS	A study by Lièvre et al in 2006 showed that colorectal cancer patients with KRAS exon 1 mutations had low cetuximab response rates.	2015-12-09 16:58:42.15204	2015-12-09 16:58:42.15204	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
37	20	DEL 755-759	ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-12-09 16:58:41.236551	2015-12-09 16:58:41.236551	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
40	20	L755W	ERBB2 L755W was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-12-09 16:58:41.305282	2015-12-09 16:58:41.305282	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
41	20	P780INS	ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-12-09 16:58:41.343376	2015-12-09 16:58:41.343376	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
42	20	R678Q	ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-12-09 16:58:41.367685	2015-12-09 16:58:41.367685	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
75	30	EXON 2 MUTATIONS	In a study by Pao et al in 2005, non-small cell lung cancer patients harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors gefinitib and erlotinib.	2015-12-09 16:58:42.182147	2015-12-09 16:58:42.182147	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
46	21	L536Q	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of these ligand binding domain mutations, and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-12-09 16:58:41.461886	2015-12-09 16:58:41.461886	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
47	21	N538G	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-12-09 16:58:41.48751	2015-12-09 16:58:41.48751	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
51	22	P235R		2015-12-09 16:58:41.584588	2015-12-09 16:58:41.584588	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
52	22	FGFR2-MGEA5	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. Additionally, tumor size reduction was achieved by both ponatinib and pazopanib treatments administered separately in a single patient with intrahepatic cholangiocarcinoma and this fusion. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-12-09 16:58:41.606917	2015-12-09 16:58:41.606917	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
53	23	FGFR3-BAIAP2L1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-12-09 16:58:41.631757	2015-12-09 16:58:41.631757	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
54	23	FGFR3-TACC3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-12-09 16:58:41.653244	2015-12-09 16:58:41.653244	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
55	24	ITD MUTATIONS	FLT3-ITD (internal tandem duplications) frequently occur in patients with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated sequence resulting in an in-frame duplication event. The length of these duplications can vary widely which may have prognostic consequences, but this has not been conclusively determined. FLT3-ITD mutations overall have generally been associated with poor prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the prognosis associated with this variant.	2015-12-09 16:58:41.681013	2015-12-09 16:58:41.681013	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
56	24	TKD MUTATIONS	FLT3 tyrosine kinase domain mutations (FLT3-TKD) are much less common than FLT3-ITD (internal tandem duplication) mutations and may not confer the same prognostic impact. Although the majority of mutations are point mutations effecting D835 (most frequently D835Y), a small proportion involve an in-frame deletion of I836. These mutations are within the activation loop of the second tyrosine kinase domain of FLT3 and thought to result in constitutive activation of the receptor.	2015-12-09 16:58:41.703141	2015-12-09 16:58:41.703141	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
57	25	EXPRESSION	GATA2 misregulation has been observed in a number of hematologic malignancies, as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung cancer.	2015-12-09 16:58:41.727903	2015-12-09 16:58:41.727903	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
76	30	G12	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-12-09 16:58:42.204582	2015-12-09 16:58:42.204582	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
58	26	R132	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-12-09 16:58:41.754218	2015-12-09 16:58:41.754218	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
62	27	R140Q/L	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival as compared to their wild-type counterparts. In myelodysplastic syndromes, however, no prognostic link was discovered between IDH2 mutation status and overall survival.	2015-12-09 16:58:41.849057	2015-12-09 16:58:41.849057	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
66	29	EXON 11 MUTATIONS	c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-12-09 16:58:41.958524	2015-12-09 16:58:41.958524	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
67	29	INTERNAL DUPLICATION	c-KIT internal duplications have been observed in exon 11, within the juxtamembrane domain. In a case study of an anal melanoma patient harboring this event, imatinib confered marked response. Also, cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-12-09 16:58:41.98381	2015-12-09 16:58:41.98381	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
68	29	EXON 13 MUTATIONS	c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations. These mutations lie within the tyrosine kinase 1 domain, and are found primarily in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations show sensitivity to imatinib and sunitinib treatment. However, only imatinib has seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant cases.	2015-12-09 16:58:42.006847	2015-12-09 16:58:42.006847	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
69	29	EXON 14 MUTATIONS	c-KIT exon 14 mutations lie within the tyrosine kinase domain of the protein. While relatively rare in primary gastrointestinal tumors, they are notably more prevalent in refractory disease, suggesting a role in imatinib resistance. Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.	2015-12-09 16:58:42.033175	2015-12-09 16:58:42.033175	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
70	29	EXON 17 MUTATIONS	c-KIT exon 17 mutations lie within the TK2 domain, containing the activation loop of the protein. In cell lines, mutations within this domain have been shown to be sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.	2015-12-09 16:58:42.056243	2015-12-09 16:58:42.056243	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
71	29	EXON 9 MUTATIONS	c-KIT exon 9 mutations lie within the dimerization motif of the protein. Relative to other KIT mutations, exon 9 mutations have been associated with better overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and in vivo.	2015-12-09 16:58:42.081677	2015-12-09 16:58:42.081677	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
72	29	L576P	KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain. It is one of the most recurrent KIT mutations in melanoma, and both in vitro and in vivo studies have shown sensitivity to imatinib.	2015-12-09 16:58:42.103638	2015-12-09 16:58:42.103638	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
77	30	G12/G13	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-12-09 16:58:42.229592	2015-12-09 16:58:42.229592	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
80	30	G13	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-12-09 16:58:42.30637	2015-12-09 16:58:42.30637	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
84	32	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-12-09 16:58:42.409742	2015-12-09 16:58:42.409742	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
85	34	PROMOTER METHYLATION	MGMT promoter methylation has been observed to impact tumor progression in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating agent carmustine has shown efficacy. In patients lacking methylation, combining carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more experiments are required. Clinical trials have also shown selective sensitivity of promoter methylation-positive patients to temozolomide, making a case for wider methylation screening in GBM patients.	2015-12-09 16:58:42.432723	2015-12-09 16:58:42.432723	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
86	35	EXON 12 MUTATIONS	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-12-09 16:58:42.459392	2015-12-09 16:58:42.459392	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
88	36	EXON 1 MUTATIONS	NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown to be correlated with poorer overall survival relative to wild-type NRAS in melanoma patients.	2015-12-09 16:58:42.505838	2015-12-09 16:58:42.505838	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
89	36	EXON 2 MUTATIONS	NRAS exon 2 mutations have been shown to be correlated with poorer overall survival in melanoma patients and colorectal cancer patients, however no prognostic impact was seen in acute myeloid leukemia patients.	2015-12-09 16:58:42.532882	2015-12-09 16:58:42.532882	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
90	36	EXON 3 MUTATIONS	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-12-09 16:58:42.563059	2015-12-09 16:58:42.563059	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
91	36	EXON 4 MUTATIONS	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-12-09 16:58:42.588323	2015-12-09 16:58:42.588323	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
92	36	G12	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-12-09 16:58:42.634699	2015-12-09 16:58:42.634699	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
94	36	Q61	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-12-09 16:58:42.68218	2015-12-09 16:58:42.68218	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
111	40	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and Arsenic Trioxide for treating these patients, and early results seem promising.	2015-12-09 16:58:43.109949	2015-12-09 16:58:43.109949	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
97	38	V536E		2015-12-09 16:58:42.754544	2015-12-09 16:58:42.754544	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
101	38	DEL I843	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-12-09 16:58:42.846563	2015-12-09 16:58:42.846563	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
102	38	DI842-843IM	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-12-09 16:58:42.872526	2015-12-09 16:58:42.872526	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
105	37	EXON 20 MUTATIONS	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-12-09 16:58:42.945613	2015-12-09 16:58:42.945613	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
106	37	EXON 9 MUTATIONS	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-12-09 16:58:42.976796	2015-12-09 16:58:42.976796	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
108	39	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	2015-12-09 16:58:43.028842	2015-12-09 16:58:43.028842	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
109	17	DNAJB1-PRKACA	This fusion has been found to be very recurrent in a rare form of adolescent liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined, and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-12-09 16:58:43.055045	2015-12-09 16:58:43.055045	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
114	44	K666N	SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients.	2015-12-09 16:58:43.185403	2015-12-09 17:02:08.912302	f	\N	\N	2	198267359	198267359	C	A	\N	\N	\N	\N	\N	\N	\N
123	45	V173G/A	While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-12-09 16:58:43.441814	2015-12-09 16:58:43.441814	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
124	46	FRAMESHIFT TRUNCATION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-12-09 16:58:43.466574	2015-12-09 16:58:43.466574	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
125	46	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-12-09 16:58:43.494301	2015-12-09 16:58:43.494301	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
126	47	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-12-09 16:58:43.519036	2015-12-09 16:58:43.519036	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
8	1	F1174L	ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells.	2015-12-09 16:58:40.520712	2015-12-09 17:02:07.729534	f	\N	\N	2	29443695	29443695	G	T	\N	\N	\N	\N	\N	\N	\N
127	48	Q157P/R	U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is less common than the S34F mutation, occurs in the second zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-12-09 16:58:43.542818	2015-12-09 16:58:43.542818	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
128	48	S34Y/F	U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is the most commonly identified variant in MDS, occurs in the first zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-12-09 16:58:43.56715	2015-12-09 16:58:43.56715	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
129	49	EXON 7 MUTATIONS	WT1 exon 7 mutations have been shown to be recurrent in acute myeloid leukemia. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 7 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-12-09 16:58:43.591779	2015-12-09 16:58:43.591779	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
130	49	EXON 9 MUTATIONS	WT1 exon 9 mutations have been shown to be recurrent in acute myeloid leukemia, although at a less frequent rate than their exon 7 counterparts. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 9 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-12-09 16:58:43.618483	2015-12-09 16:58:43.618483	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
131	6	LOSS-OF-FUNCTION	BRCA1 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.	2015-12-09 16:58:43.642702	2015-12-09 16:58:43.642702	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
132	7	LOSS-OF-FUNCTION	BRCA2 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.	2015-12-09 16:58:43.664166	2015-12-09 16:58:43.664166	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
133	19	EXON 19 DELETION	Deletions within exon 19 of EGFR are most common in lung cancer. These deletions, in non-small cell lung cancer, have been shown to be sensitive to the EGFR tyrosine kinase inhibitors gefitinib and erlotinib. There is also data to suggest that this event is a good prognostic marker in lung adenocarcinoma.	2015-12-09 16:58:43.688533	2015-12-09 16:58:43.688533	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
134	19	G719S	While not as recurrent as the L858R mutation, EGFR G719S has also been shown to be an activating mutation. In lung cancer cell lines, it also confers sensitivity to the tyrosine kinase inhibitors gefitinib and erlotinib.	2015-12-09 16:58:43.718108	2015-12-09 16:58:43.718108	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
135	50	D1643H	Activating mutations in NOTCH1, including D1643H, have been shown to be poor prognostic markers in lung cancer.	2015-12-09 16:58:43.740719	2015-12-09 16:58:43.740719	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
136	50	R2328W	Activating mutations in NOTCH1, including R2328W, have been shown to be poor prognostic markers in lung cancer.	2015-12-09 16:58:43.764439	2015-12-09 16:58:43.764439	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
137	50	V2444FS	Activating mutations in NOTCH1, including a frameshift insertion at V2444, have been shown to be poor prognostic markers in lung cancer.	2015-12-09 16:58:43.792726	2015-12-09 16:58:43.792726	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
138	50	S2275FS	Activating mutations in NOTCH1, including a frameshift insertion at S2275, have been shown to be poor prognostic markers in lung cancer.	2015-12-09 16:58:43.824763	2015-12-09 16:58:43.824763	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
139	51	L63V	Activating mutations in DDR2, including L63V, has been shown to be sensitive to dasatinib in cell lines.	2015-12-09 16:58:43.850563	2015-12-09 16:58:43.850563	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
140	51	L239R	Activating mutations in DDR2, including L239R, has been shown to be sensitive to dasatinib in cell lines.	2015-12-09 16:58:43.878182	2015-12-09 16:58:43.878182	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
141	51	G253C	Activating mutations in DDR2, including G253C, has been shown to be sensitive to dasatinib in cell lines.	2015-12-09 16:58:43.905988	2015-12-09 16:58:43.905988	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
142	51	G505S	Activating mutations in DDR2, including G505S, has been shown to be sensitive to dasatinib in cell lines.	2015-12-09 16:58:43.930917	2015-12-09 16:58:43.930917	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
143	51	I638F	DDR2 I638F has been shown to be a loss of function mutation, but also confers sensitivity to dasatinib in cell lines.	2015-12-09 16:58:43.956899	2015-12-09 16:58:43.956899	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
144	51	G774V	Activating mutations in DDR2, including G774V, has been shown to be sensitive to dasatinib in cell lines.	2015-12-09 16:58:43.999634	2015-12-09 16:58:43.999634	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
145	51	S768R	Activating mutations in DDR2, including S768R, has been shown to be sensitive to dasatinib in cell lines.	2015-12-09 16:58:44.028508	2015-12-09 16:58:44.028508	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
146	52	AMPLIFICATION	MET amplification, like EGFR T790M, has been shown to be capable of driving acquired resistance to dacomitinib in patients with lung adenocarcinoma.	2015-12-09 16:58:44.053625	2015-12-09 16:58:44.053625	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
147	30	G12V	KRAS G12V, like EGFR T790M, has been shown to be capable of driving acquired resistance to tyrosine kinase inhibitors in lung adenocarcinoma.	2015-12-09 16:58:44.077918	2015-12-09 16:58:44.077918	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
148	30	G12A	KRAS G12A, like EGFR T790M, has been shown to be capable of driving acquired resistance to tyrosine kinase inhibitors in lung adenocarcinoma.	2015-12-09 16:58:44.102804	2015-12-09 16:58:44.102804	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
149	54	EWS-FLI1	EWS-FLI1 fusions have been found to be extremely common in Ewing's sarcomas. The presence of the fusion is also thought to be a positive prognostic marker of survival.	2015-12-09 16:58:44.124565	2015-12-09 16:58:44.124565	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
150	53	EWS-FLI1	EWS-FLI1 fusions have been found to be extremely common in Ewing's sarcomas. The presence of the fusion is also thought to be a positive prognostic marker of survival.	2015-12-09 16:58:44.146067	2015-12-09 16:58:44.146067	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
151	53	TYPE 1 EWS-FLI1	The most common EWS-FLI1 fusion (type 1) links EWS exon 7 to FLI1 exon 6. This fusion product is found to be associated with better outcome than other EWS-FLI1 fusions in Ewing's Sarcoma.	2015-12-09 16:58:44.175075	2015-12-09 16:58:44.175075	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
4	2	E17K	AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal, lung, and ovarian cancer. It has been convincingly shown to be an activating mutation resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and other AKT1 mutations, are the subject of much research and development for therapeutics.	2015-12-09 16:58:40.424135	2015-12-09 17:02:07.704463	f	\N	\N	14	105246551	105246551	C	T	\N	\N	\N	\N	\N	\N	\N
9	1	R1275Q	ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma, non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This, and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC cell lines.	2015-12-09 16:58:40.542723	2015-12-09 17:02:07.750474	f	\N	\N	2	29432664	29432664	C	T	\N	\N	\N	\N	\N	\N	\N
10	3	S214C	ARAF S214C has been found to be a recurrent oncogenic mutation in non-small cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this mutation, sorafenib also acheived near-complete clinical remission. This case has brought more interest to the variant from a research and clinical perspective.	2015-12-09 16:58:40.564279	2015-12-09 17:02:07.771574	f	\N	\N	X	47426121	47426121	C	G	\N	\N	\N	\N	\N	\N	\N
11	5	V600D	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-12-09 16:58:40.589191	2015-12-09 17:02:07.793438	f	\N	\N	7	140453135	140453136	CA	AT	\N	\N	\N	\N	\N	\N	\N
12	5	V600E	BRAF V600E has been shown to be recurrent in many cancer types. It is one of the most widely studied variants in cancer. This variant is correlated with poor prognosis in certain cancer types, including colorectal cancer and papillary thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib has also shown to be effective when combined with the MEK inhibitor trametinib in colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib, and a number of combination therapies have been successful in treating V600E mutations. While the drugs cetuximab and panitumumab have been largely shown to be ineffective without supplementary treatment.	2015-12-09 16:58:40.617268	2015-12-09 17:02:07.818687	f	\N	\N	7	140453136	140453136	A	T	\N	\N	\N	\N	\N	\N	\N
15	5	V600M	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-12-09 16:58:40.703781	2015-12-09 17:02:07.85347	f	\N	\N	7	140453137	140453137	C	T	\N	\N	\N	\N	\N	\N	\N
16	5	V600R	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-12-09 16:58:40.72628	2015-12-09 17:02:07.880449	f	\N	\N	7	140453136	140453137	AC	CT	\N	\N	\N	\N	\N	\N	\N
33	19	L858R	EGFR L858R has long been recognized as a functionally significant mutation in cancer, and is one of the most prevalent single mutations in lung cancer. Best described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity to first and second generation TKI's like gefitinib and neratinib. NSCLC patients with this mutation treated with TKI's show increased overall and progression-free survival, as compared to chemotherapy alone. Third generation TKI's are currently in clinical trials that specifically focus on mutant forms of EGFR, a few of which have shown efficacy in treating patients that failed to respond to earlier generation TKI therapies.	2015-12-09 16:58:41.134082	2015-12-09 17:02:07.922464	f	\N	\N	7	55259514	55259515	CT	AG	\N	\N	\N	\N	\N	\N	\N
34	19	T790M	EGFR T790M was one of the very first mutations recognized to confer resistance to targeted therapies in non-small cell lung cancer. While successful in amplified EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients harboring this mutation before treatment is notably lower. This lack of efficacy can likely be to blame for the poorer prognosis for patients with this mutation as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired resistance to TKI inhibition have been shown to harbor this mutation, implicating it as a mechanism of acquired therapy resistence. The third generation TKI's are being developed with this resistance problem in mind, and early stage experiments have shown some efficacy of these drugs in previously resistant tumors with mutant EGFR.	2015-12-09 16:58:41.161286	2015-12-09 17:02:07.955434	f	\N	\N	7	55249071	55249071	C	T	\N	\N	\N	\N	\N	\N	\N
35	20	D769H	ERBB2 D769H was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-12-09 16:58:41.185023	2015-12-09 17:02:07.987004	f	\N	\N	17	37880261	37880261	G	C	\N	\N	\N	\N	\N	\N	\N
36	20	D769Y	ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-12-09 16:58:41.213919	2015-12-09 17:02:08.024902	f	\N	\N	17	37880261	37880261	G	T	\N	\N	\N	\N	\N	\N	\N
38	20	G309A	ERBB2 G309A was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-12-09 16:58:41.25963	2015-12-09 17:02:08.060453	f	\N	\N	17	37868205	37868205	G	C	\N	\N	\N	\N	\N	\N	\N
39	20	L755S	ERBB2 L755S was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike many of the other variants queried. This mutation was also shown to confer resistance to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.	2015-12-09 16:58:41.283461	2015-12-09 17:02:08.091793	f	\N	\N	17	37880220	37880220	T	C	\N	\N	\N	\N	\N	\N	\N
43	20	R896C	ERBB2 R896C was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-12-09 16:58:41.389468	2015-12-09 17:02:08.119925	f	\N	\N	17	37881616	37881616	C	T	\N	\N	\N	\N	\N	\N	\N
79	30	G12D	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-12-09 16:58:42.282581	2015-12-09 17:02:08.454374	f	\N	\N	12	25398284	25398284	C	T	\N	\N	\N	\N	\N	\N	\N
44	20	V777L	ERBB2 V777L was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-12-09 16:58:41.415824	2015-12-09 17:02:08.141774	f	\N	\N	17	37881000	37881000	G	T	\N	\N	\N	\N	\N	\N	\N
45	20	V842I	ERBB2 V842I was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-12-09 16:58:41.439077	2015-12-09 17:02:08.164038	f	\N	\N	17	37881332	37881332	G	A	\N	\N	\N	\N	\N	\N	\N
48	21	Y537C	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-12-09 16:58:41.509857	2015-12-09 17:02:08.185551	f	\N	\N	6	152419923	152419923	A	G	\N	\N	\N	\N	\N	\N	\N
49	21	Y537N	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-12-09 16:58:41.531327	2015-12-09 17:02:08.206538	f	\N	\N	6	152419922	152419922	T	A	\N	\N	\N	\N	\N	\N	\N
50	21	Y537S	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-12-09 16:58:41.556789	2015-12-09 17:02:08.231178	f	\N	\N	6	152419923	152419923	A	C	\N	\N	\N	\N	\N	\N	\N
59	26	R132C	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-12-09 16:58:41.77786	2015-12-09 17:02:08.264466	f	\N	\N	2	209113113	209113113	G	A	\N	\N	\N	\N	\N	\N	\N
60	26	R132H	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-12-09 16:58:41.799678	2015-12-09 17:02:08.286894	f	\N	\N	2	209113112	209113112	C	T	\N	\N	\N	\N	\N	\N	\N
61	26	R132L	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-12-09 16:58:41.821758	2015-12-09 17:02:08.306812	f	\N	\N	2	209113112	209113112	C	A	\N	\N	\N	\N	\N	\N	\N
63	27	R172K	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic syndromes, studies did not find a prognostic association between this variant and patient outcomes.	2015-12-09 16:58:41.871438	2015-12-09 17:02:08.330167	f	\N	\N	15	90631838	90631838	C	T	\N	\N	\N	\N	\N	\N	\N
64	28	V617F	JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases, occuring in around half of all MPD's. While less associated with cancer, when it is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The V617F mutation is an activating mutation, resulting in increased kinase activity. The mutation seems to be restricted to hematologic malignancies. Treatment of JAK mutant diseases with ruxolitinib has seen some clinical success.	2015-12-09 16:58:41.894582	2015-12-09 17:02:08.349154	f	\N	\N	9	5073770	5073770	G	T	\N	\N	\N	\N	\N	\N	\N
65	29	D816V	KIT D816V is a mutation observed in acute myeloid leukemia (AML). This variant has been linked to poorer prognosis and worse outcome in AML patients.	2015-12-09 16:58:41.918367	2015-12-09 17:02:08.378336	f	\N	\N	4	55599321	55599321	A	T	\N	\N	\N	\N	\N	\N	\N
73	29	V654A	KIT V654A is an exon 13 mutation that lies within the tyrosine kinase 1 domain of the protein. It has been shown to be an activating mutation by in vitro studies. This mutation is associated with imatinib resistance in melanoma patients. However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have seen success in acheiving tumor response.	2015-12-09 16:58:42.124995	2015-12-09 17:02:08.410352	f	\N	\N	4	55594258	55594258	T	C	\N	\N	\N	\N	\N	\N	\N
78	30	G12C	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-12-09 16:58:42.253465	2015-12-09 17:02:08.435788	f	\N	\N	12	25398285	25398285	C	A	\N	\N	\N	\N	\N	\N	\N
81	30	G13D	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-12-09 16:58:42.334611	2015-12-09 17:02:08.503981	f	\N	\N	12	25398281	25398281	C	T	\N	\N	\N	\N	\N	\N	\N
82	31	P124S	MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder and colon cancer to a lesser degree. The P124S mutation has been shown to contribute to AZD6244 resistance in melanoma cell lines, but considerably less so than its Q56P counterpart.	2015-12-09 16:58:42.361934	2015-12-09 17:02:08.525295	f	\N	\N	15	66729162	66729162	C	T	\N	\N	\N	\N	\N	\N	\N
83	31	Q56P	MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This mutation has been shown to confer considerable resistance to AZD6244 treatment of melanoma cell lines.	2015-12-09 16:58:42.386864	2015-12-09 17:02:08.553223	f	\N	\N	15	66727451	66727451	A	C	\N	\N	\N	\N	\N	\N	\N
87	35	W288FS	NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most common NPM1 mutation identified in acute myeloid leukemia. This mutation results in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation, it is generally grouped with other exon 12 mutations for patient analysis (see NPM1 Exon 12 variants for more information).	2015-12-09 16:58:42.483546	2015-12-09 17:02:08.588773	f	\N	\N	5	170837546	170837547	0	TCAG	\N	\N	\N	\N	\N	\N	\N
93	36	G13D	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-12-09 16:58:42.657825	2015-12-09 17:02:08.609536	f	\N	\N	1	115258744	115258744	C	T	\N	\N	\N	\N	\N	\N	\N
95	36	Q61L	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-12-09 16:58:42.706704	2015-12-09 17:02:08.631112	f	\N	\N	1	115256528	115256529	TT	CA	\N	\N	\N	\N	\N	\N	\N
96	36	Q61R	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-12-09 16:58:42.730065	2015-12-09 17:02:08.661573	f	\N	\N	1	115256528	115256529	TT	CC	\N	\N	\N	\N	\N	\N	\N
98	38	D842I	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-12-09 16:58:42.77547	2015-12-09 17:02:08.69161	f	\N	\N	4	55152092	55152093	GA	AT	\N	\N	\N	\N	\N	\N	\N
99	38	D842V	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-12-09 16:58:42.797765	2015-12-09 17:02:08.711685	f	\N	\N	4	55152093	55152093	A	T	\N	\N	\N	\N	\N	\N	\N
100	38	D842Y	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-12-09 16:58:42.82258	2015-12-09 17:02:08.737188	f	\N	\N	4	55152092	55152092	G	T	\N	\N	\N	\N	\N	\N	\N
103	37	E542K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-12-09 16:58:42.898992	2015-12-09 17:02:08.761403	f	\N	\N	3	178936082	178936082	G	A	\N	\N	\N	\N	\N	\N	\N
104	37	E545K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-12-09 16:58:42.923174	2015-12-09 17:02:08.780006	f	\N	\N	3	178936091	178936091	G	A	\N	\N	\N	\N	\N	\N	\N
107	37	H1047R	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-12-09 16:58:43.003424	2015-12-09 17:02:08.799751	f	\N	\N	3	178952085	178952085	A	G	\N	\N	\N	\N	\N	\N	\N
110	41	R233*	PTEN R233* has been shown to be a loss of function mutation, and PTEN loss has been the subject of considerable research in breast cancer. PTEN loss may sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data to support that PTEN loss is both associated with poorer prognosis, and no change in prognosis.	2015-12-09 16:58:43.081476	2015-12-09 17:02:08.8221	f	\N	\N	10	89717672	89717672	C	T	\N	\N	\N	\N	\N	\N	\N
112	42	C634W	RET C639W has been implicated as an alternate mechanism of activating RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the C639W mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib.	2015-12-09 16:58:43.135255	2015-12-09 17:02:08.848601	f	\N	\N	10	43609950	43609950	C	G	\N	\N	\N	\N	\N	\N	\N
113	42	M918T	RET M819T is the most common somatically acquired mutation in medullary thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the M918T mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T leads to more aggressive MTC with a poorer prognosis.	2015-12-09 16:58:43.159035	2015-12-09 17:02:08.879794	f	\N	\N	10	43617416	43617416	T	C	\N	\N	\N	\N	\N	\N	\N
115	44	K700E	SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients. Additionally, these mutations are the most common SF3B1 mutation observed in MDS and highly associated with subtypes of MDS that are defined by ringed sideroblasts.	2015-12-09 16:58:43.209216	2015-12-09 17:02:08.947546	f	\N	\N	2	198266834	198266834	T	C	\N	\N	\N	\N	\N	\N	\N
116	45	R175H	While loss-of-function events in TP53 are very common in cancer, the R175H variant seems not only to result in loss of tumor-suppression, but also acts as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell lines harboring this mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been shown that the R175H mutation is correlated with worse overall survival than wild-type TP53, but is not as detrimental as the R248W variant.	2015-12-09 16:58:43.238722	2015-12-09 17:02:08.994632	f	\N	\N	17	7578406	7578406	C	T	\N	\N	\N	\N	\N	\N	\N
117	45	R248Q	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival. The R248Q mutation has also shown an increased invasive behavior in cell lines. This is specific to the 248Q variant.	2015-12-09 16:58:43.270516	2015-12-09 17:02:09.044045	f	\N	\N	17	7577538	7577538	C	T	\N	\N	\N	\N	\N	\N	\N
118	45	R248W	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival.	2015-12-09 16:58:43.315699	2015-12-09 17:02:09.071514	f	\N	\N	17	7577539	7577539	G	A	\N	\N	\N	\N	\N	\N	\N
119	45	R249T	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-12-09 16:58:43.338826	2015-12-09 17:02:09.101233	f	\N	\N	17	7577535	7577535	C	G	\N	\N	\N	\N	\N	\N	\N
120	45	R249W	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-12-09 16:58:43.367981	2015-12-09 17:02:09.123792	f	\N	\N	17	7577536	7577536	T	A	\N	\N	\N	\N	\N	\N	\N
121	45	R273C	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-12-09 16:58:43.391939	2015-12-09 17:02:09.140015	f	\N	\N	17	7577121	7577121	G	A	\N	\N	\N	\N	\N	\N	\N
122	45	R273H	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-12-09 16:58:43.414558	2015-12-09 17:02:09.156885	f	\N	\N	17	7577120	7577120	C	T	\N	\N	\N	\N	\N	\N	\N
\.


--
-- Data for Name: evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_items (id, description, disease_id, source_id, variant_id, created_at, updated_at, rating, status, variant_hgvs, evidence_level, evidence_type, variant_origin, evidence_direction, clinical_significance, deleted, deleted_at, drug_interaction_type) FROM stdin;
1	In one patient with lung adenocarcinoma who had a S214C mutation, the use of sorafenib has led to more than 5 years of survival and near remission.	1	51	10	2015-12-09 16:58:44.275361	2015-12-09 16:58:44.275361	2	accepted	X:47426121-47426121 (C->G)	3	2	0	0	0	f	\N	\N
2	DNMT3A R882 mutations occur most often in de novo AML patients with intermediate cytogenic risk.	2	52	32	2015-12-09 16:58:44.367224	2015-12-09 16:58:44.367224	3	accepted	N/A	1	0	0	0	4	f	\N	\N
3	Young AML patients (<60 years old) with DNMT3A mutations (60% of which were R882) were older, had higher white blood cell counts and had higher platelet counts than patients wild-type for DNMT3A	2	53	32	2015-12-09 16:58:44.412222	2015-12-09 16:58:44.412222	3	accepted	N/A	1	0	0	0	4	f	\N	\N
4	Daunorubicin treatment resulted in similar overall survival and disease-free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	2	52	32	2015-12-09 16:58:44.45299	2015-12-09 16:58:44.45299	4	accepted	N/A	1	2	0	1	1	f	\N	\N
5	Idarubicin increases overall and disease free survival in de novo AML patients with DNMT3A R882 mutations compared to those who do not harbor this mutation.	2	52	32	2015-12-09 16:58:44.499214	2015-12-09 16:58:44.499214	4	accepted	N/A	1	2	0	0	0	f	\N	\N
6	There is no difference in the complete remission rate of de novo AML patients with DNMT3A mutation compared to those who are wild type for DNMT3A.	2	52	32	2015-12-09 16:58:44.534564	2015-12-09 16:58:44.534564	4	accepted	N/A	1	1	0	1	3	f	\N	\N
7	De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	2	54	32	2015-12-09 16:58:44.576891	2015-12-09 16:58:44.576891	4	accepted	N/A	1	1	0	0	3	f	\N	\N
8	JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).	3	55	64	2015-12-09 16:58:44.625397	2015-12-09 16:58:44.625397	4	accepted	9:5073770-5073770 (G->T)	1	0	0	0	5	f	\N	\N
9	JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical CML).	4	55	64	2015-12-09 16:58:44.684141	2015-12-09 16:58:44.684141	4	accepted	9:5073770-5073770 (G->T)	1	0	0	0	4	f	\N	\N
10	JAK2 V617F is associated with myeloid neoplasms (AML, MDS, CMML/aCML)	4	55	64	2015-12-09 16:58:44.722656	2015-12-09 16:58:44.722656	4	accepted	9:5073770-5073770 (G->T)	1	0	0	0	4	f	\N	\N
11	In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction in the percentage of cells harboring JAK2 V617F.	5	56	64	2015-12-09 16:58:44.76907	2015-12-09 16:58:44.76907	4	accepted	9:5073770-5073770 (G->T)	1	2	0	0	0	f	\N	\N
12	In acute myloid leukemia patients, D816 mutation is associated with earlier relapse and poorer prognosis than wild-type KIT	6	57	65	2015-12-09 16:58:44.81575	2015-12-09 16:58:44.81575	4	accepted	4:55599321-55599321 (A->T)	1	1	0	0	3	f	\N	\N
13	The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition by about 5 fold.	7	58	82	2015-12-09 16:58:44.908294	2015-12-09 16:58:44.908294	3	accepted	15:66729162-66729162 (C->T)	2	2	0	0	1	f	\N	\N
14	MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.	7	59	83	2015-12-09 16:58:44.960159	2015-12-09 16:58:44.960159	3	accepted	15:66727451-66727451 (A->C)	2	2	0	0	1	f	\N	\N
15	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	8	60	89	2015-12-09 16:58:45.019604	2015-12-09 16:58:45.019604	3	accepted	N/A	1	1	0	0	3	f	\N	\N
16	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	8	60	90	2015-12-09 16:58:45.059278	2015-12-09 16:58:45.059278	3	accepted	N/A	0	1	0	0	3	f	\N	\N
17	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	8	60	91	2015-12-09 16:58:45.092344	2015-12-09 16:58:45.092344	3	accepted	N/A	0	1	0	0	3	f	\N	\N
18	Likely due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic malignant melanoma with an NRAS G13D mutation	7	61	93	2015-12-09 16:58:45.139344	2015-12-09 16:58:45.139344	2	accepted	1:115258744-115258744 (C->T)	3	2	0	0	0	f	\N	\N
19	Melanoma associated with NRAS Q61 mutation was more often associated with those at the extremity than those at the trunk	7	62	94	2015-12-09 16:58:45.188279	2015-12-09 16:58:45.188279	3	accepted	N/A	1	0	0	0	4	f	\N	\N
20	Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.	7	63	94	2015-12-09 16:58:45.228578	2015-12-09 16:58:45.228578	4	accepted	N/A	1	2	0	0	1	f	\N	\N
21	In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted in disease free survival of 14 months	7	64	95	2015-12-09 16:58:45.2704	2015-12-09 16:58:45.2704	2	accepted	1:115256528-115256529 (TT->CA)	3	2	0	0	0	f	\N	\N
22	In a melanoma patient with Q61R mutation, treatment with temozolomide resulted in overall survival of 16 months	7	64	96	2015-12-09 16:58:45.325213	2015-12-09 16:58:45.325213	2	accepted	1:115256528-115256529 (TT->CC)	3	2	0	0	0	f	\N	\N
23	GIST tumors harboring PDGFRA D842V mutation are more likely to be benign than malignant.	9	65	99	2015-12-09 16:58:45.365392	2015-12-09 16:58:45.365392	3	accepted	4:55152093-55152093 (A->T)	1	0	0	0	5	f	\N	\N
24	GIST cancer with D842V mutation is resistant to imatinib.	9	66	99	2015-12-09 16:58:45.427731	2015-12-09 16:58:45.427731	4	accepted	4:55152093-55152093 (A->T)	1	2	0	0	1	f	\N	\N
25	While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib, double mutation of V561D and D842V mutants are resistant to imatinib.	9	67	99	2015-12-09 16:58:45.474456	2015-12-09 16:58:45.474456	4	accepted	4:55152093-55152093 (A->T)	1	2	0	0	1	f	\N	\N
26	Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without this mutation.	10	68	8	2015-12-09 16:58:45.541862	2015-12-09 16:58:45.541862	3	accepted	2:29443695-29443695 (G->T)	2	2	0	0	1	f	\N	\N
27	Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without this mutation.	1	68	8	2015-12-09 16:58:45.58777	2015-12-09 16:58:45.58777	3	accepted	2:29443695-29443695 (G->T)	2	2	0	0	1	f	\N	\N
28	CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.	11	69	8	2015-12-09 16:58:45.655149	2015-12-09 16:58:45.655149	3	accepted	2:29443695-29443695 (G->T)	2	2	0	0	0	f	\N	\N
29	High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing the EML4-ALK fusion containing the F1174L mutation.	11	70	8	2015-12-09 16:58:45.706401	2015-12-09 16:58:45.706401	3	accepted	2:29443695-29443695 (G->T)	2	2	0	0	0	f	\N	\N
30	TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.	11	71	9	2015-12-09 16:58:45.749486	2015-12-09 16:58:45.749486	3	accepted	2:29432664-29432664 (C->T)	2	2	0	0	0	f	\N	\N
31	The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.	11	71	9	2015-12-09 16:58:45.788289	2015-12-09 16:58:45.788289	3	accepted	2:29432664-29432664 (C->T)	2	2	0	0	1	f	\N	\N
32	Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	1	51	10	2015-12-09 16:58:45.823317	2015-12-09 16:58:45.823317	3	accepted	X:47426121-47426121 (C->G)	2	2	0	0	0	f	\N	\N
33	Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	1	51	10	2015-12-09 16:58:45.864402	2015-12-09 16:58:45.864402	3	accepted	X:47426121-47426121 (C->G)	2	2	0	0	0	f	\N	\N
34	BRAF mutations are associated with melanoma arising in non-chronic sun damaged skin and with superficial spreading melanoma	7	72	17	2015-12-09 16:58:45.916509	2015-12-09 16:58:45.916509	4	accepted	N/A	1	0	0	0	4	f	\N	\N
35	In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations were associated with reduced relapse free survival in the entire cohort as well as the subset of patients with cytogenetically normal AML	2	73	32	2015-12-09 16:58:45.965228	2015-12-09 16:58:45.965228	3	accepted	N/A	0	1	0	0	3	f	\N	\N
36	DNMT3A R882 mutations were associated with older age, higher white blood cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically normal AML patients	2	74	32	2015-12-09 16:58:46.02578	2015-12-09 16:58:46.02578	3	accepted	N/A	1	0	0	0	4	f	\N	\N
37	In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5% of which were R882) had no prognostic value on overall, relapse free and event free survival.	2	73	32	2015-12-09 16:58:46.066414	2015-12-09 16:58:46.066414	3	accepted	N/A	1	1	0	1	6	f	\N	\N
38	In a large cohort of cytogenetically normal AML patients (18-60 years old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall, relapse free and event free survival	2	73	32	2015-12-09 16:58:46.103448	2015-12-09 16:58:46.103448	3	accepted	N/A	1	1	0	1	6	f	\N	\N
39	In young AML patients (<60 years old), DNMT3A mutation status (60% of which were R882) was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	2	53	32	2015-12-09 16:58:46.139609	2015-12-09 16:58:46.139609	4	accepted	N/A	1	1	0	1	6	f	\N	\N
40	AML patients with DNMT3A mutations (59% of which were R882) showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	2	54	32	2015-12-09 16:58:46.178065	2015-12-09 16:58:46.178065	5	accepted	N/A	1	1	0	0	3	f	\N	\N
41	DNMT3A R882 mutation was associated with reduced relapse free and overall survival in ELN-unfavorable, cytogenetically normal AMLs	2	73	32	2015-12-09 16:58:46.220524	2015-12-09 16:58:46.220524	3	accepted	N/A	1	1	0	0	3	f	\N	\N
42	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including R882) were associated with worse overall survival compared to those without DNMT3A mutation.	2	54	32	2015-12-09 16:58:46.255738	2015-12-09 16:58:46.255738	5	accepted	N/A	1	1	0	0	3	f	\N	\N
43	In cytogenetically normal AML patients, DNMT3A R882 mutations are associated with lower overall and disease free survival as compared to patients with wild type DNMT3A.	2	74	32	2015-12-09 16:58:46.293105	2015-12-09 16:58:46.293105	5	accepted	N/A	1	1	0	0	3	f	\N	\N
44	In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation is prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	2	75	32	2015-12-09 16:58:46.338748	2015-12-09 16:58:46.338748	4	accepted	N/A	1	1	0	0	3	f	\N	\N
45	In young AML patients (<60 years old), DNMT3A mutations were associated with significantly reduced overall survival and relapse free survival in patients wildtype for NPM1 and FLT3	2	53	32	2015-12-09 16:58:46.37066	2015-12-09 16:58:46.37066	4	accepted	N/A	1	1	0	0	3	f	\N	\N
46	In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations other than R882 are prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	2	75	32	2015-12-09 16:58:46.40837	2015-12-09 16:58:46.40837	4	accepted	N/A	1	1	0	0	3	f	\N	\N
47	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	2	53	55	2015-12-09 16:58:46.446001	2015-12-09 16:58:46.446001	4	accepted	N/A	1	1	0	1	6	f	\N	\N
48	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated with worse overall survival compared to those without DNMT3A mutation.	2	54	55	2015-12-09 16:58:46.481103	2015-12-09 16:58:46.481103	5	accepted	N/A	1	1	0	0	3	f	\N	\N
49	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	12	76	66	2015-12-09 16:58:46.521959	2015-12-09 16:58:46.521959	3	accepted	N/A	1	1	0	1	6	f	\N	\N
50	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	7	77	66	2015-12-09 16:58:46.588265	2015-12-09 16:58:46.588265	3	accepted	N/A	1	1	0	0	2	f	\N	\N
51	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	7	77	68	2015-12-09 16:58:46.644408	2015-12-09 16:58:46.644408	3	accepted	N/A	1	1	0	0	2	f	\N	\N
52	GIST patients with exon 14 KIT mutations had reduced overall survival compared to patients wiltype for KIT	12	76	69	2015-12-09 16:58:46.685717	2015-12-09 16:58:46.685717	3	accepted	N/A	1	1	0	0	3	f	\N	\N
53	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	7	77	70	2015-12-09 16:58:46.717492	2015-12-09 16:58:46.717492	3	accepted	N/A	1	1	0	0	2	f	\N	\N
54	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	12	76	71	2015-12-09 16:58:46.758077	2015-12-09 16:58:46.758077	3	accepted	N/A	1	1	0	1	6	f	\N	\N
55	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	7	78	72	2015-12-09 16:58:46.799907	2015-12-09 16:58:46.799907	3	accepted	N/A	2	2	0	0	1	f	\N	2
56	The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation of KIT but did not alter the binding affinity of dasatinib.	7	78	72	2015-12-09 16:58:46.856704	2015-12-09 16:58:46.856704	3	accepted	N/A	2	2	0	0	0	f	\N	\N
57	In NSCLC, Exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	1	79	75	2015-12-09 16:58:46.906784	2015-12-09 16:58:46.906784	3	accepted	N/A	1	2	0	0	1	f	\N	2
58	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	2	53	86	2015-12-09 16:58:46.947381	2015-12-09 16:58:46.947381	4	accepted	N/A	1	1	0	1	6	f	\N	\N
59	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	7	80	88	2015-12-09 16:58:46.992738	2015-12-09 16:58:46.992738	3	accepted	N/A	1	1	0	0	3	f	\N	\N
60	The presence of NRAS mutation in AML patients does not impact diease prognosis (resistant disease, disease-free survival, complete remission rate, relapse rate, induction death)	2	81	89	2015-12-09 16:58:47.037417	2015-12-09 16:58:47.037417	3	accepted	N/A	1	1	0	1	6	f	\N	\N
61	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	7	80	89	2015-12-09 16:58:47.085133	2015-12-09 16:58:47.085133	3	accepted	N/A	1	1	0	0	3	f	\N	\N
62	NRAS mutation status does not impact overall, event-free or disease-free survival in patients with AML	2	82	92	2015-12-09 16:58:47.137877	2015-12-09 16:58:47.137877	3	accepted	N/A	1	1	0	1	6	f	\N	\N
63	Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation (primarily Q61) have a significantly lower response rate to cetuximab than patients wildtype for NRAS.	8	83	94	2015-12-09 16:58:47.178418	2015-12-09 16:58:47.178418	3	accepted	N/A	1	2	0	0	1	f	\N	\N
64	In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	9	84	98	2015-12-09 16:58:47.219437	2015-12-09 16:58:47.219437	4	accepted	4:55152092-55152093 (GA->AT)	2	2	0	0	0	f	\N	\N
65	In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	9	84	99	2015-12-09 16:58:47.264626	2015-12-09 16:58:47.264626	4	accepted	4:55152093-55152093 (A->T)	2	2	0	0	0	f	\N	\N
66	In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	9	84	100	2015-12-09 16:58:47.322694	2015-12-09 16:58:47.322694	4	accepted	4:55152092-55152092 (G->T)	2	2	0	0	0	f	\N	\N
67	In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	9	84	101	2015-12-09 16:58:47.369317	2015-12-09 16:58:47.369317	4	accepted	N/A	2	2	0	0	0	f	\N	\N
68	In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	9	84	102	2015-12-09 16:58:47.414115	2015-12-09 16:58:47.414115	4	accepted	N/A	2	2	0	0	0	f	\N	\N
69	A single patient with BRAF V600E/V600M bi-allelic mutation responded to the V600E drug dabrafenib.	7	85	13	2015-12-09 16:58:47.463785	2015-12-09 16:58:47.463785	3	accepted	N/A	3	2	0	0	0	f	\N	\N
70	Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib, compared to wild-type RET.	13	86	112	2015-12-09 16:58:47.534049	2015-12-09 16:58:47.534049	3	accepted	10:43609950-43609950 (C->G)	2	2	0	0	1	f	\N	\N
71	In patients with medullary carcinoma, the presence of RET M918T mutation is associated with increased probability of lymph node metastases.	13	87	113	2015-12-09 16:58:47.607239	2015-12-09 16:58:47.607239	5	accepted	10:43617416-43617416 (T->C)	1	0	0	0	4	f	\N	\N
72	Compared to those who harbor a wild type RET, patients with RET M918T mutation develop larger and more aggressive medullary thyroid cancer.	13	88	113	2015-12-09 16:58:47.650192	2015-12-09 16:58:47.650192	4	accepted	10:43617416-43617416 (T->C)	1	1	0	0	3	f	\N	\N
73	Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib, compared to wild-type RET.	13	86	113	2015-12-09 16:58:47.682685	2015-12-09 16:58:47.682685	3	accepted	10:43617416-43617416 (T->C)	2	2	0	0	1	f	\N	\N
74	The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong repression of tyroid cancer cell growth.	13	89	113	2015-12-09 16:58:47.73358	2015-12-09 16:58:47.73358	3	accepted	10:43617416-43617416 (T->C)	2	2	0	0	0	f	\N	\N
75	In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations were associated with poor progression free survival regardless of treatment (panitumumab with best supportive care or best supportive care alone)	8	90	17	2015-12-09 16:58:47.783793	2015-12-09 16:58:47.783793	3	accepted	N/A	1	2	0	0	1	f	\N	\N
76	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	7	91	17	2015-12-09 16:58:47.826843	2015-12-09 16:58:47.826843	3	accepted	N/A	1	2	0	0	0	f	\N	0
77	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	7	92	11	2015-12-09 16:58:47.874203	2015-12-09 16:58:47.874203	5	accepted	7:140453135-140453136 (CA->AT)	1	2	0	0	0	f	\N	\N
78	V600E is associated with adverse pathological features of colorectal cancer. This can be concluded as a marker of poor prognosis.	8	93	12	2015-12-09 16:58:47.924447	2015-12-09 16:58:47.924447	5	accepted	7:140453136-140453136 (A->T)	0	1	0	0	3	f	\N	\N
79	BRAF V600E is shown to be associated with the tall-cell variant of papillary thyroid cancer	14	94	12	2015-12-09 16:58:47.989378	2015-12-09 16:58:47.989378	3	accepted	7:140453136-140453136 (A->T)	1	0	0	0	4	f	\N	\N
80	Thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	14	95	12	2015-12-09 16:58:48.039691	2015-12-09 16:58:48.039691	5	accepted	7:140453136-140453136 (A->T)	1	0	0	0	4	f	\N	\N
81	Thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	14	96	12	2015-12-09 16:58:48.088786	2015-12-09 16:58:48.088786	5	accepted	7:140453136-140453136 (A->T)	1	0	0	0	4	f	\N	\N
82	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	7	97	12	2015-12-09 16:58:48.1301	2015-12-09 16:58:48.1301	2	accepted	7:140453136-140453136 (A->T)	1	2	0	1	6	f	\N	2
83	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	7	98	12	2015-12-09 16:58:48.193645	2015-12-09 16:58:48.193645	5	accepted	7:140453136-140453136 (A->T)	1	2	0	0	0	f	\N	0
84	Unlike other studies that suggest a poorer outcome, BRAF mutation in this study was not correlated with poorer prognosis in papillary thyroid cancer.	14	99	12	2015-12-09 16:58:48.257543	2015-12-09 16:58:48.257543	5	accepted	7:140453136-140453136 (A->T)	1	1	0	1	3	f	\N	\N
85	BRAF V600E is correlated with shorter disease-free and overall Survival in a Spanish cohort of melanoma patients.	7	100	12	2015-12-09 16:58:48.308517	2015-12-09 16:58:48.308517	3	accepted	7:140453136-140453136 (A->T)	1	1	0	0	3	f	\N	\N
86	BRAF mutation correlated with poor prognosis in papillary thyroid cancer in both older (>65 yo) and younger (<65 yo) cohorts.	14	94	12	2015-12-09 16:58:48.346457	2015-12-09 16:58:48.346457	3	accepted	7:140453136-140453136 (A->T)	1	1	0	0	3	f	\N	\N
87	BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in a study of 187 patients with PTC and other thyroid diseases.	14	101	12	2015-12-09 16:58:48.387012	2015-12-09 16:58:48.387012	3	accepted	7:140453136-140453136 (A->T)	1	1	0	0	3	f	\N	\N
88	V600E is correlated with disease recurrence in both age cohorts (>65 and <65 yo).	14	94	12	2015-12-09 16:58:48.422659	2015-12-09 16:58:48.422659	3	accepted	7:140453136-140453136 (A->T)	1	1	0	0	3	f	\N	\N
117	DNMT3A mutations (59% of which were R882) were associated with intermediate risk cytogenetics (including normal karyotype)	2	54	32	2015-12-09 16:58:49.964299	2015-12-09 16:58:49.964299	4	accepted	N/A	1	0	0	0	4	f	\N	\N
89	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP but does not show resistance to MEK inhibitors.	7	102	12	2015-12-09 16:58:48.462042	2015-12-09 16:58:48.462042	3	accepted	7:140453136-140453136 (A->T)	2	2	0	1	1	f	\N	\N
90	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	15	103	12	2015-12-09 16:58:48.503317	2015-12-09 16:58:48.503317	3	accepted	7:140453136-140453136 (A->T)	2	2	0	0	1	f	\N	\N
91	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP and resistance to RAF inhibitors.	7	102	12	2015-12-09 16:58:48.535556	2015-12-09 16:58:48.535556	3	accepted	7:140453136-140453136 (A->T)	2	2	0	0	1	f	\N	\N
92	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	15	104	12	2015-12-09 16:58:48.577398	2015-12-09 16:58:48.577398	3	accepted	7:140453136-140453136 (A->T)	2	2	0	0	0	f	\N	0
93	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	15	105	12	2015-12-09 16:58:48.648631	2015-12-09 16:58:48.648631	2	accepted	7:140453136-140453136 (A->T)	2	2	0	0	0	f	\N	0
94	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	8	106	12	2015-12-09 16:58:48.697739	2015-12-09 16:58:48.697739	2	accepted	7:140453136-140453136 (A->T)	2	2	0	0	0	f	\N	0
95	Cell lines expressing the BRAF V600E mutation responded better to vemurafenib treatment than cells wildtype for BRAF as measured by reduced cellular proliferation and inhibition of MET and ERK phosphorylation	8	106	12	2015-12-09 16:58:48.742413	2015-12-09 16:58:48.742413	2	accepted	7:140453136-140453136 (A->T)	2	2	0	0	0	f	\N	\N
96	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	1	107	12	2015-12-09 16:58:48.787028	2015-12-09 16:58:48.787028	2	accepted	7:140453136-140453136 (A->T)	3	2	0	0	1	f	\N	\N
97	COLO201 and COLO206F cells harboring BRAF V600E mutations were cloned to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification of the BRAF V600E gene.	8	108	14	2015-12-09 16:58:48.826103	2015-12-09 16:58:48.826103	4	accepted	N/A	4	2	0	0	1	f	\N	\N
98	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	7	92	15	2015-12-09 16:58:48.867927	2015-12-09 16:58:48.867927	5	accepted	7:140453137-140453137 (C->T)	1	2	0	0	0	f	\N	\N
99	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	7	92	16	2015-12-09 16:58:48.914008	2015-12-09 16:58:48.914008	5	accepted	7:140453136-140453137 (AC->CT)	1	2	0	0	0	f	\N	\N
100	KRAS mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	8	109	75	2015-12-09 16:58:48.961087	2015-12-09 16:58:48.961087	4	accepted	N/A	1	2	0	1	0	f	\N	\N
101	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	8	109	103	2015-12-09 16:58:49.008914	2015-12-09 16:58:49.008914	4	accepted	3:178936082-178936082 (G->A)	1	2	0	1	6	f	\N	\N
102	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	8	109	104	2015-12-09 16:58:49.061965	2015-12-09 16:58:49.061965	4	accepted	3:178936091-178936091 (G->A)	1	2	0	1	6	f	\N	\N
103	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	8	109	107	2015-12-09 16:58:49.128548	2015-12-09 16:58:49.128548	4	accepted	3:178952085-178952085 (A->G)	1	2	0	1	6	f	\N	\N
104	CH5424802 treatment resulted in significant tumor regression in xenograft models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation	1	69	7	2015-12-09 16:58:49.208096	2015-12-09 16:58:49.208096	3	accepted	N/A	2	2	0	0	0	f	\N	\N
105	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation.	11	110	8	2015-12-09 16:58:49.28362	2015-12-09 16:58:49.28362	3	accepted	2:29443695-29443695 (G->T)	2	2	0	0	1	f	\N	\N
106	SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition	11	71	8	2015-12-09 16:58:49.359703	2015-12-09 16:58:49.359703	4	accepted	2:29443695-29443695 (G->T)	2	2	0	0	0	f	\N	\N
107	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation	11	110	9	2015-12-09 16:58:49.42446	2015-12-09 16:58:49.42446	3	accepted	2:29432664-29432664 (C->T)	2	2	0	0	0	f	\N	\N
108	Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype BRAF	8	83	12	2015-12-09 16:58:49.500604	2015-12-09 16:58:49.500604	3	accepted	7:140453136-140453136 (A->T)	1	2	0	0	1	f	\N	\N
109	CEBPA mutation status had no impact on ATRA treatment response in older (>60) patients with AML	2	111	29	2015-12-09 16:58:49.566841	2015-12-09 16:58:49.566841	4	accepted	N/A	1	2	0	1	6	f	\N	\N
110	CEBPA mutation was associated with improved overall survival in older (>60) patients with AML	2	111	29	2015-12-09 16:58:49.616635	2015-12-09 16:58:49.616635	4	accepted	N/A	1	1	0	0	2	f	\N	\N
111	CEBPA mutation was significantly associated with complete remission	2	112	29	2015-12-09 16:58:49.680083	2015-12-09 16:58:49.680083	4	accepted	N/A	1	1	0	0	2	f	\N	\N
112	FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically normal AML patients with CEBPA	2	113	28	2015-12-09 16:58:49.744515	2015-12-09 16:58:49.744515	2	accepted	N/A	1	1	0	1	6	f	\N	\N
113	Overall survival was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA mutations	2	113	28	2015-12-09 16:58:49.795124	2015-12-09 16:58:49.795124	3	accepted	N/A	1	1	0	0	2	f	\N	\N
114	Remission duration was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA	2	113	28	2015-12-09 16:58:49.839628	2015-12-09 16:58:49.839628	3	accepted	N/A	1	1	0	0	2	f	\N	\N
115	Therapy-related AML was less common in patients with DNMT3A mutations (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of younger (18-60) AML patients	2	73	32	2015-12-09 16:58:49.873635	2015-12-09 16:58:49.873635	3	accepted	N/A	1	0	0	0	5	f	\N	\N
116	DNMT3A mutations (59% of which were R882) were associated with an intermediate risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes	2	54	32	2015-12-09 16:58:49.922254	2015-12-09 16:58:49.922254	5	accepted	N/A	1	0	0	0	4	f	\N	\N
118	DNMT3A mutations (64.5% R882) were associated with older age, higher white blood cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML patients	2	73	32	2015-12-09 16:58:50.004361	2015-12-09 16:58:50.004361	3	accepted	N/A	1	0	0	0	4	f	\N	\N
119	DNMT3A R882 mutations were associated with cytogenetically normal AML in a large cohort of younger (18-60) AML patients	2	73	32	2015-12-09 16:58:50.044978	2015-12-09 16:58:50.044978	3	accepted	N/A	1	0	0	0	4	f	\N	\N
120	Complete remission rates did not differ between patients with wildtype or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML	2	75	32	2015-12-09 16:58:50.085783	2015-12-09 16:58:50.085783	4	accepted	N/A	1	1	0	1	6	f	\N	\N
121	DNMT3A mutations were associated with achievement of complete remission in a large cohort of younger (18-60) AML patients	2	73	32	2015-12-09 16:58:50.124	2015-12-09 16:58:50.124	3	accepted	N/A	1	1	0	0	2	f	\N	\N
122	Complete remission rate was not different between young AML patients (<60 years old) with or without DNMT3A mutations (60% of which were R882)	2	53	32	2015-12-09 16:58:50.16392	2015-12-09 16:58:50.16392	3	accepted	N/A	1	1	0	0	3	f	\N	\N
123	DNMT3A mutations (62% of which were R882) were associated with reduced disease-free survival in patients with cytogenetically normal AML.	2	75	32	2015-12-09 16:58:50.199198	2015-12-09 16:58:50.199198	4	accepted	N/A	1	1	0	0	3	f	\N	\N
124	Young AML patients (<60 years old) with DNMT3A mutation have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	2	53	32	2015-12-09 16:58:50.252623	2015-12-09 16:58:50.252623	3	accepted	N/A	1	1	0	0	3	f	\N	\N
125	Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	2	53	32	2015-12-09 16:58:50.29308	2015-12-09 16:58:50.29308	3	accepted	N/A	1	1	0	0	3	f	\N	\N
126	Following identification of this fusion in a patient, this fusion was expressed in HEK 293T cells leading to oligomerization in the absence of ligand, induced morphologic changes in the cell and increased cellular proliferation consistent with FGFR activation. Mouse xenografts using the bladder cell line SW780 which harbors an FGFR3-BAIAP2L1 fusion showed reduced tumor burden when treated with the FGFR inhibitor PD173074.	16	114	53	2015-12-09 16:58:50.34283	2015-12-09 16:58:50.34283	3	accepted	N/A	2	2	0	0	0	f	\N	\N
127	Meta-analysis of studies involving cytogentically normal younger (<60) patients showed reduced overall and relapse-free survival for patients with FLT3-ITD	2	115	55	2015-12-09 16:58:50.409032	2015-12-09 16:58:50.409032	2	accepted	N/A	0	1	0	0	3	f	\N	\N
128	ATRA treatment did not effect overall survival in patients <60 years old with FLT3-ITD mutation	2	116	55	2015-12-09 16:58:50.457515	2015-12-09 16:58:50.457515	3	accepted	N/A	1	2	0	0	1	f	\N	\N
129	FLT3-ITD mutation without NPM1 was associated with increased relapse risk and reduced overall survival in young adult AML patients (median age 43)	2	117	55	2015-12-09 16:58:50.50715	2015-12-09 16:58:50.50715	4	accepted	N/A	1	2	0	0	1	f	\N	\N
130	Increasing level of FLT3-ITD mutant correlated with reduced disease-free and overall survival as well as increased relapse rate	2	117	55	2015-12-09 16:58:50.544133	2015-12-09 16:58:50.544133	3	accepted	N/A	1	2	0	0	1	f	\N	\N
131	FLT3-ITD duplication length did not effect prognosis in patients with an NPM1 mutation	2	118	55	2015-12-09 16:58:50.584521	2015-12-09 16:58:50.584521	4	accepted	N/A	1	1	0	1	6	f	\N	\N
132	Size of the FLT3-ITD mutant duplication had no impact on overall survival or relapse rate	2	117	55	2015-12-09 16:58:50.622639	2015-12-09 16:58:50.622639	2	accepted	N/A	1	1	0	0	6	f	\N	\N
133	A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation	2	118	55	2015-12-09 16:58:50.663556	2015-12-09 16:58:50.663556	4	accepted	N/A	1	1	0	0	3	f	\N	\N
134	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	2	118	55	2015-12-09 16:58:50.700882	2015-12-09 16:58:50.700882	4	accepted	N/A	1	1	0	0	3	f	\N	\N
135	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	2	118	55	2015-12-09 16:58:50.7368	2015-12-09 16:58:50.7368	4	accepted	N/A	1	1	0	0	3	f	\N	\N
136	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	2	119	55	2015-12-09 16:58:50.787457	2015-12-09 16:58:50.787457	3	accepted	N/A	2	2	0	0	1	f	\N	\N
137	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	2	119	55	2015-12-09 16:58:50.824947	2015-12-09 16:58:50.824947	3	accepted	N/A	2	2	0	0	1	f	\N	\N
138	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	2	120	56	2015-12-09 16:58:50.875172	2015-12-09 16:58:50.875172	3	accepted	N/A	1	1	0	0	2	f	\N	\N
139	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	2	119	56	2015-12-09 16:58:50.929643	2015-12-09 16:58:50.929643	3	accepted	N/A	2	2	0	0	1	f	\N	\N
140	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	2	119	56	2015-12-09 16:58:50.968903	2015-12-09 16:58:50.968903	3	accepted	N/A	2	2	0	0	1	f	\N	\N
141	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	2	121	58	2015-12-09 16:58:51.020812	2015-12-09 16:58:51.020812	3	accepted	N/A	1	1	0	1	6	f	\N	\N
142	Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations (primarily G12/G13) had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype KRAS	8	83	77	2015-12-09 16:58:51.056625	2015-12-09 16:58:51.056625	3	accepted	N/A	1	2	0	0	1	f	\N	\N
143	KRAS mutations were significantly associated with lack of response to cetuximab in patients with advanced colorectal cancer	8	122	77	2015-12-09 16:58:51.108096	2015-12-09 16:58:51.108096	3	accepted	N/A	2	2	0	0	1	f	\N	\N
144	KRAS mutations were significantly associated with reduced progression-free and overall survival in patients with advanced colorectal cancer	8	122	77	2015-12-09 16:58:51.144625	2015-12-09 16:58:51.144625	3	accepted	N/A	2	1	0	0	3	f	\N	\N
145	Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	15	123	81	2015-12-09 16:58:51.189022	2015-12-09 16:58:51.189022	4	accepted	12:25398281-25398281 (C->T)	2	2	0	0	1	f	\N	\N
146	Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	15	123	81	2015-12-09 16:58:51.221877	2015-12-09 16:58:51.221877	4	accepted	12:25398281-25398281 (C->T)	2	2	0	0	0	f	\N	\N
147	MGMT promoter methylation was associated with increased overall and disease-free survival as well as tumor regression in patients with gliomas.	17	124	85	2015-12-09 16:58:51.265198	2015-12-09 16:58:51.265198	4	accepted	N/A	1	1	0	0	2	f	\N	\N
148	AML with mutated NPM1 is a provisional entity in WHO classification of acute myeloid leukemia (AML). This mutation should be tested for in clinical trials and is recommended for testing in patients with cytogentically normal AML.	2	125	86	2015-12-09 16:58:51.314113	2015-12-09 16:58:51.314113	5	accepted	N/A	0	0	0	0	4	f	\N	\N
149	NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype (0/55).	2	126	86	2015-12-09 16:58:51.358975	2015-12-09 16:58:51.358975	3	accepted	N/A	1	0	0	1	4	f	\N	\N
150	NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid leukemia.	2	127	86	2015-12-09 16:58:51.402359	2015-12-09 16:58:51.402359	4	accepted	N/A	1	0	0	0	5	f	\N	\N
151	NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes	2	117	86	2015-12-09 16:58:51.439417	2015-12-09 16:58:51.439417	2	accepted	N/A	1	0	0	0	4	f	\N	\N
152	NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid leukemia.	2	127	86	2015-12-09 16:58:51.47809	2015-12-09 16:58:51.47809	4	accepted	N/A	1	0	0	0	4	f	\N	\N
153	NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid leukemia	2	126	86	2015-12-09 16:58:51.510867	2015-12-09 16:58:51.510867	3	accepted	N/A	1	0	0	0	4	f	\N	\N
154	NPM1 mutations were associated with normal karyotype in older (>60) patients	2	128	86	2015-12-09 16:58:51.573211	2015-12-09 16:58:51.573211	3	accepted	N/A	1	0	0	0	4	f	\N	\N
155	ATRA treatment did not effect overall survival in patients <60 years old with NPM1 mutation regardless of FLT3-ITD status	2	116	86	2015-12-09 16:58:51.607228	2015-12-09 16:58:51.607228	3	accepted	N/A	1	2	0	0	1	f	\N	\N
156	ATRA treatment improved overall and relapse-free survival in older (>60) patients with mutant NPM1 and without FLT3-ITD	2	128	86	2015-12-09 16:58:51.643894	2015-12-09 16:58:51.643894	2	accepted	N/A	1	2	0	0	0	f	\N	\N
157	Young patients (18-60) with NPM1 mutations had improved overall survival following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However, the adjusted p-value for multiple testing is 0.11.	2	129	86	2015-12-09 16:58:51.6921	2015-12-09 16:58:51.6921	2	accepted	N/A	1	2	0	0	0	f	\N	\N
158	For patients with NPM1 mutation that achieved CR following induction therapy, relapse-free survival was improved following treatment with valproic acid	2	130	86	2015-12-09 16:58:51.736692	2015-12-09 16:58:51.736692	2	accepted	N/A	1	2	0	0	0	f	\N	\N
159	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	2	121	86	2015-12-09 16:58:51.775864	2015-12-09 16:58:51.775864	3	accepted	N/A	1	1	0	1	6	f	\N	\N
160	NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and increased overall survival in young adult acute myeloid leukemia patients (median age 43)	2	117	86	2015-12-09 16:58:51.81421	2015-12-09 16:58:51.81421	4	accepted	N/A	1	1	0	0	2	f	\N	\N
161	Patients with NPM1 mutations were associated with improved complete remission rates as well as overall and disease free survival when all ages and karyotype were analyzed; however, when only normal karyotype patients were analyzed, only complete remission rates were improved	2	128	86	2015-12-09 16:58:51.848385	2015-12-09 16:58:51.848385	3	accepted	N/A	1	1	0	0	2	f	\N	\N
162	Complete remission rates were higher and both disease-free and overall survival were longer for patients with Exon 12 NPM1 mutations.	2	126	86	2015-12-09 16:58:51.884512	2015-12-09 16:58:51.884512	3	accepted	N/A	1	1	0	0	2	f	\N	\N
163	Complete remission rates were higher and event-free survival was longer for normal karyotype AML patients with Exon 12 NPM1 mutations.	2	127	86	2015-12-09 16:58:51.919075	2015-12-09 16:58:51.919075	5	accepted	N/A	1	1	0	0	2	f	\N	\N
164	Cytoplasmic localization of NPM in AML patients was an independent prognositic factor associated with complete remission in response to induction therapy in a multivariate analysis (p=0.019) but not in a univariate analysis (p=0.07).	2	131	86	2015-12-09 16:58:51.96672	2015-12-09 16:58:51.96672	3	accepted	N/A	1	1	0	0	2	f	\N	\N
165	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	2	120	86	2015-12-09 16:58:52.009765	2015-12-09 16:58:52.009765	3	accepted	N/A	1	1	0	0	2	f	\N	\N
166	Intermediate risk patients with mutant NPM1 had improved overall survival with the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1 and IDH2	2	129	86	2015-12-09 16:58:52.06013	2015-12-09 16:58:52.06013	2	accepted	N/A	1	1	0	0	2	f	\N	\N
167	Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved overall and relapse-free survival, complete remission rates and remission duration compared to all other combinations of mutants	2	132	86	2015-12-09 16:58:52.110642	2015-12-09 16:58:52.110642	3	accepted	N/A	1	1	0	0	2	f	\N	\N
168	NPM1 mutation was associated with higher complete remission rates, lower cumulative incidence of relapse and higher overall survival in intermediate risk AML patients	2	133	86	2015-12-09 16:58:52.163188	2015-12-09 16:58:52.163188	4	accepted	N/A	1	1	0	0	2	f	\N	\N
169	NPM1 mutation was associated with increased overall, event-free and relapse-free survival and remission after induction therapy in the entire cohort and in the cytogenetically normal subset of patients	2	134	86	2015-12-09 16:58:52.215997	2015-12-09 16:58:52.215997	3	accepted	N/A	1	1	0	0	2	f	\N	\N
170	NPM1 mutation without FLT3-ITD was significantly associated with complete remission	2	112	86	2015-12-09 16:58:52.28221	2015-12-09 16:58:52.28221	3	accepted	N/A	1	1	0	0	2	f	\N	\N
171	NPM1 mutations were associated with increased complete remission rates as well as longer overall disease free survival in normal karyotype AML patients >59 years old, particularly in those >69 years old	2	135	86	2015-12-09 16:58:52.336657	2015-12-09 16:58:52.336657	4	accepted	N/A	1	1	0	0	2	f	\N	\N
172	Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and without FLT3-ITD have increased overall survival, relapse-free survival and response to induction chemotherapy	2	136	86	2015-12-09 16:58:52.38284	2015-12-09 16:58:52.38284	4	accepted	N/A	1	1	0	0	2	f	\N	\N
173	Coocurrence of FLT3-ITD mutations significantly reduced overall survival of normal karyotype AML patients	2	126	86	2015-12-09 16:58:52.426816	2015-12-09 16:58:52.426816	4	accepted	N/A	1	1	0	0	3	f	\N	\N
174	Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction in NPM1 levels following first line therapy (N=15)	2	137	86	2015-12-09 16:58:52.483524	2015-12-09 16:58:52.483524	2	accepted	N/A	1	1	0	0	3	f	\N	\N
175	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	2	118	86	2015-12-09 16:58:52.534776	2015-12-09 16:58:52.534776	4	accepted	N/A	1	1	0	0	3	f	\N	\N
176	Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate risk AML patients	2	133	86	2015-12-09 16:58:52.582483	2015-12-09 16:58:52.582483	3	accepted	N/A	1	1	0	0	3	f	\N	\N
177	TET2 mutation reduces overall survival of normal karyotype patients with either an NPM1 mutation or NPM1 mutation without FLT3-ITD	2	138	86	2015-12-09 16:58:52.63303	2015-12-09 16:58:52.63303	3	accepted	N/A	1	1	0	0	3	f	\N	\N
178	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	2	118	86	2015-12-09 16:58:52.676435	2015-12-09 16:58:52.676435	4	accepted	N/A	1	1	0	0	3	f	\N	\N
179	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	2	139	86	2015-12-09 16:58:52.725003	2015-12-09 16:58:52.725003	3	accepted	N/A	2	2	0	0	0	f	\N	0
180	CD33 and CD123 expression was significantly increased in patients with NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33 and anti-CD123 therapy	2	140	86	2015-12-09 16:58:52.774627	2015-12-09 16:58:52.774627	2	accepted	N/A	4	2	0	0	0	f	\N	\N
181	CD33 expression was significantly increased in patients with NPM1 mutation with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy	2	140	86	2015-12-09 16:58:52.813866	2015-12-09 16:58:52.813866	2	accepted	N/A	4	2	0	0	0	f	\N	\N
182	No NPM1 mutations were identified in patients with favorable risk cytogenetics (79/215 patients)	2	54	87	2015-12-09 16:58:52.852369	2015-12-09 16:58:52.852369	4	accepted	5:170837546-170837547 (0->TCAG)	1	0	0	1	4	f	\N	\N
183	NPM1 mutations were associated with intermediate risk cytogenetics (including normal karyotype)	2	54	87	2015-12-09 16:58:52.911924	2015-12-09 16:58:52.911924	4	accepted	5:170837546-170837547 (0->TCAG)	1	0	0	0	4	f	\N	\N
184	NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.	2	139	87	2015-12-09 16:58:52.962078	2015-12-09 16:58:52.962078	3	accepted	5:170837546-170837547 (0->TCAG)	2	2	0	0	0	f	\N	\N
185	NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM in AML patients was associated with good response to induction therapy.	2	131	87	2015-12-09 16:58:53.023909	2015-12-09 16:58:53.023909	3	accepted	5:170837546-170837547 (0->TCAG)	4	2	0	0	0	f	\N	\N
186	In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status (primarily Q61) was not informative for disease control rate, progression free survival or overall survival following cetuximab plus chemotherapy.	8	83	94	2015-12-09 16:58:53.067653	2015-12-09 16:58:53.067653	3	accepted	N/A	1	2	0	1	1	f	\N	\N
187	Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	8	83	105	2015-12-09 16:58:53.108495	2015-12-09 16:58:53.108495	3	accepted	N/A	1	2	0	0	1	f	\N	\N
188	In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations (Exon 9) had no significant impact on response rate, disease control rate, progression free surivival or overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	8	83	106	2015-12-09 16:58:53.147962	2015-12-09 16:58:53.147962	3	accepted	N/A	1	2	0	1	6	f	\N	\N
189	Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit increased overall survival, increased apoptosis, reduced proliferation and reduced tumor burden compared to Tsc1 wildtype mice.	1	141	124	2015-12-09 16:58:53.194225	2015-12-09 16:58:53.194225	3	accepted	N/A	2	2	0	0	0	f	\N	\N
190	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	16	142	125	2015-12-09 16:58:53.269085	2015-12-09 16:58:53.269085	3	accepted	N/A	1	2	0	0	0	f	\N	\N
191	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	2	143	129	2015-12-09 16:58:53.330335	2015-12-09 16:58:53.330335	3	accepted	N/A	1	2	0	0	1	f	\N	\N
192	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	2	144	129	2015-12-09 16:58:53.377418	2015-12-09 16:58:53.377418	3	accepted	N/A	1	1	0	1	6	f	\N	\N
193	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML	2	144	129	2015-12-09 16:58:53.417468	2015-12-09 16:58:53.417468	4	accepted	N/A	1	1	0	1	6	f	\N	\N
194	Mutations  in WT1 were associated with increased risk of recurrence in young patients (15-50) with cytogenetically normal AML	2	145	129	2015-12-09 16:58:53.464451	2015-12-09 16:58:53.464451	2	accepted	N/A	1	1	0	0	3	f	\N	\N
195	Mutations in WT1 were associated with a worse overall prognosis than patients wildtype for WT1 in young patients (15-50), primarily because of increased risk of disease recurrence	2	145	129	2015-12-09 16:58:53.503833	2015-12-09 16:58:53.503833	2	accepted	N/A	1	1	0	0	3	f	\N	\N
196	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	2	144	129	2015-12-09 16:58:53.545937	2015-12-09 16:58:53.545937	3	accepted	N/A	1	1	0	0	3	f	\N	\N
197	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	2	146	129	2015-12-09 16:58:53.589001	2015-12-09 16:58:53.589001	3	accepted	N/A	1	1	0	0	3	f	\N	\N
198	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	2	143	129	2015-12-09 16:58:53.634102	2015-12-09 16:58:53.634102	3	accepted	N/A	1	1	0	0	3	f	\N	\N
199	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	2	143	129	2015-12-09 16:58:53.69562	2015-12-09 16:58:53.69562	3	accepted	N/A	1	1	0	0	3	f	\N	\N
200	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	2	143	130	2015-12-09 16:58:53.751061	2015-12-09 16:58:53.751061	3	accepted	N/A	1	2	0	0	1	f	\N	\N
231	Cyclin D2 overexpression is associated with the increased depth of cancer invasion in gastric cancers.	23	158	21	2015-12-09 16:58:55.318414	2015-12-09 16:58:55.318414	4	accepted	N/A	2	1	0	0	3	f	\N	\N
201	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	2	144	130	2015-12-09 16:58:53.796441	2015-12-09 16:58:53.796441	3	accepted	N/A	1	1	0	1	6	f	\N	\N
202	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML.	2	144	130	2015-12-09 16:58:53.836824	2015-12-09 16:58:53.836824	4	accepted	N/A	1	1	0	1	6	f	\N	\N
203	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	2	144	130	2015-12-09 16:58:53.881057	2015-12-09 16:58:53.881057	3	accepted	N/A	1	1	0	0	3	f	\N	\N
204	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	2	143	130	2015-12-09 16:58:53.920985	2015-12-09 16:58:53.920985	3	accepted	N/A	1	1	0	0	3	f	\N	\N
205	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	2	143	130	2015-12-09 16:58:53.959225	2015-12-09 16:58:53.959225	3	accepted	N/A	1	1	0	0	3	f	\N	\N
206	WT1 mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	2	146	130	2015-12-09 16:58:53.998555	2015-12-09 16:58:53.998555	3	accepted	N/A	1	1	0	0	3	f	\N	\N
207	The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation positive women with high grade serous or undifferentiated ovarian cancer.	18	147	131	2015-12-09 16:58:54.053419	2015-12-09 16:58:54.053419	4	accepted	N/A	1	2	1	0	0	f	\N	\N
208	The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation positive women with high grade serous or undifferentiated ovarian cancer.	18	147	132	2015-12-09 16:58:54.124247	2015-12-09 16:58:54.124247	4	accepted	N/A	1	2	1	0	0	f	\N	\N
209	The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease.	4	148	1	2015-12-09 16:58:54.182891	2015-12-09 16:58:54.182891	4	accepted	N/A	0	0	0	0	4	f	\N	\N
210	Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide results in high rates of complete remission in patients.	2	149	1	2015-12-09 16:58:54.239357	2015-12-09 16:58:54.239357	5	accepted	N/A	0	2	0	0	0	f	\N	\N
211	The clinical use of imatinib in patients with BCR-ABL fusion has resulted in drastic sensitivity to the drug.	4	148	1	2015-12-09 16:58:54.293482	2015-12-09 16:58:54.293482	5	accepted	N/A	0	2	0	0	0	f	\N	\N
212	BCR-ABL inhibitors such as imatinib have lead to significantly improved prognosis, response rate, overall survival, and patient outcome in CML patients compared to previous therapeutic regimens.	4	148	1	2015-12-09 16:58:54.354808	2015-12-09 16:58:54.354808	5	accepted	N/A	0	1	0	0	2	f	\N	\N
213	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.	4	148	1	2015-12-09 16:58:54.399126	2015-12-09 16:58:54.399126	4	accepted	N/A	1	2	0	0	0	f	\N	2
214	Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML.	4	150	1	2015-12-09 16:58:54.468706	2015-12-09 16:58:54.468706	2	accepted	N/A	2	0	0	1	4	f	\N	\N
215	BCR-ABL fusions have been found to contribute to imatinib resistance in AML cell lines.	2	149	1	2015-12-09 16:58:54.51933	2015-12-09 16:58:54.51933	3	accepted	N/A	2	2	0	0	1	f	\N	\N
216	COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	4	151	3	2015-12-09 16:58:54.570399	2015-12-09 16:58:54.570399	4	accepted	N/A	2	2	0	0	1	f	\N	\N
217	In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have shown to be resistant to imatinib treatment.	4	148	2	2015-12-09 16:58:54.617184	2015-12-09 16:58:54.617184	4	accepted	N/A	1	2	0	0	1	f	\N	\N
218	COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	4	151	2	2015-12-09 16:58:54.667439	2015-12-09 16:58:54.667439	4	accepted	N/A	2	2	0	0	1	f	\N	\N
219	PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the AKT inhibitor MK-2206 in breast cancer cell lines.	19	152	4	2015-12-09 16:58:54.722335	2015-12-09 16:58:54.722335	3	accepted	14:105246551-105246551 (C->T)	2	2	0	1	0	f	\N	\N
220	In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment with crizotinib has shown to be effective in acheiving stable disease.	1	153	5	2015-12-09 16:58:54.768607	2015-12-09 16:58:54.768607	4	accepted	N/A	1	2	0	0	0	f	\N	\N
221	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y variant has been shown to confer resistance to crizotinib.	1	153	6	2015-12-09 16:58:54.827905	2015-12-09 16:58:54.827905	4	accepted	N/A	1	2	0	0	1	f	\N	\N
222	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M variant has been shown to confer resistance to crizotinib.	1	153	7	2015-12-09 16:58:54.876867	2015-12-09 16:58:54.876867	4	accepted	N/A	1	2	0	0	1	f	\N	\N
223	Inceased copy number of CCND1 is associated with poorer overall survival.	1	154	18	2015-12-09 16:58:54.934161	2015-12-09 16:58:54.934161	3	accepted	N/A	1	1	0	0	3	f	\N	\N
224	Three studies have found no significant survival impact for CCND1 in lung cancer.	1	154	19	2015-12-09 16:58:54.972068	2015-12-09 16:58:54.972068	3	accepted	N/A	1	1	0	1	3	f	\N	\N
225	Three studies have associated CCND1 expression with poorer survival.	1	154	19	2015-12-09 16:58:55.013798	2015-12-09 16:58:55.013798	3	accepted	N/A	1	1	0	0	3	f	\N	\N
226	Cyclin D1 overexpression is associated with the ER-positive subtype of breast cancer.	20	155	20	2015-12-09 16:58:55.061361	2015-12-09 16:58:55.061361	4	accepted	N/A	1	0	0	0	4	f	\N	\N
227	In ER-positive breast cancer patients, Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis.	20	155	20	2015-12-09 16:58:55.099899	2015-12-09 16:58:55.099899	4	accepted	N/A	1	1	0	0	3	f	\N	\N
228	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in head and neck squamous cell carcinoma.	21	156	20	2015-12-09 16:58:55.178913	2015-12-09 16:58:55.178913	4	accepted	N/A	1	1	0	0	3	f	\N	\N
229	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in mantle cell lymphoma.	22	157	20	2015-12-09 16:58:55.226686	2015-12-09 16:58:55.226686	4	accepted	N/A	1	1	0	0	3	f	\N	\N
230	Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.	23	158	21	2015-12-09 16:58:55.2748	2015-12-09 16:58:55.2748	4	accepted	N/A	2	1	0	0	3	f	\N	\N
232	Cyclin D2 overexpression is associated with the presence pf lymph node metastasis in gastric cancers.	23	158	21	2015-12-09 16:58:55.356755	2015-12-09 16:58:55.356755	4	accepted	N/A	2	1	0	0	3	f	\N	\N
233	Cyclin D2 overexpression is associated with vascular invasion by cancer cells in gastric cancers.	23	158	21	2015-12-09 16:58:55.391635	2015-12-09 16:58:55.391635	4	accepted	N/A	2	1	0	0	3	f	\N	\N
234	CCND2 hypomethylation is seen to be more common in stage III and IV gastric tumors than stage I and II.	23	159	22	2015-12-09 16:58:55.436586	2015-12-09 16:58:55.436586	3	accepted	N/A	2	0	0	0	5	f	\N	\N
235	Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib (PD-0332991) significantly increased median survival.	24	160	23	2015-12-09 16:58:55.481557	2015-12-09 16:58:55.481557	3	accepted	N/A	2	2	0	0	0	f	\N	\N
236	Patients with high expression of total cyclin E and low-molecular weight cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard ratio for these patients in 13.3 times higher than those with normal cyclin E levels.	20	161	24	2015-12-09 16:58:55.527073	2015-12-09 16:58:55.527073	4	accepted	N/A	1	1	0	0	3	f	\N	\N
237	Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced stage gastric carcinoma.	25	162	24	2015-12-09 16:58:55.570755	2015-12-09 16:58:55.570755	4	accepted	N/A	1	1	0	0	3	f	\N	\N
238	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	25	162	24	2015-12-09 16:58:55.608363	2015-12-09 16:58:55.608363	4	accepted	N/A	1	1	0	0	3	f	\N	\N
239	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	26	162	24	2015-12-09 16:58:55.650084	2015-12-09 16:58:55.650084	4	accepted	N/A	1	1	0	0	3	f	\N	\N
240	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	27	163	25	2015-12-09 16:58:55.699021	2015-12-09 16:58:55.699021	3	accepted	N/A	1	1	0	0	2	f	\N	\N
241	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	27	164	25	2015-12-09 16:58:55.745874	2015-12-09 16:58:55.745874	4	accepted	N/A	2	2	0	0	0	f	\N	0
242	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	27	163	26	2015-12-09 16:58:55.790721	2015-12-09 16:58:55.790721	3	accepted	N/A	1	1	0	0	2	f	\N	\N
243	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	27	164	26	2015-12-09 16:58:55.846663	2015-12-09 16:58:55.846663	4	accepted	N/A	2	2	0	0	0	f	\N	0
244	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter overall survival.	1	165	27	2015-12-09 16:58:55.900428	2015-12-09 16:58:55.900428	3	accepted	N/A	1	1	0	0	3	f	\N	\N
245	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter time to recurrence.	1	165	27	2015-12-09 16:58:55.940032	2015-12-09 16:58:55.940032	3	accepted	N/A	1	1	0	0	3	f	\N	\N
246	In 3T3 fibroblasts expressing the G253C mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	1	166	141	2015-12-09 16:58:55.987736	2015-12-09 16:58:55.987736	4	accepted	N/A	2	2	0	0	0	f	\N	\N
247	In 3T3 fibroblasts expressing the G505S mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	1	166	142	2015-12-09 16:58:56.025036	2015-12-09 16:58:56.025036	4	accepted	N/A	2	2	0	0	0	f	\N	\N
248	In 3T3 fibroblasts expressing the G774V mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	1	166	144	2015-12-09 16:58:56.061273	2015-12-09 16:58:56.061273	4	accepted	N/A	2	2	0	0	0	f	\N	\N
249	In 3T3 fibroblasts expressing the I638F mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	1	166	143	2015-12-09 16:58:56.098076	2015-12-09 16:58:56.098076	4	accepted	N/A	2	2	0	0	0	f	\N	\N
250	In 3T3 fibroblasts expressing the L239R mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	1	166	140	2015-12-09 16:58:56.134054	2015-12-09 16:58:56.134054	4	accepted	N/A	2	2	0	0	0	f	\N	\N
251	In 3T3 fibroblasts expressing the L63V mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	1	166	139	2015-12-09 16:58:56.171663	2015-12-09 16:58:56.171663	4	accepted	N/A	2	2	0	0	0	f	\N	\N
252	In a patient expressing a DDR2 S768R mutation, treatment with dasatinib plus erlotinib acheived a partial response.	1	166	145	2015-12-09 16:58:56.207136	2015-12-09 16:58:56.207136	3	accepted	N/A	3	2	0	0	0	f	\N	0
253	Exon 19 deletion has been shown to be correlated with gefitinib response.	1	167	133	2015-12-09 16:58:56.254974	2015-12-09 16:58:56.254974	4	accepted	N/A	2	2	0	0	0	f	\N	\N
254	Gefinitib has been shown to be effective in treating cell lines with G719S missense mutations.	1	168	134	2015-12-09 16:58:56.29961	2015-12-09 16:58:56.29961	4	accepted	N/A	2	2	0	0	0	f	\N	\N
255	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.	1	169	33	2015-12-09 16:58:56.349146	2015-12-09 16:58:56.349146	3	accepted	7:55259514-55259515 (CT->AG)	1	2	0	1	6	f	\N	2
256	In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence of L858R mutation is prognostic for better progression free survival.	1	170	33	2015-12-09 16:58:56.419535	2015-12-09 16:58:56.419535	4	accepted	7:55259514-55259515 (CT->AG)	1	2	0	0	0	f	\N	\N
257	Median survival of patients with EGFR L858R mutation is better than those with wild type EGFR.	1	171	33	2015-12-09 16:58:56.460172	2015-12-09 16:58:56.460172	3	accepted	7:55259514-55259515 (CT->AG)	1	1	0	0	2	f	\N	\N
258	Gefinitib has been shown to be effective in treating cell lines with L858R missense mutations.	1	168	33	2015-12-09 16:58:56.512136	2015-12-09 16:58:56.512136	4	accepted	N/A	2	2	0	0	0	f	\N	\N
259	The T790M mutation in EGFR has been shown to confer resistance to the tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are placed on the drug are likely to relapse.	28	172	34	2015-12-09 16:58:56.564693	2015-12-09 16:58:56.564693	5	accepted	7:55249071-55249071 (C->T)	0	2	0	0	1	f	\N	\N
260	In non-small cell lung cancer, the appearance of T790M mutation leads to resistance to gefitinib.	1	173	34	2015-12-09 16:58:56.635443	2015-12-09 16:58:56.635443	3	accepted	7:55249071-55249071 (C->T)	1	2	0	0	1	f	\N	\N
261	In NSCLC patients with T790M and another activating mutations, their progression free survival is shorter compared to those who do not possess T790M mutation.	1	174	34	2015-12-09 16:58:56.678657	2015-12-09 16:58:56.678657	4	accepted	7:55249071-55249071 (C->T)	1	1	0	0	3	f	\N	\N
262	Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall survival compared to patients with L858R or other exon 19 activating mutations.	1	175	34	2015-12-09 16:58:56.723436	2015-12-09 16:58:56.723436	4	accepted	7:55249071-55249071 (C->T)	1	1	0	0	3	f	\N	\N
293	AML patients with internal tandem repeat of FLT3 has worse overall survival than patients with wild type FLT3	2	183	55	2015-12-09 16:58:58.198664	2015-12-09 16:58:58.198664	3	accepted	N/A	1	1	0	0	3	f	\N	\N
263	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.	1	176	34	2015-12-09 16:58:56.767852	2015-12-09 16:58:56.767852	3	accepted	7:55249071-55249071 (C->T)	3	2	0	0	0	f	\N	0
264	In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR, especially when an L858R mutation is also present.	1	177	34	2015-12-09 16:58:56.817646	2015-12-09 16:58:56.817646	1	accepted	7:55249071-55249071 (C->T)	4	2	0	0	0	f	\N	\N
265	In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804), MET amplification and EGFR T790M were found to be the primary drivers of resistance.	1	178	34	2015-12-09 16:58:56.866597	2015-12-09 16:58:56.866597	4	accepted	N/A	1	2	0	0	1	f	\N	\N
266	In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.	20	179	35	2015-12-09 16:58:56.920526	2015-12-09 16:58:56.920526	4	accepted	17:37880261-37880261 (G->C)	2	2	0	0	0	f	\N	\N
267	In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.	20	179	36	2015-12-09 16:58:56.980054	2015-12-09 16:58:56.980054	4	accepted	17:37880261-37880261 (G->T)	2	2	0	0	0	f	\N	\N
268	In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2, cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive to neratinib.	20	179	37	2015-12-09 16:58:57.024849	2015-12-09 16:58:57.024849	5	accepted	N/A	2	2	0	0	0	f	\N	\N
269	In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.	20	179	38	2015-12-09 16:58:57.060368	2015-12-09 16:58:57.060368	5	accepted	17:37868205-37868205 (G->C)	2	2	0	0	0	f	\N	\N
270	The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	20	179	39	2015-12-09 16:58:57.104114	2015-12-09 16:58:57.104114	5	accepted	17:37880220-37880220 (T->C)	2	2	0	0	1	f	\N	\N
271	The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	20	179	39	2015-12-09 16:58:57.158588	2015-12-09 16:58:57.158588	5	accepted	17:37880220-37880220 (T->C)	2	2	0	0	0	f	\N	\N
272	MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	20	179	40	2015-12-09 16:58:57.192901	2015-12-09 16:58:57.192901	5	accepted	N/A	2	2	0	0	0	f	\N	\N
273	In MCF10A cell lines, the in-frame insertion of a single amino acid at position 780 was shown to be sensitive to neratinib.	20	179	41	2015-12-09 16:58:57.229518	2015-12-09 16:58:57.229518	5	accepted	N/A	2	2	0	0	0	f	\N	\N
274	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	20	179	42	2015-12-09 16:58:57.27069	2015-12-09 16:58:57.27069	5	accepted	N/A	2	2	0	0	0	f	\N	2
275	In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.	20	179	43	2015-12-09 16:58:57.312456	2015-12-09 16:58:57.312456	5	accepted	17:37881616-37881616 (C->T)	2	2	0	0	0	f	\N	\N
276	In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.	20	179	44	2015-12-09 16:58:57.353567	2015-12-09 16:58:57.353567	5	accepted	17:37881000-37881000 (G->T)	2	2	0	0	0	f	\N	\N
277	In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.	20	179	45	2015-12-09 16:58:57.401162	2015-12-09 16:58:57.401162	5	accepted	17:37881332-37881332 (G->A)	2	2	0	0	0	f	\N	\N
278	MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	20	180	46	2015-12-09 16:58:57.450964	2015-12-09 16:58:57.450964	3	accepted	N/A	2	2	0	0	1	f	\N	\N
279	The L536Q ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.	20	181	46	2015-12-09 16:58:57.518857	2015-12-09 16:58:57.518857	5	accepted	N/A	2	2	0	0	0	f	\N	2
280	MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	20	180	47	2015-12-09 16:58:57.570857	2015-12-09 16:58:57.570857	3	accepted	N/A	2	2	0	0	1	f	\N	\N
281	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	20	181	47	2015-12-09 16:58:57.615813	2015-12-09 16:58:57.615813	5	accepted	N/A	2	2	0	0	0	f	\N	2
282	MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	20	180	48	2015-12-09 16:58:57.659727	2015-12-09 16:58:57.659727	3	accepted	6:152419923-152419923 (A->G)	2	2	0	0	1	f	\N	\N
283	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	20	181	48	2015-12-09 16:58:57.709078	2015-12-09 16:58:57.709078	5	accepted	6:152419923-152419923 (A->G)	2	2	0	0	0	f	\N	2
284	MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	20	180	49	2015-12-09 16:58:57.770193	2015-12-09 16:58:57.770193	3	accepted	6:152419922-152419922 (T->A)	2	2	0	0	1	f	\N	\N
285	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	20	181	49	2015-12-09 16:58:57.814265	2015-12-09 16:58:57.814265	5	accepted	6:152419922-152419922 (T->A)	2	2	0	0	0	f	\N	2
286	MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	20	180	50	2015-12-09 16:58:57.860238	2015-12-09 16:58:57.860238	3	accepted	6:152419923-152419923 (A->C)	2	2	0	0	1	f	\N	\N
287	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	20	181	50	2015-12-09 16:58:57.904247	2015-12-09 16:58:57.904247	5	accepted	6:152419923-152419923 (A->C)	2	2	0	0	0	f	\N	2
288	In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic sites and overall stable disease. However, this response did not meet criteria for RECIST partial response.	29	182	52	2015-12-09 16:58:57.964904	2015-12-09 16:58:57.964904	2	accepted	N/A	3	2	0	0	0	f	\N	\N
289	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This fusion is also shown to be recurrent in multiple other cancer types.	29	182	54	2015-12-09 16:58:58.015789	2015-12-09 16:58:58.015789	3	accepted	N/A	3	2	0	0	0	f	\N	2
290	WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared to wild type	2	183	55	2015-12-09 16:58:58.070754	2015-12-09 16:58:58.070754	3	accepted	N/A	1	0	0	0	4	f	\N	\N
291	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	2	184	55	2015-12-09 16:58:58.117786	2015-12-09 16:58:58.117786	4	accepted	N/A	1	2	0	0	0	f	\N	\N
292	Sorafenib is effective in patients with FLT3 internal tandem repeats, but not in wild type or D835 mutation.	2	185	55	2015-12-09 16:58:58.163456	2015-12-09 16:58:58.163456	4	accepted	N/A	1	2	0	0	0	f	\N	\N
294	Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor survival and inferior disease free survival in patients with AML.	2	186	55	2015-12-09 16:58:58.246199	2015-12-09 16:58:58.246199	4	accepted	N/A	1	1	0	0	3	f	\N	\N
295	AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with FLT3 internal tandem repeat	2	187	55	2015-12-09 16:58:58.291393	2015-12-09 16:58:58.291393	4	accepted	N/A	2	2	0	0	0	f	\N	\N
296	WBC counts were not significantly different in AML patients with FLT3 D835 mutations compared to wild type	2	183	56	2015-12-09 16:58:58.328722	2015-12-09 16:58:58.328722	3	accepted	N/A	1	0	0	1	4	f	\N	\N
297	Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated with sorafenib resistance	2	188	56	2015-12-09 16:58:58.371149	2015-12-09 16:58:58.371149	4	accepted	N/A	1	2	0	0	1	f	\N	\N
298	Sorafenib is effective in patients with FLT3 internal tandem repeat mutations, but not in patients with wild type or D835 mutation.	2	185	56	2015-12-09 16:58:58.426663	2015-12-09 16:58:58.426663	4	accepted	N/A	1	2	0	0	1	f	\N	\N
299	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	2	184	56	2015-12-09 16:58:58.464537	2015-12-09 16:58:58.464537	4	accepted	N/A	1	2	0	0	0	f	\N	\N
300	AML patients with D835 mutation of FLT3 did not have worse overall survival than patients with wild type FLT3	2	183	56	2015-12-09 16:58:58.500448	2015-12-09 16:58:58.500448	3	accepted	N/A	1	1	0	1	6	f	\N	\N
301	In AML patients, FLT3-TKD mutation was associated with poorer disease free survival compared to patients with wild type FLT3	2	189	56	2015-12-09 16:58:58.54197	2015-12-09 16:58:58.54197	4	accepted	N/A	1	1	0	0	3	f	\N	\N
302	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	30	190	57	2015-12-09 16:58:58.589217	2015-12-09 16:58:58.589217	4	accepted	N/A	1	2	0	0	0	f	\N	0
303	In leukemia patients, IDH1 R132 mutation is associated with adults and normal karyotype.	2	191	58	2015-12-09 16:58:58.638551	2015-12-09 16:58:58.638551	2	accepted	N/A	1	0	0	0	4	f	\N	\N
304	The complete remission and overall survival rates in patients with IDH1 R132 mutation is not significantly different from those who do not have this mutation	2	192	58	2015-12-09 16:58:58.681752	2015-12-09 16:58:58.681752	2	accepted	N/A	1	1	0	1	6	f	\N	\N
305	In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic for better survival compared to patients who harbor wild type mutation.	31	193	58	2015-12-09 16:58:58.732638	2015-12-09 16:58:58.732638	3	accepted	N/A	1	1	0	0	2	f	\N	\N
306	In patients with glioblastoma, those harboring IDH1 R132 mutation has higher overall survival compared to those who do not have IDH1 mutation	32	194	58	2015-12-09 16:58:58.774851	2015-12-09 16:58:58.774851	3	accepted	N/A	1	1	0	0	2	f	\N	\N
307	IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated with worse overall survival	33	195	58	2015-12-09 16:58:58.823607	2015-12-09 16:58:58.823607	4	accepted	N/A	1	1	0	0	3	f	\N	\N
308	IDH1 R132 mutation is associated with patients of older age, high platelet count during diagnosis, cytogenic normalcy and NPM1 mutation	2	196	59	2015-12-09 16:58:58.878449	2015-12-09 16:58:58.878449	3	accepted	2:209113113-209113113 (G->A)	1	0	0	0	4	f	\N	\N
309	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	2	196	59	2015-12-09 16:58:58.92794	2015-12-09 16:58:58.92794	2	accepted	2:209113113-209113113 (G->A)	1	1	0	1	6	f	\N	\N
310	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	2	196	60	2015-12-09 16:58:58.974387	2015-12-09 16:58:58.974387	3	accepted	2:209113112-209113112 (C->T)	1	1	0	1	6	f	\N	\N
311	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	2	196	61	2015-12-09 16:58:59.02408	2015-12-09 16:58:59.02408	3	accepted	2:209113112-209113112 (C->A)	1	1	0	1	6	f	\N	\N
312	AML patients with IDH2 mutations such as R140Q/L have event free survival and overall survival similar to those with wild-type IDH2	2	197	62	2015-12-09 16:58:59.096717	2015-12-09 16:58:59.096717	2	accepted	N/A	1	1	0	1	6	f	\N	\N
313	In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact overall survival or disease free survival.	2	198	62	2015-12-09 16:58:59.152226	2015-12-09 16:58:59.152226	3	accepted	N/A	1	1	0	1	6	f	\N	\N
314	In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L do not confer prognostic value (overall survival)	34	199	62	2015-12-09 16:58:59.201613	2015-12-09 16:58:59.201613	2	accepted	N/A	1	1	0	1	6	f	\N	\N
315	R140Q mutation in IDH2 does not have prognostic value in patients with MDS.	34	200	62	2015-12-09 16:58:59.256247	2015-12-09 16:58:59.256247	3	accepted	N/A	1	1	0	1	6	f	\N	\N
316	In AML, patients with IDH2 R140K mutation have improved overall survival compared to those with wild-type IDH2	2	201	62	2015-12-09 16:58:59.322294	2015-12-09 16:58:59.322294	3	accepted	N/A	1	1	0	0	2	f	\N	\N
317	AML patients with IDH2 mutations such as R172K have event free survival and overall survival similar to those with wild-type IDH2.	2	197	63	2015-12-09 16:58:59.368573	2015-12-09 16:58:59.368573	2	accepted	15:90631838-90631838 (C->T)	1	1	0	1	6	f	\N	\N
318	In patients suffering from MDS, the presence of IDH2 mutations such as R172K do not confer prognostic value (overall survival)	34	199	63	2015-12-09 16:58:59.422398	2015-12-09 16:58:59.422398	2	accepted	15:90631838-90631838 (C->T)	1	1	0	1	6	f	\N	\N
319	In AML, patients with an IDH2 R172K mutation have worse overall survival compared to those with wild-type IDH2	2	201	63	2015-12-09 16:58:59.470366	2015-12-09 16:58:59.470366	3	accepted	15:90631838-90631838 (C->T)	1	1	0	0	3	f	\N	\N
320	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	12	202	66	2015-12-09 16:58:59.524656	2015-12-09 16:58:59.524656	2	accepted	N/A	1	0	0	0	4	f	\N	\N
321	KIT mutations were identified in morphologically benign, incidentally discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating is not a prognostic marker for GIST	12	203	66	2015-12-09 16:58:59.592178	2015-12-09 16:58:59.592178	3	accepted	N/A	1	1	0	1	6	f	\N	\N
322	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	26	204	66	2015-12-09 16:58:59.662416	2015-12-09 16:58:59.662416	2	accepted	N/A	1	1	0	1	6	f	\N	\N
323	KIT mutation is associated with worse overall and cause-specific prognosis in patients with GIST compared to patients with wildtype KIT	12	202	66	2015-12-09 16:58:59.708187	2015-12-09 16:58:59.708187	3	accepted	N/A	1	1	0	0	3	f	\N	\N
324	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	12	202	70	2015-12-09 16:58:59.75667	2015-12-09 16:58:59.75667	3	accepted	N/A	1	0	0	0	4	f	\N	\N
325	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	26	204	71	2015-12-09 16:58:59.808561	2015-12-09 16:58:59.808561	2	accepted	N/A	1	1	0	1	6	f	\N	\N
326	In a case study, a patient with anal melanoma harboring a 7 codon duplication in the juxtamambrane region (exon 11) of KIT showed marked response 4 months after imatinib treatment.	35	205	67	2015-12-09 16:58:59.886239	2015-12-09 16:58:59.886239	3	accepted	N/A	3	2	0	0	0	f	\N	\N
327	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	1	206	72	2015-12-09 16:58:59.959965	2015-12-09 16:58:59.959965	3	accepted	N/A	2	2	0	0	0	f	\N	2
328	In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective for those that are refractory to imatinib.	12	207	73	2015-12-09 16:59:00.024617	2015-12-09 16:59:00.024617	3	accepted	4:55594258-55594258 (T->C)	1	2	0	0	0	f	\N	\N
329	KIT V654A results in imatinib resistance in GIST patient-derived cell lines.	12	67	73	2015-12-09 16:59:00.08227	2015-12-09 16:59:00.08227	3	accepted	4:55594258-55594258 (T->C)	2	2	0	0	1	f	\N	\N
330	Patients with colorectal cancer who harbor KRAS mutation have low response rate to cetuximab	8	208	74	2015-12-09 16:59:00.149567	2015-12-09 16:59:00.149567	4	accepted	N/A	1	2	0	0	1	f	\N	\N
331	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	1	209	76	2015-12-09 16:59:00.212651	2015-12-09 16:59:00.212651	3	accepted	N/A	1	0	0	1	4	f	\N	\N
332	In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12 mutation did not impact overall or disease free survival.	15	210	76	2015-12-09 16:59:00.273581	2015-12-09 16:59:00.273581	4	accepted	N/A	1	1	0	1	6	f	\N	\N
333	There is no strong association between KRAS mutation status and overall or progression free survival in patients with NSCLC	1	211	76	2015-12-09 16:59:00.325025	2015-12-09 16:59:00.325025	3	accepted	N/A	1	1	0	1	6	f	\N	\N
334	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	36	212	76	2015-12-09 16:59:00.376386	2015-12-09 16:59:00.376386	3	accepted	N/A	1	1	0	0	3	f	\N	\N
335	A meta-analysis showed KRAS mutation is associated with worse outcome in patients with NSCLC	1	213	76	2015-12-09 16:59:00.421511	2015-12-09 16:59:00.421511	3	accepted	N/A	1	1	0	0	3	f	\N	\N
336	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	37	214	148	2015-12-09 16:59:00.472542	2015-12-09 16:59:00.472542	3	accepted	N/A	1	2	0	0	1	f	\N	2
337	KRAS G12C occur more frequently in women than men	26	215	78	2015-12-09 16:59:00.53132	2015-12-09 16:59:00.53132	2	accepted	12:25398285-25398285 (C->A)	1	0	0	0	4	f	\N	\N
338	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	37	214	78	2015-12-09 16:59:00.573411	2015-12-09 16:59:00.573411	3	accepted	N/A	1	2	0	0	1	f	\N	2
339	KRAS G12D mutation occurs in never smokers significantly more often than in smokers	26	215	79	2015-12-09 16:59:00.641418	2015-12-09 16:59:00.641418	3	accepted	12:25398284-25398284 (C->T)	1	0	0	0	4	f	\N	\N
340	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.	1	216	79	2015-12-09 16:59:00.699226	2015-12-09 16:59:00.699226	4	accepted	12:25398284-25398284 (C->T)	2	2	0	0	0	f	\N	0
341	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	37	214	79	2015-12-09 16:59:00.755888	2015-12-09 16:59:00.755888	3	accepted	N/A	1	2	0	0	1	f	\N	2
342	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	37	214	147	2015-12-09 16:59:00.816729	2015-12-09 16:59:00.816729	3	accepted	N/A	1	2	0	0	1	f	\N	2
343	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	1	209	80	2015-12-09 16:59:00.882813	2015-12-09 16:59:00.882813	3	accepted	N/A	1	0	0	1	4	f	\N	\N
344	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	38	212	80	2015-12-09 16:59:00.936177	2015-12-09 16:59:00.936177	3	accepted	N/A	1	1	0	0	3	f	\N	\N
345	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	15	123	81	2015-12-09 16:59:00.979455	2015-12-09 16:59:00.979455	4	accepted	12:25398281-25398281 (C->T)	1	2	0	0	0	f	\N	\N
346	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	15	123	81	2015-12-09 16:59:01.026007	2015-12-09 16:59:01.026007	4	accepted	12:25398281-25398281 (C->T)	1	1	0	0	2	f	\N	\N
347	In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804), MET amplification and EGFR T790M were found to be the primary drivers of resistance.	1	178	146	2015-12-09 16:59:01.067763	2015-12-09 16:59:01.067763	4	accepted	N/A	1	2	0	0	1	f	\N	\N
348	In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted from temozolomide. This benefit was also methylation status dependent, as those without methylation did not see increased survival.	17	217	85	2015-12-09 16:59:01.137415	2015-12-09 16:59:01.137415	5	accepted	N/A	0	2	0	0	0	f	\N	\N
349	MGMT promoter methylation has been shown to confer sensitivity to alkylating agents, such as carmustine, in glioblastoma cells.	17	124	85	2015-12-09 16:59:01.191232	2015-12-09 16:59:01.191232	4	accepted	N/A	1	2	0	0	0	f	\N	\N
350	In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor, such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may be useful in treating patients with initial resistance to alkylating agents.	17	124	85	2015-12-09 16:59:01.237336	2015-12-09 16:59:01.237336	4	accepted	N/A	4	2	0	0	0	f	\N	0
351	NOTCH1 gain-of-function mutations, including D1643H, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	1	218	135	2015-12-09 16:59:01.297296	2015-12-09 16:59:01.297296	4	accepted	N/A	1	1	0	0	3	f	\N	\N
352	NOTCH1 gain-of-function mutations, including R2328W, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	1	218	136	2015-12-09 16:59:01.349965	2015-12-09 16:59:01.349965	4	accepted	N/A	1	1	0	0	3	f	\N	\N
353	NOTCH1 gain-of-function mutations, including S2275fs, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	1	218	138	2015-12-09 16:59:01.395928	2015-12-09 16:59:01.395928	4	accepted	N/A	1	1	0	0	3	f	\N	\N
354	NOTCH1 gain-of-function mutations, including V2444fs, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	1	218	137	2015-12-09 16:59:01.458819	2015-12-09 16:59:01.458819	4	accepted	N/A	1	1	0	0	3	f	\N	\N
355	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	20	219	103	2015-12-09 16:59:01.521865	2015-12-09 16:59:01.521865	4	accepted	3:178936082-178936082 (G->A)	2	2	0	0	0	f	\N	\N
356	Rapamycin inhibits transformation induced by mutation in PIK3CA	20	220	103	2015-12-09 16:59:01.568537	2015-12-09 16:59:01.568537	3	accepted	3:178936082-178936082 (G->A)	2	2	0	0	0	f	\N	\N
357	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	8	221	103	2015-12-09 16:59:01.612675	2015-12-09 16:59:01.612675	3	accepted	3:178936082-178936082 (G->A)	4	2	0	0	1	f	\N	2
358	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	8	222	103	2015-12-09 16:59:01.661175	2015-12-09 16:59:01.661175	2	accepted	3:178936082-178936082 (G->A)	4	1	0	0	3	f	\N	\N
359	The presence of exon 9 (such as E542K) or exon 20 mutations were associated with poorer survival compared to those with wild-type PIK3CA mutation.	8	221	103	2015-12-09 16:59:01.701112	2015-12-09 16:59:01.701112	3	accepted	3:178936082-178936082 (G->A)	4	1	0	0	3	f	\N	\N
360	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	20	219	104	2015-12-09 16:59:01.745121	2015-12-09 16:59:01.745121	4	accepted	3:178936091-178936091 (G->A)	2	2	0	0	0	f	\N	\N
361	Rapamycin inhibits transformation induced by mutation in PIK3CA	20	220	104	2015-12-09 16:59:01.788879	2015-12-09 16:59:01.788879	3	accepted	3:178936091-178936091 (G->A)	2	2	0	0	0	f	\N	\N
362	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	8	221	104	2015-12-09 16:59:01.834246	2015-12-09 16:59:01.834246	3	accepted	3:178936091-178936091 (G->A)	4	2	0	0	1	f	\N	2
363	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	8	222	104	2015-12-09 16:59:01.882523	2015-12-09 16:59:01.882523	2	accepted	3:178936091-178936091 (G->A)	4	1	0	0	3	f	\N	\N
364	The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	8	221	104	2015-12-09 16:59:01.931181	2015-12-09 16:59:01.931181	3	accepted	3:178936091-178936091 (G->A)	4	1	0	0	3	f	\N	\N
365	Breast cancer cell lines with H1047R mutation showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	20	219	107	2015-12-09 16:59:01.981869	2015-12-09 16:59:01.981869	4	accepted	3:178952085-178952085 (A->G)	2	2	0	0	0	f	\N	\N
366	Rapamycin inhibits transformation induced by mutation in PIK3CA	20	220	107	2015-12-09 16:59:02.03258	2015-12-09 16:59:02.03258	3	accepted	3:178952085-178952085 (A->G)	2	2	0	0	0	f	\N	\N
367	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.	8	221	107	2015-12-09 16:59:02.094274	2015-12-09 16:59:02.094274	3	accepted	3:178952085-178952085 (A->G)	4	2	0	0	1	f	\N	2
368	The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	8	221	107	2015-12-09 16:59:02.165189	2015-12-09 16:59:02.165189	3	accepted	3:178952085-178952085 (A->G)	4	1	0	0	3	f	\N	\N
369	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	39	223	108	2015-12-09 16:59:02.254334	2015-12-09 16:59:02.254334	5	accepted	N/A	1	2	0	0	0	f	\N	\N
370	PTEN nonsense mutations, including R233*, have been shown to be inactivating and loss-of-function, but do not have prognostic value in glioblastoma multiforme patients.	17	224	110	2015-12-09 16:59:02.316501	2015-12-09 16:59:02.316501	3	accepted	10:89717672-89717672 (C->T)	1	1	0	1	3	f	\N	\N
371	Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse to PI3K-mTOR inhibitors.	19	225	110	2015-12-09 16:59:02.429501	2015-12-09 16:59:02.429501	4	accepted	10:89717672-89717672 (C->T)	2	2	0	0	0	f	\N	\N
372	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	39	223	111	2015-12-09 16:59:02.480232	2015-12-09 16:59:02.480232	5	accepted	N/A	1	2	0	0	0	f	\N	\N
373	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	34	226	114	2015-12-09 16:59:02.550287	2015-12-09 16:59:02.550287	3	accepted	2:198267359-198267359 (C->A)	4	1	0	0	2	f	\N	\N
374	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	34	226	115	2015-12-09 16:59:02.604405	2015-12-09 16:59:02.604405	3	accepted	2:198266834-198266834 (T->C)	4	1	0	0	2	f	\N	\N
375	Breast cancer patients who harbor R175H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	20	227	116	2015-12-09 16:59:02.672956	2015-12-09 16:59:02.672956	3	accepted	17:7578406-7578406 (C->T)	1	1	0	0	3	f	\N	\N
376	Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.	20	228	116	2015-12-09 16:59:02.735966	2015-12-09 16:59:02.735966	3	accepted	17:7578406-7578406 (C->T)	2	2	0	0	0	f	\N	\N
377	Breast cancer patients who harbor R248Q mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	20	227	117	2015-12-09 16:59:02.794067	2015-12-09 16:59:02.794067	3	accepted	17:7577538-7577538 (C->T)	1	1	0	0	3	f	\N	\N
378	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R248 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	20	229	117	2015-12-09 16:59:02.846972	2015-12-09 16:59:02.846972	3	accepted	17:7577538-7577538 (C->T)	1	1	0	0	3	f	\N	\N
379	In breast cancer patients harboring R248W mutation, the prognosis is worse than any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.	20	227	118	2015-12-09 16:59:02.894405	2015-12-09 16:59:02.894405	3	accepted	17:7577539-7577539 (G->A)	1	1	0	0	3	f	\N	\N
380	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	20	229	119	2015-12-09 16:59:02.934336	2015-12-09 16:59:02.934336	3	accepted	17:7577535-7577535 (C->G)	1	1	0	0	3	f	\N	\N
381	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	20	229	120	2015-12-09 16:59:02.97919	2015-12-09 16:59:02.97919	3	accepted	17:7577536-7577536 (T->A)	1	1	0	0	3	f	\N	\N
382	Breast cancer patients who harbor R273C mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	20	227	121	2015-12-09 16:59:03.019782	2015-12-09 16:59:03.019782	3	accepted	17:7577121-7577121 (G->A)	1	1	0	0	3	f	\N	\N
383	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	20	229	121	2015-12-09 16:59:03.078327	2015-12-09 16:59:03.078327	3	accepted	17:7577121-7577121 (G->A)	1	1	0	0	3	f	\N	\N
384	Breast cancer patients who harbor R273H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	20	227	122	2015-12-09 16:59:03.119284	2015-12-09 16:59:03.119284	3	accepted	17:7577120-7577120 (C->T)	1	1	0	0	3	f	\N	\N
385	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	20	229	122	2015-12-09 16:59:03.158827	2015-12-09 16:59:03.158827	3	accepted	17:7577120-7577120 (C->T)	1	1	0	0	3	f	\N	\N
386	Breast tumors with R175H mutations are more responsive to doxorubicin than breast tumors with wild type TP53.	20	229	123	2015-12-09 16:59:03.198324	2015-12-09 16:59:03.198324	3	accepted	N/A	1	1	0	0	3	f	\N	\N
387	A patient with metastatic bladder cancer that responded well to the mTOR inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del, p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations also had mild responses to everolimus.	40	230	124	2015-12-09 16:59:03.254214	2015-12-09 16:59:03.254214	3	accepted	N/A	1	2	0	0	0	f	\N	\N
388	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	16	230	126	2015-12-09 16:59:03.297751	2015-12-09 16:59:03.297751	3	accepted	N/A	1	2	0	0	0	f	\N	\N
389	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.	2	231	127	2015-12-09 16:59:03.351897	2015-12-09 16:59:03.351897	3	accepted	N/A	4	0	0	1	4	f	\N	\N
390	In patients with AML, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	2	231	127	2015-12-09 16:59:03.39006	2015-12-09 16:59:03.39006	2	accepted	N/A	4	1	0	1	6	f	\N	\N
391	In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	34	231	127	2015-12-09 16:59:03.428417	2015-12-09 16:59:03.428417	2	accepted	N/A	4	1	0	1	6	f	\N	\N
392	After adjusting for age and cancer stage, presence of U2AF mutation such as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.	34	232	127	2015-12-09 16:59:03.476887	2015-12-09 16:59:03.476887	3	accepted	N/A	4	1	0	0	3	f	\N	\N
393	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.	2	231	128	2015-12-09 16:59:03.523449	2015-12-09 16:59:03.523449	3	accepted	N/A	4	0	0	1	4	f	\N	\N
394	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	2	231	128	2015-12-09 16:59:03.567177	2015-12-09 16:59:03.567177	2	accepted	N/A	4	1	0	1	6	f	\N	\N
395	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	34	231	128	2015-12-09 16:59:03.605444	2015-12-09 16:59:03.605444	2	accepted	N/A	4	1	0	1	6	f	\N	\N
396	After adjusting for age and cancer stage, the presence of U2AF mutations such as S34Y/F are prognostic for poorer survival outcomes in patients with MDS.	34	232	128	2015-12-09 16:59:03.640727	2015-12-09 16:59:03.640727	3	accepted	N/A	4	1	0	0	3	f	\N	\N
397	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	1	233	79	2015-12-09 16:59:03.688677	2015-12-09 16:59:03.688677	2	accepted	12:25398284-25398284 (C->T)	3	2	0	0	1	f	\N	\N
398	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	41	234	30	2015-12-09 16:59:03.759447	2015-12-09 16:59:03.759447	3	accepted	N/A	2	2	0	0	0	f	\N	2
399	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	41	234	84	2015-12-09 16:59:03.806984	2015-12-09 16:59:03.806984	3	accepted	N/A	2	2	0	0	0	f	\N	2
400	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	42	235	31	2015-12-09 16:59:03.857672	2015-12-09 16:59:03.857672	4	accepted	N/A	1	0	0	0	4	f	\N	\N
401	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	42	235	109	2015-12-09 16:59:03.898492	2015-12-09 16:59:03.898492	4	accepted	N/A	1	0	0	0	4	f	\N	\N
402	NSCLC patients with exon 19 deletions in EGFR had longer overall survival than patients with an L858R mutation when treated with gefitinib and erlotinib (38 versus 17 months; P = 0.04).	1	236	133	2015-12-09 16:59:03.980427	2015-12-09 16:59:03.980427	4	accepted	N/A	1	2	0	0	0	f	\N	2
403	In patients with Ewing's sarcoma, those with type 1 EWS-FLI1 fusions had longer overall survival than those with other types of EWS-FLI1 fusions.	43	237	151	2015-12-09 16:59:04.048472	2015-12-09 16:59:04.048472	4	accepted	N/A	1	1	0	0	4	f	\N	\N
\.


--
-- Data for Name: drugs_evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs_evidence_items (drug_id, evidence_item_id) FROM stdin;
1	1
2	2
2	3
3	4
4	5
2	6
2	7
2	8
2	9
2	10
5	11
2	12
6	13
6	14
2	15
2	16
2	17
7	18
2	19
8	20
9	21
9	22
2	23
10	24
10	25
11	26
11	27
12	28
11	29
13	30
13	31
1	32
14	33
2	34
2	35
2	36
2	37
2	38
2	39
2	40
2	41
2	42
2	43
2	44
2	45
2	46
2	47
2	48
2	49
10	50
10	51
2	52
10	53
2	54
10	55
15	55
1	55
16	56
17	57
18	57
2	58
2	59
2	60
2	61
2	62
19	63
20	64
20	65
20	66
20	67
20	68
21	69
22	70
2	71
2	72
22	73
23	74
24	75
21	76
14	76
21	77
2	78
2	79
2	80
2	81
25	82
26	82
21	83
14	83
2	84
2	85
2	86
2	87
2	88
2	89
1	90
2	91
27	92
28	92
28	93
29	93
30	94
8	94
31	94
8	95
21	96
6	97
21	98
21	99
32	100
32	101
32	102
32	103
12	104
11	105
13	106
11	107
19	108
2	109
2	110
2	111
2	112
2	113
2	114
2	115
2	116
2	117
2	118
2	119
2	120
2	121
2	122
2	123
2	124
2	125
33	126
2	127
34	128
2	129
2	130
2	131
2	132
2	133
2	134
2	135
3	136
35	137
2	138
3	139
35	140
2	141
19	142
19	143
2	144
19	145
19	146
2	147
2	148
2	149
2	150
2	151
2	152
2	153
2	154
34	155
34	156
3	157
36	158
2	159
2	160
2	161
2	162
2	163
2	164
2	165
2	166
2	167
2	168
2	169
2	170
2	171
2	172
2	173
2	174
2	175
2	176
2	177
2	178
37	179
34	179
2	180
2	181
2	182
2	183
37	184
38	185
19	186
19	187
19	188
39	189
40	190
2	191
2	192
2	193
2	194
2	195
2	196
2	197
2	198
2	199
2	200
2	201
2	202
2	203
2	204
2	205
2	206
41	207
41	208
2	209
42	210
10	211
10	212
15	213
16	213
2	214
10	215
10	216
10	217
10	218
43	219
11	220
11	221
11	222
2	223
2	224
2	225
2	226
2	227
2	228
2	229
2	230
2	231
2	232
2	233
2	234
44	235
2	236
2	237
2	238
2	239
45	240
46	241
45	242
46	243
2	244
2	245
16	246
16	247
16	248
16	249
16	250
16	251
16	252
18	252
17	253
17	254
17	255
18	255
17	255
18	255
2	256
2	257
17	258
18	259
17	260
2	261
2	262
18	263
47	263
48	264
49	265
50	266
50	267
50	268
50	269
51	270
50	271
50	272
50	273
50	274
51	274
50	275
50	276
50	277
52	278
53	279
54	279
52	280
53	281
54	281
52	282
53	283
54	283
52	284
53	285
54	285
52	286
53	287
54	287
55	288
56	289
55	289
2	290
57	291
1	292
2	293
2	294
58	295
2	296
1	297
1	298
57	299
2	300
2	301
59	302
60	302
2	303
2	304
2	305
2	306
2	307
2	308
2	309
2	310
2	311
2	312
2	313
2	314
2	315
2	316
2	317
2	318
2	319
2	320
2	321
2	322
2	323
2	324
2	325
10	326
16	327
10	327
15	327
61	328
10	329
19	330
2	331
2	332
2	333
2	334
2	335
17	336
18	336
2	337
17	338
18	338
2	339
62	340
63	340
17	341
18	341
17	342
18	342
2	343
2	344
19	345
19	346
49	347
26	348
64	349
65	350
2	351
2	352
2	353
2	354
66	355
39	356
19	357
24	357
2	358
2	359
66	360
39	361
19	362
24	362
2	363
2	364
66	365
39	366
19	367
24	367
2	368
34	369
2	370
67	371
34	372
2	373
2	374
2	375
68	376
2	377
2	378
2	379
2	380
2	381
2	382
2	383
2	384
2	385
2	386
40	387
40	388
2	389
2	390
2	391
2	392
2	393
2	394
2	395
2	396
21	397
10	398
69	398
10	399
69	399
2	400
2	401
17	402
18	402
2	403
\.


--
-- Name: drugs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('drugs_id_seq', 69, true);


--
-- Data for Name: events; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY events (id, created_at, updated_at, action, description, originating_user_id, subject_id, subject_type, state_params) FROM stdin;
\.


--
-- Name: events_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('events_id_seq', 1, false);


--
-- Name: evidence_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_items_id_seq', 403, true);


--
-- Data for Name: feeds; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY feeds (id, created_at, updated_at, owner_id, event_id, acknowledged) FROM stdin;
\.


--
-- Name: feeds_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('feeds_id_seq', 1, false);


--
-- Data for Name: gene_aliases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY gene_aliases (id, name) FROM stdin;
1	CD246
2	NBLST3
3	AKT
4	CWS6
5	PKB
6	PKB-ALPHA
7	PRKBA
8	RAC
9	RAC-ALPHA
10	A-RAF
11	ARAF1
12	PKS2
13	RAFA1
14	ABL
15	JTK7
16	bcr/abl
17	c-ABL
18	c-ABL1
19	p150
20	v-abl
21	B-RAF1
22	BRAF1
23	NS7
24	RAFB1
25	BRCAI
26	BRCC1
27	BROVCA1
28	FANCS
29	IRIS
30	PNCA4
31	PPP1R53
32	PSCP
33	RNF53
34	BRCC2
35	BROVCA2
36	FACD
37	FAD
38	FAD1
39	FANCD
40	FANCD1
41	GLM3
42	PNCA2
43	XRCC11
44	BCL1
45	D11S287E
46	PRAD1
47	U21B31
48	KIAK0002
49	MPPH3
50	CCNE
51	pCCNE1
52	MCPH12
53	PLSTIRE
54	CMM3
55	PSK-J3
56	ARF
57	CDK4I
58	CDKN2
59	CMM2
60	INK4
61	INK4A
62	MLM
63	MTS-1
64	MTS1
65	P14
66	P14ARF
67	P16
68	P16-INK4A
69	P16INK4
70	P16INK4A
71	P19
72	P19ARF
73	TP16
74	C/EBP-alpha
75	CEBP
76	HSPF1
77	Hdj1
78	Hsp40
79	RSPH16B
80	Sis1
81	PKACA
82	PPNAD4
83	DNMT3A2
84	M.HsaIIIA
85	TBRS
86	ERBB
87	ERBB1
88	HER1
89	NISBD2
90	PIG61
91	mENA
92	CD340
93	HER-2
94	HER-2/neu
95	HER2
96	MLN 19
97	NEU
98	NGL
99	TKR1
100	ER
101	ESR
102	ESRA
103	ESTRR
104	Era
105	NR3A1
106	BBDS
107	BEK
108	BFR-1
109	CD332
110	CEK3
111	CFD1
112	ECT1
113	JWS
114	K-SAM
115	KGFR
116	TK14
117	TK25
118	ACH
119	CD333
120	CEK2
121	HSFGFR3EX
122	JTK4
123	CD135
124	FLK-2
125	FLK2
126	STK1
127	DCML
128	IMD21
129	MONOMAC
130	NFE1B
131	HEL-216
132	HEL-S-26
133	IDCD
134	IDH
135	IDP
136	IDPC
137	PICD
138	D2HGA2
139	ICD-M
140	IDHM
141	IDPM
142	mNADP-IDH
143	JTK10
144	THCYT3
145	C-Kit
146	CD117
147	PBT
148	SCFR
149	C-K-RAS
150	CFC2
151	K-RAS2A
152	K-RAS2B
153	K-RAS4A
154	K-RAS4B
155	KI-RAS
156	KRAS1
157	KRAS2
158	NS
159	NS3
160	RALD
161	RASK2
162	CFC3
163	MAPKK1
164	MEK1
165	MKK1
166	PRKMK1
167	C-FMS
168	CD115
169	CSF-1R
170	CSFR
171	FIM2
172	FMS
173	HDLS
174	M-CSF-R
175	B23
176	NPM
177	ALPS4
178	CMNS
179	N-ras
180	NCMS
181	NRAS1
182	NS6
183	CLOVE
184	CWS5
185	MCAP
186	MCM
187	MCMTC
188	PI3K
189	p110-alpha
190	CD140A
191	PDGFR-2
192	PDGFR2
193	RHEPDGFRA
194	MYL
195	PP8675
196	RNF71
197	TRIM19
198	NR1B1
199	RAR
200	10q23del
201	BZS
202	CWS1
203	DEC
204	GLM2
205	MHAM
206	MMAC1
207	PTEN1
208	TEP1
209	CDHF12
210	CDHR16
211	HSCR1
212	MEN2A
213	MEN2B
214	MTC1
215	PTC
216	RET-ELE1
217	RET51
218	AML1
219	AML1-EVI-1
220	AMLCR1
221	CBF2alpha
222	CBFA2
223	EVI-1
224	PEBP2aB
225	PEBP2alpha
226	Hsh155
227	MDS
228	PRP10
229	PRPF10
230	SAP155
231	SF3b155
232	BCC7
233	LFS1
234	P53
235	TRP53
236	LAM
237	TSC
238	PPP1R160
239	TSC4
240	FP793
241	RN
242	RNU2AF1
243	U2AF35
244	U2AFBP
245	AWT1
246	EWS-WT1
247	GUD
248	NPHS4
249	WAGR
250	WIT-2
251	WT33
252	AOS5
253	AOVD1
254	TAN1
255	hN1
256	MIG20a
257	NTRKR3
258	TKT
259	TYRO10
260	AUTS9
261	DFNB97
262	HGFR
263	RCCP2
264	c-Met
265	EWSR2
266	SIC-1
267	EWS
268	EWS-FLI1
269	bK984G1.4
\.


--
-- Data for Name: gene_aliases_genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY gene_aliases_genes (gene_alias_id, gene_id) FROM stdin;
1	1
2	1
3	2
4	2
5	2
6	2
7	2
8	2
9	2
10	3
11	3
12	3
13	3
14	4
15	4
16	4
17	4
18	4
19	4
20	4
21	5
22	5
23	5
24	5
25	6
26	6
27	6
28	6
29	6
30	6
31	6
32	6
33	6
34	7
35	7
36	7
37	7
38	7
39	7
40	7
41	7
42	7
43	7
44	8
45	8
46	8
47	8
48	9
49	9
50	11
51	11
52	12
53	12
54	13
55	13
56	14
57	14
58	14
59	14
60	14
61	14
62	14
63	14
64	14
65	14
66	14
67	14
68	14
69	14
70	14
71	14
72	14
73	14
74	15
75	15
76	16
77	16
78	16
79	16
80	16
81	17
82	17
83	18
84	18
85	18
86	19
87	19
88	19
89	19
90	19
91	19
92	20
93	20
94	20
95	20
96	20
97	20
98	20
99	20
100	21
101	21
102	21
103	21
104	21
105	21
106	22
107	22
108	22
109	22
110	22
111	22
112	22
113	22
114	22
115	22
116	22
117	22
118	23
119	23
120	23
121	23
122	23
123	24
124	24
125	24
126	24
127	25
128	25
129	25
130	25
131	26
132	26
133	26
134	26
135	26
136	26
137	26
138	27
139	27
134	27
140	27
135	27
141	27
142	27
143	28
144	28
145	29
146	29
147	29
148	29
149	30
150	30
151	30
152	30
153	30
154	30
155	30
156	30
157	30
158	30
159	30
160	30
161	30
162	31
163	31
164	31
165	31
166	31
167	33
168	33
169	33
170	33
171	33
172	33
173	33
174	33
175	35
176	35
177	36
178	36
179	36
180	36
181	36
182	36
183	37
184	37
185	37
186	37
187	37
188	37
189	37
190	38
191	38
192	38
193	38
194	39
195	39
196	39
197	39
198	40
199	40
200	41
201	41
202	41
203	41
204	41
205	41
206	41
207	41
208	41
209	42
210	42
211	42
212	42
213	42
214	42
215	42
216	42
217	42
218	43
219	43
220	43
221	43
222	43
223	43
224	43
225	43
226	44
227	44
228	44
229	44
230	44
231	44
232	45
233	45
234	45
235	45
236	46
237	46
236	47
238	47
239	47
240	48
241	48
242	48
243	48
244	48
245	49
246	49
247	49
248	49
249	49
250	49
251	49
252	50
253	50
254	50
255	50
256	51
257	51
258	51
259	51
260	52
261	52
262	52
263	52
264	52
265	53
266	53
267	54
268	54
269	54
\.


--
-- Name: gene_aliases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('gene_aliases_id_seq', 269, true);


--
-- Name: genes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('genes_id_seq', 54, true);


--
-- Data for Name: genes_sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes_sources (gene_id, source_id, created_at, updated_at) FROM stdin;
1	1	2015-12-09 16:58:27.655994	2015-12-09 16:58:27.655994
1	2	2015-12-09 16:58:27.668998	2015-12-09 16:58:27.668998
2	3	2015-12-09 16:58:27.924449	2015-12-09 16:58:27.924449
2	4	2015-12-09 16:58:27.938848	2015-12-09 16:58:27.938848
3	5	2015-12-09 16:58:28.177746	2015-12-09 16:58:28.177746
3	6	2015-12-09 16:58:28.189856	2015-12-09 16:58:28.189856
4	7	2015-12-09 16:58:28.466565	2015-12-09 16:58:28.466565
4	8	2015-12-09 16:58:28.476973	2015-12-09 16:58:28.476973
5	9	2015-12-09 16:58:28.709883	2015-12-09 16:58:28.709883
5	10	2015-12-09 16:58:28.719529	2015-12-09 16:58:28.719529
6	11	2015-12-09 16:58:28.972664	2015-12-09 16:58:28.972664
7	11	2015-12-09 16:58:29.255441	2015-12-09 16:58:29.255441
8	12	2015-12-09 16:58:29.513337	2015-12-09 16:58:29.513337
8	13	2015-12-09 16:58:29.523689	2015-12-09 16:58:29.523689
9	12	2015-12-09 16:58:29.724924	2015-12-09 16:58:29.724924
9	13	2015-12-09 16:58:29.728012	2015-12-09 16:58:29.728012
10	12	2015-12-09 16:58:29.916301	2015-12-09 16:58:29.916301
10	13	2015-12-09 16:58:29.919082	2015-12-09 16:58:29.919082
11	14	2015-12-09 16:58:30.122722	2015-12-09 16:58:30.122722
12	15	2015-12-09 16:58:30.330653	2015-12-09 16:58:30.330653
12	16	2015-12-09 16:58:30.344502	2015-12-09 16:58:30.344502
13	15	2015-12-09 16:58:30.558074	2015-12-09 16:58:30.558074
13	16	2015-12-09 16:58:30.561428	2015-12-09 16:58:30.561428
14	17	2015-12-09 16:58:30.868634	2015-12-09 16:58:30.868634
14	18	2015-12-09 16:58:30.880547	2015-12-09 16:58:30.880547
15	19	2015-12-09 16:58:31.127592	2015-12-09 16:58:31.127592
15	20	2015-12-09 16:58:31.136841	2015-12-09 16:58:31.136841
15	21	2015-12-09 16:58:31.145892	2015-12-09 16:58:31.145892
16	22	2015-12-09 16:58:31.368579	2015-12-09 16:58:31.368579
17	22	2015-12-09 16:58:31.576824	2015-12-09 16:58:31.576824
17	23	2015-12-09 16:58:31.586248	2015-12-09 16:58:31.586248
18	24	2015-12-09 16:58:31.802377	2015-12-09 16:58:31.802377
18	25	2015-12-09 16:58:31.811298	2015-12-09 16:58:31.811298
19	26	2015-12-09 16:58:32.05944	2015-12-09 16:58:32.05944
19	27	2015-12-09 16:58:32.068905	2015-12-09 16:58:32.068905
20	28	2015-12-09 16:58:32.332261	2015-12-09 16:58:32.332261
20	29	2015-12-09 16:58:32.345254	2015-12-09 16:58:32.345254
21	30	2015-12-09 16:58:32.605348	2015-12-09 16:58:32.605348
21	31	2015-12-09 16:58:32.614641	2015-12-09 16:58:32.614641
22	32	2015-12-09 16:58:32.881626	2015-12-09 16:58:32.881626
22	33	2015-12-09 16:58:32.894586	2015-12-09 16:58:32.894586
23	32	2015-12-09 16:58:33.136894	2015-12-09 16:58:33.136894
23	33	2015-12-09 16:58:33.139956	2015-12-09 16:58:33.139956
24	34	2015-12-09 16:58:33.367542	2015-12-09 16:58:33.367542
24	19	2015-12-09 16:58:33.370142	2015-12-09 16:58:33.370142
25	35	2015-12-09 16:58:33.59605	2015-12-09 16:58:33.59605
25	36	2015-12-09 16:58:33.605583	2015-12-09 16:58:33.605583
26	37	2015-12-09 16:58:33.855968	2015-12-09 16:58:33.855968
26	38	2015-12-09 16:58:33.865519	2015-12-09 16:58:33.865519
27	37	2015-12-09 16:58:34.10156	2015-12-09 16:58:34.10156
27	38	2015-12-09 16:58:34.104688	2015-12-09 16:58:34.104688
27	39	2015-12-09 16:58:34.115948	2015-12-09 16:58:34.115948
28	40	2015-12-09 16:58:34.349763	2015-12-09 16:58:34.349763
29	41	2015-12-09 16:58:34.566111	2015-12-09 16:58:34.566111
30	42	2015-12-09 16:58:34.947754	2015-12-09 16:58:34.947754
35	43	2015-12-09 16:58:35.996258	2015-12-09 16:58:35.996258
35	19	2015-12-09 16:58:35.999321	2015-12-09 16:58:35.999321
42	44	2015-12-09 16:58:37.599524	2015-12-09 16:58:37.599524
42	45	2015-12-09 16:58:37.609942	2015-12-09 16:58:37.609942
44	46	2015-12-09 16:58:38.094987	2015-12-09 16:58:38.094987
44	47	2015-12-09 16:58:38.104353	2015-12-09 16:58:38.104353
45	48	2015-12-09 16:58:38.336119	2015-12-09 16:58:38.336119
48	49	2015-12-09 16:58:38.962922	2015-12-09 16:58:38.962922
48	50	2015-12-09 16:58:38.980031	2015-12-09 16:58:38.980031
\.


--
-- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY schema_migrations (version) FROM stdin;
20151112020541
20140805184308
20141021192035
20141022180523
20141022195329
20141108234203
20141108234935
20141109000136
20141114222425
20150107205621
20150115231117
20150116220720
20150116230539
20150116230632
20150204192653
20150205000010
20150205205734
20150205210725
20150210191015
20150210204445
20150219202227
20150220160805
20150220182109
20150223170106
20150227204142
20150227212146
20150303202505
20150305200429
20150319181853
20150416182633
20150501151624
20150511191452
20150512184905
20150528170506
20150528211748
20150602202928
20150609210448
20150611153328
20150611154621
20150611200713
20150709222321
20150709230109
20150722183121
20150722185207
20150722185935
20150728191648
20150805161648
20151028175026
20151028181107
20151028181641
20151029221126
20151029221742
20151030154300
20151030165159
\.


--
-- Name: sources_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('sources_id_seq', 237, true);


--
-- Data for Name: subscriptions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY subscriptions (id, user_id, subscribable_id, subscribable_type, type, created_at, updated_at, action_type, action_class) FROM stdin;
\.


--
-- Name: subscriptions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('subscriptions_id_seq', 1, false);


--
-- Data for Name: suggested_changes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY suggested_changes (id, suggested_changes, moderated_id, moderated_type, user_id, status, created_at, updated_at) FROM stdin;
\.


--
-- Name: suggested_changes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('suggested_changes_id_seq', 1, false);


--
-- Data for Name: tsv_releases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY tsv_releases (id, path, created_at, updated_at) FROM stdin;
\.


--
-- Name: tsv_releases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('tsv_releases_id_seq', 1, false);


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('users_id_seq', 1, false);


--
-- Data for Name: variant_groups; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_groups (id, name, description, created_at, updated_at, deleted, deleted_at) FROM stdin;
1	KIT Exon 17	\N	2015-12-09 16:58:44.861092	2015-12-09 16:58:44.861092	f	\N
4	KIT Exon 11	\N	2015-12-09 16:58:46.550174	2015-12-09 16:58:46.550174	f	\N
8	NPM1 Exon 12	\N	2015-12-09 16:58:52.881345	2015-12-09 16:58:52.881345	f	\N
2	Imatinib Resistance	While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. 	2015-12-09 16:58:45.389567	2015-12-09 16:59:04.177038	f	\N
11	ALK Fusions	ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase the activity of the ALK oncogene relative to normal cells. While EML4 is the most common fusion partner, other 5' partners have been observed. The EML4-ALK fusion has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. 	2015-12-09 16:58:54.798629	2015-12-09 16:59:04.201234	f	\N
12	EGFR TKI Resistance	EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine kinase inhibitors have been developed to target EGFR pathway activity. One such inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting. However, the T790M missense mutation has shown to confer resistance to this inhibitor in cell lines and case studies.  	2015-12-09 16:58:56.599109	2015-12-09 16:59:04.217803	f	\N
3	Crizotinib Resistance	The ALK oncogene has long been considered a driving factor in non-small cell lung cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive this resistance. 	2015-12-09 16:58:45.620102	2015-12-09 16:59:04.234337	f	\N
5	Other V600's	While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants have also been observed and studied to a lesser degree. At first approximation, many of these variants seem to behave similarly to V600E, and treatment with dabrafenib has been shown to be effective. 	2015-12-09 16:58:47.496939	2015-12-09 16:59:04.248938	f	\N
13	HER2 Activating	HER2-positive breast cancer is widely recognized as a molecular subtype of breast cancer. While amplification of ERBB2 is a common mechanism for activation of the pathway, missense mutations have also demonstrated activation potential. These activating missense mutations also confer senstivity to the targeted therapeutic neratinib. 	2015-12-09 16:58:56.949449	2015-12-09 16:59:04.261882	f	\N
14	ESR1 Ligand-Binding Domain	ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone therapy has been widely effective in treating the disease, however sequencing of resistant patients has uncovered a number of missense mutations in mediating this resistance. Many of these mutations lie within the ligand-binding domain, and contribute to constitutive activity of the receptor. This has lead to the development of estrogen receptor degrading agents such as fulvestrant, which have shown early efficacy in clinical trials.  	2015-12-09 16:58:57.479155	2015-12-09 16:59:04.274897	f	\N
7	FGFR fusions	FGFR fusions have been demonstrated across many cancer types to have oncogenic potential. FGFR2 and FGFR3 have been shown to be fused to many 3' partners, which may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2 fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib and pazopanib have shown efficacy in case studies.  	2015-12-09 16:58:50.370385	2015-12-09 16:59:04.287946	f	\N
15	PTEN Loss-of-Function	PTEN loss is a common event in breast cancer, as well as others. The most common single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell lines harboring this mutation behave similarly to cell lines harboring larger insertion or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded growth. 	2015-12-09 16:59:02.366392	2015-12-09 16:59:04.302553	f	\N
6	Motesanib Resistance	RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment of these patients with the targeted therapeutic motesanib has shown to be effective. However, the missense mutations C634W and M918T have shown to confer motesanib resistance in cell lines. 	2015-12-09 16:58:47.560951	2015-12-09 16:59:04.315574	f	\N
9	TSC Loss	Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types. Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor everolimus has seen significant anti-tumorigenic activity. 	2015-12-09 16:58:53.226243	2015-12-09 16:59:04.333019	f	\N
10	BRCA Germline Variants	BRCA germline variants have long been recognized as important potential predictors of breast and ovarian cancer risk. 	2015-12-09 16:58:54.085729	2015-12-09 16:59:04.350369	f	\N
\.


--
-- Data for Name: variant_group_variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_group_variants (variant_id, variant_group_id, created_at, updated_at) FROM stdin;
65	1	2015-12-09 16:58:44.882526	2015-12-09 16:58:44.882526
99	2	2015-12-09 16:58:45.403938	2015-12-09 16:58:45.403938
99	2	2015-12-09 16:58:45.453948	2015-12-09 16:58:45.453948
99	2	2015-12-09 16:58:45.512023	2015-12-09 16:58:45.512023
8	3	2015-12-09 16:58:45.632492	2015-12-09 16:58:45.632492
66	4	2015-12-09 16:58:46.565441	2015-12-09 16:58:46.565441
66	4	2015-12-09 16:58:46.613549	2015-12-09 16:58:46.613549
70	1	2015-12-09 16:58:46.744747	2015-12-09 16:58:46.744747
72	4	2015-12-09 16:58:46.841985	2015-12-09 16:58:46.841985
72	4	2015-12-09 16:58:46.885196	2015-12-09 16:58:46.885196
98	2	2015-12-09 16:58:47.247002	2015-12-09 16:58:47.247002
99	2	2015-12-09 16:58:47.30885	2015-12-09 16:58:47.30885
100	2	2015-12-09 16:58:47.354563	2015-12-09 16:58:47.354563
101	2	2015-12-09 16:58:47.397843	2015-12-09 16:58:47.397843
102	2	2015-12-09 16:58:47.442502	2015-12-09 16:58:47.442502
13	5	2015-12-09 16:58:47.50919	2015-12-09 16:58:47.50919
112	6	2015-12-09 16:58:47.576508	2015-12-09 16:58:47.576508
113	6	2015-12-09 16:58:47.708005	2015-12-09 16:58:47.708005
11	5	2015-12-09 16:58:47.903443	2015-12-09 16:58:47.903443
15	5	2015-12-09 16:58:48.900378	2015-12-09 16:58:48.900378
16	5	2015-12-09 16:58:48.93984	2015-12-09 16:58:48.93984
53	7	2015-12-09 16:58:50.385446	2015-12-09 16:58:50.385446
87	8	2015-12-09 16:58:52.897246	2015-12-09 16:58:52.897246
87	8	2015-12-09 16:58:52.944671	2015-12-09 16:58:52.944671
87	8	2015-12-09 16:58:53.009313	2015-12-09 16:58:53.009313
87	8	2015-12-09 16:58:53.053357	2015-12-09 16:58:53.053357
124	9	2015-12-09 16:58:53.240445	2015-12-09 16:58:53.240445
125	9	2015-12-09 16:58:53.301262	2015-12-09 16:58:53.301262
131	10	2015-12-09 16:58:54.100836	2015-12-09 16:58:54.100836
132	10	2015-12-09 16:58:54.15336	2015-12-09 16:58:54.15336
3	2	2015-12-09 16:58:54.599982	2015-12-09 16:58:54.599982
2	2	2015-12-09 16:58:54.649217	2015-12-09 16:58:54.649217
2	2	2015-12-09 16:58:54.695256	2015-12-09 16:58:54.695256
5	11	2015-12-09 16:58:54.812518	2015-12-09 16:58:54.812518
6	11	2015-12-09 16:58:54.85802	2015-12-09 16:58:54.85802
6	3	2015-12-09 16:58:54.862233	2015-12-09 16:58:54.862233
7	11	2015-12-09 16:58:54.906526	2015-12-09 16:58:54.906526
7	3	2015-12-09 16:58:54.910518	2015-12-09 16:58:54.910518
34	12	2015-12-09 16:58:56.61299	2015-12-09 16:58:56.61299
34	12	2015-12-09 16:58:56.897475	2015-12-09 16:58:56.897475
35	13	2015-12-09 16:58:56.963121	2015-12-09 16:58:56.963121
36	13	2015-12-09 16:58:57.008305	2015-12-09 16:58:57.008305
38	13	2015-12-09 16:58:57.088227	2015-12-09 16:58:57.088227
41	13	2015-12-09 16:58:57.255087	2015-12-09 16:58:57.255087
43	13	2015-12-09 16:58:57.339217	2015-12-09 16:58:57.339217
44	13	2015-12-09 16:58:57.386221	2015-12-09 16:58:57.386221
45	13	2015-12-09 16:58:57.428712	2015-12-09 16:58:57.428712
46	14	2015-12-09 16:58:57.491768	2015-12-09 16:58:57.491768
46	14	2015-12-09 16:58:57.555166	2015-12-09 16:58:57.555166
47	14	2015-12-09 16:58:57.599252	2015-12-09 16:58:57.599252
47	14	2015-12-09 16:58:57.645542	2015-12-09 16:58:57.645542
48	14	2015-12-09 16:58:57.693879	2015-12-09 16:58:57.693879
48	14	2015-12-09 16:58:57.741219	2015-12-09 16:58:57.741219
49	14	2015-12-09 16:58:57.799	2015-12-09 16:58:57.799
49	14	2015-12-09 16:58:57.844844	2015-12-09 16:58:57.844844
50	14	2015-12-09 16:58:57.887464	2015-12-09 16:58:57.887464
50	14	2015-12-09 16:58:57.934591	2015-12-09 16:58:57.934591
52	7	2015-12-09 16:58:57.997661	2015-12-09 16:58:57.997661
54	7	2015-12-09 16:58:58.049069	2015-12-09 16:58:58.049069
66	4	2015-12-09 16:58:59.562219	2015-12-09 16:58:59.562219
66	4	2015-12-09 16:58:59.632271	2015-12-09 16:58:59.632271
66	4	2015-12-09 16:58:59.692939	2015-12-09 16:58:59.692939
66	4	2015-12-09 16:58:59.741367	2015-12-09 16:58:59.741367
70	1	2015-12-09 16:58:59.792176	2015-12-09 16:58:59.792176
67	4	2015-12-09 16:58:59.927805	2015-12-09 16:58:59.927805
72	4	2015-12-09 16:58:59.993474	2015-12-09 16:58:59.993474
73	4	2015-12-09 16:59:00.064405	2015-12-09 16:59:00.064405
73	4	2015-12-09 16:59:00.116839	2015-12-09 16:59:00.116839
148	12	2015-12-09 16:59:00.503425	2015-12-09 16:59:00.503425
78	12	2015-12-09 16:59:00.621828	2015-12-09 16:59:00.621828
79	12	2015-12-09 16:59:00.794269	2015-12-09 16:59:00.794269
147	12	2015-12-09 16:59:00.859477	2015-12-09 16:59:00.859477
146	12	2015-12-09 16:59:01.101386	2015-12-09 16:59:01.101386
110	15	2015-12-09 16:59:02.394101	2015-12-09 16:59:02.394101
110	15	2015-12-09 16:59:02.461079	2015-12-09 16:59:02.461079
124	9	2015-12-09 16:59:03.282561	2015-12-09 16:59:03.282561
126	9	2015-12-09 16:59:03.327193	2015-12-09 16:59:03.327193
\.


--
-- Name: variant_groups_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_groups_id_seq', 15, true);


--
-- Name: variants_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variants_id_seq', 151, true);


--
-- PostgreSQL database dump complete
--

