--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

SET search_path = public, pg_catalog;

--
-- Data for Name: audits; Type: TABLE DATA; Schema: public; Owner: acoffman
--

SET SESSION AUTHORIZATION DEFAULT;

ALTER TABLE audits DISABLE TRIGGER ALL;

COPY audits (id, auditable_id, auditable_type, associated_id, associated_type, user_id, user_type, username, action, audited_changes, version, comment, remote_address, request_uuid, created_at) FROM stdin;
1	1	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 238\nname: ALK\ndescription: ALK amplifications, fusions and mutations have been shown to be driving\n  events in non-small cell lung cancer. While crizontinib has demonstrated efficacy\n  in treating the amplification, mutations in ALK have been shown to confer resistance\n  to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied\n  success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been\n  shown to be cytostatic in ALK-altered cell lines.\nofficial_name: anaplastic lymphoma receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	f7c3c750-a51f-460e-bf69-0f84588b5687	2015-02-26 02:04:56.58921
2	2	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 207\nname: AKT1\ndescription: AKT1, also referred to as protein kinase B, is a known oncogene. AKT\n  activation relies on the PI3K pathway, and is recognized as a critical node in the\n  pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been\n  shown to result in activation of the protein. Mutations in AKT1 have also been shown\n  to confer resistance to allosteric kinase inhibitors in vitro.\nofficial_name: v-akt murine thymoma viral oncogene homolog 1\nclinical_description: \n	1	\N	\N	ca25fd4d-8878-468c-8dd8-a47571abc900	2015-02-26 02:04:56.851722
3	3	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 369\nname: ARAF\ndescription: ARAF has recently become increasingly considered for its oncogenic potential.\n  Its potential as a target for informing clinical action was demonstrated by a single\n  case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated\n  with sorafenib, acheived near-complete clinical remission. This finding has brought\n  new focus on ARAF as a marker that should be assayed for in cancer treatment.\nofficial_name: A-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	f91a1ee1-9a1c-4a0b-a713-501f4d780e80	2015-02-26 02:04:57.095492
4	4	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 25\nname: ABL1\ndescription: ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein\n  that has become a signature of chronic myeloid leukemia (CML). Cells harboring this\n  fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook\n  of the disease. However, additional mutations in ABL1 have been shown to confer\n  resistance to imatinib. In these resistance cases, second-generation tyrosine kinase\n  inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are\n  currently undergoing clinical trials for treating acquired resistance in CML.\nofficial_name: ABL proto-oncogene 1, non-receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	6b2c9d1c-0e46-4c6f-ac42-67f5b90eaf61	2015-02-26 02:04:57.371248
5	5	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 673\nname: BRAF\ndescription: BRAF mutations are found to be recurrent in many cancer types. Of these,\n  the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E\n  has been determined to be an activating mutation, and cells that harbor it, along\n  with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is\n  also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK\n  inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations\n  have also been correlated with poor prognosis in many cancer types, although there\n  is at least one study that questions this conclusion in papillary thyroid cancer.\nofficial_name: B-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	4c093df4-7a38-476d-9a41-ea3038c7d7b6	2015-02-26 02:04:57.634786
6	6	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 672\nname: BRCA1\ndescription: BRCA1 mutations in the germline have become a hallmark for hereditary\n  breast and ovarian cancers. Variants that have been demonstrated to reduce the function\n  of the protein have been shown to increase risk for these cancers, as well as prostate\n  and pancreatic cancer. These findings have been the impetus for the increased popularity\n  of genetic testing of healthy indivudals to assess risk.\nofficial_name: breast cancer 1, early onset\nclinical_description: \n	1	\N	\N	3b050dd0-ff41-445c-8875-066335483fd3	2015-02-26 02:04:57.906773
7	7	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 675\nname: BRCA2\ndescription: BRCA2 mutations in the germline have become a hallmark for hereditary\n  breast and ovarian cancers. Variants that have been demonstrated to reduce the function\n  of the protein have been shown to increase risk for these cancers, as well as prostate\n  and pancreatic cancer. These findings have been the impetus for the increased popularity\n  of genetic testing of healthy indivudals to assess risk.\nofficial_name: breast cancer 2, early onset\nclinical_description: \n	1	\N	\N	d0855c25-87f9-42cc-9d5a-4ba9d584165e	2015-02-26 02:04:58.131152
8	8	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 595\nname: CCND1\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D1\nclinical_description: \n	1	\N	\N	dc542cfa-9a73-4fbf-b309-40d16bb87fa1	2015-02-26 02:04:58.35791
9	9	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 894\nname: CCND2\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D2\nclinical_description: \n	1	\N	\N	1b12b348-d850-4291-9eaa-114a7b8329e7	2015-02-26 02:04:58.584957
10	10	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 896\nname: CCND3\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared\n  to the amplifcations and overexpressions of the other cyclin D's. In a mouse study,\n  the targeted therapeutic palbociclib significantly increased the median survival\n  of the cyclin D3 knockouts.\nofficial_name: cyclin D3\nclinical_description: \n	1	\N	\N	c94e2c84-fc85-412a-912e-d17f8b7cd029	2015-02-26 02:04:58.845954
132	83	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 32\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	a03df83a-5b71-4efd-bda9-12633e4aa89f	2015-02-26 02:05:10.921025
11	11	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 898\nname: CCNE1\ndescription: Cyclin E, while currenly not as widely implicated as its cyclin D counterparts,\n  has been implicated in various carcinomas, including breast, gastric, stomach and\n  colorectal. High levels of cyclin E, either by gene amplification or overexpression,\n  have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements\n  are correlated with later stage disease. In lung cancer, neoplastic cells with higher\n  levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly\n  expressed counterparts.\nofficial_name: cyclin E1\nclinical_description: \n	1	\N	\N	fb52e913-92c2-4d01-b7b4-24301f8ffd20	2015-02-26 02:04:59.078372
12	12	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1021\nname: CDK6\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 6\nclinical_description: \n	1	\N	\N	60bbd98b-a202-4cb6-9859-d335ea149eb2	2015-02-26 02:04:59.312915
13	13	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1019\nname: CDK4\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 4\nclinical_description: \n	1	\N	\N	30353da6-d7dd-4e10-80cd-d9d87babe9a2	2015-02-26 02:04:59.551923
14	14	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1029\nname: CDKN2A\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. While no targeted therapeutic has been engaged in clinical trials, the prognostic\n  impact has been studied by a number of meta-analyses. One mechanism by which loss\n  of CDKN2A can occur is by hypermethylation of the promoter region for the gene.\n  However, the prognostic impact of promoter hypermethylation has been relatively\n  ambiguous. Many studies have suggesting poorer prognostic outcome for patients with\n  hypermethylation in colorectal, liver, and younger lung cancer patients. This being\n  said, there is still research to be done before this becomes a widely-accepted prognostic\n  indicator.\nofficial_name: cyclin-dependent kinase inhibitor 2A\nclinical_description: \n	1	\N	\N	bd127d99-698a-4d79-b778-f5646ea7b83d	2015-02-26 02:04:59.784156
15	15	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1050\nname: CEBPA\ndescription: AML with mutated CEBPA' is a provisional entity in the WHO classification\n  of acute myeloid leukemia (AML) and is recommended to be tested for in patients\n  with AML. CEBPA mutations are particularly associated with cytogenetically normal\n  AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation\n  in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha\n  protein while C-terminal mutations reduce the DNA-binding potential of this transcription\n  factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1\n  and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations\n  may have prognostic implications.\nofficial_name: CCAAT/enhancer binding protein (C/EBP), alpha\nclinical_description: \n	1	\N	\N	31c25f4a-3e3b-43db-83bf-1f182787c2c1	2015-02-26 02:05:00.03712
16	16	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3337\nname: DNAJB1\ndescription: DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating\n  p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1\n  to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In\n  one study, this fusion was observed in 15/15 FL-HCC cases examined and functional\n  studies found that the fusion retained kinase activity.\nofficial_name: DnaJ (Hsp40) homolog, subfamily B, member 1\nclinical_description: \n	1	\N	\N	a3a39427-2278-4b22-8088-a50407a43a8a	2015-02-26 02:05:00.270042
17	17	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5566\nname: PRKACA\ndescription: PRKACA has been studied in breast cancer and has been found to mediate\n  resistance to HER2 targeted therapies. It has also been found to contain a mutation\n  hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular\n  fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be\n  a diagnostic marker for this rare subtype of HCC. In one study, this fusion was\n  observed in 15/15 FL-HCC cases examined and functional studies found that the fusion\n  retained kinase activity.\nofficial_name: protein kinase, cAMP-dependent, catalytic, alpha\nclinical_description: \n	1	\N	\N	7413ff6e-b631-4664-b1dd-b77a6b64cd29	2015-02-26 02:05:00.505393
18	18	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1788\nname: DNMT3A\ndescription: DNMT3A is one of several epigenetic modifiers identified as recurrently\n  mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically\n  normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant\n  negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers.\n  AML patient bone marrow harboring R882 mutations were similarly demonstrated to\n  be hypomethylated compared to patients with wildtype DNMT3A. These studies also\n  indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner\n  from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes\n  and treatment protocols may need to be considered for these two classes of DNMT3A\n  mutations.\nofficial_name: DNA (cytosine-5-)-methyltransferase 3 alpha\nclinical_description: \n	1	\N	\N	e604d49b-db5b-4c95-885c-4c68e3451ec7	2015-02-26 02:05:00.768096
19	19	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1956\nname: EGFR\ndescription: EGFR is widely recognized for its importance in cancer. Amplification\n  and mutations have been shown to be driving events in many cancer types. Its role\n  in non-small cell lung cancer has spurred many research and drug development efforts.\n  Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably\n  gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance\n  to these drugs, particularly the variant T790M, which has been functionally characterized\n  as a resistance marker for both of these drugs. The later generation TKI's have\n  seen some success in treating these resistant cases, and targeted sequencing of\n  the EGFR locus has become a common practice in non-small cell lung cancer.\nofficial_name: epidermal growth factor receptor\nclinical_description: \n	1	\N	\N	fea2777d-70b1-47a7-af5a-94e0e66730eb	2015-02-26 02:05:01.056983
20	20	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2064\nname: ERBB2\ndescription: ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast\n  cancer, and is treated in a separate manner than the other subtypes of breast cancer.\n  Apart from being amplified, ERBB2 activating mutations have been shown to have clinical\n  importance in HER2-negative breast cancer. These mutations have shown sensitivity\n  to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical\n  sequencing efforts in treating breast cancer.\nofficial_name: erb-b2 receptor tyrosine kinase 2\nclinical_description: \n	1	\N	\N	76f28bca-c915-4824-b2e2-a4816a3c9174	2015-02-26 02:05:01.343985
21	21	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2099\nname: ESR1\ndescription: ESR1 has been a focus in breast cancer for quite some time, but has also\n  shown significance in endometrial, ovarian and other cancer types. ER-positive breast\n  cancer that is resistant to hormone therapy has instigated clinical sequencing efforts\n  to shed light on the mechanisms of this resistance. A number of mutations in the\n  ligand binding domain of the protein have been implicated in hormone resistance\n  and anti-estrogen therapies. This has spurred efforts to develop therapeutics that\n  act to degrade the protein, rather than act as an antagonist. These agents are currently\n  in clinical trials and have seen some success, highlighting the importance of sequencing\n  efforts in treating breast cancer.\nofficial_name: estrogen receptor 1\nclinical_description: \n	1	\N	\N	53ac8e12-ec91-4120-8d00-f0c7c4541b6f	2015-02-26 02:05:01.640404
22	22	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2263\nname: FGFR2\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions that lead to pathway\n  actiation have been demonstrated to have oncogenic potential across multiple cancer\n  types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success\n  in treating over-active FGFR signalling, prompting use of diagnostic sequencing\n  targeting the FGFR genes, especially in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 2\nclinical_description: \n	1	\N	\N	20210bcc-420c-4dfa-8ec7-64b730fc52f3	2015-02-26 02:05:01.917659
23	23	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2261\nname: FGFR3\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions that lead to pathway\n  actiation have been demonstrated to have oncogenic potential across multiple cancer\n  types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success\n  in treating over-active FGFR signalling, prompting use of diagnostic sequencing\n  targeting the FGFR genes, especially in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 3\nclinical_description: \n	1	\N	\N	dbacdc25-9b4b-4bbf-aff5-5af7fda71907	2015-02-26 02:05:02.184233
24	24	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2322\nname: FLT3\ndescription: FLT3 is an important cytokine receptor involved in normal hematopoiesis.\n  Mutations in this gene are common in acute myeloid leukemia (AML) and screening\n  for mutations in this gene has been recommended by the World Health Organization\n  in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML).\n  FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated\n  with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have\n  been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD\n  mutations are still up for debate.\nofficial_name: fms-related tyrosine kinase 3\nclinical_description: \n	1	\N	\N	62d6d952-2ca0-48c6-91c5-c2691995a155	2015-02-26 02:05:02.474518
25	25	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2624\nname: GATA2\ndescription: GATA2 is a transcription factor involved in stem cell maintenance with\n  key roles in hematopoietic development. GATA2 mutations are associated with a variety\n  of inherited and acquired immune disorders including myelodysplastic syndrome and\n  acute myeloid leukemia. In addition to a role in hematopoiesis, the maintenance\n  GATA2 expression has been implicated as a requirement in KRAS-driven non-small cell\n  lung cancer. Preclinical models have indicated therapeutic benefit from targeting\n  GATA2-mediated pathways in the context of KRAS-driven NSCLC.\nofficial_name: GATA binding protein 2\nclinical_description: \n	1	\N	\N	3efa61d2-0af1-45f2-be3e-92c60c701c54	2015-02-26 02:05:02.732617
26	26	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3417\nname: IDH1\ndescription: IDH1 mutations have been observed in a number of cancer types, including\n  sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in\n  the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial\n  (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described\n  as mutually exclusive in many of these cancer types. The most frequent mutations\n  involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic\n  enzyme activity. The implications of mutations in this gene vary greatly by cancer\n  type. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations\n  have been associated with worse outcome, shorter overall survival, and normal karyotype.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Unlike the association\n  with cytogenetically normal AML, in glioblastoma, IDH1 mutations have been associated\n  with specific cytogenetic abnormalities, 1p and 19q deletions.\nofficial_name: isocitrate dehydrogenase 1 (NADP+), soluble\nclinical_description: \n	1	\N	\N	1b1cb913-b38f-4cfd-ac35-2d33dccdbc8b	2015-02-26 02:05:02.980748
27	27	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3418\nname: IDH2\ndescription: IDH2 mutations have been observed in a number of cancer types, including\n  sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in\n  the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial\n  (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described\n  as mutually exclusive in many of these cancer types. The most frequent mutations\n  involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic\n  enzyme activity. Although IDH2 (R172) mutations are associated with poorer overall\n  prognosis in AML patients, its utility as a prognostic marker in MDS is still under\n  debate. Additionally, IDH2 (R140) has been associated with improved overall survival\n  in AML. IDH2 mutations have been associated with improved prognosis in gliomas.\nofficial_name: isocitrate dehydrogenase 2 (NADP+), mitochondrial\nclinical_description: \n	1	\N	\N	c8ed67e8-7bd6-4ef7-8dd8-6904b2ca0bdb	2015-02-26 02:05:03.251071
28	28	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3717\nname: JAK2\ndescription: JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative\n  diseases and cancers. The mutation V617F is the most clinically relevant variant,\n  and is seen in around half of myeloproliferative disorders. The variant is a known\n  activating mutation, and activated JAK2 is sufficient to drive myeloproliferative\n  disorders in mouse models. V617F, while most recurrent, is not the only mechanism\n  by which JAK2 can be activated in patients. JAK2 is now one of the first diagnostic\n  markers tested upon diagnosis with a myeloproliferative disorder.\nofficial_name: Janus kinase 2\nclinical_description: \n	1	\N	\N	20b16ad5-36e6-4fd5-b11f-c61131f3b1fb	2015-02-26 02:05:03.505865
29	29	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3815\nname: KIT\ndescription: c-KIT activation has been shown to have oncogenic activity in gastrointestinal\n  stromal tumors (GISTs), melanomas, lung cancer, and other tumor types. The targeted\n  therapeutics nilotinib and sunitinib have shown efficacy in treating KIT overactive\n  patients, and are in late-stage trials in melanoma and GIST. KIT overactivity can\n  be the result of many genomic events from genomic amplification to overexpression\n  to missense mutations. Missense mutations have been shown to be key players in mediating\n  clinical response and acquired resistance in patients being treated with these targeted\n  therapeutics.\nofficial_name: v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	1d88d227-bf9d-4955-855e-52ee2334825e	2015-02-26 02:05:03.744994
30	30	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3845\nname: KRAS\ndescription: Mutations in the RAS family of proteins have frequently observed across\n  cancer types. The amino acid positions account for the overwhelming majority of\n  these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also\n  behave very differently when expressed in non-native tissue types, likely due to\n  differences in the C-terminal hyper-variable regions. Mis-regulation of isoform\n  expression has been shown to be a driving event in cancer, as well as missense mutations\n  at the three hotspots previously mentioned. While highly recurrent in cancer, targeted\n  these RAS mutants has also been very elusive, and has not yet become common practice\n  in the clinic.\nofficial_name: Kirsten rat sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	19fc5986-1d90-4bda-81f0-467aa9e04eb9	2015-02-26 02:05:04.006554
31	31	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5604\nname: MAP2K1\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: mitogen-activated protein kinase kinase 1\nclinical_description: \n	1	\N	\N	786c5838-f441-4a70-9aaf-5e82b61aa412	2015-02-26 02:05:04.236988
32	32	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4209\nname: MEF2D\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: myocyte enhancer factor 2D\nclinical_description: \n	1	\N	\N	8cba7945-d534-4095-a11e-bd64b7196474	2015-02-26 02:05:04.46257
33	33	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1436\nname: CSF1R\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: colony stimulating factor 1 receptor\nclinical_description: \n	1	\N	\N	dda893c5-5931-4d57-9bf7-0eb296d2cb84	2015-02-26 02:05:04.69712
34	34	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4255\nname: MGMT\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: O-6-methylguanine-DNA methyltransferase\nclinical_description: \n	1	\N	\N	0e9e8b25-ecf3-4d20-836c-da1a59d3c805	2015-02-26 02:05:04.912226
35	35	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4869\nname: NPM1\ndescription: AML with mutated NPM1 is provisional entity in the WHO classification\n  of AML and is recommended to be tested for in patients with cytogenetically normal\n  AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they\n  have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently\n  W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations\n  in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing\n  the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia,\n  indicating it may require additional mutations to promote leukemic development.\nofficial_name: nucleophosmin (nucleolar phosphoprotein B23, numatrin)\nclinical_description: \n	1	\N	\N	8a17d79b-b130-4b84-b5d6-0bf6d32e2d78	2015-02-26 02:05:05.140744
36	36	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4893\nname: NRAS\ndescription: Mutations in the RAS family of proteins have frequently observed across\n  cancer types. The amino acid positions account for the overwhelming majority of\n  these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also\n  behave very differently when expressed in non-native tissue types, likely due to\n  differences in the C-terminal hyper-variable regions. Mis-regulation of isoform\n  expression has been shown to be a driving event in cancer, as well as missense mutations\n  at the three hotspots previously mentioned. While highly recurrent in cancer, targeted\n  these RAS mutants has also been very elusive, and has not yet become common practice\n  in the clinic.\nofficial_name: neuroblastoma RAS viral (v-ras) oncogene homolog\nclinical_description: \n	1	\N	\N	bed9ebcd-85cb-4668-aae6-da2f298965d3	2015-02-26 02:05:05.385428
37	37	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5290\nname: PIK3CA\ndescription: PIK3CA is the most recurrently mutated gene in breast cancer, and has\n  been found to important in a number of cancer types. An integral part of the PI3K\n  pathway, PIK3CA has long been described as an oncogene, with two main hotspots for\n  activating mutations, the 542/545 region of the helical domain, and the 1047 region\n  of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways,\n  is the subject of an immense amount of research and development, and PI3K inhibition\n  has seen some limited success in recent clinical trials. While monotherapies seem\n  to be limited in their potential, there is a recent interest in pursuing PI3K inhibition\n  as part of a combination therapy regiment with inhibition partners including TKI's,\n  MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.\nofficial_name: phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha\nclinical_description: \n	1	\N	\N	3d12234b-cafb-4e13-8398-618d59c762e7	2015-02-26 02:05:05.61864
38	38	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5156\nname: PDGFRA\ndescription: N/A\nofficial_name: platelet-derived growth factor receptor, alpha polypeptide\nclinical_description: \n	1	\N	\N	d3d6ccb3-6db9-4720-b87f-31701808f813	2015-02-26 02:05:05.843115
39	39	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5371\nname: PML\ndescription: The PML-RARA fusion is the result of a recurrent, balanced translocation\n  between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event\n  in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown\n  sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA\n  fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for\n  treating these patients, and early results seem promising.\nofficial_name: promyelocytic leukemia\nclinical_description: \n	1	\N	\N	9bb837a1-17a3-46bf-acbf-59b96204f771	2015-02-26 02:05:06.102181
40	40	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5914\nname: RARA\ndescription: The PML-RARA fusion is the result of a recurrent, balanced translocation\n  between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event\n  in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown\n  sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA\n  fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for\n  treating these patients, and early results seem promising.\nofficial_name: retinoic acid receptor, alpha\nclinical_description: \n	1	\N	\N	ef268ef2-1e99-4f0c-bd57-983aa1a350bd	2015-02-26 02:05:06.325182
41	41	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5728\nname: PTEN\ndescription: N/A\nofficial_name: phosphatase and tensin homolog\nclinical_description: \n	1	\N	\N	c7b2b427-e88e-4f0d-ba88-320feb42576c	2015-02-26 02:05:06.548294
42	42	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5979\nname: RET\ndescription: RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine\n  kinase receptor and are associated with thyroid cancers. RET point mutations are\n  the most common mutations identified in medullary thyroid cancer (MTC) with germline\n  and somatic mutations in RET associated with hereditary and sporadic forms, respectively.\n  The most common somatic form mutation is M918T (exon 16) and a variety of other\n  mutations effecting exons 10, 11 and 15 have been described. The prognostic significance\n  of these mutations have been hotly debated in the field, however, data suggests\n  that some RET mutation may confer drug resistence. No RET-specific agents are currently\n  clinically available but several promiscuous kinase inhibitors that target RET,\n  among others, have been approved for MTC treatment.\nofficial_name: ret proto-oncogene\nclinical_description: \n	1	\N	\N	12bd10a0-e66e-4a15-83a4-c5287a40b67f	2015-02-26 02:05:06.862311
43	43	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 861\nname: RUNX1\ndescription: N/A\nofficial_name: runt-related transcription factor 1\nclinical_description: \n	1	\N	\N	c266a6c8-2cc3-43b8-8712-02ccde13dc50	2015-02-26 02:05:07.087645
44	44	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 23451\nname: SF3B1\ndescription: SF3B1 mutations have been described in several myeloid malignancies,\n  predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies\n  and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has\n  been identified as recurrently mutated in MDS and other malignanices. The mutations\n  affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally\n  deleterious with K700E described as a major hotspot mutation. MDS patients with\n  SF3B1 mutations have been reported to have better overall and event-free survival\n  than their wildtype counterparts. Additionally, these mutations are highly associated\n  with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with\n  ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring\n  sideroblasts).\nofficial_name: splicing factor 3b, subunit 1, 155kDa\nclinical_description: \n	1	\N	\N	93ce49fd-04bb-4e4c-b92f-9742418d509d	2015-02-26 02:05:07.3175
45	45	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7157\nname: TP53\ndescription: TP53 mutations are universal across cancer types. Loss of tumor suppressors\n  is most recognized by large deleterious events, such as frameshift mutations, or\n  premature stop codons. In TP53 however, many of the observed mutations in cancer\n  are found to be single nucleotide variants, or missense mutations. These variants\n  are also very broadly distributed throughout the gene, not localizing in any particular\n  hotspot. While a large proportion of cancer genomics research is focused on somatic\n  variants, TP53 is also of note in the germline. Germline TP53 mutations are the\n  hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been\n  found to have prognostic impact on patient outcomes.\nofficial_name: tumor protein p53\nclinical_description: \n	1	\N	\N	915e2dc2-a363-45aa-a1a9-eca312fa0fd0	2015-02-26 02:05:07.574162
46	46	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7248\nname: TSC1\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: tuberous sclerosis 1\nclinical_description: \n	1	\N	\N	9b2278ea-6da5-4879-99e8-d8db1a69dc68	2015-02-26 02:05:07.794771
47	47	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7249\nname: TSC2\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: tuberous sclerosis 2\nclinical_description: \n	1	\N	\N	698b0f9d-cc60-4275-815a-60be3b99b2e2	2015-02-26 02:05:08.046789
48	48	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7307\nname: U2AF1\ndescription: U2AF1 is one of several spliceosome complex genes frequently mutated\n  in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes\n  (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot\n  mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein.\n  These mutations have been associated with altered splicing patterns in vitro and\n  in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation\n  to secondary acute myeloid leukemia, however, the impact of these mutations on overall\n  survival has been an area of debate.\nofficial_name: U2 small nuclear RNA auxiliary factor 1\nclinical_description: \n	1	\N	\N	4d1355b1-afe1-4f90-a965-e23cef5b4713	2015-02-26 02:05:08.330293
49	49	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7490\nname: WT1\ndescription: WT1 is a tumor suppressor gene associated with the development of Wilms'\n  Tumor, from which it was named. Mutations in exon 7 and 9 of WT1 have been recurrently\n  identified in acute myeloid leukemia and associated with poorer prognosis and chemotherapy\n  resistance.\nofficial_name: Wilms tumor 1\nclinical_description: \n	1	\N	\N	46cc80ab-906e-4ee0-8f93-71821213a3b3	2015-02-26 02:05:08.564722
50	1	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL\ndescription: The BCR-ABL fusion protein, commonly referred to as the philadelphia\n  chromosome, is the most well-studied fusion gene in cancer. It has widely been considered\n  the initiating event in chronic myelogenous leukemia (CML), but despite its ability\n  initiate disease in mice, its status an initiating mutation is in dispute. In what\n  is commonly used as the poster-child for targeted therapeutics, the development\n  and use of imatinib in the clinic has led to profound improvements in the prognosis\n  of the disease. However, imatinib resistance is still seen in patients with mutations\n  in the ABL kinase domain of the fusion, most notably the T315I variant. In patients\n  resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's\n  (dasatinib and nilotinib) have seen some success in delivering a tumor response.\n	1	\N	\N	d28ec0e0-7e29-4b41-ae41-30fa03a49cb3	2015-02-26 02:05:08.616822
51	2	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL T315I\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation T315I has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	44b08db5-160f-465d-9131-b2fd4e306d9b	2015-02-26 02:05:08.650009
52	3	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL E255K\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation E255K has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	db217448-fdbf-463b-a547-92c7e0e1d770	2015-02-26 02:05:08.676585
161	112	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: K666N\ndescription: SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients.\n	1	\N	\N	b4c06da1-06b8-4070-90ba-eb67af181385	2015-02-26 02:05:11.720321
53	4	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 2\nname: E17K\ndescription: AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal,\n  lung, and ovarian cancer. It has been convincingly shown to be an activating mutation\n  resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation\n  decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and\n  other AKT1 mutations, are the subject of much research and development for therapeutics.\n	1	\N	\N	29dc5107-1ded-4d3e-b060-778cf8549bcf	2015-02-26 02:05:08.699632
54	5	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK\ndescription: The EML4-ALK fusion has been seen in non-small cell lung cancer, and\n  appears to be an alternative mechanism for ALK activation. Cells with this fusion\n  have been shown to be sensitive to the ALK inhibitor crizotinib.\n	1	\N	\N	a9d32e4b-a553-45f1-be5c-f3686b5ccdcb	2015-02-26 02:05:08.728077
55	6	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK C1156Y\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	47edfcef-5b83-49de-8432-9cb14acdaa64	2015-02-26 02:05:08.753909
56	7	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK L1196M\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	68004d3f-b4cd-447b-9148-a7052d17de82	2015-02-26 02:05:08.781491
57	8	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: F1174L\ndescription: ALK F1174L has been observed as recurrent in neuroblastoma, non-small\n  cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing\n  this mutation have shown resistance to low doses of criztonib. However, increased\n  dosage can overcome this resistance in cell lines studies. TAE684 has also proven\n  effective in both NSCLC and neuroblastoma F1174L containing cells.\n	1	\N	\N	b3d90c43-20d2-4bad-8fe8-5bce0f5be933	2015-02-26 02:05:08.803834
58	9	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: R1275Q\ndescription: ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma,\n  non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma\n  cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This,\n  and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC\n  cell lines.\n	1	\N	\N	617f6c8b-b18c-4fe9-9f11-f427ce76d70e	2015-02-26 02:05:08.828952
59	10	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 3\nname: S214C\ndescription: ARAF S214C has been found to be a recurrent oncogenic mutation in non-small\n  cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib\n  in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this\n  mutation, sorafenib also acheived near-complete clinical remission. This case has\n  brought more interest to the variant from a research and clinical perspective.\n	1	\N	\N	a4f97109-4f1c-4d9d-ac8e-b23bc398ce82	2015-02-26 02:05:08.852023
60	11	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600D\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	94e1e2b8-bf44-482c-b23b-cf68b42da596	2015-02-26 02:05:08.874715
61	12	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E\ndescription: BRAF V600E has been shown to be recurrent in many cancer types. It is\n  one of the most widely studied variants in cancer. This variant is correlated with\n  poor prognosis in certain cancer types, including colorectal cancer and papillary\n  thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective\n  in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib\n  has also shown to be effective when combined with the MEK inhibitor trametinib in\n  colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA\n  mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib,\n  and a number of combination therapies have been successful in treating V600E mutations.\n  While the drugs cetuximab and panitumumab have been largely shown to be ineffective\n  without supplementary treatment.\n	1	\N	\N	62504bc0-7b54-4775-9918-198575973e57	2015-02-26 02:05:08.899216
62	13	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E/V600M\ndescription: A case study of a single patient harboring both a V600E and a V600M mutation,\n  dabrafenib was shown to acheive clinical response.\n	1	\N	\N	216f76a3-7eeb-4af8-b816-04dce2c48524	2015-02-26 02:05:08.92783
63	14	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E AMPLIFICATION\ndescription: Amplification of BRAF V600E has been shown to confer resistance to MEK\n  inhibitors. For more information on the V600 locus, see the V600E entry.\n	1	\N	\N	50f952c7-c528-47e8-9e94-574b6ff16a05	2015-02-26 02:05:08.951618
64	15	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600M\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	c93f1591-da90-45fd-913e-10ff1f7b4a76	2015-02-26 02:05:08.973591
65	16	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600R\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	42e660ec-b035-43cf-9baa-6013ba61ab05	2015-02-26 02:05:09.002473
66	17	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600\ndescription: BRAF mutations of the valine 600 residue have been shown to be recurrent\n  in many cancer types. Of the V600 mutations, V600E is the most widely researched.\n  V600 mutations as a whole have been correlated to poorer prognosis in colorectal\n  and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity\n  to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual\n  mutation pages on the left sidebar.\n	1	\N	\N	7cad3314-c521-408f-9336-4534fba28aee	2015-02-26 02:05:09.032727
67	18	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: AMPLIFICATION\ndescription: CCND1 amplification has been implicated in poorer prognosis in non-small\n  cell lung cancer.\n	1	\N	\N	23fc9265-0161-4af9-b67f-d2bd5e4ade56	2015-02-26 02:05:09.061432
68	19	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: EXPRESSION\ndescription: CCND1 expression, and its prognositc impact, is still in dispute. Three\n  experiments in non-small cell lung cancer have shown it to have no impact on survival,\n  but three additional studies have shown it results in poorer prognosis. There is\n  also some ambiguity in how the boundaries between expression and overexpression\n  are defined.\n	1	\N	\N	5309faed-a110-4274-bb3a-1d1644d0f54e	2015-02-26 02:05:09.089395
69	20	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	02db98e0-89b1-4e0e-9307-afd427c23ab1	2015-02-26 02:05:09.114274
70	21	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	d6541716-e1ad-4b3e-8b38-1ac74db67765	2015-02-26 02:05:09.144838
71	22	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: PROMOTER DEMETHYLATION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	0830500c-972d-4411-94b9-f2003fd53161	2015-02-26 02:05:09.192802
72	23	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 10\nname: LOSS\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL),\n  a seemingly unique trend when compared to the amplifcations and overexpressions\n  of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic\n  palbociclib significantly increased the median survival of a Notch-driven model\n  of T-ALL.\n	1	\N	\N	3278165b-8312-49ca-829d-098decbd3ff0	2015-02-26 02:05:09.219332
73	24	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 11\nname: OVEREXPRESSION\ndescription: Cyclin E, while currenly not as widely implicated as its counterpart,\n  cyclin D, has been implicated in various carcinomas, including breast, gastric,\n  stomach and colorectal. High levels of cyclin E, either by gene amplification or\n  overexpression, are correlated with later stage disease and have been shown to lead\n  to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer\n  specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex\n  are more radiosensitive than their lowly expressed counterparts.\n	1	\N	\N	0f34ee57-6bd7-4adf-953a-0ee9cff45428	2015-02-26 02:05:09.247187
74	25	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: EXPRESSION\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	74e03be2-780d-4081-9a20-8eeb5714ce6c	2015-02-26 02:05:09.273492
75	26	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 12\nname: EXPRESSION\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	982a141d-6b51-45d5-b3d6-2333da5a859c	2015-02-26 02:05:09.296929
76	27	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 14\nname: PROMOTER HYPERMETHYLATION\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the CDKN2A\n  promoter region. While no targeted therapeutic has been engaged in clinical trials,\n  the prognostic impact has been studied by a number of meta-analyses. The prognostic\n  impact of promoter hypermethylation has been relatively ambiguous. Many studies\n  have shown significant p-values suggesting poorer prognostic outcomes for patients\n  with hypermethylation in colorectal, liver, and younger lung cancer patients. This\n  being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\n	1	\N	\N	b7618dc0-c9f2-4f1d-9642-f8c2d24a13f8	2015-02-26 02:05:09.321248
77	28	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: N-TERMINAL FRAME SHIFT\ndescription: CEBPA N-terminal frame shift mutations that result in a premature stop\n  codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML).\n  CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3\n  mutations should also be assessed in CN-AML patients as concurrent mutations may\n  have prognostic implications.\n	1	\N	\N	cca25c5b-4708-4ec5-b9d7-f772923ffb9e	2015-02-26 02:05:09.349617
78	29	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: MUTATION\ndescription: CEBPA mutations are associated with cytogenetically normal acute myeloid\n  leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should\n  also be assessed in CN-AML patients as concurrent mutations may have prognostic\n  implications.\n	1	\N	\N	b4033517-342d-4dae-aff0-1d6db1f17d5e	2015-02-26 02:05:09.373515
79	30	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	a0c109f4-a49c-416a-b01e-62380ac3bfa0	2015-02-26 02:05:09.40189
80	31	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in the rare form of adolescent\n  liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion\n  was observed in 15/15 FL-HCC cases examined and functional studies found that the\n  fusion retained kinase activity. The presence of this fusion may be used as a diagnostic\n  marker for this rare tumor type.\n	1	\N	\N	8e98f802-8e4a-425c-aa1f-6e1a71571a41	2015-02-26 02:05:09.424979
91	42	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: R678Q\ndescription: ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	d9034e54-a982-483e-a08e-a41c2c897678	2015-02-26 02:05:09.727332
81	32	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R882\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\n	1	\N	\N	001e06dc-36fb-478b-bca6-24d193ef2ba0	2015-02-26 02:05:09.451031
82	33	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: L858R\ndescription: EGFR L858R has long been recognized as a functionally significant mutation\n  in cancer, and is one of the most prevalent single mutations in lung cancer. Best\n  described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity\n  to first and second generation TKI's like gefitinib and neratinib. NSCLC patients\n  with this mutation treated with TKI's show increased overall and progression-free\n  survival, as compared to chemotherapy alone. Third generation TKI's are currently\n  in clinical trials that specifically focus on mutant forms of EGFR, a few of which\n  have shown efficacy in treating patients that failed to respond to earlier generation\n  TKI therapies.\n	1	\N	\N	460ec475-9fb9-4e6b-b05b-78f5e1f3a649	2015-02-26 02:05:09.478038
83	34	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: T790M\ndescription: EGFR T790M was one of the very first mutations recognized to confer resistance\n  to targeted therapies. While successful in amplified EGFR, the efficacy of the first\n  and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients\n  harboring this mutation before treatment is notably lower. This lack of efficacy\n  can likely be to blame for the poorer prognosis for patients with this mutation\n  as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately\n  half of EGFR mutant tumors with acquired resistance to TKI inhibition have been\n  shown to harbor this mutation, implicating it as a mechanism of acquired therapy\n  resistence. The third generation TKI's are being developed with this resistance\n  problem in mind, and early stage experiments have shown some efficacy of these drugs\n  in previously resistant tumors with mutant EGFR.\n	1	\N	\N	5b76bd2e-bea7-4cf4-b97f-f2c53db4febf	2015-02-26 02:05:09.503309
84	35	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: D769H\ndescription: ERBB2 D769H was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	aa540eb6-1ed2-4fb1-a69a-87cfb2676401	2015-02-26 02:05:09.533987
85	36	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: D769Y\ndescription: ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	72cbb233-00c3-4ca5-ad84-792638dc629c	2015-02-26 02:05:09.558463
86	37	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: DEL 755-759\ndescription: ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants\n  to be functionally classified (Bose et al. 2012). This mutation was shown to be\n  an activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	314072b2-80d1-4344-9c32-bb69a61b2903	2015-02-26 02:05:09.587505
87	38	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: G309A\ndescription: ERBB2 G309A was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	93448b29-320d-49a1-ae08-62d74506f169	2015-02-26 02:05:09.615299
88	39	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: L755S\ndescription: ERBB2 L755S was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike\n  many of the other variants queried. This mutation was also shown to confer resistance\n  to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.\n	1	\N	\N	bbbb4c62-8666-497b-b34b-55fac7dbe234	2015-02-26 02:05:09.645295
89	40	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: L755W\ndescription: ERBB2 L755W was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	d60a8c3d-619b-4605-871d-2021ddacb203	2015-02-26 02:05:09.673671
90	41	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: P780INS\ndescription: ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally\n  classified (Bose et al. 2012). This mutation was shown to be an activating mutation\n  in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	7d598838-314b-4351-b3cc-62f17b334567	2015-02-26 02:05:09.699586
115	66	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: INTERNAL DUPLICATION\ndescription: c-KIT internal duplications have been observed in exon 11, within the\n  juxtamembrane domain. In a case study of an anal melanoma patient harboring this\n  event, imatinib confered marked response. Also, cells harboring exon 11 mutations\n  have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better\n  prognosis to patients treated with the drug in the first year.\n	1	\N	\N	e09d98f3-4af4-4823-b156-b4ced6f92270	2015-02-26 02:05:10.440233
92	43	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: R896C\ndescription: ERBB2 R896C was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	082889db-1188-400e-ba6f-11b6c8e2ec8c	2015-02-26 02:05:09.757391
93	44	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: V777L\ndescription: ERBB2 V777L was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	11639402-cf5b-492e-a88c-e0d76f1e884f	2015-02-26 02:05:09.784188
94	45	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: V842I\ndescription: ERBB2 V842I was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	e6456f54-6406-48cb-b7c5-05a70279b078	2015-02-26 02:05:09.810309
95	46	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: L536Q\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of\n  these ligand binding domain mutations, and is commonly implicated in this hormone\n  resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant,\n  may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	d7564ae9-6912-4cb7-88a2-96cc7a356ebf	2015-02-26 02:05:09.85794
96	47	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: N538G\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	20714414-e65c-4083-b140-621aeca09ee0	2015-02-26 02:05:09.883118
97	48	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537C\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	40256271-c73b-4464-9615-7d553f2555f4	2015-02-26 02:05:09.908383
98	49	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537N\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	09a19016-daea-4e91-adde-252a3b07320f	2015-02-26 02:05:09.936618
99	50	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537S\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	fc72e379-dfc9-41dd-87fb-496dd29225a4	2015-02-26 02:05:09.963329
100	51	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: FGFR2-MGEA5\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. Additionally, tumor size reduction was\n  achieved by both ponatinib and pazopanib treatments administered separately in a\n  single patient with intrahepatic cholangiocarcinoma and this fusion. The authors\n  use these cases to highlight the need for enhanced clinical sequencing efforts.\n	1	\N	\N	57bc4fe6-68fb-4e68-a87f-16162a153b12	2015-02-26 02:05:09.990384
101	52	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: FGFR3-BAIAP2L1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\n	1	\N	\N	a6b98fdd-cbb4-4d24-b732-d0404ef181ca	2015-02-26 02:05:10.020161
102	53	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: FGFR3-TACC3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\n	1	\N	\N	861755c2-c0fb-418b-9b67-04316f5f3a05	2015-02-26 02:05:10.048422
131	82	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: Q56P\ndescription: MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This\n  mutation has been shown to confer considerable resistance to AZD6244 treatment of\n  melanoma cell lines.\n	1	\N	\N	79b63bd8-0119-4080-82b7-272cc5a63536	2015-02-26 02:05:10.893906
103	54	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: ITD MUTATIONS\ndescription: FLT3-ITD (internal tandem duplications) frequently occur in patients\n  with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid\n  leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically\n  normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of\n  FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated\n  sequence resulting in an in-frame duplication event. The length of these duplications\n  can vary widely which may have prognostic consequences, but this has not been conclusively\n  determined. FLT3-ITD mutations overall have generally been associated with poor\n  prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the\n  prognosis associated with this variant.\n	1	\N	\N	6ce69146-0198-4612-8a55-f1480f2a94cf	2015-02-26 02:05:10.078196
104	55	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: TKD MUTATIONS\ndescription: FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common\n  than FLT3-ITD (internal tandem duplication) mutations and may not confer the same\n  prognostic impact. Although the majority of mutations are point mutations effecting\n  D835 (most frequently D835Y), a small proportion involve an in-frame deletion of\n  I836. These mutations are within the activation loop of the second tyrosine kinase\n  domain of FLT3 and thought to result in constitutive activation of the receptor.\n	1	\N	\N	8999f525-435e-4e77-a5d4-2c8169407d2e	2015-02-26 02:05:10.104118
105	56	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: EXPRESSION\ndescription: GATA2 misregulation has been observed in a number of hematologic malignancies,\n  as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using\n  the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung\n  cancer.\n	1	\N	\N	c81488ca-f22a-42e3-9bb6-eafc1ce63c80	2015-02-26 02:05:10.132871
106	57	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	6685d670-5876-4f68-b819-5e17ec3e8e84	2015-02-26 02:05:10.159678
107	58	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132C\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	b46ec626-3fbe-4d7e-9aa9-afdf9f125012	2015-02-26 02:05:10.190234
108	59	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132H\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	c6b3be1d-9bfc-4df3-88c7-feba2fd04443	2015-02-26 02:05:10.217822
109	60	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132L\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	bd2dca5c-cb5a-42dd-96af-d1e4f42d3378	2015-02-26 02:05:10.254611
110	61	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: R140Q/L\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival\n  as compared to their wild-type counterparts. In myelodysplastic syndromes, however,\n  no prognostic link was discovered between IDH2 mutation status and overall survival.\n	1	\N	\N	578a3fff-781a-4b27-823e-e1b3103264b2	2015-02-26 02:05:10.294571
111	62	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: R172K\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis\n  and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic\n  syndromes, studies did not find a prognostic association between this variant and\n  patient outcomes.\n	1	\N	\N	7755df66-6d2f-4968-bb0b-3e07866a9475	2015-02-26 02:05:10.321195
112	63	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: V617F\ndescription: JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases,\n  occuring in around half of all MPD's. While less associated with cancer, when it\n  is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The\n  V617F mutation is an activating mutation, resulting in increased kinase activity.\n  The mutation seems to be restricted to hematologic malignancies. Treatment of JAK\n  mutant diseases with ruxolitinib has seen some clinical success.\n	1	\N	\N	48797394-6c1d-4135-ba60-323bfe325173	2015-02-26 02:05:10.353109
113	64	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: D816V\ndescription: KIT D816V is a mutation observed in acute myeloid leukemia (AML). This\n  variant has been linked to poorer prognosis and worse outcome in AML patients.\n	1	\N	\N	918c6942-5537-49b0-b585-dd503cde2a7e	2015-02-26 02:05:10.382673
114	65	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 11 MUTATIONS\ndescription: c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are\n  very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis\n  than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity\n  to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients\n  treated with the drug in the first year.\n	1	\N	\N	08663542-4a80-445e-ad3b-3c8ca62bad4c	2015-02-26 02:05:10.411547
116	67	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 13 MUTATIONS\ndescription: c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations.\n  These mutations lie within the tyrosine kinase 1 domain, and are found primarily\n  in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations\n  show sensitivity to imatinib and sunitinib treatment. However, only imatinib has\n  seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant\n  cases.\n	1	\N	\N	067b1003-54ff-4b92-9601-f9ed66805ef1	2015-02-26 02:05:10.47257
117	68	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 14 MUTATIONS\ndescription: c-KIT exon 14 mutations lie within the tyrosine kinase domain of the\n  protein. While relatively rare in primary gastrointestinal tumors, they are notably\n  more prevalent in refractory disease, suggesting a role in imatinib resistance.\n  Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.\n	1	\N	\N	3df06753-d1f2-4b85-b89a-eb257899e833	2015-02-26 02:05:10.501468
118	69	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 17 MUTATIONS\ndescription: c-KIT exon 17 mutations lie within the TK2 domain, containing the activation\n  loop of the protein. In cell lines, mutations within this domain have been shown\n  to be sensitive to imatinib. However, in double KIT mutants in which the exon 17\n  mutation is a secondary mutation, cell lines have shown resistance to both imatinib\n  and sunitinib.\n	1	\N	\N	9dd95c35-408b-4d47-8458-f600e22e82b8	2015-02-26 02:05:10.528911
119	70	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 9 MUTATIONS\ndescription: c-KIT exon 9 mutations lie within the dimerization motif of the protein.\n  Relative to other KIT mutations, exon 9 mutations have been associated with better\n  overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and\n  in vivo.\n	1	\N	\N	d3560a19-10c3-422d-803c-a1cdf4420f69	2015-02-26 02:05:10.574839
120	71	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: L576P\ndescription: KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain.\n  It is one of the most recurrent KIT mutations in melanoma, and both in vitro and\n  in vivo studies have shown sensitivity to imatinib.\n	1	\N	\N	b2ff06bd-732e-43fe-8774-e6a45d3470fa	2015-02-26 02:05:10.601377
121	72	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: V654A\ndescription: KIT V654A is an exon 13 mutation that lies within the tyrosine kinase\n  1 domain of the protein. It has been shown to be an activating mutation by in vitro\n  studies. This mutation is associated with imatinib resistance in melanoma patients.\n  However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have\n  seen success in acheiving tumor response.\n	1	\N	\N	79181a52-2c4c-4918-8471-d760364df009	2015-02-26 02:05:10.627205
122	73	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: EXON 1 MUTATIONS\ndescription: A study by Lièvre et al in 2006 showed that colorectal cancer patients\n  with KRAS exon 1 mutations had low cetuximab response rates.\n	1	\N	\N	732294e4-7a44-49b7-9d6f-4eaba4356df6	2015-02-26 02:05:10.662522
123	74	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: EXON 2 MUTATIONS\ndescription: In a study by Pao et al in 2005, non-small cell lung cancer patients\n  harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors\n  gefinitib and erlotinib.\n	1	\N	\N	81c99b29-e3b5-490a-a00b-a5801800d017	2015-02-26 02:05:10.689913
124	75	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	fffab323-4099-4eb2-82c2-e9121cb0250c	2015-02-26 02:05:10.713783
125	76	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12/G13\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation\n  may result in a less responsive tumor when treated with first-generation TKI's like\n  gefitinib.\n	1	\N	\N	8ab34ce4-7062-4168-8c71-84103501607b	2015-02-26 02:05:10.736808
126	77	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12C\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	5024f04c-d164-4cf2-a979-de2890c04080	2015-02-26 02:05:10.761731
127	78	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12D\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	44de26c2-40c1-4576-b320-f0464a2f4543	2015-02-26 02:05:10.787913
128	79	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G13\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	04864b88-ddf9-4ee5-886f-d64e161f93e7	2015-02-26 02:05:10.812103
129	80	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G13D\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	89e822cd-429a-4d50-962b-277f717f189b	2015-02-26 02:05:10.836771
130	81	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: P124S\ndescription: MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder\n  and colon cancer to a lesser degree. The P124S mutation has been shown to contribute\n  to AZD6244 resistance in melanoma cell lines, but considerably less so than its\n  Q56P counterpart.\n	1	\N	\N	bd7879ae-e612-4a0b-bae3-8f8292090022	2015-02-26 02:05:10.868099
133	84	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: PROMOTER METHYLATION\ndescription: MGMT promoter methylation has been observed to impact tumor progression\n  in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating\n  agent carmustine has shown efficacy. In patients lacking methylation, combining\n  carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more\n  experiments are required. Clinical trials have also shown selective sensitivity\n  of promoter methylation-positive patients to temozolomide, making a case for wider\n  methylation screening in GBM patients.\n	1	\N	\N	dc7f5be8-419a-4e93-81f9-40c3efcbedc8	2015-02-26 02:05:10.94847
134	85	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 12 MUTATIONS\ndescription: NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\n	1	\N	\N	7b5bc5de-7ca5-4ba9-8968-fc2a5e85b7df	2015-02-26 02:05:10.973445
135	86	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: W288FS\ndescription: NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most\n  common NPM1 mutation identified in acute myeloid leukemia. This mutation results\n  in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response\n  to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation,\n  it is generally grouped with other exon 12 mutations for patient analysis (see NPM1\n  Exon 12 variants for more information).\n	1	\N	\N	fc3926f1-0732-48cd-b783-727cee64204d	2015-02-26 02:05:10.997762
136	87	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 1 MUTATIONS\ndescription: NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown\n  to be correlated with poorer overall survival relative to wild-type NRAS in melanoma\n  patients.\n	1	\N	\N	5d067bb5-b6e3-4a90-aa78-bb332d88328a	2015-02-26 02:05:11.025093
137	88	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 2 MUTATIONS\ndescription: NRAS exon 2 mutations have been shown to be correlated with poorer overall\n  survival in melanoma patients and colorectal cancer patients, however no prognostic\n  impact was seen in acute myeloid leukemia patients.\n	1	\N	\N	b0b07df3-07ab-46e2-9eae-05676fdc3743	2015-02-26 02:05:11.054522
138	89	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 3 MUTATIONS\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	5fd09a74-e0c1-432a-b8ae-b61908f5ba2a	2015-02-26 02:05:11.078581
139	90	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 4 MUTATIONS\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	067922f0-8ca0-487f-a83a-8d4ca404dbcb	2015-02-26 02:05:11.104753
140	91	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: G12\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	2fb1f56c-da6f-416f-b770-2406c4c90ae6	2015-02-26 02:05:11.129269
141	92	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: G13D\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	e210166a-c190-492c-bcc1-9a42eed15e11	2015-02-26 02:05:11.154341
142	93	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations\n  at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide.\n  However, in colorectal cancer patients, mutations at this locus have been shown\n  to confer resistance to cetuximab. The prognostic impact of mutations at this locus\n  is currently under study.\n	1	\N	\N	32b2d8ed-8f0c-4f03-b6b5-afd953c25140	2015-02-26 02:05:11.200624
143	94	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61L\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	de619b4a-3aa7-4fea-bc7c-68047647c2a0	2015-02-26 02:05:11.227641
144	95	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61R\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	56b8ba49-e008-48da-b9b9-e1d03ca40b62	2015-02-26 02:05:11.256071
145	96	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842I\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	238124eb-bd34-450b-8f32-26e13792fcbd	2015-02-26 02:05:11.286414
146	97	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842V\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	45c87562-6c01-4284-9d7e-f7dddb6f9d21	2015-02-26 02:05:11.313181
147	98	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842Y\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	a3161b9d-4d29-4fef-aa06-b7869175cea6	2015-02-26 02:05:11.338342
148	99	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: DEL I843\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	c5cfdc74-9005-440d-987f-b89191e57194	2015-02-26 02:05:11.365704
149	100	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: DI842-843IM\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	3daa1333-402b-4342-b050-e21219d7bb01	2015-02-26 02:05:11.394066
150	101	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: E542K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	8b8b7869-6a02-4a7c-ae7d-aef8a21f7679	2015-02-26 02:05:11.421632
151	102	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: E545K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	ba3c01e4-4e32-4434-9a9c-b277876637a0	2015-02-26 02:05:11.448735
152	103	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: EXON 20 MUTATIONS\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	5ec60e1b-56cd-47de-99d7-bf85725292d7	2015-02-26 02:05:11.478246
153	104	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: EXON 9 MUTATIONS\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	ac74c8e6-9031-4a9f-8a94-bd223a0defba	2015-02-26 02:05:11.504301
154	105	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: H1047R\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	a81714c9-be26-430b-8c72-abf31027c706	2015-02-26 02:05:11.531229
155	106	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 39\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and arsenic trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	b1fdff18-5cf8-43da-8be7-d854baba183c	2015-02-26 02:05:11.558293
156	107	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in a rare form of adolescent\n  liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found\n  Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined,\n  and functional studies found that the fusion retained kinase activity. The presence\n  of this fusion may be used as a diagnostic marker for this rare tumor type.\n	1	\N	\N	2e3179fb-47e7-4fa9-ad2f-426257d53901	2015-02-26 02:05:11.585226
157	108	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 41\nname: R233*\ndescription: PTEN R233* has been shown to be a loss of function mutation, and PTEN\n  loss has been the subject of considerable research in breast cancer. PTEN loss may\n  sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data\n  to support that PTEN loss is both associated with poorer prognosis, and no change\n  in prognosis.\n	1	\N	\N	8d6fbfad-35c9-421c-9c01-7a67c59df396	2015-02-26 02:05:11.611847
158	109	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and Arsenic Trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	c12bcd0c-1dff-4201-9869-532b2f5d0862	2015-02-26 02:05:11.639382
159	110	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: C634W\ndescription: RET C639W has been implicated as an alternate mechanism of activating\n  RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting\n  agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the C639W mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib.\n	1	\N	\N	2b6d4dd3-bfdb-40c9-9592-ccc175268c87	2015-02-26 02:05:11.666198
160	111	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: M918T\ndescription: RET M819T is the most common somatically acquired mutation in medullary\n  thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents,\n  promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the M918T mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T\n  leads to more aggressive MTC with a poorer prognosis.\n	1	\N	\N	cce516c9-ed18-461e-8959-165fd9cca48b	2015-02-26 02:05:11.692358
162	113	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: K700E\ndescription: SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients. Additionally,\n  these mutations are the most common SF3B1 mutation observed in MDS and highly associated\n  with subtypes of MDS that are defined by ringed sideroblasts.\n	1	\N	\N	2665f01e-55bc-4af6-ab89-c7963a207823	2015-02-26 02:05:11.745102
163	114	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R175H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R175H variant seems not only to result in loss of tumor-suppression, but also acts\n  as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell\n  lines harboring this mutant is also more responsive to treatment with doxorubicin\n  than its wild-type counterparts. While the prognostic impact of individual TP53\n  mutations is influenced by the cohort being studied, it has been shown that the\n  R175H mutation is correlated with worse overall survival than wild-type TP53, but\n  is not as detrimental as the R248W variant.\n	1	\N	\N	759b7d8f-2421-408a-b3bf-6553fb6fc2b1	2015-02-26 02:05:11.774085
164	115	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R248Q\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival. The R248Q mutation has also shown an increased invasive behavior in cell\n  lines. This is specific to the 248Q variant.\n	1	\N	\N	92c0d491-adcb-411e-b377-a09da95048e2	2015-02-26 02:05:11.8012
165	116	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R248W\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival.\n	1	\N	\N	2ea1817f-d2c0-41d5-b8bf-a25af6fd8059	2015-02-26 02:05:11.843299
166	117	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R249T\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	13fd35d2-95b2-4a30-a3e1-9b4f450a3335	2015-02-26 02:05:11.87138
167	118	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R249W\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	abf6a489-023d-4d66-8816-c46efe6e5bb2	2015-02-26 02:05:11.896767
168	119	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R273C\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	2a28953c-810d-4af1-a0f0-3817fb2ff26e	2015-02-26 02:05:11.921638
169	120	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R273H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	0f35644b-f5ee-48f8-a927-f946516a3650	2015-02-26 02:05:11.947393
170	121	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: V173G/A\ndescription: While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R249 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	95b19c23-6499-486e-9332-c8645595178e	2015-02-26 02:05:11.974106
171	122	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: FRAMESHIFT TRUNCATION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	919ba541-f222-432f-b3b4-0fff3ea728dd	2015-02-26 02:05:11.999365
172	123	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	3dabb7f8-1a01-47fa-a270-841e3c3bdb65	2015-02-26 02:05:12.023971
173	124	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	fce73e5b-cebd-455a-8452-5f71b1483fba	2015-02-26 02:05:12.04767
174	125	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 48\nname: Q157P/R\ndescription: U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is less common than the S34F mutation, occurs in the second zinc finger domain of\n  U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations\n  on overall survival in MDS has been debated, however, patients with U2AF1 mutations\n  were shown to be at an increased risk of transformation to secondary AML. The presence\n  of this mutation was not associated with a specific prognostic outcome in AML when\n  compared to U2AF1 wildtype patients.\n	1	\N	\N	1251d6be-edf8-4a6a-90c7-921b4ffe17b9	2015-02-26 02:05:12.070962
175	126	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 48\nname: S34Y/F\ndescription: U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is the most commonly identified variant in MDS, occurs in the first zinc finger\n  domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1\n  mutations on overall survival in MDS has been debated, however, patients with U2AF1\n  mutations were shown to be at an increased risk of transformation to secondary AML.\n  The presence of this mutation was not associated with a specific prognostic outcome\n  in AML when compared to U2AF1 wildtype patients.\n	1	\N	\N	642c4b7f-4458-4918-9bb1-0b86a56abcf6	2015-02-26 02:05:12.097137
176	127	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 49\nname: EXON 7 MUTATIONS\ndescription: WT1 exon 7 mutations have been shown to be recurrent in acute myeloid\n  leukemia. Many sources have examined the prognostic impact of these, agreeing that\n  the mutant exon 7 cohort is correlated with worse overall survival and a number\n  of poor prognistic outcomes. This may be the result of an overall poor response\n  to chemotherapy from WT1 mutant tumors.\n	1	\N	\N	6116bd49-9590-43b7-8581-9dc3e08092ed	2015-02-26 02:05:12.123456
177	128	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 49\nname: EXON 9 MUTATIONS\ndescription: WT1 exon 9 mutations have been shown to be recurrent in acute myeloid\n  leukemia, although at a less frequent rate than their exon 7 counterparts. Many\n  sources have examined the prognostic impact of these, agreeing that the mutant exon\n  9 cohort is correlated with worse overall survival and a number of poor prognistic\n  outcomes. This may be the result of an overall poor response to chemotherapy from\n  WT1 mutant tumors.\n	1	\N	\N	955fc6ec-f208-4a76-acda-29005263bff3	2015-02-26 02:05:12.151557
178	129	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: LOSS-OF-FUNCTION\ndescription: BRCA1 loss of function mutations have been shown to increase risk of\n  breast and ovarian cancer in those carrying the allele in their germline. Treating\n  BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant\n  response.\n	1	\N	\N	094cc5a5-dc10-4715-af1d-46bb48feb9a5	2015-02-26 02:05:12.17738
179	130	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: LOSS-OF-FUNCTION\ndescription: BRCA2 loss of function mutations have been shown to increase risk of\n  breast and ovarian cancer in those carrying the allele in their germline. Treating\n  BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant\n  response.\n	1	\N	\N	1d7a7b86-c0c7-4635-b4a7-adeed10a8b13	2015-02-26 02:05:12.203893
180	1	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide\n  results in high rates of complete remission in patients.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 1\nsource_id: 51\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6d35ca66-2d0c-41a5-9be3-057a87faf87a	2015-02-26 02:05:12.358158
181	2	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL fusions have been found to contribute to imatinib resistance in AML\n  cell lines.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 1\nsource_id: 51\nvariant_id: 1\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a7d988c8-ea1c-4a21-812e-e303660298f3	2015-02-26 02:05:12.432932
182	3	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The clinical use of imatinib in patients with BCR-ABL fusion has resulted in\n  drastic sensitivity to the drug.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 2\nsource_id: 52\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ff60e22b-c1bd-4bc8-84fd-fb4610c03a27	2015-02-26 02:05:12.524189
183	4	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL inhibitors such as imatinib have lead to significantly improved prognosis,\n  response rate, overall survival, and patient outcome in CML patients compared to\n  previous therapeutic regimens.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 2\nsource_id: 52\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	67aa481f-bf7a-4aa1-98ce-acd9d393f6ce	2015-02-26 02:05:12.587357
184	5	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient\n  has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may\n  imply the possibility that BCR-ABL is not universally the initiating event in CML.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 2\nsource_id: 53\nvariant_id: 1\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a3becf8b-b1e7-4425-bdfb-2671a3e4889d	2015-02-26 02:05:12.65028
185	6	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of BCR-ABL fusion is considered the characterizing feature of chronic\n  myeloid leukemia, and has been widely thought of as the initiating event in the\n  disease.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 2\nsource_id: 52\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e6344bd4-dc98-4732-b9f1-da24102e8e3c	2015-02-26 02:05:12.706918
186	7	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has been effective in\n  patients resistant to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 4\ndisease_id: 2\nsource_id: 52\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1be57ff8-d96a-43a9-b825-4ccf6f9196cf	2015-02-26 02:05:12.768309
187	8	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has been effective in\n  patients resistant to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 5\ndisease_id: 2\nsource_id: 52\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b49a851a-f23f-4f36-a09d-77cbac511c07	2015-02-26 02:05:12.829699
188	9	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 54\nvariant_id: 3\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	66363d7d-0e57-4ac1-b6b8-2b612f06e0d7	2015-02-26 02:05:12.886414
189	10	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have\n  shown to be resistant to imatinib treatment.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 2\nsource_id: 52\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c0828bba-fe07-4a96-bb7a-765fb88a2b8d	2015-02-26 02:05:13.007586
190	11	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 54\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f67d5239-bbb5-44ef-8a42-b374b5d1ac1e	2015-02-26 02:05:13.070553
191	12	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the\n  AKT inhibitor MK-2206 in breast cancer cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 6\ndisease_id: 3\nsource_id: 55\nvariant_id: 4\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 14:105246551-105246551 (C->T)\n	1	\N	\N	3460446a-ff55-429a-8b57-07860d413e1c	2015-02-26 02:05:13.137601
192	13	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment\n  with crizotinib has shown to be effective in acheiving stable disease.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 4\nsource_id: 56\nvariant_id: 5\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6f52656e-d836-43ec-9c6a-2dd667aaa405	2015-02-26 02:05:13.201325
193	14	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 4\nsource_id: 56\nvariant_id: 6\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	21d56ee6-425a-4861-90de-86c60b43e6cc	2015-02-26 02:05:13.268187
194	15	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 treatment resulted in significant tumor regression in xenograft models\n  produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 8\ndisease_id: 4\nsource_id: 57\nvariant_id: 7\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	431ef99f-c6d7-4c90-a783-19e44631ba29	2015-02-26 02:05:13.348115
195	16	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 4\nsource_id: 56\nvariant_id: 7\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8cfbba1a-d27e-4582-aaa2-a66b18c440df	2015-02-26 02:05:13.406495
196	17	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without\n  this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 5\nsource_id: 58\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29443695-29443695 (G->T)\n	1	\N	\N	438f202d-322a-49fb-97db-b7fecf92d074	2015-02-26 02:05:13.475609
197	18	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in\n  a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 8\ndisease_id: 6\nsource_id: 57\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29443695-29443695 (G->T)\n	1	\N	\N	5d175b5f-b89f-4b5f-af52-0e59b1a57cfd	2015-02-26 02:05:13.556944
198	19	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 6\nsource_id: 59\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29443695-29443695 (G->T)\n	1	\N	\N	5aae584e-df24-46fb-b061-545473d1c146	2015-02-26 02:05:13.612612
199	20	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing\n  the EML4-ALK fusion containing the F1174L mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 6\nsource_id: 60\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29443695-29443695 (G->T)\n	1	\N	\N	dfdd186b-9637-4063-b00a-bc129963bba4	2015-02-26 02:05:13.681565
200	21	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing\n  ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 9\ndisease_id: 6\nsource_id: 61\nvariant_id: 8\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29443695-29443695 (G->T)\n	1	\N	\N	69c5deb9-56c2-4a0a-93bd-1aa1ebf03d38	2015-02-26 02:05:13.751186
201	22	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without\n  this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 4\nsource_id: 58\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29443695-29443695 (G->T)\n	1	\N	\N	c0dfe27a-6632-4f52-b540-ee75e4b0e0b9	2015-02-26 02:05:13.812626
202	23	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells\n  with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 9\ndisease_id: 6\nsource_id: 61\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29432664-29432664 (C->T)\n	1	\N	\N	0bc8e832-d93b-4714-b507-17a8e95c5651	2015-02-26 02:05:13.879899
203	24	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 9\ndisease_id: 6\nsource_id: 61\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29432664-29432664 (C->T)\n	1	\N	\N	5b509452-e621-49a8-8ca3-cfe5d45d7ff9	2015-02-26 02:05:13.940111
204	25	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony\n  formation of AALE cells expressing ARAF S214C in vitro\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 4\nsource_id: 62\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: X:47426121-47426121 (C->G)\n	1	\N	\N	4229bb92-0094-4369-ba97-28224aa574d4	2015-02-26 02:05:14.011765
205	26	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In one patient with S214C mutation, the use of sorafenib has led to more than\n  5 years of survival and near remission.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 4\nsource_id: 62\nvariant_id: 10\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: X:47426121-47426121 (C->G)\n	1	\N	\N	c97fba38-a3de-4061-a2fc-7ce04707b90c	2015-02-26 02:05:14.090625
206	27	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony\n  formation of AALE cells expressing ARAF S214C in vitro\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 11\ndisease_id: 4\nsource_id: 62\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: X:47426121-47426121 (C->G)\n	1	\N	\N	04e22b8f-e2a6-48fa-851e-1ced9a16decf	2015-02-26 02:05:14.157699
207	28	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations\n  were associated with poor progression free survival regardless of treatment (panitumumab\n  with best supportive care or best supportive care alone)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 7\nsource_id: 63\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	597ee757-cdae-4f93-863c-956f0e631bea	2015-02-26 02:05:14.220184
208	29	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 13\ndisease_id: 8\nsource_id: 64\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d211e4b4-6a50-42c9-9384-6ce2d1157071	2015-02-26 02:05:14.288609
209	30	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 11\ndisease_id: 8\nsource_id: 64\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b3ee839e-7176-4551-af4d-fd2c4eb13942	2015-02-26 02:05:14.342563
210	31	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutations are associated with melanoma arising in non-chronic sun damaged\n  skin and with superficial spreading melanoma\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 8\nsource_id: 65\nvariant_id: 17\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	07ba19a6-ab2c-4dab-b28b-6394e1b105bc	2015-02-26 02:05:14.406693
211	32	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 13\ndisease_id: 8\nsource_id: 66\nvariant_id: 11\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453135-140453136 (CA->AT)\n	1	\N	\N	2e069044-eeee-4a72-8e19-7e1d8fe34fa6	2015-02-26 02:05:14.465925
212	33	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 9\nsource_id: 67\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	f018993a-ac51-45b2-8af1-54ce2e5dca3a	2015-02-26 02:05:14.559522
213	34	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 12\ndisease_id: 9\nsource_id: 67\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	ff97b898-0267-4e0a-8fbc-f5aeb3d2a2fb	2015-02-26 02:05:14.619311
214	35	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 9\nsource_id: 67\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	d5ceb948-a31f-4fb2-8a3b-310305963a3c	2015-02-26 02:05:14.670699
215	36	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 15\ndisease_id: 9\nsource_id: 68\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	2171b642-c6ae-4756-82d9-eb92ab689305	2015-02-26 02:05:14.735535
216	37	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 9\nsource_id: 68\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	1fa0aba5-a3cd-419c-8af1-949ff3c0d0be	2015-02-26 02:05:14.794417
217	38	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 9\nsource_id: 69\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	9ec328c3-6f14-492e-af50-746de0989483	2015-02-26 02:05:14.862259
218	39	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 17\ndisease_id: 9\nsource_id: 69\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	4748ebab-7aa5-4df9-9b50-8e3de07cf237	2015-02-26 02:05:14.915814
219	40	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 18\ndisease_id: 7\nsource_id: 70\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	8be37b1b-f9fa-412d-a516-384464b4c228	2015-02-26 02:05:14.991394
220	41	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 19\ndisease_id: 7\nsource_id: 70\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	882880ae-379d-48cc-accd-d7d73d9d7edc	2015-02-26 02:05:15.065939
221	42	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 20\ndisease_id: 7\nsource_id: 70\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	144ee1fe-d2ab-449c-8a7c-f233fa466f57	2015-02-26 02:05:15.126184
222	43	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations\n  had lower response and disease control rates as well as shorter progression free\n  and overall survival following cetuximab plus chemotherapy than those with wildtype\n  BRAF\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 7\nsource_id: 71\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	5674dc71-50f5-425d-a240-151e9ac2d652	2015-02-26 02:05:15.187903
223	44	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is associated with adverse pathological features of colorectal cancer.\n  This can be concluded as a marker of poor prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 72\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	076de5a2-0d30-45d9-8177-fe2b2ef68f89	2015-02-26 02:05:15.246307
224	45	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines expressing the BRAF V600E mutation responded better to vemurafenib\n  treatment than cells wildtype for BRAF as measured by reduced cellular proliferation\n  and inhibition of MET and ERK phosphorylation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 19\ndisease_id: 7\nsource_id: 70\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	5f528bcf-db2f-4a97-9d41-9f5381eeeebc	2015-02-26 02:05:15.302917
225	46	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 21\ndisease_id: 8\nsource_id: 73\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	97561b11-7a90-42e8-a486-e3e0cd36ad1a	2015-02-26 02:05:15.377535
226	47	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 22\ndisease_id: 8\nsource_id: 73\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	aa909fe7-155d-46d3-a11c-44ebb91e2d28	2015-02-26 02:05:15.427264
227	48	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 13\ndisease_id: 8\nsource_id: 74\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	2c6b9b09-a1c5-490f-9319-d787e3ec4982	2015-02-26 02:05:15.487434
228	49	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 11\ndisease_id: 8\nsource_id: 74\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	03e201be-5acc-439d-a3ae-d494149fb132	2015-02-26 02:05:15.562651
229	50	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  but does not show resistance to MEK inhibitors.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 8\nsource_id: 75\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	329b545b-439d-495e-988f-1ce07258716f	2015-02-26 02:05:15.626754
230	51	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  and resistance to RAF inhibitors.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 8\nsource_id: 75\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	a31a2837-c188-4e6d-bbfc-da0bb8c764ec	2015-02-26 02:05:15.682546
231	52	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with shorter disease-free and overall Survival in a\n  Spanish cohort of melanoma patients.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 8\nsource_id: 76\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	892d2a6f-c166-46db-a88a-7418dbe75640	2015-02-26 02:05:15.746319
232	53	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 13\ndisease_id: 4\nsource_id: 77\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	84d30fed-3a73-41f0-a97b-141ac1d7d318	2015-02-26 02:05:15.811117
233	54	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is shown to be associated with the tall-cell variant of papillary\n  thyroid cancer\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 10\nsource_id: 78\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	a000d85c-c316-4988-b6b0-3d6519d95aa6	2015-02-26 02:05:15.87262
234	55	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 10\nsource_id: 79\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	6dce6b97-657f-4005-9487-77a242743bc4	2015-02-26 02:05:15.94079
235	56	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 10\nsource_id: 80\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	5d134957-2d86-4c9a-a9c5-d612c77fff24	2015-02-26 02:05:16.006441
236	57	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Unlike other studies that suggest a poorer outcome, BRAF mutation in this study\n  was not correlated with poorer prognosis in papillary thyroid cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 10\nsource_id: 81\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	26e7a18e-4e8e-4903-95f0-10d7c383c296	2015-02-26 02:05:16.087379
237	58	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in\n  a study of 187 patients with PTC and other thyroid diseases.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 10\nsource_id: 82\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	b90faa61-e579-4ecc-a1d9-3f0becd46dec	2015-02-26 02:05:16.150961
238	59	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutation correlated with poor prognosis in papillary thyroid cancer in\n  both older (>65 yo) and younger (<65 yo) cohorts.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 10\nsource_id: 78\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	1d6bef5b-a591-4f63-875b-a85bb4872624	2015-02-26 02:05:16.207574
239	60	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is correlated with disease recurrence in both age cohorts (>65 and <65\n  yo).\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 10\nsource_id: 78\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	aa115ffa-7e44-4349-b3c2-ae452bfe3109	2015-02-26 02:05:16.269713
240	61	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COLO201 and COLO206F cells harboring BRAF V600E mutations were cloned to be\n  MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to\n  be amplification of the BRAF V600E gene.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 23\ndisease_id: 7\nsource_id: 83\nvariant_id: 14\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ce99dd96-224e-44cb-bb98-b4319daf15a3	2015-02-26 02:05:16.337325
241	62	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A single patient with BRAF V600E/V600M bi-allelic mutation responded to the\n  V600E drug dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 13\ndisease_id: 8\nsource_id: 84\nvariant_id: 13\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ede22f63-74be-4b7a-a3ae-88553f1f83ff	2015-02-26 02:05:16.398556
242	63	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 13\ndisease_id: 8\nsource_id: 66\nvariant_id: 15\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453137-140453137 (C->T)\n	1	\N	\N	a7e8aec1-c4b8-4cac-8240-1c1694512799	2015-02-26 02:05:16.459178
243	64	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 13\ndisease_id: 8\nsource_id: 66\nvariant_id: 16\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453137 (AC->CT)\n	1	\N	\N	a17de1cc-6602-4c45-aa1d-7e74ebb86938	2015-02-26 02:05:16.54718
244	65	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The combination of PARP inhibition (Olaparib) with BRCA mutation results in\n  irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation positive\n  women with high grade serous or undifferentiated ovarian cancer.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 24\ndisease_id: 11\nsource_id: 85\nvariant_id: 129\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 2\nvariant_hgvs: N/A\n	1	\N	\N	dcd68897-0d6f-40ea-88e0-753a7fc68068	2015-02-26 02:05:16.615842
245	66	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The combination of PARP inhibition (Olaparib) with BRCA mutation results in\n  irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation positive\n  women with high grade serous or undifferentiated ovarian cancer.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 24\ndisease_id: 11\nsource_id: 85\nvariant_id: 130\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 2\nvariant_hgvs: N/A\n	1	\N	\N	5ceb4b86-e611-4c73-9905-b9857f5ca071	2015-02-26 02:05:16.689994
246	67	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Inceased copy number of CCND1 is associated with poorer overall survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 86\nvariant_id: 18\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	923fafdc-9681-4c4a-8c1f-a45273da65e2	2015-02-26 02:05:16.766818
247	68	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have found no significant survival impact for CCND1 in lung cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 86\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d010cae5-d06f-4bcc-ab28-f59b07ca15b8	2015-02-26 02:05:16.826333
248	69	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have associated CCND1 expression with poorer survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 86\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	dff392a4-036d-433f-9786-6de0ef17e9ed	2015-02-26 02:05:16.88835
249	70	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with the ER-positive subtype of breast\n  cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 12\nsource_id: 87\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	355cf38c-afcd-4787-8d40-8d6423d4ddab	2015-02-26 02:05:16.956852
250	71	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In ER-positive breast cancer patients, Cyclin D1 overexpression is associated\n  with shorter overall survival and increased metastasis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 12\nsource_id: 87\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	879a2702-9176-453d-82cf-8b3879f5e143	2015-02-26 02:05:17.032668
251	72	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in head and neck squamous cell carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 13\nsource_id: 88\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	07157b0f-9cf5-4f54-9b96-c68af990801a	2015-02-26 02:05:17.099211
252	73	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in mantle cell lymphoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 14\nsource_id: 89\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8cf441df-4428-4bc6-99ed-1e2718cdca32	2015-02-26 02:05:17.157721
253	74	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the presence pf lymph node metastasis\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 15\nsource_id: 90\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9af56327-a372-41ed-b232-bde344a5c797	2015-02-26 02:05:17.222551
254	75	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the increased depth of cancer invasion\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 15\nsource_id: 90\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6c5a01e6-696a-4353-a825-b9cb3f592dc4	2015-02-26 02:05:17.283573
255	76	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 15\nsource_id: 90\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	21e3ae7d-58bc-4b5c-bb76-58931c090950	2015-02-26 02:05:17.342937
256	77	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with vascular invasion by cancer cells\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 15\nsource_id: 90\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9d227eee-763a-416d-bc15-afbf09e58b31	2015-02-26 02:05:17.406437
257	78	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CCND2 hypomethylation is seen to be more common in stage III and IV gastric\n  tumors than stage I and II.\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 15\nsource_id: 91\nvariant_id: 22\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6a866750-dcdc-40dd-9d7c-703c0b85f443	2015-02-26 02:05:17.487641
258	79	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib\n  (PD-0332991) significantly increased median survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 25\ndisease_id: 16\nsource_id: 92\nvariant_id: 23\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	676b22bf-bb5c-4cac-9593-49d679f991f4	2015-02-26 02:05:17.557143
259	80	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with high expression of total cyclin E and low-molecular weight cyclin\n  E is strongly associated with poor prognosis in breast cancer, and the hazard ratio\n  for these patients in 13.3 times higher than those with normal cyclin E levels.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 12\nsource_id: 93\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	81de2193-2918-4ad9-b107-768240241f20	2015-02-26 02:05:17.624433
260	81	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 17\nsource_id: 94\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e97a4ab6-8079-4079-8c9b-077c71060fd8	2015-02-26 02:05:17.682316
261	82	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced\n  stage gastric carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 17\nsource_id: 94\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	501fed61-8a0a-48fd-ad0c-c0a7e9fe7e51	2015-02-26 02:05:17.73883
262	83	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 18\nsource_id: 94\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	90179d42-5415-4e2c-85c6-2aa2675fd449	2015-02-26 02:05:17.810392
263	84	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 26\ndisease_id: 19\nsource_id: 95\nvariant_id: 25\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0b8a3bc4-6b31-4fd5-a8b4-81ee73e69e9f	2015-02-26 02:05:17.894356
264	85	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 27\ndisease_id: 19\nsource_id: 96\nvariant_id: 25\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	43f11e8d-e44e-41fc-bcf6-27bd6c0ae6c2	2015-02-26 02:05:17.958099
265	86	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 26\ndisease_id: 19\nsource_id: 95\nvariant_id: 26\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	21411c0c-f1e0-494d-ba67-102761fc8751	2015-02-26 02:05:18.018168
266	87	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 27\ndisease_id: 19\nsource_id: 96\nvariant_id: 26\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	91b52590-3287-44b6-9442-77a25ae0b51c	2015-02-26 02:05:18.078932
267	88	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter time to recurrence.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 97\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2f55a73d-332f-424b-8f43-3fdeb00a7d00	2015-02-26 02:05:18.142338
268	89	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter overall survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 97\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fba10fed-c4a7-4b72-834d-4bfd9104ecd5	2015-02-26 02:05:18.206419
269	90	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation status had no impact on ATRA treatment response in older (>60)\n  patients with AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 98\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7e3ad9b6-9149-4f49-b133-9edd96d653cb	2015-02-26 02:05:18.270622
270	91	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was significantly associated with complete remission\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 99\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9dcd3efe-25c2-41b9-b1fc-ff1dae356fe6	2015-02-26 02:05:18.352493
271	92	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was associated with improved overall survival in older (>60)\n  patients with AML\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 98\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d088e4f2-5ece-4457-811b-79014fd279fa	2015-02-26 02:05:18.408325
272	93	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 100\nvariant_id: 28\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bacddb03-9e37-4bda-97b8-8303f81c943e	2015-02-26 02:05:18.474518
273	94	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Overall survival was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA mutations\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 100\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f32e3500-90b9-4d56-8870-48dbeb180ed3	2015-02-26 02:05:18.551436
274	95	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Remission duration was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 100\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	db92d00b-cccb-42a2-b10f-e35654654e67	2015-02-26 02:05:18.619062
275	96	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 20\nsource_id: 101\nvariant_id: 30\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	de9765ee-5761-4e4c-afca-f8dc6461eaaf	2015-02-26 02:05:18.686115
276	97	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 20\nsource_id: 101\nvariant_id: 30\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e40bb844-1caa-4abb-b925-dbbadfacef3b	2015-02-26 02:05:18.734791
277	98	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 21\nsource_id: 102\nvariant_id: 31\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	056b2917-0bd1-4f2a-a77a-4604fc695244	2015-02-26 02:05:18.830489
278	99	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Daunorubicin treatment resulted in similar overall survival and disease free\n  survival in de novo AML patients with DNMT3A R882 mutation compared to those who\n  do not harbor this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 29\ndisease_id: 1\nsource_id: 103\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9c41d946-99f2-47be-b8e3-a17169c590e2	2015-02-26 02:05:18.896044
279	100	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Idarubicin increases the overall survival and disease free survival in de novo\n  AML patients with DNMT3A R882 mutation compared to those who do not harbor this\n  mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 30\ndisease_id: 1\nsource_id: 103\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a3ffac7b-3098-4ade-914f-32748c260744	2015-02-26 02:05:18.95607
280	101	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Therapy-related AML was less common in patients with DNMT3A mutations (64.5%\n  of which were R882) than patients wildtype for DNMT3A in a large cohort of younger\n  (18-60) AML patients\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7411981b-fb17-46eb-85e9-7a2b9db83421	2015-02-26 02:05:19.015759
281	102	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutations (60% of which were\n  R882) were older in age, had higher white blood cell counts and had higher platelet\n  counts than patients wildtype for DNMT3A\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 105\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	31307258-c9e7-48db-9ad6-f4fcacf0e346	2015-02-26 02:05:19.0814
282	103	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations occur most often in de novo AML patients with intermediate\n  cytogenic risk.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 103\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	66ba09a1-0e16-42d3-b2e4-b4fc5c326ab6	2015-02-26 02:05:19.140738
283	104	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with older age, higher white blood cell\n  count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically\n  normal AML patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 106\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9fc9f7b4-80c9-4904-91b2-5f67d587e255	2015-02-26 02:05:19.20673
284	105	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with an intermediate\n  risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 107\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	de5554ed-43b4-4fb0-af95-212fd5ee85cb	2015-02-26 02:05:19.290447
285	106	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with intermediate\n  risk cytogenetics (including normal karyotype)\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 107\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c70eac02-5a2d-40ed-a3b5-0e27b2ee93ff	2015-02-26 02:05:19.349804
286	107	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (64.5% R882) were associated with older age, higher white blood\n  cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML\n  patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0103e1f3-5a83-483f-8e1b-66cb909f98d0	2015-02-26 02:05:19.41076
287	108	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with cytogenetically normal AML in a large\n  cohort of younger (18-60) AML patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	19e786d6-fd00-457a-a13d-129df0c52ada	2015-02-26 02:05:19.472999
288	109	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of cytogenetically normal AML patients (18-60 years old),\n  DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall,\n  relapse free and event free survival\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3bf994d4-15df-4e16-a3b2-a26801e9e382	2015-02-26 02:05:19.533933
289	110	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no difference in the complete remission rate of de novo AML patients\n  with DNMT3A mutation compared to those who are wild type for DNMT3A.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 103\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	97be5d95-fbc4-423a-b8cb-5d062e908640	2015-02-26 02:05:19.592677
290	111	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status (60% of which\n  were R882) was not predictive of overall and relapse free survival in patients with\n  NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 105\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bcc6c9e0-64f3-4eef-828c-f05b0bb9ca41	2015-02-26 02:05:19.731674
291	112	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5%\n  of which were R882) had no prognostic value on overall, relapse free and event free\n  survival.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d50a6c64-68c5-4a34-bc27-b67d68bf5468	2015-02-26 02:05:19.796836
292	113	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates did not differ between patients with wildtype or mutant\n  DNMT3A (62% of which affected R882) and cytogenetically normal AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 108\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d496cb41-7563-4139-b738-7a448cdd2a48	2015-02-26 02:05:19.860392
293	114	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations were associated with achievement of complete remission in a\n  large cohort of younger (18-60) AML patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ff26d25e-cd32-45b2-bf6d-1b0ff2cd1d59	2015-02-26 02:05:19.928435
294	115	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with DNMT3A mutations (59% of which were R882) showed worse survival\n  (event-free and overall) outcome than those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 107\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	71cd54ae-b144-45eb-96cc-1863ed32cf04	2015-02-26 02:05:19.993172
295	116	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including\n  R882) were associated with worse overall survival compared to those without DNMT3A\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 107\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8ae47b7e-f425-4861-8b44-bf5eb3770577	2015-02-26 02:05:20.057336
296	117	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free\n  and overall) outcome than those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 107\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	beb8a90a-d5d9-409b-82c0-211aeb7c738e	2015-02-26 02:05:20.120994
297	118	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cytogenetically normal AML patients, DNMT3A R882 mutations are associated\n  with lower overall and disease free survival as compared to patients with wild type\n  DNMT3A.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 106\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1ffa5235-e37d-4eaf-b543-de955a989b78	2015-02-26 02:05:20.191152
298	119	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutations were associated with\n  significantly reduced overall survival and relapse free survival in patients wildtype\n  for NPM1 and FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 105\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	394008b8-0082-41ee-8b76-4cb83cb60513	2015-02-26 02:05:20.280006
299	120	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation\n  is prognostic for shorter disease free survival and overall survival compared to\n  patients without the mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 108\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	96e1ef9f-ae59-46db-a9d1-d24c567c73ce	2015-02-26 02:05:20.346008
300	121	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations\n  other than R882 are prognostic for shorter disease free survival and overall survival\n  compared to patients without the mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 108\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	670c83d9-c759-48fc-b6c4-07a01307234d	2015-02-26 02:05:20.40407
301	122	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutation was associated with reduced relapse free and overall survival\n  in ELN-unfavorable, cytogenetically normal AMLs\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2d51447f-914d-4735-9c6c-3a42b0f041db	2015-02-26 02:05:20.458463
302	123	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations\n  were associated with reduced relapse free survival in the entire cohort as well\n  as the subset of patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f4d4a06b-beeb-47f2-bcaa-6da655494c12	2015-02-26 02:05:20.51581
303	124	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (62% of which were R882) were associated with reduced disease-free\n  survival in patients with cytogenetically normal AML.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 108\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	462a8683-84f2-407b-9b59-14aa3ca0113d	2015-02-26 02:05:20.575567
304	125	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutation have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 105\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e6360750-9035-4b44-932c-059f136f24e8	2015-02-26 02:05:20.63079
305	126	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rate was not different between young AML patients (<60 years\n  old) with or without DNMT3A mutations (60% of which were R882)\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 105\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	745c2266-3042-44fb-a0a7-8cc418608215	2015-02-26 02:05:20.685325
306	127	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 105\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6245cad2-e1f4-414d-ac7e-c4c355d6a841	2015-02-26 02:05:20.771193
307	128	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Geftinib added to pemetrexed and carboplatin showed greater effectiveness than\n  premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 31\ndisease_id: 4\nsource_id: 109\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55259514-55259515 (CT->AG)\n	1	\N	\N	78523366-c58f-44a7-9bcc-787c17dd44b6	2015-02-26 02:05:20.844104
308	129	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 31\ndisease_id: 4\nsource_id: 110\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55259514-55259515 (CT->AG)\n	1	\N	\N	7e48111c-283a-49ac-95d7-bddd148a1f3a	2015-02-26 02:05:20.907534
309	130	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 32\ndisease_id: 4\nsource_id: 110\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55259514-55259515 (CT->AG)\n	1	\N	\N	4fd5f137-0ba7-4f95-9990-9383ffbedfff	2015-02-26 02:05:20.966006
310	131	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence\n  of L858R mutation is prognostic for better progression free survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 111\nvariant_id: 33\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55259514-55259515 (CT->AG)\n	1	\N	\N	6ecba970-79f7-4153-abe2-b05fbb176e8b	2015-02-26 02:05:21.037253
311	132	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Median survival of patients with EGFR L858R mutation is better than those with\n  wild type EGFR.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 112\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55259514-55259515 (CT->AG)\n	1	\N	\N	0488d75e-ee1c-436a-9271-b96351b07743	2015-02-26 02:05:21.097403
312	133	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The T790M mutation in EGFR has been shown to confer resistance to the tyrosine\n  kinase inhibitor erlotinib, and patients harboring this mutation that are placed\n  on the drug are likely to relapse.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 32\ndisease_id: 22\nsource_id: 113\nvariant_id: 34\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55249071-55249071 (C->T)\n	1	\N	\N	b55ebfd3-356e-48d4-93e3-31927b3ac9c1	2015-02-26 02:05:21.158745
313	134	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting\n  in stable disease.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 32\ndisease_id: 4\nsource_id: 114\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55249071-55249071 (C->T)\n	1	\N	\N	9afeb6bf-1013-46b4-b852-4c0c4799cd21	2015-02-26 02:05:21.233967
314	135	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting\n  in stable disease.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 33\ndisease_id: 4\nsource_id: 114\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55249071-55249071 (C->T)\n	1	\N	\N	8738c3d4-c188-4bb6-93fe-2290274c2605	2015-02-26 02:05:21.302674
315	136	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In non-small cell lung cancer, the appearance of T790M mutation leads to resistance\n  to gefitinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 31\ndisease_id: 4\nsource_id: 115\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55249071-55249071 (C->T)\n	1	\N	\N	677a2e75-a438-42b9-adc5-387347e86660	2015-02-26 02:05:21.3675
316	137	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall\n  survival compared to patients with L858R or other exon 19 activating mutations.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 116\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55249071-55249071 (C->T)\n	1	\N	\N	109f4774-a8a1-4deb-94c4-d111a14d1aa0	2015-02-26 02:05:21.424101
317	138	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients with T790M and another activating mutations, their progression\n  free survival is shorter compared to those who do not possess T790M mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 117\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55249071-55249071 (C->T)\n	1	\N	\N	b50a974a-7d7e-413e-9496-aaf7601b5325	2015-02-26 02:05:21.484363
318	139	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR,\n  especially when an L858R mutation is also present.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 34\ndisease_id: 4\nsource_id: 118\nvariant_id: 34\nrating: 1\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55249071-55249071 (C->T)\n	1	\N	\N	01f0e825-e959-421f-b683-e94e0b5483ea	2015-02-26 02:05:21.552067
319	140	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 12\nsource_id: 119\nvariant_id: 35\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37880261-37880261 (G->C)\n	1	\N	\N	ecd0245c-d7f1-4d64-b967-a36ad0f7a4e7	2015-02-26 02:05:21.618504
320	141	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 12\nsource_id: 119\nvariant_id: 36\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37880261-37880261 (G->T)\n	1	\N	\N	2972ff64-7a97-4c97-a306-414a622b4861	2015-02-26 02:05:21.685931
321	142	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2,\n  cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive\n  to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 12\nsource_id: 119\nvariant_id: 37\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	01ad7a58-6ae4-4458-bb9e-465fe5498905	2015-02-26 02:05:21.782819
322	143	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 12\nsource_id: 119\nvariant_id: 38\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37868205-37868205 (G->C)\n	1	\N	\N	9df70f8a-ec45-49ce-9ac9-5fadfb35749a	2015-02-26 02:05:21.840654
323	144	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell\n  lines retrovirally transduced with wildtype or mutant ERBB2.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 36\ndisease_id: 12\nsource_id: 119\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37880220-37880220 (T->C)\n	1	\N	\N	5340e89c-8511-42c5-b122-3f701a04c553	2015-02-26 02:05:21.903006
324	145	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally\n  transduced with wildtype or mutant ERBB2.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 12\nsource_id: 119\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37880220-37880220 (T->C)\n	1	\N	\N	9db9480d-5f27-4c5c-bde9-ff55c80f0082	2015-02-26 02:05:21.962334
325	146	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive\n  to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 12\nsource_id: 119\nvariant_id: 40\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d0ff7aa0-9bbf-48ac-8b71-993b3a572e65	2015-02-26 02:05:22.032849
326	147	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the in-frame insertion of a single amino acid at position\n  780 was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 12\nsource_id: 119\nvariant_id: 41\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a065e905-1bd7-4ed2-9462-d70357b08c62	2015-02-26 02:05:22.096389
327	148	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 12\nsource_id: 119\nvariant_id: 42\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	42f03b74-c762-4897-98f2-abe4de1e7fb1	2015-02-26 02:05:22.165512
328	149	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 36\ndisease_id: 12\nsource_id: 119\nvariant_id: 42\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	439a4489-f690-4264-b3ec-af19c6ae8921	2015-02-26 02:05:22.220676
329	150	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 12\nsource_id: 119\nvariant_id: 43\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37881616-37881616 (C->T)\n	1	\N	\N	ec3f81f5-caa1-4dc2-8119-8241e58595dc	2015-02-26 02:05:22.292957
330	151	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 12\nsource_id: 119\nvariant_id: 44\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37881000-37881000 (G->T)\n	1	\N	\N	159e1c0a-4458-47fb-bd43-3d387582c201	2015-02-26 02:05:22.365036
331	152	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 12\nsource_id: 119\nvariant_id: 45\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37881332-37881332 (G->A)\n	1	\N	\N	24a9ea2b-1183-4de3-8d43-9b6a9193bd1f	2015-02-26 02:05:22.43426
332	153	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 12\nsource_id: 120\nvariant_id: 46\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d79ad637-54d7-4d8d-b81a-27de2e4481d8	2015-02-26 02:05:22.498931
333	154	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L536Q ligand binding domain mutation have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 38\ndisease_id: 12\nsource_id: 121\nvariant_id: 46\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	19596520-62c8-4af0-8857-5590989e5eb3	2015-02-26 02:05:22.572003
334	155	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L536Q ligand binding domain mutation have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 39\ndisease_id: 12\nsource_id: 121\nvariant_id: 46\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	08c07d5f-8390-45db-8ae6-d1826f7d65c3	2015-02-26 02:05:22.623151
335	156	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 12\nsource_id: 120\nvariant_id: 47\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bd66265d-95cf-47ba-9c33-684a8db87346	2015-02-26 02:05:22.693293
336	157	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The N538G ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 38\ndisease_id: 12\nsource_id: 121\nvariant_id: 47\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f859ac0d-6ace-4dda-b8de-c224be8cc731	2015-02-26 02:05:22.775624
337	158	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The N538G ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 39\ndisease_id: 12\nsource_id: 121\nvariant_id: 47\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a6b23d98-b6c7-4d93-b340-f2407ceebda3	2015-02-26 02:05:22.827563
338	159	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 12\nsource_id: 120\nvariant_id: 48\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419923-152419923 (A->G)\n	1	\N	\N	3e5eaf5d-6db3-4676-bca5-4d463b6c1f14	2015-02-26 02:05:22.886843
339	160	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537C ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 38\ndisease_id: 12\nsource_id: 121\nvariant_id: 48\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419923-152419923 (A->G)\n	1	\N	\N	a54ae4db-a5f1-443d-82a1-cb17f8f360e0	2015-02-26 02:05:22.951927
340	161	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537C ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 39\ndisease_id: 12\nsource_id: 121\nvariant_id: 48\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419923-152419923 (A->G)\n	1	\N	\N	454595d6-5b08-43a1-95cd-159ee89e3217	2015-02-26 02:05:23.005345
341	162	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 12\nsource_id: 120\nvariant_id: 49\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419922-152419922 (T->A)\n	1	\N	\N	fcd99f12-c3f4-45d3-be35-e1669ffbdbf6	2015-02-26 02:05:23.074989
342	163	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537N ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 38\ndisease_id: 12\nsource_id: 121\nvariant_id: 49\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419922-152419922 (T->A)\n	1	\N	\N	bc469711-d1b7-4564-bee6-d67f57ef235c	2015-02-26 02:05:23.145236
343	164	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537N ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 39\ndisease_id: 12\nsource_id: 121\nvariant_id: 49\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419922-152419922 (T->A)\n	1	\N	\N	bc41653b-ff8f-48c6-8947-783eb9c260df	2015-02-26 02:05:23.213762
344	165	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 12\nsource_id: 120\nvariant_id: 50\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419923-152419923 (A->C)\n	1	\N	\N	61587f82-a73b-406d-aaeb-64323146fbc0	2015-02-26 02:05:23.278862
345	166	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537S ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 38\ndisease_id: 12\nsource_id: 121\nvariant_id: 50\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419923-152419923 (A->C)\n	1	\N	\N	643311ac-de2d-4522-ba89-9eac2709ecce	2015-02-26 02:05:23.346693
346	167	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537S ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 39\ndisease_id: 12\nsource_id: 121\nvariant_id: 50\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419923-152419923 (A->C)\n	1	\N	\N	b5219301-ff27-4ea8-b2f4-1e0d283efc9c	2015-02-26 02:05:23.398325
347	168	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib\n  treatment resulted in necrosis and shrinkage of the primary and metastatic sites\n  and overall stable disease. However, this response did not meet criteria for RECIST\n  partial response.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 40\ndisease_id: 23\nsource_id: 122\nvariant_id: 51\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	36fdb76d-2b8d-48b6-af51-fe734805a799	2015-02-26 02:05:23.466279
348	169	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Following identification of this fusion in a patient, this fusion was expressed\n  in HEK 293T cells leading to oligomerization in the absence of ligand, induced morphologic\n  changes in the cell and increased cellular proliferation consistent with FGFR activation.\n  Mouse xenografts using the bladder cell line SW780 which harbors an FGFR3-BAIAP2L1\n  fusion showed reduced tumor burden when treated with the FGFR inhibitor PD173074.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 41\ndisease_id: 24\nsource_id: 123\nvariant_id: 52\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	437aebbb-71c2-4fe7-97f6-f0e96ec053ac	2015-02-26 02:05:23.546824
349	170	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This\n  fusion is also shown to be recurrent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 42\ndisease_id: 23\nsource_id: 122\nvariant_id: 53\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7ce4d982-035d-4d6a-9d04-1a73d15f4368	2015-02-26 02:05:23.627911
350	171	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This\n  fusion is also shown to be recurrent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 40\ndisease_id: 23\nsource_id: 122\nvariant_id: 53\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6bad258f-d9ff-4b53-85ef-74ea4328cd38	2015-02-26 02:05:23.703213
351	172	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with\n  FLT3 internal tandem repeat\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 43\ndisease_id: 1\nsource_id: 124\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9cbc6fed-d85e-4ddd-8fd4-b8303e829e6d	2015-02-26 02:05:23.779171
352	173	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  FLT3-ITD mutation\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 44\ndisease_id: 1\nsource_id: 125\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1b70100f-5c1a-4514-9c3c-5671a73580d2	2015-02-26 02:05:23.838262
353	174	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML\n  patients with FLT3 mutations\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 45\ndisease_id: 1\nsource_id: 126\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	52e10236-c88e-4ca5-986a-f4fa3e04658a	2015-02-26 02:05:23.902077
354	175	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 29\ndisease_id: 1\nsource_id: 127\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c0cf154a-3eef-445f-bfdf-65eae900dd67	2015-02-26 02:05:23.963893
355	176	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared\n  to wild type\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 128\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b1df378e-3abd-4ede-8b31-7ac6d56ec0fa	2015-02-26 02:05:24.022272
356	177	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD mutation without NPM1 was associated with increased relapse risk and\n  reduced overall survival in young adult AML patients (median age 43)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 129\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	975a9830-7f7a-4dce-85a9-8a882f611771	2015-02-26 02:05:24.082589
357	178	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Increasing level of FLT3-ITD mutant correlated with reduced disease-free and\n  overall survival as well as increased relapse rate\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 129\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e9df57b3-c07b-4983-9018-2ad669493bf9	2015-02-26 02:05:24.162814
358	179	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 105\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e2ba6ca2-56d3-4991-bc79-462d0a5d01f0	2015-02-26 02:05:24.219068
359	180	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD duplication length did not effect prognosis in patients with an NPM1\n  mutation\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 130\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	adbfa824-b0ca-46c5-b25b-8830e9e4e135	2015-02-26 02:05:24.281074
360	181	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Size of the FLT3-ITD mutant duplication had no impact on overall survival or\n  relapse rate\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 129\nvariant_id: 54\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c9ca70ca-5701-43bc-a702-c96721cd81c9	2015-02-26 02:05:24.352962
361	182	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 130\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d889d98d-74f8-47d5-8e92-5ba0dfab3588	2015-02-26 02:05:24.413166
362	183	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 130\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6e87b92e-651e-4d32-9c82-b9a3f4114aa5	2015-02-26 02:05:24.46768
363	184	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free\n  and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 130\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	89628fab-ca57-44db-9473-aa3631da71f2	2015-02-26 02:05:24.527259
364	185	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with internal tandem repeat of FLT3 has worse overall survival\n  than patients with wild type FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 128\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e3944604-6ad9-4753-9a39-cb5c6c620d69	2015-02-26 02:05:24.607235
365	186	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated\n  with worse overall survival compared to those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 107\nvariant_id: 54\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	55398084-4f21-44ec-b0ce-9cc06974f413	2015-02-26 02:05:24.661405
366	187	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Meta-analysis of studies involving cytogentically normal younger (<60) patients\n  showed reduced overall and relapse-free survival for patients with FLT3-ITD\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 131\nvariant_id: 54\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ba6bf6e1-1a7f-4863-8f8e-00f272010360	2015-02-26 02:05:24.718594
367	188	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor\n  survival and inferior disease free survival in patients with AML.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 132\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	996526f8-d0b1-4358-86c0-064dd71025db	2015-02-26 02:05:24.780927
368	189	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeats, but not\n  in wild type or D835 mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 1\nsource_id: 133\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	cd5e6c9b-0cec-4725-8b08-967263fee3e2	2015-02-26 02:05:24.848781
369	190	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 46\ndisease_id: 1\nsource_id: 127\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bfb1bb98-2ba7-42cf-a96b-34e01c0bef88	2015-02-26 02:05:24.910588
370	191	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML\n  patients with FLT3 mutations\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 45\ndisease_id: 1\nsource_id: 126\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ee9c8108-6d29-4a0d-b7d6-47760bc70d54	2015-02-26 02:05:24.973308
371	192	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 29\ndisease_id: 1\nsource_id: 127\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1a7fcec8-33d1-4d63-8f4b-390cee3bc8e3	2015-02-26 02:05:25.056571
372	193	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were not significantly different in AML patients with FLT3 D835 mutations\n  compared to wild type\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 128\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	41b8b2e5-11da-4a49-8d75-4726622ea2b0	2015-02-26 02:05:25.121521
373	194	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with D835 mutation of FLT3 did not have worse overall survival\n  than patients with wild type FLT3\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 128\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0fd7d4ea-643a-4a8d-a722-bbd0326addbe	2015-02-26 02:05:25.178778
374	195	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 134\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	044d11e6-1deb-4105-8d48-352d5882540b	2015-02-26 02:05:25.248866
375	196	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients, FLT3-TKD mutation was associated with poorer disease free survival\n  compared to patients with wild type FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 135\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d0b0e50d-113c-41a4-865c-7c82cb74c156	2015-02-26 02:05:25.305909
376	197	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeat mutations,\n  but not in patients with wild type or D835 mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 1\nsource_id: 133\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f01e82ef-04f5-4e5b-ba7a-20801d0ea615	2015-02-26 02:05:25.366347
377	198	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated\n  with sorafenib resistance\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 1\nsource_id: 136\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ac6e5572-7b27-4d43-aaca-60673f1a886b	2015-02-26 02:05:25.441336
378	199	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 46\ndisease_id: 1\nsource_id: 127\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4af1a422-39aa-4356-9493-b37a9a8eec11	2015-02-26 02:05:25.524839
379	200	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 47\ndisease_id: 25\nsource_id: 137\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d17b9343-998d-4169-8a8e-4ab45e76ace4	2015-02-26 02:05:25.592132
380	201	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 48\ndisease_id: 25\nsource_id: 137\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fdead54f-6ce1-431a-9795-7adc61015364	2015-02-26 02:05:25.650292
381	202	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In leukemia patients, IDH1 R132 mutation is associated with adults and normal\n  karyotype.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 138\nvariant_id: 57\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b7f93e80-fc2e-4c47-90bc-7de1829ecff1	2015-02-26 02:05:25.712008
382	203	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 139\nvariant_id: 57\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	83055f6e-beda-4032-bae2-3893df12f54d	2015-02-26 02:05:25.770024
383	204	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The complete remission and overall survival rates in patients with IDH1 R132\n  mutation is not significantly different from those who do not have this mutation\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 140\nvariant_id: 57\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3c265f7a-ff11-4844-a332-524e5becedfa	2015-02-26 02:05:25.833485
384	205	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic\n  for better survival compared to patients who harbor wild type mutation.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 26\nsource_id: 141\nvariant_id: 57\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6a84a736-21a4-41f6-bda3-f2b0458b978f	2015-02-26 02:05:25.971828
385	206	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with glioblastoma, those harboring IDH1 R132 mutation has higher\n  overall survival compared to those who do not have IDH1 mutation\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 27\nsource_id: 142\nvariant_id: 57\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	51ff88a0-d6aa-4663-8957-cfd8ab3e6f5a	2015-02-26 02:05:26.036855
386	207	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated\n  with worse overall survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 28\nsource_id: 143\nvariant_id: 57\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0092e114-7bf1-45d7-a6d0-ecf189d7807c	2015-02-26 02:05:26.099882
387	208	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation is associated with patients of older age, high platelet count\n  during diagnosis, cytogenic normalcy and NPM1 mutation\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 144\nvariant_id: 58\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:209113113-209113113 (G->A)\n	1	\N	\N	db925efb-855e-45fb-abf8-17c91e9de1a7	2015-02-26 02:05:26.158519
388	209	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 144\nvariant_id: 58\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:209113113-209113113 (G->A)\n	1	\N	\N	5e6990a5-0ef4-44c3-a406-b2aa14de3cdd	2015-02-26 02:05:26.23035
389	210	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 144\nvariant_id: 59\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:209113112-209113112 (C->T)\n	1	\N	\N	1be03841-e19f-4e13-9cb0-11199e087447	2015-02-26 02:05:26.292331
390	211	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 144\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:209113112-209113112 (C->A)\n	1	\N	\N	24a2ee7f-b9e7-4f53-b629-d33568ca7b1f	2015-02-26 02:05:26.350179
391	212	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R140Q/L have event free survival and\n  overall survival similar to those with wild-type IDH2\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 145\nvariant_id: 61\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b5111bfe-4092-4b11-8e65-27ebd329cbe1	2015-02-26 02:05:26.412796
392	213	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact\n  overall survival or disease free survival.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 146\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3597ce90-ab5d-4cbe-86c9-9a2776a385ee	2015-02-26 02:05:26.468922
393	214	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with IDH2 R140K mutation have improved overall survival compared\n  to those with wild-type IDH2\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 147\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f3c2af6e-c48d-4130-8fba-d1462c5bb287	2015-02-26 02:05:26.521217
394	215	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: R140Q mutation in IDH2 does not have prognostic value in patients with MDS.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 29\nsource_id: 148\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	da0e55de-68e4-42b9-a18a-ae4bde53492e	2015-02-26 02:05:26.57902
395	216	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L\n  do not confer prognostic value (overall survival)\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 29\nsource_id: 149\nvariant_id: 61\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2f87ca11-a175-4d79-bf50-6dc915ff2d4c	2015-02-26 02:05:26.637329
396	217	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R172K have event free survival and\n  overall survival similar to those with wild-type IDH2.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 145\nvariant_id: 62\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 15:90631838-90631838 (C->T)\n	1	\N	\N	1669d357-8927-4d3a-be31-2e34aa83e1c9	2015-02-26 02:05:26.6966
397	218	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with an IDH2 R172K mutation have worse overall survival compared\n  to those with wild-type IDH2\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 147\nvariant_id: 62\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 15:90631838-90631838 (C->T)\n	1	\N	\N	d38d4e23-5edc-448c-81eb-c23dd0c1b680	2015-02-26 02:05:26.755873
398	219	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutations such as R172K\n  do not confer prognostic value (overall survival)\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 29\nsource_id: 149\nvariant_id: 62\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 15:90631838-90631838 (C->T)\n	1	\N	\N	15a5606f-41d9-433c-9a9d-5a66b7607b78	2015-02-26 02:05:26.815154
399	220	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical\n  CML).\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 2\nsource_id: 150\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 9:5073770-5073770 (G->T)\n	1	\N	\N	a25df531-07ff-419c-aa25-10a596b0bb46	2015-02-26 02:05:26.887736
400	221	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is associated with myeloid neoplasms (AML, MDS, CMML/aCML)\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 2\nsource_id: 150\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 9:5073770-5073770 (G->T)\n	1	\N	\N	899a1339-38af-4577-8c3e-d2b58f8b9156	2015-02-26 02:05:26.944641
401	222	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or\n  CLL).\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 30\nsource_id: 150\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 9:5073770-5073770 (G->T)\n	1	\N	\N	4600c6be-8d0b-47bb-b75e-56c1957da414	2015-02-26 02:05:26.997491
402	223	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The SNP rs10974944 in the Jak2 locus is associated with increased predisposition\n  for JAK2 V617F mutation and its associated cancer\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 31\nsource_id: 151\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 9:5073770-5073770 (G->T)\n	1	\N	\N	f7494ce8-cf14-4939-af90-5be8a2a71deb	2015-02-26 02:05:27.066255
403	224	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction\n  in the percentage of cells harboring JAK2 V617F.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 49\ndisease_id: 32\nsource_id: 152\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 9:5073770-5073770 (G->T)\n	1	\N	\N	fc7f8188-a360-49e0-becf-e7b52469865a	2015-02-26 02:05:27.132441
404	225	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 50\ndisease_id: 32\nsource_id: 153\nvariant_id: 63\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 9:5073770-5073770 (G->T)\n	1	\N	\N	98f062e4-b06e-4569-80dd-4a9947a7db79	2015-02-26 02:05:27.191109
405	226	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In acute myloid leukemia patients, D816 mutation is associated with earlier\n  relapse and poorer prognosis than wildtype KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 33\nsource_id: 154\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55599321-55599321 (A->T)\n	1	\N	\N	96fb3888-fd27-4f1a-af90-b044db1ccf23	2015-02-26 02:05:27.255072
406	227	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 34\nsource_id: 155\nvariant_id: 65\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	39e96496-8812-44f5-a962-556045bc6726	2015-02-26 02:05:27.327052
407	228	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between wildtype KIT or KIT mutations in\n  exon 9 or 11 in survival among GIST patients.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 34\nsource_id: 156\nvariant_id: 65\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	02ca8505-bc2f-4e95-a10c-cc51618e1308	2015-02-26 02:05:27.394457
408	229	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations were identified in morphologically benign, incidentally discovered\n  GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating\n  is not a prognostic marker for GIST\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 34\nsource_id: 157\nvariant_id: 65\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	579e11bd-ecb3-4251-9988-a953a34c458e	2015-02-26 02:05:27.490288
409	230	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with worse overall and cause-specific prognosis in\n  patients with GIST compared to patients with wildtype KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 34\nsource_id: 155\nvariant_id: 65\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5f915204-0c17-417d-b8c1-a922ac606928	2015-02-26 02:05:27.55446
410	231	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 18\nsource_id: 158\nvariant_id: 65\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	15541b27-78bb-4628-9325-03771d73a9df	2015-02-26 02:05:27.624257
411	232	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 8\nsource_id: 159\nvariant_id: 65\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d96f6c40-d321-4993-a88a-a837df1c435d	2015-02-26 02:05:27.697724
412	233	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 8\nsource_id: 159\nvariant_id: 67\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	12d30f73-74a9-453f-8e8f-981ea1c55ff4	2015-02-26 02:05:27.763871
413	234	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST patients with exon 14 KIT mutations had reduced overall survival compared\n  to patients wiltype for KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 34\nsource_id: 156\nvariant_id: 68\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9bac496c-3941-47fb-8e4a-45296c323ad5	2015-02-26 02:05:27.824485
414	235	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 34\nsource_id: 155\nvariant_id: 69\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8650a1be-5e19-4cfb-82a1-1e684d610886	2015-02-26 02:05:27.88065
415	236	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 8\nsource_id: 159\nvariant_id: 69\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5ede4ad5-3507-4dfa-a4ae-44c0fae03c01	2015-02-26 02:05:27.946123
416	237	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between wildtype KIT or KIT mutations in\n  exon 9 or 11 in survival among GIST patients.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 34\nsource_id: 156\nvariant_id: 70\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	272e5380-6992-401a-a4e6-1d13f9e3f063	2015-02-26 02:05:28.01559
417	238	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 18\nsource_id: 158\nvariant_id: 70\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	999d6faf-8270-4314-bbee-38f42f3ec0e1	2015-02-26 02:05:28.078939
418	239	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a case study, a patient with anal melanoma harboring a 7 codon duplication\n  in the juxtamambrane region (exon 11) of KIT showed marked response 4 months after\n  imatinib treatment.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 35\nsource_id: 160\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d5590ed0-a53b-4a0b-96cf-3a9811838b40	2015-02-26 02:05:28.139392
419	240	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive\n  to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation\n  of KIT but did not alter the binding affinity of dasatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 5\ndisease_id: 8\nsource_id: 161\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bc25a447-0c16-492c-8bfb-080ef0d412c4	2015-02-26 02:05:28.220194
420	241	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 8\nsource_id: 161\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4c0e9574-8e14-4b56-ae4c-e6b73a8281d0	2015-02-26 02:05:28.289319
421	242	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 8\nsource_id: 161\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	26b478f6-2cf6-4c02-be3c-b49921e7a989	2015-02-26 02:05:28.337515
422	243	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 8\nsource_id: 161\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	151ae246-906b-4c8f-b566-4ed79e7b7b4e	2015-02-26 02:05:28.397382
423	244	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 5\ndisease_id: 4\nsource_id: 162\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4ee1636b-417f-4df1-92ef-4e7543a1a02e	2015-02-26 02:05:28.476062
424	245	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 4\nsource_id: 162\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fd0b6314-4620-4df2-8dfe-1d6925a6948d	2015-02-26 02:05:28.532111
425	246	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 4\nsource_id: 162\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ffb0e773-aff2-47d0-a94c-c8b7e13664ef	2015-02-26 02:05:28.579964
426	247	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT V654A results in imatinib resistance in GIST patient-derived cell lines.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 34\nsource_id: 163\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55594258-55594258 (T->C)\n	1	\N	\N	2a7e6ab0-dd70-44ca-a357-d84a19f4c3f6	2015-02-26 02:05:28.647845
427	248	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective\n  for those that are refractory to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 51\ndisease_id: 34\nsource_id: 164\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55594258-55594258 (T->C)\n	1	\N	\N	f6e776f3-8773-4eef-8751-8c76d7e6d7f7	2015-02-26 02:05:28.709892
428	249	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer who harbor KRAS mutation have low response rate\n  to cetuximab\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 7\nsource_id: 165\nvariant_id: 73\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9843056b-5733-41d4-a525-4fa3269b0934	2015-02-26 02:05:28.780801
429	250	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: Sensitivity\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 52\ndisease_id: 7\nsource_id: 166\nvariant_id: 74\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	28e4adab-9bdd-46c2-8a62-48a1c5b051a9	2015-02-26 02:05:28.837637
430	251	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, Exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 31\ndisease_id: 4\nsource_id: 167\nvariant_id: 74\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1acc4f90-4234-4e1f-a56a-ecce421157db	2015-02-26 02:05:28.890913
431	252	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, Exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 32\ndisease_id: 4\nsource_id: 167\nvariant_id: 74\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	01a1c98d-7c60-4511-a7ab-86f7022cd2a1	2015-02-26 02:05:28.972979
432	253	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 36\nsource_id: 168\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	022b4979-9a3f-4d46-84fe-9ec4db297267	2015-02-26 02:05:29.03293
433	254	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12\n  mutation did not impact overall or disease free survival.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 9\nsource_id: 169\nvariant_id: 75\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f8bdd6cb-ed65-459d-9d94-6b4cac144931	2015-02-26 02:05:29.096502
434	255	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 37\nsource_id: 170\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bcb28844-38e6-4e89-b777-131a77e1a09c	2015-02-26 02:05:29.16934
435	256	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung cancer patients are not associated with smoking history,\n  age or gender\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 171\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c6cf2531-b9e3-43c7-86c8-5fc0a1bdd6ff	2015-02-26 02:05:29.229581
436	257	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no strong association between KRAS mutation status and overall or progression\n  free survival in patients with NSCLC\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 172\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7407ce31-e435-4977-af9c-efa6bc84668a	2015-02-26 02:05:29.290572
437	258	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A meta-analysis showed KRAS mutation is associated with worse outcome in patients\n  with NSCLC\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 173\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	92389f2d-60ce-45eb-9b1b-47664e71843c	2015-02-26 02:05:29.364757
438	259	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with lack of response to cetuximab\n  in patients with advanced colorectal cancer\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 7\nsource_id: 174\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6160c5ea-ad23-4edb-91b1-125fe1c6ed8e	2015-02-26 02:05:29.420816
439	260	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations\n  (primarily G12/G13) had lower response and disease control rates and shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype KRAS\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 7\nsource_id: 71\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6897eb52-40e8-4cf7-b654-be1787f45a6f	2015-02-26 02:05:29.474406
440	261	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with reduced progression-free and\n  overall survival in patients with advanced colorectal cancer\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 174\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5f3ab855-d752-42ce-b8c8-59461758b358	2015-02-26 02:05:29.529202
441	262	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12C occur more frequently in women than men\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 18\nsource_id: 175\nvariant_id: 77\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398285-25398285 (C->A)\n	1	\N	\N	caf259a9-4d6a-48be-bbc1-3dc8ec3c66b3	2015-02-26 02:05:29.586151
442	263	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12D mutation occurs in never smokers significantly more often than in\n  smokers\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 18\nsource_id: 175\nvariant_id: 78\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398284-25398284 (C->T)\n	1	\N	\N	3b2baeba-ba49-4437-ae09-dd5e4b2e14c1	2015-02-26 02:05:29.645056
443	264	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 13\ndisease_id: 4\nsource_id: 176\nvariant_id: 78\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398284-25398284 (C->T)\n	1	\N	\N	faa6e953-824f-4b39-a7ad-8b7bce860ec8	2015-02-26 02:05:29.705126
444	265	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 53\ndisease_id: 4\nsource_id: 177\nvariant_id: 78\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398284-25398284 (C->T)\n	1	\N	\N	a59df46b-13f0-4605-8e06-42f6d8c7152b	2015-02-26 02:05:29.768904
445	266	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 54\ndisease_id: 4\nsource_id: 177\nvariant_id: 78\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398284-25398284 (C->T)\n	1	\N	\N	8d9dc7e7-59be-496e-bb28-b8575f94a231	2015-02-26 02:05:29.817623
446	267	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 36\nsource_id: 168\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fac1bc45-c5d0-4f3c-8ab3-0cca59e7a304	2015-02-26 02:05:29.875516
447	268	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 38\nsource_id: 170\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3cd0d03e-76d5-4ac9-8e0c-32f254bba3c7	2015-02-26 02:05:29.938371
448	269	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung cancer patients are not associated with smoking history,\n  age or gender\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 171\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f333d021-0802-441b-b6b5-56f0c46c48c2	2015-02-26 02:05:30.00705
449	270	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft model\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 9\nsource_id: 178\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398281-25398281 (C->T)\n	1	\N	\N	59c1cade-eed3-45c7-8e02-0830164fc55e	2015-02-26 02:05:30.073728
450	271	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft model\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 9\nsource_id: 178\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398281-25398281 (C->T)\n	1	\N	\N	4a93903a-0e89-4b45-a1b2-9041031c3f9e	2015-02-26 02:05:30.138278
451	272	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 9\nsource_id: 178\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398281-25398281 (C->T)\n	1	\N	\N	8c912f2e-9ca0-4ab9-a420-3332da0f91bf	2015-02-26 02:05:30.204745
452	273	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 9\nsource_id: 178\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398281-25398281 (C->T)\n	1	\N	\N	d5df936b-7b7c-43bc-a62a-50537790b364	2015-02-26 02:05:30.270011
453	274	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition\n  by about 5 fold.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 23\ndisease_id: 8\nsource_id: 179\nvariant_id: 81\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 15:66729162-66729162 (C->T)\n	1	\N	\N	f16e3823-b9e0-4200-87ad-80a16a44de58	2015-02-26 02:05:30.333323
454	275	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 23\ndisease_id: 8\nsource_id: 180\nvariant_id: 82\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 15:66727451-66727451 (A->C)\n	1	\N	\N	31a8cc91-3cda-4363-9ef7-5d59c52ea9b0	2015-02-26 02:05:30.401471
455	276	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 20\nsource_id: 101\nvariant_id: 83\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	22fddcef-71c5-4b6c-927a-7bc987808fa0	2015-02-26 02:05:30.508468
456	277	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 20\nsource_id: 101\nvariant_id: 83\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	613388ca-4c06-4fba-aef8-566dbb934ad6	2015-02-26 02:05:30.563835
457	278	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation has been shown to confer sensitivity to alkylating\n  agents, such as carmustine, in glioblastoma cells.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 55\ndisease_id: 39\nsource_id: 181\nvariant_id: 84\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4b5e7761-bff2-4fe9-8c6b-5d18da5d2f73	2015-02-26 02:05:30.630564
458	279	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation was associated with increased overall and disease-free\n  survival as well as tumor regression in patients with gliomas.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 39\nsource_id: 181\nvariant_id: 84\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	58368b59-e2e8-4a84-b5da-d4731f5fb2d5	2015-02-26 02:05:30.701519
459	280	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor,\n  such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may\n  be useful in treating patients with initial resistance to alkylating agents.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 56\ndisease_id: 39\nsource_id: 181\nvariant_id: 84\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ca4d138f-d8e7-45b5-b382-cf1dd6f5f77f	2015-02-26 02:05:30.775806
460	281	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted\n  from temozolomide. This benefit was also methylation status dependent, as those\n  without methylation did not see increased survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 22\ndisease_id: 39\nsource_id: 182\nvariant_id: 84\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0f221ce2-31a4-488e-b5a1-e12ccf37384a	2015-02-26 02:05:30.850799
461	282	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  NPM1 mutation regardless of FLT3-ITD status\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 44\ndisease_id: 1\nsource_id: 125\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	65745293-7cb5-4972-98a0-38bed25cc17b	2015-02-26 02:05:30.919707
462	283	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment improved overall and relapse-free survival in older (>60) patients\n  with mutant NPM1 and without FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 44\ndisease_id: 1\nsource_id: 183\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ee6ffb99-2cf1-4c77-ad64-77f5650acaff	2015-02-26 02:05:30.987303
463	284	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young patients (18-60) with NPM1 mutations had improved overall survival following\n  high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However,\n  the adjusted p-value for multiple testing is 0.11.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 29\ndisease_id: 1\nsource_id: 184\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d02c557f-b869-44b1-a786-75537a4b59b4	2015-02-26 02:05:31.051642
464	285	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype\n  (0/55).\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 185\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a221dc5b-5618-4d8d-9a11-bbf52e0c437d	2015-02-26 02:05:31.117958
465	286	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid\n  leukemia.\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 186\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e10b9f1e-7d7d-4cb0-a42f-bcf2d71bb339	2015-02-26 02:05:31.18317
466	287	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid\n  leukemia\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 185\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c72bd96b-70ad-4684-aa55-9125deb9b7e0	2015-02-26 02:05:31.243277
467	288	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML with mutated NPM1 is a provisional entity in WHO classification of acute\n  myeloid leukemia (AML). This mutation should be tested for in clinical trials and\n  is recommended for testing in patients with cytogentically normal AML.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 187\nvariant_id: 85\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f9908fbd-0578-4a23-be4b-55a345bc69d5	2015-02-26 02:05:31.305423
468	289	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with normal karyotype in older (>60) patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 183\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	993c7661-7a50-4412-b020-cdd4cc56d37b	2015-02-26 02:05:31.3645
469	290	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 129\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	93f7f4b7-09fb-4da8-98ef-b73663fb70b5	2015-02-26 02:05:31.426041
470	291	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid\n  leukemia.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 186\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3ed6f4b2-c023-406d-95f3-79e800a685d4	2015-02-26 02:05:31.481276
471	292	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and\n  increased overall survival in young adult acute myeloid leukemia patients (median\n  age 43)\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 129\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	72329579-0bfc-4578-a70b-e53267c2716e	2015-02-26 02:05:31.541604
472	293	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations were associated with improved complete remission\n  rates as well as overall and disease free survival when all ages and karyotype were\n  analyzed; however, when only normal karyotype patients were analyzed, only complete\n  remission rates were improved\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 183\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7eb073e7-b22d-45fc-95d2-296b190783e3	2015-02-26 02:05:31.606428
473	294	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 and CD123 expression was significantly increased in patients with NPM1\n  mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33\n  and anti-CD123 therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 188\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	84fe93d0-b068-4fe7-96ad-92ea2b3d90b5	2015-02-26 02:05:31.664925
474	295	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 expression was significantly increased in patients with NPM1 mutation with\n  or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 188\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b9fa244e-4ad5-4a54-95b3-fd4ea6a27fa8	2015-02-26 02:05:31.719291
475	296	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 105\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1f138510-1a81-43a6-ba26-c706d96018f1	2015-02-26 02:05:31.776049
476	297	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 139\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5a86d558-cb03-4ce5-b820-05c823939edc	2015-02-26 02:05:31.844753
477	298	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was significantly associated with complete remission\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 99\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5d55ec37-626f-4c0c-bb34-6c59bcc34272	2015-02-26 02:05:31.906875
478	299	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with increased overall, event-free and relapse-free\n  survival and remission after induction therapy in the entire cohort and in the cytogenetically\n  normal subset of patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 189\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	28c8ec4c-4035-4e86-ab9e-17ecd064b4c9	2015-02-26 02:05:32.015263
479	300	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Intermediate risk patients with mutant NPM1 had improved overall survival with\n  the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1\n  and IDH2\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 184\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	dffb3bdf-2364-4335-bcda-9e69415fb204	2015-02-26 02:05:32.073049
480	301	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and\n  without FLT3-ITD have increased overall survival, relapse-free survival and response\n  to induction chemotherapy\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 190\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a5386552-d851-46eb-80f7-fd52f508f52a	2015-02-26 02:05:32.131019
481	302	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with higher complete remission rates, lower cumulative\n  incidence of relapse and higher overall survival in intermediate risk AML patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 191\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	06969bac-3ceb-490d-9c80-6c57de2b4789	2015-02-26 02:05:32.193206
482	303	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and event-free survival was longer for\n  normal karyotype AML patients with Exon 12 NPM1 mutations.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 186\nvariant_id: 85\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	11cefeb3-ef86-4f73-bd82-d5baef44cc7e	2015-02-26 02:05:32.25511
483	304	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with increased complete remission rates as well\n  as longer overall disease free survival in normal karyotype AML patients >59 years\n  old, particularly in those >69 years old\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 192\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	326543e5-c3a9-47ee-92cb-6d7a9645b4d3	2015-02-26 02:05:32.31975
484	305	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and both disease-free and overall survival\n  were longer for patients with Exon 12 NPM1 mutations.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 185\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6326b429-948c-4c2d-a3ba-df293fb42c71	2015-02-26 02:05:32.383814
485	306	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 134\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e8d435f2-8fd0-4e4a-b285-71ab02d41dd3	2015-02-26 02:05:32.4395
486	307	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved\n  overall and relapse-free survival, complete remission rates and remission duration\n  compared to all other combinations of mutants\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 193\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b4488060-6c92-44fc-9a7e-a3977c4935f9	2015-02-26 02:05:32.500264
487	308	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coocurrence of FLT3-ITD mutations significantly reduced overall survival of\n  normal karyotype AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 185\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f919b913-8a61-49a2-95dd-8b04408dbe8e	2015-02-26 02:05:32.569003
488	309	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 130\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	27d95bbd-c21a-4459-bdb8-bda196dc23cd	2015-02-26 02:05:32.62884
489	310	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 130\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1e206847-89ba-43f6-abf5-3f2ccf839ca8	2015-02-26 02:05:32.685935
490	311	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse\n  in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction\n  in NPM1 levels following first line therapy (N=15)\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 194\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1acbf3b6-8b88-4fe0-9473-4b16cfb59b81	2015-02-26 02:05:32.746804
491	312	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TET2 mutation reduces overall survival of normal karyotype patients with either\n  an NPM1 mutation or NPM1 mutation without FLT3-ITD\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 195\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	14ffd779-302f-4434-8d7c-868371c99ed0	2015-02-26 02:05:32.819471
492	313	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival\n  and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate\n  risk AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 191\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	da0ad7cf-6143-42e7-be72-8b4dc92c252e	2015-02-26 02:05:32.879433
493	314	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and\n  FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 57\ndisease_id: 1\nsource_id: 196\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	889cacde-3dc3-46b5-a38a-3847bffcbac4	2015-02-26 02:05:32.949714
494	315	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and\n  FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 44\ndisease_id: 1\nsource_id: 196\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5d0cec5a-c6c5-47cf-939a-dec6cf3fe8f5	2015-02-26 02:05:33.009537
495	316	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: For patients with NPM1 mutation that achieved CR following induction therapy,\n  relapse-free survival was improved following treatment with valproic acid\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 58\ndisease_id: 1\nsource_id: 197\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8eeabd56-6f9d-4d45-9306-ddad0062732d	2015-02-26 02:05:33.068633
496	317	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No NPM1 mutations were identified in patients with favorable risk cytogenetics\n  (79/215 patients)\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 107\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\n	1	\N	\N	63fb4a24-8670-4bcc-9ca0-414ff00ab684	2015-02-26 02:05:33.126414
497	318	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with intermediate risk cytogenetics (including\n  normal karyotype)\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 107\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\n	1	\N	\N	ea6060c5-7eb3-40b1-9bac-bd5d4f1ac85d	2015-02-26 02:05:33.197159
498	319	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected\n  into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM\n  in AML patients was associated with good response to induction therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 198\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\n	1	\N	\N	74cf658b-c0f2-4ca9-b012-ddddea51f7bd	2015-02-26 02:05:33.268642
499	320	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 57\ndisease_id: 1\nsource_id: 196\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\n	1	\N	\N	01b9762c-65e1-4d79-8c27-86320bcd1f8a	2015-02-26 02:05:33.344006
500	321	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 8\nsource_id: 199\nvariant_id: 87\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2c21a229-45d3-42ae-b44f-dd9f2eb0a325	2015-02-26 02:05:33.454602
501	322	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of NRAS mutation in AML patients does not impact diease prognosis\n  (resistant disease, disease-free survival, complete remission rate, relapse rate,\n  induction death)\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 200\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	074ab727-ff65-482b-a65e-fabbe38a1c96	2015-02-26 02:05:33.516052
502	323	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 7\nsource_id: 201\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b3c731c6-6562-404d-b4be-71208d7bea7b	2015-02-26 02:05:33.575801
503	324	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 8\nsource_id: 199\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ee8bfa86-51ff-4d62-bac2-bce988a2e81b	2015-02-26 02:05:33.636255
504	325	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 201\nvariant_id: 89\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2da950e8-0144-4f94-b79f-7dd4f4e82a88	2015-02-26 02:05:33.696119
505	326	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 201\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3cc47744-5cb3-4980-92b9-8e690fb16f46	2015-02-26 02:05:33.757541
506	327	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NRAS mutation status does not impact overall, event-free or disease-free survival\n  in patients with AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 202\nvariant_id: 91\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	35a0a138-b6cf-462c-98c8-e6a7271a0d72	2015-02-26 02:05:33.83551
507	328	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Likely due to increased reliance of mutant NRAS on HSP90 for stabilization,\n  inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic\n  malignant melanoma with an NRAS G13D mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 59\ndisease_id: 8\nsource_id: 203\nvariant_id: 92\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 1:115258744-115258744 (C->T)\n	1	\N	\N	d0c673f5-8673-4f72-8256-292a52a52516	2015-02-26 02:05:33.90161
508	329	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status\n  (primarily Q61) was not informative for disease control rate, progression free survival\n  or overall survival following cetuximab plus chemotherapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 7\nsource_id: 71\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ff45f4e3-4c6f-4223-af35-a447fddfcc96	2015-02-26 02:05:33.956613
509	330	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation\n  (primarily Q61) have a significantly lower response rate to cetuximab than patients\n  wildtype for NRAS.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 7\nsource_id: 71\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1dea8fbb-4c9e-4c11-b7f0-4d97d5c498d1	2015-02-26 02:05:34.017761
510	331	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma associated with NRAS Q61 mutation was more often associated with those\n  at the extremity than those at the trunk\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 8\nsource_id: 204\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f9c0646c-1899-4b0f-8e13-3bdf4d951363	2015-02-26 02:05:34.079457
511	332	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 19\ndisease_id: 8\nsource_id: 205\nvariant_id: 93\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	823fca18-bbb2-4289-bbca-806916328b7f	2015-02-26 02:05:34.148966
512	333	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted\n  in disease free survival of 14 months\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 60\ndisease_id: 8\nsource_id: 206\nvariant_id: 94\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 1:115256528-115256529 (TT->CA)\n	1	\N	\N	44dc13e1-ba27-4b79-bdf3-12b38ded53c6	2015-02-26 02:05:34.219922
513	334	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with Q61R mutation, treatment with temozolomide resulted\n  in overall survival of 16 months\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 60\ndisease_id: 8\nsource_id: 206\nvariant_id: 95\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 1:115256528-115256529 (TT->CC)\n	1	\N	\N	b33374b5-bd84-46f7-8c1a-6c7e395b2118	2015-02-26 02:05:34.274682
514	335	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 61\ndisease_id: 40\nsource_id: 207\nvariant_id: 96\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55152092-55152093 (GA->AT)\n	1	\N	\N	eda27325-5345-4576-bc25-27cf86e3b1c8	2015-02-26 02:05:34.343757
515	336	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 61\ndisease_id: 40\nsource_id: 207\nvariant_id: 97\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55152093-55152093 (A->T)\n	1	\N	\N	db90eb2a-2324-4910-bdb7-baa00b21d139	2015-02-26 02:05:34.408038
516	337	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST cancer with D842V mutation is resistant to imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 40\nsource_id: 208\nvariant_id: 97\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55152093-55152093 (A->T)\n	1	\N	\N	fe050228-bf81-4f4d-8b45-f91e3e7f17c9	2015-02-26 02:05:34.471588
517	338	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib,\n  double mutation of V561D and D842V mutants are resistant to imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 40\nsource_id: 163\nvariant_id: 97\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55152093-55152093 (A->T)\n	1	\N	\N	03fe8698-63d6-45b7-80c8-0d73fdea97a6	2015-02-26 02:05:34.535671
518	339	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST tumors harboring PDGFRA D842V mutation are more likely to be benign than\n  malignant.\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 40\nsource_id: 209\nvariant_id: 97\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55152093-55152093 (A->T)\n	1	\N	\N	b85df2be-8909-49cc-8768-29f39c0f1a7d	2015-02-26 02:05:34.608744
519	340	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 61\ndisease_id: 40\nsource_id: 207\nvariant_id: 98\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55152092-55152092 (G->T)\n	1	\N	\N	121fbf6f-157e-4e95-836c-2ec6440389df	2015-02-26 02:05:34.674626
520	341	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib\n  resistance, crenolanib was significantly more potent at inhibiting kinase activity\n  than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 61\ndisease_id: 40\nsource_id: 207\nvariant_id: 99\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8e06e471-b164-4650-beb4-853cf97fc72a	2015-02-26 02:05:34.738274
521	342	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 61\ndisease_id: 40\nsource_id: 207\nvariant_id: 100\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5441f3a6-2edb-4215-829a-72645608a7d9	2015-02-26 02:05:34.813094
522	343	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 62\ndisease_id: 12\nsource_id: 210\nvariant_id: 101\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936082-178936082 (G->A)\n	1	\N	\N	eb7cfbd3-3bb8-4d23-8362-52681e8c34a0	2015-02-26 02:05:34.915369
523	344	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 63\ndisease_id: 12\nsource_id: 211\nvariant_id: 101\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936082-178936082 (G->A)\n	1	\N	\N	38933432-3891-4962-9f22-1bfdc2ed5527	2015-02-26 02:05:34.98736
524	345	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 7\nsource_id: 212\nvariant_id: 101\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936082-178936082 (G->A)\n	1	\N	\N	82ea2e35-e160-4f23-a7e3-f73a0fea32cf	2015-02-26 02:05:35.055727
525	346	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 12\ndisease_id: 7\nsource_id: 212\nvariant_id: 101\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936082-178936082 (G->A)\n	1	\N	\N	e8387b9c-5701-452e-bb0e-be8bcbee8620	2015-02-26 02:05:35.108947
526	347	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 213\nvariant_id: 101\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936082-178936082 (G->A)\n	1	\N	\N	978233de-44ea-4557-ba7d-00a046c56fd4	2015-02-26 02:05:35.173375
527	348	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E542K) or exon 20 mutations were associated\n  with poorer survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 212\nvariant_id: 101\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936082-178936082 (G->A)\n	1	\N	\N	df1a4580-ac27-4bc8-8383-b209c5862544	2015-02-26 02:05:35.232404
528	349	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 52\ndisease_id: 7\nsource_id: 166\nvariant_id: 101\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936082-178936082 (G->A)\n	1	\N	\N	9f34f0a7-7e99-48fb-ad96-7bc5b339f0a2	2015-02-26 02:05:35.28892
529	350	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 62\ndisease_id: 12\nsource_id: 210\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936091-178936091 (G->A)\n	1	\N	\N	0cdafb58-9b64-4eb7-88ef-6ea32f921a9d	2015-02-26 02:05:35.3581
530	351	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 63\ndisease_id: 12\nsource_id: 211\nvariant_id: 102\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936091-178936091 (G->A)\n	1	\N	\N	92f6377b-a59d-48c0-9bd2-cc5427872725	2015-02-26 02:05:35.420961
531	352	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 7\nsource_id: 212\nvariant_id: 102\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936091-178936091 (G->A)\n	1	\N	\N	ab79dd01-9ee0-4ca6-af30-ea82e0972e02	2015-02-26 02:05:35.483024
532	353	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 12\ndisease_id: 7\nsource_id: 212\nvariant_id: 102\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936091-178936091 (G->A)\n	1	\N	\N	12b19ccb-91ca-48c9-9c6f-4ff27408bd7b	2015-02-26 02:05:35.532658
533	354	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 213\nvariant_id: 102\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936091-178936091 (G->A)\n	1	\N	\N	3cc23655-3388-462a-b96d-7caed72b63e9	2015-02-26 02:05:35.599244
534	355	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 212\nvariant_id: 102\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936091-178936091 (G->A)\n	1	\N	\N	1075b342-3374-4fe9-a2a7-187e464863cd	2015-02-26 02:05:35.659022
535	356	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 52\ndisease_id: 7\nsource_id: 166\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936091-178936091 (G->A)\n	1	\N	\N	31b4a51a-0c03-430c-9520-4c68f7544dbf	2015-02-26 02:05:35.719154
536	357	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA\n  mutations had lower response and disease control rates as well as shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype PIK3CA\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 7\nsource_id: 71\nvariant_id: 103\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4f295154-9598-42b6-bda3-dde2fabd5973	2015-02-26 02:05:35.78343
537	358	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations\n  (Exon 9) had no significant impact on response rate, disease control rate, progression\n  free surivival or overall survival following cetuximab plus chemotherapy than those\n  with wildtype PIK3CA\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 7\nsource_id: 71\nvariant_id: 104\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7c4d8f29-8ddc-4204-afe1-5739bdb31c62	2015-02-26 02:05:35.846101
538	359	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with H1047R mutation showed increased sensitivity to\n  CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 62\ndisease_id: 12\nsource_id: 210\nvariant_id: 105\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178952085-178952085 (A->G)\n	1	\N	\N	e9260ddf-d20e-454c-891b-2f9509403dd3	2015-02-26 02:05:35.916107
539	360	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 63\ndisease_id: 12\nsource_id: 211\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178952085-178952085 (A->G)\n	1	\N	\N	e0e5c93a-366c-4a2c-9998-a1e48e34a024	2015-02-26 02:05:35.974456
540	361	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 7\nsource_id: 212\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178952085-178952085 (A->G)\n	1	\N	\N	70398ef2-f772-4237-9498-dfa744f0f2a1	2015-02-26 02:05:36.035229
541	362	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 12\ndisease_id: 7\nsource_id: 212\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178952085-178952085 (A->G)\n	1	\N	\N	6b4a6223-5f1f-4960-a25b-1064f0751c46	2015-02-26 02:05:36.08497
542	363	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 212\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178952085-178952085 (A->G)\n	1	\N	\N	be1bcc8e-b2e4-4b2e-91ee-5b8823cc4a7d	2015-02-26 02:05:36.152127
543	364	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 52\ndisease_id: 7\nsource_id: 166\nvariant_id: 105\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178952085-178952085 (A->G)\n	1	\N	\N	f3cdda57-2613-4fe5-a2a8-6a640b6a7d2b	2015-02-26 02:05:36.208388
544	365	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic\n  cells at pharmalogical concentrations.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 44\ndisease_id: 41\nsource_id: 214\nvariant_id: 106\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f24c850e-c3ba-48c1-831b-bd6fffec1eca	2015-02-26 02:05:36.314786
545	366	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 21\nsource_id: 102\nvariant_id: 107\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0434b007-4095-4864-9f8a-1bdae2f3cb83	2015-02-26 02:05:36.370511
546	367	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse\n  to PI3K-mTOR inhibitors.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 3\nsource_id: 215\nvariant_id: 108\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 10:89717672-89717672 (C->T)\n	1	\N	\N	fe6cb257-5a53-4fb1-8072-55a1e30106a6	2015-02-26 02:05:36.430756
547	368	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PTEN nonsense mutations, including R233*, have been shown to be inactivating\n  and loss-of-function, but do not have prognostic value in glioblastoma multiforme\n  patients.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 39\nsource_id: 216\nvariant_id: 108\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 10:89717672-89717672 (C->T)\n	1	\N	\N	71b5edd1-3e9d-4af5-8419-375410f3fec6	2015-02-26 02:05:36.500755
548	369	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic\n  cells at pharmalogical concentrations.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 44\ndisease_id: 41\nsource_id: 214\nvariant_id: 109\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bcfc3909-2410-4914-8f1e-b142d7e36e1f	2015-02-26 02:05:36.566231
549	370	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 64\ndisease_id: 42\nsource_id: 217\nvariant_id: 110\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 10:43609950-43609950 (C->G)\n	1	\N	\N	60aa943a-5eeb-4bac-93ef-dc8c04f06223	2015-02-26 02:05:36.63366
550	371	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong\n  repression of tyroid cancer cell growth.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 65\ndisease_id: 42\nsource_id: 218\nvariant_id: 111\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 10:43617416-43617416 (T->C)\n	1	\N	\N	8d5f9bc0-3cc9-4374-9351-8715f72f519c	2015-02-26 02:05:36.720856
551	372	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 64\ndisease_id: 42\nsource_id: 217\nvariant_id: 111\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 10:43617416-43617416 (T->C)\n	1	\N	\N	bfb8168f-53e9-4cda-b108-ffc8df153e6c	2015-02-26 02:05:36.787441
552	373	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with medullary carcinoma, the presence of RET M918T mutation is\n  associated with increased probability of lymph node metastases.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 42\nsource_id: 219\nvariant_id: 111\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 10:43617416-43617416 (T->C)\n	1	\N	\N	c1de6e04-b6c0-4719-9061-8f192f455856	2015-02-26 02:05:36.856338
553	374	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Compared to those who harbor a wild type RET, patients with RET M918T mutation\n  develop larger and more aggressive medullary thyroid cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 42\nsource_id: 220\nvariant_id: 111\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 10:43617416-43617416 (T->C)\n	1	\N	\N	62c4fc0c-a8f1-41f7-8772-3bc8d17069d7	2015-02-26 02:05:36.9158
554	375	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 29\nsource_id: 221\nvariant_id: 112\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:198267359-198267359 (C->A)\n	1	\N	\N	728cde34-0b5c-4138-a94b-d43b1a654ba4	2015-02-26 02:05:36.992424
555	376	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 29\nsource_id: 221\nvariant_id: 113\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:198266834-198266834 (T->C)\n	1	\N	\N	d750f5a6-c42a-4d00-b1eb-46298a8546fe	2015-02-26 02:05:37.05897
556	377	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin\n  than breast tumors with wild type TP53.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 66\ndisease_id: 12\nsource_id: 222\nvariant_id: 114\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7578406-7578406 (C->T)\n	1	\N	\N	d001c2d0-cf3a-4669-be00-d51475759e0f	2015-02-26 02:05:37.125515
557	378	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R175H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 12\nsource_id: 223\nvariant_id: 114\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7578406-7578406 (C->T)\n	1	\N	\N	ac1eeb23-ebbb-4a2b-b891-60e9e1610dfe	2015-02-26 02:05:37.195036
558	379	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R248Q mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 12\nsource_id: 223\nvariant_id: 115\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577538-7577538 (C->T)\n	1	\N	\N	14f1b790-b4a9-479b-9332-4c0a463a0c50	2015-02-26 02:05:37.251998
559	380	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R248 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 12\nsource_id: 224\nvariant_id: 115\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577538-7577538 (C->T)\n	1	\N	\N	9388eff6-7c8b-45b1-9ca7-7a7a916d3e45	2015-02-26 02:05:37.320123
560	381	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring R248W mutation, the prognosis is worse than\n  any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 12\nsource_id: 223\nvariant_id: 116\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577539-7577539 (G->A)\n	1	\N	\N	95c5524a-a185-4b84-9c64-148a69072161	2015-02-26 02:05:37.392391
561	382	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved regions\n  such as R249 are prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 12\nsource_id: 224\nvariant_id: 117\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577535-7577535 (C->G)\n	1	\N	\N	7d8657cd-4d4f-4181-9b21-b2f76c93efc2	2015-02-26 02:05:37.452646
562	383	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved regions\n  such as R249 are prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 12\nsource_id: 224\nvariant_id: 118\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577536-7577536 (T->A)\n	1	\N	\N	750e98da-f18b-4be2-9f87-13627077c542	2015-02-26 02:05:37.514945
563	384	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 12\nsource_id: 224\nvariant_id: 119\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577121-7577121 (G->A)\n	1	\N	\N	4a99d32e-92e0-4e6b-9d60-f74384ee7eaa	2015-02-26 02:05:37.574793
564	385	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273C mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 12\nsource_id: 223\nvariant_id: 119\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577121-7577121 (G->A)\n	1	\N	\N	3330fd7e-66a4-4ebc-8b56-596890435cdd	2015-02-26 02:05:37.635398
565	386	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 12\nsource_id: 224\nvariant_id: 120\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577120-7577120 (C->T)\n	1	\N	\N	257a1852-992f-450c-9d34-125aa8f6bea2	2015-02-26 02:05:37.738529
566	387	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 12\nsource_id: 223\nvariant_id: 120\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577120-7577120 (C->T)\n	1	\N	\N	17a78fe9-9b35-4a08-b846-0326e85a5a82	2015-02-26 02:05:37.805298
567	388	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumor with R175H mutation are more responsive to doxorubicin than breast\n  tumors with wild type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 12\nsource_id: 224\nvariant_id: 121\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	70597fbc-7412-4fb5-bb2b-4721049a89ab	2015-02-26 02:05:37.874212
568	389	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A patient with metastatic bladder cancer that responded well to the mTOR inhibitor\n  everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del,\n  p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations\n  also had mild responses to everolimus.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 67\ndisease_id: 43\nsource_id: 225\nvariant_id: 122\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3bcb0d70-190c-4089-9a01-8d7620ce4553	2015-02-26 02:05:37.93523
569	390	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice\n  with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit\n  increased overall survival, increased apoptosis, reduced proliferation and reduced\n  tumor burden compared to Tsc1 wildtype mice.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 63\ndisease_id: 4\nsource_id: 226\nvariant_id: 122\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6198f2c0-9f9e-424e-bda6-1d5e094ff07f	2015-02-26 02:05:38.009174
570	391	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 67\ndisease_id: 24\nsource_id: 227\nvariant_id: 123\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ac9d9e57-8495-4b77-879b-0c925179cad8	2015-02-26 02:05:38.080911
571	392	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 67\ndisease_id: 24\nsource_id: 225\nvariant_id: 124\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bfeef2fa-8a69-4b8c-8098-bf442cce8757	2015-02-26 02:05:38.149563
572	393	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 228\nvariant_id: 125\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	196c28b6-3154-4a16-8c18-7d531ea7ba54	2015-02-26 02:05:38.212732
573	394	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 228\nvariant_id: 125\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	82eca5bd-b941-414b-aaa6-8befb6258e8b	2015-02-26 02:05:38.26949
574	395	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 29\nsource_id: 228\nvariant_id: 125\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d2200650-ed36-4d92-8fa6-08833ee79e3c	2015-02-26 02:05:38.329085
575	396	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R\n  is prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 29\nsource_id: 229\nvariant_id: 125\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2f461306-aa8b-4c00-b475-0c4ca4ba5df3	2015-02-26 02:05:38.390029
576	397	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 228\nvariant_id: 126\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5ce7453b-8ff3-4333-b90f-39f938d13f56	2015-02-26 02:05:38.449857
577	398	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 228\nvariant_id: 126\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	70badd39-b4dd-4f7f-948b-66045759d397	2015-02-26 02:05:38.511762
578	399	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 29\nsource_id: 228\nvariant_id: 126\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e2ad4ca0-13d1-437e-b815-00fa4297bcef	2015-02-26 02:05:38.567118
579	400	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, the presence of U2AF mutations such as\n  S34Y/F are prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 29\nsource_id: 229\nvariant_id: 126\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	05d2485e-5626-4a94-ba12-3f34e2e79810	2015-02-26 02:05:38.6244
580	401	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 230\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	be926d1e-d369-42b5-9169-a638ad30c09a	2015-02-26 02:05:38.690071
581	402	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No differences in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 231\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	653cb4ed-1327-4741-ac8d-92c32368f7ff	2015-02-26 02:05:38.748448
582	403	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 231\nvariant_id: 127\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7d943d1d-f933-4c45-8b75-1602e8f14132	2015-02-26 02:05:38.805118
583	404	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 230\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d6ac008d-cfe8-4a9f-9056-3619bef38ab3	2015-02-26 02:05:38.86816
584	405	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutations  in WT1 were associated with increased risk of recurrence in young\n  patients (15-50) with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 232\nvariant_id: 127\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9e92bedb-13df-4fa9-8c6c-be2449cc3ab6	2015-02-26 02:05:38.940249
585	406	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 230\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9156ea04-2496-4742-b304-b2ef2921d7ab	2015-02-26 02:05:39.005351
586	407	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutations in WT1 were associated with a worse overall prognosis than patients\n  wildtype for WT1 in young patients (15-50), primarily because of increased risk\n  of disease recurrence\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 232\nvariant_id: 127\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	468bba68-2f12-4fc3-addb-e41065991cba	2015-02-26 02:05:39.116585
587	408	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 231\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9f568e44-e443-423c-ad02-39e2815563e6	2015-02-26 02:05:39.17444
588	409	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 233\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5e0d233e-1ac7-45a5-bb60-34a8d647f3fb	2015-02-26 02:05:39.243382
589	410	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 230\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	80df75b0-4383-4a4d-896e-a44040357f31	2015-02-26 02:05:39.301059
590	411	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 231\nvariant_id: 128\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	322dfa2f-f951-4f5e-9483-2d7779121c54	2015-02-26 02:05:39.369608
591	412	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No differences in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 231\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fb15d1cd-a2c8-4f1e-9970-629f1a0a0205	2015-02-26 02:05:39.432842
592	413	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 231\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	adefa193-a293-4368-a57b-4ffbfd789bd8	2015-02-26 02:05:39.498052
593	414	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 230\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	75e01ddf-dded-4525-b60b-e5ee2e4eacbd	2015-02-26 02:05:39.556819
594	415	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 230\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ecc8869a-6dac-43e6-9b5b-8c415175e870	2015-02-26 02:05:39.616776
595	416	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 233\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	42656bc9-156f-455e-9cd1-13b5b4b6748e	2015-02-26 02:05:39.676546
\.


ALTER TABLE audits ENABLE TRIGGER ALL;

--
-- Name: audits_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('audits_id_seq', 595, true);


--
-- Data for Name: authorizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE authorizations DISABLE TRIGGER ALL;

COPY authorizations (id, user_id, provider, uid, created_at, updated_at) FROM stdin;
\.


ALTER TABLE authorizations ENABLE TRIGGER ALL;

--
-- Name: authorizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('authorizations_id_seq', 1, false);


--
-- Data for Name: comments; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE comments DISABLE TRIGGER ALL;

COPY comments (id, title, comment, commentable_id, commentable_type, user_id, role, created_at, updated_at) FROM stdin;
\.


ALTER TABLE comments ENABLE TRIGGER ALL;

--
-- Name: comments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('comments_id_seq', 1, false);


--
-- Data for Name: data_versions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE data_versions DISABLE TRIGGER ALL;

COPY data_versions (id, version) FROM stdin;
\.


ALTER TABLE data_versions ENABLE TRIGGER ALL;

--
-- Name: data_versions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('data_versions_id_seq', 1, false);


--
-- Data for Name: definitions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE definitions DISABLE TRIGGER ALL;

COPY definitions (id, term, text, created_at, updated_at) FROM stdin;
\.


ALTER TABLE definitions ENABLE TRIGGER ALL;

--
-- Name: definitions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('definitions_id_seq', 1, false);


--
-- Data for Name: delayed_jobs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE delayed_jobs DISABLE TRIGGER ALL;

COPY delayed_jobs (id, priority, attempts, handler, last_error, run_at, locked_at, failed_at, locked_by, queue, created_at, updated_at) FROM stdin;
\.


ALTER TABLE delayed_jobs ENABLE TRIGGER ALL;

--
-- Name: delayed_jobs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('delayed_jobs_id_seq', 1, false);


--
-- Data for Name: diseases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE diseases DISABLE TRIGGER ALL;

COPY diseases (id, doid, name, created_at, updated_at) FROM stdin;
3	1612	Breast Cancer	2015-02-26 02:05:13.120983	2015-02-26 02:05:13.120983
6	769	Neuroblastoma	2015-02-26 02:05:13.53938	2015-02-26 02:05:13.53938
8	1909	Melanoma	2015-02-26 02:05:14.271238	2015-02-26 02:05:14.271238
23	4947	Cholangiocarcinoma	2015-02-26 02:05:23.449661	2015-02-26 02:05:23.449661
26	3069	Astrocytoma	2015-02-26 02:05:25.953842	2015-02-26 02:05:25.953842
36	12603	Acute Leukemia	2015-02-26 02:05:29.016246	2015-02-26 02:05:29.016246
1	9119	Acute Myeloid Leukemia	2015-02-26 02:05:12.329749	2015-02-26 02:07:42.916744
2	8552	Chronic Myeloid Leukemia	2015-02-26 02:05:12.504996	2015-02-26 02:07:43.120702
4	3908	Non-small Cell Lung Carcinoma	2015-02-26 02:05:13.182608	2015-02-26 02:07:43.918731
5	50905		2015-02-26 02:05:13.458748	2015-02-26 02:07:44.103636
7	9256	Colorectal Cancer	2015-02-26 02:05:14.20105	2015-02-26 02:07:44.592015
9	9256	Colorectal Cancer	2015-02-26 02:05:14.521272	2015-02-26 02:07:45.127264
10	1781	Thyroid Cancer	2015-02-26 02:05:15.855151	2015-02-26 02:07:45.382037
11	2394	Ovarian Cancer	2015-02-26 02:05:16.598007	2015-02-26 02:07:45.741468
12	1612	Breast Cancer	2015-02-26 02:05:16.940523	2015-02-26 02:07:46.307924
13	5520	Head And Neck Squamous Cell Carcinoma	2015-02-26 02:05:17.074161	2015-02-26 02:07:46.497541
14	50746		2015-02-26 02:05:17.140814	2015-02-26 02:07:46.671411
15	10534	Stomach Cancer	2015-02-26 02:05:17.203304	2015-02-26 02:07:47.126898
16	715	T-cell Leukemia	2015-02-26 02:05:17.536743	2015-02-26 02:07:47.343165
17	5635	Gastric Adenosquamous Carcinoma	2015-02-26 02:05:17.666281	2015-02-26 02:07:47.560159
18	1324	Lung Cancer	2015-02-26 02:05:17.793921	2015-02-26 02:07:47.910628
19	60075		2015-02-26 02:05:17.860042	2015-02-26 02:07:48.088011
20	9952	Acute Lymphocytic Leukemia	2015-02-26 02:05:18.669673	2015-02-26 02:07:48.334908
21	5015	Hepatocellular Fibrolamellar Carcinoma	2015-02-26 02:05:18.806339	2015-02-26 02:07:48.544103
22	3908	Non-small Cell Lung Carcinoma	2015-02-26 02:05:21.142429	2015-02-26 02:07:48.825707
24	4007	Bladder Carcinoma	2015-02-26 02:05:23.527889	2015-02-26 02:07:49.376039
25	3909	Bronchogenic Lung Adenocarcinoma	2015-02-26 02:05:25.57501	2015-02-26 02:07:49.580787
27	3068	Glioblastoma Multiforme	2015-02-26 02:05:26.020113	2015-02-26 02:07:50.296559
28	50908		2015-02-26 02:05:26.082824	2015-02-26 02:07:50.478796
29	50908		2015-02-26 02:05:26.563649	2015-02-26 02:07:50.656891
30	10747	Lymphoid Leukemia	2015-02-26 02:05:26.982264	2015-02-26 02:07:51.98406
31	4960	Bone Marrow Cancer	2015-02-26 02:05:27.046524	2015-02-26 02:07:52.227875
32	8997	Polycythemia Vera	2015-02-26 02:05:27.116964	2015-02-26 02:07:52.425034
33	9119	Acute Myeloid Leukemia	2015-02-26 02:05:27.237407	2015-02-26 02:07:52.651642
34	9253	Gastrointestinal Stromal Tumor	2015-02-26 02:05:27.307317	2015-02-26 02:07:52.8566
35	14145	Malignant Anus Melanoma	2015-02-26 02:05:28.122413	2015-02-26 02:07:53.048497
37	9538	Multiple Myeloma	2015-02-26 02:05:29.144571	2015-02-26 02:07:53.521501
38	9119	Acute Myeloid Leukemia	2015-02-26 02:05:29.920481	2015-02-26 02:07:53.731073
39	3068	Glioblastoma Multiforme	2015-02-26 02:05:30.610848	2015-02-26 02:07:54.021366
40	9253	Gastrointestinal Stromal Tumor	2015-02-26 02:05:34.320739	2015-02-26 02:07:54.215878
41	9119	Acute Myeloid Leukemia	2015-02-26 02:05:36.296076	2015-02-26 02:07:54.449705
42	3973	Thyroid Medullary Carcinoma	2015-02-26 02:05:36.61413	2015-02-26 02:07:54.678313
43	6477	Invasive Bladder Transitional Cell Carcinoma	2015-02-26 02:05:37.919119	2015-02-26 02:07:55.006595
\.


ALTER TABLE diseases ENABLE TRIGGER ALL;

--
-- Name: diseases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('diseases_id_seq', 43, true);


--
-- Data for Name: drugs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE drugs DISABLE TRIGGER ALL;

COPY drugs (id, name, pubchem_id, created_at, updated_at) FROM stdin;
1	Arsenic Trioxide	\N	2015-02-26 02:05:12.323678	2015-02-26 02:05:12.323678
2	Imatinib	\N	2015-02-26 02:05:12.409628	2015-02-26 02:05:12.409628
3	N/A	\N	2015-02-26 02:05:12.628433	2015-02-26 02:05:12.628433
4	Nilotinib	\N	2015-02-26 02:05:12.749123	2015-02-26 02:05:12.749123
5	Dasatinib	\N	2015-02-26 02:05:12.807169	2015-02-26 02:05:12.807169
6	MK-2206	\N	2015-02-26 02:05:13.115679	2015-02-26 02:05:13.115679
7	Crizotinib	\N	2015-02-26 02:05:13.179036	2015-02-26 02:05:13.179036
8	CH5424802	\N	2015-02-26 02:05:13.32755	2015-02-26 02:05:13.32755
9	TAE684	\N	2015-02-26 02:05:13.726172	2015-02-26 02:05:13.726172
10	Sorafenib	\N	2015-02-26 02:05:13.986421	2015-02-26 02:05:13.986421
11	Trametinib	\N	2015-02-26 02:05:14.132886	2015-02-26 02:05:14.132886
12	Panitumumab	\N	2015-02-26 02:05:14.19751	2015-02-26 02:05:14.19751
13	Dabrafenib	\N	2015-02-26 02:05:14.266105	2015-02-26 02:05:14.266105
14	Cetuximab	\N	2015-02-26 02:05:14.517229	2015-02-26 02:05:14.517229
15	PD0325901	\N	2015-02-26 02:05:14.71516	2015-02-26 02:05:14.71516
16	PLX4720	\N	2015-02-26 02:05:14.773826	2015-02-26 02:05:14.773826
17	Nutlin-3	\N	2015-02-26 02:05:14.897518	2015-02-26 02:05:14.897518
18	Capecitabine	\N	2015-02-26 02:05:14.965601	2015-02-26 02:05:14.965601
19	Vemurafenib	\N	2015-02-26 02:05:15.02522	2015-02-26 02:05:15.02522
20	Bevacizumab	\N	2015-02-26 02:05:15.108635	2015-02-26 02:05:15.108635
21	Dacarbazine	\N	2015-02-26 02:05:15.349183	2015-02-26 02:05:15.349183
22	Temozolomide	\N	2015-02-26 02:05:15.409598	2015-02-26 02:05:15.409598
23	AZD6244	\N	2015-02-26 02:05:16.311902	2015-02-26 02:05:16.311902
24	Olaparib	\N	2015-02-26 02:05:16.594593	2015-02-26 02:05:16.594593
25	Palbociclib (PD-0332991)	\N	2015-02-26 02:05:17.53302	2015-02-26 02:05:17.53302
26	BYL719	\N	2015-02-26 02:05:17.855289	2015-02-26 02:05:17.855289
27	Palbociclib	\N	2015-02-26 02:05:17.939355	2015-02-26 02:05:17.939355
28	GW-2580	\N	2015-02-26 02:05:18.717743	2015-02-26 02:05:18.717743
29	Daunorubicin	\N	2015-02-26 02:05:18.877103	2015-02-26 02:05:18.877103
30	Idarubicin	\N	2015-02-26 02:05:18.936639	2015-02-26 02:05:18.936639
31	Gefitinib	\N	2015-02-26 02:05:20.825241	2015-02-26 02:05:20.825241
32	Erlotinib	\N	2015-02-26 02:05:20.947507	2015-02-26 02:05:20.947507
33	Premetrexed	\N	2015-02-26 02:05:21.285015	2015-02-26 02:05:21.285015
34	Stauroporine	\N	2015-02-26 02:05:21.529687	2015-02-26 02:05:21.529687
35	Neratinib	\N	2015-02-26 02:05:21.598901	2015-02-26 02:05:21.598901
36	Lapatinib	\N	2015-02-26 02:05:21.885656	2015-02-26 02:05:21.885656
37	Hormone Therapy	\N	2015-02-26 02:05:22.481092	2015-02-26 02:05:22.481092
38	Tamoxifen	\N	2015-02-26 02:05:22.553878	2015-02-26 02:05:22.553878
39	Fulvestrant	\N	2015-02-26 02:05:22.606922	2015-02-26 02:05:22.606922
40	Ponatinib	\N	2015-02-26 02:05:23.445799	2015-02-26 02:05:23.445799
41	PD173074	\N	2015-02-26 02:05:23.523367	2015-02-26 02:05:23.523367
42	Pazopanib	\N	2015-02-26 02:05:23.602618	2015-02-26 02:05:23.602618
43	AG1296	\N	2015-02-26 02:05:23.755437	2015-02-26 02:05:23.755437
44	ATRA	\N	2015-02-26 02:05:23.819184	2015-02-26 02:05:23.819184
45	CEP701	\N	2015-02-26 02:05:23.881475	2015-02-26 02:05:23.881475
46	SU5614	\N	2015-02-26 02:05:24.891535	2015-02-26 02:05:24.891535
47	Bortezomib	\N	2015-02-26 02:05:25.570478	2015-02-26 02:05:25.570478
48	Fasudil	\N	2015-02-26 02:05:25.628897	2015-02-26 02:05:25.628897
49	Pegylated IFN-α–2a	\N	2015-02-26 02:05:27.113488	2015-02-26 02:05:27.113488
50	TG101348	\N	2015-02-26 02:05:27.173699	2015-02-26 02:05:27.173699
51	SU11248	\N	2015-02-26 02:05:28.692399	2015-02-26 02:05:28.692399
52	Regorafenib	\N	2015-02-26 02:05:28.820002	2015-02-26 02:05:28.820002
53	NVP-BEZ235	\N	2015-02-26 02:05:29.750303	2015-02-26 02:05:29.750303
54	ARRY-142886	\N	2015-02-26 02:05:29.800636	2015-02-26 02:05:29.800636
55	Carmustine	\N	2015-02-26 02:05:30.607163	2015-02-26 02:05:30.607163
56	O(6)-benzylguanine	\N	2015-02-26 02:05:30.750614	2015-02-26 02:05:30.750614
57	NSC348884	\N	2015-02-26 02:05:32.928445	2015-02-26 02:05:32.928445
58	Valproic Acid	\N	2015-02-26 02:05:33.050556	2015-02-26 02:05:33.050556
59	17-AAG	\N	2015-02-26 02:05:33.876722	2015-02-26 02:05:33.876722
60	Temozolomide 	\N	2015-02-26 02:05:34.201641	2015-02-26 02:05:34.201641
61	Crenolanib	\N	2015-02-26 02:05:34.317134	2015-02-26 02:05:34.317134
62	CH5132799	\N	2015-02-26 02:05:34.895694	2015-02-26 02:05:34.895694
63	Rapamycin	\N	2015-02-26 02:05:34.960044	2015-02-26 02:05:34.960044
64	Motesanib	\N	2015-02-26 02:05:36.610368	2015-02-26 02:05:36.610368
65	AZD1480	\N	2015-02-26 02:05:36.698489	2015-02-26 02:05:36.698489
66	Doxorubicin	\N	2015-02-26 02:05:37.106895	2015-02-26 02:05:37.106895
67	Everolimus	\N	2015-02-26 02:05:37.915543	2015-02-26 02:05:37.915543
\.


ALTER TABLE drugs ENABLE TRIGGER ALL;

--
-- Name: drugs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('drugs_id_seq', 67, true);


--
-- Data for Name: variant_origins; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE variant_origins DISABLE TRIGGER ALL;

COPY variant_origins (id, origin) FROM stdin;
1	Somatic
2	Germline
\.


ALTER TABLE variant_origins ENABLE TRIGGER ALL;

--
-- Data for Name: evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE evidence_items DISABLE TRIGGER ALL;

COPY evidence_items (id, text, clinical_significance, evidence_direction, evidence_type_id, evidence_level_id, drug_id, disease_id, source_id, variant_id, created_at, updated_at, rating, status, remote_errors, remote_ids, variant_origin_id, variant_hgvs) FROM stdin;
1	Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide results in high rates of complete remission in patients.	Sensitivity	Supports	1	1	1	1	51	1	2015-02-26 02:05:12.339832	2015-02-26 02:05:12.339832	5	accepted	\N	\N	1	N/A
2	BCR-ABL fusions have been found to contribute to imatinib resistance in AML cell lines.	Resistance or Non-Response	Supports	1	2	2	1	51	1	2015-02-26 02:05:12.416468	2015-02-26 02:05:12.416468	3	accepted	\N	\N	1	N/A
3	The clinical use of imatinib in patients with BCR-ABL fusion has resulted in drastic sensitivity to the drug.	Sensitivity	Supports	1	1	2	2	52	1	2015-02-26 02:05:12.507614	2015-02-26 02:05:12.507614	5	accepted	\N	\N	1	N/A
4	BCR-ABL inhibitors such as imatinib have lead to significantly improved prognosis, response rate, overall survival, and patient outcome in CML patients compared to previous therapeutic regimens.	Better Outcome	Supports	2	1	2	2	52	1	2015-02-26 02:05:12.573146	2015-02-26 02:05:12.573146	5	accepted	\N	\N	1	N/A
5	Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML.	Positive	Does Not Support	3	3	3	2	53	1	2015-02-26 02:05:12.635812	2015-02-26 02:05:12.635812	2	accepted	\N	\N	1	N/A
6	The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease.	Positive	Supports	3	1	3	2	52	1	2015-02-26 02:05:12.691613	2015-02-26 02:05:12.691613	4	accepted	\N	\N	1	N/A
7	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.	Sensitivity	Supports	1	2	4	2	52	1	2015-02-26 02:05:12.751815	2015-02-26 02:05:12.751815	4	accepted	\N	\N	1	N/A
8	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.	Sensitivity	Supports	1	2	5	2	52	1	2015-02-26 02:05:12.811759	2015-02-26 02:05:12.811759	4	accepted	\N	\N	1	N/A
9	COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	Resistance or Non-Response	Supports	1	3	2	2	54	3	2015-02-26 02:05:12.87164	2015-02-26 02:05:12.87164	4	accepted	\N	\N	1	N/A
10	In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have shown to be resistant to imatinib treatment.	Resistance or Non-Response	Supports	1	2	2	2	52	2	2015-02-26 02:05:12.965046	2015-02-26 02:05:12.965046	4	accepted	\N	\N	1	N/A
11	COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	Resistance or Non-Response	Supports	1	3	2	2	54	2	2015-02-26 02:05:13.05519	2015-02-26 02:05:13.05519	4	accepted	\N	\N	1	N/A
12	PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the AKT inhibitor MK-2206 in breast cancer cell lines.	Sensitivity	Does Not Support	1	2	6	3	55	4	2015-02-26 02:05:13.123176	2015-02-26 02:05:13.123176	3	accepted	\N	\N	1	14:105246551-105246551 (C->T)
13	In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment with crizotinib has shown to be effective in acheiving stable disease.	Sensitivity	Supports	1	2	7	4	56	5	2015-02-26 02:05:13.184936	2015-02-26 02:05:13.184936	4	accepted	\N	\N	1	N/A
14	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y variant has been shown to confer resistance to crizotinib.	Resistance or Non-Response	Supports	1	2	7	4	56	6	2015-02-26 02:05:13.253572	2015-02-26 02:05:13.253572	4	accepted	\N	\N	1	N/A
15	CH5424802 treatment resulted in significant tumor regression in xenograft models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation	Sensitivity	Supports	1	3	8	4	57	7	2015-02-26 02:05:13.333521	2015-02-26 02:05:13.333521	3	accepted	\N	\N	1	N/A
16	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M variant has been shown to confer resistance to crizotinib.	Resistance or Non-Response	Supports	1	2	7	4	56	7	2015-02-26 02:05:13.38883	2015-02-26 02:05:13.38883	4	accepted	\N	\N	1	N/A
17	Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without this mutation.	Resistance or Non-Response	Supports	1	3	7	5	58	8	2015-02-26 02:05:13.461012	2015-02-26 02:05:13.461012	3	accepted	\N	\N	1	2:29443695-29443695 (G->T)
18	CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.	Sensitivity	Supports	1	3	8	6	57	8	2015-02-26 02:05:13.542035	2015-02-26 02:05:13.542035	3	accepted	\N	\N	1	2:29443695-29443695 (G->T)
19	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation.	Resistance or Non-Response	Supports	1	3	7	6	59	8	2015-02-26 02:05:13.598524	2015-02-26 02:05:13.598524	3	accepted	\N	\N	1	2:29443695-29443695 (G->T)
20	High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing the EML4-ALK fusion containing the F1174L mutation.	Sensitivity	Supports	1	3	7	6	60	8	2015-02-26 02:05:13.667018	2015-02-26 02:05:13.667018	3	accepted	\N	\N	1	2:29443695-29443695 (G->T)
21	SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition	Sensitivity	Supports	1	3	9	6	61	8	2015-02-26 02:05:13.730745	2015-02-26 02:05:13.730745	4	accepted	\N	\N	1	2:29443695-29443695 (G->T)
22	Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without this mutation.	Resistance or Non-Response	Supports	1	3	7	4	58	8	2015-02-26 02:05:13.797827	2015-02-26 02:05:13.797827	3	accepted	\N	\N	1	2:29443695-29443695 (G->T)
23	TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.	Sensitivity	Supports	1	3	9	6	61	9	2015-02-26 02:05:13.859358	2015-02-26 02:05:13.859358	3	accepted	\N	\N	1	2:29432664-29432664 (C->T)
24	The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.	Resistance or Non-Response	Supports	1	3	9	6	61	9	2015-02-26 02:05:13.919261	2015-02-26 02:05:13.919261	3	accepted	\N	\N	1	2:29432664-29432664 (C->T)
25	Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	Sensitivity	Supports	1	3	10	4	62	10	2015-02-26 02:05:13.992519	2015-02-26 02:05:13.992519	3	accepted	\N	\N	1	X:47426121-47426121 (C->G)
26	In one patient with S214C mutation, the use of sorafenib has led to more than 5 years of survival and near remission.	Sensitivity	Supports	1	2	10	4	62	10	2015-02-26 02:05:14.076489	2015-02-26 02:05:14.076489	2	accepted	\N	\N	1	X:47426121-47426121 (C->G)
27	Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	Sensitivity	Supports	1	3	11	4	62	10	2015-02-26 02:05:14.137676	2015-02-26 02:05:14.137676	3	accepted	\N	\N	1	X:47426121-47426121 (C->G)
83	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	Poor Outcome	Supports	2	2	3	18	94	24	2015-02-26 02:05:17.796324	2015-02-26 02:05:17.796324	4	accepted	\N	\N	1	N/A
28	In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations were associated with poor progression free survival regardless of treatment (panitumumab with best supportive care or best supportive care alone)	Resistance or Non-Response	Supports	1	2	12	7	63	17	2015-02-26 02:05:14.203283	2015-02-26 02:05:14.203283	3	accepted	\N	\N	1	N/A
29	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	Sensitivity	Supports	1	2	13	8	64	17	2015-02-26 02:05:14.273918	2015-02-26 02:05:14.273918	3	accepted	\N	\N	1	N/A
30	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	Sensitivity	Supports	1	2	11	8	64	17	2015-02-26 02:05:14.325592	2015-02-26 02:05:14.325592	3	accepted	\N	\N	1	N/A
31	BRAF mutations are associated with melanoma arising in non-chronic sun damaged skin and with superficial spreading melanoma	Positive	Supports	3	2	3	8	65	17	2015-02-26 02:05:14.392843	2015-02-26 02:05:14.392843	4	accepted	\N	\N	1	N/A
32	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	1	2	13	8	66	11	2015-02-26 02:05:14.450314	2015-02-26 02:05:14.450314	5	accepted	\N	\N	1	7:140453135-140453136 (CA->AT)
33	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	1	3	14	9	67	12	2015-02-26 02:05:14.523559	2015-02-26 02:05:14.523559	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
34	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	1	3	12	9	67	12	2015-02-26 02:05:14.601441	2015-02-26 02:05:14.601441	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
35	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	1	3	10	9	67	12	2015-02-26 02:05:14.655096	2015-02-26 02:05:14.655096	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
36	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	Sensitivity	Supports	1	3	15	9	68	12	2015-02-26 02:05:14.720382	2015-02-26 02:05:14.720382	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
37	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	Sensitivity	Supports	1	3	16	9	68	12	2015-02-26 02:05:14.776611	2015-02-26 02:05:14.776611	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
38	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	Sensitivity	Supports	1	3	16	9	69	12	2015-02-26 02:05:14.846565	2015-02-26 02:05:14.846565	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
39	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	Sensitivity	Supports	1	3	17	9	69	12	2015-02-26 02:05:14.90023	2015-02-26 02:05:14.90023	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
40	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	1	3	18	7	70	12	2015-02-26 02:05:14.970009	2015-02-26 02:05:14.970009	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
41	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	1	3	19	7	70	12	2015-02-26 02:05:15.028126	2015-02-26 02:05:15.028126	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
42	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	1	3	20	7	70	12	2015-02-26 02:05:15.111789	2015-02-26 02:05:15.111789	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
43	Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype BRAF	Resistance or Non-Response	Supports	1	2	14	7	71	12	2015-02-26 02:05:15.172413	2015-02-26 02:05:15.172413	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
44	V600E is associated with adverse pathological features of colorectal cancer. This can be concluded as a marker of poor prognosis.	Poor Outcome	Supports	2	1	3	7	72	12	2015-02-26 02:05:15.231832	2015-02-26 02:05:15.231832	5	accepted	\N	\N	1	7:140453136-140453136 (A->T)
45	Cell lines expressing the BRAF V600E mutation responded better to vemurafenib treatment than cells wildtype for BRAF as measured by reduced cellular proliferation and inhibition of MET and ERK phosphorylation	Sensitivity	Supports	1	3	19	7	70	12	2015-02-26 02:05:15.287277	2015-02-26 02:05:15.287277	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
46	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	N / A	Does Not Support	1	2	21	8	73	12	2015-02-26 02:05:15.357143	2015-02-26 02:05:15.357143	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
47	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	N / A	Does Not Support	1	2	22	8	73	12	2015-02-26 02:05:15.412381	2015-02-26 02:05:15.412381	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
48	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	Sensitivity	Supports	1	2	13	8	74	12	2015-02-26 02:05:15.472573	2015-02-26 02:05:15.472573	5	accepted	\N	\N	1	7:140453136-140453136 (A->T)
49	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	Sensitivity	Supports	1	2	11	8	74	12	2015-02-26 02:05:15.526913	2015-02-26 02:05:15.526913	5	accepted	\N	\N	1	7:140453136-140453136 (A->T)
50	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP but does not show resistance to MEK inhibitors.	Resistance or Non-Response	Does Not Support	1	3	3	8	75	12	2015-02-26 02:05:15.609485	2015-02-26 02:05:15.609485	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
51	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP and resistance to RAF inhibitors.	Resistance or Non-Response	Supports	1	3	3	8	75	12	2015-02-26 02:05:15.668327	2015-02-26 02:05:15.668327	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
52	BRAF V600E is correlated with shorter disease-free and overall Survival in a Spanish cohort of melanoma patients.	Poor Outcome	Supports	2	2	3	8	76	12	2015-02-26 02:05:15.727582	2015-02-26 02:05:15.727582	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
53	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	Resistance or Non-Response	Supports	1	2	13	4	77	12	2015-02-26 02:05:15.793881	2015-02-26 02:05:15.793881	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
54	BRAF V600E is shown to be associated with the tall-cell variant of papillary thyroid cancer	Positive	Supports	3	2	3	10	78	12	2015-02-26 02:05:15.857886	2015-02-26 02:05:15.857886	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
55	Thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	Positive	Supports	3	2	3	10	79	12	2015-02-26 02:05:15.924381	2015-02-26 02:05:15.924381	5	accepted	\N	\N	1	7:140453136-140453136 (A->T)
56	Thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	Positive	Supports	3	2	3	10	80	12	2015-02-26 02:05:15.98582	2015-02-26 02:05:15.98582	5	accepted	\N	\N	1	7:140453136-140453136 (A->T)
57	Unlike other studies that suggest a poorer outcome, BRAF mutation in this study was not correlated with poorer prognosis in papillary thyroid cancer.	Poor Outcome	Does Not Support	2	2	3	10	81	12	2015-02-26 02:05:16.067627	2015-02-26 02:05:16.067627	5	accepted	\N	\N	1	7:140453136-140453136 (A->T)
58	BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in a study of 187 patients with PTC and other thyroid diseases.	Poor Outcome	Supports	2	2	3	10	82	12	2015-02-26 02:05:16.137351	2015-02-26 02:05:16.137351	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
59	BRAF mutation correlated with poor prognosis in papillary thyroid cancer in both older (>65 yo) and younger (<65 yo) cohorts.	Poor Outcome	Supports	2	2	3	10	78	12	2015-02-26 02:05:16.193004	2015-02-26 02:05:16.193004	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
60	V600E is correlated with disease recurrence in both age cohorts (>65 and <65 yo).	Poor Outcome	Supports	2	2	3	10	78	12	2015-02-26 02:05:16.254194	2015-02-26 02:05:16.254194	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
61	COLO201 and COLO206F cells harboring BRAF V600E mutations were cloned to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification of the BRAF V600E gene.	Resistance or Non-Response	Supports	1	4	23	7	83	14	2015-02-26 02:05:16.320321	2015-02-26 02:05:16.320321	4	accepted	\N	\N	1	N/A
62	A single patient with BRAF V600E/V600M bi-allelic mutation responded to the V600E drug dabrafenib.	Sensitivity	Supports	1	2	13	8	84	13	2015-02-26 02:05:16.384805	2015-02-26 02:05:16.384805	3	accepted	\N	\N	1	N/A
63	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	1	2	13	8	66	15	2015-02-26 02:05:16.444634	2015-02-26 02:05:16.444634	5	accepted	\N	\N	1	7:140453137-140453137 (C->T)
64	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	1	2	13	8	66	16	2015-02-26 02:05:16.529382	2015-02-26 02:05:16.529382	5	accepted	\N	\N	1	7:140453136-140453137 (AC->CT)
65	The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation positive women with high grade serous or undifferentiated ovarian cancer.	Sensitivity	Supports	1	2	24	11	85	129	2015-02-26 02:05:16.60137	2015-02-26 02:05:16.60137	4	accepted	\N	\N	2	N/A
66	The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation positive women with high grade serous or undifferentiated ovarian cancer.	Sensitivity	Supports	1	2	24	11	85	130	2015-02-26 02:05:16.67554	2015-02-26 02:05:16.67554	4	accepted	\N	\N	2	N/A
67	Inceased copy number of CCND1 is associated with poorer overall survival.	Poor Outcome	Supports	2	2	3	4	86	18	2015-02-26 02:05:16.749277	2015-02-26 02:05:16.749277	3	accepted	\N	\N	1	N/A
68	Three studies have found no significant survival impact for CCND1 in lung cancer.	Poor Outcome	Does Not Support	2	2	3	4	86	19	2015-02-26 02:05:16.811694	2015-02-26 02:05:16.811694	3	accepted	\N	\N	1	N/A
69	Three studies have associated CCND1 expression with poorer survival.	Poor Outcome	Supports	2	2	3	4	86	19	2015-02-26 02:05:16.869496	2015-02-26 02:05:16.869496	3	accepted	\N	\N	1	N/A
70	Cyclin D1 overexpression is associated with the ER-positive subtype of breast cancer.	Positive	Supports	3	2	3	12	87	20	2015-02-26 02:05:16.942854	2015-02-26 02:05:16.942854	4	accepted	\N	\N	1	N/A
71	In ER-positive breast cancer patients, Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis.	Poor Outcome	Supports	2	2	3	12	87	20	2015-02-26 02:05:17.018054	2015-02-26 02:05:17.018054	4	accepted	\N	\N	1	N/A
72	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in head and neck squamous cell carcinoma.	Poor Outcome	Supports	2	2	3	13	88	20	2015-02-26 02:05:17.076384	2015-02-26 02:05:17.076384	4	accepted	\N	\N	1	N/A
73	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in mantle cell lymphoma.	Poor Outcome	Supports	2	2	3	14	89	20	2015-02-26 02:05:17.143191	2015-02-26 02:05:17.143191	4	accepted	\N	\N	1	N/A
74	Cyclin D2 overexpression is associated with the presence pf lymph node metastasis in gastric cancers.	Poor Outcome	Supports	2	3	3	15	90	21	2015-02-26 02:05:17.207927	2015-02-26 02:05:17.207927	4	accepted	\N	\N	1	N/A
75	Cyclin D2 overexpression is associated with the increased depth of cancer invasion in gastric cancers.	Poor Outcome	Supports	2	3	3	15	90	21	2015-02-26 02:05:17.269052	2015-02-26 02:05:17.269052	4	accepted	\N	\N	1	N/A
76	Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.	Poor Outcome	Supports	2	3	3	15	90	21	2015-02-26 02:05:17.328463	2015-02-26 02:05:17.328463	4	accepted	\N	\N	1	N/A
77	Cyclin D2 overexpression is associated with vascular invasion by cancer cells in gastric cancers.	Poor Outcome	Supports	2	3	3	15	90	21	2015-02-26 02:05:17.383408	2015-02-26 02:05:17.383408	4	accepted	\N	\N	1	N/A
78	CCND2 hypomethylation is seen to be more common in stage III and IV gastric tumors than stage I and II.	Negative	Supports	3	3	3	15	91	22	2015-02-26 02:05:17.472616	2015-02-26 02:05:17.472616	3	accepted	\N	\N	1	N/A
79	Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib (PD-0332991) significantly increased median survival.	Sensitivity	Supports	1	3	25	16	92	23	2015-02-26 02:05:17.539361	2015-02-26 02:05:17.539361	3	accepted	\N	\N	1	N/A
80	Patients with high expression of total cyclin E and low-molecular weight cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard ratio for these patients in 13.3 times higher than those with normal cyclin E levels.	Poor Outcome	Supports	2	2	3	12	93	24	2015-02-26 02:05:17.604137	2015-02-26 02:05:17.604137	4	accepted	\N	\N	1	N/A
81	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	Poor Outcome	Supports	2	2	3	17	94	24	2015-02-26 02:05:17.668537	2015-02-26 02:05:17.668537	4	accepted	\N	\N	1	N/A
82	Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced stage gastric carcinoma.	Poor Outcome	Supports	2	2	3	17	94	24	2015-02-26 02:05:17.723522	2015-02-26 02:05:17.723522	4	accepted	\N	\N	1	N/A
84	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	Sensitivity	Supports	1	3	26	19	95	25	2015-02-26 02:05:17.86308	2015-02-26 02:05:17.86308	4	accepted	\N	\N	1	N/A
85	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	Better Outcome	Supports	2	2	27	19	96	25	2015-02-26 02:05:17.943944	2015-02-26 02:05:17.943944	3	accepted	\N	\N	1	N/A
86	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	Sensitivity	Supports	1	3	26	19	95	26	2015-02-26 02:05:17.997887	2015-02-26 02:05:17.997887	4	accepted	\N	\N	1	N/A
87	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	Better Outcome	Supports	2	2	27	19	96	26	2015-02-26 02:05:18.064162	2015-02-26 02:05:18.064162	3	accepted	\N	\N	1	N/A
88	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter time to recurrence.	Poor Outcome	Supports	2	2	3	4	97	27	2015-02-26 02:05:18.128045	2015-02-26 02:05:18.128045	3	accepted	\N	\N	1	N/A
89	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter overall survival.	Poor Outcome	Supports	2	2	3	4	97	27	2015-02-26 02:05:18.189182	2015-02-26 02:05:18.189182	3	accepted	\N	\N	1	N/A
90	CEBPA mutation status had no impact on ATRA treatment response in older (>60) patients with AML	N / A	Does Not Support	1	2	3	1	98	29	2015-02-26 02:05:18.253122	2015-02-26 02:05:18.253122	4	accepted	\N	\N	1	N/A
91	CEBPA mutation was significantly associated with complete remission	Better Outcome	Supports	2	2	3	1	99	29	2015-02-26 02:05:18.315495	2015-02-26 02:05:18.315495	4	accepted	\N	\N	1	N/A
92	CEBPA mutation was associated with improved overall survival in older (>60) patients with AML	Better Outcome	Supports	2	2	3	1	98	29	2015-02-26 02:05:18.393982	2015-02-26 02:05:18.393982	4	accepted	\N	\N	1	N/A
93	FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically normal AML patients with CEBPA	N / A	Does Not Support	2	2	3	1	100	28	2015-02-26 02:05:18.451852	2015-02-26 02:05:18.451852	2	accepted	\N	\N	1	N/A
94	Overall survival was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA mutations	Better Outcome	Supports	2	2	3	1	100	28	2015-02-26 02:05:18.529843	2015-02-26 02:05:18.529843	3	accepted	\N	\N	1	N/A
95	Remission duration was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA	Better Outcome	Supports	2	2	3	1	100	28	2015-02-26 02:05:18.596643	2015-02-26 02:05:18.596643	3	accepted	\N	\N	1	N/A
96	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	3	2	20	101	30	2015-02-26 02:05:18.672055	2015-02-26 02:05:18.672055	3	accepted	\N	\N	1	N/A
97	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	3	28	20	101	30	2015-02-26 02:05:18.720611	2015-02-26 02:05:18.720611	3	accepted	\N	\N	1	N/A
98	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	Positive	Supports	3	2	3	21	102	31	2015-02-26 02:05:18.810348	2015-02-26 02:05:18.810348	4	accepted	\N	\N	1	N/A
99	Daunorubicin treatment resulted in similar overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	Resistance or Non-Response	Does Not Support	1	2	29	1	103	32	2015-02-26 02:05:18.881579	2015-02-26 02:05:18.881579	4	accepted	\N	\N	1	N/A
100	Idarubicin increases the overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	Sensitivity	Supports	1	2	30	1	103	32	2015-02-26 02:05:18.941302	2015-02-26 02:05:18.941302	4	accepted	\N	\N	1	N/A
101	Therapy-related AML was less common in patients with DNMT3A mutations (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of younger (18-60) AML patients	Negative	Supports	3	2	3	1	104	32	2015-02-26 02:05:19.000539	2015-02-26 02:05:19.000539	3	accepted	\N	\N	1	N/A
102	Young AML patients (<60 years old) with DNMT3A mutations (60% of which were R882) were older in age, had higher white blood cell counts and had higher platelet counts than patients wildtype for DNMT3A	Positive	Supports	3	2	3	1	105	32	2015-02-26 02:05:19.067334	2015-02-26 02:05:19.067334	3	accepted	\N	\N	1	N/A
103	DNMT3A R882 mutations occur most often in de novo AML patients with intermediate cytogenic risk.	Positive	Supports	3	2	3	1	103	32	2015-02-26 02:05:19.125748	2015-02-26 02:05:19.125748	3	accepted	\N	\N	1	N/A
104	DNMT3A R882 mutations were associated with older age, higher white blood cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically normal AML patients	Positive	Supports	3	2	3	1	106	32	2015-02-26 02:05:19.192207	2015-02-26 02:05:19.192207	3	accepted	\N	\N	1	N/A
105	DNMT3A mutations (59% of which were R882) were associated with an intermediate risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes	Positive	Supports	3	2	3	1	107	32	2015-02-26 02:05:19.27601	2015-02-26 02:05:19.27601	5	accepted	\N	\N	1	N/A
106	DNMT3A mutations (59% of which were R882) were associated with intermediate risk cytogenetics (including normal karyotype)	Positive	Supports	3	2	3	1	107	32	2015-02-26 02:05:19.330536	2015-02-26 02:05:19.330536	4	accepted	\N	\N	1	N/A
107	DNMT3A mutations (64.5% R882) were associated with older age, higher white blood cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML patients	Positive	Supports	3	2	3	1	104	32	2015-02-26 02:05:19.391276	2015-02-26 02:05:19.391276	3	accepted	\N	\N	1	N/A
108	DNMT3A R882 mutations were associated with cytogenetically normal AML in a large cohort of younger (18-60) AML patients	Positive	Supports	3	2	3	1	104	32	2015-02-26 02:05:19.450483	2015-02-26 02:05:19.450483	3	accepted	\N	\N	1	N/A
109	In a large cohort of cytogenetically normal AML patients (18-60 years old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall, relapse free and event free survival	N / A	Does Not Support	2	2	3	1	104	32	2015-02-26 02:05:19.519149	2015-02-26 02:05:19.519149	3	accepted	\N	\N	1	N/A
110	There is no difference in the complete remission rate of de novo AML patients with DNMT3A mutation compared to those who are wild type for DNMT3A.	N / A	Does Not Support	2	2	3	1	103	32	2015-02-26 02:05:19.5779	2015-02-26 02:05:19.5779	4	accepted	\N	\N	1	N/A
111	In young AML patients (<60 years old), DNMT3A mutation status (60% of which were R882) was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N / A	Does Not Support	2	2	3	1	105	32	2015-02-26 02:05:19.717072	2015-02-26 02:05:19.717072	4	accepted	\N	\N	1	N/A
112	In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5% of which were R882) had no prognostic value on overall, relapse free and event free survival.	N / A	Does Not Support	2	2	3	1	104	32	2015-02-26 02:05:19.779162	2015-02-26 02:05:19.779162	3	accepted	\N	\N	1	N/A
113	Complete remission rates did not differ between patients with wildtype or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML	N / A	Does Not Support	2	2	3	1	108	32	2015-02-26 02:05:19.843619	2015-02-26 02:05:19.843619	4	accepted	\N	\N	1	N/A
114	DNMT3A mutations were associated with achievement of complete remission in a large cohort of younger (18-60) AML patients	Better Outcome	Supports	2	2	3	1	104	32	2015-02-26 02:05:19.908592	2015-02-26 02:05:19.908592	3	accepted	\N	\N	1	N/A
115	AML patients with DNMT3A mutations (59% of which were R882) showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	Poor Outcome	Supports	2	2	3	1	107	32	2015-02-26 02:05:19.977418	2015-02-26 02:05:19.977418	5	accepted	\N	\N	1	N/A
116	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including R882) were associated with worse overall survival compared to those without DNMT3A mutation.	Poor Outcome	Supports	2	2	3	1	107	32	2015-02-26 02:05:20.040486	2015-02-26 02:05:20.040486	5	accepted	\N	\N	1	N/A
117	De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	Poor Outcome	Supports	2	2	3	1	107	32	2015-02-26 02:05:20.104576	2015-02-26 02:05:20.104576	4	accepted	\N	\N	1	N/A
118	In cytogenetically normal AML patients, DNMT3A R882 mutations are associated with lower overall and disease free survival as compared to patients with wild type DNMT3A.	Poor Outcome	Supports	2	2	3	1	106	32	2015-02-26 02:05:20.17497	2015-02-26 02:05:20.17497	5	accepted	\N	\N	1	N/A
119	In young AML patients (<60 years old), DNMT3A mutations were associated with significantly reduced overall survival and relapse free survival in patients wildtype for NPM1 and FLT3	Poor Outcome	Supports	2	2	3	1	105	32	2015-02-26 02:05:20.263017	2015-02-26 02:05:20.263017	4	accepted	\N	\N	1	N/A
120	In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation is prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	Poor Outcome	Supports	2	2	3	1	108	32	2015-02-26 02:05:20.323368	2015-02-26 02:05:20.323368	4	accepted	\N	\N	1	N/A
121	In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations other than R882 are prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	Poor Outcome	Supports	2	2	3	1	108	32	2015-02-26 02:05:20.389641	2015-02-26 02:05:20.389641	4	accepted	\N	\N	1	N/A
122	DNMT3A R882 mutation was associated with reduced relapse free and overall survival in ELN-unfavorable, cytogenetically normal AMLs	Poor Outcome	Supports	2	2	3	1	104	32	2015-02-26 02:05:20.444416	2015-02-26 02:05:20.444416	3	accepted	\N	\N	1	N/A
123	In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations were associated with reduced relapse free survival in the entire cohort as well as the subset of patients with cytogenetically normal AML	Poor Outcome	Supports	2	1	3	1	104	32	2015-02-26 02:05:20.501741	2015-02-26 02:05:20.501741	3	accepted	\N	\N	1	N/A
124	DNMT3A mutations (62% of which were R882) were associated with reduced disease-free survival in patients with cytogenetically normal AML.	Poor Outcome	Supports	2	2	3	1	108	32	2015-02-26 02:05:20.556586	2015-02-26 02:05:20.556586	4	accepted	\N	\N	1	N/A
125	Young AML patients (<60 years old) with DNMT3A mutation have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	Poor Outcome	Supports	2	2	3	1	105	32	2015-02-26 02:05:20.616912	2015-02-26 02:05:20.616912	3	accepted	\N	\N	1	N/A
126	Complete remission rate was not different between young AML patients (<60 years old) with or without DNMT3A mutations (60% of which were R882)	Poor Outcome	Supports	2	2	3	1	105	32	2015-02-26 02:05:20.671352	2015-02-26 02:05:20.671352	3	accepted	\N	\N	1	N/A
127	Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	Poor Outcome	Supports	2	2	3	1	105	32	2015-02-26 02:05:20.755112	2015-02-26 02:05:20.755112	3	accepted	\N	\N	1	N/A
128	Geftinib added to pemetrexed and carboplatin showed greater effectiveness than premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.	Sensitivity	Supports	1	2	31	4	109	33	2015-02-26 02:05:20.829582	2015-02-26 02:05:20.829582	3	accepted	\N	\N	1	7:55259514-55259515 (CT->AG)
129	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.	N / A	Does Not Support	1	2	31	4	110	33	2015-02-26 02:05:20.891398	2015-02-26 02:05:20.891398	3	accepted	\N	\N	1	7:55259514-55259515 (CT->AG)
130	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.	N / A	Does Not Support	1	2	32	4	110	33	2015-02-26 02:05:20.950604	2015-02-26 02:05:20.950604	3	accepted	\N	\N	1	7:55259514-55259515 (CT->AG)
131	In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence of L858R mutation is prognostic for better progression free survival.	Sensitivity	Supports	1	2	3	4	111	33	2015-02-26 02:05:21.015642	2015-02-26 02:05:21.015642	4	accepted	\N	\N	1	7:55259514-55259515 (CT->AG)
132	Median survival of patients with EGFR L858R mutation is better than those with wild type EGFR.	Better Outcome	Supports	2	2	3	4	112	33	2015-02-26 02:05:21.079877	2015-02-26 02:05:21.079877	3	accepted	\N	\N	1	7:55259514-55259515 (CT->AG)
133	The T790M mutation in EGFR has been shown to confer resistance to the tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are placed on the drug are likely to relapse.	Resistance or Non-Response	Supports	1	1	32	22	113	34	2015-02-26 02:05:21.144717	2015-02-26 02:05:21.144717	5	accepted	\N	\N	1	7:55249071-55249071 (C->T)
134	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.	Sensitivity	Supports	1	2	32	4	114	34	2015-02-26 02:05:21.214431	2015-02-26 02:05:21.214431	3	accepted	\N	\N	1	7:55249071-55249071 (C->T)
135	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.	Sensitivity	Supports	1	2	33	4	114	34	2015-02-26 02:05:21.287881	2015-02-26 02:05:21.287881	3	accepted	\N	\N	1	7:55249071-55249071 (C->T)
136	In non-small cell lung cancer, the appearance of T790M mutation leads to resistance to gefitinib.	Resistance or Non-Response	Supports	1	2	31	4	115	34	2015-02-26 02:05:21.34714	2015-02-26 02:05:21.34714	3	accepted	\N	\N	1	7:55249071-55249071 (C->T)
137	Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall survival compared to patients with L858R or other exon 19 activating mutations.	Poor Outcome	Supports	2	2	3	4	116	34	2015-02-26 02:05:21.409968	2015-02-26 02:05:21.409968	4	accepted	\N	\N	1	7:55249071-55249071 (C->T)
193	WBC counts were not significantly different in AML patients with FLT3 D835 mutations compared to wild type	Positive	Does Not Support	3	2	3	1	128	55	2015-02-26 02:05:25.107303	2015-02-26 02:05:25.107303	3	accepted	\N	\N	1	N/A
138	In NSCLC patients with T790M and another activating mutations, their progression free survival is shorter compared to those who do not possess T790M mutation.	Poor Outcome	Supports	2	2	3	4	117	34	2015-02-26 02:05:21.468716	2015-02-26 02:05:21.468716	4	accepted	\N	\N	1	7:55249071-55249071 (C->T)
139	In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR, especially when an L858R mutation is also present.	Sensitivity	Supports	1	4	34	4	118	34	2015-02-26 02:05:21.536223	2015-02-26 02:05:21.536223	1	accepted	\N	\N	1	7:55249071-55249071 (C->T)
140	In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	35	12	119	35	2015-02-26 02:05:21.603886	2015-02-26 02:05:21.603886	4	accepted	\N	\N	1	17:37880261-37880261 (G->C)
141	In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	35	12	119	36	2015-02-26 02:05:21.671363	2015-02-26 02:05:21.671363	4	accepted	\N	\N	1	17:37880261-37880261 (G->T)
142	In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2, cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	35	12	119	37	2015-02-26 02:05:21.738047	2015-02-26 02:05:21.738047	5	accepted	\N	\N	1	N/A
143	In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	35	12	119	38	2015-02-26 02:05:21.826181	2015-02-26 02:05:21.826181	5	accepted	\N	\N	1	17:37868205-37868205 (G->C)
144	The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	Resistance or Non-Response	Supports	1	3	36	12	119	39	2015-02-26 02:05:21.888471	2015-02-26 02:05:21.888471	5	accepted	\N	\N	1	17:37880220-37880220 (T->C)
145	The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	Sensitivity	Supports	1	3	35	12	119	39	2015-02-26 02:05:21.948397	2015-02-26 02:05:21.948397	5	accepted	\N	\N	1	17:37880220-37880220 (T->C)
146	MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	Sensitivity	Supports	1	3	35	12	119	40	2015-02-26 02:05:22.01441	2015-02-26 02:05:22.01441	5	accepted	\N	\N	1	N/A
147	In MCF10A cell lines, the in-frame insertion of a single amino acid at position 780 was shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	35	12	119	41	2015-02-26 02:05:22.079995	2015-02-26 02:05:22.079995	5	accepted	\N	\N	1	N/A
148	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	Sensitivity	Supports	1	3	35	12	119	42	2015-02-26 02:05:22.151443	2015-02-26 02:05:22.151443	5	accepted	\N	\N	1	N/A
149	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	Sensitivity	Supports	1	3	36	12	119	42	2015-02-26 02:05:22.204525	2015-02-26 02:05:22.204525	5	accepted	\N	\N	1	N/A
150	In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	35	12	119	43	2015-02-26 02:05:22.278439	2015-02-26 02:05:22.278439	5	accepted	\N	\N	1	17:37881616-37881616 (C->T)
151	In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	35	12	119	44	2015-02-26 02:05:22.344386	2015-02-26 02:05:22.344386	5	accepted	\N	\N	1	17:37881000-37881000 (G->T)
152	In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	35	12	119	45	2015-02-26 02:05:22.4193	2015-02-26 02:05:22.4193	5	accepted	\N	\N	1	17:37881332-37881332 (G->A)
153	MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	1	3	37	12	120	46	2015-02-26 02:05:22.485538	2015-02-26 02:05:22.485538	3	accepted	\N	\N	1	N/A
154	The L536Q ligand binding domain mutation have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.	Sensitivity	Supports	1	3	38	12	121	46	2015-02-26 02:05:22.558264	2015-02-26 02:05:22.558264	5	accepted	\N	\N	1	N/A
155	The L536Q ligand binding domain mutation have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.	Sensitivity	Supports	1	3	39	12	121	46	2015-02-26 02:05:22.6098	2015-02-26 02:05:22.6098	5	accepted	\N	\N	1	N/A
156	MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	1	3	37	12	120	47	2015-02-26 02:05:22.679245	2015-02-26 02:05:22.679245	3	accepted	\N	\N	1	N/A
157	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	38	12	121	47	2015-02-26 02:05:22.76091	2015-02-26 02:05:22.76091	5	accepted	\N	\N	1	N/A
158	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	39	12	121	47	2015-02-26 02:05:22.81225	2015-02-26 02:05:22.81225	5	accepted	\N	\N	1	N/A
159	MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	1	3	37	12	120	48	2015-02-26 02:05:22.872998	2015-02-26 02:05:22.872998	3	accepted	\N	\N	1	6:152419923-152419923 (A->G)
160	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	38	12	121	48	2015-02-26 02:05:22.937555	2015-02-26 02:05:22.937555	5	accepted	\N	\N	1	6:152419923-152419923 (A->G)
161	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	39	12	121	48	2015-02-26 02:05:22.990908	2015-02-26 02:05:22.990908	5	accepted	\N	\N	1	6:152419923-152419923 (A->G)
162	MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	1	3	37	12	120	49	2015-02-26 02:05:23.057707	2015-02-26 02:05:23.057707	3	accepted	\N	\N	1	6:152419922-152419922 (T->A)
163	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	38	12	121	49	2015-02-26 02:05:23.127662	2015-02-26 02:05:23.127662	5	accepted	\N	\N	1	6:152419922-152419922 (T->A)
164	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	39	12	121	49	2015-02-26 02:05:23.181975	2015-02-26 02:05:23.181975	5	accepted	\N	\N	1	6:152419922-152419922 (T->A)
165	MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	1	3	37	12	120	50	2015-02-26 02:05:23.264237	2015-02-26 02:05:23.264237	3	accepted	\N	\N	1	6:152419923-152419923 (A->C)
166	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	38	12	121	50	2015-02-26 02:05:23.331319	2015-02-26 02:05:23.331319	5	accepted	\N	\N	1	6:152419923-152419923 (A->C)
167	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	39	12	121	50	2015-02-26 02:05:23.384296	2015-02-26 02:05:23.384296	5	accepted	\N	\N	1	6:152419923-152419923 (A->C)
168	In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic sites and overall stable disease. However, this response did not meet criteria for RECIST partial response.	Sensitivity	Supports	1	2	40	23	122	51	2015-02-26 02:05:23.452103	2015-02-26 02:05:23.452103	2	accepted	\N	\N	1	N/A
169	Following identification of this fusion in a patient, this fusion was expressed in HEK 293T cells leading to oligomerization in the absence of ligand, induced morphologic changes in the cell and increased cellular proliferation consistent with FGFR activation. Mouse xenografts using the bladder cell line SW780 which harbors an FGFR3-BAIAP2L1 fusion showed reduced tumor burden when treated with the FGFR inhibitor PD173074.	Sensitivity	Supports	1	3	41	24	123	52	2015-02-26 02:05:23.53048	2015-02-26 02:05:23.53048	3	accepted	\N	\N	1	N/A
170	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This fusion is also shown to be recurrent in multiple other cancer types.	Sensitivity	Supports	1	2	42	23	122	53	2015-02-26 02:05:23.607407	2015-02-26 02:05:23.607407	3	accepted	\N	\N	1	N/A
171	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This fusion is also shown to be recurrent in multiple other cancer types.	Sensitivity	Supports	1	2	40	23	122	53	2015-02-26 02:05:23.665867	2015-02-26 02:05:23.665867	3	accepted	\N	\N	1	N/A
172	AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with FLT3 internal tandem repeat	Sensitivity	Supports	1	3	43	1	124	54	2015-02-26 02:05:23.760095	2015-02-26 02:05:23.760095	4	accepted	\N	\N	1	N/A
173	ATRA treatment did not effect overall survival in patients <60 years old with FLT3-ITD mutation	Resistance or Non-Response	Supports	1	2	44	1	125	54	2015-02-26 02:05:23.823935	2015-02-26 02:05:23.823935	3	accepted	\N	\N	1	N/A
174	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	Sensitivity	Supports	1	2	45	1	126	54	2015-02-26 02:05:23.887285	2015-02-26 02:05:23.887285	4	accepted	\N	\N	1	N/A
175	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	Resistance or Non-Response	Supports	1	3	29	1	127	54	2015-02-26 02:05:23.949465	2015-02-26 02:05:23.949465	3	accepted	\N	\N	1	N/A
176	WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared to wild type	Positive	Supports	3	2	3	1	128	54	2015-02-26 02:05:24.007989	2015-02-26 02:05:24.007989	3	accepted	\N	\N	1	N/A
177	FLT3-ITD mutation without NPM1 was associated with increased relapse risk and reduced overall survival in young adult AML patients (median age 43)	Resistance or Non-Response	Supports	1	2	3	1	129	54	2015-02-26 02:05:24.068287	2015-02-26 02:05:24.068287	4	accepted	\N	\N	1	N/A
178	Increasing level of FLT3-ITD mutant correlated with reduced disease-free and overall survival as well as increased relapse rate	Resistance or Non-Response	Supports	1	2	3	1	129	54	2015-02-26 02:05:24.147996	2015-02-26 02:05:24.147996	3	accepted	\N	\N	1	N/A
179	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N / A	Does Not Support	2	2	3	1	105	54	2015-02-26 02:05:24.204056	2015-02-26 02:05:24.204056	4	accepted	\N	\N	1	N/A
180	FLT3-ITD duplication length did not effect prognosis in patients with an NPM1 mutation	N / A	Does Not Support	2	2	3	1	130	54	2015-02-26 02:05:24.266959	2015-02-26 02:05:24.266959	4	accepted	\N	\N	1	N/A
181	Size of the FLT3-ITD mutant duplication had no impact on overall survival or relapse rate	N / A	Does Not Support	2	2	3	1	129	54	2015-02-26 02:05:24.330289	2015-02-26 02:05:24.330289	2	accepted	\N	\N	1	N/A
182	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	Poor Outcome	Supports	2	2	3	1	130	54	2015-02-26 02:05:24.393791	2015-02-26 02:05:24.393791	4	accepted	\N	\N	1	N/A
183	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	Poor Outcome	Supports	2	2	3	1	130	54	2015-02-26 02:05:24.453234	2015-02-26 02:05:24.453234	4	accepted	\N	\N	1	N/A
184	A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation	Poor Outcome	Supports	2	2	3	1	130	54	2015-02-26 02:05:24.513195	2015-02-26 02:05:24.513195	4	accepted	\N	\N	1	N/A
185	AML patients with internal tandem repeat of FLT3 has worse overall survival than patients with wild type FLT3	Poor Outcome	Supports	2	2	3	1	128	54	2015-02-26 02:05:24.589699	2015-02-26 02:05:24.589699	3	accepted	\N	\N	1	N/A
186	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated with worse overall survival compared to those without DNMT3A mutation.	Poor Outcome	Supports	2	2	3	1	107	54	2015-02-26 02:05:24.647562	2015-02-26 02:05:24.647562	5	accepted	\N	\N	1	N/A
187	Meta-analysis of studies involving cytogentically normal younger (<60) patients showed reduced overall and relapse-free survival for patients with FLT3-ITD	Poor Outcome	Supports	2	1	3	1	131	54	2015-02-26 02:05:24.704389	2015-02-26 02:05:24.704389	2	accepted	\N	\N	1	N/A
188	Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor survival and inferior disease free survival in patients with AML.	Poor Outcome	Supports	2	2	3	1	132	54	2015-02-26 02:05:24.763128	2015-02-26 02:05:24.763128	4	accepted	\N	\N	1	N/A
189	Sorafenib is effective in patients with FLT3 internal tandem repeats, but not in wild type or D835 mutation.	Sensitivity	Supports	1	3	10	1	133	54	2015-02-26 02:05:24.831779	2015-02-26 02:05:24.831779	4	accepted	\N	\N	1	N/A
190	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	Resistance or Non-Response	Supports	1	3	46	1	127	54	2015-02-26 02:05:24.894773	2015-02-26 02:05:24.894773	3	accepted	\N	\N	1	N/A
191	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	Sensitivity	Supports	1	2	45	1	126	55	2015-02-26 02:05:24.956502	2015-02-26 02:05:24.956502	4	accepted	\N	\N	1	N/A
192	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	Resistance or Non-Response	Supports	1	3	29	1	127	55	2015-02-26 02:05:25.03929	2015-02-26 02:05:25.03929	3	accepted	\N	\N	1	N/A
194	AML patients with D835 mutation of FLT3 did not have worse overall survival than patients with wild type FLT3	N / A	Does Not Support	2	2	3	1	128	55	2015-02-26 02:05:25.164264	2015-02-26 02:05:25.164264	3	accepted	\N	\N	1	N/A
195	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	Better Outcome	Supports	2	2	3	1	134	55	2015-02-26 02:05:25.228619	2015-02-26 02:05:25.228619	3	accepted	\N	\N	1	N/A
196	In AML patients, FLT3-TKD mutation was associated with poorer disease free survival compared to patients with wild type FLT3	Poor Outcome	Supports	2	2	3	1	135	55	2015-02-26 02:05:25.291558	2015-02-26 02:05:25.291558	4	accepted	\N	\N	1	N/A
197	Sorafenib is effective in patients with FLT3 internal tandem repeat mutations, but not in patients with wild type or D835 mutation.	Resistance or Non-Response	Supports	1	3	10	1	133	55	2015-02-26 02:05:25.351885	2015-02-26 02:05:25.351885	4	accepted	\N	\N	1	N/A
198	Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated with sorafenib resistance	Resistance or Non-Response	Supports	1	2	10	1	136	55	2015-02-26 02:05:25.418785	2015-02-26 02:05:25.418785	4	accepted	\N	\N	1	N/A
199	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	Resistance or Non-Response	Supports	1	3	46	1	127	55	2015-02-26 02:05:25.506928	2015-02-26 02:05:25.506928	3	accepted	\N	\N	1	N/A
200	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	Sensitivity	Supports	1	2	47	25	137	56	2015-02-26 02:05:25.577947	2015-02-26 02:05:25.577947	4	accepted	\N	\N	1	N/A
201	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	Sensitivity	Supports	1	2	48	25	137	56	2015-02-26 02:05:25.631712	2015-02-26 02:05:25.631712	4	accepted	\N	\N	1	N/A
202	In leukemia patients, IDH1 R132 mutation is associated with adults and normal karyotype.	Positive	Supports	3	2	3	1	138	57	2015-02-26 02:05:25.697546	2015-02-26 02:05:25.697546	2	accepted	\N	\N	1	N/A
203	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	N / A	Does Not Support	2	2	3	1	139	57	2015-02-26 02:05:25.755769	2015-02-26 02:05:25.755769	3	accepted	\N	\N	1	N/A
204	The complete remission and overall survival rates in patients with IDH1 R132 mutation is not significantly different from those who do not have this mutation	N / A	Does Not Support	2	2	3	1	140	57	2015-02-26 02:05:25.814712	2015-02-26 02:05:25.814712	2	accepted	\N	\N	1	N/A
205	In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic for better survival compared to patients who harbor wild type mutation.	Better Outcome	Supports	2	2	3	26	141	57	2015-02-26 02:05:25.957281	2015-02-26 02:05:25.957281	3	accepted	\N	\N	1	N/A
206	In patients with glioblastoma, those harboring IDH1 R132 mutation has higher overall survival compared to those who do not have IDH1 mutation	Better Outcome	Supports	2	2	3	27	142	57	2015-02-26 02:05:26.022451	2015-02-26 02:05:26.022451	3	accepted	\N	\N	1	N/A
207	IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated with worse overall survival	Poor Outcome	Supports	2	2	3	28	143	57	2015-02-26 02:05:26.085555	2015-02-26 02:05:26.085555	4	accepted	\N	\N	1	N/A
208	IDH1 R132 mutation is associated with patients of older age, high platelet count during diagnosis, cytogenic normalcy and NPM1 mutation	Positive	Supports	3	2	3	1	144	58	2015-02-26 02:05:26.144553	2015-02-26 02:05:26.144553	3	accepted	\N	\N	1	2:209113113-209113113 (G->A)
209	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N / A	Does Not Support	2	2	3	1	144	58	2015-02-26 02:05:26.215858	2015-02-26 02:05:26.215858	2	accepted	\N	\N	1	2:209113113-209113113 (G->A)
210	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N / A	Does Not Support	2	2	3	1	144	59	2015-02-26 02:05:26.277571	2015-02-26 02:05:26.277571	3	accepted	\N	\N	1	2:209113112-209113112 (C->T)
211	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N / A	Does Not Support	2	2	3	1	144	60	2015-02-26 02:05:26.335398	2015-02-26 02:05:26.335398	3	accepted	\N	\N	1	2:209113112-209113112 (C->A)
212	AML patients with IDH2 mutations such as R140Q/L have event free survival and overall survival similar to those with wild-type IDH2	N / A	Does Not Support	2	2	3	1	145	61	2015-02-26 02:05:26.39151	2015-02-26 02:05:26.39151	2	accepted	\N	\N	1	N/A
213	In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact overall survival or disease free survival.	N / A	Does Not Support	2	2	3	1	146	61	2015-02-26 02:05:26.45569	2015-02-26 02:05:26.45569	3	accepted	\N	\N	1	N/A
214	In AML, patients with IDH2 R140K mutation have improved overall survival compared to those with wild-type IDH2	Better Outcome	Supports	2	2	3	1	147	61	2015-02-26 02:05:26.508507	2015-02-26 02:05:26.508507	3	accepted	\N	\N	1	N/A
215	R140Q mutation in IDH2 does not have prognostic value in patients with MDS.	N / A	Does Not Support	2	2	3	29	148	61	2015-02-26 02:05:26.566112	2015-02-26 02:05:26.566112	3	accepted	\N	\N	1	N/A
216	In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L do not confer prognostic value (overall survival)	N / A	Does Not Support	2	2	3	29	149	61	2015-02-26 02:05:26.624361	2015-02-26 02:05:26.624361	2	accepted	\N	\N	1	N/A
217	AML patients with IDH2 mutations such as R172K have event free survival and overall survival similar to those with wild-type IDH2.	N / A	Does Not Support	2	2	3	1	145	62	2015-02-26 02:05:26.675015	2015-02-26 02:05:26.675015	2	accepted	\N	\N	1	15:90631838-90631838 (C->T)
218	In AML, patients with an IDH2 R172K mutation have worse overall survival compared to those with wild-type IDH2	Poor Outcome	Supports	2	2	3	1	147	62	2015-02-26 02:05:26.740416	2015-02-26 02:05:26.740416	3	accepted	\N	\N	1	15:90631838-90631838 (C->T)
219	In patients suffering from MDS, the presence of IDH2 mutations such as R172K do not confer prognostic value (overall survival)	N / A	Does Not Support	2	2	3	29	149	62	2015-02-26 02:05:26.794675	2015-02-26 02:05:26.794675	2	accepted	\N	\N	1	15:90631838-90631838 (C->T)
220	JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical CML).	Positive	Supports	3	2	3	2	150	63	2015-02-26 02:05:26.868851	2015-02-26 02:05:26.868851	4	accepted	\N	\N	1	9:5073770-5073770 (G->T)
221	JAK2 V617F is associated with myeloid neoplasms (AML, MDS, CMML/aCML)	Positive	Supports	3	2	3	2	150	63	2015-02-26 02:05:26.931339	2015-02-26 02:05:26.931339	4	accepted	\N	\N	1	9:5073770-5073770 (G->T)
222	JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).	Negative	Supports	3	2	3	30	150	63	2015-02-26 02:05:26.984533	2015-02-26 02:05:26.984533	4	accepted	\N	\N	1	9:5073770-5073770 (G->T)
223	The SNP rs10974944 in the Jak2 locus is associated with increased predisposition for JAK2 V617F mutation and its associated cancer	Positive	Supports	3	2	3	31	151	63	2015-02-26 02:05:27.049316	2015-02-26 02:05:27.049316	4	accepted	\N	\N	1	9:5073770-5073770 (G->T)
224	In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction in the percentage of cells harboring JAK2 V617F.	Sensitivity	Supports	1	2	49	32	152	63	2015-02-26 02:05:27.119088	2015-02-26 02:05:27.119088	4	accepted	\N	\N	1	9:5073770-5073770 (G->T)
225	TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells	Sensitivity	Supports	1	3	50	32	153	63	2015-02-26 02:05:27.17798	2015-02-26 02:05:27.17798	3	accepted	\N	\N	1	9:5073770-5073770 (G->T)
226	In acute myloid leukemia patients, D816 mutation is associated with earlier relapse and poorer prognosis than wildtype KIT	Poor Outcome	Supports	2	2	3	33	154	64	2015-02-26 02:05:27.239806	2015-02-26 02:05:27.239806	4	accepted	\N	\N	1	4:55599321-55599321 (A->T)
227	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	Positive	Supports	3	2	3	34	155	65	2015-02-26 02:05:27.309873	2015-02-26 02:05:27.309873	2	accepted	\N	\N	1	N/A
228	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	N / A	Does Not Support	2	2	3	34	156	65	2015-02-26 02:05:27.380362	2015-02-26 02:05:27.380362	3	accepted	\N	\N	1	N/A
229	KIT mutations were identified in morphologically benign, incidentally discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating is not a prognostic marker for GIST	N / A	Does Not Support	2	2	3	34	157	65	2015-02-26 02:05:27.471097	2015-02-26 02:05:27.471097	3	accepted	\N	\N	1	N/A
230	KIT mutation is associated with worse overall and cause-specific prognosis in patients with GIST compared to patients with wildtype KIT	Poor Outcome	Supports	2	2	3	34	155	65	2015-02-26 02:05:27.540074	2015-02-26 02:05:27.540074	3	accepted	\N	\N	1	N/A
231	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	N / A	Does Not Support	2	2	3	18	158	65	2015-02-26 02:05:27.607499	2015-02-26 02:05:27.607499	2	accepted	\N	\N	1	N/A
232	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	2	2	2	8	159	65	2015-02-26 02:05:27.678731	2015-02-26 02:05:27.678731	3	accepted	\N	\N	1	N/A
233	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	2	2	2	8	159	67	2015-02-26 02:05:27.749388	2015-02-26 02:05:27.749388	3	accepted	\N	\N	1	N/A
234	GIST patients with exon 14 KIT mutations had reduced overall survival compared to patients wiltype for KIT	Poor Outcome	Supports	2	2	3	34	156	68	2015-02-26 02:05:27.810498	2015-02-26 02:05:27.810498	3	accepted	\N	\N	1	N/A
235	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	Positive	Supports	3	2	3	34	155	69	2015-02-26 02:05:27.865143	2015-02-26 02:05:27.865143	3	accepted	\N	\N	1	N/A
236	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	2	2	2	8	159	69	2015-02-26 02:05:27.930915	2015-02-26 02:05:27.930915	3	accepted	\N	\N	1	N/A
237	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	N / A	Does Not Support	2	2	3	34	156	70	2015-02-26 02:05:27.997407	2015-02-26 02:05:27.997407	3	accepted	\N	\N	1	N/A
238	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	N / A	Does Not Support	2	2	3	18	158	70	2015-02-26 02:05:28.064632	2015-02-26 02:05:28.064632	2	accepted	\N	\N	1	N/A
239	In a case study, a patient with anal melanoma harboring a 7 codon duplication in the juxtamambrane region (exon 11) of KIT showed marked response 4 months after imatinib treatment.	Sensitivity	Supports	1	2	2	35	160	66	2015-02-26 02:05:28.124915	2015-02-26 02:05:28.124915	3	accepted	\N	\N	1	N/A
240	The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation of KIT but did not alter the binding affinity of dasatinib.	Sensitivity	Supports	1	3	5	8	161	71	2015-02-26 02:05:28.19657	2015-02-26 02:05:28.19657	3	accepted	\N	\N	1	N/A
241	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	1	3	2	8	161	71	2015-02-26 02:05:28.27501	2015-02-26 02:05:28.27501	3	accepted	\N	\N	1	N/A
242	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	1	3	4	8	161	71	2015-02-26 02:05:28.323105	2015-02-26 02:05:28.323105	3	accepted	\N	\N	1	N/A
243	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	1	3	10	8	161	71	2015-02-26 02:05:28.373758	2015-02-26 02:05:28.373758	3	accepted	\N	\N	1	N/A
244	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	1	3	5	4	162	71	2015-02-26 02:05:28.455797	2015-02-26 02:05:28.455797	3	accepted	\N	\N	1	N/A
245	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	1	3	2	4	162	71	2015-02-26 02:05:28.518659	2015-02-26 02:05:28.518659	3	accepted	\N	\N	1	N/A
246	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	1	3	4	4	162	71	2015-02-26 02:05:28.566629	2015-02-26 02:05:28.566629	3	accepted	\N	\N	1	N/A
247	KIT V654A results in imatinib resistance in GIST patient-derived cell lines.	Resistance or Non-Response	Supports	1	3	2	34	163	72	2015-02-26 02:05:28.629103	2015-02-26 02:05:28.629103	3	accepted	\N	\N	1	4:55594258-55594258 (T->C)
248	In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective for those that are refractory to imatinib.	Sensitivity	Supports	1	2	51	34	164	72	2015-02-26 02:05:28.696872	2015-02-26 02:05:28.696872	3	accepted	\N	\N	1	4:55594258-55594258 (T->C)
249	Patients with colorectal cancer who harbor KRAS mutation have low response rate to cetuximab	Resistance or Non-Response	Supports	1	2	14	7	165	73	2015-02-26 02:05:28.759467	2015-02-26 02:05:28.759467	4	accepted	\N	\N	1	N/A
250	KRAS mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	Sensitivity	Does Not Support	1	2	52	7	166	74	2015-02-26 02:05:28.824351	2015-02-26 02:05:28.824351	4	accepted	\N	\N	1	N/A
251	In NSCLC, Exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	Resistance or Non-Response	Supports	1	2	31	4	167	74	2015-02-26 02:05:28.877992	2015-02-26 02:05:28.877992	3	accepted	\N	\N	1	N/A
252	In NSCLC, Exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	Resistance or Non-Response	Supports	1	2	32	4	167	74	2015-02-26 02:05:28.958219	2015-02-26 02:05:28.958219	3	accepted	\N	\N	1	N/A
253	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	Positive	Supports	3	2	3	36	168	75	2015-02-26 02:05:29.018787	2015-02-26 02:05:29.018787	3	accepted	\N	\N	1	N/A
254	In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12 mutation did not impact overall or disease free survival.	N / A	Does Not Support	2	2	3	9	169	75	2015-02-26 02:05:29.081295	2015-02-26 02:05:29.081295	4	accepted	\N	\N	1	N/A
255	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	Poor Outcome	Supports	2	2	3	37	170	75	2015-02-26 02:05:29.146987	2015-02-26 02:05:29.146987	3	accepted	\N	\N	1	N/A
256	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	Positive	Does Not Support	3	2	3	4	171	75	2015-02-26 02:05:29.216085	2015-02-26 02:05:29.216085	3	accepted	\N	\N	1	N/A
257	There is no strong association between KRAS mutation status and overall or progression free survival in patients with NSCLC	N / A	Does Not Support	2	2	3	4	172	75	2015-02-26 02:05:29.276791	2015-02-26 02:05:29.276791	3	accepted	\N	\N	1	N/A
258	A meta-analysis showed KRAS mutation is associated with worse outcome in patients with NSCLC	Poor Outcome	Supports	2	2	3	4	173	75	2015-02-26 02:05:29.342528	2015-02-26 02:05:29.342528	3	accepted	\N	\N	1	N/A
259	KRAS mutations were significantly associated with lack of response to cetuximab in patients with advanced colorectal cancer	Resistance or Non-Response	Supports	1	3	14	7	174	76	2015-02-26 02:05:29.407049	2015-02-26 02:05:29.407049	3	accepted	\N	\N	1	N/A
260	Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations (primarily G12/G13) had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype KRAS	Resistance or Non-Response	Supports	1	2	14	7	71	76	2015-02-26 02:05:29.460456	2015-02-26 02:05:29.460456	3	accepted	\N	\N	1	N/A
261	KRAS mutations were significantly associated with reduced progression-free and overall survival in patients with advanced colorectal cancer	Poor Outcome	Supports	2	3	3	7	174	76	2015-02-26 02:05:29.51484	2015-02-26 02:05:29.51484	3	accepted	\N	\N	1	N/A
262	KRAS G12C occur more frequently in women than men	Positive	Supports	3	2	3	18	175	77	2015-02-26 02:05:29.572371	2015-02-26 02:05:29.572371	2	accepted	\N	\N	1	12:25398285-25398285 (C->A)
263	KRAS G12D mutation occurs in never smokers significantly more often than in smokers	Positive	Supports	3	2	3	18	175	78	2015-02-26 02:05:29.631047	2015-02-26 02:05:29.631047	3	accepted	\N	\N	1	12:25398284-25398284 (C->T)
264	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	Resistance or Non-Response	Supports	1	2	13	4	176	78	2015-02-26 02:05:29.690742	2015-02-26 02:05:29.690742	2	accepted	\N	\N	1	12:25398284-25398284 (C->T)
265	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.	Sensitivity	Supports	1	3	53	4	177	78	2015-02-26 02:05:29.7549	2015-02-26 02:05:29.7549	4	accepted	\N	\N	1	12:25398284-25398284 (C->T)
266	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.	Sensitivity	Supports	1	3	54	4	177	78	2015-02-26 02:05:29.803461	2015-02-26 02:05:29.803461	4	accepted	\N	\N	1	12:25398284-25398284 (C->T)
267	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	Positive	Supports	3	2	3	36	168	79	2015-02-26 02:05:29.859084	2015-02-26 02:05:29.859084	3	accepted	\N	\N	1	N/A
268	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	Poor Outcome	Supports	2	2	3	38	170	79	2015-02-26 02:05:29.923165	2015-02-26 02:05:29.923165	3	accepted	\N	\N	1	N/A
269	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	Positive	Does Not Support	3	2	3	4	171	79	2015-02-26 02:05:29.987197	2015-02-26 02:05:29.987197	3	accepted	\N	\N	1	N/A
270	Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	Resistance or Non-Response	Supports	1	3	14	9	178	80	2015-02-26 02:05:30.057302	2015-02-26 02:05:30.057302	4	accepted	\N	\N	1	12:25398281-25398281 (C->T)
271	Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	Sensitivity	Supports	1	3	14	9	178	80	2015-02-26 02:05:30.121258	2015-02-26 02:05:30.121258	4	accepted	\N	\N	1	12:25398281-25398281 (C->T)
272	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	Sensitivity	Supports	1	2	14	9	178	80	2015-02-26 02:05:30.185919	2015-02-26 02:05:30.185919	4	accepted	\N	\N	1	12:25398281-25398281 (C->T)
273	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	Better Outcome	Supports	2	2	14	9	178	80	2015-02-26 02:05:30.255141	2015-02-26 02:05:30.255141	4	accepted	\N	\N	1	12:25398281-25398281 (C->T)
274	The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition by about 5 fold.	Resistance or Non-Response	Supports	1	3	23	8	179	81	2015-02-26 02:05:30.318181	2015-02-26 02:05:30.318181	3	accepted	\N	\N	1	15:66729162-66729162 (C->T)
275	MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.	Resistance or Non-Response	Supports	1	3	23	8	180	82	2015-02-26 02:05:30.38714	2015-02-26 02:05:30.38714	3	accepted	\N	\N	1	15:66727451-66727451 (A->C)
276	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	3	2	20	101	83	2015-02-26 02:05:30.494061	2015-02-26 02:05:30.494061	3	accepted	\N	\N	1	N/A
277	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	3	28	20	101	83	2015-02-26 02:05:30.547434	2015-02-26 02:05:30.547434	3	accepted	\N	\N	1	N/A
278	MGMT promoter methylation has been shown to confer sensitivity to alkylating agents, such as carmustine, in glioblastoma cells.	Sensitivity	Supports	1	2	55	39	181	84	2015-02-26 02:05:30.613215	2015-02-26 02:05:30.613215	4	accepted	\N	\N	1	N/A
279	MGMT promoter methylation was associated with increased overall and disease-free survival as well as tumor regression in patients with gliomas.	Better Outcome	Supports	2	2	3	39	181	84	2015-02-26 02:05:30.684545	2015-02-26 02:05:30.684545	4	accepted	\N	\N	1	N/A
280	In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor, such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may be useful in treating patients with initial resistance to alkylating agents.	Sensitivity	Supports	1	4	56	39	181	84	2015-02-26 02:05:30.754874	2015-02-26 02:05:30.754874	4	accepted	\N	\N	1	N/A
281	In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted from temozolomide. This benefit was also methylation status dependent, as those without methylation did not see increased survival.	Sensitivity	Supports	1	1	22	39	182	84	2015-02-26 02:05:30.832066	2015-02-26 02:05:30.832066	5	accepted	\N	\N	1	N/A
282	ATRA treatment did not effect overall survival in patients <60 years old with NPM1 mutation regardless of FLT3-ITD status	Resistance or Non-Response	Supports	1	2	44	1	125	85	2015-02-26 02:05:30.896931	2015-02-26 02:05:30.896931	3	accepted	\N	\N	1	N/A
283	ATRA treatment improved overall and relapse-free survival in older (>60) patients with mutant NPM1 and without FLT3-ITD	Sensitivity	Supports	1	2	44	1	183	85	2015-02-26 02:05:30.968028	2015-02-26 02:05:30.968028	2	accepted	\N	\N	1	N/A
284	Young patients (18-60) with NPM1 mutations had improved overall survival following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However, the adjusted p-value for multiple testing is 0.11.	Sensitivity	Supports	1	2	29	1	184	85	2015-02-26 02:05:31.037781	2015-02-26 02:05:31.037781	2	accepted	\N	\N	1	N/A
285	NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype (0/55).	Positive	Does Not Support	3	2	3	1	185	85	2015-02-26 02:05:31.104336	2015-02-26 02:05:31.104336	3	accepted	\N	\N	1	N/A
286	NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid leukemia.	Negative	Supports	3	2	3	1	186	85	2015-02-26 02:05:31.163571	2015-02-26 02:05:31.163571	4	accepted	\N	\N	1	N/A
287	NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid leukemia	Positive	Supports	3	2	3	1	185	85	2015-02-26 02:05:31.22777	2015-02-26 02:05:31.22777	3	accepted	\N	\N	1	N/A
288	AML with mutated NPM1 is a provisional entity in WHO classification of acute myeloid leukemia (AML). This mutation should be tested for in clinical trials and is recommended for testing in patients with cytogentically normal AML.	Positive	Supports	3	1	3	1	187	85	2015-02-26 02:05:31.291339	2015-02-26 02:05:31.291339	5	accepted	\N	\N	1	N/A
289	NPM1 mutations were associated with normal karyotype in older (>60) patients	Positive	Supports	3	2	3	1	183	85	2015-02-26 02:05:31.350257	2015-02-26 02:05:31.350257	3	accepted	\N	\N	1	N/A
290	NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes	Positive	Supports	3	2	3	1	129	85	2015-02-26 02:05:31.412072	2015-02-26 02:05:31.412072	2	accepted	\N	\N	1	N/A
291	NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid leukemia.	Positive	Supports	3	2	3	1	186	85	2015-02-26 02:05:31.466854	2015-02-26 02:05:31.466854	4	accepted	\N	\N	1	N/A
292	NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and increased overall survival in young adult acute myeloid leukemia patients (median age 43)	Sensitivity	Supports	1	2	3	1	129	85	2015-02-26 02:05:31.526361	2015-02-26 02:05:31.526361	4	accepted	\N	\N	1	N/A
293	Patients with NPM1 mutations were associated with improved complete remission rates as well as overall and disease free survival when all ages and karyotype were analyzed; however, when only normal karyotype patients were analyzed, only complete remission rates were improved	Sensitivity	Supports	1	2	3	1	183	85	2015-02-26 02:05:31.591085	2015-02-26 02:05:31.591085	3	accepted	\N	\N	1	N/A
294	CD33 and CD123 expression was significantly increased in patients with NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33 and anti-CD123 therapy	Sensitivity	Supports	1	4	3	1	188	85	2015-02-26 02:05:31.650565	2015-02-26 02:05:31.650565	2	accepted	\N	\N	1	N/A
295	CD33 expression was significantly increased in patients with NPM1 mutation with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy	Sensitivity	Supports	1	4	3	1	188	85	2015-02-26 02:05:31.704997	2015-02-26 02:05:31.704997	2	accepted	\N	\N	1	N/A
296	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N / A	Does Not Support	2	2	3	1	105	85	2015-02-26 02:05:31.761676	2015-02-26 02:05:31.761676	4	accepted	\N	\N	1	N/A
297	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	N / A	Does Not Support	2	2	3	1	139	85	2015-02-26 02:05:31.822059	2015-02-26 02:05:31.822059	3	accepted	\N	\N	1	N/A
298	NPM1 mutation without FLT3-ITD was significantly associated with complete remission	Better Outcome	Supports	2	2	3	1	99	85	2015-02-26 02:05:31.888329	2015-02-26 02:05:31.888329	3	accepted	\N	\N	1	N/A
299	NPM1 mutation was associated with increased overall, event-free and relapse-free survival and remission after induction therapy in the entire cohort and in the cytogenetically normal subset of patients	Better Outcome	Supports	2	2	3	1	189	85	2015-02-26 02:05:31.999169	2015-02-26 02:05:31.999169	3	accepted	\N	\N	1	N/A
300	Intermediate risk patients with mutant NPM1 had improved overall survival with the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1 and IDH2	Better Outcome	Supports	2	2	3	1	184	85	2015-02-26 02:05:32.056028	2015-02-26 02:05:32.056028	2	accepted	\N	\N	1	N/A
301	Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and without FLT3-ITD have increased overall survival, relapse-free survival and response to induction chemotherapy	Better Outcome	Supports	2	2	3	1	190	85	2015-02-26 02:05:32.117224	2015-02-26 02:05:32.117224	4	accepted	\N	\N	1	N/A
302	NPM1 mutation was associated with higher complete remission rates, lower cumulative incidence of relapse and higher overall survival in intermediate risk AML patients	Better Outcome	Supports	2	2	3	1	191	85	2015-02-26 02:05:32.179448	2015-02-26 02:05:32.179448	4	accepted	\N	\N	1	N/A
303	Complete remission rates were higher and event-free survival was longer for normal karyotype AML patients with Exon 12 NPM1 mutations.	Better Outcome	Supports	2	2	3	1	186	85	2015-02-26 02:05:32.237698	2015-02-26 02:05:32.237698	5	accepted	\N	\N	1	N/A
304	NPM1 mutations were associated with increased complete remission rates as well as longer overall disease free survival in normal karyotype AML patients >59 years old, particularly in those >69 years old	Better Outcome	Supports	2	2	3	1	192	85	2015-02-26 02:05:32.299292	2015-02-26 02:05:32.299292	4	accepted	\N	\N	1	N/A
305	Complete remission rates were higher and both disease-free and overall survival were longer for patients with Exon 12 NPM1 mutations.	Better Outcome	Supports	2	2	3	1	185	85	2015-02-26 02:05:32.369937	2015-02-26 02:05:32.369937	3	accepted	\N	\N	1	N/A
306	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	Better Outcome	Supports	2	2	3	1	134	85	2015-02-26 02:05:32.424852	2015-02-26 02:05:32.424852	3	accepted	\N	\N	1	N/A
307	Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved overall and relapse-free survival, complete remission rates and remission duration compared to all other combinations of mutants	Better Outcome	Supports	2	2	3	1	193	85	2015-02-26 02:05:32.486438	2015-02-26 02:05:32.486438	3	accepted	\N	\N	1	N/A
308	Coocurrence of FLT3-ITD mutations significantly reduced overall survival of normal karyotype AML patients	Poor Outcome	Supports	2	2	3	1	185	85	2015-02-26 02:05:32.546664	2015-02-26 02:05:32.546664	4	accepted	\N	\N	1	N/A
309	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	Poor Outcome	Supports	2	2	3	1	130	85	2015-02-26 02:05:32.615118	2015-02-26 02:05:32.615118	4	accepted	\N	\N	1	N/A
310	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	Poor Outcome	Supports	2	2	3	1	130	85	2015-02-26 02:05:32.670645	2015-02-26 02:05:32.670645	4	accepted	\N	\N	1	N/A
311	Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction in NPM1 levels following first line therapy (N=15)	Poor Outcome	Supports	2	2	3	1	194	85	2015-02-26 02:05:32.732968	2015-02-26 02:05:32.732968	2	accepted	\N	\N	1	N/A
312	TET2 mutation reduces overall survival of normal karyotype patients with either an NPM1 mutation or NPM1 mutation without FLT3-ITD	Poor Outcome	Supports	2	2	3	1	195	85	2015-02-26 02:05:32.803392	2015-02-26 02:05:32.803392	3	accepted	\N	\N	1	N/A
313	Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate risk AML patients	Poor Outcome	Supports	2	2	3	1	191	85	2015-02-26 02:05:32.86344	2015-02-26 02:05:32.86344	3	accepted	\N	\N	1	N/A
314	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	Sensitivity	Supports	1	3	57	1	196	85	2015-02-26 02:05:32.934674	2015-02-26 02:05:32.934674	3	accepted	\N	\N	1	N/A
315	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	Sensitivity	Supports	1	3	44	1	196	85	2015-02-26 02:05:32.993165	2015-02-26 02:05:32.993165	3	accepted	\N	\N	1	N/A
316	For patients with NPM1 mutation that achieved CR following induction therapy, relapse-free survival was improved following treatment with valproic acid	Sensitivity	Supports	1	2	58	1	197	85	2015-02-26 02:05:33.054894	2015-02-26 02:05:33.054894	2	accepted	\N	\N	1	N/A
317	No NPM1 mutations were identified in patients with favorable risk cytogenetics (79/215 patients)	Positive	Does Not Support	3	2	3	1	107	86	2015-02-26 02:05:33.111961	2015-02-26 02:05:33.111961	4	accepted	\N	\N	1	5:170837546-170837547 (0->TCAG)
318	NPM1 mutations were associated with intermediate risk cytogenetics (including normal karyotype)	Positive	Supports	3	2	3	1	107	86	2015-02-26 02:05:33.183382	2015-02-26 02:05:33.183382	4	accepted	\N	\N	1	5:170837546-170837547 (0->TCAG)
319	NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM in AML patients was associated with good response to induction therapy.	Sensitivity	Supports	1	4	3	1	198	86	2015-02-26 02:05:33.245661	2015-02-26 02:05:33.245661	3	accepted	\N	\N	1	5:170837546-170837547 (0->TCAG)
320	NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.	Sensitivity	Supports	1	3	57	1	196	86	2015-02-26 02:05:33.329084	2015-02-26 02:05:33.329084	3	accepted	\N	\N	1	5:170837546-170837547 (0->TCAG)
321	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	Poor Outcome	Supports	2	2	3	8	199	87	2015-02-26 02:05:33.439958	2015-02-26 02:05:33.439958	3	accepted	\N	\N	1	N/A
322	The presence of NRAS mutation in AML patients does not impact diease prognosis (resistant disease, disease-free survival, complete remission rate, relapse rate, induction death)	N / A	Does Not Support	2	2	3	1	200	88	2015-02-26 02:05:33.501437	2015-02-26 02:05:33.501437	3	accepted	\N	\N	1	N/A
323	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	Poor Outcome	Supports	2	2	3	7	201	88	2015-02-26 02:05:33.561577	2015-02-26 02:05:33.561577	3	accepted	\N	\N	1	N/A
324	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	Poor Outcome	Supports	2	2	3	8	199	88	2015-02-26 02:05:33.622118	2015-02-26 02:05:33.622118	3	accepted	\N	\N	1	N/A
325	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	Poor Outcome	Supports	2	1	3	7	201	89	2015-02-26 02:05:33.68185	2015-02-26 02:05:33.68185	3	accepted	\N	\N	1	N/A
326	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	Poor Outcome	Supports	2	1	3	7	201	90	2015-02-26 02:05:33.741415	2015-02-26 02:05:33.741415	3	accepted	\N	\N	1	N/A
327	NRAS mutation status does not impact overall, event-free or disease-free survival in patients with AML	N / A	Does Not Support	2	2	3	1	202	91	2015-02-26 02:05:33.815501	2015-02-26 02:05:33.815501	3	accepted	\N	\N	1	N/A
328	Likely due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic malignant melanoma with an NRAS G13D mutation	Sensitivity	Supports	1	2	59	8	203	92	2015-02-26 02:05:33.881354	2015-02-26 02:05:33.881354	2	accepted	\N	\N	1	1:115258744-115258744 (C->T)
329	In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status (primarily Q61) was not informative for disease control rate, progression free survival or overall survival following cetuximab plus chemotherapy.	Resistance or Non-Response	Does Not Support	1	2	14	7	71	93	2015-02-26 02:05:33.941694	2015-02-26 02:05:33.941694	3	accepted	\N	\N	1	N/A
330	Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation (primarily Q61) have a significantly lower response rate to cetuximab than patients wildtype for NRAS.	Resistance or Non-Response	Supports	1	2	14	7	71	93	2015-02-26 02:05:33.996735	2015-02-26 02:05:33.996735	3	accepted	\N	\N	1	N/A
331	Melanoma associated with NRAS Q61 mutation was more often associated with those at the extremity than those at the trunk	Positive	Supports	3	2	3	8	204	93	2015-02-26 02:05:34.065273	2015-02-26 02:05:34.065273	3	accepted	\N	\N	1	N/A
332	Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.	Resistance or Non-Response	Supports	1	2	19	8	205	93	2015-02-26 02:05:34.126527	2015-02-26 02:05:34.126527	4	accepted	\N	\N	1	N/A
333	In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted in disease free survival of 14 months	Sensitivity	Supports	1	2	60	8	206	94	2015-02-26 02:05:34.206155	2015-02-26 02:05:34.206155	2	accepted	\N	\N	1	1:115256528-115256529 (TT->CA)
334	In a melanoma patient with Q61R mutation, treatment with temozolomide resulted in overall survival of 16 months	Sensitivity	Supports	1	2	60	8	206	95	2015-02-26 02:05:34.260175	2015-02-26 02:05:34.260175	2	accepted	\N	\N	1	1:115256528-115256529 (TT->CC)
335	In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	1	3	61	40	207	96	2015-02-26 02:05:34.323001	2015-02-26 02:05:34.323001	4	accepted	\N	\N	1	4:55152092-55152093 (GA->AT)
336	In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	1	3	61	40	207	97	2015-02-26 02:05:34.393748	2015-02-26 02:05:34.393748	4	accepted	\N	\N	1	4:55152093-55152093 (A->T)
337	GIST cancer with D842V mutation is resistant to imatinib.	Resistance or Non-Response	Supports	1	2	2	40	208	97	2015-02-26 02:05:34.457464	2015-02-26 02:05:34.457464	4	accepted	\N	\N	1	4:55152093-55152093 (A->T)
338	While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib, double mutation of V561D and D842V mutants are resistant to imatinib.	Resistance or Non-Response	Supports	1	2	2	40	163	97	2015-02-26 02:05:34.521055	2015-02-26 02:05:34.521055	4	accepted	\N	\N	1	4:55152093-55152093 (A->T)
339	GIST tumors harboring PDGFRA D842V mutation are more likely to be benign than malignant.	Negative	Supports	3	2	3	40	209	97	2015-02-26 02:05:34.591116	2015-02-26 02:05:34.591116	3	accepted	\N	\N	1	4:55152093-55152093 (A->T)
340	In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	1	3	61	40	207	98	2015-02-26 02:05:34.654471	2015-02-26 02:05:34.654471	4	accepted	\N	\N	1	4:55152092-55152092 (G->T)
341	In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	1	3	61	40	207	99	2015-02-26 02:05:34.723376	2015-02-26 02:05:34.723376	4	accepted	\N	\N	1	N/A
342	In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	1	3	61	40	207	100	2015-02-26 02:05:34.789861	2015-02-26 02:05:34.789861	4	accepted	\N	\N	1	N/A
343	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	1	3	62	12	210	101	2015-02-26 02:05:34.900862	2015-02-26 02:05:34.900862	4	accepted	\N	\N	1	3:178936082-178936082 (G->A)
344	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	1	3	63	12	211	101	2015-02-26 02:05:34.966922	2015-02-26 02:05:34.966922	3	accepted	\N	\N	1	3:178936082-178936082 (G->A)
345	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	1	4	14	7	212	101	2015-02-26 02:05:35.040392	2015-02-26 02:05:35.040392	3	accepted	\N	\N	1	3:178936082-178936082 (G->A)
346	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	1	4	12	7	212	101	2015-02-26 02:05:35.093243	2015-02-26 02:05:35.093243	3	accepted	\N	\N	1	3:178936082-178936082 (G->A)
347	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	Poor Outcome	Supports	2	4	3	7	213	101	2015-02-26 02:05:35.156414	2015-02-26 02:05:35.156414	2	accepted	\N	\N	1	3:178936082-178936082 (G->A)
348	The presence of exon 9 (such as E542K) or exon 20 mutations were associated with poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	2	4	3	7	212	101	2015-02-26 02:05:35.217128	2015-02-26 02:05:35.217128	3	accepted	\N	\N	1	3:178936082-178936082 (G->A)
349	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N / A	Does Not Support	1	2	52	7	166	101	2015-02-26 02:05:35.27411	2015-02-26 02:05:35.27411	4	accepted	\N	\N	1	3:178936082-178936082 (G->A)
350	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	1	3	62	12	210	102	2015-02-26 02:05:35.335034	2015-02-26 02:05:35.335034	4	accepted	\N	\N	1	3:178936091-178936091 (G->A)
351	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	1	3	63	12	211	102	2015-02-26 02:05:35.40634	2015-02-26 02:05:35.40634	3	accepted	\N	\N	1	3:178936091-178936091 (G->A)
352	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	1	4	14	7	212	102	2015-02-26 02:05:35.468553	2015-02-26 02:05:35.468553	3	accepted	\N	\N	1	3:178936091-178936091 (G->A)
353	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	1	4	12	7	212	102	2015-02-26 02:05:35.517858	2015-02-26 02:05:35.517858	3	accepted	\N	\N	1	3:178936091-178936091 (G->A)
354	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	Poor Outcome	Supports	2	4	3	7	213	102	2015-02-26 02:05:35.579769	2015-02-26 02:05:35.579769	2	accepted	\N	\N	1	3:178936091-178936091 (G->A)
355	The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	2	4	3	7	212	102	2015-02-26 02:05:35.644403	2015-02-26 02:05:35.644403	3	accepted	\N	\N	1	3:178936091-178936091 (G->A)
356	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N / A	Does Not Support	1	2	52	7	166	102	2015-02-26 02:05:35.704383	2015-02-26 02:05:35.704383	4	accepted	\N	\N	1	3:178936091-178936091 (G->A)
357	Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	Resistance or Non-Response	Supports	1	2	14	7	71	103	2015-02-26 02:05:35.761832	2015-02-26 02:05:35.761832	3	accepted	\N	\N	1	N/A
358	In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations (Exon 9) had no significant impact on response rate, disease control rate, progression free surivival or overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	N / A	Does Not Support	1	2	14	7	71	104	2015-02-26 02:05:35.82695	2015-02-26 02:05:35.82695	3	accepted	\N	\N	1	N/A
359	Breast cancer cell lines with H1047R mutation showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	1	3	62	12	210	105	2015-02-26 02:05:35.896227	2015-02-26 02:05:35.896227	4	accepted	\N	\N	1	3:178952085-178952085 (A->G)
360	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	1	3	63	12	211	105	2015-02-26 02:05:35.960178	2015-02-26 02:05:35.960178	3	accepted	\N	\N	1	3:178952085-178952085 (A->G)
361	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	1	4	14	7	212	105	2015-02-26 02:05:36.019956	2015-02-26 02:05:36.019956	3	accepted	\N	\N	1	3:178952085-178952085 (A->G)
362	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	1	4	12	7	212	105	2015-02-26 02:05:36.069938	2015-02-26 02:05:36.069938	3	accepted	\N	\N	1	3:178952085-178952085 (A->G)
363	The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	2	4	3	7	212	105	2015-02-26 02:05:36.13729	2015-02-26 02:05:36.13729	3	accepted	\N	\N	1	3:178952085-178952085 (A->G)
364	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N / A	Does Not Support	1	2	52	7	166	105	2015-02-26 02:05:36.193873	2015-02-26 02:05:36.193873	4	accepted	\N	\N	1	3:178952085-178952085 (A->G)
365	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	Sensitivity	Supports	1	2	44	41	214	106	2015-02-26 02:05:36.298445	2015-02-26 02:05:36.298445	5	accepted	\N	\N	1	N/A
366	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	Positive	Supports	3	2	3	21	102	107	2015-02-26 02:05:36.356238	2015-02-26 02:05:36.356238	4	accepted	\N	\N	1	N/A
367	Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse to PI3K-mTOR inhibitors.	Sensitivity	Supports	1	3	3	3	215	108	2015-02-26 02:05:36.414301	2015-02-26 02:05:36.414301	4	accepted	\N	\N	1	10:89717672-89717672 (C->T)
368	PTEN nonsense mutations, including R233*, have been shown to be inactivating and loss-of-function, but do not have prognostic value in glioblastoma multiforme patients.	Poor Outcome	Does Not Support	2	2	3	39	216	108	2015-02-26 02:05:36.486449	2015-02-26 02:05:36.486449	3	accepted	\N	\N	1	10:89717672-89717672 (C->T)
369	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	Sensitivity	Supports	1	2	44	41	214	109	2015-02-26 02:05:36.551665	2015-02-26 02:05:36.551665	5	accepted	\N	\N	1	N/A
370	Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib, compared to wild-type RET.	Resistance or Non-Response	Supports	1	3	64	42	217	110	2015-02-26 02:05:36.61629	2015-02-26 02:05:36.61629	3	accepted	\N	\N	1	10:43609950-43609950 (C->G)
371	The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong repression of tyroid cancer cell growth.	Sensitivity	Supports	1	3	65	42	218	111	2015-02-26 02:05:36.703138	2015-02-26 02:05:36.703138	3	accepted	\N	\N	1	10:43617416-43617416 (T->C)
372	Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib, compared to wild-type RET.	Resistance or Non-Response	Supports	1	3	64	42	217	111	2015-02-26 02:05:36.773081	2015-02-26 02:05:36.773081	3	accepted	\N	\N	1	10:43617416-43617416 (T->C)
373	In patients with medullary carcinoma, the presence of RET M918T mutation is associated with increased probability of lymph node metastases.	Positive	Supports	3	2	3	42	219	111	2015-02-26 02:05:36.841881	2015-02-26 02:05:36.841881	5	accepted	\N	\N	1	10:43617416-43617416 (T->C)
374	Compared to those who harbor a wild type RET, patients with RET M918T mutation develop larger and more aggressive medullary thyroid cancer.	Poor Outcome	Supports	2	2	3	42	220	111	2015-02-26 02:05:36.901275	2015-02-26 02:05:36.901275	4	accepted	\N	\N	1	10:43617416-43617416 (T->C)
375	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	Better Outcome	Supports	2	4	3	29	221	112	2015-02-26 02:05:36.970059	2015-02-26 02:05:36.970059	3	accepted	\N	\N	1	2:198267359-198267359 (C->A)
376	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	Better Outcome	Supports	2	4	3	29	221	113	2015-02-26 02:05:37.038327	2015-02-26 02:05:37.038327	3	accepted	\N	\N	1	2:198266834-198266834 (T->C)
377	Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.	Sensitivity	Supports	1	3	66	12	222	114	2015-02-26 02:05:37.111324	2015-02-26 02:05:37.111324	3	accepted	\N	\N	1	17:7578406-7578406 (C->T)
378	Breast cancer patients who harbor R175H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	2	2	3	12	223	114	2015-02-26 02:05:37.175943	2015-02-26 02:05:37.175943	3	accepted	\N	\N	1	17:7578406-7578406 (C->T)
379	Breast cancer patients who harbor R248Q mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	2	2	3	12	223	115	2015-02-26 02:05:37.237704	2015-02-26 02:05:37.237704	3	accepted	\N	\N	1	17:7577538-7577538 (C->T)
380	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R248 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	2	2	3	12	224	115	2015-02-26 02:05:37.304173	2015-02-26 02:05:37.304173	3	accepted	\N	\N	1	17:7577538-7577538 (C->T)
381	In breast cancer patients harboring R248W mutation, the prognosis is worse than any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.	Poor Outcome	Supports	2	2	3	12	223	116	2015-02-26 02:05:37.376639	2015-02-26 02:05:37.376639	3	accepted	\N	\N	1	17:7577539-7577539 (G->A)
382	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	Poor Outcome	Supports	2	2	3	12	224	117	2015-02-26 02:05:37.437997	2015-02-26 02:05:37.437997	3	accepted	\N	\N	1	17:7577535-7577535 (C->G)
383	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	Poor Outcome	Supports	2	2	3	12	224	118	2015-02-26 02:05:37.496827	2015-02-26 02:05:37.496827	3	accepted	\N	\N	1	17:7577536-7577536 (T->A)
384	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	2	2	3	12	224	119	2015-02-26 02:05:37.560409	2015-02-26 02:05:37.560409	3	accepted	\N	\N	1	17:7577121-7577121 (G->A)
385	Breast cancer patients who harbor R273C mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	2	2	3	12	223	119	2015-02-26 02:05:37.620722	2015-02-26 02:05:37.620722	3	accepted	\N	\N	1	17:7577121-7577121 (G->A)
386	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	2	2	3	12	224	120	2015-02-26 02:05:37.724288	2015-02-26 02:05:37.724288	3	accepted	\N	\N	1	17:7577120-7577120 (C->T)
387	Breast cancer patients who harbor R273H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	2	2	3	12	223	120	2015-02-26 02:05:37.791109	2015-02-26 02:05:37.791109	3	accepted	\N	\N	1	17:7577120-7577120 (C->T)
388	Breast tumor with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.	Poor Outcome	Supports	2	2	3	12	224	121	2015-02-26 02:05:37.855484	2015-02-26 02:05:37.855484	3	accepted	\N	\N	1	N/A
389	A patient with metastatic bladder cancer that responded well to the mTOR inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del, p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations also had mild responses to everolimus.	Sensitivity	Supports	1	2	67	43	225	122	2015-02-26 02:05:37.921355	2015-02-26 02:05:37.921355	3	accepted	\N	\N	1	N/A
390	Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit increased overall survival, increased apoptosis, reduced proliferation and reduced tumor burden compared to Tsc1 wildtype mice.	Sensitivity	Supports	1	3	63	4	226	122	2015-02-26 02:05:37.991103	2015-02-26 02:05:37.991103	3	accepted	\N	\N	1	N/A
391	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	Sensitivity	Supports	1	2	67	24	227	123	2015-02-26 02:05:38.06661	2015-02-26 02:05:38.06661	3	accepted	\N	\N	1	N/A
392	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	Sensitivity	Supports	1	2	67	24	225	124	2015-02-26 02:05:38.135297	2015-02-26 02:05:38.135297	3	accepted	\N	\N	1	N/A
393	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.	Positive	Does Not Support	3	4	3	1	228	125	2015-02-26 02:05:38.198277	2015-02-26 02:05:38.198277	3	accepted	\N	\N	1	N/A
394	In patients with AML, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	N / A	Does Not Support	2	4	3	1	228	125	2015-02-26 02:05:38.254671	2015-02-26 02:05:38.254671	2	accepted	\N	\N	1	N/A
395	In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	N / A	Does Not Support	2	4	3	29	228	125	2015-02-26 02:05:38.314821	2015-02-26 02:05:38.314821	2	accepted	\N	\N	1	N/A
396	After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.	Poor Outcome	Supports	2	4	3	29	229	125	2015-02-26 02:05:38.37261	2015-02-26 02:05:38.37261	3	accepted	\N	\N	1	N/A
397	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.	Positive	Does Not Support	3	4	3	1	228	126	2015-02-26 02:05:38.432984	2015-02-26 02:05:38.432984	3	accepted	\N	\N	1	N/A
398	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	N / A	Does Not Support	2	4	3	1	228	126	2015-02-26 02:05:38.495398	2015-02-26 02:05:38.495398	2	accepted	\N	\N	1	N/A
399	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	N / A	Does Not Support	2	4	3	29	228	126	2015-02-26 02:05:38.55283	2015-02-26 02:05:38.55283	2	accepted	\N	\N	1	N/A
400	After adjust for age and cancer stage, the presence of U2AF mutations such as S34Y/F are prognostic for poorer survival outcomes in patients with MDS.	Poor Outcome	Supports	2	4	3	29	229	126	2015-02-26 02:05:38.607868	2015-02-26 02:05:38.607868	3	accepted	\N	\N	1	N/A
401	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	Resistance or Non-Response	Supports	1	2	3	1	230	127	2015-02-26 02:05:38.675675	2015-02-26 02:05:38.675675	3	accepted	\N	\N	1	N/A
402	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	N / A	Does Not Support	2	2	3	1	231	127	2015-02-26 02:05:38.733669	2015-02-26 02:05:38.733669	3	accepted	\N	\N	1	N/A
403	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML	N / A	Does Not Support	2	2	3	1	231	127	2015-02-26 02:05:38.790182	2015-02-26 02:05:38.790182	4	accepted	\N	\N	1	N/A
404	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	2	2	3	1	230	127	2015-02-26 02:05:38.852973	2015-02-26 02:05:38.852973	3	accepted	\N	\N	1	N/A
405	Mutations  in WT1 were associated with increased risk of recurrence in young patients (15-50) with cytogenetically normal AML	Poor Outcome	Supports	2	2	3	1	232	127	2015-02-26 02:05:38.92449	2015-02-26 02:05:38.92449	2	accepted	\N	\N	1	N/A
406	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	2	2	3	1	230	127	2015-02-26 02:05:38.982868	2015-02-26 02:05:38.982868	3	accepted	\N	\N	1	N/A
407	Mutations in WT1 were associated with a worse overall prognosis than patients wildtype for WT1 in young patients (15-50), primarily because of increased risk of disease recurrence	Poor Outcome	Supports	2	2	3	1	232	127	2015-02-26 02:05:39.094662	2015-02-26 02:05:39.094662	2	accepted	\N	\N	1	N/A
408	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	Poor Outcome	Supports	2	2	3	1	231	127	2015-02-26 02:05:39.158473	2015-02-26 02:05:39.158473	3	accepted	\N	\N	1	N/A
409	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	Poor Outcome	Supports	2	2	3	1	233	127	2015-02-26 02:05:39.228275	2015-02-26 02:05:39.228275	3	accepted	\N	\N	1	N/A
410	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	Resistance or Non-Response	Supports	1	2	3	1	230	128	2015-02-26 02:05:39.286246	2015-02-26 02:05:39.286246	3	accepted	\N	\N	1	N/A
411	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML.	N / A	Does Not Support	2	2	3	1	231	128	2015-02-26 02:05:39.346255	2015-02-26 02:05:39.346255	4	accepted	\N	\N	1	N/A
412	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	N / A	Does Not Support	2	2	3	1	231	128	2015-02-26 02:05:39.414957	2015-02-26 02:05:39.414957	3	accepted	\N	\N	1	N/A
413	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	Poor Outcome	Supports	2	2	3	1	231	128	2015-02-26 02:05:39.475229	2015-02-26 02:05:39.475229	3	accepted	\N	\N	1	N/A
414	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	2	2	3	1	230	128	2015-02-26 02:05:39.542775	2015-02-26 02:05:39.542775	3	accepted	\N	\N	1	N/A
415	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	2	2	3	1	230	128	2015-02-26 02:05:39.602454	2015-02-26 02:05:39.602454	3	accepted	\N	\N	1	N/A
416	WT1 mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	Poor Outcome	Supports	2	2	3	1	233	128	2015-02-26 02:05:39.662491	2015-02-26 02:05:39.662491	3	accepted	\N	\N	1	N/A
\.


ALTER TABLE evidence_items ENABLE TRIGGER ALL;

--
-- Name: evidence_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_items_id_seq', 416, true);


--
-- Data for Name: evidence_levels; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE evidence_levels DISABLE TRIGGER ALL;

COPY evidence_levels (id, level, description, created_at, updated_at) FROM stdin;
1	A	\N	2015-02-26 02:05:12.335766	2015-02-26 02:05:12.335766
2	B	\N	2015-02-26 02:05:12.414091	2015-02-26 02:05:12.414091
3	C	\N	2015-02-26 02:05:12.634114	2015-02-26 02:05:12.634114
4	D	\N	2015-02-26 02:05:16.317532	2015-02-26 02:05:16.317532
\.


ALTER TABLE evidence_levels ENABLE TRIGGER ALL;

--
-- Name: evidence_levels_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_levels_id_seq', 4, true);


--
-- Data for Name: evidence_types; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE evidence_types DISABLE TRIGGER ALL;

COPY evidence_types (id, evidence_type, created_at, updated_at) FROM stdin;
1	Predictive	2015-02-26 02:05:12.332727	2015-02-26 02:05:12.332727
2	Prognostic	2015-02-26 02:05:12.57101	2015-02-26 02:05:12.57101
3	Diagnostic	2015-02-26 02:05:12.632424	2015-02-26 02:05:12.632424
\.


ALTER TABLE evidence_types ENABLE TRIGGER ALL;

--
-- Name: evidence_types_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_types_id_seq', 3, true);


--
-- Data for Name: genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE genes DISABLE TRIGGER ALL;

COPY genes (id, entrez_id, name, description, official_name, created_at, updated_at, clinical_description) FROM stdin;
1	238	ALK	ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines.	anaplastic lymphoma receptor tyrosine kinase	2015-02-26 02:04:56.54554	2015-02-26 02:04:56.54554	\N
2	207	AKT1	AKT1, also referred to as protein kinase B, is a known oncogene. AKT activation relies on the PI3K pathway, and is recognized as a critical node in the pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been shown to result in activation of the protein. Mutations in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors in vitro.	v-akt murine thymoma viral oncogene homolog 1	2015-02-26 02:04:56.830749	2015-02-26 02:04:56.830749	\N
3	369	ARAF	ARAF has recently become increasingly considered for its oncogenic potential. Its potential as a target for informing clinical action was demonstrated by a single case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated with sorafenib, acheived near-complete clinical remission. This finding has brought new focus on ARAF as a marker that should be assayed for in cancer treatment.	A-Raf proto-oncogene, serine/threonine kinase	2015-02-26 02:04:57.074904	2015-02-26 02:04:57.074904	\N
4	25	ABL1	ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein that has become a signature of chronic myeloid leukemia (CML). Cells harboring this fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook of the disease. However, additional mutations in ABL1 have been shown to confer resistance to imatinib. In these resistance cases, second-generation tyrosine kinase inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are currently undergoing clinical trials for treating acquired resistance in CML.	ABL proto-oncogene 1, non-receptor tyrosine kinase	2015-02-26 02:04:57.350556	2015-02-26 02:04:57.350556	\N
5	673	BRAF	BRAF mutations are found to be recurrent in many cancer types. Of these, the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E has been determined to be an activating mutation, and cells that harbor it, along with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations have also been correlated with poor prognosis in many cancer types, although there is at least one study that questions this conclusion in papillary thyroid cancer.	B-Raf proto-oncogene, serine/threonine kinase	2015-02-26 02:04:57.614013	2015-02-26 02:04:57.614013	\N
6	672	BRCA1	BRCA1 mutations in the germline have become a hallmark for hereditary breast and ovarian cancers. Variants that have been demonstrated to reduce the function of the protein have been shown to increase risk for these cancers, as well as prostate and pancreatic cancer. These findings have been the impetus for the increased popularity of genetic testing of healthy indivudals to assess risk.	breast cancer 1, early onset	2015-02-26 02:04:57.886997	2015-02-26 02:04:57.886997	\N
7	675	BRCA2	BRCA2 mutations in the germline have become a hallmark for hereditary breast and ovarian cancers. Variants that have been demonstrated to reduce the function of the protein have been shown to increase risk for these cancers, as well as prostate and pancreatic cancer. These findings have been the impetus for the increased popularity of genetic testing of healthy indivudals to assess risk.	breast cancer 2, early onset	2015-02-26 02:04:58.11663	2015-02-26 02:04:58.11663	\N
8	595	CCND1	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D1	2015-02-26 02:04:58.336866	2015-02-26 02:04:58.336866	\N
9	894	CCND2	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D2	2015-02-26 02:04:58.5697	2015-02-26 02:04:58.5697	\N
10	896	CCND3	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. In a mouse study, the targeted therapeutic palbociclib significantly increased the median survival of the cyclin D3 knockouts.	cyclin D3	2015-02-26 02:04:58.822846	2015-02-26 02:04:58.822846	\N
11	898	CCNE1	Cyclin E, while currenly not as widely implicated as its cyclin D counterparts, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements are correlated with later stage disease. In lung cancer, neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly expressed counterparts.	cyclin E1	2015-02-26 02:04:59.057641	2015-02-26 02:04:59.057641	\N
12	1021	CDK6	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 6	2015-02-26 02:04:59.289466	2015-02-26 02:04:59.289466	\N
13	1019	CDK4	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 4	2015-02-26 02:04:59.53405	2015-02-26 02:04:59.53405	\N
14	1029	CDKN2A	CDKN2A loss has been shown to be a significant event in a number of cancer types. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. One mechanism by which loss of CDKN2A can occur is by hypermethylation of the promoter region for the gene. However, the prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have suggesting poorer prognostic outcome for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	cyclin-dependent kinase inhibitor 2A	2015-02-26 02:04:59.763831	2015-02-26 02:04:59.763831	\N
15	1050	CEBPA	AML with mutated CEBPA' is a provisional entity in the WHO classification of acute myeloid leukemia (AML) and is recommended to be tested for in patients with AML. CEBPA mutations are particularly associated with cytogenetically normal AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha protein while C-terminal mutations reduce the DNA-binding potential of this transcription factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	CCAAT/enhancer binding protein (C/EBP), alpha	2015-02-26 02:05:00.011016	2015-02-26 02:05:00.011016	\N
16	3337	DNAJB1	DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	DnaJ (Hsp40) homolog, subfamily B, member 1	2015-02-26 02:05:00.2545	2015-02-26 02:05:00.2545	\N
17	5566	PRKACA	PRKACA has been studied in breast cancer and has been found to mediate resistance to HER2 targeted therapies. It has also been found to contain a mutation hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	protein kinase, cAMP-dependent, catalytic, alpha	2015-02-26 02:05:00.484359	2015-02-26 02:05:00.484359	\N
18	1788	DNMT3A	DNMT3A is one of several epigenetic modifiers identified as recurrently mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers. AML patient bone marrow harboring R882 mutations were similarly demonstrated to be hypomethylated compared to patients with wildtype DNMT3A. These studies also indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes and treatment protocols may need to be considered for these two classes of DNMT3A mutations.	DNA (cytosine-5-)-methyltransferase 3 alpha	2015-02-26 02:05:00.746868	2015-02-26 02:05:00.746868	\N
19	1956	EGFR	EGFR is widely recognized for its importance in cancer. Amplification and mutations have been shown to be driving events in many cancer types. Its role in non-small cell lung cancer has spurred many research and drug development efforts. Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance to these drugs, particularly the variant T790M, which has been functionally characterized as a resistance marker for both of these drugs. The later generation TKI's have seen some success in treating these resistant cases, and targeted sequencing of the EGFR locus has become a common practice in non-small cell lung cancer.	epidermal growth factor receptor	2015-02-26 02:05:01.036186	2015-02-26 02:05:01.036186	\N
20	2064	ERBB2	ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast cancer, and is treated in a separate manner than the other subtypes of breast cancer. Apart from being amplified, ERBB2 activating mutations have been shown to have clinical importance in HER2-negative breast cancer. These mutations have shown sensitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical sequencing efforts in treating breast cancer.	erb-b2 receptor tyrosine kinase 2	2015-02-26 02:05:01.322919	2015-02-26 02:05:01.322919	\N
21	2099	ESR1	ESR1 has been a focus in breast cancer for quite some time, but has also shown significance in endometrial, ovarian and other cancer types. ER-positive breast cancer that is resistant to hormone therapy has instigated clinical sequencing efforts to shed light on the mechanisms of this resistance. A number of mutations in the ligand binding domain of the protein have been implicated in hormone resistance and anti-estrogen therapies. This has spurred efforts to develop therapeutics that act to degrade the protein, rather than act as an antagonist. These agents are currently in clinical trials and have seen some success, highlighting the importance of sequencing efforts in treating breast cancer.	estrogen receptor 1	2015-02-26 02:05:01.613895	2015-02-26 02:05:01.613895	\N
22	2263	FGFR2	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions that lead to pathway actiation have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 2	2015-02-26 02:05:01.892338	2015-02-26 02:05:01.892338	\N
23	2261	FGFR3	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions that lead to pathway actiation have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 3	2015-02-26 02:05:02.164891	2015-02-26 02:05:02.164891	\N
24	2322	FLT3	FLT3 is an important cytokine receptor involved in normal hematopoiesis. Mutations in this gene are common in acute myeloid leukemia (AML) and screening for mutations in this gene has been recommended by the World Health Organization in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML). FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD mutations are still up for debate.	fms-related tyrosine kinase 3	2015-02-26 02:05:02.453283	2015-02-26 02:05:02.453283	\N
25	2624	GATA2	GATA2 is a transcription factor involved in stem cell maintenance with key roles in hematopoietic development. GATA2 mutations are associated with a variety of inherited and acquired immune disorders including myelodysplastic syndrome and acute myeloid leukemia. In addition to a role in hematopoiesis, the maintenance GATA2 expression has been implicated as a requirement in KRAS-driven non-small cell lung cancer. Preclinical models have indicated therapeutic benefit from targeting GATA2-mediated pathways in the context of KRAS-driven NSCLC.	GATA binding protein 2	2015-02-26 02:05:02.70778	2015-02-26 02:05:02.70778	\N
26	3417	IDH1	IDH1 mutations have been observed in a number of cancer types, including sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described as mutually exclusive in many of these cancer types. The most frequent mutations involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic enzyme activity. The implications of mutations in this gene vary greatly by cancer type. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations have been associated with worse outcome, shorter overall survival, and normal karyotype. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Unlike the association with cytogenetically normal AML, in glioblastoma, IDH1 mutations have been associated with specific cytogenetic abnormalities, 1p and 19q deletions.	isocitrate dehydrogenase 1 (NADP+), soluble	2015-02-26 02:05:02.955965	2015-02-26 02:05:02.955965	\N
27	3418	IDH2	IDH2 mutations have been observed in a number of cancer types, including sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described as mutually exclusive in many of these cancer types. The most frequent mutations involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic enzyme activity. Although IDH2 (R172) mutations are associated with poorer overall prognosis in AML patients, its utility as a prognostic marker in MDS is still under debate. Additionally, IDH2 (R140) has been associated with improved overall survival in AML. IDH2 mutations have been associated with improved prognosis in gliomas.	isocitrate dehydrogenase 2 (NADP+), mitochondrial	2015-02-26 02:05:03.225075	2015-02-26 02:05:03.225075	\N
28	3717	JAK2	JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative diseases and cancers. The mutation V617F is the most clinically relevant variant, and is seen in around half of myeloproliferative disorders. The variant is a known activating mutation, and activated JAK2 is sufficient to drive myeloproliferative disorders in mouse models. V617F, while most recurrent, is not the only mechanism by which JAK2 can be activated in patients. JAK2 is now one of the first diagnostic markers tested upon diagnosis with a myeloproliferative disorder.	Janus kinase 2	2015-02-26 02:05:03.48818	2015-02-26 02:05:03.48818	\N
29	3815	KIT	c-KIT activation has been shown to have oncogenic activity in gastrointestinal stromal tumors (GISTs), melanomas, lung cancer, and other tumor types. The targeted therapeutics nilotinib and sunitinib have shown efficacy in treating KIT overactive patients, and are in late-stage trials in melanoma and GIST. KIT overactivity can be the result of many genomic events from genomic amplification to overexpression to missense mutations. Missense mutations have been shown to be key players in mediating clinical response and acquired resistance in patients being treated with these targeted therapeutics.	v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog	2015-02-26 02:05:03.728698	2015-02-26 02:05:03.728698	\N
30	3845	KRAS	Mutations in the RAS family of proteins have frequently observed across cancer types. The amino acid positions account for the overwhelming majority of these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also behave very differently when expressed in non-native tissue types, likely due to differences in the C-terminal hyper-variable regions. Mis-regulation of isoform expression has been shown to be a driving event in cancer, as well as missense mutations at the three hotspots previously mentioned. While highly recurrent in cancer, targeted these RAS mutants has also been very elusive, and has not yet become common practice in the clinic.	Kirsten rat sarcoma viral oncogene homolog	2015-02-26 02:05:03.983895	2015-02-26 02:05:03.983895	\N
31	5604	MAP2K1	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	mitogen-activated protein kinase kinase 1	2015-02-26 02:05:04.224638	2015-02-26 02:05:04.224638	\N
32	4209	MEF2D	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	myocyte enhancer factor 2D	2015-02-26 02:05:04.452121	2015-02-26 02:05:04.452121	\N
33	1436	CSF1R	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	colony stimulating factor 1 receptor	2015-02-26 02:05:04.68801	2015-02-26 02:05:04.68801	\N
34	4255	MGMT	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	O-6-methylguanine-DNA methyltransferase	2015-02-26 02:05:04.900021	2015-02-26 02:05:04.900021	\N
35	4869	NPM1	AML with mutated NPM1 is provisional entity in the WHO classification of AML and is recommended to be tested for in patients with cytogenetically normal AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia, indicating it may require additional mutations to promote leukemic development.	nucleophosmin (nucleolar phosphoprotein B23, numatrin)	2015-02-26 02:05:05.118993	2015-02-26 02:05:05.118993	\N
36	4893	NRAS	Mutations in the RAS family of proteins have frequently observed across cancer types. The amino acid positions account for the overwhelming majority of these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also behave very differently when expressed in non-native tissue types, likely due to differences in the C-terminal hyper-variable regions. Mis-regulation of isoform expression has been shown to be a driving event in cancer, as well as missense mutations at the three hotspots previously mentioned. While highly recurrent in cancer, targeted these RAS mutants has also been very elusive, and has not yet become common practice in the clinic.	neuroblastoma RAS viral (v-ras) oncogene homolog	2015-02-26 02:05:05.372785	2015-02-26 02:05:05.372785	\N
37	5290	PIK3CA	PIK3CA is the most recurrently mutated gene in breast cancer, and has been found to important in a number of cancer types. An integral part of the PI3K pathway, PIK3CA has long been described as an oncogene, with two main hotspots for activating mutations, the 542/545 region of the helical domain, and the 1047 region of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the subject of an immense amount of research and development, and PI3K inhibition has seen some limited success in recent clinical trials. While monotherapies seem to be limited in their potential, there is a recent interest in pursuing PI3K inhibition as part of a combination therapy regiment with inhibition partners including TKI's, MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.	phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha	2015-02-26 02:05:05.605358	2015-02-26 02:05:05.605358	\N
38	5156	PDGFRA	N/A	platelet-derived growth factor receptor, alpha polypeptide	2015-02-26 02:05:05.831357	2015-02-26 02:05:05.831357	\N
39	5371	PML	The PML-RARA fusion is the result of a recurrent, balanced translocation between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	promyelocytic leukemia	2015-02-26 02:05:06.08972	2015-02-26 02:05:06.08972	\N
40	5914	RARA	The PML-RARA fusion is the result of a recurrent, balanced translocation between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	retinoic acid receptor, alpha	2015-02-26 02:05:06.312624	2015-02-26 02:05:06.312624	\N
41	5728	PTEN	N/A	phosphatase and tensin homolog	2015-02-26 02:05:06.536446	2015-02-26 02:05:06.536446	\N
42	5979	RET	RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine kinase receptor and are associated with thyroid cancers. RET point mutations are the most common mutations identified in medullary thyroid cancer (MTC) with germline and somatic mutations in RET associated with hereditary and sporadic forms, respectively. The most common somatic form mutation is M918T (exon 16) and a variety of other mutations effecting exons 10, 11 and 15 have been described. The prognostic significance of these mutations have been hotly debated in the field, however, data suggests that some RET mutation may confer drug resistence. No RET-specific agents are currently clinically available but several promiscuous kinase inhibitors that target RET, among others, have been approved for MTC treatment.	ret proto-oncogene	2015-02-26 02:05:06.839311	2015-02-26 02:05:06.839311	\N
43	861	RUNX1	N/A	runt-related transcription factor 1	2015-02-26 02:05:07.075112	2015-02-26 02:05:07.075112	\N
44	23451	SF3B1	SF3B1 mutations have been described in several myeloid malignancies, predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has been identified as recurrently mutated in MDS and other malignanices. The mutations affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally deleterious with K700E described as a major hotspot mutation. MDS patients with SF3B1 mutations have been reported to have better overall and event-free survival than their wildtype counterparts. Additionally, these mutations are highly associated with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring sideroblasts).	splicing factor 3b, subunit 1, 155kDa	2015-02-26 02:05:07.295908	2015-02-26 02:05:07.295908	\N
45	7157	TP53	TP53 mutations are universal across cancer types. Loss of tumor suppressors is most recognized by large deleterious events, such as frameshift mutations, or premature stop codons. In TP53 however, many of the observed mutations in cancer are found to be single nucleotide variants, or missense mutations. These variants are also very broadly distributed throughout the gene, not localizing in any particular hotspot. While a large proportion of cancer genomics research is focused on somatic variants, TP53 is also of note in the germline. Germline TP53 mutations are the hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been found to have prognostic impact on patient outcomes.	tumor protein p53	2015-02-26 02:05:07.55639	2015-02-26 02:05:07.55639	\N
46	7248	TSC1	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	tuberous sclerosis 1	2015-02-26 02:05:07.78231	2015-02-26 02:05:07.78231	\N
47	7249	TSC2	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	tuberous sclerosis 2	2015-02-26 02:05:08.033932	2015-02-26 02:05:08.033932	\N
48	7307	U2AF1	U2AF1 is one of several spliceosome complex genes frequently mutated in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein. These mutations have been associated with altered splicing patterns in vitro and in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation to secondary acute myeloid leukemia, however, the impact of these mutations on overall survival has been an area of debate.	U2 small nuclear RNA auxiliary factor 1	2015-02-26 02:05:08.308545	2015-02-26 02:05:08.308545	\N
49	7490	WT1	WT1 is a tumor suppressor gene associated with the development of Wilms' Tumor, from which it was named. Mutations in exon 7 and 9 of WT1 have been recurrently identified in acute myeloid leukemia and associated with poorer prognosis and chemotherapy resistance.	Wilms tumor 1	2015-02-26 02:05:08.552196	2015-02-26 02:05:08.552196	\N
\.


ALTER TABLE genes ENABLE TRIGGER ALL;

--
-- Name: genes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('genes_id_seq', 49, true);


--
-- Data for Name: genes_sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE genes_sources DISABLE TRIGGER ALL;

COPY genes_sources (gene_id, source_id, created_at, updated_at) FROM stdin;
1	1	2015-02-26 02:04:56.55758	2015-02-26 02:04:56.55758
1	2	2015-02-26 02:04:56.561806	2015-02-26 02:04:56.561806
2	3	2015-02-26 02:04:56.836152	2015-02-26 02:04:56.836152
2	4	2015-02-26 02:04:56.840778	2015-02-26 02:04:56.840778
3	5	2015-02-26 02:04:57.080241	2015-02-26 02:04:57.080241
3	6	2015-02-26 02:04:57.084829	2015-02-26 02:04:57.084829
4	7	2015-02-26 02:04:57.355858	2015-02-26 02:04:57.355858
4	8	2015-02-26 02:04:57.360269	2015-02-26 02:04:57.360269
5	9	2015-02-26 02:04:57.619748	2015-02-26 02:04:57.619748
5	10	2015-02-26 02:04:57.62474	2015-02-26 02:04:57.62474
6	11	2015-02-26 02:04:57.892065	2015-02-26 02:04:57.892065
7	11	2015-02-26 02:04:58.119136	2015-02-26 02:04:58.119136
8	12	2015-02-26 02:04:58.341959	2015-02-26 02:04:58.341959
8	13	2015-02-26 02:04:58.347643	2015-02-26 02:04:58.347643
9	12	2015-02-26 02:04:58.571622	2015-02-26 02:04:58.571622
9	13	2015-02-26 02:04:58.573767	2015-02-26 02:04:58.573767
10	12	2015-02-26 02:04:58.825914	2015-02-26 02:04:58.825914
10	13	2015-02-26 02:04:58.829045	2015-02-26 02:04:58.829045
11	14	2015-02-26 02:04:59.062534	2015-02-26 02:04:59.062534
12	15	2015-02-26 02:04:59.294728	2015-02-26 02:04:59.294728
12	16	2015-02-26 02:04:59.30018	2015-02-26 02:04:59.30018
13	15	2015-02-26 02:04:59.536997	2015-02-26 02:04:59.536997
13	16	2015-02-26 02:04:59.540346	2015-02-26 02:04:59.540346
14	17	2015-02-26 02:04:59.768837	2015-02-26 02:04:59.768837
14	18	2015-02-26 02:04:59.773233	2015-02-26 02:04:59.773233
15	19	2015-02-26 02:05:00.01663	2015-02-26 02:05:00.01663
15	20	2015-02-26 02:05:00.020514	2015-02-26 02:05:00.020514
15	21	2015-02-26 02:05:00.025108	2015-02-26 02:05:00.025108
16	22	2015-02-26 02:05:00.258762	2015-02-26 02:05:00.258762
17	22	2015-02-26 02:05:00.487479	2015-02-26 02:05:00.487479
17	23	2015-02-26 02:05:00.493835	2015-02-26 02:05:00.493835
18	24	2015-02-26 02:05:00.752172	2015-02-26 02:05:00.752172
18	25	2015-02-26 02:05:00.756664	2015-02-26 02:05:00.756664
19	26	2015-02-26 02:05:01.041377	2015-02-26 02:05:01.041377
19	27	2015-02-26 02:05:01.045837	2015-02-26 02:05:01.045837
20	28	2015-02-26 02:05:01.328461	2015-02-26 02:05:01.328461
20	29	2015-02-26 02:05:01.333656	2015-02-26 02:05:01.333656
21	30	2015-02-26 02:05:01.61987	2015-02-26 02:05:01.61987
21	31	2015-02-26 02:05:01.624752	2015-02-26 02:05:01.624752
22	32	2015-02-26 02:05:01.898091	2015-02-26 02:05:01.898091
22	33	2015-02-26 02:05:01.903443	2015-02-26 02:05:01.903443
23	32	2015-02-26 02:05:02.16816	2015-02-26 02:05:02.16816
23	33	2015-02-26 02:05:02.171465	2015-02-26 02:05:02.171465
24	34	2015-02-26 02:05:02.458675	2015-02-26 02:05:02.458675
24	19	2015-02-26 02:05:02.461703	2015-02-26 02:05:02.461703
25	35	2015-02-26 02:05:02.713326	2015-02-26 02:05:02.713326
25	36	2015-02-26 02:05:02.719973	2015-02-26 02:05:02.719973
26	37	2015-02-26 02:05:02.96202	2015-02-26 02:05:02.96202
26	38	2015-02-26 02:05:02.96737	2015-02-26 02:05:02.96737
27	37	2015-02-26 02:05:03.228232	2015-02-26 02:05:03.228232
27	38	2015-02-26 02:05:03.231425	2015-02-26 02:05:03.231425
27	39	2015-02-26 02:05:03.235648	2015-02-26 02:05:03.235648
28	40	2015-02-26 02:05:03.493716	2015-02-26 02:05:03.493716
29	41	2015-02-26 02:05:03.734033	2015-02-26 02:05:03.734033
30	42	2015-02-26 02:05:03.990647	2015-02-26 02:05:03.990647
35	43	2015-02-26 02:05:05.124662	2015-02-26 02:05:05.124662
35	19	2015-02-26 02:05:05.127892	2015-02-26 02:05:05.127892
42	44	2015-02-26 02:05:06.844574	2015-02-26 02:05:06.844574
42	45	2015-02-26 02:05:06.850541	2015-02-26 02:05:06.850541
44	46	2015-02-26 02:05:07.301253	2015-02-26 02:05:07.301253
44	47	2015-02-26 02:05:07.306396	2015-02-26 02:05:07.306396
45	48	2015-02-26 02:05:07.562359	2015-02-26 02:05:07.562359
48	49	2015-02-26 02:05:08.314057	2015-02-26 02:05:08.314057
48	50	2015-02-26 02:05:08.319211	2015-02-26 02:05:08.319211
\.


ALTER TABLE genes_sources ENABLE TRIGGER ALL;

--
-- Data for Name: notifications; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE notifications DISABLE TRIGGER ALL;

COPY notifications (id, subscription_id, user_id, subscribable_id, subscribable_type, content, url, acknowledged, delivered, created_at, updated_at) FROM stdin;
\.


ALTER TABLE notifications ENABLE TRIGGER ALL;

--
-- Name: notifications_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('notifications_id_seq', 1, false);


--
-- Data for Name: ratings; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE ratings DISABLE TRIGGER ALL;

COPY ratings (id, value, evidence_item_id, user_id, created_at, updated_at) FROM stdin;
\.


ALTER TABLE ratings ENABLE TRIGGER ALL;

--
-- Name: ratings_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('ratings_id_seq', 1, false);


--
-- Data for Name: roles; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE roles DISABLE TRIGGER ALL;

COPY roles (id, name, description, created_at, updated_at) FROM stdin;
\.


ALTER TABLE roles ENABLE TRIGGER ALL;

--
-- Name: roles_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('roles_id_seq', 1, false);


--
-- Data for Name: roles_users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE roles_users DISABLE TRIGGER ALL;

COPY roles_users (role_id, user_id, created_at, updated_at) FROM stdin;
\.


ALTER TABLE roles_users ENABLE TRIGGER ALL;

--
-- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE schema_migrations DISABLE TRIGGER ALL;

COPY schema_migrations (version) FROM stdin;
20150223170106
20140805184308
20141021192035
20141022180523
20141022195329
20141108234203
20141108234935
20141109000136
20141114222425
20150107205621
20150115231117
20150116220720
20150116230539
20150116230632
20150204192653
20150205000010
20150205205734
20150205210725
20150210191015
20150210204445
20150219202227
20150220160805
20150220182109
\.


ALTER TABLE schema_migrations ENABLE TRIGGER ALL;

--
-- Data for Name: sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE sources DISABLE TRIGGER ALL;

COPY sources (id, pubmed_id, study_type, description, created_at, updated_at) FROM stdin;
1	24889366	\N	Rossi et al., 2014, Int. J. Oncol.	2015-02-26 02:04:56.555446	2015-02-26 02:05:40.474628
2	23401436	\N	Shaw et al., 2013, J. Clin. Oncol.	2015-02-26 02:04:56.56036	2015-02-26 02:05:41.075168
3	16095999	\N	Bellacosa et al., 2005, Adv. Cancer Res.	2015-02-26 02:04:56.833896	2015-02-26 02:05:41.672518
4	15023437	\N	Fresno Vara et al., 2004, Cancer Treat. Rev.	2015-02-26 02:04:56.838955	2015-02-26 02:05:42.066688
5	15676015	\N	Lee et al., 2005, APMIS	2015-02-26 02:04:57.078001	2015-02-26 02:05:42.424362
6	24569458	\N	Imielinski et al., 2014, J. Clin. Invest.	2015-02-26 02:04:57.083083	2015-02-26 02:05:42.917427
7	15719031	\N	Ren, 2005, Nat. Rev. Cancer	2015-02-26 02:04:57.35372	2015-02-26 02:05:43.326646
8	17457302	\N	Weisberg et al., 2007, Nat. Rev. Cancer	2015-02-26 02:04:57.358621	2015-02-26 02:05:43.734888
9	19724843	\N	Li et al., 2009, Oncol. Rep.	2015-02-26 02:04:57.61715	2015-02-26 02:05:44.131204
10	23594689	\N	Pakneshan et al., 2013, Pathology	2015-02-26 02:04:57.623025	2015-02-26 02:05:44.555475
11	24366442	\N	Nelson et al., 2014, Ann. Intern. Med.	2015-02-26 02:04:57.889987	2015-02-26 02:05:45.270238
12	12432268	\N	Diehl, Cancer Biol. Ther.	2015-02-26 02:04:58.339866	2015-02-26 02:05:45.72454
13	24387133	\N	Casimiro et al., 2014, Expert Opin Investig Drugs	2015-02-26 02:04:58.345415	2015-02-26 02:05:46.193059
14	14965268	\N	Mazumder et al., 2004, Curr Cancer Drug Targets	2015-02-26 02:04:59.060499	2015-02-26 02:05:46.767649
15	20370706	\N	Graf et al., 2010, Mini Rev Med Chem	2015-02-26 02:04:59.292457	2015-02-26 02:05:47.216029
16	24089445	\N	Sheppard et al., 2013, Clin. Cancer Res.	2015-02-26 02:04:59.298035	2015-02-26 02:05:47.627483
17	20473920	\N	Shima et al., 2011, Int. J. Cancer	2015-02-26 02:04:59.766852	2015-02-26 02:05:48.195658
18	23111194	\N	Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-26 02:04:59.771573	2015-02-26 02:05:48.65062
19	19357394	\N	Vardiman et al., 2009, Blood	2015-02-26 02:05:00.014505	2015-02-26 02:05:49.878026
20	11242107	\N	Pabst et al., 2001, Nat. Genet.	2015-02-26 02:05:00.019038	2015-02-26 02:05:50.287047
21	9012825	\N	Zhang et al., 1997, Proc. Natl. Acad. Sci. U.S.A.	2015-02-26 02:05:00.023299	2015-02-26 02:05:52.028975
22	24578576	\N	Honeyman et al., 2014, Science	2015-02-26 02:05:00.256971	2015-02-26 02:05:52.442087
23	24909179	\N	Moody et al., 2014, Oncogene	2015-02-26 02:05:00.491266	2015-02-26 02:05:52.950262
24	24167195	\N	Kim et al., 2013, Blood	2015-02-26 02:05:00.749934	2015-02-26 02:05:53.871865
25	24656771	\N	Russler-Germain et al., 2014, Cancer Cell	2015-02-26 02:05:00.755015	2015-02-26 02:05:54.387548
26	23953842	\N	Yewale et al., 2013, Biomaterials	2015-02-26 02:05:01.039304	2015-02-26 02:05:54.792608
27	18712184	\N	Charpidou et al., In Vivo	2015-02-26 02:05:01.044244	2015-02-26 02:05:55.204997
28	16947083	\N	Badache et al., 2006, J Mammary Gland Biol Neoplasia	2015-02-26 02:05:01.326099	2015-02-26 02:05:55.614795
29	10878580	\N	Yu et al., 2000, Bioessays	2015-02-26 02:05:01.331788	2015-02-26 02:05:56.02233
30	24185510	\N	Robinson et al., 2013, Nat. Genet.	2015-02-26 02:05:01.617363	2015-02-26 02:05:56.537986
31	22279420	\N	Mohibi et al., 2011, J Carcinog	2015-02-26 02:05:01.622724	2015-02-26 02:05:56.944788
32	18636142	\N	Katoh, 2008, Int. J. Oncol.	2015-02-26 02:05:01.895638	2015-02-26 02:05:57.353545
33	23558953	\N	Wu et al., 2013, Cancer Discov	2015-02-26 02:05:01.901199	2015-02-26 02:05:57.789679
34	12951584	\N	Stirewalt et al., 2003, Nat. Rev. Cancer	2015-02-26 02:05:02.456625	2015-02-26 02:05:58.276335
35	22541434	\N	Kumar et al., 2012, Cell	2015-02-26 02:05:02.711041	2015-02-26 02:05:58.686516
36	25619630	\N	Mir et al., 2015, Cancer Med	2015-02-26 02:05:02.717464	2015-02-26 02:05:59.197037
37	24065766	\N	Lu et al., 2013, Genes Dev.	2015-02-26 02:05:02.959539	2015-02-26 02:05:59.71153
38	19228619	\N	Yan et al., 2009, N. Engl. J. Med.	2015-02-26 02:05:02.965229	2015-02-26 02:06:00.166079
39	22417203	\N	Patel et al., 2012, N. Engl. J. Med.	2015-02-26 02:05:03.233994	2015-02-26 02:06:00.700748
40	17133099	\N	Levine et al., 2007, Curr. Opin. Hematol.	2015-02-26 02:05:03.491333	2015-02-26 02:06:01.347451
41	23944364	\N	Stankov et al., 2014, Curr. Pharm. Des.	2015-02-26 02:05:03.731927	2015-02-26 02:06:01.858884
42	22589270	\N	Prior et al., 2012, Cancer Res.	2015-02-26 02:05:03.987847	2015-02-26 02:06:02.271462
43	23226219	\N	Chou et al., 2012, PLoS ONE	2015-02-26 02:05:05.122246	2015-02-26 02:06:02.863617
44	18073307	\N	Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-26 02:05:06.842448	2015-02-26 02:06:03.396331
45	25465739	\N	Perri et al., 2014, Crit. Rev. Oncol. Hematol.	2015-02-26 02:05:06.847737	2015-02-26 02:06:03.907452
46	21995386	\N	Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-26 02:05:07.299109	2015-02-26 02:06:04.326891
47	24136165	\N	Cazzola et al., 2013, Blood	2015-02-26 02:05:07.304378	2015-02-26 02:06:04.730354
48	20182602	\N	Olivier et al., 2010, Cold Spring Harb Perspect Biol	2015-02-26 02:05:07.5599	2015-02-26 02:06:05.472063
49	25311244	\N	Okeyo-Owuor et al., 2014, Leukemia	2015-02-26 02:05:08.31177	2015-02-26 02:06:05.831517
50	22158538	\N	Graubert et al., 2012, Nat. Genet.	2015-02-26 02:05:08.3174	2015-02-26 02:06:06.320253
51	12476305		Nimmanapalli et al., 2002, Oncogene	2015-02-26 02:05:12.326624	2015-02-26 02:06:06.774898
52	20537386		An et al., 2010, Leuk. Res.	2015-02-26 02:05:12.503	2015-02-26 02:06:07.184575
53	25212276		Schmidt et al., 2014, Leukemia	2015-02-26 02:05:12.630348	2015-02-26 02:06:07.552407
54	15194504	Cell Line	Yamamoto et al., 2004, Biochem. Biophys. Res. Commun.	2015-02-26 02:05:12.868964	2015-02-26 02:06:08.002259
55	23888070		Beaver et al., 2013, Clin. Cancer Res.	2015-02-26 02:05:13.118557	2015-02-26 02:06:08.41728
56	20979473	Clinical Trial	Choi et al., 2010, N. Engl. J. Med.	2015-02-26 02:05:13.180839	2015-02-26 02:06:08.827778
57	21575866		Sakamoto et al., 2011, Cancer Cell	2015-02-26 02:05:13.330351	2015-02-26 02:06:09.238163
58	21030459		Sasaki et al., 2010, Cancer Res.	2015-02-26 02:05:13.456891	2015-02-26 02:06:09.751192
59	22072639		Bresler et al., 2011, Sci Transl Med	2015-02-26 02:05:13.595926	2015-02-26 02:06:10.156584
60	21948233		Heuckmann et al., 2011, Clin. Cancer Res.	2015-02-26 02:05:13.66398	2015-02-26 02:06:10.626784
61	18923525		George et al., 2008, Nature	2015-02-26 02:05:13.727993	2015-02-26 02:06:11.184822
62	24569458		Imielinski et al., 2014, J. Clin. Invest.	2015-02-26 02:05:13.989275	2015-02-26 02:06:11.692733
63	23325582	Clinical trial	Peeters et al., 2013, Clin. Cancer Res.	2015-02-26 02:05:14.199331	2015-02-26 02:06:12.202787
64	23020132	Clinical trial	Flaherty et al., 2012, N. Engl. J. Med.	2015-02-26 02:05:14.268868	2015-02-26 02:06:12.77004
65	21166657	Meta analysis	Lee et al., 2011, Br. J. Dermatol.	2015-02-26 02:05:14.390153	2015-02-26 02:06:13.227552
66	23463675	Clinical trial	Ponti et al., 2013, J. Clin. Pathol.	2015-02-26 02:05:14.446906	2015-02-26 02:06:13.637876
67	19001320	In vitro study	Di Nicolantonio et al., 2008, J. Clin. Oncol.	2015-02-26 02:05:14.519474	2015-02-26 02:06:14.019297
68	23845441	In vitro study	Rad et al., 2013, Cancer Cell	2015-02-26 02:05:14.717062	2015-02-26 02:06:14.455925
69	23812671	In vitro study	Ji et al., 2013, Clin. Cancer Res.	2015-02-26 02:05:14.84386	2015-02-26 02:06:14.907273
70	22180495	In vivo study	Yang et al., 2012, Cancer Res.	2015-02-26 02:05:14.967505	2015-02-26 02:06:15.479346
71	20619739	Retrospective study	De Roock et al., 2010, Lancet Oncol.	2015-02-26 02:05:15.167889	2015-02-26 02:06:15.913742
72	24594804	Metaanalysis	Chen et al., 2014, PLoS ONE	2015-02-26 02:05:15.229245	2015-02-26 02:06:16.567157
73	24586605	Retrospective study	Meckbach et al., 2014, PLoS ONE	2015-02-26 02:05:15.35173	2015-02-26 02:06:17.118853
74	24583796	Clinical trial	Menzies et al., 2014, Clin. Cancer Res.	2015-02-26 02:05:15.469928	2015-02-26 02:06:17.538402
75	24576830	In vitro study	Nissan et al., 2014, Cancer Res.	2015-02-26 02:05:15.606869	2015-02-26 02:06:17.935461
76	24388723	Retrospective study	Nagore et al., 2014, J. Am. Acad. Dermatol.	2015-02-26 02:05:15.724796	2015-02-26 02:06:18.359002
77	23524406	Case report	Rudin et al., 2013, J Thorac Oncol	2015-02-26 02:05:15.789231	2015-02-26 02:06:18.858458
78	21594703	Retrospective study	Howell et al., 2011, Ann. Surg. Oncol.	2015-02-26 02:05:15.852983	2015-02-26 02:06:19.883462
79	24570209	Retrospective study	Crescenzi et al., 2014, Horm. Metab. Res.	2015-02-26 02:05:15.921636	2015-02-26 02:06:22.360002
80	24588959	Perspective study	Zhang et al., 2014, Diagn Pathol	2015-02-26 02:05:15.981339	2015-02-26 02:06:22.869232
81	24354346	Retrospective study	Walczyk et al., 2014, Clin. Endocrinol. (Oxf)	2015-02-26 02:05:16.064955	2015-02-26 02:06:23.263816
82	24396464	Retrospective study	He et al., 2014, Oncol Lett	2015-02-26 02:05:16.134611	2015-02-26 02:06:23.772101
83	21098728	In vitro study	Corcoran et al., 2010, Sci Signal	2015-02-26 02:05:16.313696	2015-02-26 02:06:24.199697
84	23031422	Case report	Ponti et al., 2012, J Hematol Oncol	2015-02-26 02:05:16.382095	2015-02-26 02:06:24.593623
85	23346317	Review	Goff, 2013, J Gynecol Oncol	2015-02-26 02:05:16.596318	2015-02-26 02:06:25.107271
86	17070615		Gautschi et al., 2007, Lung Cancer	2015-02-26 02:05:16.7466	2015-02-26 02:06:25.499168
87	15961768		Arnold et al., 2005, J. Clin. Oncol.	2015-02-26 02:05:16.938656	2015-02-26 02:06:25.88285
88	16309541		Thomas et al., 2005, Int J Exp Pathol	2015-02-26 02:05:17.072315	2015-02-26 02:06:26.513531
89	17891190		Jares et al., 2007, Nat. Rev. Cancer	2015-02-26 02:05:17.138874	2015-02-26 02:06:26.948158
90	10547574		Takano et al., 1999, J. Pathol.	2015-02-26 02:05:17.200904	2015-02-26 02:06:27.458722
91	14612939		Oshimo et al., 2003, Int. J. Oncol.	2015-02-26 02:05:17.468865	2015-02-26 02:06:27.870541
92	23079656		Sawai et al., 2012, Cancer Cell	2015-02-26 02:05:17.534823	2015-02-26 02:06:28.384197
93	12432043		Keyomarsi et al., 2002, N. Engl. J. Med.	2015-02-26 02:05:17.599673	2015-02-26 02:06:28.793021
94	10224221		Donnellan et al., 1999, FASEB J.	2015-02-26 02:05:17.664447	2015-02-26 02:06:29.200021
95	25002028		Vora et al., 2014, Cancer Cell	2015-02-26 02:05:17.857291	2015-02-26 02:06:29.713623
96	23898052		Logan et al., 2013, Anticancer Res.	2015-02-26 02:05:17.941286	2015-02-26 02:06:30.131833
97	23111194		Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-26 02:05:18.124954	2015-02-26 02:06:30.63635
98	19965647	Retrospective study	Burnett et al., 2010, Blood	2015-02-26 02:05:18.250404	2015-02-26 02:06:31.045641
99	18450602	Retrospective study	Schlenk et al., 2008, N. Engl. J. Med.	2015-02-26 02:05:18.312814	2015-02-26 02:06:31.715937
100	14726504	Retrospective study	Fröhling et al., 2004, J. Clin. Oncol.	2015-02-26 02:05:18.449146	2015-02-26 02:06:32.172811
101	24186003		Lilljebjörn et al., 2014, Leukemia	2015-02-26 02:05:18.66754	2015-02-26 02:06:32.580037
102	24578576	Retrospective study	Honeyman et al., 2014, Science	2015-02-26 02:05:18.804399	2015-02-26 02:06:32.994555
103	22081665	Retrospective study	LaRochelle et al., 2011, Oncotarget	2015-02-26 02:05:18.87905	2015-02-26 02:06:33.453205
104	23632886	Retrospective study	Gaidzik et al., 2013, Blood	2015-02-26 02:05:18.997854	2015-02-26 02:06:33.911606
105	22490330	Retrospective study	Ribeiro et al., 2012, Blood	2015-02-26 02:05:19.064762	2015-02-26 02:06:34.4214
106	24512939	Retrospective study	El Ghannam et al., Blood Cells Mol. Dis.	2015-02-26 02:05:19.189394	2015-02-26 02:06:35.03809
107	21067377	Retrospective study	Ley et al., 2010, N. Engl. J. Med.	2015-02-26 02:05:19.273138	2015-02-26 02:06:35.501631
108	22291079	Retrospective study	Marcucci et al., 2012, J. Clin. Oncol.	2015-02-26 02:05:19.840593	2015-02-26 02:06:36.635186
109	24585406		Li et al., 2014, Ann. Surg. Oncol.	2015-02-26 02:05:20.827025	2015-02-26 02:06:37.087015
110	24736073		Lim et al., 2014, J Thorac Oncol	2015-02-26 02:05:20.886825	2015-02-26 02:06:37.701791
111	24457318		Fukihara et al., 2014, Oncology	2015-02-26 02:05:21.010963	2015-02-26 02:06:38.212491
112	24662454		Douillard et al., 2014, J Thorac Oncol	2015-02-26 02:05:21.07731	2015-02-26 02:06:39.605303
113	25668228	Review	Denis et al., 2015, Clin. Chim. Acta	2015-02-26 02:05:21.140567	2015-02-26 02:06:40.05354
114	24636847		Li et al., 2014, Lung Cancer	2015-02-26 02:05:21.210029	2015-02-26 02:06:40.566409
115	15728811		Kobayashi et al., 2005, N. Engl. J. Med.	2015-02-26 02:05:21.342612	2015-02-26 02:06:40.977272
116	24729716		Li et al., 2014, Onco Targets Ther	2015-02-26 02:05:21.407294	2015-02-26 02:06:41.387416
117	24623981		Ding et al., 2014, Onco Targets Ther	2015-02-26 02:05:21.465972	2015-02-26 02:06:41.750088
118	24658966		Ai et al., 2014, Amino Acids	2015-02-26 02:05:21.53244	2015-02-26 02:06:42.204867
119	23220880		Bose et al., 2013, Cancer Discov	2015-02-26 02:05:21.601106	2015-02-26 02:06:42.619177
120	24185512		Toy et al., 2013, Nat. Genet.	2015-02-26 02:05:22.482945	2015-02-26 02:06:43.026976
121	24185510		Robinson et al., 2013, Nat. Genet.	2015-02-26 02:05:22.555673	2015-02-26 02:06:43.678421
122	24550739	Case study	Borad et al., 2014, PLoS Genet.	2015-02-26 02:05:23.447774	2015-02-26 02:06:44.164462
123	23558953		Wu et al., 2013, Cancer Discov	2015-02-26 02:05:23.525693	2015-02-26 02:06:44.665152
124	12357354	In vitro study	Tse et al., 2002, Leukemia	2015-02-26 02:05:23.757511	2015-02-26 02:06:45.277199
125	19965647		Burnett et al., 2010, Blood	2015-02-26 02:05:23.821058	2015-02-26 02:06:45.682196
126	14726387	Clinical trial	Smith et al., 2004, Blood	2015-02-26 02:05:23.883784	2015-02-26 02:06:46.093607
127	15626738	In vitro study	Bagrintseva et al., 2005, Blood	2015-02-26 02:05:23.946295	2015-02-26 02:06:46.506921
128	11290608	Retrospective study	Yamamoto et al., 2001, Blood	2015-02-26 02:05:24.004812	2015-02-26 02:06:46.917118
129	17957027	Retrospective study	Gale et al., 2008, Blood	2015-02-26 02:05:24.065298	2015-02-26 02:06:47.326427
130	21537333		Schnittger et al., 2011, Leukemia	2015-02-26 02:05:24.26434	2015-02-26 02:06:47.736437
131	24801015		Port et al., 2014, Ann. Hematol.	2015-02-26 02:05:24.701607	2015-02-26 02:06:48.147973
132	11585760	Retrospective study	Whitman et al., 2001, Cancer Res.	2015-02-26 02:05:24.758704	2015-02-26 02:06:48.657461
133	18230792	In vitro study	Zhang et al., 2008, J. Natl. Cancer Inst.	2015-02-26 02:05:24.828868	2015-02-26 02:06:49.171263
134	17965322	Retrospective study	Bacher et al., 2008, Blood	2015-02-26 02:05:25.22415	2015-02-26 02:06:49.781883
135	17940205	Retrospective study	Whitman et al., 2008, Blood	2015-02-26 02:05:25.288595	2015-02-26 02:06:50.8096
136	22368270	Clinical trial	Man et al., 2012, Blood	2015-02-26 02:05:25.415724	2015-02-26 02:06:51.219271
137	22624710		Barbacid, 2012, Cancer Cell	2015-02-26 02:05:25.572261	2015-02-26 02:06:51.727899
138	20376086	Retrospective study	Ho et al., 2010, Leukemia	2015-02-26 02:05:25.694268	2015-02-26 02:06:52.2412
139	20805365	Retrospective study	Schnittger et al., 2010, Blood	2015-02-26 02:05:25.752932	2015-02-26 02:06:52.75631
140	20368538	Retrospective study	Wagner et al., 2010, J. Clin. Oncol.	2015-02-26 02:05:25.81158	2015-02-26 02:06:53.175303
141	19933982	Retrospective study	Dubbink et al., 2009, Neurology	2015-02-26 02:05:25.878081	2015-02-26 02:06:53.564384
142	20127344	Retrospective study	Bleeker et al., 2010, Acta Neuropathol.	2015-02-26 02:05:26.018081	2015-02-26 02:06:54.039443
143	20494930	Retrospective study	Thol et al., 2010, Haematologica	2015-02-26 02:05:26.080298	2015-02-26 02:06:54.495033
144	20538800	Retrospective study	Abbas et al., 2010, Blood	2015-02-26 02:05:26.141829	2015-02-26 02:06:55.006719
145	22616558		Zhou et al., 2012, Leuk. Lymphoma	2015-02-26 02:05:26.389054	2015-02-26 02:06:55.448696
146	20421455	Retrospective study	Thol et al., 2010, Blood	2015-02-26 02:05:26.453189	2015-02-26 02:06:55.92826
147	21596855		Green et al., 2011, Blood	2015-02-26 02:05:26.505843	2015-02-26 02:06:56.335176
148	22033490		Patnaik et al., 2012, Leukemia	2015-02-26 02:05:26.56135	2015-02-26 02:06:56.906618
149	21997850		Lin et al., 2012, Ann. Hematol.	2015-02-26 02:05:26.621883	2015-02-26 02:06:57.413993
150	16081687	Retrospective study	Levine et al., 2005, Blood	2015-02-26 02:05:26.864605	2015-02-26 02:06:57.873801
151	19287384	Retrospective study	Kilpivaara et al., 2009, Nat. Genet.	2015-02-26 02:05:27.04377	2015-02-26 02:06:58.266899
152	16709929	Clinical trial	Kiladjian et al., 2006, Blood	2015-02-26 02:05:27.115335	2015-02-26 02:06:58.692724
153	18394554	In vitro study	Wernig et al., 2008, Cancer Cell	2015-02-26 02:05:27.175533	2015-02-26 02:06:59.102229
154	16384925	Retrospective study	Cairoli et al., 2006, Blood	2015-02-26 02:05:27.235377	2015-02-26 02:07:00.021573
155	10485475	Retrospective study	Taniguchi et al., 1999, Cancer Res.	2015-02-26 02:05:27.305517	2015-02-26 02:07:00.841841
156	16551858	Retrospective study	Wardelmann et al., 2006, Clin. Cancer Res.	2015-02-26 02:05:27.377641	2015-02-26 02:07:02.070967
157	12000708	Retrospective study	Corless et al., 2002, Am. J. Pathol.	2015-02-26 02:05:27.46851	2015-02-26 02:07:02.454887
158	15217946	Retrospective study	Boldrini et al., 2004, Clin. Cancer Res.	2015-02-26 02:05:27.60481	2015-02-26 02:07:02.894453
159	23775962	Clinical trial	Hodi et al., 2013, J. Clin. Oncol.	2015-02-26 02:05:27.675976	2015-02-26 02:07:03.459221
160	18421059	Observational study	Hodi et al., 2008, J. Clin. Oncol.	2015-02-26 02:05:28.120083	2015-02-26 02:07:04.221837
161	19671763	In vitro study	Woodman et al., 2009, Mol. Cancer Ther.	2015-02-26 02:05:28.193868	2015-02-26 02:07:04.63244
162	17372901	In vitro study	Antonescu et al., 2007, Int. J. Cancer	2015-02-26 02:05:28.45133	2015-02-26 02:07:05.352908
163	16954519		Heinrich et al., 2006, J. Clin. Oncol.	2015-02-26 02:05:28.624941	2015-02-26 02:07:05.757778
164	16638875		Prenen et al., 2006, Clin. Cancer Res.	2015-02-26 02:05:28.694239	2015-02-26 02:07:06.167854
165	16618717	Retrospective study	Lièvre et al., 2006, Cancer Res.	2015-02-26 02:05:28.756923	2015-02-26 02:07:06.579278
166	24559322	Clinical trial	Sartore-Bianchi et al., 2014, Expert Rev Anticancer Ther	2015-02-26 02:05:28.821709	2015-02-26 02:07:06.961475
167	15696205	Retrospective study	Pao et al., 2005, PLoS Med.	2015-02-26 02:05:28.875554	2015-02-26 02:07:07.397613
168	24571676	Retrospective study	Andrade et al., 2014, BMC Cancer	2015-02-26 02:05:29.014311	2015-02-26 02:07:07.90986
169	19934290	Retrospective study	Ogino et al., 2009, Clin. Cancer Res.	2015-02-26 02:05:29.078537	2015-02-26 02:07:08.320729
170	18528420	Retrospective study	Chng et al., 2008, Leukemia	2015-02-26 02:05:29.142501	2015-02-26 02:07:08.732919
171	18794081	Retrospective study	Riely et al., 2008, Clin. Cancer Res.	2015-02-26 02:05:29.213216	2015-02-26 02:07:09.137502
172	11208838	Retrospective study	Schiller et al., 2001, J. Clin. Oncol.	2015-02-26 02:05:29.274072	2015-02-26 02:07:09.651988
173	15597105	Metaanalysis	Mascaux et al., 2005, Br. J. Cancer	2015-02-26 02:05:29.339936	2015-02-26 02:07:10.159882
174	18202412	In vitro study	Lièvre et al., 2008, J. Clin. Oncol.	2015-02-26 02:05:29.403965	2015-02-26 02:07:10.572679
175	23014527	Retrospective study	Dogan et al., 2012, Clin. Cancer Res.	2015-02-26 02:05:29.569703	2015-02-26 02:07:11.084255
176	23524406	case report	Rudin et al., 2013, J Thorac Oncol	2015-02-26 02:05:29.688135	2015-02-26 02:07:11.493863
177	19029981	In vivo study	Engelman et al., 2008, Nat. Med.	2015-02-26 02:05:29.752396	2015-02-26 02:07:12.007692
178	20978259	Clinical trial	De Roock et al., 2010, JAMA	2015-02-26 02:05:30.054386	2015-02-26 02:07:12.519446
179	19915144		Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-02-26 02:05:30.315345	2015-02-26 02:07:12.895108
180	19915144	In vitro study	Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-02-26 02:05:30.384473	2015-02-26 02:07:13.439259
181	11070098		Esteller et al., 2000, N. Engl. J. Med.	2015-02-26 02:05:30.609047	2015-02-26 02:07:13.952301
182	15758010	Clinical Trial	Hegi et al., 2005, N. Engl. J. Med.	2015-02-26 02:05:30.827682	2015-02-26 02:07:14.359872
183	19059939	Retrospective study	Schlenk et al., 2009, Haematologica	2015-02-26 02:05:30.965126	2015-02-26 02:07:14.876435
184	22417203	Retrospective study	Patel et al., 2012, N. Engl. J. Med.	2015-02-26 02:05:31.035259	2015-02-26 02:07:15.896895
185	16455956	Retrospective study	Thiede et al., 2006, Blood	2015-02-26 02:05:31.101676	2015-02-26 02:07:16.317706
186	16076867	Retrospective study	Schnittger et al., 2005, Blood	2015-02-26 02:05:31.159197	2015-02-26 02:07:16.817479
187	19357394		Vardiman et al., 2009, Blood	2015-02-26 02:05:31.288568	2015-02-26 02:07:17.332993
188	24927407		Ehninger et al., 2014, Blood Cancer J	2015-02-26 02:05:31.64778	2015-02-26 02:07:17.740119
189	22430270	Retrospective study	Gaidzik et al., 2012, J. Clin. Oncol.	2015-02-26 02:05:31.995628	2015-02-26 02:07:18.32651
190	16051734	Retrospective study	Döhner et al., 2005, Blood	2015-02-26 02:05:32.114559	2015-02-26 02:07:18.762798
191	24855211		Linch et al., 2014, Blood	2015-02-26 02:05:32.176813	2015-02-26 02:07:19.174282
192	20026798	Retrospective study	Becker et al., 2010, J. Clin. Oncol.	2015-02-26 02:05:32.296611	2015-02-26 02:07:19.891148
193	19047294	Retrospective study	Büchner et al., 2009, J. Clin. Oncol.	2015-02-26 02:05:32.483807	2015-02-26 02:07:20.404672
194	19587375	Retrospective study	Schnittger et al., 2009, Blood	2015-02-26 02:05:32.730268	2015-02-26 02:07:20.811562
195	24859829		Tian et al., 2014, Int. J. Hematol.	2015-02-26 02:05:32.79993	2015-02-26 02:07:21.224421
196	21719597	In vitro study	Balusu et al., 2011, Blood	2015-02-26 02:05:32.930235	2015-02-26 02:07:21.732777
197	24797300		Tassara et al., 2014, Blood	2015-02-26 02:05:33.052376	2015-02-26 02:07:22.250372
198	15659725		Falini et al., 2005, N. Engl. J. Med.	2015-02-26 02:05:33.242879	2015-02-26 02:07:22.654421
199	22180178	Clinical trial	Jakob et al., 2012, Cancer	2015-02-26 02:05:33.436933	2015-02-26 02:07:25.935212
200	15951308	Retrospective study	Bowen et al., 2005, Blood	2015-02-26 02:05:33.498596	2015-02-26 02:07:26.442948
201	24666267	Meta analysis	Therkildsen et al., 2014, Acta Oncol	2015-02-26 02:05:33.558814	2015-02-26 02:07:26.832654
202	16434492	Clinical trial	Bacher et al., 2006, Blood	2015-02-26 02:05:33.811015	2015-02-26 02:07:27.26179
203	18375819	Case report	Banerji et al., 2008, Mol. Cancer Ther.	2015-02-26 02:05:33.878577	2015-02-26 02:07:27.670808
204	23861977	Retrospective study	Tschandl et al., 2013, PLoS ONE	2015-02-26 02:05:34.062547	2015-02-26 02:07:28.185859
205	23569304	Clinical trial	Trunzer et al., 2013, J. Clin. Oncol.	2015-02-26 02:05:34.123825	2015-02-26 02:07:28.594661
206	21576590	Case report	Soon et al., 2011, Arch Dermatol	2015-02-26 02:05:34.203553	2015-02-26 02:07:29.003992
207	22745105		Heinrich et al., 2012, Clin. Cancer Res.	2015-02-26 02:05:34.318961	2015-02-26 02:07:29.412383
208	15928335		Corless et al., 2005, J. Clin. Oncol.	2015-02-26 02:05:34.454863	2015-02-26 02:07:29.925146
209	15146165		Lasota et al., 2004, Lab. Invest.	2015-02-26 02:05:34.586433	2015-02-26 02:07:30.434949
210	21558396		Tanaka et al., 2011, Clin. Cancer Res.	2015-02-26 02:05:34.898055	2015-02-26 02:07:31.357776
211	15647370		Kang et al., 2005, Proc. Natl. Acad. Sci. U.S.A.	2015-02-26 02:05:34.962224	2015-02-26 02:07:31.870406
212	19223544		Sartore-Bianchi et al., 2009, Cancer Res.	2015-02-26 02:05:35.035994	2015-02-26 02:07:32.382683
213	22357840		Liao et al., 2012, Clin. Cancer Res.	2015-02-26 02:05:35.153763	2015-02-26 02:07:32.764954
214	8674046		Yoshida et al., 1996, Cancer Res.	2015-02-26 02:05:36.294177	2015-02-26 02:07:33.305855
215	20085938		Courtney et al., 2010, J. Clin. Oncol.	2015-02-26 02:05:36.411597	2015-02-26 02:07:33.771522
216	22479427		Carico et al., 2012, PLoS ONE	2015-02-26 02:05:36.48371	2015-02-26 02:07:34.329268
217	21422803		Coxon et al., 2012, J. Endocrinol. Invest.	2015-02-26 02:05:36.612206	2015-02-26 02:07:35.044636
218	23056499		Couto et al., 2012, PLoS ONE	2015-02-26 02:05:36.700409	2015-02-26 02:07:35.514551
219	18073307		Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-26 02:05:36.838739	2015-02-26 02:07:35.965498
220	9839497		Egawa et al., 1998, Jpn. J. Clin. Oncol.	2015-02-26 02:05:36.898644	2015-02-26 02:07:36.47865
221	21995386		Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-26 02:05:36.964777	2015-02-26 02:07:36.889662
222	22698404		Jackson et al., 2012, Cancer Cell	2015-02-26 02:05:37.108823	2015-02-26 02:07:37.404652
223	16489069		Olivier et al., 2006, Clin. Cancer Res.	2015-02-26 02:05:37.173182	2015-02-26 02:07:37.918963
224	9569050		Berns et al., 1998, Br. J. Cancer	2015-02-26 02:05:37.301254	2015-02-26 02:07:38.424488
225	22923433		Iyer et al., 2012, Science	2015-02-26 02:05:37.917393	2015-02-26 02:07:38.936103
226	19966866	Mouse model	Liang et al., 2010, Oncogene	2015-02-26 02:05:37.98853	2015-02-26 02:07:39.360452
227	22923433	Retrospective study	Iyer et al., 2012, Science	2015-02-26 02:05:38.063604	2015-02-26 02:07:39.856519
228	23029227		Qian et al., 2012, PLoS ONE	2015-02-26 02:05:38.195643	2015-02-26 02:07:40.268703
229	23861105		Wu et al., 2013, Am. J. Hematol.	2015-02-26 02:05:38.370044	2015-02-26 02:07:40.678809
230	18591546	Retrospective study	Virappane et al., 2008, J. Clin. Oncol.	2015-02-26 02:05:38.672635	2015-02-26 02:07:41.189084
231	19221039	Retrospective study	Gaidzik et al., 2009, Blood	2015-02-26 02:05:38.730858	2015-02-26 02:07:41.602348
232	19536888	Retrospective study	Renneville et al., 2009, Cancer	2015-02-26 02:05:38.920878	2015-02-26 02:07:42.15118
233	18559874	Retrospective study	Paschka et al., 2008, J. Clin. Oncol.	2015-02-26 02:05:39.224693	2015-02-26 02:07:42.625364
\.


ALTER TABLE sources ENABLE TRIGGER ALL;

--
-- Name: sources_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('sources_id_seq', 233, true);


--
-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE users DISABLE TRIGGER ALL;

COPY users (id, email, name, url, nickname, created_at, updated_at) FROM stdin;
\.


ALTER TABLE users ENABLE TRIGGER ALL;

--
-- Data for Name: subscriptions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE subscriptions DISABLE TRIGGER ALL;

COPY subscriptions (id, user_id, subscribable_id, subscribable_type, type, created_at, updated_at) FROM stdin;
\.


ALTER TABLE subscriptions ENABLE TRIGGER ALL;

--
-- Name: subscriptions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('subscriptions_id_seq', 1, false);


--
-- Data for Name: suggested_changes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE suggested_changes DISABLE TRIGGER ALL;

COPY suggested_changes (id, suggested_changes, moderated_id, moderated_type, user_id, status, created_at, updated_at) FROM stdin;
\.


ALTER TABLE suggested_changes ENABLE TRIGGER ALL;

--
-- Name: suggested_changes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('suggested_changes_id_seq', 1, false);


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('users_id_seq', 1, false);


--
-- Data for Name: variant_group_variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE variant_group_variants DISABLE TRIGGER ALL;

COPY variant_group_variants (variant_id, variant_group_id, created_at, updated_at) FROM stdin;
3	1	2015-02-26 02:05:12.948352	2015-02-26 02:05:12.948352
2	1	2015-02-26 02:05:13.039375	2015-02-26 02:05:13.039375
2	1	2015-02-26 02:05:13.099459	2015-02-26 02:05:13.099459
5	2	2015-02-26 02:05:13.233478	2015-02-26 02:05:13.233478
6	2	2015-02-26 02:05:13.306467	2015-02-26 02:05:13.306467
6	3	2015-02-26 02:05:13.311861	2015-02-26 02:05:13.311861
7	2	2015-02-26 02:05:13.437703	2015-02-26 02:05:13.437703
7	3	2015-02-26 02:05:13.441612	2015-02-26 02:05:13.441612
8	3	2015-02-26 02:05:13.843362	2015-02-26 02:05:13.843362
11	4	2015-02-26 02:05:14.498522	2015-02-26 02:05:14.498522
13	4	2015-02-26 02:05:16.42926	2015-02-26 02:05:16.42926
15	4	2015-02-26 02:05:16.513063	2015-02-26 02:05:16.513063
16	4	2015-02-26 02:05:16.5786	2015-02-26 02:05:16.5786
129	5	2015-02-26 02:05:16.654851	2015-02-26 02:05:16.654851
130	5	2015-02-26 02:05:16.728694	2015-02-26 02:05:16.728694
34	6	2015-02-26 02:05:21.19412	2015-02-26 02:05:21.19412
35	7	2015-02-26 02:05:21.6507	2015-02-26 02:05:21.6507
36	7	2015-02-26 02:05:21.721609	2015-02-26 02:05:21.721609
38	7	2015-02-26 02:05:21.870372	2015-02-26 02:05:21.870372
41	7	2015-02-26 02:05:22.130084	2015-02-26 02:05:22.130084
43	7	2015-02-26 02:05:22.326206	2015-02-26 02:05:22.326206
44	7	2015-02-26 02:05:22.401989	2015-02-26 02:05:22.401989
45	7	2015-02-26 02:05:22.464167	2015-02-26 02:05:22.464167
46	8	2015-02-26 02:05:22.537409	2015-02-26 02:05:22.537409
46	8	2015-02-26 02:05:22.66237	2015-02-26 02:05:22.66237
47	8	2015-02-26 02:05:22.723262	2015-02-26 02:05:22.723262
47	8	2015-02-26 02:05:22.857162	2015-02-26 02:05:22.857162
48	8	2015-02-26 02:05:22.917456	2015-02-26 02:05:22.917456
48	8	2015-02-26 02:05:23.041507	2015-02-26 02:05:23.041507
49	8	2015-02-26 02:05:23.109054	2015-02-26 02:05:23.109054
49	8	2015-02-26 02:05:23.248631	2015-02-26 02:05:23.248631
50	8	2015-02-26 02:05:23.310522	2015-02-26 02:05:23.310522
50	8	2015-02-26 02:05:23.428825	2015-02-26 02:05:23.428825
51	9	2015-02-26 02:05:23.507448	2015-02-26 02:05:23.507448
52	9	2015-02-26 02:05:23.587286	2015-02-26 02:05:23.587286
53	9	2015-02-26 02:05:23.737447	2015-02-26 02:05:23.737447
64	10	2015-02-26 02:05:27.289786	2015-02-26 02:05:27.289786
65	11	2015-02-26 02:05:27.363688	2015-02-26 02:05:27.363688
65	11	2015-02-26 02:05:27.45249	2015-02-26 02:05:27.45249
65	11	2015-02-26 02:05:27.522446	2015-02-26 02:05:27.522446
65	11	2015-02-26 02:05:27.588962	2015-02-26 02:05:27.588962
65	11	2015-02-26 02:05:27.659349	2015-02-26 02:05:27.659349
65	11	2015-02-26 02:05:27.730356	2015-02-26 02:05:27.730356
69	10	2015-02-26 02:05:27.915172	2015-02-26 02:05:27.915172
69	10	2015-02-26 02:05:27.981362	2015-02-26 02:05:27.981362
66	11	2015-02-26 02:05:28.178724	2015-02-26 02:05:28.178724
71	11	2015-02-26 02:05:28.255214	2015-02-26 02:05:28.255214
71	11	2015-02-26 02:05:28.43303	2015-02-26 02:05:28.43303
71	11	2015-02-26 02:05:28.609541	2015-02-26 02:05:28.609541
72	11	2015-02-26 02:05:28.674725	2015-02-26 02:05:28.674725
72	11	2015-02-26 02:05:28.741155	2015-02-26 02:05:28.741155
86	12	2015-02-26 02:05:33.16553	2015-02-26 02:05:33.16553
86	12	2015-02-26 02:05:33.227763	2015-02-26 02:05:33.227763
86	12	2015-02-26 02:05:33.312281	2015-02-26 02:05:33.312281
86	12	2015-02-26 02:05:33.417135	2015-02-26 02:05:33.417135
96	1	2015-02-26 02:05:34.377837	2015-02-26 02:05:34.377837
97	1	2015-02-26 02:05:34.438613	2015-02-26 02:05:34.438613
97	1	2015-02-26 02:05:34.501923	2015-02-26 02:05:34.501923
97	1	2015-02-26 02:05:34.566833	2015-02-26 02:05:34.566833
97	1	2015-02-26 02:05:34.639213	2015-02-26 02:05:34.639213
98	1	2015-02-26 02:05:34.70493	2015-02-26 02:05:34.70493
99	1	2015-02-26 02:05:34.773994	2015-02-26 02:05:34.773994
100	1	2015-02-26 02:05:34.880349	2015-02-26 02:05:34.880349
108	13	2015-02-26 02:05:36.465773	2015-02-26 02:05:36.465773
108	13	2015-02-26 02:05:36.53588	2015-02-26 02:05:36.53588
110	14	2015-02-26 02:05:36.681504	2015-02-26 02:05:36.681504
111	14	2015-02-26 02:05:36.822909	2015-02-26 02:05:36.822909
122	15	2015-02-26 02:05:37.968312	2015-02-26 02:05:37.968312
122	15	2015-02-26 02:05:38.043441	2015-02-26 02:05:38.043441
123	15	2015-02-26 02:05:38.119618	2015-02-26 02:05:38.119618
124	15	2015-02-26 02:05:38.180325	2015-02-26 02:05:38.180325
\.


ALTER TABLE variant_group_variants ENABLE TRIGGER ALL;

--
-- Data for Name: variant_groups; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE variant_groups DISABLE TRIGGER ALL;

COPY variant_groups (id, name, description, created_at, updated_at) FROM stdin;
10	KIT Exon 17	\N	2015-02-26 02:05:27.286689	2015-02-26 02:05:27.286689
11	KIT Exon 11	\N	2015-02-26 02:05:27.361213	2015-02-26 02:05:27.361213
1	Imatinib Resistance	While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. 	2015-02-26 02:05:12.939945	2015-02-26 02:05:39.756697
2	ALK Fusions	ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase the activity of the ALK oncogene relative to normal cells. While EML4 is the most common fusion partner, other 5' partners have been observed. The EML4-ALK fusion has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. 	2015-02-26 02:05:13.230386	2015-02-26 02:05:39.76141
6	Erlotinib Resistance	EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine kinase inhibitors have been developed to target EGFR pathway activity. One such inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting. However, the T790M missense mutation has shown to confer resistance to this inhibitor in cell lines and case studies.  	2015-02-26 02:05:21.191339	2015-02-26 02:05:39.764404
3	Crizotinib Resistance	The ALK oncogene has long been considered a driving factor in non-small cell lung cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive this resistance. 	2015-02-26 02:05:13.309411	2015-02-26 02:05:39.767315
4	Other V600's	While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants have also been observed and studied to a lesser degree. At first approximation, many of these variants seem to behave similarly to V600E, and treatment with dabrafenib has been shown to be effective. 	2015-02-26 02:05:14.495906	2015-02-26 02:05:39.770242
7	HER2 Activating	HER2-positive breast cancer is widely recognized as a molecular subtype of breast cancer. While amplification of ERBB2 is a common mechanism for activation of the pathway, missense mutations have also demonstrated activation potential. These activating missense mutations also confer senstivity to the targeted therapeutic neratinib. 	2015-02-26 02:05:21.647805	2015-02-26 02:05:39.77358
8	ESR1 Ligand-Binding Domain	ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone therapy has been widely effective in treating the disease, however sequencing of resistant patients has uncovered a number of missense mutations in mediating this resistance. Many of these mutations lie within the ligand-binding domain, and contribute to constitutive activity of the receptor. This has lead to the development of estrogen receptor degrading agents such as fulvestrant, which have shown early efficacy in clinical trials.  	2015-02-26 02:05:22.534306	2015-02-26 02:05:39.7775
9	FGFR fusions	FGFR fusions have been demonstrated across many cancer types to have oncogenic potential. FGFR2 and FGFR3 have been shown to be fused to many 3' partners, which may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2 fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib and pazopanib have shown efficacy in case studies.  	2015-02-26 02:05:23.504616	2015-02-26 02:05:39.781477
12	NPM1 exon 12	To be filled in by Dr. KK, resident blood expert and knower of all things hematology. :) :) :) 	2015-02-26 02:05:33.161344	2015-02-26 02:05:39.785107
13	PTEN Loss-of-Function	PTEN loss is a common event in breast cancer, as well as others. The most common single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell lines harboring this mutation behave similarly to cell lines harboring larger insertion or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded growth. 	2015-02-26 02:05:36.46316	2015-02-26 02:05:39.788083
14	Motesanib Resistance	RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment of these patients with the targeted therapeutic motesanib has shown to be effective. However, the missense mutations C634W and M918T have shown to confer motesanib resistance in cell lines. 	2015-02-26 02:05:36.678557	2015-02-26 02:05:39.790782
15	TSC Loss	Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types. Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor everolimus has seen significant anti-tumorigenic activity. 	2015-02-26 02:05:37.965377	2015-02-26 02:05:39.793234
5	BRCA Germline Variants	BRCA germline variants have long been recognized as important potential predictors of breast and ovarian cancer risk. 	2015-02-26 02:05:16.650602	2015-02-26 02:05:39.795476
\.


ALTER TABLE variant_groups ENABLE TRIGGER ALL;

--
-- Name: variant_groups_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_groups_id_seq', 15, true);


--
-- Name: variant_origins_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_origins_id_seq', 2, true);


--
-- Data for Name: variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

ALTER TABLE variants DISABLE TRIGGER ALL;

COPY variants (id, gene_id, name, description, created_at, updated_at) FROM stdin;
1	4	BCR-ABL	The BCR-ABL fusion protein, commonly referred to as the philadelphia chromosome, is the most well-studied fusion gene in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML), but despite its ability initiate disease in mice, its status an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and nilotinib) have seen some success in delivering a tumor response.	2015-02-26 02:05:08.604543	2015-02-26 02:05:08.604543
2	4	BCR-ABL T315I	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-02-26 02:05:08.637466	2015-02-26 02:05:08.637466
3	4	BCR-ABL E255K	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-02-26 02:05:08.668949	2015-02-26 02:05:08.668949
4	2	E17K	AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal, lung, and ovarian cancer. It has been convincingly shown to be an activating mutation resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and other AKT1 mutations, are the subject of much research and development for therapeutics.	2015-02-26 02:05:08.692461	2015-02-26 02:05:08.692461
5	1	EML4-ALK	The EML4-ALK fusion has been seen in non-small cell lung cancer, and appears to be an alternative mechanism for ALK activation. Cells with this fusion have been shown to be sensitive to the ALK inhibitor crizotinib.	2015-02-26 02:05:08.716081	2015-02-26 02:05:08.716081
6	1	EML4-ALK C1156Y	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-26 02:05:08.745563	2015-02-26 02:05:08.745563
7	1	EML4-ALK L1196M	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-26 02:05:08.774438	2015-02-26 02:05:08.774438
8	1	F1174L	ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells.	2015-02-26 02:05:08.796314	2015-02-26 02:05:08.796314
9	1	R1275Q	ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma, non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This, and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC cell lines.	2015-02-26 02:05:08.820784	2015-02-26 02:05:08.820784
10	3	S214C	ARAF S214C has been found to be a recurrent oncogenic mutation in non-small cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this mutation, sorafenib also acheived near-complete clinical remission. This case has brought more interest to the variant from a research and clinical perspective.	2015-02-26 02:05:08.844655	2015-02-26 02:05:08.844655
11	5	V600D	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-26 02:05:08.867761	2015-02-26 02:05:08.867761
12	5	V600E	BRAF V600E has been shown to be recurrent in many cancer types. It is one of the most widely studied variants in cancer. This variant is correlated with poor prognosis in certain cancer types, including colorectal cancer and papillary thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib has also shown to be effective when combined with the MEK inhibitor trametinib in colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib, and a number of combination therapies have been successful in treating V600E mutations. While the drugs cetuximab and panitumumab have been largely shown to be ineffective without supplementary treatment.	2015-02-26 02:05:08.891582	2015-02-26 02:05:08.891582
13	5	V600E/V600M	A case study of a single patient harboring both a V600E and a V600M mutation, dabrafenib was shown to acheive clinical response.	2015-02-26 02:05:08.920301	2015-02-26 02:05:08.920301
14	5	V600E AMPLIFICATION	Amplification of BRAF V600E has been shown to confer resistance to MEK inhibitors. For more information on the V600 locus, see the V600E entry.	2015-02-26 02:05:08.944102	2015-02-26 02:05:08.944102
15	5	V600M	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-26 02:05:08.966553	2015-02-26 02:05:08.966553
16	5	V600R	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-26 02:05:08.991965	2015-02-26 02:05:08.991965
17	5	V600	BRAF mutations of the valine 600 residue have been shown to be recurrent in many cancer types. Of the V600 mutations, V600E is the most widely researched. V600 mutations as a whole have been correlated to poorer prognosis in colorectal and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual mutation pages on the left sidebar.	2015-02-26 02:05:09.022981	2015-02-26 02:05:09.022981
18	8	AMPLIFICATION	CCND1 amplification has been implicated in poorer prognosis in non-small cell lung cancer.	2015-02-26 02:05:09.054523	2015-02-26 02:05:09.054523
19	8	EXPRESSION	CCND1 expression, and its prognositc impact, is still in dispute. Three experiments in non-small cell lung cancer have shown it to have no impact on survival, but three additional studies have shown it results in poorer prognosis. There is also some ambiguity in how the boundaries between expression and overexpression are defined.	2015-02-26 02:05:09.081111	2015-02-26 02:05:09.081111
96	38	D842I	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-26 02:05:11.276959	2015-02-26 02:05:11.276959
20	8	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-26 02:05:09.105466	2015-02-26 02:05:09.105466
21	9	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-26 02:05:09.136943	2015-02-26 02:05:09.136943
22	9	PROMOTER DEMETHYLATION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-26 02:05:09.18503	2015-02-26 02:05:09.18503
23	10	LOSS	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL), a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic palbociclib significantly increased the median survival of a Notch-driven model of T-ALL.	2015-02-26 02:05:09.210412	2015-02-26 02:05:09.210412
24	11	OVEREXPRESSION	Cyclin E, while currenly not as widely implicated as its counterpart, cyclin D, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, are correlated with later stage disease and have been shown to lead to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	2015-02-26 02:05:09.239297	2015-02-26 02:05:09.239297
25	13	EXPRESSION	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-26 02:05:09.266061	2015-02-26 02:05:09.266061
26	12	EXPRESSION	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-26 02:05:09.289433	2015-02-26 02:05:09.289433
27	14	PROMOTER HYPERMETHYLATION	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the CDKN2A promoter region. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcomes for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	2015-02-26 02:05:09.313201	2015-02-26 02:05:09.313201
28	15	N-TERMINAL FRAME SHIFT	CEBPA N-terminal frame shift mutations that result in a premature stop codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML). CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-26 02:05:09.339682	2015-02-26 02:05:09.339682
29	15	MUTATION	CEBPA mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-26 02:05:09.366197	2015-02-26 02:05:09.366197
30	33	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-26 02:05:09.394659	2015-02-26 02:05:09.394659
31	16	DNAJB1-PRKACA	This fusion has been found to be very recurrent in the rare form of adolescent liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-26 02:05:09.4171	2015-02-26 02:05:09.4171
32	18	R882	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-02-26 02:05:09.441704	2015-02-26 02:05:09.441704
97	38	D842V	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-26 02:05:11.305898	2015-02-26 02:05:11.305898
33	19	L858R	EGFR L858R has long been recognized as a functionally significant mutation in cancer, and is one of the most prevalent single mutations in lung cancer. Best described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity to first and second generation TKI's like gefitinib and neratinib. NSCLC patients with this mutation treated with TKI's show increased overall and progression-free survival, as compared to chemotherapy alone. Third generation TKI's are currently in clinical trials that specifically focus on mutant forms of EGFR, a few of which have shown efficacy in treating patients that failed to respond to earlier generation TKI therapies.	2015-02-26 02:05:09.469744	2015-02-26 02:05:09.469744
34	19	T790M	EGFR T790M was one of the very first mutations recognized to confer resistance to targeted therapies. While successful in amplified EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients harboring this mutation before treatment is notably lower. This lack of efficacy can likely be to blame for the poorer prognosis for patients with this mutation as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired resistance to TKI inhibition have been shown to harbor this mutation, implicating it as a mechanism of acquired therapy resistence. The third generation TKI's are being developed with this resistance problem in mind, and early stage experiments have shown some efficacy of these drugs in previously resistant tumors with mutant EGFR.	2015-02-26 02:05:09.495027	2015-02-26 02:05:09.495027
35	20	D769H	ERBB2 D769H was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-26 02:05:09.525989	2015-02-26 02:05:09.525989
36	20	D769Y	ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-26 02:05:09.550862	2015-02-26 02:05:09.550862
37	20	DEL 755-759	ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-26 02:05:09.578155	2015-02-26 02:05:09.578155
38	20	G309A	ERBB2 G309A was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-26 02:05:09.606008	2015-02-26 02:05:09.606008
39	20	L755S	ERBB2 L755S was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike many of the other variants queried. This mutation was also shown to confer resistance to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.	2015-02-26 02:05:09.637087	2015-02-26 02:05:09.637087
40	20	L755W	ERBB2 L755W was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-26 02:05:09.663662	2015-02-26 02:05:09.663662
41	20	P780INS	ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-26 02:05:09.691047	2015-02-26 02:05:09.691047
42	20	R678Q	ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-26 02:05:09.717405	2015-02-26 02:05:09.717405
43	20	R896C	ERBB2 R896C was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-26 02:05:09.748551	2015-02-26 02:05:09.748551
44	20	V777L	ERBB2 V777L was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-26 02:05:09.775372	2015-02-26 02:05:09.775372
45	20	V842I	ERBB2 V842I was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-26 02:05:09.801818	2015-02-26 02:05:09.801818
46	21	L536Q	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of these ligand binding domain mutations, and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-26 02:05:09.830356	2015-02-26 02:05:09.830356
47	21	N538G	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-26 02:05:09.875487	2015-02-26 02:05:09.875487
48	21	Y537C	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-26 02:05:09.900234	2015-02-26 02:05:09.900234
49	21	Y537N	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-26 02:05:09.928036	2015-02-26 02:05:09.928036
50	21	Y537S	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-26 02:05:09.954724	2015-02-26 02:05:09.954724
51	22	FGFR2-MGEA5	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. Additionally, tumor size reduction was achieved by both ponatinib and pazopanib treatments administered separately in a single patient with intrahepatic cholangiocarcinoma and this fusion. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-26 02:05:09.981741	2015-02-26 02:05:09.981741
52	23	FGFR3-BAIAP2L1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-26 02:05:10.011716	2015-02-26 02:05:10.011716
53	23	FGFR3-TACC3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-26 02:05:10.038699	2015-02-26 02:05:10.038699
54	24	ITD MUTATIONS	FLT3-ITD (internal tandem duplications) frequently occur in patients with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated sequence resulting in an in-frame duplication event. The length of these duplications can vary widely which may have prognostic consequences, but this has not been conclusively determined. FLT3-ITD mutations overall have generally been associated with poor prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the prognosis associated with this variant.	2015-02-26 02:05:10.068383	2015-02-26 02:05:10.068383
55	24	TKD MUTATIONS	FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common than FLT3-ITD (internal tandem duplication) mutations and may not confer the same prognostic impact. Although the majority of mutations are point mutations effecting D835 (most frequently D835Y), a small proportion involve an in-frame deletion of I836. These mutations are within the activation loop of the second tyrosine kinase domain of FLT3 and thought to result in constitutive activation of the receptor.	2015-02-26 02:05:10.096021	2015-02-26 02:05:10.096021
56	25	EXPRESSION	GATA2 misregulation has been observed in a number of hematologic malignancies, as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung cancer.	2015-02-26 02:05:10.122567	2015-02-26 02:05:10.122567
57	26	R132	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-26 02:05:10.150835	2015-02-26 02:05:10.150835
58	26	R132C	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-26 02:05:10.181002	2015-02-26 02:05:10.181002
59	26	R132H	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-26 02:05:10.208857	2015-02-26 02:05:10.208857
60	26	R132L	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-26 02:05:10.240208	2015-02-26 02:05:10.240208
61	27	R140Q/L	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival as compared to their wild-type counterparts. In myelodysplastic syndromes, however, no prognostic link was discovered between IDH2 mutation status and overall survival.	2015-02-26 02:05:10.284594	2015-02-26 02:05:10.284594
62	27	R172K	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic syndromes, studies did not find a prognostic association between this variant and patient outcomes.	2015-02-26 02:05:10.312126	2015-02-26 02:05:10.312126
63	28	V617F	JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases, occuring in around half of all MPD's. While less associated with cancer, when it is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The V617F mutation is an activating mutation, resulting in increased kinase activity. The mutation seems to be restricted to hematologic malignancies. Treatment of JAK mutant diseases with ruxolitinib has seen some clinical success.	2015-02-26 02:05:10.343489	2015-02-26 02:05:10.343489
64	29	D816V	KIT D816V is a mutation observed in acute myeloid leukemia (AML). This variant has been linked to poorer prognosis and worse outcome in AML patients.	2015-02-26 02:05:10.373933	2015-02-26 02:05:10.373933
65	29	EXON 11 MUTATIONS	c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-02-26 02:05:10.401881	2015-02-26 02:05:10.401881
66	29	INTERNAL DUPLICATION	c-KIT internal duplications have been observed in exon 11, within the juxtamembrane domain. In a case study of an anal melanoma patient harboring this event, imatinib confered marked response. Also, cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-02-26 02:05:10.430503	2015-02-26 02:05:10.430503
67	29	EXON 13 MUTATIONS	c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations. These mutations lie within the tyrosine kinase 1 domain, and are found primarily in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations show sensitivity to imatinib and sunitinib treatment. However, only imatinib has seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant cases.	2015-02-26 02:05:10.463111	2015-02-26 02:05:10.463111
68	29	EXON 14 MUTATIONS	c-KIT exon 14 mutations lie within the tyrosine kinase domain of the protein. While relatively rare in primary gastrointestinal tumors, they are notably more prevalent in refractory disease, suggesting a role in imatinib resistance. Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.	2015-02-26 02:05:10.492078	2015-02-26 02:05:10.492078
69	29	EXON 17 MUTATIONS	c-KIT exon 17 mutations lie within the TK2 domain, containing the activation loop of the protein. In cell lines, mutations within this domain have been shown to be sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.	2015-02-26 02:05:10.519339	2015-02-26 02:05:10.519339
70	29	EXON 9 MUTATIONS	c-KIT exon 9 mutations lie within the dimerization motif of the protein. Relative to other KIT mutations, exon 9 mutations have been associated with better overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and in vivo.	2015-02-26 02:05:10.565867	2015-02-26 02:05:10.565867
71	29	L576P	KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain. It is one of the most recurrent KIT mutations in melanoma, and both in vitro and in vivo studies have shown sensitivity to imatinib.	2015-02-26 02:05:10.593193	2015-02-26 02:05:10.593193
72	29	V654A	KIT V654A is an exon 13 mutation that lies within the tyrosine kinase 1 domain of the protein. It has been shown to be an activating mutation by in vitro studies. This mutation is associated with imatinib resistance in melanoma patients. However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have seen success in acheiving tumor response.	2015-02-26 02:05:10.619312	2015-02-26 02:05:10.619312
73	30	EXON 1 MUTATIONS	A study by Lièvre et al in 2006 showed that colorectal cancer patients with KRAS exon 1 mutations had low cetuximab response rates.	2015-02-26 02:05:10.645216	2015-02-26 02:05:10.645216
74	30	EXON 2 MUTATIONS	In a study by Pao et al in 2005, non-small cell lung cancer patients harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors gefinitib and erlotinib.	2015-02-26 02:05:10.681878	2015-02-26 02:05:10.681878
75	30	G12	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-26 02:05:10.706323	2015-02-26 02:05:10.706323
76	30	G12/G13	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-26 02:05:10.729315	2015-02-26 02:05:10.729315
77	30	G12C	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-26 02:05:10.753646	2015-02-26 02:05:10.753646
78	30	G12D	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-26 02:05:10.779142	2015-02-26 02:05:10.779142
79	30	G13	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-26 02:05:10.804273	2015-02-26 02:05:10.804273
80	30	G13D	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-26 02:05:10.827746	2015-02-26 02:05:10.827746
81	31	P124S	MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder and colon cancer to a lesser degree. The P124S mutation has been shown to contribute to AZD6244 resistance in melanoma cell lines, but considerably less so than its Q56P counterpart.	2015-02-26 02:05:10.858234	2015-02-26 02:05:10.858234
82	31	Q56P	MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This mutation has been shown to confer considerable resistance to AZD6244 treatment of melanoma cell lines.	2015-02-26 02:05:10.886377	2015-02-26 02:05:10.886377
83	32	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-26 02:05:10.913786	2015-02-26 02:05:10.913786
84	34	PROMOTER METHYLATION	MGMT promoter methylation has been observed to impact tumor progression in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating agent carmustine has shown efficacy. In patients lacking methylation, combining carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more experiments are required. Clinical trials have also shown selective sensitivity of promoter methylation-positive patients to temozolomide, making a case for wider methylation screening in GBM patients.	2015-02-26 02:05:10.940182	2015-02-26 02:05:10.940182
85	35	EXON 12 MUTATIONS	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-02-26 02:05:10.96529	2015-02-26 02:05:10.96529
86	35	W288FS	NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most common NPM1 mutation identified in acute myeloid leukemia. This mutation results in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation, it is generally grouped with other exon 12 mutations for patient analysis (see NPM1 Exon 12 variants for more information).	2015-02-26 02:05:10.990408	2015-02-26 02:05:10.990408
87	36	EXON 1 MUTATIONS	NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown to be correlated with poorer overall survival relative to wild-type NRAS in melanoma patients.	2015-02-26 02:05:11.0168	2015-02-26 02:05:11.0168
88	36	EXON 2 MUTATIONS	NRAS exon 2 mutations have been shown to be correlated with poorer overall survival in melanoma patients and colorectal cancer patients, however no prognostic impact was seen in acute myeloid leukemia patients.	2015-02-26 02:05:11.047147	2015-02-26 02:05:11.047147
89	36	EXON 3 MUTATIONS	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-26 02:05:11.071304	2015-02-26 02:05:11.071304
90	36	EXON 4 MUTATIONS	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-26 02:05:11.09714	2015-02-26 02:05:11.09714
91	36	G12	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-26 02:05:11.121795	2015-02-26 02:05:11.121795
92	36	G13D	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-26 02:05:11.14537	2015-02-26 02:05:11.14537
93	36	Q61	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-26 02:05:11.174772	2015-02-26 02:05:11.174772
94	36	Q61L	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-26 02:05:11.218982	2015-02-26 02:05:11.218982
95	36	Q61R	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-26 02:05:11.247194	2015-02-26 02:05:11.247194
98	38	D842Y	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-26 02:05:11.329904	2015-02-26 02:05:11.329904
99	38	DEL I843	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-26 02:05:11.357875	2015-02-26 02:05:11.357875
100	38	DI842-843IM	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-26 02:05:11.385559	2015-02-26 02:05:11.385559
101	37	E542K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-26 02:05:11.412856	2015-02-26 02:05:11.412856
102	37	E545K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-26 02:05:11.440615	2015-02-26 02:05:11.440615
103	37	EXON 20 MUTATIONS	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-26 02:05:11.468913	2015-02-26 02:05:11.468913
104	37	EXON 9 MUTATIONS	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-26 02:05:11.496226	2015-02-26 02:05:11.496226
105	37	H1047R	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-26 02:05:11.522368	2015-02-26 02:05:11.522368
106	39	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	2015-02-26 02:05:11.550158	2015-02-26 02:05:11.550158
107	17	DNAJB1-PRKACA	This fusion has been found to be very recurrent in a rare form of adolescent liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined, and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-26 02:05:11.57595	2015-02-26 02:05:11.57595
108	41	R233*	PTEN R233* has been shown to be a loss of function mutation, and PTEN loss has been the subject of considerable research in breast cancer. PTEN loss may sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data to support that PTEN loss is both associated with poorer prognosis, and no change in prognosis.	2015-02-26 02:05:11.604154	2015-02-26 02:05:11.604154
109	40	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and Arsenic Trioxide for treating these patients, and early results seem promising.	2015-02-26 02:05:11.630766	2015-02-26 02:05:11.630766
110	42	C634W	RET C639W has been implicated as an alternate mechanism of activating RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the C639W mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib.	2015-02-26 02:05:11.657424	2015-02-26 02:05:11.657424
111	42	M918T	RET M819T is the most common somatically acquired mutation in medullary thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the M918T mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T leads to more aggressive MTC with a poorer prognosis.	2015-02-26 02:05:11.684224	2015-02-26 02:05:11.684224
112	44	K666N	SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients.	2015-02-26 02:05:11.711056	2015-02-26 02:05:11.711056
113	44	K700E	SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients. Additionally, these mutations are the most common SF3B1 mutation observed in MDS and highly associated with subtypes of MDS that are defined by ringed sideroblasts.	2015-02-26 02:05:11.737088	2015-02-26 02:05:11.737088
129	6	LOSS-OF-FUNCTION	BRCA1 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.	2015-02-26 02:05:12.168443	2015-02-26 02:05:12.168443
114	45	R175H	While loss-of-function events in TP53 are very common in cancer, the R175H variant seems not only to result in loss of tumor-suppression, but also acts as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell lines harboring this mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been shown that the R175H mutation is correlated with worse overall survival than wild-type TP53, but is not as detrimental as the R248W variant.	2015-02-26 02:05:11.76576	2015-02-26 02:05:11.76576
115	45	R248Q	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival. The R248Q mutation has also shown an increased invasive behavior in cell lines. This is specific to the 248Q variant.	2015-02-26 02:05:11.792784	2015-02-26 02:05:11.792784
116	45	R248W	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival.	2015-02-26 02:05:11.818819	2015-02-26 02:05:11.818819
117	45	R249T	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-26 02:05:11.860962	2015-02-26 02:05:11.860962
118	45	R249W	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-26 02:05:11.889026	2015-02-26 02:05:11.889026
119	45	R273C	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-26 02:05:11.913736	2015-02-26 02:05:11.913736
120	45	R273H	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-26 02:05:11.939359	2015-02-26 02:05:11.939359
121	45	V173G/A	While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-26 02:05:11.966017	2015-02-26 02:05:11.966017
122	46	FRAMESHIFT TRUNCATION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-26 02:05:11.991733	2015-02-26 02:05:11.991733
123	46	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-26 02:05:12.016025	2015-02-26 02:05:12.016025
124	47	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-26 02:05:12.040172	2015-02-26 02:05:12.040172
125	48	Q157P/R	U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is less common than the S34F mutation, occurs in the second zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-02-26 02:05:12.063396	2015-02-26 02:05:12.063396
126	48	S34Y/F	U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is the most commonly identified variant in MDS, occurs in the first zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-02-26 02:05:12.087904	2015-02-26 02:05:12.087904
127	49	EXON 7 MUTATIONS	WT1 exon 7 mutations have been shown to be recurrent in acute myeloid leukemia. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 7 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-26 02:05:12.116016	2015-02-26 02:05:12.116016
128	49	EXON 9 MUTATIONS	WT1 exon 9 mutations have been shown to be recurrent in acute myeloid leukemia, although at a less frequent rate than their exon 7 counterparts. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 9 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-26 02:05:12.14251	2015-02-26 02:05:12.14251
130	7	LOSS-OF-FUNCTION	BRCA2 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.	2015-02-26 02:05:12.195399	2015-02-26 02:05:12.195399
\.


ALTER TABLE variants ENABLE TRIGGER ALL;

--
-- Name: variants_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variants_id_seq', 130, true);


--
-- PostgreSQL database dump complete
--

