--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

SET search_path = public, pg_catalog;

--
-- Data for Name: organizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY organizations (id, name, url, description) FROM stdin;
1	Test Org		This is a test organization with one member.
\.


--
-- Data for Name: audits; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY audits (id, auditable_id, auditable_type, associated_id, associated_type, user_id, user_type, username, action, audited_changes, version, comment, remote_address, request_uuid, created_at) FROM stdin;
1	1	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 238\nname: ALK\ndescription: ALK amplifications, fusions and mutations have been shown to be driving\n  events in non-small cell lung cancer. While crizontinib has demonstrated efficacy\n  in treating the amplification, mutations in ALK have been shown to confer resistance\n  to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied\n  success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been\n  shown to be cytostatic in ALK-altered cell lines.\nofficial_name: anaplastic lymphoma receptor tyrosine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	c69abf47-faeb-4d3e-ad3f-d45a904ee9db	2015-07-22 18:46:55.395966
2	2	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 207\nname: AKT1\ndescription: AKT1, also referred to as protein kinase B, is a known oncogene. AKT\n  activation relies on the PI3K pathway, and is recognized as a critical node in the\n  pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been\n  shown to result in activation of the protein. Mutations in AKT1 have also been shown\n  to confer resistance to allosteric kinase inhibitors in vitro.\nofficial_name: v-akt murine thymoma viral oncogene homolog 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	2390ff02-bb9c-47e3-a3c9-f33c6c386205	2015-07-22 18:46:55.746246
3	3	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 369\nname: ARAF\ndescription: ARAF has recently become increasingly considered for its oncogenic potential.\n  Its potential as a target for informing clinical action was demonstrated by a single\n  case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated\n  with sorafenib, acheived near-complete clinical remission. This finding has brought\n  new focus on ARAF as a marker that should be assayed for in cancer treatment.\nofficial_name: A-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	f507a101-7738-40dd-9872-7158a9dc078c	2015-07-22 18:46:55.989746
4	4	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 25\nname: ABL1\ndescription: ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein\n  that has become a signature of chronic myeloid leukemia (CML). Cells harboring this\n  fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook\n  of the disease. However, additional mutations in ABL1 have been shown to confer\n  resistance to imatinib. In these resistance cases, second-generation tyrosine kinase\n  inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are\n  currently undergoing clinical trials for treating acquired resistance in CML.\nofficial_name: ABL proto-oncogene 1, non-receptor tyrosine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	d07738ef-a69d-4ccc-a80a-4f7f3716ec87	2015-07-22 18:46:56.270726
5	5	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 673\nname: BRAF\ndescription: BRAF mutations are found to be recurrent in many cancer types. Of these,\n  the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E\n  has been determined to be an activating mutation, and cells that harbor it, along\n  with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is\n  also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK\n  inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations\n  have also been correlated with poor prognosis in many cancer types, although there\n  is at least one study that questions this conclusion in papillary thyroid cancer.\nofficial_name: B-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	0757a879-c5c8-4338-9c25-e4191fb3ecfc	2015-07-22 18:46:56.569677
6	6	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 672\nname: BRCA1\ndescription: BRCA1 mutations in the germline have become a hallmark for hereditary\n  breast and ovarian cancers. Variants that have been demonstrated to reduce the function\n  of the protein have been shown to increase risk for these cancers, as well as prostate\n  and pancreatic cancer. These findings have been the impetus for the increased popularity\n  of genetic testing of healthy indivudals to assess risk.\nofficial_name: breast cancer 1, early onset\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	8b6b3c16-c185-408c-a5d3-549b01eae80f	2015-07-22 18:46:56.845926
7	7	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 675\nname: BRCA2\ndescription: BRCA2 mutations in the germline have become a hallmark for hereditary\n  breast and ovarian cancers. Variants that have been demonstrated to reduce the function\n  of the protein have been shown to increase risk for these cancers, as well as prostate\n  and pancreatic cancer. These findings have been the impetus for the increased popularity\n  of genetic testing of healthy indivudals to assess risk.\nofficial_name: breast cancer 2, early onset\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	dcdade67-b035-432d-b5ed-e704fa668d4a	2015-07-22 18:46:57.110632
8	8	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 595\nname: CCND1\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	5bdaf0aa-393d-4cbf-a63f-8ee98eb35429	2015-07-22 18:46:57.341057
9	9	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 894\nname: CCND2\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	5eb0dd96-1837-4de0-9058-3aafb3b4a2a3	2015-07-22 18:46:57.56789
10	10	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 896\nname: CCND3\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared\n  to the amplifcations and overexpressions of the other cyclin D's. In a mouse study,\n  the targeted therapeutic palbociclib significantly increased the median survival\n  of the cyclin D3 knockouts.\nofficial_name: cyclin D3\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	2779c9ca-1ba4-42eb-93e4-53ede4bdded7	2015-07-22 18:46:57.755391
11	11	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 898\nname: CCNE1\ndescription: Cyclin E, while currenly not as widely implicated as its cyclin D counterparts,\n  has been implicated in various carcinomas, including breast, gastric, stomach and\n  colorectal. High levels of cyclin E, either by gene amplification or overexpression,\n  have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements\n  are correlated with later stage disease. In lung cancer, neoplastic cells with higher\n  levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly\n  expressed counterparts.\nofficial_name: cyclin E1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	db8a2a26-fee0-43ad-8139-e9ac15581806	2015-07-22 18:46:57.951798
12	12	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1021\nname: CDK6\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 6\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	2791ecc8-ed95-4e1d-9865-f1f212237427	2015-07-22 18:46:58.168778
13	13	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1019\nname: CDK4\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 4\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	a6065dc1-8a42-4452-8388-1bc39f417b19	2015-07-22 18:46:58.379514
14	14	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1029\nname: CDKN2A\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. While no targeted therapeutic has been engaged in clinical trials, the prognostic\n  impact has been studied by a number of meta-analyses. One mechanism by which loss\n  of CDKN2A can occur is by hypermethylation of the promoter region for the gene.\n  However, the prognostic impact of promoter hypermethylation has been relatively\n  ambiguous. Many studies have suggesting poorer prognostic outcome for patients with\n  hypermethylation in colorectal, liver, and younger lung cancer patients. This being\n  said, there is still research to be done before this becomes a widely-accepted prognostic\n  indicator.\nofficial_name: cyclin-dependent kinase inhibitor 2A\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	89dc3a3d-3ae2-4bc2-b166-520f5bbce69a	2015-07-22 18:46:58.661485
15	15	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1050\nname: CEBPA\ndescription: AML with mutated CEBPA' is a provisional entity in the WHO classification\n  of acute myeloid leukemia (AML) and is recommended to be tested for in patients\n  with AML. CEBPA mutations are particularly associated with cytogenetically normal\n  AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation\n  in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha\n  protein while C-terminal mutations reduce the DNA-binding potential of this transcription\n  factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1\n  and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations\n  may have prognostic implications.\nofficial_name: CCAAT/enhancer binding protein (C/EBP), alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	e9c22a19-0604-455b-8851-0e188d154130	2015-07-22 18:46:58.912433
16	16	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3337\nname: DNAJB1\ndescription: DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating\n  p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1\n  to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In\n  one study, this fusion was observed in 15/15 FL-HCC cases examined and functional\n  studies found that the fusion retained kinase activity.\nofficial_name: DnaJ (Hsp40) homolog, subfamily B, member 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	fc3df232-634c-48ef-850e-4718fa9b3287	2015-07-22 18:46:59.156318
17	17	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5566\nname: PRKACA\ndescription: PRKACA has been studied in breast cancer and has been found to mediate\n  resistance to HER2 targeted therapies. It has also been found to contain a mutation\n  hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular\n  fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be\n  a diagnostic marker for this rare subtype of HCC. In one study, this fusion was\n  observed in 15/15 FL-HCC cases examined and functional studies found that the fusion\n  retained kinase activity.\nofficial_name: protein kinase, cAMP-dependent, catalytic, alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	6cc7acc7-61d6-43d1-b032-e8cb1cb2fc09	2015-07-22 18:46:59.382833
18	18	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1788\nname: DNMT3A\ndescription: DNMT3A is one of several epigenetic modifiers identified as recurrently\n  mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically\n  normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant\n  negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers.\n  AML patient bone marrow harboring R882 mutations were similarly demonstrated to\n  be hypomethylated compared to patients with wildtype DNMT3A. These studies also\n  indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner\n  from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes\n  and treatment protocols may need to be considered for these two classes of DNMT3A\n  mutations.\nofficial_name: DNA (cytosine-5-)-methyltransferase 3 alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	bb5e38aa-8a8f-4f1c-b014-541bc4a4fca5	2015-07-22 18:46:59.594025
19	19	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1956\nname: EGFR\ndescription: EGFR is widely recognized for its importance in cancer. Amplification\n  and mutations have been shown to be driving events in many cancer types. Its role\n  in non-small cell lung cancer has spurred many research and drug development efforts.\n  Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably\n  gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance\n  to these drugs, particularly the variant T790M, which has been functionally characterized\n  as a resistance marker for both of these drugs. The later generation TKI's have\n  seen some success in treating these resistant cases, and targeted sequencing of\n  the EGFR locus has become a common practice in non-small cell lung cancer.\nofficial_name: epidermal growth factor receptor\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b21e17e2-82be-4aa7-bf4b-d568f7893173	2015-07-22 18:46:59.85438
57	57	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5966\nname: REL\ndescription: ''\nofficial_name: v-rel avian reticuloendotheliosis viral oncogene homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	44b1f3d4-abd5-4e2c-8b9a-2c7c4664f8fe	2015-07-22 18:47:08.741419
59	59	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 596\nname: BCL2\ndescription: ''\nofficial_name: B-cell CLL/lymphoma 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	be57c579-aaeb-46ac-8fbe-737c151a9b34	2015-07-22 18:47:09.147126
20	20	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2064\nname: ERBB2\ndescription: ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast\n  cancer, and is treated in a separate manner than the other subtypes of breast cancer.\n  Apart from being amplified, ERBB2 activating mutations have been shown to have clinical\n  importance in HER2-negative breast cancer. These mutations have shown sensitivity\n  to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical\n  sequencing efforts in treating breast cancer.\nofficial_name: erb-b2 receptor tyrosine kinase 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	bb6a8028-61bd-48d2-ae77-aa1a7e89890f	2015-07-22 18:47:00.142769
21	21	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2099\nname: ESR1\ndescription: ESR1 has been a focus in breast cancer for quite some time, but has also\n  shown significance in endometrial, ovarian and other cancer types. ER-positive breast\n  cancer that is resistant to hormone therapy has instigated clinical sequencing efforts\n  to shed light on the mechanisms of this resistance. A number of mutations in the\n  ligand binding domain of the protein have been implicated in hormone resistance\n  and anti-estrogen therapies. This has spurred efforts to develop therapeutics that\n  act to degrade the protein, rather than act as an antagonist. These agents are currently\n  in clinical trials and have seen some success, highlighting the importance of sequencing\n  efforts in treating breast cancer.\nofficial_name: estrogen receptor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	a15b1524-4977-436c-8873-2476ecef1b6a	2015-07-22 18:47:00.417986
22	22	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2263\nname: FGFR2\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions that lead to pathway\n  actiation have been demonstrated to have oncogenic potential across multiple cancer\n  types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success\n  in treating over-active FGFR signalling, prompting use of diagnostic sequencing\n  targeting the FGFR genes, especially in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	2c88e8ea-1d17-4b65-b5a4-ededf38f88a5	2015-07-22 18:47:00.702192
23	23	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2261\nname: FGFR3\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions that lead to pathway\n  actiation have been demonstrated to have oncogenic potential across multiple cancer\n  types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success\n  in treating over-active FGFR signalling, prompting use of diagnostic sequencing\n  targeting the FGFR genes, especially in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 3\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	2f7f9dc3-63f2-4875-86dc-570f87f7b192	2015-07-22 18:47:00.957378
24	24	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2322\nname: FLT3\ndescription: FLT3 is an important cytokine receptor involved in normal hematopoiesis.\n  Mutations in this gene are common in acute myeloid leukemia (AML) and screening\n  for mutations in this gene has been recommended by the World Health Organization\n  in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML).\n  FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated\n  with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have\n  been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD\n  mutations are still up for debate.\nofficial_name: fms-related tyrosine kinase 3\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	d1499bca-c2ec-4dea-b634-8bc959230161	2015-07-22 18:47:01.178909
25	25	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2624\nname: GATA2\ndescription: GATA2 is a transcription factor involved in stem cell maintenance with\n  key roles in hematopoietic development. GATA2 mutations are associated with a variety\n  of inherited and acquired immune disorders including myelodysplastic syndrome and\n  acute myeloid leukemia. In addition to a role in hematopoiesis, the maintenance\n  GATA2 expression has been implicated as a requirement in KRAS-driven non-small cell\n  lung cancer. Preclinical models have indicated therapeutic benefit from targeting\n  GATA2-mediated pathways in the context of KRAS-driven NSCLC.\nofficial_name: GATA binding protein 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	10940be3-4d20-4e9b-b38d-2bd88e2bc724	2015-07-22 18:47:01.405509
26	26	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3417\nname: IDH1\ndescription: IDH1 mutations have been observed in a number of cancer types, including\n  sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in\n  the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial\n  (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described\n  as mutually exclusive in many of these cancer types. The most frequent mutations\n  involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic\n  enzyme activity. The implications of mutations in this gene vary greatly by cancer\n  type. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations\n  have been associated with worse outcome, shorter overall survival, and normal karyotype.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Unlike the association\n  with cytogenetically normal AML, in glioblastoma, IDH1 mutations have been associated\n  with specific cytogenetic abnormalities, 1p and 19q deletions.\nofficial_name: isocitrate dehydrogenase 1 (NADP+), soluble\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	9bc3332b-1279-4a34-a6bc-fb2a45427730	2015-07-22 18:47:01.658509
27	27	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3418\nname: IDH2\ndescription: IDH2 mutations have been observed in a number of cancer types, including\n  sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in\n  the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial\n  (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described\n  as mutually exclusive in many of these cancer types. The most frequent mutations\n  involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic\n  enzyme activity. Although IDH2 (R172) mutations are associated with poorer overall\n  prognosis in AML patients, its utility as a prognostic marker in MDS is still under\n  debate. Additionally, IDH2 (R140) has been associated with improved overall survival\n  in AML. IDH2 mutations have been associated with improved prognosis in gliomas.\nofficial_name: isocitrate dehydrogenase 2 (NADP+), mitochondrial\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	736bd24e-4b79-467d-ac16-8b08d54a7ba1	2015-07-22 18:47:01.912093
28	28	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3717\nname: JAK2\ndescription: JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative\n  diseases and cancers. The mutation V617F is the most clinically relevant variant,\n  and is seen in around half of myeloproliferative disorders. The variant is a known\n  activating mutation, and activated JAK2 is sufficient to drive myeloproliferative\n  disorders in mouse models. V617F, while most recurrent, is not the only mechanism\n  by which JAK2 can be activated in patients. JAK2 is now one of the first diagnostic\n  markers tested upon diagnosis with a myeloproliferative disorder.\nofficial_name: Janus kinase 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	da50d67f-eb57-48d9-9d8f-14393bbfa5cb	2015-07-22 18:47:02.133728
29	29	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3815\nname: KIT\ndescription: c-KIT activation has been shown to have oncogenic activity in gastrointestinal\n  stromal tumors (GISTs), melanomas, lung cancer, and other tumor types. The targeted\n  therapeutics nilotinib and sunitinib have shown efficacy in treating KIT overactive\n  patients, and are in late-stage trials in melanoma and GIST. KIT overactivity can\n  be the result of many genomic events from genomic amplification to overexpression\n  to missense mutations. Missense mutations have been shown to be key players in mediating\n  clinical response and acquired resistance in patients being treated with these targeted\n  therapeutics.\nofficial_name: v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	1acfc137-9bec-4a72-bc34-4b3923c7ff79	2015-07-22 18:47:02.34645
30	30	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3845\nname: KRAS\ndescription: Mutations in the RAS family of proteins have frequently observed across\n  cancer types. The amino acid positions account for the overwhelming majority of\n  these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also\n  behave very differently when expressed in non-native tissue types, likely due to\n  differences in the C-terminal hyper-variable regions. Mis-regulation of isoform\n  expression has been shown to be a driving event in cancer, as well as missense mutations\n  at the three hotspots previously mentioned. While highly recurrent in cancer, targeted\n  these RAS mutants has also been very elusive, and has not yet become common practice\n  in the clinic.\nofficial_name: Kirsten rat sarcoma viral oncogene homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	61481939-30f9-4c7f-87b3-5e243618848b	2015-07-22 18:47:02.611077
31	31	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5604\nname: MAP2K1\ndescription: ''\nofficial_name: mitogen-activated protein kinase kinase 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	7e2f2e0a-b80c-409b-b8c2-255400ec3f55	2015-07-22 18:47:02.845261
32	32	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4209\nname: MEF2D\ndescription: ''\nofficial_name: myocyte enhancer factor 2D\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	9889b9e1-e619-43f9-9c85-35b1fe4cb6aa	2015-07-22 18:47:03.041479
33	33	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1436\nname: CSF1R\ndescription: ''\nofficial_name: colony stimulating factor 1 receptor\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b4618645-31b3-4e10-9642-76783a6708ef	2015-07-22 18:47:03.255537
34	34	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4255\nname: MGMT\ndescription: ''\nofficial_name: O-6-methylguanine-DNA methyltransferase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	a57771fc-673f-40c6-bcbf-3bafc70f14fe	2015-07-22 18:47:03.443263
35	35	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4869\nname: NPM1\ndescription: AML with mutated NPM1 is provisional entity in the WHO classification\n  of AML and is recommended to be tested for in patients with cytogenetically normal\n  AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they\n  have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently\n  W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations\n  in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing\n  the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia,\n  indicating it may require additional mutations to promote leukemic development.\nofficial_name: nucleophosmin (nucleolar phosphoprotein B23, numatrin)\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	7ffa48f3-79cd-47f4-8725-e58d5e37ddaa	2015-07-22 18:47:03.707128
36	36	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4893\nname: NRAS\ndescription: Mutations in the RAS family of proteins have frequently observed across\n  cancer types. The amino acid positions account for the overwhelming majority of\n  these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also\n  behave very differently when expressed in non-native tissue types, likely due to\n  differences in the C-terminal hyper-variable regions. Mis-regulation of isoform\n  expression has been shown to be a driving event in cancer, as well as missense mutations\n  at the three hotspots previously mentioned. While highly recurrent in cancer, targeted\n  these RAS mutants has also been very elusive, and has not yet become common practice\n  in the clinic.\nofficial_name: neuroblastoma RAS viral (v-ras) oncogene homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	5976316f-045d-42d8-b040-3ff7e1969e61	2015-07-22 18:47:03.934555
37	37	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5290\nname: PIK3CA\ndescription: PIK3CA is the most recurrently mutated gene in breast cancer, and has\n  been found to important in a number of cancer types. An integral part of the PI3K\n  pathway, PIK3CA has long been described as an oncogene, with two main hotspots for\n  activating mutations, the 542/545 region of the helical domain, and the 1047 region\n  of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways,\n  is the subject of an immense amount of research and development, and PI3K inhibition\n  has seen some limited success in recent clinical trials. While monotherapies seem\n  to be limited in their potential, there is a recent interest in pursuing PI3K inhibition\n  as part of a combination therapy regiment with inhibition partners including TKI's,\n  MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.\nofficial_name: phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	837387ed-3ea8-4e83-8206-74bda784c0b1	2015-07-22 18:47:04.164689
38	38	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5156\nname: PDGFRA\ndescription: ''\nofficial_name: platelet-derived growth factor receptor, alpha polypeptide\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	50f17c80-14f2-4722-8b5d-f1ea2b5411f6	2015-07-22 18:47:04.377732
39	39	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5371\nname: PML\ndescription: The PML-RARA fusion is the result of a recurrent, balanced translocation\n  between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event\n  in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown\n  sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA\n  fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for\n  treating these patients, and early results seem promising.\nofficial_name: promyelocytic leukemia\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	47e4c906-a0e2-4c07-920d-8c0e680ae948	2015-07-22 18:47:04.58937
58	58	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7428\nname: VHL\ndescription: ''\nofficial_name: von Hippel-Lindau tumor suppressor, E3 ubiquitin protein ligase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	eeacc4f3-2a67-4a34-9364-f19bc20852d8	2015-07-22 18:47:08.952058
40	40	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5914\nname: RARA\ndescription: The PML-RARA fusion is the result of a recurrent, balanced translocation\n  between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event\n  in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown\n  sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA\n  fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for\n  treating these patients, and early results seem promising.\nofficial_name: retinoic acid receptor, alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	edc6b65d-ca0f-483c-b065-5b9d1fb886b3	2015-07-22 18:47:04.800125
41	41	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5728\nname: PTEN\ndescription: ''\nofficial_name: phosphatase and tensin homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	63a1a7a1-c617-4ea5-abe7-ad27cffd06cb	2015-07-22 18:47:05.097441
42	42	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5979\nname: RET\ndescription: RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine\n  kinase receptor and are associated with thyroid cancers. RET point mutations are\n  the most common mutations identified in medullary thyroid cancer (MTC) with germline\n  and somatic mutations in RET associated with hereditary and sporadic forms, respectively.\n  The most common somatic form mutation is M918T (exon 16) and a variety of other\n  mutations effecting exons 10, 11 and 15 have been described. The prognostic significance\n  of these mutations have been hotly debated in the field, however, data suggests\n  that some RET mutation may confer drug resistence. No RET-specific agents are currently\n  clinically available but several promiscuous kinase inhibitors that target RET,\n  among others, have been approved for MTC treatment.\nofficial_name: ret proto-oncogene\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	f2dd3cf9-d277-4343-af2c-457e77085d86	2015-07-22 18:47:05.373279
43	43	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 861\nname: RUNX1\ndescription: ''\nofficial_name: runt-related transcription factor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	601c9bf4-c260-4e3a-a963-067664950125	2015-07-22 18:47:05.63589
44	44	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 23451\nname: SF3B1\ndescription: SF3B1 mutations have been described in several myeloid malignancies,\n  predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies\n  and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has\n  been identified as recurrently mutated in MDS and other malignanices. The mutations\n  affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally\n  deleterious with K700E described as a major hotspot mutation. MDS patients with\n  SF3B1 mutations have been reported to have better overall and event-free survival\n  than their wildtype counterparts. Additionally, these mutations are highly associated\n  with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with\n  ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring\n  sideroblasts).\nofficial_name: splicing factor 3b, subunit 1, 155kDa\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	f2a56b39-e653-4ef0-b16c-20901a45a922	2015-07-22 18:47:05.884875
45	45	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7157\nname: TP53\ndescription: TP53 mutations are universal across cancer types. Loss of tumor suppressors\n  is most recognized by large deleterious events, such as frameshift mutations, or\n  premature stop codons. In TP53 however, many of the observed mutations in cancer\n  are found to be single nucleotide variants, or missense mutations. These variants\n  are also very broadly distributed throughout the gene, not localizing in any particular\n  hotspot. While a large proportion of cancer genomics research is focused on somatic\n  variants, TP53 is also of note in the germline. Germline TP53 mutations are the\n  hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been\n  found to have prognostic impact on patient outcomes.\nofficial_name: tumor protein p53\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	3a700fee-e1b8-41a2-ad46-764223fc5c75	2015-07-22 18:47:06.160061
46	46	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7248\nname: TSC1\ndescription: ''\nofficial_name: tuberous sclerosis 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	8088011f-78cd-4f68-94b8-e036854b1dad	2015-07-22 18:47:06.42284
47	47	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7249\nname: TSC2\ndescription: ''\nofficial_name: tuberous sclerosis 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	f8f45a70-1a5a-4f39-bc00-99e0ae6d71f9	2015-07-22 18:47:06.623922
48	48	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7307\nname: U2AF1\ndescription: U2AF1 is one of several spliceosome complex genes frequently mutated\n  in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes\n  (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot\n  mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein.\n  These mutations have been associated with altered splicing patterns in vitro and\n  in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation\n  to secondary acute myeloid leukemia, however, the impact of these mutations on overall\n  survival has been an area of debate.\nofficial_name: U2 small nuclear RNA auxiliary factor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	fc1a4558-2baa-4d7e-96cb-f7cda6f78b02	2015-07-22 18:47:06.843259
49	49	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7490\nname: WT1\ndescription: WT1 is a tumor suppressor gene associated with the development of Wilms'\n  Tumor, from which it was named. Mutations in exon 7 and 9 of WT1 have been recurrently\n  identified in acute myeloid leukemia and associated with poorer prognosis and chemotherapy\n  resistance.\nofficial_name: Wilms tumor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	89296862-63d0-4223-ab22-f329b30dc107	2015-07-22 18:47:07.088836
50	50	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4851\nname: NOTCH1\ndescription: ''\nofficial_name: notch 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	4d592088-9168-47d0-aeb1-37951baf7a11	2015-07-22 18:47:07.304086
51	51	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4921\nname: DDR2\ndescription: ''\nofficial_name: discoidin domain receptor tyrosine kinase 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	d0a4428e-c20e-463a-a1b4-af27e0705061	2015-07-22 18:47:07.512128
52	52	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4233\nname: MET\ndescription: ''\nofficial_name: MET proto-oncogene, receptor tyrosine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	50bebbcb-32d2-41ce-aa7e-a0fb247ee8eb	2015-07-22 18:47:07.729846
53	53	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2313\nname: FLI1\ndescription: ''\nofficial_name: Fli-1 proto-oncogene, ETS transcription factor\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	6223c6cd-509e-44fc-9f4d-467cf8616b5e	2015-07-22 18:47:07.944047
54	54	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2130\nname: EWSR1\ndescription: ''\nofficial_name: EWS RNA-binding protein 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	122b0f22-958c-4b41-aab7-6dfa673ca82d	2015-07-22 18:47:08.144228
55	55	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 54790\nname: TET2\ndescription: ''\nofficial_name: tet methylcytosine dioxygenase 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	3a223727-d76c-42ca-86c4-6b2715e4f2d5	2015-07-22 18:47:08.337827
56	56	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 27086\nname: FOXP1\ndescription: ''\nofficial_name: forkhead box P1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	425b71d9-1071-41ee-8fb4-599c62dc5de0	2015-07-22 18:47:08.54555
60	60	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3481\nname: IGF2\ndescription: ''\nofficial_name: insulin-like growth factor 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	df5079f6-5b2b-450c-aa1f-c48d7727ac5f	2015-07-22 18:47:09.343875
61	61	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 6790\nname: AURKA\ndescription: ''\nofficial_name: aurora kinase A\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	221e172f-3ddf-424f-b5cd-13979ac4adf2	2015-07-22 18:47:09.578705
62	62	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 55193\nname: PBRM1\ndescription: ''\nofficial_name: polybromo 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	02ec0cdf-742b-47d9-933a-18c9764329a8	2015-07-22 18:47:09.779634
63	63	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2146\nname: EZH2\ndescription: ''\nofficial_name: enhancer of zeste 2 polycomb repressive complex 2 subunit\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	451bf31c-a1f4-4086-aaa9-654f601b60f9	2015-07-22 18:47:10.031133
64	1	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL\ndescription: The BCR-ABL fusion protein, commonly referred to as the philadelphia\n  chromosome, is the most well-studied fusion gene in cancer. It has widely been considered\n  the initiating event in chronic myelogenous leukemia (CML), but despite its ability\n  initiate disease in mice, its status an initiating mutation is in dispute. In what\n  is commonly used as the poster-child for targeted therapeutics, the development\n  and use of imatinib in the clinic has led to profound improvements in the prognosis\n  of the disease. However, imatinib resistance is still seen in patients with mutations\n  in the ABL kinase domain of the fusion, most notably the T315I variant. In patients\n  resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's\n  (dasatinib and nilotinib) have seen some success in delivering a tumor response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	51e995e0-51cb-44e5-951c-50a57e4cf4aa	2015-07-22 18:47:10.150844
65	2	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL T315I\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation T315I has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	1428bcf3-184f-4708-b7cf-2845790efc3e	2015-07-22 18:47:10.174344
66	3	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL E255K\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation E255K has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	9bfa69ab-fd9a-422a-9ab8-1ad5ac779f6a	2015-07-22 18:47:10.201452
67	4	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 2\nname: E17K\ndescription: AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal,\n  lung, and ovarian cancer. It has been convincingly shown to be an activating mutation\n  resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation\n  decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and\n  other AKT1 mutations, are the subject of much research and development for therapeutics.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d8db273a-915e-423c-9da5-1703271cb587	2015-07-22 18:47:10.226359
68	5	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK\ndescription: The EML4-ALK fusion has been seen in non-small cell lung cancer, and\n  appears to be an alternative mechanism for ALK activation. Cells with this fusion\n  have been shown to be sensitive to the ALK inhibitor crizotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	33b476e2-87f0-4d1f-9f23-1f0f8581930b	2015-07-22 18:47:10.256956
69	6	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK C1156Y\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	e4d12535-383a-452b-a361-7a5af26c179c	2015-07-22 18:47:10.282822
70	7	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK L1196M\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	94b87537-9761-43d4-b52d-5a5dc2a1d14e	2015-07-22 18:47:10.308261
71	8	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: F1174L\ndescription: ALK F1174L has been observed as recurrent in neuroblastoma, non-small\n  cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing\n  this mutation have shown resistance to low doses of criztonib. However, increased\n  dosage can overcome this resistance in cell lines studies. TAE684 has also proven\n  effective in both NSCLC and neuroblastoma F1174L containing cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	356c2ad4-431d-4f2c-81f0-4e8844fb92c9	2015-07-22 18:47:10.330583
72	9	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: R1275Q\ndescription: ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma,\n  non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma\n  cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This,\n  and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC\n  cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	61cdb518-29c5-4dd4-beab-f30b7bf48679	2015-07-22 18:47:10.351457
93	30	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d4fa77c9-dfff-4937-af98-eaa87a17d49f	2015-07-22 18:47:10.819244
73	10	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 3\nname: S214C\ndescription: ARAF S214C has been found to be a recurrent oncogenic mutation in non-small\n  cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib\n  in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this\n  mutation, sorafenib also acheived near-complete clinical remission. This case has\n  brought more interest to the variant from a research and clinical perspective.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	9b0150e2-2f31-4109-9b86-50f7d5722979	2015-07-22 18:47:10.369647
74	11	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600D\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	64da4d16-4b5b-49e0-95bf-8f804ac8abd1	2015-07-22 18:47:10.391175
75	12	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E\ndescription: BRAF V600E has been shown to be recurrent in many cancer types. It is\n  one of the most widely studied variants in cancer. This variant is correlated with\n  poor prognosis in certain cancer types, including colorectal cancer and papillary\n  thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective\n  in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib\n  has also shown to be effective when combined with the MEK inhibitor trametinib in\n  colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA\n  mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib,\n  and a number of combination therapies have been successful in treating V600E mutations.\n  While the drugs cetuximab and panitumumab have been largely shown to be ineffective\n  without supplementary treatment.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d4b84b43-9184-4e13-9e9c-ee2f2ca2df89	2015-07-22 18:47:10.410665
76	13	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E/V600M\ndescription: A case study of a single patient harboring both a V600E and a V600M mutation,\n  dabrafenib was shown to acheive clinical response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	a8592c8d-7fa3-4b2e-a9d6-c23bb7b00392	2015-07-22 18:47:10.435014
77	14	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E AMPLIFICATION\ndescription: Amplification of BRAF V600E has been shown to confer resistance to MEK\n  inhibitors. For more information on the V600 locus, see the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	536d5fd3-2fca-43c6-a1a2-bf6633cefc39	2015-07-22 18:47:10.461541
78	15	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600M\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	75759aef-894d-4ee8-addb-c1b70e0f6ab1	2015-07-22 18:47:10.479967
79	16	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600R\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	bb0f5052-8c36-46ad-9455-6f69e09eab03	2015-07-22 18:47:10.502815
80	17	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600\ndescription: BRAF mutations of the valine 600 residue have been shown to be recurrent\n  in many cancer types. Of the V600 mutations, V600E is the most widely researched.\n  V600 mutations as a whole have been correlated to poorer prognosis in colorectal\n  and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity\n  to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual\n  mutation pages on the left sidebar.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d7924e5a-f9b6-4130-861d-edf30d69c43c	2015-07-22 18:47:10.522952
81	18	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: AMPLIFICATION\ndescription: CCND1 amplification has been implicated in poorer prognosis in non-small\n  cell lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	67cf55c7-1289-4f77-8bdc-c052a13cc776	2015-07-22 18:47:10.541287
82	19	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: EXPRESSION\ndescription: CCND1 expression, and its prognositc impact, is still in dispute. Three\n  experiments in non-small cell lung cancer have shown it to have no impact on survival,\n  but three additional studies have shown it results in poorer prognosis. There is\n  also some ambiguity in how the boundaries between expression and overexpression\n  are defined.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c1e64fdc-49bb-49b7-a40c-dfafbe3fccf1	2015-07-22 18:47:10.563251
83	20	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	66bc9550-af39-4010-ad0b-4cd9f775face	2015-07-22 18:47:10.583518
102	39	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: L755S\ndescription: ERBB2 L755S was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike\n  many of the other variants queried. This mutation was also shown to confer resistance\n  to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	6441e15d-aa2f-4c39-9500-0df11b85e5c1	2015-07-22 18:47:11.020224
84	21	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	b6993152-9dbc-47e4-92c5-e51ba0141941	2015-07-22 18:47:10.603918
85	22	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: PROMOTER DEMETHYLATION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	bab10561-cc2f-42e8-850d-674cebc546ad	2015-07-22 18:47:10.630121
86	23	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 10\nname: LOSS\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL),\n  a seemingly unique trend when compared to the amplifcations and overexpressions\n  of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic\n  palbociclib significantly increased the median survival of a Notch-driven model\n  of T-ALL.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ced1dd46-b239-4f57-8b2d-0f350face958	2015-07-22 18:47:10.672347
87	24	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 11\nname: OVEREXPRESSION\ndescription: Cyclin E, while currenly not as widely implicated as its counterpart,\n  cyclin D, has been implicated in various carcinomas, including breast, gastric,\n  stomach and colorectal. High levels of cyclin E, either by gene amplification or\n  overexpression, are correlated with later stage disease and have been shown to lead\n  to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer\n  specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex\n  are more radiosensitive than their lowly expressed counterparts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	171abdbf-0c02-4974-a45d-fe0dc2846de4	2015-07-22 18:47:10.692998
88	25	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: EXPRESSION\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	7a3c4883-1f5e-4e73-b9a9-3110e4138cb7	2015-07-22 18:47:10.713406
89	26	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 12\nname: EXPRESSION\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	fc4e2009-9f52-472b-8d2c-c4569da1972a	2015-07-22 18:47:10.73267
90	27	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 14\nname: PROMOTER HYPERMETHYLATION\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the CDKN2A\n  promoter region. While no targeted therapeutic has been engaged in clinical trials,\n  the prognostic impact has been studied by a number of meta-analyses. The prognostic\n  impact of promoter hypermethylation has been relatively ambiguous. Many studies\n  have shown significant p-values suggesting poorer prognostic outcomes for patients\n  with hypermethylation in colorectal, liver, and younger lung cancer patients. This\n  being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	28ff57b5-dd5d-4b68-9edf-1afcc382cef4	2015-07-22 18:47:10.758058
91	28	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: N-TERMINAL FRAME SHIFT\ndescription: CEBPA N-terminal frame shift mutations that result in a premature stop\n  codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML).\n  CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3\n  mutations should also be assessed in CN-AML patients as concurrent mutations may\n  have prognostic implications.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	7ffc6fda-7aaf-4fe0-9d60-7ce2594a99e3	2015-07-22 18:47:10.776769
92	29	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: MUTATION\ndescription: CEBPA mutations are associated with cytogenetically normal acute myeloid\n  leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should\n  also be assessed in CN-AML patients as concurrent mutations may have prognostic\n  implications.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c8cd86df-2acb-4fd3-ac00-294c2cd38ef7	2015-07-22 18:47:10.796579
94	31	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in the rare form of adolescent\n  liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion\n  was observed in 15/15 FL-HCC cases examined and functional studies found that the\n  fusion retained kinase activity. The presence of this fusion may be used as a diagnostic\n  marker for this rare tumor type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	3e8706b1-0558-4f37-ac05-9218eafbb319	2015-07-22 18:47:10.842021
95	32	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R882\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	941fae42-940e-4c66-aa72-ab68643faa04	2015-07-22 18:47:10.86969
96	33	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: L858R\ndescription: EGFR L858R has long been recognized as a functionally significant mutation\n  in cancer, and is one of the most prevalent single mutations in lung cancer. Best\n  described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity\n  to first and second generation TKI's like gefitinib and neratinib. NSCLC patients\n  with this mutation treated with TKI's show increased overall and progression-free\n  survival, as compared to chemotherapy alone. Third generation TKI's are currently\n  in clinical trials that specifically focus on mutant forms of EGFR, a few of which\n  have shown efficacy in treating patients that failed to respond to earlier generation\n  TKI therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	2ed5c4f2-4298-4e88-867e-f6775a596f5b	2015-07-22 18:47:10.891304
97	34	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: T790M\ndescription: EGFR T790M was one of the very first mutations recognized to confer resistance\n  to targeted therapies in non-small cell lung cancer. While successful in amplified\n  EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib,\n  neratinib) in treating patients harboring this mutation before treatment is notably\n  lower. This lack of efficacy can likely be to blame for the poorer prognosis for\n  patients with this mutation as compared to patients with wildtype EGFR or other\n  types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired\n  resistance to TKI inhibition have been shown to harbor this mutation, implicating\n  it as a mechanism of acquired therapy resistence. The third generation TKI's are\n  being developed with this resistance problem in mind, and early stage experiments\n  have shown some efficacy of these drugs in previously resistant tumors with mutant\n  EGFR.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	36e422e8-d682-48c3-bd49-8c05c9cb6af8	2015-07-22 18:47:10.912514
98	35	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: D769H\ndescription: ERBB2 D769H was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	5394a4ac-a6b9-4e97-bf93-c5c4d9f05ef1	2015-07-22 18:47:10.934673
99	36	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: D769Y\ndescription: ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	abb04ab6-3863-4c60-991b-1f67febd18d7	2015-07-22 18:47:10.957659
100	37	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: DEL 755-759\ndescription: ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants\n  to be functionally classified (Bose et al. 2012). This mutation was shown to be\n  an activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	b8de548c-e657-4ad1-80bf-76fc4e2bd2f2	2015-07-22 18:47:10.977762
101	38	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: G309A\ndescription: ERBB2 G309A was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	48ed7b7d-7d69-45f9-bf30-79c856f0d2fb	2015-07-22 18:47:10.998191
155	92	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: G12\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	07e5a224-b82a-481c-a08c-77debcae7f6b	2015-07-22 18:47:12.406749
103	40	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: L755W\ndescription: ERBB2 L755W was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	8915c3cd-f149-42ec-927b-bacbde767c6c	2015-07-22 18:47:11.041715
104	41	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: P780INS\ndescription: ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally\n  classified (Bose et al. 2012). This mutation was shown to be an activating mutation\n  in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c5fe7419-33cb-47c8-8d6b-5645ca61dda3	2015-07-22 18:47:11.065927
105	42	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: R678Q\ndescription: ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	2732df46-63e1-4473-a908-a9b6c516064d	2015-07-22 18:47:11.085926
106	43	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: R896C\ndescription: ERBB2 R896C was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	a106acc8-02d7-4e07-8a14-3e3fd5cc5201	2015-07-22 18:47:11.11261
107	44	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: V777L\ndescription: ERBB2 V777L was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	9a022df6-4ba5-48d7-824b-830400ba1973	2015-07-22 18:47:11.159006
108	45	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: V842I\ndescription: ERBB2 V842I was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	93ddbf75-cf00-44fb-b851-3f51bac8897c	2015-07-22 18:47:11.183452
109	46	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: L536Q\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of\n  these ligand binding domain mutations, and is commonly implicated in this hormone\n  resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant,\n  may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	cf3b00ba-ac17-4a9b-9482-e044f834c1bd	2015-07-22 18:47:11.208541
110	47	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: N538G\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c86389f6-5c93-4ceb-be0d-37e090193917	2015-07-22 18:47:11.233473
111	48	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537C\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d3c68998-7f77-4907-b7e2-42365b5ba3bc	2015-07-22 18:47:11.268251
112	49	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537N\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	59e182d5-3643-495b-a945-4a372208b970	2015-07-22 18:47:11.297067
113	50	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537S\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	bdc437eb-e4fc-4bf5-bfd3-50767dc1a1a5	2015-07-22 18:47:11.341464
114	51	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: P235R\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	45dd4fb5-4c70-4a63-8397-253f40a7efae	2015-07-22 18:47:11.369769
115	52	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: FGFR2-MGEA5\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. Additionally, tumor size reduction was\n  achieved by both ponatinib and pazopanib treatments administered separately in a\n  single patient with intrahepatic cholangiocarcinoma and this fusion. The authors\n  use these cases to highlight the need for enhanced clinical sequencing efforts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	6b9744a0-3780-46ae-a205-09f0230288e8	2015-07-22 18:47:11.400581
116	53	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: FGFR3-BAIAP2L1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	427ed069-873c-4a9f-b4b2-811751b10af9	2015-07-22 18:47:11.430622
117	54	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: FGFR3-TACC3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	14e59a39-cd6b-4711-8e25-58099b36b59f	2015-07-22 18:47:11.459298
118	55	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: ITD MUTATIONS\ndescription: FLT3-ITD (internal tandem duplications) frequently occur in patients\n  with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid\n  leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically\n  normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of\n  FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated\n  sequence resulting in an in-frame duplication event. The length of these duplications\n  can vary widely which may have prognostic consequences, but this has not been conclusively\n  determined. FLT3-ITD mutations overall have generally been associated with poor\n  prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the\n  prognosis associated with this variant.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	6fff1a0c-c6ba-4d32-9118-89ce2c727c02	2015-07-22 18:47:11.483684
119	56	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: TKD MUTATIONS\ndescription: FLT3 tyrosine kinase domain mutations (FLT3-TKD) are much less common\n  than FLT3-ITD (internal tandem duplication) mutations and may not confer the same\n  prognostic impact. Although the majority of mutations are point mutations effecting\n  D835 (most frequently D835Y), a small proportion involve an in-frame deletion of\n  I836. These mutations are within the activation loop of the second tyrosine kinase\n  domain of FLT3 and thought to result in constitutive activation of the receptor.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	13b3b9af-cbaa-4255-b6f2-c370be28b2e6	2015-07-22 18:47:11.508253
120	57	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: EXPRESSION\ndescription: GATA2 misregulation has been observed in a number of hematologic malignancies,\n  as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using\n  the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung\n  cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ae13009b-ed73-4734-9181-d035bddf3581	2015-07-22 18:47:11.536643
130	67	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: INTERNAL DUPLICATION\ndescription: c-KIT internal duplications have been observed in exon 11, within the\n  juxtamembrane domain. In a case study of an anal melanoma patient harboring this\n  event, imatinib confered marked response. Also, cells harboring exon 11 mutations\n  have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better\n  prognosis to patients treated with the drug in the first year.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	fb5f3ce5-b91a-4adf-841b-c97fce2e0cd8	2015-07-22 18:47:11.781157
121	58	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	72e8ecca-e44d-411c-b0f8-3b4da4f6dead	2015-07-22 18:47:11.564399
122	59	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132C\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	4347dedc-3eaa-4457-a0bf-f172d862f0cb	2015-07-22 18:47:11.590605
123	60	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132H\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	21409c1e-9b97-41a6-9a55-90d01b1c6968	2015-07-22 18:47:11.610476
124	61	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132L\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	0ffa59ae-56c7-4bb0-abc8-24bc9552f9b1	2015-07-22 18:47:11.635659
125	62	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: R140Q/L\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival\n  as compared to their wild-type counterparts. In myelodysplastic syndromes, however,\n  no prognostic link was discovered between IDH2 mutation status and overall survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	493bd176-b5b8-4bbe-a5b1-350899d49597	2015-07-22 18:47:11.660508
126	63	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: R172K\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis\n  and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic\n  syndromes, studies did not find a prognostic association between this variant and\n  patient outcomes.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	6a488677-ddf0-48bb-87f4-fff54a765e15	2015-07-22 18:47:11.678772
127	64	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: V617F\ndescription: JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases,\n  occuring in around half of all MPD's. While less associated with cancer, when it\n  is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The\n  V617F mutation is an activating mutation, resulting in increased kinase activity.\n  The mutation seems to be restricted to hematologic malignancies. Treatment of JAK\n  mutant diseases with ruxolitinib has seen some clinical success.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	e52c13a6-3c55-4694-963a-c3cfe12f779d	2015-07-22 18:47:11.703299
128	65	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: D816V\ndescription: KIT D816V is a mutation observed in acute myeloid leukemia (AML). This\n  variant has been linked to poorer prognosis and worse outcome in AML patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	bb54ba6a-adf8-4a49-be96-f5159ce9fa6c	2015-07-22 18:47:11.736743
129	66	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 11 MUTATIONS\ndescription: c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are\n  very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis\n  than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity\n  to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients\n  treated with the drug in the first year.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	70731d8d-70b5-4594-a063-4e2a42cc1002	2015-07-22 18:47:11.761334
152	89	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 2 MUTATIONS\ndescription: NRAS exon 2 mutations have been shown to be correlated with poorer overall\n  survival in melanoma patients and colorectal cancer patients, however no prognostic\n  impact was seen in acute myeloid leukemia patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c93c355e-0312-479f-b3d2-a0f057fe8ef9	2015-07-22 18:47:12.346675
131	68	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 13 MUTATIONS\ndescription: c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations.\n  These mutations lie within the tyrosine kinase 1 domain, and are found primarily\n  in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations\n  show sensitivity to imatinib and sunitinib treatment. However, only imatinib has\n  seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant\n  cases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	30807a1e-6dfe-491b-ad32-489b7e2fa55f	2015-07-22 18:47:11.801878
132	69	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 14 MUTATIONS\ndescription: c-KIT exon 14 mutations lie within the tyrosine kinase domain of the\n  protein. While relatively rare in primary gastrointestinal tumors, they are notably\n  more prevalent in refractory disease, suggesting a role in imatinib resistance.\n  Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	057f6ca7-520b-438d-935b-d1baa06b77df	2015-07-22 18:47:11.822646
133	70	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 17 MUTATIONS\ndescription: c-KIT exon 17 mutations lie within the TK2 domain, containing the activation\n  loop of the protein. In cell lines, mutations within this domain have been shown\n  to be sensitive to imatinib. However, in double KIT mutants in which the exon 17\n  mutation is a secondary mutation, cell lines have shown resistance to both imatinib\n  and sunitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	3a0face0-9894-42fe-beaf-5cacb36990ec	2015-07-22 18:47:11.845646
134	71	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 9 MUTATIONS\ndescription: c-KIT exon 9 mutations lie within the dimerization motif of the protein.\n  Relative to other KIT mutations, exon 9 mutations have been associated with better\n  overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and\n  in vivo.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c8eaa043-5ce0-4490-88a8-8b08022d45c5	2015-07-22 18:47:11.873201
135	72	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: L576P\ndescription: KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain.\n  It is one of the most recurrent KIT mutations in melanoma, and both in vitro and\n  in vivo studies have shown sensitivity to imatinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	f8776b2f-b40a-4086-bed4-2b7aded52910	2015-07-22 18:47:11.894341
136	73	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: V654A\ndescription: KIT V654A is an exon 13 mutation that lies within the tyrosine kinase\n  1 domain of the protein. It has been shown to be an activating mutation by in vitro\n  studies. This mutation is associated with imatinib resistance in melanoma patients.\n  However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have\n  seen success in acheiving tumor response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	43ea016c-4b4e-480a-a2a3-fbc269c28dca	2015-07-22 18:47:11.914254
137	74	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: EXON 1 MUTATIONS\ndescription: A study by Lièvre et al in 2006 showed that colorectal cancer patients\n  with KRAS exon 1 mutations had low cetuximab response rates.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d686bc6c-fb77-4adb-a576-52e266bebbaf	2015-07-22 18:47:11.94273
138	75	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: EXON 2 MUTATIONS\ndescription: In a study by Pao et al in 2005, non-small cell lung cancer patients\n  harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors\n  gefinitib and erlotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ade7f098-6eaa-4705-bf32-f77fbcee2abc	2015-07-22 18:47:11.974211
139	76	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	003e2eb2-9836-47a8-8eaf-ba43a256807f	2015-07-22 18:47:11.999498
140	77	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12/G13\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation\n  may result in a less responsive tumor when treated with first-generation TKI's like\n  gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	72127446-7518-46e0-93f7-8d6119271c68	2015-07-22 18:47:12.02788
141	78	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12C\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	8dc4718a-d180-4a48-b75e-f822a0961ab4	2015-07-22 18:47:12.057983
153	90	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 3 MUTATIONS\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	6d44b0e5-80a9-40ad-a79c-1251618b729f	2015-07-22 18:47:12.369294
142	79	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12D\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	b1d056f0-c624-488c-afd6-3bc825fe1d98	2015-07-22 18:47:12.085513
143	80	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G13\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	bc825ae2-db77-4748-ad50-ee548710f554	2015-07-22 18:47:12.114764
144	81	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G13D\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	0fb1d6b8-e8fc-4d47-9d5c-df905cfca614	2015-07-22 18:47:12.145457
145	82	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: P124S\ndescription: MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder\n  and colon cancer to a lesser degree. The P124S mutation has been shown to contribute\n  to AZD6244 resistance in melanoma cell lines, but considerably less so than its\n  Q56P counterpart.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	17cf9b9b-1b81-48e6-ac8c-7efa4b0cd0bf	2015-07-22 18:47:12.173701
146	83	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: Q56P\ndescription: MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This\n  mutation has been shown to confer considerable resistance to AZD6244 treatment of\n  melanoma cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d0abeffb-14d6-4214-a324-26b9c13a4c14	2015-07-22 18:47:12.200032
147	84	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 32\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	e315094b-6de7-4253-ad82-dbcf51dfa2fe	2015-07-22 18:47:12.222009
148	85	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: PROMOTER METHYLATION\ndescription: MGMT promoter methylation has been observed to impact tumor progression\n  in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating\n  agent carmustine has shown efficacy. In patients lacking methylation, combining\n  carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more\n  experiments are required. Clinical trials have also shown selective sensitivity\n  of promoter methylation-positive patients to temozolomide, making a case for wider\n  methylation screening in GBM patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	78253ec9-2bed-4206-add2-44a8d2dc5291	2015-07-22 18:47:12.263926
149	86	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 12 MUTATIONS\ndescription: NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	7f24b476-9316-4a36-9c3d-616d2473400a	2015-07-22 18:47:12.284647
150	87	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: W288FS\ndescription: NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most\n  common NPM1 mutation identified in acute myeloid leukemia. This mutation results\n  in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response\n  to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation,\n  it is generally grouped with other exon 12 mutations for patient analysis (see NPM1\n  Exon 12 variants for more information).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	bbb0b158-c0bc-46d0-abd2-3c583b0a0eb5	2015-07-22 18:47:12.304487
151	88	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 1 MUTATIONS\ndescription: NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown\n  to be correlated with poorer overall survival relative to wild-type NRAS in melanoma\n  patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	36c7050a-2cf0-4696-a67d-1120b6df9067	2015-07-22 18:47:12.324804
154	91	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 4 MUTATIONS\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	bd739d58-b811-4e03-a496-213ed714297a	2015-07-22 18:47:12.387961
156	93	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: G13D\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	64c3157c-a7c2-4fd4-bfb9-f0ff7dc52b56	2015-07-22 18:47:12.428386
157	94	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations\n  at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide.\n  However, in colorectal cancer patients, mutations at this locus have been shown\n  to confer resistance to cetuximab. The prognostic impact of mutations at this locus\n  is currently under study.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	a42e18a7-7eed-4ac0-930b-fcab22f7f20d	2015-07-22 18:47:12.451689
158	95	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61L\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	16c5593d-f679-4c75-9ec9-027d6fca3d27	2015-07-22 18:47:12.478728
159	96	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61R\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	196db203-85f6-4b13-9f7f-1b4100481f5a	2015-07-22 18:47:12.498894
160	97	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: V536E\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	85f44a4e-81af-4dad-9aa7-12870ed94448	2015-07-22 18:47:12.518723
161	98	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842I\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	4fa2ad46-8a5e-4953-a27e-6cbfa3b0373b	2015-07-22 18:47:12.537873
162	99	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842V\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	432c3e79-d2ad-4082-aecc-4a2105ace792	2015-07-22 18:47:12.561409
163	100	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842Y\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	f1d3c815-992b-44f4-9d1e-31b526fdba17	2015-07-22 18:47:12.582007
164	101	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: DEL I843\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	726bcae9-fe9f-48db-b130-2f2e67db49b4	2015-07-22 18:47:12.604218
165	102	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: DI842-843IM\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	1ae64dde-b631-454d-9e49-3de171547361	2015-07-22 18:47:12.627552
166	103	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: E542K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	6db91d33-fbef-482c-86ae-063db401d27e	2015-07-22 18:47:12.650262
216	153	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 56\nname: AMPLIFICATION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	56694fc7-e626-4557-b6cd-7d5f251e78db	2015-07-22 18:47:13.983932
167	104	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: E545K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	a930d034-4f66-4f15-9814-df0bafb0774e	2015-07-22 18:47:12.686706
168	105	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: EXON 20 MUTATIONS\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	a7a9b04e-620c-42d2-9499-6a4be8659ef4	2015-07-22 18:47:12.710458
169	106	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: EXON 9 MUTATIONS\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	e6fed418-c54e-455b-ae67-992b33d0c346	2015-07-22 18:47:12.754264
170	107	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: H1047R\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d2ff944e-74bf-4dea-af98-1772b2363b5e	2015-07-22 18:47:12.793628
171	108	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 39\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and arsenic trioxide for treating these patients, and early results\n  seem promising.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	bbd0a7d7-bd07-437d-8832-b24acb3c57a3	2015-07-22 18:47:12.822105
172	109	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in a rare form of adolescent\n  liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found\n  Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined,\n  and functional studies found that the fusion retained kinase activity. The presence\n  of this fusion may be used as a diagnostic marker for this rare tumor type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	a3c15d55-8e4d-4e29-851c-e50266e84025	2015-07-22 18:47:12.867147
173	110	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 41\nname: R233*\ndescription: PTEN R233* has been shown to be a loss of function mutation, and PTEN\n  loss has been the subject of considerable research in breast cancer. PTEN loss may\n  sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data\n  to support that PTEN loss is both associated with poorer prognosis, and no change\n  in prognosis.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d9499bb2-6f7e-481f-b147-ba275471f1e7	2015-07-22 18:47:12.890794
174	111	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and Arsenic Trioxide for treating these patients, and early results\n  seem promising.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	0584fa4f-c36b-4e90-9a35-c0511e538c2e	2015-07-22 18:47:12.919192
175	112	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: C634W\ndescription: RET C639W has been implicated as an alternate mechanism of activating\n  RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting\n  agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the C639W mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	330037ea-4929-48c5-b88c-fcffca7ab0e6	2015-07-22 18:47:12.945575
176	113	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: M918T\ndescription: RET M819T is the most common somatically acquired mutation in medullary\n  thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents,\n  promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the M918T mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T\n  leads to more aggressive MTC with a poorer prognosis.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	6a6b01d0-8c9d-4e53-83cd-a9fdaf479a8a	2015-07-22 18:47:12.969547
716	81	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '12'\nstart:\n- \n- '25398281'\nstop:\n- \n- '25398281'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	22acfc57-a994-4362-b972-5a023cfdae55	2015-07-22 18:49:40.860093
177	114	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: K666N\ndescription: SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	0f83d12c-2ddc-42bf-bb1b-acd21caa965f	2015-07-22 18:47:12.990591
178	115	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: K700E\ndescription: SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients. Additionally,\n  these mutations are the most common SF3B1 mutation observed in MDS and highly associated\n  with subtypes of MDS that are defined by ringed sideroblasts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	78921fb5-e16e-4dc2-81b4-2c9a49d69cb2	2015-07-22 18:47:13.013711
179	116	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R175H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R175H variant seems not only to result in loss of tumor-suppression, but also acts\n  as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell\n  lines harboring this mutant is also more responsive to treatment with doxorubicin\n  than its wild-type counterparts. While the prognostic impact of individual TP53\n  mutations is influenced by the cohort being studied, it has been shown that the\n  R175H mutation is correlated with worse overall survival than wild-type TP53, but\n  is not as detrimental as the R248W variant.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	3dac6ba1-0857-44c7-8c9c-ca5c8b1864f3	2015-07-22 18:47:13.038913
180	117	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R248Q\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival. The R248Q mutation has also shown an increased invasive behavior in cell\n  lines. This is specific to the 248Q variant.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	06023e9a-7956-40f2-b220-6d92894d8ebb	2015-07-22 18:47:13.060481
181	118	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R248W\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c4eaeb2d-0074-47e1-bd75-86d1e07d61e2	2015-07-22 18:47:13.085181
182	119	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R249T\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	cbe651db-b6f7-470e-9ba4-af40f7401895	2015-07-22 18:47:13.106555
183	120	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R249W\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	25b54a22-b0ba-42c6-bf13-a750de0c9f73	2015-07-22 18:47:13.128195
184	121	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R273C\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ef49ac03-f620-49ea-be3d-5a93e9aa7f12	2015-07-22 18:47:13.149066
185	122	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R273H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	de31b170-6039-4f85-8236-e814846db2bd	2015-07-22 18:47:13.171327
186	123	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: V173G/A\ndescription: While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R249 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	805cb1ed-b83d-40d0-903d-b1edb7b666a4	2015-07-22 18:47:13.191853
736	118	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577539'\nstop:\n- \n- '7577539'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	1aa6cf2b-fc86-4858-a686-67915e426c02	2015-07-22 18:49:41.2442
187	124	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: FRAMESHIFT TRUNCATION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	9af1b50d-a293-4d0b-9f41-b3605aae1444	2015-07-22 18:47:13.218344
188	125	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c06c4add-8483-4cd1-86f3-4c951d452ecc	2015-07-22 18:47:13.239289
189	126	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d26591fe-2a36-49f5-b858-dec2f9baec4f	2015-07-22 18:47:13.259526
190	127	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 48\nname: Q157P/R\ndescription: U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is less common than the S34F mutation, occurs in the second zinc finger domain of\n  U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations\n  on overall survival in MDS has been debated, however, patients with U2AF1 mutations\n  were shown to be at an increased risk of transformation to secondary AML. The presence\n  of this mutation was not associated with a specific prognostic outcome in AML when\n  compared to U2AF1 wildtype patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	45817853-404d-4470-95ac-84b1bf3be251	2015-07-22 18:47:13.285859
191	128	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 48\nname: S34Y/F\ndescription: U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is the most commonly identified variant in MDS, occurs in the first zinc finger\n  domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1\n  mutations on overall survival in MDS has been debated, however, patients with U2AF1\n  mutations were shown to be at an increased risk of transformation to secondary AML.\n  The presence of this mutation was not associated with a specific prognostic outcome\n  in AML when compared to U2AF1 wildtype patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	6c205ded-6012-465b-93ee-fb89ed1ec7ac	2015-07-22 18:47:13.307912
192	129	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 49\nname: EXON 7 MUTATIONS\ndescription: WT1 exon 7 mutations have been shown to be recurrent in acute myeloid\n  leukemia. Many sources have examined the prognostic impact of these, agreeing that\n  the mutant exon 7 cohort is correlated with worse overall survival and a number\n  of poor prognistic outcomes. This may be the result of an overall poor response\n  to chemotherapy from WT1 mutant tumors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	a95a7e3f-52d1-446d-95eb-6a1076fe3459	2015-07-22 18:47:13.329684
193	130	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 49\nname: EXON 9 MUTATIONS\ndescription: WT1 exon 9 mutations have been shown to be recurrent in acute myeloid\n  leukemia, although at a less frequent rate than their exon 7 counterparts. Many\n  sources have examined the prognostic impact of these, agreeing that the mutant exon\n  9 cohort is correlated with worse overall survival and a number of poor prognistic\n  outcomes. This may be the result of an overall poor response to chemotherapy from\n  WT1 mutant tumors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	a47d2077-225a-4e02-9a8a-a72d45ac83a8	2015-07-22 18:47:13.352787
194	131	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: LOSS-OF-FUNCTION\ndescription: BRCA1 loss of function mutations have been shown to increase risk of\n  breast and ovarian cancer in those carrying the allele in their germline. Treating\n  BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant\n  response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	4e6d9ab3-72a7-48d6-aa07-cb315a7616b2	2015-07-22 18:47:13.389707
195	132	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: LOSS-OF-FUNCTION\ndescription: BRCA2 loss of function mutations have been shown to increase risk of\n  breast and ovarian cancer in those carrying the allele in their germline. Treating\n  BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant\n  response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	a6e551ad-e9aa-43bf-b2dd-b86103fd2008	2015-07-22 18:47:13.429571
196	133	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: EXON 19 DELETION\ndescription: Deletions within exon 19 of EGFR are most common in lung cancer. These\n  deletions, in non-small cell lung cancer, have been shown to be sensitive to the\n  EGFR tyrosine kinase inhibitors gefitinib and erlotinib. There is also data to suggest\n  that this event is a good prognostic marker in lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	6ba10dcf-91fe-45ad-b666-c49194a3f5e4	2015-07-22 18:47:13.461099
197	134	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: G719S\ndescription: While not as recurrent as the L858R mutation, EGFR G719S has also been\n  shown to be an activating mutation. In lung cancer cell lines, it also confers sensitivity\n  to the tyrosine kinase inhibitors gefitinib and erlotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d6c2d3bc-4b1c-46cb-b690-60b39d683d95	2015-07-22 18:47:13.487273
198	135	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: D1643H\ndescription: Activating mutations in NOTCH1, including D1643H, have been shown to\n  be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d611a94d-1f17-4f06-ac83-27f7f43ed736	2015-07-22 18:47:13.520634
199	136	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: R2328W\ndescription: Activating mutations in NOTCH1, including R2328W, have been shown to\n  be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	2ff51065-ca86-4df1-9026-a794ee3394a2	2015-07-22 18:47:13.542727
200	137	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: V2444FS\ndescription: Activating mutations in NOTCH1, including a frameshift insertion at V2444,\n  have been shown to be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ba33fec2-b39c-4235-93de-2dd430320e3b	2015-07-22 18:47:13.564967
201	138	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: S2275FS\ndescription: Activating mutations in NOTCH1, including a frameshift insertion at S2275,\n  have been shown to be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ddb4c432-8b71-408d-8cff-92649f816d3d	2015-07-22 18:47:13.590796
202	139	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: L63V\ndescription: Activating mutations in DDR2, including L63V, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	5bc02fdd-4886-42c1-8520-d3ff3129c971	2015-07-22 18:47:13.618366
203	140	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: L239R\ndescription: Activating mutations in DDR2, including L239R, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c507bafc-6562-40b9-855e-6d97bd07d068	2015-07-22 18:47:13.643188
204	141	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: G253C\ndescription: Activating mutations in DDR2, including G253C, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	1d1162c7-de8a-4d94-b000-2a7ec0604ffd	2015-07-22 18:47:13.67247
205	142	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: G505S\ndescription: Activating mutations in DDR2, including G505S, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	dcea04c5-2408-4c4f-b180-720f52403244	2015-07-22 18:47:13.694471
206	143	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: I638F\ndescription: DDR2 I638F has been shown to be a loss of function mutation, but also\n  confers sensitivity to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	643452b0-2997-4120-83cc-6fa1dbffbef6	2015-07-22 18:47:13.720975
207	144	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: G774V\ndescription: Activating mutations in DDR2, including G774V, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	22ebcbac-7b46-48bd-9144-f40b17b0f4be	2015-07-22 18:47:13.745901
208	145	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: S768R\ndescription: Activating mutations in DDR2, including S768R, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	7b210f76-91c9-4874-b589-896839b653d2	2015-07-22 18:47:13.77328
209	146	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 52\nname: AMPLIFICATION\ndescription: MET amplification, like EGFR T790M, has been shown to be capable of driving\n  acquired resistance to dacomitinib in patients with lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	4788f81e-4dcf-4a05-87fb-f2d85ab53502	2015-07-22 18:47:13.80431
210	147	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12V\ndescription: KRAS G12V, like EGFR T790M, has been shown to be capable of driving acquired\n  resistance to tyrosine kinase inhibitors in lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	2feb130d-7357-4547-b14e-5e71d0b8265d	2015-07-22 18:47:13.836069
211	148	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12A\ndescription: KRAS G12A, like EGFR T790M, has been shown to be capable of driving acquired\n  resistance to tyrosine kinase inhibitors in lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	da73eca2-9712-404c-88c8-fd4a1345313a	2015-07-22 18:47:13.862483
212	149	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 54\nname: EWS-FLI1\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	bd464242-efe3-4b46-b865-33e87890257b	2015-07-22 18:47:13.893518
213	150	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 53\nname: EWS-FLI1\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ab630e24-f5af-4677-bf7b-001ca898ea99	2015-07-22 18:47:13.915811
214	151	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 59\nname: T(14;18)\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	583f32db-45bc-4030-90ea-fca5bfd2e55d	2015-07-22 18:47:13.936712
215	152	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 59\nname: EXPRESSION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	0808d994-6305-45ea-8655-361481d6b497	2015-07-22 18:47:13.961207
737	119	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577535'\nstop:\n- \n- '7577535'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- G\n	2	\N	\N	ac86e7e0-4b09-4654-9613-317826ea16c0	2015-07-22 18:49:41.262521
217	154	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 57\nname: AMPLIFICATION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	6d75d04e-21d1-48c0-87f5-be489075e395	2015-07-22 18:47:14.007931
218	155	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: MUTATION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	a648e48d-6207-45ea-856e-f1b6de81c885	2015-07-22 18:47:14.043833
219	156	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 60\nname: OVEREXPRESSION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	b7ce40b3-e16a-4d8d-8b17-baf99a2c2df4	2015-07-22 18:47:14.06367
220	157	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 55\nname: MUTATION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d8a57c5e-8316-4c37-958a-db3073e864d5	2015-07-22 18:47:14.087503
221	158	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 61\nname: EXPRESSION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	e337da32-4c81-4b6e-8eed-2c7726f19163	2015-07-22 18:47:14.10918
222	159	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 58\nname: LOSS OF FUNCTION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c5178063-5df0-461f-a808-4a743887c381	2015-07-22 18:47:14.128664
223	160	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 58\nname: MUTATION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	2210ce43-56eb-4d2b-a5f8-113afc5aebb4	2015-07-22 18:47:14.148994
224	161	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 62\nname: MUTATION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c4beae99-a3e4-4e87-a4e6-93e7bcbef599	2015-07-22 18:47:14.170106
225	162	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 57\nname: EXPRESSION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	06bdfd4b-29ea-48da-b930-d8af5b06a94e	2015-07-22 18:47:14.197142
226	163	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 63\nname: MUTATION\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	31d92dbc-dda3-41aa-92aa-db02a440f7b0	2015-07-22 18:47:14.225893
227	164	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 53\nname: TYPE 1 EWS-FLI1\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	1079223d-6f1b-45a3-9a6f-b85102c158c4	2015-07-22 18:47:14.249824
228	1	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL\n  or CLL).\ndisease_id: 1\nsource_id: 51\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4e835b38-10b3-4f78-b689-1e5d81905058	2015-07-22 18:47:14.422353
229	2	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: GIST tumors harboring PDGFRA D842V mutation are more likely to be benign\n  than malignant.\ndisease_id: 2\nsource_id: 52\nvariant_id: 99\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152093-55152093 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1f35b11d-1a25-4497-b084-258cdd80d7fc	2015-07-22 18:47:14.495634
230	1	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Imatinib Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	af6064da-b6da-47db-8650-16070bebf89c	2015-07-22 18:47:14.561627
231	3	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutations occur most often in de novo AML patients with intermediate\n  cytogenic risk.\ndisease_id: 3\nsource_id: 53\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1a380771-db1f-408b-874c-2d80d359c6a0	2015-07-22 18:47:14.637026
232	4	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young AML patients (<60 years old) with DNMT3A mutations (60% of which\n  were R882) were older in age, had higher white blood cell counts and had higher\n  platelet counts than patients wildtype for DNMT3A\ndisease_id: 3\nsource_id: 54\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2425e947-d0f1-42f9-8499-362ed27dbfdd	2015-07-22 18:47:14.68528
233	5	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical\n  CML).\ndisease_id: 4\nsource_id: 51\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d5182f5c-46e1-46b1-aee7-28a120452eba	2015-07-22 18:47:14.720114
234	6	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: JAK2 V617F is associated with myeloid neoplasms (AML, MDS, CMML/aCML)\ndisease_id: 4\nsource_id: 51\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ff9e99d6-f397-4031-af3f-2b4e86e4b9d0	2015-07-22 18:47:14.753241
235	7	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The SNP rs10974944 in the Jak2 locus is associated with increased predisposition\n  for JAK2 V617F mutation and its associated cancer\ndisease_id: 5\nsource_id: 55\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bb321294-7633-45a8-b629-3286db895257	2015-07-22 18:47:14.798192
236	8	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Children with early age leukaemia who had second hand smoke exposure\n  are more likely to harbor KRAS mutation.\ndisease_id: 6\nsource_id: 56\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	705232d7-cabc-4f64-9dee-e1a87479b311	2015-07-22 18:47:14.840029
237	9	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Children with early age leukaemia who had second hand smoke exposure\n  are more likely to harbor KRAS mutation.\ndisease_id: 6\nsource_id: 56\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1d38fba1-3807-401c-af76-4869e6d1da79	2015-07-22 18:47:14.872507
238	10	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma associated with NRAS Q61 mutation was more often associated\n  with those at the extremity than those at the trunk\ndisease_id: 7\nsource_id: 57\nvariant_id: 94\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	575fe536-17f1-4115-8e13-83357149cb4e	2015-07-22 18:47:14.918186
239	11	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Daunorubicin treatment resulted in similar overall survival and disease\n  free survival in de novo AML patients with DNMT3A R882 mutation compared to those\n  who do not harbor this mutation.\ndisease_id: 3\nsource_id: 53\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c883efeb-7e2d-4ef6-bcab-a4f0dd9d1366	2015-07-22 18:47:14.956357
240	12	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition\n  by about 5 fold.\ndisease_id: 7\nsource_id: 58\nvariant_id: 82\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:66729162-66729162 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a031f24b-6015-40da-b9e5-cc93ce2365e2	2015-07-22 18:47:15.012735
241	13	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by\n  100 fold.\ndisease_id: 7\nsource_id: 59\nvariant_id: 83\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:66727451-66727451 (A->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bac6e12b-5bed-4f8f-b6cd-5dc3b274a964	2015-07-22 18:47:15.05858
242	14	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.\ndisease_id: 7\nsource_id: 60\nvariant_id: 94\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	433c5076-e812-4180-a888-a7460137322a	2015-07-22 18:47:15.109936
243	15	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: GIST cancer with D842V mutation is resistant to imatinib.\ndisease_id: 2\nsource_id: 61\nvariant_id: 99\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152093-55152093 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d612347d-5d4a-4859-abd0-9b4476562b36	2015-07-22 18:47:15.170987
244	16	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib,\n  double mutation of V561D and D842V mutants are resistant to imatinib.\ndisease_id: 2\nsource_id: 62\nvariant_id: 99\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152093-55152093 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5f2a9b24-1b0a-42a3-a491-6a06251f8ae8	2015-07-22 18:47:15.231205
245	17	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In one patient with S214C mutation, the use of sorafenib has led to more\n  than 5 years of survival and near remission.\ndisease_id: 8\nsource_id: 63\nvariant_id: 10\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: X:47426121-47426121 (C->G)\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0d92347f-f467-44a1-996b-8af7252f47ae	2015-07-22 18:47:15.296183
246	18	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Idarubicin increases the overall survival and disease free survival in\n  de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor\n  this mutation.\ndisease_id: 3\nsource_id: 53\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9d9e3a9c-ed12-4ece-81b5-a70f26960b4e	2015-07-22 18:47:15.338249
247	19	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads\n  to reduction in the percentage of cells harboring JAK2 V617F.\ndisease_id: 9\nsource_id: 64\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1f64d926-0597-4c76-9a77-681838ff9c5a	2015-07-22 18:47:15.392456
248	20	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells\ndisease_id: 9\nsource_id: 65\nvariant_id: 64\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1c9c9dff-4f1b-4735-b9d1-70ff5299b433	2015-07-22 18:47:15.448801
249	21	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Likely due to increased reliance of mutant NRAS on HSP90 for stabilization,\n  inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic\n  malignant melanoma with an NRAS G13D mutation\ndisease_id: 7\nsource_id: 66\nvariant_id: 93\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 1:115258744-115258744 (C->T)\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8f18c688-cdb0-4d7b-a1af-eaa6edadc85c	2015-07-22 18:47:15.506292
250	22	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide\n  resulted in disease free survival of 14 months\ndisease_id: 7\nsource_id: 67\nvariant_id: 95\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 1:115256528-115256529 (TT->CA)\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cce14f06-3d9a-4a4b-b188-5d9325e4ab03	2015-07-22 18:47:15.561181
251	23	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a melanoma patient with Q61R mutation, treatment with temozolomide\n  resulted in overall survival of 16 months\ndisease_id: 7\nsource_id: 67\nvariant_id: 96\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 1:115256528-115256529 (TT->CC)\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	70f8eb31-2f1e-4ecb-bfc8-bed704f0e9c1	2015-07-22 18:47:15.612979
252	24	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no difference in the complete remission rate of de novo AML\n  patients with DNMT3A mutation compared to those who are wild type for DNMT3A.\ndisease_id: 3\nsource_id: 53\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7ea5ac9f-2e3b-476d-8999-5a51f5bab59f	2015-07-22 18:47:15.657555
253	25	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: De novo AML patients with DNMT3A D882 mutation showed worse survival\n  (event-free and overall) outcome than those without DNMT3A mutation.\ndisease_id: 3\nsource_id: 68\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ee1fb848-a529-4183-a904-fff2e49e1a8c	2015-07-22 18:47:15.719849
254	26	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In acute myloid leukemia patients, D816 mutation is associated with earlier\n  relapse and poorer prognosis than wildtype KIT\ndisease_id: 10\nsource_id: 69\nvariant_id: 65\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55599321-55599321 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a457faeb-5c9e-4869-81e0-1cde80af368a	2015-07-22 18:47:15.762956
255	2	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: KIT Exon 17\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	27a2f0a7-7550-4106-b004-0e013b03219c	2015-07-22 18:47:15.786395
256	27	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\ndisease_id: 11\nsource_id: 70\nvariant_id: 89\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3272210e-087d-453d-8ae8-b2d6dadb9907	2015-07-22 18:47:15.828554
257	28	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\ndisease_id: 11\nsource_id: 70\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	64a75584-41f8-4ea2-af92-ab9178f61e90	2015-07-22 18:47:15.866176
258	29	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\ndisease_id: 11\nsource_id: 70\nvariant_id: 91\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9c1a4c02-1b38-4b20-ac50-f5cbd541daa1	2015-07-22 18:47:15.899701
259	30	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF mutations are associated with melanoma arising in non-chronic sun\n  damaged skin and with superficial spreading melanoma\ndisease_id: 7\nsource_id: 71\nvariant_id: 17\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fbbc7d41-b0e3-441d-b5e6-c2d1fcc1d5e1	2015-07-22 18:47:15.948438
260	31	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutations were associated with older age, higher white blood\n  cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of\n  cytogenetically normal AML patients\ndisease_id: 3\nsource_id: 72\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	052de9c6-4a90-4365-8c41-bb84b7f90167	2015-07-22 18:47:15.994354
261	32	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation\n  were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK\n  without this mutation.\ndisease_id: 12\nsource_id: 73\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d92e9870-b158-49c0-acc8-610c04ffbabe	2015-07-22 18:47:16.042575
262	33	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation\n  were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK\n  without this mutation.\ndisease_id: 8\nsource_id: 73\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	797557db-ef86-41cc-9a8b-3148fb7ec793	2015-07-22 18:47:16.085062
263	3	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Crizotinib Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	a599e48a-e43c-467b-a845-2874f1b043ff	2015-07-22 18:47:16.111698
290	59	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with KIT mutation but not KIT amplification showed\n  response to imatinib treatment in a small cohort of patients.\ndisease_id: 7\nsource_id: 85\nvariant_id: 68\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	458dff0a-9046-4b37-8443-5e7b464108ec	2015-07-22 18:47:17.294452
264	34	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The melanoma cell line WM3211, which harbors the L576P mutation, is not\n  sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the\n  L576P mutation alters the conformation of KIT and reduces the binding affinity of\n  imatinib.\ndisease_id: 7\nsource_id: 74\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	0fa11e1a-cfdc-48ac-8f33-87af2a31ef11	2015-07-22 18:47:16.163848
265	4	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: KIT Exon 11\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	2adf78af-24b0-49ac-9a2e-bc35cdc59954	2015-07-22 18:47:16.196155
266	35	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC, Exon 2 KRAS mutations were associated with resistance to the\n  EGFR kinase inhibitors gefitinib and erlotinib\ndisease_id: 8\nsource_id: 75\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	c09ef8d9-943c-449a-8fd5-6143033680e5	2015-07-22 18:47:16.23311
267	36	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring NRAS\n  mutation (primarily Q61) have a significantly lower response rate to cetuximab than\n  patients wildtype for NRAS.\ndisease_id: 11\nsource_id: 76\nvariant_id: 94\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c9a5056f-0163-4938-a3b4-f1709af69bac	2015-07-22 18:47:16.305015
268	37	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant\n  in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.\ndisease_id: 13\nsource_id: 77\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cbaf2e7e-87e0-4b7e-b671-00632458cdc4	2015-07-22 18:47:16.353095
269	38	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: High levels of crizotinib can overcome drug resistance of Ba/F3 cells\n  expressing the EML4-ALK fusion containing the F1174L mutation.\ndisease_id: 13\nsource_id: 78\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	36f25ad5-fd52-4252-8249-fa34c03a2704	2015-07-22 18:47:16.399534
270	39	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however,\n  cells with the R1275Q mutation were inhibited at a higher IC50 than those expressing\n  F1174L.\ndisease_id: 13\nsource_id: 79\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29432664-29432664 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	136bc800-37e5-4bc9-8125-018861bfd17f	2015-07-22 18:47:16.440543
271	40	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar\n  colony formation of AALE cells expressing ARAF S214C in vitro\ndisease_id: 8\nsource_id: 63\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: X:47426121-47426121 (C->G)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7c71b92b-9a5d-4928-84a6-62eca2b3bd57	2015-07-22 18:47:16.472804
272	41	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar\n  colony formation of AALE cells expressing ARAF S214C in vitro\ndisease_id: 8\nsource_id: 63\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: X:47426121-47426121 (C->G)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	06bdd395-c99e-463a-96b4-a75810ef3d60	2015-07-22 18:47:16.514671
273	42	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive\n  to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation\n  of KIT but did not alter the binding affinity of dasatinib.\ndisease_id: 7\nsource_id: 74\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b978d6a7-31b8-460c-b683-74ed19b25eb7	2015-07-22 18:47:16.554059
274	43	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\ndisease_id: 2\nsource_id: 80\nvariant_id: 98\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152092-55152093 (GA->AT)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	dd15808a-0237-4d9a-a054-8b3f09b7c8a7	2015-07-22 18:47:16.619595
275	44	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\ndisease_id: 2\nsource_id: 80\nvariant_id: 99\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152093-55152093 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	15578d74-13bb-4542-a1e1-c503dc89cfaf	2015-07-22 18:47:16.672094
276	45	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\ndisease_id: 2\nsource_id: 80\nvariant_id: 100\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152092-55152092 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ff58ec78-115e-4a5d-90a2-40905bd5eecd	2015-07-22 18:47:16.719985
717	82	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '15'\nstart:\n- \n- '66729162'\nstop:\n- \n- '66729162'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	f712ddaf-bdcd-4db2-bec4-55bd1ddf6816	2015-07-22 18:49:40.875766
718	83	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '15'\nstart:\n- \n- '66727451'\nstop:\n- \n- '66727451'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- C\n	2	\N	\N	8eaad86b-046c-4586-91cf-615261403ca8	2015-07-22 18:49:40.89457
277	46	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib\n  resistance, crenolanib was significantly more potent at inhibiting kinase activity\n  than imatinib.\ndisease_id: 2\nsource_id: 80\nvariant_id: 101\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	504db835-b3cf-4cb1-8c13-f958166abd07	2015-07-22 18:47:16.768344
278	47	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib\n  resistance, crenolanib was significantly more potent at inhibiting kinase activity\n  than imatinib.\ndisease_id: 2\nsource_id: 80\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	df146931-180f-4ddd-a718-323d3c79e98d	2015-07-22 18:47:16.815495
279	48	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The SMS-KCNR cell line harboring the R1275Q mutation was resistant to\n  TAE684.\ndisease_id: 13\nsource_id: 79\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29432664-29432664 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9fdf79ee-b62c-4e33-8df7-923ddeb7686d	2015-07-22 18:47:16.852535
280	49	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a large cohort of AML patients (mean = 48 years), DNMT3A mutation\n  (64.5% of which were R882) had no prognostic value on overall, relapse free and\n  event free survival.\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b5940070-dfbc-40c0-98f2-a8763113f33e	2015-07-22 18:47:16.906395
281	50	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a large cohort of cytogenetically normal AML patients (18-60 years\n  old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value\n  for overall, relapse free and event free survival\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bca954be-dc7b-4a07-8246-1369102f0297	2015-07-22 18:47:16.937387
282	51	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutation status (60% of\n  which were R882) was not predictive of overall and relapse free survival in patients\n  with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\ndisease_id: 3\nsource_id: 54\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0620c8bf-bd68-4e52-8bab-2c9fdf023ad0	2015-07-22 18:47:16.969694
283	52	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutation status was not\n  predictive of overall and relapse free survival in patients with NPM1 mutations\n  and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\ndisease_id: 3\nsource_id: 54\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0b386963-f38d-427a-9f4d-cab2eb98cbd1	2015-07-22 18:47:17.005427
284	53	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no significant association between wildtype KIT or KIT mutations\n  in exon 9 or 11 in survival among GIST patients.\ndisease_id: 14\nsource_id: 82\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1a8ae2cf-b94d-4d1b-a3fb-0240d627b966	2015-07-22 18:47:17.049762
285	54	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no significant association between wildtype KIT or KIT mutations\n  in exon 9 or 11 in survival among GIST patients.\ndisease_id: 14\nsource_id: 82\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	af45ad15-c5c6-4274-b213-973602443346	2015-07-22 18:47:17.094587
286	55	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutation status was not\n  predictive of overall and relapse free survival in patients with NPM1 mutations\n  and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\ndisease_id: 3\nsource_id: 54\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	04f120b8-394a-4112-8923-a36de05552c1	2015-07-22 18:47:17.127619
287	56	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of NRAS mutation in AML patients does not impact diease\n  prognosis (resistant disease, disease-free survival, complete remission rate, relapse\n  rate, induction death)\ndisease_id: 3\nsource_id: 83\nvariant_id: 89\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	50e6e691-c64b-49ab-bfdb-30758bd829cf	2015-07-22 18:47:17.168284
288	57	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NRAS mutation status does not impact overall, event-free or disease-free\n  survival in patients with AML\ndisease_id: 3\nsource_id: 84\nvariant_id: 92\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5c4d6e10-3a52-41d6-bf5b-33c5f5b49031	2015-07-22 18:47:17.21116
289	58	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with KIT mutation but not KIT amplification showed\n  response to imatinib treatment in a small cohort of patients.\ndisease_id: 7\nsource_id: 85\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0b932a9e-de16-4cdb-9ff7-f58df3642d7a	2015-07-22 18:47:17.249856
719	87	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '5'\nstart:\n- \n- '170837546'\nstop:\n- \n- '170837547'\nreference_bases:\n- \n- '0'\nvariant_bases:\n- \n- TCAG\n	2	\N	\N	94b5edaa-4dcc-41f3-9463-df3da630cf9e	2015-07-22 18:49:40.923239
291	60	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with KIT mutation but not KIT amplification showed\n  response to imatinib treatment in a small cohort of patients.\ndisease_id: 7\nsource_id: 85\nvariant_id: 70\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fdc30066-da34-4d29-b1a6-75e601a979ef	2015-07-22 18:47:17.333755
292	61	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a large cohort of young AML patients (18-60 years old), DNMT3A R882\n  mutations were associated with reduced relapse free survival in the entire cohort\n  as well as the subset of patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	40730066-f0ff-44b1-9ce3-7710827a1054	2015-07-22 18:47:17.412236
293	62	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with DNMT3A mutations (59% of which were R882) showed worse\n  survival (event-free and overall) outcome than those without DNMT3A mutation.\ndisease_id: 3\nsource_id: 68\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	69e57199-d295-4a89-a7db-0f9480894336	2015-07-22 18:47:17.449965
294	63	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutation was associated with reduced relapse free and overall\n  survival in ELN-unfavorable, cytogenetically normal AMLs\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bed5c2e4-f3cc-4116-87c5-7a8a5e4cea27	2015-07-22 18:47:17.479936
295	64	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations\n  (including R882) were associated with worse overall survival compared to those without\n  DNMT3A mutation.\ndisease_id: 3\nsource_id: 68\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4c9b04f6-c862-49aa-a77c-ba8ead4f6800	2015-07-22 18:47:17.518006
296	65	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In cytogenetically normal AML patients, DNMT3A R882 mutations are associated\n  with lower overall and disease free survival as compared to patients with wild type\n  DNMT3A.\ndisease_id: 3\nsource_id: 72\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4432aa36-bac0-4729-9484-5039853735ed	2015-07-22 18:47:17.551869
297	66	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In older cytogenetically normal AML patients (>59 years), DNMT3A R882\n  mutation is prognostic for shorter disease free survival and overall survival compared\n  to patients without the mutation.\ndisease_id: 3\nsource_id: 86\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	17c701c3-ddba-4753-b3c1-97172e1a969b	2015-07-22 18:47:17.603945
298	67	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutations were associated\n  with significantly reduced overall survival and relapse free survival in patients\n  wildtype for NPM1 and FLT3\ndisease_id: 3\nsource_id: 54\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	286d3ac7-9f62-41b9-87fc-50095e19295c	2015-07-22 18:47:17.634982
299	68	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations\n  other than R882 are prognostic for shorter disease free survival and overall survival\n  compared to patients without the mutation.\ndisease_id: 3\nsource_id: 86\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0f499f3d-4d5b-432e-9431-01bc449829d1	2015-07-22 18:47:17.67848
300	69	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations\n  were associated with worse overall survival compared to those without DNMT3A mutation.\ndisease_id: 3\nsource_id: 68\nvariant_id: 55\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6fff8923-3a06-4bb0-bb85-364f57c2acb3	2015-07-22 18:47:17.718478
301	70	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: GIST patients with exon 14 KIT mutations had reduced overall survival\n  compared to patients wiltype for KIT\ndisease_id: 14\nsource_id: 82\nvariant_id: 69\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1fc356cf-4be5-4c96-b4b5-156afcc454d9	2015-07-22 18:47:17.756332
302	71	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with stage IV melanoma, NRAS mutation was associated with\n  reduced median survival compared to patients with wildtype NRAS\ndisease_id: 7\nsource_id: 87\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	07734c88-7e87-4446-83cd-6658d6c2c8ae	2015-07-22 18:47:17.804453
303	72	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with stage IV melanoma, NRAS mutation was associated with\n  reduced median survival compared to patients with wildtype NRAS\ndisease_id: 7\nsource_id: 87\nvariant_id: 89\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	858fb542-93c7-4d52-9782-af6c9885cc62	2015-07-22 18:47:17.842566
720	93	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '1'\nstart:\n- \n- '115258744'\nstop:\n- \n- '115258744'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	85b7a975-8189-47b4-b07e-30411e77cf65	2015-07-22 18:49:40.944541
304	73	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A single patient with BRAF V600E/V600M bi-allelic mutation responded\n  to the V600E drug dabrafenib.\ndisease_id: 7\nsource_id: 88\nvariant_id: 13\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	67a17c38-1fdb-4b27-a5e7-2556d8bcc438	2015-07-22 18:47:17.894463
305	5	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Other V600's\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	364d5a22-105c-4b02-8570-bedf797c01a0	2015-07-22 18:47:17.920052
306	74	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with medullary carcinoma, the presence of RET M918T mutation\n  is associated with increased probability of lymph node metastases.\ndisease_id: 15\nsource_id: 89\nvariant_id: 113\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43617416-43617416 (T->C)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f9c3cc68-70c3-43ee-a1d6-1b88b2ad1f73	2015-07-22 18:47:17.971319
307	75	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Medullary thyroid cancer cells with RET C634W mutation are insensitive\n  to motesanib, compared to wild-type RET.\ndisease_id: 15\nsource_id: 90\nvariant_id: 112\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43609950-43609950 (C->G)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	52accb2a-2dd8-40ea-af41-a9f7ca0630aa	2015-07-22 18:47:18.016774
308	6	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Motesanib Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	9c1a0acc-6d65-466a-95e9-ce7c96a431d4	2015-07-22 18:47:18.046814
309	76	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Medullary thyroid cancer cells with RET M918T mutation are insensitive\n  to motesanib, compared to wild-type RET.\ndisease_id: 15\nsource_id: 90\nvariant_id: 113\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43617416-43617416 (T->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ee08877a-7e55-44f0-85e6-56558562dcd6	2015-07-22 18:47:18.078999
310	77	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led\n  to strong repression of tyroid cancer cell growth.\ndisease_id: 15\nsource_id: 91\nvariant_id: 113\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43617416-43617416 (T->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c900dbcc-b1bb-4c27-80e8-707401cbec64	2015-07-22 18:47:18.128308
311	78	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Compared to those who harbor a wild type RET, patients with RET M918T\n  mutation develop larger and more aggressive medullary thyroid cancer.\ndisease_id: 15\nsource_id: 92\nvariant_id: 113\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43617416-43617416 (T->C)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bec01af9-1d5f-4812-b793-37617e193251	2015-07-22 18:47:18.171281
312	79	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF V600E is shown to be associated with the tall-cell variant of papillary\n  thyroid cancer\ndisease_id: 16\nsource_id: 93\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	85ac4cd7-f78a-4098-9583-c9a3529d04d1	2015-07-22 18:47:18.218556
313	80	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\ndisease_id: 16\nsource_id: 94\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d33eb2c3-1156-457f-b8eb-91b1ae0bdc11	2015-07-22 18:47:18.256845
314	81	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\ndisease_id: 16\nsource_id: 95\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	54efaf8b-daaf-4d55-9f17-8bc720d22b5e	2015-07-22 18:47:18.296229
315	82	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\ndisease_id: 7\nsource_id: 96\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	560ef7b3-788c-4771-ad52-9fb2f9e908a9	2015-07-22 18:47:18.347762
316	83	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation status was not predictive of response to regorafenib\n  treatment in patients that had received standard therapy and progressed within 3\n  months of their last treatment\ndisease_id: 11\nsource_id: 97\nvariant_id: 103\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f9913a41-727f-4ede-a2f2-328b9570c78a	2015-07-22 18:47:18.41063
317	84	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation status was not predictive of response to regorafenib\n  treatment in patients that had received standard therapy and progressed within 3\n  months of their last treatment\ndisease_id: 11\nsource_id: 97\nvariant_id: 104\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2569a419-9d6f-4238-a7ae-93b9738ef726	2015-07-22 18:47:18.461948
318	85	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation status was not predictive of response to regorafenib\n  treatment in patients that had received standard therapy and progressed within 3\n  months of their last treatment\ndisease_id: 11\nsource_id: 97\nvariant_id: 107\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bfdc2e45-c8c0-429a-a64e-e571935a22fb	2015-07-22 18:47:18.494922
319	86	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation\n  of RAS-GTP but does not show resistance to MEK inhibitors.\ndisease_id: 7\nsource_id: 98\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a7eec3f0-0e86-4ae3-ae4a-a3fa35422d91	2015-07-22 18:47:18.540745
320	87	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\ndisease_id: 11\nsource_id: 97\nvariant_id: 75\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	31efe7d9-3ec4-4c49-b150-1e025159efd8	2015-07-22 18:47:18.576277
321	88	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations\n  were associated with poor progression free survival regardless of treatment (panitumumab\n  with best supportive care or best supportive care alone)\ndisease_id: 11\nsource_id: 99\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a93bc048-b9f1-46ac-baf9-72b4d48db6c7	2015-07-22 18:47:18.624815
322	89	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation\n  unless BRAF inhibitor such as Sorafenib is introduced.\ndisease_id: 17\nsource_id: 100\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	dbc4f8b4-68d0-471a-adda-49ae8313f169	2015-07-22 18:47:18.667157
323	90	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation\n  of RAS-GTP and resistance to RAF inhibitors.\ndisease_id: 7\nsource_id: 98\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b175610c-af62-4961-8238-2f827256e0a7	2015-07-22 18:47:18.699896
324	91	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\ndisease_id: 8\nsource_id: 101\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	46f87cbe-d02e-4d0a-8b31-20d9637ea1e8	2015-07-22 18:47:18.744788
325	92	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: COLO201 and COLO206F cells harboring BRAF V600E mutations were cloned\n  to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown\n  to be amplification of the BRAF V600E gene.\ndisease_id: 11\nsource_id: 102\nvariant_id: 14\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0de15192-de91-4dc7-a1b5-4baf0acb3ca9	2015-07-22 18:47:18.786785
326	93	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with BRAF V600E/K mutations had longer progression-free\n  survival, increased tumor regression,  and increased duration of response to combined\n  dabrafenib and trametinib treatment compared to dabrafenib alone\ndisease_id: 7\nsource_id: 103\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	2ca5eb05-81f5-4479-95d0-c27730d7146e	2015-07-22 18:47:18.834584
327	94	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with other BRAF V600 mutations also respond well to the V600E\n  drug dabrafenib.\ndisease_id: 7\nsource_id: 104\nvariant_id: 11\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453135-140453136 (CA->AT)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1be53536-0bcd-4350-83a6-d136822f30ed	2015-07-22 18:47:18.876294
328	95	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\ndisease_id: 7\nsource_id: 105\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	ab6eccc1-6b9a-44ae-8d7f-07f21383177f	2015-07-22 18:47:18.942944
329	96	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely\n  injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited\n  tumor growth and reduced cellular proliferation\ndisease_id: 17\nsource_id: 106\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	02344b2e-ee69-426c-aa9f-e47aa616a065	2015-07-22 18:47:19.009532
330	97	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\ndisease_id: 17\nsource_id: 107\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	73fba558-2b62-43e5-897e-737296052335	2015-07-22 18:47:19.067518
345	111	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid\n  leukemia.\ndisease_id: 3\nsource_id: 114\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	30afa14f-1162-4248-ba6f-54faf14e5446	2015-07-22 18:47:19.853882
331	98	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\ndisease_id: 11\nsource_id: 108\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	e7f9c6bd-6e4d-4a0b-aef1-b54fad32f800	2015-07-22 18:47:19.126744
332	99	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cell lines expressing the BRAF V600E mutation responded better to vemurafenib\n  treatment than cells wildtype for BRAF as measured by reduced cellular proliferation\n  and inhibition of MET and ERK phosphorylation\ndisease_id: 11\nsource_id: 108\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ab684dc0-d58b-4403-924a-1ee074751dcc	2015-07-22 18:47:19.17263
333	100	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with other BRAF V600 mutations also respond well to the V600E\n  drug dabrafenib.\ndisease_id: 7\nsource_id: 104\nvariant_id: 15\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453137-140453137 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1bb9c82f-23bd-410d-92fb-bd275e253ad0	2015-07-22 18:47:19.213338
334	101	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with other BRAF V600 mutations also respond well to the V600E\n  drug dabrafenib.\ndisease_id: 7\nsource_id: 104\nvariant_id: 16\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453137 (AC->CT)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1549fbf9-7e70-4add-baf9-830175666bc7	2015-07-22 18:47:19.265195
335	102	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Unlike other studies that suggest a poorer outcome, BRAF mutation in\n  this study was not correlated with poorer prognosis in papillary thyroid cancer.\ndisease_id: 16\nsource_id: 109\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	08d53d1d-5601-405d-b495-488517ac1a6f	2015-07-22 18:47:19.332343
336	103	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: V600E is associated with adverse pathological features of colorectal\n  cancer. This can be concluded as a marker of poor prognosis.\ndisease_id: 11\nsource_id: 110\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	88f2a262-6a7f-4603-9dd1-1301222c3b61	2015-07-22 18:47:19.386928
337	104	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF V600E is correlated with shorter disease-free and overall Survival\n  in a Spanish cohort of melanoma patients.\ndisease_id: 7\nsource_id: 111\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	170909b8-3ec1-4255-ac5c-8ae3c470821d	2015-07-22 18:47:19.451011
338	105	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF mutation correlated with poor prognosis in papillary thyroid cancer\n  in both older (>65 yo) and younger (<65 yo) cohorts.\ndisease_id: 16\nsource_id: 93\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e24e9740-f541-4b1b-94dd-4a8bbd6e52ee	2015-07-22 18:47:19.515026
339	106	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF V600E is correlated with poor prognosis in papillary thyroid cancer\n  in a study of 187 patients with PTC and other thyroid diseases.\ndisease_id: 16\nsource_id: 112\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6613c8d0-d7db-4668-9973-86b269806b38	2015-07-22 18:47:19.561657
340	107	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: V600E is correlated with disease recurrence in both age cohorts (>65\n  and <65 yo).\ndisease_id: 16\nsource_id: 93\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	16779b47-2c20-4853-99f9-b4b6d0cc0231	2015-07-22 18:47:19.601458
341	108	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were not associated with the M3 acute myeloid leukemia\n  FAB subtype (0/55).\ndisease_id: 3\nsource_id: 113\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c519b522-f64e-44fc-8d67-f3331c2feb3a	2015-07-22 18:47:19.663581
342	109	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: No NPM1 mutations were identified in patients with favorable risk cytogenetics\n  (79/215 patients)\ndisease_id: 3\nsource_id: 68\nvariant_id: 87\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5ba6cc3a-6dd9-45c3-816d-16fed31eaa09	2015-07-22 18:47:19.709065
343	7	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: NPM1 Exon 12\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	41156519-3c51-49d8-9988-ba8c4b071b05	2015-07-22 18:47:19.74264
344	110	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Therapy-related AML was less common in patients with DNMT3A mutations\n  (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of\n  younger (18-60) AML patients\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	383047cd-34a7-487c-a4ad-367742d990f8	2015-07-22 18:47:19.792197
346	112	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (59% of which were R882) were associated with an intermediate\n  risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes\ndisease_id: 3\nsource_id: 68\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a1223521-6a83-4012-9fe1-d453aac7021f	2015-07-22 18:47:19.898133
347	113	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (59% of which were R882) were associated with intermediate\n  risk cytogenetics (including normal karyotype)\ndisease_id: 3\nsource_id: 68\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	985ea02a-59ed-4e6e-90ba-5e1b34998f43	2015-07-22 18:47:19.943036
348	114	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (64.5% R882) were associated with older age, higher\n  white blood cell count and cytogenetically normal AML in a large cohort of younger\n  (18-60) AML patients\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9511390a-1b27-40ab-a63f-f4752448f918	2015-07-22 18:47:19.983913
349	115	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutations were associated with cytogenetically normal AML\n  in a large cohort of younger (18-60) AML patients\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5a4c447c-04fb-44bb-a247-d015f9efed42	2015-07-22 18:47:20.027512
350	116	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML with mutated NPM1 is a provisional entity in WHO classification of\n  acute myeloid leukemia (AML). This mutation should be tested for in clinical trials\n  and is recommended for testing in patients with cytogentically normal AML.\ndisease_id: 3\nsource_id: 115\nvariant_id: 86\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	20fce3b4-18d1-420a-8274-2c9422dacbf2	2015-07-22 18:47:20.081098
351	117	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with de novo AML as well as M4 and M5\n  FAB subtypes\ndisease_id: 3\nsource_id: 116\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4e5fa091-4750-483a-99cf-b56944b2e4ea	2015-07-22 18:47:20.135552
352	118	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute\n  myeloid leukemia.\ndisease_id: 3\nsource_id: 114\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bce86571-bfea-4789-85b9-96b35492f8f1	2015-07-22 18:47:20.17697
353	119	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with M5a and M5b FAB subtypes of acute\n  myeloid leukemia\ndisease_id: 3\nsource_id: 113\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	11736a95-2d11-404f-b3a7-4cdb8032eb6e	2015-07-22 18:47:20.221318
354	120	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with normal karyotype in older (>60) patients\ndisease_id: 3\nsource_id: 117\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c9e1a76a-ecd8-4ed8-92aa-6ee274e83cf1	2015-07-22 18:47:20.275179
355	121	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with intermediate risk cytogenetics (including\n  normal karyotype)\ndisease_id: 3\nsource_id: 68\nvariant_id: 87\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	80ea01d5-de42-49ac-8bcc-148dca468b94	2015-07-22 18:47:20.318852
356	122	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEBPA mutation status had no impact on ATRA treatment response in older\n  (>60) patients with AML\ndisease_id: 3\nsource_id: 118\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b8021eb5-3379-413c-9d64-5a2a228fd7fb	2015-07-22 18:47:20.395971
357	123	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA\n  mutations (Exon 9) had no significant impact on response rate, disease control rate,\n  progression free surivival or overall survival following cetuximab plus chemotherapy\n  than those with wildtype PIK3CA\ndisease_id: 11\nsource_id: 76\nvariant_id: 106\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	433688e7-93de-487d-ad06-4618d263f885	2015-07-22 18:47:20.451088
358	124	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In chemotherapy-refractory patients with colorectal cancer, NRAS mutation\n  status (primarily Q61) was not informative for disease control rate, progression\n  free survival or overall survival following cetuximab plus chemotherapy.\ndisease_id: 11\nsource_id: 76\nvariant_id: 94\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	29bc5c1b-4299-4bc0-a680-365346429123	2015-07-22 18:47:20.496696
721	95	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '1'\nstart:\n- \n- '115256528'\nstop:\n- \n- '115256529'\nreference_bases:\n- \n- TT\nvariant_bases:\n- \n- CA\n	2	\N	\N	e1c40e5b-e756-4f2b-9a15-431026a9f546	2015-07-22 18:49:40.964701
722	96	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '1'\nstart:\n- \n- '115256528'\nstop:\n- \n- '115256529'\nreference_bases:\n- \n- TT\nvariant_bases:\n- \n- CC\n	2	\N	\N	83502481-0d6c-4f1e-9498-00064094c35d	2015-07-22 18:49:40.984312
359	125	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation.\ndisease_id: 13\nsource_id: 119\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ce71c09b-8cef-410b-a509-6386fe025bb3	2015-07-22 18:47:20.584303
360	126	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring BRAF\n  mutations had lower response and disease control rates as well as shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype BRAF\ndisease_id: 11\nsource_id: 76\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7a52677c-6cd7-4e32-b0b0-86814ba3180d	2015-07-22 18:47:20.635864
361	127	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: ATRA treatment did not effect overall survival in patients <60 years\n  old with FLT3-ITD mutation\ndisease_id: 3\nsource_id: 120\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	62d2a235-b3f9-408d-8300-822abcf78d48	2015-07-22 18:47:20.709703
362	128	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: FLT3-ITD mutation without NPM1 was associated with increased relapse\n  risk and reduced overall survival in young adult AML patients (median age 43)\ndisease_id: 3\nsource_id: 116\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	218d6c54-df09-4f81-928d-9dccc69c9faf	2015-07-22 18:47:20.78894
363	129	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Increasing level of FLT3-ITD mutant correlated with reduced disease-free\n  and overall survival as well as increased relapse rate\ndisease_id: 3\nsource_id: 116\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d01e1a8b-3155-4f1c-a45b-360b347cea6e	2015-07-22 18:47:20.867633
364	130	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to daunorubicin than cells with FLT3-ITD alone.\ndisease_id: 3\nsource_id: 121\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7ef479f4-2af1-45cd-bb73-bc3aeade8718	2015-07-22 18:47:20.944934
365	131	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells\n  with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\ndisease_id: 3\nsource_id: 121\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2d7c6abf-3faf-41ea-9291-0ecdc7e3ea81	2015-07-22 18:47:21.00642
366	132	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to daunorubicin than cells with FLT3-ITD alone.\ndisease_id: 3\nsource_id: 121\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cedaeba6-565b-41ca-9e6f-be9c0e66eeda	2015-07-22 18:47:21.06527
367	133	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells\n  with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\ndisease_id: 3\nsource_id: 121\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	096e3650-ff11-42c9-87ed-9e70937505e8	2015-07-22 18:47:21.105594
368	134	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring KRAS\n  mutations (primarily G12/G13) had lower response and disease control rates and shorter\n  progression free and overall survival following cetuximab plus chemotherapy than\n  those with wildtype KRAS\ndisease_id: 11\nsource_id: 76\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e867232c-c45d-40e6-ab5d-fdc7d3d794f6	2015-07-22 18:47:21.152185
369	135	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations were significantly associated with lack of response to\n  cetuximab in patients with advanced colorectal cancer\ndisease_id: 11\nsource_id: 122\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8dbfcd99-dbef-4d74-8332-6a24eaf30bea	2015-07-22 18:47:21.216685
370	136	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment\n  in isogenic SW48 cells and in a mouse xenograft model\ndisease_id: 17\nsource_id: 123\nvariant_id: 81\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398281-25398281 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ff93c8d3-1fc1-45e4-914d-954114e9f420	2015-07-22 18:47:21.287586
371	137	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: ATRA treatment did not effect overall survival in patients <60 years\n  old with NPM1 mutation regardless of FLT3-ITD status\ndisease_id: 3\nsource_id: 120\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bcf6a8e6-176d-4996-97d9-c941787dd847	2015-07-22 18:47:21.340657
372	138	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring Exon\n  20 PIK3CA mutations had lower response and disease control rates as well as shorter\n  progression free and overall survival following cetuximab plus chemotherapy than\n  those with wildtype PIK3CA\ndisease_id: 11\nsource_id: 76\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a4e15c84-f6f6-4683-a9c0-2e2761d9c85d	2015-07-22 18:47:21.379429
373	139	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with an inferior response to induction\n  chemotherapy with a higher rate of resistant disease in young (15-60+, median 45)\n  patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 124\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	dfad2c97-b426-4c15-bc33-47e8b4647347	2015-07-22 18:47:21.438378
374	140	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with an inferior response to induction\n  chemotherapy with a higher rate of resistant disease in young (15-60+, median 45)\n  patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 124\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	081883bc-b211-4415-8fe3-cf017e71234b	2015-07-22 18:47:21.493736
375	141	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CH5424802 treatment resulted in significant tumor regression in xenograft\n  models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M\n  mutation\ndisease_id: 8\nsource_id: 77\nvariant_id: 7\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	47ba9136-edd9-45f6-93fa-219626b7c925	2015-07-22 18:47:21.540432
376	142	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing\n  ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition\ndisease_id: 13\nsource_id: 79\nvariant_id: 8\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2385e8ca-df59-4e2b-91fc-65bf31e95b24	2015-07-22 18:47:21.577675
377	143	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation\ndisease_id: 13\nsource_id: 119\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29432664-29432664 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	35c9abd3-7cc7-47d0-94db-ad3150a6dc9f	2015-07-22 18:47:21.619563
378	144	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Following identification of this fusion in a patient, this fusion was\n  expressed in HEK 293T cells leading to oligomerization in the absence of ligand,\n  induced morphologic changes in the cell and increased cellular proliferation consistent\n  with FGFR activation. Mouse xenografts using the bladder cell line SW780 which harbors\n  an FGFR3-BAIAP2L1 fusion showed reduced tumor burden when treated with the FGFR\n  inhibitor PD173074.\ndisease_id: 18\nsource_id: 125\nvariant_id: 53\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	970910a5-4393-49e2-b134-1edede0d071b	2015-07-22 18:47:21.676417
379	8	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: FGFR fusions\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	1e743bca-ba70-4abf-9143-1d70cd30650b	2015-07-22 18:47:21.724098
380	145	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment\n  in isogenic SW48 cells and in a mouse xenograft model\ndisease_id: 17\nsource_id: 123\nvariant_id: 81\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398281-25398281 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3801e1a3-20a2-43b1-b1bd-dea87f897449	2015-07-22 18:47:21.758687
381	146	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: ATRA treatment improved overall and relapse-free survival in older (>60)\n  patients with mutant NPM1 and without FLT3-ITD\ndisease_id: 3\nsource_id: 117\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	23d54a3f-91ab-493f-9d29-a0ba650395d7	2015-07-22 18:47:21.795892
382	147	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young patients (18-60) with NPM1 mutations had improved overall survival\n  following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis.\n  However, the adjusted p-value for multiple testing is 0.11.\ndisease_id: 3\nsource_id: 126\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a4e7afbc-d3f3-44bc-86ef-4cae85ce2237	2015-07-22 18:47:21.850577
383	148	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: For patients with NPM1 mutation that achieved CR following induction\n  therapy, relapse-free survival was improved following treatment with valproic acid\ndisease_id: 3\nsource_id: 127\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fc449fb7-4788-468b-81a3-ca203855d591	2015-07-22 18:47:21.905239
384	149	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NSC348884 combined with ATRA treatment induces apoptosis in primary AML\n  cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1\n  and FLT3-ITD\ndisease_id: 3\nsource_id: 128\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	ed01ae21-3de5-437e-af00-00da17a0836d	2015-07-22 18:47:21.971081
738	120	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577536'\nstop:\n- \n- '7577536'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- A\n	2	\N	\N	40d561be-e329-4544-91da-8f61897b81fe	2015-07-22 18:49:41.280578
385	150	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CD33 and CD123 expression was significantly increased in patients with\n  NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33\n  and anti-CD123 therapy\ndisease_id: 3\nsource_id: 129\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	072b9bef-99b8-4db0-b0c7-c0a9b33c4991	2015-07-22 18:47:22.04183
386	151	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CD33 expression was significantly increased in patients with NPM1 mutation\n  with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy\ndisease_id: 3\nsource_id: 129\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b6c9378d-60b3-4433-8d16-bb93cb50f888	2015-07-22 18:47:22.084866
387	152	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.\ndisease_id: 3\nsource_id: 128\nvariant_id: 87\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8153852d-a3b4-45a9-8ea3-8fcb2e6be919	2015-07-22 18:47:22.132257
388	153	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM\n  when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization\n  of NPM in AML patients was associated with good response to induction therapy.\ndisease_id: 3\nsource_id: 130\nvariant_id: 87\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	58eeb597-3626-4e62-a1c8-ea0919565f1d	2015-07-22 18:47:22.197943
389	154	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Following rapamycin treatment of mice with KrasG12D-induced lung tumors,\n  mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung\n  exhibit increased overall survival, increased apoptosis, reduced proliferation and\n  reduced tumor burden compared to Tsc1 wildtype mice.\ndisease_id: 8\nsource_id: 131\nvariant_id: 124\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a2f89193-b038-4cc1-b151-fefc68974b16	2015-07-22 18:47:22.268644
390	9	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: TSC Loss\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	cf7f6266-4257-429a-af6e-357f98189e84	2015-07-22 18:47:22.297371
391	155	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity\n  to everolimus than their wild-type counterparts.\ndisease_id: 18\nsource_id: 132\nvariant_id: 125\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cbf7cbf8-1527-420c-8140-001b6bbb2d82	2015-07-22 18:47:22.34972
392	156	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\ndisease_id: 3\nsource_id: 133\nvariant_id: 28\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b37ef444-1adc-4196-9838-65e3409fb5b2	2015-07-22 18:47:22.413393
393	157	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rates did not differ between patients with wildtype\n  or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML\ndisease_id: 3\nsource_id: 86\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	da865a5c-b4ff-4440-84da-3e9f020eae5a	2015-07-22 18:47:22.462168
394	158	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: FLT3-ITD duplication length did not effect prognosis in patients with\n  an NPM1 mutation\ndisease_id: 3\nsource_id: 134\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	56098a37-853c-4d76-b100-9d5266a2d133	2015-07-22 18:47:22.518373
395	159	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Size of the FLT3-ITD mutant duplication had no impact on overall survival\n  or relapse rate\ndisease_id: 3\nsource_id: 116\nvariant_id: 55\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	80b2c705-39a2-401d-83c9-f495c9c5574c	2015-07-22 18:47:22.570266
396	160	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutation status of IDH1 did not change event-free survival of patients\n  with an NPM1 mutation\ndisease_id: 3\nsource_id: 135\nvariant_id: 58\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2a528d58-54be-4dea-b39c-16ed20c4515e	2015-07-22 18:47:22.63177
397	161	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutation status of IDH1 did not change event-free survival of patients\n  with an NPM1 mutation\ndisease_id: 3\nsource_id: 135\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4e3cae14-2211-41a2-90cc-ee9e5480fa8f	2015-07-22 18:47:22.676691
398	162	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: No differences in relapse-free or overall survival were identified between\n  young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in\n  cytogenetically normal AML\ndisease_id: 3\nsource_id: 136\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f7e7e610-efcd-4695-b479-a9b2a037b1f8	2015-07-22 18:47:22.732832
723	98	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55152092'\nstop:\n- \n- '55152093'\nreference_bases:\n- \n- GA\nvariant_bases:\n- \n- AT\n	2	\N	\N	96ba75ec-8e9c-4265-ab1b-c0ee9abfdf52	2015-07-22 18:49:40.999585
399	163	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rates of complete remission and refractory disease are not different\n  in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young\n  (16-60) patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 136\nvariant_id: 129\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	542f9d43-af6f-4578-ac65-935b7f96365c	2015-07-22 18:47:22.76277
400	164	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: No differences in relapse-free or overall survival were identified between\n  young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in\n  cytogenetically normal AML\ndisease_id: 3\nsource_id: 136\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d2e4959e-0c5d-4d67-992a-1a2899d1399f	2015-07-22 18:47:22.799397
401	165	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rates of complete remission and refractory disease are not different\n  in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young\n  (16-60) patients with cytogenetically normal AML.\ndisease_id: 3\nsource_id: 136\nvariant_id: 130\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	68883e9a-7946-4dd9-b7f1-671730bcea70	2015-07-22 18:47:22.833322
402	166	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEBPA mutation was associated with improved overall survival in older\n  (>60) patients with AML\ndisease_id: 3\nsource_id: 118\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8efa3bb7-2516-48b5-bcca-a14402996a48	2015-07-22 18:47:22.869629
403	167	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEBPA mutation was significantly associated with complete remission\ndisease_id: 3\nsource_id: 137\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	337c0198-9387-446f-9ac2-2831289d6598	2015-07-22 18:47:22.927038
404	168	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Overall survival was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA mutations\ndisease_id: 3\nsource_id: 133\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	eb5e3f1f-5f28-4e4d-996b-ae9f530b0aeb	2015-07-22 18:47:22.963942
405	169	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Remission duration was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\ndisease_id: 3\nsource_id: 133\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1a8289d3-dc63-4da8-a709-c446d934729d	2015-07-22 18:47:22.995714
406	170	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations were associated with achievement of complete remission\n  in a large cohort of younger (18-60) AML patients\ndisease_id: 3\nsource_id: 81\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f64fadcf-8745-42b3-b265-8a81b23f3aaf	2015-07-22 18:47:23.03136
407	171	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Event-free survival was significantly improved in patients with both\n  NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone\n  2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis\n  3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal\n  karyotype patients were analyzed\ndisease_id: 3\nsource_id: 138\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3c71561e-ae72-4eff-ad32-31896902d2c4	2015-07-22 18:47:23.070265
408	172	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MGMT promoter methylation was associated with increased overall and disease-free\n  survival as well as tumor regression in patients with gliomas.\ndisease_id: 19\nsource_id: 139\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2a08df1e-85d7-4e75-9cd5-03f25fa9bbd6	2015-07-22 18:47:23.124761
409	173	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation without FLT3-ITD was associated with reduced relapse risk\n  and increased overall survival in young adult acute myeloid leukemia patients (median\n  age 43)\ndisease_id: 3\nsource_id: 116\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	58aca3b5-ea61-4123-8e11-34267f7102c3	2015-07-22 18:47:23.159001
410	174	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutations were associated with improved complete remission\n  rates as well as overall and disease free survival when all ages and karyotype were\n  analyzed; however, when only normal karyotype patients were analyzed, only complete\n  remission rates were improved\ndisease_id: 3\nsource_id: 117\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	04980a71-030c-4fcd-9fdf-a1b97a342f87	2015-07-22 18:47:23.1873
411	175	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rates were higher and both disease-free and overall\n  survival were longer for patients with Exon 12 NPM1 mutations.\ndisease_id: 3\nsource_id: 113\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c5bb12db-52dc-44f2-874c-53d6904fe3b0	2015-07-22 18:47:23.222799
412	176	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rates were higher and event-free survival was longer\n  for normal karyotype AML patients with Exon 12 NPM1 mutations.\ndisease_id: 3\nsource_id: 114\nvariant_id: 86\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f15fe14a-c0c4-4f13-8da8-8e33f1bf39ac	2015-07-22 18:47:23.265398
413	177	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cytoplasmic localization of NPM in AML patients was an independent prognositic\n  factor associated with complete remission in response to induction therapy in a\n  multivariate analysis (p=0.019) but not in a univariate analysis (p=0.07).\ndisease_id: 3\nsource_id: 130\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b46de7e6-35c8-4518-9ba1-dcd3603687ec	2015-07-22 18:47:23.308656
414	178	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Event-free survival was significantly improved in patients with both\n  NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone\n  2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis\n  3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal\n  karyotype patients were analyzed\ndisease_id: 3\nsource_id: 138\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	92073eb1-cbad-45f5-a6f7-41575a4f789b	2015-07-22 18:47:23.356132
415	179	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Intermediate risk patients with mutant NPM1 had improved overall survival\n  with the presence of either IDH1 or IDH2 mutations than those wildtype for both\n  IDH1 and IDH2\ndisease_id: 3\nsource_id: 126\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a4ac99ed-a42d-4e51-bfb4-969196fe4a93	2015-07-22 18:47:23.397725
416	180	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Normal karyotype patients with NPM1 mutation and without FLT3-ITD had\n  improved overall and relapse-free survival, complete remission rates and remission\n  duration compared to all other combinations of mutants\ndisease_id: 3\nsource_id: 140\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e08d294f-2f05-4309-9b76-8cf97dba9e99	2015-07-22 18:47:23.451093
417	181	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation was associated with higher complete remission rates, lower\n  cumulative incidence of relapse and higher overall survival in intermediate risk\n  AML patients\ndisease_id: 3\nsource_id: 141\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f014d79e-b125-474a-9486-945a537d691e	2015-07-22 18:47:23.503576
418	182	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation was associated with increased overall, event-free and relapse-free\n  survival and remission after induction therapy in the entire cohort and in the cytogenetically\n  normal subset of patients\ndisease_id: 3\nsource_id: 142\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e3713a0b-b252-49ee-a2e9-410dd13a55a9	2015-07-22 18:47:23.556951
419	183	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation without FLT3-ITD was significantly associated with complete\n  remission\ndisease_id: 3\nsource_id: 137\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ec2e0739-1f38-4abc-af84-94be1ed5e14d	2015-07-22 18:47:23.587622
420	184	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with increased complete remission rates\n  as well as longer overall disease free survival in normal karyotype AML patients\n  >59 years old, particularly in those >69 years old\ndisease_id: 3\nsource_id: 143\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cf0ab052-ec80-4a39-b1de-bb88f2398cdf	2015-07-22 18:47:23.670668
421	185	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation\n  and without FLT3-ITD have increased overall survival, relapse-free survival and\n  response to induction chemotherapy\ndisease_id: 3\nsource_id: 144\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e7556554-1427-4743-8007-a6fcd9f02624	2015-07-22 18:47:23.732744
422	186	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rate was not different between young AML patients\n  (<60 years old) with or without DNMT3A mutations (60% of which were R882)\ndisease_id: 3\nsource_id: 54\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ee152b41-8fb4-4522-86b6-9e2d28a79e73	2015-07-22 18:47:23.78612
423	187	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (62% of which were R882) were associated with reduced\n  disease-free survival in patients with cytogenetically normal AML.\ndisease_id: 3\nsource_id: 86\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d2a50d8c-83b3-4048-a741-1477ac617300	2015-07-22 18:47:23.83087
424	188	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young AML patients (<60 years old) with DNMT3A mutation have shorter\n  overall survival and relapse-free survival than patients with wildtype DNMT3A\ndisease_id: 3\nsource_id: 54\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b8de36fe-4590-4d7b-97e8-527afe176767	2015-07-22 18:47:23.895419
425	189	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter\n  overall survival and relapse-free survival than patients with wildtype DNMT3A\ndisease_id: 3\nsource_id: 54\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	76fb75d2-b5a1-4a4f-80c3-5c039efb1fc4	2015-07-22 18:47:23.95528
426	190	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Meta-analysis of studies involving cytogentically normal younger (<60)\n  patients showed reduced overall and relapse-free survival for patients with FLT3-ITD\ndisease_id: 3\nsource_id: 145\nvariant_id: 55\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	966ec166-59cb-4b98-b9fd-c7d338071b6d	2015-07-22 18:47:24.007372
427	191	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced\n  event-free and overall survival in old (>59) and younger patients (<60) with an\n  NPM1 mutation\ndisease_id: 3\nsource_id: 134\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b900ab1c-564b-44d3-96c4-ed284fdbaf94	2015-07-22 18:47:24.055735
428	192	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival\n  compared to patients with wildtype FLT3\ndisease_id: 3\nsource_id: 134\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bdf78ada-e03d-4bd9-a992-8fb2b242708e	2015-07-22 18:47:24.098126
429	193	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable,\n  a higher ratio was associated with reduced event-free survival in patients with\n  an NPM1 mutation\ndisease_id: 3\nsource_id: 134\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1cb16100-f168-49ad-85ab-5788c6300538	2015-07-22 18:47:24.142754
430	194	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations were significantly associated with reduced progression-free\n  and overall survival in patients with advanced colorectal cancer\ndisease_id: 11\nsource_id: 122\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ef9a3ba5-edcc-41f8-a228-20d01d86c860	2015-07-22 18:47:24.187399
431	195	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Coocurrence of FLT3-ITD mutations significantly reduced overall survival\n  of normal karyotype AML patients\ndisease_id: 3\nsource_id: 113\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4a1faa2b-2dd7-46e8-946a-0db7f104368a	2015-07-22 18:47:24.223179
432	196	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse\n  in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction\n  in NPM1 levels following first line therapy (N=15)\ndisease_id: 3\nsource_id: 146\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4ef1f891-7afe-4a9d-850a-da86b0b623df	2015-07-22 18:47:24.282303
433	197	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival\n  compared to patients with wildtype FLT3\ndisease_id: 3\nsource_id: 134\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	19b8ea21-484d-4418-bd51-5bf1e6ef33c5	2015-07-22 18:47:24.321074
434	198	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall\n  survival and cumulative incidence of relapse than those with NPM1 mutations alone\n  in intermediate risk AML patients\ndisease_id: 3\nsource_id: 141\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	60382cda-ecb4-4885-b62e-b9fbc9103357	2015-07-22 18:47:24.36377
435	199	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: TET2 mutation reduces overall survival of normal karyotype patients with\n  either an NPM1 mutation or NPM1 mutation without FLT3-ITD\ndisease_id: 3\nsource_id: 147\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	35961ced-3610-4f8d-be3d-4854ddd02db1	2015-07-22 18:47:24.417443
436	200	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable,\n  a higher ratio was associated with reduced event-free survival in patients with\n  an NPM1 mutation\ndisease_id: 3\nsource_id: 134\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c588580b-fe03-4f56-932a-4ea7fa1f1da5	2015-07-22 18:47:24.477473
437	201	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutations  in WT1 were associated with increased risk of recurrence in\n  young patients (15-50) with cytogenetically normal AML\ndisease_id: 3\nsource_id: 148\nvariant_id: 129\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e951b791-927d-4628-bb9a-dc752b753970	2015-07-22 18:47:24.532873
724	99	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55152093'\nstop:\n- \n- '55152093'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- T\n	2	\N	\N	146138a1-de7a-4222-b125-1d56a471898c	2015-07-22 18:49:41.01687
725	100	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55152092'\nstop:\n- \n- '55152092'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	9dc75097-4897-490f-9612-048552909966	2015-07-22 18:49:41.037518
438	202	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutations in WT1 were associated with a worse overall prognosis than\n  patients wildtype for WT1 in young patients (15-50), primarily because of increased\n  risk of disease recurrence\ndisease_id: 3\nsource_id: 148\nvariant_id: 129\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b96869b5-1aa2-4e43-af96-0aca535c1444	2015-07-22 18:47:24.581544
439	203	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had\n  significantly lower complete remission and higher refractory disease rates than\n  those with wildtype WT1 and without FLT3-ITD following induction therapy in young\n  (16-60) patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 136\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ab3230f7-92de-476e-b313-77efee35774b	2015-07-22 18:47:24.619042
440	204	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\ndisease_id: 3\nsource_id: 149\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	60e769aa-6c7a-4516-b6b1-96a0a1942c3d	2015-07-22 18:47:24.669667
441	205	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were a negative prognostic factor for overall survival\n  in young (15-60+, median 45) patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 124\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b7b6c81c-145e-48d3-ab7b-fe74af99e0a2	2015-07-22 18:47:24.716546
442	206	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\ndisease_id: 3\nsource_id: 124\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a3a80702-c1b8-4b9f-9485-3c506888835e	2015-07-22 18:47:24.769346
443	207	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had\n  significantly lower complete remission and higher refractory disease rates than\n  those with wildtype WT1 and without FLT3-ITD following induction therapy in young\n  (16-60) patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 136\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cecc05a5-7a6c-4791-b7de-1ac9ac514252	2015-07-22 18:47:24.82502
444	208	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were a negative prognostic factor for overall survival\n  in young (15-60+, median 45) patients with cytogenetically normal AML\ndisease_id: 3\nsource_id: 124\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	43ea2db8-d98d-4b4f-a85a-703874dfc073	2015-07-22 18:47:24.877788
445	209	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\ndisease_id: 3\nsource_id: 124\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3869d55e-e2e6-4cba-bb0e-d96b7ef24db6	2015-07-22 18:47:24.925615
446	210	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\ndisease_id: 3\nsource_id: 149\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3b57c078-bba9-4a9a-b604-f708497adf7d	2015-07-22 18:47:24.964071
447	211	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The combination of PARP inhibition (Olaparib) with BRCA mutation results\n  in irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation\n  positive women with high grade serous or undifferentiated ovarian cancer.\ndisease_id: 20\nsource_id: 150\nvariant_id: 131\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 1\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4146f294-7e20-4bd1-9e26-2452147babb5	2015-07-22 18:47:25.02287
448	10	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: BRCA Germline Variants\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	e8eea1bf-e780-4f49-b3ef-73b698cd3a14	2015-07-22 18:47:25.059962
449	212	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The combination of PARP inhibition (Olaparib) with BRCA mutation results\n  in irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation\n  positive women with high grade serous or undifferentiated ovarian cancer.\ndisease_id: 20\nsource_id: 150\nvariant_id: 132\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 1\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a5b05fe7-a3e4-4a13-8d80-220823870b56	2015-07-22 18:47:25.103843
450	213	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Data from deep sequencing of a Ph-negative clone of a Ph-positive CML\n  patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion,\n  and may imply the possibility that BCR-ABL is not universally the initiating event\n  in CML.\ndisease_id: 4\nsource_id: 151\nvariant_id: 1\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e063d6cc-4922-462a-9be6-a7a9016c32aa	2015-07-22 18:47:25.161952
739	121	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577121'\nstop:\n- \n- '7577121'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	32c58897-4059-4359-845b-52585cf6d777	2015-07-22 18:49:41.299063
451	214	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WBC counts were not significantly different in AML patients with FLT3\n  D835 mutations compared to wild type\ndisease_id: 3\nsource_id: 152\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7884b755-9433-449a-8e15-e0f98e1c8131	2015-07-22 18:47:25.216425
452	215	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations in lung cancer patients are not associated with smoking\n  history, age or gender\ndisease_id: 8\nsource_id: 153\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	590badab-0ade-4099-86c1-2a1d84a82be1	2015-07-22 18:47:25.272829
453	216	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations in lung cancer patients are not associated with smoking\n  history, age or gender\ndisease_id: 8\nsource_id: 153\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	da06dbc3-b3b4-42ea-8b21-aef4d6743e9d	2015-07-22 18:47:25.315569
454	217	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Age, sex, FAB subtype and karyotypes were not statistically significant\n  between AML patients with U2AF Q157P/R mutations and those who harbor wild type\n  U2AF.\ndisease_id: 3\nsource_id: 154\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3d8f29dd-081b-40e8-9be2-a3c104952f11	2015-07-22 18:47:25.371559
455	218	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Age, sex, FAB subtype and karyotypes were not statistically significant\n  between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.\ndisease_id: 3\nsource_id: 154\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	29c4115d-d963-4fe1-a1a0-f73901a70bbb	2015-07-22 18:47:25.422825
456	219	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CCND2 hypomethylation is seen to be more common in stage III and IV gastric\n  tumors than stage I and II.\ndisease_id: 21\nsource_id: 155\nvariant_id: 22\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e73d5062-be54-45a0-98ad-3410658bf815	2015-07-22 18:47:25.483426
457	220	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of BCR-ABL fusion is considered the characterizing feature\n  of chronic myeloid leukemia, and has been widely thought of as the initiating event\n  in the disease.\ndisease_id: 4\nsource_id: 156\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bf1868d3-c000-45f1-9d7c-e8c1bcf487a8	2015-07-22 18:47:25.552349
458	221	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D1 overexpression is associated with the ER-positive subtype of\n  breast cancer.\ndisease_id: 22\nsource_id: 157\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d6ecb2e9-0691-469a-996a-07bae4003687	2015-07-22 18:47:25.61803
459	222	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WBC counts were higher in AML patients with internal tandem repeat of\n  Flt3 compared to wild type\ndisease_id: 3\nsource_id: 152\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2b317739-e912-43c8-b64a-1a3c4115d72a	2015-07-22 18:47:25.667247
460	223	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In leukemia patients, IDH1 R132 mutation is associated with adults and\n  normal karyotype.\ndisease_id: 3\nsource_id: 158\nvariant_id: 58\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	73c4cc9f-0d7f-4856-8302-cc999b06f346	2015-07-22 18:47:25.741511
461	224	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation is associated with patients of older age, high platelet\n  count during diagnosis, cytogenic normalcy and NPM1 mutation\ndisease_id: 3\nsource_id: 159\nvariant_id: 59\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:209113113-209113113 (G->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a7974c19-5605-4985-b471-0444010e174c	2015-07-22 18:47:25.800555
462	225	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutation is associated with larger, more invasive tumors, greater\n  pathologic histology and older patients compared to tumors with wildtype KIT\ndisease_id: 14\nsource_id: 160\nvariant_id: 66\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	64e7b17e-9882-426d-9b3a-6a0eb8ce34a1	2015-07-22 18:47:25.865907
463	226	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutation is associated with larger, more invasive tumors, greater\n  pathologic histology and older patients compared to tumors with wildtype KIT\ndisease_id: 14\nsource_id: 160\nvariant_id: 70\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	11925b00-730f-4a59-b582-f59669fdd817	2015-07-22 18:47:25.91446
464	227	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12C occur more frequently in women than men\ndisease_id: 23\nsource_id: 161\nvariant_id: 78\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398285-25398285 (C->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d46788af-3e22-4b3d-a5d4-4e6a7054ceb8	2015-07-22 18:47:25.968686
726	103	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '3'\nstart:\n- \n- '178936082'\nstop:\n- \n- '178936082'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	2c337240-0d17-4a2b-a0bb-37b02784203f	2015-07-22 18:49:41.053596
465	228	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12D mutation occurs in never smokers significantly more often than\n  in smokers\ndisease_id: 23\nsource_id: 161\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398284-25398284 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	953d746b-90f3-42e2-82bd-80df390040ae	2015-07-22 18:47:26.000203
466	229	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no statistical difference in progression free survival between\n  lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.\ndisease_id: 8\nsource_id: 162\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55259514-55259515 (CT->AG)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	b5e51ae4-e364-4830-969a-0dfa81e0284f	2015-07-22 18:47:26.050527
467	230	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no statistical difference in progression free survival between\n  lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.\ndisease_id: 8\nsource_id: 162\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55259514-55259515 (CT->AG)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	c7d70f33-aa0c-46e3-a743-0d524b9da50a	2015-07-22 18:47:26.093573
468	231	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity\n  to the AKT inhibitor MK-2206 in breast cancer cell lines.\ndisease_id: 24\nsource_id: 163\nvariant_id: 4\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 14:105246551-105246551 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9ffba439-90d8-400d-b1fc-c352b575da77	2015-07-22 18:47:26.159892
469	232	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BCR-ABL fusions have been found to contribute to imatinib resistance\n  in AML cell lines.\ndisease_id: 3\nsource_id: 164\nvariant_id: 1\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	54bdf7c9-ac8c-4f12-a864-a1ffb46417e8	2015-07-22 18:47:26.21288
470	233	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\ndisease_id: 4\nsource_id: 165\nvariant_id: 3\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a43b8b72-f34b-4480-874d-1ed4575d6a20	2015-07-22 18:47:26.267066
471	234	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In chronic myeloid leukemia patients with the ABL T315I mutation, tumors\n  have shown to be resistant to imatinib treatment.\ndisease_id: 4\nsource_id: 156\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c0454566-9b83-4361-8046-77fa7ea3a7a8	2015-07-22 18:47:26.316553
472	235	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\ndisease_id: 4\nsource_id: 165\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e0be50f5-0fc1-4708-aa5d-72477d9a2c05	2015-07-22 18:47:26.368969
473	236	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with non-small cell lung cancer harboring EML4-ALK fusion,\n  the C1156Y variant has been shown to confer resistance to crizotinib.\ndisease_id: 8\nsource_id: 166\nvariant_id: 6\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	697d8e4c-2be3-4868-8ea8-08f8735650a1	2015-07-22 18:47:26.426153
474	11	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: ALK Fusions\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	4380e344-2656-42eb-b695-347659b1c3e9	2015-07-22 18:47:26.457882
475	237	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with non-small cell lung cancer harboring EML4-ALK fusion,\n  the L1196M variant has been shown to confer resistance to crizotinib.\ndisease_id: 8\nsource_id: 166\nvariant_id: 7\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bd84d244-cada-4f59-bd41-0e816bc5e848	2015-07-22 18:47:26.501308
476	238	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The T790M mutation in EGFR has been shown to confer resistance to the\n  tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are\n  placed on the drug are likely to relapse.\ndisease_id: 25\nsource_id: 167\nvariant_id: 34\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	76ed77cf-20f3-4c39-b196-066171c5c628	2015-07-22 18:47:26.570928
477	12	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: EGFR TKI Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	738caa10-9544-4bc1-bbb2-2ad2d8fc289f	2015-07-22 18:47:26.599235
478	239	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In non-small cell lung cancer, the appearance of T790M mutation leads\n  to resistance to gefitinib.\ndisease_id: 8\nsource_id: 168\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b7d221d3-9d82-4ac4-ab5b-5711bc60ab37	2015-07-22 18:47:26.666704
479	240	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804),\n  MET amplification and EGFR T790M were found to be the primary drivers of resistance.\ndisease_id: 8\nsource_id: 169\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a1fdbb89-58b2-47a7-a954-604b57bad8e5	2015-07-22 18:47:26.733543
480	241	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The L755S mutation was shown to confer resistance to lapatinib in MCF10A\n  cell lines retrovirally transduced with wildtype or mutant ERBB2.\ndisease_id: 22\nsource_id: 170\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37880220-37880220 (T->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c02151cb-7f1a-4e4a-b3a6-7013144514ba	2015-07-22 18:47:26.794042
481	242	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 22\nsource_id: 171\nvariant_id: 46\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4c673e6d-6cde-4563-83b9-ef26c5d4dd5b	2015-07-22 18:47:26.850334
482	13	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: ESR1 Ligand-Binding Domain\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	e7d8b02e-217c-4c71-9b57-be840de4a0e3	2015-07-22 18:47:26.878899
483	243	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the N538G mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 22\nsource_id: 171\nvariant_id: 47\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bd592966-e522-40c7-b674-ff3880dc23e7	2015-07-22 18:47:26.927695
484	244	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 22\nsource_id: 171\nvariant_id: 48\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419923-152419923 (A->G)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4357d5fa-9f4e-4c01-9583-ca55f3570e9a	2015-07-22 18:47:26.980262
485	245	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 22\nsource_id: 171\nvariant_id: 49\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419922-152419922 (T->A)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fb12feae-bf3e-4e27-8a2d-2870d6a9e1da	2015-07-22 18:47:27.034294
486	246	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the L537S mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 22\nsource_id: 171\nvariant_id: 50\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419923-152419923 (A->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a9d829da-152c-4504-b73e-64b44ebfcab7	2015-07-22 18:47:27.083623
487	247	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Emergence of D835 mutation in AML patients with FLT3 tandem repeat is\n  associated with sorafenib resistance\ndisease_id: 3\nsource_id: 172\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e94666e2-2b5f-4c37-946b-543b8d61ec1c	2015-07-22 18:47:27.141027
488	248	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Sorafenib is effective in patients with FLT3 internal tandem repeat mutations,\n  but not in patients with wild type or D835 mutation.\ndisease_id: 3\nsource_id: 173\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	78cba486-acb8-4405-8f3f-b89af043bd19	2015-07-22 18:47:27.204571
489	249	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT V654A results in imatinib resistance in GIST patient-derived cell\n  lines.\ndisease_id: 14\nsource_id: 62\nvariant_id: 73\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55594258-55594258 (T->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0979b560-071a-46bf-85a2-fb1e7b175662	2015-07-22 18:47:27.24168
490	250	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer who harbor KRAS mutation have low response\n  rate to cetuximab\ndisease_id: 11\nsource_id: 174\nvariant_id: 74\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0c974cf3-1029-419c-abdb-ec9ea9a27957	2015-07-22 18:47:27.295941
491	251	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 26\nsource_id: 175\nvariant_id: 148\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	af83e5ae-e6fc-47c7-8483-0cab8fca2356	2015-07-22 18:47:27.359008
492	252	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 26\nsource_id: 175\nvariant_id: 78\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	fd5f4246-bdac-432d-bffd-0764d599286a	2015-07-22 18:47:27.41329
493	253	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 26\nsource_id: 175\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	38ef349a-8609-4fa2-a7ba-bcc855ad198c	2015-07-22 18:47:27.46731
494	254	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 26\nsource_id: 175\nvariant_id: 147\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	307d0c57-9750-4369-9d26-cd3eff03e2ed	2015-07-22 18:47:27.526283
495	255	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804),\n  MET amplification and EGFR T790M were found to be the primary drivers of resistance.\ndisease_id: 8\nsource_id: 169\nvariant_id: 146\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f2a4effb-6942-40c6-a3ab-fa09b73089ac	2015-07-22 18:47:27.579846
496	256	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20\n  were associated with lack of panitumumab or cetuximab response.\ndisease_id: 11\nsource_id: 176\nvariant_id: 103\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	c7dc09c8-5ed3-4ff6-b169-0d8ec1d3c24a	2015-07-22 18:47:27.671358
497	257	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20\n  were associated with lack of panitumumab or cetuximab response.\ndisease_id: 11\nsource_id: 176\nvariant_id: 104\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	bc22bf67-a12c-4246-af3e-2e50cb4ed7d3	2015-07-22 18:47:27.726264
498	258	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R)\n  were associated with lack of panitumumab or cetuximab response.\ndisease_id: 11\nsource_id: 176\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	82a42037-daea-4c14-851e-8f18f0963dae	2015-07-22 18:47:27.774754
499	259	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Treatment of Philadelphia Chromosome positive leukemias with Arsenic\n  Trioxide results in high rates of complete remission in patients.\ndisease_id: 3\nsource_id: 164\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2753e830-9159-435e-9f1f-8cf5a8174122	2015-07-22 18:47:27.815964
500	260	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The clinical use of imatinib in patients with BCR-ABL fusion has resulted\n  in drastic sensitivity to the drug.\ndisease_id: 4\nsource_id: 156\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0f011f31-a888-4f54-9011-e3fb229fc8d7	2015-07-22 18:47:27.853968
501	261	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The use of second-generation BCR-ABL targeted therapies has been effective\n  in patients resistant to imatinib.\ndisease_id: 4\nsource_id: 156\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	58adbe28-fb05-4232-8a50-533a3f745e32	2015-07-22 18:47:27.900616
502	262	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with non-small cell lung cancer harboring the EML4-ALK fusion,\n  treatment with crizotinib has shown to be effective in acheiving stable disease.\ndisease_id: 8\nsource_id: 166\nvariant_id: 5\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	07de92e7-b0b1-46d4-8a09-59ced3c6a665	2015-07-22 18:47:27.943336
503	263	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with\n  Palbociclib (PD-0332991) significantly increased median survival.\ndisease_id: 27\nsource_id: 177\nvariant_id: 23\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	dd6eb7d2-a61c-4338-b43c-d186b89c965f	2015-07-22 18:47:27.999154
504	264	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\ndisease_id: 28\nsource_id: 178\nvariant_id: 25\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	7aa067e4-cd70-4287-8f84-f99271b304d6	2015-07-22 18:47:28.064123
505	265	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\ndisease_id: 28\nsource_id: 178\nvariant_id: 26\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	089e96d7-7255-4fa2-8260-e265c0b3b880	2015-07-22 18:47:28.104305
506	266	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the G253C mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 8\nsource_id: 179\nvariant_id: 141\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7ae6c7b7-4ea1-412b-8631-8f2cddb5d92c	2015-07-22 18:47:28.151518
507	267	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the G505S mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 8\nsource_id: 179\nvariant_id: 142\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	034d1362-a075-4213-bcd1-1384ca44d742	2015-07-22 18:47:28.189443
727	104	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '3'\nstart:\n- \n- '178936091'\nstop:\n- \n- '178936091'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	ea7e7507-dc97-4d8b-980c-7d48cead46d3	2015-07-22 18:49:41.073208
728	107	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '3'\nstart:\n- \n- '178952085'\nstop:\n- \n- '178952085'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- G\n	2	\N	\N	f2adff75-7e8b-4493-bb5a-5bbf4fb434b3	2015-07-22 18:49:41.092752
508	268	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the G774V mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 8\nsource_id: 179\nvariant_id: 144\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fe6059a6-e0f2-477f-ba13-02efb2b8ce41	2015-07-22 18:47:28.229467
509	269	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the I638F mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 8\nsource_id: 179\nvariant_id: 143\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9f35b7fc-e5ad-42fc-9ff8-f673043110bc	2015-07-22 18:47:28.28489
510	270	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the L239R mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 8\nsource_id: 179\nvariant_id: 140\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	acc57d1a-252e-4046-8af1-c0590999cbdf	2015-07-22 18:47:28.351256
511	271	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the L63V mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 8\nsource_id: 179\nvariant_id: 139\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	279a5cde-ac59-4515-b85e-c7df7957d3b9	2015-07-22 18:47:28.413186
512	272	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a patient expressing a DDR2 S768R mutation, treatment with dasatinib\n  plus erlotinib acheived a partial response.\ndisease_id: 8\nsource_id: 179\nvariant_id: 145\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	8a65c120-1c71-438e-bfe2-06e23e58c4ff	2015-07-22 18:47:28.462361
513	273	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Exon 19 deletion has been shown to be correlated with gefitinib response.\ndisease_id: 8\nsource_id: 180\nvariant_id: 133\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8e7cbd3b-e8b0-4228-ad5e-e95dd1ffab68	2015-07-22 18:47:28.523027
514	274	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Gefinitib has been shown to be effective in treating cell lines with\n  G719S missense mutations.\ndisease_id: 8\nsource_id: 181\nvariant_id: 134\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b5be0435-d675-454d-8189-f9e11b42449f	2015-07-22 18:47:28.574199
515	275	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence\n  of L858R mutation is prognostic for better progression free survival.\ndisease_id: 8\nsource_id: 182\nvariant_id: 33\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55259514-55259515 (CT->AG)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	07ccb80a-d5c3-4823-abbd-55ae7f425fa7	2015-07-22 18:47:28.622172
516	276	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Gefinitib has been shown to be effective in treating cell lines with\n  L858R missense mutations.\ndisease_id: 8\nsource_id: 181\nvariant_id: 33\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d5ae6ec0-299d-4b9b-948c-da8b5f9dc488	2015-07-22 18:47:28.680436
517	277	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a NSCLC patient with T790M and L858R mutation, combination treatment\n  with erlotinib and premetrexed has shown to be effective, reducing tumor size and\n  resulting in stable disease.\ndisease_id: 8\nsource_id: 183\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	61569042-ec5f-4d99-ac8b-ba4a6071a0ff	2015-07-22 18:47:28.731746
518	278	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC containing a T790M mutation, staurosporine may be inhibitive\n  of EGFR, especially when an L858R mutation is also present.\ndisease_id: 8\nsource_id: 184\nvariant_id: 34\nrating: 1\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	da5ba74e-4a00-4799-8382-a497eb1575aa	2015-07-22 18:47:28.789052
519	279	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the D769H mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 35\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37880261-37880261 (G->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6acee337-03c7-477e-a009-5f01350de21d	2015-07-22 18:47:28.825982
520	14	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: HER2 Activating\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	93dab98a-9035-42d1-9f18-7328e656070d	2015-07-22 18:47:28.848496
521	280	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the D769Y mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 36\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37880261-37880261 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	62a631b7-a1c2-4f98-8673-c44d18b35161	2015-07-22 18:47:28.88409
522	281	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines retrovirally transduced with wildtype or mutant\n  ERBB2, cells expressing the in-frame deletion of amino acids 755-759 were shown\n  to be sensitive to neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 37\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e76c41bc-b118-46df-a288-253d7970ae49	2015-07-22 18:47:28.928173
523	282	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the G309A mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 38\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37868205-37868205 (G->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c50f0e03-e969-4487-b615-7bbe31f2f119	2015-07-22 18:47:28.963822
524	283	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The L755S mutation showed mild response to neratinib in MCF10A cell lines\n  retrovirally transduced with wildtype or mutant ERBB2.\ndisease_id: 22\nsource_id: 170\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37880220-37880220 (T->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b3449565-889a-4b38-a53a-f7ff59e550e9	2015-07-22 18:47:29.026015
525	284	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive\n  to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.\ndisease_id: 22\nsource_id: 170\nvariant_id: 40\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2ae66077-951c-4301-b5c0-5656fdcdbfef	2015-07-22 18:47:29.070062
526	285	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the in-frame insertion of a single amino acid at\n  position 780 was shown to be sensitive to neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 41\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	26eff9c9-882f-4ed7-ac84-c7d082845254	2015-07-22 18:47:29.104937
527	286	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\ndisease_id: 22\nsource_id: 170\nvariant_id: 42\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	ecc984d8-258d-49f6-b826-34fdfae74f28	2015-07-22 18:47:29.151105
528	287	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the R896C mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 43\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37881616-37881616 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c6f09806-a0bc-41ec-8ccb-60736867d803	2015-07-22 18:47:29.198216
529	288	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the V777L mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 44\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37881000-37881000 (G->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	11b59e0a-2127-4779-b84e-02d65a73714a	2015-07-22 18:47:29.253317
530	289	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the V842I mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 22\nsource_id: 170\nvariant_id: 45\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37881332-37881332 (G->A)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7145d3f3-dd3f-4d48-a910-f7e71c3482b0	2015-07-22 18:47:29.307032
531	290	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The L536Q ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  MCF7 cell lines.\ndisease_id: 22\nsource_id: 185\nvariant_id: 46\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	ddf50bdf-dc75-4494-bee5-f6282806c54e	2015-07-22 18:47:29.37486
532	291	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The N538G ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 22\nsource_id: 185\nvariant_id: 47\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	abc6a299-e239-4e6b-9f0f-8b2945baa52f	2015-07-22 18:47:29.473413
533	292	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The Y537C ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 22\nsource_id: 185\nvariant_id: 48\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419923-152419923 (A->G)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	f60cd0a6-98c2-4d6b-b726-496a0fac5e8e	2015-07-22 18:47:29.532475
534	293	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The Y537N ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 22\nsource_id: 185\nvariant_id: 49\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419922-152419922 (T->A)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	0ae84473-0dad-4d1b-acd2-43f7106ebdef	2015-07-22 18:47:29.595188
535	294	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The Y537S ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 22\nsource_id: 185\nvariant_id: 50\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419923-152419923 (A->C)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	5e1c9f6e-54d1-4fda-9cd3-2e1b6891b7bc	2015-07-22 18:47:29.650338
729	110	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '10'\nstart:\n- \n- '89717672'\nstop:\n- \n- '89717672'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	2b520f34-1d18-47c2-8fda-05ef7430282f	2015-07-22 18:49:41.111752
730	112	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '10'\nstart:\n- \n- '43609950'\nstop:\n- \n- '43609950'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- G\n	2	\N	\N	4ef78e28-d0ea-4871-ae04-bd8f8add8fda	2015-07-22 18:49:41.130507
536	295	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a single patient with this fusion in intrahepatic cholangiocarcinoma.\n  Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic\n  sites and overall stable disease. However, this response did not meet criteria for\n  RECIST partial response.\ndisease_id: 29\nsource_id: 186\nvariant_id: 52\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3b4b7efa-fdc4-41e6-885f-a2484dbac8cd	2015-07-22 18:47:29.720906
537	296	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This\n  fusion is also shown to be recurrent in multiple other cancer types.\ndisease_id: 29\nsource_id: 186\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	95097219-5c82-4f88-a79c-8668d6499715	2015-07-22 18:47:29.787778
538	297	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated\n  AML patients with FLT3 mutations\ndisease_id: 3\nsource_id: 187\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	34bb0237-8448-4b33-927d-57e5675c096a	2015-07-22 18:47:29.862015
539	298	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Sorafenib is effective in patients with FLT3 internal tandem repeats,\n  but not in wild type or D835 mutation.\ndisease_id: 3\nsource_id: 173\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b5e6d21d-3c04-4a86-9ee0-05a572e409e2	2015-07-22 18:47:29.906867
540	299	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML\n  cells with FLT3 internal tandem repeat\ndisease_id: 3\nsource_id: 188\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fc7231b4-0bc1-49cf-812c-1945a287f694	2015-07-22 18:47:29.956162
541	300	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated\n  AML patients with FLT3 mutations\ndisease_id: 3\nsource_id: 187\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	aff7807a-d9ae-4213-a565-5a16655692ac	2015-07-22 18:47:29.993231
542	301	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib\n  and fasudil led to dramatic tumor regression.\ndisease_id: 30\nsource_id: 189\nvariant_id: 57\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	bc00bf77-ef5b-47dc-9249-6ce0da764def	2015-07-22 18:47:30.044425
543	302	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a case study, a patient with anal melanoma harboring a 7 codon duplication\n  in the juxtamambrane region (exon 11) of KIT showed marked response 4 months after\n  imatinib treatment.\ndisease_id: 31\nsource_id: 190\nvariant_id: 67\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	caed395a-91cd-44dd-812e-18b2b665afed	2015-07-22 18:47:30.108668
544	303	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib,\n  nilotinib and imatinib.\ndisease_id: 8\nsource_id: 191\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	d2a63cc2-7d9c-449a-871a-04eec1552c1d	2015-07-22 18:47:30.163385
545	304	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In GIST tumor cells and patients harboring KIT V654A mutation, SU11248\n  is effective for those that are refractory to imatinib.\ndisease_id: 14\nsource_id: 192\nvariant_id: 73\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55594258-55594258 (T->C)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bee8dd0b-7aa3-4d36-8b15-c269cd5a563b	2015-07-22 18:47:30.220342
546	305	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\ndisease_id: 8\nsource_id: 193\nvariant_id: 79\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398284-25398284 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	917ad23b-2e6e-4e6a-b090-9ef69ad337be	2015-07-22 18:47:30.276593
547	306	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G13D mutation is associated with better response to Cetuximab with\n  longer progression-free and overall survival in colorectal patients compared to\n  other KRAS\ndisease_id: 17\nsource_id: 123\nvariant_id: 81\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398281-25398281 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f76a54b2-c0f5-4cd4-9b0e-c392c57422e9	2015-07-22 18:47:30.318262
548	307	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a randomized clinical trial, patients with MGMT promoter methyaltion\n  benefitted from temozolomide. This benefit was also methylation status dependent,\n  as those without methylation did not see increased survival.\ndisease_id: 19\nsource_id: 194\nvariant_id: 85\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4535aee5-0522-45bc-acfa-d9fdabde64c5	2015-07-22 18:47:30.363583
731	113	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '10'\nstart:\n- \n- '43617416'\nstop:\n- \n- '43617416'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	4a551dc3-a5ab-4195-9474-7ec82fef1f18	2015-07-22 18:49:41.148348
549	308	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MGMT promoter methylation has been shown to confer sensitivity to alkylating\n  agents, such as carmustine, in glioblastoma cells.\ndisease_id: 19\nsource_id: 139\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	909bf223-7b60-42b1-a5ba-b5c5325cdcd4	2015-07-22 18:47:30.400851
550	309	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor,\n  such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may\n  be useful in treating patients with initial resistance to alkylating agents.\ndisease_id: 19\nsource_id: 139\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	d319d7f8-e066-4526-a006-700d79ab0a67	2015-07-22 18:47:30.43511
551	310	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer cell lines with E542K/E545K mutations showed increased\n  sensitivity to CH5132799 than cells with wild-type PIK3CA gene.\ndisease_id: 22\nsource_id: 195\nvariant_id: 103\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2290ea8d-335e-43b0-ac1c-51b7b692a099	2015-07-22 18:47:30.482635
552	311	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rapamycin inhibits transformation induced by mutation in PIK3CA\ndisease_id: 22\nsource_id: 196\nvariant_id: 103\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d14b110a-376c-4cc7-b3d8-3ae60c830dc6	2015-07-22 18:47:30.529944
553	312	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer cell lines with E542K/E545K mutations showed increased\n  sensitivity to CH5132799 than cells with wild-type PIK3CA gene.\ndisease_id: 22\nsource_id: 195\nvariant_id: 104\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	87138368-d38d-434d-ac07-19f2812b7e1c	2015-07-22 18:47:30.56199
554	313	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rapamycin inhibits transformation induced by mutation in PIK3CA\ndisease_id: 22\nsource_id: 196\nvariant_id: 104\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	85b8b993-03cc-4051-9778-7fbcfbb8831c	2015-07-22 18:47:30.600988
555	314	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer cell lines with H1047R mutation showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\ndisease_id: 22\nsource_id: 195\nvariant_id: 107\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9fd7ddb9-8f1d-4638-8a6b-db4230c25d09	2015-07-22 18:47:30.636635
556	315	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rapamycin inhibits transformation induced by mutation in PIK3CA\ndisease_id: 22\nsource_id: 196\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4f87c44b-6fe3-429d-a513-423a1044da0b	2015-07-22 18:47:30.670483
557	316	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Fusion protein degradation is accelerated upon ATRA treatment of acute\n  promyelocytic cells at pharmalogical concentrations.\ndisease_id: 32\nsource_id: 197\nvariant_id: 108\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a71c10ab-1b5b-4ff2-8f31-4fcea0eff40b	2015-07-22 18:47:30.719125
558	317	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells with PTEN deficiency have been shown to exhibit slowed growth in\n  reponse to PI3K-mTOR inhibitors.\ndisease_id: 24\nsource_id: 198\nvariant_id: 110\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:89717672-89717672 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	963dff56-bec2-4f30-ba99-492da1fd6d98	2015-07-22 18:47:30.76132
559	15	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: PTEN Loss-of-Function\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b4d315b4-758a-4b7b-a31d-117cd3b13d41	2015-07-22 18:47:30.788579
560	318	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Fusion protein degradation is accelerated upon ATRA treatment of acute\n  promyelocytic cells at pharmalogical concentrations.\ndisease_id: 32\nsource_id: 197\nvariant_id: 111\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e7372646-82af-4c07-94c0-530771040fdb	2015-07-22 18:47:30.821143
561	319	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast tumors in a mouse model with R175H mutation are more responsive\n  to doxorubicin than breast tumors with wild type TP53.\ndisease_id: 22\nsource_id: 199\nvariant_id: 116\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7578406-7578406 (C->T)\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5b5f3a4c-f46f-4f32-910b-00ac2b47d25b	2015-07-22 18:47:30.864904
562	320	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A patient with metastatic bladder cancer that responded well to the mTOR\n  inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del,\n  p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations\n  also had mild responses to everolimus.\ndisease_id: 33\nsource_id: 200\nvariant_id: 124\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6dd667d8-363a-4583-9ea1-74731d864154	2015-07-22 18:47:30.913001
732	114	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '198267359'\nstop:\n- \n- '198267359'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- A\n	2	\N	\N	16536581-a4a8-4542-af8b-ba5cff6e581a	2015-07-22 18:49:41.169961
563	321	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity\n  to everolimus than their wild-type counterparts.\ndisease_id: 18\nsource_id: 200\nvariant_id: 126\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ac5c02b3-ab01-4ee9-b618-0e8c4308bb1f	2015-07-22 18:47:30.957346
564	322	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with D835 mutation of FLT3 did not have worse overall survival\n  than patients with wild type FLT3\ndisease_id: 3\nsource_id: 152\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	15afd69e-e9b7-42c0-a9c1-bcca56d45dbe	2015-07-22 18:47:31.001689
565	323	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The complete remission and overall survival rates in patients with IDH1\n  R132 mutation is not significantly different from those who do not have this mutation\ndisease_id: 3\nsource_id: 201\nvariant_id: 58\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5cf0e997-73fc-4318-a64a-4359027a101b	2015-07-22 18:47:31.077037
566	324	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\ndisease_id: 3\nsource_id: 159\nvariant_id: 59\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:209113113-209113113 (G->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ca9615d8-801b-4e70-80c1-cb3387a2f980	2015-07-22 18:47:31.113123
567	325	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\ndisease_id: 3\nsource_id: 159\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:209113112-209113112 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6ca8b056-e1bf-401b-b1c9-81b2b67b0e68	2015-07-22 18:47:31.157437
568	326	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\ndisease_id: 3\nsource_id: 159\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:209113112-209113112 (C->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	70c5149e-9a43-4e5e-979c-39d3fbe936c4	2015-07-22 18:47:31.203245
569	327	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with IDH2 mutations such as R140Q/L have event free survival\n  and overall survival similar to those with wild-type IDH2\ndisease_id: 3\nsource_id: 202\nvariant_id: 62\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6f3bfdf7-4a55-4c9b-982d-2f820fb54a6b	2015-07-22 18:47:31.253776
570	328	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with IDH2 R140Q/L mutation, the presence of mutation does\n  not impact overall survival or disease free survival.\ndisease_id: 3\nsource_id: 203\nvariant_id: 62\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	460e34e7-68c4-4311-b138-903af1c9b793	2015-07-22 18:47:31.308194
571	329	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients suffering from MDS, the presence of IDH2 mutation such as\n  R140Q/L do not confer prognostic value (overall survival)\ndisease_id: 34\nsource_id: 204\nvariant_id: 62\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a1c3a78f-9b75-4e71-ab59-3f1de01a6c89	2015-07-22 18:47:31.358813
572	330	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: R140Q mutation in IDH2 does not have prognostic value in patients with\n  MDS.\ndisease_id: 34\nsource_id: 205\nvariant_id: 62\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	95c122e8-7c35-4b13-ae41-b327964760e1	2015-07-22 18:47:31.406498
573	331	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with IDH2 mutations such as R172K have event free survival\n  and overall survival similar to those with wild-type IDH2.\ndisease_id: 3\nsource_id: 202\nvariant_id: 63\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:90631838-90631838 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7fa9db61-37a0-49cf-9ec0-3f3ee098b9fd	2015-07-22 18:47:31.446501
574	332	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients suffering from MDS, the presence of IDH2 mutations such as\n  R172K do not confer prognostic value (overall survival)\ndisease_id: 34\nsource_id: 204\nvariant_id: 63\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:90631838-90631838 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	268aee96-08d3-4bf8-b870-b0039de9add4	2015-07-22 18:47:31.486791
575	333	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutations were identified in morphologically benign, incidentally\n  discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients\n  indicating is not a prognostic marker for GIST\ndisease_id: 14\nsource_id: 206\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8a0f1784-7cef-4dc7-a5db-4648d461a21f	2015-07-22 18:47:31.53552
576	334	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutations detected in 5/60 patients showed no prognostic significance\n  in patients with small cell lung cancer\ndisease_id: 23\nsource_id: 207\nvariant_id: 66\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a7f0fa84-2b4e-449e-89ae-ecc37110c594	2015-07-22 18:47:31.599526
577	335	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutations detected in 5/60 patients showed no prognostic significance\n  in patients with small cell lung cancer\ndisease_id: 23\nsource_id: 207\nvariant_id: 71\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cf68ffea-7797-43f9-ac75-d4eae93bd1e2	2015-07-22 18:47:31.642988
578	336	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with stage III colorectal cancer undergoing chemotherapy,\n  KRAS G12 mutation did not impact overall or disease free survival.\ndisease_id: 17\nsource_id: 208\nvariant_id: 76\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a2b597a5-360a-4c13-9892-b95842132089	2015-07-22 18:47:31.689251
579	337	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no strong association between KRAS mutation status and overall\n  or progression free survival in patients with NSCLC\ndisease_id: 8\nsource_id: 209\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	07f1d531-990f-4cac-9e56-f4836447f2db	2015-07-22 18:47:31.757795
580	338	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with AML, those who harbor Q157P/R mutation of U2AF do not\n  show statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\ndisease_id: 3\nsource_id: 154\nvariant_id: 127\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3dc671e0-c2cf-4b2e-b549-4ad2f7063678	2015-07-22 18:47:31.800852
581	339	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with MDS, those who harbor Q157P/R mutation of U2AF do not\n  show statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\ndisease_id: 34\nsource_id: 154\nvariant_id: 127\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7b68e8f4-6eed-4678-9738-c0e7fbd3f587	2015-07-22 18:47:31.846483
582	340	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with AML, complete remission rates are not different between\n  patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\ndisease_id: 3\nsource_id: 154\nvariant_id: 128\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	eb91e064-59a3-4ab7-a9e7-2ea7ab904487	2015-07-22 18:47:31.891719
583	341	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with AML, complete remission rates are not different between\n  patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\ndisease_id: 34\nsource_id: 154\nvariant_id: 128\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7e2cba31-24d7-4c6a-a34d-f45383d6a0e3	2015-07-22 18:47:31.934516
584	342	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Three studies have found no significant survival impact for CCND1 in\n  lung cancer.\ndisease_id: 8\nsource_id: 210\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1b3110ee-211e-424e-be2c-9c93076b82e0	2015-07-22 18:47:32.021884
585	343	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PTEN nonsense mutations, including R233*, have been shown to be inactivating\n  and loss-of-function, but do not have prognostic value in glioblastoma multiforme\n  patients.\ndisease_id: 19\nsource_id: 211\nvariant_id: 110\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:89717672-89717672 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c2c21f6b-aa63-4706-a850-8f1f261b0967	2015-07-22 18:47:32.082895
586	344	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BCR-ABL inhibitors such as imatinib have lead to significantly improved\n  prognosis, response rate, overall survival, and patient outcome in CML patients\n  compared to previous therapeutic regimens.\ndisease_id: 4\nsource_id: 156\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e1691042-892a-4177-b63f-c697535e5200	2015-07-22 18:47:32.150379
587	345	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\ndisease_id: 28\nsource_id: 212\nvariant_id: 25\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bf7cf6f7-d3e3-4115-9c54-7d7fe8ba5478	2015-07-22 18:47:32.213152
588	346	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\ndisease_id: 28\nsource_id: 212\nvariant_id: 26\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	68a8d2ba-1661-42d2-a96b-971d41ba9a53	2015-07-22 18:47:32.266843
589	347	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Median survival of patients with EGFR L858R mutation is better than those\n  with wild type EGFR.\ndisease_id: 8\nsource_id: 213\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55259514-55259515 (CT->AG)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f7e6fee0-7137-4d2c-bf75-22e5748e44b4	2015-07-22 18:47:32.332938
733	115	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '198266834'\nstop:\n- \n- '198266834'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	821d5517-442b-462c-a5ba-507a85f0dbc8	2015-07-22 18:49:41.188467
734	116	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7578406'\nstop:\n- \n- '7578406'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	3e812e57-8f44-4ae7-83f1-a8697652e4b5	2015-07-22 18:49:41.207029
590	348	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic\n  for better survival compared to patients who harbor wild type mutation.\ndisease_id: 35\nsource_id: 214\nvariant_id: 58\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c892c364-7c37-440d-8b2e-11bf6ca21c8b	2015-07-22 18:47:32.398189
591	349	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with glioblastoma, those harboring IDH1 R132 mutation has\n  higher overall survival compared to those who do not have IDH1 mutation\ndisease_id: 36\nsource_id: 215\nvariant_id: 58\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	711b736a-9ac0-4f15-970c-656873a3a102	2015-07-22 18:47:32.457175
592	350	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML, patients with IDH2 R140K mutation have improved overall survival\n  compared to those with wild-type IDH2\ndisease_id: 3\nsource_id: 216\nvariant_id: 62\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a8f3e592-72c0-4127-93dc-3f1ab0cf3ef5	2015-07-22 18:47:32.518187
593	351	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G13D mutation is associated with better response to Cetuximab with\n  longer progression-free and overall survival in colorectal patients compared to\n  other KRAS\ndisease_id: 17\nsource_id: 123\nvariant_id: 81\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398281-25398281 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9bd41336-79ac-4083-a3b1-2d2ce14d8e39	2015-07-22 18:47:32.561821
594	352	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with SF3B1 mutations had a statistically significant longer\n  overall survival as well as event free-survival. Both before and after adjustment\n  for age, karyotype and sex.\ndisease_id: 34\nsource_id: 217\nvariant_id: 114\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:198267359-198267359 (C->A)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5bfed4f4-d697-4f69-b3a0-78fde8f93121	2015-07-22 18:47:32.621215
595	353	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with SF3B1 mutations had a statistically significant longer\n  overall survival as well as event free-survival. Both before and after adjustment\n  for age, karyotype and sex.\ndisease_id: 34\nsource_id: 217\nvariant_id: 115\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:198266834-198266834 (T->C)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bd5c0025-adda-48b3-a134-da3b8f0e7f22	2015-07-22 18:47:32.663012
596	354	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Inceased copy number of CCND1 is associated with poorer overall survival.\ndisease_id: 8\nsource_id: 210\nvariant_id: 18\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3ba182aa-6468-44aa-ac19-9649ec60913c	2015-07-22 18:47:32.70559
597	355	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Three studies have associated CCND1 expression with poorer survival.\ndisease_id: 8\nsource_id: 210\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	65544f72-b285-46ee-96e9-1819dfdb64ae	2015-07-22 18:47:32.743309
598	356	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In ER-positive breast cancer patients, Cyclin D1 overexpression is associated\n  with shorter overall survival and increased metastasis.\ndisease_id: 22\nsource_id: 157\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	725c89ad-776d-40b3-98a8-a8c15e8af77b	2015-07-22 18:47:32.795047
599	357	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D1 overexpression is associated with shorter overall survival\n  and increased metastasis in head and neck squamous cell carcinoma.\ndisease_id: 37\nsource_id: 218\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	90244cde-70f5-44f9-ad14-30e7afe2bc56	2015-07-22 18:47:32.893349
600	358	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D1 overexpression is associated with shorter overall survival\n  and increased metastasis in mantle cell lymphoma.\ndisease_id: 38\nsource_id: 219\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5b501394-c191-4f23-8cfe-db2391af0f53	2015-07-22 18:47:32.956221
601	359	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with poor prognosis in gastric\n  cancers.\ndisease_id: 21\nsource_id: 220\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	973bbfd4-625c-4d6d-afd6-81b98d50453a	2015-07-22 18:47:33.013405
602	360	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with the increased depth of cancer\n  invasion in gastric cancers.\ndisease_id: 21\nsource_id: 220\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	87ad5d26-a226-4ad0-b0bd-85063cb8babb	2015-07-22 18:47:33.056741
603	361	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with the presence pf lymph node\n  metastasis in gastric cancers.\ndisease_id: 21\nsource_id: 220\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	071f7c7c-890b-4886-8a78-7f6559318e31	2015-07-22 18:47:33.098193
604	362	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with vascular invasion by cancer\n  cells in gastric cancers.\ndisease_id: 21\nsource_id: 220\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	852b4aa7-c5c8-4e09-8ece-a9235ee75289	2015-07-22 18:47:33.136004
605	363	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with high expression of total cyclin E and low-molecular weight\n  cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard\n  ratio for these patients in 13.3 times higher than those with normal cyclin E levels.\ndisease_id: 22\nsource_id: 221\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	419ff342-8cb8-4899-8223-c617ce557172	2015-07-22 18:47:33.188348
606	364	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced\n  stage gastric carcinoma.\ndisease_id: 39\nsource_id: 222\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fb09915e-4da9-4096-90e5-f8367e7aa476	2015-07-22 18:47:33.24633
607	365	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\ndisease_id: 39\nsource_id: 222\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d5ee287c-500b-45d4-8d8f-c2b271549413	2015-07-22 18:47:33.29043
608	366	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\ndisease_id: 23\nsource_id: 222\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7ee71bc8-a41e-437f-bd07-ff7d95d81fc6	2015-07-22 18:47:33.336805
609	367	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A)\n  promoter hypermethylation is associated with shorter overall survival.\ndisease_id: 8\nsource_id: 223\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5f7d72f3-3045-4b9d-a7e6-0583c7f669a4	2015-07-22 18:47:33.399499
610	368	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A)\n  promoter hypermethylation is associated with shorter time to recurrence.\ndisease_id: 8\nsource_id: 223\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c229d922-3ddf-4e2d-8828-06fc3e3b636c	2015-07-22 18:47:33.44784
611	369	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients with T790M and another activating mutations, their\n  progression free survival is shorter compared to those who do not possess T790M\n  mutation.\ndisease_id: 8\nsource_id: 224\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4501e8bf-dec7-4b44-a4d5-e5716b6ee7b5	2015-07-22 18:47:33.508483
612	370	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NSCLC harboring EGFR T790M mutation have statistically\n  worse overall survival compared to patients with L858R or other exon 19 activating\n  mutations.\ndisease_id: 8\nsource_id: 225\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	25ec78f3-2e00-4233-ba4d-14eff9703a95	2015-07-22 18:47:33.576456
613	371	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with internal tandem repeat of FLT3 has worse overall survival\n  than patients with wild type FLT3\ndisease_id: 3\nsource_id: 152\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bf0d7b2c-7580-44dc-8c9e-f0cc87835af4	2015-07-22 18:47:33.626295
614	372	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads\n  to poor survival and inferior disease free survival in patients with AML.\ndisease_id: 3\nsource_id: 226\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4f539f80-f859-432d-8d88-579ab5526c77	2015-07-22 18:47:33.679207
615	373	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML patients, FLT3-TKD mutation was associated with poorer disease\n  free survival compared to patients with wild type FLT3\ndisease_id: 3\nsource_id: 227\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	60601e73-49c3-4292-a709-5c3a2b046083	2015-07-22 18:47:33.731798
616	374	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients suffering from myelodysplastic syndromes\n  is associated with worse overall survival\ndisease_id: 40\nsource_id: 228\nvariant_id: 58\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d43838ab-5ef3-4738-b907-95b16954e614	2015-07-22 18:47:33.785034
681	8	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'FGFR fusions have been demonstrated across many cancer types to have oncogenic\n  potential. FGFR2 and FGFR3 have been shown to be fused to many 3'' partners, which\n  may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2\n  fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib\n  and pazopanib have shown efficacy in case studies.  '\n	2	\N	\N	9b62d684-68a5-4af1-957c-a5d137bb47b9	2015-07-22 18:47:37.210812
617	375	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML, patients with an IDH2 R172K mutation have worse overall survival\n  compared to those with wild-type IDH2\ndisease_id: 3\nsource_id: 216\nvariant_id: 63\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:90631838-90631838 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	31e41a93-472b-4030-b881-f2e7bb881bbb	2015-07-22 18:47:33.829082
618	376	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutation is associated with worse overall and cause-specific prognosis\n  in patients with GIST compared to patients with wildtype KIT\ndisease_id: 14\nsource_id: 160\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cc37bec2-0f1e-4016-be25-c3a7f7dc29e8	2015-07-22 18:47:33.871957
619	377	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of KRAS mutations in MM patients is prognostic for shorter\n  overall and progression free survival\ndisease_id: 41\nsource_id: 229\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	50c76eb5-1b59-489a-8c58-54e2b1f2bdc4	2015-07-22 18:47:33.947232
620	378	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A meta-analysis showed KRAS mutation is associated with worse outcome\n  in patients with NSCLC\ndisease_id: 8\nsource_id: 230\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	208b2917-bd27-4358-8413-5b2cee83c276	2015-07-22 18:47:34.008015
621	379	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of KRAS mutations in MM patients is prognostic for shorter\n  overall and progression free survival\ndisease_id: 42\nsource_id: 229\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9d23f131-f5b6-43dc-ad6c-adc5638bd462	2015-07-22 18:47:34.047615
622	380	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including D1643H, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 8\nsource_id: 231\nvariant_id: 135\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6a5b9951-05da-4103-849f-fef9ec2e619d	2015-07-22 18:47:34.095547
623	381	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including R2328W, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 8\nsource_id: 231\nvariant_id: 136\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4b52afba-2d0c-4367-8039-7934b2d92649	2015-07-22 18:47:34.140106
624	382	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including S2275fs, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 8\nsource_id: 231\nvariant_id: 138\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3d72c82d-43ce-4696-84dc-50f5d37be8db	2015-07-22 18:47:34.185192
625	383	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including V2444fs, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 8\nsource_id: 231\nvariant_id: 137\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a979d5e9-2d2a-4bbe-88fd-e56946352cbe	2015-07-22 18:47:34.236015
626	384	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not\n  either alone, is prognostic for worse overall survival and cancer specific survival.\ndisease_id: 11\nsource_id: 232\nvariant_id: 103\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2eb62071-4612-4e1a-afbc-e109d705684d	2015-07-22 18:47:34.29998
627	385	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of exon 9 (such as E542K) or exon 20 mutations were associated\n  with poorer survival compared to those with wild-type PIK3CA mutation.\ndisease_id: 11\nsource_id: 176\nvariant_id: 103\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	114804b5-5da4-4be4-9626-4ce557c95a24	2015-07-22 18:47:34.346279
628	386	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not\n  either alone, is prognostic for worse overall survival and cancer specific survival.\ndisease_id: 11\nsource_id: 232\nvariant_id: 104\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5ad30d2c-2845-41e4-af11-4fd6557d08a1	2015-07-22 18:47:34.389141
629	387	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of exon 9 (such as E545K) or exon 20 mutation may result\n  in poorer survival compared to those with wild-type PIK3CA mutation.\ndisease_id: 11\nsource_id: 176\nvariant_id: 104\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	044a72ee-39b2-416a-a4a1-f204ee642b4a	2015-07-22 18:47:34.442725
710	63	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '15'\nstart:\n- \n- '90631838'\nstop:\n- \n- '90631838'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	c8f737f7-3249-47e0-b08d-b30bf8e1a658	2015-07-22 18:49:40.550789
711	64	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '9'\nstart:\n- \n- '5073770'\nstop:\n- \n- '5073770'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	7371b21b-401f-407c-9265-0785b2ca356b	2015-07-22 18:49:40.572419
630	388	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of exon 20 (such as H1047R) or exon 9 mutation may result\n  in poorer survival compared to those with wild-type PIK3CA mutation.\ndisease_id: 11\nsource_id: 176\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	45dd9d40-9006-49e6-8ce7-dbaab1913fb5	2015-07-22 18:47:34.485856
631	389	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R175H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 22\nsource_id: 233\nvariant_id: 116\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7578406-7578406 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	620932ad-c080-4b63-8028-530229b08f92	2015-07-22 18:47:34.543353
632	390	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R248Q mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 22\nsource_id: 233\nvariant_id: 117\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577538-7577538 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1f21b08f-1db5-4bea-aaac-8fee51f7b645	2015-07-22 18:47:34.615768
633	391	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R248 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\ndisease_id: 22\nsource_id: 234\nvariant_id: 117\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577538-7577538 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fd9a5993-9c70-4168-8363-1284dc2cf3a3	2015-07-22 18:47:34.677359
634	392	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring R248W mutation, the prognosis is\n  worse than any other hotspot TP53 mutation, as well as worse than patients with\n  wild type TP53.\ndisease_id: 22\nsource_id: 233\nvariant_id: 118\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577539-7577539 (G->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d9803f9f-f3cc-43ca-84b7-bff05ef79247	2015-07-22 18:47:34.747569
635	393	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in conserved\n  regions such as R249 are prognostic for a worse overall survival compared to those\n  harboring wild-type TP53.\ndisease_id: 22\nsource_id: 234\nvariant_id: 119\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577535-7577535 (C->G)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	35ff4399-1e3d-45d2-a5ac-ca4938cd1a2d	2015-07-22 18:47:34.822292
636	394	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in conserved\n  regions such as R249 are prognostic for a worse overall survival compared to those\n  harboring wild-type TP53.\ndisease_id: 22\nsource_id: 234\nvariant_id: 120\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577536-7577536 (T->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3fbf9988-e022-45a5-a91a-70ae0e24bc88	2015-07-22 18:47:34.873658
637	395	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R273C mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 22\nsource_id: 233\nvariant_id: 121\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577121-7577121 (G->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0e65c90c-43cb-45bf-bfb5-807bcb90a6b1	2015-07-22 18:47:34.926982
638	396	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients  harboring TP53 mutation, mutations in DNA\n  contact regions such as R273 are prognostic for a worse relapse-free survival compared\n  to other non-silent mutations in TP53.\ndisease_id: 22\nsource_id: 234\nvariant_id: 121\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577121-7577121 (G->A)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	951dfa3c-6f4a-4708-b6f0-4c5c4b0501e3	2015-07-22 18:47:34.973092
639	397	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R273H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 22\nsource_id: 233\nvariant_id: 122\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577120-7577120 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	61c01abf-8969-4e07-802d-06f5ef6a2d2d	2015-07-22 18:47:35.028253
640	398	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\ndisease_id: 22\nsource_id: 234\nvariant_id: 122\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577120-7577120 (C->T)\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d269f0e9-cdfc-4e9a-9879-917c9d3790a1	2015-07-22 18:47:35.079202
641	399	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast tumors with R175H mutations are more responsive to doxorubicin\n  than breast tumors with wild type TP53.\ndisease_id: 22\nsource_id: 234\nvariant_id: 123\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d0c6b9a8-cf1e-4662-91bc-0e7f55e043e2	2015-07-22 18:47:35.131528
712	65	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55599321'\nstop:\n- \n- '55599321'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- T\n	2	\N	\N	eb1dd4f8-a454-4173-a7ee-c79b083df2c9	2015-07-22 18:49:40.59357
713	73	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55594258'\nstop:\n- \n- '55594258'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	d905f6a3-5424-446e-bb60-703b252cbb71	2015-07-22 18:49:40.628265
642	400	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: After adjusting for age and cancer stage, presence of U2AF mutation such\n  as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.\ndisease_id: 34\nsource_id: 235\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	479a741a-cdbe-486b-8511-67b8cb5fb044	2015-07-22 18:47:35.194066
643	401	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: After adjusting for age and cancer stage, the presence of U2AF mutations\n  such as S34Y/F are prognostic for poorer survival outcomes in patients with MDS.\ndisease_id: 34\nsource_id: 235\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	441e1656-35fe-41f7-b3b9-4af1a51aec48	2015-07-22 18:47:35.246807
644	402	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\ndisease_id: 8\nsource_id: 236\nvariant_id: 79\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398284-25398284 (C->T)\ndrug_interaction_description: \nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	dda3bdcb-e70b-4235-a5c2-7eddeaa3c55b	2015-07-22 18:47:35.312137
645	403	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase\n  inhibitors Imatinib and GW-2580\ndisease_id: 43\nsource_id: 237\nvariant_id: 30\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	2c684e5b-53b2-4833-9129-a97ef2d28f95	2015-07-22 18:47:35.379903
646	404	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase\n  inhibitors Imatinib and GW-2580\ndisease_id: 43\nsource_id: 237\nvariant_id: 84\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	55c7e756-885c-4c0b-b232-a6b2c1bd04e9	2015-07-22 18:47:35.434065
647	405	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA\n  resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\ndisease_id: 44\nsource_id: 238\nvariant_id: 31\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2e5dac06-f244-4943-b057-5f5ea0480d06	2015-07-22 18:47:35.488011
648	406	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA\n  resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\ndisease_id: 44\nsource_id: 238\nvariant_id: 109\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bae091a2-0526-4b2d-8b32-52737249a8fc	2015-07-22 18:47:35.535804
649	407	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The t(14;18) transloaction of BCL2 is shown to be associated with germinal-center\n  B-cell–like diffuse large-B-cell lymphomas.\ndisease_id: 45\nsource_id: 239\nvariant_id: 151\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7cde2a2f-ac44-4ff8-b84e-5b663902895f	2015-07-22 18:47:35.612694
650	408	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Amplification of FOXP1 is shown to be associated with ABC DLBCL\ndisease_id: 45\nsource_id: 240\nvariant_id: 153\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	44f859bd-5cc2-4a0b-9a52-f5ea1a5672d4	2015-07-22 18:47:35.682035
651	409	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Amplification of REL is shown to be associated with germinal-center B-cell–like\n  diffuse large-B-cell lymphomas.\ndisease_id: 45\nsource_id: 239\nvariant_id: 154\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d6ab93a4-7588-48c2-83a2-1449d13e7990	2015-07-22 18:47:35.729123
652	410	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Amplification of REL is shown to be associated with germinal-center B-cell–like\n  diffuse large-B-cell lymphomas.\ndisease_id: 45\nsource_id: 239\nvariant_id: 154\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c8cf4bc2-eaae-47ea-8c4f-46170ee0ae47	2015-07-22 18:47:35.78475
653	411	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In non-APL patients with AML, those who had mutations in RUNX1 and were\n  treated with allogeneic hematopoietic stem-cell transplantation had longer relapse-free\n  survival than those treated with conventional post remmision treatment.\ndisease_id: 3\nsource_id: 241\nvariant_id: 155\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	df90bba3-2191-46b6-bbf3-6f5cea1c34c6	2015-07-22 18:47:35.85596
714	78	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '12'\nstart:\n- \n- '25398285'\nstop:\n- \n- '25398285'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- A\n	2	\N	\N	154b4b21-aa0c-42e9-a1c8-08a1a6b08ce5	2015-07-22 18:49:40.652285
715	79	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '12'\nstart:\n- \n- '25398284'\nstop:\n- \n- '25398284'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	1ed67639-4040-4fce-a09b-9c1da261ee15	2015-07-22 18:49:40.843847
654	412	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: OSI-906, an IGF1R/INSR inhibitor, does not have a significant effect\n  on chemotherapy resistant castration-resistant prostate cancer cell lines.\ndisease_id: 46\nsource_id: 242\nvariant_id: 156\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4c94ab6f-126c-4c50-b0b2-7dc08e24823e	2015-07-22 18:47:35.930412
655	413	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NSCLC patients with exon 19 deletions in EGFR had longer overall survival\n  than patients with an L858R mutation when treated with gefitinib and erlotinib (38\n  versus 17 months; P = 0.04).\ndisease_id: 8\nsource_id: 243\nvariant_id: 133\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	0a99bf4f-c017-465d-96ff-51ced76f850d	2015-07-22 18:47:35.995194
656	414	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: OSI-906, an IGF1R/INSR inhibitor, sensitizes cell lines to docetaxel\n  and cabazitaxel, resulting in better overall survival in chemotherapy resistant\n  CRPC cell lines.\ndisease_id: 46\nsource_id: 244\nvariant_id: 156\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	dbd43b65-7a59-43f9-b408-d106f07c8a04	2015-07-22 18:47:36.073101
657	415	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a cohort of 783 young adult patients with acute myeloid leukemia,\n  those with TET2 mutations did not have significantly different rates of complete\n  recovery, refractory disease, or hypoplastic death.\ndisease_id: 3\nsource_id: 245\nvariant_id: 157\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	27ec0d5e-2413-4560-8564-020adf6b2c2a	2015-07-22 18:47:36.144946
658	416	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with ovarian carcinomas, those with Aurora kinase A expression\n  (as measured by immunohistochemistry) had increased progression-free and overall\n  survival compared to those without expression.\ndisease_id: 47\nsource_id: 246\nvariant_id: 158\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	387ffbab-33c5-4faf-9bc3-7c2429064a5c	2015-07-22 18:47:36.191587
659	417	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with myelodysplastic syndromes, those with TET2 mutations\n  have been shown to have better overall survival than patients without TET2 mutations.\ndisease_id: 34\nsource_id: 247\nvariant_id: 157\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b24a7c2a-3648-411e-a6d5-d0bc4a55d698	2015-07-22 18:47:36.241378
660	418	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with renal clear cell carcinoma, patients with VHL mutations\n  had longer cancer-specific survival.\ndisease_id: 48\nsource_id: 248\nvariant_id: 160\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a2274bc0-dcfb-4405-872a-f40a68e9f9a5	2015-07-22 18:47:36.297644
661	419	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with cytogenetically normal acute myeloid leukemia, those\n  in the intermediate-I risk group with TET2 mutations had higher rates of complete\n  remission.\ndisease_id: 3\nsource_id: 245\nvariant_id: 157\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4f802f94-b164-4d04-a26a-a60f5b61b204	2015-07-22 18:47:36.340289
662	420	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In renal clear cell carcinoma, patients with PBRM1 mutations did not\n  have a significantly different cancer specific survival.\ndisease_id: 48\nsource_id: 249\nvariant_id: 161\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	085d2140-5ee4-4652-b8ef-d5f6043f843e	2015-07-22 18:47:36.395814
663	421	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with cytogenetically normal acute myeloid leukemia, those\n  in the intermediate-I risk group with TET2 mutations did not have significantly\n  different rates of event-free survival, complete remission, disease-free survival,\n  or overall survival.\ndisease_id: 3\nsource_id: 250\nvariant_id: 157\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	04a07c80-41c9-437a-8016-a35bc9fe712a	2015-07-22 18:47:36.478725
664	422	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In renal clear cell carcinoma, patients with VHL mutations did not have\n  a significant impact on cancer specific survival.\ndisease_id: 48\nsource_id: 249\nvariant_id: 160\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0cf6c978-b92d-4f9a-a02c-04ecfcf5bfb8	2015-07-22 18:47:36.518783
665	423	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with Diffuse Large B-Cell Lymphoma, those with increased\n  levels of BCL2 were shown to have decreased overall, disease-free, and relapse-free\n  survival.\ndisease_id: 45\nsource_id: 251\nvariant_id: 152\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e2bf0a0c-41ce-488b-b98a-f66d93053b97	2015-07-22 18:47:36.571988
666	424	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with Diffuse Large B-Cell Lymphoma, those with increased\n  levels of BCL2 were shown to have decreased OS, DFS, and RFS.\ndisease_id: 45\nsource_id: 251\nvariant_id: 152\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cb3441dd-0f0e-43df-a074-cd66fdc2ccb9	2015-07-22 18:47:36.613657
735	117	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577538'\nstop:\n- \n- '7577538'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	35e6fd31-1e71-4bd8-b342-765990cf806d	2015-07-22 18:49:41.22724
667	425	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In myelodysplastic syndrome, patients with point mutations in EZH2 have\n  worse overall survival that their wild-type counterparts.\ndisease_id: 49\nsource_id: 252\nvariant_id: 163\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cdfc60d9-be57-45f7-8d15-18760f1eaeed	2015-07-22 18:47:36.678871
668	426	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients classified as intermediate-risk via cytogenetics, TET2 mutations\n  have been shown to be correlated with poor prognosis.\ndisease_id: 3\nsource_id: 253\nvariant_id: 157\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7df3c9a1-dca9-4f79-9a0b-11e6f5c027d5	2015-07-22 18:47:36.737581
669	427	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with cytogenetically normal acute myeloid leukemia, those\n  in the ELN favorable-risk group with TET2 mutations had shorter event-free survival,\n  lower rates of complete remission, and shorter disease-free survival.\ndisease_id: 3\nsource_id: 250\nvariant_id: 157\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f52e0dbb-81e9-4045-9a7c-8c697862b137	2015-07-22 18:47:36.781422
670	428	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with renal clear cell carcinoma, patients with loss of function\n  VHL mutations had shorter tumor-specific survival.\ndisease_id: 48\nsource_id: 254\nvariant_id: 159\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e72cba63-5a53-449f-ab78-b2fdd7cc3d45	2015-07-22 18:47:36.83161
671	429	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In non-APL patients with AML, those who had mutations in RUNX1 had higher\n  rates of refractory disease than those without.\ndisease_id: 3\nsource_id: 241\nvariant_id: 155\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b65fb19a-c02b-4e46-abc1-d3a4948c8cd6	2015-07-22 18:47:36.867458
672	430	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In non-APL patients with AML, those who had mutations in RUNX1 had shorter\n  event-free survival and relapse-free survival relative to wild-type.\ndisease_id: 3\nsource_id: 241\nvariant_id: 155\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	53560754-43da-48b7-98a2-f8d782d55509	2015-07-22 18:47:36.914955
673	431	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with Ewing's sarcoma, those with type 1 EWS-FLI1 fusions\n  had longer overall survival than those with other types of EWS-FLI1 fusions.\ndisease_id: 50\nsource_id: 255\nvariant_id: 164\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\ndrug_interaction_description: \nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1644004c-febb-43ed-94f7-01e75fc1f3d4	2015-07-22 18:47:36.967966
674	1	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'While imatinib has shown to be incredibly successful in treating philadelphia chromosome\n  positive CML, patients that have shown primary or secondary resistance to the drug\n  have been observed to harbor T315I and E255K ABL kinase domain mutations. These\n  mutations, among others, have been observed both in primary refractory disease and\n  acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations\n  have also been shown to confer resistance to imatinib. '\n	2	\N	\N	c0d807e2-fb4b-4900-8566-606b55c5758e	2015-07-22 18:47:37.075382
675	11	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset\n  of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase\n  the activity of the ALK oncogene relative to normal cells. While EML4 is the most\n  common fusion partner, other 5'' partners have been observed. The EML4-ALK fusion\n  has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. '\n	2	\N	\N	200327bf-c825-4bf4-9c3f-cfa94c1cd8f5	2015-07-22 18:47:37.103097
676	12	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine\n  kinase inhibitors have been developed to target EGFR pathway activity. One such\n  inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting.\n  However, the T790M missense mutation has shown to confer resistance to this inhibitor\n  in cell lines and case studies.  '\n	2	\N	\N	918a49e2-d3f3-4a2e-a371-ebb8bbf43888	2015-07-22 18:47:37.125907
677	3	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'The ALK oncogene has long been considered a driving factor in non-small cell lung\n  cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be\n  effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance\n  to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive\n  this resistance. '\n	2	\N	\N	68e45a91-6254-4393-ba5c-d42336c34d42	2015-07-22 18:47:37.148363
678	5	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants\n  have also been observed and studied to a lesser degree. At first approximation,\n  many of these variants seem to behave similarly to V600E, and treatment with dabrafenib\n  has been shown to be effective. '\n	2	\N	\N	321c3d7b-8fb3-409e-94ec-3e78a8cfedee	2015-07-22 18:47:37.164475
679	14	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'HER2-positive breast cancer is widely recognized as a molecular subtype of breast\n  cancer. While amplification of ERBB2 is a common mechanism for activation of the\n  pathway, missense mutations have also demonstrated activation potential. These activating\n  missense mutations also confer senstivity to the targeted therapeutic neratinib. '\n	2	\N	\N	25bf623e-8089-425c-86a5-608eff72b408	2015-07-22 18:47:37.178901
680	13	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone\n  therapy has been widely effective in treating the disease, however sequencing of\n  resistant patients has uncovered a number of missense mutations in mediating this\n  resistance. Many of these mutations lie within the ligand-binding domain, and contribute\n  to constitutive activity of the receptor. This has lead to the development of estrogen\n  receptor degrading agents such as fulvestrant, which have shown early efficacy in\n  clinical trials.  '\n	2	\N	\N	cbc0f27e-fca0-4f47-ad26-133a8a2e96e1	2015-07-22 18:47:37.194786
682	7	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\n	2	\N	\N	06903f6b-5e1f-401a-85b1-c02c87f6dabf	2015-07-22 18:47:37.228873
683	15	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'PTEN loss is a common event in breast cancer, as well as others. The most common\n  single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell\n  lines harboring this mutation behave similarly to cell lines harboring larger insertion\n  or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded\n  growth. '\n	2	\N	\N	26dfab84-4656-4d87-acfe-1507e78234f7	2015-07-22 18:47:37.245936
684	6	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment\n  of these patients with the targeted therapeutic motesanib has shown to be effective.\n  However, the missense mutations C634W and M918T have shown to confer motesanib resistance\n  in cell lines. '\n	2	\N	\N	5941fd63-b6d8-4fc8-9177-07ccfe4c734d	2015-07-22 18:47:37.259489
685	9	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types.\n  Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor\n  everolimus has seen significant anti-tumorigenic activity. '\n	2	\N	\N	069496c4-c6ac-451a-9e67-bb23d9ccf1c0	2015-07-22 18:47:37.274694
686	10	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'BRCA germline variants have long been recognized as important potential predictors\n  of breast and ovarian cancer risk. '\n	2	\N	\N	114b998f-ea1c-46f1-a13c-e326a7c21a07	2015-07-22 18:47:37.291022
687	4	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '14'\nstart:\n- \n- '105246551'\nstop:\n- \n- '105246551'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	4cb8f809-3bd0-452a-a1d4-5fa2cf07ae0f	2015-07-22 18:49:40.122149
688	8	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '29443695'\nstop:\n- \n- '29443695'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	0f9872aa-7588-4ba4-bbfc-816ff92e1a8c	2015-07-22 18:49:40.139555
689	9	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '29432664'\nstop:\n- \n- '29432664'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	76875151-8b7e-4af1-bf24-5cf2d0c1ee2b	2015-07-22 18:49:40.154143
690	10	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- X\nstart:\n- \n- '47426121'\nstop:\n- \n- '47426121'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- G\n	2	\N	\N	dcfb4d98-5178-40e2-8328-5989b132e505	2015-07-22 18:49:40.166428
691	11	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453135'\nstop:\n- \n- '140453136'\nreference_bases:\n- \n- CA\nvariant_bases:\n- \n- AT\n	2	\N	\N	e97eebd7-68fd-4bc9-a4db-83301ebf3201	2015-07-22 18:49:40.183053
692	12	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453136'\nstop:\n- \n- '140453136'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- T\n	2	\N	\N	f672edfd-2ef3-436a-9153-869bc8bc31ae	2015-07-22 18:49:40.195537
693	15	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453137'\nstop:\n- \n- '140453137'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	07a13816-c711-4e68-a070-a285739b96e4	2015-07-22 18:49:40.209092
694	16	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453136'\nstop:\n- \n- '140453137'\nreference_bases:\n- \n- AC\nvariant_bases:\n- \n- CT\n	2	\N	\N	cc7bd1ba-c381-4030-9046-77ac2f68c52c	2015-07-22 18:49:40.223658
695	33	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '55259514'\nstop:\n- \n- '55259515'\nreference_bases:\n- \n- CT\nvariant_bases:\n- \n- AG\n	2	\N	\N	ebf80527-a046-452d-a6eb-15a253a58704	2015-07-22 18:49:40.244442
696	34	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '55249071'\nstop:\n- \n- '55249071'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	25228a58-ad0d-4ae6-a636-381b5c8fb4b6	2015-07-22 18:49:40.257354
697	35	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37880261'\nstop:\n- \n- '37880261'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- C\n	2	\N	\N	bf3ef9c1-3d8c-4253-89ba-842130272675	2015-07-22 18:49:40.275212
698	36	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37880261'\nstop:\n- \n- '37880261'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	a27fcaa1-be45-4a8c-a0af-91911766dd39	2015-07-22 18:49:40.29203
699	38	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37868205'\nstop:\n- \n- '37868205'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- C\n	2	\N	\N	2b80799c-df97-4b2a-81ec-73ba32598cd1	2015-07-22 18:49:40.3093
700	39	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37880220'\nstop:\n- \n- '37880220'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	ecb3bca0-24c5-4620-a026-d2ea0e04ad2a	2015-07-22 18:49:40.33492
701	43	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37881616'\nstop:\n- \n- '37881616'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	67d9a37e-33b1-4404-9063-d5e698ec0524	2015-07-22 18:49:40.355776
702	44	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37881000'\nstop:\n- \n- '37881000'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	3cb0f57c-8a3a-4be3-aa19-a37ddc93e3b1	2015-07-22 18:49:40.376336
703	45	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37881332'\nstop:\n- \n- '37881332'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	647414a2-1eac-4684-8033-7177f61ac82f	2015-07-22 18:49:40.396244
704	48	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '6'\nstart:\n- \n- '152419923'\nstop:\n- \n- '152419923'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- G\n	2	\N	\N	cf432ffb-851d-4df6-824a-2166492facee	2015-07-22 18:49:40.421517
705	49	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '6'\nstart:\n- \n- '152419922'\nstop:\n- \n- '152419922'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- A\n	2	\N	\N	7ec8bded-2a4b-47aa-a7ca-405e101e8055	2015-07-22 18:49:40.442524
706	50	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '6'\nstart:\n- \n- '152419923'\nstop:\n- \n- '152419923'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- C\n	2	\N	\N	7fb9f2db-4fd8-43b1-8aca-e96fffb45928	2015-07-22 18:49:40.46602
707	59	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '209113113'\nstop:\n- \n- '209113113'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	a008837d-a35f-4845-b307-3a7d31d9734a	2015-07-22 18:49:40.48784
708	60	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '209113112'\nstop:\n- \n- '209113112'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	2edc2531-9b78-4960-a26a-b65116d29aca	2015-07-22 18:49:40.506018
709	61	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '209113112'\nstop:\n- \n- '209113112'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- A\n	2	\N	\N	f9de9c3c-4c5c-4c69-8963-365285e1fe7d	2015-07-22 18:49:40.527552
740	122	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577120'\nstop:\n- \n- '7577120'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	158fde36-b56a-49d1-9e99-e5156ccbdad6	2015-07-22 18:49:41.31747
\.


--
-- Name: audits_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('audits_id_seq', 740, true);


--
-- Data for Name: authorizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY authorizations (id, user_id, provider, uid, created_at, updated_at) FROM stdin;
1	1	github	13370	2015-07-22 18:55:30.732669	2015-07-22 18:55:30.732669
\.


--
-- Name: authorizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('authorizations_id_seq', 1, true);


--
-- Data for Name: comments; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY comments (id, title, comment, commentable_id, commentable_type, user_id, role, created_at, updated_at) FROM stdin;
\.


--
-- Name: comments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('comments_id_seq', 1, false);


--
-- Data for Name: data_versions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY data_versions (id, version) FROM stdin;
\.


--
-- Name: data_versions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('data_versions_id_seq', 1, false);


--
-- Data for Name: definitions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY definitions (id, term, text, created_at, updated_at) FROM stdin;
\.


--
-- Name: definitions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('definitions_id_seq', 1, false);


--
-- Data for Name: delayed_jobs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY delayed_jobs (id, priority, attempts, handler, last_error, run_at, locked_at, failed_at, locked_by, queue, created_at, updated_at) FROM stdin;
1	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6840a321-b661-40cd-8a2c-6df4a18771bc\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/1\n	\N	2015-07-22 18:46:55.492627	\N	\N	\N	default	2015-07-22 18:46:55.49286	2015-07-22 18:46:55.49286
2	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 12ccd14e-e7ad-4737-b0ea-b58574581471\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/2\n	\N	2015-07-22 18:46:55.511165	\N	\N	\N	default	2015-07-22 18:46:55.511276	2015-07-22 18:46:55.511276
3	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 961ca8bc-dbda-43f3-bd8b-2198d0e03a74\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/3\n	\N	2015-07-22 18:46:55.760894	\N	\N	\N	default	2015-07-22 18:46:55.761019	2015-07-22 18:46:55.761019
4	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ddb344dd-2730-4e05-b4b6-905d492b55d1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/4\n	\N	2015-07-22 18:46:55.769908	\N	\N	\N	default	2015-07-22 18:46:55.770014	2015-07-22 18:46:55.770014
5	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5d1e97b0-7df8-44bc-bcc6-ac83ef66e8c8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/5\n	\N	2015-07-22 18:46:56.004294	\N	\N	\N	default	2015-07-22 18:46:56.004553	2015-07-22 18:46:56.004553
6	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ea8724af-87cb-4723-9bac-941c074d968d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/6\n	\N	2015-07-22 18:46:56.028365	\N	\N	\N	default	2015-07-22 18:46:56.02848	2015-07-22 18:46:56.02848
7	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 484d7e7d-0db4-486e-bc44-4a34737b8d26\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/7\n	\N	2015-07-22 18:46:56.283254	\N	\N	\N	default	2015-07-22 18:46:56.283379	2015-07-22 18:46:56.283379
8	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a9aa2328-519e-46bd-a693-d56bae9763d1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/8\n	\N	2015-07-22 18:46:56.294503	\N	\N	\N	default	2015-07-22 18:46:56.294659	2015-07-22 18:46:56.294659
9	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 81ac6e50-c7b4-42dc-af3c-05d80a682706\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/9\n	\N	2015-07-22 18:46:56.581058	\N	\N	\N	default	2015-07-22 18:46:56.581169	2015-07-22 18:46:56.581169
10	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 572501aa-6cc8-4c08-bb2e-2e615fca4812\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/10\n	\N	2015-07-22 18:46:56.590101	\N	\N	\N	default	2015-07-22 18:46:56.590211	2015-07-22 18:46:56.590211
11	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7d38c359-b96a-4675-a907-27d3632b7f7c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/11\n	\N	2015-07-22 18:46:56.85815	\N	\N	\N	default	2015-07-22 18:46:56.858318	2015-07-22 18:46:56.858318
12	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 48fbc13d-80af-486b-9964-6ab112375ea6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/12\n	\N	2015-07-22 18:46:57.35505	\N	\N	\N	default	2015-07-22 18:46:57.355162	2015-07-22 18:46:57.355162
13	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 887b6b98-e5d1-4fd6-ab16-519abd5918da\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/13\n	\N	2015-07-22 18:46:57.366098	\N	\N	\N	default	2015-07-22 18:46:57.366224	2015-07-22 18:46:57.366224
14	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8fd88702-09e2-4daa-9536-dd42b2e849c7\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/14\n	\N	2015-07-22 18:46:57.963432	\N	\N	\N	default	2015-07-22 18:46:57.963581	2015-07-22 18:46:57.963581
15	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e9ddc88e-0d1a-4967-9094-36a501d1f6e4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/15\n	\N	2015-07-22 18:46:58.181305	\N	\N	\N	default	2015-07-22 18:46:58.181414	2015-07-22 18:46:58.181414
16	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: adc0a91d-9966-40e6-a201-81de49d0866e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/16\n	\N	2015-07-22 18:46:58.190209	\N	\N	\N	default	2015-07-22 18:46:58.190317	2015-07-22 18:46:58.190317
17	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f7e898bc-8354-4011-93c3-ea3131aaf40b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/17\n	\N	2015-07-22 18:46:58.672595	\N	\N	\N	default	2015-07-22 18:46:58.672761	2015-07-22 18:46:58.672761
18	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e325cab5-1aa4-40da-9b53-6beb9c0ca983\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/18\n	\N	2015-07-22 18:46:58.681625	\N	\N	\N	default	2015-07-22 18:46:58.681732	2015-07-22 18:46:58.681732
19	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: bcde1aa6-c0e6-44df-9945-d19441128431\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/19\n	\N	2015-07-22 18:46:58.92646	\N	\N	\N	default	2015-07-22 18:46:58.926592	2015-07-22 18:46:58.926592
20	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 872bbe4b-ce72-4071-977c-33194967256d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/20\n	\N	2015-07-22 18:46:58.936653	\N	\N	\N	default	2015-07-22 18:46:58.93676	2015-07-22 18:46:58.93676
21	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 56e58780-71ff-4021-a00f-912de29c449e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/21\n	\N	2015-07-22 18:46:58.945753	\N	\N	\N	default	2015-07-22 18:46:58.945873	2015-07-22 18:46:58.945873
22	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f42b6e28-29b2-4d79-bd42-858a55b1e15c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/22\n	\N	2015-07-22 18:46:59.171283	\N	\N	\N	default	2015-07-22 18:46:59.171393	2015-07-22 18:46:59.171393
23	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a263e329-dcc4-4a7d-b663-dd2e79600449\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/23\n	\N	2015-07-22 18:46:59.397305	\N	\N	\N	default	2015-07-22 18:46:59.397535	2015-07-22 18:46:59.397535
24	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8ec3eabb-c250-47eb-9a40-000188edfc57\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/24\n	\N	2015-07-22 18:46:59.609413	\N	\N	\N	default	2015-07-22 18:46:59.60954	2015-07-22 18:46:59.60954
25	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d42a1ba4-3294-428f-96c1-7f5d1d19903b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/25\n	\N	2015-07-22 18:46:59.619606	\N	\N	\N	default	2015-07-22 18:46:59.619734	2015-07-22 18:46:59.619734
26	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 07cca4af-615b-451d-90cd-a3ad4e770ab3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/26\n	\N	2015-07-22 18:46:59.878421	\N	\N	\N	default	2015-07-22 18:46:59.878552	2015-07-22 18:46:59.878552
27	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 13c9fb2d-a403-4222-89e3-4f584e82f7dc\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/27\n	\N	2015-07-22 18:46:59.89219	\N	\N	\N	default	2015-07-22 18:46:59.892301	2015-07-22 18:46:59.892301
28	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3fff4db8-8338-421d-84e0-c2d787109e2c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/28\n	\N	2015-07-22 18:47:00.156594	\N	\N	\N	default	2015-07-22 18:47:00.156758	2015-07-22 18:47:00.156758
29	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 24853890-9755-4430-9451-aa8571cf0c55\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/29\n	\N	2015-07-22 18:47:00.171643	\N	\N	\N	default	2015-07-22 18:47:00.171771	2015-07-22 18:47:00.171771
30	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 207efab1-409b-47e8-a92f-bf1d1e6b6eae\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/30\n	\N	2015-07-22 18:47:00.431956	\N	\N	\N	default	2015-07-22 18:47:00.432087	2015-07-22 18:47:00.432087
31	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2fe10498-3032-4daa-9a31-4d0c8fc8768a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/31\n	\N	2015-07-22 18:47:00.443278	\N	\N	\N	default	2015-07-22 18:47:00.44347	2015-07-22 18:47:00.44347
32	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7cd6a069-327d-409b-be0c-f2bef3f86fda\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/32\n	\N	2015-07-22 18:47:00.71391	\N	\N	\N	default	2015-07-22 18:47:00.714025	2015-07-22 18:47:00.714025
33	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7c35e59c-2f50-4519-84d9-777028814730\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/33\n	\N	2015-07-22 18:47:00.722957	\N	\N	\N	default	2015-07-22 18:47:00.723064	2015-07-22 18:47:00.723064
34	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 80f71e15-d10a-4590-b47a-f266c0ff7547\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/34\n	\N	2015-07-22 18:47:01.195209	\N	\N	\N	default	2015-07-22 18:47:01.19535	2015-07-22 18:47:01.19535
35	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: fbf3b98d-85c3-4010-86da-138423f63df0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/35\n	\N	2015-07-22 18:47:01.415603	\N	\N	\N	default	2015-07-22 18:47:01.415729	2015-07-22 18:47:01.415729
36	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1fb7f46c-7de9-411e-8e29-19c009d3ef5a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/36\n	\N	2015-07-22 18:47:01.42475	\N	\N	\N	default	2015-07-22 18:47:01.424859	2015-07-22 18:47:01.424859
37	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2a391bb0-8fa5-47b8-bbbe-87bc5e683c8e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/37\n	\N	2015-07-22 18:47:01.668974	\N	\N	\N	default	2015-07-22 18:47:01.669083	2015-07-22 18:47:01.669083
38	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1faad215-ae8d-43ec-8e52-99ef9561f678\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/38\n	\N	2015-07-22 18:47:01.680576	\N	\N	\N	default	2015-07-22 18:47:01.680687	2015-07-22 18:47:01.680687
39	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ca4877b8-890c-4e4f-b4f8-99d89a6af0c6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/39\n	\N	2015-07-22 18:47:01.928395	\N	\N	\N	default	2015-07-22 18:47:01.928504	2015-07-22 18:47:01.928504
40	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1758ff88-ea15-472a-8948-d3153d65b163\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/40\n	\N	2015-07-22 18:47:02.144895	\N	\N	\N	default	2015-07-22 18:47:02.14501	2015-07-22 18:47:02.14501
41	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6c2d3459-28ba-434c-92c9-e5b20b5ba596\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/41\n	\N	2015-07-22 18:47:02.357695	\N	\N	\N	default	2015-07-22 18:47:02.357807	2015-07-22 18:47:02.357807
42	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6bcf4dfe-62ca-454c-93ee-f520dbf1125b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/42\n	\N	2015-07-22 18:47:02.621256	\N	\N	\N	default	2015-07-22 18:47:02.621365	2015-07-22 18:47:02.621365
43	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: cef8a4c4-46f4-4812-81c1-a08c795733d5\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/43\n	\N	2015-07-22 18:47:03.722457	\N	\N	\N	default	2015-07-22 18:47:03.722591	2015-07-22 18:47:03.722591
44	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c08ff1bb-2f24-44ee-adca-09eb2c555bc5\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/44\n	\N	2015-07-22 18:47:05.38952	\N	\N	\N	default	2015-07-22 18:47:05.389661	2015-07-22 18:47:05.389661
45	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 20e35b20-5944-4fd5-9e41-5980f2e9e86a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/45\n	\N	2015-07-22 18:47:05.401405	\N	\N	\N	default	2015-07-22 18:47:05.401548	2015-07-22 18:47:05.401548
46	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7d3ca681-163a-46b0-826d-c0ed9115822c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/46\n	\N	2015-07-22 18:47:05.900418	\N	\N	\N	default	2015-07-22 18:47:05.900549	2015-07-22 18:47:05.900549
47	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 16f678f5-165a-48bd-988f-a1c1c890ae2b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/47\n	\N	2015-07-22 18:47:05.913137	\N	\N	\N	default	2015-07-22 18:47:05.913331	2015-07-22 18:47:05.913331
48	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b3936447-2ed1-4316-bc10-5717c3034894\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/48\n	\N	2015-07-22 18:47:06.17584	\N	\N	\N	default	2015-07-22 18:47:06.175982	2015-07-22 18:47:06.175982
49	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 62712b76-9098-497e-931c-db791dc28cfd\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/49\n	\N	2015-07-22 18:47:06.857647	\N	\N	\N	default	2015-07-22 18:47:06.857843	2015-07-22 18:47:06.857843
50	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6c901219-f5c6-4b97-ae66-8ef200708bd8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/50\n	\N	2015-07-22 18:47:06.86927	\N	\N	\N	default	2015-07-22 18:47:06.869393	2015-07-22 18:47:06.869393
51	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 354262b8-3744-4baa-a84c-fec665bfd927\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/51\n	\N	2015-07-22 18:47:14.400599	\N	\N	\N	default	2015-07-22 18:47:14.400728	2015-07-22 18:47:14.400728
52	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 415982ac-3ff5-4fe3-bbf2-b05dbd77d772\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/52\n	\N	2015-07-22 18:47:14.480121	\N	\N	\N	default	2015-07-22 18:47:14.480236	2015-07-22 18:47:14.480236
53	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 95b4f295-2d52-4ab1-b533-92b352076ac0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/53\n	\N	2015-07-22 18:47:14.602877	\N	\N	\N	default	2015-07-22 18:47:14.603027	2015-07-22 18:47:14.603027
54	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7c43360b-e3e9-4091-b2c0-604a4aa794f3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/54\n	\N	2015-07-22 18:47:14.667301	\N	\N	\N	default	2015-07-22 18:47:14.667409	2015-07-22 18:47:14.667409
55	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a6365eec-fdb1-491e-943b-0802f2d762cd\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/55\n	\N	2015-07-22 18:47:14.784206	\N	\N	\N	default	2015-07-22 18:47:14.784323	2015-07-22 18:47:14.784323
56	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c5e4e308-7153-47dd-8cd5-ebdcaf989bb1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/56\n	\N	2015-07-22 18:47:14.8257	\N	\N	\N	default	2015-07-22 18:47:14.825809	2015-07-22 18:47:14.825809
57	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7e0f9200-c37f-4394-8d7f-1c0d6ba446f9\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/57\n	\N	2015-07-22 18:47:14.902689	\N	\N	\N	default	2015-07-22 18:47:14.90284	2015-07-22 18:47:14.90284
58	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1f79550d-dfe1-4f40-84ae-c32721fd0ca9\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/58\n	\N	2015-07-22 18:47:14.997794	\N	\N	\N	default	2015-07-22 18:47:14.997918	2015-07-22 18:47:14.997918
59	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 51bda9db-74ab-4a5e-b7f4-4bb1e1c77024\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/59\n	\N	2015-07-22 18:47:15.046153	\N	\N	\N	default	2015-07-22 18:47:15.046295	2015-07-22 18:47:15.046295
60	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 82f1c819-dc92-4d53-b445-d7f6179b97be\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/60\n	\N	2015-07-22 18:47:15.09345	\N	\N	\N	default	2015-07-22 18:47:15.093591	2015-07-22 18:47:15.093591
61	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 306d4edf-fc03-4609-9388-f82ba2dc15ab\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/61\n	\N	2015-07-22 18:47:15.158177	\N	\N	\N	default	2015-07-22 18:47:15.158366	2015-07-22 18:47:15.158366
62	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 34a5a6b6-9873-408e-9a69-97f6e3a8595f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/62\n	\N	2015-07-22 18:47:15.216056	\N	\N	\N	default	2015-07-22 18:47:15.216257	2015-07-22 18:47:15.216257
63	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5a8c443d-7390-4438-91e1-125c7d8014c2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/63\n	\N	2015-07-22 18:47:15.278446	\N	\N	\N	default	2015-07-22 18:47:15.279247	2015-07-22 18:47:15.279247
64	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d1ac3846-1373-41e7-94cf-3482313e6029\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/64\n	\N	2015-07-22 18:47:15.371051	\N	\N	\N	default	2015-07-22 18:47:15.371172	2015-07-22 18:47:15.371172
65	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8137987a-2931-4cea-b9a8-643aedbccd38\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/65\n	\N	2015-07-22 18:47:15.431494	\N	\N	\N	default	2015-07-22 18:47:15.431606	2015-07-22 18:47:15.431606
66	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0d029ed7-0690-41f4-9f4b-294fc7f313c3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/66\n	\N	2015-07-22 18:47:15.491006	\N	\N	\N	default	2015-07-22 18:47:15.49113	2015-07-22 18:47:15.49113
67	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a95cf301-46c1-459f-b11a-0450e160ba8c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/67\n	\N	2015-07-22 18:47:15.546231	\N	\N	\N	default	2015-07-22 18:47:15.546341	2015-07-22 18:47:15.546341
68	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 32401031-93a2-49ed-9931-4b78ad891d08\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/68\n	\N	2015-07-22 18:47:15.708739	\N	\N	\N	default	2015-07-22 18:47:15.708851	2015-07-22 18:47:15.708851
69	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c5c392fa-e5ee-41b9-b5a4-c684e39e63f5\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/69\n	\N	2015-07-22 18:47:15.750086	\N	\N	\N	default	2015-07-22 18:47:15.750208	2015-07-22 18:47:15.750208
70	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: bf6ab43c-7c78-4a56-bbe8-b33053fde3f9\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/70\n	\N	2015-07-22 18:47:15.814563	\N	\N	\N	default	2015-07-22 18:47:15.814685	2015-07-22 18:47:15.814685
71	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 04a13833-c302-41ee-b674-1e8d017af9cb\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/71\n	\N	2015-07-22 18:47:15.930308	\N	\N	\N	default	2015-07-22 18:47:15.930419	2015-07-22 18:47:15.930419
72	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d9196e2d-0668-41a3-b82d-d6344ddac94c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/72\n	\N	2015-07-22 18:47:15.979607	\N	\N	\N	default	2015-07-22 18:47:15.979744	2015-07-22 18:47:15.979744
73	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d6900231-fce9-4946-97da-ed1a9975ceda\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/73\n	\N	2015-07-22 18:47:16.02735	\N	\N	\N	default	2015-07-22 18:47:16.02748	2015-07-22 18:47:16.02748
74	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6b64a4a4-b91b-4be6-8e58-03c786c83608\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/74\n	\N	2015-07-22 18:47:16.147529	\N	\N	\N	default	2015-07-22 18:47:16.147657	2015-07-22 18:47:16.147657
75	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4ad3bbac-d183-40cc-8a44-297310d37b9e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/75\n	\N	2015-07-22 18:47:16.221649	\N	\N	\N	default	2015-07-22 18:47:16.221792	2015-07-22 18:47:16.221792
76	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8701cc56-9bec-46b9-b7d4-eb4b7e6c847b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/76\n	\N	2015-07-22 18:47:16.290978	\N	\N	\N	default	2015-07-22 18:47:16.291089	2015-07-22 18:47:16.291089
77	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 16ed1363-7ef6-4965-abe4-5a9083329e61\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/77\n	\N	2015-07-22 18:47:16.338108	\N	\N	\N	default	2015-07-22 18:47:16.338214	2015-07-22 18:47:16.338214
78	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f47cf182-bcbc-4e9d-ae02-23d47ca24455\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/78\n	\N	2015-07-22 18:47:16.384945	\N	\N	\N	default	2015-07-22 18:47:16.385075	2015-07-22 18:47:16.385075
79	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3cb52819-23d7-4bef-b011-732e2953cdc8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/79\n	\N	2015-07-22 18:47:16.429302	\N	\N	\N	default	2015-07-22 18:47:16.429411	2015-07-22 18:47:16.429411
80	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 407e051a-8839-4ceb-b71f-039d71cad33e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/80\n	\N	2015-07-22 18:47:16.601817	\N	\N	\N	default	2015-07-22 18:47:16.601945	2015-07-22 18:47:16.601945
81	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: fd873fbf-4764-4001-906d-cfe54171c41c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/81\n	\N	2015-07-22 18:47:16.895159	\N	\N	\N	default	2015-07-22 18:47:16.895278	2015-07-22 18:47:16.895278
82	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5fc561b9-2236-4b13-aaca-c2318158157b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/82\n	\N	2015-07-22 18:47:17.034914	\N	\N	\N	default	2015-07-22 18:47:17.035022	2015-07-22 18:47:17.035022
83	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c15309a0-d90f-4ba6-9186-61ef5fd7fc65\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/83\n	\N	2015-07-22 18:47:17.155241	\N	\N	\N	default	2015-07-22 18:47:17.155518	2015-07-22 18:47:17.155518
84	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3fb8be60-6012-4edb-87f0-0262a5ff4a6c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/84\n	\N	2015-07-22 18:47:17.199352	\N	\N	\N	default	2015-07-22 18:47:17.199466	2015-07-22 18:47:17.199466
85	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: fb5375f5-d751-4a78-903d-e837e453940b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/85\n	\N	2015-07-22 18:47:17.238137	\N	\N	\N	default	2015-07-22 18:47:17.238252	2015-07-22 18:47:17.238252
86	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4cbec4c0-12b0-40cd-8076-256e9da7724e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/86\n	\N	2015-07-22 18:47:17.589286	\N	\N	\N	default	2015-07-22 18:47:17.589402	2015-07-22 18:47:17.589402
87	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0e9f2870-218a-4519-8b5d-44ed2e3dfdcf\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/87\n	\N	2015-07-22 18:47:17.789535	\N	\N	\N	default	2015-07-22 18:47:17.789674	2015-07-22 18:47:17.789674
88	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d7c0782d-9102-4d5e-87a7-1833b5e0637f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/88\n	\N	2015-07-22 18:47:17.877522	\N	\N	\N	default	2015-07-22 18:47:17.877651	2015-07-22 18:47:17.877651
89	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f79bc281-8a70-4075-9b94-998733b188ca\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/89\n	\N	2015-07-22 18:47:17.958344	\N	\N	\N	default	2015-07-22 18:47:17.958459	2015-07-22 18:47:17.958459
90	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 94abd2fd-4665-4977-887d-5e5524dc9a52\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/90\n	\N	2015-07-22 18:47:18.000486	\N	\N	\N	default	2015-07-22 18:47:18.000618	2015-07-22 18:47:18.000618
91	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5444ffc9-2f59-49b2-96d8-a1b6a8b64fbe\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/91\n	\N	2015-07-22 18:47:18.115969	\N	\N	\N	default	2015-07-22 18:47:18.116079	2015-07-22 18:47:18.116079
92	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 47ce08aa-17ae-44e8-8a59-f944ff308e92\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/92\n	\N	2015-07-22 18:47:18.158123	\N	\N	\N	default	2015-07-22 18:47:18.158243	2015-07-22 18:47:18.158243
93	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a58c9f5a-5e5d-4aac-b87c-c3106dea91ad\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/93\n	\N	2015-07-22 18:47:18.202929	\N	\N	\N	default	2015-07-22 18:47:18.203043	2015-07-22 18:47:18.203043
94	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a34794ff-04d9-438c-b55c-b37db9a8559c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/94\n	\N	2015-07-22 18:47:18.245291	\N	\N	\N	default	2015-07-22 18:47:18.245398	2015-07-22 18:47:18.245398
95	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: efec162a-ae5f-4d89-b273-1b034592f315\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/95\n	\N	2015-07-22 18:47:18.283784	\N	\N	\N	default	2015-07-22 18:47:18.283893	2015-07-22 18:47:18.283893
96	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1f895f5c-7147-416f-920b-5954c83ae34c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/96\n	\N	2015-07-22 18:47:18.334168	\N	\N	\N	default	2015-07-22 18:47:18.334282	2015-07-22 18:47:18.334282
97	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 87937328-ff19-4849-aa4d-f618e6cb99cf\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/97\n	\N	2015-07-22 18:47:18.394992	\N	\N	\N	default	2015-07-22 18:47:18.395106	2015-07-22 18:47:18.395106
98	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 17b402e6-04a3-47e0-b6c9-cb85ce422a4c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/98\n	\N	2015-07-22 18:47:18.528802	\N	\N	\N	default	2015-07-22 18:47:18.528943	2015-07-22 18:47:18.528943
99	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e773bfb2-ae1f-43e1-ac07-c7fe039a6084\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/99\n	\N	2015-07-22 18:47:18.610238	\N	\N	\N	default	2015-07-22 18:47:18.61035	2015-07-22 18:47:18.61035
100	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 34bc1235-5ea2-4a0a-8f15-e1d695108f24\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/100\n	\N	2015-07-22 18:47:18.6532	\N	\N	\N	default	2015-07-22 18:47:18.653357	2015-07-22 18:47:18.653357
101	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8479f08a-fbd3-40ca-8463-a4a0fd04b357\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/101\n	\N	2015-07-22 18:47:18.730953	\N	\N	\N	default	2015-07-22 18:47:18.731064	2015-07-22 18:47:18.731064
102	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9dcf6c28-1e35-4a8e-9a77-d06b8a5bfb6a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/102\n	\N	2015-07-22 18:47:18.774228	\N	\N	\N	default	2015-07-22 18:47:18.774355	2015-07-22 18:47:18.774355
103	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5a7873a0-9019-4293-9291-cff81670d9f4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/103\n	\N	2015-07-22 18:47:18.822065	\N	\N	\N	default	2015-07-22 18:47:18.822202	2015-07-22 18:47:18.822202
104	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 83e04298-6ad8-41b3-a352-96d7ab018491\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/104\n	\N	2015-07-22 18:47:18.863677	\N	\N	\N	default	2015-07-22 18:47:18.863792	2015-07-22 18:47:18.863792
105	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: dc682ab3-86dc-4fa9-a7fc-2892f479e540\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/105\n	\N	2015-07-22 18:47:18.91365	\N	\N	\N	default	2015-07-22 18:47:18.91376	2015-07-22 18:47:18.91376
106	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 500b6309-4bae-47dc-aa8d-f730245baabf\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/106\n	\N	2015-07-22 18:47:18.988776	\N	\N	\N	default	2015-07-22 18:47:18.988908	2015-07-22 18:47:18.988908
107	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0b9e5e7f-cd17-441c-953e-e9400f822e21\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/107\n	\N	2015-07-22 18:47:19.051668	\N	\N	\N	default	2015-07-22 18:47:19.051881	2015-07-22 18:47:19.051881
108	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8932268d-4c13-45b9-a28d-189e7c577f30\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/108\n	\N	2015-07-22 18:47:19.110753	\N	\N	\N	default	2015-07-22 18:47:19.110978	2015-07-22 18:47:19.110978
109	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f208ecbe-94a4-4c42-b4c0-08187d0774b0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/109\n	\N	2015-07-22 18:47:19.313868	\N	\N	\N	default	2015-07-22 18:47:19.314021	2015-07-22 18:47:19.314021
110	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0e204654-ec18-49cb-a069-aa727d1889d4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/110\n	\N	2015-07-22 18:47:19.371984	\N	\N	\N	default	2015-07-22 18:47:19.372194	2015-07-22 18:47:19.372194
111	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9961e3ab-a8bd-45c2-805c-9909ff7f3e6f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/111\n	\N	2015-07-22 18:47:19.430618	\N	\N	\N	default	2015-07-22 18:47:19.430747	2015-07-22 18:47:19.430747
112	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ba81501a-37f5-43c5-a752-bafdbce7c48a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/112\n	\N	2015-07-22 18:47:19.54772	\N	\N	\N	default	2015-07-22 18:47:19.547921	2015-07-22 18:47:19.547921
113	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ab79547c-a740-4aae-99d8-1bf119bb3219\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/113\n	\N	2015-07-22 18:47:19.646291	\N	\N	\N	default	2015-07-22 18:47:19.646554	2015-07-22 18:47:19.646554
114	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: fa83214e-49a3-469b-987d-e3678d4df0dd\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/114\n	\N	2015-07-22 18:47:19.836447	\N	\N	\N	default	2015-07-22 18:47:19.836649	2015-07-22 18:47:19.836649
115	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4d5a307a-5ace-4afc-ab20-78ad7bc65071\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/115\n	\N	2015-07-22 18:47:20.064942	\N	\N	\N	default	2015-07-22 18:47:20.065154	2015-07-22 18:47:20.065154
116	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c06f3c59-dea1-4ba2-9fc7-8edd5636e53f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/116\n	\N	2015-07-22 18:47:20.119578	\N	\N	\N	default	2015-07-22 18:47:20.119745	2015-07-22 18:47:20.119745
117	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 21a5e7b5-871f-48d6-99fc-d1fe0cb835e3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/117\n	\N	2015-07-22 18:47:20.258971	\N	\N	\N	default	2015-07-22 18:47:20.259171	2015-07-22 18:47:20.259171
118	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d10c07f0-5b3a-49fb-a3a9-8d9cbd4d0cd7\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/118\n	\N	2015-07-22 18:47:20.373358	\N	\N	\N	default	2015-07-22 18:47:20.373606	2015-07-22 18:47:20.373606
119	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 39eac379-ad1d-49b9-be9d-7bc66b90dbf8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/119\n	\N	2015-07-22 18:47:20.561343	\N	\N	\N	default	2015-07-22 18:47:20.561472	2015-07-22 18:47:20.561472
120	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 112c8f99-57b5-4516-bafc-d10956de3433\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/120\n	\N	2015-07-22 18:47:20.686556	\N	\N	\N	default	2015-07-22 18:47:20.686854	2015-07-22 18:47:20.686854
121	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a84c410d-73e4-4e20-9bcb-2b301a3c6156\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/121\n	\N	2015-07-22 18:47:20.92174	\N	\N	\N	default	2015-07-22 18:47:20.921977	2015-07-22 18:47:20.921977
122	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 83c7df55-e77a-4a77-9ce2-3e71b2707962\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/122\n	\N	2015-07-22 18:47:21.198764	\N	\N	\N	default	2015-07-22 18:47:21.198956	2015-07-22 18:47:21.198956
123	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 67ff415d-2121-4e62-b37b-65d8518048a4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/123\n	\N	2015-07-22 18:47:21.260802	\N	\N	\N	default	2015-07-22 18:47:21.260978	2015-07-22 18:47:21.260978
124	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 564643fc-3761-4a7f-91d3-6252654d73c7\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/124\n	\N	2015-07-22 18:47:21.411498	\N	\N	\N	default	2015-07-22 18:47:21.411627	2015-07-22 18:47:21.411627
125	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d88671c0-ae10-4ec1-9a49-5779d8121ba8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/125\n	\N	2015-07-22 18:47:21.653534	\N	\N	\N	default	2015-07-22 18:47:21.653663	2015-07-22 18:47:21.653663
126	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 067f8804-047b-4ccf-915c-4e8a36f685c7\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/126\n	\N	2015-07-22 18:47:21.832046	\N	\N	\N	default	2015-07-22 18:47:21.832246	2015-07-22 18:47:21.832246
127	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 590f69a9-5890-4649-8d5a-7f3688c63949\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/127\n	\N	2015-07-22 18:47:21.886486	\N	\N	\N	default	2015-07-22 18:47:21.886612	2015-07-22 18:47:21.886612
128	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e9140dbf-20f7-49be-b0dc-c4a13c05ef6f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/128\n	\N	2015-07-22 18:47:21.954374	\N	\N	\N	default	2015-07-22 18:47:21.954603	2015-07-22 18:47:21.954603
129	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0de9906c-2335-49ae-91ac-fe67e7f65896\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/129\n	\N	2015-07-22 18:47:22.021066	\N	\N	\N	default	2015-07-22 18:47:22.021217	2015-07-22 18:47:22.021217
130	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 109ab6ae-2dd2-4758-89c7-8c61f95c7bd3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/130\n	\N	2015-07-22 18:47:22.182769	\N	\N	\N	default	2015-07-22 18:47:22.182897	2015-07-22 18:47:22.182897
131	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ad2fe2b5-dfb4-4d56-af75-cd52a6ae6fca\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/131\n	\N	2015-07-22 18:47:22.252287	\N	\N	\N	default	2015-07-22 18:47:22.252479	2015-07-22 18:47:22.252479
132	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d4a11f66-8eab-420b-a467-f444b50b2b17\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/132\n	\N	2015-07-22 18:47:22.334832	\N	\N	\N	default	2015-07-22 18:47:22.334987	2015-07-22 18:47:22.334987
133	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 30ced5d3-2754-416b-a57f-000d9d9dd192\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/133\n	\N	2015-07-22 18:47:22.398426	\N	\N	\N	default	2015-07-22 18:47:22.398597	2015-07-22 18:47:22.398597
134	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 14cd646a-e57e-402f-a91a-357910613f92\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/134\n	\N	2015-07-22 18:47:22.501737	\N	\N	\N	default	2015-07-22 18:47:22.502054	2015-07-22 18:47:22.502054
135	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f8cc7857-0adc-4e0e-b552-663b287cc109\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/135\n	\N	2015-07-22 18:47:22.614439	\N	\N	\N	default	2015-07-22 18:47:22.614599	2015-07-22 18:47:22.614599
136	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: eb672bd1-a253-44fe-b010-b36b57e60814\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/136\n	\N	2015-07-22 18:47:22.71783	\N	\N	\N	default	2015-07-22 18:47:22.71794	2015-07-22 18:47:22.71794
137	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1b477fde-6459-46bc-8037-e396c784add3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/137\n	\N	2015-07-22 18:47:22.908027	\N	\N	\N	default	2015-07-22 18:47:22.908217	2015-07-22 18:47:22.908217
138	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2e642fc8-220f-4721-8f47-77001a29fa7a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/138\n	\N	2015-07-22 18:47:23.058512	\N	\N	\N	default	2015-07-22 18:47:23.058678	2015-07-22 18:47:23.058678
139	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 44e18f5d-d7c9-4dc8-9d2c-6e2da995e591\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/139\n	\N	2015-07-22 18:47:23.103613	\N	\N	\N	default	2015-07-22 18:47:23.10375	2015-07-22 18:47:23.10375
140	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a2f08c2f-9dfe-4f2e-b07c-9f5af924e8fe\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/140\n	\N	2015-07-22 18:47:23.437194	\N	\N	\N	default	2015-07-22 18:47:23.437355	2015-07-22 18:47:23.437355
141	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: eb9aae00-a30d-40f7-8fad-0db6c7bb9dbe\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/141\n	\N	2015-07-22 18:47:23.483716	\N	\N	\N	default	2015-07-22 18:47:23.483872	2015-07-22 18:47:23.483872
142	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f713c348-c37b-4e7c-92f7-fe32fe48c4a0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/142\n	\N	2015-07-22 18:47:23.544022	\N	\N	\N	default	2015-07-22 18:47:23.544209	2015-07-22 18:47:23.544209
143	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7e847bb1-1c17-4793-9c6a-280e0ebd2387\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/143\n	\N	2015-07-22 18:47:23.655584	\N	\N	\N	default	2015-07-22 18:47:23.655728	2015-07-22 18:47:23.655728
144	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4dd0cc06-814f-4748-a234-db5f621c1d2d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/144\n	\N	2015-07-22 18:47:23.712686	\N	\N	\N	default	2015-07-22 18:47:23.712825	2015-07-22 18:47:23.712825
145	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f1219377-bde1-4568-a780-ede28cfd754e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/145\n	\N	2015-07-22 18:47:23.992514	\N	\N	\N	default	2015-07-22 18:47:23.992648	2015-07-22 18:47:23.992648
146	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 08aebeda-67ad-4fdd-871c-a64e05adac11\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/146\n	\N	2015-07-22 18:47:24.265974	\N	\N	\N	default	2015-07-22 18:47:24.266157	2015-07-22 18:47:24.266157
147	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ddab127f-b313-4a0f-bd0d-58e7e36f8dd2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/147\n	\N	2015-07-22 18:47:24.401896	\N	\N	\N	default	2015-07-22 18:47:24.402061	2015-07-22 18:47:24.402061
148	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: bc6f6df0-c1ba-419c-a0de-aaaf8b3a0f5b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/148\n	\N	2015-07-22 18:47:24.517563	\N	\N	\N	default	2015-07-22 18:47:24.517755	2015-07-22 18:47:24.517755
149	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 807bb526-41a9-48c6-b08d-3238642a3251\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/149\n	\N	2015-07-22 18:47:24.655265	\N	\N	\N	default	2015-07-22 18:47:24.655489	2015-07-22 18:47:24.655489
150	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d19aec54-30b7-4c5d-a556-5ab2691e146e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/150\n	\N	2015-07-22 18:47:25.002229	\N	\N	\N	default	2015-07-22 18:47:25.00256	2015-07-22 18:47:25.00256
151	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: eb000df5-d11a-4963-b1ba-82b7c9d5e63e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/151\n	\N	2015-07-22 18:47:25.148497	\N	\N	\N	default	2015-07-22 18:47:25.148637	2015-07-22 18:47:25.148637
152	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2bbad99b-abfd-439f-ab34-d1b1a6096dfc\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/152\n	\N	2015-07-22 18:47:25.198192	\N	\N	\N	default	2015-07-22 18:47:25.1984	2015-07-22 18:47:25.1984
153	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ba353b31-1539-4b39-9fc7-be465364199c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/153\n	\N	2015-07-22 18:47:25.257203	\N	\N	\N	default	2015-07-22 18:47:25.257332	2015-07-22 18:47:25.257332
154	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ca332ccf-4bb7-4a26-b95a-02066c19d7b8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/154\n	\N	2015-07-22 18:47:25.353589	\N	\N	\N	default	2015-07-22 18:47:25.353725	2015-07-22 18:47:25.353725
155	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 78b9dbd1-290f-4da4-808e-d861c477bcda\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/155\n	\N	2015-07-22 18:47:25.464293	\N	\N	\N	default	2015-07-22 18:47:25.464452	2015-07-22 18:47:25.464452
156	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b0d4f428-5460-48f4-8cae-55ad28728bac\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/156\n	\N	2015-07-22 18:47:25.533432	\N	\N	\N	default	2015-07-22 18:47:25.533596	2015-07-22 18:47:25.533596
157	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 743a0b0b-39fc-4653-b3eb-c600bb1b2725\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/157\n	\N	2015-07-22 18:47:25.598565	\N	\N	\N	default	2015-07-22 18:47:25.598778	2015-07-22 18:47:25.598778
158	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f239b9a6-2dbb-4a45-94b2-a1d65b832b3f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/158\n	\N	2015-07-22 18:47:25.702694	\N	\N	\N	default	2015-07-22 18:47:25.702817	2015-07-22 18:47:25.702817
159	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 55962c3a-8087-4a4e-8cde-b2dc44f60da3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/159\n	\N	2015-07-22 18:47:25.781444	\N	\N	\N	default	2015-07-22 18:47:25.781609	2015-07-22 18:47:25.781609
160	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0cc2ee89-5bd1-424f-a1b2-96ef3eac6ef6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/160\n	\N	2015-07-22 18:47:25.840456	\N	\N	\N	default	2015-07-22 18:47:25.840628	2015-07-22 18:47:25.840628
161	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f558a81c-29d9-41d6-86f6-2d2843eb0863\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/161\n	\N	2015-07-22 18:47:25.954343	\N	\N	\N	default	2015-07-22 18:47:25.954473	2015-07-22 18:47:25.954473
162	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 30ad0416-7ce6-4504-a148-40340dbed47d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/162\n	\N	2015-07-22 18:47:26.03497	\N	\N	\N	default	2015-07-22 18:47:26.035114	2015-07-22 18:47:26.035114
163	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1f3b170c-51fe-437e-8ac0-652d0d3bb3b1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/163\n	\N	2015-07-22 18:47:26.139557	\N	\N	\N	default	2015-07-22 18:47:26.139721	2015-07-22 18:47:26.139721
164	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b3cb81d2-8ad5-4faa-a4e6-fcbd4d9e1819\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/164\n	\N	2015-07-22 18:47:26.197684	\N	\N	\N	default	2015-07-22 18:47:26.197848	2015-07-22 18:47:26.197848
165	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d13d7a6c-8b82-4b64-ad9a-fbd7771a555f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/165\n	\N	2015-07-22 18:47:26.252226	\N	\N	\N	default	2015-07-22 18:47:26.252358	2015-07-22 18:47:26.252358
166	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: dbb9b096-b388-4142-8851-88d232205ef3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/166\n	\N	2015-07-22 18:47:26.412161	\N	\N	\N	default	2015-07-22 18:47:26.412287	2015-07-22 18:47:26.412287
167	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a2c59f02-9601-48cd-a030-eaf2081adcbc\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/167\n	\N	2015-07-22 18:47:26.553056	\N	\N	\N	default	2015-07-22 18:47:26.553282	2015-07-22 18:47:26.553282
168	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 020295bf-d3c1-43cf-bab7-0393fa5ed8c5\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/168\n	\N	2015-07-22 18:47:26.6444	\N	\N	\N	default	2015-07-22 18:47:26.644586	2015-07-22 18:47:26.644586
169	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: beb1108e-b653-417b-b740-9a0e019f5a5b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/169\n	\N	2015-07-22 18:47:26.715585	\N	\N	\N	default	2015-07-22 18:47:26.71579	2015-07-22 18:47:26.71579
170	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9723ed42-4380-4f8c-9073-cf211cda6be6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/170\n	\N	2015-07-22 18:47:26.780067	\N	\N	\N	default	2015-07-22 18:47:26.780212	2015-07-22 18:47:26.780212
171	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e2576452-6651-4393-b991-d71a40d83582\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/171\n	\N	2015-07-22 18:47:26.834397	\N	\N	\N	default	2015-07-22 18:47:26.834569	2015-07-22 18:47:26.834569
172	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b706a926-d95f-4673-b438-5b964deaaec2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/172\n	\N	2015-07-22 18:47:27.123745	\N	\N	\N	default	2015-07-22 18:47:27.123874	2015-07-22 18:47:27.123874
173	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: dfae26d5-474c-4eab-b4ad-e5bc0674bdff\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/173\n	\N	2015-07-22 18:47:27.190777	\N	\N	\N	default	2015-07-22 18:47:27.190983	2015-07-22 18:47:27.190983
174	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9078017d-1f03-4e29-aecc-6a0f0eb85c6d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/174\n	\N	2015-07-22 18:47:27.281498	\N	\N	\N	default	2015-07-22 18:47:27.281629	2015-07-22 18:47:27.281629
175	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 73490902-cb7b-48db-9814-705580848b32\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/175\n	\N	2015-07-22 18:47:27.337102	\N	\N	\N	default	2015-07-22 18:47:27.337367	2015-07-22 18:47:27.337367
176	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1018d9e3-cc86-4429-a654-8a5cefc3574f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/176\n	\N	2015-07-22 18:47:27.659061	\N	\N	\N	default	2015-07-22 18:47:27.659195	2015-07-22 18:47:27.659195
177	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5a3d64ee-8187-47d3-901d-62202af12626\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/177\n	\N	2015-07-22 18:47:27.983953	\N	\N	\N	default	2015-07-22 18:47:27.984082	2015-07-22 18:47:27.984082
178	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ba861b36-1db8-4b85-8864-7b87c12ebbd6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/178\n	\N	2015-07-22 18:47:28.043075	\N	\N	\N	default	2015-07-22 18:47:28.043612	2015-07-22 18:47:28.043612
179	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3589b6c6-3d81-4139-ae2e-ce31f1b97dcd\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/179\n	\N	2015-07-22 18:47:28.13545	\N	\N	\N	default	2015-07-22 18:47:28.135607	2015-07-22 18:47:28.135607
180	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 35e325ab-d5bc-4789-bee3-60dde42e3b5b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/180\n	\N	2015-07-22 18:47:28.505082	\N	\N	\N	default	2015-07-22 18:47:28.505214	2015-07-22 18:47:28.505214
181	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c2febf5b-14d0-438c-abde-414e5cfe42f9\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/181\n	\N	2015-07-22 18:47:28.558662	\N	\N	\N	default	2015-07-22 18:47:28.55879	2015-07-22 18:47:28.55879
182	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9ef6aa98-b097-4000-81a1-74ca22d786c6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/182\n	\N	2015-07-22 18:47:28.605802	\N	\N	\N	default	2015-07-22 18:47:28.606245	2015-07-22 18:47:28.606245
183	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e9109bb4-b7ef-4dd3-9067-48d395c05b0b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/183\n	\N	2015-07-22 18:47:28.717591	\N	\N	\N	default	2015-07-22 18:47:28.717721	2015-07-22 18:47:28.717721
184	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 73867bf2-07fe-467b-a7cf-1b7d65b4d175\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/184\n	\N	2015-07-22 18:47:28.774251	\N	\N	\N	default	2015-07-22 18:47:28.774379	2015-07-22 18:47:28.774379
185	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 01987f85-1538-4e26-abd5-ddb031e75711\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/185\n	\N	2015-07-22 18:47:29.354281	\N	\N	\N	default	2015-07-22 18:47:29.354423	2015-07-22 18:47:29.354423
186	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a1ff3985-f86d-495b-a82a-52aaa6b86b1b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/186\n	\N	2015-07-22 18:47:29.704204	\N	\N	\N	default	2015-07-22 18:47:29.70438	2015-07-22 18:47:29.70438
187	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 71b04265-c43b-4ddb-971a-e3b781b7c4f3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/187\n	\N	2015-07-22 18:47:29.843477	\N	\N	\N	default	2015-07-22 18:47:29.84376	2015-07-22 18:47:29.84376
188	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9f028ca3-22dd-45c8-807a-8233e01ef60b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/188\n	\N	2015-07-22 18:47:29.941273	\N	\N	\N	default	2015-07-22 18:47:29.941489	2015-07-22 18:47:29.941489
189	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1c115e4b-19b9-4e95-8758-530921062d0f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/189\n	\N	2015-07-22 18:47:30.029601	\N	\N	\N	default	2015-07-22 18:47:30.029706	2015-07-22 18:47:30.029706
190	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 383a63b8-b29a-4d4c-a665-ac8ffe6e11c4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/190\n	\N	2015-07-22 18:47:30.093894	\N	\N	\N	default	2015-07-22 18:47:30.094029	2015-07-22 18:47:30.094029
191	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 91357a8f-d2e0-43ca-8740-719ff687fe88\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/191\n	\N	2015-07-22 18:47:30.150118	\N	\N	\N	default	2015-07-22 18:47:30.150231	2015-07-22 18:47:30.150231
192	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 56aa2865-ca04-4134-85f4-8479d506b826\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/192\n	\N	2015-07-22 18:47:30.206123	\N	\N	\N	default	2015-07-22 18:47:30.206362	2015-07-22 18:47:30.206362
193	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b6f0354f-e104-44d3-8a61-d3f3a3f59042\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/193\n	\N	2015-07-22 18:47:30.261038	\N	\N	\N	default	2015-07-22 18:47:30.26117	2015-07-22 18:47:30.26117
194	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 60e7eef5-6e30-4126-a1dc-77999dc89b46\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/194\n	\N	2015-07-22 18:47:30.350366	\N	\N	\N	default	2015-07-22 18:47:30.350477	2015-07-22 18:47:30.350477
195	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9355c04a-ae93-449e-8fe9-33c535ca0dbb\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/195\n	\N	2015-07-22 18:47:30.46809	\N	\N	\N	default	2015-07-22 18:47:30.468202	2015-07-22 18:47:30.468202
196	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 68992ac5-0a96-4514-82d4-8de6fdbd7f1a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/196\n	\N	2015-07-22 18:47:30.515085	\N	\N	\N	default	2015-07-22 18:47:30.515228	2015-07-22 18:47:30.515228
197	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e497754b-3c4c-4021-a7bf-1d4219aaf1e1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/197\n	\N	2015-07-22 18:47:30.70456	\N	\N	\N	default	2015-07-22 18:47:30.704754	2015-07-22 18:47:30.704754
198	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a846ae9b-9ab3-4227-a206-cc688652971f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/198\n	\N	2015-07-22 18:47:30.749978	\N	\N	\N	default	2015-07-22 18:47:30.750096	2015-07-22 18:47:30.750096
199	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ad54c665-4b45-44d1-af06-03be481d5777\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/199\n	\N	2015-07-22 18:47:30.851816	\N	\N	\N	default	2015-07-22 18:47:30.851974	2015-07-22 18:47:30.851974
200	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5eb53ddc-47b6-44b7-9896-e01ddf620348\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/200\n	\N	2015-07-22 18:47:30.899298	\N	\N	\N	default	2015-07-22 18:47:30.899457	2015-07-22 18:47:30.899457
201	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 22084462-ac86-4ff3-9cc4-9feeed71aa55\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/201\n	\N	2015-07-22 18:47:31.033717	\N	\N	\N	default	2015-07-22 18:47:31.033941	2015-07-22 18:47:31.033941
202	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f372f94f-dac7-4cbb-9af7-4160d403bf04\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/202\n	\N	2015-07-22 18:47:31.238875	\N	\N	\N	default	2015-07-22 18:47:31.23905	2015-07-22 18:47:31.23905
203	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 369eb2e0-ec2f-43a6-baed-a93e68b7e63f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/203\n	\N	2015-07-22 18:47:31.291864	\N	\N	\N	default	2015-07-22 18:47:31.292007	2015-07-22 18:47:31.292007
204	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3a5ec201-1615-4c1c-9352-b8455969b199\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/204\n	\N	2015-07-22 18:47:31.339348	\N	\N	\N	default	2015-07-22 18:47:31.339537	2015-07-22 18:47:31.339537
205	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e7ec7463-eaaa-43f4-a9e5-b43be8b7bb06\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/205\n	\N	2015-07-22 18:47:31.392716	\N	\N	\N	default	2015-07-22 18:47:31.392851	2015-07-22 18:47:31.392851
206	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e371fab5-2c69-4c93-bf5f-832eee45c957\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/206\n	\N	2015-07-22 18:47:31.522886	\N	\N	\N	default	2015-07-22 18:47:31.523071	2015-07-22 18:47:31.523071
207	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9fdb5b76-0f17-44cb-a016-9b74313bc7bf\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/207\n	\N	2015-07-22 18:47:31.586156	\N	\N	\N	default	2015-07-22 18:47:31.586297	2015-07-22 18:47:31.586297
208	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c9fa7014-38d0-4570-8bac-8cb39194959c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/208\n	\N	2015-07-22 18:47:31.673786	\N	\N	\N	default	2015-07-22 18:47:31.673924	2015-07-22 18:47:31.673924
209	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4669ab98-1aa9-466b-9982-0df0a1dd62bd\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/209\n	\N	2015-07-22 18:47:31.730499	\N	\N	\N	default	2015-07-22 18:47:31.730775	2015-07-22 18:47:31.730775
210	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ae4e2e5d-a046-49e2-a9ce-1f6a8d9f4c36\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/210\n	\N	2015-07-22 18:47:31.977648	\N	\N	\N	default	2015-07-22 18:47:31.977864	2015-07-22 18:47:31.977864
211	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 374d419c-fa08-4bc6-bc5f-dc6f890d7ea1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/211\n	\N	2015-07-22 18:47:32.064099	\N	\N	\N	default	2015-07-22 18:47:32.064292	2015-07-22 18:47:32.064292
212	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 57d3e7cd-ddb1-4e76-95f8-553f1e9f0d5d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/212\n	\N	2015-07-22 18:47:32.194624	\N	\N	\N	default	2015-07-22 18:47:32.194785	2015-07-22 18:47:32.194785
213	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a712d3a4-02a5-4a22-a8db-74e8ef2fba74\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/213\n	\N	2015-07-22 18:47:32.314332	\N	\N	\N	default	2015-07-22 18:47:32.314528	2015-07-22 18:47:32.314528
214	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 540b9708-623c-469e-9b81-dd6d2049f57b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/214\n	\N	2015-07-22 18:47:32.375527	\N	\N	\N	default	2015-07-22 18:47:32.375696	2015-07-22 18:47:32.375696
215	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f1359a3b-7dc1-4c47-ae5d-50d8d51b3004\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/215\n	\N	2015-07-22 18:47:32.437333	\N	\N	\N	default	2015-07-22 18:47:32.437581	2015-07-22 18:47:32.437581
216	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: cdc1daae-9335-4655-b7cb-9f6c61c9c1be\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/216\n	\N	2015-07-22 18:47:32.503776	\N	\N	\N	default	2015-07-22 18:47:32.50404	2015-07-22 18:47:32.50404
217	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a94aab40-7f05-41f6-b351-b502621411bc\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/217\n	\N	2015-07-22 18:47:32.60378	\N	\N	\N	default	2015-07-22 18:47:32.604039	2015-07-22 18:47:32.604039
218	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 82105cb0-57b0-41f8-b421-ea5c578cf16d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/218\n	\N	2015-07-22 18:47:32.86749	\N	\N	\N	default	2015-07-22 18:47:32.867618	2015-07-22 18:47:32.867618
219	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: bfb8ccd7-eb9b-4938-8757-0d18a38d83b1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/219\n	\N	2015-07-22 18:47:32.938255	\N	\N	\N	default	2015-07-22 18:47:32.938414	2015-07-22 18:47:32.938414
220	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 077a2686-5598-4fb7-abc1-e8659d175f24\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/220\n	\N	2015-07-22 18:47:32.997083	\N	\N	\N	default	2015-07-22 18:47:32.997255	2015-07-22 18:47:32.997255
221	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a8f1f6ef-a679-4187-91ec-4a5ff2fd43b7\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/221\n	\N	2015-07-22 18:47:33.173169	\N	\N	\N	default	2015-07-22 18:47:33.173328	2015-07-22 18:47:33.173328
222	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a5fba91e-7830-4e73-8949-4e96b9df50e3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/222\n	\N	2015-07-22 18:47:33.227009	\N	\N	\N	default	2015-07-22 18:47:33.227181	2015-07-22 18:47:33.227181
223	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: aa3cf697-6cd9-4610-b94b-0925fd12c1f0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/223\n	\N	2015-07-22 18:47:33.374089	\N	\N	\N	default	2015-07-22 18:47:33.374403	2015-07-22 18:47:33.374403
224	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5f12f12b-2d78-46df-ad5c-b9367ad5a30e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/224\n	\N	2015-07-22 18:47:33.48434	\N	\N	\N	default	2015-07-22 18:47:33.484474	2015-07-22 18:47:33.484474
225	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 33c12129-a1dc-4a57-9a3c-ac034cf363ff\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/225\n	\N	2015-07-22 18:47:33.54879	\N	\N	\N	default	2015-07-22 18:47:33.548964	2015-07-22 18:47:33.548964
226	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8f80fedd-f5d5-4a87-b686-b241ec991d2c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/226\n	\N	2015-07-22 18:47:33.663763	\N	\N	\N	default	2015-07-22 18:47:33.663946	2015-07-22 18:47:33.663946
227	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f6e75464-9350-4690-9aa8-3122283d1e00\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/227\n	\N	2015-07-22 18:47:33.717037	\N	\N	\N	default	2015-07-22 18:47:33.717169	2015-07-22 18:47:33.717169
228	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 87dc2090-9b06-4f87-b8cc-d0cd2b0c2764\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/228\n	\N	2015-07-22 18:47:33.765007	\N	\N	\N	default	2015-07-22 18:47:33.765133	2015-07-22 18:47:33.765133
229	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7f1b2671-29c2-4888-806b-ac61ca271eac\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/229\n	\N	2015-07-22 18:47:33.926452	\N	\N	\N	default	2015-07-22 18:47:33.926584	2015-07-22 18:47:33.926584
230	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 025b2cbb-0881-4c85-a6ac-88dba63db00c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/230\n	\N	2015-07-22 18:47:33.988348	\N	\N	\N	default	2015-07-22 18:47:33.988622	2015-07-22 18:47:33.988622
231	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 37b76c69-5f01-4812-bf09-3c782ffbd95c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/231\n	\N	2015-07-22 18:47:34.082337	\N	\N	\N	default	2015-07-22 18:47:34.082469	2015-07-22 18:47:34.082469
232	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5b9c13d9-2362-4fa0-8510-dd812cd57689\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/232\n	\N	2015-07-22 18:47:34.278336	\N	\N	\N	default	2015-07-22 18:47:34.278513	2015-07-22 18:47:34.278513
233	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d4b18a4d-d26c-4d6f-9d20-a7978c69f4a6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/233\n	\N	2015-07-22 18:47:34.525881	\N	\N	\N	default	2015-07-22 18:47:34.526065	2015-07-22 18:47:34.526065
234	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 01a788df-0f42-405b-b016-f5b31e52aedf\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/234\n	\N	2015-07-22 18:47:34.657344	\N	\N	\N	default	2015-07-22 18:47:34.657493	2015-07-22 18:47:34.657493
235	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0cf10a55-15a1-4cd8-a569-370c5fa0ce80\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/235\n	\N	2015-07-22 18:47:35.17494	\N	\N	\N	default	2015-07-22 18:47:35.175096	2015-07-22 18:47:35.175096
236	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a09982e0-803f-480f-9630-a175251dafb5\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/236\n	\N	2015-07-22 18:47:35.288798	\N	\N	\N	default	2015-07-22 18:47:35.289001	2015-07-22 18:47:35.289001
237	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e55a2362-9964-40de-bd18-5787954509ae\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/237\n	\N	2015-07-22 18:47:35.359524	\N	\N	\N	default	2015-07-22 18:47:35.359723	2015-07-22 18:47:35.359723
238	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 40d07581-edb2-454e-8660-071af2f30388\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/238\n	\N	2015-07-22 18:47:35.472868	\N	\N	\N	default	2015-07-22 18:47:35.473012	2015-07-22 18:47:35.473012
239	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 26f915ba-3ad2-490e-8e90-0996fe9c4da1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/239\n	\N	2015-07-22 18:47:35.58387	\N	\N	\N	default	2015-07-22 18:47:35.584039	2015-07-22 18:47:35.584039
240	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 251ad2b4-4385-4a42-988c-b623740ffeeb\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/240\n	\N	2015-07-22 18:47:35.664901	\N	\N	\N	default	2015-07-22 18:47:35.665032	2015-07-22 18:47:35.665032
241	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b6433180-ff0b-4d78-852d-041ac90768a4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/241\n	\N	2015-07-22 18:47:35.83095	\N	\N	\N	default	2015-07-22 18:47:35.831179	2015-07-22 18:47:35.831179
242	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0a981d0f-8160-4f3d-b060-c78c259be898\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/242\n	\N	2015-07-22 18:47:35.906905	\N	\N	\N	default	2015-07-22 18:47:35.907102	2015-07-22 18:47:35.907102
243	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 799ac69e-fc90-4b36-98f4-a7d17dd6f6f2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/243\n	\N	2015-07-22 18:47:35.973313	\N	\N	\N	default	2015-07-22 18:47:35.97357	2015-07-22 18:47:35.97357
244	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c8d0c597-735c-4fe8-b2eb-381c170a88b8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/244\n	\N	2015-07-22 18:47:36.053298	\N	\N	\N	default	2015-07-22 18:47:36.053542	2015-07-22 18:47:36.053542
245	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 78e87edc-a476-4ae7-9e68-dd5deff52130\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/245\n	\N	2015-07-22 18:47:36.128674	\N	\N	\N	default	2015-07-22 18:47:36.128846	2015-07-22 18:47:36.128846
246	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: eaf01a9a-551c-44fd-a234-09c451dcbf32\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/246\n	\N	2015-07-22 18:47:36.177434	\N	\N	\N	default	2015-07-22 18:47:36.177557	2015-07-22 18:47:36.177557
247	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0ece4863-e9a8-4280-bc83-d1923bdb889f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/247\n	\N	2015-07-22 18:47:36.224973	\N	\N	\N	default	2015-07-22 18:47:36.225149	2015-07-22 18:47:36.225149
248	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0d962819-87eb-417c-8412-f7a87f4cd737\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/248\n	\N	2015-07-22 18:47:36.279566	\N	\N	\N	default	2015-07-22 18:47:36.279733	2015-07-22 18:47:36.279733
249	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b23aab6a-1989-422b-b2c8-997d394527e6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/249\n	\N	2015-07-22 18:47:36.380833	\N	\N	\N	default	2015-07-22 18:47:36.381042	2015-07-22 18:47:36.381042
250	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c760394b-8a11-41c3-a1d2-c24a8a40750a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/250\n	\N	2015-07-22 18:47:36.463159	\N	\N	\N	default	2015-07-22 18:47:36.46333	2015-07-22 18:47:36.46333
251	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3f466945-05e5-47f4-8e2b-4b00c489d29b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/251\n	\N	2015-07-22 18:47:36.556335	\N	\N	\N	default	2015-07-22 18:47:36.556505	2015-07-22 18:47:36.556505
252	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0c7e09bf-2946-43d3-8a35-e32a885e6bb3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/252\n	\N	2015-07-22 18:47:36.658234	\N	\N	\N	default	2015-07-22 18:47:36.658452	2015-07-22 18:47:36.658452
253	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e6635952-1a6c-437e-b2b6-51b03014f777\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/253\n	\N	2015-07-22 18:47:36.722734	\N	\N	\N	default	2015-07-22 18:47:36.72289	2015-07-22 18:47:36.72289
254	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1c23652d-ef8a-419e-8c30-814fdd21d015\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/254\n	\N	2015-07-22 18:47:36.813671	\N	\N	\N	default	2015-07-22 18:47:36.8138	2015-07-22 18:47:36.8138
255	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 22470677-7394-4065-b894-a8f91d882a17\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/255\n	\N	2015-07-22 18:47:36.950391	\N	\N	\N	default	2015-07-22 18:47:36.950573	2015-07-22 18:47:36.950573
\.


--
-- Name: delayed_jobs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('delayed_jobs_id_seq', 255, true);


--
-- Data for Name: disease_ontology_mirrors; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY disease_ontology_mirrors (id, doid, name) FROM stdin;
\.


--
-- Name: disease_ontology_mirrors_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('disease_ontology_mirrors_id_seq', 1, false);


--
-- Data for Name: diseases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY diseases (id, doid, name, created_at, updated_at) FROM stdin;
6	12603	Acute Leukemia	2015-07-22 18:47:14.827998	2015-07-22 18:47:14.827998
7	1909	Melanoma	2015-07-22 18:47:14.905447	2015-07-22 18:47:14.905447
13	769	Neuroblastoma	2015-07-22 18:47:16.341301	2015-07-22 18:47:16.341301
24	1612	Breast Cancer	2015-07-22 18:47:26.143388	2015-07-22 18:47:26.143388
29	4947	Cholangiocarcinoma	2015-07-22 18:47:29.70768	2015-07-22 18:47:29.70768
35	3069	Astrocytoma	2015-07-22 18:47:32.378892	2015-07-22 18:47:32.378892
1	10747	Lymphoid Leukemia	2015-07-22 18:47:14.403699	2015-07-22 18:49:30.221043
2	9253	Gastrointestinal Stromal Tumor	2015-07-22 18:47:14.482693	2015-07-22 18:49:30.426641
3	9119	Acute Myeloid Leukemia	2015-07-22 18:47:14.605947	2015-07-22 18:49:30.673605
4	8552	Chronic Myeloid Leukemia	2015-07-22 18:47:14.707677	2015-07-22 18:49:30.836312
5	4960	Bone Marrow Cancer	2015-07-22 18:47:14.78661	2015-07-22 18:49:31.080235
8	3908	Non-small Cell Lung Carcinoma	2015-07-22 18:47:15.283611	2015-07-22 18:49:31.730923
9	8997	Polycythemia Vera	2015-07-22 18:47:15.373552	2015-07-22 18:49:31.930513
10	9119	Acute Myeloid Leukemia	2015-07-22 18:47:15.752389	2015-07-22 18:49:32.101169
11	9256	Colorectal Cancer	2015-07-22 18:47:15.81718	2015-07-22 18:49:32.276632
12	0050905	Inflammatory Myofibroblastic Tumor	2015-07-22 18:47:16.030247	2015-07-22 18:49:32.523989
14	9253	Gastrointestinal Stromal Tumor	2015-07-22 18:47:17.037569	2015-07-22 18:49:32.848685
15	3973	Thyroid Medullary Carcinoma	2015-07-22 18:47:17.96067	2015-07-22 18:49:33.033246
16	1781	Thyroid Cancer	2015-07-22 18:47:18.206871	2015-07-22 18:49:33.290211
17	9256	Colorectal Cancer	2015-07-22 18:47:18.655797	2015-07-22 18:49:33.464142
18	4007	Bladder Carcinoma	2015-07-22 18:47:21.656427	2015-07-22 18:49:33.692925
19	3068	Glioblastoma Multiforme	2015-07-22 18:47:23.106916	2015-07-22 18:49:33.93338
20	2394	Ovarian Cancer	2015-07-22 18:47:25.006475	2015-07-22 18:49:34.199172
21	10534	Stomach Cancer	2015-07-22 18:47:25.467231	2015-07-22 18:49:34.459659
22	1612	Breast Cancer	2015-07-22 18:47:25.602106	2015-07-22 18:49:34.805711
23	1324	Lung Cancer	2015-07-22 18:47:25.956999	2015-07-22 18:49:35.079238
25	3908	Non-small Cell Lung Carcinoma	2015-07-22 18:47:26.556067	2015-07-22 18:49:35.604575
26	3910	Lung Adenocarcinoma	2015-07-22 18:47:27.340358	2015-07-22 18:49:35.775569
27	715	T-cell Leukemia	2015-07-22 18:47:27.986565	2015-07-22 18:49:35.943706
28	0060075	Estrogen-receptor Positive Breast Cancer	2015-07-22 18:47:28.046466	2015-07-22 18:49:36.070046
30	3909	Bronchogenic Lung Adenocarcinoma	2015-07-22 18:47:30.031797	2015-07-22 18:49:36.562729
31	14145	Malignant Anus Melanoma	2015-07-22 18:47:30.096808	2015-07-22 18:49:36.72717
32	9119	Acute Myeloid Leukemia	2015-07-22 18:47:30.707191	2015-07-22 18:49:36.863491
33	6477	Invasive Bladder Transitional Cell Carcinoma	2015-07-22 18:47:30.902122	2015-07-22 18:49:37.048379
34	0050908	Myelodysplastic Syndrome	2015-07-22 18:47:31.341982	2015-07-22 18:49:37.176009
36	3068	Glioblastoma Multiforme	2015-07-22 18:47:32.440725	2015-07-22 18:49:37.659974
37	5520	Head And Neck Squamous Cell Carcinoma	2015-07-22 18:47:32.87053	2015-07-22 18:49:37.81755
38	0050746	Mantle Cell Lymphoma	2015-07-22 18:47:32.941504	2015-07-22 18:49:37.935779
39	5635	Gastric Adenosquamous Carcinoma	2015-07-22 18:47:33.229954	2015-07-22 18:49:38.049009
40	0050908	Myelodysplastic Syndrome	2015-07-22 18:47:33.768972	2015-07-22 18:49:38.16625
41	9538	Multiple Myeloma	2015-07-22 18:47:33.929493	2015-07-22 18:49:38.408495
42	9119	Acute Myeloid Leukemia	2015-07-22 18:47:34.035672	2015-07-22 18:49:38.609805
43	9952	Acute Lymphocytic Leukemia	2015-07-22 18:47:35.362747	2015-07-22 18:49:38.795196
44	5015	Hepatocellular Fibrolamellar Carcinoma	2015-07-22 18:47:35.475331	2015-07-22 18:49:38.929489
45	0050745	Diffuse Large B-cell Lymphoma	2015-07-22 18:47:35.588411	2015-07-22 18:49:39.072376
46	10283	Prostate Cancer	2015-07-22 18:47:35.913218	2015-07-22 18:49:39.280374
47	4001	Ovarian Carcinoma	2015-07-22 18:47:36.179866	2015-07-22 18:49:39.529072
48	4467	Renal Clear Cell Carcinoma	2015-07-22 18:47:36.28299	2015-07-22 18:49:39.68909
49	0050908	Myelodysplastic Syndrome	2015-07-22 18:47:36.661276	2015-07-22 18:49:39.838277
50	3369	Peripheral Primitive Neuroectodermal Tumor	2015-07-22 18:47:36.953348	2015-07-22 18:49:40.026274
\.


--
-- Name: diseases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('diseases_id_seq', 50, true);


--
-- Data for Name: drugs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs (id, name, pubchem_id, created_at, updated_at) FROM stdin;
1	N/A	\N	2015-07-22 18:47:14.445469	2015-07-22 18:47:14.445469
2	Daunorubicin	\N	2015-07-22 18:47:14.970924	2015-07-22 18:47:14.970924
3	AZD6244	\N	2015-07-22 18:47:15.023042	2015-07-22 18:47:15.023042
4	Vemurafenib	\N	2015-07-22 18:47:15.134914	2015-07-22 18:47:15.134914
5	Imatinib	\N	2015-07-22 18:47:15.183988	2015-07-22 18:47:15.183988
6	Sorafenib	\N	2015-07-22 18:47:15.309907	2015-07-22 18:47:15.309907
7	Idarubicin	\N	2015-07-22 18:47:15.3475	2015-07-22 18:47:15.3475
8	Pegylated IFN-α–2a	\N	2015-07-22 18:47:15.403962	2015-07-22 18:47:15.403962
9	TG101348	\N	2015-07-22 18:47:15.461093	2015-07-22 18:47:15.461093
10	17-AAG	\N	2015-07-22 18:47:15.520691	2015-07-22 18:47:15.520691
11	Temozolomide 	\N	2015-07-22 18:47:15.575849	2015-07-22 18:47:15.575849
12	Crizotinib	\N	2015-07-22 18:47:16.052835	2015-07-22 18:47:16.052835
13	Nilotinib	\N	2015-07-22 18:47:16.177657	2015-07-22 18:47:16.177657
14	Gefitinib	\N	2015-07-22 18:47:16.244055	2015-07-22 18:47:16.244055
15	Erlotinib	\N	2015-07-22 18:47:16.25202	2015-07-22 18:47:16.25202
16	Cetuximab	\N	2015-07-22 18:47:16.315212	2015-07-22 18:47:16.315212
17	CH5424802	\N	2015-07-22 18:47:16.361742	2015-07-22 18:47:16.361742
18	TAE684	\N	2015-07-22 18:47:16.448475	2015-07-22 18:47:16.448475
19	Trametinib	\N	2015-07-22 18:47:16.523747	2015-07-22 18:47:16.523747
20	Dasatinib	\N	2015-07-22 18:47:16.565283	2015-07-22 18:47:16.565283
21	Crenolanib	\N	2015-07-22 18:47:16.630145	2015-07-22 18:47:16.630145
22	Dabrafenib	\N	2015-07-22 18:47:17.90798	2015-07-22 18:47:17.90798
23	Motesanib	\N	2015-07-22 18:47:18.029442	2015-07-22 18:47:18.029442
24	AZD1480	\N	2015-07-22 18:47:18.137294	2015-07-22 18:47:18.137294
25	Dacarbazine	\N	2015-07-22 18:47:18.358473	2015-07-22 18:47:18.358473
26	Temozolomide	\N	2015-07-22 18:47:18.365616	2015-07-22 18:47:18.365616
27	Regorafenib	\N	2015-07-22 18:47:18.419468	2015-07-22 18:47:18.419468
28	Panitumumab	\N	2015-07-22 18:47:18.633617	2015-07-22 18:47:18.633617
29	PD0325901	\N	2015-07-22 18:47:19.020122	2015-07-22 18:47:19.020122
30	PLX4720	\N	2015-07-22 18:47:19.026317	2015-07-22 18:47:19.026317
31	Nutlin-3	\N	2015-07-22 18:47:19.081812	2015-07-22 18:47:19.081812
32	Capecitabine	\N	2015-07-22 18:47:19.136609	2015-07-22 18:47:19.136609
33	Bevacizumab	\N	2015-07-22 18:47:19.146997	2015-07-22 18:47:19.146997
34	ATRA	\N	2015-07-22 18:47:20.733639	2015-07-22 18:47:20.733639
35	SU5614	\N	2015-07-22 18:47:21.027208	2015-07-22 18:47:21.027208
36	PD173074	\N	2015-07-22 18:47:21.691041	2015-07-22 18:47:21.691041
37	Valproic Acid	\N	2015-07-22 18:47:21.917679	2015-07-22 18:47:21.917679
38	NSC348884	\N	2015-07-22 18:47:21.984329	2015-07-22 18:47:21.984329
39	Induction Therapy	\N	2015-07-22 18:47:22.210286	2015-07-22 18:47:22.210286
40	Rapamycin	\N	2015-07-22 18:47:22.280618	2015-07-22 18:47:22.280618
41	Everolimus	\N	2015-07-22 18:47:22.360737	2015-07-22 18:47:22.360737
42	Olaparib	\N	2015-07-22 18:47:25.03425	2015-07-22 18:47:25.03425
43	MK-2206	\N	2015-07-22 18:47:26.17122	2015-07-22 18:47:26.17122
44	Dacomitinib	\N	2015-07-22 18:47:26.745075	2015-07-22 18:47:26.745075
45	Lapatinib	\N	2015-07-22 18:47:26.805305	2015-07-22 18:47:26.805305
46	Hormone Therapy	\N	2015-07-22 18:47:26.863791	2015-07-22 18:47:26.863791
47	Arsenic Trioxide	\N	2015-07-22 18:47:27.825754	2015-07-22 18:47:27.825754
48	Palbociclib (PD-0332991)	\N	2015-07-22 18:47:28.014777	2015-07-22 18:47:28.014777
49	BYL719	\N	2015-07-22 18:47:28.075458	2015-07-22 18:47:28.075458
50	Premetrexed	\N	2015-07-22 18:47:28.74631	2015-07-22 18:47:28.74631
51	Stauroporine	\N	2015-07-22 18:47:28.798657	2015-07-22 18:47:28.798657
52	Neratinib	\N	2015-07-22 18:47:28.834854	2015-07-22 18:47:28.834854
53	Tamoxifen	\N	2015-07-22 18:47:29.385261	2015-07-22 18:47:29.385261
54	Fulvestrant	\N	2015-07-22 18:47:29.391482	2015-07-22 18:47:29.391482
55	Ponatinib	\N	2015-07-22 18:47:29.733116	2015-07-22 18:47:29.733116
56	Pazopanib	\N	2015-07-22 18:47:29.800391	2015-07-22 18:47:29.800391
57	CEP701	\N	2015-07-22 18:47:29.874776	2015-07-22 18:47:29.874776
58	AG1296	\N	2015-07-22 18:47:29.966971	2015-07-22 18:47:29.966971
59	Bortezomib	\N	2015-07-22 18:47:30.056989	2015-07-22 18:47:30.056989
60	Fasudil	\N	2015-07-22 18:47:30.064856	2015-07-22 18:47:30.064856
61	SU11248	\N	2015-07-22 18:47:30.229414	2015-07-22 18:47:30.229414
62	NVP-BEZ235	\N	2015-07-22 18:47:30.286205	2015-07-22 18:47:30.286205
63	ARRY-142886	\N	2015-07-22 18:47:30.293707	2015-07-22 18:47:30.293707
64	Carmustine	\N	2015-07-22 18:47:30.410838	2015-07-22 18:47:30.410838
65	O(6)-benzylguanine	\N	2015-07-22 18:47:30.445124	2015-07-22 18:47:30.445124
66	CH5132799	\N	2015-07-22 18:47:30.493859	2015-07-22 18:47:30.493859
67	MTOR Inhibitors	\N	2015-07-22 18:47:30.770928	2015-07-22 18:47:30.770928
68	Doxorubicin	\N	2015-07-22 18:47:30.87519	2015-07-22 18:47:30.87519
69	Palbociclib	\N	2015-07-22 18:47:32.225607	2015-07-22 18:47:32.225607
70	GW-2580	\N	2015-07-22 18:47:35.396718	2015-07-22 18:47:35.396718
71	Allogeneic Hemopoietic Stem-cell Transplantation	\N	2015-07-22 18:47:35.869276	2015-07-22 18:47:35.869276
72	OSI-906 (Linsitinib)	\N	2015-07-22 18:47:35.941871	2015-07-22 18:47:35.941871
73	Docetaxel	\N	2015-07-22 18:47:36.089689	2015-07-22 18:47:36.089689
74	Cabazitaxel	\N	2015-07-22 18:47:36.096476	2015-07-22 18:47:36.096476
\.


--
-- Data for Name: genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes (id, entrez_id, name, description, official_name, created_at, updated_at, clinical_description, deleted, deleted_at) FROM stdin;
1	238	ALK	ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines.	anaplastic lymphoma receptor tyrosine kinase	2015-07-22 18:46:55.37169	2015-07-22 18:46:55.37169	\N	f	\N
2	207	AKT1	AKT1, also referred to as protein kinase B, is a known oncogene. AKT activation relies on the PI3K pathway, and is recognized as a critical node in the pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been shown to result in activation of the protein. Mutations in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors in vitro.	v-akt murine thymoma viral oncogene homolog 1	2015-07-22 18:46:55.739924	2015-07-22 18:46:55.739924	\N	f	\N
3	369	ARAF	ARAF has recently become increasingly considered for its oncogenic potential. Its potential as a target for informing clinical action was demonstrated by a single case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated with sorafenib, acheived near-complete clinical remission. This finding has brought new focus on ARAF as a marker that should be assayed for in cancer treatment.	A-Raf proto-oncogene, serine/threonine kinase	2015-07-22 18:46:55.982708	2015-07-22 18:46:55.982708	\N	f	\N
4	25	ABL1	ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein that has become a signature of chronic myeloid leukemia (CML). Cells harboring this fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook of the disease. However, additional mutations in ABL1 have been shown to confer resistance to imatinib. In these resistance cases, second-generation tyrosine kinase inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are currently undergoing clinical trials for treating acquired resistance in CML.	ABL proto-oncogene 1, non-receptor tyrosine kinase	2015-07-22 18:46:56.262899	2015-07-22 18:46:56.262899	\N	f	\N
5	673	BRAF	BRAF mutations are found to be recurrent in many cancer types. Of these, the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E has been determined to be an activating mutation, and cells that harbor it, along with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations have also been correlated with poor prognosis in many cancer types, although there is at least one study that questions this conclusion in papillary thyroid cancer.	B-Raf proto-oncogene, serine/threonine kinase	2015-07-22 18:46:56.563464	2015-07-22 18:46:56.563464	\N	f	\N
6	672	BRCA1	BRCA1 mutations in the germline have become a hallmark for hereditary breast and ovarian cancers. Variants that have been demonstrated to reduce the function of the protein have been shown to increase risk for these cancers, as well as prostate and pancreatic cancer. These findings have been the impetus for the increased popularity of genetic testing of healthy indivudals to assess risk.	breast cancer 1, early onset	2015-07-22 18:46:56.833662	2015-07-22 18:46:56.833662	\N	f	\N
7	675	BRCA2	BRCA2 mutations in the germline have become a hallmark for hereditary breast and ovarian cancers. Variants that have been demonstrated to reduce the function of the protein have been shown to increase risk for these cancers, as well as prostate and pancreatic cancer. These findings have been the impetus for the increased popularity of genetic testing of healthy indivudals to assess risk.	breast cancer 2, early onset	2015-07-22 18:46:57.104555	2015-07-22 18:46:57.104555	\N	f	\N
8	595	CCND1	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D1	2015-07-22 18:46:57.334277	2015-07-22 18:46:57.334277	\N	f	\N
9	894	CCND2	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D2	2015-07-22 18:46:57.561765	2015-07-22 18:46:57.561765	\N	f	\N
10	896	CCND3	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. In a mouse study, the targeted therapeutic palbociclib significantly increased the median survival of the cyclin D3 knockouts.	cyclin D3	2015-07-22 18:46:57.744404	2015-07-22 18:46:57.744404	\N	f	\N
11	898	CCNE1	Cyclin E, while currenly not as widely implicated as its cyclin D counterparts, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements are correlated with later stage disease. In lung cancer, neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly expressed counterparts.	cyclin E1	2015-07-22 18:46:57.94328	2015-07-22 18:46:57.94328	\N	f	\N
12	1021	CDK6	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 6	2015-07-22 18:46:58.157179	2015-07-22 18:46:58.157179	\N	f	\N
13	1019	CDK4	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 4	2015-07-22 18:46:58.372476	2015-07-22 18:46:58.372476	\N	f	\N
14	1029	CDKN2A	CDKN2A loss has been shown to be a significant event in a number of cancer types. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. One mechanism by which loss of CDKN2A can occur is by hypermethylation of the promoter region for the gene. However, the prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have suggesting poorer prognostic outcome for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	cyclin-dependent kinase inhibitor 2A	2015-07-22 18:46:58.650565	2015-07-22 18:46:58.650565	\N	f	\N
15	1050	CEBPA	AML with mutated CEBPA' is a provisional entity in the WHO classification of acute myeloid leukemia (AML) and is recommended to be tested for in patients with AML. CEBPA mutations are particularly associated with cytogenetically normal AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha protein while C-terminal mutations reduce the DNA-binding potential of this transcription factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	CCAAT/enhancer binding protein (C/EBP), alpha	2015-07-22 18:46:58.906066	2015-07-22 18:46:58.906066	\N	f	\N
16	3337	DNAJB1	DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	DnaJ (Hsp40) homolog, subfamily B, member 1	2015-07-22 18:46:59.15031	2015-07-22 18:46:59.15031	\N	f	\N
17	5566	PRKACA	PRKACA has been studied in breast cancer and has been found to mediate resistance to HER2 targeted therapies. It has also been found to contain a mutation hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	protein kinase, cAMP-dependent, catalytic, alpha	2015-07-22 18:46:59.375427	2015-07-22 18:46:59.375427	\N	f	\N
18	1788	DNMT3A	DNMT3A is one of several epigenetic modifiers identified as recurrently mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers. AML patient bone marrow harboring R882 mutations were similarly demonstrated to be hypomethylated compared to patients with wildtype DNMT3A. These studies also indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes and treatment protocols may need to be considered for these two classes of DNMT3A mutations.	DNA (cytosine-5-)-methyltransferase 3 alpha	2015-07-22 18:46:59.58746	2015-07-22 18:46:59.58746	\N	f	\N
19	1956	EGFR	EGFR is widely recognized for its importance in cancer. Amplification and mutations have been shown to be driving events in many cancer types. Its role in non-small cell lung cancer has spurred many research and drug development efforts. Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance to these drugs, particularly the variant T790M, which has been functionally characterized as a resistance marker for both of these drugs. The later generation TKI's have seen some success in treating these resistant cases, and targeted sequencing of the EGFR locus has become a common practice in non-small cell lung cancer.	epidermal growth factor receptor	2015-07-22 18:46:59.847388	2015-07-22 18:46:59.847388	\N	f	\N
20	2064	ERBB2	ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast cancer, and is treated in a separate manner than the other subtypes of breast cancer. Apart from being amplified, ERBB2 activating mutations have been shown to have clinical importance in HER2-negative breast cancer. These mutations have shown sensitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical sequencing efforts in treating breast cancer.	erb-b2 receptor tyrosine kinase 2	2015-07-22 18:47:00.136365	2015-07-22 18:47:00.136365	\N	f	\N
21	2099	ESR1	ESR1 has been a focus in breast cancer for quite some time, but has also shown significance in endometrial, ovarian and other cancer types. ER-positive breast cancer that is resistant to hormone therapy has instigated clinical sequencing efforts to shed light on the mechanisms of this resistance. A number of mutations in the ligand binding domain of the protein have been implicated in hormone resistance and anti-estrogen therapies. This has spurred efforts to develop therapeutics that act to degrade the protein, rather than act as an antagonist. These agents are currently in clinical trials and have seen some success, highlighting the importance of sequencing efforts in treating breast cancer.	estrogen receptor 1	2015-07-22 18:47:00.407739	2015-07-22 18:47:00.407739	\N	f	\N
22	2263	FGFR2	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions that lead to pathway actiation have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 2	2015-07-22 18:47:00.695514	2015-07-22 18:47:00.695514	\N	f	\N
23	2261	FGFR3	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions that lead to pathway actiation have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 3	2015-07-22 18:47:00.95088	2015-07-22 18:47:00.95088	\N	f	\N
24	2322	FLT3	FLT3 is an important cytokine receptor involved in normal hematopoiesis. Mutations in this gene are common in acute myeloid leukemia (AML) and screening for mutations in this gene has been recommended by the World Health Organization in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML). FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD mutations are still up for debate.	fms-related tyrosine kinase 3	2015-07-22 18:47:01.171436	2015-07-22 18:47:01.171436	\N	f	\N
25	2624	GATA2	GATA2 is a transcription factor involved in stem cell maintenance with key roles in hematopoietic development. GATA2 mutations are associated with a variety of inherited and acquired immune disorders including myelodysplastic syndrome and acute myeloid leukemia. In addition to a role in hematopoiesis, the maintenance GATA2 expression has been implicated as a requirement in KRAS-driven non-small cell lung cancer. Preclinical models have indicated therapeutic benefit from targeting GATA2-mediated pathways in the context of KRAS-driven NSCLC.	GATA binding protein 2	2015-07-22 18:47:01.399293	2015-07-22 18:47:01.399293	\N	f	\N
26	3417	IDH1	IDH1 mutations have been observed in a number of cancer types, including sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described as mutually exclusive in many of these cancer types. The most frequent mutations involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic enzyme activity. The implications of mutations in this gene vary greatly by cancer type. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations have been associated with worse outcome, shorter overall survival, and normal karyotype. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Unlike the association with cytogenetically normal AML, in glioblastoma, IDH1 mutations have been associated with specific cytogenetic abnormalities, 1p and 19q deletions.	isocitrate dehydrogenase 1 (NADP+), soluble	2015-07-22 18:47:01.651951	2015-07-22 18:47:01.651951	\N	f	\N
27	3418	IDH2	IDH2 mutations have been observed in a number of cancer types, including sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described as mutually exclusive in many of these cancer types. The most frequent mutations involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic enzyme activity. Although IDH2 (R172) mutations are associated with poorer overall prognosis in AML patients, its utility as a prognostic marker in MDS is still under debate. Additionally, IDH2 (R140) has been associated with improved overall survival in AML. IDH2 mutations have been associated with improved prognosis in gliomas.	isocitrate dehydrogenase 2 (NADP+), mitochondrial	2015-07-22 18:47:01.905458	2015-07-22 18:47:01.905458	\N	f	\N
28	3717	JAK2	JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative diseases and cancers. The mutation V617F is the most clinically relevant variant, and is seen in around half of myeloproliferative disorders. The variant is a known activating mutation, and activated JAK2 is sufficient to drive myeloproliferative disorders in mouse models. V617F, while most recurrent, is not the only mechanism by which JAK2 can be activated in patients. JAK2 is now one of the first diagnostic markers tested upon diagnosis with a myeloproliferative disorder.	Janus kinase 2	2015-07-22 18:47:02.127411	2015-07-22 18:47:02.127411	\N	f	\N
29	3815	KIT	c-KIT activation has been shown to have oncogenic activity in gastrointestinal stromal tumors (GISTs), melanomas, lung cancer, and other tumor types. The targeted therapeutics nilotinib and sunitinib have shown efficacy in treating KIT overactive patients, and are in late-stage trials in melanoma and GIST. KIT overactivity can be the result of many genomic events from genomic amplification to overexpression to missense mutations. Missense mutations have been shown to be key players in mediating clinical response and acquired resistance in patients being treated with these targeted therapeutics.	v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog	2015-07-22 18:47:02.340371	2015-07-22 18:47:02.340371	\N	f	\N
30	3845	KRAS	Mutations in the RAS family of proteins have frequently observed across cancer types. The amino acid positions account for the overwhelming majority of these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also behave very differently when expressed in non-native tissue types, likely due to differences in the C-terminal hyper-variable regions. Mis-regulation of isoform expression has been shown to be a driving event in cancer, as well as missense mutations at the three hotspots previously mentioned. While highly recurrent in cancer, targeted these RAS mutants has also been very elusive, and has not yet become common practice in the clinic.	Kirsten rat sarcoma viral oncogene homolog	2015-07-22 18:47:02.60495	2015-07-22 18:47:02.60495	\N	f	\N
31	5604	MAP2K1		mitogen-activated protein kinase kinase 1	2015-07-22 18:47:02.839543	2015-07-22 18:47:02.839543	\N	f	\N
32	4209	MEF2D		myocyte enhancer factor 2D	2015-07-22 18:47:03.031105	2015-07-22 18:47:03.031105	\N	f	\N
33	1436	CSF1R		colony stimulating factor 1 receptor	2015-07-22 18:47:03.249777	2015-07-22 18:47:03.249777	\N	f	\N
34	4255	MGMT		O-6-methylguanine-DNA methyltransferase	2015-07-22 18:47:03.434063	2015-07-22 18:47:03.434063	\N	f	\N
35	4869	NPM1	AML with mutated NPM1 is provisional entity in the WHO classification of AML and is recommended to be tested for in patients with cytogenetically normal AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia, indicating it may require additional mutations to promote leukemic development.	nucleophosmin (nucleolar phosphoprotein B23, numatrin)	2015-07-22 18:47:03.700326	2015-07-22 18:47:03.700326	\N	f	\N
36	4893	NRAS	Mutations in the RAS family of proteins have frequently observed across cancer types. The amino acid positions account for the overwhelming majority of these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also behave very differently when expressed in non-native tissue types, likely due to differences in the C-terminal hyper-variable regions. Mis-regulation of isoform expression has been shown to be a driving event in cancer, as well as missense mutations at the three hotspots previously mentioned. While highly recurrent in cancer, targeted these RAS mutants has also been very elusive, and has not yet become common practice in the clinic.	neuroblastoma RAS viral (v-ras) oncogene homolog	2015-07-22 18:47:03.927344	2015-07-22 18:47:03.927344	\N	f	\N
37	5290	PIK3CA	PIK3CA is the most recurrently mutated gene in breast cancer, and has been found to important in a number of cancer types. An integral part of the PI3K pathway, PIK3CA has long been described as an oncogene, with two main hotspots for activating mutations, the 542/545 region of the helical domain, and the 1047 region of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the subject of an immense amount of research and development, and PI3K inhibition has seen some limited success in recent clinical trials. While monotherapies seem to be limited in their potential, there is a recent interest in pursuing PI3K inhibition as part of a combination therapy regiment with inhibition partners including TKI's, MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.	phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha	2015-07-22 18:47:04.157659	2015-07-22 18:47:04.157659	\N	f	\N
38	5156	PDGFRA		platelet-derived growth factor receptor, alpha polypeptide	2015-07-22 18:47:04.371226	2015-07-22 18:47:04.371226	\N	f	\N
39	5371	PML	The PML-RARA fusion is the result of a recurrent, balanced translocation between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	promyelocytic leukemia	2015-07-22 18:47:04.581562	2015-07-22 18:47:04.581562	\N	f	\N
40	5914	RARA	The PML-RARA fusion is the result of a recurrent, balanced translocation between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	retinoic acid receptor, alpha	2015-07-22 18:47:04.793094	2015-07-22 18:47:04.793094	\N	f	\N
41	5728	PTEN		phosphatase and tensin homolog	2015-07-22 18:47:05.089338	2015-07-22 18:47:05.089338	\N	f	\N
42	5979	RET	RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine kinase receptor and are associated with thyroid cancers. RET point mutations are the most common mutations identified in medullary thyroid cancer (MTC) with germline and somatic mutations in RET associated with hereditary and sporadic forms, respectively. The most common somatic form mutation is M918T (exon 16) and a variety of other mutations effecting exons 10, 11 and 15 have been described. The prognostic significance of these mutations have been hotly debated in the field, however, data suggests that some RET mutation may confer drug resistence. No RET-specific agents are currently clinically available but several promiscuous kinase inhibitors that target RET, among others, have been approved for MTC treatment.	ret proto-oncogene	2015-07-22 18:47:05.363732	2015-07-22 18:47:05.363732	\N	f	\N
43	861	RUNX1		runt-related transcription factor 1	2015-07-22 18:47:05.629792	2015-07-22 18:47:05.629792	\N	f	\N
44	23451	SF3B1	SF3B1 mutations have been described in several myeloid malignancies, predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has been identified as recurrently mutated in MDS and other malignanices. The mutations affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally deleterious with K700E described as a major hotspot mutation. MDS patients with SF3B1 mutations have been reported to have better overall and event-free survival than their wildtype counterparts. Additionally, these mutations are highly associated with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring sideroblasts).	splicing factor 3b, subunit 1, 155kDa	2015-07-22 18:47:05.876207	2015-07-22 18:47:05.876207	\N	f	\N
45	7157	TP53	TP53 mutations are universal across cancer types. Loss of tumor suppressors is most recognized by large deleterious events, such as frameshift mutations, or premature stop codons. In TP53 however, many of the observed mutations in cancer are found to be single nucleotide variants, or missense mutations. These variants are also very broadly distributed throughout the gene, not localizing in any particular hotspot. While a large proportion of cancer genomics research is focused on somatic variants, TP53 is also of note in the germline. Germline TP53 mutations are the hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been found to have prognostic impact on patient outcomes.	tumor protein p53	2015-07-22 18:47:06.15295	2015-07-22 18:47:06.15295	\N	f	\N
46	7248	TSC1		tuberous sclerosis 1	2015-07-22 18:47:06.410703	2015-07-22 18:47:06.410703	\N	f	\N
47	7249	TSC2		tuberous sclerosis 2	2015-07-22 18:47:06.61665	2015-07-22 18:47:06.61665	\N	f	\N
48	7307	U2AF1	U2AF1 is one of several spliceosome complex genes frequently mutated in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein. These mutations have been associated with altered splicing patterns in vitro and in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation to secondary acute myeloid leukemia, however, the impact of these mutations on overall survival has been an area of debate.	U2 small nuclear RNA auxiliary factor 1	2015-07-22 18:47:06.83709	2015-07-22 18:47:06.83709	\N	f	\N
49	7490	WT1	WT1 is a tumor suppressor gene associated with the development of Wilms' Tumor, from which it was named. Mutations in exon 7 and 9 of WT1 have been recurrently identified in acute myeloid leukemia and associated with poorer prognosis and chemotherapy resistance.	Wilms tumor 1	2015-07-22 18:47:07.081917	2015-07-22 18:47:07.081917	\N	f	\N
50	4851	NOTCH1		notch 1	2015-07-22 18:47:07.297986	2015-07-22 18:47:07.297986	\N	f	\N
51	4921	DDR2		discoidin domain receptor tyrosine kinase 2	2015-07-22 18:47:07.505277	2015-07-22 18:47:07.505277	\N	f	\N
52	4233	MET		MET proto-oncogene, receptor tyrosine kinase	2015-07-22 18:47:07.722814	2015-07-22 18:47:07.722814	\N	f	\N
53	2313	FLI1		Fli-1 proto-oncogene, ETS transcription factor	2015-07-22 18:47:07.937092	2015-07-22 18:47:07.937092	\N	f	\N
54	2130	EWSR1		EWS RNA-binding protein 1	2015-07-22 18:47:08.138373	2015-07-22 18:47:08.138373	\N	f	\N
55	54790	TET2		tet methylcytosine dioxygenase 2	2015-07-22 18:47:08.331018	2015-07-22 18:47:08.331018	\N	f	\N
56	27086	FOXP1		forkhead box P1	2015-07-22 18:47:08.539779	2015-07-22 18:47:08.539779	\N	f	\N
57	5966	REL		v-rel avian reticuloendotheliosis viral oncogene homolog	2015-07-22 18:47:08.732314	2015-07-22 18:47:08.732314	\N	f	\N
58	7428	VHL		von Hippel-Lindau tumor suppressor, E3 ubiquitin protein ligase	2015-07-22 18:47:08.9467	2015-07-22 18:47:08.9467	\N	f	\N
59	596	BCL2		B-cell CLL/lymphoma 2	2015-07-22 18:47:09.141665	2015-07-22 18:47:09.141665	\N	f	\N
60	3481	IGF2		insulin-like growth factor 2	2015-07-22 18:47:09.333353	2015-07-22 18:47:09.333353	\N	f	\N
61	6790	AURKA		aurora kinase A	2015-07-22 18:47:09.573199	2015-07-22 18:47:09.573199	\N	f	\N
62	55193	PBRM1		polybromo 1	2015-07-22 18:47:09.770023	2015-07-22 18:47:09.770023	\N	f	\N
63	2146	EZH2		enhancer of zeste 2 polycomb repressive complex 2 subunit	2015-07-22 18:47:10.021172	2015-07-22 18:47:10.021172	\N	f	\N
\.


--
-- Data for Name: sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY sources (id, pubmed_id, study_type, description, created_at, updated_at) FROM stdin;
1	24889366	\N	Rossi et al., 2014, Int. J. Oncol.	2015-07-22 18:46:55.43118	2015-07-22 18:47:38.806368
2	23401436	\N	Shaw et al., 2013, J. Clin. Oncol.	2015-07-22 18:46:55.506192	2015-07-22 18:47:39.640197
3	16095999	\N	Bellacosa et al., 2005, Adv. Cancer Res.	2015-07-22 18:46:55.752573	2015-07-22 18:47:40.054112
4	15023437	\N	Fresno Vara et al., 2004, Cancer Treat. Rev.	2015-07-22 18:46:55.76539	2015-07-22 18:47:40.66607
5	15676015	\N	Lee et al., 2005, APMIS	2015-07-22 18:46:55.995847	2015-07-22 18:47:41.042292
6	24569458	\N	Imielinski et al., 2014, J. Clin. Invest.	2015-07-22 18:46:56.023352	2015-07-22 18:47:41.399895
7	15719031	\N	Ren, 2005, Nat. Rev. Cancer	2015-07-22 18:46:56.277349	2015-07-22 18:47:41.853709
8	17457302	\N	Weisberg et al., 2007, Nat. Rev. Cancer	2015-07-22 18:46:56.28962	2015-07-22 18:47:42.24658
9	19724843	\N	Li et al., 2009, Oncol. Rep.	2015-07-22 18:46:56.5762	2015-07-22 18:47:42.658112
10	23594689	\N	Pakneshan et al., 2013, Pathology	2015-07-22 18:46:56.585588	2015-07-22 18:47:43.061314
11	24366442	\N	Nelson et al., 2014, Ann. Intern. Med.	2015-07-22 18:46:56.85307	2015-07-22 18:47:43.41924
12	12432268	\N	Diehl, Cancer Biol. Ther.	2015-07-22 18:46:57.349766	2015-07-22 18:47:43.820392
13	24387133	\N	Casimiro et al., 2014, Expert Opin Investig Drugs	2015-07-22 18:46:57.360944	2015-07-22 18:47:44.217143
14	14965268	\N	Mazumder et al., 2004, Curr Cancer Drug Targets	2015-07-22 18:46:57.957789	2015-07-22 18:47:44.744953
15	20370706	\N	Graf et al., 2010, Mini Rev Med Chem	2015-07-22 18:46:58.176208	2015-07-22 18:47:45.156668
16	24089445	\N	Sheppard et al., 2013, Clin. Cancer Res.	2015-07-22 18:46:58.185749	2015-07-22 18:47:45.57531
17	20473920	\N	Shima et al., 2011, Int. J. Cancer	2015-07-22 18:46:58.667328	2015-07-22 18:47:45.995005
18	23111194	\N	Bradly et al., 2012, Diagn. Mol. Pathol.	2015-07-22 18:46:58.67731	2015-07-22 18:47:46.403057
19	19357394	\N	Vardiman et al., 2009, Blood	2015-07-22 18:46:58.920286	2015-07-22 18:47:46.788382
20	11242107	\N	Pabst et al., 2001, Nat. Genet.	2015-07-22 18:46:58.931741	2015-07-22 18:47:47.350595
21	9012825	\N	Zhang et al., 1997, Proc. Natl. Acad. Sci. U.S.A.	2015-07-22 18:46:58.940943	2015-07-22 18:47:47.768661
22	24578576	\N	Honeyman et al., 2014, Science	2015-07-22 18:46:59.166529	2015-07-22 18:47:48.152956
23	24909179	\N	Moody et al., 2015, Oncogene	2015-07-22 18:46:59.392741	2015-07-22 18:47:48.605781
24	24167195	\N	Kim et al., 2013, Blood	2015-07-22 18:46:59.603582	2015-07-22 18:47:48.916834
25	24656771	\N	Russler-Germain et al., 2014, Cancer Cell	2015-07-22 18:46:59.614522	2015-07-22 18:47:49.352281
26	23953842	\N	Yewale et al., 2013, Biomaterials	2015-07-22 18:46:59.86095	2015-07-22 18:47:49.71399
27	18712184	\N	Charpidou et al., In Vivo	2015-07-22 18:46:59.884981	2015-07-22 18:47:50.04017
28	16947083	\N	Badache et al., 2006, J Mammary Gland Biol Neoplasia	2015-07-22 18:47:00.148521	2015-07-22 18:47:50.481766
29	10878580	\N	Yu et al., 2000, Bioessays	2015-07-22 18:47:00.16255	2015-07-22 18:47:50.9012
30	24185510	\N	Robinson et al., 2013, Nat. Genet.	2015-07-22 18:47:00.425214	2015-07-22 18:47:51.32243
31	22279420	\N	Mohibi et al., 2011, J Carcinog	2015-07-22 18:47:00.437413	2015-07-22 18:47:51.736742
32	18636142	\N	Katoh, 2008, Int. J. Oncol.	2015-07-22 18:47:00.708504	2015-07-22 18:47:52.154122
33	23558953	\N	Wu et al., 2013, Cancer Discov	2015-07-22 18:47:00.718479	2015-07-22 18:47:52.783532
34	12951584	\N	Stirewalt et al., 2003, Nat. Rev. Cancer	2015-07-22 18:47:01.188074	2015-07-22 18:47:53.207522
35	22541434	\N	Kumar et al., 2012, Cell	2015-07-22 18:47:01.41117	2015-07-22 18:47:53.616806
36	25619630	\N	Mir et al., 2015, Cancer Med	2015-07-22 18:47:01.419853	2015-07-22 18:47:54.035402
37	24065766	\N	Lu et al., 2013, Genes Dev.	2015-07-22 18:47:01.664263	2015-07-22 18:47:54.453537
38	19228619	\N	Yan et al., 2009, N. Engl. J. Med.	2015-07-22 18:47:01.674236	2015-07-22 18:47:54.874536
39	22417203	\N	Patel et al., 2012, N. Engl. J. Med.	2015-07-22 18:47:01.923588	2015-07-22 18:47:55.289072
40	17133099	\N	Levine et al., 2007, Curr. Opin. Hematol.	2015-07-22 18:47:02.139886	2015-07-22 18:47:55.879013
41	23944364	\N	Stankov et al., 2014, Curr. Pharm. Des.	2015-07-22 18:47:02.352874	2015-07-22 18:47:56.267196
42	22589270	\N	Prior et al., 2012, Cancer Res.	2015-07-22 18:47:02.616601	2015-07-22 18:47:56.678938
43	23226219	\N	Chou et al., 2012, PLoS ONE	2015-07-22 18:47:03.716396	2015-07-22 18:47:57.031916
44	18073307	\N	Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-07-22 18:47:05.380845	2015-07-22 18:47:57.584894
45	25465739	\N	Perri et al., 2015, Crit. Rev. Oncol. Hematol.	2015-07-22 18:47:05.39452	2015-07-22 18:47:58.002233
46	21995386	\N	Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-07-22 18:47:05.894017	2015-07-22 18:47:58.425488
47	24136165	\N	Cazzola et al., 2013, Blood	2015-07-22 18:47:05.906308	2015-07-22 18:47:58.843089
48	20182602	\N	Olivier et al., 2010, Cold Spring Harb Perspect Biol	2015-07-22 18:47:06.169949	2015-07-22 18:47:59.261694
49	25311244	\N	Okeyo-Owuor et al., 2015, Leukemia	2015-07-22 18:47:06.848912	2015-07-22 18:47:59.989806
50	22158538	\N	Graubert et al., 2012, Nat. Genet.	2015-07-22 18:47:06.864148	2015-07-22 18:48:00.407523
51	16081687	Retrospective study	Levine et al., 2005, Blood	2015-07-22 18:47:14.393978	2015-07-22 18:48:00.889497
52	15146165		Lasota et al., 2004, Lab. Invest.	2015-07-22 18:47:14.473074	2015-07-22 18:48:01.25142
53	22081665	Retrospective study	LaRochelle et al., 2011, Oncotarget	2015-07-22 18:47:14.596956	2015-07-22 18:48:01.660153
54	22490330	Retrospective study	Ribeiro et al., 2012, Blood	2015-07-22 18:47:14.66251	2015-07-22 18:48:02.077289
55	19287384	Retrospective study	Kilpivaara et al., 2009, Nat. Genet.	2015-07-22 18:47:14.778885	2015-07-22 18:48:02.494139
56	24571676	Retrospective study	Andrade et al., 2014, BMC Cancer	2015-07-22 18:47:14.821155	2015-07-22 18:48:02.91332
57	23861977	Retrospective study	Tschandl et al., 2013, PLoS ONE	2015-07-22 18:47:14.897693	2015-07-22 18:48:03.33371
58	19915144		Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-07-22 18:47:14.992254	2015-07-22 18:48:03.753102
59	19915144	In vitro study	Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-07-22 18:47:15.040358	2015-07-22 18:48:04.378427
60	23569304	Clinical trial	Trunzer et al., 2013, J. Clin. Oncol.	2015-07-22 18:47:15.087454	2015-07-22 18:48:04.796834
61	15928335		Corless et al., 2005, J. Clin. Oncol.	2015-07-22 18:47:15.153094	2015-07-22 18:48:05.211878
62	16954519		Heinrich et al., 2006, J. Clin. Oncol.	2015-07-22 18:47:15.209368	2015-07-22 18:48:05.571148
63	24569458		Imielinski et al., 2014, J. Clin. Invest.	2015-07-22 18:47:15.271626	2015-07-22 18:48:05.940585
64	16709929	Clinical trial	Kiladjian et al., 2006, Blood	2015-07-22 18:47:15.365829	2015-07-22 18:48:06.464386
65	18394554	In vitro study	Wernig et al., 2008, Cancer Cell	2015-07-22 18:47:15.423901	2015-07-22 18:48:06.886024
66	18375819	Case report	Banerji et al., 2008, Mol. Cancer Ther.	2015-07-22 18:47:15.483082	2015-07-22 18:48:07.241891
67	21576590	Case report	Soon et al., 2011, Arch Dermatol	2015-07-22 18:47:15.539691	2015-07-22 18:48:07.623895
68	21067377	Retrospective study	Ley et al., 2010, N. Engl. J. Med.	2015-07-22 18:47:15.70304	2015-07-22 18:48:08.042883
69	16384925	Retrospective study	Cairoli et al., 2006, Blood	2015-07-22 18:47:15.744654	2015-07-22 18:48:08.450418
70	24666267	Meta analysis	Therkildsen et al., 2014, Acta Oncol	2015-07-22 18:47:15.809423	2015-07-22 18:48:08.865612
71	21166657	Meta analysis	Lee et al., 2011, Br. J. Dermatol.	2015-07-22 18:47:15.925429	2015-07-22 18:48:09.353197
72	24512939	Retrospective study	El Ghannam et al., Blood Cells Mol. Dis.	2015-07-22 18:47:15.974528	2015-07-22 18:48:09.675532
73	21030459		Sasaki et al., 2010, Cancer Res.	2015-07-22 18:47:16.020808	2015-07-22 18:48:10.120947
74	19671763	In vitro study	Woodman et al., 2009, Mol. Cancer Ther.	2015-07-22 18:47:16.141373	2015-07-22 18:48:10.568894
75	15696205	Retrospective study	Pao et al., 2005, PLoS Med.	2015-07-22 18:47:16.216723	2015-07-22 18:48:11.002562
76	20619739	Retrospective study	De Roock et al., 2010, Lancet Oncol.	2015-07-22 18:47:16.27025	2015-07-22 18:48:11.390143
77	21575866		Sakamoto et al., 2011, Cancer Cell	2015-07-22 18:47:16.333596	2015-07-22 18:48:11.794195
78	21948233		Heuckmann et al., 2011, Clin. Cancer Res.	2015-07-22 18:47:16.37954	2015-07-22 18:48:12.210171
79	18923525		George et al., 2008, Nature	2015-07-22 18:47:16.423474	2015-07-22 18:48:12.732491
80	22745105		Heinrich et al., 2012, Clin. Cancer Res.	2015-07-22 18:47:16.594726	2015-07-22 18:48:13.100735
81	23632886	Retrospective study	Gaidzik et al., 2013, Blood	2015-07-22 18:47:16.888604	2015-07-22 18:48:13.565998
82	16551858	Retrospective study	Wardelmann et al., 2006, Clin. Cancer Res.	2015-07-22 18:47:17.029868	2015-07-22 18:48:13.930473
83	15951308	Retrospective study	Bowen et al., 2005, Blood	2015-07-22 18:47:17.150666	2015-07-22 18:48:14.296176
84	16434492	Clinical trial	Bacher et al., 2006, Blood	2015-07-22 18:47:17.194273	2015-07-22 18:48:14.719034
85	23775962	Clinical trial	Hodi et al., 2013, J. Clin. Oncol.	2015-07-22 18:47:17.233385	2015-07-22 18:48:15.105366
86	22291079	Retrospective study	Marcucci et al., 2012, J. Clin. Oncol.	2015-07-22 18:47:17.584005	2015-07-22 18:48:15.463859
87	22180178	Clinical trial	Jakob et al., 2012, Cancer	2015-07-22 18:47:17.782344	2015-07-22 18:48:16.124537
88	23031422	Case report	Ponti et al., 2012, J Hematol Oncol	2015-07-22 18:47:17.872874	2015-07-22 18:48:16.551234
89	18073307		Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-07-22 18:47:17.953445	2015-07-22 18:48:17.01435
90	21422803		Coxon et al., 2012, J. Endocrinol. Invest.	2015-07-22 18:47:17.993772	2015-07-22 18:48:17.343365
91	23056499		Couto et al., 2012, PLoS ONE	2015-07-22 18:47:18.111371	2015-07-22 18:48:17.745739
92	9839497		Egawa et al., 1998, Jpn. J. Clin. Oncol.	2015-07-22 18:47:18.153253	2015-07-22 18:48:18.162698
93	21594703	Retrospective study	Howell et al., 2011, Ann. Surg. Oncol.	2015-07-22 18:47:18.195378	2015-07-22 18:48:18.511341
94	24570209	Retrospective study	Crescenzi et al., 2014, Horm. Metab. Res.	2015-07-22 18:47:18.240711	2015-07-22 18:48:19.992964
95	24588959	Perspective study	Zhang et al., 2014, Diagn Pathol	2015-07-22 18:47:18.27926	2015-07-22 18:48:20.461354
96	24586605	Retrospective study	Meckbach et al., 2014, PLoS ONE	2015-07-22 18:47:18.329236	2015-07-22 18:48:20.983648
97	24559322	Clinical trial	Sartore-Bianchi et al., 2014, Expert Rev Anticancer Ther	2015-07-22 18:47:18.388616	2015-07-22 18:48:21.403268
98	24576830	In vitro study	Nissan et al., 2014, Cancer Res.	2015-07-22 18:47:18.523278	2015-07-22 18:48:21.777705
99	23325582	Clinical trial	Peeters et al., 2013, Clin. Cancer Res.	2015-07-22 18:47:18.604846	2015-07-22 18:48:22.154878
100	19001320	In vitro study	Di Nicolantonio et al., 2008, J. Clin. Oncol.	2015-07-22 18:47:18.647732	2015-07-22 18:48:22.558512
101	23524406	Case report	Rudin et al., 2013, J Thorac Oncol	2015-07-22 18:47:18.725329	2015-07-22 18:48:22.965085
102	21098728	In vitro study	Corcoran et al., 2010, Sci Signal	2015-07-22 18:47:18.768646	2015-07-22 18:48:23.383893
103	23020132	Clinical trial	Flaherty et al., 2012, N. Engl. J. Med.	2015-07-22 18:47:18.817106	2015-07-22 18:48:23.803474
104	23463675	Clinical trial	Ponti et al., 2013, J. Clin. Pathol.	2015-07-22 18:47:18.858822	2015-07-22 18:48:24.252206
105	24583796	Clinical trial	Menzies et al., 2014, Clin. Cancer Res.	2015-07-22 18:47:18.90902	2015-07-22 18:48:24.577538
106	23845441	In vitro study	Rad et al., 2013, Cancer Cell	2015-07-22 18:47:18.979895	2015-07-22 18:48:24.952508
107	23812671	In vitro study	Ji et al., 2013, Clin. Cancer Res.	2015-07-22 18:47:19.045764	2015-07-22 18:48:25.372176
108	22180495	In vivo study	Yang et al., 2012, Cancer Res.	2015-07-22 18:47:19.102863	2015-07-22 18:48:25.790941
109	24354346	Retrospective study	Walczyk et al., 2014, Clin. Endocrinol. (Oxf)	2015-07-22 18:47:19.304618	2015-07-22 18:48:26.151325
110	24594804	Metaanalysis	Chen et al., 2014, PLoS ONE	2015-07-22 18:47:19.365459	2015-07-22 18:48:26.537969
111	24388723	Retrospective study	Nagore et al., 2014, J. Am. Acad. Dermatol.	2015-07-22 18:47:19.424022	2015-07-22 18:48:26.93445
112	24396464	Retrospective study	He et al., 2014, Oncol Lett	2015-07-22 18:47:19.541291	2015-07-22 18:48:27.252338
113	16455956	Retrospective study	Thiede et al., 2006, Blood	2015-07-22 18:47:19.639632	2015-07-22 18:48:27.683615
114	16076867	Retrospective study	Schnittger et al., 2005, Blood	2015-07-22 18:47:19.829047	2015-07-22 18:48:28.050898
115	19357394		Vardiman et al., 2009, Blood	2015-07-22 18:47:20.057979	2015-07-22 18:48:28.400534
116	17957027	Retrospective study	Gale et al., 2008, Blood	2015-07-22 18:47:20.11192	2015-07-22 18:48:28.922045
117	19059939	Retrospective study	Schlenk et al., 2009, Haematologica	2015-07-22 18:47:20.251401	2015-07-22 18:48:29.342068
118	19965647	Retrospective study	Burnett et al., 2010, Blood	2015-07-22 18:47:20.366873	2015-07-22 18:48:29.827004
119	22072639		Bresler et al., 2011, Sci Transl Med	2015-07-22 18:47:20.542039	2015-07-22 18:48:30.297004
120	19965647		Burnett et al., 2010, Blood	2015-07-22 18:47:20.680324	2015-07-22 18:48:30.683848
121	15626738	In vitro study	Bagrintseva et al., 2005, Blood	2015-07-22 18:47:20.908385	2015-07-22 18:48:31.016693
122	18202412	In vitro study	Lièvre et al., 2008, J. Clin. Oncol.	2015-07-22 18:47:21.18133	2015-07-22 18:48:31.430783
123	20978259	Clinical trial	De Roock et al., 2010, JAMA	2015-07-22 18:47:21.250695	2015-07-22 18:48:31.849062
124	18591546	Retrospective study	Virappane et al., 2008, J. Clin. Oncol.	2015-07-22 18:47:21.406163	2015-07-22 18:48:32.26462
125	23558953		Wu et al., 2013, Cancer Discov	2015-07-22 18:47:21.64801	2015-07-22 18:48:32.681507
126	22417203	Retrospective study	Patel et al., 2012, N. Engl. J. Med.	2015-07-22 18:47:21.825729	2015-07-22 18:48:33.062417
127	24797300		Tassara et al., 2014, Blood	2015-07-22 18:47:21.880607	2015-07-22 18:48:33.410637
128	21719597	In vitro study	Balusu et al., 2011, Blood	2015-07-22 18:47:21.945545	2015-07-22 18:48:33.844152
129	24927407		Ehninger et al., 2014, Blood Cancer J	2015-07-22 18:47:22.013968	2015-07-22 18:48:34.238452
130	15659725		Falini et al., 2005, N. Engl. J. Med.	2015-07-22 18:47:22.176584	2015-07-22 18:48:34.619622
131	19966866	Mouse model	Liang et al., 2010, Oncogene	2015-07-22 18:47:22.245527	2015-07-22 18:48:34.987472
132	22923433	Retrospective study	Iyer et al., 2012, Science	2015-07-22 18:47:22.327708	2015-07-22 18:48:35.396688
133	14726504	Retrospective study	Fröhling et al., 2004, J. Clin. Oncol.	2015-07-22 18:47:22.391124	2015-07-22 18:48:35.733889
134	21537333		Schnittger et al., 2011, Leukemia	2015-07-22 18:47:22.494463	2015-07-22 18:48:36.130486
135	20805365	Retrospective study	Schnittger et al., 2010, Blood	2015-07-22 18:47:22.6048	2015-07-22 18:48:36.540861
136	19221039	Retrospective study	Gaidzik et al., 2009, Blood	2015-07-22 18:47:22.711398	2015-07-22 18:48:36.88539
137	18450602	Retrospective study	Schlenk et al., 2008, N. Engl. J. Med.	2015-07-22 18:47:22.900765	2015-07-22 18:48:37.25595
138	17965322	Retrospective study	Bacher et al., 2008, Blood	2015-07-22 18:47:23.052904	2015-07-22 18:48:37.656597
139	11070098		Esteller et al., 2000, N. Engl. J. Med.	2015-07-22 18:47:23.096403	2015-07-22 18:48:38.082233
140	19047294	Retrospective study	Büchner et al., 2009, J. Clin. Oncol.	2015-07-22 18:47:23.424921	2015-07-22 18:48:38.530085
141	24855211		Linch et al., 2014, Blood	2015-07-22 18:47:23.478825	2015-07-22 18:48:38.947779
142	22430270	Retrospective study	Gaidzik et al., 2012, J. Clin. Oncol.	2015-07-22 18:47:23.53826	2015-07-22 18:48:39.278932
143	20026798	Retrospective study	Becker et al., 2010, J. Clin. Oncol.	2015-07-22 18:47:23.614624	2015-07-22 18:48:39.725528
144	16051734	Retrospective study	Döhner et al., 2005, Blood	2015-07-22 18:47:23.703673	2015-07-22 18:48:40.11921
145	24801015		Port et al., 2014, Ann. Hematol.	2015-07-22 18:47:23.986111	2015-07-22 18:48:40.618871
146	19587375	Retrospective study	Schnittger et al., 2009, Blood	2015-07-22 18:47:24.258998	2015-07-22 18:48:41.036535
147	24859829		Tian et al., 2014, Int. J. Hematol.	2015-07-22 18:47:24.39415	2015-07-22 18:48:41.453639
148	19536888	Retrospective study	Renneville et al., 2009, Cancer	2015-07-22 18:47:24.511901	2015-07-22 18:48:41.875506
149	18559874	Retrospective study	Paschka et al., 2008, J. Clin. Oncol.	2015-07-22 18:47:24.647468	2015-07-22 18:48:42.293584
150	23346317	Review	Goff, 2013, J Gynecol Oncol	2015-07-22 18:47:24.994953	2015-07-22 18:48:42.711849
151	25212276		Schmidt et al., 2014, Leukemia	2015-07-22 18:47:25.142832	2015-07-22 18:48:43.141025
152	11290608	Retrospective study	Yamamoto et al., 2001, Blood	2015-07-22 18:47:25.187962	2015-07-22 18:48:43.544521
153	18794081	Retrospective study	Riely et al., 2008, Clin. Cancer Res.	2015-07-22 18:47:25.249656	2015-07-22 18:48:44.066089
154	23029227		Qian et al., 2012, PLoS ONE	2015-07-22 18:47:25.347432	2015-07-22 18:48:44.483551
155	14612939		Oshimo et al., 2003, Int. J. Oncol.	2015-07-22 18:47:25.4556	2015-07-22 18:48:44.90082
156	20537386		An et al., 2010, Leuk. Res.	2015-07-22 18:47:25.525749	2015-07-22 18:48:45.318709
157	15961768		Arnold et al., 2005, J. Clin. Oncol.	2015-07-22 18:47:25.5894	2015-07-22 18:48:45.734869
158	20376086	Retrospective study	Ho et al., 2010, Leukemia	2015-07-22 18:47:25.697247	2015-07-22 18:48:46.156011
159	20538800	Retrospective study	Abbas et al., 2010, Blood	2015-07-22 18:47:25.775538	2015-07-22 18:48:47.597651
160	10485475	Retrospective study	Taniguchi et al., 1999, Cancer Res.	2015-07-22 18:47:25.833664	2015-07-22 18:48:48.137036
161	23014527	Retrospective study	Dogan et al., 2012, Clin. Cancer Res.	2015-07-22 18:47:25.949135	2015-07-22 18:48:48.558857
162	24736073		Lim et al., 2014, J Thorac Oncol	2015-07-22 18:47:26.02931	2015-07-22 18:48:48.976642
163	23888070		Beaver et al., 2013, Clin. Cancer Res.	2015-07-22 18:47:26.133107	2015-07-22 18:48:49.396782
164	12476305		Nimmanapalli et al., 2002, Oncogene	2015-07-22 18:47:26.192049	2015-07-22 18:48:49.760405
165	15194504	Cell Line	Yamamoto et al., 2004, Biochem. Biophys. Res. Commun.	2015-07-22 18:47:26.243935	2015-07-22 18:48:50.136506
166	20979473	Clinical Trial	Choi et al., 2010, N. Engl. J. Med.	2015-07-22 18:47:26.406881	2015-07-22 18:48:50.544426
167	25668228	Review	Denis et al., 2015, Clin. Chim. Acta	2015-07-22 18:47:26.545394	2015-07-22 18:48:50.959842
168	15728811		Kobayashi et al., 2005, N. Engl. J. Med.	2015-07-22 18:47:26.638223	2015-07-22 18:48:51.480227
169	21430269		Sequist et al., 2011, Sci Transl Med	2015-07-22 18:47:26.70712	2015-07-22 18:48:51.935822
170	23220880		Bose et al., 2013, Cancer Discov	2015-07-22 18:47:26.774085	2015-07-22 18:48:52.288426
171	24185512		Toy et al., 2013, Nat. Genet.	2015-07-22 18:47:26.827575	2015-07-22 18:48:52.73704
172	22368270	Clinical trial	Man et al., 2012, Blood	2015-07-22 18:47:27.117254	2015-07-22 18:48:53.159622
173	18230792	In vitro study	Zhang et al., 2008, J. Natl. Cancer Inst.	2015-07-22 18:47:27.179082	2015-07-22 18:48:53.57354
174	16618717	Retrospective study	Lièvre et al., 2006, Cancer Res.	2015-07-22 18:47:27.275536	2015-07-22 18:48:53.987915
175	19794967		Marchetti et al., 2009, Neoplasia	2015-07-22 18:47:27.329748	2015-07-22 18:48:54.40848
176	19223544		Sartore-Bianchi et al., 2009, Cancer Res.	2015-07-22 18:47:27.651639	2015-07-22 18:48:54.827208
177	23079656		Sawai et al., 2012, Cancer Cell	2015-07-22 18:47:27.978373	2015-07-22 18:48:55.246979
178	25002028		Vora et al., 2014, Cancer Cell	2015-07-22 18:47:28.035705	2015-07-22 18:48:55.979431
179	22328973		Hammerman et al., 2011, Cancer Discov	2015-07-22 18:47:28.129983	2015-07-22 18:48:56.394974
180	16730237		Ji et al., 2006, Cancer Cell	2015-07-22 18:47:28.49531	2015-07-22 18:48:56.721852
181	15118125		Paez et al., 2004, Science	2015-07-22 18:47:28.553415	2015-07-22 18:48:57.173264
182	24457318		Fukihara et al., 2014, Oncology	2015-07-22 18:47:28.599316	2015-07-22 18:48:57.556593
183	24636847		Li et al., 2014, Lung Cancer	2015-07-22 18:47:28.712446	2015-07-22 18:48:57.960376
184	24658966		Ai et al., 2014, Amino Acids	2015-07-22 18:47:28.766836	2015-07-22 18:48:58.375025
185	24185510		Robinson et al., 2013, Nat. Genet.	2015-07-22 18:47:29.34833	2015-07-22 18:48:58.792929
186	24550739	Case study	Borad et al., 2014, PLoS Genet.	2015-07-22 18:47:29.697787	2015-07-22 18:48:59.250326
187	14726387	Clinical trial	Smith et al., 2004, Blood	2015-07-22 18:47:29.835373	2015-07-22 18:48:59.639165
188	12357354	In vitro study	Tse et al., 2002, Leukemia	2015-07-22 18:47:29.934967	2015-07-22 18:49:00.048979
189	22624710		Barbacid, 2012, Cancer Cell	2015-07-22 18:47:30.024898	2015-07-22 18:49:00.465807
190	18421059	Observational study	Hodi et al., 2008, J. Clin. Oncol.	2015-07-22 18:47:30.088845	2015-07-22 18:49:00.988635
191	17372901	In vitro study	Antonescu et al., 2007, Int. J. Cancer	2015-07-22 18:47:30.144622	2015-07-22 18:49:01.405494
192	16638875		Prenen et al., 2006, Clin. Cancer Res.	2015-07-22 18:47:30.20057	2015-07-22 18:49:01.845108
193	19029981	In vivo study	Engelman et al., 2008, Nat. Med.	2015-07-22 18:47:30.256394	2015-07-22 18:49:02.242037
194	15758010	Clinical Trial	Hegi et al., 2005, N. Engl. J. Med.	2015-07-22 18:47:30.344916	2015-07-22 18:49:02.659005
195	21558396		Tanaka et al., 2011, Clin. Cancer Res.	2015-07-22 18:47:30.462641	2015-07-22 18:49:03.076503
196	15647370		Kang et al., 2005, Proc. Natl. Acad. Sci. U.S.A.	2015-07-22 18:47:30.509433	2015-07-22 18:49:03.493009
197	8674046		Yoshida et al., 1996, Cancer Res.	2015-07-22 18:47:30.6993	2015-07-22 18:49:04.330606
198	20085938		Courtney et al., 2010, J. Clin. Oncol.	2015-07-22 18:47:30.745055	2015-07-22 18:49:04.782577
199	22698404		Jackson et al., 2012, Cancer Cell	2015-07-22 18:47:30.84619	2015-07-22 18:49:05.426408
200	22923433		Iyer et al., 2012, Science	2015-07-22 18:47:30.892672	2015-07-22 18:49:05.805552
201	20368538	Retrospective study	Wagner et al., 2010, J. Clin. Oncol.	2015-07-22 18:47:31.02774	2015-07-22 18:49:06.209749
202	22616558		Zhou et al., 2012, Leuk. Lymphoma	2015-07-22 18:47:31.231387	2015-07-22 18:49:06.608151
203	20421455	Retrospective study	Thol et al., 2010, Blood	2015-07-22 18:47:31.286266	2015-07-22 18:49:07.045524
204	21997850		Lin et al., 2012, Ann. Hematol.	2015-07-22 18:47:31.333583	2015-07-22 18:49:07.88126
205	22033490		Patnaik et al., 2012, Leukemia	2015-07-22 18:47:31.386757	2015-07-22 18:49:08.400895
206	12000708	Retrospective study	Corless et al., 2002, Am. J. Pathol.	2015-07-22 18:47:31.514769	2015-07-22 18:49:08.788819
207	15217946	Retrospective study	Boldrini et al., 2004, Clin. Cancer Res.	2015-07-22 18:47:31.578363	2015-07-22 18:49:09.239378
208	19934290	Retrospective study	Ogino et al., 2009, Clin. Cancer Res.	2015-07-22 18:47:31.668232	2015-07-22 18:49:09.661176
209	11208838	Retrospective study	Schiller et al., 2001, J. Clin. Oncol.	2015-07-22 18:47:31.714154	2015-07-22 18:49:10.075256
210	17070615		Gautschi et al., 2007, Lung Cancer	2015-07-22 18:47:31.969707	2015-07-22 18:49:10.491886
211	22479427		Carico et al., 2012, PLoS ONE	2015-07-22 18:47:32.055828	2015-07-22 18:49:10.912952
212	23898052		Logan et al., 2013, Anticancer Res.	2015-07-22 18:47:32.186956	2015-07-22 18:49:11.329057
213	24662454		Douillard et al., 2014, J Thorac Oncol	2015-07-22 18:47:32.306483	2015-07-22 18:49:11.746987
214	19933982	Retrospective study	Dubbink et al., 2009, Neurology	2015-07-22 18:47:32.369173	2015-07-22 18:49:12.163018
215	20127344	Retrospective study	Bleeker et al., 2010, Acta Neuropathol.	2015-07-22 18:47:32.429987	2015-07-22 18:49:12.580844
216	21596855		Green et al., 2011, Blood	2015-07-22 18:47:32.496949	2015-07-22 18:49:12.999142
217	21995386		Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-07-22 18:47:32.596857	2015-07-22 18:49:13.418416
218	16309541		Thomas et al., 2005, Int J Exp Pathol	2015-07-22 18:47:32.859779	2015-07-22 18:49:13.79159
219	17891190		Jares et al., 2007, Nat. Rev. Cancer	2015-07-22 18:47:32.931747	2015-07-22 18:49:14.260869
220	10547574		Takano et al., 1999, J. Pathol.	2015-07-22 18:47:32.987188	2015-07-22 18:49:14.773302
221	12432043		Keyomarsi et al., 2002, N. Engl. J. Med.	2015-07-22 18:47:33.166166	2015-07-22 18:49:15.905906
222	10224221		Donnellan et al., 1999, FASEB J.	2015-07-22 18:47:33.220258	2015-07-22 18:49:16.340854
223	23111194		Bradly et al., 2012, Diagn. Mol. Pathol.	2015-07-22 18:47:33.364279	2015-07-22 18:49:16.8692
224	24623981		Ding et al., 2014, Onco Targets Ther	2015-07-22 18:47:33.477236	2015-07-22 18:49:17.211113
225	24729716		Li et al., 2014, Onco Targets Ther	2015-07-22 18:47:33.540654	2015-07-22 18:49:17.574523
226	11585760	Retrospective study	Whitman et al., 2001, Cancer Res.	2015-07-22 18:47:33.655189	2015-07-22 18:49:18.013809
227	17940205	Retrospective study	Whitman et al., 2008, Blood	2015-07-22 18:47:33.711393	2015-07-22 18:49:18.429716
228	20494930	Retrospective study	Thol et al., 2010, Haematologica	2015-07-22 18:47:33.759063	2015-07-22 18:49:18.865621
229	18528420	Retrospective study	Chng et al., 2008, Leukemia	2015-07-22 18:47:33.918893	2015-07-22 18:49:19.265461
230	15597105	Metaanalysis	Mascaux et al., 2005, Br. J. Cancer	2015-07-22 18:47:33.979693	2015-07-22 18:49:19.645713
231	20007775		Westhoff et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-07-22 18:47:34.075924	2015-07-22 18:49:20.007514
232	22357840		Liao et al., 2012, Clin. Cancer Res.	2015-07-22 18:47:34.270385	2015-07-22 18:49:20.41467
233	16489069		Olivier et al., 2006, Clin. Cancer Res.	2015-07-22 18:47:34.51859	2015-07-22 18:49:20.743283
234	9569050		Berns et al., 1998, Br. J. Cancer	2015-07-22 18:47:34.649565	2015-07-22 18:49:21.145208
235	23861105		Wu et al., 2013, Am. J. Hematol.	2015-07-22 18:47:35.165524	2015-07-22 18:49:21.562412
236	23524406	case report	Rudin et al., 2013, J Thorac Oncol	2015-07-22 18:47:35.279944	2015-07-22 18:49:21.982541
237	24186003		Lilljebjörn et al., 2014, Leukemia	2015-07-22 18:47:35.351231	2015-07-22 18:49:22.398563
238	24578576	Retrospective study	Honeyman et al., 2014, Science	2015-07-22 18:47:35.467155	2015-07-22 18:49:22.818124
239	12075054	Retrospective Study	Rosenwald et al., 2002, N. Engl. J. Med.	2015-07-22 18:47:35.575208	2015-07-22 18:49:23.199827
240	18765795	Retrospective Study	Lenz et al., 2008, Proc. Natl. Acad. Sci. U.S.A.	2015-07-22 18:47:35.658999	2015-07-22 18:49:23.580037
241	21343560	Retrospective study	Gaidzik et al., 2011, J. Clin. Oncol.	2015-07-22 18:47:35.821606	2015-07-22 18:49:24.07113
242	25670080		Vidal et al., 2015, Cancer Cell	2015-07-22 18:47:35.897821	2015-07-22 18:49:24.416646
243	16818686	Retrospective study	Jackman et al., 2006, Clin. Cancer Res.	2015-07-22 18:47:35.965676	2015-07-22 18:49:24.905689
244	25670080	PDX	Vidal et al., 2015, Cancer Cell	2015-07-22 18:47:36.04477	2015-07-22 18:49:25.324648
245	22430270	Retrospective Study	Gaidzik et al., 2012, J. Clin. Oncol.	2015-07-22 18:47:36.119635	2015-07-22 18:49:25.741237
246	19157502	Retrospective Study	Mendiola et al., 2009, Hum. Pathol.	2015-07-22 18:47:36.172073	2015-07-22 18:49:26.159604
247	19666869	Retrospective study	Kosmider et al., 2009, Blood	2015-07-22 18:47:36.217995	2015-07-22 18:49:26.569215
248	18464292	Retrospective Study	Patard et al., 2008, Int. J. Cancer	2015-07-22 18:47:36.272504	2015-07-22 18:49:26.94531
249	23620406	Retrospective Study	Hakimi et al., 2013, Clin. Cancer Res.	2015-07-22 18:47:36.375285	2015-07-22 18:49:27.359773
250	21343549	Retrospective Study	Metzeler et al., 2011, J. Clin. Oncol.	2015-07-22 18:47:36.42859	2015-07-22 18:49:27.746483
251	9207459	Retrospective Study	Gascoyne et al., 1997, Blood	2015-07-22 18:47:36.549847	2015-07-22 18:49:28.143672
252	20601954	Retrospective Study	Nikoloski et al., 2010, Nat. Genet.	2015-07-22 18:47:36.652136	2015-07-22 18:49:28.560262
253	21828143	Retrospective study	Chou et al., 2011, Blood	2015-07-22 18:47:36.712827	2015-07-22 18:49:28.978503
254	11793370	Retrospective Study	Schraml et al., 2002, J. Pathol.	2015-07-22 18:47:36.807709	2015-07-22 18:49:29.398038
255	9552022	Retrospective study	de Alava et al., 1998, J. Clin. Oncol.	2015-07-22 18:47:36.94396	2015-07-22 18:49:29.8262
\.


--
-- Data for Name: variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variants (id, gene_id, name, description, created_at, updated_at, deleted, deleted_at, genome_build, chromosome, start, stop, reference_bases, variant_bases, representative_transcript, chromosome2, start2, stop2, reference_build) FROM stdin;
1	4	BCR-ABL	The BCR-ABL fusion protein, commonly referred to as the philadelphia chromosome, is the most well-studied fusion gene in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML), but despite its ability initiate disease in mice, its status an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and nilotinib) have seen some success in delivering a tumor response.	2015-07-22 18:47:10.126202	2015-07-22 18:47:10.126202	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
2	4	BCR-ABL T315I	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-07-22 18:47:10.165055	2015-07-22 18:47:10.165055	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
3	4	BCR-ABL E255K	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-07-22 18:47:10.188981	2015-07-22 18:47:10.188981	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
5	1	EML4-ALK	The EML4-ALK fusion has been seen in non-small cell lung cancer, and appears to be an alternative mechanism for ALK activation. Cells with this fusion have been shown to be sensitive to the ALK inhibitor crizotinib.	2015-07-22 18:47:10.246757	2015-07-22 18:47:10.246757	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
6	1	EML4-ALK C1156Y	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-07-22 18:47:10.272301	2015-07-22 18:47:10.272301	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
7	1	EML4-ALK L1196M	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-07-22 18:47:10.298591	2015-07-22 18:47:10.298591	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
13	5	V600E/V600M	A case study of a single patient harboring both a V600E and a V600M mutation, dabrafenib was shown to acheive clinical response.	2015-07-22 18:47:10.426656	2015-07-22 18:47:10.426656	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
14	5	V600E AMPLIFICATION	Amplification of BRAF V600E has been shown to confer resistance to MEK inhibitors. For more information on the V600 locus, see the V600E entry.	2015-07-22 18:47:10.454595	2015-07-22 18:47:10.454595	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
17	5	V600	BRAF mutations of the valine 600 residue have been shown to be recurrent in many cancer types. Of the V600 mutations, V600E is the most widely researched. V600 mutations as a whole have been correlated to poorer prognosis in colorectal and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual mutation pages on the left sidebar.	2015-07-22 18:47:10.515277	2015-07-22 18:47:10.515277	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
18	8	AMPLIFICATION	CCND1 amplification has been implicated in poorer prognosis in non-small cell lung cancer.	2015-07-22 18:47:10.534137	2015-07-22 18:47:10.534137	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
19	8	EXPRESSION	CCND1 expression, and its prognositc impact, is still in dispute. Three experiments in non-small cell lung cancer have shown it to have no impact on survival, but three additional studies have shown it results in poorer prognosis. There is also some ambiguity in how the boundaries between expression and overexpression are defined.	2015-07-22 18:47:10.556435	2015-07-22 18:47:10.556435	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
20	8	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-07-22 18:47:10.575886	2015-07-22 18:47:10.575886	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
21	9	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-07-22 18:47:10.596198	2015-07-22 18:47:10.596198	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
22	9	PROMOTER DEMETHYLATION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-07-22 18:47:10.619112	2015-07-22 18:47:10.619112	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
23	10	LOSS	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL), a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic palbociclib significantly increased the median survival of a Notch-driven model of T-ALL.	2015-07-22 18:47:10.65182	2015-07-22 18:47:10.65182	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
24	11	OVEREXPRESSION	Cyclin E, while currenly not as widely implicated as its counterpart, cyclin D, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, are correlated with later stage disease and have been shown to lead to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	2015-07-22 18:47:10.683945	2015-07-22 18:47:10.683945	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
25	13	EXPRESSION	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-07-22 18:47:10.705781	2015-07-22 18:47:10.705781	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
26	12	EXPRESSION	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-07-22 18:47:10.72508	2015-07-22 18:47:10.72508	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
27	14	PROMOTER HYPERMETHYLATION	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the CDKN2A promoter region. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcomes for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	2015-07-22 18:47:10.746919	2015-07-22 18:47:10.746919	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
28	15	N-TERMINAL FRAME SHIFT	CEBPA N-terminal frame shift mutations that result in a premature stop codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML). CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-07-22 18:47:10.76936	2015-07-22 18:47:10.76936	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
29	15	MUTATION	CEBPA mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-07-22 18:47:10.788624	2015-07-22 18:47:10.788624	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
30	33	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-07-22 18:47:10.811194	2015-07-22 18:47:10.811194	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
31	16	DNAJB1-PRKACA	This fusion has been found to be very recurrent in the rare form of adolescent liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-07-22 18:47:10.833674	2015-07-22 18:47:10.833674	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
32	18	R882	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-07-22 18:47:10.861092	2015-07-22 18:47:10.861092	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
74	30	EXON 1 MUTATIONS	A study by Lièvre et al in 2006 showed that colorectal cancer patients with KRAS exon 1 mutations had low cetuximab response rates.	2015-07-22 18:47:11.926866	2015-07-22 18:47:11.926866	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
37	20	DEL 755-759	ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-22 18:47:10.969871	2015-07-22 18:47:10.969871	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
40	20	L755W	ERBB2 L755W was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-22 18:47:11.033366	2015-07-22 18:47:11.033366	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
41	20	P780INS	ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-22 18:47:11.058649	2015-07-22 18:47:11.058649	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
42	20	R678Q	ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-22 18:47:11.078533	2015-07-22 18:47:11.078533	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
75	30	EXON 2 MUTATIONS	In a study by Pao et al in 2005, non-small cell lung cancer patients harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors gefinitib and erlotinib.	2015-07-22 18:47:11.963392	2015-07-22 18:47:11.963392	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
46	21	L536Q	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of these ligand binding domain mutations, and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-07-22 18:47:11.197974	2015-07-22 18:47:11.197974	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
47	21	N538G	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-07-22 18:47:11.22261	2015-07-22 18:47:11.22261	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
51	22	P235R		2015-07-22 18:47:11.36035	2015-07-22 18:47:11.36035	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
52	22	FGFR2-MGEA5	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. Additionally, tumor size reduction was achieved by both ponatinib and pazopanib treatments administered separately in a single patient with intrahepatic cholangiocarcinoma and this fusion. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-07-22 18:47:11.384421	2015-07-22 18:47:11.384421	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
53	23	FGFR3-BAIAP2L1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-07-22 18:47:11.417861	2015-07-22 18:47:11.417861	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
54	23	FGFR3-TACC3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-07-22 18:47:11.446216	2015-07-22 18:47:11.446216	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
55	24	ITD MUTATIONS	FLT3-ITD (internal tandem duplications) frequently occur in patients with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated sequence resulting in an in-frame duplication event. The length of these duplications can vary widely which may have prognostic consequences, but this has not been conclusively determined. FLT3-ITD mutations overall have generally been associated with poor prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the prognosis associated with this variant.	2015-07-22 18:47:11.47324	2015-07-22 18:47:11.47324	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
56	24	TKD MUTATIONS	FLT3 tyrosine kinase domain mutations (FLT3-TKD) are much less common than FLT3-ITD (internal tandem duplication) mutations and may not confer the same prognostic impact. Although the majority of mutations are point mutations effecting D835 (most frequently D835Y), a small proportion involve an in-frame deletion of I836. These mutations are within the activation loop of the second tyrosine kinase domain of FLT3 and thought to result in constitutive activation of the receptor.	2015-07-22 18:47:11.497597	2015-07-22 18:47:11.497597	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
57	25	EXPRESSION	GATA2 misregulation has been observed in a number of hematologic malignancies, as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung cancer.	2015-07-22 18:47:11.52575	2015-07-22 18:47:11.52575	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
76	30	G12	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-07-22 18:47:11.987727	2015-07-22 18:47:11.987727	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
58	26	R132	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-07-22 18:47:11.55419	2015-07-22 18:47:11.55419	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
62	27	R140Q/L	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival as compared to their wild-type counterparts. In myelodysplastic syndromes, however, no prognostic link was discovered between IDH2 mutation status and overall survival.	2015-07-22 18:47:11.648021	2015-07-22 18:47:11.648021	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
66	29	EXON 11 MUTATIONS	c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-07-22 18:47:11.748756	2015-07-22 18:47:11.748756	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
67	29	INTERNAL DUPLICATION	c-KIT internal duplications have been observed in exon 11, within the juxtamembrane domain. In a case study of an anal melanoma patient harboring this event, imatinib confered marked response. Also, cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-07-22 18:47:11.77329	2015-07-22 18:47:11.77329	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
68	29	EXON 13 MUTATIONS	c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations. These mutations lie within the tyrosine kinase 1 domain, and are found primarily in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations show sensitivity to imatinib and sunitinib treatment. However, only imatinib has seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant cases.	2015-07-22 18:47:11.793913	2015-07-22 18:47:11.793913	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
69	29	EXON 14 MUTATIONS	c-KIT exon 14 mutations lie within the tyrosine kinase domain of the protein. While relatively rare in primary gastrointestinal tumors, they are notably more prevalent in refractory disease, suggesting a role in imatinib resistance. Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.	2015-07-22 18:47:11.814072	2015-07-22 18:47:11.814072	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
70	29	EXON 17 MUTATIONS	c-KIT exon 17 mutations lie within the TK2 domain, containing the activation loop of the protein. In cell lines, mutations within this domain have been shown to be sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.	2015-07-22 18:47:11.836188	2015-07-22 18:47:11.836188	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
71	29	EXON 9 MUTATIONS	c-KIT exon 9 mutations lie within the dimerization motif of the protein. Relative to other KIT mutations, exon 9 mutations have been associated with better overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and in vivo.	2015-07-22 18:47:11.863978	2015-07-22 18:47:11.863978	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
72	29	L576P	KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain. It is one of the most recurrent KIT mutations in melanoma, and both in vitro and in vivo studies have shown sensitivity to imatinib.	2015-07-22 18:47:11.885718	2015-07-22 18:47:11.885718	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
77	30	G12/G13	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-07-22 18:47:12.017754	2015-07-22 18:47:12.017754	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
80	30	G13	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-07-22 18:47:12.103223	2015-07-22 18:47:12.103223	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
84	32	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-07-22 18:47:12.213962	2015-07-22 18:47:12.213962	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
85	34	PROMOTER METHYLATION	MGMT promoter methylation has been observed to impact tumor progression in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating agent carmustine has shown efficacy. In patients lacking methylation, combining carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more experiments are required. Clinical trials have also shown selective sensitivity of promoter methylation-positive patients to temozolomide, making a case for wider methylation screening in GBM patients.	2015-07-22 18:47:12.236727	2015-07-22 18:47:12.236727	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
86	35	EXON 12 MUTATIONS	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-07-22 18:47:12.276265	2015-07-22 18:47:12.276265	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
88	36	EXON 1 MUTATIONS	NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown to be correlated with poorer overall survival relative to wild-type NRAS in melanoma patients.	2015-07-22 18:47:12.316309	2015-07-22 18:47:12.316309	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
89	36	EXON 2 MUTATIONS	NRAS exon 2 mutations have been shown to be correlated with poorer overall survival in melanoma patients and colorectal cancer patients, however no prognostic impact was seen in acute myeloid leukemia patients.	2015-07-22 18:47:12.33916	2015-07-22 18:47:12.33916	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
90	36	EXON 3 MUTATIONS	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-07-22 18:47:12.361883	2015-07-22 18:47:12.361883	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
91	36	EXON 4 MUTATIONS	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-07-22 18:47:12.380696	2015-07-22 18:47:12.380696	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
92	36	G12	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-07-22 18:47:12.399452	2015-07-22 18:47:12.399452	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
94	36	Q61	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-07-22 18:47:12.441783	2015-07-22 18:47:12.441783	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
111	40	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and Arsenic Trioxide for treating these patients, and early results seem promising.	2015-07-22 18:47:12.90835	2015-07-22 18:47:12.90835	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
97	38	V536E		2015-07-22 18:47:12.510719	2015-07-22 18:47:12.510719	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
101	38	DEL I843	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-07-22 18:47:12.595903	2015-07-22 18:47:12.595903	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
102	38	DI842-843IM	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-07-22 18:47:12.619673	2015-07-22 18:47:12.619673	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
105	37	EXON 20 MUTATIONS	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-07-22 18:47:12.700054	2015-07-22 18:47:12.700054	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
106	37	EXON 9 MUTATIONS	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-07-22 18:47:12.740817	2015-07-22 18:47:12.740817	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
108	39	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	2015-07-22 18:47:12.809235	2015-07-22 18:47:12.809235	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
109	17	DNAJB1-PRKACA	This fusion has been found to be very recurrent in a rare form of adolescent liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined, and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-07-22 18:47:12.855829	2015-07-22 18:47:12.855829	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
123	45	V173G/A	While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-07-22 18:47:13.182799	2015-07-22 18:47:13.182799	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
124	46	FRAMESHIFT TRUNCATION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-07-22 18:47:13.209936	2015-07-22 18:47:13.209936	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
125	46	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-07-22 18:47:13.230226	2015-07-22 18:47:13.230226	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
126	47	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-07-22 18:47:13.250949	2015-07-22 18:47:13.250949	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
4	2	E17K	AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal, lung, and ovarian cancer. It has been convincingly shown to be an activating mutation resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and other AKT1 mutations, are the subject of much research and development for therapeutics.	2015-07-22 18:47:10.217461	2015-07-22 18:49:40.116941	f	\N	\N	14	105246551	105246551	C	T	\N	\N	\N	\N	\N
127	48	Q157P/R	U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is less common than the S34F mutation, occurs in the second zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-07-22 18:47:13.276623	2015-07-22 18:47:13.276623	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
128	48	S34Y/F	U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is the most commonly identified variant in MDS, occurs in the first zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-07-22 18:47:13.299896	2015-07-22 18:47:13.299896	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
129	49	EXON 7 MUTATIONS	WT1 exon 7 mutations have been shown to be recurrent in acute myeloid leukemia. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 7 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-07-22 18:47:13.321173	2015-07-22 18:47:13.321173	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
130	49	EXON 9 MUTATIONS	WT1 exon 9 mutations have been shown to be recurrent in acute myeloid leukemia, although at a less frequent rate than their exon 7 counterparts. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 9 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-07-22 18:47:13.34226	2015-07-22 18:47:13.34226	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
131	6	LOSS-OF-FUNCTION	BRCA1 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.	2015-07-22 18:47:13.377435	2015-07-22 18:47:13.377435	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
132	7	LOSS-OF-FUNCTION	BRCA2 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.	2015-07-22 18:47:13.422366	2015-07-22 18:47:13.422366	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
133	19	EXON 19 DELETION	Deletions within exon 19 of EGFR are most common in lung cancer. These deletions, in non-small cell lung cancer, have been shown to be sensitive to the EGFR tyrosine kinase inhibitors gefitinib and erlotinib. There is also data to suggest that this event is a good prognostic marker in lung adenocarcinoma.	2015-07-22 18:47:13.447573	2015-07-22 18:47:13.447573	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
134	19	G719S	While not as recurrent as the L858R mutation, EGFR G719S has also been shown to be an activating mutation. In lung cancer cell lines, it also confers sensitivity to the tyrosine kinase inhibitors gefitinib and erlotinib.	2015-07-22 18:47:13.476845	2015-07-22 18:47:13.476845	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
135	50	D1643H	Activating mutations in NOTCH1, including D1643H, have been shown to be poor prognostic markers in lung cancer.	2015-07-22 18:47:13.501282	2015-07-22 18:47:13.501282	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
136	50	R2328W	Activating mutations in NOTCH1, including R2328W, have been shown to be poor prognostic markers in lung cancer.	2015-07-22 18:47:13.533943	2015-07-22 18:47:13.533943	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
137	50	V2444FS	Activating mutations in NOTCH1, including a frameshift insertion at V2444, have been shown to be poor prognostic markers in lung cancer.	2015-07-22 18:47:13.556062	2015-07-22 18:47:13.556062	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
138	50	S2275FS	Activating mutations in NOTCH1, including a frameshift insertion at S2275, have been shown to be poor prognostic markers in lung cancer.	2015-07-22 18:47:13.580438	2015-07-22 18:47:13.580438	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
139	51	L63V	Activating mutations in DDR2, including L63V, has been shown to be sensitive to dasatinib in cell lines.	2015-07-22 18:47:13.608342	2015-07-22 18:47:13.608342	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
140	51	L239R	Activating mutations in DDR2, including L239R, has been shown to be sensitive to dasatinib in cell lines.	2015-07-22 18:47:13.63379	2015-07-22 18:47:13.63379	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
141	51	G253C	Activating mutations in DDR2, including G253C, has been shown to be sensitive to dasatinib in cell lines.	2015-07-22 18:47:13.660399	2015-07-22 18:47:13.660399	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
142	51	G505S	Activating mutations in DDR2, including G505S, has been shown to be sensitive to dasatinib in cell lines.	2015-07-22 18:47:13.685102	2015-07-22 18:47:13.685102	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
143	51	I638F	DDR2 I638F has been shown to be a loss of function mutation, but also confers sensitivity to dasatinib in cell lines.	2015-07-22 18:47:13.711116	2015-07-22 18:47:13.711116	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
144	51	G774V	Activating mutations in DDR2, including G774V, has been shown to be sensitive to dasatinib in cell lines.	2015-07-22 18:47:13.736087	2015-07-22 18:47:13.736087	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
145	51	S768R	Activating mutations in DDR2, including S768R, has been shown to be sensitive to dasatinib in cell lines.	2015-07-22 18:47:13.761487	2015-07-22 18:47:13.761487	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
146	52	AMPLIFICATION	MET amplification, like EGFR T790M, has been shown to be capable of driving acquired resistance to dacomitinib in patients with lung adenocarcinoma.	2015-07-22 18:47:13.787647	2015-07-22 18:47:13.787647	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
147	30	G12V	KRAS G12V, like EGFR T790M, has been shown to be capable of driving acquired resistance to tyrosine kinase inhibitors in lung adenocarcinoma.	2015-07-22 18:47:13.826326	2015-07-22 18:47:13.826326	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
148	30	G12A	KRAS G12A, like EGFR T790M, has been shown to be capable of driving acquired resistance to tyrosine kinase inhibitors in lung adenocarcinoma.	2015-07-22 18:47:13.850277	2015-07-22 18:47:13.850277	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
149	54	EWS-FLI1		2015-07-22 18:47:13.882497	2015-07-22 18:47:13.882497	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
150	53	EWS-FLI1		2015-07-22 18:47:13.907059	2015-07-22 18:47:13.907059	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
151	59	T(14;18)		2015-07-22 18:47:13.928894	2015-07-22 18:47:13.928894	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
152	59	EXPRESSION		2015-07-22 18:47:13.953289	2015-07-22 18:47:13.953289	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
153	56	AMPLIFICATION		2015-07-22 18:47:13.975262	2015-07-22 18:47:13.975262	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
154	57	AMPLIFICATION		2015-07-22 18:47:13.998851	2015-07-22 18:47:13.998851	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
155	43	MUTATION		2015-07-22 18:47:14.036692	2015-07-22 18:47:14.036692	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
156	60	OVEREXPRESSION		2015-07-22 18:47:14.054999	2015-07-22 18:47:14.054999	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
157	55	MUTATION		2015-07-22 18:47:14.078313	2015-07-22 18:47:14.078313	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
158	61	EXPRESSION		2015-07-22 18:47:14.101182	2015-07-22 18:47:14.101182	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
159	58	LOSS OF FUNCTION		2015-07-22 18:47:14.121477	2015-07-22 18:47:14.121477	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
160	58	MUTATION		2015-07-22 18:47:14.141277	2015-07-22 18:47:14.141277	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
161	62	MUTATION		2015-07-22 18:47:14.162596	2015-07-22 18:47:14.162596	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
162	57	EXPRESSION		2015-07-22 18:47:14.186205	2015-07-22 18:47:14.186205	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
163	63	MUTATION		2015-07-22 18:47:14.214459	2015-07-22 18:47:14.214459	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
164	53	TYPE 1 EWS-FLI1		2015-07-22 18:47:14.240244	2015-07-22 18:47:14.240244	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
8	1	F1174L	ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells.	2015-07-22 18:47:10.322233	2015-07-22 18:49:40.135159	f	\N	\N	2	29443695	29443695	G	T	\N	\N	\N	\N	\N
9	1	R1275Q	ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma, non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This, and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC cell lines.	2015-07-22 18:47:10.342437	2015-07-22 18:49:40.149631	f	\N	\N	2	29432664	29432664	C	T	\N	\N	\N	\N	\N
10	3	S214C	ARAF S214C has been found to be a recurrent oncogenic mutation in non-small cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this mutation, sorafenib also acheived near-complete clinical remission. This case has brought more interest to the variant from a research and clinical perspective.	2015-07-22 18:47:10.362432	2015-07-22 18:49:40.162233	f	\N	\N	X	47426121	47426121	C	G	\N	\N	\N	\N	\N
11	5	V600D	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-07-22 18:47:10.381977	2015-07-22 18:49:40.177927	f	\N	\N	7	140453135	140453136	CA	AT	\N	\N	\N	\N	\N
12	5	V600E	BRAF V600E has been shown to be recurrent in many cancer types. It is one of the most widely studied variants in cancer. This variant is correlated with poor prognosis in certain cancer types, including colorectal cancer and papillary thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib has also shown to be effective when combined with the MEK inhibitor trametinib in colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib, and a number of combination therapies have been successful in treating V600E mutations. While the drugs cetuximab and panitumumab have been largely shown to be ineffective without supplementary treatment.	2015-07-22 18:47:10.403269	2015-07-22 18:49:40.191432	f	\N	\N	7	140453136	140453136	A	T	\N	\N	\N	\N	\N
15	5	V600M	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-07-22 18:47:10.471968	2015-07-22 18:49:40.204851	f	\N	\N	7	140453137	140453137	C	T	\N	\N	\N	\N	\N
16	5	V600R	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-07-22 18:47:10.494041	2015-07-22 18:49:40.219498	f	\N	\N	7	140453136	140453137	AC	CT	\N	\N	\N	\N	\N
33	19	L858R	EGFR L858R has long been recognized as a functionally significant mutation in cancer, and is one of the most prevalent single mutations in lung cancer. Best described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity to first and second generation TKI's like gefitinib and neratinib. NSCLC patients with this mutation treated with TKI's show increased overall and progression-free survival, as compared to chemotherapy alone. Third generation TKI's are currently in clinical trials that specifically focus on mutant forms of EGFR, a few of which have shown efficacy in treating patients that failed to respond to earlier generation TKI therapies.	2015-07-22 18:47:10.883013	2015-07-22 18:49:40.23686	f	\N	\N	7	55259514	55259515	CT	AG	\N	\N	\N	\N	\N
34	19	T790M	EGFR T790M was one of the very first mutations recognized to confer resistance to targeted therapies in non-small cell lung cancer. While successful in amplified EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients harboring this mutation before treatment is notably lower. This lack of efficacy can likely be to blame for the poorer prognosis for patients with this mutation as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired resistance to TKI inhibition have been shown to harbor this mutation, implicating it as a mechanism of acquired therapy resistence. The third generation TKI's are being developed with this resistance problem in mind, and early stage experiments have shown some efficacy of these drugs in previously resistant tumors with mutant EGFR.	2015-07-22 18:47:10.903263	2015-07-22 18:49:40.253099	f	\N	\N	7	55249071	55249071	C	T	\N	\N	\N	\N	\N
35	20	D769H	ERBB2 D769H was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-22 18:47:10.926071	2015-07-22 18:49:40.267587	f	\N	\N	17	37880261	37880261	G	C	\N	\N	\N	\N	\N
36	20	D769Y	ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-22 18:47:10.947322	2015-07-22 18:49:40.286764	f	\N	\N	17	37880261	37880261	G	T	\N	\N	\N	\N	\N
38	20	G309A	ERBB2 G309A was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-22 18:47:10.989321	2015-07-22 18:49:40.301987	f	\N	\N	17	37868205	37868205	G	C	\N	\N	\N	\N	\N
39	20	L755S	ERBB2 L755S was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike many of the other variants queried. This mutation was also shown to confer resistance to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.	2015-07-22 18:47:11.01182	2015-07-22 18:49:40.32717	f	\N	\N	17	37880220	37880220	T	C	\N	\N	\N	\N	\N
43	20	R896C	ERBB2 R896C was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-22 18:47:11.102823	2015-07-22 18:49:40.348791	f	\N	\N	17	37881616	37881616	C	T	\N	\N	\N	\N	\N
44	20	V777L	ERBB2 V777L was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-22 18:47:11.127415	2015-07-22 18:49:40.36942	f	\N	\N	17	37881000	37881000	G	T	\N	\N	\N	\N	\N
45	20	V842I	ERBB2 V842I was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-22 18:47:11.172914	2015-07-22 18:49:40.388542	f	\N	\N	17	37881332	37881332	G	A	\N	\N	\N	\N	\N
48	21	Y537C	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-07-22 18:47:11.253867	2015-07-22 18:49:40.413299	f	\N	\N	6	152419923	152419923	A	G	\N	\N	\N	\N	\N
49	21	Y537N	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-07-22 18:47:11.286146	2015-07-22 18:49:40.434453	f	\N	\N	6	152419922	152419922	T	A	\N	\N	\N	\N	\N
50	21	Y537S	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-07-22 18:47:11.319729	2015-07-22 18:49:40.456638	f	\N	\N	6	152419923	152419923	A	C	\N	\N	\N	\N	\N
59	26	R132C	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-07-22 18:47:11.581347	2015-07-22 18:49:40.480352	f	\N	\N	2	209113113	209113113	G	A	\N	\N	\N	\N	\N
60	26	R132H	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-07-22 18:47:11.602804	2015-07-22 18:49:40.499289	f	\N	\N	2	209113112	209113112	C	T	\N	\N	\N	\N	\N
61	26	R132L	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-07-22 18:47:11.626326	2015-07-22 18:49:40.519769	f	\N	\N	2	209113112	209113112	C	A	\N	\N	\N	\N	\N
63	27	R172K	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic syndromes, studies did not find a prognostic association between this variant and patient outcomes.	2015-07-22 18:47:11.671145	2015-07-22 18:49:40.542991	f	\N	\N	15	90631838	90631838	C	T	\N	\N	\N	\N	\N
64	28	V617F	JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases, occuring in around half of all MPD's. While less associated with cancer, when it is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The V617F mutation is an activating mutation, resulting in increased kinase activity. The mutation seems to be restricted to hematologic malignancies. Treatment of JAK mutant diseases with ruxolitinib has seen some clinical success.	2015-07-22 18:47:11.694682	2015-07-22 18:49:40.563811	f	\N	\N	9	5073770	5073770	G	T	\N	\N	\N	\N	\N
65	29	D816V	KIT D816V is a mutation observed in acute myeloid leukemia (AML). This variant has been linked to poorer prognosis and worse outcome in AML patients.	2015-07-22 18:47:11.728367	2015-07-22 18:49:40.586058	f	\N	\N	4	55599321	55599321	A	T	\N	\N	\N	\N	\N
73	29	V654A	KIT V654A is an exon 13 mutation that lies within the tyrosine kinase 1 domain of the protein. It has been shown to be an activating mutation by in vitro studies. This mutation is associated with imatinib resistance in melanoma patients. However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have seen success in acheiving tumor response.	2015-07-22 18:47:11.90637	2015-07-22 18:49:40.608243	f	\N	\N	4	55594258	55594258	T	C	\N	\N	\N	\N	\N
78	30	G12C	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-07-22 18:47:12.044516	2015-07-22 18:49:40.643854	f	\N	\N	12	25398285	25398285	C	A	\N	\N	\N	\N	\N
79	30	G12D	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-07-22 18:47:12.074114	2015-07-22 18:49:40.838622	f	\N	\N	12	25398284	25398284	C	T	\N	\N	\N	\N	\N
81	30	G13D	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-07-22 18:47:12.1357	2015-07-22 18:49:40.854183	f	\N	\N	12	25398281	25398281	C	T	\N	\N	\N	\N	\N
82	31	P124S	MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder and colon cancer to a lesser degree. The P124S mutation has been shown to contribute to AZD6244 resistance in melanoma cell lines, but considerably less so than its Q56P counterpart.	2015-07-22 18:47:12.163737	2015-07-22 18:49:40.870562	f	\N	\N	15	66729162	66729162	C	T	\N	\N	\N	\N	\N
83	31	Q56P	MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This mutation has been shown to confer considerable resistance to AZD6244 treatment of melanoma cell lines.	2015-07-22 18:47:12.1912	2015-07-22 18:49:40.887624	f	\N	\N	15	66727451	66727451	A	C	\N	\N	\N	\N	\N
87	35	W288FS	NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most common NPM1 mutation identified in acute myeloid leukemia. This mutation results in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation, it is generally grouped with other exon 12 mutations for patient analysis (see NPM1 Exon 12 variants for more information).	2015-07-22 18:47:12.296465	2015-07-22 18:49:40.910096	f	\N	\N	5	170837546	170837547	0	TCAG	\N	\N	\N	\N	\N
93	36	G13D	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-07-22 18:47:12.41886	2015-07-22 18:49:40.937285	f	\N	\N	1	115258744	115258744	C	T	\N	\N	\N	\N	\N
95	36	Q61L	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-07-22 18:47:12.47096	2015-07-22 18:49:40.958318	f	\N	\N	1	115256528	115256529	TT	CA	\N	\N	\N	\N	\N
96	36	Q61R	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-07-22 18:47:12.49055	2015-07-22 18:49:40.978226	f	\N	\N	1	115256528	115256529	TT	CC	\N	\N	\N	\N	\N
98	38	D842I	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-07-22 18:47:12.529743	2015-07-22 18:49:40.994381	f	\N	\N	4	55152092	55152093	GA	AT	\N	\N	\N	\N	\N
99	38	D842V	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-07-22 18:47:12.550418	2015-07-22 18:49:41.011837	f	\N	\N	4	55152093	55152093	A	T	\N	\N	\N	\N	\N
100	38	D842Y	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-07-22 18:47:12.574591	2015-07-22 18:49:41.031804	f	\N	\N	4	55152092	55152092	G	T	\N	\N	\N	\N	\N
103	37	E542K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-07-22 18:47:12.642558	2015-07-22 18:49:41.047961	f	\N	\N	3	178936082	178936082	G	A	\N	\N	\N	\N	\N
104	37	E545K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-07-22 18:47:12.67451	2015-07-22 18:49:41.065958	f	\N	\N	3	178936091	178936091	G	A	\N	\N	\N	\N	\N
107	37	H1047R	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-07-22 18:47:12.783065	2015-07-22 18:49:41.085787	f	\N	\N	3	178952085	178952085	A	G	\N	\N	\N	\N	\N
110	41	R233*	PTEN R233* has been shown to be a loss of function mutation, and PTEN loss has been the subject of considerable research in breast cancer. PTEN loss may sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data to support that PTEN loss is both associated with poorer prognosis, and no change in prognosis.	2015-07-22 18:47:12.880669	2015-07-22 18:49:41.105488	f	\N	\N	10	89717672	89717672	C	T	\N	\N	\N	\N	\N
112	42	C634W	RET C639W has been implicated as an alternate mechanism of activating RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the C639W mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib.	2015-07-22 18:47:12.936623	2015-07-22 18:49:41.125049	f	\N	\N	10	43609950	43609950	C	G	\N	\N	\N	\N	\N
113	42	M918T	RET M819T is the most common somatically acquired mutation in medullary thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the M918T mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T leads to more aggressive MTC with a poorer prognosis.	2015-07-22 18:47:12.959587	2015-07-22 18:49:41.142504	f	\N	\N	10	43617416	43617416	T	C	\N	\N	\N	\N	\N
114	44	K666N	SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients.	2015-07-22 18:47:12.982538	2015-07-22 18:49:41.158964	f	\N	\N	2	198267359	198267359	C	A	\N	\N	\N	\N	\N
115	44	K700E	SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients. Additionally, these mutations are the most common SF3B1 mutation observed in MDS and highly associated with subtypes of MDS that are defined by ringed sideroblasts.	2015-07-22 18:47:13.002558	2015-07-22 18:49:41.181679	f	\N	\N	2	198266834	198266834	T	C	\N	\N	\N	\N	\N
116	45	R175H	While loss-of-function events in TP53 are very common in cancer, the R175H variant seems not only to result in loss of tumor-suppression, but also acts as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell lines harboring this mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been shown that the R175H mutation is correlated with worse overall survival than wild-type TP53, but is not as detrimental as the R248W variant.	2015-07-22 18:47:13.030354	2015-07-22 18:49:41.200192	f	\N	\N	17	7578406	7578406	C	T	\N	\N	\N	\N	\N
117	45	R248Q	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival. The R248Q mutation has also shown an increased invasive behavior in cell lines. This is specific to the 248Q variant.	2015-07-22 18:47:13.051859	2015-07-22 18:49:41.218554	f	\N	\N	17	7577538	7577538	C	T	\N	\N	\N	\N	\N
118	45	R248W	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival.	2015-07-22 18:47:13.077104	2015-07-22 18:49:41.237734	f	\N	\N	17	7577539	7577539	G	A	\N	\N	\N	\N	\N
119	45	R249T	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-07-22 18:47:13.098074	2015-07-22 18:49:41.255731	f	\N	\N	17	7577535	7577535	C	G	\N	\N	\N	\N	\N
120	45	R249W	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-07-22 18:47:13.119015	2015-07-22 18:49:41.274296	f	\N	\N	17	7577536	7577536	T	A	\N	\N	\N	\N	\N
121	45	R273C	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-07-22 18:47:13.1415	2015-07-22 18:49:41.292282	f	\N	\N	17	7577121	7577121	G	A	\N	\N	\N	\N	\N
122	45	R273H	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-07-22 18:47:13.161369	2015-07-22 18:49:41.310384	f	\N	\N	17	7577120	7577120	C	T	\N	\N	\N	\N	\N
\.


--
-- Data for Name: evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_items (id, description, disease_id, source_id, variant_id, created_at, updated_at, rating, status, remote_errors, remote_ids, variant_hgvs, evidence_level, evidence_type, variant_origin, evidence_direction, clinical_significance, deleted, deleted_at, drug_interaction_type) FROM stdin;
1	JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).	1	51	64	2015-07-22 18:47:14.406475	2015-07-22 18:47:14.406475	4	accepted	\N	\N	9:5073770-5073770 (G->T)	1	0	0	0	5	f	\N	\N
2	GIST tumors harboring PDGFRA D842V mutation are more likely to be benign than malignant.	2	52	99	2015-07-22 18:47:14.484386	2015-07-22 18:47:14.484386	3	accepted	\N	\N	4:55152093-55152093 (A->T)	1	0	0	0	5	f	\N	\N
3	DNMT3A R882 mutations occur most often in de novo AML patients with intermediate cytogenic risk.	3	53	32	2015-07-22 18:47:14.607997	2015-07-22 18:47:14.607997	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
4	Young AML patients (<60 years old) with DNMT3A mutations (60% of which were R882) were older in age, had higher white blood cell counts and had higher platelet counts than patients wildtype for DNMT3A	3	54	32	2015-07-22 18:47:14.670563	2015-07-22 18:47:14.670563	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
5	JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical CML).	4	51	64	2015-07-22 18:47:14.709547	2015-07-22 18:47:14.709547	4	accepted	\N	\N	9:5073770-5073770 (G->T)	1	0	0	0	4	f	\N	\N
6	JAK2 V617F is associated with myeloid neoplasms (AML, MDS, CMML/aCML)	4	51	64	2015-07-22 18:47:14.743426	2015-07-22 18:47:14.743426	4	accepted	\N	\N	9:5073770-5073770 (G->T)	1	0	0	0	4	f	\N	\N
7	The SNP rs10974944 in the Jak2 locus is associated with increased predisposition for JAK2 V617F mutation and its associated cancer	5	55	64	2015-07-22 18:47:14.78835	2015-07-22 18:47:14.78835	4	accepted	\N	\N	9:5073770-5073770 (G->T)	1	0	0	0	4	f	\N	\N
8	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	6	56	76	2015-07-22 18:47:14.829615	2015-07-22 18:47:14.829615	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
9	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	6	56	80	2015-07-22 18:47:14.862773	2015-07-22 18:47:14.862773	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
10	Melanoma associated with NRAS Q61 mutation was more often associated with those at the extremity than those at the trunk	7	57	94	2015-07-22 18:47:14.907117	2015-07-22 18:47:14.907117	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
11	Daunorubicin treatment resulted in similar overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	3	53	32	2015-07-22 18:47:14.944624	2015-07-22 18:47:14.944624	4	accepted	\N	\N	N/A	1	2	0	1	1	f	\N	\N
12	The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition by about 5 fold.	7	58	82	2015-07-22 18:47:15.000578	2015-07-22 18:47:15.000578	3	accepted	\N	\N	15:66729162-66729162 (C->T)	2	2	0	0	1	f	\N	\N
13	MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.	7	59	83	2015-07-22 18:47:15.04879	2015-07-22 18:47:15.04879	3	accepted	\N	\N	15:66727451-66727451 (A->C)	2	2	0	0	1	f	\N	\N
14	Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.	7	60	94	2015-07-22 18:47:15.096409	2015-07-22 18:47:15.096409	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
15	GIST cancer with D842V mutation is resistant to imatinib.	2	61	99	2015-07-22 18:47:15.160678	2015-07-22 18:47:15.160678	4	accepted	\N	\N	4:55152093-55152093 (A->T)	1	2	0	0	1	f	\N	\N
16	While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib, double mutation of V561D and D842V mutants are resistant to imatinib.	2	62	99	2015-07-22 18:47:15.219288	2015-07-22 18:47:15.219288	4	accepted	\N	\N	4:55152093-55152093 (A->T)	1	2	0	0	1	f	\N	\N
17	In one patient with S214C mutation, the use of sorafenib has led to more than 5 years of survival and near remission.	8	63	10	2015-07-22 18:47:15.285543	2015-07-22 18:47:15.285543	2	accepted	\N	\N	X:47426121-47426121 (C->G)	3	2	0	0	0	f	\N	\N
18	Idarubicin increases the overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	3	53	32	2015-07-22 18:47:15.328909	2015-07-22 18:47:15.328909	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
19	In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction in the percentage of cells harboring JAK2 V617F.	9	64	64	2015-07-22 18:47:15.375555	2015-07-22 18:47:15.375555	4	accepted	\N	\N	9:5073770-5073770 (G->T)	1	2	0	0	0	f	\N	\N
20	TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells	9	65	64	2015-07-22 18:47:15.435242	2015-07-22 18:47:15.435242	3	accepted	\N	\N	9:5073770-5073770 (G->T)	2	2	0	0	0	f	\N	\N
21	Likely due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic malignant melanoma with an NRAS G13D mutation	7	66	93	2015-07-22 18:47:15.493599	2015-07-22 18:47:15.493599	2	accepted	\N	\N	1:115258744-115258744 (C->T)	3	2	0	0	0	f	\N	\N
22	In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted in disease free survival of 14 months	7	67	95	2015-07-22 18:47:15.549378	2015-07-22 18:47:15.549378	2	accepted	\N	\N	1:115256528-115256529 (TT->CA)	3	2	0	0	0	f	\N	\N
23	In a melanoma patient with Q61R mutation, treatment with temozolomide resulted in overall survival of 16 months	7	67	96	2015-07-22 18:47:15.601135	2015-07-22 18:47:15.601135	2	accepted	\N	\N	1:115256528-115256529 (TT->CC)	3	2	0	0	0	f	\N	\N
24	There is no difference in the complete remission rate of de novo AML patients with DNMT3A mutation compared to those who are wild type for DNMT3A.	3	53	32	2015-07-22 18:47:15.646387	2015-07-22 18:47:15.646387	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
25	De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	3	68	32	2015-07-22 18:47:15.711091	2015-07-22 18:47:15.711091	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
26	In acute myloid leukemia patients, D816 mutation is associated with earlier relapse and poorer prognosis than wildtype KIT	10	69	65	2015-07-22 18:47:15.75392	2015-07-22 18:47:15.75392	4	accepted	\N	\N	4:55599321-55599321 (A->T)	1	1	0	0	3	f	\N	\N
27	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	11	70	89	2015-07-22 18:47:15.819352	2015-07-22 18:47:15.819352	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
28	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	11	70	90	2015-07-22 18:47:15.856446	2015-07-22 18:47:15.856446	3	accepted	\N	\N	N/A	0	1	0	0	3	f	\N	\N
29	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	11	70	91	2015-07-22 18:47:15.890581	2015-07-22 18:47:15.890581	3	accepted	\N	\N	N/A	0	1	0	0	3	f	\N	\N
30	BRAF mutations are associated with melanoma arising in non-chronic sun damaged skin and with superficial spreading melanoma	7	71	17	2015-07-22 18:47:15.934503	2015-07-22 18:47:15.934503	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
31	DNMT3A R882 mutations were associated with older age, higher white blood cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically normal AML patients	3	72	32	2015-07-22 18:47:15.982399	2015-07-22 18:47:15.982399	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
32	Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without this mutation.	12	73	8	2015-07-22 18:47:16.032133	2015-07-22 18:47:16.032133	3	accepted	\N	\N	2:29443695-29443695 (G->T)	2	2	0	0	1	f	\N	\N
33	Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without this mutation.	8	73	8	2015-07-22 18:47:16.072233	2015-07-22 18:47:16.072233	3	accepted	\N	\N	2:29443695-29443695 (G->T)	2	2	0	0	1	f	\N	\N
34	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	7	74	72	2015-07-22 18:47:16.150387	2015-07-22 18:47:16.150387	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	2
35	In NSCLC, Exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	8	75	75	2015-07-22 18:47:16.224131	2015-07-22 18:47:16.224131	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	2
36	Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation (primarily Q61) have a significantly lower response rate to cetuximab than patients wildtype for NRAS.	11	76	94	2015-07-22 18:47:16.293461	2015-07-22 18:47:16.293461	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
37	CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.	13	77	8	2015-07-22 18:47:16.343532	2015-07-22 18:47:16.343532	3	accepted	\N	\N	2:29443695-29443695 (G->T)	2	2	0	0	0	f	\N	\N
38	High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing the EML4-ALK fusion containing the F1174L mutation.	13	78	8	2015-07-22 18:47:16.389848	2015-07-22 18:47:16.389848	3	accepted	\N	\N	2:29443695-29443695 (G->T)	2	2	0	0	0	f	\N	\N
39	TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.	13	79	9	2015-07-22 18:47:16.431678	2015-07-22 18:47:16.431678	3	accepted	\N	\N	2:29432664-29432664 (C->T)	2	2	0	0	0	f	\N	\N
40	Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	8	63	10	2015-07-22 18:47:16.464993	2015-07-22 18:47:16.464993	3	accepted	\N	\N	X:47426121-47426121 (C->G)	2	2	0	0	0	f	\N	\N
41	Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	8	63	10	2015-07-22 18:47:16.504089	2015-07-22 18:47:16.504089	3	accepted	\N	\N	X:47426121-47426121 (C->G)	2	2	0	0	0	f	\N	\N
42	The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation of KIT but did not alter the binding affinity of dasatinib.	7	74	72	2015-07-22 18:47:16.542848	2015-07-22 18:47:16.542848	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
43	In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	2	80	98	2015-07-22 18:47:16.604615	2015-07-22 18:47:16.604615	4	accepted	\N	\N	4:55152092-55152093 (GA->AT)	2	2	0	0	0	f	\N	\N
44	In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	2	80	99	2015-07-22 18:47:16.660478	2015-07-22 18:47:16.660478	4	accepted	\N	\N	4:55152093-55152093 (A->T)	2	2	0	0	0	f	\N	\N
45	In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	2	80	100	2015-07-22 18:47:16.710141	2015-07-22 18:47:16.710141	4	accepted	\N	\N	4:55152092-55152092 (G->T)	2	2	0	0	0	f	\N	\N
46	In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	2	80	101	2015-07-22 18:47:16.75643	2015-07-22 18:47:16.75643	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
47	In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	2	80	102	2015-07-22 18:47:16.807096	2015-07-22 18:47:16.807096	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
48	The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.	13	79	9	2015-07-22 18:47:16.843596	2015-07-22 18:47:16.843596	3	accepted	\N	\N	2:29432664-29432664 (C->T)	2	2	0	0	1	f	\N	\N
49	In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5% of which were R882) had no prognostic value on overall, relapse free and event free survival.	3	81	32	2015-07-22 18:47:16.897723	2015-07-22 18:47:16.897723	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
50	In a large cohort of cytogenetically normal AML patients (18-60 years old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall, relapse free and event free survival	3	81	32	2015-07-22 18:47:16.927619	2015-07-22 18:47:16.927619	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
51	In young AML patients (<60 years old), DNMT3A mutation status (60% of which were R882) was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	3	54	32	2015-07-22 18:47:16.961028	2015-07-22 18:47:16.961028	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
52	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	3	54	55	2015-07-22 18:47:16.994842	2015-07-22 18:47:16.994842	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
53	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	14	82	66	2015-07-22 18:47:17.040285	2015-07-22 18:47:17.040285	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
54	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	14	82	71	2015-07-22 18:47:17.082491	2015-07-22 18:47:17.082491	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
55	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	3	54	86	2015-07-22 18:47:17.11753	2015-07-22 18:47:17.11753	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
56	The presence of NRAS mutation in AML patients does not impact diease prognosis (resistant disease, disease-free survival, complete remission rate, relapse rate, induction death)	3	83	89	2015-07-22 18:47:17.158503	2015-07-22 18:47:17.158503	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
57	NRAS mutation status does not impact overall, event-free or disease-free survival in patients with AML	3	84	92	2015-07-22 18:47:17.201767	2015-07-22 18:47:17.201767	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
58	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	7	85	66	2015-07-22 18:47:17.240723	2015-07-22 18:47:17.240723	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
59	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	7	85	68	2015-07-22 18:47:17.281152	2015-07-22 18:47:17.281152	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
60	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	7	85	70	2015-07-22 18:47:17.322874	2015-07-22 18:47:17.322874	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
61	In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations were associated with reduced relapse free survival in the entire cohort as well as the subset of patients with cytogenetically normal AML	3	81	32	2015-07-22 18:47:17.374999	2015-07-22 18:47:17.374999	3	accepted	\N	\N	N/A	0	1	0	0	3	f	\N	\N
62	AML patients with DNMT3A mutations (59% of which were R882) showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	3	68	32	2015-07-22 18:47:17.439516	2015-07-22 18:47:17.439516	5	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
63	DNMT3A R882 mutation was associated with reduced relapse free and overall survival in ELN-unfavorable, cytogenetically normal AMLs	3	81	32	2015-07-22 18:47:17.471398	2015-07-22 18:47:17.471398	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
64	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including R882) were associated with worse overall survival compared to those without DNMT3A mutation.	3	68	32	2015-07-22 18:47:17.507868	2015-07-22 18:47:17.507868	5	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
65	In cytogenetically normal AML patients, DNMT3A R882 mutations are associated with lower overall and disease free survival as compared to patients with wild type DNMT3A.	3	72	32	2015-07-22 18:47:17.540167	2015-07-22 18:47:17.540167	5	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
66	In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation is prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	3	86	32	2015-07-22 18:47:17.591748	2015-07-22 18:47:17.591748	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
67	In young AML patients (<60 years old), DNMT3A mutations were associated with significantly reduced overall survival and relapse free survival in patients wildtype for NPM1 and FLT3	3	54	32	2015-07-22 18:47:17.625981	2015-07-22 18:47:17.625981	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
68	In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations other than R882 are prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	3	86	32	2015-07-22 18:47:17.665826	2015-07-22 18:47:17.665826	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
69	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated with worse overall survival compared to those without DNMT3A mutation.	3	68	55	2015-07-22 18:47:17.7091	2015-07-22 18:47:17.7091	5	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
70	GIST patients with exon 14 KIT mutations had reduced overall survival compared to patients wiltype for KIT	14	82	69	2015-07-22 18:47:17.745055	2015-07-22 18:47:17.745055	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
71	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	7	87	88	2015-07-22 18:47:17.79235	2015-07-22 18:47:17.79235	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
72	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	7	87	89	2015-07-22 18:47:17.830407	2015-07-22 18:47:17.830407	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
73	A single patient with BRAF V600E/V600M bi-allelic mutation responded to the V600E drug dabrafenib.	7	88	13	2015-07-22 18:47:17.880149	2015-07-22 18:47:17.880149	3	accepted	\N	\N	N/A	3	2	0	0	0	f	\N	\N
74	In patients with medullary carcinoma, the presence of RET M918T mutation is associated with increased probability of lymph node metastases.	15	89	113	2015-07-22 18:47:17.962675	2015-07-22 18:47:17.962675	5	accepted	\N	\N	10:43617416-43617416 (T->C)	1	0	0	0	4	f	\N	\N
75	Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib, compared to wild-type RET.	15	90	112	2015-07-22 18:47:18.004623	2015-07-22 18:47:18.004623	3	accepted	\N	\N	10:43609950-43609950 (C->G)	2	2	0	0	1	f	\N	\N
76	Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib, compared to wild-type RET.	15	90	113	2015-07-22 18:47:18.070471	2015-07-22 18:47:18.070471	3	accepted	\N	\N	10:43617416-43617416 (T->C)	2	2	0	0	1	f	\N	\N
77	The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong repression of tyroid cancer cell growth.	15	91	113	2015-07-22 18:47:18.118198	2015-07-22 18:47:18.118198	3	accepted	\N	\N	10:43617416-43617416 (T->C)	2	2	0	0	0	f	\N	\N
78	Compared to those who harbor a wild type RET, patients with RET M918T mutation develop larger and more aggressive medullary thyroid cancer.	15	92	113	2015-07-22 18:47:18.160518	2015-07-22 18:47:18.160518	4	accepted	\N	\N	10:43617416-43617416 (T->C)	1	1	0	0	3	f	\N	\N
79	BRAF V600E is shown to be associated with the tall-cell variant of papillary thyroid cancer	16	93	12	2015-07-22 18:47:18.209002	2015-07-22 18:47:18.209002	3	accepted	\N	\N	7:140453136-140453136 (A->T)	1	0	0	0	4	f	\N	\N
80	Thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	16	94	12	2015-07-22 18:47:18.247713	2015-07-22 18:47:18.247713	5	accepted	\N	\N	7:140453136-140453136 (A->T)	1	0	0	0	4	f	\N	\N
81	Thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	16	95	12	2015-07-22 18:47:18.286216	2015-07-22 18:47:18.286216	5	accepted	\N	\N	7:140453136-140453136 (A->T)	1	0	0	0	4	f	\N	\N
82	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	7	96	12	2015-07-22 18:47:18.337046	2015-07-22 18:47:18.337046	2	accepted	\N	\N	7:140453136-140453136 (A->T)	1	2	0	1	6	f	\N	2
83	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	11	97	103	2015-07-22 18:47:18.397434	2015-07-22 18:47:18.397434	4	accepted	\N	\N	3:178936082-178936082 (G->A)	1	2	0	1	6	f	\N	\N
84	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	11	97	104	2015-07-22 18:47:18.437322	2015-07-22 18:47:18.437322	4	accepted	\N	\N	3:178936091-178936091 (G->A)	1	2	0	1	6	f	\N	\N
85	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	11	97	107	2015-07-22 18:47:18.484739	2015-07-22 18:47:18.484739	4	accepted	\N	\N	3:178952085-178952085 (A->G)	1	2	0	1	6	f	\N	\N
86	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP but does not show resistance to MEK inhibitors.	7	98	12	2015-07-22 18:47:18.531708	2015-07-22 18:47:18.531708	3	accepted	\N	\N	7:140453136-140453136 (A->T)	2	2	0	1	1	f	\N	\N
87	KRAS mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	11	97	75	2015-07-22 18:47:18.566506	2015-07-22 18:47:18.566506	4	accepted	\N	\N	N/A	1	2	0	1	0	f	\N	\N
88	In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations were associated with poor progression free survival regardless of treatment (panitumumab with best supportive care or best supportive care alone)	11	99	17	2015-07-22 18:47:18.612917	2015-07-22 18:47:18.612917	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
89	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	17	100	12	2015-07-22 18:47:18.657576	2015-07-22 18:47:18.657576	3	accepted	\N	\N	7:140453136-140453136 (A->T)	2	2	0	0	1	f	\N	\N
90	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP and resistance to RAF inhibitors.	7	98	12	2015-07-22 18:47:18.69064	2015-07-22 18:47:18.69064	3	accepted	\N	\N	7:140453136-140453136 (A->T)	2	2	0	0	1	f	\N	\N
91	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	8	101	12	2015-07-22 18:47:18.733193	2015-07-22 18:47:18.733193	2	accepted	\N	\N	7:140453136-140453136 (A->T)	3	2	0	0	1	f	\N	\N
92	COLO201 and COLO206F cells harboring BRAF V600E mutations were cloned to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification of the BRAF V600E gene.	11	102	14	2015-07-22 18:47:18.776667	2015-07-22 18:47:18.776667	4	accepted	\N	\N	N/A	4	2	0	0	1	f	\N	\N
93	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	7	103	17	2015-07-22 18:47:18.824622	2015-07-22 18:47:18.824622	3	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	0
94	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	7	104	11	2015-07-22 18:47:18.866594	2015-07-22 18:47:18.866594	5	accepted	\N	\N	7:140453135-140453136 (CA->AT)	1	2	0	0	0	f	\N	\N
95	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	7	105	12	2015-07-22 18:47:18.929738	2015-07-22 18:47:18.929738	5	accepted	\N	\N	7:140453136-140453136 (A->T)	1	2	0	0	0	f	\N	0
96	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	17	106	12	2015-07-22 18:47:18.992019	2015-07-22 18:47:18.992019	3	accepted	\N	\N	7:140453136-140453136 (A->T)	2	2	0	0	0	f	\N	0
97	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	17	107	12	2015-07-22 18:47:19.055317	2015-07-22 18:47:19.055317	2	accepted	\N	\N	7:140453136-140453136 (A->T)	2	2	0	0	0	f	\N	0
98	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	11	108	12	2015-07-22 18:47:19.113876	2015-07-22 18:47:19.113876	2	accepted	\N	\N	7:140453136-140453136 (A->T)	2	2	0	0	0	f	\N	0
99	Cell lines expressing the BRAF V600E mutation responded better to vemurafenib treatment than cells wildtype for BRAF as measured by reduced cellular proliferation and inhibition of MET and ERK phosphorylation	11	108	12	2015-07-22 18:47:19.162928	2015-07-22 18:47:19.162928	2	accepted	\N	\N	7:140453136-140453136 (A->T)	2	2	0	0	0	f	\N	\N
100	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	7	104	15	2015-07-22 18:47:19.200113	2015-07-22 18:47:19.200113	5	accepted	\N	\N	7:140453137-140453137 (C->T)	1	2	0	0	0	f	\N	\N
101	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	7	104	16	2015-07-22 18:47:19.254086	2015-07-22 18:47:19.254086	5	accepted	\N	\N	7:140453136-140453137 (AC->CT)	1	2	0	0	0	f	\N	\N
102	Unlike other studies that suggest a poorer outcome, BRAF mutation in this study was not correlated with poorer prognosis in papillary thyroid cancer.	16	109	12	2015-07-22 18:47:19.317508	2015-07-22 18:47:19.317508	5	accepted	\N	\N	7:140453136-140453136 (A->T)	1	1	0	1	3	f	\N	\N
103	V600E is associated with adverse pathological features of colorectal cancer. This can be concluded as a marker of poor prognosis.	11	110	12	2015-07-22 18:47:19.375541	2015-07-22 18:47:19.375541	5	accepted	\N	\N	7:140453136-140453136 (A->T)	0	1	0	0	3	f	\N	\N
104	BRAF V600E is correlated with shorter disease-free and overall Survival in a Spanish cohort of melanoma patients.	7	111	12	2015-07-22 18:47:19.435787	2015-07-22 18:47:19.435787	3	accepted	\N	\N	7:140453136-140453136 (A->T)	1	1	0	0	3	f	\N	\N
105	BRAF mutation correlated with poor prognosis in papillary thyroid cancer in both older (>65 yo) and younger (<65 yo) cohorts.	16	93	12	2015-07-22 18:47:19.477604	2015-07-22 18:47:19.477604	3	accepted	\N	\N	7:140453136-140453136 (A->T)	1	1	0	0	3	f	\N	\N
106	BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in a study of 187 patients with PTC and other thyroid diseases.	16	112	12	2015-07-22 18:47:19.551156	2015-07-22 18:47:19.551156	3	accepted	\N	\N	7:140453136-140453136 (A->T)	1	1	0	0	3	f	\N	\N
107	V600E is correlated with disease recurrence in both age cohorts (>65 and <65 yo).	16	93	12	2015-07-22 18:47:19.58924	2015-07-22 18:47:19.58924	3	accepted	\N	\N	7:140453136-140453136 (A->T)	1	1	0	0	3	f	\N	\N
108	NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype (0/55).	3	113	86	2015-07-22 18:47:19.649958	2015-07-22 18:47:19.649958	3	accepted	\N	\N	N/A	1	0	0	1	4	f	\N	\N
109	No NPM1 mutations were identified in patients with favorable risk cytogenetics (79/215 patients)	3	68	87	2015-07-22 18:47:19.695294	2015-07-22 18:47:19.695294	4	accepted	\N	\N	5:170837546-170837547 (0->TCAG)	1	0	0	1	4	f	\N	\N
110	Therapy-related AML was less common in patients with DNMT3A mutations (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of younger (18-60) AML patients	3	81	32	2015-07-22 18:47:19.775505	2015-07-22 18:47:19.775505	3	accepted	\N	\N	N/A	1	0	0	0	5	f	\N	\N
111	NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid leukemia.	3	114	86	2015-07-22 18:47:19.840495	2015-07-22 18:47:19.840495	4	accepted	\N	\N	N/A	1	0	0	0	5	f	\N	\N
112	DNMT3A mutations (59% of which were R882) were associated with an intermediate risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes	3	68	32	2015-07-22 18:47:19.8865	2015-07-22 18:47:19.8865	5	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
113	DNMT3A mutations (59% of which were R882) were associated with intermediate risk cytogenetics (including normal karyotype)	3	68	32	2015-07-22 18:47:19.930118	2015-07-22 18:47:19.930118	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
114	DNMT3A mutations (64.5% R882) were associated with older age, higher white blood cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML patients	3	81	32	2015-07-22 18:47:19.97199	2015-07-22 18:47:19.97199	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
115	DNMT3A R882 mutations were associated with cytogenetically normal AML in a large cohort of younger (18-60) AML patients	3	81	32	2015-07-22 18:47:20.01639	2015-07-22 18:47:20.01639	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
116	AML with mutated NPM1 is a provisional entity in WHO classification of acute myeloid leukemia (AML). This mutation should be tested for in clinical trials and is recommended for testing in patients with cytogentically normal AML.	3	115	86	2015-07-22 18:47:20.068246	2015-07-22 18:47:20.068246	5	accepted	\N	\N	N/A	0	0	0	0	4	f	\N	\N
117	NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes	3	116	86	2015-07-22 18:47:20.12341	2015-07-22 18:47:20.12341	2	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
118	NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid leukemia.	3	114	86	2015-07-22 18:47:20.1654	2015-07-22 18:47:20.1654	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
119	NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid leukemia	3	113	86	2015-07-22 18:47:20.209133	2015-07-22 18:47:20.209133	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
120	NPM1 mutations were associated with normal karyotype in older (>60) patients	3	117	86	2015-07-22 18:47:20.262916	2015-07-22 18:47:20.262916	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
121	NPM1 mutations were associated with intermediate risk cytogenetics (including normal karyotype)	3	68	87	2015-07-22 18:47:20.305161	2015-07-22 18:47:20.305161	4	accepted	\N	\N	5:170837546-170837547 (0->TCAG)	1	0	0	0	4	f	\N	\N
122	CEBPA mutation status had no impact on ATRA treatment response in older (>60) patients with AML	3	118	29	2015-07-22 18:47:20.380736	2015-07-22 18:47:20.380736	4	accepted	\N	\N	N/A	1	2	0	1	6	f	\N	\N
123	In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations (Exon 9) had no significant impact on response rate, disease control rate, progression free surivival or overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	11	76	106	2015-07-22 18:47:20.430992	2015-07-22 18:47:20.430992	3	accepted	\N	\N	N/A	1	2	0	1	6	f	\N	\N
124	In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status (primarily Q61) was not informative for disease control rate, progression free survival or overall survival following cetuximab plus chemotherapy.	11	76	94	2015-07-22 18:47:20.485271	2015-07-22 18:47:20.485271	3	accepted	\N	\N	N/A	1	2	0	1	1	f	\N	\N
125	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation.	13	119	8	2015-07-22 18:47:20.565365	2015-07-22 18:47:20.565365	3	accepted	\N	\N	2:29443695-29443695 (G->T)	2	2	0	0	1	f	\N	\N
126	Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype BRAF	11	76	12	2015-07-22 18:47:20.621551	2015-07-22 18:47:20.621551	3	accepted	\N	\N	7:140453136-140453136 (A->T)	1	2	0	0	1	f	\N	\N
127	ATRA treatment did not effect overall survival in patients <60 years old with FLT3-ITD mutation	3	120	55	2015-07-22 18:47:20.691057	2015-07-22 18:47:20.691057	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
128	FLT3-ITD mutation without NPM1 was associated with increased relapse risk and reduced overall survival in young adult AML patients (median age 43)	3	116	55	2015-07-22 18:47:20.769037	2015-07-22 18:47:20.769037	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
129	Increasing level of FLT3-ITD mutant correlated with reduced disease-free and overall survival as well as increased relapse rate	3	116	55	2015-07-22 18:47:20.850177	2015-07-22 18:47:20.850177	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
130	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	3	121	55	2015-07-22 18:47:20.927715	2015-07-22 18:47:20.927715	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
131	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	3	121	55	2015-07-22 18:47:20.988135	2015-07-22 18:47:20.988135	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
132	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	3	121	56	2015-07-22 18:47:21.051705	2015-07-22 18:47:21.051705	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
133	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	3	121	56	2015-07-22 18:47:21.095065	2015-07-22 18:47:21.095065	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
134	Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations (primarily G12/G13) had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype KRAS	11	76	77	2015-07-22 18:47:21.141581	2015-07-22 18:47:21.141581	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
135	KRAS mutations were significantly associated with lack of response to cetuximab in patients with advanced colorectal cancer	11	122	77	2015-07-22 18:47:21.202403	2015-07-22 18:47:21.202403	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
136	Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	17	123	81	2015-07-22 18:47:21.265758	2015-07-22 18:47:21.265758	4	accepted	\N	\N	12:25398281-25398281 (C->T)	2	2	0	0	1	f	\N	\N
137	ATRA treatment did not effect overall survival in patients <60 years old with NPM1 mutation regardless of FLT3-ITD status	3	120	86	2015-07-22 18:47:21.322127	2015-07-22 18:47:21.322127	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
138	Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	11	76	105	2015-07-22 18:47:21.368653	2015-07-22 18:47:21.368653	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
139	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	3	124	129	2015-07-22 18:47:21.425674	2015-07-22 18:47:21.425674	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
140	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	3	124	130	2015-07-22 18:47:21.477801	2015-07-22 18:47:21.477801	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
141	CH5424802 treatment resulted in significant tumor regression in xenograft models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation	8	77	7	2015-07-22 18:47:21.530327	2015-07-22 18:47:21.530327	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
142	SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition	13	79	8	2015-07-22 18:47:21.566612	2015-07-22 18:47:21.566612	4	accepted	\N	\N	2:29443695-29443695 (G->T)	2	2	0	0	0	f	\N	\N
143	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation	13	119	9	2015-07-22 18:47:21.609361	2015-07-22 18:47:21.609361	3	accepted	\N	\N	2:29432664-29432664 (C->T)	2	2	0	0	0	f	\N	\N
144	Following identification of this fusion in a patient, this fusion was expressed in HEK 293T cells leading to oligomerization in the absence of ligand, induced morphologic changes in the cell and increased cellular proliferation consistent with FGFR activation. Mouse xenografts using the bladder cell line SW780 which harbors an FGFR3-BAIAP2L1 fusion showed reduced tumor burden when treated with the FGFR inhibitor PD173074.	18	125	53	2015-07-22 18:47:21.658428	2015-07-22 18:47:21.658428	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
145	Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	17	123	81	2015-07-22 18:47:21.745896	2015-07-22 18:47:21.745896	4	accepted	\N	\N	12:25398281-25398281 (C->T)	2	2	0	0	0	f	\N	\N
146	ATRA treatment improved overall and relapse-free survival in older (>60) patients with mutant NPM1 and without FLT3-ITD	3	117	86	2015-07-22 18:47:21.785353	2015-07-22 18:47:21.785353	2	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
147	Young patients (18-60) with NPM1 mutations had improved overall survival following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However, the adjusted p-value for multiple testing is 0.11.	3	126	86	2015-07-22 18:47:21.836494	2015-07-22 18:47:21.836494	2	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
148	For patients with NPM1 mutation that achieved CR following induction therapy, relapse-free survival was improved following treatment with valproic acid	3	127	86	2015-07-22 18:47:21.889423	2015-07-22 18:47:21.889423	2	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
149	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	3	128	86	2015-07-22 18:47:21.958683	2015-07-22 18:47:21.958683	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	0
150	CD33 and CD123 expression was significantly increased in patients with NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33 and anti-CD123 therapy	3	129	86	2015-07-22 18:47:22.028865	2015-07-22 18:47:22.028865	2	accepted	\N	\N	N/A	4	2	0	0	0	f	\N	\N
151	CD33 expression was significantly increased in patients with NPM1 mutation with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy	3	129	86	2015-07-22 18:47:22.072446	2015-07-22 18:47:22.072446	2	accepted	\N	\N	N/A	4	2	0	0	0	f	\N	\N
152	NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.	3	128	87	2015-07-22 18:47:22.114573	2015-07-22 18:47:22.114573	3	accepted	\N	\N	5:170837546-170837547 (0->TCAG)	2	2	0	0	0	f	\N	\N
153	NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM in AML patients was associated with good response to induction therapy.	3	130	87	2015-07-22 18:47:22.185545	2015-07-22 18:47:22.185545	3	accepted	\N	\N	5:170837546-170837547 (0->TCAG)	4	2	0	0	0	f	\N	\N
154	Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit increased overall survival, increased apoptosis, reduced proliferation and reduced tumor burden compared to Tsc1 wildtype mice.	8	131	124	2015-07-22 18:47:22.255969	2015-07-22 18:47:22.255969	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
155	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	18	132	125	2015-07-22 18:47:22.338254	2015-07-22 18:47:22.338254	3	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
156	FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically normal AML patients with CEBPA	3	133	28	2015-07-22 18:47:22.401708	2015-07-22 18:47:22.401708	2	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
157	Complete remission rates did not differ between patients with wildtype or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML	3	86	32	2015-07-22 18:47:22.448791	2015-07-22 18:47:22.448791	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
158	FLT3-ITD duplication length did not effect prognosis in patients with an NPM1 mutation	3	134	55	2015-07-22 18:47:22.505154	2015-07-22 18:47:22.505154	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
159	Size of the FLT3-ITD mutant duplication had no impact on overall survival or relapse rate	3	116	55	2015-07-22 18:47:22.557021	2015-07-22 18:47:22.557021	2	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
160	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	3	135	58	2015-07-22 18:47:22.617751	2015-07-22 18:47:22.617751	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
161	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	3	135	86	2015-07-22 18:47:22.663429	2015-07-22 18:47:22.663429	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
162	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	3	136	129	2015-07-22 18:47:22.720128	2015-07-22 18:47:22.720128	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
163	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML	3	136	129	2015-07-22 18:47:22.75391	2015-07-22 18:47:22.75391	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
164	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	3	136	130	2015-07-22 18:47:22.790821	2015-07-22 18:47:22.790821	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
165	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML.	3	136	130	2015-07-22 18:47:22.820733	2015-07-22 18:47:22.820733	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
166	CEBPA mutation was associated with improved overall survival in older (>60) patients with AML	3	118	29	2015-07-22 18:47:22.857976	2015-07-22 18:47:22.857976	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
167	CEBPA mutation was significantly associated with complete remission	3	137	29	2015-07-22 18:47:22.911373	2015-07-22 18:47:22.911373	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
168	Overall survival was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA mutations	3	133	28	2015-07-22 18:47:22.955422	2015-07-22 18:47:22.955422	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
169	Remission duration was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA	3	133	28	2015-07-22 18:47:22.98741	2015-07-22 18:47:22.98741	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
170	DNMT3A mutations were associated with achievement of complete remission in a large cohort of younger (18-60) AML patients	3	81	32	2015-07-22 18:47:23.019939	2015-07-22 18:47:23.019939	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
171	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	3	138	56	2015-07-22 18:47:23.061322	2015-07-22 18:47:23.061322	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
172	MGMT promoter methylation was associated with increased overall and disease-free survival as well as tumor regression in patients with gliomas.	19	139	85	2015-07-22 18:47:23.109583	2015-07-22 18:47:23.109583	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
173	NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and increased overall survival in young adult acute myeloid leukemia patients (median age 43)	3	116	86	2015-07-22 18:47:23.150394	2015-07-22 18:47:23.150394	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
174	Patients with NPM1 mutations were associated with improved complete remission rates as well as overall and disease free survival when all ages and karyotype were analyzed; however, when only normal karyotype patients were analyzed, only complete remission rates were improved	3	117	86	2015-07-22 18:47:23.178757	2015-07-22 18:47:23.178757	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
175	Complete remission rates were higher and both disease-free and overall survival were longer for patients with Exon 12 NPM1 mutations.	3	113	86	2015-07-22 18:47:23.2081	2015-07-22 18:47:23.2081	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
176	Complete remission rates were higher and event-free survival was longer for normal karyotype AML patients with Exon 12 NPM1 mutations.	3	114	86	2015-07-22 18:47:23.253529	2015-07-22 18:47:23.253529	5	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
177	Cytoplasmic localization of NPM in AML patients was an independent prognositic factor associated with complete remission in response to induction therapy in a multivariate analysis (p=0.019) but not in a univariate analysis (p=0.07).	3	130	86	2015-07-22 18:47:23.295473	2015-07-22 18:47:23.295473	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
178	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	3	138	86	2015-07-22 18:47:23.345783	2015-07-22 18:47:23.345783	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
179	Intermediate risk patients with mutant NPM1 had improved overall survival with the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1 and IDH2	3	126	86	2015-07-22 18:47:23.386341	2015-07-22 18:47:23.386341	2	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
180	Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved overall and relapse-free survival, complete remission rates and remission duration compared to all other combinations of mutants	3	140	86	2015-07-22 18:47:23.440429	2015-07-22 18:47:23.440429	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
181	NPM1 mutation was associated with higher complete remission rates, lower cumulative incidence of relapse and higher overall survival in intermediate risk AML patients	3	141	86	2015-07-22 18:47:23.486899	2015-07-22 18:47:23.486899	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
182	NPM1 mutation was associated with increased overall, event-free and relapse-free survival and remission after induction therapy in the entire cohort and in the cytogenetically normal subset of patients	3	142	86	2015-07-22 18:47:23.547958	2015-07-22 18:47:23.547958	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
183	NPM1 mutation without FLT3-ITD was significantly associated with complete remission	3	137	86	2015-07-22 18:47:23.578927	2015-07-22 18:47:23.578927	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
184	NPM1 mutations were associated with increased complete remission rates as well as longer overall disease free survival in normal karyotype AML patients >59 years old, particularly in those >69 years old	3	143	86	2015-07-22 18:47:23.658421	2015-07-22 18:47:23.658421	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
185	Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and without FLT3-ITD have increased overall survival, relapse-free survival and response to induction chemotherapy	3	144	86	2015-07-22 18:47:23.71629	2015-07-22 18:47:23.71629	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
186	Complete remission rate was not different between young AML patients (<60 years old) with or without DNMT3A mutations (60% of which were R882)	3	54	32	2015-07-22 18:47:23.769615	2015-07-22 18:47:23.769615	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
187	DNMT3A mutations (62% of which were R882) were associated with reduced disease-free survival in patients with cytogenetically normal AML.	3	86	32	2015-07-22 18:47:23.817893	2015-07-22 18:47:23.817893	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
188	Young AML patients (<60 years old) with DNMT3A mutation have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	3	54	32	2015-07-22 18:47:23.875883	2015-07-22 18:47:23.875883	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
189	Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	3	54	32	2015-07-22 18:47:23.94162	2015-07-22 18:47:23.94162	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
190	Meta-analysis of studies involving cytogentically normal younger (<60) patients showed reduced overall and relapse-free survival for patients with FLT3-ITD	3	145	55	2015-07-22 18:47:23.995513	2015-07-22 18:47:23.995513	2	accepted	\N	\N	N/A	0	1	0	0	3	f	\N	\N
191	A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation	3	134	55	2015-07-22 18:47:24.042339	2015-07-22 18:47:24.042339	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
192	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	3	134	55	2015-07-22 18:47:24.085452	2015-07-22 18:47:24.085452	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
193	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	3	134	55	2015-07-22 18:47:24.126603	2015-07-22 18:47:24.126603	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
194	KRAS mutations were significantly associated with reduced progression-free and overall survival in patients with advanced colorectal cancer	11	122	77	2015-07-22 18:47:24.174726	2015-07-22 18:47:24.174726	3	accepted	\N	\N	N/A	2	1	0	0	3	f	\N	\N
195	Coocurrence of FLT3-ITD mutations significantly reduced overall survival of normal karyotype AML patients	3	113	86	2015-07-22 18:47:24.213375	2015-07-22 18:47:24.213375	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
196	Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction in NPM1 levels following first line therapy (N=15)	3	146	86	2015-07-22 18:47:24.269273	2015-07-22 18:47:24.269273	2	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
197	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	3	134	86	2015-07-22 18:47:24.309174	2015-07-22 18:47:24.309174	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
198	Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate risk AML patients	3	141	86	2015-07-22 18:47:24.35184	2015-07-22 18:47:24.35184	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
199	TET2 mutation reduces overall survival of normal karyotype patients with either an NPM1 mutation or NPM1 mutation without FLT3-ITD	3	147	86	2015-07-22 18:47:24.405139	2015-07-22 18:47:24.405139	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
200	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	3	134	86	2015-07-22 18:47:24.464677	2015-07-22 18:47:24.464677	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
201	Mutations  in WT1 were associated with increased risk of recurrence in young patients (15-50) with cytogenetically normal AML	3	148	129	2015-07-22 18:47:24.521729	2015-07-22 18:47:24.521729	2	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
202	Mutations in WT1 were associated with a worse overall prognosis than patients wildtype for WT1 in young patients (15-50), primarily because of increased risk of disease recurrence	3	148	129	2015-07-22 18:47:24.570159	2015-07-22 18:47:24.570159	2	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
203	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	3	136	129	2015-07-22 18:47:24.60848	2015-07-22 18:47:24.60848	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
204	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	3	149	129	2015-07-22 18:47:24.659111	2015-07-22 18:47:24.659111	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
205	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	3	124	129	2015-07-22 18:47:24.700706	2015-07-22 18:47:24.700706	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
206	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	3	124	129	2015-07-22 18:47:24.752381	2015-07-22 18:47:24.752381	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
207	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	3	136	130	2015-07-22 18:47:24.80868	2015-07-22 18:47:24.80868	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
208	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	3	124	130	2015-07-22 18:47:24.861443	2015-07-22 18:47:24.861443	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
209	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	3	124	130	2015-07-22 18:47:24.913518	2015-07-22 18:47:24.913518	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
210	WT1 mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	3	149	130	2015-07-22 18:47:24.954054	2015-07-22 18:47:24.954054	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
211	The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation positive women with high grade serous or undifferentiated ovarian cancer.	20	150	131	2015-07-22 18:47:25.009334	2015-07-22 18:47:25.009334	4	accepted	\N	\N	N/A	1	2	1	0	0	f	\N	\N
212	The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation positive women with high grade serous or undifferentiated ovarian cancer.	20	150	132	2015-07-22 18:47:25.090944	2015-07-22 18:47:25.090944	4	accepted	\N	\N	N/A	1	2	1	0	0	f	\N	\N
213	Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML.	4	151	1	2015-07-22 18:47:25.151462	2015-07-22 18:47:25.151462	2	accepted	\N	\N	N/A	2	0	0	1	4	f	\N	\N
214	WBC counts were not significantly different in AML patients with FLT3 D835 mutations compared to wild type	3	152	56	2015-07-22 18:47:25.20343	2015-07-22 18:47:25.20343	3	accepted	\N	\N	N/A	1	0	0	1	4	f	\N	\N
215	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	8	153	76	2015-07-22 18:47:25.260411	2015-07-22 18:47:25.260411	3	accepted	\N	\N	N/A	1	0	0	1	4	f	\N	\N
216	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	8	153	80	2015-07-22 18:47:25.304332	2015-07-22 18:47:25.304332	3	accepted	\N	\N	N/A	1	0	0	1	4	f	\N	\N
217	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.	3	154	127	2015-07-22 18:47:25.35681	2015-07-22 18:47:25.35681	3	accepted	\N	\N	N/A	4	0	0	1	4	f	\N	\N
218	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.	3	154	128	2015-07-22 18:47:25.407666	2015-07-22 18:47:25.407666	3	accepted	\N	\N	N/A	4	0	0	1	4	f	\N	\N
219	CCND2 hypomethylation is seen to be more common in stage III and IV gastric tumors than stage I and II.	21	155	22	2015-07-22 18:47:25.469747	2015-07-22 18:47:25.469747	3	accepted	\N	\N	N/A	2	0	0	0	5	f	\N	\N
220	The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease.	4	156	1	2015-07-22 18:47:25.537485	2015-07-22 18:47:25.537485	4	accepted	\N	\N	N/A	0	0	0	0	4	f	\N	\N
221	Cyclin D1 overexpression is associated with the ER-positive subtype of breast cancer.	22	157	20	2015-07-22 18:47:25.604055	2015-07-22 18:47:25.604055	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
222	WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared to wild type	3	152	55	2015-07-22 18:47:25.653148	2015-07-22 18:47:25.653148	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
223	In leukemia patients, IDH1 R132 mutation is associated with adults and normal karyotype.	3	158	58	2015-07-22 18:47:25.731608	2015-07-22 18:47:25.731608	2	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
224	IDH1 R132 mutation is associated with patients of older age, high platelet count during diagnosis, cytogenic normalcy and NPM1 mutation	3	159	59	2015-07-22 18:47:25.784494	2015-07-22 18:47:25.784494	3	accepted	\N	\N	2:209113113-209113113 (G->A)	1	0	0	0	4	f	\N	\N
225	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	14	160	66	2015-07-22 18:47:25.846817	2015-07-22 18:47:25.846817	2	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
226	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	14	160	70	2015-07-22 18:47:25.904479	2015-07-22 18:47:25.904479	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
227	KRAS G12C occur more frequently in women than men	23	161	78	2015-07-22 18:47:25.958922	2015-07-22 18:47:25.958922	2	accepted	\N	\N	12:25398285-25398285 (C->A)	1	0	0	0	4	f	\N	\N
228	KRAS G12D mutation occurs in never smokers significantly more often than in smokers	23	161	79	2015-07-22 18:47:25.991136	2015-07-22 18:47:25.991136	3	accepted	\N	\N	12:25398284-25398284 (C->T)	1	0	0	0	4	f	\N	\N
229	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.	8	162	33	2015-07-22 18:47:26.037837	2015-07-22 18:47:26.037837	3	accepted	\N	\N	7:55259514-55259515 (CT->AG)	1	2	0	1	6	f	\N	2
230	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.	8	162	33	2015-07-22 18:47:26.081078	2015-07-22 18:47:26.081078	3	accepted	\N	\N	7:55259514-55259515 (CT->AG)	1	2	0	1	6	f	\N	2
231	PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the AKT inhibitor MK-2206 in breast cancer cell lines.	24	163	4	2015-07-22 18:47:26.146772	2015-07-22 18:47:26.146772	3	accepted	\N	\N	14:105246551-105246551 (C->T)	2	2	0	1	0	f	\N	\N
232	BCR-ABL fusions have been found to contribute to imatinib resistance in AML cell lines.	3	164	1	2015-07-22 18:47:26.200829	2015-07-22 18:47:26.200829	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
233	COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	4	165	3	2015-07-22 18:47:26.254925	2015-07-22 18:47:26.254925	4	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
234	In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have shown to be resistant to imatinib treatment.	4	156	2	2015-07-22 18:47:26.30618	2015-07-22 18:47:26.30618	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
235	COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	4	165	2	2015-07-22 18:47:26.357176	2015-07-22 18:47:26.357176	4	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
236	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y variant has been shown to confer resistance to crizotinib.	8	166	6	2015-07-22 18:47:26.41514	2015-07-22 18:47:26.41514	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
237	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M variant has been shown to confer resistance to crizotinib.	8	166	7	2015-07-22 18:47:26.491665	2015-07-22 18:47:26.491665	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
238	The T790M mutation in EGFR has been shown to confer resistance to the tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are placed on the drug are likely to relapse.	25	167	34	2015-07-22 18:47:26.558507	2015-07-22 18:47:26.558507	5	accepted	\N	\N	7:55249071-55249071 (C->T)	0	2	0	0	1	f	\N	\N
239	In non-small cell lung cancer, the appearance of T790M mutation leads to resistance to gefitinib.	8	168	34	2015-07-22 18:47:26.648413	2015-07-22 18:47:26.648413	3	accepted	\N	\N	7:55249071-55249071 (C->T)	1	2	0	0	1	f	\N	\N
240	In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804), MET amplification and EGFR T790M were found to be the primary drivers of resistance.	8	169	34	2015-07-22 18:47:26.719818	2015-07-22 18:47:26.719818	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
241	The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	22	170	39	2015-07-22 18:47:26.783005	2015-07-22 18:47:26.783005	5	accepted	\N	\N	17:37880220-37880220 (T->C)	2	2	0	0	1	f	\N	\N
242	MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	22	171	46	2015-07-22 18:47:26.837651	2015-07-22 18:47:26.837651	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
243	MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	22	171	47	2015-07-22 18:47:26.912421	2015-07-22 18:47:26.912421	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
244	MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	22	171	48	2015-07-22 18:47:26.968644	2015-07-22 18:47:26.968644	3	accepted	\N	\N	6:152419923-152419923 (A->G)	2	2	0	0	1	f	\N	\N
245	MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	22	171	49	2015-07-22 18:47:27.020718	2015-07-22 18:47:27.020718	3	accepted	\N	\N	6:152419922-152419922 (T->A)	2	2	0	0	1	f	\N	\N
246	MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	22	171	50	2015-07-22 18:47:27.072593	2015-07-22 18:47:27.072593	3	accepted	\N	\N	6:152419923-152419923 (A->C)	2	2	0	0	1	f	\N	\N
247	Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated with sorafenib resistance	3	172	56	2015-07-22 18:47:27.1274	2015-07-22 18:47:27.1274	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
248	Sorafenib is effective in patients with FLT3 internal tandem repeat mutations, but not in patients with wild type or D835 mutation.	3	173	56	2015-07-22 18:47:27.19489	2015-07-22 18:47:27.19489	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
249	KIT V654A results in imatinib resistance in GIST patient-derived cell lines.	14	62	73	2015-07-22 18:47:27.230851	2015-07-22 18:47:27.230851	3	accepted	\N	\N	4:55594258-55594258 (T->C)	2	2	0	0	1	f	\N	\N
250	Patients with colorectal cancer who harbor KRAS mutation have low response rate to cetuximab	11	174	74	2015-07-22 18:47:27.284294	2015-07-22 18:47:27.284294	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
251	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	26	175	148	2015-07-22 18:47:27.343157	2015-07-22 18:47:27.343157	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	2
252	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	26	175	78	2015-07-22 18:47:27.402414	2015-07-22 18:47:27.402414	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	2
253	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	26	175	79	2015-07-22 18:47:27.454782	2015-07-22 18:47:27.454782	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	2
254	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	26	175	147	2015-07-22 18:47:27.51214	2015-07-22 18:47:27.51214	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	2
255	In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804), MET amplification and EGFR T790M were found to be the primary drivers of resistance.	8	169	146	2015-07-22 18:47:27.569407	2015-07-22 18:47:27.569407	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
256	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	11	176	103	2015-07-22 18:47:27.662111	2015-07-22 18:47:27.662111	3	accepted	\N	\N	3:178936082-178936082 (G->A)	4	2	0	0	1	f	\N	2
257	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	11	176	104	2015-07-22 18:47:27.713403	2015-07-22 18:47:27.713403	3	accepted	\N	\N	3:178936091-178936091 (G->A)	4	2	0	0	1	f	\N	2
258	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.	11	176	107	2015-07-22 18:47:27.762025	2015-07-22 18:47:27.762025	3	accepted	\N	\N	3:178952085-178952085 (A->G)	4	2	0	0	1	f	\N	2
259	Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide results in high rates of complete remission in patients.	3	164	1	2015-07-22 18:47:27.804121	2015-07-22 18:47:27.804121	5	accepted	\N	\N	N/A	0	2	0	0	0	f	\N	\N
260	The clinical use of imatinib in patients with BCR-ABL fusion has resulted in drastic sensitivity to the drug.	4	156	1	2015-07-22 18:47:27.842417	2015-07-22 18:47:27.842417	5	accepted	\N	\N	N/A	0	2	0	0	0	f	\N	\N
261	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.	4	156	1	2015-07-22 18:47:27.884021	2015-07-22 18:47:27.884021	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	2
262	In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment with crizotinib has shown to be effective in acheiving stable disease.	8	166	5	2015-07-22 18:47:27.931412	2015-07-22 18:47:27.931412	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
263	Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib (PD-0332991) significantly increased median survival.	27	177	23	2015-07-22 18:47:27.98883	2015-07-22 18:47:27.98883	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
264	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	28	178	25	2015-07-22 18:47:28.049097	2015-07-22 18:47:28.049097	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	0
265	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	28	178	26	2015-07-22 18:47:28.093945	2015-07-22 18:47:28.093945	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	0
266	In 3T3 fibroblasts expressing the G253C mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	8	179	141	2015-07-22 18:47:28.138182	2015-07-22 18:47:28.138182	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
267	In 3T3 fibroblasts expressing the G505S mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	8	179	142	2015-07-22 18:47:28.17991	2015-07-22 18:47:28.17991	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
268	In 3T3 fibroblasts expressing the G774V mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	8	179	144	2015-07-22 18:47:28.219047	2015-07-22 18:47:28.219047	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
269	In 3T3 fibroblasts expressing the I638F mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	8	179	143	2015-07-22 18:47:28.267239	2015-07-22 18:47:28.267239	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
270	In 3T3 fibroblasts expressing the L239R mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	8	179	140	2015-07-22 18:47:28.330328	2015-07-22 18:47:28.330328	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
271	In 3T3 fibroblasts expressing the L63V mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	8	179	139	2015-07-22 18:47:28.398292	2015-07-22 18:47:28.398292	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
272	In a patient expressing a DDR2 S768R mutation, treatment with dasatinib plus erlotinib acheived a partial response.	8	179	145	2015-07-22 18:47:28.446437	2015-07-22 18:47:28.446437	3	accepted	\N	\N	N/A	3	2	0	0	0	f	\N	0
273	Exon 19 deletion has been shown to be correlated with gefitinib response.	8	180	133	2015-07-22 18:47:28.507703	2015-07-22 18:47:28.507703	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
274	Gefinitib has been shown to be effective in treating cell lines with G719S missense mutations.	8	181	134	2015-07-22 18:47:28.561504	2015-07-22 18:47:28.561504	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
275	In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence of L858R mutation is prognostic for better progression free survival.	8	182	33	2015-07-22 18:47:28.608878	2015-07-22 18:47:28.608878	4	accepted	\N	\N	7:55259514-55259515 (CT->AG)	1	2	0	0	0	f	\N	\N
276	Gefinitib has been shown to be effective in treating cell lines with L858R missense mutations.	8	181	33	2015-07-22 18:47:28.661581	2015-07-22 18:47:28.661581	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
277	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.	8	183	34	2015-07-22 18:47:28.720277	2015-07-22 18:47:28.720277	3	accepted	\N	\N	7:55249071-55249071 (C->T)	3	2	0	0	0	f	\N	0
278	In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR, especially when an L858R mutation is also present.	8	184	34	2015-07-22 18:47:28.777231	2015-07-22 18:47:28.777231	1	accepted	\N	\N	7:55249071-55249071 (C->T)	4	2	0	0	0	f	\N	\N
279	In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.	22	170	35	2015-07-22 18:47:28.815507	2015-07-22 18:47:28.815507	4	accepted	\N	\N	17:37880261-37880261 (G->C)	2	2	0	0	0	f	\N	\N
280	In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.	22	170	36	2015-07-22 18:47:28.873848	2015-07-22 18:47:28.873848	4	accepted	\N	\N	17:37880261-37880261 (G->T)	2	2	0	0	0	f	\N	\N
281	In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2, cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive to neratinib.	22	170	37	2015-07-22 18:47:28.918161	2015-07-22 18:47:28.918161	5	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
282	In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.	22	170	38	2015-07-22 18:47:28.952311	2015-07-22 18:47:28.952311	5	accepted	\N	\N	17:37868205-37868205 (G->C)	2	2	0	0	0	f	\N	\N
283	The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	22	170	39	2015-07-22 18:47:29.009466	2015-07-22 18:47:29.009466	5	accepted	\N	\N	17:37880220-37880220 (T->C)	2	2	0	0	0	f	\N	\N
284	MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	22	170	40	2015-07-22 18:47:29.058218	2015-07-22 18:47:29.058218	5	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
285	In MCF10A cell lines, the in-frame insertion of a single amino acid at position 780 was shown to be sensitive to neratinib.	22	170	41	2015-07-22 18:47:29.094868	2015-07-22 18:47:29.094868	5	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
286	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	22	170	42	2015-07-22 18:47:29.138882	2015-07-22 18:47:29.138882	5	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	2
287	In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.	22	170	43	2015-07-22 18:47:29.184246	2015-07-22 18:47:29.184246	5	accepted	\N	\N	17:37881616-37881616 (C->T)	2	2	0	0	0	f	\N	\N
288	In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.	22	170	44	2015-07-22 18:47:29.241085	2015-07-22 18:47:29.241085	5	accepted	\N	\N	17:37881000-37881000 (G->T)	2	2	0	0	0	f	\N	\N
289	In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.	22	170	45	2015-07-22 18:47:29.294543	2015-07-22 18:47:29.294543	5	accepted	\N	\N	17:37881332-37881332 (G->A)	2	2	0	0	0	f	\N	\N
290	The L536Q ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.	22	185	46	2015-07-22 18:47:29.358157	2015-07-22 18:47:29.358157	5	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	2
291	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	22	185	47	2015-07-22 18:47:29.431373	2015-07-22 18:47:29.431373	5	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	2
292	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	22	185	48	2015-07-22 18:47:29.520947	2015-07-22 18:47:29.520947	5	accepted	\N	\N	6:152419923-152419923 (A->G)	2	2	0	0	0	f	\N	2
293	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	22	185	49	2015-07-22 18:47:29.580293	2015-07-22 18:47:29.580293	5	accepted	\N	\N	6:152419922-152419922 (T->A)	2	2	0	0	0	f	\N	2
294	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	22	185	50	2015-07-22 18:47:29.636687	2015-07-22 18:47:29.636687	5	accepted	\N	\N	6:152419923-152419923 (A->C)	2	2	0	0	0	f	\N	2
295	In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic sites and overall stable disease. However, this response did not meet criteria for RECIST partial response.	29	186	52	2015-07-22 18:47:29.710246	2015-07-22 18:47:29.710246	2	accepted	\N	\N	N/A	3	2	0	0	0	f	\N	\N
296	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This fusion is also shown to be recurrent in multiple other cancer types.	29	186	54	2015-07-22 18:47:29.773222	2015-07-22 18:47:29.773222	3	accepted	\N	\N	N/A	3	2	0	0	0	f	\N	2
297	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	3	187	55	2015-07-22 18:47:29.847731	2015-07-22 18:47:29.847731	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
298	Sorafenib is effective in patients with FLT3 internal tandem repeats, but not in wild type or D835 mutation.	3	173	55	2015-07-22 18:47:29.896387	2015-07-22 18:47:29.896387	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
299	AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with FLT3 internal tandem repeat	3	188	55	2015-07-22 18:47:29.944501	2015-07-22 18:47:29.944501	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
300	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	3	187	56	2015-07-22 18:47:29.985127	2015-07-22 18:47:29.985127	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
301	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	30	189	57	2015-07-22 18:47:30.033624	2015-07-22 18:47:30.033624	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	0
302	In a case study, a patient with anal melanoma harboring a 7 codon duplication in the juxtamambrane region (exon 11) of KIT showed marked response 4 months after imatinib treatment.	31	190	67	2015-07-22 18:47:30.098719	2015-07-22 18:47:30.098719	3	accepted	\N	\N	N/A	3	2	0	0	0	f	\N	\N
303	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	8	191	72	2015-07-22 18:47:30.152959	2015-07-22 18:47:30.152959	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	2
304	In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective for those that are refractory to imatinib.	14	192	73	2015-07-22 18:47:30.208967	2015-07-22 18:47:30.208967	3	accepted	\N	\N	4:55594258-55594258 (T->C)	1	2	0	0	0	f	\N	\N
305	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.	8	193	79	2015-07-22 18:47:30.263855	2015-07-22 18:47:30.263855	4	accepted	\N	\N	12:25398284-25398284 (C->T)	2	2	0	0	0	f	\N	0
306	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	17	123	81	2015-07-22 18:47:30.309777	2015-07-22 18:47:30.309777	4	accepted	\N	\N	12:25398281-25398281 (C->T)	1	2	0	0	0	f	\N	\N
307	In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted from temozolomide. This benefit was also methylation status dependent, as those without methylation did not see increased survival.	19	194	85	2015-07-22 18:47:30.35287	2015-07-22 18:47:30.35287	5	accepted	\N	\N	N/A	0	2	0	0	0	f	\N	\N
308	MGMT promoter methylation has been shown to confer sensitivity to alkylating agents, such as carmustine, in glioblastoma cells.	19	139	85	2015-07-22 18:47:30.391032	2015-07-22 18:47:30.391032	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
309	In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor, such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may be useful in treating patients with initial resistance to alkylating agents.	19	139	85	2015-07-22 18:47:30.426959	2015-07-22 18:47:30.426959	4	accepted	\N	\N	N/A	4	2	0	0	0	f	\N	0
310	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	22	195	103	2015-07-22 18:47:30.470758	2015-07-22 18:47:30.470758	4	accepted	\N	\N	3:178936082-178936082 (G->A)	2	2	0	0	0	f	\N	\N
311	Rapamycin inhibits transformation induced by mutation in PIK3CA	22	196	103	2015-07-22 18:47:30.517783	2015-07-22 18:47:30.517783	3	accepted	\N	\N	3:178936082-178936082 (G->A)	2	2	0	0	0	f	\N	\N
312	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	22	195	104	2015-07-22 18:47:30.551676	2015-07-22 18:47:30.551676	4	accepted	\N	\N	3:178936091-178936091 (G->A)	2	2	0	0	0	f	\N	\N
313	Rapamycin inhibits transformation induced by mutation in PIK3CA	22	196	104	2015-07-22 18:47:30.590373	2015-07-22 18:47:30.590373	3	accepted	\N	\N	3:178936091-178936091 (G->A)	2	2	0	0	0	f	\N	\N
314	Breast cancer cell lines with H1047R mutation showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	22	195	107	2015-07-22 18:47:30.625033	2015-07-22 18:47:30.625033	4	accepted	\N	\N	3:178952085-178952085 (A->G)	2	2	0	0	0	f	\N	\N
315	Rapamycin inhibits transformation induced by mutation in PIK3CA	22	196	107	2015-07-22 18:47:30.660911	2015-07-22 18:47:30.660911	3	accepted	\N	\N	3:178952085-178952085 (A->G)	2	2	0	0	0	f	\N	\N
316	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	32	197	108	2015-07-22 18:47:30.709236	2015-07-22 18:47:30.709236	5	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
317	Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse to PI3K-mTOR inhibitors.	24	198	110	2015-07-22 18:47:30.75244	2015-07-22 18:47:30.75244	4	accepted	\N	\N	10:89717672-89717672 (C->T)	2	2	0	0	0	f	\N	\N
318	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	32	197	111	2015-07-22 18:47:30.811208	2015-07-22 18:47:30.811208	5	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
319	Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.	22	199	116	2015-07-22 18:47:30.854667	2015-07-22 18:47:30.854667	3	accepted	\N	\N	17:7578406-7578406 (C->T)	2	2	0	0	0	f	\N	\N
320	A patient with metastatic bladder cancer that responded well to the mTOR inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del, p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations also had mild responses to everolimus.	33	200	124	2015-07-22 18:47:30.904297	2015-07-22 18:47:30.904297	3	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
321	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	18	200	126	2015-07-22 18:47:30.946838	2015-07-22 18:47:30.946838	3	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
322	AML patients with D835 mutation of FLT3 did not have worse overall survival than patients with wild type FLT3	3	152	56	2015-07-22 18:47:30.99209	2015-07-22 18:47:30.99209	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
323	The complete remission and overall survival rates in patients with IDH1 R132 mutation is not significantly different from those who do not have this mutation	3	201	58	2015-07-22 18:47:31.037557	2015-07-22 18:47:31.037557	2	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
324	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	3	159	59	2015-07-22 18:47:31.102054	2015-07-22 18:47:31.102054	2	accepted	\N	\N	2:209113113-209113113 (G->A)	1	1	0	1	6	f	\N	\N
325	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	3	159	60	2015-07-22 18:47:31.14219	2015-07-22 18:47:31.14219	3	accepted	\N	\N	2:209113112-209113112 (C->T)	1	1	0	1	6	f	\N	\N
326	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	3	159	61	2015-07-22 18:47:31.190178	2015-07-22 18:47:31.190178	3	accepted	\N	\N	2:209113112-209113112 (C->A)	1	1	0	1	6	f	\N	\N
327	AML patients with IDH2 mutations such as R140Q/L have event free survival and overall survival similar to those with wild-type IDH2	3	202	62	2015-07-22 18:47:31.241872	2015-07-22 18:47:31.241872	2	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
328	In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact overall survival or disease free survival.	3	203	62	2015-07-22 18:47:31.294683	2015-07-22 18:47:31.294683	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
329	In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L do not confer prognostic value (overall survival)	34	204	62	2015-07-22 18:47:31.34396	2015-07-22 18:47:31.34396	2	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
330	R140Q mutation in IDH2 does not have prognostic value in patients with MDS.	34	205	62	2015-07-22 18:47:31.395521	2015-07-22 18:47:31.395521	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
331	AML patients with IDH2 mutations such as R172K have event free survival and overall survival similar to those with wild-type IDH2.	3	202	63	2015-07-22 18:47:31.436563	2015-07-22 18:47:31.436563	2	accepted	\N	\N	15:90631838-90631838 (C->T)	1	1	0	1	6	f	\N	\N
332	In patients suffering from MDS, the presence of IDH2 mutations such as R172K do not confer prognostic value (overall survival)	34	204	63	2015-07-22 18:47:31.474225	2015-07-22 18:47:31.474225	2	accepted	\N	\N	15:90631838-90631838 (C->T)	1	1	0	1	6	f	\N	\N
333	KIT mutations were identified in morphologically benign, incidentally discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating is not a prognostic marker for GIST	14	206	66	2015-07-22 18:47:31.525683	2015-07-22 18:47:31.525683	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
334	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	23	207	66	2015-07-22 18:47:31.588989	2015-07-22 18:47:31.588989	2	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
335	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	23	207	71	2015-07-22 18:47:31.632576	2015-07-22 18:47:31.632576	2	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
336	In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12 mutation did not impact overall or disease free survival.	17	208	76	2015-07-22 18:47:31.676551	2015-07-22 18:47:31.676551	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
337	There is no strong association between KRAS mutation status and overall or progression free survival in patients with NSCLC	8	209	76	2015-07-22 18:47:31.737752	2015-07-22 18:47:31.737752	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
338	In patients with AML, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	3	154	127	2015-07-22 18:47:31.79032	2015-07-22 18:47:31.79032	2	accepted	\N	\N	N/A	4	1	0	1	6	f	\N	\N
339	In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	34	154	127	2015-07-22 18:47:31.835718	2015-07-22 18:47:31.835718	2	accepted	\N	\N	N/A	4	1	0	1	6	f	\N	\N
340	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	3	154	128	2015-07-22 18:47:31.875652	2015-07-22 18:47:31.875652	2	accepted	\N	\N	N/A	4	1	0	1	6	f	\N	\N
341	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	34	154	128	2015-07-22 18:47:31.922521	2015-07-22 18:47:31.922521	2	accepted	\N	\N	N/A	4	1	0	1	6	f	\N	\N
342	Three studies have found no significant survival impact for CCND1 in lung cancer.	8	210	19	2015-07-22 18:47:31.981489	2015-07-22 18:47:31.981489	3	accepted	\N	\N	N/A	1	1	0	1	3	f	\N	\N
343	PTEN nonsense mutations, including R233*, have been shown to be inactivating and loss-of-function, but do not have prognostic value in glioblastoma multiforme patients.	19	211	110	2015-07-22 18:47:32.067964	2015-07-22 18:47:32.067964	3	accepted	\N	\N	10:89717672-89717672 (C->T)	1	1	0	1	3	f	\N	\N
344	BCR-ABL inhibitors such as imatinib have lead to significantly improved prognosis, response rate, overall survival, and patient outcome in CML patients compared to previous therapeutic regimens.	4	156	1	2015-07-22 18:47:32.13744	2015-07-22 18:47:32.13744	5	accepted	\N	\N	N/A	0	1	0	0	2	f	\N	\N
345	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	28	212	25	2015-07-22 18:47:32.198636	2015-07-22 18:47:32.198636	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
346	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	28	212	26	2015-07-22 18:47:32.249104	2015-07-22 18:47:32.249104	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
347	Median survival of patients with EGFR L858R mutation is better than those with wild type EGFR.	8	213	33	2015-07-22 18:47:32.318202	2015-07-22 18:47:32.318202	3	accepted	\N	\N	7:55259514-55259515 (CT->AG)	1	1	0	0	2	f	\N	\N
348	In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic for better survival compared to patients who harbor wild type mutation.	35	214	58	2015-07-22 18:47:32.38178	2015-07-22 18:47:32.38178	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
349	In patients with glioblastoma, those harboring IDH1 R132 mutation has higher overall survival compared to those who do not have IDH1 mutation	36	215	58	2015-07-22 18:47:32.443513	2015-07-22 18:47:32.443513	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
350	In AML, patients with IDH2 R140K mutation have improved overall survival compared to those with wild-type IDH2	3	216	62	2015-07-22 18:47:32.507148	2015-07-22 18:47:32.507148	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
351	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	17	123	81	2015-07-22 18:47:32.549009	2015-07-22 18:47:32.549009	4	accepted	\N	\N	12:25398281-25398281 (C->T)	1	1	0	0	2	f	\N	\N
352	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	34	217	114	2015-07-22 18:47:32.608024	2015-07-22 18:47:32.608024	3	accepted	\N	\N	2:198267359-198267359 (C->A)	4	1	0	0	2	f	\N	\N
353	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	34	217	115	2015-07-22 18:47:32.652071	2015-07-22 18:47:32.652071	3	accepted	\N	\N	2:198266834-198266834 (T->C)	4	1	0	0	2	f	\N	\N
354	Inceased copy number of CCND1 is associated with poorer overall survival.	8	210	18	2015-07-22 18:47:32.694746	2015-07-22 18:47:32.694746	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
355	Three studies have associated CCND1 expression with poorer survival.	8	210	19	2015-07-22 18:47:32.732338	2015-07-22 18:47:32.732338	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
356	In ER-positive breast cancer patients, Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis.	22	157	20	2015-07-22 18:47:32.777914	2015-07-22 18:47:32.777914	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
357	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in head and neck squamous cell carcinoma.	37	218	20	2015-07-22 18:47:32.873631	2015-07-22 18:47:32.873631	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
358	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in mantle cell lymphoma.	38	219	20	2015-07-22 18:47:32.944129	2015-07-22 18:47:32.944129	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
359	Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.	21	220	21	2015-07-22 18:47:33.000454	2015-07-22 18:47:33.000454	4	accepted	\N	\N	N/A	2	1	0	0	3	f	\N	\N
360	Cyclin D2 overexpression is associated with the increased depth of cancer invasion in gastric cancers.	21	220	21	2015-07-22 18:47:33.045261	2015-07-22 18:47:33.045261	4	accepted	\N	\N	N/A	2	1	0	0	3	f	\N	\N
361	Cyclin D2 overexpression is associated with the presence pf lymph node metastasis in gastric cancers.	21	220	21	2015-07-22 18:47:33.083515	2015-07-22 18:47:33.083515	4	accepted	\N	\N	N/A	2	1	0	0	3	f	\N	\N
362	Cyclin D2 overexpression is associated with vascular invasion by cancer cells in gastric cancers.	21	220	21	2015-07-22 18:47:33.125267	2015-07-22 18:47:33.125267	4	accepted	\N	\N	N/A	2	1	0	0	3	f	\N	\N
363	Patients with high expression of total cyclin E and low-molecular weight cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard ratio for these patients in 13.3 times higher than those with normal cyclin E levels.	22	221	24	2015-07-22 18:47:33.176532	2015-07-22 18:47:33.176532	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
364	Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced stage gastric carcinoma.	39	222	24	2015-07-22 18:47:33.232545	2015-07-22 18:47:33.232545	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
365	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	39	222	24	2015-07-22 18:47:33.278323	2015-07-22 18:47:33.278323	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
366	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	23	222	24	2015-07-22 18:47:33.324268	2015-07-22 18:47:33.324268	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
367	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter overall survival.	8	223	27	2015-07-22 18:47:33.378653	2015-07-22 18:47:33.378653	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
368	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter time to recurrence.	8	223	27	2015-07-22 18:47:33.435172	2015-07-22 18:47:33.435172	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
369	In NSCLC patients with T790M and another activating mutations, their progression free survival is shorter compared to those who do not possess T790M mutation.	8	224	34	2015-07-22 18:47:33.48827	2015-07-22 18:47:33.48827	4	accepted	\N	\N	7:55249071-55249071 (C->T)	1	1	0	0	3	f	\N	\N
370	Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall survival compared to patients with L858R or other exon 19 activating mutations.	8	225	34	2015-07-22 18:47:33.552449	2015-07-22 18:47:33.552449	4	accepted	\N	\N	7:55249071-55249071 (C->T)	1	1	0	0	3	f	\N	\N
371	AML patients with internal tandem repeat of FLT3 has worse overall survival than patients with wild type FLT3	3	152	55	2015-07-22 18:47:33.614405	2015-07-22 18:47:33.614405	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
372	Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor survival and inferior disease free survival in patients with AML.	3	226	55	2015-07-22 18:47:33.667091	2015-07-22 18:47:33.667091	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
373	In AML patients, FLT3-TKD mutation was associated with poorer disease free survival compared to patients with wild type FLT3	3	227	56	2015-07-22 18:47:33.719705	2015-07-22 18:47:33.719705	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
374	IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated with worse overall survival	40	228	58	2015-07-22 18:47:33.771125	2015-07-22 18:47:33.771125	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
375	In AML, patients with an IDH2 R172K mutation have worse overall survival compared to those with wild-type IDH2	3	216	63	2015-07-22 18:47:33.815946	2015-07-22 18:47:33.815946	3	accepted	\N	\N	15:90631838-90631838 (C->T)	1	1	0	0	3	f	\N	\N
376	KIT mutation is associated with worse overall and cause-specific prognosis in patients with GIST compared to patients with wildtype KIT	14	160	66	2015-07-22 18:47:33.860481	2015-07-22 18:47:33.860481	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
377	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	41	229	76	2015-07-22 18:47:33.931376	2015-07-22 18:47:33.931376	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
378	A meta-analysis showed KRAS mutation is associated with worse outcome in patients with NSCLC	8	230	76	2015-07-22 18:47:33.991444	2015-07-22 18:47:33.991444	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
379	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	42	229	80	2015-07-22 18:47:34.037698	2015-07-22 18:47:34.037698	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
380	NOTCH1 gain-of-function mutations, including D1643H, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	8	231	135	2015-07-22 18:47:34.085089	2015-07-22 18:47:34.085089	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
381	NOTCH1 gain-of-function mutations, including R2328W, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	8	231	136	2015-07-22 18:47:34.128133	2015-07-22 18:47:34.128133	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
382	NOTCH1 gain-of-function mutations, including S2275fs, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	8	231	138	2015-07-22 18:47:34.171832	2015-07-22 18:47:34.171832	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
383	NOTCH1 gain-of-function mutations, including V2444fs, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	8	231	137	2015-07-22 18:47:34.224334	2015-07-22 18:47:34.224334	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
384	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	11	232	103	2015-07-22 18:47:34.281503	2015-07-22 18:47:34.281503	2	accepted	\N	\N	3:178936082-178936082 (G->A)	4	1	0	0	3	f	\N	\N
385	The presence of exon 9 (such as E542K) or exon 20 mutations were associated with poorer survival compared to those with wild-type PIK3CA mutation.	11	176	103	2015-07-22 18:47:34.33169	2015-07-22 18:47:34.33169	3	accepted	\N	\N	3:178936082-178936082 (G->A)	4	1	0	0	3	f	\N	\N
386	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	11	232	104	2015-07-22 18:47:34.374575	2015-07-22 18:47:34.374575	2	accepted	\N	\N	3:178936091-178936091 (G->A)	4	1	0	0	3	f	\N	\N
387	The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	11	176	104	2015-07-22 18:47:34.429972	2015-07-22 18:47:34.429972	3	accepted	\N	\N	3:178936091-178936091 (G->A)	4	1	0	0	3	f	\N	\N
388	The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	11	176	107	2015-07-22 18:47:34.475535	2015-07-22 18:47:34.475535	3	accepted	\N	\N	3:178952085-178952085 (A->G)	4	1	0	0	3	f	\N	\N
389	Breast cancer patients who harbor R175H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	22	233	116	2015-07-22 18:47:34.52929	2015-07-22 18:47:34.52929	3	accepted	\N	\N	17:7578406-7578406 (C->T)	1	1	0	0	3	f	\N	\N
390	Breast cancer patients who harbor R248Q mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	22	233	117	2015-07-22 18:47:34.600224	2015-07-22 18:47:34.600224	3	accepted	\N	\N	17:7577538-7577538 (C->T)	1	1	0	0	3	f	\N	\N
391	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R248 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	22	234	117	2015-07-22 18:47:34.662003	2015-07-22 18:47:34.662003	3	accepted	\N	\N	17:7577538-7577538 (C->T)	1	1	0	0	3	f	\N	\N
392	In breast cancer patients harboring R248W mutation, the prognosis is worse than any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.	22	233	118	2015-07-22 18:47:34.720958	2015-07-22 18:47:34.720958	3	accepted	\N	\N	17:7577539-7577539 (G->A)	1	1	0	0	3	f	\N	\N
393	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	22	234	119	2015-07-22 18:47:34.799137	2015-07-22 18:47:34.799137	3	accepted	\N	\N	17:7577535-7577535 (C->G)	1	1	0	0	3	f	\N	\N
394	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	22	234	120	2015-07-22 18:47:34.856162	2015-07-22 18:47:34.856162	3	accepted	\N	\N	17:7577536-7577536 (T->A)	1	1	0	0	3	f	\N	\N
395	Breast cancer patients who harbor R273C mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	22	233	121	2015-07-22 18:47:34.911378	2015-07-22 18:47:34.911378	3	accepted	\N	\N	17:7577121-7577121 (G->A)	1	1	0	0	3	f	\N	\N
396	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	22	234	121	2015-07-22 18:47:34.959363	2015-07-22 18:47:34.959363	3	accepted	\N	\N	17:7577121-7577121 (G->A)	1	1	0	0	3	f	\N	\N
397	Breast cancer patients who harbor R273H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	22	233	122	2015-07-22 18:47:35.013789	2015-07-22 18:47:35.013789	3	accepted	\N	\N	17:7577120-7577120 (C->T)	1	1	0	0	3	f	\N	\N
398	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	22	234	122	2015-07-22 18:47:35.061448	2015-07-22 18:47:35.061448	3	accepted	\N	\N	17:7577120-7577120 (C->T)	1	1	0	0	3	f	\N	\N
399	Breast tumors with R175H mutations are more responsive to doxorubicin than breast tumors with wild type TP53.	22	234	123	2015-07-22 18:47:35.116688	2015-07-22 18:47:35.116688	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
400	After adjusting for age and cancer stage, presence of U2AF mutation such as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.	34	235	127	2015-07-22 18:47:35.17881	2015-07-22 18:47:35.17881	3	accepted	\N	\N	N/A	4	1	0	0	3	f	\N	\N
401	After adjusting for age and cancer stage, the presence of U2AF mutations such as S34Y/F are prognostic for poorer survival outcomes in patients with MDS.	34	235	128	2015-07-22 18:47:35.232772	2015-07-22 18:47:35.232772	3	accepted	\N	\N	N/A	4	1	0	0	3	f	\N	\N
402	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	8	236	79	2015-07-22 18:47:35.293631	2015-07-22 18:47:35.293631	2	accepted	\N	\N	12:25398284-25398284 (C->T)	3	2	0	0	1	f	\N	\N
403	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	43	237	30	2015-07-22 18:47:35.365646	2015-07-22 18:47:35.365646	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	2
404	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	43	237	84	2015-07-22 18:47:35.422176	2015-07-22 18:47:35.422176	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	2
405	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	44	238	31	2015-07-22 18:47:35.477321	2015-07-22 18:47:35.477321	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
406	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	44	238	109	2015-07-22 18:47:35.518697	2015-07-22 18:47:35.518697	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
407	The t(14;18) transloaction of BCL2 is shown to be associated with germinal-center B-cell–like diffuse large-B-cell lymphomas.	45	239	151	2015-07-22 18:47:35.592593	2015-07-22 18:47:35.592593	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
408	Amplification of FOXP1 is shown to be associated with ABC DLBCL	45	240	153	2015-07-22 18:47:35.667812	2015-07-22 18:47:35.667812	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
409	Amplification of REL is shown to be associated with germinal-center B-cell–like diffuse large-B-cell lymphomas.	45	239	154	2015-07-22 18:47:35.717173	2015-07-22 18:47:35.717173	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
410	Amplification of REL is shown to be associated with germinal-center B-cell–like diffuse large-B-cell lymphomas.	45	239	154	2015-07-22 18:47:35.766892	2015-07-22 18:47:35.766892	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
411	In non-APL patients with AML, those who had mutations in RUNX1 and were treated with allogeneic hematopoietic stem-cell transplantation had longer relapse-free survival than those treated with conventional post remmision treatment.	3	241	155	2015-07-22 18:47:35.835747	2015-07-22 18:47:35.835747	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
412	OSI-906, an IGF1R/INSR inhibitor, does not have a significant effect on chemotherapy resistant castration-resistant prostate cancer cell lines.	46	242	156	2015-07-22 18:47:35.916425	2015-07-22 18:47:35.916425	3	accepted	\N	\N	N/A	2	2	0	0	6	f	\N	\N
413	NSCLC patients with exon 19 deletions in EGFR had longer overall survival than patients with an L858R mutation when treated with gefitinib and erlotinib (38 versus 17 months; P = 0.04).	8	243	133	2015-07-22 18:47:35.976806	2015-07-22 18:47:35.976806	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	2
414	OSI-906, an IGF1R/INSR inhibitor, sensitizes cell lines to docetaxel and cabazitaxel, resulting in better overall survival in chemotherapy resistant CRPC cell lines.	46	244	156	2015-07-22 18:47:36.059137	2015-07-22 18:47:36.059137	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	0
415	In a cohort of 783 young adult patients with acute myeloid leukemia, those with TET2 mutations did not have significantly different rates of complete recovery, refractory disease, or hypoplastic death.	3	245	157	2015-07-22 18:47:36.13323	2015-07-22 18:47:36.13323	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
416	In patients with ovarian carcinomas, those with Aurora kinase A expression (as measured by immunohistochemistry) had increased progression-free and overall survival compared to those without expression.	47	246	158	2015-07-22 18:47:36.182055	2015-07-22 18:47:36.182055	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
417	In patients with myelodysplastic syndromes, those with TET2 mutations have been shown to have better overall survival than patients without TET2 mutations.	34	247	157	2015-07-22 18:47:36.228349	2015-07-22 18:47:36.228349	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
418	In patients with renal clear cell carcinoma, patients with VHL mutations had longer cancer-specific survival.	48	248	160	2015-07-22 18:47:36.285493	2015-07-22 18:47:36.285493	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
419	In patients with cytogenetically normal acute myeloid leukemia, those in the intermediate-I risk group with TET2 mutations had higher rates of complete remission.	3	245	157	2015-07-22 18:47:36.328936	2015-07-22 18:47:36.328936	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
420	In renal clear cell carcinoma, patients with PBRM1 mutations did not have a significantly different cancer specific survival.	48	249	161	2015-07-22 18:47:36.383774	2015-07-22 18:47:36.383774	3	accepted	\N	\N	N/A	1	1	0	0	6	f	\N	\N
421	In patients with cytogenetically normal acute myeloid leukemia, those in the intermediate-I risk group with TET2 mutations did not have significantly different rates of event-free survival, complete remission, disease-free survival, or overall survival.	3	250	157	2015-07-22 18:47:36.466542	2015-07-22 18:47:36.466542	4	accepted	\N	\N	N/A	1	1	0	0	6	f	\N	\N
422	In renal clear cell carcinoma, patients with VHL mutations did not have a significant impact on cancer specific survival.	48	249	160	2015-07-22 18:47:36.506762	2015-07-22 18:47:36.506762	3	accepted	\N	\N	N/A	1	1	0	0	6	f	\N	\N
423	In patients with Diffuse Large B-Cell Lymphoma, those with increased levels of BCL2 were shown to have decreased overall, disease-free, and relapse-free survival.	45	251	152	2015-07-22 18:47:36.559547	2015-07-22 18:47:36.559547	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
424	In patients with Diffuse Large B-Cell Lymphoma, those with increased levels of BCL2 were shown to have decreased OS, DFS, and RFS.	45	251	152	2015-07-22 18:47:36.601306	2015-07-22 18:47:36.601306	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
425	In myelodysplastic syndrome, patients with point mutations in EZH2 have worse overall survival that their wild-type counterparts.	49	252	163	2015-07-22 18:47:36.663327	2015-07-22 18:47:36.663327	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
426	In patients classified as intermediate-risk via cytogenetics, TET2 mutations have been shown to be correlated with poor prognosis.	3	253	157	2015-07-22 18:47:36.726093	2015-07-22 18:47:36.726093	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
427	In patients with cytogenetically normal acute myeloid leukemia, those in the ELN favorable-risk group with TET2 mutations had shorter event-free survival, lower rates of complete remission, and shorter disease-free survival.	3	250	157	2015-07-22 18:47:36.769202	2015-07-22 18:47:36.769202	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
428	In patients with renal clear cell carcinoma, patients with loss of function VHL mutations had shorter tumor-specific survival.	48	254	159	2015-07-22 18:47:36.816307	2015-07-22 18:47:36.816307	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
429	In non-APL patients with AML, those who had mutations in RUNX1 had higher rates of refractory disease than those without.	3	241	155	2015-07-22 18:47:36.855787	2015-07-22 18:47:36.855787	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
430	In non-APL patients with AML, those who had mutations in RUNX1 had shorter event-free survival and relapse-free survival relative to wild-type.	3	241	155	2015-07-22 18:47:36.902588	2015-07-22 18:47:36.902588	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
431	In patients with Ewing's sarcoma, those with type 1 EWS-FLI1 fusions had longer overall survival than those with other types of EWS-FLI1 fusions.	50	255	164	2015-07-22 18:47:36.955489	2015-07-22 18:47:36.955489	4	accepted	\N	\N	N/A	1	1	0	0	4	f	\N	\N
\.


--
-- Data for Name: drugs_evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs_evidence_items (drug_id, evidence_item_id) FROM stdin;
1	1
1	2
1	3
1	4
1	5
1	6
1	7
1	8
1	9
1	10
2	11
3	12
3	13
4	14
5	15
5	16
6	17
7	18
8	19
9	20
10	21
11	22
11	23
1	24
1	25
1	26
1	27
1	28
1	29
1	30
1	31
12	32
12	33
5	34
13	34
6	34
14	35
15	35
16	36
17	37
12	38
18	39
6	40
19	41
20	42
21	43
21	44
21	45
21	46
21	47
18	48
1	49
1	50
1	51
1	52
1	53
1	54
1	55
1	56
1	57
5	58
5	59
5	60
1	61
1	62
1	63
1	64
1	65
1	66
1	67
1	68
1	69
1	70
1	71
1	72
22	73
1	74
23	75
23	76
24	77
1	78
1	79
1	80
1	81
25	82
26	82
27	83
27	84
27	85
1	86
27	87
28	88
6	89
1	90
22	91
3	92
22	93
19	93
22	94
22	95
19	95
29	96
30	96
30	97
31	97
32	98
4	98
33	98
4	99
22	100
22	101
1	102
1	103
1	104
1	105
1	106
1	107
1	108
1	109
1	110
1	111
1	112
1	113
1	114
1	115
1	116
1	117
1	118
1	119
1	120
1	121
1	122
16	123
16	124
12	125
16	126
34	127
1	128
1	129
2	130
35	131
2	132
35	133
16	134
16	135
16	136
34	137
16	138
1	139
1	140
17	141
18	142
12	143
36	144
16	145
34	146
2	147
37	148
38	149
34	149
1	150
1	151
38	152
39	153
40	154
41	155
1	156
1	157
1	158
1	159
1	160
1	161
1	162
1	163
1	164
1	165
1	166
1	167
1	168
1	169
1	170
1	171
1	172
1	173
1	174
1	175
1	176
1	177
1	178
1	179
1	180
1	181
1	182
1	183
1	184
1	185
1	186
1	187
1	188
1	189
1	190
1	191
1	192
1	193
1	194
1	195
1	196
1	197
1	198
1	199
1	200
1	201
1	202
1	203
1	204
1	205
1	206
1	207
1	208
1	209
1	210
42	211
42	212
1	213
1	214
1	215
1	216
1	217
1	218
1	219
1	220
1	221
1	222
1	223
1	224
1	225
1	226
1	227
1	228
14	229
15	229
14	230
15	230
43	231
5	232
5	233
5	234
5	235
12	236
12	237
15	238
14	239
44	240
45	241
46	242
46	243
46	244
46	245
46	246
6	247
6	248
5	249
16	250
14	251
15	251
14	252
15	252
14	253
15	253
14	254
15	254
44	255
16	256
28	256
16	257
28	257
16	258
28	258
47	259
5	260
13	261
20	261
12	262
48	263
49	264
49	265
20	266
20	267
20	268
20	269
20	270
20	271
20	272
15	272
14	273
14	274
1	275
14	276
15	277
50	277
51	278
52	279
52	280
52	281
52	282
52	283
52	284
52	285
52	286
45	286
52	287
52	288
52	289
53	290
54	290
53	291
54	291
53	292
54	292
53	293
54	293
53	294
54	294
55	295
56	296
55	296
57	297
6	298
58	299
57	300
59	301
60	301
5	302
20	303
5	303
13	303
61	304
62	305
63	305
16	306
26	307
64	308
65	309
66	310
40	311
66	312
40	313
66	314
40	315
34	316
67	317
34	318
68	319
41	320
41	321
1	322
1	323
1	324
1	325
1	326
1	327
1	328
1	329
1	330
1	331
1	332
1	333
1	334
1	335
1	336
1	337
1	338
1	339
1	340
1	341
1	342
1	343
5	344
69	345
69	346
1	347
1	348
1	349
1	350
16	351
1	352
1	353
1	354
1	355
1	356
1	357
1	358
1	359
1	360
1	361
1	362
1	363
1	364
1	365
1	366
1	367
1	368
1	369
1	370
1	371
1	372
1	373
1	374
1	375
1	376
1	377
1	378
1	379
1	380
1	381
1	382
1	383
1	384
1	385
1	386
1	387
1	388
1	389
1	390
1	391
1	392
1	393
1	394
1	395
1	396
1	397
1	398
1	399
1	400
1	401
22	402
5	403
70	403
5	404
70	404
1	405
1	406
1	407
1	408
1	409
1	410
71	411
72	412
14	413
15	413
72	414
73	414
74	414
1	415
1	416
1	417
1	418
1	419
1	420
1	421
1	422
1	423
1	424
1	425
1	426
1	427
1	428
1	429
1	430
1	431
\.


--
-- Name: drugs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('drugs_id_seq', 74, true);


--
-- Data for Name: events; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY events (id, created_at, updated_at, action, description, originating_user_id, subject_id, subject_type) FROM stdin;
\.


--
-- Name: events_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('events_id_seq', 1, false);


--
-- Name: evidence_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_items_id_seq', 431, true);


--
-- Data for Name: feeds; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY feeds (id, created_at, updated_at, owner_id, event_id, acknowledged) FROM stdin;
\.


--
-- Name: feeds_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('feeds_id_seq', 1, false);


--
-- Data for Name: gene_aliases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY gene_aliases (id, name) FROM stdin;
1	CD246
2	NBLST3
3	AKT
4	CWS6
5	PKB
6	PKB-ALPHA
7	PRKBA
8	RAC
9	RAC-ALPHA
10	A-RAF
11	ARAF1
12	PKS2
13	RAFA1
14	ABL
15	JTK7
16	bcr/abl
17	c-ABL
18	c-ABL1
19	p150
20	v-abl
21	B-RAF1
22	BRAF1
23	NS7
24	RAFB1
25	BRCAI
26	BRCC1
27	BROVCA1
28	FANCS
29	IRIS
30	PNCA4
31	PPP1R53
32	PSCP
33	RNF53
34	BRCC2
35	BROVCA2
36	FACD
37	FAD
38	FAD1
39	FANCD
40	FANCD1
41	GLM3
42	PNCA2
43	XRCC11
44	BCL1
45	D11S287E
46	PRAD1
47	U21B31
48	KIAK0002
49	MPPH3
50	CCNE
51	pCCNE1
52	MCPH12
53	PLSTIRE
54	CMM3
55	PSK-J3
56	ARF
57	CDK4I
58	CDKN2
59	CMM2
60	INK4
61	INK4A
62	MLM
63	MTS-1
64	MTS1
65	P14
66	P14ARF
67	P16
68	P16-INK4A
69	P16INK4
70	P16INK4A
71	P19
72	P19ARF
73	TP16
74	C/EBP-alpha
75	CEBP
76	HSPF1
77	Hdj1
78	Hsp40
79	RSPH16B
80	Sis1
81	PKACA
82	PPNAD4
83	DNMT3A2
84	M.HsaIIIA
85	TBRS
86	ERBB
87	ERBB1
88	HER1
89	NISBD2
90	PIG61
91	mENA
92	CD340
93	HER-2
94	HER-2/neu
95	HER2
96	MLN 19
97	NEU
98	NGL
99	TKR1
100	ER
101	ESR
102	ESRA
103	ESTRR
104	Era
105	NR3A1
106	BBDS
107	BEK
108	BFR-1
109	CD332
110	CEK3
111	CFD1
112	ECT1
113	JWS
114	K-SAM
115	KGFR
116	TK14
117	TK25
118	ACH
119	CD333
120	CEK2
121	HSFGFR3EX
122	JTK4
123	CD135
124	FLK-2
125	FLK2
126	STK1
127	DCML
128	IMD21
129	MONOMAC
130	NFE1B
131	HEL-216
132	HEL-S-26
133	IDCD
134	IDH
135	IDP
136	IDPC
137	PICD
138	D2HGA2
139	ICD-M
140	IDHM
141	IDPM
142	mNADP-IDH
143	JTK10
144	THCYT3
145	C-Kit
146	CD117
147	PBT
148	SCFR
149	C-K-RAS
150	CFC2
151	K-RAS2A
152	K-RAS2B
153	K-RAS4A
154	K-RAS4B
155	KI-RAS
156	KRAS1
157	KRAS2
158	NS
159	NS3
160	RASK2
161	CFC3
162	MAPKK1
163	MEK1
164	MKK1
165	PRKMK1
166	C-FMS
167	CD115
168	CSF-1R
169	CSFR
170	FIM2
171	FMS
172	HDLS
173	M-CSF-R
174	B23
175	NPM
176	ALPS4
177	CMNS
178	N-ras
179	NCMS
180	NRAS1
181	NS6
182	CLOVE
183	CWS5
184	MCAP
185	MCM
186	MCMTC
187	PI3K
188	p110-alpha
189	CD140A
190	PDGFR-2
191	PDGFR2
192	RHEPDGFRA
193	MYL
194	PP8675
195	RNF71
196	TRIM19
197	NR1B1
198	RAR
199	10q23del
200	BZS
201	CWS1
202	DEC
203	GLM2
204	MHAM
205	MMAC1
206	PTEN1
207	TEP1
208	CDHF12
209	CDHR16
210	HSCR1
211	MEN2A
212	MEN2B
213	MTC1
214	PTC
215	RET-ELE1
216	RET51
217	AML1
218	AML1-EVI-1
219	AMLCR1
220	CBFA2
221	EVI-1
222	PEBP2aB
223	Hsh155
224	MDS
225	PRP10
226	PRPF10
227	SAP155
228	SF3b155
229	BCC7
230	LFS1
231	P53
232	TRP53
233	LAM
234	TSC
235	PPP1R160
236	TSC4
237	FP793
238	RN
239	RNU2AF1
240	U2AF35
241	U2AFBP
242	AWT1
243	EWS-WT1
244	GUD
245	NPHS4
246	WAGR
247	WIT-2
248	WT33
249	AOS5
250	AOVD1
251	TAN1
252	hN1
253	MIG20a
254	NTRKR3
255	TKT
256	TYRO10
257	AUTS9
258	DFNB97
259	HGFR
260	RCCP2
261	c-Met
262	EWSR2
263	SIC-1
264	EWS
265	EWS-FLI1
266	bK984G1.4
267	KIAA1546
268	12CC4
269	HSPC215
270	MFH
271	QRF1
272	hFKH1B
273	C-Rel
274	HRCA1
275	RCA1
276	VHL1
277	pVHL
278	Bcl-2
279	PPP1R50
280	C11orf43
281	IGF-II
282	PP9974
283	AIK
284	ARK1
285	AURA
286	AURORA2
287	BTAK
288	PPP1R47
289	STK15
290	STK6
291	STK7
292	BAF180
293	PB1
294	ENX-1
295	ENX1
296	EZH1
297	EZH2b
298	KMT6
299	KMT6A
300	WVS
301	WVS2
\.


--
-- Data for Name: gene_aliases_genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY gene_aliases_genes (gene_alias_id, gene_id) FROM stdin;
1	1
2	1
3	2
4	2
5	2
6	2
7	2
8	2
9	2
10	3
11	3
12	3
13	3
14	4
15	4
16	4
17	4
18	4
19	4
20	4
21	5
22	5
23	5
24	5
25	6
26	6
27	6
28	6
29	6
30	6
31	6
32	6
33	6
34	7
35	7
36	7
37	7
38	7
39	7
40	7
41	7
42	7
43	7
44	8
45	8
46	8
47	8
48	9
49	9
50	11
51	11
52	12
53	12
54	13
55	13
56	14
57	14
58	14
59	14
60	14
61	14
62	14
63	14
64	14
65	14
66	14
67	14
68	14
69	14
70	14
71	14
72	14
73	14
74	15
75	15
76	16
77	16
78	16
79	16
80	16
81	17
82	17
83	18
84	18
85	18
86	19
87	19
88	19
89	19
90	19
91	19
92	20
93	20
94	20
95	20
96	20
97	20
98	20
99	20
100	21
101	21
102	21
103	21
104	21
105	21
106	22
107	22
108	22
109	22
110	22
111	22
112	22
113	22
114	22
115	22
116	22
117	22
118	23
119	23
120	23
121	23
122	23
123	24
124	24
125	24
126	24
127	25
128	25
129	25
130	25
131	26
132	26
133	26
134	26
135	26
136	26
137	26
138	27
139	27
134	27
140	27
135	27
141	27
142	27
143	28
144	28
145	29
146	29
147	29
148	29
149	30
150	30
151	30
152	30
153	30
154	30
155	30
156	30
157	30
158	30
159	30
160	30
161	31
162	31
163	31
164	31
165	31
166	33
167	33
168	33
169	33
170	33
171	33
172	33
173	33
174	35
175	35
176	36
177	36
178	36
179	36
180	36
181	36
182	37
183	37
184	37
185	37
186	37
187	37
188	37
189	38
190	38
191	38
192	38
193	39
194	39
195	39
196	39
197	40
198	40
199	41
200	41
201	41
202	41
203	41
204	41
205	41
206	41
207	41
208	42
209	42
210	42
211	42
212	42
213	42
214	42
215	42
216	42
217	43
218	43
219	43
220	43
221	43
222	43
223	44
224	44
225	44
226	44
227	44
228	44
229	45
230	45
231	45
232	45
233	46
234	46
233	47
235	47
236	47
237	48
238	48
239	48
240	48
241	48
242	49
243	49
244	49
245	49
246	49
247	49
248	49
249	50
250	50
251	50
252	50
253	51
254	51
255	51
256	51
257	52
258	52
259	52
260	52
261	52
262	53
263	53
264	54
265	54
266	54
267	55
224	55
268	56
269	56
270	56
271	56
272	56
273	57
274	58
275	58
276	58
277	58
278	59
279	59
280	60
281	60
282	60
283	61
284	61
285	61
286	61
287	61
288	61
289	61
290	61
291	61
292	62
293	62
294	63
295	63
296	63
297	63
298	63
299	63
300	63
301	63
\.


--
-- Name: gene_aliases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('gene_aliases_id_seq', 301, true);


--
-- Name: genes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('genes_id_seq', 63, true);


--
-- Data for Name: genes_sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes_sources (gene_id, source_id, created_at, updated_at) FROM stdin;
1	1	2015-07-22 18:46:55.501885	2015-07-22 18:46:55.501885
1	2	2015-07-22 18:46:55.513413	2015-07-22 18:46:55.513413
2	3	2015-07-22 18:46:55.763005	2015-07-22 18:46:55.763005
2	4	2015-07-22 18:46:55.772003	2015-07-22 18:46:55.772003
3	5	2015-07-22 18:46:56.020553	2015-07-22 18:46:56.020553
3	6	2015-07-22 18:46:56.030415	2015-07-22 18:46:56.030415
4	7	2015-07-22 18:46:56.286645	2015-07-22 18:46:56.286645
4	8	2015-07-22 18:46:56.297278	2015-07-22 18:46:56.297278
5	9	2015-07-22 18:46:56.583097	2015-07-22 18:46:56.583097
5	10	2015-07-22 18:46:56.592078	2015-07-22 18:46:56.592078
6	11	2015-07-22 18:46:56.861237	2015-07-22 18:46:56.861237
7	11	2015-07-22 18:46:57.116243	2015-07-22 18:46:57.116243
8	12	2015-07-22 18:46:57.357998	2015-07-22 18:46:57.357998
8	13	2015-07-22 18:46:57.368081	2015-07-22 18:46:57.368081
9	12	2015-07-22 18:46:57.573354	2015-07-22 18:46:57.573354
9	13	2015-07-22 18:46:57.576114	2015-07-22 18:46:57.576114
10	12	2015-07-22 18:46:57.762437	2015-07-22 18:46:57.762437
10	13	2015-07-22 18:46:57.765413	2015-07-22 18:46:57.765413
11	14	2015-07-22 18:46:57.966537	2015-07-22 18:46:57.966537
12	15	2015-07-22 18:46:58.183314	2015-07-22 18:46:58.183314
12	16	2015-07-22 18:46:58.192134	2015-07-22 18:46:58.192134
13	15	2015-07-22 18:46:58.38512	2015-07-22 18:46:58.38512
13	16	2015-07-22 18:46:58.387555	2015-07-22 18:46:58.387555
14	17	2015-07-22 18:46:58.674873	2015-07-22 18:46:58.674873
14	18	2015-07-22 18:46:58.68378	2015-07-22 18:46:58.68378
15	19	2015-07-22 18:46:58.929185	2015-07-22 18:46:58.929185
15	20	2015-07-22 18:46:58.938627	2015-07-22 18:46:58.938627
15	21	2015-07-22 18:46:58.947871	2015-07-22 18:46:58.947871
16	22	2015-07-22 18:46:59.173187	2015-07-22 18:46:59.173187
17	22	2015-07-22 18:46:59.390373	2015-07-22 18:46:59.390373
17	23	2015-07-22 18:46:59.40021	2015-07-22 18:46:59.40021
18	24	2015-07-22 18:46:59.611817	2015-07-22 18:46:59.611817
18	25	2015-07-22 18:46:59.622317	2015-07-22 18:46:59.622317
19	26	2015-07-22 18:46:59.880775	2015-07-22 18:46:59.880775
19	27	2015-07-22 18:46:59.89447	2015-07-22 18:46:59.89447
20	28	2015-07-22 18:47:00.15944	2015-07-22 18:47:00.15944
20	29	2015-07-22 18:47:00.173923	2015-07-22 18:47:00.173923
21	30	2015-07-22 18:47:00.434352	2015-07-22 18:47:00.434352
21	31	2015-07-22 18:47:00.445501	2015-07-22 18:47:00.445501
22	32	2015-07-22 18:47:00.71608	2015-07-22 18:47:00.71608
22	33	2015-07-22 18:47:00.725913	2015-07-22 18:47:00.725913
23	32	2015-07-22 18:47:00.963492	2015-07-22 18:47:00.963492
23	33	2015-07-22 18:47:00.966237	2015-07-22 18:47:00.966237
24	34	2015-07-22 18:47:01.197805	2015-07-22 18:47:01.197805
24	19	2015-07-22 18:47:01.200534	2015-07-22 18:47:01.200534
25	35	2015-07-22 18:47:01.417558	2015-07-22 18:47:01.417558
25	36	2015-07-22 18:47:01.426995	2015-07-22 18:47:01.426995
26	37	2015-07-22 18:47:01.671879	2015-07-22 18:47:01.671879
26	38	2015-07-22 18:47:01.682494	2015-07-22 18:47:01.682494
27	37	2015-07-22 18:47:01.918443	2015-07-22 18:47:01.918443
27	38	2015-07-22 18:47:01.921321	2015-07-22 18:47:01.921321
27	39	2015-07-22 18:47:01.930443	2015-07-22 18:47:01.930443
28	40	2015-07-22 18:47:02.147011	2015-07-22 18:47:02.147011
29	41	2015-07-22 18:47:02.3596	2015-07-22 18:47:02.3596
30	42	2015-07-22 18:47:02.623487	2015-07-22 18:47:02.623487
35	43	2015-07-22 18:47:03.72482	2015-07-22 18:47:03.72482
35	19	2015-07-22 18:47:03.727223	2015-07-22 18:47:03.727223
42	44	2015-07-22 18:47:05.391857	2015-07-22 18:47:05.391857
42	45	2015-07-22 18:47:05.404272	2015-07-22 18:47:05.404272
44	46	2015-07-22 18:47:05.902669	2015-07-22 18:47:05.902669
44	47	2015-07-22 18:47:05.916319	2015-07-22 18:47:05.916319
45	48	2015-07-22 18:47:06.178732	2015-07-22 18:47:06.178732
48	49	2015-07-22 18:47:06.860841	2015-07-22 18:47:06.860841
48	50	2015-07-22 18:47:06.871586	2015-07-22 18:47:06.871586
\.


--
-- Name: organizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('organizations_id_seq', 1, true);


--
-- Data for Name: ratings; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY ratings (id, value, evidence_item_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Name: ratings_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('ratings_id_seq', 1, false);


--
-- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY schema_migrations (version) FROM stdin;
20140805184308
20141021192035
20141022180523
20141022195329
20141108234203
20141108234935
20141109000136
20141114222425
20150107205621
20150115231117
20150116220720
20150116230539
20150116230632
20150204192653
20150205000010
20150205205734
20150205210725
20150210191015
20150210204445
20150219202227
20150220160805
20150220182109
20150223170106
20150227204142
20150227212146
20150303202505
20150305200429
20150319181853
20150416182633
20150501151624
20150511191452
20150512184905
20150528170506
20150528211748
20150602202928
20150609210448
20150611153328
20150611154621
20150611200713
20150709222321
20150709230109
20150722183121
20150722185207
20150722185935
\.


--
-- Name: sources_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('sources_id_seq', 255, true);


--
-- Data for Name: subscriptions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY subscriptions (id, user_id, subscribable_id, subscribable_type, type, created_at, updated_at, action_type, action_class) FROM stdin;
\.


--
-- Name: subscriptions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('subscriptions_id_seq', 1, false);


--
-- Data for Name: suggested_changes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY suggested_changes (id, suggested_changes, moderated_id, moderated_type, user_id, status, created_at, updated_at) FROM stdin;
\.


--
-- Name: suggested_changes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('suggested_changes_id_seq', 1, false);


--
-- Data for Name: tsv_releases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY tsv_releases (id, path, created_at, updated_at) FROM stdin;
\.


--
-- Name: tsv_releases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('tsv_releases_id_seq', 1, false);


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('users_id_seq', 1, true);


--
-- Data for Name: variant_groups; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_groups (id, name, description, created_at, updated_at, deleted, deleted_at) FROM stdin;
2	KIT Exon 17	\N	2015-07-22 18:47:15.778183	2015-07-22 18:47:15.778183	f	\N
4	KIT Exon 11	\N	2015-07-22 18:47:16.190222	2015-07-22 18:47:16.190222	f	\N
1	Imatinib Resistance	While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. 	2015-07-22 18:47:14.553378	2015-07-22 18:47:37.064481	f	\N
11	ALK Fusions	ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase the activity of the ALK oncogene relative to normal cells. While EML4 is the most common fusion partner, other 5' partners have been observed. The EML4-ALK fusion has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. 	2015-07-22 18:47:26.448063	2015-07-22 18:47:37.095469	f	\N
12	EGFR TKI Resistance	EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine kinase inhibitors have been developed to target EGFR pathway activity. One such inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting. However, the T790M missense mutation has shown to confer resistance to this inhibitor in cell lines and case studies.  	2015-07-22 18:47:26.592132	2015-07-22 18:47:37.118409	f	\N
3	Crizotinib Resistance	The ALK oncogene has long been considered a driving factor in non-small cell lung cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive this resistance. 	2015-07-22 18:47:16.104004	2015-07-22 18:47:37.142	f	\N
5	Other V600's	While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants have also been observed and studied to a lesser degree. At first approximation, many of these variants seem to behave similarly to V600E, and treatment with dabrafenib has been shown to be effective. 	2015-07-22 18:47:17.91533	2015-07-22 18:47:37.158559	f	\N
14	HER2 Activating	HER2-positive breast cancer is widely recognized as a molecular subtype of breast cancer. While amplification of ERBB2 is a common mechanism for activation of the pathway, missense mutations have also demonstrated activation potential. These activating missense mutations also confer senstivity to the targeted therapeutic neratinib. 	2015-07-22 18:47:28.843172	2015-07-22 18:47:37.174851	f	\N
13	ESR1 Ligand-Binding Domain	ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone therapy has been widely effective in treating the disease, however sequencing of resistant patients has uncovered a number of missense mutations in mediating this resistance. Many of these mutations lie within the ligand-binding domain, and contribute to constitutive activity of the receptor. This has lead to the development of estrogen receptor degrading agents such as fulvestrant, which have shown early efficacy in clinical trials.  	2015-07-22 18:47:26.873406	2015-07-22 18:47:37.190017	f	\N
8	FGFR fusions	FGFR fusions have been demonstrated across many cancer types to have oncogenic potential. FGFR2 and FGFR3 have been shown to be fused to many 3' partners, which may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2 fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib and pazopanib have shown efficacy in case studies.  	2015-07-22 18:47:21.699553	2015-07-22 18:47:37.205739	f	\N
7	NPM1 Exon 12	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-07-22 18:47:19.735025	2015-07-22 18:47:37.223561	f	\N
15	PTEN Loss-of-Function	PTEN loss is a common event in breast cancer, as well as others. The most common single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell lines harboring this mutation behave similarly to cell lines harboring larger insertion or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded growth. 	2015-07-22 18:47:30.782706	2015-07-22 18:47:37.24083	f	\N
6	Motesanib Resistance	RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment of these patients with the targeted therapeutic motesanib has shown to be effective. However, the missense mutations C634W and M918T have shown to confer motesanib resistance in cell lines. 	2015-07-22 18:47:18.040739	2015-07-22 18:47:37.255414	f	\N
9	TSC Loss	Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types. Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor everolimus has seen significant anti-tumorigenic activity. 	2015-07-22 18:47:22.291321	2015-07-22 18:47:37.270395	f	\N
10	BRCA Germline Variants	BRCA germline variants have long been recognized as important potential predictors of breast and ovarian cancer risk. 	2015-07-22 18:47:25.049008	2015-07-22 18:47:37.286865	f	\N
\.


--
-- Data for Name: variant_group_variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_group_variants (variant_id, variant_group_id, created_at, updated_at) FROM stdin;
99	1	2015-07-22 18:47:14.577475	2015-07-22 18:47:14.577475
99	1	2015-07-22 18:47:15.193697	2015-07-22 18:47:15.193697
99	1	2015-07-22 18:47:15.252776	2015-07-22 18:47:15.252776
65	2	2015-07-22 18:47:15.793912	2015-07-22 18:47:15.793912
8	3	2015-07-22 18:47:16.11982	2015-07-22 18:47:16.11982
72	4	2015-07-22 18:47:16.202909	2015-07-22 18:47:16.202909
72	4	2015-07-22 18:47:16.575558	2015-07-22 18:47:16.575558
98	1	2015-07-22 18:47:16.642891	2015-07-22 18:47:16.642891
99	1	2015-07-22 18:47:16.693658	2015-07-22 18:47:16.693658
100	1	2015-07-22 18:47:16.737411	2015-07-22 18:47:16.737411
101	1	2015-07-22 18:47:16.789733	2015-07-22 18:47:16.789733
102	1	2015-07-22 18:47:16.830456	2015-07-22 18:47:16.830456
66	4	2015-07-22 18:47:17.068002	2015-07-22 18:47:17.068002
66	4	2015-07-22 18:47:17.266623	2015-07-22 18:47:17.266623
70	2	2015-07-22 18:47:17.355078	2015-07-22 18:47:17.355078
13	5	2015-07-22 18:47:17.926572	2015-07-22 18:47:17.926572
112	6	2015-07-22 18:47:18.054514	2015-07-22 18:47:18.054514
113	6	2015-07-22 18:47:18.094705	2015-07-22 18:47:18.094705
11	5	2015-07-22 18:47:18.892702	2015-07-22 18:47:18.892702
15	5	2015-07-22 18:47:19.236894	2015-07-22 18:47:19.236894
16	5	2015-07-22 18:47:19.285685	2015-07-22 18:47:19.285685
87	7	2015-07-22 18:47:19.754647	2015-07-22 18:47:19.754647
87	7	2015-07-22 18:47:20.337148	2015-07-22 18:47:20.337148
53	8	2015-07-22 18:47:21.731737	2015-07-22 18:47:21.731737
87	7	2015-07-22 18:47:22.157854	2015-07-22 18:47:22.157854
87	7	2015-07-22 18:47:22.222329	2015-07-22 18:47:22.222329
124	9	2015-07-22 18:47:22.305553	2015-07-22 18:47:22.305553
125	9	2015-07-22 18:47:22.372605	2015-07-22 18:47:22.372605
131	10	2015-07-22 18:47:25.069953	2015-07-22 18:47:25.069953
132	10	2015-07-22 18:47:25.124436	2015-07-22 18:47:25.124436
66	4	2015-07-22 18:47:25.8869	2015-07-22 18:47:25.8869
70	2	2015-07-22 18:47:25.931433	2015-07-22 18:47:25.931433
3	1	2015-07-22 18:47:26.285262	2015-07-22 18:47:26.285262
2	1	2015-07-22 18:47:26.336486	2015-07-22 18:47:26.336486
2	1	2015-07-22 18:47:26.389788	2015-07-22 18:47:26.389788
6	11	2015-07-22 18:47:26.467523	2015-07-22 18:47:26.467523
6	3	2015-07-22 18:47:26.472576	2015-07-22 18:47:26.472576
7	11	2015-07-22 18:47:26.525203	2015-07-22 18:47:26.525203
7	3	2015-07-22 18:47:26.528999	2015-07-22 18:47:26.528999
34	12	2015-07-22 18:47:26.611666	2015-07-22 18:47:26.611666
34	12	2015-07-22 18:47:26.757057	2015-07-22 18:47:26.757057
46	13	2015-07-22 18:47:26.886547	2015-07-22 18:47:26.886547
47	13	2015-07-22 18:47:26.948636	2015-07-22 18:47:26.948636
48	13	2015-07-22 18:47:27.002044	2015-07-22 18:47:27.002044
49	13	2015-07-22 18:47:27.052303	2015-07-22 18:47:27.052303
50	13	2015-07-22 18:47:27.101178	2015-07-22 18:47:27.101178
73	4	2015-07-22 18:47:27.259898	2015-07-22 18:47:27.259898
148	12	2015-07-22 18:47:27.382228	2015-07-22 18:47:27.382228
78	12	2015-07-22 18:47:27.433908	2015-07-22 18:47:27.433908
79	12	2015-07-22 18:47:27.491468	2015-07-22 18:47:27.491468
147	12	2015-07-22 18:47:27.54881	2015-07-22 18:47:27.54881
146	12	2015-07-22 18:47:27.597049	2015-07-22 18:47:27.597049
5	11	2015-07-22 18:47:27.962612	2015-07-22 18:47:27.962612
35	14	2015-07-22 18:47:28.856306	2015-07-22 18:47:28.856306
36	14	2015-07-22 18:47:28.902374	2015-07-22 18:47:28.902374
38	14	2015-07-22 18:47:28.990264	2015-07-22 18:47:28.990264
41	14	2015-07-22 18:47:29.121628	2015-07-22 18:47:29.121628
43	14	2015-07-22 18:47:29.219414	2015-07-22 18:47:29.219414
44	14	2015-07-22 18:47:29.27721	2015-07-22 18:47:29.27721
45	14	2015-07-22 18:47:29.328331	2015-07-22 18:47:29.328331
46	13	2015-07-22 18:47:29.40546	2015-07-22 18:47:29.40546
47	13	2015-07-22 18:47:29.499301	2015-07-22 18:47:29.499301
48	13	2015-07-22 18:47:29.557418	2015-07-22 18:47:29.557418
49	13	2015-07-22 18:47:29.617684	2015-07-22 18:47:29.617684
50	13	2015-07-22 18:47:29.678369	2015-07-22 18:47:29.678369
52	8	2015-07-22 18:47:29.746624	2015-07-22 18:47:29.746624
54	8	2015-07-22 18:47:29.816475	2015-07-22 18:47:29.816475
67	4	2015-07-22 18:47:30.127505	2015-07-22 18:47:30.127505
72	4	2015-07-22 18:47:30.18776	2015-07-22 18:47:30.18776
73	4	2015-07-22 18:47:30.238992	2015-07-22 18:47:30.238992
110	15	2015-07-22 18:47:30.795105	2015-07-22 18:47:30.795105
124	9	2015-07-22 18:47:30.931185	2015-07-22 18:47:30.931185
126	9	2015-07-22 18:47:30.973578	2015-07-22 18:47:30.973578
66	4	2015-07-22 18:47:31.553538	2015-07-22 18:47:31.553538
66	4	2015-07-22 18:47:31.617264	2015-07-22 18:47:31.617264
110	15	2015-07-22 18:47:32.114379	2015-07-22 18:47:32.114379
66	4	2015-07-22 18:47:33.894328	2015-07-22 18:47:33.894328
\.


--
-- Name: variant_groups_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_groups_id_seq', 15, true);


--
-- Name: variants_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variants_id_seq', 164, true);


--
-- PostgreSQL database dump complete
--

