--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

SET search_path = public, pg_catalog;

--
-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY users (id, email, name, url, nickname, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: audits; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY audits (id, auditable_id, auditable_type, associated_id, associated_type, user_id, user_type, username, action, audited_changes, version, comment, remote_address, request_uuid, created_at) FROM stdin;
1	1	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 238\nname: ALK\ndescription: ALK amplifications, fusions and mutations have been shown to be driving\n  events in non-small cell lung cancer. While crizontinib has demonstrated efficacy\n  in treating the amplification, mutations in ALK have been shown to confer resistance\n  to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied\n  success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been\n  shown to be cytostatic in ALK-altered cell lines.\nofficial_name: anaplastic lymphoma receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	125e1ce2-3d11-47f0-af3e-404f42998565	2015-03-04 17:25:26.504482
2	2	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 207\nname: AKT1\ndescription: AKT1, also referred to as protein kinase B, is a known oncogene. AKT\n  activation relies on the PI3K pathway, and is recognized as a critical node in the\n  pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been\n  shown to result in activation of the protein. Mutations in AKT1 have also been shown\n  to confer resistance to allosteric kinase inhibitors in vitro.\nofficial_name: v-akt murine thymoma viral oncogene homolog 1\nclinical_description: \n	1	\N	\N	e3a80b89-11c9-44fa-a277-186a44d52401	2015-03-04 17:25:26.783067
3	3	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 369\nname: ARAF\ndescription: ARAF has recently become increasingly considered for its oncogenic potential.\n  Its potential as a target for informing clinical action was demonstrated by a single\n  case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated\n  with sorafenib, acheived near-complete clinical remission. This finding has brought\n  new focus on ARAF as a marker that should be assayed for in cancer treatment.\nofficial_name: A-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	bfc3a77b-d1fa-4125-91f1-ec0df9326776	2015-03-04 17:25:27.015827
4	4	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 25\nname: ABL1\ndescription: ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein\n  that has become a signature of chronic myeloid leukemia (CML). Cells harboring this\n  fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook\n  of the disease. However, additional mutations in ABL1 have been shown to confer\n  resistance to imatinib. In these resistance cases, second-generation tyrosine kinase\n  inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are\n  currently undergoing clinical trials for treating acquired resistance in CML.\nofficial_name: ABL proto-oncogene 1, non-receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	ed7032cd-b0d0-4d03-bc71-62d00dd09d8e	2015-03-04 17:25:27.303132
5	5	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 673\nname: BRAF\ndescription: BRAF mutations are found to be recurrent in many cancer types. Of these,\n  the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E\n  has been determined to be an activating mutation, and cells that harbor it, along\n  with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is\n  also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK\n  inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations\n  have also been correlated with poor prognosis in many cancer types, although there\n  is at least one study that questions this conclusion in papillary thyroid cancer.\nofficial_name: B-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	b3f783a7-6998-4e24-a0f6-2d4433b5547f	2015-03-04 17:25:27.542017
6	6	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 672\nname: BRCA1\ndescription: BRCA1 mutations in the germline have become a hallmark for hereditary\n  breast and ovarian cancers. Variants that have been demonstrated to reduce the function\n  of the protein have been shown to increase risk for these cancers, as well as prostate\n  and pancreatic cancer. These findings have been the impetus for the increased popularity\n  of genetic testing of healthy indivudals to assess risk.\nofficial_name: breast cancer 1, early onset\nclinical_description: \n	1	\N	\N	a3fef212-2cd9-4506-9621-ad5ad8d5a0c6	2015-03-04 17:25:27.798885
7	7	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 675\nname: BRCA2\ndescription: BRCA2 mutations in the germline have become a hallmark for hereditary\n  breast and ovarian cancers. Variants that have been demonstrated to reduce the function\n  of the protein have been shown to increase risk for these cancers, as well as prostate\n  and pancreatic cancer. These findings have been the impetus for the increased popularity\n  of genetic testing of healthy indivudals to assess risk.\nofficial_name: breast cancer 2, early onset\nclinical_description: \n	1	\N	\N	7c76a06d-46fd-40c3-9bca-afa3a28769bc	2015-03-04 17:25:28.081293
8	8	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 595\nname: CCND1\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D1\nclinical_description: \n	1	\N	\N	beb935cf-360d-409f-b882-c3c37f281ec6	2015-03-04 17:25:28.339385
9	9	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 894\nname: CCND2\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D2\nclinical_description: \n	1	\N	\N	d6f8794e-fa8f-4ba2-8daf-bd0562a83b15	2015-03-04 17:25:28.567635
10	10	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 896\nname: CCND3\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared\n  to the amplifcations and overexpressions of the other cyclin D's. In a mouse study,\n  the targeted therapeutic palbociclib significantly increased the median survival\n  of the cyclin D3 knockouts.\nofficial_name: cyclin D3\nclinical_description: \n	1	\N	\N	41f554c8-80df-4e17-8998-a538b10b84c4	2015-03-04 17:25:28.783866
41	41	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5728\nname: PTEN\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: phosphatase and tensin homolog\nclinical_description: \n	1	\N	\N	5fe8950f-de5f-46b0-a3ed-8c8582175663	2015-03-04 17:25:36.490753
11	11	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 898\nname: CCNE1\ndescription: Cyclin E, while currenly not as widely implicated as its cyclin D counterparts,\n  has been implicated in various carcinomas, including breast, gastric, stomach and\n  colorectal. High levels of cyclin E, either by gene amplification or overexpression,\n  have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements\n  are correlated with later stage disease. In lung cancer, neoplastic cells with higher\n  levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly\n  expressed counterparts.\nofficial_name: cyclin E1\nclinical_description: \n	1	\N	\N	eb755b2f-8346-477d-8eca-8d31f3910a16	2015-03-04 17:25:28.992722
12	12	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1021\nname: CDK6\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 6\nclinical_description: \n	1	\N	\N	3347ce81-ed12-4d6e-bea5-abaec215d2e0	2015-03-04 17:25:29.212483
13	13	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1019\nname: CDK4\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 4\nclinical_description: \n	1	\N	\N	0b055dd9-04b2-4803-ac4f-555cafd0b29c	2015-03-04 17:25:29.422785
14	14	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1029\nname: CDKN2A\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. While no targeted therapeutic has been engaged in clinical trials, the prognostic\n  impact has been studied by a number of meta-analyses. One mechanism by which loss\n  of CDKN2A can occur is by hypermethylation of the promoter region for the gene.\n  However, the prognostic impact of promoter hypermethylation has been relatively\n  ambiguous. Many studies have suggesting poorer prognostic outcome for patients with\n  hypermethylation in colorectal, liver, and younger lung cancer patients. This being\n  said, there is still research to be done before this becomes a widely-accepted prognostic\n  indicator.\nofficial_name: cyclin-dependent kinase inhibitor 2A\nclinical_description: \n	1	\N	\N	5d6097e2-cca4-49f7-8a72-49adc25e2c17	2015-03-04 17:25:29.772837
15	15	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1050\nname: CEBPA\ndescription: AML with mutated CEBPA' is a provisional entity in the WHO classification\n  of acute myeloid leukemia (AML) and is recommended to be tested for in patients\n  with AML. CEBPA mutations are particularly associated with cytogenetically normal\n  AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation\n  in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha\n  protein while C-terminal mutations reduce the DNA-binding potential of this transcription\n  factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1\n  and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations\n  may have prognostic implications.\nofficial_name: CCAAT/enhancer binding protein (C/EBP), alpha\nclinical_description: \n	1	\N	\N	097951ad-6483-4218-9966-943b8d1c8509	2015-03-04 17:25:30.103884
16	16	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3337\nname: DNAJB1\ndescription: DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating\n  p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1\n  to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In\n  one study, this fusion was observed in 15/15 FL-HCC cases examined and functional\n  studies found that the fusion retained kinase activity.\nofficial_name: DnaJ (Hsp40) homolog, subfamily B, member 1\nclinical_description: \n	1	\N	\N	a45ce17f-b209-4daf-85bd-e2323b855a0b	2015-03-04 17:25:30.339436
17	17	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5566\nname: PRKACA\ndescription: PRKACA has been studied in breast cancer and has been found to mediate\n  resistance to HER2 targeted therapies. It has also been found to contain a mutation\n  hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular\n  fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be\n  a diagnostic marker for this rare subtype of HCC. In one study, this fusion was\n  observed in 15/15 FL-HCC cases examined and functional studies found that the fusion\n  retained kinase activity.\nofficial_name: protein kinase, cAMP-dependent, catalytic, alpha\nclinical_description: \n	1	\N	\N	1b5abd0f-019c-45ff-9c4f-82eadc5ad78d	2015-03-04 17:25:30.559724
18	18	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1788\nname: DNMT3A\ndescription: DNMT3A is one of several epigenetic modifiers identified as recurrently\n  mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically\n  normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant\n  negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers.\n  AML patient bone marrow harboring R882 mutations were similarly demonstrated to\n  be hypomethylated compared to patients with wildtype DNMT3A. These studies also\n  indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner\n  from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes\n  and treatment protocols may need to be considered for these two classes of DNMT3A\n  mutations.\nofficial_name: DNA (cytosine-5-)-methyltransferase 3 alpha\nclinical_description: \n	1	\N	\N	417dd180-ed31-4b17-867c-6066007b527d	2015-03-04 17:25:30.858058
19	19	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1956\nname: EGFR\ndescription: EGFR is widely recognized for its importance in cancer. Amplification\n  and mutations have been shown to be driving events in many cancer types. Its role\n  in non-small cell lung cancer has spurred many research and drug development efforts.\n  Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably\n  gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance\n  to these drugs, particularly the variant T790M, which has been functionally characterized\n  as a resistance marker for both of these drugs. The later generation TKI's have\n  seen some success in treating these resistant cases, and targeted sequencing of\n  the EGFR locus has become a common practice in non-small cell lung cancer.\nofficial_name: epidermal growth factor receptor\nclinical_description: \n	1	\N	\N	a0607b02-f4a1-4f8e-8146-332a998b0c3e	2015-03-04 17:25:31.113345
20	20	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2064\nname: ERBB2\ndescription: ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast\n  cancer, and is treated in a separate manner than the other subtypes of breast cancer.\n  Apart from being amplified, ERBB2 activating mutations have been shown to have clinical\n  importance in HER2-negative breast cancer. These mutations have shown sensitivity\n  to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical\n  sequencing efforts in treating breast cancer.\nofficial_name: erb-b2 receptor tyrosine kinase 2\nclinical_description: \n	1	\N	\N	6f564722-59e6-488e-8bdd-9de1761816b0	2015-03-04 17:25:31.387085
21	21	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2099\nname: ESR1\ndescription: ESR1 has been a focus in breast cancer for quite some time, but has also\n  shown significance in endometrial, ovarian and other cancer types. ER-positive breast\n  cancer that is resistant to hormone therapy has instigated clinical sequencing efforts\n  to shed light on the mechanisms of this resistance. A number of mutations in the\n  ligand binding domain of the protein have been implicated in hormone resistance\n  and anti-estrogen therapies. This has spurred efforts to develop therapeutics that\n  act to degrade the protein, rather than act as an antagonist. These agents are currently\n  in clinical trials and have seen some success, highlighting the importance of sequencing\n  efforts in treating breast cancer.\nofficial_name: estrogen receptor 1\nclinical_description: \n	1	\N	\N	dc18a3e6-1087-4a60-b08a-1f12d8610830	2015-03-04 17:25:31.646865
22	22	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2263\nname: FGFR2\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions that lead to pathway\n  actiation have been demonstrated to have oncogenic potential across multiple cancer\n  types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success\n  in treating over-active FGFR signalling, prompting use of diagnostic sequencing\n  targeting the FGFR genes, especially in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 2\nclinical_description: \n	1	\N	\N	fe32faa2-c5cd-4449-b5ab-493be77f7316	2015-03-04 17:25:31.932708
23	23	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2261\nname: FGFR3\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions that lead to pathway\n  actiation have been demonstrated to have oncogenic potential across multiple cancer\n  types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success\n  in treating over-active FGFR signalling, prompting use of diagnostic sequencing\n  targeting the FGFR genes, especially in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 3\nclinical_description: \n	1	\N	\N	ae7659fa-08de-4154-a1a1-5451e63d2faf	2015-03-04 17:25:32.228491
24	24	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2322\nname: FLT3\ndescription: FLT3 is an important cytokine receptor involved in normal hematopoiesis.\n  Mutations in this gene are common in acute myeloid leukemia (AML) and screening\n  for mutations in this gene has been recommended by the World Health Organization\n  in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML).\n  FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated\n  with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have\n  been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD\n  mutations are still up for debate.\nofficial_name: fms-related tyrosine kinase 3\nclinical_description: \n	1	\N	\N	aefaa2f8-64b5-4c24-b024-6c0513257276	2015-03-04 17:25:32.463161
25	25	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2624\nname: GATA2\ndescription: GATA2 is a transcription factor involved in stem cell maintenance with\n  key roles in hematopoietic development. GATA2 mutations are associated with a variety\n  of inherited and acquired immune disorders including myelodysplastic syndrome and\n  acute myeloid leukemia. In addition to a role in hematopoiesis, the maintenance\n  GATA2 expression has been implicated as a requirement in KRAS-driven non-small cell\n  lung cancer. Preclinical models have indicated therapeutic benefit from targeting\n  GATA2-mediated pathways in the context of KRAS-driven NSCLC.\nofficial_name: GATA binding protein 2\nclinical_description: \n	1	\N	\N	3fff1357-b87f-49ce-8973-a9de79ac928f	2015-03-04 17:25:32.706104
26	26	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3417\nname: IDH1\ndescription: IDH1 mutations have been observed in a number of cancer types, including\n  sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in\n  the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial\n  (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described\n  as mutually exclusive in many of these cancer types. The most frequent mutations\n  involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic\n  enzyme activity. The implications of mutations in this gene vary greatly by cancer\n  type. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations\n  have been associated with worse outcome, shorter overall survival, and normal karyotype.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Unlike the association\n  with cytogenetically normal AML, in glioblastoma, IDH1 mutations have been associated\n  with specific cytogenetic abnormalities, 1p and 19q deletions.\nofficial_name: isocitrate dehydrogenase 1 (NADP+), soluble\nclinical_description: \n	1	\N	\N	79444541-2cb1-4d24-b2d0-fc21276ade46	2015-03-04 17:25:32.984658
27	27	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3418\nname: IDH2\ndescription: IDH2 mutations have been observed in a number of cancer types, including\n  sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in\n  the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial\n  (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described\n  as mutually exclusive in many of these cancer types. The most frequent mutations\n  involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic\n  enzyme activity. Although IDH2 (R172) mutations are associated with poorer overall\n  prognosis in AML patients, its utility as a prognostic marker in MDS is still under\n  debate. Additionally, IDH2 (R140) has been associated with improved overall survival\n  in AML. IDH2 mutations have been associated with improved prognosis in gliomas.\nofficial_name: isocitrate dehydrogenase 2 (NADP+), mitochondrial\nclinical_description: \n	1	\N	\N	52d73236-2800-4e05-97ba-4af1ccc091fb	2015-03-04 17:25:33.252797
28	28	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3717\nname: JAK2\ndescription: JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative\n  diseases and cancers. The mutation V617F is the most clinically relevant variant,\n  and is seen in around half of myeloproliferative disorders. The variant is a known\n  activating mutation, and activated JAK2 is sufficient to drive myeloproliferative\n  disorders in mouse models. V617F, while most recurrent, is not the only mechanism\n  by which JAK2 can be activated in patients. JAK2 is now one of the first diagnostic\n  markers tested upon diagnosis with a myeloproliferative disorder.\nofficial_name: Janus kinase 2\nclinical_description: \n	1	\N	\N	d5eb98fc-8a7f-48b6-8bd5-3f3e6f26b127	2015-03-04 17:25:33.469685
193	140	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 52\nname: G505S\ndescription: Activating mutations in DDR2, including G505S, has been shown to be sensitive\n  to dasatinib in cell lines.\n	1	\N	\N	545aff0b-0562-4443-adcd-a257c4783f4e	2015-03-04 17:25:43.567492
29	29	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3815\nname: KIT\ndescription: c-KIT activation has been shown to have oncogenic activity in gastrointestinal\n  stromal tumors (GISTs), melanomas, lung cancer, and other tumor types. The targeted\n  therapeutics nilotinib and sunitinib have shown efficacy in treating KIT overactive\n  patients, and are in late-stage trials in melanoma and GIST. KIT overactivity can\n  be the result of many genomic events from genomic amplification to overexpression\n  to missense mutations. Missense mutations have been shown to be key players in mediating\n  clinical response and acquired resistance in patients being treated with these targeted\n  therapeutics.\nofficial_name: v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	51189c89-e83a-4c96-b92b-92b8bd15ce9a	2015-03-04 17:25:33.705093
30	30	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3845\nname: KRAS\ndescription: Mutations in the RAS family of proteins have frequently observed across\n  cancer types. The amino acid positions account for the overwhelming majority of\n  these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also\n  behave very differently when expressed in non-native tissue types, likely due to\n  differences in the C-terminal hyper-variable regions. Mis-regulation of isoform\n  expression has been shown to be a driving event in cancer, as well as missense mutations\n  at the three hotspots previously mentioned. While highly recurrent in cancer, targeted\n  these RAS mutants has also been very elusive, and has not yet become common practice\n  in the clinic.\nofficial_name: Kirsten rat sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	15165124-a4ae-430d-a7e5-4890b89a7270	2015-03-04 17:25:33.995649
31	31	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5604\nname: MAP2K1\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: mitogen-activated protein kinase kinase 1\nclinical_description: \n	1	\N	\N	f478b9c2-165b-49e6-8679-b4ad8b4be614	2015-03-04 17:25:34.223992
32	32	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4209\nname: MEF2D\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: myocyte enhancer factor 2D\nclinical_description: \n	1	\N	\N	7a84f967-8e8a-445e-ab53-ed7c5e5a7d41	2015-03-04 17:25:34.41661
33	33	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1436\nname: CSF1R\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: colony stimulating factor 1 receptor\nclinical_description: \n	1	\N	\N	9be91075-1fdd-4296-a80a-ec415220ade1	2015-03-04 17:25:34.669021
34	34	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4255\nname: MGMT\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: O-6-methylguanine-DNA methyltransferase\nclinical_description: \n	1	\N	\N	a99b5410-9b4d-4946-9d05-0089f6eac1da	2015-03-04 17:25:34.866417
35	35	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4869\nname: NPM1\ndescription: AML with mutated NPM1 is provisional entity in the WHO classification\n  of AML and is recommended to be tested for in patients with cytogenetically normal\n  AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they\n  have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently\n  W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations\n  in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing\n  the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia,\n  indicating it may require additional mutations to promote leukemic development.\nofficial_name: nucleophosmin (nucleolar phosphoprotein B23, numatrin)\nclinical_description: \n	1	\N	\N	5598b65f-e283-421d-ae36-9bccb322dbfb	2015-03-04 17:25:35.087953
36	36	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4893\nname: NRAS\ndescription: Mutations in the RAS family of proteins have frequently observed across\n  cancer types. The amino acid positions account for the overwhelming majority of\n  these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also\n  behave very differently when expressed in non-native tissue types, likely due to\n  differences in the C-terminal hyper-variable regions. Mis-regulation of isoform\n  expression has been shown to be a driving event in cancer, as well as missense mutations\n  at the three hotspots previously mentioned. While highly recurrent in cancer, targeted\n  these RAS mutants has also been very elusive, and has not yet become common practice\n  in the clinic.\nofficial_name: neuroblastoma RAS viral (v-ras) oncogene homolog\nclinical_description: \n	1	\N	\N	1b566d8a-e0d8-4760-8dee-7a94b1919a9d	2015-03-04 17:25:35.333383
37	37	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5290\nname: PIK3CA\ndescription: PIK3CA is the most recurrently mutated gene in breast cancer, and has\n  been found to important in a number of cancer types. An integral part of the PI3K\n  pathway, PIK3CA has long been described as an oncogene, with two main hotspots for\n  activating mutations, the 542/545 region of the helical domain, and the 1047 region\n  of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways,\n  is the subject of an immense amount of research and development, and PI3K inhibition\n  has seen some limited success in recent clinical trials. While monotherapies seem\n  to be limited in their potential, there is a recent interest in pursuing PI3K inhibition\n  as part of a combination therapy regiment with inhibition partners including TKI's,\n  MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.\nofficial_name: phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha\nclinical_description: \n	1	\N	\N	167e6e03-ac6f-4c6f-95fe-570338ea7f06	2015-03-04 17:25:35.581685
38	38	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5156\nname: PDGFRA\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: platelet-derived growth factor receptor, alpha polypeptide\nclinical_description: \n	1	\N	\N	693e5618-a673-4111-a203-d46aa463957f	2015-03-04 17:25:35.807228
39	39	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5371\nname: PML\ndescription: The PML-RARA fusion is the result of a recurrent, balanced translocation\n  between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event\n  in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown\n  sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA\n  fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for\n  treating these patients, and early results seem promising.\nofficial_name: promyelocytic leukemia\nclinical_description: \n	1	\N	\N	c82b97ac-2ac0-421a-b4f8-94ae4309d991	2015-03-04 17:25:36.027924
40	40	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5914\nname: RARA\ndescription: The PML-RARA fusion is the result of a recurrent, balanced translocation\n  between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event\n  in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown\n  sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA\n  fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for\n  treating these patients, and early results seem promising.\nofficial_name: retinoic acid receptor, alpha\nclinical_description: \n	1	\N	\N	9fd3f00d-993a-4e95-ba92-2b4488c23815	2015-03-04 17:25:36.227348
42	42	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5979\nname: RET\ndescription: RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine\n  kinase receptor and are associated with thyroid cancers. RET point mutations are\n  the most common mutations identified in medullary thyroid cancer (MTC) with germline\n  and somatic mutations in RET associated with hereditary and sporadic forms, respectively.\n  The most common somatic form mutation is M918T (exon 16) and a variety of other\n  mutations effecting exons 10, 11 and 15 have been described. The prognostic significance\n  of these mutations have been hotly debated in the field, however, data suggests\n  that some RET mutation may confer drug resistence. No RET-specific agents are currently\n  clinically available but several promiscuous kinase inhibitors that target RET,\n  among others, have been approved for MTC treatment.\nofficial_name: ret proto-oncogene\nclinical_description: \n	1	\N	\N	da8d8f94-edf4-4ab6-9191-ecaf1f780166	2015-03-04 17:25:36.768518
43	43	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 861\nname: RUNX1\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: runt-related transcription factor 1\nclinical_description: \n	1	\N	\N	4f9004e5-827e-4ce9-8a10-62527d208e86	2015-03-04 17:25:37.013535
44	44	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 23451\nname: SF3B1\ndescription: SF3B1 mutations have been described in several myeloid malignancies,\n  predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies\n  and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has\n  been identified as recurrently mutated in MDS and other malignanices. The mutations\n  affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally\n  deleterious with K700E described as a major hotspot mutation. MDS patients with\n  SF3B1 mutations have been reported to have better overall and event-free survival\n  than their wildtype counterparts. Additionally, these mutations are highly associated\n  with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with\n  ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring\n  sideroblasts).\nofficial_name: splicing factor 3b, subunit 1, 155kDa\nclinical_description: \n	1	\N	\N	3a869857-94f2-4401-954c-9c9cc5080acd	2015-03-04 17:25:37.340214
45	45	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7157\nname: TP53\ndescription: TP53 mutations are universal across cancer types. Loss of tumor suppressors\n  is most recognized by large deleterious events, such as frameshift mutations, or\n  premature stop codons. In TP53 however, many of the observed mutations in cancer\n  are found to be single nucleotide variants, or missense mutations. These variants\n  are also very broadly distributed throughout the gene, not localizing in any particular\n  hotspot. While a large proportion of cancer genomics research is focused on somatic\n  variants, TP53 is also of note in the germline. Germline TP53 mutations are the\n  hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been\n  found to have prognostic impact on patient outcomes.\nofficial_name: tumor protein p53\nclinical_description: \n	1	\N	\N	94090e45-5a33-4d1c-b0ae-bd3c97cb8d64	2015-03-04 17:25:37.62358
46	46	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7248\nname: TSC1\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: tuberous sclerosis 1\nclinical_description: \n	1	\N	\N	c70cd3d2-7afe-4fa0-85ec-8ef9ff00688d	2015-03-04 17:25:37.833935
47	47	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7249\nname: TSC2\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: tuberous sclerosis 2\nclinical_description: \n	1	\N	\N	d53a6cad-39ca-49c3-a8fb-7e47de81c00c	2015-03-04 17:25:38.045302
48	48	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7307\nname: U2AF1\ndescription: U2AF1 is one of several spliceosome complex genes frequently mutated\n  in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes\n  (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot\n  mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein.\n  These mutations have been associated with altered splicing patterns in vitro and\n  in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation\n  to secondary acute myeloid leukemia, however, the impact of these mutations on overall\n  survival has been an area of debate.\nofficial_name: U2 small nuclear RNA auxiliary factor 1\nclinical_description: \n	1	\N	\N	80aeda81-8705-44dc-8347-6eec51698161	2015-03-04 17:25:38.294332
49	49	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7490\nname: WT1\ndescription: WT1 is a tumor suppressor gene associated with the development of Wilms'\n  Tumor, from which it was named. Mutations in exon 7 and 9 of WT1 have been recurrently\n  identified in acute myeloid leukemia and associated with poorer prognosis and chemotherapy\n  resistance.\nofficial_name: Wilms tumor 1\nclinical_description: \n	1	\N	\N	11684e95-c486-4925-bf0a-1b7857a6ac12	2015-03-04 17:25:38.542119
50	50	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2475\nname: MTOR\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: mechanistic target of rapamycin (serine/threonine kinase)\nclinical_description: \n	1	\N	\N	c9c63402-fc94-4c04-a13a-ea1c22fe5163	2015-03-04 17:25:38.775255
51	51	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4851\nname: NOTCH1\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: notch 1\nclinical_description: \n	1	\N	\N	26acf547-8620-41f9-9e11-f33348e0d6e7	2015-03-04 17:25:38.995924
52	52	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4921\nname: DDR2\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: discoidin domain receptor tyrosine kinase 2\nclinical_description: \n	1	\N	\N	d8b36763-5736-4705-ac6a-4d362c6fdbcd	2015-03-04 17:25:39.21663
53	53	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4233\nname: MET\ndescription: A summary for this gene has yet to be developed! Add one now by clicking\n  the "Edit Gene" button.\nofficial_name: MET proto-oncogene, receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	cee4921c-9632-4a20-9b5f-cbbaaac26a8e	2015-03-04 17:25:39.444402
54	1	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL\ndescription: The BCR-ABL fusion protein, commonly referred to as the philadelphia\n  chromosome, is the most well-studied fusion gene in cancer. It has widely been considered\n  the initiating event in chronic myelogenous leukemia (CML), but despite its ability\n  initiate disease in mice, its status an initiating mutation is in dispute. In what\n  is commonly used as the poster-child for targeted therapeutics, the development\n  and use of imatinib in the clinic has led to profound improvements in the prognosis\n  of the disease. However, imatinib resistance is still seen in patients with mutations\n  in the ABL kinase domain of the fusion, most notably the T315I variant. In patients\n  resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's\n  (dasatinib and nilotinib) have seen some success in delivering a tumor response.\n	1	\N	\N	068c85cf-4337-41e4-bc90-64348675fee4	2015-03-04 17:25:39.539908
194	141	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 52\nname: I638F\ndescription: DDR2 I638F has been shown to be a loss of function mutation, but also\n  confers sensitivity to dasatinib in cell lines.\n	1	\N	\N	59dcfb49-d73b-4a1d-b3e0-7cb45b4cc14b	2015-03-04 17:25:43.596749
55	2	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL T315I\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation T315I has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	51f9ca85-9ce8-4310-8919-5113f1ebfb8b	2015-03-04 17:25:39.569152
56	3	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL E255K\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation E255K has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	b082cffd-ac91-42bf-a35a-4af7d5bf324d	2015-03-04 17:25:39.598173
57	4	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 2\nname: E17K\ndescription: AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal,\n  lung, and ovarian cancer. It has been convincingly shown to be an activating mutation\n  resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation\n  decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and\n  other AKT1 mutations, are the subject of much research and development for therapeutics.\n	1	\N	\N	cbd553c1-c2d6-4a41-8d60-38446b6a5b7b	2015-03-04 17:25:39.622488
58	5	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK\ndescription: The EML4-ALK fusion has been seen in non-small cell lung cancer, and\n  appears to be an alternative mechanism for ALK activation. Cells with this fusion\n  have been shown to be sensitive to the ALK inhibitor crizotinib.\n	1	\N	\N	7e3626b3-714b-4295-88b0-f1b592e61cbc	2015-03-04 17:25:39.65205
59	6	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK C1156Y\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	4399a032-c1b6-495b-a0fa-17e478cdc098	2015-03-04 17:25:39.681327
60	7	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK L1196M\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	c137e40f-8e0f-47b1-85a9-5382685a1dcf	2015-03-04 17:25:39.705664
61	8	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: F1174L\ndescription: ALK F1174L has been observed as recurrent in neuroblastoma, non-small\n  cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing\n  this mutation have shown resistance to low doses of criztonib. However, increased\n  dosage can overcome this resistance in cell lines studies. TAE684 has also proven\n  effective in both NSCLC and neuroblastoma F1174L containing cells.\n	1	\N	\N	89e63107-161f-492b-af33-73dbb0a6cae1	2015-03-04 17:25:39.731264
62	9	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: R1275Q\ndescription: ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma,\n  non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma\n  cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This,\n  and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC\n  cell lines.\n	1	\N	\N	5b61141c-e54f-4af1-93e8-9802aaa06ad4	2015-03-04 17:25:39.756978
63	10	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 3\nname: S214C\ndescription: ARAF S214C has been found to be a recurrent oncogenic mutation in non-small\n  cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib\n  in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this\n  mutation, sorafenib also acheived near-complete clinical remission. This case has\n  brought more interest to the variant from a research and clinical perspective.\n	1	\N	\N	14702e53-19f7-4da5-a7d0-b00785353eb2	2015-03-04 17:25:39.786424
64	11	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600D\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	99e9cb7b-257d-4f11-8856-8908f7af8785	2015-03-04 17:25:39.820038
65	12	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E\ndescription: BRAF V600E has been shown to be recurrent in many cancer types. It is\n  one of the most widely studied variants in cancer. This variant is correlated with\n  poor prognosis in certain cancer types, including colorectal cancer and papillary\n  thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective\n  in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib\n  has also shown to be effective when combined with the MEK inhibitor trametinib in\n  colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA\n  mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib,\n  and a number of combination therapies have been successful in treating V600E mutations.\n  While the drugs cetuximab and panitumumab have been largely shown to be ineffective\n  without supplementary treatment.\n	1	\N	\N	c155f279-ee7e-47e1-bdb1-6c0eda189d0c	2015-03-04 17:25:39.848987
66	13	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E/V600M\ndescription: A case study of a single patient harboring both a V600E and a V600M mutation,\n  dabrafenib was shown to acheive clinical response.\n	1	\N	\N	796ea8b0-b41b-4b28-91c7-6f22f77356da	2015-03-04 17:25:39.880386
67	14	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E AMPLIFICATION\ndescription: Amplification of BRAF V600E has been shown to confer resistance to MEK\n  inhibitors. For more information on the V600 locus, see the V600E entry.\n	1	\N	\N	63bbfc6b-ef57-411a-be6e-ec595454988e	2015-03-04 17:25:39.90956
68	15	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600M\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	755a4743-b354-4d61-ba38-e0f42c53fb08	2015-03-04 17:25:39.932324
69	16	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600R\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	c9af2d4d-8d42-4b33-8158-66a96fe4722e	2015-03-04 17:25:39.954783
70	17	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600\ndescription: BRAF mutations of the valine 600 residue have been shown to be recurrent\n  in many cancer types. Of the V600 mutations, V600E is the most widely researched.\n  V600 mutations as a whole have been correlated to poorer prognosis in colorectal\n  and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity\n  to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual\n  mutation pages on the left sidebar.\n	1	\N	\N	a9102006-0702-42ca-8719-0eb5711a1313	2015-03-04 17:25:39.979387
71	18	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: AMPLIFICATION\ndescription: CCND1 amplification has been implicated in poorer prognosis in non-small\n  cell lung cancer.\n	1	\N	\N	b3f34df5-9fbc-41b5-86a1-1d9df80195c0	2015-03-04 17:25:40.009176
72	19	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: EXPRESSION\ndescription: CCND1 expression, and its prognositc impact, is still in dispute. Three\n  experiments in non-small cell lung cancer have shown it to have no impact on survival,\n  but three additional studies have shown it results in poorer prognosis. There is\n  also some ambiguity in how the boundaries between expression and overexpression\n  are defined.\n	1	\N	\N	3241a03a-33fa-423f-9e65-e061a6e7f8e1	2015-03-04 17:25:40.037597
73	20	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	afde7fff-0f3c-47c1-aab1-156d8663e107	2015-03-04 17:25:40.071204
74	21	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	2ef7c3d0-a2c1-4445-bd72-ae79fce921b4	2015-03-04 17:25:40.09907
75	22	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: PROMOTER DEMETHYLATION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	1718c6e6-18e1-4963-a4c6-6ed1ad8c65fd	2015-03-04 17:25:40.129094
76	23	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 10\nname: LOSS\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL),\n  a seemingly unique trend when compared to the amplifcations and overexpressions\n  of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic\n  palbociclib significantly increased the median survival of a Notch-driven model\n  of T-ALL.\n	1	\N	\N	c51f3faa-8b62-43e3-ba1d-d97b644d8abe	2015-03-04 17:25:40.168047
77	24	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 11\nname: OVEREXPRESSION\ndescription: Cyclin E, while currenly not as widely implicated as its counterpart,\n  cyclin D, has been implicated in various carcinomas, including breast, gastric,\n  stomach and colorectal. High levels of cyclin E, either by gene amplification or\n  overexpression, are correlated with later stage disease and have been shown to lead\n  to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer\n  specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex\n  are more radiosensitive than their lowly expressed counterparts.\n	1	\N	\N	688718ab-7b75-41af-8fd4-246334c397fb	2015-03-04 17:25:40.208982
78	25	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: EXPRESSION\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	69a9005d-b480-4406-b044-2ed46037d464	2015-03-04 17:25:40.235598
79	26	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 12\nname: EXPRESSION\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	eac4a08e-8f46-429e-b3da-d0c8ecfadd22	2015-03-04 17:25:40.262162
80	27	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 14\nname: PROMOTER HYPERMETHYLATION\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the CDKN2A\n  promoter region. While no targeted therapeutic has been engaged in clinical trials,\n  the prognostic impact has been studied by a number of meta-analyses. The prognostic\n  impact of promoter hypermethylation has been relatively ambiguous. Many studies\n  have shown significant p-values suggesting poorer prognostic outcomes for patients\n  with hypermethylation in colorectal, liver, and younger lung cancer patients. This\n  being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\n	1	\N	\N	bc825394-9223-4e52-8604-b88b96e4a993	2015-03-04 17:25:40.286879
81	28	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: N-TERMINAL FRAME SHIFT\ndescription: CEBPA N-terminal frame shift mutations that result in a premature stop\n  codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML).\n  CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3\n  mutations should also be assessed in CN-AML patients as concurrent mutations may\n  have prognostic implications.\n	1	\N	\N	7ce1e699-ca46-4116-83da-f2fd16c9dfd3	2015-03-04 17:25:40.31289
82	29	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: MUTATION\ndescription: CEBPA mutations are associated with cytogenetically normal acute myeloid\n  leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should\n  also be assessed in CN-AML patients as concurrent mutations may have prognostic\n  implications.\n	1	\N	\N	b6ba5731-74a3-41a8-b247-9f66a56c2dd5	2015-03-04 17:25:40.336955
83	30	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	0d378373-c429-464e-923c-58c26b4eca8c	2015-03-04 17:25:40.364487
84	31	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in the rare form of adolescent\n  liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion\n  was observed in 15/15 FL-HCC cases examined and functional studies found that the\n  fusion retained kinase activity. The presence of this fusion may be used as a diagnostic\n  marker for this rare tumor type.\n	1	\N	\N	0c79c6f5-1704-4ba9-a32b-c17107e415ae	2015-03-04 17:25:40.393655
85	32	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R882\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\n	1	\N	\N	80163c51-b82f-4e7a-8775-c074bd73cc61	2015-03-04 17:25:40.422598
86	33	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: L858R\ndescription: EGFR L858R has long been recognized as a functionally significant mutation\n  in cancer, and is one of the most prevalent single mutations in lung cancer. Best\n  described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity\n  to first and second generation TKI's like gefitinib and neratinib. NSCLC patients\n  with this mutation treated with TKI's show increased overall and progression-free\n  survival, as compared to chemotherapy alone. Third generation TKI's are currently\n  in clinical trials that specifically focus on mutant forms of EGFR, a few of which\n  have shown efficacy in treating patients that failed to respond to earlier generation\n  TKI therapies.\n	1	\N	\N	d9edb2ae-bfed-44b1-bd11-01b2c0cd5e9e	2015-03-04 17:25:40.450525
87	34	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: T790M\ndescription: EGFR T790M was one of the very first mutations recognized to confer resistance\n  to targeted therapies. While successful in amplified EGFR, the efficacy of the first\n  and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients\n  harboring this mutation before treatment is notably lower. This lack of efficacy\n  can likely be to blame for the poorer prognosis for patients with this mutation\n  as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately\n  half of EGFR mutant tumors with acquired resistance to TKI inhibition have been\n  shown to harbor this mutation, implicating it as a mechanism of acquired therapy\n  resistence. The third generation TKI's are being developed with this resistance\n  problem in mind, and early stage experiments have shown some efficacy of these drugs\n  in previously resistant tumors with mutant EGFR.\n	1	\N	\N	d62316ab-82bf-4962-8df8-72b16ceaa568	2015-03-04 17:25:40.479231
88	35	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: D769H\ndescription: ERBB2 D769H was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	5e38541d-11d2-4eb8-895c-d0baa4f532d2	2015-03-04 17:25:40.505992
89	36	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: D769Y\ndescription: ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	40677a51-7df8-48e1-b766-1cf3cccbed07	2015-03-04 17:25:40.530882
90	37	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: DEL 755-759\ndescription: ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants\n  to be functionally classified (Bose et al. 2012). This mutation was shown to be\n  an activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	d2ce170c-1a0b-4417-b8e3-e5b79c3626c5	2015-03-04 17:25:40.556643
91	38	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: G309A\ndescription: ERBB2 G309A was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	cc33d8a3-18ea-4a07-b019-13b75e0908b5	2015-03-04 17:25:40.583077
92	39	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: L755S\ndescription: ERBB2 L755S was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike\n  many of the other variants queried. This mutation was also shown to confer resistance\n  to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.\n	1	\N	\N	5c6fff2c-b264-4707-a505-70b015134515	2015-03-04 17:25:40.612452
93	40	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: L755W\ndescription: ERBB2 L755W was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	9e69777b-30c7-4b13-b2f4-31eb42cd579e	2015-03-04 17:25:40.636807
94	41	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: P780INS\ndescription: ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally\n  classified (Bose et al. 2012). This mutation was shown to be an activating mutation\n  in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	5ed90420-cf47-4309-9a44-7fdb46736c5b	2015-03-04 17:25:40.668763
95	42	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: R678Q\ndescription: ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	058fc792-e62f-43e4-9a2f-428735386142	2015-03-04 17:25:40.697525
96	43	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: R896C\ndescription: ERBB2 R896C was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	b2711e0d-c367-4f4a-9367-13482682a806	2015-03-04 17:25:40.723666
97	44	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: V777L\ndescription: ERBB2 V777L was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	2f47e869-5cb6-42a2-9d36-f0d9e686302a	2015-03-04 17:25:40.75388
98	45	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: V842I\ndescription: ERBB2 V842I was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	1dd37740-a32f-45fd-bd1c-3698cc7ded5c	2015-03-04 17:25:40.788745
99	46	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: L536Q\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of\n  these ligand binding domain mutations, and is commonly implicated in this hormone\n  resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant,\n  may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	a02ce7b4-f47e-4be5-b019-a6361ae2e503	2015-03-04 17:25:40.818042
100	47	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: N538G\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	27fa9645-bede-4f7a-bbe8-ac235ef3901b	2015-03-04 17:25:40.863689
101	48	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537C\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	1bf068a5-b9c7-4fec-b852-4592596d9855	2015-03-04 17:25:40.890448
102	49	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537N\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	c02b5066-a0cf-4e63-aa73-90beacccb140	2015-03-04 17:25:40.915197
103	50	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537S\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	d3a024c0-cb8b-4b28-8b8d-d5600a867b43	2015-03-04 17:25:40.94169
104	51	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: FGFR2-MGEA5\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. Additionally, tumor size reduction was\n  achieved by both ponatinib and pazopanib treatments administered separately in a\n  single patient with intrahepatic cholangiocarcinoma and this fusion. The authors\n  use these cases to highlight the need for enhanced clinical sequencing efforts.\n	1	\N	\N	22412888-13d1-427c-9e38-3e94d4927b17	2015-03-04 17:25:40.970701
105	52	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: FGFR3-BAIAP2L1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\n	1	\N	\N	28cff08d-dcb4-4195-8628-b8eab392f9d3	2015-03-04 17:25:40.99625
106	53	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: FGFR3-TACC3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\n	1	\N	\N	0cbb366e-b4bd-486f-a2b1-99a38b25b944	2015-03-04 17:25:41.024825
107	54	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: ITD MUTATIONS\ndescription: FLT3-ITD (internal tandem duplications) frequently occur in patients\n  with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid\n  leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically\n  normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of\n  FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated\n  sequence resulting in an in-frame duplication event. The length of these duplications\n  can vary widely which may have prognostic consequences, but this has not been conclusively\n  determined. FLT3-ITD mutations overall have generally been associated with poor\n  prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the\n  prognosis associated with this variant.\n	1	\N	\N	f79bb8da-7c02-4963-89d7-049b0e63b733	2015-03-04 17:25:41.051213
108	55	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: TKD MUTATIONS\ndescription: FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common\n  than FLT3-ITD (internal tandem duplication) mutations and may not confer the same\n  prognostic impact. Although the majority of mutations are point mutations effecting\n  D835 (most frequently D835Y), a small proportion involve an in-frame deletion of\n  I836. These mutations are within the activation loop of the second tyrosine kinase\n  domain of FLT3 and thought to result in constitutive activation of the receptor.\n	1	\N	\N	892d3fce-50b5-4074-ae40-2de3e5d3fc51	2015-03-04 17:25:41.082008
109	56	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: EXPRESSION\ndescription: GATA2 misregulation has been observed in a number of hematologic malignancies,\n  as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using\n  the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung\n  cancer.\n	1	\N	\N	9c063187-2e2c-414e-b92d-b938d8284ad7	2015-03-04 17:25:41.106101
110	57	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	c87ebf74-636e-4f77-951e-09e74f812e68	2015-03-04 17:25:41.132812
111	58	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132C\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	60d99ec5-b666-4a05-931b-250eae241772	2015-03-04 17:25:41.159141
112	59	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132H\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	54668b66-ad2e-43ae-ba10-68916279d035	2015-03-04 17:25:41.185044
113	60	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132L\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	1ed827f0-0458-44b9-8c78-638a3b5199c9	2015-03-04 17:25:41.210895
114	61	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: R140Q/L\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival\n  as compared to their wild-type counterparts. In myelodysplastic syndromes, however,\n  no prognostic link was discovered between IDH2 mutation status and overall survival.\n	1	\N	\N	3c4cd8b0-58cc-4e1f-a738-99e06e41d677	2015-03-04 17:25:41.247514
115	62	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: R172K\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis\n  and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic\n  syndromes, studies did not find a prognostic association between this variant and\n  patient outcomes.\n	1	\N	\N	02068de7-6ccd-4c0b-a701-bd0f7f059768	2015-03-04 17:25:41.272579
116	63	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: V617F\ndescription: JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases,\n  occuring in around half of all MPD's. While less associated with cancer, when it\n  is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The\n  V617F mutation is an activating mutation, resulting in increased kinase activity.\n  The mutation seems to be restricted to hematologic malignancies. Treatment of JAK\n  mutant diseases with ruxolitinib has seen some clinical success.\n	1	\N	\N	72ac98ef-1ac6-415d-8417-c2a6156551a3	2015-03-04 17:25:41.302704
117	64	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: D816V\ndescription: KIT D816V is a mutation observed in acute myeloid leukemia (AML). This\n  variant has been linked to poorer prognosis and worse outcome in AML patients.\n	1	\N	\N	cb085a02-f5c6-4293-8039-b77c91423011	2015-03-04 17:25:41.326922
118	65	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 11 MUTATIONS\ndescription: c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are\n  very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis\n  than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity\n  to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients\n  treated with the drug in the first year.\n	1	\N	\N	a043b8ae-511d-43cb-88ab-63cc7169d923	2015-03-04 17:25:41.354823
119	66	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: INTERNAL DUPLICATION\ndescription: c-KIT internal duplications have been observed in exon 11, within the\n  juxtamembrane domain. In a case study of an anal melanoma patient harboring this\n  event, imatinib confered marked response. Also, cells harboring exon 11 mutations\n  have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better\n  prognosis to patients treated with the drug in the first year.\n	1	\N	\N	77639b76-c88d-4f2c-b821-c64705695d1d	2015-03-04 17:25:41.387686
120	67	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 13 MUTATIONS\ndescription: c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations.\n  These mutations lie within the tyrosine kinase 1 domain, and are found primarily\n  in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations\n  show sensitivity to imatinib and sunitinib treatment. However, only imatinib has\n  seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant\n  cases.\n	1	\N	\N	4baaac4f-c267-44da-b030-9f8fdb8e977c	2015-03-04 17:25:41.417967
121	68	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 14 MUTATIONS\ndescription: c-KIT exon 14 mutations lie within the tyrosine kinase domain of the\n  protein. While relatively rare in primary gastrointestinal tumors, they are notably\n  more prevalent in refractory disease, suggesting a role in imatinib resistance.\n  Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.\n	1	\N	\N	153a8ee2-9996-43a8-a11a-88a9a0286651	2015-03-04 17:25:41.448843
122	69	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 17 MUTATIONS\ndescription: c-KIT exon 17 mutations lie within the TK2 domain, containing the activation\n  loop of the protein. In cell lines, mutations within this domain have been shown\n  to be sensitive to imatinib. However, in double KIT mutants in which the exon 17\n  mutation is a secondary mutation, cell lines have shown resistance to both imatinib\n  and sunitinib.\n	1	\N	\N	d1977fd8-8ef2-44db-91ed-a18c03b4f4b6	2015-03-04 17:25:41.492478
123	70	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 9 MUTATIONS\ndescription: c-KIT exon 9 mutations lie within the dimerization motif of the protein.\n  Relative to other KIT mutations, exon 9 mutations have been associated with better\n  overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and\n  in vivo.\n	1	\N	\N	30961589-9c38-4ce5-84eb-cab58403f5ab	2015-03-04 17:25:41.520881
124	71	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: L576P\ndescription: KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain.\n  It is one of the most recurrent KIT mutations in melanoma, and both in vitro and\n  in vivo studies have shown sensitivity to imatinib.\n	1	\N	\N	157abe37-efac-4734-8117-8fd34cc2b539	2015-03-04 17:25:41.546459
125	72	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: V654A\ndescription: KIT V654A is an exon 13 mutation that lies within the tyrosine kinase\n  1 domain of the protein. It has been shown to be an activating mutation by in vitro\n  studies. This mutation is associated with imatinib resistance in melanoma patients.\n  However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have\n  seen success in acheiving tumor response.\n	1	\N	\N	594369e9-5a2e-4044-9b34-8c10e218ea47	2015-03-04 17:25:41.575725
126	73	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: EXON 1 MUTATIONS\ndescription: A study by Lièvre et al in 2006 showed that colorectal cancer patients\n  with KRAS exon 1 mutations had low cetuximab response rates.\n	1	\N	\N	fd736e9f-34ee-4fd3-958b-98b2aa2bde2a	2015-03-04 17:25:41.605855
127	74	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: EXON 2 MUTATIONS\ndescription: In a study by Pao et al in 2005, non-small cell lung cancer patients\n  harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors\n  gefinitib and erlotinib.\n	1	\N	\N	22ab49d5-1166-47cc-9003-04374f2dd034	2015-03-04 17:25:41.631072
128	75	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	37135f5e-e9ba-4d8d-a694-83185ff33385	2015-03-04 17:25:41.661739
129	76	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12/G13\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation\n  may result in a less responsive tumor when treated with first-generation TKI's like\n  gefitinib.\n	1	\N	\N	676f3731-8013-40c7-804a-06634286a598	2015-03-04 17:25:41.696075
130	77	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12C\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	f01a37dd-d48b-4e6b-b2e0-cb1473d85357	2015-03-04 17:25:41.726784
131	78	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12D\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	73f9db17-488d-4545-8a7d-3d13f288e870	2015-03-04 17:25:41.751557
132	79	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G13\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	7bb5bc0d-0474-4061-954f-b6ce3fa6f9c5	2015-03-04 17:25:41.782131
133	80	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G13D\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	dce785e0-b814-4000-b873-f52d4ea15f90	2015-03-04 17:25:41.807994
134	81	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: P124S\ndescription: MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder\n  and colon cancer to a lesser degree. The P124S mutation has been shown to contribute\n  to AZD6244 resistance in melanoma cell lines, but considerably less so than its\n  Q56P counterpart.\n	1	\N	\N	0aeb59b2-202d-4016-b1bf-917a854fe921	2015-03-04 17:25:41.837508
135	82	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: Q56P\ndescription: MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This\n  mutation has been shown to confer considerable resistance to AZD6244 treatment of\n  melanoma cell lines.\n	1	\N	\N	b04f36f3-a855-4811-b809-7403577b0290	2015-03-04 17:25:41.865751
136	83	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 32\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	21a6b894-7d29-48a9-8dbe-af35a2da4423	2015-03-04 17:25:41.891203
137	84	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: PROMOTER METHYLATION\ndescription: MGMT promoter methylation has been observed to impact tumor progression\n  in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating\n  agent carmustine has shown efficacy. In patients lacking methylation, combining\n  carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more\n  experiments are required. Clinical trials have also shown selective sensitivity\n  of promoter methylation-positive patients to temozolomide, making a case for wider\n  methylation screening in GBM patients.\n	1	\N	\N	db9f16ca-9743-473e-83ed-d9b4ba006027	2015-03-04 17:25:41.92719
138	85	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 12 MUTATIONS\ndescription: NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\n	1	\N	\N	fe25e524-23d4-47c8-a9d0-74b1dc3224d0	2015-03-04 17:25:41.952843
139	86	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: W288FS\ndescription: NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most\n  common NPM1 mutation identified in acute myeloid leukemia. This mutation results\n  in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response\n  to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation,\n  it is generally grouped with other exon 12 mutations for patient analysis (see NPM1\n  Exon 12 variants for more information).\n	1	\N	\N	e2e459c6-0be2-4809-9f06-bf8d3de432e9	2015-03-04 17:25:41.977904
140	87	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 1 MUTATIONS\ndescription: NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown\n  to be correlated with poorer overall survival relative to wild-type NRAS in melanoma\n  patients.\n	1	\N	\N	45d31dd6-d068-4304-90d6-3360706603bb	2015-03-04 17:25:42.008016
141	88	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 2 MUTATIONS\ndescription: NRAS exon 2 mutations have been shown to be correlated with poorer overall\n  survival in melanoma patients and colorectal cancer patients, however no prognostic\n  impact was seen in acute myeloid leukemia patients.\n	1	\N	\N	1201ab08-145b-46fe-b696-aaa2f858a05a	2015-03-04 17:25:42.033527
142	89	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 3 MUTATIONS\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	9ebb3600-335c-47c1-ac89-37ad2588afd4	2015-03-04 17:25:42.060409
143	90	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 4 MUTATIONS\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	f4e473d9-7acc-4b9e-b864-abe5e139b56a	2015-03-04 17:25:42.084012
144	91	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: G12\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	c461c6eb-0593-485f-ac7d-ff6d018c48c7	2015-03-04 17:25:42.135559
145	92	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: G13D\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	92d1c71f-5b46-4598-b50f-5a4864f5a679	2015-03-04 17:25:42.162362
146	93	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations\n  at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide.\n  However, in colorectal cancer patients, mutations at this locus have been shown\n  to confer resistance to cetuximab. The prognostic impact of mutations at this locus\n  is currently under study.\n	1	\N	\N	af03eca2-63b0-4fb6-a4ab-ca6b2fcc22dd	2015-03-04 17:25:42.18961
147	94	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61L\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	40d7b8d7-3667-464f-9337-0beb7bf58a73	2015-03-04 17:25:42.215979
148	95	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61R\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	fd5579fd-1842-4192-a0e6-50804d7862c7	2015-03-04 17:25:42.241597
195	142	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 52\nname: G774V\ndescription: Activating mutations in DDR2, including G774V, has been shown to be sensitive\n  to dasatinib in cell lines.\n	1	\N	\N	ba8e5d23-99f4-4149-9690-d69b57f35bc4	2015-03-04 17:25:43.621018
149	96	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842I\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	034e9454-7034-41a8-8185-262e29807825	2015-03-04 17:25:42.268584
150	97	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842V\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	7263e692-3059-44f3-a737-68cd4dc287eb	2015-03-04 17:25:42.293899
151	98	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842Y\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	dbb7f86d-e9ba-458f-b8f3-968dc2397117	2015-03-04 17:25:42.328338
152	99	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: DEL I843\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	40f7e157-7e89-4147-9ba0-14868d2376b6	2015-03-04 17:25:42.356858
153	100	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: DI842-843IM\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	e163e022-aae1-4b60-b8dd-495304e22cd7	2015-03-04 17:25:42.388496
154	101	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: E542K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	4844b843-52a7-4be1-a787-c426d41ee088	2015-03-04 17:25:42.413794
155	102	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: E545K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	bcc86a7f-3d6f-4abe-8138-d275b19bc525	2015-03-04 17:25:42.444906
156	103	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: EXON 20 MUTATIONS\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	575d6dba-a319-43ea-9dee-86c2450f7354	2015-03-04 17:25:42.475652
157	104	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: EXON 9 MUTATIONS\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	e475e547-1a37-4faf-966c-742821e602c9	2015-03-04 17:25:42.500502
158	105	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: H1047R\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	cbefeb0a-6ece-421e-affc-b4f011d3c1a0	2015-03-04 17:25:42.525694
159	106	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 39\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and arsenic trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	6a841efa-9781-4836-9801-06d87bcf6aa1	2015-03-04 17:25:42.553322
160	107	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in a rare form of adolescent\n  liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found\n  Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined,\n  and functional studies found that the fusion retained kinase activity. The presence\n  of this fusion may be used as a diagnostic marker for this rare tumor type.\n	1	\N	\N	02eee43f-9148-4a75-8273-e4f01ea505d2	2015-03-04 17:25:42.580802
161	108	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 41\nname: R233*\ndescription: PTEN R233* has been shown to be a loss of function mutation, and PTEN\n  loss has been the subject of considerable research in breast cancer. PTEN loss may\n  sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data\n  to support that PTEN loss is both associated with poorer prognosis, and no change\n  in prognosis.\n	1	\N	\N	bcf9409c-09fc-4ebb-9236-5e30cf097c4b	2015-03-04 17:25:42.611414
162	109	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and Arsenic Trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	46a8c503-aed4-4181-835e-a268050ef098	2015-03-04 17:25:42.63719
163	110	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: C634W\ndescription: RET C639W has been implicated as an alternate mechanism of activating\n  RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting\n  agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the C639W mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib.\n	1	\N	\N	be36e93e-9cc1-42d5-a632-98e3fdd699f6	2015-03-04 17:25:42.661262
164	111	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: M918T\ndescription: RET M819T is the most common somatically acquired mutation in medullary\n  thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents,\n  promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the M918T mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T\n  leads to more aggressive MTC with a poorer prognosis.\n	1	\N	\N	2254e247-999d-4a47-8b5b-ebfde23e02c1	2015-03-04 17:25:42.69205
165	112	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: K666N\ndescription: SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients.\n	1	\N	\N	46823e62-e85d-42f0-9765-223909e98290	2015-03-04 17:25:42.736774
166	113	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: K700E\ndescription: SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients. Additionally,\n  these mutations are the most common SF3B1 mutation observed in MDS and highly associated\n  with subtypes of MDS that are defined by ringed sideroblasts.\n	1	\N	\N	40c3ef00-d546-4b44-bca4-05f9d6f0eb88	2015-03-04 17:25:42.760542
167	114	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R175H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R175H variant seems not only to result in loss of tumor-suppression, but also acts\n  as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell\n  lines harboring this mutant is also more responsive to treatment with doxorubicin\n  than its wild-type counterparts. While the prognostic impact of individual TP53\n  mutations is influenced by the cohort being studied, it has been shown that the\n  R175H mutation is correlated with worse overall survival than wild-type TP53, but\n  is not as detrimental as the R248W variant.\n	1	\N	\N	26c67992-bbfe-43a1-98c5-4eef0b3da9a5	2015-03-04 17:25:42.78553
168	115	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R248Q\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival. The R248Q mutation has also shown an increased invasive behavior in cell\n  lines. This is specific to the 248Q variant.\n	1	\N	\N	2c47797f-c1a8-41dc-afa0-15c635bdfbb1	2015-03-04 17:25:42.810138
169	116	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R248W\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival.\n	1	\N	\N	ee6fdfc2-4456-4acd-abb2-9fba07935761	2015-03-04 17:25:42.844846
170	117	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R249T\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	2014c218-34be-4e5e-b32f-d6a822ee78c8	2015-03-04 17:25:42.877231
171	118	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R249W\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	deccfaf8-04cd-4ec1-8ba0-2bdd67494620	2015-03-04 17:25:42.905849
172	119	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R273C\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	5f693056-edd2-4e54-90ed-619222fd05d2	2015-03-04 17:25:42.93102
173	120	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R273H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	9cc1e4f2-b88e-4860-9ef4-8c89a21e3c5d	2015-03-04 17:25:42.955818
174	121	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: V173G/A\ndescription: While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R249 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	b0c8fe8c-ef65-4c3d-b6b1-1b9652d5a381	2015-03-04 17:25:42.982549
196	143	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 52\nname: S768R\ndescription: Activating mutations in DDR2, including S768R, has been shown to be sensitive\n  to dasatinib in cell lines.\n	1	\N	\N	423edaf1-15d1-4100-ac2a-67e20212e75d	2015-03-04 17:25:43.647857
175	122	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: FRAMESHIFT TRUNCATION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	0fe3dafb-7d0c-40b9-b5f2-9e0489890487	2015-03-04 17:25:43.008693
176	123	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	6e8e2589-cf67-4e04-b9e8-fce7f5ce4695	2015-03-04 17:25:43.036372
177	124	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	b3f1cfe4-84d0-42fc-a87e-99ca8acc12a1	2015-03-04 17:25:43.065544
178	125	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 48\nname: Q157P/R\ndescription: U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is less common than the S34F mutation, occurs in the second zinc finger domain of\n  U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations\n  on overall survival in MDS has been debated, however, patients with U2AF1 mutations\n  were shown to be at an increased risk of transformation to secondary AML. The presence\n  of this mutation was not associated with a specific prognostic outcome in AML when\n  compared to U2AF1 wildtype patients.\n	1	\N	\N	12e4a0e8-f3f3-4c40-92d4-dd401ff1c368	2015-03-04 17:25:43.094453
179	126	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 48\nname: S34Y/F\ndescription: U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is the most commonly identified variant in MDS, occurs in the first zinc finger\n  domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1\n  mutations on overall survival in MDS has been debated, however, patients with U2AF1\n  mutations were shown to be at an increased risk of transformation to secondary AML.\n  The presence of this mutation was not associated with a specific prognostic outcome\n  in AML when compared to U2AF1 wildtype patients.\n	1	\N	\N	d52d20e1-de50-4a29-8cad-00e6e4046de0	2015-03-04 17:25:43.124679
180	127	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 49\nname: EXON 7 MUTATIONS\ndescription: WT1 exon 7 mutations have been shown to be recurrent in acute myeloid\n  leukemia. Many sources have examined the prognostic impact of these, agreeing that\n  the mutant exon 7 cohort is correlated with worse overall survival and a number\n  of poor prognistic outcomes. This may be the result of an overall poor response\n  to chemotherapy from WT1 mutant tumors.\n	1	\N	\N	5d561177-ea17-4ab9-abe6-3b9e48e5ab7c	2015-03-04 17:25:43.153628
181	128	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 49\nname: EXON 9 MUTATIONS\ndescription: WT1 exon 9 mutations have been shown to be recurrent in acute myeloid\n  leukemia, although at a less frequent rate than their exon 7 counterparts. Many\n  sources have examined the prognostic impact of these, agreeing that the mutant exon\n  9 cohort is correlated with worse overall survival and a number of poor prognistic\n  outcomes. This may be the result of an overall poor response to chemotherapy from\n  WT1 mutant tumors.\n	1	\N	\N	44a3251d-d967-4bd8-9444-74f6b8da2f59	2015-03-04 17:25:43.186717
182	129	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: LOSS-OF-FUNCTION\ndescription: BRCA1 loss of function mutations have been shown to increase risk of\n  breast and ovarian cancer in those carrying the allele in their germline. Treating\n  BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant\n  response.\n	1	\N	\N	6ef63f35-cd35-403e-bca3-9a2b462fdce5	2015-03-04 17:25:43.220137
183	130	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: LOSS-OF-FUNCTION\ndescription: BRCA2 loss of function mutations have been shown to increase risk of\n  breast and ovarian cancer in those carrying the allele in their germline. Treating\n  BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant\n  response.\n	1	\N	\N	eda868cb-281d-46a7-829e-9144a1c13ea1	2015-03-04 17:25:43.254052
184	131	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: EXON 19 DELETION\ndescription: Deletions within exon 19 of EGFR are most common in lung cancer. These\n  deletions, in non-small cell lung cancer, have been shown to be sensitive to the\n  EGFR tyrosine kinase inhibitors gefitinib and erlotinib. There is also data to suggest\n  that this event is a good prognostic marker in lung adenocarcinoma.\n	1	\N	\N	dadba64b-f6ea-4d1c-8185-ec54be6c4510	2015-03-04 17:25:43.287838
185	132	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: G719S\ndescription: While not as recurrent as the L858R mutation, EGFR G719S has also been\n  shown to be an activating mutation. In lung cancer cell lines, it also confers sensitivity\n  to the tyrosine kinase inhibitors gefitinib and erlotinib.\n	1	\N	\N	9bf478b6-f0fa-44b5-8981-6ff57807fed6	2015-03-04 17:25:43.321963
186	133	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: D1643H\ndescription: Activating mutations in NOTCH1, including D1643H, have been shown to\n  be poor prognostic markers in lung cancer.\n	1	\N	\N	db39d8b4-54ec-4af6-b4af-20a0425016c6	2015-03-04 17:25:43.379147
187	134	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: R2328W\ndescription: Activating mutations in NOTCH1, including R2328W, have been shown to\n  be poor prognostic markers in lung cancer.\n	1	\N	\N	fdd5eddb-8ccb-4de0-8e52-cef656f3588c	2015-03-04 17:25:43.412272
188	135	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: V2444FS\ndescription: Activating mutations in NOTCH1, including a frameshift insertion at V2444,\n  have been shown to be poor prognostic markers in lung cancer.\n	1	\N	\N	ef017645-3d3a-4801-bbf9-46f8bca985e9	2015-03-04 17:25:43.436625
189	136	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: S2275FS\ndescription: Activating mutations in NOTCH1, including a frameshift insertion at S2275,\n  have been shown to be poor prognostic markers in lung cancer.\n	1	\N	\N	032bcc0b-280d-4851-8a50-816c1836ba20	2015-03-04 17:25:43.468787
190	137	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 52\nname: L63V\ndescription: Activating mutations in DDR2, including L63V, has been shown to be sensitive\n  to dasatinib in cell lines.\n	1	\N	\N	b99a2ebb-1d88-4dd1-8f30-1b66c02c6ec2	2015-03-04 17:25:43.494159
191	138	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 52\nname: L239R\ndescription: Activating mutations in DDR2, including L239R, has been shown to be sensitive\n  to dasatinib in cell lines.\n	1	\N	\N	1ab34ec0-e549-4865-b5d3-fc86ed8021fd	2015-03-04 17:25:43.517486
192	139	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 52\nname: G253C\ndescription: Activating mutations in DDR2, including G253C, has been shown to be sensitive\n  to dasatinib in cell lines.\n	1	\N	\N	8f9a618f-f91e-4ae3-a778-ee8ddab6e1a4	2015-03-04 17:25:43.544412
197	144	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 53\nname: AMPLIFICATION\ndescription: MET amplification, like EGFR T790M, has been shown to be capable of driving\n  acquired resistance to dacomitinib in patients with lung adenocarcinoma.\n	1	\N	\N	38358e68-fb63-47a3-9457-16adc781d3c2	2015-03-04 17:25:43.680624
198	145	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12V\ndescription: KRAS G12V, like EGFR T790M, has been shown to be capable of driving acquired\n  resistance to tyrosine kinase inhibitors in lung adenocarcinoma.\n	1	\N	\N	53df3cea-40ce-4c90-8997-02ff61e71408	2015-03-04 17:25:43.706894
199	146	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12A\ndescription: KRAS G12A, like EGFR T790M, has been shown to be capable of driving acquired\n  resistance to tyrosine kinase inhibitors in lung adenocarcinoma.\n	1	\N	\N	56220d80-5e4f-42ee-83a1-71ef6ec1c97c	2015-03-04 17:25:43.735274
200	147	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: Q2223K\ndescription: A summary for this variant has yet to be developed! Add one now by clicking\n  the "Edit Variant" button.\n	1	\N	\N	87afb147-8486-40f3-a470-7d68512dcc17	2015-03-04 17:25:43.762557
201	148	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: G853D\ndescription: A summary for this variant has yet to be developed! Add one now by clicking\n  the "Edit Variant" button.\n	1	\N	\N	d7a2320a-9a3c-43db-bbd6-e2001df49724	2015-03-04 17:25:43.787097
202	149	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: G469L\ndescription: A summary for this variant has yet to be developed! Add one now by clicking\n  the "Edit Variant" button.\n	1	\N	\N	2fae51d3-a2ab-458b-9c02-740789dec2f8	2015-03-04 17:25:43.821828
203	150	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: K642E\ndescription: A summary for this variant has yet to be developed! Add one now by clicking\n  the "Edit Variant" button.\n	1	\N	\N	a8412d6d-b8b1-4130-a75d-ee364ef9fd93	2015-03-04 17:25:43.850883
204	1	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient\n  has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may\n  imply the possibility that BCR-ABL is not universally the initiating event in CML.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 1\nsource_id: 51\nvariant_id: 1\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6db45dc8-4b59-45f8-a69e-ec7bde327f9f	2015-03-04 17:25:44.041547
205	2	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of BCR-ABL fusion is considered the characterizing feature of chronic\n  myeloid leukemia, and has been widely thought of as the initiating event in the\n  disease.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 1\nsource_id: 52\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f1748a13-0ffc-49fe-9c8d-1c571b4691ce	2015-03-04 17:25:44.119228
206	3	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has been effective in\n  patients resistant to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 1\nsource_id: 52\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	20e4aaa1-2edc-4952-ac06-a604e0e3573a	2015-03-04 17:25:44.192029
207	4	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has been effective in\n  patients resistant to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 1\nsource_id: 52\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	dc0c7996-9b1c-4ed6-a07c-dd282abf9563	2015-03-04 17:25:44.252512
208	5	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 1\nsource_id: 53\nvariant_id: 3\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	96cc3219-4a83-4a4c-a20a-4462f17142d5	2015-03-04 17:25:44.321639
209	6	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 1\nsource_id: 53\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	15c2a68e-63e2-4ab7-b663-f685d157cbad	2015-03-04 17:25:44.428334
210	7	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have\n  shown to be resistant to imatinib treatment.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 1\nsource_id: 52\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	84d1ca9a-097b-45a5-94db-5fdc221d4054	2015-03-04 17:25:44.523456
211	8	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the\n  AKT inhibitor MK-2206 in breast cancer cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 5\ndisease_id: 2\nsource_id: 54\nvariant_id: 4\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 14:105246551-105246551 (C->T)\n	1	\N	\N	d0619797-53fe-4e30-ad70-36e1ea163981	2015-03-04 17:25:44.602434
212	9	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment\n  with crizotinib has shown to be effective in acheiving stable disease.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 6\ndisease_id: 3\nsource_id: 55\nvariant_id: 5\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fd0d5b78-e757-4299-a19c-56f2df358788	2015-03-04 17:25:44.674098
213	10	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 6\ndisease_id: 3\nsource_id: 55\nvariant_id: 6\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4264c275-a577-437d-8364-f4a8550c65de	2015-03-04 17:25:44.746007
214	11	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 treatment resulted in significant tumor regression in xenograft models\n  produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 3\nsource_id: 56\nvariant_id: 7\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d1d3c543-8320-49fb-978f-b4af76128902	2015-03-04 17:25:44.824005
215	12	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 6\ndisease_id: 3\nsource_id: 55\nvariant_id: 7\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	849f437c-b293-47cb-89b9-5cf3ebd881db	2015-03-04 17:25:44.885562
216	13	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without\n  this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 4\nsource_id: 57\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29443695-29443695 (G->T)\n	1	\N	\N	63ecd347-32a9-4941-bce6-6733b6778794	2015-03-04 17:25:44.960505
217	14	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in\n  a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 5\nsource_id: 56\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29443695-29443695 (G->T)\n	1	\N	\N	077690ac-3116-4625-aa7f-1c251851f06e	2015-03-04 17:25:45.06361
218	15	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 5\nsource_id: 58\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29443695-29443695 (G->T)\n	1	\N	\N	453e412e-5b71-4c97-9ef1-812d9d4c3098	2015-03-04 17:25:45.130778
219	16	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing\n  the EML4-ALK fusion containing the F1174L mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 5\nsource_id: 59\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29443695-29443695 (G->T)\n	1	\N	\N	6febdd6b-6219-4a4c-b82f-1eea527a7a2a	2015-03-04 17:25:45.197187
220	17	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing\n  ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 8\ndisease_id: 5\nsource_id: 60\nvariant_id: 8\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29443695-29443695 (G->T)\n	1	\N	\N	d5f74cfe-0f48-48b4-a470-bbb41e2559e1	2015-03-04 17:25:45.26689
221	18	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without\n  this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 3\nsource_id: 57\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29443695-29443695 (G->T)\n	1	\N	\N	e8092536-db3b-4694-9e44-39df69f7c303	2015-03-04 17:25:45.330309
222	19	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 5\nsource_id: 58\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29432664-29432664 (C->T)\n	1	\N	\N	e3f8b534-d61f-4a9c-85e0-a7032d7bed3b	2015-03-04 17:25:45.410928
223	20	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells\n  with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 8\ndisease_id: 5\nsource_id: 60\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29432664-29432664 (C->T)\n	1	\N	\N	94ff2699-001e-42d7-ad75-b339219cc7b7	2015-03-04 17:25:45.492365
224	21	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 8\ndisease_id: 5\nsource_id: 60\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:29432664-29432664 (C->T)\n	1	\N	\N	1375fc3d-8ce3-470f-ac60-5356cbfaac29	2015-03-04 17:25:45.586747
225	22	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In one patient with S214C mutation, the use of sorafenib has led to more than\n  5 years of survival and near remission.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 9\ndisease_id: 3\nsource_id: 61\nvariant_id: 10\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: X:47426121-47426121 (C->G)\n	1	\N	\N	193230eb-fba9-4423-883c-f20ea065b9cd	2015-03-04 17:25:45.668215
226	23	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony\n  formation of AALE cells expressing ARAF S214C in vitro\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 9\ndisease_id: 3\nsource_id: 61\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: X:47426121-47426121 (C->G)\n	1	\N	\N	bcbae773-602f-4f6a-af78-a557d252b8e3	2015-03-04 17:25:45.736734
227	24	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony\n  formation of AALE cells expressing ARAF S214C in vitro\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 3\nsource_id: 61\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: X:47426121-47426121 (C->G)\n	1	\N	\N	4b3b21b9-6bf8-4008-a27e-bbb880aaee63	2015-03-04 17:25:45.816334
228	25	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations\n  were associated with poor progression free survival regardless of treatment (panitumumab\n  with best supportive care or best supportive care alone)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 11\ndisease_id: 6\nsource_id: 62\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b950c358-e94d-4a88-80cb-22d9ae157529	2015-03-04 17:25:45.899168
229	26	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 12\ndisease_id: 7\nsource_id: 63\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0518da11-d0af-40b7-8572-d2101dbec747	2015-03-04 17:25:45.981915
230	27	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 7\nsource_id: 63\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	df76fbb2-ed8d-4303-8f7c-88d7b3a5fd56	2015-03-04 17:25:46.043413
231	28	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutations are associated with melanoma arising in non-chronic sun damaged\n  skin and with superficial spreading melanoma\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 7\nsource_id: 64\nvariant_id: 17\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1dcadd3c-2145-41eb-a8f5-a2c0bd0aaf21	2015-03-04 17:25:46.140369
232	29	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 12\ndisease_id: 7\nsource_id: 65\nvariant_id: 11\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453135-140453136 (CA->AT)\n	1	\N	\N	75896037-51dd-4c79-9c10-9b21770caa28	2015-03-04 17:25:46.214226
233	30	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 66\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	8d3ca064-0c9e-400a-ac74-a0c61926ba82	2015-03-04 17:25:46.308244
234	31	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 8\nsource_id: 66\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	d9a068c7-3e69-4574-9e22-38161d511a3c	2015-03-04 17:25:46.370417
235	32	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 9\ndisease_id: 8\nsource_id: 66\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	9448ac86-757c-4f14-a830-f8dd18b1c91a	2015-03-04 17:25:46.431647
236	33	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 8\nsource_id: 67\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	6e8d3b73-91ee-4950-bce2-edf4d33e9be4	2015-03-04 17:25:46.506431
237	34	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 15\ndisease_id: 8\nsource_id: 67\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	9e1477c5-307b-4f86-89b6-e1f4cf809c0e	2015-03-04 17:25:46.563203
238	35	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 15\ndisease_id: 8\nsource_id: 68\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	80558d97-d26d-4245-9cf8-15d2e9d4a7f4	2015-03-04 17:25:46.634324
239	36	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 16\ndisease_id: 8\nsource_id: 68\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	9b817c5d-4e92-455f-a65d-c71a6fee2459	2015-03-04 17:25:46.709358
240	37	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 17\ndisease_id: 6\nsource_id: 69\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	33534a6a-ee02-4984-9d16-e49c0c80bb2c	2015-03-04 17:25:46.786599
241	38	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 18\ndisease_id: 6\nsource_id: 69\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	d302b873-2ba6-48ae-a027-dfca509ec1b7	2015-03-04 17:25:46.840151
242	39	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 19\ndisease_id: 6\nsource_id: 69\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	2b3816d5-dae9-4bd0-bdf5-3fe31d9448cf	2015-03-04 17:25:46.911233
243	40	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations\n  had lower response and disease control rates as well as shorter progression free\n  and overall survival following cetuximab plus chemotherapy than those with wildtype\n  BRAF\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 13\ndisease_id: 6\nsource_id: 70\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	43f3402a-a0d2-4496-a8d3-90e4f8c4a2bf	2015-03-04 17:25:46.980098
244	41	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is associated with adverse pathological features of colorectal cancer.\n  This can be concluded as a marker of poor prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 6\nsource_id: 71\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	7c9282ec-4653-4fe8-ae77-3d5a1b5558e9	2015-03-04 17:25:47.051739
245	42	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines expressing the BRAF V600E mutation responded better to vemurafenib\n  treatment than cells wildtype for BRAF as measured by reduced cellular proliferation\n  and inhibition of MET and ERK phosphorylation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 18\ndisease_id: 6\nsource_id: 69\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	d8c0b3f7-e498-4fdb-807f-45970b6f8a6e	2015-03-04 17:25:47.121454
246	43	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 20\ndisease_id: 7\nsource_id: 72\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	0369ed79-3493-4e8d-bd33-de7c76ac1a2d	2015-03-04 17:25:47.212717
247	44	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 21\ndisease_id: 7\nsource_id: 72\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	f8641599-a8fc-4b18-ba5c-61cbf4bc3fdc	2015-03-04 17:25:47.273848
248	45	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 12\ndisease_id: 7\nsource_id: 73\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	6029fdda-d2d3-4cf7-b4f5-c6ea844a1e74	2015-03-04 17:25:47.343952
249	46	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 7\nsource_id: 73\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	1bbb9c94-83ad-4f0d-a9be-584fc65e6789	2015-03-04 17:25:47.411167
250	47	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  but does not show resistance to MEK inhibitors.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 7\nsource_id: 74\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	bc204be1-362b-4e80-9405-0d83e003ac53	2015-03-04 17:25:47.478065
251	48	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  and resistance to RAF inhibitors.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 7\nsource_id: 74\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	2f8c5437-c643-44ad-8ca5-8b8cdf6a92c1	2015-03-04 17:25:47.535986
252	49	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with shorter disease-free and overall Survival in a\n  Spanish cohort of melanoma patients.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 7\nsource_id: 75\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	9bd60507-880d-4ccc-8c0e-2e396619efe6	2015-03-04 17:25:47.605751
253	50	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 12\ndisease_id: 3\nsource_id: 76\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	db94bc85-2684-495b-9d0a-76b437748b47	2015-03-04 17:25:47.699647
254	51	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is shown to be associated with the tall-cell variant of papillary\n  thyroid cancer\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 9\nsource_id: 77\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	c429680b-321e-4fea-8ce3-078639382cc9	2015-03-04 17:25:47.771854
255	52	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 9\nsource_id: 78\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	78e0e5eb-79c9-4777-8485-e29a42e803cc	2015-03-04 17:25:47.832973
256	53	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 9\nsource_id: 79\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	eba2a5e7-9834-4f32-bfed-68b6cfdb7813	2015-03-04 17:25:47.89568
257	54	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Unlike other studies that suggest a poorer outcome, BRAF mutation in this study\n  was not correlated with poorer prognosis in papillary thyroid cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 9\nsource_id: 80\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	b22cd9b5-9cb5-4e7a-a631-4a942dceedb6	2015-03-04 17:25:47.967057
258	55	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutation correlated with poor prognosis in papillary thyroid cancer in\n  both older (>65 yo) and younger (<65 yo) cohorts.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 9\nsource_id: 77\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	bb08ab8a-f168-48cf-aeac-d5faa50f0e43	2015-03-04 17:25:48.046103
259	56	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in\n  a study of 187 patients with PTC and other thyroid diseases.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 9\nsource_id: 81\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	4fa03903-2b2e-46cf-92ee-d38c704fbfb5	2015-03-04 17:25:48.147076
260	57	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is correlated with disease recurrence in both age cohorts (>65 and <65\n  yo).\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 9\nsource_id: 77\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453136 (A->T)\n	1	\N	\N	982f7216-d74d-46ed-87b2-c78dafcde695	2015-03-04 17:25:48.216945
261	58	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COLO201 and COLO206F cells harboring BRAF V600E mutations were cloned to be\n  MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to\n  be amplification of the BRAF V600E gene.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 22\ndisease_id: 6\nsource_id: 82\nvariant_id: 14\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	39081229-9a5c-471c-a236-19917127bc20	2015-03-04 17:25:48.293729
262	59	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A single patient with BRAF V600E/V600M bi-allelic mutation responded to the\n  V600E drug dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 12\ndisease_id: 7\nsource_id: 83\nvariant_id: 13\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7e53f8ed-8adf-4cdd-8ea5-2c003af57420	2015-03-04 17:25:48.365416
263	60	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 12\ndisease_id: 7\nsource_id: 65\nvariant_id: 15\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453137-140453137 (C->T)\n	1	\N	\N	ffe98cd0-79f4-4d40-b7c4-0e28dfb0c74d	2015-03-04 17:25:48.44577
264	61	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 12\ndisease_id: 7\nsource_id: 65\nvariant_id: 16\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:140453136-140453137 (AC->CT)\n	1	\N	\N	8f49eb15-2ff5-4ab3-8fc6-541e761eda71	2015-03-04 17:25:48.527326
265	62	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The combination of PARP inhibition (Olaparib) with BRCA mutation results in\n  irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation positive\n  women with high grade serous or undifferentiated ovarian cancer.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 23\ndisease_id: 10\nsource_id: 84\nvariant_id: 129\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 2\nvariant_hgvs: N/A\n	1	\N	\N	e492f769-275b-402d-af6e-b1bdce238b34	2015-03-04 17:25:48.696629
266	63	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The combination of PARP inhibition (Olaparib) with BRCA mutation results in\n  irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation positive\n  women with high grade serous or undifferentiated ovarian cancer.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 23\ndisease_id: 10\nsource_id: 84\nvariant_id: 130\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 2\nvariant_hgvs: N/A\n	1	\N	\N	1d1474c5-a8d4-4a0b-aa5b-4d1481c11c24	2015-03-04 17:25:48.782121
267	64	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Inceased copy number of CCND1 is associated with poorer overall survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 85\nvariant_id: 18\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bce866f3-f059-4d64-8073-9e65ceaa71e7	2015-03-04 17:25:48.86101
268	65	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have found no significant survival impact for CCND1 in lung cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 85\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	00afe03a-ecac-4f4b-a944-b4a3d708dad5	2015-03-04 17:25:48.934827
269	66	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have associated CCND1 expression with poorer survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 85\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4ff425e4-5d73-47a5-9c0c-74df123ab5c3	2015-03-04 17:25:49.011257
270	67	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with the ER-positive subtype of breast\n  cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 11\nsource_id: 86\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f09fac08-a820-4dc0-85f8-8f8e6abe1e16	2015-03-04 17:25:49.093587
271	68	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In ER-positive breast cancer patients, Cyclin D1 overexpression is associated\n  with shorter overall survival and increased metastasis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 11\nsource_id: 86\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7b92526b-0cc2-4f4f-8a7a-1bf29297f39f	2015-03-04 17:25:49.174235
272	69	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in head and neck squamous cell carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 12\nsource_id: 87\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	96e7e53d-5ff5-4825-a553-fb7a104ba127	2015-03-04 17:25:49.261464
273	70	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in mantle cell lymphoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 13\nsource_id: 88\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d2789bbd-34e2-4bfe-87ff-4c863d464e3a	2015-03-04 17:25:49.352194
274	71	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 14\nsource_id: 89\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	57d1a8a6-c2e8-4737-bc0d-accaf914bcf4	2015-03-04 17:25:49.465004
275	72	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the increased depth of cancer invasion\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 14\nsource_id: 89\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5e04c47a-2726-440b-8211-7520955b02b0	2015-03-04 17:25:49.558519
276	73	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the presence pf lymph node metastasis\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 14\nsource_id: 89\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	591e503a-091f-468b-b4b8-b28238241e17	2015-03-04 17:25:49.646508
277	74	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with vascular invasion by cancer cells\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 14\nsource_id: 89\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ed74fb58-ecd8-4bcb-8b35-9c970e7d574d	2015-03-04 17:25:49.732642
278	75	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CCND2 hypomethylation is seen to be more common in stage III and IV gastric\n  tumors than stage I and II.\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 14\nsource_id: 90\nvariant_id: 22\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	83b5a0a1-da79-408e-ad43-87a098b1a6bf	2015-03-04 17:25:49.820787
279	76	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib\n  (PD-0332991) significantly increased median survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 24\ndisease_id: 15\nsource_id: 91\nvariant_id: 23\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6a2f0fbe-d4d0-4ea0-8bd0-22fe58bed682	2015-03-04 17:25:49.907099
280	77	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with high expression of total cyclin E and low-molecular weight cyclin\n  E is strongly associated with poor prognosis in breast cancer, and the hazard ratio\n  for these patients in 13.3 times higher than those with normal cyclin E levels.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 11\nsource_id: 92\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6ebc8234-47a0-49ec-a17e-2a949fc0a313	2015-03-04 17:25:49.995155
281	78	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced\n  stage gastric carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 16\nsource_id: 93\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	09bd5d68-3a8e-44a1-92a8-3432d74354f2	2015-03-04 17:25:50.093501
282	79	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 16\nsource_id: 93\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	744cf50e-59b0-4326-a054-4588e9da5fdf	2015-03-04 17:25:50.1798
283	80	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 17\nsource_id: 93\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ca051992-8a2b-4a0d-9d16-1550d317f8fe	2015-03-04 17:25:50.24709
284	81	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 25\ndisease_id: 18\nsource_id: 94\nvariant_id: 25\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3c504901-f054-4c70-8b5a-b26bb7f5a8b8	2015-03-04 17:25:50.308954
285	82	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 26\ndisease_id: 18\nsource_id: 95\nvariant_id: 25\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	056eda01-e901-4bed-a566-0a6236d46edb	2015-03-04 17:25:50.390596
286	83	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 25\ndisease_id: 18\nsource_id: 94\nvariant_id: 26\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	78addf5b-d937-41bc-801a-4f87cb51b790	2015-03-04 17:25:50.453921
287	84	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 26\ndisease_id: 18\nsource_id: 95\nvariant_id: 26\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d8f6bc28-cf6c-4fef-a6f8-2dd0465fd10d	2015-03-04 17:25:50.530919
288	85	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter overall survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 96\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0ea5d42d-07bc-475f-85eb-5006f17e5263	2015-03-04 17:25:50.608149
289	86	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter time to recurrence.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 96\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f4703254-09f6-4adb-971c-61ffcb9b10ae	2015-03-04 17:25:50.678885
290	87	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation status had no impact on ATRA treatment response in older (>60)\n  patients with AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 97\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	927c5845-611a-4d8d-b2ea-8f304813dced	2015-03-04 17:25:50.754186
291	88	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was associated with improved overall survival in older (>60)\n  patients with AML\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 97\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fca2fdeb-b1f9-4654-a293-a4ae15060da5	2015-03-04 17:25:50.846319
292	89	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was significantly associated with complete remission\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 98\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1e4db166-a0bc-4d8c-9ff3-272ec3e5ca84	2015-03-04 17:25:50.91819
293	90	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 99\nvariant_id: 28\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	dd3620c8-fb5d-4fda-abb4-f87c2329f5f4	2015-03-04 17:25:50.987854
294	91	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Overall survival was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA mutations\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 99\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7cd5ad8e-9535-4660-9388-450b8e190cb8	2015-03-04 17:25:51.061034
295	92	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Remission duration was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 99\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	559bb2df-4977-48f8-852a-77f2509a2120	2015-03-04 17:25:51.136516
296	93	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 20\nsource_id: 100\nvariant_id: 30\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	79abb557-d40f-48fe-9800-2e22c08c8d0a	2015-03-04 17:25:51.223384
297	94	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 27\ndisease_id: 20\nsource_id: 100\nvariant_id: 30\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	488b7f39-b362-4ed0-ace9-18c60ca32fdc	2015-03-04 17:25:51.290074
298	95	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In 3T3 fibroblasts expressing the G253C mutation have to be sensitive to the\n  tyrosine kinase inhibitor dasatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 101\nvariant_id: 139\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7ac7dfe5-8b30-4df8-acab-8a01e572da2f	2015-03-04 17:25:51.374653
299	96	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In 3T3 fibroblasts expressing the G505S mutation have to be sensitive to the\n  tyrosine kinase inhibitor dasatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 101\nvariant_id: 140\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	dffdf3c8-704f-4b38-bdb5-bc3ff4af342d	2015-03-04 17:25:51.519359
300	97	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In 3T3 fibroblasts expressing the G774V mutation have to be sensitive to the\n  tyrosine kinase inhibitor dasatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 101\nvariant_id: 142\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b701d84f-d754-4598-bd83-6d48373af335	2015-03-04 17:25:51.625618
301	98	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In 3T3 fibroblasts expressing the I638F mutation have to be sensitive to the\n  tyrosine kinase inhibitor dasatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 101\nvariant_id: 141\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	da6662da-eae9-487a-ba89-15934e915de5	2015-03-04 17:25:51.705939
302	99	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In 3T3 fibroblasts expressing the L239R mutation have to be sensitive to the\n  tyrosine kinase inhibitor dasatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 101\nvariant_id: 138\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2cf06417-cd10-447a-b39d-79e4d9a30572	2015-03-04 17:25:51.769795
303	100	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In 3T3 fibroblasts expressing the L63V mutation have to be sensitive to the\n  tyrosine kinase inhibitor dasatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 101\nvariant_id: 137\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	df9d10ff-d9c8-4bde-b4a1-1505330d109f	2015-03-04 17:25:51.831599
304	101	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient expressing a DDR2 S768R mutation, treatment with dasatinib plus\n  erlotinib acheived a partial response.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 3\nsource_id: 101\nvariant_id: 143\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3786fd24-0ab5-4d50-b8b3-f81754f4b904	2015-03-04 17:25:51.905052
305	102	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient expressing a DDR2 S768R mutation, treatment with dasatinib plus\n  erlotinib acheived a partial response.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 3\nsource_id: 101\nvariant_id: 143\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7c666ff0-0c74-479c-9ece-d785f5b4f3d6	2015-03-04 17:25:51.968202
306	103	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 21\nsource_id: 102\nvariant_id: 31\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1b57c0c8-ee1c-4911-a266-823a9bcdbd1c	2015-03-04 17:25:52.040713
307	104	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Daunorubicin treatment resulted in similar overall survival and disease free\n  survival in de novo AML patients with DNMT3A R882 mutation compared to those who\n  do not harbor this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 29\ndisease_id: 19\nsource_id: 103\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7c119f72-b008-43be-ac92-da20f4ac209c	2015-03-04 17:25:52.130733
308	105	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Idarubicin increases the overall survival and disease free survival in de novo\n  AML patients with DNMT3A R882 mutation compared to those who do not harbor this\n  mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 30\ndisease_id: 19\nsource_id: 103\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b86356de-c0d8-43be-848c-c0f632f76d69	2015-03-04 17:25:52.198975
309	106	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Therapy-related AML was less common in patients with DNMT3A mutations (64.5%\n  of which were R882) than patients wildtype for DNMT3A in a large cohort of younger\n  (18-60) AML patients\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2378c7c0-62de-47ea-82c8-7ae878505f6b	2015-03-04 17:25:52.261799
310	107	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with an intermediate\n  risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 105\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	406a2230-cbff-4dc0-8fd5-9e8543b88af4	2015-03-04 17:25:52.325882
311	108	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with intermediate\n  risk cytogenetics (including normal karyotype)\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 105\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	45f50cca-c39d-4c85-a02d-134b04db1b02	2015-03-04 17:25:52.391086
312	109	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (64.5% R882) were associated with older age, higher white blood\n  cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML\n  patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c0a5ed2d-d18e-483b-9049-2af66895739f	2015-03-04 17:25:52.459617
313	110	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations occur most often in de novo AML patients with intermediate\n  cytogenic risk.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 103\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	af933a0e-17e6-48eb-9cca-7fcb54fdd359	2015-03-04 17:25:52.524524
314	111	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with cytogenetically normal AML in a large\n  cohort of younger (18-60) AML patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a156d0ea-043e-41cc-9798-5381b4a0e370	2015-03-04 17:25:52.585157
315	112	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with older age, higher white blood cell\n  count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically\n  normal AML patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 106\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bb83c9cb-cd69-4823-ba5c-01a473bb914a	2015-03-04 17:25:52.675705
316	113	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutations (60% of which were\n  R882) were older in age, had higher white blood cell counts and had higher platelet\n  counts than patients wildtype for DNMT3A\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 107\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f4c2fc31-551c-4505-b033-bbbabf8ed218	2015-03-04 17:25:52.747149
317	114	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates did not differ between patients with wildtype or mutant\n  DNMT3A (62% of which affected R882) and cytogenetically normal AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 108\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	32b227fe-75dc-475d-9a14-d26ea90954a1	2015-03-04 17:25:52.821298
318	115	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5%\n  of which were R882) had no prognostic value on overall, relapse free and event free\n  survival.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	70596e3e-4cfd-401f-8170-2d2e689b5571	2015-03-04 17:25:52.883792
319	116	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of cytogenetically normal AML patients (18-60 years old),\n  DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall,\n  relapse free and event free survival\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	cb299e9e-e572-4080-b20b-47415c903673	2015-03-04 17:25:52.942316
320	117	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status (60% of which\n  were R882) was not predictive of overall and relapse free survival in patients with\n  NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 107\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	40509e2f-959c-4edd-997c-9bb620238abb	2015-03-04 17:25:53.006262
321	118	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no difference in the complete remission rate of de novo AML patients\n  with DNMT3A mutation compared to those who are wild type for DNMT3A.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 103\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b87194dc-4f27-499e-9db1-48ef10ff82f5	2015-03-04 17:25:53.070282
322	119	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations were associated with achievement of complete remission in a\n  large cohort of younger (18-60) AML patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	acc6b4a3-8d4a-466e-9e42-396c0ca19bf8	2015-03-04 17:25:53.137787
323	120	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with DNMT3A mutations (59% of which were R882) showed worse survival\n  (event-free and overall) outcome than those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 105\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	adc16e6f-a402-490a-b15a-2e1d15cbfc6b	2015-03-04 17:25:53.228285
324	121	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rate was not different between young AML patients (<60 years\n  old) with or without DNMT3A mutations (60% of which were R882)\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 107\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f61b570b-3007-4f9f-a571-20f5a4371199	2015-03-04 17:25:53.296529
325	122	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free\n  and overall) outcome than those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 105\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d0f59a74-a758-47b8-9a91-7194351a9e94	2015-03-04 17:25:53.366463
326	123	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (62% of which were R882) were associated with reduced disease-free\n  survival in patients with cytogenetically normal AML.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 108\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bbca3722-c3ac-4fe8-a142-ef638cb8585e	2015-03-04 17:25:53.428578
327	124	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutation was associated with reduced relapse free and overall survival\n  in ELN-unfavorable, cytogenetically normal AMLs\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	35c40a1c-6339-4d6e-a8bf-c1573e7c53a8	2015-03-04 17:25:53.489814
328	125	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations\n  were associated with reduced relapse free survival in the entire cohort as well\n  as the subset of patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 19\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6295b4bf-08cc-434c-9370-72fcef05deb0	2015-03-04 17:25:53.558161
329	126	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including\n  R882) were associated with worse overall survival compared to those without DNMT3A\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 105\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0ff582c7-4495-4826-a1ba-44c21a889424	2015-03-04 17:25:53.618811
330	127	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cytogenetically normal AML patients, DNMT3A R882 mutations are associated\n  with lower overall and disease free survival as compared to patients with wild type\n  DNMT3A.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 106\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	67169012-b7a0-4f47-9c94-8f8b87ce6cf8	2015-03-04 17:25:53.684049
331	128	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation\n  is prognostic for shorter disease free survival and overall survival compared to\n  patients without the mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 108\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8f17cc20-6b55-4883-a286-7b853e161ecf	2015-03-04 17:25:53.765905
332	129	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutations were associated with\n  significantly reduced overall survival and relapse free survival in patients wildtype\n  for NPM1 and FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 107\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0360de4e-7d5b-44de-8170-2d7b9d4d90e3	2015-03-04 17:25:53.835261
333	130	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations\n  other than R882 are prognostic for shorter disease free survival and overall survival\n  compared to patients without the mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 108\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d17276f5-aa84-4320-870e-7467aa8089a9	2015-03-04 17:25:53.898165
334	131	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutation have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 107\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bf8344e1-a212-48db-8531-069798591641	2015-03-04 17:25:53.972062
335	132	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 107\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	883f8f11-1b9e-429c-a0dc-2802fdd1283b	2015-03-04 17:25:54.031098
336	133	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Exon 19 deletion has been shown to be correlated with gefitinib response.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 3\nsource_id: 109\nvariant_id: 131\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9011a03e-9146-43ac-869c-aa5ec6080f0b	2015-03-04 17:25:54.102425
337	134	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Gefinitib has been shown to be effective in treating cell lines with G719S missense\n  mutations.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 3\nsource_id: 110\nvariant_id: 132\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e686a7e9-efca-4867-979e-da13aa4bf8fb	2015-03-04 17:25:54.161882
338	135	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 31\ndisease_id: 3\nsource_id: 111\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55259514-55259515 (CT->AG)\n	1	\N	\N	2d5fe488-5b2d-48b3-85a0-6ff3949a1740	2015-03-04 17:25:54.228786
339	136	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 3\nsource_id: 111\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55259514-55259515 (CT->AG)\n	1	\N	\N	1de7e04e-4b9c-419a-a1ea-142acc0b7f3c	2015-03-04 17:25:54.31377
340	137	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence\n  of L858R mutation is prognostic for better progression free survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 112\nvariant_id: 33\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55259514-55259515 (CT->AG)\n	1	\N	\N	ea0aa23e-f6a3-4473-a93c-a9d4376be6a3	2015-03-04 17:25:54.391317
341	138	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Median survival of patients with EGFR L858R mutation is better than those with\n  wild type EGFR.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 113\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55259514-55259515 (CT->AG)\n	1	\N	\N	ace0c177-53d4-462d-8321-8357603d8edb	2015-03-04 17:25:54.451844
342	139	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Gefinitib has been shown to be effective in treating cell lines with L858R missense\n  mutations.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 3\nsource_id: 110\nvariant_id: 33\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ac5be9f4-4fcd-4459-8999-e63846f83e10	2015-03-04 17:25:54.513529
343	140	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The T790M mutation in EGFR has been shown to confer resistance to the tyrosine\n  kinase inhibitor erlotinib, and patients harboring this mutation that are placed\n  on the drug are likely to relapse.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 28\ndisease_id: 22\nsource_id: 114\nvariant_id: 34\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55249071-55249071 (C->T)\n	1	\N	\N	e3610537-548e-4d3b-ba77-708df55cd091	2015-03-04 17:25:54.580901
344	141	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting\n  in stable disease.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 3\nsource_id: 115\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55249071-55249071 (C->T)\n	1	\N	\N	3601e9b6-3c8a-4439-89bb-77dea949015d	2015-03-04 17:25:54.652531
345	142	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting\n  in stable disease.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 32\ndisease_id: 3\nsource_id: 115\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55249071-55249071 (C->T)\n	1	\N	\N	032fd521-43a4-481d-b5c4-405be0f91e83	2015-03-04 17:25:54.712535
346	143	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In non-small cell lung cancer, the appearance of T790M mutation leads to resistance\n  to gefitinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 31\ndisease_id: 3\nsource_id: 116\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55249071-55249071 (C->T)\n	1	\N	\N	90cd49c8-a873-426e-b3c7-423b33b515de	2015-03-04 17:25:54.798026
347	144	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients with T790M and another activating mutations, their progression\n  free survival is shorter compared to those who do not possess T790M mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 117\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55249071-55249071 (C->T)\n	1	\N	\N	b8d217b7-ac6d-4ef3-b7fc-f0dad2f08de2	2015-03-04 17:25:54.871584
348	145	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall\n  survival compared to patients with L858R or other exon 19 activating mutations.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 118\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55249071-55249071 (C->T)\n	1	\N	\N	7d5f9fe0-e74e-4871-804d-26b20492fd67	2015-03-04 17:25:54.93812
349	146	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR,\n  especially when an L858R mutation is also present.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 33\ndisease_id: 3\nsource_id: 119\nvariant_id: 34\nrating: 1\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 7:55249071-55249071 (C->T)\n	1	\N	\N	9c543ce2-173f-458c-888f-50c275667b3c	2015-03-04 17:25:55.009384
350	147	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804),\n  MET amplification and EGFR T790M were found to be the primary drivers of resistance.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 3\nsource_id: 120\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	dfc2ea76-9e65-42f5-a477-a5da2686f6f3	2015-03-04 17:25:55.078588
351	148	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 35\ndisease_id: 11\nsource_id: 121\nvariant_id: 35\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37880261-37880261 (G->C)\n	1	\N	\N	15d66c9a-f7af-43ba-bb87-f220b1e0960c	2015-03-04 17:25:55.155626
352	149	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 35\ndisease_id: 11\nsource_id: 121\nvariant_id: 36\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37880261-37880261 (G->T)\n	1	\N	\N	fa7aac73-1031-47f6-a428-cb1012937edb	2015-03-04 17:25:55.241604
353	150	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2,\n  cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive\n  to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 35\ndisease_id: 11\nsource_id: 121\nvariant_id: 37\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2dc58eb8-abe6-41e8-89a2-6a400d6fa526	2015-03-04 17:25:55.329708
354	151	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 35\ndisease_id: 11\nsource_id: 121\nvariant_id: 38\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37868205-37868205 (G->C)\n	1	\N	\N	438395ba-c473-4538-abd5-34614160b19b	2015-03-04 17:25:55.391566
355	152	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell\n  lines retrovirally transduced with wildtype or mutant ERBB2.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 36\ndisease_id: 11\nsource_id: 121\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37880220-37880220 (T->C)\n	1	\N	\N	ea39dcde-610d-43d1-8bb0-c8df0ec16ce8	2015-03-04 17:25:55.468224
356	153	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally\n  transduced with wildtype or mutant ERBB2.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 35\ndisease_id: 11\nsource_id: 121\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37880220-37880220 (T->C)\n	1	\N	\N	39627298-0bc4-4f2f-9e0f-1b7293727c37	2015-03-04 17:25:55.541688
357	154	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive\n  to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 35\ndisease_id: 11\nsource_id: 121\nvariant_id: 40\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6f468b4d-ba2a-4301-8356-26f5e3344a64	2015-03-04 17:25:55.60581
358	155	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the in-frame insertion of a single amino acid at position\n  780 was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 35\ndisease_id: 11\nsource_id: 121\nvariant_id: 41\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ebb35dbd-f781-479c-8869-e7e66f1cc955	2015-03-04 17:25:55.685217
359	156	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 35\ndisease_id: 11\nsource_id: 121\nvariant_id: 42\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	04c2d893-68e1-4851-a3cc-5be8653cf5e0	2015-03-04 17:25:55.749144
360	157	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 36\ndisease_id: 11\nsource_id: 121\nvariant_id: 42\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0dfe1426-e7d5-48b1-827f-7870c39ca5e2	2015-03-04 17:25:55.826009
361	158	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 35\ndisease_id: 11\nsource_id: 121\nvariant_id: 43\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37881616-37881616 (C->T)\n	1	\N	\N	46046fd1-6440-43a8-b44e-4deee1bdca26	2015-03-04 17:25:55.889772
362	159	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 35\ndisease_id: 11\nsource_id: 121\nvariant_id: 44\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37881000-37881000 (G->T)\n	1	\N	\N	f806ea6a-7637-41ce-bc20-8e506b6d1f73	2015-03-04 17:25:55.955205
363	160	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 35\ndisease_id: 11\nsource_id: 121\nvariant_id: 45\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:37881332-37881332 (G->A)\n	1	\N	\N	29bf04dd-2eeb-4eb1-818e-ed00bf934989	2015-03-04 17:25:56.032511
364	161	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 37\ndisease_id: 11\nsource_id: 122\nvariant_id: 46\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8b46e228-fa2a-4fa7-9a7a-ede081b17dd0	2015-03-04 17:25:56.100556
365	162	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L536Q ligand binding domain mutation have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 11\nsource_id: 123\nvariant_id: 46\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0e344e38-25d8-48cd-92a7-d1b4f34b7415	2015-03-04 17:25:56.177846
366	163	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L536Q ligand binding domain mutation have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 39\ndisease_id: 11\nsource_id: 123\nvariant_id: 46\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	77dec223-42bd-4efc-bcca-336169a40e32	2015-03-04 17:25:56.24396
367	164	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 37\ndisease_id: 11\nsource_id: 122\nvariant_id: 47\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f72f1bb1-4ea5-4e14-a75d-bf587e915082	2015-03-04 17:25:56.343497
368	165	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The N538G ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 11\nsource_id: 123\nvariant_id: 47\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7d10fbbd-3cea-42bf-8835-2449f835fa13	2015-03-04 17:25:56.418885
369	166	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The N538G ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 39\ndisease_id: 11\nsource_id: 123\nvariant_id: 47\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6d8a558f-d6ce-42b2-a6e6-ae36cfe36a6c	2015-03-04 17:25:56.475577
370	167	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 37\ndisease_id: 11\nsource_id: 122\nvariant_id: 48\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419923-152419923 (A->G)\n	1	\N	\N	64ed9e17-1e7f-40c7-a4f4-3df561fc314a	2015-03-04 17:25:56.549041
371	168	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537C ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 11\nsource_id: 123\nvariant_id: 48\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419923-152419923 (A->G)\n	1	\N	\N	73261e05-d14b-45b7-a560-1b047e4f03b7	2015-03-04 17:25:56.613728
372	169	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537C ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 39\ndisease_id: 11\nsource_id: 123\nvariant_id: 48\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419923-152419923 (A->G)\n	1	\N	\N	9347c9f3-7fcc-4ba2-abd6-e116e6ef8254	2015-03-04 17:25:56.674713
373	170	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 37\ndisease_id: 11\nsource_id: 122\nvariant_id: 49\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419922-152419922 (T->A)\n	1	\N	\N	d3f72228-9bfc-457a-afb2-7e39a8b98972	2015-03-04 17:25:56.741207
374	171	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537N ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 11\nsource_id: 123\nvariant_id: 49\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419922-152419922 (T->A)\n	1	\N	\N	0fef9cf0-eb2d-4fb1-a5aa-24ad7ba18496	2015-03-04 17:25:56.835708
375	172	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537N ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 39\ndisease_id: 11\nsource_id: 123\nvariant_id: 49\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419922-152419922 (T->A)\n	1	\N	\N	5c127897-00c8-43a8-a4dd-5568d62ef038	2015-03-04 17:25:56.892378
376	173	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 37\ndisease_id: 11\nsource_id: 122\nvariant_id: 50\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419923-152419923 (A->C)\n	1	\N	\N	9083e1ed-748b-42ee-84d8-c6fec37d3bbe	2015-03-04 17:25:56.961822
377	174	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537S ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 11\nsource_id: 123\nvariant_id: 50\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419923-152419923 (A->C)\n	1	\N	\N	cc00f49a-bc96-471d-87cb-d0395c9116f7	2015-03-04 17:25:57.030908
378	175	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537S ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 39\ndisease_id: 11\nsource_id: 123\nvariant_id: 50\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 6:152419923-152419923 (A->C)\n	1	\N	\N	a0bd3a62-00d9-4c90-ac9a-25f575e6c35d	2015-03-04 17:25:57.092383
379	176	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib\n  treatment resulted in necrosis and shrinkage of the primary and metastatic sites\n  and overall stable disease. However, this response did not meet criteria for RECIST\n  partial response.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 40\ndisease_id: 23\nsource_id: 124\nvariant_id: 51\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b52fa20c-107e-42d8-8266-df1c9d6990f1	2015-03-04 17:25:57.165369
380	177	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Following identification of this fusion in a patient, this fusion was expressed\n  in HEK 293T cells leading to oligomerization in the absence of ligand, induced morphologic\n  changes in the cell and increased cellular proliferation consistent with FGFR activation.\n  Mouse xenografts using the bladder cell line SW780 which harbors an FGFR3-BAIAP2L1\n  fusion showed reduced tumor burden when treated with the FGFR inhibitor PD173074.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 41\ndisease_id: 24\nsource_id: 125\nvariant_id: 52\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	99f10fc6-3a13-415a-9450-12bf19a83c12	2015-03-04 17:25:57.25578
381	178	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This\n  fusion is also shown to be recurrent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 42\ndisease_id: 23\nsource_id: 124\nvariant_id: 53\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	dc740c78-9b95-45c6-ae12-ec6ffeb58285	2015-03-04 17:25:57.350767
382	179	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This\n  fusion is also shown to be recurrent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 40\ndisease_id: 23\nsource_id: 124\nvariant_id: 53\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9b93d895-b7b5-4102-91cc-9d9d6877b927	2015-03-04 17:25:57.413327
383	180	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with\n  FLT3 internal tandem repeat\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 43\ndisease_id: 19\nsource_id: 126\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c50c4770-f584-4d9c-8b56-6c4aaa9b873a	2015-03-04 17:25:57.484495
384	181	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  FLT3-ITD mutation\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 44\ndisease_id: 19\nsource_id: 127\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5fc14366-a251-4c27-ba6a-e1adb874b19e	2015-03-04 17:25:57.56413
385	182	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML\n  patients with FLT3 mutations\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 45\ndisease_id: 19\nsource_id: 128\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f2d35fa8-7c6a-4379-9aa7-340a9265bcc7	2015-03-04 17:25:57.640089
475	272	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12C occur more frequently in women than men\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 17\nsource_id: 178\nvariant_id: 77\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398285-25398285 (C->A)\n	1	\N	\N	b7863cc2-4e31-4a6d-9f23-97807c624e65	2015-03-04 17:26:04.359222
386	183	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 19\nsource_id: 129\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fe4faf44-182c-4809-9a91-3c2e3de032d5	2015-03-04 17:25:57.705765
387	184	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared\n  to wild type\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 130\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c777d2f3-3972-4e24-b180-cbec1ea7d427	2015-03-04 17:25:57.773922
388	185	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD mutation without NPM1 was associated with increased relapse risk and\n  reduced overall survival in young adult AML patients (median age 43)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 131\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4a47e8d6-12d9-43e8-9c64-cf9a6a88e34c	2015-03-04 17:25:57.865127
389	186	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Increasing level of FLT3-ITD mutant correlated with reduced disease-free and\n  overall survival as well as increased relapse rate\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 131\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f3ae10ca-1ebd-4d29-96c5-78887e88e63d	2015-03-04 17:25:57.927744
390	187	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD duplication length did not effect prognosis in patients with an NPM1\n  mutation\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 132\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	521815e2-2ac0-4e0c-9611-18c9432893f2	2015-03-04 17:25:57.992032
391	188	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 107\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9712d162-50e4-487a-a5a9-dbf4202f020d	2015-03-04 17:25:58.052034
392	189	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Size of the FLT3-ITD mutant duplication had no impact on overall survival or\n  relapse rate\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 131\nvariant_id: 54\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f7f51b17-8f14-49ef-b241-b28e3bb737b0	2015-03-04 17:25:58.118451
393	190	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free\n  and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 132\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8997bbcf-cdba-4b84-8eee-4c70c24d3d55	2015-03-04 17:25:58.178365
394	191	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with internal tandem repeat of FLT3 has worse overall survival\n  than patients with wild type FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 130\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	98fbfa0d-d298-4491-968c-fd45cf2e2e3b	2015-03-04 17:25:58.268965
395	192	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated\n  with worse overall survival compared to those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 105\nvariant_id: 54\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ae030ee5-0c39-4333-8621-cd9212a39ee0	2015-03-04 17:25:58.33121
396	193	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Meta-analysis of studies involving cytogentically normal younger (<60) patients\n  showed reduced overall and relapse-free survival for patients with FLT3-ITD\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 19\nsource_id: 133\nvariant_id: 54\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4de55ed5-3128-4bb3-bf1e-29ab28586155	2015-03-04 17:25:58.415217
397	194	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 132\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5197f0ae-108a-416d-b1c0-e0b63078a501	2015-03-04 17:25:58.481953
398	195	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor\n  survival and inferior disease free survival in patients with AML.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 134\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8e13c4ca-9c25-4c10-9427-043012b63e2e	2015-03-04 17:25:58.552302
399	196	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 132\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4f444cd7-7a32-4c5f-9c68-b4e7584bf025	2015-03-04 17:25:58.629925
400	197	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeats, but not\n  in wild type or D835 mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 9\ndisease_id: 19\nsource_id: 135\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e7280494-2111-4d8e-9b50-66aa1761341d	2015-03-04 17:25:58.7
401	198	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 46\ndisease_id: 19\nsource_id: 129\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	496a44d3-27fe-41b1-8c8b-37e9f41bb09c	2015-03-04 17:25:58.850799
402	199	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML\n  patients with FLT3 mutations\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 45\ndisease_id: 19\nsource_id: 128\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8685ac25-f93a-4830-a10a-4aa2120b75fe	2015-03-04 17:25:58.919206
403	200	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 19\nsource_id: 129\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	768b2b48-59ac-478d-b160-6b484a2cc87d	2015-03-04 17:25:58.983393
404	201	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were not significantly different in AML patients with FLT3 D835 mutations\n  compared to wild type\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 130\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8a9605e5-c491-424b-ba14-8db15646cc0d	2015-03-04 17:25:59.054487
405	202	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with D835 mutation of FLT3 did not have worse overall survival\n  than patients with wild type FLT3\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 130\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fab098e7-47ae-45c7-8486-56c70a28180f	2015-03-04 17:25:59.119331
406	203	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 136\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	490c60af-8c5f-4871-b3b7-ed71a3da4894	2015-03-04 17:25:59.182625
407	204	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients, FLT3-TKD mutation was associated with poorer disease free survival\n  compared to patients with wild type FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 137\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c4d2ada1-6ee0-43ed-b343-96b6462222cd	2015-03-04 17:25:59.249965
408	205	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated\n  with sorafenib resistance\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 9\ndisease_id: 19\nsource_id: 138\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9d6453cf-f68c-4bd5-a6d1-a153e09fe789	2015-03-04 17:25:59.335722
409	206	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeat mutations,\n  but not in patients with wild type or D835 mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 9\ndisease_id: 19\nsource_id: 135\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	810cb476-179c-40fd-958b-025ac0a5ce29	2015-03-04 17:25:59.407276
410	207	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 46\ndisease_id: 19\nsource_id: 129\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	511406c5-a953-4497-91d3-da075e68933d	2015-03-04 17:25:59.469446
411	208	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 47\ndisease_id: 25\nsource_id: 139\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0ef9a426-f275-4845-8c24-e80d1ba67f09	2015-03-04 17:25:59.546113
412	209	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 48\ndisease_id: 25\nsource_id: 139\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7a913243-5e59-4b5f-b8d6-df5096eea363	2015-03-04 17:25:59.604372
413	210	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In leukemia patients, IDH1 R132 mutation is associated with adults and normal\n  karyotype.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 140\nvariant_id: 57\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	14e62975-b216-42b7-ac65-a3b0a5217cde	2015-03-04 17:25:59.670063
414	211	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 141\nvariant_id: 57\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	31e16c01-64d4-4cb0-9a57-b32ce09c58e0	2015-03-04 17:25:59.824814
415	212	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The complete remission and overall survival rates in patients with IDH1 R132\n  mutation is not significantly different from those who do not have this mutation\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 142\nvariant_id: 57\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f948238f-fee1-4dc8-9e9f-3bb1e127754a	2015-03-04 17:25:59.883964
416	213	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic\n  for better survival compared to patients who harbor wild type mutation.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 26\nsource_id: 143\nvariant_id: 57\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	195df650-37d7-4a7a-b8a0-044edd5ff715	2015-03-04 17:25:59.950386
417	214	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with glioblastoma, those harboring IDH1 R132 mutation has higher\n  overall survival compared to those who do not have IDH1 mutation\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 27\nsource_id: 144\nvariant_id: 57\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	97f7d2ad-4b65-4d67-ab6f-335d1e64569f	2015-03-04 17:26:00.022982
418	215	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated\n  with worse overall survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 28\nsource_id: 145\nvariant_id: 57\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	93d36e73-24f7-4fe8-af64-dfc69b1d9c4f	2015-03-04 17:26:00.091682
419	216	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation is associated with patients of older age, high platelet count\n  during diagnosis, cytogenic normalcy and NPM1 mutation\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 146\nvariant_id: 58\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:209113113-209113113 (G->A)\n	1	\N	\N	00dfa545-5b81-4ad9-89e7-a7a8196825f4	2015-03-04 17:26:00.160395
420	217	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 146\nvariant_id: 58\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:209113113-209113113 (G->A)\n	1	\N	\N	4ccab1bc-9910-40e6-86ca-116a759cdc1d	2015-03-04 17:26:00.237214
421	218	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 146\nvariant_id: 59\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:209113112-209113112 (C->T)\n	1	\N	\N	d75bc2ff-a24a-4641-927d-5e44874ab9d6	2015-03-04 17:26:00.302041
422	219	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 146\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:209113112-209113112 (C->A)\n	1	\N	\N	32ef1d3f-f36c-48d3-a577-e323bfb6d3d8	2015-03-04 17:26:00.37241
423	220	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R140Q/L have event free survival and\n  overall survival similar to those with wild-type IDH2\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 147\nvariant_id: 61\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a5442db9-cd12-46bb-a740-0a62243e9e17	2015-03-04 17:26:00.450179
424	221	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact\n  overall survival or disease free survival.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 148\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bd5cb570-29d8-4572-ade0-ab11dc22c1f7	2015-03-04 17:26:00.510165
425	222	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with IDH2 R140K mutation have improved overall survival compared\n  to those with wild-type IDH2\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 149\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5127781f-cd65-40f9-9e0b-69e27b128de4	2015-03-04 17:26:00.57353
426	223	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L\n  do not confer prognostic value (overall survival)\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 29\nsource_id: 150\nvariant_id: 61\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8f59d50c-d008-407a-8c57-76e529c219f5	2015-03-04 17:26:00.645405
427	224	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: R140Q mutation in IDH2 does not have prognostic value in patients with MDS.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 29\nsource_id: 151\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9e8082c2-39d6-4beb-afdc-d2ca2976b65d	2015-03-04 17:26:00.701806
428	225	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R172K have event free survival and\n  overall survival similar to those with wild-type IDH2.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 147\nvariant_id: 62\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 15:90631838-90631838 (C->T)\n	1	\N	\N	e1662b51-c6c7-43f2-8e31-1e455f13b016	2015-03-04 17:26:00.764751
429	226	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with an IDH2 R172K mutation have worse overall survival compared\n  to those with wild-type IDH2\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 149\nvariant_id: 62\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 15:90631838-90631838 (C->T)\n	1	\N	\N	7f7a57fb-9e06-4984-a134-9c262a3d8108	2015-03-04 17:26:00.822937
430	227	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutations such as R172K\n  do not confer prognostic value (overall survival)\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 29\nsource_id: 150\nvariant_id: 62\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 15:90631838-90631838 (C->T)\n	1	\N	\N	24109bff-b44c-4765-b6ff-adc3e409edd6	2015-03-04 17:26:00.895127
431	228	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical\n  CML).\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 1\nsource_id: 152\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 9:5073770-5073770 (G->T)\n	1	\N	\N	97d019e5-2324-44cb-ab28-c7686e73d4e2	2015-03-04 17:26:00.959094
432	229	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is associated with myeloid neoplasms (AML, MDS, CMML/aCML)\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 1\nsource_id: 152\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 9:5073770-5073770 (G->T)\n	1	\N	\N	8da1e122-7115-4110-bb2e-626e465f94ad	2015-03-04 17:26:01.01934
433	230	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or\n  CLL).\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 30\nsource_id: 152\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 9:5073770-5073770 (G->T)\n	1	\N	\N	d1cd953f-2771-4e6b-9129-f9ef3fa9a61f	2015-03-04 17:26:01.087873
434	231	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The SNP rs10974944 in the Jak2 locus is associated with increased predisposition\n  for JAK2 V617F mutation and its associated cancer\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 31\nsource_id: 153\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 9:5073770-5073770 (G->T)\n	1	\N	\N	41b8376e-41b4-479b-b54e-fa18d90d461b	2015-03-04 17:26:01.155811
435	232	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction\n  in the percentage of cells harboring JAK2 V617F.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 49\ndisease_id: 32\nsource_id: 154\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 9:5073770-5073770 (G->T)\n	1	\N	\N	3dbd7263-bd9c-4249-ac70-3d97ccfdfbb6	2015-03-04 17:26:01.230759
436	233	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 50\ndisease_id: 32\nsource_id: 155\nvariant_id: 63\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 9:5073770-5073770 (G->T)\n	1	\N	\N	0ce00fe4-ece4-42b8-b183-4418bdd1cdd7	2015-03-04 17:26:01.295301
437	234	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In acute myloid leukemia patients, D816 mutation is associated with earlier\n  relapse and poorer prognosis than wildtype KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 33\nsource_id: 156\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55599321-55599321 (A->T)\n	1	\N	\N	3bf2abb6-43e3-494a-b235-8705622e3079	2015-03-04 17:26:01.358098
438	235	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 34\nsource_id: 157\nvariant_id: 65\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e95b8410-af44-4fae-8e1f-3f453181b933	2015-03-04 17:26:01.453301
439	236	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations were identified in morphologically benign, incidentally discovered\n  GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating\n  is not a prognostic marker for GIST\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 34\nsource_id: 158\nvariant_id: 65\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	81c156a2-9e5b-4fe8-a57d-2e293fcf299b	2015-03-04 17:26:01.531484
440	237	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between wildtype KIT or KIT mutations in\n  exon 9 or 11 in survival among GIST patients.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 34\nsource_id: 159\nvariant_id: 65\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4e6ec44b-5b96-431a-baf9-77420f347f9d	2015-03-04 17:26:01.602791
441	238	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with worse overall and cause-specific prognosis in\n  patients with GIST compared to patients with wildtype KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 34\nsource_id: 157\nvariant_id: 65\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d2c88356-5ed2-4566-a8b4-c20f93ee7eea	2015-03-04 17:26:01.679812
442	239	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 17\nsource_id: 160\nvariant_id: 65\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	559b2c06-a0c4-4127-9450-6b5d7b80bb8b	2015-03-04 17:26:01.754601
443	240	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 7\nsource_id: 161\nvariant_id: 65\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a5058193-cd26-403f-badd-d0e9f1fe35eb	2015-03-04 17:26:01.843166
444	241	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 7\nsource_id: 161\nvariant_id: 67\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6d5c5114-dc4f-4954-af48-07cfb3f925c6	2015-03-04 17:26:01.931743
445	242	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST patients with exon 14 KIT mutations had reduced overall survival compared\n  to patients wiltype for KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 34\nsource_id: 159\nvariant_id: 68\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5c95fbe2-131e-4dae-9601-d77b012240b2	2015-03-04 17:26:02.008627
446	243	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 34\nsource_id: 157\nvariant_id: 69\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	93f2c8b3-8ce6-4cd6-a916-7149eea6c9bc	2015-03-04 17:26:02.088949
447	244	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 7\nsource_id: 161\nvariant_id: 69\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	97f7cd1d-bd21-4ed9-91a3-af3007bf738e	2015-03-04 17:26:02.173959
448	245	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between wildtype KIT or KIT mutations in\n  exon 9 or 11 in survival among GIST patients.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 34\nsource_id: 159\nvariant_id: 70\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8b01907f-d95f-4d29-b2af-9223906661ab	2015-03-04 17:26:02.268051
449	246	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 17\nsource_id: 160\nvariant_id: 70\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2b5faf23-15e8-441d-86d2-1fc759e635cc	2015-03-04 17:26:02.34423
450	247	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a case study, a patient with anal melanoma harboring a 7 codon duplication\n  in the juxtamambrane region (exon 11) of KIT showed marked response 4 months after\n  imatinib treatment.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 35\nsource_id: 162\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	72ea71a7-0309-41f7-87bf-61bc3bfdfaba	2015-03-04 17:26:02.425319
451	248	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive\n  to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation\n  of KIT but did not alter the binding affinity of dasatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 163\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b65a8520-7a85-459d-811b-1abbdbaddd60	2015-03-04 17:26:02.500067
452	249	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 7\nsource_id: 163\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	80f22c26-6bc1-4405-921e-b8b4f31f9ff8	2015-03-04 17:26:02.597241
453	250	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 7\nsource_id: 163\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	da4ea6b3-5880-436a-b1b3-041f3c20572c	2015-03-04 17:26:02.669123
454	251	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 9\ndisease_id: 7\nsource_id: 163\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1a3ca7a9-f5b1-43e6-9387-42ce214fd71d	2015-03-04 17:26:02.725526
455	252	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 164\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b7b8fc52-54b8-4cfe-ad81-90e6424a3060	2015-03-04 17:26:02.807759
456	253	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 3\nsource_id: 164\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	35b1ff82-d4cb-400b-a458-4d8916849d9b	2015-03-04 17:26:02.871058
457	254	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 164\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1f10fa3f-1adf-42a2-bf05-0b80520abba4	2015-03-04 17:26:02.928547
458	255	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT V654A results in imatinib resistance in GIST patient-derived cell lines.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 34\nsource_id: 165\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55594258-55594258 (T->C)\n	1	\N	\N	ca014e13-c0d8-4871-a47a-d8895e7a9341	2015-03-04 17:26:03.016552
459	256	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective\n  for those that are refractory to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 51\ndisease_id: 34\nsource_id: 166\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55594258-55594258 (T->C)\n	1	\N	\N	75991319-a9b8-4433-aa41-b87934881b21	2015-03-04 17:26:03.096777
460	257	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer who harbor KRAS mutation have low response rate\n  to cetuximab\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 13\ndisease_id: 6\nsource_id: 167\nvariant_id: 73\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	31966c6d-33a8-42db-acbd-b9615c65a72a	2015-03-04 17:26:03.175006
461	258	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: Sensitivity\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 52\ndisease_id: 6\nsource_id: 168\nvariant_id: 74\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	55f4d7cb-18b4-411d-baa9-2df3582cebfc	2015-03-04 17:26:03.30225
462	259	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, Exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 31\ndisease_id: 3\nsource_id: 169\nvariant_id: 74\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	29356985-6e8f-4a9e-85af-12ccb81cc802	2015-03-04 17:26:03.38155
463	260	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, Exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 3\nsource_id: 169\nvariant_id: 74\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4d562cae-a545-45ae-93a9-e75ba00de5f0	2015-03-04 17:26:03.445609
464	261	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 36\nsource_id: 170\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e95b11a4-6925-414d-aaa6-9a27b106577b	2015-03-04 17:26:03.522281
465	262	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12\n  mutation did not impact overall or disease free survival.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 8\nsource_id: 171\nvariant_id: 75\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7801c797-dc2d-4afe-851d-3d3cab46a395	2015-03-04 17:26:03.593621
466	263	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 37\nsource_id: 172\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	31742dc9-0d83-455a-80b9-5a29b3d394b2	2015-03-04 17:26:03.67218
467	264	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung cancer patients are not associated with smoking history,\n  age or gender\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 173\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	287a3b8b-7cb0-4a20-8be1-e6f1cf09258a	2015-03-04 17:26:03.740846
468	265	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no strong association between KRAS mutation status and overall or progression\n  free survival in patients with NSCLC\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 174\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b0381911-238a-4a08-a4c2-82a78880e334	2015-03-04 17:26:03.816037
469	266	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A meta-analysis showed KRAS mutation is associated with worse outcome in patients\n  with NSCLC\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 175\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1b3a5e2a-d732-47a3-8f39-61461e47eb4a	2015-03-04 17:26:03.897175
470	267	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations\n  (primarily G12/G13) had lower response and disease control rates and shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype KRAS\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 13\ndisease_id: 6\nsource_id: 70\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0d05de35-995e-4eeb-b592-bcc677fa57b1	2015-03-04 17:26:03.978036
471	268	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with lack of response to cetuximab\n  in patients with advanced colorectal cancer\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 6\nsource_id: 176\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6065a2a2-b5ad-4946-b7fa-451b5d1542c3	2015-03-04 17:26:04.05404
472	269	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with reduced progression-free and\n  overall survival in patients with advanced colorectal cancer\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 6\nsource_id: 176\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	00591d62-94d3-40f4-971b-c4229d76af4b	2015-03-04 17:26:04.124093
473	270	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients\n  with lung adenocarcinoma.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 31\ndisease_id: 38\nsource_id: 177\nvariant_id: 146\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	aec4aa6c-df87-4611-be7a-0749301076f2	2015-03-04 17:26:04.202565
474	271	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients\n  with lung adenocarcinoma.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 38\nsource_id: 177\nvariant_id: 146\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	46093d73-1bde-4431-98be-357b19d7bc1c	2015-03-04 17:26:04.270205
476	273	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients\n  with lung adenocarcinoma.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 31\ndisease_id: 38\nsource_id: 177\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	753c17f5-1772-42ce-ab00-2d2dd43315cd	2015-03-04 17:26:04.442377
477	274	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients\n  with lung adenocarcinoma.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 38\nsource_id: 177\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d0e6bd8e-3369-437e-8247-7222814edcf6	2015-03-04 17:26:04.511696
478	275	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12D mutation occurs in never smokers significantly more often than in\n  smokers\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 17\nsource_id: 178\nvariant_id: 78\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398284-25398284 (C->T)\n	1	\N	\N	26fadeab-1e1e-4352-b576-451b314af867	2015-03-04 17:26:04.598211
479	276	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 12\ndisease_id: 3\nsource_id: 179\nvariant_id: 78\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398284-25398284 (C->T)\n	1	\N	\N	0338f6a1-b0cb-4fea-a9d2-5683ed9e5f58	2015-03-04 17:26:04.682436
480	277	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 53\ndisease_id: 3\nsource_id: 180\nvariant_id: 78\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398284-25398284 (C->T)\n	1	\N	\N	7f0fc6ba-b1eb-4bc9-8fbd-df00475b7bfd	2015-03-04 17:26:04.769008
481	278	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 54\ndisease_id: 3\nsource_id: 180\nvariant_id: 78\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398284-25398284 (C->T)\n	1	\N	\N	403b90c8-a5fd-4880-8a79-da1b798f4049	2015-03-04 17:26:04.836166
482	279	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients\n  with lung adenocarcinoma.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 31\ndisease_id: 38\nsource_id: 177\nvariant_id: 78\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	40ed6f30-25ec-4849-9b8d-762d3d441d99	2015-03-04 17:26:04.910794
483	280	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients\n  with lung adenocarcinoma.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 38\nsource_id: 177\nvariant_id: 78\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0e4b31a7-ff57-45d0-9e50-77a10e1116fa	2015-03-04 17:26:04.980925
484	281	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients\n  with lung adenocarcinoma.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 31\ndisease_id: 38\nsource_id: 177\nvariant_id: 145\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	99d76990-e6b4-4d05-bbc0-43b000dc6007	2015-03-04 17:26:05.12616
485	282	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients\n  with lung adenocarcinoma.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 38\nsource_id: 177\nvariant_id: 145\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f394bf99-97cf-49a8-b22d-fe9152d74312	2015-03-04 17:26:05.205328
486	283	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 36\nsource_id: 170\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	04a15f53-ad46-4e26-bbea-6e2ac53d8ada	2015-03-04 17:26:05.287624
487	284	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 39\nsource_id: 172\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	50a4c68a-c8aa-4d03-9c4c-3119fa1cbc63	2015-03-04 17:26:05.380458
488	285	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung cancer patients are not associated with smoking history,\n  age or gender\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 173\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c1866fb8-0b1e-45fd-95b9-b15ea4f37ed6	2015-03-04 17:26:05.469483
489	286	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft model\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 181\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398281-25398281 (C->T)\n	1	\N	\N	a8a864f9-013e-482c-b98a-b2eae9e31ea1	2015-03-04 17:26:05.554722
490	287	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft model\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 181\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398281-25398281 (C->T)\n	1	\N	\N	faa1a3b1-5ac8-4a6a-9171-0016d2906495	2015-03-04 17:26:05.626381
491	288	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 13\ndisease_id: 8\nsource_id: 181\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398281-25398281 (C->T)\n	1	\N	\N	a7823582-320f-43ea-a2c2-2ca2b8cdb3aa	2015-03-04 17:26:05.704825
492	289	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 13\ndisease_id: 8\nsource_id: 181\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 12:25398281-25398281 (C->T)\n	1	\N	\N	5bfb9bc9-b987-4cde-be9e-76963842b746	2015-03-04 17:26:05.78652
493	290	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition\n  by about 5 fold.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 22\ndisease_id: 7\nsource_id: 182\nvariant_id: 81\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 15:66729162-66729162 (C->T)\n	1	\N	\N	a868dde8-eb42-48c4-955f-a4a292b5fc74	2015-03-04 17:26:05.865147
494	291	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 22\ndisease_id: 7\nsource_id: 183\nvariant_id: 82\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 15:66727451-66727451 (A->C)\n	1	\N	\N	f2bd7625-8282-4be0-bb55-3de62f69ff77	2015-03-04 17:26:05.942579
495	292	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 20\nsource_id: 100\nvariant_id: 83\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8b0d0547-6216-44ba-bbbd-8504da49d4f4	2015-03-04 17:26:06.020697
496	293	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 27\ndisease_id: 20\nsource_id: 100\nvariant_id: 83\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	08d69039-9103-4a5a-a97f-28ce10de79cf	2015-03-04 17:26:06.086748
497	294	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804),\n  MET amplification and EGFR T790M were found to be the primary drivers of resistance.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 3\nsource_id: 120\nvariant_id: 144\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	83a59c7a-8423-44c0-89a8-22807b75efb7	2015-03-04 17:26:06.163215
498	295	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation has been shown to confer sensitivity to alkylating\n  agents, such as carmustine, in glioblastoma cells.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 55\ndisease_id: 40\nsource_id: 184\nvariant_id: 84\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6ce1a90d-3efb-4d6d-b98a-9ac11dcd4b44	2015-03-04 17:26:06.251584
499	296	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation was associated with increased overall and disease-free\n  survival as well as tumor regression in patients with gliomas.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 40\nsource_id: 184\nvariant_id: 84\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a3c67dfe-f34c-4883-b7c7-8da5ff7353ac	2015-03-04 17:26:06.322386
500	297	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor,\n  such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may\n  be useful in treating patients with initial resistance to alkylating agents.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 56\ndisease_id: 40\nsource_id: 184\nvariant_id: 84\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	40298212-c045-4917-a947-34a1cf9dc4b4	2015-03-04 17:26:06.394834
501	298	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted\n  from temozolomide. This benefit was also methylation status dependent, as those\n  without methylation did not see increased survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 21\ndisease_id: 40\nsource_id: 185\nvariant_id: 84\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8dd3c8b0-fb8b-4df2-b835-6841e4dcaf9b	2015-03-04 17:26:06.467234
502	299	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NOTCH1 gain-of-function mutations, including D1643H, have shown to be correlated\n  to poor prognosis in lung cancer. This is in a TP53 wild-type context.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 186\nvariant_id: 133\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f2360f5c-0fe1-4d73-9580-cb164c26a289	2015-03-04 17:26:06.548244
503	300	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NOTCH1 gain-of-function mutations, including R2328W, have shown to be correlated\n  to poor prognosis in lung cancer. This is in a TP53 wild-type context.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 186\nvariant_id: 134\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7e15cb54-6624-4b1a-b26f-8651fba99fbe	2015-03-04 17:26:06.631206
504	301	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NOTCH1 gain-of-function mutations, including S2275fs, have shown to be correlated\n  to poor prognosis in lung cancer. This is in a TP53 wild-type context.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 186\nvariant_id: 136\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	07f4d5b2-0e91-40b1-b6e5-b7b3350f74ae	2015-03-04 17:26:06.694476
505	302	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NOTCH1 gain-of-function mutations, including V2444fs, have shown to be correlated\n  to poor prognosis in lung cancer. This is in a TP53 wild-type context.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 3\nsource_id: 186\nvariant_id: 135\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1ad0b4b8-bf22-498d-9de0-0840c8d6f488	2015-03-04 17:26:06.762833
506	303	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  NPM1 mutation regardless of FLT3-ITD status\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 44\ndisease_id: 19\nsource_id: 127\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	96a47591-6d85-473d-8777-58c6188913bb	2015-03-04 17:26:06.828944
507	304	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment improved overall and relapse-free survival in older (>60) patients\n  with mutant NPM1 and without FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 44\ndisease_id: 19\nsource_id: 187\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b9eccad8-6b3f-40e4-8903-d0d40c1f95ac	2015-03-04 17:26:06.946463
508	305	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young patients (18-60) with NPM1 mutations had improved overall survival following\n  high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However,\n  the adjusted p-value for multiple testing is 0.11.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 29\ndisease_id: 19\nsource_id: 188\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	935d3202-0978-40a8-9166-20010685c27c	2015-03-04 17:26:07.022267
509	306	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype\n  (0/55).\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 189\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5bcbdb6b-b281-42cc-88dd-5dbbc7676b8a	2015-03-04 17:26:07.095717
510	307	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid\n  leukemia.\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 190\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6ab614d6-5c5b-4c42-8fdb-cfc5657f0f17	2015-03-04 17:26:07.167154
511	308	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML with mutated NPM1 is a provisional entity in WHO classification of acute\n  myeloid leukemia (AML). This mutation should be tested for in clinical trials and\n  is recommended for testing in patients with cytogentically normal AML.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 19\nsource_id: 191\nvariant_id: 85\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	319d1059-92ca-436b-87b3-9b5aa0ef8a86	2015-03-04 17:26:07.235995
512	309	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 131\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	dd2491a6-432b-4d96-8af7-c7477cc13aaf	2015-03-04 17:26:07.300948
513	310	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid\n  leukemia.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 190\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9a4aa6c9-6b73-448d-843a-a85f3c3395b7	2015-03-04 17:26:07.369728
514	311	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid\n  leukemia\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 189\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7c7b502f-e812-4062-b06d-b63dc4b6f019	2015-03-04 17:26:07.443158
515	312	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with normal karyotype in older (>60) patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 187\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6e0ae6b7-f0f7-489a-9101-3e06e39f99a0	2015-03-04 17:26:07.522882
516	313	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 and CD123 expression was significantly increased in patients with NPM1\n  mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33\n  and anti-CD123 therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 19\nsource_id: 192\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fc925e7d-9e03-48f3-8c72-e9cf6046ecb8	2015-03-04 17:26:07.602846
517	314	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 expression was significantly increased in patients with NPM1 mutation with\n  or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 19\nsource_id: 192\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	295d1855-a40a-46fc-a3cb-14fb21e72df2	2015-03-04 17:26:07.685383
518	315	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and\n  increased overall survival in young adult acute myeloid leukemia patients (median\n  age 43)\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 131\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9a6c82d6-4280-40d0-afcd-77b3875d81fb	2015-03-04 17:26:07.772339
519	316	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations were associated with improved complete remission\n  rates as well as overall and disease free survival when all ages and karyotype were\n  analyzed; however, when only normal karyotype patients were analyzed, only complete\n  remission rates were improved\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 187\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d3295fef-e695-4cd3-95cd-34be23d7d0c0	2015-03-04 17:26:07.847343
520	317	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 107\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ac7f06fa-62d2-4458-81fd-f8f947e568bc	2015-03-04 17:26:07.925084
521	318	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 141\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6544e339-6568-4473-ba4b-bf7256c52a16	2015-03-04 17:26:08.006436
522	319	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and both disease-free and overall survival\n  were longer for patients with Exon 12 NPM1 mutations.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 189\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e0808c7a-9b65-45cc-a98d-22989183dff6	2015-03-04 17:26:08.079886
523	320	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and event-free survival was longer for\n  normal karyotype AML patients with Exon 12 NPM1 mutations.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 190\nvariant_id: 85\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4c4236e7-4668-4003-83df-1c1e81006397	2015-03-04 17:26:08.156088
524	321	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cytoplasmic localization of NPM in AML patients was an independent prognositic\n  factor associated with complete remission in response to induction therapy in a\n  multivariate analysis (p=0.019) but not in a univariate analysis (p=0.07).\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 193\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	408f8533-67bd-4f90-9072-afb2fae8b040	2015-03-04 17:26:08.226961
525	322	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 136\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b4b23d3a-fc6e-49e4-8040-5805170989cc	2015-03-04 17:26:08.304125
526	323	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Intermediate risk patients with mutant NPM1 had improved overall survival with\n  the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1\n  and IDH2\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 188\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d92f9410-2f2a-4231-855b-8a371ddf1049	2015-03-04 17:26:08.376911
527	324	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved\n  overall and relapse-free survival, complete remission rates and remission duration\n  compared to all other combinations of mutants\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 194\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6f2dcff7-5423-4e05-8cbe-5b133d7258aa	2015-03-04 17:26:08.45645
528	325	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with higher complete remission rates, lower cumulative\n  incidence of relapse and higher overall survival in intermediate risk AML patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 195\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	01feee6e-82f7-4174-a2a5-24a1ca2c39de	2015-03-04 17:26:08.531077
529	326	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with increased overall, event-free and relapse-free\n  survival and remission after induction therapy in the entire cohort and in the cytogenetically\n  normal subset of patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 196\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ede69e28-ffaa-4c3f-a8a3-0d95b6101e24	2015-03-04 17:26:08.649836
530	327	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was significantly associated with complete remission\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 98\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0ad181bc-8008-457b-b1da-b2b88921aba5	2015-03-04 17:26:08.725569
531	328	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with increased complete remission rates as well\n  as longer overall disease free survival in normal karyotype AML patients >59 years\n  old, particularly in those >69 years old\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 197\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	061b3739-c730-41c7-8fa0-dfba0ca5fb8a	2015-03-04 17:26:08.798472
532	329	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and\n  without FLT3-ITD have increased overall survival, relapse-free survival and response\n  to induction chemotherapy\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 198\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ddb6eec3-0bde-4872-aef2-b15a72e09918	2015-03-04 17:26:08.870477
533	330	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coocurrence of FLT3-ITD mutations significantly reduced overall survival of\n  normal karyotype AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 189\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5d438c64-720b-417f-8255-02db20ba4c84	2015-03-04 17:26:08.943431
534	331	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse\n  in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction\n  in NPM1 levels following first line therapy (N=15)\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 199\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b9c05ea6-253e-4a69-8665-39f037098be4	2015-03-04 17:26:09.016415
535	332	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 132\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ec1eddc2-19db-4015-a3ca-9505b068b9ef	2015-03-04 17:26:09.090801
536	333	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival\n  and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate\n  risk AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 195\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e658ca89-166e-441b-894f-7ab8588dee06	2015-03-04 17:26:09.158599
537	334	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TET2 mutation reduces overall survival of normal karyotype patients with either\n  an NPM1 mutation or NPM1 mutation without FLT3-ITD\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 200\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ce0ce875-aee7-42ab-be27-2726c9754a6e	2015-03-04 17:26:09.237512
538	335	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 132\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3ec91b55-04d4-4895-a774-71ba25457527	2015-03-04 17:26:09.311795
539	336	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and\n  FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 57\ndisease_id: 19\nsource_id: 201\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	76e65066-3a37-4192-ae48-597938e9a4f8	2015-03-04 17:26:09.392171
540	337	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and\n  FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 44\ndisease_id: 19\nsource_id: 201\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	51e4f748-3978-4796-a698-ac5ba2cf2660	2015-03-04 17:26:09.449446
541	338	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: For patients with NPM1 mutation that achieved CR following induction therapy,\n  relapse-free survival was improved following treatment with valproic acid\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 58\ndisease_id: 19\nsource_id: 202\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	55d61110-9846-45f0-93f6-414cfc368fab	2015-03-04 17:26:09.528325
542	339	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No NPM1 mutations were identified in patients with favorable risk cytogenetics\n  (79/215 patients)\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 105\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\n	1	\N	\N	949fd9a9-edae-4c78-9008-c0170872ba2a	2015-03-04 17:26:09.607276
543	340	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with intermediate risk cytogenetics (including\n  normal karyotype)\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 105\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\n	1	\N	\N	b21af8fc-d1b2-40e3-b06d-cbc8481dc04a	2015-03-04 17:26:09.692556
544	341	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected\n  into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM\n  in AML patients was associated with good response to induction therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 19\nsource_id: 193\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\n	1	\N	\N	269d82be-078c-477d-82ff-a309037e64f3	2015-03-04 17:26:09.7787
545	342	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 57\ndisease_id: 19\nsource_id: 201\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\n	1	\N	\N	eed8e069-10de-4862-b76a-bb499a8e80dc	2015-03-04 17:26:09.866893
546	343	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 7\nsource_id: 203\nvariant_id: 87\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	89f42ae0-a43a-4876-8087-5569e75a3f19	2015-03-04 17:26:09.946996
547	344	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of NRAS mutation in AML patients does not impact diease prognosis\n  (resistant disease, disease-free survival, complete remission rate, relapse rate,\n  induction death)\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 204\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d69c452c-e609-4842-9d28-d2f00da4f41d	2015-03-04 17:26:10.019721
548	345	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 6\nsource_id: 205\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	31d8aeb3-8fca-4a0e-bce5-6543da290313	2015-03-04 17:26:10.094807
549	346	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 7\nsource_id: 203\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5f5522ce-adbf-42b5-bac4-30dd71a01d18	2015-03-04 17:26:10.162219
550	347	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 6\nsource_id: 205\nvariant_id: 89\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7b5db3c0-ec65-40b6-8055-4f88717eb241	2015-03-04 17:26:10.237125
551	348	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 6\nsource_id: 205\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a56142ef-eb17-4d4e-b40b-0cb2bea9dce4	2015-03-04 17:26:10.358088
552	349	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NRAS mutation status does not impact overall, event-free or disease-free survival\n  in patients with AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 206\nvariant_id: 91\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b602e724-b0c0-4f84-bc14-399337946ca5	2015-03-04 17:26:10.433667
553	350	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Likely due to increased reliance of mutant NRAS on HSP90 for stabilization,\n  inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic\n  malignant melanoma with an NRAS G13D mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 59\ndisease_id: 7\nsource_id: 207\nvariant_id: 92\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 1:115258744-115258744 (C->T)\n	1	\N	\N	dd40764a-e848-410e-b76e-d157b6a60d96	2015-03-04 17:26:10.5119
554	351	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status\n  (primarily Q61) was not informative for disease control rate, progression free survival\n  or overall survival following cetuximab plus chemotherapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 13\ndisease_id: 6\nsource_id: 70\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	46f67ba2-5084-4736-b906-f281629a57eb	2015-03-04 17:26:10.588882
555	352	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation\n  (primarily Q61) have a significantly lower response rate to cetuximab than patients\n  wildtype for NRAS.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 13\ndisease_id: 6\nsource_id: 70\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f45ec9c6-d0ca-422f-a14c-62ab076979eb	2015-03-04 17:26:10.662405
556	353	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma associated with NRAS Q61 mutation was more often associated with those\n  at the extremity than those at the trunk\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 7\nsource_id: 208\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	cc70f817-37a6-42b5-9564-76ac4b672e33	2015-03-04 17:26:10.739655
557	354	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 18\ndisease_id: 7\nsource_id: 209\nvariant_id: 93\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b92b6702-4d08-45a1-8128-f45836e9443e	2015-03-04 17:26:10.808555
558	355	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted\n  in disease free survival of 14 months\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 21\ndisease_id: 7\nsource_id: 210\nvariant_id: 94\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 1:115256528-115256529 (TT->CA)\n	1	\N	\N	97c17005-ab00-4b86-9414-c1f3b37dd6d1	2015-03-04 17:26:10.893469
559	356	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with Q61R mutation, treatment with temozolomide resulted\n  in overall survival of 16 months\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 21\ndisease_id: 7\nsource_id: 210\nvariant_id: 95\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 1:115256528-115256529 (TT->CC)\n	1	\N	\N	143c0791-7d49-4b71-83bc-1f422ad2e986	2015-03-04 17:26:10.96552
560	357	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 60\ndisease_id: 41\nsource_id: 211\nvariant_id: 96\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55152092-55152093 (GA->AT)\n	1	\N	\N	13384050-56fe-47ac-a178-4796374302aa	2015-03-04 17:26:11.05781
561	358	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 60\ndisease_id: 41\nsource_id: 211\nvariant_id: 97\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55152093-55152093 (A->T)\n	1	\N	\N	1a573500-a53a-4100-a8fa-1250ba62c7e9	2015-03-04 17:26:11.134657
562	359	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST cancer with D842V mutation is resistant to imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 41\nsource_id: 212\nvariant_id: 97\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55152093-55152093 (A->T)\n	1	\N	\N	3237ce38-24f0-4cd2-8556-f533a97ea9de	2015-03-04 17:26:11.215216
563	360	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib,\n  double mutation of V561D and D842V mutants are resistant to imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 41\nsource_id: 165\nvariant_id: 97\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55152093-55152093 (A->T)\n	1	\N	\N	e648642d-9e8d-4164-a3bb-3fa788e99968	2015-03-04 17:26:11.29607
564	361	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST tumors harboring PDGFRA D842V mutation are more likely to be benign than\n  malignant.\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 41\nsource_id: 213\nvariant_id: 97\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55152093-55152093 (A->T)\n	1	\N	\N	d5a9d405-c49b-4875-bef5-da6c464c26d8	2015-03-04 17:26:11.380101
565	362	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 60\ndisease_id: 41\nsource_id: 211\nvariant_id: 98\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 4:55152092-55152092 (G->T)\n	1	\N	\N	63e3818f-3d2c-491a-aa40-7d4d48679f37	2015-03-04 17:26:11.459072
566	363	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib\n  resistance, crenolanib was significantly more potent at inhibiting kinase activity\n  than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 60\ndisease_id: 41\nsource_id: 211\nvariant_id: 99\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f74279db-266c-4c5a-9156-ee2e9a88751b	2015-03-04 17:26:11.538128
567	364	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 60\ndisease_id: 41\nsource_id: 211\nvariant_id: 100\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f83292a6-7883-4e05-9824-78cb204985e5	2015-03-04 17:26:11.621289
568	365	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 61\ndisease_id: 11\nsource_id: 214\nvariant_id: 101\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936082-178936082 (G->A)\n	1	\N	\N	56333cac-bf7b-47c7-90d2-3f865837feda	2015-03-04 17:26:11.706233
569	366	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 62\ndisease_id: 11\nsource_id: 215\nvariant_id: 101\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936082-178936082 (G->A)\n	1	\N	\N	1da41516-7d75-4edd-bd5b-93e4b616ee69	2015-03-04 17:26:11.791836
570	367	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 13\ndisease_id: 6\nsource_id: 216\nvariant_id: 101\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936082-178936082 (G->A)\n	1	\N	\N	bb76e475-2faa-4011-b037-bdfe80cd31a6	2015-03-04 17:26:11.871884
571	368	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 11\ndisease_id: 6\nsource_id: 216\nvariant_id: 101\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936082-178936082 (G->A)\n	1	\N	\N	8e02fbc6-d345-4bec-bf1e-b136cd9d2f8b	2015-03-04 17:26:11.932984
572	369	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 6\nsource_id: 217\nvariant_id: 101\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936082-178936082 (G->A)\n	1	\N	\N	1dd531bf-cbf4-4f07-9771-68d629a7bcab	2015-03-04 17:26:12.054483
573	370	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E542K) or exon 20 mutations were associated\n  with poorer survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 6\nsource_id: 216\nvariant_id: 101\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936082-178936082 (G->A)\n	1	\N	\N	e25289b9-29a3-46ea-b031-4a037c3ef0f9	2015-03-04 17:26:12.123407
574	371	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 52\ndisease_id: 6\nsource_id: 168\nvariant_id: 101\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936082-178936082 (G->A)\n	1	\N	\N	fdc3856b-e14e-4fa5-8f55-0b355726368d	2015-03-04 17:26:12.195071
575	372	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 61\ndisease_id: 11\nsource_id: 214\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936091-178936091 (G->A)\n	1	\N	\N	a404d17e-df0c-4c6e-a665-b2472d09efa2	2015-03-04 17:26:12.264057
576	373	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 62\ndisease_id: 11\nsource_id: 215\nvariant_id: 102\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936091-178936091 (G->A)\n	1	\N	\N	8fcd6835-a7f2-4512-8b8c-63f13f6b730b	2015-03-04 17:26:12.333692
577	374	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 13\ndisease_id: 6\nsource_id: 216\nvariant_id: 102\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936091-178936091 (G->A)\n	1	\N	\N	40d16225-b1b0-4e01-bfab-e98e3389d144	2015-03-04 17:26:12.406776
578	375	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 11\ndisease_id: 6\nsource_id: 216\nvariant_id: 102\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936091-178936091 (G->A)\n	1	\N	\N	1f461f2d-bff1-43c5-9b9f-d6273ddb4bf8	2015-03-04 17:26:12.46583
579	376	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 6\nsource_id: 217\nvariant_id: 102\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936091-178936091 (G->A)\n	1	\N	\N	1e37cb23-24fe-439f-bfcc-07357b099f0c	2015-03-04 17:26:12.532054
580	377	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 6\nsource_id: 216\nvariant_id: 102\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936091-178936091 (G->A)\n	1	\N	\N	1ae5628c-5ffe-4085-b2ab-d4ca6bc3196e	2015-03-04 17:26:12.599837
581	378	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 52\ndisease_id: 6\nsource_id: 168\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178936091-178936091 (G->A)\n	1	\N	\N	909984f4-dfd3-4606-aec3-b9c215522f12	2015-03-04 17:26:12.664791
582	379	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA\n  mutations had lower response and disease control rates as well as shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype PIK3CA\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 13\ndisease_id: 6\nsource_id: 70\nvariant_id: 103\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0c0ef2ad-2761-4ab6-89ad-ca137fc5265b	2015-03-04 17:26:12.730643
583	380	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations\n  (Exon 9) had no significant impact on response rate, disease control rate, progression\n  free surivival or overall survival following cetuximab plus chemotherapy than those\n  with wildtype PIK3CA\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 13\ndisease_id: 6\nsource_id: 70\nvariant_id: 104\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5d3cf85a-12d8-4b8d-95bf-2c475e583d30	2015-03-04 17:26:12.797041
584	381	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with H1047R mutation showed increased sensitivity to\n  CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 61\ndisease_id: 11\nsource_id: 214\nvariant_id: 105\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178952085-178952085 (A->G)\n	1	\N	\N	f9b40472-1d89-4f90-b4ff-4e46aa9ff33f	2015-03-04 17:26:12.877011
585	382	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 62\ndisease_id: 11\nsource_id: 215\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178952085-178952085 (A->G)\n	1	\N	\N	76b3665e-aace-43b8-819c-82dd7de99d72	2015-03-04 17:26:12.945904
586	383	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 13\ndisease_id: 6\nsource_id: 216\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178952085-178952085 (A->G)\n	1	\N	\N	0494060f-eb84-4fc6-8157-e79f9aed4a45	2015-03-04 17:26:13.019317
587	384	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 11\ndisease_id: 6\nsource_id: 216\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178952085-178952085 (A->G)\n	1	\N	\N	c46f35ad-eede-4b26-b1d4-1b7b6d67506b	2015-03-04 17:26:13.073928
588	385	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 6\nsource_id: 216\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178952085-178952085 (A->G)\n	1	\N	\N	6bda570a-3b24-4796-9aa3-03158c07d282	2015-03-04 17:26:13.147728
589	386	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 52\ndisease_id: 6\nsource_id: 168\nvariant_id: 105\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 3:178952085-178952085 (A->G)\n	1	\N	\N	61942b3b-0a6f-423d-bd53-9460a64c6da8	2015-03-04 17:26:13.222361
590	387	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic\n  cells at pharmalogical concentrations.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 44\ndisease_id: 42\nsource_id: 218\nvariant_id: 106\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c887a3ce-a067-4aa7-a0f3-be044ae086ff	2015-03-04 17:26:13.293817
591	388	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 21\nsource_id: 102\nvariant_id: 107\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f2a6ce82-cefa-4f74-aa13-b5538383368b	2015-03-04 17:26:13.369484
592	389	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse\n  to PI3K-mTOR inhibitors.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 2\nsource_id: 219\nvariant_id: 108\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 10:89717672-89717672 (C->T)\n	1	\N	\N	56ad6538-2e77-4bb4-b08d-727cbb2d1021	2015-03-04 17:26:13.450669
593	390	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PTEN nonsense mutations, including R233*, have been shown to be inactivating\n  and loss-of-function, but do not have prognostic value in glioblastoma multiforme\n  patients.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 40\nsource_id: 220\nvariant_id: 108\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 10:89717672-89717672 (C->T)\n	1	\N	\N	0bb0e1ac-d7d8-4564-99b3-2a44fc424a16	2015-03-04 17:26:13.534263
594	391	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic\n  cells at pharmalogical concentrations.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 44\ndisease_id: 42\nsource_id: 218\nvariant_id: 109\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9da2f9fb-7746-4c04-8a1e-f8abff23606d	2015-03-04 17:26:13.652882
595	392	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 63\ndisease_id: 43\nsource_id: 221\nvariant_id: 110\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 10:43609950-43609950 (C->G)\n	1	\N	\N	9f11f4c8-ae1a-40c1-96f5-d10e709508e7	2015-03-04 17:26:13.728848
596	393	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong\n  repression of tyroid cancer cell growth.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 64\ndisease_id: 43\nsource_id: 222\nvariant_id: 111\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 10:43617416-43617416 (T->C)\n	1	\N	\N	d1a61ccf-1d7d-48c3-88c3-d47ec2916398	2015-03-04 17:26:13.818129
597	394	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 63\ndisease_id: 43\nsource_id: 221\nvariant_id: 111\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 10:43617416-43617416 (T->C)\n	1	\N	\N	ebfbec01-2956-417c-8bd9-063198792f43	2015-03-04 17:26:13.890113
598	395	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with medullary carcinoma, the presence of RET M918T mutation is\n  associated with increased probability of lymph node metastases.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 43\nsource_id: 223\nvariant_id: 111\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 10:43617416-43617416 (T->C)\n	1	\N	\N	934c2b2b-2db5-43e9-b9d0-4a5790672bfd	2015-03-04 17:26:13.968145
599	396	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Compared to those who harbor a wild type RET, patients with RET M918T mutation\n  develop larger and more aggressive medullary thyroid cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 43\nsource_id: 224\nvariant_id: 111\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 10:43617416-43617416 (T->C)\n	1	\N	\N	aff8e3d8-efcb-4b7d-8bd8-df0674ff99e5	2015-03-04 17:26:14.050492
600	397	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 29\nsource_id: 225\nvariant_id: 112\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:198267359-198267359 (C->A)\n	1	\N	\N	1dfc9ab7-41fc-428a-8b15-bcd3eff61f6e	2015-03-04 17:26:14.122344
601	398	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 29\nsource_id: 225\nvariant_id: 113\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 2:198266834-198266834 (T->C)\n	1	\N	\N	30338a10-ad94-49a2-b93b-90b884a8032d	2015-03-04 17:26:14.198856
602	399	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin\n  than breast tumors with wild type TP53.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 65\ndisease_id: 11\nsource_id: 226\nvariant_id: 114\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7578406-7578406 (C->T)\n	1	\N	\N	57db3a75-ed8f-494c-bc78-39d9ab43c082	2015-03-04 17:26:14.278546
603	400	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R175H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 11\nsource_id: 227\nvariant_id: 114\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7578406-7578406 (C->T)\n	1	\N	\N	8efc501e-2dee-457c-8d97-246be6dab6b3	2015-03-04 17:26:14.359207
604	401	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R248Q mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 11\nsource_id: 227\nvariant_id: 115\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577538-7577538 (C->T)\n	1	\N	\N	d40db417-c546-4f1c-b829-1582dc4f54f7	2015-03-04 17:26:14.437787
605	402	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R248 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 11\nsource_id: 228\nvariant_id: 115\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577538-7577538 (C->T)\n	1	\N	\N	75bd4260-f339-4b8f-85e4-71ac6991d7a4	2015-03-04 17:26:14.51354
606	403	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring R248W mutation, the prognosis is worse than\n  any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 11\nsource_id: 227\nvariant_id: 116\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577539-7577539 (G->A)\n	1	\N	\N	990d3271-b25e-4391-946f-75604645f096	2015-03-04 17:26:14.58958
607	404	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved regions\n  such as R249 are prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 11\nsource_id: 228\nvariant_id: 117\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577535-7577535 (C->G)\n	1	\N	\N	3731bf8b-e3d6-4556-b759-6bb95b4ab37e	2015-03-04 17:26:14.665025
608	405	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved regions\n  such as R249 are prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 11\nsource_id: 228\nvariant_id: 118\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577536-7577536 (T->A)\n	1	\N	\N	82759533-db4e-47cd-9048-0d550cde1fe4	2015-03-04 17:26:14.741018
609	406	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273C mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 11\nsource_id: 227\nvariant_id: 119\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577121-7577121 (G->A)\n	1	\N	\N	e2906dfa-1c7e-4d27-9d8d-2d3c68be3096	2015-03-04 17:26:14.820862
610	407	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 11\nsource_id: 228\nvariant_id: 119\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577121-7577121 (G->A)\n	1	\N	\N	8e37fece-e7f0-4720-9b4d-f37de62205f2	2015-03-04 17:26:14.900051
611	408	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 11\nsource_id: 227\nvariant_id: 120\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577120-7577120 (C->T)\n	1	\N	\N	78b3280b-d207-41db-b8b3-4dd7689555ae	2015-03-04 17:26:15.021035
612	409	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 11\nsource_id: 228\nvariant_id: 120\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: 17:7577120-7577120 (C->T)\n	1	\N	\N	972b9fa4-7c97-45f5-acda-9c1d8b57f05c	2015-03-04 17:26:15.119479
613	410	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumor with R175H mutation are more responsive to doxorubicin than breast\n  tumors with wild type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 11\nsource_id: 228\nvariant_id: 121\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ebebe8e8-3750-4e46-b432-5ff32576e63b	2015-03-04 17:26:15.199015
614	411	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A patient with metastatic bladder cancer that responded well to the mTOR inhibitor\n  everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del,\n  p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations\n  also had mild responses to everolimus.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 66\ndisease_id: 44\nsource_id: 229\nvariant_id: 122\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	091713c9-13c3-492b-b048-38e3d1cb51f7	2015-03-04 17:26:15.279264
615	412	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice\n  with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit\n  increased overall survival, increased apoptosis, reduced proliferation and reduced\n  tumor burden compared to Tsc1 wildtype mice.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 62\ndisease_id: 3\nsource_id: 230\nvariant_id: 122\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c883bf3c-d371-4b46-821c-9bf1e03dac41	2015-03-04 17:26:15.416618
616	413	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 66\ndisease_id: 24\nsource_id: 231\nvariant_id: 123\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bc630876-3d24-47e6-82a0-08afd17e2ae5	2015-03-04 17:26:15.500163
617	414	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 66\ndisease_id: 24\nsource_id: 229\nvariant_id: 124\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2dc4fa57-714d-4099-9f00-6a99a12013cb	2015-03-04 17:26:15.586819
618	415	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 19\nsource_id: 232\nvariant_id: 125\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	51104384-3803-469e-aefe-6b9389088fc5	2015-03-04 17:26:15.665645
619	416	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 19\nsource_id: 232\nvariant_id: 125\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3a5cb3fb-5391-43f2-bb49-cbaa5e8c45e2	2015-03-04 17:26:15.744943
620	417	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 29\nsource_id: 232\nvariant_id: 125\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c13c6986-cf3d-4b39-a43e-ddf1c68acd8c	2015-03-04 17:26:15.81939
621	418	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R\n  is prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 29\nsource_id: 233\nvariant_id: 125\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f2c9b62b-92e9-415c-96f7-a7bcefbda138	2015-03-04 17:26:15.893443
622	419	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 19\nsource_id: 232\nvariant_id: 126\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4c75a05b-7000-4880-b022-7593572a194d	2015-03-04 17:26:15.972961
623	420	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 19\nsource_id: 232\nvariant_id: 126\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	250bd957-b220-4dd3-9d1f-145c277a0ca3	2015-03-04 17:26:16.037091
624	421	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 29\nsource_id: 232\nvariant_id: 126\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a6651334-94ae-478e-beb6-d3d123887b6a	2015-03-04 17:26:16.108564
625	422	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, the presence of U2AF mutations such as\n  S34Y/F are prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 1\ndisease_id: 29\nsource_id: 233\nvariant_id: 126\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	826fc88b-6987-42b9-a4dc-9727fab77b2f	2015-03-04 17:26:16.173692
626	423	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 234\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ec07e8d5-c6b0-4d3f-ac77-c42b8c4af27c	2015-03-04 17:26:16.240498
627	424	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No differences in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 235\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	564ce7f0-ab3f-4d8c-b701-2cf6f6f0a507	2015-03-04 17:26:16.313349
628	425	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 235\nvariant_id: 127\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2b5dbc67-8d3e-44f0-97f6-1d9f02989bd0	2015-03-04 17:26:16.378644
629	426	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutations  in WT1 were associated with increased risk of recurrence in young\n  patients (15-50) with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 236\nvariant_id: 127\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0946a8fe-5000-4652-9475-46ca58ce3e60	2015-03-04 17:26:16.447571
630	427	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutations in WT1 were associated with a worse overall prognosis than patients\n  wildtype for WT1 in young patients (15-50), primarily because of increased risk\n  of disease recurrence\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 236\nvariant_id: 127\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5e6888f9-a6a1-49a2-ade8-fc0f89fdb73a	2015-03-04 17:26:16.515762
631	428	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 235\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	91c48e09-eb7a-4e2b-80df-a430f2f84eb0	2015-03-04 17:26:16.587623
632	429	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 237\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8a7ceb27-fab5-42b7-bf76-314c45704606	2015-03-04 17:26:16.660117
633	430	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 234\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	07e4ddce-7509-4e18-b70e-97ee4f3b3d36	2015-03-04 17:26:16.73273
634	431	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 234\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c8308625-83cf-402b-a09f-4437cddbd60f	2015-03-04 17:26:16.804278
635	432	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 234\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f10806d3-d619-47d2-bbff-78d0e92d91ce	2015-03-04 17:26:16.941406
636	433	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No differences in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 235\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	70fd4f5b-8ce7-41bf-a742-13ef8f441c37	2015-03-04 17:26:17.032415
637	434	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 235\nvariant_id: 128\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0c361285-b95b-4cd2-a558-2177079b7f6f	2015-03-04 17:26:17.111972
638	435	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 235\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0deaa074-2fdb-4dfa-be8d-9342cc9d6c19	2015-03-04 17:26:17.200301
639	436	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 234\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fb653acf-7fe7-4d83-b164-a7308e663f0b	2015-03-04 17:26:17.287464
640	437	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 234\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2b0dd01d-3540-43a1-92c6-8542d04405e8	2015-03-04 17:26:17.361128
641	438	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 237\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fc35a274-dc7b-4b47-8ebf-39b93b8636fb	2015-03-04 17:26:17.436352
642	439	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: '"Two metastatic melanoma patients with the L576P KIT mutation were treated\n  with dasatinib, including one patient previously treated with imatinib. Both patients\n  had marked reduction (>50%) and elimination of tumor F18-fluorodeoxyglucose (FDG)-avidity\n  by positron emission tomography (PET) imaging after dasatinib treatment. These data\n  support the selective inhibitory effect of dasatinib against cells harboring the\n  most common KIT mutation in melanoma, and thus has therapeutic implications for\n  acrallentiginous, chronic sun-damaged, and mucosal melanomas."'\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 45\nsource_id: 163\nvariant_id: 71\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6888acaf-3f2d-4bec-bf0d-56df5c53cdb8	2015-03-04 17:26:17.509324
643	440	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: '"To formally test this hypothesis, we took advantage of HCT116 cells in which\n  the KRAS D13 mutant allele had been genetically deleted by homologous recombination\n  (15). Strikingly, we found that HCT116-derivative cells retaining only the KRAS\n  WT allele (named HKh-2 and HKe-3; Supplemental Table 3) were sensitive to everolimus,\n  while the parental and the isogenic cells carrying mutated KRAS were equally resistant\n  to this compound (Figure 2A)."'\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 66\ndisease_id: 46\nsource_id: 238\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e9ebdc31-ab82-4b61-9343-c8ab9c64db90	2015-03-04 17:26:17.593355
644	441	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Figure 1 of paper shows that V654A was associated with progression of disease\n  in one case of a mucosal melanoma. Results were from a Phase II clinical trial.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 45\nsource_id: 239\nvariant_id: 72\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4b7c5a7d-3c33-484f-8b63-f4072e96a2aa	2015-03-04 17:26:17.670828
645	442	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: '"Dose-dependent, complete response to imatinib of a metastatic mucosal melanoma\n  with a K642E KIT mutation."'\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 45\nsource_id: 240\nvariant_id: 150\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	282f9a6b-6350-4876-81b0-fbb79ca47878	2015-03-04 17:26:17.740351
646	443	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: '"From the screen, we discovered that proteasome inhibitors show selectivity\n  toward BRAF V600Emutant cells, irrespective of PTEN or RB1 expression. Preferential\n  targeting of BRAF-mutant cells by proteasome inhibitors was corroborated in a second\n  BRAF V600E isogenic model, as well as in a panel of colorectal cancer cell lines\n  by the use of the proteasome inhibitor carfilzomib. Notably, carfilzomib also showed\n  striking in vivo activity in a BRAF-mutant human colorectal cancer xenograft model.\n  Vulnerability to proteasome inhibitors is dependent on persistent BRAF signaling,\n  because BRAF V600E blockade by PLX4720 reversed sensitivity to carfilzomib in BRAF-mutant\n  colorectal cancer cells. Our findings indicated that proteasome inhibition might\n  represent a valuable targeting strategy in BRAF V600Emutant colorectal tumors."'\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 67\ndisease_id: 47\nsource_id: 241\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	71ba393d-2b5f-4542-ba7f-b59d31003a53	2015-03-04 17:26:17.81246
647	444	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PDGFRA G853D mutation found in an acral melanoma sample. In vitro testing showed\n  that G853D induced IL-3independent growth could be inhibited by either imatinib\n  or crenolanib to different extent.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 48\nsource_id: 242\nvariant_id: 148\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ffaba3d9-0ac5-487f-8311-4fa178090b8b	2015-03-04 17:26:17.887962
648	445	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: '"Here, we report the case of a patient with metastatic lung adenocarcinoma\n  with BRAF G469L mutation refractory to vemurafenib. We calculated a structure model\n  of this very rare type of mutated BRAF kinase to explain the molecular mechanism\n  of drug resistance."'\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 68\ndisease_id: 49\nsource_id: 243\nvariant_id: 149\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	40ae638b-5bdd-4185-b3eb-bc4419bdd902	2015-03-04 17:26:17.958069
649	446	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: '"Authors of study investigated genomic changes in metastatic renal cancer patients\n  with long term benefit from rapalog treatment. Patient 3 (treated with everolimus),\n  in study, was found to have a novel MTOR mutation, Q2223K. This residue was in the\n  PI3K kinase domain structure. In vitro, functional validation confirmed that this\n  variant was activating and was sensitive to rapamycin and rapalogs, everolimus and\n  temsirolimus."'\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 66\ndisease_id: 50\nsource_id: 244\nvariant_id: 147\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	eab077d7-9b2d-4005-93cf-409d63bababd	2015-03-04 17:26:18.030854
\.


--
-- Name: audits_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('audits_id_seq', 649, true);


--
-- Data for Name: authorizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY authorizations (id, user_id, provider, uid, created_at, updated_at) FROM stdin;
\.


--
-- Name: authorizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('authorizations_id_seq', 1, false);


--
-- Data for Name: comments; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY comments (id, title, comment, commentable_id, commentable_type, user_id, role, created_at, updated_at) FROM stdin;
\.


--
-- Name: comments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('comments_id_seq', 1, false);


--
-- Data for Name: data_versions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY data_versions (id, version) FROM stdin;
\.


--
-- Name: data_versions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('data_versions_id_seq', 1, false);


--
-- Data for Name: definitions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY definitions (id, term, text, created_at, updated_at) FROM stdin;
\.


--
-- Name: definitions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('definitions_id_seq', 1, false);


--
-- Data for Name: delayed_jobs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY delayed_jobs (id, priority, attempts, handler, last_error, run_at, locked_at, failed_at, locked_by, queue, created_at, updated_at) FROM stdin;
\.


--
-- Name: delayed_jobs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('delayed_jobs_id_seq', 1, false);


--
-- Data for Name: diseases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY diseases (id, doid, name, created_at, updated_at) FROM stdin;
2	1612	Breast Cancer	2015-03-04 17:25:44.579654	2015-03-04 17:25:44.579654
5	769	Neuroblastoma	2015-03-04 17:25:45.014947	2015-03-04 17:25:45.014947
7	1909	Melanoma	2015-03-04 17:25:45.960049	2015-03-04 17:25:45.960049
23	4947	Cholangiocarcinoma	2015-03-04 17:25:57.147903	2015-03-04 17:25:57.147903
26	3069	Astrocytoma	2015-03-04 17:25:59.927362	2015-03-04 17:25:59.927362
36	12603	Acute Leukemia	2015-03-04 17:26:03.498692	2015-03-04 17:26:03.498692
45	0050929	Mucosal Melanoma	2015-03-04 17:26:17.487534	2015-03-04 17:26:17.487534
46	1520	Colon Carcinoma	2015-03-04 17:26:17.567816	2015-03-04 17:26:17.567816
47	9256	Colorectal Cancer	2015-03-04 17:26:17.792815	2015-03-04 17:26:17.792815
48	6367	Acral Lentiginous Melanoma	2015-03-04 17:26:17.866023	2015-03-04 17:26:17.866023
49	3910	Lung Adenocarcinoma	2015-03-04 17:26:17.936637	2015-03-04 17:26:17.936637
50	4450	Renal Cell Carcinoma	2015-03-04 17:26:18.012109	2015-03-04 17:26:18.012109
1	8552	Chronic Myeloid Leukemia	2015-03-04 17:25:44.007693	2015-03-04 17:28:58.830492
3	3908	Non-small Cell Lung Carcinoma	2015-03-04 17:25:44.654387	2015-03-04 17:28:59.271702
4	0050905	Inflammatory Myofibroblastic Tumor	2015-03-04 17:25:44.942543	2015-03-04 17:28:59.400783
6	9256	Colorectal Cancer	2015-03-04 17:25:45.875005	2015-03-04 17:28:59.706524
8	9256	Colorectal Cancer	2015-03-04 17:25:46.285078	2015-03-04 17:29:00.042402
9	1781	Thyroid Cancer	2015-03-04 17:25:47.748485	2015-03-04 17:29:00.254025
10	2394	Ovarian Cancer	2015-03-04 17:25:48.592098	2015-03-04 17:29:00.468722
11	1612	Breast Cancer	2015-03-04 17:25:49.072986	2015-03-04 17:29:01.108774
12	5520	Head And Neck Squamous Cell Carcinoma	2015-03-04 17:25:49.231144	2015-03-04 17:29:01.288744
13	0050746	Mantle Cell Lymphoma	2015-03-04 17:25:49.331899	2015-03-04 17:29:01.484475
14	10534	Stomach Cancer	2015-03-04 17:25:49.43805	2015-03-04 17:29:01.876758
15	715	T-cell Leukemia	2015-03-04 17:25:49.883949	2015-03-04 17:29:02.026274
16	5635	Gastric Adenosquamous Carcinoma	2015-03-04 17:25:50.064777	2015-03-04 17:29:02.216648
17	1324	Lung Cancer	2015-03-04 17:25:50.227455	2015-03-04 17:29:02.521759
18	0060075	Estrogen-receptor Positive Breast Cancer	2015-03-04 17:25:50.292298	2015-03-04 17:29:02.76017
19	9119	Acute Myeloid Leukemia	2015-03-04 17:25:50.7343	2015-03-04 17:29:02.983291
20	9952	Acute Lymphocytic Leukemia	2015-03-04 17:25:51.203245	2015-03-04 17:29:03.185189
21	5015	Hepatocellular Fibrolamellar Carcinoma	2015-03-04 17:25:52.023417	2015-03-04 17:29:03.348726
22	3908	Non-small Cell Lung Carcinoma	2015-03-04 17:25:54.559626	2015-03-04 17:29:03.609608
24	4007	Bladder Carcinoma	2015-03-04 17:25:57.229825	2015-03-04 17:29:04.453221
25	3909	Bronchogenic Lung Adenocarcinoma	2015-03-04 17:25:59.527252	2015-03-04 17:29:04.587501
27	3068	Glioblastoma Multiforme	2015-03-04 17:25:59.999331	2015-03-04 17:29:05.125642
28	0050908	Myelodysplastic Syndrome	2015-03-04 17:26:00.070133	2015-03-04 17:29:06.496233
29	0050908	Myelodysplastic Syndrome	2015-03-04 17:26:00.621592	2015-03-04 17:29:06.674934
30	10747	Lymphoid Leukemia	2015-03-04 17:26:01.063667	2015-03-04 17:29:06.828909
31	4960	Bone Marrow Cancer	2015-03-04 17:26:01.135241	2015-03-04 17:29:06.990167
32	8997	Polycythemia Vera	2015-03-04 17:26:01.21493	2015-03-04 17:29:07.149675
33	9119	Acute Myeloid Leukemia	2015-03-04 17:26:01.34104	2015-03-04 17:29:07.319829
34	9253	Gastrointestinal Stromal Tumor	2015-03-04 17:26:01.433255	2015-03-04 17:29:07.49963
35	14145	Malignant Anus Melanoma	2015-03-04 17:26:02.405902	2015-03-04 17:29:07.661386
37	9538	Multiple Myeloma	2015-03-04 17:26:03.653266	2015-03-04 17:29:08.998614
38	3910	Lung Adenocarcinoma	2015-03-04 17:26:04.181611	2015-03-04 17:29:09.342415
39	9119	Acute Myeloid Leukemia	2015-03-04 17:26:05.355537	2015-03-04 17:29:09.615606
40	3068	Glioblastoma Multiforme	2015-03-04 17:26:06.228856	2015-03-04 17:29:09.974872
41	9253	Gastrointestinal Stromal Tumor	2015-03-04 17:26:11.033944	2015-03-04 17:29:10.131515
42	9119	Acute Myeloid Leukemia	2015-03-04 17:26:13.272937	2015-03-04 17:29:10.300528
43	3973	Thyroid Medullary Carcinoma	2015-03-04 17:26:13.706665	2015-03-04 17:29:10.484055
44	6477	Invasive Bladder Transitional Cell Carcinoma	2015-03-04 17:26:15.255618	2015-03-04 17:29:10.886147
\.


--
-- Name: diseases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('diseases_id_seq', 50, true);


--
-- Data for Name: drugs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs (id, name, pubchem_id, created_at, updated_at) FROM stdin;
1	N/A	\N	2015-03-04 17:25:44.000406	2015-03-04 17:25:44.000406
2	Nilotinib	\N	2015-03-04 17:25:44.168847	2015-03-04 17:25:44.168847
3	Dasatinib	\N	2015-03-04 17:25:44.233593	2015-03-04 17:25:44.233593
4	Imatinib	\N	2015-03-04 17:25:44.299996	2015-03-04 17:25:44.299996
5	MK-2206	\N	2015-03-04 17:25:44.575713	2015-03-04 17:25:44.575713
6	Crizotinib	\N	2015-03-04 17:25:44.650357	2015-03-04 17:25:44.650357
7	CH5424802	\N	2015-03-04 17:25:44.804808	2015-03-04 17:25:44.804808
8	TAE684	\N	2015-03-04 17:25:45.241937	2015-03-04 17:25:45.241937
9	Sorafenib	\N	2015-03-04 17:25:45.641356	2015-03-04 17:25:45.641356
10	Trametinib	\N	2015-03-04 17:25:45.790881	2015-03-04 17:25:45.790881
11	Panitumumab	\N	2015-03-04 17:25:45.870713	2015-03-04 17:25:45.870713
12	Dabrafenib	\N	2015-03-04 17:25:45.95627	2015-03-04 17:25:45.95627
13	Cetuximab	\N	2015-03-04 17:25:46.27921	2015-03-04 17:25:46.27921
14	PD0325901	\N	2015-03-04 17:25:46.483996	2015-03-04 17:25:46.483996
15	PLX4720	\N	2015-03-04 17:25:46.542659	2015-03-04 17:25:46.542659
16	Nutlin-3	\N	2015-03-04 17:25:46.690116	2015-03-04 17:25:46.690116
17	Capecitabine	\N	2015-03-04 17:25:46.762361	2015-03-04 17:25:46.762361
18	Vemurafenib	\N	2015-03-04 17:25:46.82249	2015-03-04 17:25:46.82249
19	Bevacizumab	\N	2015-03-04 17:25:46.884951	2015-03-04 17:25:46.884951
20	Dacarbazine	\N	2015-03-04 17:25:47.190327	2015-03-04 17:25:47.190327
21	Temozolomide	\N	2015-03-04 17:25:47.253186	2015-03-04 17:25:47.253186
22	AZD6244	\N	2015-03-04 17:25:48.267271	2015-03-04 17:25:48.267271
23	Olaparib	\N	2015-03-04 17:25:48.587482	2015-03-04 17:25:48.587482
24	Palbociclib (PD-0332991)	\N	2015-03-04 17:25:49.877918	2015-03-04 17:25:49.877918
25	BYL719	\N	2015-03-04 17:25:50.288489	2015-03-04 17:25:50.288489
26	Palbociclib	\N	2015-03-04 17:25:50.36662	2015-03-04 17:25:50.36662
27	GW-2580	\N	2015-03-04 17:25:51.267691	2015-03-04 17:25:51.267691
28	Erlotinib	\N	2015-03-04 17:25:51.947591	2015-03-04 17:25:51.947591
29	Daunorubicin	\N	2015-03-04 17:25:52.110593	2015-03-04 17:25:52.110593
30	Idarubicin	\N	2015-03-04 17:25:52.180842	2015-03-04 17:25:52.180842
31	Gefitinib	\N	2015-03-04 17:25:54.078059	2015-03-04 17:25:54.078059
32	Premetrexed	\N	2015-03-04 17:25:54.692604	2015-03-04 17:25:54.692604
33	Stauroporine	\N	2015-03-04 17:25:54.985616	2015-03-04 17:25:54.985616
34	Dacomitinib	\N	2015-03-04 17:25:55.058892	2015-03-04 17:25:55.058892
35	Neratinib	\N	2015-03-04 17:25:55.133623	2015-03-04 17:25:55.133623
36	Lapatinib	\N	2015-03-04 17:25:55.442236	2015-03-04 17:25:55.442236
37	Hormone Therapy	\N	2015-03-04 17:25:56.081518	2015-03-04 17:25:56.081518
38	Tamoxifen	\N	2015-03-04 17:25:56.150298	2015-03-04 17:25:56.150298
39	Fulvestrant	\N	2015-03-04 17:25:56.221643	2015-03-04 17:25:56.221643
40	Ponatinib	\N	2015-03-04 17:25:57.143846	2015-03-04 17:25:57.143846
41	PD173074	\N	2015-03-04 17:25:57.225493	2015-03-04 17:25:57.225493
42	Pazopanib	\N	2015-03-04 17:25:57.331453	2015-03-04 17:25:57.331453
43	AG1296	\N	2015-03-04 17:25:57.461673	2015-03-04 17:25:57.461673
44	ATRA	\N	2015-03-04 17:25:57.534816	2015-03-04 17:25:57.534816
45	CEP701	\N	2015-03-04 17:25:57.621044	2015-03-04 17:25:57.621044
46	SU5614	\N	2015-03-04 17:25:58.832731	2015-03-04 17:25:58.832731
47	Bortezomib	\N	2015-03-04 17:25:59.521281	2015-03-04 17:25:59.521281
48	Fasudil	\N	2015-03-04 17:25:59.584875	2015-03-04 17:25:59.584875
49	Pegylated IFN-2a	\N	2015-03-04 17:26:01.210855	2015-03-04 17:26:01.210855
50	TG101348	\N	2015-03-04 17:26:01.275276	2015-03-04 17:26:01.275276
51	SU11248	\N	2015-03-04 17:26:03.072938	2015-03-04 17:26:03.072938
52	Regorafenib	\N	2015-03-04 17:26:03.27614	2015-03-04 17:26:03.27614
53	NVP-BEZ235	\N	2015-03-04 17:26:04.745185	2015-03-04 17:26:04.745185
54	ARRY-142886	\N	2015-03-04 17:26:04.816272	2015-03-04 17:26:04.816272
55	Carmustine	\N	2015-03-04 17:26:06.222963	2015-03-04 17:26:06.222963
56	O(6)-benzylguanine	\N	2015-03-04 17:26:06.376	2015-03-04 17:26:06.376
57	NSC348884	\N	2015-03-04 17:26:09.3708	2015-03-04 17:26:09.3708
58	Valproic Acid	\N	2015-03-04 17:26:09.503616	2015-03-04 17:26:09.503616
59	17-AAG	\N	2015-03-04 17:26:10.484146	2015-03-04 17:26:10.484146
60	Crenolanib	\N	2015-03-04 17:26:11.027153	2015-03-04 17:26:11.027153
61	CH5132799	\N	2015-03-04 17:26:11.682982	2015-03-04 17:26:11.682982
62	Rapamycin	\N	2015-03-04 17:26:11.761314	2015-03-04 17:26:11.761314
63	Motesanib	\N	2015-03-04 17:26:13.702983	2015-03-04 17:26:13.702983
64	AZD1480	\N	2015-03-04 17:26:13.794433	2015-03-04 17:26:13.794433
65	Doxorubicin	\N	2015-03-04 17:26:14.25595	2015-03-04 17:26:14.25595
66	Everolimus	\N	2015-03-04 17:26:15.251215	2015-03-04 17:26:15.251215
67	Carfilzomib	\N	2015-03-04 17:26:17.789106	2015-03-04 17:26:17.789106
68	Plx4032	\N	2015-03-04 17:26:17.932603	2015-03-04 17:26:17.932603
\.


--
-- Name: drugs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('drugs_id_seq', 68, true);


--
-- Data for Name: evidence_levels; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_levels (id, level, description, created_at, updated_at) FROM stdin;
1	C	\N	2015-03-04 17:25:44.013037	2015-03-04 17:25:44.013037
2	A	\N	2015-03-04 17:25:44.096582	2015-03-04 17:25:44.096582
3	B	\N	2015-03-04 17:25:44.173818	2015-03-04 17:25:44.173818
4	D	\N	2015-03-04 17:25:48.272162	2015-03-04 17:25:48.272162
\.


--
-- Data for Name: evidence_types; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_types (id, evidence_type, created_at, updated_at) FROM stdin;
1	Diagnostic	2015-03-04 17:25:44.0107	2015-03-04 17:25:44.0107
2	Predictive	2015-03-04 17:25:44.171825	2015-03-04 17:25:44.171825
3	Prognostic	2015-03-04 17:25:47.02851	2015-03-04 17:25:47.02851
\.


--
-- Data for Name: genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes (id, entrez_id, name, description, official_name, created_at, updated_at, clinical_description) FROM stdin;
1	238	ALK	ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines.	anaplastic lymphoma receptor tyrosine kinase	2015-03-04 17:25:26.459258	2015-03-04 17:25:26.459258	\N
2	207	AKT1	AKT1, also referred to as protein kinase B, is a known oncogene. AKT activation relies on the PI3K pathway, and is recognized as a critical node in the pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been shown to result in activation of the protein. Mutations in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors in vitro.	v-akt murine thymoma viral oncogene homolog 1	2015-03-04 17:25:26.746404	2015-03-04 17:25:26.746404	\N
3	369	ARAF	ARAF has recently become increasingly considered for its oncogenic potential. Its potential as a target for informing clinical action was demonstrated by a single case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated with sorafenib, acheived near-complete clinical remission. This finding has brought new focus on ARAF as a marker that should be assayed for in cancer treatment.	A-Raf proto-oncogene, serine/threonine kinase	2015-03-04 17:25:26.987268	2015-03-04 17:25:26.987268	\N
4	25	ABL1	ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein that has become a signature of chronic myeloid leukemia (CML). Cells harboring this fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook of the disease. However, additional mutations in ABL1 have been shown to confer resistance to imatinib. In these resistance cases, second-generation tyrosine kinase inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are currently undergoing clinical trials for treating acquired resistance in CML.	ABL proto-oncogene 1, non-receptor tyrosine kinase	2015-03-04 17:25:27.258027	2015-03-04 17:25:27.258027	\N
5	673	BRAF	BRAF mutations are found to be recurrent in many cancer types. Of these, the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E has been determined to be an activating mutation, and cells that harbor it, along with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations have also been correlated with poor prognosis in many cancer types, although there is at least one study that questions this conclusion in papillary thyroid cancer.	B-Raf proto-oncogene, serine/threonine kinase	2015-03-04 17:25:27.514894	2015-03-04 17:25:27.514894	\N
6	672	BRCA1	BRCA1 mutations in the germline have become a hallmark for hereditary breast and ovarian cancers. Variants that have been demonstrated to reduce the function of the protein have been shown to increase risk for these cancers, as well as prostate and pancreatic cancer. These findings have been the impetus for the increased popularity of genetic testing of healthy indivudals to assess risk.	breast cancer 1, early onset	2015-03-04 17:25:27.768412	2015-03-04 17:25:27.768412	\N
7	675	BRCA2	BRCA2 mutations in the germline have become a hallmark for hereditary breast and ovarian cancers. Variants that have been demonstrated to reduce the function of the protein have been shown to increase risk for these cancers, as well as prostate and pancreatic cancer. These findings have been the impetus for the increased popularity of genetic testing of healthy indivudals to assess risk.	breast cancer 2, early onset	2015-03-04 17:25:28.039733	2015-03-04 17:25:28.039733	\N
8	595	CCND1	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D1	2015-03-04 17:25:28.305093	2015-03-04 17:25:28.305093	\N
9	894	CCND2	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D2	2015-03-04 17:25:28.542837	2015-03-04 17:25:28.542837	\N
10	896	CCND3	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. In a mouse study, the targeted therapeutic palbociclib significantly increased the median survival of the cyclin D3 knockouts.	cyclin D3	2015-03-04 17:25:28.765395	2015-03-04 17:25:28.765395	\N
11	898	CCNE1	Cyclin E, while currenly not as widely implicated as its cyclin D counterparts, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements are correlated with later stage disease. In lung cancer, neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly expressed counterparts.	cyclin E1	2015-03-04 17:25:28.972469	2015-03-04 17:25:28.972469	\N
12	1021	CDK6	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 6	2015-03-04 17:25:29.188955	2015-03-04 17:25:29.188955	\N
13	1019	CDK4	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 4	2015-03-04 17:25:29.4035	2015-03-04 17:25:29.4035	\N
14	1029	CDKN2A	CDKN2A loss has been shown to be a significant event in a number of cancer types. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. One mechanism by which loss of CDKN2A can occur is by hypermethylation of the promoter region for the gene. However, the prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have suggesting poorer prognostic outcome for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	cyclin-dependent kinase inhibitor 2A	2015-03-04 17:25:29.701994	2015-03-04 17:25:29.701994	\N
15	1050	CEBPA	AML with mutated CEBPA' is a provisional entity in the WHO classification of acute myeloid leukemia (AML) and is recommended to be tested for in patients with AML. CEBPA mutations are particularly associated with cytogenetically normal AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha protein while C-terminal mutations reduce the DNA-binding potential of this transcription factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	CCAAT/enhancer binding protein (C/EBP), alpha	2015-03-04 17:25:30.071926	2015-03-04 17:25:30.071926	\N
16	3337	DNAJB1	DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	DnaJ (Hsp40) homolog, subfamily B, member 1	2015-03-04 17:25:30.315357	2015-03-04 17:25:30.315357	\N
17	5566	PRKACA	PRKACA has been studied in breast cancer and has been found to mediate resistance to HER2 targeted therapies. It has also been found to contain a mutation hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	protein kinase, cAMP-dependent, catalytic, alpha	2015-03-04 17:25:30.53847	2015-03-04 17:25:30.53847	\N
18	1788	DNMT3A	DNMT3A is one of several epigenetic modifiers identified as recurrently mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers. AML patient bone marrow harboring R882 mutations were similarly demonstrated to be hypomethylated compared to patients with wildtype DNMT3A. These studies also indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes and treatment protocols may need to be considered for these two classes of DNMT3A mutations.	DNA (cytosine-5-)-methyltransferase 3 alpha	2015-03-04 17:25:30.83245	2015-03-04 17:25:30.83245	\N
19	1956	EGFR	EGFR is widely recognized for its importance in cancer. Amplification and mutations have been shown to be driving events in many cancer types. Its role in non-small cell lung cancer has spurred many research and drug development efforts. Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance to these drugs, particularly the variant T790M, which has been functionally characterized as a resistance marker for both of these drugs. The later generation TKI's have seen some success in treating these resistant cases, and targeted sequencing of the EGFR locus has become a common practice in non-small cell lung cancer.	epidermal growth factor receptor	2015-03-04 17:25:31.082287	2015-03-04 17:25:31.082287	\N
20	2064	ERBB2	ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast cancer, and is treated in a separate manner than the other subtypes of breast cancer. Apart from being amplified, ERBB2 activating mutations have been shown to have clinical importance in HER2-negative breast cancer. These mutations have shown sensitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical sequencing efforts in treating breast cancer.	erb-b2 receptor tyrosine kinase 2	2015-03-04 17:25:31.34963	2015-03-04 17:25:31.34963	\N
21	2099	ESR1	ESR1 has been a focus in breast cancer for quite some time, but has also shown significance in endometrial, ovarian and other cancer types. ER-positive breast cancer that is resistant to hormone therapy has instigated clinical sequencing efforts to shed light on the mechanisms of this resistance. A number of mutations in the ligand binding domain of the protein have been implicated in hormone resistance and anti-estrogen therapies. This has spurred efforts to develop therapeutics that act to degrade the protein, rather than act as an antagonist. These agents are currently in clinical trials and have seen some success, highlighting the importance of sequencing efforts in treating breast cancer.	estrogen receptor 1	2015-03-04 17:25:31.616132	2015-03-04 17:25:31.616132	\N
22	2263	FGFR2	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions that lead to pathway actiation have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 2	2015-03-04 17:25:31.889937	2015-03-04 17:25:31.889937	\N
23	2261	FGFR3	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions that lead to pathway actiation have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 3	2015-03-04 17:25:32.201189	2015-03-04 17:25:32.201189	\N
24	2322	FLT3	FLT3 is an important cytokine receptor involved in normal hematopoiesis. Mutations in this gene are common in acute myeloid leukemia (AML) and screening for mutations in this gene has been recommended by the World Health Organization in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML). FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD mutations are still up for debate.	fms-related tyrosine kinase 3	2015-03-04 17:25:32.436425	2015-03-04 17:25:32.436425	\N
25	2624	GATA2	GATA2 is a transcription factor involved in stem cell maintenance with key roles in hematopoietic development. GATA2 mutations are associated with a variety of inherited and acquired immune disorders including myelodysplastic syndrome and acute myeloid leukemia. In addition to a role in hematopoiesis, the maintenance GATA2 expression has been implicated as a requirement in KRAS-driven non-small cell lung cancer. Preclinical models have indicated therapeutic benefit from targeting GATA2-mediated pathways in the context of KRAS-driven NSCLC.	GATA binding protein 2	2015-03-04 17:25:32.678646	2015-03-04 17:25:32.678646	\N
26	3417	IDH1	IDH1 mutations have been observed in a number of cancer types, including sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described as mutually exclusive in many of these cancer types. The most frequent mutations involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic enzyme activity. The implications of mutations in this gene vary greatly by cancer type. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations have been associated with worse outcome, shorter overall survival, and normal karyotype. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Unlike the association with cytogenetically normal AML, in glioblastoma, IDH1 mutations have been associated with specific cytogenetic abnormalities, 1p and 19q deletions.	isocitrate dehydrogenase 1 (NADP+), soluble	2015-03-04 17:25:32.950987	2015-03-04 17:25:32.950987	\N
27	3418	IDH2	IDH2 mutations have been observed in a number of cancer types, including sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described as mutually exclusive in many of these cancer types. The most frequent mutations involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic enzyme activity. Although IDH2 (R172) mutations are associated with poorer overall prognosis in AML patients, its utility as a prognostic marker in MDS is still under debate. Additionally, IDH2 (R140) has been associated with improved overall survival in AML. IDH2 mutations have been associated with improved prognosis in gliomas.	isocitrate dehydrogenase 2 (NADP+), mitochondrial	2015-03-04 17:25:33.214535	2015-03-04 17:25:33.214535	\N
28	3717	JAK2	JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative diseases and cancers. The mutation V617F is the most clinically relevant variant, and is seen in around half of myeloproliferative disorders. The variant is a known activating mutation, and activated JAK2 is sufficient to drive myeloproliferative disorders in mouse models. V617F, while most recurrent, is not the only mechanism by which JAK2 can be activated in patients. JAK2 is now one of the first diagnostic markers tested upon diagnosis with a myeloproliferative disorder.	Janus kinase 2	2015-03-04 17:25:33.449997	2015-03-04 17:25:33.449997	\N
29	3815	KIT	c-KIT activation has been shown to have oncogenic activity in gastrointestinal stromal tumors (GISTs), melanomas, lung cancer, and other tumor types. The targeted therapeutics nilotinib and sunitinib have shown efficacy in treating KIT overactive patients, and are in late-stage trials in melanoma and GIST. KIT overactivity can be the result of many genomic events from genomic amplification to overexpression to missense mutations. Missense mutations have been shown to be key players in mediating clinical response and acquired resistance in patients being treated with these targeted therapeutics.	v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog	2015-03-04 17:25:33.681263	2015-03-04 17:25:33.681263	\N
30	3845	KRAS	Mutations in the RAS family of proteins have frequently observed across cancer types. The amino acid positions account for the overwhelming majority of these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also behave very differently when expressed in non-native tissue types, likely due to differences in the C-terminal hyper-variable regions. Mis-regulation of isoform expression has been shown to be a driving event in cancer, as well as missense mutations at the three hotspots previously mentioned. While highly recurrent in cancer, targeted these RAS mutants has also been very elusive, and has not yet become common practice in the clinic.	Kirsten rat sarcoma viral oncogene homolog	2015-03-04 17:25:33.955933	2015-03-04 17:25:33.955933	\N
31	5604	MAP2K1	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	mitogen-activated protein kinase kinase 1	2015-03-04 17:25:34.204222	2015-03-04 17:25:34.204222	\N
32	4209	MEF2D	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	myocyte enhancer factor 2D	2015-03-04 17:25:34.404977	2015-03-04 17:25:34.404977	\N
33	1436	CSF1R	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	colony stimulating factor 1 receptor	2015-03-04 17:25:34.640151	2015-03-04 17:25:34.640151	\N
34	4255	MGMT	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	O-6-methylguanine-DNA methyltransferase	2015-03-04 17:25:34.85423	2015-03-04 17:25:34.85423	\N
35	4869	NPM1	AML with mutated NPM1 is provisional entity in the WHO classification of AML and is recommended to be tested for in patients with cytogenetically normal AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia, indicating it may require additional mutations to promote leukemic development.	nucleophosmin (nucleolar phosphoprotein B23, numatrin)	2015-03-04 17:25:35.066789	2015-03-04 17:25:35.066789	\N
36	4893	NRAS	Mutations in the RAS family of proteins have frequently observed across cancer types. The amino acid positions account for the overwhelming majority of these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also behave very differently when expressed in non-native tissue types, likely due to differences in the C-terminal hyper-variable regions. Mis-regulation of isoform expression has been shown to be a driving event in cancer, as well as missense mutations at the three hotspots previously mentioned. While highly recurrent in cancer, targeted these RAS mutants has also been very elusive, and has not yet become common practice in the clinic.	neuroblastoma RAS viral (v-ras) oncogene homolog	2015-03-04 17:25:35.311333	2015-03-04 17:25:35.311333	\N
37	5290	PIK3CA	PIK3CA is the most recurrently mutated gene in breast cancer, and has been found to important in a number of cancer types. An integral part of the PI3K pathway, PIK3CA has long been described as an oncogene, with two main hotspots for activating mutations, the 542/545 region of the helical domain, and the 1047 region of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the subject of an immense amount of research and development, and PI3K inhibition has seen some limited success in recent clinical trials. While monotherapies seem to be limited in their potential, there is a recent interest in pursuing PI3K inhibition as part of a combination therapy regiment with inhibition partners including TKI's, MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.	phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha	2015-03-04 17:25:35.557405	2015-03-04 17:25:35.557405	\N
38	5156	PDGFRA	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	platelet-derived growth factor receptor, alpha polypeptide	2015-03-04 17:25:35.788983	2015-03-04 17:25:35.788983	\N
39	5371	PML	The PML-RARA fusion is the result of a recurrent, balanced translocation between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	promyelocytic leukemia	2015-03-04 17:25:36.008596	2015-03-04 17:25:36.008596	\N
40	5914	RARA	The PML-RARA fusion is the result of a recurrent, balanced translocation between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	retinoic acid receptor, alpha	2015-03-04 17:25:36.213481	2015-03-04 17:25:36.213481	\N
41	5728	PTEN	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	phosphatase and tensin homolog	2015-03-04 17:25:36.46266	2015-03-04 17:25:36.46266	\N
42	5979	RET	RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine kinase receptor and are associated with thyroid cancers. RET point mutations are the most common mutations identified in medullary thyroid cancer (MTC) with germline and somatic mutations in RET associated with hereditary and sporadic forms, respectively. The most common somatic form mutation is M918T (exon 16) and a variety of other mutations effecting exons 10, 11 and 15 have been described. The prognostic significance of these mutations have been hotly debated in the field, however, data suggests that some RET mutation may confer drug resistence. No RET-specific agents are currently clinically available but several promiscuous kinase inhibitors that target RET, among others, have been approved for MTC treatment.	ret proto-oncogene	2015-03-04 17:25:36.727918	2015-03-04 17:25:36.727918	\N
43	861	RUNX1	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	runt-related transcription factor 1	2015-03-04 17:25:36.987508	2015-03-04 17:25:36.987508	\N
44	23451	SF3B1	SF3B1 mutations have been described in several myeloid malignancies, predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has been identified as recurrently mutated in MDS and other malignanices. The mutations affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally deleterious with K700E described as a major hotspot mutation. MDS patients with SF3B1 mutations have been reported to have better overall and event-free survival than their wildtype counterparts. Additionally, these mutations are highly associated with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring sideroblasts).	splicing factor 3b, subunit 1, 155kDa	2015-03-04 17:25:37.288057	2015-03-04 17:25:37.288057	\N
45	7157	TP53	TP53 mutations are universal across cancer types. Loss of tumor suppressors is most recognized by large deleterious events, such as frameshift mutations, or premature stop codons. In TP53 however, many of the observed mutations in cancer are found to be single nucleotide variants, or missense mutations. These variants are also very broadly distributed throughout the gene, not localizing in any particular hotspot. While a large proportion of cancer genomics research is focused on somatic variants, TP53 is also of note in the germline. Germline TP53 mutations are the hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been found to have prognostic impact on patient outcomes.	tumor protein p53	2015-03-04 17:25:37.599676	2015-03-04 17:25:37.599676	\N
46	7248	TSC1	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	tuberous sclerosis 1	2015-03-04 17:25:37.817444	2015-03-04 17:25:37.817444	\N
47	7249	TSC2	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	tuberous sclerosis 2	2015-03-04 17:25:38.029937	2015-03-04 17:25:38.029937	\N
48	7307	U2AF1	U2AF1 is one of several spliceosome complex genes frequently mutated in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein. These mutations have been associated with altered splicing patterns in vitro and in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation to secondary acute myeloid leukemia, however, the impact of these mutations on overall survival has been an area of debate.	U2 small nuclear RNA auxiliary factor 1	2015-03-04 17:25:38.266331	2015-03-04 17:25:38.266331	\N
49	7490	WT1	WT1 is a tumor suppressor gene associated with the development of Wilms' Tumor, from which it was named. Mutations in exon 7 and 9 of WT1 have been recurrently identified in acute myeloid leukemia and associated with poorer prognosis and chemotherapy resistance.	Wilms tumor 1	2015-03-04 17:25:38.513154	2015-03-04 17:25:38.513154	\N
50	2475	MTOR	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	mechanistic target of rapamycin (serine/threonine kinase)	2015-03-04 17:25:38.755111	2015-03-04 17:25:38.755111	\N
51	4851	NOTCH1	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	notch 1	2015-03-04 17:25:38.978967	2015-03-04 17:25:38.978967	\N
52	4921	DDR2	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	discoidin domain receptor tyrosine kinase 2	2015-03-04 17:25:39.196791	2015-03-04 17:25:39.196791	\N
53	4233	MET	A summary for this gene has yet to be developed! Add one now by clicking the "Edit Gene" button.	MET proto-oncogene, receptor tyrosine kinase	2015-03-04 17:25:39.425266	2015-03-04 17:25:39.425266	\N
\.


--
-- Data for Name: sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY sources (id, pubmed_id, study_type, description, created_at, updated_at) FROM stdin;
1	24889366	\N	Rossi et al., 2014, Int. J. Oncol.	2015-03-04 17:25:26.463162	2015-03-04 17:26:18.626338
2	23401436	\N	Shaw et al., 2013, J. Clin. Oncol.	2015-03-04 17:25:26.47057	2015-03-04 17:26:19.045922
3	16095999	\N	Bellacosa et al., 2005, Adv. Cancer Res.	2015-03-04 17:25:26.748625	2015-03-04 17:26:19.485132
4	15023437	\N	Fresno Vara et al., 2004, Cancer Treat. Rev.	2015-03-04 17:25:26.754202	2015-03-04 17:26:20.114438
5	15676015	\N	Lee et al., 2005, APMIS	2015-03-04 17:25:26.989244	2015-03-04 17:26:20.528879
6	24569458	\N	Imielinski et al., 2014, J. Clin. Invest.	2015-03-04 17:25:26.992967	2015-03-04 17:26:20.913424
7	15719031	\N	Ren, 2005, Nat. Rev. Cancer	2015-03-04 17:25:27.261535	2015-03-04 17:26:21.297254
8	17457302	\N	Weisberg et al., 2007, Nat. Rev. Cancer	2015-03-04 17:25:27.266151	2015-03-04 17:26:21.784541
9	19724843	\N	Li et al., 2009, Oncol. Rep.	2015-03-04 17:25:27.517325	2015-03-04 17:26:22.305222
10	23594689	\N	Pakneshan et al., 2013, Pathology	2015-03-04 17:25:27.521363	2015-03-04 17:26:23.162394
11	24366442	\N	Nelson et al., 2014, Ann. Intern. Med.	2015-03-04 17:25:27.770366	2015-03-04 17:26:23.560418
12	12432268	\N	Diehl, Cancer Biol. Ther.	2015-03-04 17:25:28.307869	2015-03-04 17:26:24.602948
13	24387133	\N	Casimiro et al., 2014, Expert Opin Investig Drugs	2015-03-04 17:25:28.312712	2015-03-04 17:26:24.984729
14	14965268	\N	Mazumder et al., 2004, Curr Cancer Drug Targets	2015-03-04 17:25:28.974586	2015-03-04 17:26:26.978131
15	20370706	\N	Graf et al., 2010, Mini Rev Med Chem	2015-03-04 17:25:29.190893	2015-03-04 17:26:27.324602
16	24089445	\N	Sheppard et al., 2013, Clin. Cancer Res.	2015-03-04 17:25:29.195463	2015-03-04 17:26:32.35233
17	20473920	\N	Shima et al., 2011, Int. J. Cancer	2015-03-04 17:25:29.704765	2015-03-04 17:26:33.498643
18	23111194	\N	Bradly et al., 2012, Diagn. Mol. Pathol.	2015-03-04 17:25:29.708833	2015-03-04 17:26:33.909877
19	19357394	\N	Vardiman et al., 2009, Blood	2015-03-04 17:25:30.074485	2015-03-04 17:26:34.386418
20	11242107	\N	Pabst et al., 2001, Nat. Genet.	2015-03-04 17:25:30.079084	2015-03-04 17:26:34.754992
21	9012825	\N	Zhang et al., 1997, Proc. Natl. Acad. Sci. U.S.A.	2015-03-04 17:25:30.083186	2015-03-04 17:26:35.166203
22	24578576	\N	Honeyman et al., 2014, Science	2015-03-04 17:25:30.317249	2015-03-04 17:26:35.680059
23	24909179	\N	Moody et al., 2014, Oncogene	2015-03-04 17:25:30.543444	2015-03-04 17:26:36.09403
24	24167195	\N	Kim et al., 2013, Blood	2015-03-04 17:25:30.834477	2015-03-04 17:26:36.476645
25	24656771	\N	Russler-Germain et al., 2014, Cancer Cell	2015-03-04 17:25:30.838449	2015-03-04 17:26:36.879932
26	23953842	\N	Yewale et al., 2013, Biomaterials	2015-03-04 17:25:31.084215	2015-03-04 17:26:37.270314
27	18712184	\N	Charpidou et al., In Vivo	2015-03-04 17:25:31.087835	2015-03-04 17:26:37.643334
28	16947083	\N	Badache et al., 2006, J Mammary Gland Biol Neoplasia	2015-03-04 17:25:31.351667	2015-03-04 17:26:38.011477
29	10878580	\N	Yu et al., 2000, Bioessays	2015-03-04 17:25:31.355307	2015-03-04 17:26:38.463328
30	24185510	\N	Robinson et al., 2013, Nat. Genet.	2015-03-04 17:25:31.61813	2015-03-04 17:26:38.922095
31	22279420	\N	Mohibi et al., 2011, J Carcinog	2015-03-04 17:25:31.621968	2015-03-04 17:26:39.327022
32	18636142	\N	Katoh, 2008, Int. J. Oncol.	2015-03-04 17:25:31.892281	2015-03-04 17:26:39.748318
33	23558953	\N	Wu et al., 2013, Cancer Discov	2015-03-04 17:25:31.896226	2015-03-04 17:26:40.175443
34	12951584	\N	Stirewalt et al., 2003, Nat. Rev. Cancer	2015-03-04 17:25:32.438568	2015-03-04 17:26:40.897981
35	22541434	\N	Kumar et al., 2012, Cell	2015-03-04 17:25:32.680686	2015-03-04 17:26:41.318352
36	25619630	\N	Mir et al., 2015, Cancer Med	2015-03-04 17:25:32.684807	2015-03-04 17:26:41.704829
37	24065766	\N	Lu et al., 2013, Genes Dev.	2015-03-04 17:25:32.952926	2015-03-04 17:26:43.701684
38	19228619	\N	Yan et al., 2009, N. Engl. J. Med.	2015-03-04 17:25:32.956602	2015-03-04 17:26:44.137712
39	22417203	\N	Patel et al., 2012, N. Engl. J. Med.	2015-03-04 17:25:33.222156	2015-03-04 17:26:44.723763
40	17133099	\N	Levine et al., 2007, Curr. Opin. Hematol.	2015-03-04 17:25:33.45217	2015-03-04 17:26:45.178881
41	23944364	\N	Stankov et al., 2014, Curr. Pharm. Des.	2015-03-04 17:25:33.683445	2015-03-04 17:26:45.597042
42	22589270	\N	Prior et al., 2012, Cancer Res.	2015-03-04 17:25:33.958205	2015-03-04 17:26:46.05535
43	23226219	\N	Chou et al., 2012, PLoS ONE	2015-03-04 17:25:35.068779	2015-03-04 17:26:46.417488
44	18073307	\N	Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-03-04 17:25:36.730409	2015-03-04 17:26:46.831468
45	25465739	\N	Perri et al., 2014, Crit. Rev. Oncol. Hematol.	2015-03-04 17:25:36.735698	2015-03-04 17:26:47.209904
46	21995386	\N	Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-03-04 17:25:37.291113	2015-03-04 17:26:47.60846
47	24136165	\N	Cazzola et al., 2013, Blood	2015-03-04 17:25:37.297371	2015-03-04 17:26:48.007807
48	20182602	\N	Olivier et al., 2010, Cold Spring Harb Perspect Biol	2015-03-04 17:25:37.60189	2015-03-04 17:26:48.402955
49	25311244	\N	Okeyo-Owuor et al., 2014, Leukemia	2015-03-04 17:25:38.268286	2015-03-04 17:26:48.837176
50	22158538	\N	Graubert et al., 2012, Nat. Genet.	2015-03-04 17:25:38.271893	2015-03-04 17:26:51.038912
51	25212276		Schmidt et al., 2014, Leukemia	2015-03-04 17:25:44.003673	2015-03-04 17:26:51.442438
52	20537386		An et al., 2010, Leuk. Res.	2015-03-04 17:25:44.092438	2015-03-04 17:26:51.868519
53	15194504	Cell Line	Yamamoto et al., 2004, Biochem. Biophys. Res. Commun.	2015-03-04 17:25:44.302985	2015-03-04 17:26:52.281485
54	23888070		Beaver et al., 2013, Clin. Cancer Res.	2015-03-04 17:25:44.577777	2015-03-04 17:26:54.079169
55	20979473	Clinical Trial	Choi et al., 2010, N. Engl. J. Med.	2015-03-04 17:25:44.652294	2015-03-04 17:26:54.475215
56	21575866		Sakamoto et al., 2011, Cancer Cell	2015-03-04 17:25:44.806918	2015-03-04 17:26:54.904396
57	21030459		Sasaki et al., 2010, Cancer Res.	2015-03-04 17:25:44.94051	2015-03-04 17:26:55.348874
58	22072639		Bresler et al., 2011, Sci Transl Med	2015-03-04 17:25:45.109068	2015-03-04 17:26:55.754874
59	21948233		Heuckmann et al., 2011, Clin. Cancer Res.	2015-03-04 17:25:45.173933	2015-03-04 17:26:56.112089
60	18923525		George et al., 2008, Nature	2015-03-04 17:25:45.243814	2015-03-04 17:26:56.589891
61	24569458		Imielinski et al., 2014, J. Clin. Invest.	2015-03-04 17:25:45.644755	2015-03-04 17:26:56.988748
62	23325582	Clinical trial	Peeters et al., 2013, Clin. Cancer Res.	2015-03-04 17:25:45.872708	2015-03-04 17:26:57.692607
63	23020132	Clinical trial	Flaherty et al., 2012, N. Engl. J. Med.	2015-03-04 17:25:45.958156	2015-03-04 17:26:58.096471
64	21166657	Meta analysis	Lee et al., 2011, Br. J. Dermatol.	2015-03-04 17:25:46.094546	2015-03-04 17:26:58.54665
65	23463675	Clinical trial	Ponti et al., 2013, J. Clin. Pathol.	2015-03-04 17:25:46.191513	2015-03-04 17:26:58.967646
66	19001320	In vitro study	Di Nicolantonio et al., 2008, J. Clin. Oncol.	2015-03-04 17:25:46.282282	2015-03-04 17:26:59.386647
67	23845441	In vitro study	Rad et al., 2013, Cancer Cell	2015-03-04 17:25:46.486055	2015-03-04 17:26:59.889085
68	23812671	In vitro study	Ji et al., 2013, Clin. Cancer Res.	2015-03-04 17:25:46.611269	2015-03-04 17:27:00.431631
69	22180495	In vivo study	Yang et al., 2012, Cancer Res.	2015-03-04 17:25:46.764481	2015-03-04 17:27:00.861772
70	20619739	Retrospective study	De Roock et al., 2010, Lancet Oncol.	2015-03-04 17:25:46.956311	2015-03-04 17:27:01.261164
71	24594804	Metaanalysis	Chen et al., 2014, PLoS ONE	2015-03-04 17:25:47.024773	2015-03-04 17:27:01.685598
72	24586605	Retrospective study	Meckbach et al., 2014, PLoS ONE	2015-03-04 17:25:47.192174	2015-03-04 17:27:02.4264
73	24583796	Clinical trial	Menzies et al., 2014, Clin. Cancer Res.	2015-03-04 17:25:47.316864	2015-03-04 17:27:02.847346
74	24576830	In vitro study	Nissan et al., 2014, Cancer Res.	2015-03-04 17:25:47.453667	2015-03-04 17:27:22.465093
75	24388723	Retrospective study	Nagore et al., 2014, J. Am. Acad. Dermatol.	2015-03-04 17:25:47.583014	2015-03-04 17:27:23.930394
76	23524406	Case report	Rudin et al., 2013, J Thorac Oncol	2015-03-04 17:25:47.676555	2015-03-04 17:27:24.348173
77	21594703	Retrospective study	Howell et al., 2011, Ann. Surg. Oncol.	2015-03-04 17:25:47.746197	2015-03-04 17:27:24.767553
78	24570209	Retrospective study	Crescenzi et al., 2014, Horm. Metab. Res.	2015-03-04 17:25:47.814479	2015-03-04 17:27:25.182469
79	24588959	Perspective study	Zhang et al., 2014, Diagn Pathol	2015-03-04 17:25:47.87625	2015-03-04 17:27:25.599232
80	24354346	Retrospective study	Walczyk et al., 2014, Clin. Endocrinol. (Oxf)	2015-03-04 17:25:47.946194	2015-03-04 17:27:26.008095
81	24396464	Retrospective study	He et al., 2014, Oncol Lett	2015-03-04 17:25:48.122514	2015-03-04 17:27:26.374463
82	21098728	In vitro study	Corcoran et al., 2010, Sci Signal	2015-03-04 17:25:48.269582	2015-03-04 17:27:26.757768
83	23031422	Case report	Ponti et al., 2012, J Hematol Oncol	2015-03-04 17:25:48.343496	2015-03-04 17:27:27.149657
84	23346317	Review	Goff, 2013, J Gynecol Oncol	2015-03-04 17:25:48.589809	2015-03-04 17:27:27.531955
85	17070615		Gautschi et al., 2007, Lung Cancer	2015-03-04 17:25:48.838951	2015-03-04 17:27:27.924983
86	15961768		Arnold et al., 2005, J. Clin. Oncol.	2015-03-04 17:25:49.069564	2015-03-04 17:27:28.316463
87	16309541		Thomas et al., 2005, Int J Exp Pathol	2015-03-04 17:25:49.228656	2015-03-04 17:27:30.227245
88	17891190		Jares et al., 2007, Nat. Rev. Cancer	2015-03-04 17:25:49.329244	2015-03-04 17:27:30.720769
89	10547574		Takano et al., 1999, J. Pathol.	2015-03-04 17:25:49.435638	2015-03-04 17:27:32.270849
90	14612939		Oshimo et al., 2003, Int. J. Oncol.	2015-03-04 17:25:49.792677	2015-03-04 17:27:32.703906
91	23079656		Sawai et al., 2012, Cancer Cell	2015-03-04 17:25:49.880808	2015-03-04 17:27:33.125609
92	12432043		Keyomarsi et al., 2002, N. Engl. J. Med.	2015-03-04 17:25:49.968953	2015-03-04 17:27:33.511103
93	10224221		Donnellan et al., 1999, FASEB J.	2015-03-04 17:25:50.06263	2015-03-04 17:27:34.203876
94	25002028		Vora et al., 2014, Cancer Cell	2015-03-04 17:25:50.290445	2015-03-04 17:27:34.688772
95	23898052		Logan et al., 2013, Anticancer Res.	2015-03-04 17:25:50.369427	2015-03-04 17:27:35.015752
96	23111194		Bradly et al., 2012, Diagn. Mol. Pathol.	2015-03-04 17:25:50.582851	2015-03-04 17:27:35.419403
97	19965647	Retrospective study	Burnett et al., 2010, Blood	2015-03-04 17:25:50.731868	2015-03-04 17:27:35.802818
98	18450602	Retrospective study	Schlenk et al., 2008, N. Engl. J. Med.	2015-03-04 17:25:50.897909	2015-03-04 17:27:36.16788
99	14726504	Retrospective study	Fröhling et al., 2004, J. Clin. Oncol.	2015-03-04 17:25:50.967384	2015-03-04 17:27:36.566303
100	24186003		Lilljebjörn et al., 2014, Leukemia	2015-03-04 17:25:51.200132	2015-03-04 17:27:37.348464
101	22328973		Hammerman et al., 2011, Cancer Discov	2015-03-04 17:25:51.350352	2015-03-04 17:27:37.902896
102	24578576	Retrospective study	Honeyman et al., 2014, Science	2015-03-04 17:25:52.021122	2015-03-04 17:27:38.634553
103	22081665	Retrospective study	LaRochelle et al., 2011, Oncotarget	2015-03-04 17:25:52.113	2015-03-04 17:27:39.047515
104	23632886	Retrospective study	Gaidzik et al., 2013, Blood	2015-03-04 17:25:52.244775	2015-03-04 17:27:39.458826
105	21067377	Retrospective study	Ley et al., 2010, N. Engl. J. Med.	2015-03-04 17:25:52.307315	2015-03-04 17:27:39.882983
106	24512939	Retrospective study	El Ghannam et al., Blood Cells Mol. Dis.	2015-03-04 17:25:52.655337	2015-03-04 17:27:40.248792
107	22490330	Retrospective study		2015-03-04 17:25:52.725924	2015-03-04 17:27:40.592658
108	22291079	Retrospective study		2015-03-04 17:25:52.793798	2015-03-04 17:27:40.922478
109	16730237			2015-03-04 17:25:54.080956	2015-03-04 17:27:41.296159
110	15118125		Paez et al., 2004, Science	2015-03-04 17:25:54.144791	2015-03-04 17:27:41.687511
111	24736073		Lim et al., 2014, J Thorac Oncol	2015-03-04 17:25:54.210668	2015-03-04 17:27:42.105578
112	24457318		Fukihara et al., 2014, Oncology	2015-03-04 17:25:54.371373	2015-03-04 17:27:42.529339
113	24662454		Douillard et al., 2014, J Thorac Oncol	2015-03-04 17:25:54.433242	2015-03-04 17:27:42.917374
114	25668228	Review		2015-03-04 17:25:54.557693	2015-03-04 17:27:43.356068
115	24636847		Li et al., 2014, Lung Cancer	2015-03-04 17:25:54.635311	2015-03-04 17:27:43.793127
116	15728811		Kobayashi et al., 2005, N. Engl. J. Med.	2015-03-04 17:25:54.778835	2015-03-04 17:27:44.281403
117	24623981		Ding et al., 2014, Onco Targets Ther	2015-03-04 17:25:54.84613	2015-03-04 17:27:44.680545
118	24729716		Li et al., 2014, Onco Targets Ther	2015-03-04 17:25:54.915888	2015-03-04 17:27:45.133275
119	24658966		Ai et al., 2014, Amino Acids	2015-03-04 17:25:54.987566	2015-03-04 17:27:45.492822
120	21430269		Sequist et al., 2011, Sci Transl Med	2015-03-04 17:25:55.060767	2015-03-04 17:27:45.889284
121	23220880		Bose et al., 2013, Cancer Discov	2015-03-04 17:25:55.135461	2015-03-04 17:27:46.388852
122	24185512		Toy et al., 2013, Nat. Genet.	2015-03-04 17:25:56.083629	2015-03-04 17:27:46.912023
123	24185510		Robinson et al., 2013, Nat. Genet.	2015-03-04 17:25:56.152246	2015-03-04 17:27:47.305225
124	24550739	Case study	Borad et al., 2014, PLoS Genet.	2015-03-04 17:25:57.146057	2015-03-04 17:27:47.72806
125	23558953		Wu et al., 2013, Cancer Discov	2015-03-04 17:25:57.227465	2015-03-04 17:27:48.180676
126	12357354	In vitro study	Tse et al., 2002, Leukemia	2015-03-04 17:25:57.464678	2015-03-04 17:27:48.580789
127	19965647		Burnett et al., 2010, Blood	2015-03-04 17:25:57.536786	2015-03-04 17:27:48.964822
128	14726387	Clinical trial	Smith et al., 2004, Blood	2015-03-04 17:25:57.622937	2015-03-04 17:27:49.411696
129	15626738	In vitro study	Bagrintseva et al., 2005, Blood	2015-03-04 17:25:57.681568	2015-03-04 17:27:49.836142
130	11290608	Retrospective study	Yamamoto et al., 2001, Blood	2015-03-04 17:25:57.751846	2015-03-04 17:27:50.250918
131	17957027	Retrospective study	Gale et al., 2008, Blood	2015-03-04 17:25:57.839282	2015-03-04 17:27:50.610585
132	21537333		Schnittger et al., 2011, Leukemia	2015-03-04 17:25:57.96888	2015-03-04 17:27:50.98119
133	24801015		Port et al., 2014, Ann. Hematol.	2015-03-04 17:25:58.394984	2015-03-04 17:27:51.402663
134	11585760	Retrospective study	Whitman et al., 2001, Cancer Res.	2015-03-04 17:25:58.530466	2015-03-04 17:27:52.45306
135	18230792	In vitro study	Zhang et al., 2008, J. Natl. Cancer Inst.	2015-03-04 17:25:58.674687	2015-03-04 17:27:52.863825
136	17965322	Retrospective study	Bacher et al., 2008, Blood	2015-03-04 17:25:59.162955	2015-03-04 17:27:53.257465
137	17940205	Retrospective study	Whitman et al., 2008, Blood	2015-03-04 17:25:59.229916	2015-03-04 17:27:53.809824
138	22368270	Clinical trial	Man et al., 2012, Blood	2015-03-04 17:25:59.316798	2015-03-04 17:27:54.743438
139	22624710		Barbacid, 2012, Cancer Cell	2015-03-04 17:25:59.524424	2015-03-04 17:27:55.092998
140	20376086	Retrospective study	Ho et al., 2010, Leukemia	2015-03-04 17:25:59.650388	2015-03-04 17:27:55.482597
141	20805365	Retrospective study	Schnittger et al., 2010, Blood	2015-03-04 17:25:59.802151	2015-03-04 17:27:55.890962
142	20368538	Retrospective study	Wagner et al., 2010, J. Clin. Oncol.	2015-03-04 17:25:59.866273	2015-03-04 17:27:56.265295
143	19933982	Retrospective study	Dubbink et al., 2009, Neurology	2015-03-04 17:25:59.925418	2015-03-04 17:27:56.697175
144	20127344	Retrospective study	Bleeker et al., 2010, Acta Neuropathol.	2015-03-04 17:25:59.997313	2015-03-04 17:27:57.250651
145	20494930	Retrospective study	Thol et al., 2010, Haematologica	2015-03-04 17:26:00.068096	2015-03-04 17:27:57.734239
146	20538800	Retrospective study	Abbas et al., 2010, Blood	2015-03-04 17:26:00.137573	2015-03-04 17:27:58.132202
147	22616558		Zhou et al., 2012, Leuk. Lymphoma	2015-03-04 17:26:00.430253	2015-03-04 17:27:58.576256
148	20421455	Retrospective study	Thol et al., 2010, Blood	2015-03-04 17:26:00.494229	2015-03-04 17:27:58.968327
149	21596855		Green et al., 2011, Blood	2015-03-04 17:26:00.556792	2015-03-04 17:27:59.381503
150	21997850		Lin et al., 2012, Ann. Hematol.	2015-03-04 17:26:00.61983	2015-03-04 17:27:59.85697
151	22033490		Patnaik et al., 2012, Leukemia	2015-03-04 17:26:00.686075	2015-03-04 17:28:00.219071
152	16081687	Retrospective study	Levine et al., 2005, Blood	2015-03-04 17:26:00.935428	2015-03-04 17:28:01.249313
153	19287384	Retrospective study	Kilpivaara et al., 2009, Nat. Genet.	2015-03-04 17:26:01.133154	2015-03-04 17:28:01.737627
154	16709929	Clinical trial	Kiladjian et al., 2006, Blood	2015-03-04 17:26:01.213215	2015-03-04 17:28:02.159435
155	18394554	In vitro study	Wernig et al., 2008, Cancer Cell	2015-03-04 17:26:01.277059	2015-03-04 17:28:02.577273
156	16384925	Retrospective study	Cairoli et al., 2006, Blood	2015-03-04 17:26:01.339271	2015-03-04 17:28:03.117643
157	10485475	Retrospective study	Taniguchi et al., 1999, Cancer Res.	2015-03-04 17:26:01.431003	2015-03-04 17:28:03.491491
158	12000708	Retrospective study	Corless et al., 2002, Am. J. Pathol.	2015-03-04 17:26:01.509664	2015-03-04 17:28:03.937329
159	16551858	Retrospective study	Wardelmann et al., 2006, Clin. Cancer Res.	2015-03-04 17:26:01.58375	2015-03-04 17:28:04.351245
160	15217946	Retrospective study	Boldrini et al., 2004, Clin. Cancer Res.	2015-03-04 17:26:01.733487	2015-03-04 17:28:05.598223
161	23775962	Clinical trial	Hodi et al., 2013, J. Clin. Oncol.	2015-03-04 17:26:01.821104	2015-03-04 17:28:06.589007
162	18421059	Observational study	Hodi et al., 2008, J. Clin. Oncol.	2015-03-04 17:26:02.403307	2015-03-04 17:28:07.27641
163	19671763	In vitro study	Woodman et al., 2009, Mol. Cancer Ther.	2015-03-04 17:26:02.478167	2015-03-04 17:28:07.696203
164	17372901	In vitro study	Antonescu et al., 2007, Int. J. Cancer	2015-03-04 17:26:02.786633	2015-03-04 17:28:08.112495
165	16954519		Heinrich et al., 2006, J. Clin. Oncol.	2015-03-04 17:26:02.987523	2015-03-04 17:28:09.054352
166	16638875		Prenen et al., 2006, Clin. Cancer Res.	2015-03-04 17:26:03.075124	2015-03-04 17:28:09.469203
167	16618717	Retrospective study	Lièvre et al., 2006, Cancer Res.	2015-03-04 17:26:03.154849	2015-03-04 17:28:09.888366
168	24559322	Clinical trial	Sartore-Bianchi et al., 2014, Expert Rev Anticancer Ther	2015-03-04 17:26:03.278717	2015-03-04 17:28:10.27954
169	15696205	Retrospective study	Pao et al., 2005, PLoS Med.	2015-03-04 17:26:03.359442	2015-03-04 17:28:13.755053
170	24571676	Retrospective study	Andrade et al., 2014, BMC Cancer	2015-03-04 17:26:03.495249	2015-03-04 17:28:14.173187
171	19934290	Retrospective study	Ogino et al., 2009, Clin. Cancer Res.	2015-03-04 17:26:03.575407	2015-03-04 17:28:14.599162
172	18528420	Retrospective study	Chng et al., 2008, Leukemia	2015-03-04 17:26:03.65137	2015-03-04 17:28:15.007099
173	18794081	Retrospective study	Riely et al., 2008, Clin. Cancer Res.	2015-03-04 17:26:03.719144	2015-03-04 17:28:15.37092
174	11208838	Retrospective study	Schiller et al., 2001, J. Clin. Oncol.	2015-03-04 17:26:03.793792	2015-03-04 17:28:15.801281
175	15597105	Metaanalysis	Mascaux et al., 2005, Br. J. Cancer	2015-03-04 17:26:03.872358	2015-03-04 17:28:16.262818
176	18202412	In vitro study	Lièvre et al., 2008, J. Clin. Oncol.	2015-03-04 17:26:04.033656	2015-03-04 17:28:16.677883
177	19794967		Marchetti et al., 2009, Neoplasia	2015-03-04 17:26:04.179211	2015-03-04 17:28:17.096249
178	23014527	Retrospective study	Dogan et al., 2012, Clin. Cancer Res.	2015-03-04 17:26:04.338821	2015-03-04 17:28:17.521804
179	23524406	case report	Rudin et al., 2013, J Thorac Oncol	2015-03-04 17:26:04.655333	2015-03-04 17:28:18.035828
180	19029981	In vivo study	Engelman et al., 2008, Nat. Med.	2015-03-04 17:26:04.747709	2015-03-04 17:28:18.387281
181	20978259	Clinical trial	De Roock et al., 2010, JAMA	2015-03-04 17:26:05.530011	2015-03-04 17:28:18.884629
182	19915144		Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-03-04 17:26:05.841657	2015-03-04 17:28:19.291978
183	19915144	In vitro study	Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-03-04 17:26:05.918665	2015-03-04 17:28:19.700663
184	11070098		Esteller et al., 2000, N. Engl. J. Med.	2015-03-04 17:26:06.226033	2015-03-04 17:28:20.059003
185	15758010	Clinical Trial	Hegi et al., 2005, N. Engl. J. Med.	2015-03-04 17:26:06.448237	2015-03-04 17:28:21.50901
186	20007775		Westhoff et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-03-04 17:26:06.525488	2015-03-04 17:28:22.005882
187	19059939	Retrospective study	Schlenk et al., 2009, Haematologica	2015-03-04 17:26:06.926959	2015-03-04 17:28:23.507182
188	22417203	Retrospective study	Patel et al., 2012, N. Engl. J. Med.	2015-03-04 17:26:06.998891	2015-03-04 17:28:24.660354
189	16455956	Retrospective study	Thiede et al., 2006, Blood	2015-03-04 17:26:07.073424	2015-03-04 17:28:25.108044
190	16076867	Retrospective study	Schnittger et al., 2005, Blood	2015-03-04 17:26:07.146019	2015-03-04 17:28:25.55765
191	19357394		Vardiman et al., 2009, Blood	2015-03-04 17:26:07.214692	2015-03-04 17:28:25.973238
192	24927407		Ehninger et al., 2014, Blood Cancer J	2015-03-04 17:26:07.579421	2015-03-04 17:28:26.394859
193	15659725		Falini et al., 2005, N. Engl. J. Med.	2015-03-04 17:26:08.206276	2015-03-04 17:28:26.810692
194	19047294	Retrospective study	Büchner et al., 2009, J. Clin. Oncol.	2015-03-04 17:26:08.428452	2015-03-04 17:28:27.231113
195	24855211		Linch et al., 2014, Blood	2015-03-04 17:26:08.50773	2015-03-04 17:28:27.648641
196	22430270	Retrospective study	Gaidzik et al., 2012, J. Clin. Oncol.	2015-03-04 17:26:08.581519	2015-03-04 17:28:29.000818
197	20026798	Retrospective study	Becker et al., 2010, J. Clin. Oncol.	2015-03-04 17:26:08.773342	2015-03-04 17:28:29.405889
198	16051734	Retrospective study	Döhner et al., 2005, Blood	2015-03-04 17:26:08.850834	2015-03-04 17:28:29.788855
199	19587375	Retrospective study	Schnittger et al., 2009, Blood	2015-03-04 17:26:08.994123	2015-03-04 17:28:30.415119
200	24859829		Tian et al., 2014, Int. J. Hematol.	2015-03-04 17:26:09.214594	2015-03-04 17:28:30.880332
201	21719597	In vitro study	Balusu et al., 2011, Blood	2015-03-04 17:26:09.372953	2015-03-04 17:28:31.340984
202	24797300		Tassara et al., 2014, Blood	2015-03-04 17:26:09.505871	2015-03-04 17:28:31.71867
203	22180178	Clinical trial	Jakob et al., 2012, Cancer	2015-03-04 17:26:09.926341	2015-03-04 17:28:32.135695
204	15951308	Retrospective study	Bowen et al., 2005, Blood	2015-03-04 17:26:09.995836	2015-03-04 17:28:32.553365
205	24666267	Meta analysis	Therkildsen et al., 2014, Acta Oncol	2015-03-04 17:26:10.071451	2015-03-04 17:28:35.997488
206	16434492	Clinical trial	Bacher et al., 2006, Blood	2015-03-04 17:26:10.414254	2015-03-04 17:28:36.41649
207	18375819	Case report	Banerji et al., 2008, Mol. Cancer Ther.	2015-03-04 17:26:10.486965	2015-03-04 17:28:36.939667
208	23861977	Retrospective study	Tschandl et al., 2013, PLoS ONE	2015-03-04 17:26:10.721884	2015-03-04 17:28:37.465721
209	23569304	Clinical trial	Trunzer et al., 2013, J. Clin. Oncol.	2015-03-04 17:26:10.787201	2015-03-04 17:28:37.881467
210	21576590	Case report	Soon et al., 2011, Arch Dermatol	2015-03-04 17:26:10.865119	2015-03-04 17:28:38.611794
211	22745105		Heinrich et al., 2012, Clin. Cancer Res.	2015-03-04 17:26:11.03087	2015-03-04 17:28:39.131817
212	15928335		Corless et al., 2005, J. Clin. Oncol.	2015-03-04 17:26:11.19365	2015-03-04 17:28:39.678772
213	15146165		Lasota et al., 2004, Lab. Invest.	2015-03-04 17:26:11.355564	2015-03-04 17:28:41.787836
214	21558396		Tanaka et al., 2011, Clin. Cancer Res.	2015-03-04 17:26:11.685513	2015-03-04 17:28:42.171014
215	15647370		Kang et al., 2005, Proc. Natl. Acad. Sci. U.S.A.	2015-03-04 17:26:11.765342	2015-03-04 17:28:42.516849
216	19223544		Sartore-Bianchi et al., 2009, Cancer Res.	2015-03-04 17:26:11.8518	2015-03-04 17:28:42.913064
217	22357840		Liao et al., 2012, Clin. Cancer Res.	2015-03-04 17:26:11.980127	2015-03-04 17:28:43.310703
218	8674046		Yoshida et al., 1996, Cancer Res.	2015-03-04 17:26:13.270004	2015-03-04 17:28:43.938042
219	20085938		Courtney et al., 2010, J. Clin. Oncol.	2015-03-04 17:26:13.427166	2015-03-04 17:28:45.089623
220	22479427		Carico et al., 2012, PLoS ONE	2015-03-04 17:26:13.511564	2015-03-04 17:28:45.55083
221	21422803		Coxon et al., 2012, J. Endocrinol. Invest.	2015-03-04 17:26:13.704877	2015-03-04 17:28:47.323164
222	23056499		Couto et al., 2012, PLoS ONE	2015-03-04 17:26:13.79665	2015-03-04 17:28:47.718185
223	18073307		Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-03-04 17:26:13.947275	2015-03-04 17:28:48.123281
224	9839497		Egawa et al., 1998, Jpn. J. Clin. Oncol.	2015-03-04 17:26:14.026521	2015-03-04 17:28:48.53598
225	21995386		Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-03-04 17:26:14.101145	2015-03-04 17:28:48.965265
226	22698404		Jackson et al., 2012, Cancer Cell	2015-03-04 17:26:14.257972	2015-03-04 17:28:49.353956
227	16489069		Olivier et al., 2006, Clin. Cancer Res.	2015-03-04 17:26:14.333607	2015-03-04 17:28:49.890958
228	9569050		Berns et al., 1998, Br. J. Cancer	2015-03-04 17:26:14.489598	2015-03-04 17:28:50.274863
229	22923433		Iyer et al., 2012, Science	2015-03-04 17:26:15.253637	2015-03-04 17:28:50.683233
230	19966866	Mouse model	Liang et al., 2010, Oncogene	2015-03-04 17:26:15.391536	2015-03-04 17:28:51.14147
231	22923433	Retrospective study	Iyer et al., 2012, Science	2015-03-04 17:26:15.475801	2015-03-04 17:28:51.565431
232	23029227		Qian et al., 2012, PLoS ONE	2015-03-04 17:26:15.642631	2015-03-04 17:28:51.98218
233	23861105		Wu et al., 2013, Am. J. Hematol.	2015-03-04 17:26:15.870956	2015-03-04 17:28:52.744032
234	18591546	Retrospective study	Virappane et al., 2008, J. Clin. Oncol.	2015-03-04 17:26:16.220126	2015-03-04 17:28:53.341795
235	19221039	Retrospective study	Gaidzik et al., 2009, Blood	2015-03-04 17:26:16.286602	2015-03-04 17:28:53.867955
236	19536888	Retrospective study	Renneville et al., 2009, Cancer	2015-03-04 17:26:16.42731	2015-03-04 17:28:54.228556
237	18559874	Retrospective study	Paschka et al., 2008, J. Clin. Oncol.	2015-03-04 17:26:16.638481	2015-03-04 17:28:54.855035
238	20664172		Di Nicolantonio et al., 2010, J. Clin. Invest.	2015-03-04 17:26:17.565504	2015-03-04 17:28:55.22225
239	21642685		Carvajal et al., 2011, JAMA	2015-03-04 17:26:17.644413	2015-03-04 17:28:55.951912
240	18510589		Lutzky et al., 2008, Pigment Cell Melanoma Res	2015-03-04 17:26:17.717373	2015-03-04 17:28:56.366182
241	24107445		Zecchin et al., 2013, Mol. Cancer Ther.	2015-03-04 17:26:17.791104	2015-03-04 17:28:56.790997
242	24132921		Dai et al., 2013, Clin. Cancer Res.	2015-03-04 17:26:17.864052	2015-03-04 17:28:57.525273
243	24035431		Gautschi et al., 2013, Lung Cancer	2015-03-04 17:26:17.934545	2015-03-04 17:28:58.038724
244	24622468		Voss et al., 2014, Clin. Cancer Res.	2015-03-04 17:26:18.009445	2015-03-04 17:28:58.520141
\.


--
-- Data for Name: variant_origins; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_origins (id, origin) FROM stdin;
1	Somatic
2	Germline
\.


--
-- Data for Name: variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variants (id, gene_id, name, description, created_at, updated_at) FROM stdin;
1	4	BCR-ABL	The BCR-ABL fusion protein, commonly referred to as the philadelphia chromosome, is the most well-studied fusion gene in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML), but despite its ability initiate disease in mice, its status an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and nilotinib) have seen some success in delivering a tumor response.	2015-03-04 17:25:39.509976	2015-03-04 17:25:39.509976
2	4	BCR-ABL T315I	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-03-04 17:25:39.559147	2015-03-04 17:25:39.559147
3	4	BCR-ABL E255K	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-03-04 17:25:39.589313	2015-03-04 17:25:39.589313
4	2	E17K	AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal, lung, and ovarian cancer. It has been convincingly shown to be an activating mutation resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and other AKT1 mutations, are the subject of much research and development for therapeutics.	2015-03-04 17:25:39.61424	2015-03-04 17:25:39.61424
5	1	EML4-ALK	The EML4-ALK fusion has been seen in non-small cell lung cancer, and appears to be an alternative mechanism for ALK activation. Cells with this fusion have been shown to be sensitive to the ALK inhibitor crizotinib.	2015-03-04 17:25:39.644077	2015-03-04 17:25:39.644077
6	1	EML4-ALK C1156Y	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-03-04 17:25:39.66899	2015-03-04 17:25:39.66899
7	1	EML4-ALK L1196M	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-03-04 17:25:39.698097	2015-03-04 17:25:39.698097
8	1	F1174L	ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells.	2015-03-04 17:25:39.723044	2015-03-04 17:25:39.723044
9	1	R1275Q	ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma, non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This, and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC cell lines.	2015-03-04 17:25:39.748573	2015-03-04 17:25:39.748573
10	3	S214C	ARAF S214C has been found to be a recurrent oncogenic mutation in non-small cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this mutation, sorafenib also acheived near-complete clinical remission. This case has brought more interest to the variant from a research and clinical perspective.	2015-03-04 17:25:39.774814	2015-03-04 17:25:39.774814
11	5	V600D	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-03-04 17:25:39.811721	2015-03-04 17:25:39.811721
12	5	V600E	BRAF V600E has been shown to be recurrent in many cancer types. It is one of the most widely studied variants in cancer. This variant is correlated with poor prognosis in certain cancer types, including colorectal cancer and papillary thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib has also shown to be effective when combined with the MEK inhibitor trametinib in colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib, and a number of combination therapies have been successful in treating V600E mutations. While the drugs cetuximab and panitumumab have been largely shown to be ineffective without supplementary treatment.	2015-03-04 17:25:39.839854	2015-03-04 17:25:39.839854
13	5	V600E/V600M	A case study of a single patient harboring both a V600E and a V600M mutation, dabrafenib was shown to acheive clinical response.	2015-03-04 17:25:39.872888	2015-03-04 17:25:39.872888
14	5	V600E AMPLIFICATION	Amplification of BRAF V600E has been shown to confer resistance to MEK inhibitors. For more information on the V600 locus, see the V600E entry.	2015-03-04 17:25:39.899025	2015-03-04 17:25:39.899025
15	5	V600M	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-03-04 17:25:39.92531	2015-03-04 17:25:39.92531
16	5	V600R	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-03-04 17:25:39.947648	2015-03-04 17:25:39.947648
17	5	V600	BRAF mutations of the valine 600 residue have been shown to be recurrent in many cancer types. Of the V600 mutations, V600E is the most widely researched. V600 mutations as a whole have been correlated to poorer prognosis in colorectal and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual mutation pages on the left sidebar.	2015-03-04 17:25:39.971562	2015-03-04 17:25:39.971562
18	8	AMPLIFICATION	CCND1 amplification has been implicated in poorer prognosis in non-small cell lung cancer.	2015-03-04 17:25:39.999658	2015-03-04 17:25:39.999658
19	8	EXPRESSION	CCND1 expression, and its prognositc impact, is still in dispute. Three experiments in non-small cell lung cancer have shown it to have no impact on survival, but three additional studies have shown it results in poorer prognosis. There is also some ambiguity in how the boundaries between expression and overexpression are defined.	2015-03-04 17:25:40.029081	2015-03-04 17:25:40.029081
96	38	D842I	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-03-04 17:25:42.260108	2015-03-04 17:25:42.260108
20	8	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-03-04 17:25:40.055663	2015-03-04 17:25:40.055663
21	9	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-03-04 17:25:40.091341	2015-03-04 17:25:40.091341
22	9	PROMOTER DEMETHYLATION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-03-04 17:25:40.119412	2015-03-04 17:25:40.119412
23	10	LOSS	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL), a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic palbociclib significantly increased the median survival of a Notch-driven model of T-ALL.	2015-03-04 17:25:40.158073	2015-03-04 17:25:40.158073
24	11	OVEREXPRESSION	Cyclin E, while currenly not as widely implicated as its counterpart, cyclin D, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, are correlated with later stage disease and have been shown to lead to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	2015-03-04 17:25:40.186525	2015-03-04 17:25:40.186525
25	13	EXPRESSION	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-03-04 17:25:40.226156	2015-03-04 17:25:40.226156
26	12	EXPRESSION	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-03-04 17:25:40.254753	2015-03-04 17:25:40.254753
27	14	PROMOTER HYPERMETHYLATION	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the CDKN2A promoter region. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcomes for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	2015-03-04 17:25:40.278951	2015-03-04 17:25:40.278951
28	15	N-TERMINAL FRAME SHIFT	CEBPA N-terminal frame shift mutations that result in a premature stop codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML). CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-03-04 17:25:40.304443	2015-03-04 17:25:40.304443
29	15	MUTATION	CEBPA mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-03-04 17:25:40.328732	2015-03-04 17:25:40.328732
30	33	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-03-04 17:25:40.353603	2015-03-04 17:25:40.353603
31	16	DNAJB1-PRKACA	This fusion has been found to be very recurrent in the rare form of adolescent liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-03-04 17:25:40.386129	2015-03-04 17:25:40.386129
32	18	R882	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-03-04 17:25:40.414002	2015-03-04 17:25:40.414002
97	38	D842V	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-03-04 17:25:42.284936	2015-03-04 17:25:42.284936
33	19	L858R	EGFR L858R has long been recognized as a functionally significant mutation in cancer, and is one of the most prevalent single mutations in lung cancer. Best described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity to first and second generation TKI's like gefitinib and neratinib. NSCLC patients with this mutation treated with TKI's show increased overall and progression-free survival, as compared to chemotherapy alone. Third generation TKI's are currently in clinical trials that specifically focus on mutant forms of EGFR, a few of which have shown efficacy in treating patients that failed to respond to earlier generation TKI therapies.	2015-03-04 17:25:40.440619	2015-03-04 17:25:40.440619
34	19	T790M	EGFR T790M was one of the very first mutations recognized to confer resistance to targeted therapies. While successful in amplified EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients harboring this mutation before treatment is notably lower. This lack of efficacy can likely be to blame for the poorer prognosis for patients with this mutation as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired resistance to TKI inhibition have been shown to harbor this mutation, implicating it as a mechanism of acquired therapy resistence. The third generation TKI's are being developed with this resistance problem in mind, and early stage experiments have shown some efficacy of these drugs in previously resistant tumors with mutant EGFR.	2015-03-04 17:25:40.467931	2015-03-04 17:25:40.467931
35	20	D769H	ERBB2 D769H was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-03-04 17:25:40.49845	2015-03-04 17:25:40.49845
36	20	D769Y	ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-03-04 17:25:40.523287	2015-03-04 17:25:40.523287
37	20	DEL 755-759	ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-03-04 17:25:40.549156	2015-03-04 17:25:40.549156
38	20	G309A	ERBB2 G309A was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-03-04 17:25:40.5726	2015-03-04 17:25:40.5726
39	20	L755S	ERBB2 L755S was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike many of the other variants queried. This mutation was also shown to confer resistance to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.	2015-03-04 17:25:40.604315	2015-03-04 17:25:40.604315
40	20	L755W	ERBB2 L755W was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-03-04 17:25:40.629033	2015-03-04 17:25:40.629033
41	20	P780INS	ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-03-04 17:25:40.660595	2015-03-04 17:25:40.660595
42	20	R678Q	ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-03-04 17:25:40.68772	2015-03-04 17:25:40.68772
43	20	R896C	ERBB2 R896C was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-03-04 17:25:40.715764	2015-03-04 17:25:40.715764
44	20	V777L	ERBB2 V777L was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-03-04 17:25:40.742698	2015-03-04 17:25:40.742698
45	20	V842I	ERBB2 V842I was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-03-04 17:25:40.780689	2015-03-04 17:25:40.780689
46	21	L536Q	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of these ligand binding domain mutations, and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-03-04 17:25:40.805837	2015-03-04 17:25:40.805837
47	21	N538G	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-03-04 17:25:40.852633	2015-03-04 17:25:40.852633
48	21	Y537C	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-03-04 17:25:40.881795	2015-03-04 17:25:40.881795
49	21	Y537N	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-03-04 17:25:40.907554	2015-03-04 17:25:40.907554
50	21	Y537S	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-03-04 17:25:40.93375	2015-03-04 17:25:40.93375
51	22	FGFR2-MGEA5	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. Additionally, tumor size reduction was achieved by both ponatinib and pazopanib treatments administered separately in a single patient with intrahepatic cholangiocarcinoma and this fusion. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-03-04 17:25:40.961786	2015-03-04 17:25:40.961786
52	23	FGFR3-BAIAP2L1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-03-04 17:25:40.988436	2015-03-04 17:25:40.988436
53	23	FGFR3-TACC3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-03-04 17:25:41.016729	2015-03-04 17:25:41.016729
54	24	ITD MUTATIONS	FLT3-ITD (internal tandem duplications) frequently occur in patients with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated sequence resulting in an in-frame duplication event. The length of these duplications can vary widely which may have prognostic consequences, but this has not been conclusively determined. FLT3-ITD mutations overall have generally been associated with poor prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the prognosis associated with this variant.	2015-03-04 17:25:41.041824	2015-03-04 17:25:41.041824
55	24	TKD MUTATIONS	FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common than FLT3-ITD (internal tandem duplication) mutations and may not confer the same prognostic impact. Although the majority of mutations are point mutations effecting D835 (most frequently D835Y), a small proportion involve an in-frame deletion of I836. These mutations are within the activation loop of the second tyrosine kinase domain of FLT3 and thought to result in constitutive activation of the receptor.	2015-03-04 17:25:41.074427	2015-03-04 17:25:41.074427
56	25	EXPRESSION	GATA2 misregulation has been observed in a number of hematologic malignancies, as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung cancer.	2015-03-04 17:25:41.098687	2015-03-04 17:25:41.098687
57	26	R132	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-03-04 17:25:41.12444	2015-03-04 17:25:41.12444
58	26	R132C	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-03-04 17:25:41.151003	2015-03-04 17:25:41.151003
59	26	R132H	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-03-04 17:25:41.177352	2015-03-04 17:25:41.177352
60	26	R132L	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-03-04 17:25:41.20321	2015-03-04 17:25:41.20321
61	27	R140Q/L	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival as compared to their wild-type counterparts. In myelodysplastic syndromes, however, no prognostic link was discovered between IDH2 mutation status and overall survival.	2015-03-04 17:25:41.237182	2015-03-04 17:25:41.237182
62	27	R172K	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic syndromes, studies did not find a prognostic association between this variant and patient outcomes.	2015-03-04 17:25:41.264665	2015-03-04 17:25:41.264665
63	28	V617F	JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases, occuring in around half of all MPD's. While less associated with cancer, when it is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The V617F mutation is an activating mutation, resulting in increased kinase activity. The mutation seems to be restricted to hematologic malignancies. Treatment of JAK mutant diseases with ruxolitinib has seen some clinical success.	2015-03-04 17:25:41.294921	2015-03-04 17:25:41.294921
64	29	D816V	KIT D816V is a mutation observed in acute myeloid leukemia (AML). This variant has been linked to poorer prognosis and worse outcome in AML patients.	2015-03-04 17:25:41.319606	2015-03-04 17:25:41.319606
65	29	EXON 11 MUTATIONS	c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-03-04 17:25:41.34726	2015-03-04 17:25:41.34726
66	29	INTERNAL DUPLICATION	c-KIT internal duplications have been observed in exon 11, within the juxtamembrane domain. In a case study of an anal melanoma patient harboring this event, imatinib confered marked response. Also, cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-03-04 17:25:41.37939	2015-03-04 17:25:41.37939
67	29	EXON 13 MUTATIONS	c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations. These mutations lie within the tyrosine kinase 1 domain, and are found primarily in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations show sensitivity to imatinib and sunitinib treatment. However, only imatinib has seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant cases.	2015-03-04 17:25:41.41049	2015-03-04 17:25:41.41049
68	29	EXON 14 MUTATIONS	c-KIT exon 14 mutations lie within the tyrosine kinase domain of the protein. While relatively rare in primary gastrointestinal tumors, they are notably more prevalent in refractory disease, suggesting a role in imatinib resistance. Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.	2015-03-04 17:25:41.439741	2015-03-04 17:25:41.439741
69	29	EXON 17 MUTATIONS	c-KIT exon 17 mutations lie within the TK2 domain, containing the activation loop of the protein. In cell lines, mutations within this domain have been shown to be sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.	2015-03-04 17:25:41.484836	2015-03-04 17:25:41.484836
70	29	EXON 9 MUTATIONS	c-KIT exon 9 mutations lie within the dimerization motif of the protein. Relative to other KIT mutations, exon 9 mutations have been associated with better overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and in vivo.	2015-03-04 17:25:41.510242	2015-03-04 17:25:41.510242
71	29	L576P	KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain. It is one of the most recurrent KIT mutations in melanoma, and both in vitro and in vivo studies have shown sensitivity to imatinib.	2015-03-04 17:25:41.539493	2015-03-04 17:25:41.539493
72	29	V654A	KIT V654A is an exon 13 mutation that lies within the tyrosine kinase 1 domain of the protein. It has been shown to be an activating mutation by in vitro studies. This mutation is associated with imatinib resistance in melanoma patients. However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have seen success in acheiving tumor response.	2015-03-04 17:25:41.567953	2015-03-04 17:25:41.567953
73	30	EXON 1 MUTATIONS	A study by Lièvre et al in 2006 showed that colorectal cancer patients with KRAS exon 1 mutations had low cetuximab response rates.	2015-03-04 17:25:41.591636	2015-03-04 17:25:41.591636
74	30	EXON 2 MUTATIONS	In a study by Pao et al in 2005, non-small cell lung cancer patients harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors gefinitib and erlotinib.	2015-03-04 17:25:41.621699	2015-03-04 17:25:41.621699
75	30	G12	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-03-04 17:25:41.654048	2015-03-04 17:25:41.654048
76	30	G12/G13	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-03-04 17:25:41.687494	2015-03-04 17:25:41.687494
77	30	G12C	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-03-04 17:25:41.716399	2015-03-04 17:25:41.716399
78	30	G12D	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-03-04 17:25:41.743966	2015-03-04 17:25:41.743966
79	30	G13	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-03-04 17:25:41.773815	2015-03-04 17:25:41.773815
80	30	G13D	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-03-04 17:25:41.80036	2015-03-04 17:25:41.80036
81	31	P124S	MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder and colon cancer to a lesser degree. The P124S mutation has been shown to contribute to AZD6244 resistance in melanoma cell lines, but considerably less so than its Q56P counterpart.	2015-03-04 17:25:41.830165	2015-03-04 17:25:41.830165
82	31	Q56P	MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This mutation has been shown to confer considerable resistance to AZD6244 treatment of melanoma cell lines.	2015-03-04 17:25:41.858065	2015-03-04 17:25:41.858065
83	32	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-03-04 17:25:41.883597	2015-03-04 17:25:41.883597
84	34	PROMOTER METHYLATION	MGMT promoter methylation has been observed to impact tumor progression in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating agent carmustine has shown efficacy. In patients lacking methylation, combining carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more experiments are required. Clinical trials have also shown selective sensitivity of promoter methylation-positive patients to temozolomide, making a case for wider methylation screening in GBM patients.	2015-03-04 17:25:41.918709	2015-03-04 17:25:41.918709
85	35	EXON 12 MUTATIONS	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-03-04 17:25:41.944473	2015-03-04 17:25:41.944473
86	35	W288FS	NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most common NPM1 mutation identified in acute myeloid leukemia. This mutation results in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation, it is generally grouped with other exon 12 mutations for patient analysis (see NPM1 Exon 12 variants for more information).	2015-03-04 17:25:41.969976	2015-03-04 17:25:41.969976
87	36	EXON 1 MUTATIONS	NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown to be correlated with poorer overall survival relative to wild-type NRAS in melanoma patients.	2015-03-04 17:25:41.999748	2015-03-04 17:25:41.999748
88	36	EXON 2 MUTATIONS	NRAS exon 2 mutations have been shown to be correlated with poorer overall survival in melanoma patients and colorectal cancer patients, however no prognostic impact was seen in acute myeloid leukemia patients.	2015-03-04 17:25:42.026209	2015-03-04 17:25:42.026209
89	36	EXON 3 MUTATIONS	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-03-04 17:25:42.052202	2015-03-04 17:25:42.052202
90	36	EXON 4 MUTATIONS	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-03-04 17:25:42.076146	2015-03-04 17:25:42.076146
91	36	G12	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-03-04 17:25:42.126298	2015-03-04 17:25:42.126298
92	36	G13D	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-03-04 17:25:42.154923	2015-03-04 17:25:42.154923
93	36	Q61	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-03-04 17:25:42.177872	2015-03-04 17:25:42.177872
94	36	Q61L	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-03-04 17:25:42.20785	2015-03-04 17:25:42.20785
95	36	Q61R	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-03-04 17:25:42.233837	2015-03-04 17:25:42.233837
98	38	D842Y	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-03-04 17:25:42.319092	2015-03-04 17:25:42.319092
99	38	DEL I843	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-03-04 17:25:42.347652	2015-03-04 17:25:42.347652
100	38	DI842-843IM	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-03-04 17:25:42.380055	2015-03-04 17:25:42.380055
101	37	E542K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-03-04 17:25:42.405553	2015-03-04 17:25:42.405553
102	37	E545K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-03-04 17:25:42.435728	2015-03-04 17:25:42.435728
103	37	EXON 20 MUTATIONS	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-03-04 17:25:42.465209	2015-03-04 17:25:42.465209
104	37	EXON 9 MUTATIONS	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-03-04 17:25:42.492909	2015-03-04 17:25:42.492909
105	37	H1047R	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-03-04 17:25:42.517642	2015-03-04 17:25:42.517642
106	39	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	2015-03-04 17:25:42.545734	2015-03-04 17:25:42.545734
107	17	DNAJB1-PRKACA	This fusion has been found to be very recurrent in a rare form of adolescent liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined, and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-03-04 17:25:42.570979	2015-03-04 17:25:42.570979
108	41	R233*	PTEN R233* has been shown to be a loss of function mutation, and PTEN loss has been the subject of considerable research in breast cancer. PTEN loss may sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data to support that PTEN loss is both associated with poorer prognosis, and no change in prognosis.	2015-03-04 17:25:42.603209	2015-03-04 17:25:42.603209
109	40	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and Arsenic Trioxide for treating these patients, and early results seem promising.	2015-03-04 17:25:42.628494	2015-03-04 17:25:42.628494
110	42	C634W	RET C639W has been implicated as an alternate mechanism of activating RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the C639W mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib.	2015-03-04 17:25:42.653455	2015-03-04 17:25:42.653455
111	42	M918T	RET M819T is the most common somatically acquired mutation in medullary thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the M918T mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T leads to more aggressive MTC with a poorer prognosis.	2015-03-04 17:25:42.683965	2015-03-04 17:25:42.683965
112	44	K666N	SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients.	2015-03-04 17:25:42.728548	2015-03-04 17:25:42.728548
113	44	K700E	SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients. Additionally, these mutations are the most common SF3B1 mutation observed in MDS and highly associated with subtypes of MDS that are defined by ringed sideroblasts.	2015-03-04 17:25:42.753102	2015-03-04 17:25:42.753102
129	6	LOSS-OF-FUNCTION	BRCA1 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.	2015-03-04 17:25:43.209698	2015-03-04 17:25:43.209698
114	45	R175H	While loss-of-function events in TP53 are very common in cancer, the R175H variant seems not only to result in loss of tumor-suppression, but also acts as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell lines harboring this mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been shown that the R175H mutation is correlated with worse overall survival than wild-type TP53, but is not as detrimental as the R248W variant.	2015-03-04 17:25:42.777737	2015-03-04 17:25:42.777737
115	45	R248Q	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival. The R248Q mutation has also shown an increased invasive behavior in cell lines. This is specific to the 248Q variant.	2015-03-04 17:25:42.802467	2015-03-04 17:25:42.802467
116	45	R248W	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival.	2015-03-04 17:25:42.832415	2015-03-04 17:25:42.832415
117	45	R249T	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-03-04 17:25:42.865213	2015-03-04 17:25:42.865213
118	45	R249W	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-03-04 17:25:42.896476	2015-03-04 17:25:42.896476
119	45	R273C	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-03-04 17:25:42.923056	2015-03-04 17:25:42.923056
120	45	R273H	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-03-04 17:25:42.948205	2015-03-04 17:25:42.948205
121	45	V173G/A	While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-03-04 17:25:42.975363	2015-03-04 17:25:42.975363
122	46	FRAMESHIFT TRUNCATION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-03-04 17:25:42.999905	2015-03-04 17:25:42.999905
123	46	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-03-04 17:25:43.028412	2015-03-04 17:25:43.028412
124	47	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-03-04 17:25:43.056347	2015-03-04 17:25:43.056347
125	48	Q157P/R	U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is less common than the S34F mutation, occurs in the second zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-03-04 17:25:43.08425	2015-03-04 17:25:43.08425
126	48	S34Y/F	U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is the most commonly identified variant in MDS, occurs in the first zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-03-04 17:25:43.113419	2015-03-04 17:25:43.113419
127	49	EXON 7 MUTATIONS	WT1 exon 7 mutations have been shown to be recurrent in acute myeloid leukemia. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 7 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-03-04 17:25:43.144269	2015-03-04 17:25:43.144269
128	49	EXON 9 MUTATIONS	WT1 exon 9 mutations have been shown to be recurrent in acute myeloid leukemia, although at a less frequent rate than their exon 7 counterparts. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 9 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-03-04 17:25:43.176332	2015-03-04 17:25:43.176332
130	7	LOSS-OF-FUNCTION	BRCA2 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.	2015-03-04 17:25:43.243273	2015-03-04 17:25:43.243273
131	19	EXON 19 DELETION	Deletions within exon 19 of EGFR are most common in lung cancer. These deletions, in non-small cell lung cancer, have been shown to be sensitive to the EGFR tyrosine kinase inhibitors gefitinib and erlotinib. There is also data to suggest that this event is a good prognostic marker in lung adenocarcinoma.	2015-03-04 17:25:43.275215	2015-03-04 17:25:43.275215
132	19	G719S	While not as recurrent as the L858R mutation, EGFR G719S has also been shown to be an activating mutation. In lung cancer cell lines, it also confers sensitivity to the tyrosine kinase inhibitors gefitinib and erlotinib.	2015-03-04 17:25:43.31029	2015-03-04 17:25:43.31029
133	51	D1643H	Activating mutations in NOTCH1, including D1643H, have been shown to be poor prognostic markers in lung cancer.	2015-03-04 17:25:43.368803	2015-03-04 17:25:43.368803
134	51	R2328W	Activating mutations in NOTCH1, including R2328W, have been shown to be poor prognostic markers in lung cancer.	2015-03-04 17:25:43.402791	2015-03-04 17:25:43.402791
135	51	V2444FS	Activating mutations in NOTCH1, including a frameshift insertion at V2444, have been shown to be poor prognostic markers in lung cancer.	2015-03-04 17:25:43.42864	2015-03-04 17:25:43.42864
136	51	S2275FS	Activating mutations in NOTCH1, including a frameshift insertion at S2275, have been shown to be poor prognostic markers in lung cancer.	2015-03-04 17:25:43.459112	2015-03-04 17:25:43.459112
137	52	L63V	Activating mutations in DDR2, including L63V, has been shown to be sensitive to dasatinib in cell lines.	2015-03-04 17:25:43.486235	2015-03-04 17:25:43.486235
138	52	L239R	Activating mutations in DDR2, including L239R, has been shown to be sensitive to dasatinib in cell lines.	2015-03-04 17:25:43.509748	2015-03-04 17:25:43.509748
139	52	G253C	Activating mutations in DDR2, including G253C, has been shown to be sensitive to dasatinib in cell lines.	2015-03-04 17:25:43.535695	2015-03-04 17:25:43.535695
140	52	G505S	Activating mutations in DDR2, including G505S, has been shown to be sensitive to dasatinib in cell lines.	2015-03-04 17:25:43.560285	2015-03-04 17:25:43.560285
141	52	I638F	DDR2 I638F has been shown to be a loss of function mutation, but also confers sensitivity to dasatinib in cell lines.	2015-03-04 17:25:43.588508	2015-03-04 17:25:43.588508
142	52	G774V	Activating mutations in DDR2, including G774V, has been shown to be sensitive to dasatinib in cell lines.	2015-03-04 17:25:43.613359	2015-03-04 17:25:43.613359
143	52	S768R	Activating mutations in DDR2, including S768R, has been shown to be sensitive to dasatinib in cell lines.	2015-03-04 17:25:43.640421	2015-03-04 17:25:43.640421
144	53	AMPLIFICATION	MET amplification, like EGFR T790M, has been shown to be capable of driving acquired resistance to dacomitinib in patients with lung adenocarcinoma.	2015-03-04 17:25:43.668436	2015-03-04 17:25:43.668436
145	30	G12V	KRAS G12V, like EGFR T790M, has been shown to be capable of driving acquired resistance to tyrosine kinase inhibitors in lung adenocarcinoma.	2015-03-04 17:25:43.699006	2015-03-04 17:25:43.699006
146	30	G12A	KRAS G12A, like EGFR T790M, has been shown to be capable of driving acquired resistance to tyrosine kinase inhibitors in lung adenocarcinoma.	2015-03-04 17:25:43.727822	2015-03-04 17:25:43.727822
147	50	Q2223K	A summary for this variant has yet to be developed! Add one now by clicking the "Edit Variant" button.	2015-03-04 17:25:43.754979	2015-03-04 17:25:43.754979
148	38	G853D	A summary for this variant has yet to be developed! Add one now by clicking the "Edit Variant" button.	2015-03-04 17:25:43.780112	2015-03-04 17:25:43.780112
149	5	G469L	A summary for this variant has yet to be developed! Add one now by clicking the "Edit Variant" button.	2015-03-04 17:25:43.805761	2015-03-04 17:25:43.805761
150	29	K642E	A summary for this variant has yet to be developed! Add one now by clicking the "Edit Variant" button.	2015-03-04 17:25:43.842969	2015-03-04 17:25:43.842969
\.


--
-- Data for Name: evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_items (id, text, clinical_significance, evidence_direction, evidence_type_id, evidence_level_id, drug_id, disease_id, source_id, variant_id, created_at, updated_at, rating, status, remote_errors, remote_ids, variant_origin_id, variant_hgvs) FROM stdin;
1	Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML.	Positive	Does Not Support	1	1	1	1	51	1	2015-03-04 17:25:44.01694	2015-03-04 17:25:44.01694	2	accepted	\N	\N	1	N/A
2	The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease.	Positive	Supports	1	2	1	1	52	1	2015-03-04 17:25:44.099603	2015-03-04 17:25:44.099603	4	accepted	\N	\N	1	N/A
3	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.	Sensitivity	Supports	2	3	2	1	52	1	2015-03-04 17:25:44.176003	2015-03-04 17:25:44.176003	4	accepted	\N	\N	1	N/A
4	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.	Sensitivity	Supports	2	3	3	1	52	1	2015-03-04 17:25:44.237012	2015-03-04 17:25:44.237012	4	accepted	\N	\N	1	N/A
5	COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	Resistance or Non-Response	Supports	2	1	4	1	53	3	2015-03-04 17:25:44.306237	2015-03-04 17:25:44.306237	4	accepted	\N	\N	1	N/A
6	COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	Resistance or Non-Response	Supports	2	1	4	1	53	2	2015-03-04 17:25:44.412995	2015-03-04 17:25:44.412995	4	accepted	\N	\N	1	N/A
7	In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have shown to be resistant to imatinib treatment.	Resistance or Non-Response	Supports	2	3	4	1	52	2	2015-03-04 17:25:44.487539	2015-03-04 17:25:44.487539	4	accepted	\N	\N	1	N/A
8	PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the AKT inhibitor MK-2206 in breast cancer cell lines.	Sensitivity	Does Not Support	2	3	5	2	54	4	2015-03-04 17:25:44.583778	2015-03-04 17:25:44.583778	3	accepted	\N	\N	1	14:105246551-105246551 (C->T)
9	In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment with crizotinib has shown to be effective in acheiving stable disease.	Sensitivity	Supports	2	3	6	3	55	5	2015-03-04 17:25:44.657662	2015-03-04 17:25:44.657662	4	accepted	\N	\N	1	N/A
10	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y variant has been shown to confer resistance to crizotinib.	Resistance or Non-Response	Supports	2	3	6	3	55	6	2015-03-04 17:25:44.726435	2015-03-04 17:25:44.726435	4	accepted	\N	\N	1	N/A
11	CH5424802 treatment resulted in significant tumor regression in xenograft models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation	Sensitivity	Supports	2	1	7	3	56	7	2015-03-04 17:25:44.809475	2015-03-04 17:25:44.809475	3	accepted	\N	\N	1	N/A
12	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M variant has been shown to confer resistance to crizotinib.	Resistance or Non-Response	Supports	2	3	6	3	55	7	2015-03-04 17:25:44.870541	2015-03-04 17:25:44.870541	4	accepted	\N	\N	1	N/A
13	Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without this mutation.	Resistance or Non-Response	Supports	2	1	6	4	57	8	2015-03-04 17:25:44.944857	2015-03-04 17:25:44.944857	3	accepted	\N	\N	1	2:29443695-29443695 (G->T)
14	CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.	Sensitivity	Supports	2	1	7	5	56	8	2015-03-04 17:25:45.017984	2015-03-04 17:25:45.017984	3	accepted	\N	\N	1	2:29443695-29443695 (G->T)
15	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation.	Resistance or Non-Response	Supports	2	1	6	5	58	8	2015-03-04 17:25:45.112995	2015-03-04 17:25:45.112995	3	accepted	\N	\N	1	2:29443695-29443695 (G->T)
16	High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing the EML4-ALK fusion containing the F1174L mutation.	Sensitivity	Supports	2	1	6	5	59	8	2015-03-04 17:25:45.176576	2015-03-04 17:25:45.176576	3	accepted	\N	\N	1	2:29443695-29443695 (G->T)
17	SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition	Sensitivity	Supports	2	1	8	5	60	8	2015-03-04 17:25:45.248026	2015-03-04 17:25:45.248026	4	accepted	\N	\N	1	2:29443695-29443695 (G->T)
18	Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without this mutation.	Resistance or Non-Response	Supports	2	1	6	3	57	8	2015-03-04 17:25:45.314538	2015-03-04 17:25:45.314538	3	accepted	\N	\N	1	2:29443695-29443695 (G->T)
19	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation	Sensitivity	Supports	2	1	6	5	58	9	2015-03-04 17:25:45.389902	2015-03-04 17:25:45.389902	3	accepted	\N	\N	1	2:29432664-29432664 (C->T)
20	TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.	Sensitivity	Supports	2	1	8	5	60	9	2015-03-04 17:25:45.472125	2015-03-04 17:25:45.472125	3	accepted	\N	\N	1	2:29432664-29432664 (C->T)
21	The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.	Resistance or Non-Response	Supports	2	1	8	5	60	9	2015-03-04 17:25:45.546804	2015-03-04 17:25:45.546804	3	accepted	\N	\N	1	2:29432664-29432664 (C->T)
22	In one patient with S214C mutation, the use of sorafenib has led to more than 5 years of survival and near remission.	Sensitivity	Supports	2	3	9	3	61	10	2015-03-04 17:25:45.648137	2015-03-04 17:25:45.648137	2	accepted	\N	\N	1	X:47426121-47426121 (C->G)
23	Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	Sensitivity	Supports	2	1	9	3	61	10	2015-03-04 17:25:45.718553	2015-03-04 17:25:45.718553	3	accepted	\N	\N	1	X:47426121-47426121 (C->G)
24	Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	Sensitivity	Supports	2	1	10	3	61	10	2015-03-04 17:25:45.795107	2015-03-04 17:25:45.795107	3	accepted	\N	\N	1	X:47426121-47426121 (C->G)
25	In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations were associated with poor progression free survival regardless of treatment (panitumumab with best supportive care or best supportive care alone)	Resistance or Non-Response	Supports	2	3	11	6	62	17	2015-03-04 17:25:45.878198	2015-03-04 17:25:45.878198	3	accepted	\N	\N	1	N/A
26	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	Sensitivity	Supports	2	3	12	7	63	17	2015-03-04 17:25:45.963686	2015-03-04 17:25:45.963686	3	accepted	\N	\N	1	N/A
52	Thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	Positive	Supports	1	3	1	9	78	12	2015-03-04 17:25:47.817475	2015-03-04 17:25:47.817475	5	accepted	\N	\N	1	7:140453136-140453136 (A->T)
27	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	Sensitivity	Supports	2	3	10	7	63	17	2015-03-04 17:25:46.02576	2015-03-04 17:25:46.02576	3	accepted	\N	\N	1	N/A
28	BRAF mutations are associated with melanoma arising in non-chronic sun damaged skin and with superficial spreading melanoma	Positive	Supports	1	3	1	7	64	17	2015-03-04 17:25:46.097455	2015-03-04 17:25:46.097455	4	accepted	\N	\N	1	N/A
29	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	2	3	12	7	65	11	2015-03-04 17:25:46.194324	2015-03-04 17:25:46.194324	5	accepted	\N	\N	1	7:140453135-140453136 (CA->AT)
30	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	2	1	13	8	66	12	2015-03-04 17:25:46.289199	2015-03-04 17:25:46.289199	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
31	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	2	1	11	8	66	12	2015-03-04 17:25:46.350052	2015-03-04 17:25:46.350052	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
32	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	2	1	9	8	66	12	2015-03-04 17:25:46.414015	2015-03-04 17:25:46.414015	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
33	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	Sensitivity	Supports	2	1	14	8	67	12	2015-03-04 17:25:46.488598	2015-03-04 17:25:46.488598	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
34	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	Sensitivity	Supports	2	1	15	8	67	12	2015-03-04 17:25:46.545576	2015-03-04 17:25:46.545576	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
35	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	Sensitivity	Supports	2	1	15	8	68	12	2015-03-04 17:25:46.615704	2015-03-04 17:25:46.615704	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
36	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	Sensitivity	Supports	2	1	16	8	68	12	2015-03-04 17:25:46.692911	2015-03-04 17:25:46.692911	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
37	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	2	1	17	6	69	12	2015-03-04 17:25:46.767045	2015-03-04 17:25:46.767045	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
38	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	2	1	18	6	69	12	2015-03-04 17:25:46.82547	2015-03-04 17:25:46.82547	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
39	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	2	1	19	6	69	12	2015-03-04 17:25:46.888313	2015-03-04 17:25:46.888313	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
40	Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype BRAF	Resistance or Non-Response	Supports	2	3	13	6	70	12	2015-03-04 17:25:46.958946	2015-03-04 17:25:46.958946	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
41	V600E is associated with adverse pathological features of colorectal cancer. This can be concluded as a marker of poor prognosis.	Poor Outcome	Supports	3	2	1	6	71	12	2015-03-04 17:25:47.030816	2015-03-04 17:25:47.030816	5	accepted	\N	\N	1	7:140453136-140453136 (A->T)
42	Cell lines expressing the BRAF V600E mutation responded better to vemurafenib treatment than cells wildtype for BRAF as measured by reduced cellular proliferation and inhibition of MET and ERK phosphorylation	Sensitivity	Supports	2	1	18	6	69	12	2015-03-04 17:25:47.106132	2015-03-04 17:25:47.106132	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
43	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	N / A	Does Not Support	2	3	20	7	72	12	2015-03-04 17:25:47.194911	2015-03-04 17:25:47.194911	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
44	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	N / A	Does Not Support	2	3	21	7	72	12	2015-03-04 17:25:47.257362	2015-03-04 17:25:47.257362	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
45	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	Sensitivity	Supports	2	3	12	7	73	12	2015-03-04 17:25:47.319595	2015-03-04 17:25:47.319595	5	accepted	\N	\N	1	7:140453136-140453136 (A->T)
46	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	Sensitivity	Supports	2	3	10	7	73	12	2015-03-04 17:25:47.390224	2015-03-04 17:25:47.390224	5	accepted	\N	\N	1	7:140453136-140453136 (A->T)
47	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP but does not show resistance to MEK inhibitors.	Resistance or Non-Response	Does Not Support	2	1	1	7	74	12	2015-03-04 17:25:47.458178	2015-03-04 17:25:47.458178	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
48	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP and resistance to RAF inhibitors.	Resistance or Non-Response	Supports	2	1	1	7	74	12	2015-03-04 17:25:47.521245	2015-03-04 17:25:47.521245	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
49	BRAF V600E is correlated with shorter disease-free and overall Survival in a Spanish cohort of melanoma patients.	Poor Outcome	Supports	3	3	1	7	75	12	2015-03-04 17:25:47.586087	2015-03-04 17:25:47.586087	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
50	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	Resistance or Non-Response	Supports	2	3	12	3	76	12	2015-03-04 17:25:47.679335	2015-03-04 17:25:47.679335	2	accepted	\N	\N	1	7:140453136-140453136 (A->T)
51	BRAF V600E is shown to be associated with the tall-cell variant of papillary thyroid cancer	Positive	Supports	1	3	1	9	77	12	2015-03-04 17:25:47.750948	2015-03-04 17:25:47.750948	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
53	Thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	Positive	Supports	1	3	1	9	79	12	2015-03-04 17:25:47.880713	2015-03-04 17:25:47.880713	5	accepted	\N	\N	1	7:140453136-140453136 (A->T)
54	Unlike other studies that suggest a poorer outcome, BRAF mutation in this study was not correlated with poorer prognosis in papillary thyroid cancer.	Poor Outcome	Does Not Support	3	3	1	9	80	12	2015-03-04 17:25:47.949356	2015-03-04 17:25:47.949356	5	accepted	\N	\N	1	7:140453136-140453136 (A->T)
55	BRAF mutation correlated with poor prognosis in papillary thyroid cancer in both older (>65 yo) and younger (<65 yo) cohorts.	Poor Outcome	Supports	3	3	1	9	77	12	2015-03-04 17:25:48.023823	2015-03-04 17:25:48.023823	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
56	BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in a study of 187 patients with PTC and other thyroid diseases.	Poor Outcome	Supports	3	3	1	9	81	12	2015-03-04 17:25:48.126329	2015-03-04 17:25:48.126329	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
57	V600E is correlated with disease recurrence in both age cohorts (>65 and <65 yo).	Poor Outcome	Supports	3	3	1	9	77	12	2015-03-04 17:25:48.198541	2015-03-04 17:25:48.198541	3	accepted	\N	\N	1	7:140453136-140453136 (A->T)
58	COLO201 and COLO206F cells harboring BRAF V600E mutations were cloned to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification of the BRAF V600E gene.	Resistance or Non-Response	Supports	2	4	22	6	82	14	2015-03-04 17:25:48.274175	2015-03-04 17:25:48.274175	4	accepted	\N	\N	1	N/A
59	A single patient with BRAF V600E/V600M bi-allelic mutation responded to the V600E drug dabrafenib.	Sensitivity	Supports	2	3	12	7	83	13	2015-03-04 17:25:48.34752	2015-03-04 17:25:48.34752	3	accepted	\N	\N	1	N/A
60	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	2	3	12	7	65	15	2015-03-04 17:25:48.427528	2015-03-04 17:25:48.427528	5	accepted	\N	\N	1	7:140453137-140453137 (C->T)
61	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	2	3	12	7	65	16	2015-03-04 17:25:48.509132	2015-03-04 17:25:48.509132	5	accepted	\N	\N	1	7:140453136-140453137 (AC->CT)
62	The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation positive women with high grade serous or undifferentiated ovarian cancer.	Sensitivity	Supports	2	3	23	10	84	129	2015-03-04 17:25:48.595951	2015-03-04 17:25:48.595951	4	accepted	\N	\N	2	N/A
63	The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation positive women with high grade serous or undifferentiated ovarian cancer.	Sensitivity	Supports	2	3	23	10	84	130	2015-03-04 17:25:48.76237	2015-03-04 17:25:48.76237	4	accepted	\N	\N	2	N/A
64	Inceased copy number of CCND1 is associated with poorer overall survival.	Poor Outcome	Supports	3	3	1	3	85	18	2015-03-04 17:25:48.842581	2015-03-04 17:25:48.842581	3	accepted	\N	\N	1	N/A
65	Three studies have found no significant survival impact for CCND1 in lung cancer.	Poor Outcome	Does Not Support	3	3	1	3	85	19	2015-03-04 17:25:48.914237	2015-03-04 17:25:48.914237	3	accepted	\N	\N	1	N/A
66	Three studies have associated CCND1 expression with poorer survival.	Poor Outcome	Supports	3	3	1	3	85	19	2015-03-04 17:25:48.990401	2015-03-04 17:25:48.990401	3	accepted	\N	\N	1	N/A
67	Cyclin D1 overexpression is associated with the ER-positive subtype of breast cancer.	Positive	Supports	1	3	1	11	86	20	2015-03-04 17:25:49.076309	2015-03-04 17:25:49.076309	4	accepted	\N	\N	1	N/A
68	In ER-positive breast cancer patients, Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis.	Poor Outcome	Supports	3	3	1	11	86	20	2015-03-04 17:25:49.154249	2015-03-04 17:25:49.154249	4	accepted	\N	\N	1	N/A
69	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in head and neck squamous cell carcinoma.	Poor Outcome	Supports	3	3	1	12	87	20	2015-03-04 17:25:49.233779	2015-03-04 17:25:49.233779	4	accepted	\N	\N	1	N/A
70	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in mantle cell lymphoma.	Poor Outcome	Supports	3	3	1	13	88	20	2015-03-04 17:25:49.334437	2015-03-04 17:25:49.334437	4	accepted	\N	\N	1	N/A
71	Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.	Poor Outcome	Supports	3	1	1	14	89	21	2015-03-04 17:25:49.441167	2015-03-04 17:25:49.441167	4	accepted	\N	\N	1	N/A
72	Cyclin D2 overexpression is associated with the increased depth of cancer invasion in gastric cancers.	Poor Outcome	Supports	3	1	1	14	89	21	2015-03-04 17:25:49.535514	2015-03-04 17:25:49.535514	4	accepted	\N	\N	1	N/A
73	Cyclin D2 overexpression is associated with the presence pf lymph node metastasis in gastric cancers.	Poor Outcome	Supports	3	1	1	14	89	21	2015-03-04 17:25:49.619935	2015-03-04 17:25:49.619935	4	accepted	\N	\N	1	N/A
74	Cyclin D2 overexpression is associated with vascular invasion by cancer cells in gastric cancers.	Poor Outcome	Supports	3	1	1	14	89	21	2015-03-04 17:25:49.706197	2015-03-04 17:25:49.706197	4	accepted	\N	\N	1	N/A
75	CCND2 hypomethylation is seen to be more common in stage III and IV gastric tumors than stage I and II.	Negative	Supports	1	1	1	14	90	22	2015-03-04 17:25:49.796761	2015-03-04 17:25:49.796761	3	accepted	\N	\N	1	N/A
76	Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib (PD-0332991) significantly increased median survival.	Sensitivity	Supports	2	1	24	15	91	23	2015-03-04 17:25:49.887054	2015-03-04 17:25:49.887054	3	accepted	\N	\N	1	N/A
77	Patients with high expression of total cyclin E and low-molecular weight cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard ratio for these patients in 13.3 times higher than those with normal cyclin E levels.	Poor Outcome	Supports	3	3	1	11	92	24	2015-03-04 17:25:49.972546	2015-03-04 17:25:49.972546	4	accepted	\N	\N	1	N/A
78	Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced stage gastric carcinoma.	Poor Outcome	Supports	3	3	1	16	93	24	2015-03-04 17:25:50.068487	2015-03-04 17:25:50.068487	4	accepted	\N	\N	1	N/A
79	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	Poor Outcome	Supports	3	3	1	16	93	24	2015-03-04 17:25:50.140304	2015-03-04 17:25:50.140304	4	accepted	\N	\N	1	N/A
80	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	Poor Outcome	Supports	3	3	1	17	93	24	2015-03-04 17:25:50.230493	2015-03-04 17:25:50.230493	4	accepted	\N	\N	1	N/A
81	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	Sensitivity	Supports	2	1	25	18	94	25	2015-03-04 17:25:50.294603	2015-03-04 17:25:50.294603	4	accepted	\N	\N	1	N/A
82	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	Better Outcome	Supports	3	3	26	18	95	25	2015-03-04 17:25:50.372884	2015-03-04 17:25:50.372884	3	accepted	\N	\N	1	N/A
254	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	2	1	2	3	164	71	2015-03-04 17:26:02.911491	2015-03-04 17:26:02.911491	3	accepted	\N	\N	1	N/A
83	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	Sensitivity	Supports	2	1	25	18	94	26	2015-03-04 17:25:50.439739	2015-03-04 17:25:50.439739	4	accepted	\N	\N	1	N/A
84	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	Better Outcome	Supports	3	3	26	18	95	26	2015-03-04 17:25:50.507277	2015-03-04 17:25:50.507277	3	accepted	\N	\N	1	N/A
85	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter overall survival.	Poor Outcome	Supports	3	3	1	3	96	27	2015-03-04 17:25:50.586437	2015-03-04 17:25:50.586437	3	accepted	\N	\N	1	N/A
86	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter time to recurrence.	Poor Outcome	Supports	3	3	1	3	96	27	2015-03-04 17:25:50.660489	2015-03-04 17:25:50.660489	3	accepted	\N	\N	1	N/A
87	CEBPA mutation status had no impact on ATRA treatment response in older (>60) patients with AML	N / A	Does Not Support	2	3	1	19	97	29	2015-03-04 17:25:50.737221	2015-03-04 17:25:50.737221	4	accepted	\N	\N	1	N/A
88	CEBPA mutation was associated with improved overall survival in older (>60) patients with AML	Better Outcome	Supports	3	3	1	19	97	29	2015-03-04 17:25:50.828491	2015-03-04 17:25:50.828491	4	accepted	\N	\N	1	N/A
89	CEBPA mutation was significantly associated with complete remission	Better Outcome	Supports	3	3	1	19	98	29	2015-03-04 17:25:50.901426	2015-03-04 17:25:50.901426	4	accepted	\N	\N	1	N/A
90	FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically normal AML patients with CEBPA	N / A	Does Not Support	3	3	1	19	99	28	2015-03-04 17:25:50.970744	2015-03-04 17:25:50.970744	2	accepted	\N	\N	1	N/A
91	Overall survival was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA mutations	Better Outcome	Supports	3	3	1	19	99	28	2015-03-04 17:25:51.043148	2015-03-04 17:25:51.043148	3	accepted	\N	\N	1	N/A
92	Remission duration was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA	Better Outcome	Supports	3	3	1	19	99	28	2015-03-04 17:25:51.118018	2015-03-04 17:25:51.118018	3	accepted	\N	\N	1	N/A
93	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	2	1	4	20	100	30	2015-03-04 17:25:51.206204	2015-03-04 17:25:51.206204	3	accepted	\N	\N	1	N/A
94	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	2	1	27	20	100	30	2015-03-04 17:25:51.272103	2015-03-04 17:25:51.272103	3	accepted	\N	\N	1	N/A
95	In 3T3 fibroblasts expressing the G253C mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	Sensitivity	Supports	2	1	3	3	101	139	2015-03-04 17:25:51.354995	2015-03-04 17:25:51.354995	4	accepted	\N	\N	1	N/A
96	In 3T3 fibroblasts expressing the G505S mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	Sensitivity	Supports	2	1	3	3	101	140	2015-03-04 17:25:51.489236	2015-03-04 17:25:51.489236	4	accepted	\N	\N	1	N/A
97	In 3T3 fibroblasts expressing the G774V mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	Sensitivity	Supports	2	1	3	3	101	142	2015-03-04 17:25:51.605379	2015-03-04 17:25:51.605379	4	accepted	\N	\N	1	N/A
98	In 3T3 fibroblasts expressing the I638F mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	Sensitivity	Supports	2	1	3	3	101	141	2015-03-04 17:25:51.68492	2015-03-04 17:25:51.68492	4	accepted	\N	\N	1	N/A
99	In 3T3 fibroblasts expressing the L239R mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	Sensitivity	Supports	2	1	3	3	101	138	2015-03-04 17:25:51.752553	2015-03-04 17:25:51.752553	4	accepted	\N	\N	1	N/A
100	In 3T3 fibroblasts expressing the L63V mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	Sensitivity	Supports	2	1	3	3	101	137	2015-03-04 17:25:51.816199	2015-03-04 17:25:51.816199	4	accepted	\N	\N	1	N/A
101	In a patient expressing a DDR2 S768R mutation, treatment with dasatinib plus erlotinib acheived a partial response.	Sensitivity	Supports	2	3	3	3	101	143	2015-03-04 17:25:51.886942	2015-03-04 17:25:51.886942	3	accepted	\N	\N	1	N/A
102	In a patient expressing a DDR2 S768R mutation, treatment with dasatinib plus erlotinib acheived a partial response.	Sensitivity	Supports	2	3	28	3	101	143	2015-03-04 17:25:51.951039	2015-03-04 17:25:51.951039	3	accepted	\N	\N	1	N/A
103	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	Positive	Supports	1	3	1	21	102	31	2015-03-04 17:25:52.026047	2015-03-04 17:25:52.026047	4	accepted	\N	\N	1	N/A
104	Daunorubicin treatment resulted in similar overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	Resistance or Non-Response	Does Not Support	2	3	29	19	103	32	2015-03-04 17:25:52.116398	2015-03-04 17:25:52.116398	4	accepted	\N	\N	1	N/A
105	Idarubicin increases the overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	Sensitivity	Supports	2	3	30	19	103	32	2015-03-04 17:25:52.18386	2015-03-04 17:25:52.18386	4	accepted	\N	\N	1	N/A
106	Therapy-related AML was less common in patients with DNMT3A mutations (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of younger (18-60) AML patients	Negative	Supports	1	3	1	19	104	32	2015-03-04 17:25:52.247472	2015-03-04 17:25:52.247472	3	accepted	\N	\N	1	N/A
107	DNMT3A mutations (59% of which were R882) were associated with an intermediate risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes	Positive	Supports	1	3	1	19	105	32	2015-03-04 17:25:52.311123	2015-03-04 17:25:52.311123	5	accepted	\N	\N	1	N/A
108	DNMT3A mutations (59% of which were R882) were associated with intermediate risk cytogenetics (including normal karyotype)	Positive	Supports	1	3	1	19	105	32	2015-03-04 17:25:52.3768	2015-03-04 17:25:52.3768	4	accepted	\N	\N	1	N/A
109	DNMT3A mutations (64.5% R882) were associated with older age, higher white blood cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML patients	Positive	Supports	1	3	1	19	104	32	2015-03-04 17:25:52.444268	2015-03-04 17:25:52.444268	3	accepted	\N	\N	1	N/A
110	DNMT3A R882 mutations occur most often in de novo AML patients with intermediate cytogenic risk.	Positive	Supports	1	3	1	19	103	32	2015-03-04 17:25:52.508931	2015-03-04 17:25:52.508931	3	accepted	\N	\N	1	N/A
111	DNMT3A R882 mutations were associated with cytogenetically normal AML in a large cohort of younger (18-60) AML patients	Positive	Supports	1	3	1	19	104	32	2015-03-04 17:25:52.56648	2015-03-04 17:25:52.56648	3	accepted	\N	\N	1	N/A
112	DNMT3A R882 mutations were associated with older age, higher white blood cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically normal AML patients	Positive	Supports	1	3	1	19	106	32	2015-03-04 17:25:52.6592	2015-03-04 17:25:52.6592	3	accepted	\N	\N	1	N/A
113	Young AML patients (<60 years old) with DNMT3A mutations (60% of which were R882) were older in age, had higher white blood cell counts and had higher platelet counts than patients wildtype for DNMT3A	Positive	Supports	1	3	1	19	107	32	2015-03-04 17:25:52.729944	2015-03-04 17:25:52.729944	3	accepted	\N	\N	1	N/A
114	Complete remission rates did not differ between patients with wildtype or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML	N / A	Does Not Support	3	3	1	19	108	32	2015-03-04 17:25:52.798256	2015-03-04 17:25:52.798256	4	accepted	\N	\N	1	N/A
115	In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5% of which were R882) had no prognostic value on overall, relapse free and event free survival.	N / A	Does Not Support	3	3	1	19	104	32	2015-03-04 17:25:52.868642	2015-03-04 17:25:52.868642	3	accepted	\N	\N	1	N/A
116	In a large cohort of cytogenetically normal AML patients (18-60 years old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall, relapse free and event free survival	N / A	Does Not Support	3	3	1	19	104	32	2015-03-04 17:25:52.924851	2015-03-04 17:25:52.924851	3	accepted	\N	\N	1	N/A
117	In young AML patients (<60 years old), DNMT3A mutation status (60% of which were R882) was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N / A	Does Not Support	3	3	1	19	107	32	2015-03-04 17:25:52.988952	2015-03-04 17:25:52.988952	4	accepted	\N	\N	1	N/A
118	There is no difference in the complete remission rate of de novo AML patients with DNMT3A mutation compared to those who are wild type for DNMT3A.	N / A	Does Not Support	3	3	1	19	103	32	2015-03-04 17:25:53.055236	2015-03-04 17:25:53.055236	4	accepted	\N	\N	1	N/A
119	DNMT3A mutations were associated with achievement of complete remission in a large cohort of younger (18-60) AML patients	Better Outcome	Supports	3	3	1	19	104	32	2015-03-04 17:25:53.119403	2015-03-04 17:25:53.119403	3	accepted	\N	\N	1	N/A
120	AML patients with DNMT3A mutations (59% of which were R882) showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	Poor Outcome	Supports	3	3	1	19	105	32	2015-03-04 17:25:53.211541	2015-03-04 17:25:53.211541	5	accepted	\N	\N	1	N/A
121	Complete remission rate was not different between young AML patients (<60 years old) with or without DNMT3A mutations (60% of which were R882)	Poor Outcome	Supports	3	3	1	19	107	32	2015-03-04 17:25:53.279973	2015-03-04 17:25:53.279973	3	accepted	\N	\N	1	N/A
122	De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	Poor Outcome	Supports	3	3	1	19	105	32	2015-03-04 17:25:53.348752	2015-03-04 17:25:53.348752	4	accepted	\N	\N	1	N/A
123	DNMT3A mutations (62% of which were R882) were associated with reduced disease-free survival in patients with cytogenetically normal AML.	Poor Outcome	Supports	3	3	1	19	108	32	2015-03-04 17:25:53.409032	2015-03-04 17:25:53.409032	4	accepted	\N	\N	1	N/A
124	DNMT3A R882 mutation was associated with reduced relapse free and overall survival in ELN-unfavorable, cytogenetically normal AMLs	Poor Outcome	Supports	3	3	1	19	104	32	2015-03-04 17:25:53.474907	2015-03-04 17:25:53.474907	3	accepted	\N	\N	1	N/A
125	In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations were associated with reduced relapse free survival in the entire cohort as well as the subset of patients with cytogenetically normal AML	Poor Outcome	Supports	3	2	1	19	104	32	2015-03-04 17:25:53.534496	2015-03-04 17:25:53.534496	3	accepted	\N	\N	1	N/A
126	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including R882) were associated with worse overall survival compared to those without DNMT3A mutation.	Poor Outcome	Supports	3	3	1	19	105	32	2015-03-04 17:25:53.604364	2015-03-04 17:25:53.604364	5	accepted	\N	\N	1	N/A
127	In cytogenetically normal AML patients, DNMT3A R882 mutations are associated with lower overall and disease free survival as compared to patients with wild type DNMT3A.	Poor Outcome	Supports	3	3	1	19	106	32	2015-03-04 17:25:53.668913	2015-03-04 17:25:53.668913	5	accepted	\N	\N	1	N/A
128	In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation is prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	Poor Outcome	Supports	3	3	1	19	108	32	2015-03-04 17:25:53.751429	2015-03-04 17:25:53.751429	4	accepted	\N	\N	1	N/A
129	In young AML patients (<60 years old), DNMT3A mutations were associated with significantly reduced overall survival and relapse free survival in patients wildtype for NPM1 and FLT3	Poor Outcome	Supports	3	3	1	19	107	32	2015-03-04 17:25:53.812867	2015-03-04 17:25:53.812867	4	accepted	\N	\N	1	N/A
130	In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations other than R882 are prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	Poor Outcome	Supports	3	3	1	19	108	32	2015-03-04 17:25:53.882291	2015-03-04 17:25:53.882291	4	accepted	\N	\N	1	N/A
131	Young AML patients (<60 years old) with DNMT3A mutation have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	Poor Outcome	Supports	3	3	1	19	107	32	2015-03-04 17:25:53.95126	2015-03-04 17:25:53.95126	3	accepted	\N	\N	1	N/A
132	Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	Poor Outcome	Supports	3	3	1	19	107	32	2015-03-04 17:25:54.014906	2015-03-04 17:25:54.014906	3	accepted	\N	\N	1	N/A
133	Exon 19 deletion has been shown to be correlated with gefitinib response.	Sensitivity	Supports	2	1	31	3	109	131	2015-03-04 17:25:54.084089	2015-03-04 17:25:54.084089	4	accepted	\N	\N	1	N/A
134	Gefinitib has been shown to be effective in treating cell lines with G719S missense mutations.	Sensitivity	Supports	2	1	31	3	110	132	2015-03-04 17:25:54.147481	2015-03-04 17:25:54.147481	4	accepted	\N	\N	1	N/A
135	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.	N / A	Does Not Support	2	3	31	3	111	33	2015-03-04 17:25:54.213534	2015-03-04 17:25:54.213534	3	accepted	\N	\N	1	7:55259514-55259515 (CT->AG)
136	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.	N / A	Does Not Support	2	3	28	3	111	33	2015-03-04 17:25:54.294034	2015-03-04 17:25:54.294034	3	accepted	\N	\N	1	7:55259514-55259515 (CT->AG)
137	In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence of L858R mutation is prognostic for better progression free survival.	Sensitivity	Supports	2	3	1	3	112	33	2015-03-04 17:25:54.374128	2015-03-04 17:25:54.374128	4	accepted	\N	\N	1	7:55259514-55259515 (CT->AG)
138	Median survival of patients with EGFR L858R mutation is better than those with wild type EGFR.	Better Outcome	Supports	3	3	1	3	113	33	2015-03-04 17:25:54.437462	2015-03-04 17:25:54.437462	3	accepted	\N	\N	1	7:55259514-55259515 (CT->AG)
139	Gefinitib has been shown to be effective in treating cell lines with L858R missense mutations.	Sensitivity	Supports	2	1	31	3	110	33	2015-03-04 17:25:54.498185	2015-03-04 17:25:54.498185	4	accepted	\N	\N	1	N/A
195	Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor survival and inferior disease free survival in patients with AML.	Poor Outcome	Supports	3	3	1	19	134	54	2015-03-04 17:25:58.533149	2015-03-04 17:25:58.533149	4	accepted	\N	\N	1	N/A
140	The T790M mutation in EGFR has been shown to confer resistance to the tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are placed on the drug are likely to relapse.	Resistance or Non-Response	Supports	2	2	28	22	114	34	2015-03-04 17:25:54.561971	2015-03-04 17:25:54.561971	5	accepted	\N	\N	1	7:55249071-55249071 (C->T)
141	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.	Sensitivity	Supports	2	3	28	3	115	34	2015-03-04 17:25:54.638122	2015-03-04 17:25:54.638122	3	accepted	\N	\N	1	7:55249071-55249071 (C->T)
142	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.	Sensitivity	Supports	2	3	32	3	115	34	2015-03-04 17:25:54.69552	2015-03-04 17:25:54.69552	3	accepted	\N	\N	1	7:55249071-55249071 (C->T)
143	In non-small cell lung cancer, the appearance of T790M mutation leads to resistance to gefitinib.	Resistance or Non-Response	Supports	2	3	31	3	116	34	2015-03-04 17:25:54.782767	2015-03-04 17:25:54.782767	3	accepted	\N	\N	1	7:55249071-55249071 (C->T)
144	In NSCLC patients with T790M and another activating mutations, their progression free survival is shorter compared to those who do not possess T790M mutation.	Poor Outcome	Supports	3	3	1	3	117	34	2015-03-04 17:25:54.848844	2015-03-04 17:25:54.848844	4	accepted	\N	\N	1	7:55249071-55249071 (C->T)
145	Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall survival compared to patients with L858R or other exon 19 activating mutations.	Poor Outcome	Supports	3	3	1	3	118	34	2015-03-04 17:25:54.918836	2015-03-04 17:25:54.918836	4	accepted	\N	\N	1	7:55249071-55249071 (C->T)
146	In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR, especially when an L858R mutation is also present.	Sensitivity	Supports	2	4	33	3	119	34	2015-03-04 17:25:54.990157	2015-03-04 17:25:54.990157	1	accepted	\N	\N	1	7:55249071-55249071 (C->T)
147	In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804), MET amplification and EGFR T790M were found to be the primary drivers of resistance.	Resistance or Non-Response	Supports	2	3	34	3	120	34	2015-03-04 17:25:55.063295	2015-03-04 17:25:55.063295	4	accepted	\N	\N	1	N/A
148	In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	2	1	35	11	121	35	2015-03-04 17:25:55.137945	2015-03-04 17:25:55.137945	4	accepted	\N	\N	1	17:37880261-37880261 (G->C)
149	In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	2	1	35	11	121	36	2015-03-04 17:25:55.217992	2015-03-04 17:25:55.217992	4	accepted	\N	\N	1	17:37880261-37880261 (G->T)
150	In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2, cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive to neratinib.	Sensitivity	Supports	2	1	35	11	121	37	2015-03-04 17:25:55.314618	2015-03-04 17:25:55.314618	5	accepted	\N	\N	1	N/A
151	In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	2	1	35	11	121	38	2015-03-04 17:25:55.375211	2015-03-04 17:25:55.375211	5	accepted	\N	\N	1	17:37868205-37868205 (G->C)
152	The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	Resistance or Non-Response	Supports	2	1	36	11	121	39	2015-03-04 17:25:55.446873	2015-03-04 17:25:55.446873	5	accepted	\N	\N	1	17:37880220-37880220 (T->C)
153	The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	Sensitivity	Supports	2	1	35	11	121	39	2015-03-04 17:25:55.519978	2015-03-04 17:25:55.519978	5	accepted	\N	\N	1	17:37880220-37880220 (T->C)
154	MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	Sensitivity	Supports	2	1	35	11	121	40	2015-03-04 17:25:55.590365	2015-03-04 17:25:55.590365	5	accepted	\N	\N	1	N/A
155	In MCF10A cell lines, the in-frame insertion of a single amino acid at position 780 was shown to be sensitive to neratinib.	Sensitivity	Supports	2	1	35	11	121	41	2015-03-04 17:25:55.667298	2015-03-04 17:25:55.667298	5	accepted	\N	\N	1	N/A
156	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	Sensitivity	Supports	2	1	35	11	121	42	2015-03-04 17:25:55.734177	2015-03-04 17:25:55.734177	5	accepted	\N	\N	1	N/A
157	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	Sensitivity	Supports	2	1	36	11	121	42	2015-03-04 17:25:55.788474	2015-03-04 17:25:55.788474	5	accepted	\N	\N	1	N/A
158	In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	2	1	35	11	121	43	2015-03-04 17:25:55.874981	2015-03-04 17:25:55.874981	5	accepted	\N	\N	1	17:37881616-37881616 (C->T)
159	In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	2	1	35	11	121	44	2015-03-04 17:25:55.936494	2015-03-04 17:25:55.936494	5	accepted	\N	\N	1	17:37881000-37881000 (G->T)
160	In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	2	1	35	11	121	45	2015-03-04 17:25:56.011718	2015-03-04 17:25:56.011718	5	accepted	\N	\N	1	17:37881332-37881332 (G->A)
161	MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	2	1	37	11	122	46	2015-03-04 17:25:56.086145	2015-03-04 17:25:56.086145	3	accepted	\N	\N	1	N/A
162	The L536Q ligand binding domain mutation have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.	Sensitivity	Supports	2	1	38	11	123	46	2015-03-04 17:25:56.154789	2015-03-04 17:25:56.154789	5	accepted	\N	\N	1	N/A
163	The L536Q ligand binding domain mutation have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.	Sensitivity	Supports	2	1	39	11	123	46	2015-03-04 17:25:56.22461	2015-03-04 17:25:56.22461	5	accepted	\N	\N	1	N/A
164	MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	2	1	37	11	122	47	2015-03-04 17:25:56.302203	2015-03-04 17:25:56.302203	3	accepted	\N	\N	1	N/A
165	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	2	1	38	11	123	47	2015-03-04 17:25:56.403719	2015-03-04 17:25:56.403719	5	accepted	\N	\N	1	N/A
166	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	2	1	39	11	123	47	2015-03-04 17:25:56.458106	2015-03-04 17:25:56.458106	5	accepted	\N	\N	1	N/A
167	MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	2	1	37	11	122	48	2015-03-04 17:25:56.531168	2015-03-04 17:25:56.531168	3	accepted	\N	\N	1	6:152419923-152419923 (A->G)
168	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	2	1	38	11	123	48	2015-03-04 17:25:56.598561	2015-03-04 17:25:56.598561	5	accepted	\N	\N	1	6:152419923-152419923 (A->G)
169	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	2	1	39	11	123	48	2015-03-04 17:25:56.658845	2015-03-04 17:25:56.658845	5	accepted	\N	\N	1	6:152419923-152419923 (A->G)
170	MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	2	1	37	11	122	49	2015-03-04 17:25:56.726223	2015-03-04 17:25:56.726223	3	accepted	\N	\N	1	6:152419922-152419922 (T->A)
171	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	2	1	38	11	123	49	2015-03-04 17:25:56.798096	2015-03-04 17:25:56.798096	5	accepted	\N	\N	1	6:152419922-152419922 (T->A)
172	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	2	1	39	11	123	49	2015-03-04 17:25:56.873445	2015-03-04 17:25:56.873445	5	accepted	\N	\N	1	6:152419922-152419922 (T->A)
173	MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	2	1	37	11	122	50	2015-03-04 17:25:56.943008	2015-03-04 17:25:56.943008	3	accepted	\N	\N	1	6:152419923-152419923 (A->C)
174	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	2	1	38	11	123	50	2015-03-04 17:25:57.013808	2015-03-04 17:25:57.013808	5	accepted	\N	\N	1	6:152419923-152419923 (A->C)
175	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	2	1	39	11	123	50	2015-03-04 17:25:57.070742	2015-03-04 17:25:57.070742	5	accepted	\N	\N	1	6:152419923-152419923 (A->C)
176	In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic sites and overall stable disease. However, this response did not meet criteria for RECIST partial response.	Sensitivity	Supports	2	3	40	23	124	51	2015-03-04 17:25:57.150235	2015-03-04 17:25:57.150235	2	accepted	\N	\N	1	N/A
177	Following identification of this fusion in a patient, this fusion was expressed in HEK 293T cells leading to oligomerization in the absence of ligand, induced morphologic changes in the cell and increased cellular proliferation consistent with FGFR activation. Mouse xenografts using the bladder cell line SW780 which harbors an FGFR3-BAIAP2L1 fusion showed reduced tumor burden when treated with the FGFR inhibitor PD173074.	Sensitivity	Supports	2	1	41	24	125	52	2015-03-04 17:25:57.232449	2015-03-04 17:25:57.232449	3	accepted	\N	\N	1	N/A
178	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This fusion is also shown to be recurrent in multiple other cancer types.	Sensitivity	Supports	2	3	42	23	124	53	2015-03-04 17:25:57.334492	2015-03-04 17:25:57.334492	3	accepted	\N	\N	1	N/A
179	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This fusion is also shown to be recurrent in multiple other cancer types.	Sensitivity	Supports	2	3	40	23	124	53	2015-03-04 17:25:57.397372	2015-03-04 17:25:57.397372	3	accepted	\N	\N	1	N/A
180	AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with FLT3 internal tandem repeat	Sensitivity	Supports	2	1	43	19	126	54	2015-03-04 17:25:57.468597	2015-03-04 17:25:57.468597	4	accepted	\N	\N	1	N/A
181	ATRA treatment did not effect overall survival in patients <60 years old with FLT3-ITD mutation	Resistance or Non-Response	Supports	2	3	44	19	127	54	2015-03-04 17:25:57.539305	2015-03-04 17:25:57.539305	3	accepted	\N	\N	1	N/A
182	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	Sensitivity	Supports	2	3	45	19	128	54	2015-03-04 17:25:57.625454	2015-03-04 17:25:57.625454	4	accepted	\N	\N	1	N/A
183	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	Resistance or Non-Response	Supports	2	1	29	19	129	54	2015-03-04 17:25:57.685936	2015-03-04 17:25:57.685936	3	accepted	\N	\N	1	N/A
184	WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared to wild type	Positive	Supports	1	3	1	19	130	54	2015-03-04 17:25:57.754656	2015-03-04 17:25:57.754656	3	accepted	\N	\N	1	N/A
185	FLT3-ITD mutation without NPM1 was associated with increased relapse risk and reduced overall survival in young adult AML patients (median age 43)	Resistance or Non-Response	Supports	2	3	1	19	131	54	2015-03-04 17:25:57.841942	2015-03-04 17:25:57.841942	4	accepted	\N	\N	1	N/A
186	Increasing level of FLT3-ITD mutant correlated with reduced disease-free and overall survival as well as increased relapse rate	Resistance or Non-Response	Supports	2	3	1	19	131	54	2015-03-04 17:25:57.907938	2015-03-04 17:25:57.907938	3	accepted	\N	\N	1	N/A
187	FLT3-ITD duplication length did not effect prognosis in patients with an NPM1 mutation	N / A	Does Not Support	3	3	1	19	132	54	2015-03-04 17:25:57.973382	2015-03-04 17:25:57.973382	4	accepted	\N	\N	1	N/A
188	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N / A	Does Not Support	3	3	1	19	107	54	2015-03-04 17:25:58.037641	2015-03-04 17:25:58.037641	4	accepted	\N	\N	1	N/A
189	Size of the FLT3-ITD mutant duplication had no impact on overall survival or relapse rate	N / A	Does Not Support	3	3	1	19	131	54	2015-03-04 17:25:58.100646	2015-03-04 17:25:58.100646	2	accepted	\N	\N	1	N/A
190	A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation	Poor Outcome	Supports	3	3	1	19	132	54	2015-03-04 17:25:58.162942	2015-03-04 17:25:58.162942	4	accepted	\N	\N	1	N/A
191	AML patients with internal tandem repeat of FLT3 has worse overall survival than patients with wild type FLT3	Poor Outcome	Supports	3	3	1	19	130	54	2015-03-04 17:25:58.228798	2015-03-04 17:25:58.228798	3	accepted	\N	\N	1	N/A
192	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated with worse overall survival compared to those without DNMT3A mutation.	Poor Outcome	Supports	3	3	1	19	105	54	2015-03-04 17:25:58.311276	2015-03-04 17:25:58.311276	5	accepted	\N	\N	1	N/A
193	Meta-analysis of studies involving cytogentically normal younger (<60) patients showed reduced overall and relapse-free survival for patients with FLT3-ITD	Poor Outcome	Supports	3	2	1	19	133	54	2015-03-04 17:25:58.399461	2015-03-04 17:25:58.399461	2	accepted	\N	\N	1	N/A
194	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	Poor Outcome	Supports	3	3	1	19	132	54	2015-03-04 17:25:58.459403	2015-03-04 17:25:58.459403	4	accepted	\N	\N	1	N/A
196	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	Poor Outcome	Supports	3	3	1	19	132	54	2015-03-04 17:25:58.607002	2015-03-04 17:25:58.607002	4	accepted	\N	\N	1	N/A
197	Sorafenib is effective in patients with FLT3 internal tandem repeats, but not in wild type or D835 mutation.	Sensitivity	Supports	2	1	9	19	135	54	2015-03-04 17:25:58.677431	2015-03-04 17:25:58.677431	4	accepted	\N	\N	1	N/A
198	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	Resistance or Non-Response	Supports	2	1	46	19	129	54	2015-03-04 17:25:58.83618	2015-03-04 17:25:58.83618	3	accepted	\N	\N	1	N/A
199	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	Sensitivity	Supports	2	3	45	19	128	55	2015-03-04 17:25:58.896286	2015-03-04 17:25:58.896286	4	accepted	\N	\N	1	N/A
200	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	Resistance or Non-Response	Supports	2	1	29	19	129	55	2015-03-04 17:25:58.963827	2015-03-04 17:25:58.963827	3	accepted	\N	\N	1	N/A
201	WBC counts were not significantly different in AML patients with FLT3 D835 mutations compared to wild type	Positive	Does Not Support	1	3	1	19	130	55	2015-03-04 17:25:59.036639	2015-03-04 17:25:59.036639	3	accepted	\N	\N	1	N/A
202	AML patients with D835 mutation of FLT3 did not have worse overall survival than patients with wild type FLT3	N / A	Does Not Support	3	3	1	19	130	55	2015-03-04 17:25:59.103195	2015-03-04 17:25:59.103195	3	accepted	\N	\N	1	N/A
203	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	Better Outcome	Supports	3	3	1	19	136	55	2015-03-04 17:25:59.166032	2015-03-04 17:25:59.166032	3	accepted	\N	\N	1	N/A
204	In AML patients, FLT3-TKD mutation was associated with poorer disease free survival compared to patients with wild type FLT3	Poor Outcome	Supports	3	3	1	19	137	55	2015-03-04 17:25:59.232918	2015-03-04 17:25:59.232918	4	accepted	\N	\N	1	N/A
205	Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated with sorafenib resistance	Resistance or Non-Response	Supports	2	3	9	19	138	55	2015-03-04 17:25:59.319639	2015-03-04 17:25:59.319639	4	accepted	\N	\N	1	N/A
206	Sorafenib is effective in patients with FLT3 internal tandem repeat mutations, but not in patients with wild type or D835 mutation.	Resistance or Non-Response	Supports	2	1	9	19	135	55	2015-03-04 17:25:59.386755	2015-03-04 17:25:59.386755	4	accepted	\N	\N	1	N/A
207	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	Resistance or Non-Response	Supports	2	1	46	19	129	55	2015-03-04 17:25:59.452251	2015-03-04 17:25:59.452251	3	accepted	\N	\N	1	N/A
208	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	Sensitivity	Supports	2	3	47	25	139	56	2015-03-04 17:25:59.530079	2015-03-04 17:25:59.530079	4	accepted	\N	\N	1	N/A
209	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	Sensitivity	Supports	2	3	48	25	139	56	2015-03-04 17:25:59.588035	2015-03-04 17:25:59.588035	4	accepted	\N	\N	1	N/A
210	In leukemia patients, IDH1 R132 mutation is associated with adults and normal karyotype.	Positive	Supports	1	3	1	19	140	57	2015-03-04 17:25:59.65333	2015-03-04 17:25:59.65333	2	accepted	\N	\N	1	N/A
211	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	N / A	Does Not Support	3	3	1	19	141	57	2015-03-04 17:25:59.805446	2015-03-04 17:25:59.805446	3	accepted	\N	\N	1	N/A
212	The complete remission and overall survival rates in patients with IDH1 R132 mutation is not significantly different from those who do not have this mutation	N / A	Does Not Support	3	3	1	19	142	57	2015-03-04 17:25:59.869343	2015-03-04 17:25:59.869343	2	accepted	\N	\N	1	N/A
213	In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic for better survival compared to patients who harbor wild type mutation.	Better Outcome	Supports	3	3	1	26	143	57	2015-03-04 17:25:59.931365	2015-03-04 17:25:59.931365	3	accepted	\N	\N	1	N/A
214	In patients with glioblastoma, those harboring IDH1 R132 mutation has higher overall survival compared to those who do not have IDH1 mutation	Better Outcome	Supports	3	3	1	27	144	57	2015-03-04 17:26:00.002482	2015-03-04 17:26:00.002482	3	accepted	\N	\N	1	N/A
215	IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated with worse overall survival	Poor Outcome	Supports	3	3	1	28	145	57	2015-03-04 17:26:00.072403	2015-03-04 17:26:00.072403	4	accepted	\N	\N	1	N/A
216	IDH1 R132 mutation is associated with patients of older age, high platelet count during diagnosis, cytogenic normalcy and NPM1 mutation	Positive	Supports	1	3	1	19	146	58	2015-03-04 17:26:00.142235	2015-03-04 17:26:00.142235	3	accepted	\N	\N	1	2:209113113-209113113 (G->A)
217	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N / A	Does Not Support	3	3	1	19	146	58	2015-03-04 17:26:00.217173	2015-03-04 17:26:00.217173	2	accepted	\N	\N	1	2:209113113-209113113 (G->A)
218	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N / A	Does Not Support	3	3	1	19	146	59	2015-03-04 17:26:00.282618	2015-03-04 17:26:00.282618	3	accepted	\N	\N	1	2:209113112-209113112 (C->T)
219	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N / A	Does Not Support	3	3	1	19	146	60	2015-03-04 17:26:00.352029	2015-03-04 17:26:00.352029	3	accepted	\N	\N	1	2:209113112-209113112 (C->A)
220	AML patients with IDH2 mutations such as R140Q/L have event free survival and overall survival similar to those with wild-type IDH2	N / A	Does Not Support	3	3	1	19	147	61	2015-03-04 17:26:00.43271	2015-03-04 17:26:00.43271	2	accepted	\N	\N	1	N/A
221	In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact overall survival or disease free survival.	N / A	Does Not Support	3	3	1	19	148	61	2015-03-04 17:26:00.496815	2015-03-04 17:26:00.496815	3	accepted	\N	\N	1	N/A
222	In AML, patients with IDH2 R140K mutation have improved overall survival compared to those with wild-type IDH2	Better Outcome	Supports	3	3	1	19	149	61	2015-03-04 17:26:00.559953	2015-03-04 17:26:00.559953	3	accepted	\N	\N	1	N/A
223	In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L do not confer prognostic value (overall survival)	N / A	Does Not Support	3	3	1	29	150	61	2015-03-04 17:26:00.623735	2015-03-04 17:26:00.623735	2	accepted	\N	\N	1	N/A
224	R140Q mutation in IDH2 does not have prognostic value in patients with MDS.	N / A	Does Not Support	3	3	1	29	151	61	2015-03-04 17:26:00.688649	2015-03-04 17:26:00.688649	3	accepted	\N	\N	1	N/A
225	AML patients with IDH2 mutations such as R172K have event free survival and overall survival similar to those with wild-type IDH2.	N / A	Does Not Support	3	3	1	19	147	62	2015-03-04 17:26:00.745541	2015-03-04 17:26:00.745541	2	accepted	\N	\N	1	15:90631838-90631838 (C->T)
226	In AML, patients with an IDH2 R172K mutation have worse overall survival compared to those with wild-type IDH2	Poor Outcome	Supports	3	3	1	19	149	62	2015-03-04 17:26:00.809415	2015-03-04 17:26:00.809415	3	accepted	\N	\N	1	15:90631838-90631838 (C->T)
227	In patients suffering from MDS, the presence of IDH2 mutations such as R172K do not confer prognostic value (overall survival)	N / A	Does Not Support	3	3	1	29	150	62	2015-03-04 17:26:00.875578	2015-03-04 17:26:00.875578	2	accepted	\N	\N	1	15:90631838-90631838 (C->T)
228	JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical CML).	Positive	Supports	1	3	1	1	152	63	2015-03-04 17:26:00.939718	2015-03-04 17:26:00.939718	4	accepted	\N	\N	1	9:5073770-5073770 (G->T)
229	JAK2 V617F is associated with myeloid neoplasms (AML, MDS, CMML/aCML)	Positive	Supports	1	3	1	1	152	63	2015-03-04 17:26:01.003884	2015-03-04 17:26:01.003884	4	accepted	\N	\N	1	9:5073770-5073770 (G->T)
230	JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).	Negative	Supports	1	3	1	30	152	63	2015-03-04 17:26:01.068205	2015-03-04 17:26:01.068205	4	accepted	\N	\N	1	9:5073770-5073770 (G->T)
231	The SNP rs10974944 in the Jak2 locus is associated with increased predisposition for JAK2 V617F mutation and its associated cancer	Positive	Supports	1	3	1	31	153	63	2015-03-04 17:26:01.137649	2015-03-04 17:26:01.137649	4	accepted	\N	\N	1	9:5073770-5073770 (G->T)
232	In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction in the percentage of cells harboring JAK2 V617F.	Sensitivity	Supports	2	3	49	32	154	63	2015-03-04 17:26:01.217157	2015-03-04 17:26:01.217157	4	accepted	\N	\N	1	9:5073770-5073770 (G->T)
233	TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells	Sensitivity	Supports	2	1	50	32	155	63	2015-03-04 17:26:01.281198	2015-03-04 17:26:01.281198	3	accepted	\N	\N	1	9:5073770-5073770 (G->T)
234	In acute myloid leukemia patients, D816 mutation is associated with earlier relapse and poorer prognosis than wildtype KIT	Poor Outcome	Supports	3	3	1	33	156	64	2015-03-04 17:26:01.343207	2015-03-04 17:26:01.343207	4	accepted	\N	\N	1	4:55599321-55599321 (A->T)
235	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	Positive	Supports	1	3	1	34	157	65	2015-03-04 17:26:01.436084	2015-03-04 17:26:01.436084	2	accepted	\N	\N	1	N/A
236	KIT mutations were identified in morphologically benign, incidentally discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating is not a prognostic marker for GIST	N / A	Does Not Support	3	3	1	34	158	65	2015-03-04 17:26:01.513963	2015-03-04 17:26:01.513963	3	accepted	\N	\N	1	N/A
237	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	N / A	Does Not Support	3	3	1	34	159	65	2015-03-04 17:26:01.586262	2015-03-04 17:26:01.586262	3	accepted	\N	\N	1	N/A
238	KIT mutation is associated with worse overall and cause-specific prognosis in patients with GIST compared to patients with wildtype KIT	Poor Outcome	Supports	3	3	1	34	157	65	2015-03-04 17:26:01.663042	2015-03-04 17:26:01.663042	3	accepted	\N	\N	1	N/A
239	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	N / A	Does Not Support	3	3	1	17	160	65	2015-03-04 17:26:01.736503	2015-03-04 17:26:01.736503	2	accepted	\N	\N	1	N/A
240	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	3	3	4	7	161	65	2015-03-04 17:26:01.82455	2015-03-04 17:26:01.82455	3	accepted	\N	\N	1	N/A
241	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	3	3	4	7	161	67	2015-03-04 17:26:01.909067	2015-03-04 17:26:01.909067	3	accepted	\N	\N	1	N/A
242	GIST patients with exon 14 KIT mutations had reduced overall survival compared to patients wiltype for KIT	Poor Outcome	Supports	3	3	1	34	159	68	2015-03-04 17:26:01.98786	2015-03-04 17:26:01.98786	3	accepted	\N	\N	1	N/A
243	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	Positive	Supports	1	3	1	34	157	69	2015-03-04 17:26:02.068798	2015-03-04 17:26:02.068798	3	accepted	\N	\N	1	N/A
244	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	3	3	4	7	161	69	2015-03-04 17:26:02.15395	2015-03-04 17:26:02.15395	3	accepted	\N	\N	1	N/A
245	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	N / A	Does Not Support	3	3	1	34	159	70	2015-03-04 17:26:02.247415	2015-03-04 17:26:02.247415	3	accepted	\N	\N	1	N/A
246	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	N / A	Does Not Support	3	3	1	17	160	70	2015-03-04 17:26:02.324018	2015-03-04 17:26:02.324018	2	accepted	\N	\N	1	N/A
247	In a case study, a patient with anal melanoma harboring a 7 codon duplication in the juxtamambrane region (exon 11) of KIT showed marked response 4 months after imatinib treatment.	Sensitivity	Supports	2	3	4	35	162	66	2015-03-04 17:26:02.409147	2015-03-04 17:26:02.409147	3	accepted	\N	\N	1	N/A
248	The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation of KIT but did not alter the binding affinity of dasatinib.	Sensitivity	Supports	2	1	3	7	163	71	2015-03-04 17:26:02.480747	2015-03-04 17:26:02.480747	3	accepted	\N	\N	1	N/A
249	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	2	1	4	7	163	71	2015-03-04 17:26:02.57444	2015-03-04 17:26:02.57444	3	accepted	\N	\N	1	N/A
250	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	2	1	2	7	163	71	2015-03-04 17:26:02.645934	2015-03-04 17:26:02.645934	3	accepted	\N	\N	1	N/A
251	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	2	1	9	7	163	71	2015-03-04 17:26:02.707758	2015-03-04 17:26:02.707758	3	accepted	\N	\N	1	N/A
252	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	2	1	3	3	164	71	2015-03-04 17:26:02.789845	2015-03-04 17:26:02.789845	3	accepted	\N	\N	1	N/A
253	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	2	1	4	3	164	71	2015-03-04 17:26:02.851523	2015-03-04 17:26:02.851523	3	accepted	\N	\N	1	N/A
255	KIT V654A results in imatinib resistance in GIST patient-derived cell lines.	Resistance or Non-Response	Supports	2	1	4	34	165	72	2015-03-04 17:26:02.990349	2015-03-04 17:26:02.990349	3	accepted	\N	\N	1	4:55594258-55594258 (T->C)
256	In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective for those that are refractory to imatinib.	Sensitivity	Supports	2	3	51	34	166	72	2015-03-04 17:26:03.077676	2015-03-04 17:26:03.077676	3	accepted	\N	\N	1	4:55594258-55594258 (T->C)
257	Patients with colorectal cancer who harbor KRAS mutation have low response rate to cetuximab	Resistance or Non-Response	Supports	2	3	13	6	167	73	2015-03-04 17:26:03.157751	2015-03-04 17:26:03.157751	4	accepted	\N	\N	1	N/A
258	KRAS mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	Sensitivity	Does Not Support	2	3	52	6	168	74	2015-03-04 17:26:03.282783	2015-03-04 17:26:03.282783	4	accepted	\N	\N	1	N/A
259	In NSCLC, Exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	Resistance or Non-Response	Supports	2	3	31	3	169	74	2015-03-04 17:26:03.362389	2015-03-04 17:26:03.362389	3	accepted	\N	\N	1	N/A
260	In NSCLC, Exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	Resistance or Non-Response	Supports	2	3	28	3	169	74	2015-03-04 17:26:03.425344	2015-03-04 17:26:03.425344	3	accepted	\N	\N	1	N/A
261	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	Positive	Supports	1	3	1	36	170	75	2015-03-04 17:26:03.501911	2015-03-04 17:26:03.501911	3	accepted	\N	\N	1	N/A
262	In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12 mutation did not impact overall or disease free survival.	N / A	Does Not Support	3	3	1	8	171	75	2015-03-04 17:26:03.578774	2015-03-04 17:26:03.578774	4	accepted	\N	\N	1	N/A
263	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	Poor Outcome	Supports	3	3	1	37	172	75	2015-03-04 17:26:03.655481	2015-03-04 17:26:03.655481	3	accepted	\N	\N	1	N/A
264	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	Positive	Does Not Support	1	3	1	3	173	75	2015-03-04 17:26:03.722531	2015-03-04 17:26:03.722531	3	accepted	\N	\N	1	N/A
265	There is no strong association between KRAS mutation status and overall or progression free survival in patients with NSCLC	N / A	Does Not Support	3	3	1	3	174	75	2015-03-04 17:26:03.798261	2015-03-04 17:26:03.798261	3	accepted	\N	\N	1	N/A
266	A meta-analysis showed KRAS mutation is associated with worse outcome in patients with NSCLC	Poor Outcome	Supports	3	3	1	3	175	75	2015-03-04 17:26:03.875499	2015-03-04 17:26:03.875499	3	accepted	\N	\N	1	N/A
267	Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations (primarily G12/G13) had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype KRAS	Resistance or Non-Response	Supports	2	3	13	6	70	76	2015-03-04 17:26:03.957426	2015-03-04 17:26:03.957426	3	accepted	\N	\N	1	N/A
268	KRAS mutations were significantly associated with lack of response to cetuximab in patients with advanced colorectal cancer	Resistance or Non-Response	Supports	2	1	13	6	176	76	2015-03-04 17:26:04.036434	2015-03-04 17:26:04.036434	3	accepted	\N	\N	1	N/A
269	KRAS mutations were significantly associated with reduced progression-free and overall survival in patients with advanced colorectal cancer	Poor Outcome	Supports	3	1	1	6	176	76	2015-03-04 17:26:04.102857	2015-03-04 17:26:04.102857	3	accepted	\N	\N	1	N/A
270	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	Resistance or Non-Response	Supports	2	3	31	38	177	146	2015-03-04 17:26:04.184463	2015-03-04 17:26:04.184463	3	accepted	\N	\N	1	N/A
271	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	Resistance or Non-Response	Supports	2	3	28	38	177	146	2015-03-04 17:26:04.24946	2015-03-04 17:26:04.24946	3	accepted	\N	\N	1	N/A
272	KRAS G12C occur more frequently in women than men	Positive	Supports	1	3	1	17	178	77	2015-03-04 17:26:04.341741	2015-03-04 17:26:04.341741	2	accepted	\N	\N	1	12:25398285-25398285 (C->A)
273	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	Resistance or Non-Response	Supports	2	3	31	38	177	77	2015-03-04 17:26:04.42001	2015-03-04 17:26:04.42001	3	accepted	\N	\N	1	N/A
274	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	Resistance or Non-Response	Supports	2	3	28	38	177	77	2015-03-04 17:26:04.487596	2015-03-04 17:26:04.487596	3	accepted	\N	\N	1	N/A
275	KRAS G12D mutation occurs in never smokers significantly more often than in smokers	Positive	Supports	1	3	1	17	178	78	2015-03-04 17:26:04.577061	2015-03-04 17:26:04.577061	3	accepted	\N	\N	1	12:25398284-25398284 (C->T)
276	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	Resistance or Non-Response	Supports	2	3	12	3	179	78	2015-03-04 17:26:04.659126	2015-03-04 17:26:04.659126	2	accepted	\N	\N	1	12:25398284-25398284 (C->T)
277	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.	Sensitivity	Supports	2	1	53	3	180	78	2015-03-04 17:26:04.750932	2015-03-04 17:26:04.750932	4	accepted	\N	\N	1	12:25398284-25398284 (C->T)
278	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.	Sensitivity	Supports	2	1	54	3	180	78	2015-03-04 17:26:04.819247	2015-03-04 17:26:04.819247	4	accepted	\N	\N	1	12:25398284-25398284 (C->T)
279	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	Resistance or Non-Response	Supports	2	3	31	38	177	78	2015-03-04 17:26:04.893698	2015-03-04 17:26:04.893698	3	accepted	\N	\N	1	N/A
280	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	Resistance or Non-Response	Supports	2	3	28	38	177	78	2015-03-04 17:26:04.959679	2015-03-04 17:26:04.959679	3	accepted	\N	\N	1	N/A
281	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	Resistance or Non-Response	Supports	2	3	31	38	177	145	2015-03-04 17:26:05.105525	2015-03-04 17:26:05.105525	3	accepted	\N	\N	1	N/A
282	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	Resistance or Non-Response	Supports	2	3	28	38	177	145	2015-03-04 17:26:05.180178	2015-03-04 17:26:05.180178	3	accepted	\N	\N	1	N/A
283	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	Positive	Supports	1	3	1	36	170	79	2015-03-04 17:26:05.266803	2015-03-04 17:26:05.266803	3	accepted	\N	\N	1	N/A
284	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	Poor Outcome	Supports	3	3	1	39	172	79	2015-03-04 17:26:05.360184	2015-03-04 17:26:05.360184	3	accepted	\N	\N	1	N/A
285	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	Positive	Does Not Support	1	3	1	3	173	79	2015-03-04 17:26:05.443657	2015-03-04 17:26:05.443657	3	accepted	\N	\N	1	N/A
286	Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	Resistance or Non-Response	Supports	2	1	13	8	181	80	2015-03-04 17:26:05.535856	2015-03-04 17:26:05.535856	4	accepted	\N	\N	1	12:25398281-25398281 (C->T)
287	Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	Sensitivity	Supports	2	1	13	8	181	80	2015-03-04 17:26:05.607227	2015-03-04 17:26:05.607227	4	accepted	\N	\N	1	12:25398281-25398281 (C->T)
288	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	Sensitivity	Supports	2	3	13	8	181	80	2015-03-04 17:26:05.682905	2015-03-04 17:26:05.682905	4	accepted	\N	\N	1	12:25398281-25398281 (C->T)
289	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	Better Outcome	Supports	3	3	13	8	181	80	2015-03-04 17:26:05.764067	2015-03-04 17:26:05.764067	4	accepted	\N	\N	1	12:25398281-25398281 (C->T)
290	The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition by about 5 fold.	Resistance or Non-Response	Supports	2	1	22	7	182	81	2015-03-04 17:26:05.845297	2015-03-04 17:26:05.845297	3	accepted	\N	\N	1	15:66729162-66729162 (C->T)
291	MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.	Resistance or Non-Response	Supports	2	1	22	7	183	82	2015-03-04 17:26:05.922129	2015-03-04 17:26:05.922129	3	accepted	\N	\N	1	15:66727451-66727451 (A->C)
292	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	2	1	4	20	100	83	2015-03-04 17:26:05.997196	2015-03-04 17:26:05.997196	3	accepted	\N	\N	1	N/A
293	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	2	1	27	20	100	83	2015-03-04 17:26:06.067061	2015-03-04 17:26:06.067061	3	accepted	\N	\N	1	N/A
294	In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804), MET amplification and EGFR T790M were found to be the primary drivers of resistance.	Resistance or Non-Response	Supports	2	3	34	3	120	144	2015-03-04 17:26:06.144409	2015-03-04 17:26:06.144409	4	accepted	\N	\N	1	N/A
295	MGMT promoter methylation has been shown to confer sensitivity to alkylating agents, such as carmustine, in glioblastoma cells.	Sensitivity	Supports	2	3	55	40	184	84	2015-03-04 17:26:06.231932	2015-03-04 17:26:06.231932	4	accepted	\N	\N	1	N/A
296	MGMT promoter methylation was associated with increased overall and disease-free survival as well as tumor regression in patients with gliomas.	Better Outcome	Supports	3	3	1	40	184	84	2015-03-04 17:26:06.305027	2015-03-04 17:26:06.305027	4	accepted	\N	\N	1	N/A
297	In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor, such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may be useful in treating patients with initial resistance to alkylating agents.	Sensitivity	Supports	2	4	56	40	184	84	2015-03-04 17:26:06.379373	2015-03-04 17:26:06.379373	4	accepted	\N	\N	1	N/A
298	In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted from temozolomide. This benefit was also methylation status dependent, as those without methylation did not see increased survival.	Sensitivity	Supports	2	2	21	40	185	84	2015-03-04 17:26:06.451467	2015-03-04 17:26:06.451467	5	accepted	\N	\N	1	N/A
299	NOTCH1 gain-of-function mutations, including D1643H, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	Poor Outcome	Supports	3	3	1	3	186	133	2015-03-04 17:26:06.529443	2015-03-04 17:26:06.529443	4	accepted	\N	\N	1	N/A
300	NOTCH1 gain-of-function mutations, including R2328W, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	Poor Outcome	Supports	3	3	1	3	186	134	2015-03-04 17:26:06.607329	2015-03-04 17:26:06.607329	4	accepted	\N	\N	1	N/A
301	NOTCH1 gain-of-function mutations, including S2275fs, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	Poor Outcome	Supports	3	3	1	3	186	136	2015-03-04 17:26:06.677184	2015-03-04 17:26:06.677184	4	accepted	\N	\N	1	N/A
302	NOTCH1 gain-of-function mutations, including V2444fs, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	Poor Outcome	Supports	3	3	1	3	186	135	2015-03-04 17:26:06.74401	2015-03-04 17:26:06.74401	4	accepted	\N	\N	1	N/A
303	ATRA treatment did not effect overall survival in patients <60 years old with NPM1 mutation regardless of FLT3-ITD status	Resistance or Non-Response	Supports	2	3	44	19	127	85	2015-03-04 17:26:06.815453	2015-03-04 17:26:06.815453	3	accepted	\N	\N	1	N/A
304	ATRA treatment improved overall and relapse-free survival in older (>60) patients with mutant NPM1 and without FLT3-ITD	Sensitivity	Supports	2	3	44	19	187	85	2015-03-04 17:26:06.929936	2015-03-04 17:26:06.929936	2	accepted	\N	\N	1	N/A
305	Young patients (18-60) with NPM1 mutations had improved overall survival following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However, the adjusted p-value for multiple testing is 0.11.	Sensitivity	Supports	2	3	29	19	188	85	2015-03-04 17:26:07.003393	2015-03-04 17:26:07.003393	2	accepted	\N	\N	1	N/A
306	NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype (0/55).	Positive	Does Not Support	1	3	1	19	189	85	2015-03-04 17:26:07.076067	2015-03-04 17:26:07.076067	3	accepted	\N	\N	1	N/A
307	NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid leukemia.	Negative	Supports	1	3	1	19	190	85	2015-03-04 17:26:07.149494	2015-03-04 17:26:07.149494	4	accepted	\N	\N	1	N/A
308	AML with mutated NPM1 is a provisional entity in WHO classification of acute myeloid leukemia (AML). This mutation should be tested for in clinical trials and is recommended for testing in patients with cytogentically normal AML.	Positive	Supports	1	2	1	19	191	85	2015-03-04 17:26:07.218141	2015-03-04 17:26:07.218141	5	accepted	\N	\N	1	N/A
309	NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes	Positive	Supports	1	3	1	19	131	85	2015-03-04 17:26:07.283738	2015-03-04 17:26:07.283738	2	accepted	\N	\N	1	N/A
310	NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid leukemia.	Positive	Supports	1	3	1	19	190	85	2015-03-04 17:26:07.351501	2015-03-04 17:26:07.351501	4	accepted	\N	\N	1	N/A
311	NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid leukemia	Positive	Supports	1	3	1	19	189	85	2015-03-04 17:26:07.424541	2015-03-04 17:26:07.424541	3	accepted	\N	\N	1	N/A
312	NPM1 mutations were associated with normal karyotype in older (>60) patients	Positive	Supports	1	3	1	19	187	85	2015-03-04 17:26:07.501074	2015-03-04 17:26:07.501074	3	accepted	\N	\N	1	N/A
313	CD33 and CD123 expression was significantly increased in patients with NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33 and anti-CD123 therapy	Sensitivity	Supports	2	4	1	19	192	85	2015-03-04 17:26:07.582538	2015-03-04 17:26:07.582538	2	accepted	\N	\N	1	N/A
314	CD33 expression was significantly increased in patients with NPM1 mutation with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy	Sensitivity	Supports	2	4	1	19	192	85	2015-03-04 17:26:07.662817	2015-03-04 17:26:07.662817	2	accepted	\N	\N	1	N/A
315	NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and increased overall survival in young adult acute myeloid leukemia patients (median age 43)	Sensitivity	Supports	2	3	1	19	131	85	2015-03-04 17:26:07.748489	2015-03-04 17:26:07.748489	4	accepted	\N	\N	1	N/A
316	Patients with NPM1 mutations were associated with improved complete remission rates as well as overall and disease free survival when all ages and karyotype were analyzed; however, when only normal karyotype patients were analyzed, only complete remission rates were improved	Sensitivity	Supports	2	3	1	19	187	85	2015-03-04 17:26:07.828663	2015-03-04 17:26:07.828663	3	accepted	\N	\N	1	N/A
317	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N / A	Does Not Support	3	3	1	19	107	85	2015-03-04 17:26:07.903038	2015-03-04 17:26:07.903038	4	accepted	\N	\N	1	N/A
318	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	N / A	Does Not Support	3	3	1	19	141	85	2015-03-04 17:26:07.987356	2015-03-04 17:26:07.987356	3	accepted	\N	\N	1	N/A
319	Complete remission rates were higher and both disease-free and overall survival were longer for patients with Exon 12 NPM1 mutations.	Better Outcome	Supports	3	3	1	19	189	85	2015-03-04 17:26:08.06029	2015-03-04 17:26:08.06029	3	accepted	\N	\N	1	N/A
320	Complete remission rates were higher and event-free survival was longer for normal karyotype AML patients with Exon 12 NPM1 mutations.	Better Outcome	Supports	3	3	1	19	190	85	2015-03-04 17:26:08.134453	2015-03-04 17:26:08.134453	5	accepted	\N	\N	1	N/A
321	Cytoplasmic localization of NPM in AML patients was an independent prognositic factor associated with complete remission in response to induction therapy in a multivariate analysis (p=0.019) but not in a univariate analysis (p=0.07).	Better Outcome	Supports	3	3	1	19	193	85	2015-03-04 17:26:08.209193	2015-03-04 17:26:08.209193	3	accepted	\N	\N	1	N/A
322	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	Better Outcome	Supports	3	3	1	19	136	85	2015-03-04 17:26:08.284045	2015-03-04 17:26:08.284045	3	accepted	\N	\N	1	N/A
323	Intermediate risk patients with mutant NPM1 had improved overall survival with the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1 and IDH2	Better Outcome	Supports	3	3	1	19	188	85	2015-03-04 17:26:08.35811	2015-03-04 17:26:08.35811	2	accepted	\N	\N	1	N/A
324	Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved overall and relapse-free survival, complete remission rates and remission duration compared to all other combinations of mutants	Better Outcome	Supports	3	3	1	19	194	85	2015-03-04 17:26:08.431769	2015-03-04 17:26:08.431769	3	accepted	\N	\N	1	N/A
325	NPM1 mutation was associated with higher complete remission rates, lower cumulative incidence of relapse and higher overall survival in intermediate risk AML patients	Better Outcome	Supports	3	3	1	19	195	85	2015-03-04 17:26:08.511144	2015-03-04 17:26:08.511144	4	accepted	\N	\N	1	N/A
326	NPM1 mutation was associated with increased overall, event-free and relapse-free survival and remission after induction therapy in the entire cohort and in the cytogenetically normal subset of patients	Better Outcome	Supports	3	3	1	19	196	85	2015-03-04 17:26:08.584606	2015-03-04 17:26:08.584606	3	accepted	\N	\N	1	N/A
327	NPM1 mutation without FLT3-ITD was significantly associated with complete remission	Better Outcome	Supports	3	3	1	19	98	85	2015-03-04 17:26:08.706122	2015-03-04 17:26:08.706122	3	accepted	\N	\N	1	N/A
328	NPM1 mutations were associated with increased complete remission rates as well as longer overall disease free survival in normal karyotype AML patients >59 years old, particularly in those >69 years old	Better Outcome	Supports	3	3	1	19	197	85	2015-03-04 17:26:08.777137	2015-03-04 17:26:08.777137	4	accepted	\N	\N	1	N/A
329	Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and without FLT3-ITD have increased overall survival, relapse-free survival and response to induction chemotherapy	Better Outcome	Supports	3	3	1	19	198	85	2015-03-04 17:26:08.853704	2015-03-04 17:26:08.853704	4	accepted	\N	\N	1	N/A
330	Coocurrence of FLT3-ITD mutations significantly reduced overall survival of normal karyotype AML patients	Poor Outcome	Supports	3	3	1	19	189	85	2015-03-04 17:26:08.923848	2015-03-04 17:26:08.923848	4	accepted	\N	\N	1	N/A
331	Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction in NPM1 levels following first line therapy (N=15)	Poor Outcome	Supports	3	3	1	19	199	85	2015-03-04 17:26:08.997892	2015-03-04 17:26:08.997892	2	accepted	\N	\N	1	N/A
332	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	Poor Outcome	Supports	3	3	1	19	132	85	2015-03-04 17:26:09.070285	2015-03-04 17:26:09.070285	4	accepted	\N	\N	1	N/A
333	Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate risk AML patients	Poor Outcome	Supports	3	3	1	19	195	85	2015-03-04 17:26:09.143344	2015-03-04 17:26:09.143344	3	accepted	\N	\N	1	N/A
334	TET2 mutation reduces overall survival of normal karyotype patients with either an NPM1 mutation or NPM1 mutation without FLT3-ITD	Poor Outcome	Supports	3	3	1	19	200	85	2015-03-04 17:26:09.218035	2015-03-04 17:26:09.218035	3	accepted	\N	\N	1	N/A
335	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	Poor Outcome	Supports	3	3	1	19	132	85	2015-03-04 17:26:09.293169	2015-03-04 17:26:09.293169	4	accepted	\N	\N	1	N/A
336	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	Sensitivity	Supports	2	1	57	19	201	85	2015-03-04 17:26:09.376432	2015-03-04 17:26:09.376432	3	accepted	\N	\N	1	N/A
337	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	Sensitivity	Supports	2	1	44	19	201	85	2015-03-04 17:26:09.429296	2015-03-04 17:26:09.429296	3	accepted	\N	\N	1	N/A
338	For patients with NPM1 mutation that achieved CR following induction therapy, relapse-free survival was improved following treatment with valproic acid	Sensitivity	Supports	2	3	58	19	202	85	2015-03-04 17:26:09.508937	2015-03-04 17:26:09.508937	2	accepted	\N	\N	1	N/A
339	No NPM1 mutations were identified in patients with favorable risk cytogenetics (79/215 patients)	Positive	Does Not Support	1	3	1	19	105	86	2015-03-04 17:26:09.585142	2015-03-04 17:26:09.585142	4	accepted	\N	\N	1	5:170837546-170837547 (0->TCAG)
340	NPM1 mutations were associated with intermediate risk cytogenetics (including normal karyotype)	Positive	Supports	1	3	1	19	105	86	2015-03-04 17:26:09.667035	2015-03-04 17:26:09.667035	4	accepted	\N	\N	1	5:170837546-170837547 (0->TCAG)
341	NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM in AML patients was associated with good response to induction therapy.	Sensitivity	Supports	2	4	1	19	193	86	2015-03-04 17:26:09.751772	2015-03-04 17:26:09.751772	3	accepted	\N	\N	1	5:170837546-170837547 (0->TCAG)
342	NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.	Sensitivity	Supports	2	1	57	19	201	86	2015-03-04 17:26:09.850327	2015-03-04 17:26:09.850327	3	accepted	\N	\N	1	5:170837546-170837547 (0->TCAG)
343	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	Poor Outcome	Supports	3	3	1	7	203	87	2015-03-04 17:26:09.929684	2015-03-04 17:26:09.929684	3	accepted	\N	\N	1	N/A
344	The presence of NRAS mutation in AML patients does not impact diease prognosis (resistant disease, disease-free survival, complete remission rate, relapse rate, induction death)	N / A	Does Not Support	3	3	1	19	204	88	2015-03-04 17:26:10.000393	2015-03-04 17:26:10.000393	3	accepted	\N	\N	1	N/A
345	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	Poor Outcome	Supports	3	3	1	6	205	88	2015-03-04 17:26:10.074089	2015-03-04 17:26:10.074089	3	accepted	\N	\N	1	N/A
346	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	Poor Outcome	Supports	3	3	1	7	203	88	2015-03-04 17:26:10.142168	2015-03-04 17:26:10.142168	3	accepted	\N	\N	1	N/A
347	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	Poor Outcome	Supports	3	2	1	6	205	89	2015-03-04 17:26:10.21613	2015-03-04 17:26:10.21613	3	accepted	\N	\N	1	N/A
348	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	Poor Outcome	Supports	3	2	1	6	205	90	2015-03-04 17:26:10.339722	2015-03-04 17:26:10.339722	3	accepted	\N	\N	1	N/A
349	NRAS mutation status does not impact overall, event-free or disease-free survival in patients with AML	N / A	Does Not Support	3	3	1	19	206	91	2015-03-04 17:26:10.417081	2015-03-04 17:26:10.417081	3	accepted	\N	\N	1	N/A
350	Likely due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic malignant melanoma with an NRAS G13D mutation	Sensitivity	Supports	2	3	59	7	207	92	2015-03-04 17:26:10.490486	2015-03-04 17:26:10.490486	2	accepted	\N	\N	1	1:115258744-115258744 (C->T)
351	In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status (primarily Q61) was not informative for disease control rate, progression free survival or overall survival following cetuximab plus chemotherapy.	Resistance or Non-Response	Does Not Support	2	3	13	6	70	93	2015-03-04 17:26:10.57071	2015-03-04 17:26:10.57071	3	accepted	\N	\N	1	N/A
352	Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation (primarily Q61) have a significantly lower response rate to cetuximab than patients wildtype for NRAS.	Resistance or Non-Response	Supports	2	3	13	6	70	93	2015-03-04 17:26:10.643412	2015-03-04 17:26:10.643412	3	accepted	\N	\N	1	N/A
353	Melanoma associated with NRAS Q61 mutation was more often associated with those at the extremity than those at the trunk	Positive	Supports	1	3	1	7	208	93	2015-03-04 17:26:10.724887	2015-03-04 17:26:10.724887	3	accepted	\N	\N	1	N/A
354	Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.	Resistance or Non-Response	Supports	2	3	18	7	209	93	2015-03-04 17:26:10.790297	2015-03-04 17:26:10.790297	4	accepted	\N	\N	1	N/A
355	In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted in disease free survival of 14 months	Sensitivity	Supports	2	3	21	7	210	94	2015-03-04 17:26:10.869101	2015-03-04 17:26:10.869101	2	accepted	\N	\N	1	1:115256528-115256529 (TT->CA)
356	In a melanoma patient with Q61R mutation, treatment with temozolomide resulted in overall survival of 16 months	Sensitivity	Supports	2	3	21	7	210	95	2015-03-04 17:26:10.948386	2015-03-04 17:26:10.948386	2	accepted	\N	\N	1	1:115256528-115256529 (TT->CC)
357	In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	2	1	60	41	211	96	2015-03-04 17:26:11.038333	2015-03-04 17:26:11.038333	4	accepted	\N	\N	1	4:55152092-55152093 (GA->AT)
358	In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	2	1	60	41	211	97	2015-03-04 17:26:11.115434	2015-03-04 17:26:11.115434	4	accepted	\N	\N	1	4:55152093-55152093 (A->T)
359	GIST cancer with D842V mutation is resistant to imatinib.	Resistance or Non-Response	Supports	2	3	4	41	212	97	2015-03-04 17:26:11.197039	2015-03-04 17:26:11.197039	4	accepted	\N	\N	1	4:55152093-55152093 (A->T)
360	While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib, double mutation of V561D and D842V mutants are resistant to imatinib.	Resistance or Non-Response	Supports	2	3	4	41	165	97	2015-03-04 17:26:11.276561	2015-03-04 17:26:11.276561	4	accepted	\N	\N	1	4:55152093-55152093 (A->T)
361	GIST tumors harboring PDGFRA D842V mutation are more likely to be benign than malignant.	Negative	Supports	1	3	1	41	213	97	2015-03-04 17:26:11.359906	2015-03-04 17:26:11.359906	3	accepted	\N	\N	1	4:55152093-55152093 (A->T)
362	In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	2	1	60	41	211	98	2015-03-04 17:26:11.441307	2015-03-04 17:26:11.441307	4	accepted	\N	\N	1	4:55152092-55152092 (G->T)
363	In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	2	1	60	41	211	99	2015-03-04 17:26:11.520167	2015-03-04 17:26:11.520167	4	accepted	\N	\N	1	N/A
364	In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	2	1	60	41	211	100	2015-03-04 17:26:11.601723	2015-03-04 17:26:11.601723	4	accepted	\N	\N	1	N/A
365	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	2	1	61	11	214	101	2015-03-04 17:26:11.688199	2015-03-04 17:26:11.688199	4	accepted	\N	\N	1	3:178936082-178936082 (G->A)
366	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	2	1	62	11	215	101	2015-03-04 17:26:11.769766	2015-03-04 17:26:11.769766	3	accepted	\N	\N	1	3:178936082-178936082 (G->A)
367	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	2	4	13	6	216	101	2015-03-04 17:26:11.854831	2015-03-04 17:26:11.854831	3	accepted	\N	\N	1	3:178936082-178936082 (G->A)
368	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	2	4	11	6	216	101	2015-03-04 17:26:11.911689	2015-03-04 17:26:11.911689	3	accepted	\N	\N	1	3:178936082-178936082 (G->A)
369	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	Poor Outcome	Supports	3	4	1	6	217	101	2015-03-04 17:26:11.983566	2015-03-04 17:26:11.983566	2	accepted	\N	\N	1	3:178936082-178936082 (G->A)
370	The presence of exon 9 (such as E542K) or exon 20 mutations were associated with poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	3	4	1	6	216	101	2015-03-04 17:26:12.106328	2015-03-04 17:26:12.106328	3	accepted	\N	\N	1	3:178936082-178936082 (G->A)
371	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N / A	Does Not Support	2	3	52	6	168	101	2015-03-04 17:26:12.173844	2015-03-04 17:26:12.173844	4	accepted	\N	\N	1	3:178936082-178936082 (G->A)
372	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	2	1	61	11	214	102	2015-03-04 17:26:12.249193	2015-03-04 17:26:12.249193	4	accepted	\N	\N	1	3:178936091-178936091 (G->A)
373	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	2	1	62	11	215	102	2015-03-04 17:26:12.315748	2015-03-04 17:26:12.315748	3	accepted	\N	\N	1	3:178936091-178936091 (G->A)
374	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	2	4	13	6	216	102	2015-03-04 17:26:12.389264	2015-03-04 17:26:12.389264	3	accepted	\N	\N	1	3:178936091-178936091 (G->A)
375	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	2	4	11	6	216	102	2015-03-04 17:26:12.450838	2015-03-04 17:26:12.450838	3	accepted	\N	\N	1	3:178936091-178936091 (G->A)
376	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	Poor Outcome	Supports	3	4	1	6	217	102	2015-03-04 17:26:12.514467	2015-03-04 17:26:12.514467	2	accepted	\N	\N	1	3:178936091-178936091 (G->A)
377	The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	3	4	1	6	216	102	2015-03-04 17:26:12.584282	2015-03-04 17:26:12.584282	3	accepted	\N	\N	1	3:178936091-178936091 (G->A)
378	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N / A	Does Not Support	2	3	52	6	168	102	2015-03-04 17:26:12.649567	2015-03-04 17:26:12.649567	4	accepted	\N	\N	1	3:178936091-178936091 (G->A)
379	Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	Resistance or Non-Response	Supports	2	3	13	6	70	103	2015-03-04 17:26:12.712208	2015-03-04 17:26:12.712208	3	accepted	\N	\N	1	N/A
380	In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations (Exon 9) had no significant impact on response rate, disease control rate, progression free surivival or overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	N / A	Does Not Support	2	3	13	6	70	104	2015-03-04 17:26:12.776067	2015-03-04 17:26:12.776067	3	accepted	\N	\N	1	N/A
381	Breast cancer cell lines with H1047R mutation showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	2	1	61	11	214	105	2015-03-04 17:26:12.852223	2015-03-04 17:26:12.852223	4	accepted	\N	\N	1	3:178952085-178952085 (A->G)
382	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	2	1	62	11	215	105	2015-03-04 17:26:12.926743	2015-03-04 17:26:12.926743	3	accepted	\N	\N	1	3:178952085-178952085 (A->G)
383	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	2	4	13	6	216	105	2015-03-04 17:26:12.994065	2015-03-04 17:26:12.994065	3	accepted	\N	\N	1	3:178952085-178952085 (A->G)
384	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	2	4	11	6	216	105	2015-03-04 17:26:13.058705	2015-03-04 17:26:13.058705	3	accepted	\N	\N	1	3:178952085-178952085 (A->G)
385	The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	3	4	1	6	216	105	2015-03-04 17:26:13.12729	2015-03-04 17:26:13.12729	3	accepted	\N	\N	1	3:178952085-178952085 (A->G)
386	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N / A	Does Not Support	2	3	52	6	168	105	2015-03-04 17:26:13.203218	2015-03-04 17:26:13.203218	4	accepted	\N	\N	1	3:178952085-178952085 (A->G)
387	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	Sensitivity	Supports	2	3	44	42	218	106	2015-03-04 17:26:13.2763	2015-03-04 17:26:13.2763	5	accepted	\N	\N	1	N/A
388	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	Positive	Supports	1	3	1	21	102	107	2015-03-04 17:26:13.348148	2015-03-04 17:26:13.348148	4	accepted	\N	\N	1	N/A
389	Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse to PI3K-mTOR inhibitors.	Sensitivity	Supports	2	1	1	2	219	108	2015-03-04 17:26:13.430461	2015-03-04 17:26:13.430461	4	accepted	\N	\N	1	10:89717672-89717672 (C->T)
390	PTEN nonsense mutations, including R233*, have been shown to be inactivating and loss-of-function, but do not have prognostic value in glioblastoma multiforme patients.	Poor Outcome	Does Not Support	3	3	1	40	220	108	2015-03-04 17:26:13.515089	2015-03-04 17:26:13.515089	3	accepted	\N	\N	1	10:89717672-89717672 (C->T)
391	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	Sensitivity	Supports	2	3	44	42	218	109	2015-03-04 17:26:13.6347	2015-03-04 17:26:13.6347	5	accepted	\N	\N	1	N/A
392	Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib, compared to wild-type RET.	Resistance or Non-Response	Supports	2	1	63	43	221	110	2015-03-04 17:26:13.70975	2015-03-04 17:26:13.70975	3	accepted	\N	\N	1	10:43609950-43609950 (C->G)
393	The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong repression of tyroid cancer cell growth.	Sensitivity	Supports	2	1	64	43	222	111	2015-03-04 17:26:13.801073	2015-03-04 17:26:13.801073	3	accepted	\N	\N	1	10:43617416-43617416 (T->C)
394	Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib, compared to wild-type RET.	Resistance or Non-Response	Supports	2	1	63	43	221	111	2015-03-04 17:26:13.871284	2015-03-04 17:26:13.871284	3	accepted	\N	\N	1	10:43617416-43617416 (T->C)
395	In patients with medullary carcinoma, the presence of RET M918T mutation is associated with increased probability of lymph node metastases.	Positive	Supports	1	3	1	43	223	111	2015-03-04 17:26:13.950476	2015-03-04 17:26:13.950476	5	accepted	\N	\N	1	10:43617416-43617416 (T->C)
396	Compared to those who harbor a wild type RET, patients with RET M918T mutation develop larger and more aggressive medullary thyroid cancer.	Poor Outcome	Supports	3	3	1	43	224	111	2015-03-04 17:26:14.029744	2015-03-04 17:26:14.029744	4	accepted	\N	\N	1	10:43617416-43617416 (T->C)
397	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	Better Outcome	Supports	3	4	1	29	225	112	2015-03-04 17:26:14.104199	2015-03-04 17:26:14.104199	3	accepted	\N	\N	1	2:198267359-198267359 (C->A)
398	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	Better Outcome	Supports	3	4	1	29	225	113	2015-03-04 17:26:14.179919	2015-03-04 17:26:14.179919	3	accepted	\N	\N	1	2:198266834-198266834 (T->C)
399	Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.	Sensitivity	Supports	2	1	65	11	226	114	2015-03-04 17:26:14.260912	2015-03-04 17:26:14.260912	3	accepted	\N	\N	1	17:7578406-7578406 (C->T)
400	Breast cancer patients who harbor R175H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	3	3	1	11	227	114	2015-03-04 17:26:14.337537	2015-03-04 17:26:14.337537	3	accepted	\N	\N	1	17:7578406-7578406 (C->T)
401	Breast cancer patients who harbor R248Q mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	3	3	1	11	227	115	2015-03-04 17:26:14.420292	2015-03-04 17:26:14.420292	3	accepted	\N	\N	1	17:7577538-7577538 (C->T)
402	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R248 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	3	3	1	11	228	115	2015-03-04 17:26:14.492841	2015-03-04 17:26:14.492841	3	accepted	\N	\N	1	17:7577538-7577538 (C->T)
403	In breast cancer patients harboring R248W mutation, the prognosis is worse than any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.	Poor Outcome	Supports	3	3	1	11	227	116	2015-03-04 17:26:14.568453	2015-03-04 17:26:14.568453	3	accepted	\N	\N	1	17:7577539-7577539 (G->A)
404	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	Poor Outcome	Supports	3	3	1	11	228	117	2015-03-04 17:26:14.642443	2015-03-04 17:26:14.642443	3	accepted	\N	\N	1	17:7577535-7577535 (C->G)
405	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	Poor Outcome	Supports	3	3	1	11	228	118	2015-03-04 17:26:14.721253	2015-03-04 17:26:14.721253	3	accepted	\N	\N	1	17:7577536-7577536 (T->A)
406	Breast cancer patients who harbor R273C mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	3	3	1	11	227	119	2015-03-04 17:26:14.802442	2015-03-04 17:26:14.802442	3	accepted	\N	\N	1	17:7577121-7577121 (G->A)
407	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	3	3	1	11	228	119	2015-03-04 17:26:14.875797	2015-03-04 17:26:14.875797	3	accepted	\N	\N	1	17:7577121-7577121 (G->A)
408	Breast cancer patients who harbor R273H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	3	3	1	11	227	120	2015-03-04 17:26:14.973971	2015-03-04 17:26:14.973971	3	accepted	\N	\N	1	17:7577120-7577120 (C->T)
409	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	3	3	1	11	228	120	2015-03-04 17:26:15.096989	2015-03-04 17:26:15.096989	3	accepted	\N	\N	1	17:7577120-7577120 (C->T)
410	Breast tumor with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.	Poor Outcome	Supports	3	3	1	11	228	121	2015-03-04 17:26:15.177179	2015-03-04 17:26:15.177179	3	accepted	\N	\N	1	N/A
411	A patient with metastatic bladder cancer that responded well to the mTOR inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del, p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations also had mild responses to everolimus.	Sensitivity	Supports	2	3	66	44	229	122	2015-03-04 17:26:15.258122	2015-03-04 17:26:15.258122	3	accepted	\N	\N	1	N/A
412	Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit increased overall survival, increased apoptosis, reduced proliferation and reduced tumor burden compared to Tsc1 wildtype mice.	Sensitivity	Supports	2	1	62	3	230	122	2015-03-04 17:26:15.396028	2015-03-04 17:26:15.396028	3	accepted	\N	\N	1	N/A
413	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	Sensitivity	Supports	2	3	66	24	231	123	2015-03-04 17:26:15.480004	2015-03-04 17:26:15.480004	3	accepted	\N	\N	1	N/A
414	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	Sensitivity	Supports	2	3	66	24	229	124	2015-03-04 17:26:15.563419	2015-03-04 17:26:15.563419	3	accepted	\N	\N	1	N/A
415	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.	Positive	Does Not Support	1	4	1	19	232	125	2015-03-04 17:26:15.646155	2015-03-04 17:26:15.646155	3	accepted	\N	\N	1	N/A
416	In patients with AML, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	N / A	Does Not Support	3	4	1	19	232	125	2015-03-04 17:26:15.726214	2015-03-04 17:26:15.726214	2	accepted	\N	\N	1	N/A
417	In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	N / A	Does Not Support	3	4	1	29	232	125	2015-03-04 17:26:15.80003	2015-03-04 17:26:15.80003	2	accepted	\N	\N	1	N/A
418	After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.	Poor Outcome	Supports	3	4	1	29	233	125	2015-03-04 17:26:15.874078	2015-03-04 17:26:15.874078	3	accepted	\N	\N	1	N/A
419	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.	Positive	Does Not Support	1	4	1	19	232	126	2015-03-04 17:26:15.952312	2015-03-04 17:26:15.952312	3	accepted	\N	\N	1	N/A
420	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	N / A	Does Not Support	3	4	1	19	232	126	2015-03-04 17:26:16.019384	2015-03-04 17:26:16.019384	2	accepted	\N	\N	1	N/A
421	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	N / A	Does Not Support	3	4	1	29	232	126	2015-03-04 17:26:16.088751	2015-03-04 17:26:16.088751	2	accepted	\N	\N	1	N/A
422	After adjust for age and cancer stage, the presence of U2AF mutations such as S34Y/F are prognostic for poorer survival outcomes in patients with MDS.	Poor Outcome	Supports	3	4	1	29	233	126	2015-03-04 17:26:16.15551	2015-03-04 17:26:16.15551	3	accepted	\N	\N	1	N/A
423	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	Resistance or Non-Response	Supports	2	3	1	19	234	127	2015-03-04 17:26:16.223149	2015-03-04 17:26:16.223149	3	accepted	\N	\N	1	N/A
424	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	N / A	Does Not Support	3	3	1	19	235	127	2015-03-04 17:26:16.289513	2015-03-04 17:26:16.289513	3	accepted	\N	\N	1	N/A
425	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML	N / A	Does Not Support	3	3	1	19	235	127	2015-03-04 17:26:16.35532	2015-03-04 17:26:16.35532	4	accepted	\N	\N	1	N/A
426	Mutations  in WT1 were associated with increased risk of recurrence in young patients (15-50) with cytogenetically normal AML	Poor Outcome	Supports	3	3	1	19	236	127	2015-03-04 17:26:16.430161	2015-03-04 17:26:16.430161	2	accepted	\N	\N	1	N/A
427	Mutations in WT1 were associated with a worse overall prognosis than patients wildtype for WT1 in young patients (15-50), primarily because of increased risk of disease recurrence	Poor Outcome	Supports	3	3	1	19	236	127	2015-03-04 17:26:16.495817	2015-03-04 17:26:16.495817	2	accepted	\N	\N	1	N/A
428	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	Poor Outcome	Supports	3	3	1	19	235	127	2015-03-04 17:26:16.566029	2015-03-04 17:26:16.566029	3	accepted	\N	\N	1	N/A
429	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	Poor Outcome	Supports	3	3	1	19	237	127	2015-03-04 17:26:16.643442	2015-03-04 17:26:16.643442	3	accepted	\N	\N	1	N/A
430	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	3	3	1	19	234	127	2015-03-04 17:26:16.713638	2015-03-04 17:26:16.713638	3	accepted	\N	\N	1	N/A
431	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	3	3	1	19	234	127	2015-03-04 17:26:16.784696	2015-03-04 17:26:16.784696	3	accepted	\N	\N	1	N/A
432	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	Resistance or Non-Response	Supports	2	3	1	19	234	128	2015-03-04 17:26:16.865429	2015-03-04 17:26:16.865429	3	accepted	\N	\N	1	N/A
433	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	N / A	Does Not Support	3	3	1	19	235	128	2015-03-04 17:26:17.002948	2015-03-04 17:26:17.002948	3	accepted	\N	\N	1	N/A
434	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML.	N / A	Does Not Support	3	3	1	19	235	128	2015-03-04 17:26:17.090235	2015-03-04 17:26:17.090235	4	accepted	\N	\N	1	N/A
435	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	Poor Outcome	Supports	3	3	1	19	235	128	2015-03-04 17:26:17.176624	2015-03-04 17:26:17.176624	3	accepted	\N	\N	1	N/A
436	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	3	3	1	19	234	128	2015-03-04 17:26:17.268678	2015-03-04 17:26:17.268678	3	accepted	\N	\N	1	N/A
437	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	3	3	1	19	234	128	2015-03-04 17:26:17.342131	2015-03-04 17:26:17.342131	3	accepted	\N	\N	1	N/A
438	WT1 mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	Poor Outcome	Supports	3	3	1	19	237	128	2015-03-04 17:26:17.416274	2015-03-04 17:26:17.416274	3	accepted	\N	\N	1	N/A
439	"Two metastatic melanoma patients with the L576P KIT mutation were treated with dasatinib, including one patient previously treated with imatinib. Both patients had marked reduction (>50%) and elimination of tumor F18-fluorodeoxyglucose (FDG)-avidity by positron emission tomography (PET) imaging after dasatinib treatment. These data support the selective inhibitory effect of dasatinib against cells harboring the most common KIT mutation in melanoma, and thus has therapeutic implications for acrallentiginous, chronic sun-damaged, and mucosal melanomas."	Sensitivity	Supports	2	3	3	45	163	71	2015-03-04 17:26:17.490403	2015-03-04 17:26:17.490403	5	accepted	\N	\N	1	N/A
440	"To formally test this hypothesis, we took advantage of HCT116 cells in which the KRAS D13 mutant allele had been genetically deleted by homologous recombination (15). Strikingly, we found that HCT116-derivative cells retaining only the KRAS WT allele (named HKh-2 and HKe-3; Supplemental Table 3) were sensitive to everolimus, while the parental and the isogenic cells carrying mutated KRAS were equally resistant to this compound (Figure 2A)."	Resistance or Non-Response	Supports	2	1	66	46	238	80	2015-03-04 17:26:17.571013	2015-03-04 17:26:17.571013	4	accepted	\N	\N	1	N/A
441	Figure 1 of paper shows that V654A was associated with progression of disease in one case of a mucosal melanoma. Results were from a Phase II clinical trial.	Resistance or Non-Response	Supports	2	3	4	45	239	72	2015-03-04 17:26:17.64914	2015-03-04 17:26:17.64914	5	accepted	\N	\N	1	N/A
442	"Dose-dependent, complete response to imatinib of a metastatic mucosal melanoma with a K642E KIT mutation."	Sensitivity	Supports	2	3	4	45	240	150	2015-03-04 17:26:17.720351	2015-03-04 17:26:17.720351	5	accepted	\N	\N	1	N/A
443	"From the screen, we discovered that proteasome inhibitors show selectivity toward BRAF V600Emutant cells, irrespective of PTEN or RB1 expression. Preferential targeting of BRAF-mutant cells by proteasome inhibitors was corroborated in a second BRAF V600E isogenic model, as well as in a panel of colorectal cancer cell lines by the use of the proteasome inhibitor carfilzomib. Notably, carfilzomib also showed striking in vivo activity in a BRAF-mutant human colorectal cancer xenograft model. Vulnerability to proteasome inhibitors is dependent on persistent BRAF signaling, because BRAF V600E blockade by PLX4720 reversed sensitivity to carfilzomib in BRAF-mutant colorectal cancer cells. Our findings indicated that proteasome inhibition might represent a valuable targeting strategy in BRAF V600Emutant colorectal tumors."	Sensitivity	Supports	2	1	67	47	241	12	2015-03-04 17:26:17.795088	2015-03-04 17:26:17.795088	3	accepted	\N	\N	1	N/A
444	PDGFRA G853D mutation found in an acral melanoma sample. In vitro testing showed that G853D induced IL-3independent growth could be inhibited by either imatinib or crenolanib to different extent.	Sensitivity	Supports	2	1	4	48	242	148	2015-03-04 17:26:17.868435	2015-03-04 17:26:17.868435	4	accepted	\N	\N	1	N/A
445	"Here, we report the case of a patient with metastatic lung adenocarcinoma with BRAF G469L mutation refractory to vemurafenib. We calculated a structure model of this very rare type of mutated BRAF kinase to explain the molecular mechanism of drug resistance."	Resistance or Non-Response	Supports	2	3	68	49	243	149	2015-03-04 17:26:17.93916	2015-03-04 17:26:17.93916	3	accepted	\N	\N	1	N/A
446	"Authors of study investigated genomic changes in metastatic renal cancer patients with long term benefit from rapalog treatment. Patient 3 (treated with everolimus), in study, was found to have a novel MTOR mutation, Q2223K. This residue was in the PI3K kinase domain structure. In vitro, functional validation confirmed that this variant was activating and was sensitive to rapamycin and rapalogs, everolimus and temsirolimus."	Sensitivity	Supports	2	3	66	50	244	147	2015-03-04 17:26:18.015317	2015-03-04 17:26:18.015317	5	accepted	\N	\N	1	N/A
\.


--
-- Name: evidence_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_items_id_seq', 446, true);


--
-- Name: evidence_levels_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_levels_id_seq', 4, true);


--
-- Name: evidence_types_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_types_id_seq', 3, true);


--
-- Data for Name: gene_aliases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY gene_aliases (id, name) FROM stdin;
1	CD246
2	NBLST3
3	AKT
4	CWS6
5	PKB
6	PKB-ALPHA
7	PRKBA
8	RAC
9	RAC-ALPHA
10	A-RAF
11	ARAF1
12	PKS2
13	RAFA1
14	ABL
15	JTK7
16	bcr/abl
17	c-ABL
18	c-ABL1
19	p150
20	v-abl
21	B-RAF1
22	BRAF1
23	NS7
24	RAFB1
25	BRCAI
26	BRCC1
27	BROVCA1
28	FANCS
29	IRIS
30	PNCA4
31	PPP1R53
32	PSCP
33	RNF53
34	BRCC2
35	BROVCA2
36	FACD
37	FAD
38	FAD1
39	FANCD
40	FANCD1
41	GLM3
42	PNCA2
43	XRCC11
44	BCL1
45	D11S287E
46	PRAD1
47	U21B31
48	KIAK0002
49	MPPH3
50	CCNE
51	MCPH12
52	PLSTIRE
53	CMM3
54	PSK-J3
55	ARF
56	CDK4I
57	CDKN2
58	CMM2
59	INK4
60	INK4A
61	MLM
62	MTS-1
63	MTS1
64	P14
65	P14ARF
66	P16
67	P16-INK4A
68	P16INK4
69	P16INK4A
70	P19
71	P19ARF
72	TP16
73	C/EBP-alpha
74	CEBP
75	HSPF1
76	Hdj1
77	Hsp40
78	RSPH16B
79	Sis1
80	PKACA
81	PPNAD4
82	DNMT3A2
83	M.HsaIIIA
84	TBRS
85	ERBB
86	ERBB1
87	HER1
88	NISBD2
89	PIG61
90	mENA
91	CD340
92	HER-2
93	HER-2/neu
94	HER2
95	MLN 19
96	NEU
97	NGL
98	TKR1
99	ER
100	ESR
101	ESRA
102	ESTRR
103	Era
104	NR3A1
105	BBDS
106	BEK
107	BFR-1
108	CD332
109	CEK3
110	CFD1
111	ECT1
112	JWS
113	K-SAM
114	KGFR
115	TK14
116	TK25
117	ACH
118	CD333
119	CEK2
120	HSFGFR3EX
121	JTK4
122	CD135
123	FLK-2
124	FLK2
125	STK1
126	DCML
127	IMD21
128	MONOMAC
129	NFE1B
130	HEL-216
131	HEL-S-26
132	IDCD
133	IDH
134	IDP
135	IDPC
136	PICD
137	D2HGA2
138	ICD-M
139	IDHM
140	IDPM
141	mNADP-IDH
142	JTK10
143	THCYT3
144	C-Kit
145	CD117
146	PBT
147	SCFR
148	C-K-RAS
149	CFC2
150	K-RAS2A
151	K-RAS2B
152	K-RAS4A
153	K-RAS4B
154	KI-RAS
155	KRAS1
156	KRAS2
157	NS
158	NS3
159	RASK2
160	CFC3
161	MAPKK1
162	MEK1
163	MKK1
164	PRKMK1
165	C-FMS
166	CD115
167	CSF-1R
168	CSFR
169	FIM2
170	FMS
171	HDLS
172	M-CSF-R
173	B23
174	NPM
175	ALPS4
176	CMNS
177	N-ras
178	NCMS
179	NRAS1
180	NS6
181	CLOVE
182	CWS5
183	MCAP
184	MCM
185	MCMTC
186	PI3K
187	p110-alpha
188	CD140A
189	PDGFR-2
190	PDGFR2
191	RHEPDGFRA
192	MYL
193	PP8675
194	RNF71
195	TRIM19
196	NR1B1
197	RAR
198	10q23del
199	BZS
200	CWS1
201	DEC
202	GLM2
203	MHAM
204	MMAC1
205	PTEN1
206	TEP1
207	CDHF12
208	CDHR16
209	HSCR1
210	MEN2A
211	MEN2B
212	MTC1
213	PTC
214	RET-ELE1
215	RET51
216	AML1
217	AML1-EVI-1
218	AMLCR1
219	CBFA2
220	EVI-1
221	PEBP2aB
222	Hsh155
223	MDS
224	PRP10
225	PRPF10
226	SAP155
227	SF3b155
228	BCC7
229	LFS1
230	P53
231	TRP53
232	LAM
233	TSC
234	PPP1R160
235	TSC4
236	FP793
237	RN
238	RNU2AF1
239	U2AF35
240	U2AFBP
241	AWT1
242	EWS-WT1
243	GUD
244	NPHS4
245	WAGR
246	WIT-2
247	WT33
248	FRAP
249	FRAP1
250	FRAP2
251	RAFT1
252	RAPT1
253	AOS5
254	AOVD1
255	TAN1
256	hN1
257	MIG20a
258	NTRKR3
259	TKT
260	TYRO10
261	AUTS9
262	HGFR
263	RCCP2
264	c-Met
\.


--
-- Data for Name: gene_aliases_genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY gene_aliases_genes (gene_alias_id, gene_id) FROM stdin;
1	1
2	1
3	2
4	2
5	2
6	2
7	2
8	2
9	2
10	3
11	3
12	3
13	3
14	4
15	4
16	4
17	4
18	4
19	4
20	4
21	5
22	5
23	5
24	5
25	6
26	6
27	6
28	6
29	6
30	6
31	6
32	6
33	6
34	7
35	7
36	7
37	7
38	7
39	7
40	7
41	7
42	7
43	7
44	8
45	8
46	8
47	8
48	9
49	9
50	11
51	12
52	12
53	13
54	13
55	14
56	14
57	14
58	14
59	14
60	14
61	14
62	14
63	14
64	14
65	14
66	14
67	14
68	14
69	14
70	14
71	14
72	14
73	15
74	15
75	16
76	16
77	16
78	16
79	16
80	17
81	17
82	18
83	18
84	18
85	19
86	19
87	19
88	19
89	19
90	19
91	20
92	20
93	20
94	20
95	20
96	20
97	20
98	20
99	21
100	21
101	21
102	21
103	21
104	21
105	22
106	22
107	22
108	22
109	22
110	22
111	22
112	22
113	22
114	22
115	22
116	22
117	23
118	23
119	23
120	23
121	23
122	24
123	24
124	24
125	24
126	25
127	25
128	25
129	25
130	26
131	26
132	26
133	26
134	26
135	26
136	26
137	27
138	27
133	27
139	27
134	27
140	27
141	27
142	28
143	28
144	29
145	29
146	29
147	29
148	30
149	30
150	30
151	30
152	30
153	30
154	30
155	30
156	30
157	30
158	30
159	30
160	31
161	31
162	31
163	31
164	31
165	33
166	33
167	33
168	33
169	33
170	33
171	33
172	33
173	35
174	35
175	36
176	36
177	36
178	36
179	36
180	36
181	37
182	37
183	37
184	37
185	37
186	37
187	37
188	38
189	38
190	38
191	38
192	39
193	39
194	39
195	39
196	40
197	40
198	41
199	41
200	41
201	41
202	41
203	41
204	41
205	41
206	41
207	42
208	42
209	42
210	42
211	42
212	42
213	42
214	42
215	42
216	43
217	43
218	43
219	43
220	43
221	43
222	44
223	44
224	44
225	44
226	44
227	44
228	45
229	45
230	45
231	45
232	46
233	46
232	47
234	47
235	47
236	48
237	48
238	48
239	48
240	48
241	49
242	49
243	49
244	49
245	49
246	49
247	49
248	50
249	50
250	50
251	50
252	50
253	51
254	51
255	51
256	51
257	52
258	52
259	52
260	52
261	53
262	53
263	53
264	53
\.


--
-- Name: gene_aliases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('gene_aliases_id_seq', 264, true);


--
-- Name: genes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('genes_id_seq', 53, true);


--
-- Data for Name: genes_sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes_sources (gene_id, source_id, created_at, updated_at) FROM stdin;
1	1	2015-03-04 17:25:26.465931	2015-03-04 17:25:26.465931
1	2	2015-03-04 17:25:26.473022	2015-03-04 17:25:26.473022
2	3	2015-03-04 17:25:26.750587	2015-03-04 17:25:26.750587
2	4	2015-03-04 17:25:26.756117	2015-03-04 17:25:26.756117
3	5	2015-03-04 17:25:26.990699	2015-03-04 17:25:26.990699
3	6	2015-03-04 17:25:26.994393	2015-03-04 17:25:26.994393
4	7	2015-03-04 17:25:27.263444	2015-03-04 17:25:27.263444
4	8	2015-03-04 17:25:27.268029	2015-03-04 17:25:27.268029
5	9	2015-03-04 17:25:27.518974	2015-03-04 17:25:27.518974
5	10	2015-03-04 17:25:27.522847	2015-03-04 17:25:27.522847
6	11	2015-03-04 17:25:27.771777	2015-03-04 17:25:27.771777
7	11	2015-03-04 17:25:28.042495	2015-03-04 17:25:28.042495
8	12	2015-03-04 17:25:28.309976	2015-03-04 17:25:28.309976
8	13	2015-03-04 17:25:28.31435	2015-03-04 17:25:28.31435
9	12	2015-03-04 17:25:28.546209	2015-03-04 17:25:28.546209
9	13	2015-03-04 17:25:28.548704	2015-03-04 17:25:28.548704
10	12	2015-03-04 17:25:28.767964	2015-03-04 17:25:28.767964
10	13	2015-03-04 17:25:28.770586	2015-03-04 17:25:28.770586
11	14	2015-03-04 17:25:28.97641	2015-03-04 17:25:28.97641
12	15	2015-03-04 17:25:29.192359	2015-03-04 17:25:29.192359
12	16	2015-03-04 17:25:29.197552	2015-03-04 17:25:29.197552
13	15	2015-03-04 17:25:29.405332	2015-03-04 17:25:29.405332
13	16	2015-03-04 17:25:29.407589	2015-03-04 17:25:29.407589
14	17	2015-03-04 17:25:29.706441	2015-03-04 17:25:29.706441
14	18	2015-03-04 17:25:29.710473	2015-03-04 17:25:29.710473
15	19	2015-03-04 17:25:30.076272	2015-03-04 17:25:30.076272
15	20	2015-03-04 17:25:30.080763	2015-03-04 17:25:30.080763
15	21	2015-03-04 17:25:30.085523	2015-03-04 17:25:30.085523
16	22	2015-03-04 17:25:30.318853	2015-03-04 17:25:30.318853
17	22	2015-03-04 17:25:30.540735	2015-03-04 17:25:30.540735
17	23	2015-03-04 17:25:30.545165	2015-03-04 17:25:30.545165
18	24	2015-03-04 17:25:30.836015	2015-03-04 17:25:30.836015
18	25	2015-03-04 17:25:30.840007	2015-03-04 17:25:30.840007
19	26	2015-03-04 17:25:31.085597	2015-03-04 17:25:31.085597
19	27	2015-03-04 17:25:31.089221	2015-03-04 17:25:31.089221
20	28	2015-03-04 17:25:31.353094	2015-03-04 17:25:31.353094
20	29	2015-03-04 17:25:31.356679	2015-03-04 17:25:31.356679
21	30	2015-03-04 17:25:31.619543	2015-03-04 17:25:31.619543
21	31	2015-03-04 17:25:31.623455	2015-03-04 17:25:31.623455
22	32	2015-03-04 17:25:31.893935	2015-03-04 17:25:31.893935
22	33	2015-03-04 17:25:31.897624	2015-03-04 17:25:31.897624
23	32	2015-03-04 17:25:32.203546	2015-03-04 17:25:32.203546
23	33	2015-03-04 17:25:32.205977	2015-03-04 17:25:32.205977
24	34	2015-03-04 17:25:32.4403	2015-03-04 17:25:32.4403
24	19	2015-03-04 17:25:32.442717	2015-03-04 17:25:32.442717
25	35	2015-03-04 17:25:32.682338	2015-03-04 17:25:32.682338
25	36	2015-03-04 17:25:32.686387	2015-03-04 17:25:32.686387
26	37	2015-03-04 17:25:32.954355	2015-03-04 17:25:32.954355
26	38	2015-03-04 17:25:32.958064	2015-03-04 17:25:32.958064
27	37	2015-03-04 17:25:33.216658	2015-03-04 17:25:33.216658
27	38	2015-03-04 17:25:33.219638	2015-03-04 17:25:33.219638
27	39	2015-03-04 17:25:33.223876	2015-03-04 17:25:33.223876
28	40	2015-03-04 17:25:33.453853	2015-03-04 17:25:33.453853
29	41	2015-03-04 17:25:33.685093	2015-03-04 17:25:33.685093
30	42	2015-03-04 17:25:33.96021	2015-03-04 17:25:33.96021
35	43	2015-03-04 17:25:35.070361	2015-03-04 17:25:35.070361
35	19	2015-03-04 17:25:35.072796	2015-03-04 17:25:35.072796
42	44	2015-03-04 17:25:36.732396	2015-03-04 17:25:36.732396
42	45	2015-03-04 17:25:36.737756	2015-03-04 17:25:36.737756
44	46	2015-03-04 17:25:37.293956	2015-03-04 17:25:37.293956
44	47	2015-03-04 17:25:37.314592	2015-03-04 17:25:37.314592
45	48	2015-03-04 17:25:37.603481	2015-03-04 17:25:37.603481
48	49	2015-03-04 17:25:38.269667	2015-03-04 17:25:38.269667
48	50	2015-03-04 17:25:38.273284	2015-03-04 17:25:38.273284
\.


--
-- Data for Name: subscriptions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY subscriptions (id, user_id, subscribable_id, subscribable_type, type, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: notifications; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY notifications (id, subscription_id, user_id, subscribable_id, subscribable_type, content, url, acknowledged, delivered, created_at, updated_at) FROM stdin;
\.


--
-- Name: notifications_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('notifications_id_seq', 1, false);


--
-- Data for Name: ratings; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY ratings (id, value, evidence_item_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Name: ratings_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('ratings_id_seq', 1, false);


--
-- Data for Name: roles; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles (id, name, description, created_at, updated_at) FROM stdin;
\.


--
-- Name: roles_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('roles_id_seq', 1, false);


--
-- Data for Name: roles_users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles_users (role_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY schema_migrations (version) FROM stdin;
20150303202505
20140805184308
20141021192035
20141022180523
20141022195329
20141108234203
20141108234935
20141109000136
20141114222425
20150107205621
20150115231117
20150116220720
20150116230539
20150116230632
20150204192653
20150205000010
20150205205734
20150205210725
20150210191015
20150210204445
20150219202227
20150220160805
20150220182109
20150223170106
20150227204142
20150227212146
\.


--
-- Name: sources_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('sources_id_seq', 244, true);


--
-- Name: subscriptions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('subscriptions_id_seq', 1, false);


--
-- Data for Name: suggested_changes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY suggested_changes (id, suggested_changes, moderated_id, moderated_type, user_id, status, created_at, updated_at) FROM stdin;
\.


--
-- Name: suggested_changes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('suggested_changes_id_seq', 1, false);


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('users_id_seq', 1, false);


--
-- Data for Name: variant_groups; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_groups (id, name, description, created_at, updated_at) FROM stdin;
10	KIT Exon 17	\N	2015-03-04 17:26:01.413031	2015-03-04 17:26:01.413031
11	KIT Exon 11	\N	2015-03-04 17:26:01.491908	2015-03-04 17:26:01.491908
1	Imatinib Resistance	While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. 	2015-03-04 17:25:44.385335	2015-03-04 17:26:18.12504
2	ALK Fusions	ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase the activity of the ALK oncogene relative to normal cells. While EML4 is the most common fusion partner, other 5' partners have been observed. The EML4-ALK fusion has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. 	2015-03-04 17:25:44.707599	2015-03-04 17:26:18.12944
6	TKI Resistance	EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine kinase inhibitors have been developed to target EGFR pathway activity. One such inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting. However, the T790M missense mutation has shown to confer resistance to this inhibitor in cell lines and case studies.  	2015-03-04 17:25:54.616214	2015-03-04 17:26:18.13262
3	Crizotinib Resistance	The ALK oncogene has long been considered a driving factor in non-small cell lung cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive this resistance. 	2015-03-04 17:25:44.786026	2015-03-04 17:26:18.135517
4	Other V600's	While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants have also been observed and studied to a lesser degree. At first approximation, many of these variants seem to behave similarly to V600E, and treatment with dabrafenib has been shown to be effective. 	2015-03-04 17:25:46.252591	2015-03-04 17:26:18.137922
7	HER2 Activating	HER2-positive breast cancer is widely recognized as a molecular subtype of breast cancer. While amplification of ERBB2 is a common mechanism for activation of the pathway, missense mutations have also demonstrated activation potential. These activating missense mutations also confer senstivity to the targeted therapeutic neratinib. 	2015-03-04 17:25:55.196757	2015-03-04 17:26:18.140273
8	ESR1 Ligand-Binding Domain	ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone therapy has been widely effective in treating the disease, however sequencing of resistant patients has uncovered a number of missense mutations in mediating this resistance. Many of these mutations lie within the ligand-binding domain, and contribute to constitutive activity of the receptor. This has lead to the development of estrogen receptor degrading agents such as fulvestrant, which have shown early efficacy in clinical trials.  	2015-03-04 17:25:56.130103	2015-03-04 17:26:18.142707
9	FGFR fusions	FGFR fusions have been demonstrated across many cancer types to have oncogenic potential. FGFR2 and FGFR3 have been shown to be fused to many 3' partners, which may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2 fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib and pazopanib have shown efficacy in case studies.  	2015-03-04 17:25:57.203603	2015-03-04 17:26:18.145219
12	NPM1 Exon 12	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-03-04 17:26:09.642424	2015-03-04 17:26:18.148371
13	PTEN Loss-of-Function	PTEN loss is a common event in breast cancer, as well as others. The most common single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell lines harboring this mutation behave similarly to cell lines harboring larger insertion or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded growth. 	2015-03-04 17:26:13.488777	2015-03-04 17:26:18.151584
14	Motesanib Resistance	RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment of these patients with the targeted therapeutic motesanib has shown to be effective. However, the missense mutations C634W and M918T have shown to confer motesanib resistance in cell lines. 	2015-03-04 17:26:13.768609	2015-03-04 17:26:18.154793
15	TSC Loss	Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types. Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor everolimus has seen significant anti-tumorigenic activity. 	2015-03-04 17:26:15.369038	2015-03-04 17:26:18.157906
5	BRCA Germline Variants	BRCA germline variants have long been recognized as important potential predictors of breast and ovarian cancer risk. 	2015-03-04 17:25:48.73627	2015-03-04 17:26:18.160786
\.


--
-- Data for Name: variant_group_variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_group_variants (variant_id, variant_group_id, created_at, updated_at) FROM stdin;
3	1	2015-03-04 17:25:44.394377	2015-03-04 17:25:44.394377
2	1	2015-03-04 17:25:44.470308	2015-03-04 17:25:44.470308
2	1	2015-03-04 17:25:44.559186	2015-03-04 17:25:44.559186
5	2	2015-03-04 17:25:44.710295	2015-03-04 17:25:44.710295
6	2	2015-03-04 17:25:44.783494	2015-03-04 17:25:44.783494
6	3	2015-03-04 17:25:44.788576	2015-03-04 17:25:44.788576
7	2	2015-03-04 17:25:44.9204	2015-03-04 17:25:44.9204
7	3	2015-03-04 17:25:44.923868	2015-03-04 17:25:44.923868
8	3	2015-03-04 17:25:45.369573	2015-03-04 17:25:45.369573
11	4	2015-03-04 17:25:46.257095	2015-03-04 17:25:46.257095
13	4	2015-03-04 17:25:48.406679	2015-03-04 17:25:48.406679
15	4	2015-03-04 17:25:48.48807	2015-03-04 17:25:48.48807
16	4	2015-03-04 17:25:48.566922	2015-03-04 17:25:48.566922
129	5	2015-03-04 17:25:48.739457	2015-03-04 17:25:48.739457
130	5	2015-03-04 17:25:48.819722	2015-03-04 17:25:48.819722
34	6	2015-03-04 17:25:54.619108	2015-03-04 17:25:54.619108
34	6	2015-03-04 17:25:55.115628	2015-03-04 17:25:55.115628
35	7	2015-03-04 17:25:55.200531	2015-03-04 17:25:55.200531
36	7	2015-03-04 17:25:55.277615	2015-03-04 17:25:55.277615
38	7	2015-03-04 17:25:55.427075	2015-03-04 17:25:55.427075
41	7	2015-03-04 17:25:55.717273	2015-03-04 17:25:55.717273
43	7	2015-03-04 17:25:55.92051	2015-03-04 17:25:55.92051
44	7	2015-03-04 17:25:55.991538	2015-03-04 17:25:55.991538
45	7	2015-03-04 17:25:56.063712	2015-03-04 17:25:56.063712
46	8	2015-03-04 17:25:56.133015	2015-03-04 17:25:56.133015
46	8	2015-03-04 17:25:56.284767	2015-03-04 17:25:56.284767
47	8	2015-03-04 17:25:56.384884	2015-03-04 17:25:56.384884
47	8	2015-03-04 17:25:56.510852	2015-03-04 17:25:56.510852
48	8	2015-03-04 17:25:56.581611	2015-03-04 17:25:56.581611
48	8	2015-03-04 17:25:56.71054	2015-03-04 17:25:56.71054
49	8	2015-03-04 17:25:56.779796	2015-03-04 17:25:56.779796
49	8	2015-03-04 17:25:56.926606	2015-03-04 17:25:56.926606
50	8	2015-03-04 17:25:56.994577	2015-03-04 17:25:56.994577
50	8	2015-03-04 17:25:57.126498	2015-03-04 17:25:57.126498
51	9	2015-03-04 17:25:57.207382	2015-03-04 17:25:57.207382
52	9	2015-03-04 17:25:57.313618	2015-03-04 17:25:57.313618
53	9	2015-03-04 17:25:57.44337	2015-03-04 17:25:57.44337
64	10	2015-03-04 17:26:01.415723	2015-03-04 17:26:01.415723
65	11	2015-03-04 17:26:01.494526	2015-03-04 17:26:01.494526
65	11	2015-03-04 17:26:01.568385	2015-03-04 17:26:01.568385
65	11	2015-03-04 17:26:01.642889	2015-03-04 17:26:01.642889
65	11	2015-03-04 17:26:01.714786	2015-03-04 17:26:01.714786
65	11	2015-03-04 17:26:01.796649	2015-03-04 17:26:01.796649
65	11	2015-03-04 17:26:01.887118	2015-03-04 17:26:01.887118
69	10	2015-03-04 17:26:02.129969	2015-03-04 17:26:02.129969
69	10	2015-03-04 17:26:02.218374	2015-03-04 17:26:02.218374
66	11	2015-03-04 17:26:02.461782	2015-03-04 17:26:02.461782
71	11	2015-03-04 17:26:02.547072	2015-03-04 17:26:02.547072
71	11	2015-03-04 17:26:02.763625	2015-03-04 17:26:02.763625
71	11	2015-03-04 17:26:02.968369	2015-03-04 17:26:02.968369
72	11	2015-03-04 17:26:03.054709	2015-03-04 17:26:03.054709
72	11	2015-03-04 17:26:03.134217	2015-03-04 17:26:03.134217
146	6	2015-03-04 17:26:04.312988	2015-03-04 17:26:04.312988
77	6	2015-03-04 17:26:04.553634	2015-03-04 17:26:04.553634
78	6	2015-03-04 17:26:05.077293	2015-03-04 17:26:05.077293
145	6	2015-03-04 17:26:05.241859	2015-03-04 17:26:05.241859
144	6	2015-03-04 17:26:06.204848	2015-03-04 17:26:06.204848
86	12	2015-03-04 17:26:09.645977	2015-03-04 17:26:09.645977
86	12	2015-03-04 17:26:09.731853	2015-03-04 17:26:09.731853
86	12	2015-03-04 17:26:09.819327	2015-03-04 17:26:09.819327
86	12	2015-03-04 17:26:09.905022	2015-03-04 17:26:09.905022
96	1	2015-03-04 17:26:11.094761	2015-03-04 17:26:11.094761
97	1	2015-03-04 17:26:11.174443	2015-03-04 17:26:11.174443
97	1	2015-03-04 17:26:11.253542	2015-03-04 17:26:11.253542
97	1	2015-03-04 17:26:11.335146	2015-03-04 17:26:11.335146
97	1	2015-03-04 17:26:11.419142	2015-03-04 17:26:11.419142
98	1	2015-03-04 17:26:11.500409	2015-03-04 17:26:11.500409
99	1	2015-03-04 17:26:11.581447	2015-03-04 17:26:11.581447
100	1	2015-03-04 17:26:11.661559	2015-03-04 17:26:11.661559
108	13	2015-03-04 17:26:13.49238	2015-03-04 17:26:13.49238
108	13	2015-03-04 17:26:13.61546	2015-03-04 17:26:13.61546
110	14	2015-03-04 17:26:13.772591	2015-03-04 17:26:13.772591
111	14	2015-03-04 17:26:13.930716	2015-03-04 17:26:13.930716
122	15	2015-03-04 17:26:15.371911	2015-03-04 17:26:15.371911
122	15	2015-03-04 17:26:15.453601	2015-03-04 17:26:15.453601
123	15	2015-03-04 17:26:15.53927	2015-03-04 17:26:15.53927
124	15	2015-03-04 17:26:15.622834	2015-03-04 17:26:15.622834
\.


--
-- Name: variant_groups_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_groups_id_seq', 15, true);


--
-- Name: variant_origins_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_origins_id_seq', 2, true);


--
-- Name: variants_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variants_id_seq', 150, true);


--
-- PostgreSQL database dump complete
--

