--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

SET search_path = public, pg_catalog;

--
-- Data for Name: audits; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY audits (id, auditable_id, auditable_type, associated_id, associated_type, user_id, user_type, username, action, audited_changes, version, comment, remote_address, request_uuid, created_at) FROM stdin;
1	1	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 238\nname: ALK\ndescription: ALK amplifications, fusions and mutations have been shown to be driving\n  events in non-small cell lung cancer. While crizontinib has demonstrated efficacy\n  in treating the amplification, mutations in ALK have been shown to confer resistance\n  to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied\n  success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been\n  shown to be cytostatic in ALK-altered cell lines.\nofficial_name: anaplastic lymphoma receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	9b4b2179-c7dd-441c-b345-6283b701e257	2015-02-24 20:42:38.667873
2	2	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 207\nname: AKT1\ndescription: AKT1, also referred to as protein kinase B, is a known oncogene. AKT\n  activation relies on the PI3K pathway, and is recognized as a critical node in the\n  pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been\n  shown to result in activation of the protein. Mutations in AKT1 have also been shown\n  to confer resistance to allosteric kinase inhibitors in vitro.\nofficial_name: v-akt murine thymoma viral oncogene homolog 1\nclinical_description: \n	1	\N	\N	eadecd90-569e-42a8-8359-ebd83a842411	2015-02-24 20:42:38.887093
3	3	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 369\nname: ARAF\ndescription: ARAF has recently become increasingly considered for its oncogenic potential.\n  Its potential as a target for informing clinical action was demonstrated by a single\n  case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated\n  with sorafenib, acheived near-complete clinical remission. This finding has brought\n  new focus on ARAF as a marker that should be assayed for in cancer treatment.\nofficial_name: A-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	fb1d604f-9690-4cb0-b018-fa635f634100	2015-02-24 20:42:39.089398
4	4	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 25\nname: ABL1\ndescription: ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein\n  that has become a signature of chronic myeloid leukemia (CML). Cells harboring this\n  fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook\n  of the disease. However, additional mutations in ABL1 have been shown to confer\n  resistance to imatinib. In these resistance cases, second-generation tyrosine kinase\n  inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are\n  currently undergoing clinical trials for treating acquired resistance in CML.\nofficial_name: ABL proto-oncogene 1, non-receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	a0e7c1ad-5e27-4245-a81e-689d21cb00e8	2015-02-24 20:42:39.302976
5	5	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 673\nname: BRAF\ndescription: BRAF mutations are found to be recurrent in many cancer types. Of these,\n  the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E\n  has been determined to be an activating mutation, and cells that harbor it, along\n  with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is\n  also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK\n  inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations\n  have also been correlated with poor prognosis in many cancer types, although there\n  is at least one study that questions this conclusion in papillary thyroid cancer.\nofficial_name: B-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	c0523f15-7fc3-4c99-92f1-e90990eb68a7	2015-02-24 20:42:39.517732
6	6	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 672\nname: BRCA1\ndescription: N/A\nofficial_name: breast cancer 1, early onset\nclinical_description: \n	1	\N	\N	37abd00d-ed5d-4789-ad72-237af9f60ba8	2015-02-24 20:42:39.727054
7	7	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 675\nname: BRCA2\ndescription: N/A\nofficial_name: breast cancer 2, early onset\nclinical_description: \n	1	\N	\N	6bd052b5-161e-4de7-82ff-3f5a1ad737b5	2015-02-24 20:42:39.916184
8	8	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 595\nname: CCND1\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D1\nclinical_description: \n	1	\N	\N	69f93a84-696b-49af-a1a9-8fde7d988b56	2015-02-24 20:42:40.123859
9	9	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 894\nname: CCND2\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D2\nclinical_description: \n	1	\N	\N	0b94c2b6-3f77-4c81-af77-ff5d71defe2e	2015-02-24 20:42:40.328379
10	10	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 896\nname: CCND3\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared\n  to the amplifcations and overexpressions of the other cyclin D's. In a mouse study,\n  the targeted therapeutic palbociclib significantly increased the median survival\n  of the cyclin D3 knockouts.\nofficial_name: cyclin D3\nclinical_description: \n	1	\N	\N	cc9c19cc-2811-47a8-9432-6e6378393aa0	2015-02-24 20:42:40.579622
11	11	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 898\nname: CCNE1\ndescription: Cyclin E, while currenly not as widely implicated as its cyclin D counterparts,\n  has been implicated in various carcinomas, including breast, gastric, stomach and\n  colorectal. High levels of cyclin E, either by gene amplification or overexpression,\n  have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements\n  are correlated with later stage disease. In lung cancer, neoplastic cells with higher\n  levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly\n  expressed counterparts.\nofficial_name: cyclin E1\nclinical_description: \n	1	\N	\N	7cad9438-4775-4f35-84a5-25f1b75c50c3	2015-02-24 20:42:40.789334
30	30	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3845\nname: KRAS\ndescription: N/A\nofficial_name: Kirsten rat sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	126ffe17-a6de-420f-a2ed-d3948df4b61d	2015-02-24 20:42:44.762583
31	31	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5604\nname: MAP2K1\ndescription: N/A\nofficial_name: mitogen-activated protein kinase kinase 1\nclinical_description: \n	1	\N	\N	2825b163-38d6-4a3e-871a-25a5dbe911e5	2015-02-24 20:42:44.98548
12	12	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1021\nname: CDK6\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 6\nclinical_description: \n	1	\N	\N	edeea83b-5d2d-457a-ab14-6f1774bbc591	2015-02-24 20:42:40.992192
13	13	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1019\nname: CDK4\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 4\nclinical_description: \n	1	\N	\N	1f72b483-c2f3-4a18-a65c-91d0ea152308	2015-02-24 20:42:41.193469
14	14	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1029\nname: CDKN2A\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. While no targeted therapeutic has been engaged in clinical trials, the prognostic\n  impact has been studied by a number of meta-analyses. One mechanism by which loss\n  of CDKN2A can occur is by hypermethylation of the promoter region for the gene.\n  However, the prognostic impact of promoter hypermethylation has been relatively\n  ambiguous. Many studies have suggesting poorer prognostic outcome for patients with\n  hypermethylation in colorectal, liver, and younger lung cancer patients. This being\n  said, there is still research to be done before this becomes a widely-accepted prognostic\n  indicator.\nofficial_name: cyclin-dependent kinase inhibitor 2A\nclinical_description: \n	1	\N	\N	3a59b0ff-bcdc-4544-9e24-d9993bdfd2b2	2015-02-24 20:42:41.395546
15	15	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1050\nname: CEBPA\ndescription: AML with mutated CEBPA' is a provisional entity in the WHO classification\n  of acute myeloid leukemia (AML) and is recommended to be tested for in patients\n  with AML. CEBPA mutations are particularly associated with cytogenetically normal\n  AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation\n  in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha\n  protein while C-terminal mutations reduce the DNA-binding potential of this transcription\n  factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1\n  and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations\n  may have prognostic implications.\nofficial_name: CCAAT/enhancer binding protein (C/EBP), alpha\nclinical_description: \n	1	\N	\N	e59c5bfc-efce-4014-af88-99c50eb8aa88	2015-02-24 20:42:41.619947
16	16	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3337\nname: DNAJB1\ndescription: DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating\n  p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1\n  to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In\n  one study, this fusion was observed in 15/15 FL-HCC cases examined and functional\n  studies found that the fusion retained kinase activity.\nofficial_name: DnaJ (Hsp40) homolog, subfamily B, member 1\nclinical_description: \n	1	\N	\N	3b0ef6c9-8063-46c9-a3d2-2ebf40c92a10	2015-02-24 20:42:41.819643
17	17	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5566\nname: PRKACA\ndescription: PRKACA has been studied in breast cancer and has been found to mediate\n  resistance to HER2 targeted therapies. It has also been found to contain a mutation\n  hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular\n  fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be\n  a diagnostic marker for this rare subtype of HCC. In one study, this fusion was\n  observed in 15/15 FL-HCC cases examined and functional studies found that the fusion\n  retained kinase activity.\nofficial_name: protein kinase, cAMP-dependent, catalytic, alpha\nclinical_description: \n	1	\N	\N	ded01167-fe88-48eb-b709-d690d2d2bb34	2015-02-24 20:42:42.021182
18	18	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1788\nname: DNMT3A\ndescription: DNMT3A is one of several epigenetic modifiers identified as recurrently\n  mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically\n  normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant\n  negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers.\n  AML patient bone marrow harboring R882 mutations were similarly demonstrated to\n  be hypomethylated compared to patients with wildtype DNMT3A. These studies also\n  indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner\n  from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes\n  and treatment protocols may need to be considered for these two classes of DNMT3A\n  mutations.\nofficial_name: DNA (cytosine-5-)-methyltransferase 3 alpha\nclinical_description: \n	1	\N	\N	91d6295f-a7c1-4f4d-8a32-7a4e16fc5847	2015-02-24 20:42:42.23168
19	19	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1956\nname: EGFR\ndescription: EGFR is widely recognized for its importance in cancer. Amplification\n  and mutations have been shown to be driving events in many cancer types. Its role\n  in non-small cell lung cancer has spurred many research and drug development efforts.\n  Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably\n  gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance\n  to these drugs, particularly the variant T790M, which has been functionally characterized\n  as a resistance marker for both of these drugs. The later generation TKI's have\n  seen some success in treating these resistant cases, and targeted sequencing of\n  the EGFR locus has become a common practice in non-small cell lung cancer.\nofficial_name: epidermal growth factor receptor\nclinical_description: \n	1	\N	\N	35eaf80d-8b6c-490f-9e32-e551a17d0f79	2015-02-24 20:42:42.440286
20	20	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2064\nname: ERBB2\ndescription: ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast\n  cancer, and is treated in a separate manner than the other subtypes of breast cancer.\n  Apart from being amplified, ERBB2 activating mutations have been shown to have clinical\n  importance in HER2-negative breast cancer. These mutations have shown sensitivity\n  to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical\n  sequencing efforts in treating breast cancer.\nofficial_name: erb-b2 receptor tyrosine kinase 2\nclinical_description: \n	1	\N	\N	8aeea7e6-f967-40d0-a346-c6a82f34b79b	2015-02-24 20:42:42.649297
32	32	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4209\nname: MEF2D\ndescription: N/A\nofficial_name: myocyte enhancer factor 2D\nclinical_description: \n	1	\N	\N	438d7333-608e-4f72-96e5-e68b92cb7e3f	2015-02-24 20:42:45.172302
33	33	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1436\nname: CSF1R\ndescription: N/A\nofficial_name: colony stimulating factor 1 receptor\nclinical_description: \n	1	\N	\N	b2034f65-ca22-4665-9c18-36f2c3b43f67	2015-02-24 20:42:45.362914
34	34	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4255\nname: MGMT\ndescription: N/A\nofficial_name: O-6-methylguanine-DNA methyltransferase\nclinical_description: \n	1	\N	\N	791325e1-abd9-4ef3-9d4a-1bdc0ea40c4c	2015-02-24 20:42:45.554897
21	21	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2099\nname: ESR1\ndescription: ESR1 has been a focus in breast cancer for quite some time, but has also\n  shown significance in endometrial, ovarian and other cancer types. ER-positive breast\n  cancer that is resistant to hormone therapy has instigated clinical sequencing efforts\n  to shed light on the mechanisms of this resistance. A number of mutations in the\n  ligand binding domain of the protein have been implicated in hormone resistance\n  and anti-estrogen therapies. This has spurred efforts to develop therapeutics that\n  act to degrade the protein, rather than act as an antagonist. These agents are currently\n  in clinical trials and have seen some success, highlighting the importance of sequencing\n  efforts in treating breast cancer.\nofficial_name: estrogen receptor 1\nclinical_description: \n	1	\N	\N	1af3436c-ac2a-4d16-97c1-b2cd5885c300	2015-02-24 20:42:42.877851
22	22	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2263\nname: FGFR2\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions that lead to pathway\n  actiation have been demonstrated to have oncogenic potential across multiple cancer\n  types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success\n  in treating over-active FGFR signalling, prompting use of diagnostic sequencing\n  targeting the FGFR genes, especially in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 2\nclinical_description: \n	1	\N	\N	9c51ed28-a5b7-4610-b241-8691c2cac75f	2015-02-24 20:42:43.084763
23	23	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2261\nname: FGFR3\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions that lead to pathway\n  actiation have been demonstrated to have oncogenic potential across multiple cancer\n  types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success\n  in treating over-active FGFR signalling, prompting use of diagnostic sequencing\n  targeting the FGFR genes, especially in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 3\nclinical_description: \n	1	\N	\N	87fd95a0-3b12-4e4f-8d4f-1905043ed3ab	2015-02-24 20:42:43.296827
24	24	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2322\nname: FLT3\ndescription: FLT3 is an important cytokine receptor involved in normal hematopoiesis.\n  Mutations in this gene are common in acute myeloid leukemia (AML) and screening\n  for mutations in this gene has been recommended by the World Health Organization\n  in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML).\n  FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated\n  with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have\n  been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD\n  mutations are still up for debate.\nofficial_name: fms-related tyrosine kinase 3\nclinical_description: \n	1	\N	\N	2420f125-c58e-47ad-8b6d-5a119a88983b	2015-02-24 20:42:43.513275
25	25	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2624\nname: GATA2\ndescription: GATA2 is a transcription factor involved in stem cell maintenance with\n  key roles in hematopoietic development. GATA2 mutations are associated with a variety\n  of inherited and acquired immune disorders including myelodysplastic syndrome and\n  acute myeloid leukemia. In addition to a role in hematopoiesis, the maintenance\n  GATA2 expression has been implicated as a requirement in KRAS-driven non-small cell\n  lung cancer. Preclinical models have indicated therapeutic benefit from targeting\n  GATA2-mediated pathways in the context of KRAS-driven NSCLC.\nofficial_name: GATA binding protein 2\nclinical_description: \n	1	\N	\N	bd8c17a4-b69b-48e7-9a29-667a80a0143c	2015-02-24 20:42:43.718408
26	26	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3417\nname: IDH1\ndescription: IDH1 mutations have been observed in a number of cancer types, including\n  sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in\n  the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial\n  (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described\n  as mutually exclusive in many of these cancer types. The most frequent mutations\n  involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic\n  enzyme activity. The implications of mutations in this gene vary greatly by cancer\n  type. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations\n  have been associated with worse outcome, shorter overall survival, and normal karyotype.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Unlike the association\n  with cytogenetically normal AML, in glioblastoma, IDH1 mutations have been associated\n  with specific cytogenetic abnormalities, 1p and 19q deletions.\nofficial_name: isocitrate dehydrogenase 1 (NADP+), soluble\nclinical_description: \n	1	\N	\N	7a0c3a59-7118-4977-a700-38258e598c33	2015-02-24 20:42:43.945786
27	27	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3418\nname: IDH2\ndescription: IDH2 mutations have been observed in a number of cancer types, including\n  sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in\n  the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial\n  (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described\n  as mutually exclusive in many of these cancer types. The most frequent mutations\n  involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic\n  enzyme activity. Although IDH2 (R172) mutations are associated with poorer overall\n  prognosis in AML patients, its utility as a prognostic marker in MDS is still under\n  debate. Additionally, IDH2 (R140) has been associated with improved overall survival\n  in AML. IDH2 mutations have been associated with improved prognosis in gliomas.\nofficial_name: isocitrate dehydrogenase 2 (NADP+), mitochondrial\nclinical_description: \n	1	\N	\N	8beee39f-5399-4e2c-b103-c3ef550acc9a	2015-02-24 20:42:44.169885
28	28	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3717\nname: JAK2\ndescription: JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative\n  diseases and cancers. The mutation V617F is the most clinically relevant variant,\n  and is seen in around half of myeloproliferative disorders. The variant is a known\n  activating mutation, and activated JAK2 is sufficient to drive myeloproliferative\n  disorders in mouse models. V617F, while most recurrent, is not the only mechanism\n  by which JAK2 can be activated in patients. JAK2 is now one of the first diagnostic\n  markers tested upon diagnosis with a myeloproliferative disorder.\nofficial_name: Janus kinase 2\nclinical_description: \n	1	\N	\N	61adf9a6-5212-4d3a-bbe1-d1cffee51c35	2015-02-24 20:42:44.381379
29	29	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3815\nname: KIT\ndescription: N/A\nofficial_name: v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	759b92fc-40d5-496f-8131-4ba610811284	2015-02-24 20:42:44.571952
35	35	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4869\nname: NPM1\ndescription: AML with mutated NPM1 is provisional entity in the WHO classification\n  of AML and is recommended to be tested for in patients with cytogenetically normal\n  AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they\n  have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently\n  W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations\n  in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing\n  the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia,\n  indicating it may require additional mutations to promote leukemic development.\nofficial_name: nucleophosmin (nucleolar phosphoprotein B23, numatrin)\nclinical_description: \n	1	\N	\N	6fa594a6-d04d-49ae-9154-f232e8b4195c	2015-02-24 20:42:45.762786
36	36	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4893\nname: NRAS\ndescription: N/A\nofficial_name: neuroblastoma RAS viral (v-ras) oncogene homolog\nclinical_description: \n	1	\N	\N	26ecfc9e-2955-4926-836b-07fc67b8c229	2015-02-24 20:42:45.958193
37	37	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5290\nname: PIK3CA\ndescription: PIK3CA is the most recurrently mutated gene in breast cancer, and has\n  been found to important in a number of cancer types. An integral part of the PI3K\n  pathway, PIK3CA has long been described as an oncogene, with two main hotspots for\n  activating mutations, the 542/545 region of the helical domain, and the 1047 region\n  of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways,\n  is the subject of an immense amount of research and development, and PI3K inhibition\n  has seen some limited success in recent clinical trials. While monotherapies seem\n  to be limited in their potential, there is a recent interest in pursuing PI3K inhibition\n  as part of a combination therapy regiment with inhibition partners including TKI's,\n  MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.\nofficial_name: phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha\nclinical_description: \n	1	\N	\N	de7f5063-1f11-4e42-b1e1-c695745f83e6	2015-02-24 20:42:46.168352
38	38	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5156\nname: PDGFRA\ndescription: N/A\nofficial_name: platelet-derived growth factor receptor, alpha polypeptide\nclinical_description: \n	1	\N	\N	2298c2e9-3ef0-4f83-8ac4-704b2dc3bb98	2015-02-24 20:42:46.3792
39	39	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5371\nname: PML\ndescription: The PML-RARA fusion is the result of a recurrent, balanced translocation\n  between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event\n  in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown\n  sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA\n  fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for\n  treating these patients, and early results seem promising.\nofficial_name: promyelocytic leukemia\nclinical_description: \n	1	\N	\N	105bcd95-e02d-49fe-81d3-774f1d9a47aa	2015-02-24 20:42:46.576676
40	40	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5914\nname: RARA\ndescription: The PML-RARA fusion is the result of a recurrent, balanced translocation\n  between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event\n  in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown\n  sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA\n  fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for\n  treating these patients, and early results seem promising.\nofficial_name: retinoic acid receptor, alpha\nclinical_description: \n	1	\N	\N	1e8c59db-b29b-4407-883f-6c3347f11dbd	2015-02-24 20:42:46.76926
41	41	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5728\nname: PTEN\ndescription: N/A\nofficial_name: phosphatase and tensin homolog\nclinical_description: \n	1	\N	\N	afabc731-6855-49b7-aa07-63d7f44b6aea	2015-02-24 20:42:46.962297
42	42	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5979\nname: RET\ndescription: RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine\n  kinase receptor and are associated with thyroid cancers. RET point mutations are\n  the most common mutations identified in medullary thyroid cancer (MTC) with germline\n  and somatic mutations in RET associated with hereditary and sporadic forms, respectively.\n  The most common somatic form mutation is M918T (exon 16) and a variety of other\n  mutations effecting exons 10, 11 and 15 have been described. The prognostic significance\n  of these mutations have been hotly debated in the field, however, data suggests\n  that some RET mutation may confer drug resistence. No RET-specific agents are currently\n  clinically available but several promiscuous kinase inhibitors that target RET,\n  among others, have been approved for MTC treatment.\nofficial_name: ret proto-oncogene\nclinical_description: \n	1	\N	\N	e24dfe83-f05b-4e7c-b408-63ec6ee7c434	2015-02-24 20:42:47.169365
43	43	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 861\nname: RUNX1\ndescription: N/A\nofficial_name: runt-related transcription factor 1\nclinical_description: \n	1	\N	\N	a4b48dcd-711a-49ad-959b-5e3c6e6fbc37	2015-02-24 20:42:47.358723
44	44	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 23451\nname: SF3B1\ndescription: SF3B1 mutations have been described in several myeloid malignancies,\n  predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies\n  and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has\n  been identified as recurrently mutated in MDS and other malignanices. The mutations\n  affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally\n  deleterious with K700E described as a major hotspot mutation. MDS patients with\n  SF3B1 mutations have been reported to have better overall and event-free survival\n  than their wildtype counterparts. Additionally, these mutations are highly associated\n  with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with\n  ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring\n  sideroblasts).\nofficial_name: splicing factor 3b, subunit 1, 155kDa\nclinical_description: \n	1	\N	\N	92df120d-4761-4e4a-9f03-292808a8c9d0	2015-02-24 20:42:47.564091
45	45	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7157\nname: TP53\ndescription: TP53 mutations are universal across cancer types. Loss of tumor suppressors\n  is most recognized by large deleterious events, such as frameshift mutations, or\n  premature stop codons. In TP53 however, many of the observed mutations in cancer\n  are found to be single nucleotide variants, or missense mutations. These variants\n  are also very broadly distributed throughout the gene, not localizing in any particular\n  hotspot. While a large proportion of cancer genomics research is focused on somatic\n  variants, TP53 is also of note in the germline. Germline TP53 mutations are the\n  hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been\n  found to have prognostic impact on patient outcomes.\nofficial_name: tumor protein p53\nclinical_description: \n	1	\N	\N	2b1ccf37-111b-45e4-840a-eee16ec922a5	2015-02-24 20:42:47.809835
46	46	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7248\nname: TSC1\ndescription: N/A\nofficial_name: tuberous sclerosis 1\nclinical_description: \n	1	\N	\N	27625b73-699d-4043-93e5-cc26aea18161	2015-02-24 20:42:48.005119
47	47	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7249\nname: TSC2\ndescription: N/A\nofficial_name: tuberous sclerosis 2\nclinical_description: \n	1	\N	\N	584d1797-c150-4e94-abc1-ad67a400daf8	2015-02-24 20:42:48.206252
48	48	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7307\nname: U2AF1\ndescription: U2AF1 is one of several spliceosome complex genes frequently mutated\n  in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes\n  (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot\n  mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein.\n  These mutations have been associated with altered splicing patterns in vitro and\n  in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation\n  to secondary acute myeloid leukemia, however, the impact of these mutations on overall\n  survival has been an area of debate.\nofficial_name: U2 small nuclear RNA auxiliary factor 1\nclinical_description: \n	1	\N	\N	5d6e297a-6804-4b97-92f7-141f71fd69b4	2015-02-24 20:42:48.436572
49	49	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7490\nname: WT1\ndescription: WT1 is a tumor suppressor gene associated with the development of Wilms'\n  Tumor, from which it was named. Mutations in exon 7 and 9 of WT1 have been recurrently\n  identified in acute myeloid leukemia and associated with poorer prognosis and chemotherapy\n  resistance.\nofficial_name: Wilms tumor 1\nclinical_description: \n	1	\N	\N	bb78a2ef-86c2-4960-b352-22d5a5b139e0	2015-02-24 20:42:48.644964
50	1	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL\ndescription: The BCR-ABL fusion protein, commonly referred to as the philadelphia\n  chromosome, is the most well-studied fusion gene in cancer. It has widely been considered\n  the initiating event in chronic myelogenous leukemia (CML), but despite its ability\n  initiate disease in mice, its status an initiating mutation is in dispute. In what\n  is commonly used as the poster-child for targeted therapeutics, the development\n  and use of imatinib in the clinic has led to profound improvements in the prognosis\n  of the disease. However, imatinib resistance is still seen in patients with mutations\n  in the ABL kinase domain of the fusion, most notably the T315I variant. In patients\n  resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's\n  (dasatinib and nilotinib) have seen some success in delivering a tumor response.\n	1	\N	\N	040e65e3-7971-480b-9c2e-516c4a967264	2015-02-24 20:42:48.701666
51	2	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL T315I\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation T315I has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	74120e66-44ea-4cc2-92c6-87222d1863e4	2015-02-24 20:42:48.73369
52	3	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL E255K\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation E255K has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	54c1afd0-f996-401d-9fcc-98d7977e05f4	2015-02-24 20:42:48.76399
53	4	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 2\nname: E17K\ndescription: AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal,\n  lung, and ovarian cancer. It has been convincingly shown to be an activating mutation\n  resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation\n  decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and\n  other AKT1 mutations, are the subject of much research and development for therapeutics.\n	1	\N	\N	6968c735-5ae7-4562-bca5-8c5dc511d924	2015-02-24 20:42:48.788811
54	5	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK\ndescription: The EML4-ALK fusion has been seen in non-small cell lung cancer, and\n  appears to be an alternative mechanism for ALK activation. Cells with this fusion\n  have been shown to be sensitive to the ALK inhibitor crizotinib.\n	1	\N	\N	640c0baf-1693-4fad-9fe4-b370045a8c7e	2015-02-24 20:42:48.812689
55	6	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK C1156Y\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	e30699cc-999c-42b0-a2fd-d69094205f84	2015-02-24 20:42:48.836468
56	7	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK L1196M\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	6f116511-d49b-4756-a7cf-bcfb42d3c9e3	2015-02-24 20:42:48.859114
57	8	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: F1174L\ndescription: ALK F1174L has been observed as recurrent in neuroblastoma, non-small\n  cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing\n  this mutation have shown resistance to low doses of criztonib. However, increased\n  dosage can overcome this resistance in cell lines studies. TAE684 has also proven\n  effective in both NSCLC and neuroblastoma F1174L containing cells.\n	1	\N	\N	33a049b6-c0a9-49b8-b7d8-5de9dd710ba9	2015-02-24 20:42:48.889636
58	9	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: R1275Q\ndescription: ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma,\n  non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma\n  cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This,\n  and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC\n  cell lines.\n	1	\N	\N	23bd4472-853c-4942-ad14-a165fcd3ce4e	2015-02-24 20:42:48.91514
59	10	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 3\nname: S214C\ndescription: ARAF S214C has been found to be a recurrent oncogenic mutation in non-small\n  cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib\n  in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this\n  mutation, sorafenib also acheived near-complete clinical remission. This case has\n  brought more interest to the variant from a research and clinical perspective.\n	1	\N	\N	b2cdb8a2-4839-480d-a336-aa4a8c7b14e7	2015-02-24 20:42:48.942163
60	11	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600D\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	40719537-9b9c-4d6c-955a-ceaa1829bed1	2015-02-24 20:42:48.969404
74	25	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: EXPRESSION\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	f2e21f14-cb82-4092-b1bc-1773fdad842e	2015-02-24 20:42:49.377754
139	90	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 4 MUTATIONS\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	cfc0d2ac-0837-4d3f-b729-d2c65d6d1553	2015-02-24 20:42:51.351508
61	12	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E\ndescription: BRAF V600E has been shown to be recurrent in many cancer types. It is\n  one of the most widely studied variants in cancer. This variant is correlated with\n  poor prognosis in certain cancer types, including colorectal cancer and papillary\n  thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective\n  in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib\n  has also shown to be effective when combined with the MEK inhibitor trametinib in\n  colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA\n  mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib,\n  and a number of combination therapies have been successful in treating V600E mutations.\n  While the drugs cetuximab and panitumumab have been largely shown to be ineffective\n  without supplementary treatment.\n	1	\N	\N	5da04d70-5494-4906-9482-092e57b11c8b	2015-02-24 20:42:48.994846
62	13	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E/V600M\ndescription: A case study of a single patient harboring both a V600E and a V600M mutation,\n  dabrafenib was shown to acheive clinical response.\n	1	\N	\N	5b52d8ab-41d6-441c-afd2-51180513af99	2015-02-24 20:42:49.030281
63	14	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E AMPLIFICATION\ndescription: Amplification of BRAF V600E has been shown to confer resistance to MEK\n  inhibitors. For more information on the V600 locus, see the V600E entry.\n	1	\N	\N	b0ccbbf0-94c7-44a3-a01c-713e24f7fe14	2015-02-24 20:42:49.055864
64	15	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600M\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	5d3cf19d-1a82-4297-863a-ec471ece2191	2015-02-24 20:42:49.086475
65	16	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600R\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	74fb4e3a-c300-4b53-8526-dfb9e215c8aa	2015-02-24 20:42:49.111088
66	17	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600\ndescription: BRAF mutations of the valine 600 residue have been shown to be recurrent\n  in many cancer types. Of the V600 mutations, V600E is the most widely researched.\n  V600 mutations as a whole have been correlated to poorer prognosis in colorectal\n  and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity\n  to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual\n  mutation pages on the left sidebar.\n	1	\N	\N	40a95563-e2a5-4192-bf84-bc011b332678	2015-02-24 20:42:49.140257
67	18	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: AMPLIFICATION\ndescription: CCND1 amplification has been implicated in poorer prognosis in non-small\n  cell lung cancer.\n	1	\N	\N	00b14940-1fa5-46d5-bac6-6912a2bd2a74	2015-02-24 20:42:49.16912
68	19	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: EXPRESSION\ndescription: CCND1 expression, and its prognositc impact, is still in dispute. Three\n  experiments in non-small cell lung cancer have shown it to have no impact on survival,\n  but three additional studies have shown it results in poorer prognosis. There is\n  also some ambiguity in how the boundaries between expression and overexpression\n  are defined.\n	1	\N	\N	638bfc52-e954-4197-a388-f658318244dc	2015-02-24 20:42:49.193031
69	20	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	67e69932-9651-48a4-b782-0bef4bb6f4c5	2015-02-24 20:42:49.218389
70	21	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	bfaaaea4-4fb5-47a1-8a0d-d65cf67f34b3	2015-02-24 20:42:49.244402
71	22	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: PROMOTER DEMETHYLATION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	fc721d17-fbda-4b95-8ecf-f9273eaf1898	2015-02-24 20:42:49.275264
72	23	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 10\nname: LOSS\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL),\n  a seemingly unique trend when compared to the amplifcations and overexpressions\n  of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic\n  palbociclib significantly increased the median survival of a Notch-driven model\n  of T-ALL.\n	1	\N	\N	862592eb-1518-4490-bd21-bf883eadaa66	2015-02-24 20:42:49.321054
73	24	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 11\nname: OVEREXPRESSION\ndescription: Cyclin E, while currenly not as widely implicated as its counterpart,\n  cyclin D, has been implicated in various carcinomas, including breast, gastric,\n  stomach and colorectal. High levels of cyclin E, either by gene amplification or\n  overexpression, are correlated with later stage disease and have been shown to lead\n  to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer\n  specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex\n  are more radiosensitive than their lowly expressed counterparts.\n	1	\N	\N	f36784bc-cede-4faa-89cd-09c70792b0ed	2015-02-24 20:42:49.350945
75	26	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 12\nname: EXPRESSION\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	b1fd2aae-d9f5-4226-9794-73b2b0c29dc7	2015-02-24 20:42:49.410266
76	27	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 14\nname: PROMOTER HYPERMETHYLATION\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the CDKN2A\n  promoter region. While no targeted therapeutic has been engaged in clinical trials,\n  the prognostic impact has been studied by a number of meta-analyses. The prognostic\n  impact of promoter hypermethylation has been relatively ambiguous. Many studies\n  have shown significant p-values suggesting poorer prognostic outcomes for patients\n  with hypermethylation in colorectal, liver, and younger lung cancer patients. This\n  being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\n	1	\N	\N	63cb74e8-bc94-4b96-830b-6f717f44590e	2015-02-24 20:42:49.434261
77	28	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: N-TERMINAL FRAME SHIFT\ndescription: CEBPA N-terminal frame shift mutations that result in a premature stop\n  codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML).\n  CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3\n  mutations should also be assessed in CN-AML patients as concurrent mutations may\n  have prognostic implications.\n	1	\N	\N	aef827a0-2c87-4daa-a872-935fd177347c	2015-02-24 20:42:49.46444
78	29	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: MUTATION\ndescription: CEBPA mutations are associated with cytogenetically normal acute myeloid\n  leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should\n  also be assessed in CN-AML patients as concurrent mutations may have prognostic\n  implications.\n	1	\N	\N	3d16cc0f-416f-4b85-a5de-3a2e0e5120f6	2015-02-24 20:42:49.496256
79	30	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	a3c43bdc-357c-46d1-87b6-3432d9c4d2ad	2015-02-24 20:42:49.521314
80	31	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in the rare form of adolescent\n  liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion\n  was observed in 15/15 FL-HCC cases examined and functional studies found that the\n  fusion retained kinase activity. The presence of this fusion may be used as a diagnostic\n  marker for this rare tumor type.\n	1	\N	\N	c2b3ec08-14a4-4493-b17c-e9f346562fae	2015-02-24 20:42:49.548293
81	32	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R882\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\n	1	\N	\N	11d9c7d1-00a2-4041-9b82-80117a4ad7f5	2015-02-24 20:42:49.576929
82	33	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: L858R\ndescription: EGFR L858R has long been recognized as a functionally significant mutation\n  in cancer, and is one of the most prevalent single mutations in lung cancer. Best\n  described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity\n  to first and second generation TKI's like gefitinib and neratinib. NSCLC patients\n  with this mutation treated with TKI's show increased overall and progression-free\n  survival, as compared to chemotherapy alone. Third generation TKI's are currently\n  in clinical trials that specifically focus on mutant forms of EGFR, a few of which\n  have shown efficacy in treating patients that failed to respond to earlier generation\n  TKI therapies.\n	1	\N	\N	44ab4dd0-2d5f-48d2-9349-c813d25a77ec	2015-02-24 20:42:49.605819
83	34	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: T790M\ndescription: EGFR T790M was one of the very first mutations recognized to confer resistance\n  to targeted therapies. While successful in amplified EGFR, the efficacy of the first\n  and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients\n  harboring this mutation before treatment is notably lower. This lack of efficacy\n  can likely be to blame for the poorer prognosis for patients with this mutation\n  as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately\n  half of EGFR mutant tumors with acquired resistance to TKI inhibition have been\n  shown to harbor this mutation, implicating it as a mechanism of acquired therapy\n  resistence. The third generation TKI's are being developed with this resistance\n  problem in mind, and early stage experiments have shown some efficacy of these drugs\n  in previously resistant tumors with mutant EGFR.\n	1	\N	\N	28df9fd1-4372-4c8f-8a9d-1f7573d76f5f	2015-02-24 20:42:49.631052
84	35	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: D769H\ndescription: ERBB2 D769H was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	ef28517f-5ac2-4202-a6d3-47be62edfe4c	2015-02-24 20:42:49.664919
85	36	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: D769Y\ndescription: ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	314f18d2-1259-4110-bc81-afccd4cbe019	2015-02-24 20:42:49.690866
123	74	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: EXON 2 MUTATIONS\ndescription: In a study by Pao et al in 2005, non-small cell lung cancer patients\n  harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors\n  gefinitib and erlotinib.\n	1	\N	\N	b9fb317e-83f8-4c56-a593-8847167bf4a6	2015-02-24 20:42:50.853494
140	91	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: G12\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	8eb3ff98-53af-4f68-8171-e3bb46309bd2	2015-02-24 20:42:51.376516
86	37	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: DEL 755-759\ndescription: ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants\n  to be functionally classified (Bose et al. 2012). This mutation was shown to be\n  an activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	6c955d6d-a697-4392-a019-90afa6251831	2015-02-24 20:42:49.716349
87	38	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: G309A\ndescription: ERBB2 G309A was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	a3a3b858-ea7f-43ca-adb7-316d39133abc	2015-02-24 20:42:49.74585
88	39	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: L755S\ndescription: ERBB2 L755S was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike\n  many of the other variants queried. This mutation was also shown to confer resistance\n  to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.\n	1	\N	\N	5c4f176b-0ddb-45e1-8f74-362ec895233c	2015-02-24 20:42:49.773787
89	40	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: L755W\ndescription: ERBB2 L755W was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	acdac105-d374-4953-b890-90079b7a3b51	2015-02-24 20:42:49.806433
90	41	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: P780INS\ndescription: ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally\n  classified (Bose et al. 2012). This mutation was shown to be an activating mutation\n  in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	e0a52f09-f1d9-45f8-bf48-fa307270a5ef	2015-02-24 20:42:49.834047
91	42	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: R678Q\ndescription: ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	c2dbf5fc-27fe-49e4-8aea-20a7af2ba362	2015-02-24 20:42:49.865082
92	43	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: R896C\ndescription: ERBB2 R896C was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	b3962378-6a25-4e06-afc8-4ff0edec5fe2	2015-02-24 20:42:49.890551
93	44	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: V777L\ndescription: ERBB2 V777L was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	f353dc98-4a36-4897-8e81-3b723ce01aad	2015-02-24 20:42:49.914735
94	45	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: V842I\ndescription: ERBB2 V842I was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	7e512a0f-af4f-4ea3-a725-71f1f87a4930	2015-02-24 20:42:49.940484
95	46	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: L536Q\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of\n  these ligand binding domain mutations, and is commonly implicated in this hormone\n  resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant,\n  may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	02c7c21c-96ed-4520-83a4-c43111564100	2015-02-24 20:42:49.972023
96	47	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: N538G\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	bddef5b6-e5af-4c6d-b476-6794f7bfd8e9	2015-02-24 20:42:50.005895
138	89	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 3 MUTATIONS\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	9f840efe-4209-4e53-bf2e-d810ddebf7b3	2015-02-24 20:42:51.327594
97	48	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537C\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	67df644d-0ec3-4c80-9663-c0a898774ba2	2015-02-24 20:42:50.04833
98	49	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537N\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	a6829198-1aa8-46aa-a13a-7b2572238b3e	2015-02-24 20:42:50.076449
99	50	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537S\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	73291184-1648-4907-8695-6ba54e7852cf	2015-02-24 20:42:50.107353
100	51	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: FGFR2-MGEA5\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. Additionally, tumor size reduction was\n  achieved by both ponatinib and pazopanib treatments administered separately in a\n  single patient with intrahepatic cholangiocarcinoma and this fusion. The authors\n  use these cases to highlight the need for enhanced clinical sequencing efforts.\n	1	\N	\N	25626132-85af-40c2-8620-f28639a08abe	2015-02-24 20:42:50.139569
101	52	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: FGFR3-BAIAP2L1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\n	1	\N	\N	943d654f-d0a3-4ca9-948f-0bc036188228	2015-02-24 20:42:50.174014
102	53	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: FGFR3-TACC3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\n	1	\N	\N	00ae10b7-0884-4685-b015-67b785c742d3	2015-02-24 20:42:50.204059
103	54	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: ITD MUTATIONS\ndescription: FLT3-ITD (internal tandem duplications) frequently occur in patients\n  with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid\n  leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically\n  normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of\n  FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated\n  sequence resulting in an in-frame duplication event. The length of these duplications\n  can vary widely which may have prognostic consequences, but this has not been conclusively\n  determined. FLT3-ITD mutations overall have generally been associated with poor\n  prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the\n  prognosis associated with this variant.\n	1	\N	\N	179c3737-a05e-4cb5-a408-698571aa6b1e	2015-02-24 20:42:50.22951
104	55	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: TKD MUTATIONS\ndescription: FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common\n  than FLT3-ITD (internal tandem duplication) mutations and may not confer the same\n  prognostic impact. Although the majority of mutations are point mutations effecting\n  D835 (most frequently D835Y), a small proportion involve an in-frame deletion of\n  I836. These mutations are within the activation loop of the second tyrosine kinase\n  domain of FLT3 and thought to result in constitutive activation of the receptor.\n	1	\N	\N	80d774c3-c311-4299-8680-483afc56aa00	2015-02-24 20:42:50.256352
105	56	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: EXPRESSION\ndescription: GATA2 misregulation has been observed in a number of hematologic malignancies,\n  as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using\n  the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung\n  cancer.\n	1	\N	\N	f4ba7697-9ff3-48ca-93d2-70236639dc56	2015-02-24 20:42:50.285275
106	57	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	49c14564-d4d4-4bf3-b58b-54027c98f533	2015-02-24 20:42:50.315691
107	58	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132C\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	b84730e0-7939-4d87-a8f1-bd33785ee481	2015-02-24 20:42:50.341497
108	59	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132H\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	887e4615-c66d-4850-aead-25a6550d56b7	2015-02-24 20:42:50.378259
109	60	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132L\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	779cd7a9-e5e3-4855-a082-fe56fc783cc4	2015-02-24 20:42:50.407552
110	61	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: R140Q/L\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival\n  as compared to their wild-type counterparts. In myelodysplastic syndromes, however,\n  no prognostic link was discovered between IDH2 mutation status and overall survival.\n	1	\N	\N	4718d524-0d9b-4b1a-8429-40e9122e51e6	2015-02-24 20:42:50.435661
111	62	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: R172K\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis\n  and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic\n  syndromes, studies did not find a prognostic association between this variant and\n  patient outcomes.\n	1	\N	\N	ea757020-ad47-43af-ba10-001db90961b4	2015-02-24 20:42:50.465513
112	63	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: V617F\ndescription: JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases,\n  occuring in around half of all MPD's. While less associated with cancer, when it\n  is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The\n  V617F mutation is an activating mutation, resulting in increased kinase activity.\n  The mutation seems to be restricted to hematologic malignancies. Treatment of JAK\n  mutant diseases with ruxolitinib has seen some clinical success.\n	1	\N	\N	1fe102a7-807c-47a6-9d13-b9506fd9601e	2015-02-24 20:42:50.49416
113	64	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: D816V\ndescription: KIT D816V is a mutation observed in acute myeloid leukemia (AML). This\n  variant has been linked to poorer prognosis and worse outcome in AML patients.\n	1	\N	\N	c21e6740-a24e-49c8-94dd-2b79b6aaed98	2015-02-24 20:42:50.522192
114	65	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 11 MUTATIONS\ndescription: c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are\n  very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis\n  than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity\n  to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients\n  treated with the drug in the first year.\n	1	\N	\N	d78a98fe-64fe-4234-bbbe-fe4f7cd700ae	2015-02-24 20:42:50.548639
115	66	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: INTERNAL DUPLICATION\ndescription: c-KIT internal duplications have been observed in exon 11, within the\n  juxtamembrane domain. In a case study of an anal melanoma patient harboring this\n  event, imatinib confered marked response. Also, cells harboring exon 11 mutations\n  have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better\n  prognosis to patients treated with the drug in the first year.\n	1	\N	\N	d7c0419c-0b74-4535-89d6-7fdf1fb7458f	2015-02-24 20:42:50.577204
116	67	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 13 MUTATIONS\ndescription: c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations.\n  These mutations lie within the tyrosine kinase 1 domain, and are found primarily\n  in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations\n  show sensitivity to imatinib and sunitinib treatment. However, only imatinib has\n  seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant\n  cases.\n	1	\N	\N	371c5730-eeab-40ef-9939-bc0dba5d0e0e	2015-02-24 20:42:50.606923
117	68	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 14 MUTATIONS\ndescription: c-KIT exon 14 mutations lie within the tyrosine kinase domain of the\n  protein. While relatively rare in primary gastrointestinal tumors, they are notably\n  more prevalent in refractory disease, suggesting a role in imatinib resistance.\n  Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.\n	1	\N	\N	2a5de9a4-92f8-4cca-807f-ec0a12c9c817	2015-02-24 20:42:50.636441
118	69	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 17 MUTATIONS\ndescription: c-KIT exon 17 mutations lie within the TK2 domain, containing the activation\n  loop of the protein. In cell lines, mutations within this domain have been shown\n  to be sensitive to imatinib. However, in double KIT mutants in which the exon 17\n  mutation is a secondary mutation, cell lines have shown resistance to both imatinib\n  and sunitinib.\n	1	\N	\N	33c189f4-8fe2-4411-ae18-a975b7e9f222	2015-02-24 20:42:50.666537
119	70	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 9 MUTATIONS\ndescription: c-KIT exon 9 mutations lie within the dimerization motif of the protein.\n  Relative to other KIT mutations, exon 9 mutations have been associated with better\n  overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and\n  in vivo.\n	1	\N	\N	ea4b46e0-e0ed-4345-81e4-dd94da409b26	2015-02-24 20:42:50.69879
120	71	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: L576P\ndescription: KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain.\n  It is one of the most recurrent KIT mutations in melanoma, and both in vitro and\n  in vivo studies have shown sensitivity to imatinib.\n	1	\N	\N	85b33844-4d98-4f2f-8daa-e8b3c25f3fc3	2015-02-24 20:42:50.752877
121	72	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: V654A\ndescription: KIT V654A is an exon 13 mutation that lies within the tyrosine kinase\n  1 domain of the protein. It has been shown to be an activating mutation by in vitro\n  studies. This mutation is associated with imatinib resistance in melanoma patients.\n  However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have\n  seen success in acheiving tumor response.\n	1	\N	\N	acf99b81-55e5-47c0-96f8-4c003e7beabf	2015-02-24 20:42:50.786147
122	73	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: EXON 1 MUTATIONS\ndescription: A study by Lièvre et al in 2006 showed that colorectal cancer patients\n  with KRAS exon 1 mutations had low cetuximab response rates.\n	1	\N	\N	5f15f229-bd06-4c75-b909-bba9d19e8e0a	2015-02-24 20:42:50.822497
124	75	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	6e9f6d62-93b4-40f9-830f-5f9c76231efb	2015-02-24 20:42:50.884377
125	76	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12/G13\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation\n  may result in a less responsive tumor when treated with first-generation TKI's like\n  gefitinib.\n	1	\N	\N	13371e1a-f633-4c9c-a999-e1983b59b475	2015-02-24 20:42:50.91588
126	77	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12C\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	1e3b29de-e2bd-4202-8cb0-24aa4f707b82	2015-02-24 20:42:50.94681
127	78	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12D\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	08dc760e-b9c2-4922-b1b7-e4fed03ea18a	2015-02-24 20:42:50.978885
128	79	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G13\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	8148e54d-ae5a-411b-b79c-70df6031c00c	2015-02-24 20:42:51.014831
129	80	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G13D\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	c6f7697b-c434-4062-8308-5f9abd153c58	2015-02-24 20:42:51.046759
130	81	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: P124S\ndescription: MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder\n  and colon cancer to a lesser degree. The P124S mutation has been shown to contribute\n  to AZD6244 resistance in melanoma cell lines, but considerably less so than its\n  Q56P counterpart.\n	1	\N	\N	5124a0b7-7436-4f8c-8852-d15d9fc14bae	2015-02-24 20:42:51.078212
131	82	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: Q56P\ndescription: MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This\n  mutation has been shown to confer considerable resistance to AZD6244 treatment of\n  melanoma cell lines.\n	1	\N	\N	ba6078d8-f121-4fb5-81b1-7b9716e64504	2015-02-24 20:42:51.109596
132	83	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 32\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	9d5770d3-b010-4656-b774-457231891cb9	2015-02-24 20:42:51.143611
133	84	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: PROMOTER METHYLATION\ndescription: MGMT promoter methylation has been observed to impact tumor progression\n  in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating\n  agent carmustine has shown efficacy. In patients lacking methylation, combining\n  carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more\n  experiments are required. Clinical trials have also shown selective sensitivity\n  of promoter methylation-positive patients to temozolomide, making a case for wider\n  methylation screening in GBM patients.\n	1	\N	\N	e18c99aa-3f2b-4a50-87dd-90000dbe8b70	2015-02-24 20:42:51.175771
134	85	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 12 MUTATIONS\ndescription: NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\n	1	\N	\N	000dcc90-6c59-4f6a-9b21-97c65b6f56bb	2015-02-24 20:42:51.211074
135	86	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: W288FS\ndescription: NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most\n  common NPM1 mutation identified in acute myeloid leukemia. This mutation results\n  in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response\n  to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation,\n  it is generally grouped with other exon 12 mutations for patient analysis (see NPM1\n  Exon 12 variants for more information).\n	1	\N	\N	203b2511-9df1-4443-8a6f-bc3280e6f634	2015-02-24 20:42:51.245792
136	87	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 1 MUTATIONS\ndescription: NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown\n  to be correlated with poorer overall survival relative to wild-type NRAS in melanoma\n  patients.\n	1	\N	\N	00986fee-f837-4063-a8c0-94eda110a689	2015-02-24 20:42:51.270878
137	88	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 2 MUTATIONS\ndescription: NRAS exon 2 mutations have been shown to be correlated with poorer overall\n  survival in melanoma patients and colorectal cancer patients, however no prognostic\n  impact was seen in acute myeloid leukemia patients.\n	1	\N	\N	9d948ea9-9f90-49a6-8ff5-60e8be06ad29	2015-02-24 20:42:51.295135
141	92	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: G13D\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	d8d6d888-3c77-4637-8704-1b29aeb123b3	2015-02-24 20:42:51.407008
142	93	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations\n  at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide.\n  However, in colorectal cancer patients, mutations at this locus have been shown\n  to confer resistance to cetuximab. The prognostic impact of mutations at this locus\n  is currently under study.\n	1	\N	\N	3624eab9-d93d-4255-8cad-a00f9905779a	2015-02-24 20:42:51.430851
143	94	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61L\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	66a219ee-15c2-4f82-a55f-02729512e33a	2015-02-24 20:42:51.456954
144	95	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61R\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	e8d5fed7-f8a8-487f-9599-106fa927c043	2015-02-24 20:42:51.506313
145	96	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842I\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	dbbbc324-f39b-4076-9c21-644286ed60fe	2015-02-24 20:42:51.532928
146	97	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842V\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	10a18b96-9014-4f06-9806-eaf47da5af8a	2015-02-24 20:42:51.564352
147	98	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842Y\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	c6157ca6-079f-4cbf-9e9d-bdc57f93f6dc	2015-02-24 20:42:51.592404
148	99	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: DEL I843\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	6cff4122-700a-49a4-bd24-f621779459ee	2015-02-24 20:42:51.622021
149	100	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: DI842-843IM\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	3a98fec8-2ffb-4d4a-9c2f-21c60e8bf8a1	2015-02-24 20:42:51.649521
150	101	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: E542K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	58284795-364c-49b4-9c42-0db6b0fc75af	2015-02-24 20:42:51.678686
151	102	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: E545K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	43541f01-cb8a-496e-b941-25f48eb41d58	2015-02-24 20:42:51.707922
152	103	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: EXON 20 MUTATIONS\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	3a6ed199-63c9-42ac-ab6e-370950d81df7	2015-02-24 20:42:51.738435
153	104	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: EXON 9 MUTATIONS\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	c8608154-7c22-43d0-8cc4-c1de2412976a	2015-02-24 20:42:51.776449
154	105	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: H1047R\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	f6fb850f-20a9-4710-8e01-4de634ffb194	2015-02-24 20:42:51.816498
155	106	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 39\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and arsenic trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	6048d943-6ecc-4273-a9e0-9588d494a2ec	2015-02-24 20:42:51.860941
156	107	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in a rare form of adolescent\n  liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found\n  Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined,\n  and functional studies found that the fusion retained kinase activity. The presence\n  of this fusion may be used as a diagnostic marker for this rare tumor type.\n	1	\N	\N	fb0adc41-2437-4d52-bcc7-d10f49b5d722	2015-02-24 20:42:51.91511
157	108	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 41\nname: R233*\ndescription: PTEN R233* has been shown to be a loss of function mutation, and PTEN\n  loss has been the subject of considerable research in breast cancer. PTEN loss may\n  sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data\n  to support that PTEN loss is both associated with poorer prognosis, and no change\n  in prognosis.\n	1	\N	\N	9e8a026b-93d2-45f9-a890-905e0e1bb9fa	2015-02-24 20:42:51.970181
158	109	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and Arsenic Trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	a9d5b82b-236f-4bd7-9416-71e1df898d9a	2015-02-24 20:42:52.017334
159	110	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: C634W\ndescription: RET C639W has been implicated as an alternate mechanism of activating\n  RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting\n  agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the C639W mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib.\n	1	\N	\N	363cb837-f52a-4fee-91f4-aaa14db433de	2015-02-24 20:42:52.058101
160	111	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: M918T\ndescription: RET M819T is the most common somatically acquired mutation in medullary\n  thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents,\n  promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the M918T mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T\n  leads to more aggressive MTC with a poorer prognosis.\n	1	\N	\N	48434b8b-42f9-4707-b3ab-fb9997d9487c	2015-02-24 20:42:52.099049
161	112	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: K666N\ndescription: SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients.\n	1	\N	\N	da633d77-0b1d-45dc-83ea-cfc81dab1159	2015-02-24 20:42:52.133565
162	113	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: K700E\ndescription: SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients. Additionally,\n  these mutations are the most common SF3B1 mutation observed in MDS and highly associated\n  with subtypes of MDS that are defined by ringed sideroblasts.\n	1	\N	\N	5b33f031-3911-453d-be4c-f092a3d12925	2015-02-24 20:42:52.161609
163	114	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R175H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R175H variant seems not only to result in loss of tumor-suppression, but also acts\n  as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell\n  lines harboring this mutant is also more responsive to treatment with doxorubicin\n  than its wild-type counterparts. While the prognostic impact of individual TP53\n  mutations is influenced by the cohort being studied, it has been shown that the\n  R175H mutation is correlated with worse overall survival than wild-type TP53, but\n  is not as detrimental as the R248W variant.\n	1	\N	\N	4e6ee537-89d4-4447-91e0-30efd381cdb6	2015-02-24 20:42:52.188539
164	115	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R248Q\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival. The R248Q mutation has also shown an increased invasive behavior in cell\n  lines. This is specific to the 248Q variant.\n	1	\N	\N	a94decc4-01c0-46db-a0cb-0a302cc018f2	2015-02-24 20:42:52.221161
165	116	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R248W\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival.\n	1	\N	\N	ee15c49f-0003-4d38-9803-7939c5c70733	2015-02-24 20:42:52.250468
166	117	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R249T\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	766ae908-83d8-43dc-a334-083b79e63c3c	2015-02-24 20:42:52.282605
167	118	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R249W\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	39c814fd-b122-42b1-9618-53a72d760ef6	2015-02-24 20:42:52.327638
168	119	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R273C\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	cb09a5e6-5245-44e0-b151-e71735127369	2015-02-24 20:42:52.353519
169	120	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R273H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	b4d8f20e-bff7-47cc-a6e4-1aeb2a3ebc07	2015-02-24 20:42:52.378836
170	121	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: V173G/A\ndescription: While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R249 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	26ac7293-2a67-4e16-a399-18eefb31ee07	2015-02-24 20:42:52.408373
171	122	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: FRAMESHIFT TRUNCATION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	d27ef322-33d9-465f-9893-a67053016a92	2015-02-24 20:42:52.436472
172	123	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	b50fcda7-4473-4daf-9628-6002536656f7	2015-02-24 20:42:52.464819
173	124	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	64f4324f-f20d-420f-8106-8f16f0cee636	2015-02-24 20:42:52.494357
174	125	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 48\nname: Q157P/R\ndescription: U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is less common than the S34F mutation, occurs in the second zinc finger domain of\n  U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations\n  on overall survival in MDS has been debated, however, patients with U2AF1 mutations\n  were shown to be at an increased risk of transformation to secondary AML. The presence\n  of this mutation was not associated with a specific prognostic outcome in AML when\n  compared to U2AF1 wildtype patients.\n	1	\N	\N	cdf8c669-5ea2-4150-8ae0-09dc942858de	2015-02-24 20:42:52.522807
175	126	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 48\nname: S34Y/F\ndescription: U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is the most commonly identified variant in MDS, occurs in the first zinc finger\n  domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1\n  mutations on overall survival in MDS has been debated, however, patients with U2AF1\n  mutations were shown to be at an increased risk of transformation to secondary AML.\n  The presence of this mutation was not associated with a specific prognostic outcome\n  in AML when compared to U2AF1 wildtype patients.\n	1	\N	\N	0e95f6df-67ba-4d67-90ef-1273d69db05e	2015-02-24 20:42:52.549904
176	127	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 49\nname: EXON 7 MUTATIONS\ndescription: WT1 exon 7 mutations have been shown to be recurrent in acute myeloid\n  leukemia. Many sources have examined the prognostic impact of these, agreeing that\n  the mutant exon 7 cohort is correlated with worse overall survival and a number\n  of poor prognistic outcomes. This may be the result of an overall poor response\n  to chemotherapy from WT1 mutant tumors.\n	1	\N	\N	62ca48a5-20b4-4883-8e06-7e81756684ec	2015-02-24 20:42:52.575775
177	128	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 49\nname: EXON 9 MUTATIONS\ndescription: WT1 exon 9 mutations have been shown to be recurrent in acute myeloid\n  leukemia, although at a less frequent rate than their exon 7 counterparts. Many\n  sources have examined the prognostic impact of these, agreeing that the mutant exon\n  9 cohort is correlated with worse overall survival and a number of poor prognistic\n  outcomes. This may be the result of an overall poor response to chemotherapy from\n  WT1 mutant tumors.\n	1	\N	\N	77104f09-88e5-4a18-a302-b096abb9eb32	2015-02-24 20:42:52.599501
178	129	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: LOSS-OF-FUNCTION\ndescription: BRCA1 loss of function mutations have been shown to increase risk of\n  breast and ovarian cancer in those carrying the allele in their germline. Treating\n  BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant\n  response.\n	1	\N	\N	683d37dc-bd5b-401a-9061-200755788133	2015-02-24 20:42:52.630202
179	130	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: LOSS-OF-FUNCTION\ndescription: BRCA2 loss of function mutations have been shown to increase risk of\n  breast and ovarian cancer in those carrying the allele in their germline. Treating\n  BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant\n  response.\n	1	\N	\N	19e2624b-a472-4873-b2e0-c5f32bc5579a	2015-02-24 20:42:52.655398
180	1	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL inhibitors such as imatinib have lead to "significantly improved prognosis,\n  response rate, overall survival, and patient outcome in CML patients compared to\n  previous therapeutic regimens."\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 1\nsource_id: 48\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8cd6e4b6-02ba-4615-84d8-522228dfc391	2015-02-24 20:42:52.810842
211	32	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with shorter disease-free and overall Survival in a\n  Spanish cohort of melanoma patients.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 7\nsource_id: 63\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a05af245-ed06-47bc-b9a3-3840a41d7bb2	2015-02-24 20:42:55.295945
181	2	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient\n  has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may\n  imply the possibility that BCR-ABL is not universally the initiating event in CML.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 1\nsource_id: 49\nvariant_id: 1\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b276d6fc-5092-463b-b42d-82968a8da43e	2015-02-24 20:42:52.911332
182	3	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of BCR-ABL fusion is considered the characterizing feature of chronic\n  myeloid leukemia, and has been widely thought of as the initiating event in the\n  disease.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 1\nsource_id: 48\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	371bea7a-8e4a-49df-b684-79156bff4474	2015-02-24 20:42:53.001363
183	4	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL fusions have been found to contribute to imatinib resistance in AML\n  cell lines.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 2\nsource_id: 50\nvariant_id: 1\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	cf2fd3f2-6c69-4f55-80d5-c696fb0c06f9	2015-02-24 20:42:53.085693
184	5	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 1\nsource_id: 51\nvariant_id: 3\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3fbb06da-a72d-4d1d-979a-cc469540179a	2015-02-24 20:42:53.151775
185	6	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have\n  shown to be resistant to imatinib treatment.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 1\nsource_id: 48\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	460596d8-ff05-40f6-893b-8b39f6ec72c0	2015-02-24 20:42:53.253159
186	7	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 1\nsource_id: 51\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e4320966-1022-482c-882a-1325460de56f	2015-02-24 20:42:53.340718
187	8	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide\n  results in high rates of complete remission in patients.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 2\nsource_id: 50\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bf591fc8-388f-4ff8-a9c6-718867e2fd20	2015-02-24 20:42:53.424604
188	9	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The clinical use of imatinib in patients with BCR-ABL fusion has resulted in\n  drastic sensitivity to the drug.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 1\nsource_id: 48\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	71843927-a970-455c-b566-feaab84433e6	2015-02-24 20:42:53.484998
189	10	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has been effective in\n  patients resistant to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 1\nsource_id: 48\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d6e0d219-a42a-4583-bd1f-d3979c05e076	2015-02-24 20:42:53.554523
190	11	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has been effective in\n  patients resistant to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 5\ndisease_id: 1\nsource_id: 48\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7774f82d-131d-45ee-994e-e19c7c3e5a15	2015-02-24 20:42:53.646712
191	12	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the\n  AKT inhibitor MK-2206 in breast cancer cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 6\ndisease_id: 3\nsource_id: 52\nvariant_id: 4\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	32aa9d2b-3858-4f01-ad47-b60a3228b102	2015-02-24 20:42:53.718086
192	13	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 4\nsource_id: 53\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b9bfa30a-4a2c-49a8-93a1-a1e365be3a8a	2015-02-24 20:42:53.786362
193	14	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 8\ndisease_id: 5\nsource_id: 54\nvariant_id: 6\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ae2b808b-6820-4d56-8cd9-ab1d10540f71	2015-02-24 20:42:53.858956
194	15	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without\n  this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 8\ndisease_id: 6\nsource_id: 55\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5aeca675-2759-44b0-9883-83209454b136	2015-02-24 20:42:53.945326
195	16	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 8\ndisease_id: 4\nsource_id: 56\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	dae88bb5-adab-40b4-8171-2a059686aa31	2015-02-24 20:42:54.015794
196	17	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without\n  this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 8\ndisease_id: 5\nsource_id: 55\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	10870cbe-4150-4ad3-b424-27b8dd7a04e9	2015-02-24 20:42:54.08715
197	18	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 8\ndisease_id: 5\nsource_id: 54\nvariant_id: 7\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0f0ee2a5-d836-4ef8-a58a-3550140243ea	2015-02-24 20:42:54.169771
198	19	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment\n  with crizotinib has shown to be effective in acheiving stable disease.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 8\ndisease_id: 5\nsource_id: 54\nvariant_id: 5\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3a1a9ba2-edea-4b68-b501-2cd3866b0f62	2015-02-24 20:42:54.274362
199	20	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in\n  a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 9\ndisease_id: 4\nsource_id: 57\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8e305591-72b1-4ad0-8813-0c311ca4bfad	2015-02-24 20:42:54.350441
200	21	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing\n  the EML4-ALK fusion containing the F1174L mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 8\ndisease_id: 4\nsource_id: 58\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b1b5f171-ff65-4cf5-9b83-5b03874a667a	2015-02-24 20:42:54.418516
201	22	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing\n  ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 4\nsource_id: 53\nvariant_id: 8\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6b14e523-5189-4352-afcd-7bb10c62eca4	2015-02-24 20:42:54.491684
202	23	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 treatment resulted in significant tumor regression in xenograft models\n  produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 9\ndisease_id: 5\nsource_id: 57\nvariant_id: 7\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	632f82d7-ffac-4e65-a745-7dfcded6a9a0	2015-02-24 20:42:54.577595
203	24	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells\n  with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 4\nsource_id: 53\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2f14058d-f68f-4367-91ce-f8bc7a39248a	2015-02-24 20:42:54.658524
204	25	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In one patient with S214C mutation, the use of sorafenib has led to more than\n  5 years of survival and near remission.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 5\nsource_id: 59\nvariant_id: 10\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b044933a-0921-41f0-aece-93c996219156	2015-02-24 20:42:54.73913
205	26	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony\n  formation of AALE cells expressing ARAF S214C in vitro\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 11\ndisease_id: 5\nsource_id: 59\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b9873f26-db28-471f-b312-1caf6841fa55	2015-02-24 20:42:54.839989
206	27	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony\n  formation of AALE cells expressing ARAF S214C in vitro\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 5\nsource_id: 59\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6e006135-1a90-418e-9142-d3906d67b47a	2015-02-24 20:42:54.913607
207	28	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 12\ndisease_id: 7\nsource_id: 60\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5607430b-a3c1-462e-8adc-adc48bc0d1bb	2015-02-24 20:42:54.999959
208	29	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 13\ndisease_id: 7\nsource_id: 60\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0da5fc0e-f3d5-4ad2-a60c-0750b39208ce	2015-02-24 20:42:55.064864
209	30	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Unlike other studies that suggest a poorer outcome, BRAF mutation in this study\n  was not correlated with poorer prognosis in papillary thyroid cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 8\nsource_id: 61\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	94ca70c1-7f3a-4240-9943-20202d7859ba	2015-02-24 20:42:55.146411
210	31	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is associated with adverse pathological features of colorectal cancer.\n  This can be concluded as a marker of poor prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 9\nsource_id: 62\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4b0e0be4-38ba-45d8-98aa-6a4514b10370	2015-02-24 20:42:55.226371
212	33	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is correlated with disease recurrence in both age cohorts (>65 and <65\n  yo).\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 8\nsource_id: 64\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ee05ed86-a690-452d-a982-027f838ba89b	2015-02-24 20:42:55.372047
213	34	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in\n  a study of 187 patients with PTC and other thyroid diseases.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 8\nsource_id: 65\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7cef5588-fedd-4958-a45e-fa367b6d9a92	2015-02-24 20:42:55.465438
214	35	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutation correlated with poor prognosis in papillary thyroid cancer in\n  both older (>65 yo) and younger (<65 yo) cohorts.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 8\nsource_id: 64\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7e34f37f-2085-4399-9f21-3eea5b7613b3	2015-02-24 20:42:55.541475
215	36	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutations are associated with melanoma arising in non-chronic sun damaged\n  skin and with superficial spreading melanoma\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 7\nsource_id: 66\nvariant_id: 17\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	02bb564e-7f11-4af3-8c2b-f82fb6d82da3	2015-02-24 20:42:55.625932
216	37	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 8\nsource_id: 67\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fb65b873-34cd-4d1e-b483-ae5e2d77347e	2015-02-24 20:42:55.714408
217	38	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 8\nsource_id: 68\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3a08b3c2-a965-45ae-bec2-ca8463bb2ce8	2015-02-24 20:42:55.784335
218	39	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is shown to be associated with the tall-cell variant of papillary\n  thyroid cancer\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 8\nsource_id: 64\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f7a17e02-3f54-4e88-b143-9f07f6ce8f46	2015-02-24 20:42:55.852362
219	40	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  but does not show resistance to MEK inhibitors.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 7\nsource_id: 69\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ca32638e-4adb-43c4-ac0e-5b88aa3ed6c3	2015-02-24 20:42:55.925551
220	41	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations\n  were associated with poor progression free survival regardless of treatment (panitumumab\n  with best supportive care or best supportive care alone)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 9\nsource_id: 70\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	cd9cfe38-0d74-440e-94f5-0455d470c493	2015-02-24 20:42:56.000134
221	42	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 15\ndisease_id: 10\nsource_id: 71\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	09338cca-7623-4377-92bf-99a6b31c5f8a	2015-02-24 20:42:56.114989
222	43	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 10\nsource_id: 71\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f4a69265-322a-4c74-8b1e-ed2d86491e7c	2015-02-24 20:42:56.182051
223	44	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 10\nsource_id: 71\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c6f2e138-ffd5-424c-bb7d-5b0c6318f5ee	2015-02-24 20:42:56.236728
224	45	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations\n  had lower response and disease control rates as well as shorter progression free\n  and overall survival following cetuximab plus chemotherapy than those with wildtype\n  BRAF\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 15\ndisease_id: 9\nsource_id: 72\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7696d3a1-2b99-4a44-9dfd-dd318c30ec43	2015-02-24 20:42:56.311719
225	46	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  and resistance to RAF inhibitors.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 7\nsource_id: 69\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d08ce3f3-e552-4807-b3af-9062c37f0e98	2015-02-24 20:42:56.378347
226	47	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 5\nsource_id: 73\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	cefc734a-56eb-460c-9494-5d0e4186bd0f	2015-02-24 20:42:56.468961
227	48	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COLO201 and COLO206F cell harboring BRAF V600E mutations were cloned to be MEK\n  inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification\n  of the BRAF V600E gene.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 17\ndisease_id: 9\nsource_id: 74\nvariant_id: 14\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	20b8fab2-c5fd-4228-b910-6f77f616e175	2015-02-24 20:42:56.548106
228	49	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 7\nsource_id: 75\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c7796689-371e-4b4b-8bd9-9e15c474be02	2015-02-24 20:42:56.62699
229	50	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 11\ndisease_id: 7\nsource_id: 75\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	73673696-aed8-4786-8fa9-1c7833de655e	2015-02-24 20:42:56.710435
230	51	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 7\nsource_id: 76\nvariant_id: 11\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	121a20df-0a0e-478a-bde3-2fa958296afc	2015-02-24 20:42:56.784596
231	52	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 18\ndisease_id: 10\nsource_id: 77\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	03208935-1667-40ba-8d6b-d152e3408a78	2015-02-24 20:42:56.868093
232	53	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 19\ndisease_id: 10\nsource_id: 77\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1a198213-5d45-416d-af69-865314168f6d	2015-02-24 20:42:56.927979
233	54	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 19\ndisease_id: 10\nsource_id: 78\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	49fdd81d-9c29-447a-9df0-35b909cb2dbd	2015-02-24 20:42:56.994093
234	55	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 20\ndisease_id: 10\nsource_id: 78\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	94cea70d-c71a-41c7-93db-0e616136ade3	2015-02-24 20:42:57.067097
235	56	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 21\ndisease_id: 9\nsource_id: 79\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f599d902-0622-4bfd-9e18-203e62d841b4	2015-02-24 20:42:57.180778
236	57	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 22\ndisease_id: 9\nsource_id: 79\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	52a1a781-eaa2-44d7-b9cd-9daa80d08d75	2015-02-24 20:42:57.27102
237	58	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 23\ndisease_id: 9\nsource_id: 79\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1def44df-0fbb-4cd3-a7d8-ecd61053cfcd	2015-02-24 20:42:57.339271
238	59	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines expressing the BRAF V600E mutation responded better to vemurafenib\n  treatment than cells wildtype for BRAF as measured by reduced cellular proliferation\n  and inhibition of MET and ERK phosphorylation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 22\ndisease_id: 9\nsource_id: 79\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	09e79496-f4b0-44fd-8de4-1974c4b5a739	2015-02-24 20:42:57.40916
239	60	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 7\nsource_id: 80\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	402a8e14-ac97-45c4-b04a-8650c6133666	2015-02-24 20:42:57.474869
240	61	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 11\ndisease_id: 7\nsource_id: 80\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d427422a-4652-42e7-8546-ca60079194dd	2015-02-24 20:42:57.534785
241	62	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A single patient with BRAF V600E/V600M bi-allelic mutation responded to the\n  V600E drug dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 7\nsource_id: 81\nvariant_id: 13\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2b382c3e-f2ea-4111-a9e6-0b63edf56a3d	2015-02-24 20:42:57.602505
242	63	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 7\nsource_id: 76\nvariant_id: 15\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	efa16fca-a2a6-4616-8969-9281787f702b	2015-02-24 20:42:57.674438
243	64	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 7\nsource_id: 76\nvariant_id: 16\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5e06422c-e6c0-4cfa-826b-1e0dc937cced	2015-02-24 20:42:57.748967
244	65	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have found no significant survival impact for CCND1 in lung cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 5\nsource_id: 82\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	54814e55-883c-44f1-aa98-4d051f5fe5de	2015-02-24 20:42:57.848974
245	66	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Inceased copy number of CCND1 is associated with poorer overall survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 5\nsource_id: 82\nvariant_id: 18\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9447169a-8966-4174-aac2-5561004ff5af	2015-02-24 20:42:57.915578
246	67	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have associated CCND1 expression with poorer survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 5\nsource_id: 82\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0c19f6f3-6a74-4db2-9080-d72d4d18691a	2015-02-24 20:42:57.987105
247	68	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In ER-positive breast cancer patients, Cyclin D1 overexpression is associated\n  with shorter overall survival and increased metastasis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 11\nsource_id: 83\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	328a3253-733f-4196-98f8-8b8911d07b9a	2015-02-24 20:42:58.066081
248	69	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in head and neck squamous cell carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 12\nsource_id: 84\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b57c5d81-7876-402f-8237-d2e6a3cbbfb4	2015-02-24 20:42:58.133121
249	70	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in mantle cell lymphoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 13\nsource_id: 85\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	663c8221-155d-4b3c-8d17-e369c7473433	2015-02-24 20:42:58.210889
250	71	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with the ER-positive subtype of breast\n  cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 11\nsource_id: 83\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ce5395d1-ab35-479e-87ea-b19100467f30	2015-02-24 20:42:58.281154
251	72	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CCND2 hypomethylation is seen to be more common in stage III and IV gastric\n  tumors than stage I and II.\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 14\nsource_id: 86\nvariant_id: 22\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	53c252be-d878-4b6c-b76a-4ea4a63a54b8	2015-02-24 20:42:58.379379
252	73	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the presence pf lymph node metastasis\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 14\nsource_id: 87\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	dc4b1f07-8167-4f71-bd86-2e2521e0b2ee	2015-02-24 20:42:58.447634
253	74	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the increased depth of cancer invasion\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 14\nsource_id: 87\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3b225efc-b7f2-4b79-8c2e-e8f045b00940	2015-02-24 20:42:58.510306
254	75	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 14\nsource_id: 87\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7c26e30f-5cdf-4e28-ab92-8388eb7626ec	2015-02-24 20:42:58.581817
255	76	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with vascular invasion by cancer cells\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 14\nsource_id: 87\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4c39e1fc-9b90-430e-92b8-a59677890337	2015-02-24 20:42:58.640167
256	77	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib\n  (PD-0332991) significantly increased median survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 24\ndisease_id: 15\nsource_id: 88\nvariant_id: 23\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	095cd49c-fd25-490b-9597-aad0ed4c6bd2	2015-02-24 20:42:58.710771
257	78	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with high expression of total cyclin E and low-molecular weight cyclin\n  E is strongly associated with poor prognosis in breast cancer, and the hazard ratio\n  for these patients in 13.3 times higher than those with normal cyclin E levels.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 11\nsource_id: 89\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c55b515c-0aff-4dce-bdc0-51001087e41e	2015-02-24 20:42:58.768201
258	79	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced\n  stage gastric carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 16\nsource_id: 90\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	30110e2c-e8db-4b52-b481-11b3d3901c99	2015-02-24 20:42:58.854763
259	80	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 16\nsource_id: 90\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	405af31d-4137-4541-9fb3-85518ec1e6be	2015-02-24 20:42:58.928638
260	81	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 17\nsource_id: 90\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b898fe6c-86f4-4022-8b0d-957c8903faf7	2015-02-24 20:42:58.996689
261	82	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 25\ndisease_id: 18\nsource_id: 91\nvariant_id: 25\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	10f88ec0-8e80-4e65-bbd4-6e2b8b7ef7c1	2015-02-24 20:42:59.067374
262	83	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 26\ndisease_id: 18\nsource_id: 92\nvariant_id: 25\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3012cfe9-4b83-4610-9914-6e9200053636	2015-02-24 20:42:59.140914
263	84	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 25\ndisease_id: 18\nsource_id: 91\nvariant_id: 26\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7dee366f-cdd1-4694-a774-2de411b713a5	2015-02-24 20:42:59.203342
264	85	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 26\ndisease_id: 18\nsource_id: 92\nvariant_id: 26\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c3e2cb27-676a-44a8-944b-9a1e37df9a26	2015-02-24 20:42:59.271315
265	86	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter time to recurrence.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 5\nsource_id: 93\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a9b2d74a-fe43-4349-b1da-a4b0c3e0237c	2015-02-24 20:42:59.356512
266	87	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter overall survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 5\nsource_id: 93\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a6c9315a-f107-43bd-b572-660954c0a181	2015-02-24 20:42:59.428332
267	88	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was significantly associated with complete remission\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 94\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d788ab31-bdaf-47c5-8cbd-83b3cfce23ee	2015-02-24 20:42:59.499043
268	89	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was associated with improved overall survival in older (>60)\n  patients with AML\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 95\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c1aa955e-5586-4895-a327-9e3fa54b60f8	2015-02-24 20:42:59.565404
269	90	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Overall survival was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA mutations\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 96\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	120d857b-48b0-4168-905b-ce3f5d263655	2015-02-24 20:42:59.63342
270	91	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Remission duration was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 96\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	59508f7d-ad92-40c3-9aaf-45b646b20f9e	2015-02-24 20:42:59.697114
271	92	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation status had no impact on ATRA treatment response in older (>60)\n  patients with AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 95\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	dcad3315-521b-4382-a96d-44f62197412a	2015-02-24 20:42:59.763168
272	93	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 96\nvariant_id: 28\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	cf8e1088-db6e-4f03-b254-68cb2aadb4ef	2015-02-24 20:42:59.852536
273	94	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 19\nsource_id: 97\nvariant_id: 30\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	15d92414-79af-455e-ac03-dc33e81efdb4	2015-02-24 20:42:59.925448
274	95	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 27\ndisease_id: 19\nsource_id: 97\nvariant_id: 30\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5a65689d-deea-4e20-bf05-37d0ea8c2bfd	2015-02-24 20:42:59.976432
275	96	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 20\nsource_id: 98\nvariant_id: 31\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0de60ee4-a46d-458b-b138-29904e368536	2015-02-24 20:43:00.046644
276	97	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations were associated with achievement of complete remission in a\n  large cohort of younger (18-60) AML patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 99\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f57a9e8c-7ce8-4b2c-96eb-1a5a58a2d23a	2015-02-24 20:43:00.122211
277	98	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no difference in the complete remission rate of de novo AML patients\n  with DNMT3A mutation compared to those who are wild type for DNMT3A.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 100\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0e0059ee-3ddc-4e60-a5b7-7023797f9fc4	2015-02-24 20:43:00.187283
278	99	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status (60% of which\n  were R882) was not predictive of overall and relapse free survival in patients with\n  NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 101\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e67e24a4-68e3-48a3-84f7-14006711293b	2015-02-24 20:43:00.278328
279	100	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5%\n  of which were R882) had no prognostic value on overall, relapse free and event free\n  survival.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 99\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	860d2a8f-c9d9-4be3-a261-66a397474dd3	2015-02-24 20:43:00.341384
280	101	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of cytogenetically normal AML patients (18-60 years old),\n  DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall,\n  relapse free and event free survival\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 99\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	51a61c08-3b13-43cd-b11d-92323e9596ff	2015-02-24 20:43:00.40291
281	102	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates did not differ between patients with wildtype or mutant\n  DNMT3A (62% of which affected R882) and cytogenetically normal AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 102\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9915d5db-b4e1-4590-87cc-c6178fb9c440	2015-02-24 20:43:00.467533
282	103	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Therapy-related AML was less common in patients with DNMT3A mutations (64.5%\n  of which were R882) than patients wildtype for DNMT3A in a large cohort of younger\n  (18-60) AML patients\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 99\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bc579c5b-9d54-4a5f-b03e-d1b3087218d4	2015-02-24 20:43:00.528659
283	104	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free\n  and overall) outcome than those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 103\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8dc71e13-1a18-496b-b3d4-a54dec422263	2015-02-24 20:43:00.595431
284	105	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with DNMT3A mutations (59% of which were R882) showed worse survival\n  (event-free and overall) outcome than those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 103\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b3ee5b04-2968-474b-9e2d-9790efe14ee8	2015-02-24 20:43:00.657576
285	106	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including\n  R882) were associated with worse overall survival compared to those without DNMT3A\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 103\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9f499101-86ba-4fac-8846-d947e8771252	2015-02-24 20:43:00.737335
286	107	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cytogenetically normal AML patients, DNMT3A R882 mutations are associated\n  with lower overall and disease free survival as compared to patients with wild type\n  DNMT3A.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 104\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	cf761ada-39a8-4c67-94e3-2e0ff81f93e5	2015-02-24 20:43:00.79587
287	108	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutations were associated with\n  significantly reduced overall survival and relapse free survival in patients wildtype\n  for NPM1 and FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 101\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c39ced50-09cb-45bc-84f8-3260d59c2d16	2015-02-24 20:43:00.864145
288	109	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation\n  is prognostic for shorter disease free survival and overall survival compared to\n  patients without the mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 102\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a1df9c63-f24a-43d3-8a20-5a125d2deaa0	2015-02-24 20:43:00.926941
289	110	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations\n  other than R882 are prognostic for shorter disease free survival and overall survival\n  compared to patients without the mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 102\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	050bb4da-a6d8-4464-aafd-df8357053718	2015-02-24 20:43:00.989015
290	111	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutation was associated with reduced relapse free and overall survival\n  in ELN-unfavorable, cytogenetically normal AMLs\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 99\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	50c9ed5b-7fbd-4910-a7e7-63950fd39955	2015-02-24 20:43:01.052355
291	112	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations\n  were associated with reduced relapse free survival in the entire cohort as well\n  as the subset of patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 2\nsource_id: 99\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8ff3f9c2-6223-46f2-8fcc-aa0d2dab75be	2015-02-24 20:43:01.11385
292	113	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (62% of which were R882) were associated with reduced disease-free\n  survival in patients with cytogenetically normal AML.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 102\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	41c4c3db-fdb0-4309-b678-1d071a8bdde7	2015-02-24 20:43:01.256017
293	114	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutation have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 101\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	60847d36-57af-4613-9839-958bc0f5d35a	2015-02-24 20:43:01.316402
294	115	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rate was not different between young AML patients (<60 years\n  old) with or without DNMT3A mutations (60% of which were R882)\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 101\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	593dc71d-51bd-488a-8cba-81a7710a9cfa	2015-02-24 20:43:01.37866
295	116	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 101\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0cbf2a0a-f72c-48b7-bbbe-e7cdb73204a5	2015-02-24 20:43:01.448113
296	117	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutations (60% of which were\n  R882) were older in age, had higher white blood cell counts and had higher platelet\n  counts than patients wildtype for DNMT3A\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 101\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c605a2d3-266f-44df-9597-f9684161514c	2015-02-24 20:43:01.516462
297	118	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations occur most often in de novo AML patients with intermediate\n  cytogenic risk.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 100\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ddf9b96b-ec05-4f5f-a32b-06cea1bc9913	2015-02-24 20:43:01.581872
298	119	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with older age, higher white blood cell\n  count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically\n  normal AML patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 104\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ee83e003-f576-4162-a0b9-8d23eb32b82e	2015-02-24 20:43:01.645876
299	120	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with an intermediate\n  risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 103\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9d837ecb-fb01-4bee-a633-dd2229877fd9	2015-02-24 20:43:01.719118
300	121	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with intermediate\n  risk cytogenetics (including normal karyotype)\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 103\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ca5122da-4655-4b11-9dbd-50bd660c2415	2015-02-24 20:43:01.809529
301	122	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (64.5% R882) were associated with older age, higher white blood\n  cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML\n  patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 99\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b498b74f-b9ed-4ff4-91a8-f14ffa883f05	2015-02-24 20:43:01.887817
302	123	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with cytogenetically normal AML in a large\n  cohort of younger (18-60) AML patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 99\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	81ee6a54-39f5-41fe-bd9d-5b4108eb22f2	2015-02-24 20:43:01.953218
303	124	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Daunorubicin treatment resulted in similar overall survival and disease free\n  survival in de novo AML patients with DNMT3A R882 mutation compared to those who\n  do not harbor this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 2\nsource_id: 100\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1765b040-e879-4936-88e9-7277e2198e17	2015-02-24 20:43:02.019184
304	125	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Idarubicin increases the overall survival and disease free survival in de novo\n  AML patients with DNMT3A R882 mutation compared to those who do not harbor this\n  mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 29\ndisease_id: 2\nsource_id: 100\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	82ecd79c-995b-4dff-9554-3fdf1d4d6fa8	2015-02-24 20:43:02.081409
305	126	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Median survival of patients with EGFR L858R mutation is better than those with\n  wild type EGFR.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 5\nsource_id: 105\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	35121ba7-b7f9-4f5d-874d-765c3c9633c5	2015-02-24 20:43:02.158824
306	127	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 30\ndisease_id: 5\nsource_id: 106\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6a4a47c5-1425-45f4-a3a5-92efe840e118	2015-02-24 20:43:02.22563
307	128	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 31\ndisease_id: 5\nsource_id: 106\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ffacc074-1b4c-4436-9efc-1f8e75312ecc	2015-02-24 20:43:02.278626
308	129	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall\n  survival compared to patients with L858R or other exon 19 activating mutations.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 5\nsource_id: 107\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bdeaa84f-6bbf-4d47-a6c3-5fe477680fe6	2015-02-24 20:43:02.363324
309	130	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients with T790M and another activating mutations, their progression\n  free survival is shorter compared to those who do not possess T790M mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 5\nsource_id: 108\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ad702e48-09b2-44cb-bd00-044c3f58ae94	2015-02-24 20:43:02.428551
310	131	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The T790M mutation in EGFR has been shown to confer resistance to the tyrosine\n  kinase inhibitor erlotinib, and patients harboring this mutation that are placed\n  on the drug are likely to relapse.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 21\nsource_id: 109\nvariant_id: 34\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	49a80ad4-3d02-4c9d-8fcc-02ea935ce62e	2015-02-24 20:43:02.494949
311	132	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In non-small cell lung cancer, the appearance of T790M mutation leads to resistance\n  to gefitinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 30\ndisease_id: 5\nsource_id: 110\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fa221f61-d769-4569-9bcc-cd56f04178c0	2015-02-24 20:43:02.56633
312	133	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence\n  of L858R mutation is prognostic for better progression free survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 5\nsource_id: 111\nvariant_id: 33\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	18542bc7-b1f5-4910-97e3-e553650c2059	2015-02-24 20:43:02.6356
328	149	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 35\ndisease_id: 11\nsource_id: 115\nvariant_id: 44\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a4a30879-2a9f-4cec-9158-b640f88e6a49	2015-02-24 20:43:03.738096
313	134	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Geftinib added to pemetrexed and carboplatin showed greater effectiveness than\n  premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 30\ndisease_id: 5\nsource_id: 112\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	43be4d48-1357-45e5-87e6-78d89bfb88ae	2015-02-24 20:43:02.700322
314	135	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting\n  in stable disease.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 31\ndisease_id: 5\nsource_id: 113\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8ab6111c-968c-4bd9-9783-d407f2a9f1a5	2015-02-24 20:43:02.770554
315	136	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting\n  in stable disease.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 32\ndisease_id: 5\nsource_id: 113\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fdb607f9-6e04-454b-937f-43773d850df0	2015-02-24 20:43:02.855532
316	137	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR,\n  especially when an L858R mutation is also present.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 33\ndisease_id: 5\nsource_id: 114\nvariant_id: 34\nrating: 1\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	882f741c-80c2-4e15-ba89-3392e52c2d98	2015-02-24 20:43:02.930392
317	138	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell\n  lines retrovirally transduced with wildtype or mutant ERBB2.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 34\ndisease_id: 11\nsource_id: 115\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	068187bc-3109-468e-bf43-3faff28ad3a1	2015-02-24 20:43:03.005885
318	139	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 35\ndisease_id: 11\nsource_id: 115\nvariant_id: 35\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	38172417-6463-4169-95e8-3fe05e18dd37	2015-02-24 20:43:03.067669
319	140	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 35\ndisease_id: 11\nsource_id: 115\nvariant_id: 36\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d311705a-f378-49ce-b51b-a90dd157abfd	2015-02-24 20:43:03.133897
320	141	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2,\n  cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive\n  to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 35\ndisease_id: 11\nsource_id: 115\nvariant_id: 37\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	47d3f144-8285-45fe-8fda-8cff9f3081c6	2015-02-24 20:43:03.203479
321	142	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 35\ndisease_id: 11\nsource_id: 115\nvariant_id: 38\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3e84ea75-bdcd-426c-8a6e-9fa8458cdde9	2015-02-24 20:43:03.26873
322	143	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally\n  transduced with wildtype or mutant ERBB2.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 35\ndisease_id: 11\nsource_id: 115\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c8ac2392-0848-474d-ad88-b1706942ce45	2015-02-24 20:43:03.332006
323	144	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive\n  to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 35\ndisease_id: 11\nsource_id: 115\nvariant_id: 40\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	26c18cb3-4abb-4b76-bf3b-1f0d2c46ecff	2015-02-24 20:43:03.422361
324	145	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the in-frame insertion of a single amino acid at position\n  780 was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 35\ndisease_id: 11\nsource_id: 115\nvariant_id: 41\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1cecd0d2-4322-41e7-87e8-bf73a409d7f8	2015-02-24 20:43:03.489108
325	146	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 35\ndisease_id: 11\nsource_id: 115\nvariant_id: 42\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c4d7c36c-0995-4a23-87a7-4d893f0dd974	2015-02-24 20:43:03.55519
326	147	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 34\ndisease_id: 11\nsource_id: 115\nvariant_id: 42\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bb124eb2-d652-4395-8136-1552beb764d2	2015-02-24 20:43:03.608175
327	148	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 35\ndisease_id: 11\nsource_id: 115\nvariant_id: 43\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2fecc640-1401-4446-aef8-309685ddd492	2015-02-24 20:43:03.669523
329	150	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 35\ndisease_id: 11\nsource_id: 115\nvariant_id: 45\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3ea9d961-d44b-4d51-a247-d9e86e4b16e6	2015-02-24 20:43:03.806698
330	151	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L536Q ligand binding domain mutation have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 36\ndisease_id: 11\nsource_id: 116\nvariant_id: 46\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7c1afe81-abdb-47bd-98f9-39ea0eccfc1a	2015-02-24 20:43:03.902765
331	152	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L536Q ligand binding domain mutation have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 37\ndisease_id: 11\nsource_id: 116\nvariant_id: 46\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	212ef75a-6ff7-451a-935c-5892c7e5ba73	2015-02-24 20:43:03.954401
332	153	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 38\ndisease_id: 11\nsource_id: 117\nvariant_id: 46\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	555ea31e-79fd-4416-9497-8512dd326717	2015-02-24 20:43:04.035367
333	154	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The N538G ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 36\ndisease_id: 11\nsource_id: 116\nvariant_id: 47\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	843511c2-f81a-44e8-a508-0fd8a639b94f	2015-02-24 20:43:04.118794
334	155	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The N538G ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 37\ndisease_id: 11\nsource_id: 116\nvariant_id: 47\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	691bdb4f-6955-49e1-b956-cfca412c71d9	2015-02-24 20:43:04.170125
335	156	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 38\ndisease_id: 11\nsource_id: 117\nvariant_id: 47\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c36fadee-c90a-4572-a925-bb3e78110b74	2015-02-24 20:43:04.24179
336	157	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537C ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 36\ndisease_id: 11\nsource_id: 116\nvariant_id: 48\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	396e6de5-3aab-45e0-b6b6-64817ffd3036	2015-02-24 20:43:04.312486
337	158	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537C ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 37\ndisease_id: 11\nsource_id: 116\nvariant_id: 48\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e0d8a9b5-9315-44c6-8987-94a8e7576214	2015-02-24 20:43:04.366712
338	159	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 38\ndisease_id: 11\nsource_id: 117\nvariant_id: 48\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e059fb1f-bfa8-4d64-93b5-02c61e7d7a9b	2015-02-24 20:43:04.450148
339	160	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537N ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 36\ndisease_id: 11\nsource_id: 116\nvariant_id: 49\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	95be2bb8-3af0-46a5-8f8c-4c5211425779	2015-02-24 20:43:04.517627
340	161	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537N ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 37\ndisease_id: 11\nsource_id: 116\nvariant_id: 49\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8e58f857-af01-4328-9358-17e1ba67ba78	2015-02-24 20:43:04.574481
341	162	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 38\ndisease_id: 11\nsource_id: 117\nvariant_id: 49\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	aea9b2f1-eac9-434c-8fe0-2566f6a4e639	2015-02-24 20:43:04.641623
342	163	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537S ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 36\ndisease_id: 11\nsource_id: 116\nvariant_id: 50\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b0172655-a438-4482-814d-ef59fb5a4d35	2015-02-24 20:43:04.712568
343	164	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537S ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 37\ndisease_id: 11\nsource_id: 116\nvariant_id: 50\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8b23f9db-c81e-478f-ac88-62bf8048e708	2015-02-24 20:43:04.765308
344	165	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 38\ndisease_id: 11\nsource_id: 117\nvariant_id: 50\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	340e3fa6-6c0f-4cd0-bd5f-3c4aae1e96ea	2015-02-24 20:43:04.838999
345	166	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib\n  treatment resulted in necrosis and shrinkage of the primary and metastatic sites\n  and overall stable disease. However, this response did not meet criteria for RECIST\n  partial response.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 39\ndisease_id: 22\nsource_id: 118\nvariant_id: 51\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4be7b1fa-33a0-4009-a13f-60cd5202bc9e	2015-02-24 20:43:04.935694
346	167	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Following identification of this fusion in a patient, this fusion was expressed\n  in HEK 293T cells leading to oligomerization in the absence of ligand, induced morphologic\n  changes in the cell and increased cellular proliferation consistent with FGFR activation.\n  Mouse xenografts using the bladder cell line SW780 which harbors an FGFR3-BAIAP2L1\n  fusion showed reduced tumor burden when treated with the FGFR inhibitor PD173074.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 40\ndisease_id: 23\nsource_id: 119\nvariant_id: 52\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	083fff45-e1c0-43d4-ad5b-451b10b1d593	2015-02-24 20:43:05.010524
347	168	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be recurrent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 41\ndisease_id: 22\nsource_id: 118\nvariant_id: 53\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3f63f44f-7a30-438e-b4e2-95f8f3ee4eaf	2015-02-24 20:43:05.083921
348	169	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be recurrent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 39\ndisease_id: 22\nsource_id: 118\nvariant_id: 53\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3be4a9d6-4ec0-45a1-ac37-1db842b15c1f	2015-02-24 20:43:05.142721
349	170	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 120\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1d39c532-62aa-4640-896e-2674ea760b6d	2015-02-24 20:43:05.220634
350	171	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 101\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bf4a8cc3-fa75-4eee-878e-d543403013c4	2015-02-24 20:43:05.284138
351	172	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD duplication length did not effect prognosis in patients with an NPM1\n  mutation\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 121\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f9d22429-079f-486d-bd80-75d286ddafb5	2015-02-24 20:43:05.365153
352	173	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Size of the FLT3-ITD mutant duplication had no impact on overall survival or\n  relapse rate\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 122\nvariant_id: 54\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	62420c88-df4c-4073-8362-8073277374cd	2015-02-24 20:43:05.456679
353	174	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with D835 mutation of FLT3 did not have worse overall survival\n  than patients with wild type FLT3\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 123\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ad8cb988-9427-4076-9746-be80b209e3ee	2015-02-24 20:43:05.526357
354	175	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated\n  with worse overall survival compared to those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 103\nvariant_id: 54\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	498d2f7b-6d43-4cf7-99f1-a511ba702393	2015-02-24 20:43:05.589219
355	176	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 121\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f8d2e9a3-51e5-42c6-a1e8-b342d0313b46	2015-02-24 20:43:05.65529
356	177	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 121\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5b8755c4-acae-4083-bb49-5248f8ef2000	2015-02-24 20:43:05.720182
357	178	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free\n  and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 121\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b031a6cf-024a-48f0-b772-0691c3002d44	2015-02-24 20:43:05.790223
358	179	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Meta-analysis of studies involving cytogentically normal younger (<60) patients\n  showed reduced overall and relapse-free survival for patients with FLT3-ITD\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 2\nsource_id: 124\nvariant_id: 54\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	596fc6f5-157e-4132-b36c-8aca64c575cb	2015-02-24 20:43:05.855529
359	180	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor\n  survival and inferior disease free survival in patients with AML.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 125\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	66f5e99a-c1ea-4473-bc94-fbe70fb15d28	2015-02-24 20:43:05.941431
360	181	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with internal tandem repeat of FLT3 has worse overall survival\n  than patients with wild type FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 123\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	dab64768-784e-40f3-8165-5ff81fb2d95c	2015-02-24 20:43:06.011097
361	182	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients, FLT3-TKD mutation was associated with poorer disease free survival\n  compared to patients with wild type FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 126\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ec90b881-2bfd-4db4-8f0e-d49057c55094	2015-02-24 20:43:06.082859
362	183	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were not significantly different in AML patients with FLT3 D835 mutations\n  compared to wild type\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 123\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	264b83ea-335a-45af-8888-d683abc890af	2015-02-24 20:43:06.147656
363	184	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared\n  to wild type\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 123\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8f369e43-0166-4b49-9e1f-5a0e89f4751c	2015-02-24 20:43:06.217387
364	185	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD mutation without NPM1 was associated with increased relapse risk and\n  reduced overall survival in young adult AML patients (median age 43)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 122\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ee4d62ce-a622-486f-892d-51544140a6f3	2015-02-24 20:43:06.286797
365	186	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 42\ndisease_id: 2\nsource_id: 127\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f868dbec-0ade-448a-ab2e-73a61eb384cf	2015-02-24 20:43:06.364045
366	187	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  FLT3-ITD mutation\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 43\ndisease_id: 2\nsource_id: 128\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8aa789d0-fe06-4d4f-9cb1-72a925313fa9	2015-02-24 20:43:06.466396
367	188	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 28\ndisease_id: 2\nsource_id: 127\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	751d4aeb-35cb-4e1c-b7c4-df221b899597	2015-02-24 20:43:06.535814
368	189	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Increasing level of FLT3-ITD mutant correlated with reduced disease-free and\n  overall survival as well as increased relapse rate\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 122\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7676e337-ba83-4ebc-80e4-6ed9f164fc86	2015-02-24 20:43:06.602456
369	190	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 42\ndisease_id: 2\nsource_id: 127\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a3616d0c-595f-4c2c-81b4-cc84821c143e	2015-02-24 20:43:06.677262
370	191	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 28\ndisease_id: 2\nsource_id: 127\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d7c01f80-a3ea-4fe5-81ad-0a464a56b3d6	2015-02-24 20:43:06.755698
371	192	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated\n  with sorafenib resistance\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 2\nsource_id: 129\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9d748186-ea80-4c14-9c6b-794f99228810	2015-02-24 20:43:06.822103
372	193	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeat mutations,\n  but not in patients with wild type or D835 mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 2\nsource_id: 130\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2e7e59c9-04bf-4e88-b464-8f44e9042dd7	2015-02-24 20:43:06.888353
373	194	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeats, but not\n  in wild type or D835 mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 2\nsource_id: 130\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	874f8e45-d93f-49a9-be0f-f512fb73fdba	2015-02-24 20:43:06.971878
374	195	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML\n  patients with FLT3 mutations\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 44\ndisease_id: 2\nsource_id: 131\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6f579b8f-df0b-42ed-af0c-fe6eba57ca81	2015-02-24 20:43:07.046769
375	196	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with\n  FLT3 internal tandem repeat\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 45\ndisease_id: 2\nsource_id: 132\nvariant_id: 54\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6bb8084e-bf20-4bea-b6fd-ed3853242f20	2015-02-24 20:43:07.107479
376	197	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML\n  patients with FLT3 mutations\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 44\ndisease_id: 2\nsource_id: 131\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ffec247f-9013-4e69-a18e-5fc0c45efc83	2015-02-24 20:43:07.169604
377	198	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 46\ndisease_id: 24\nsource_id: 133\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e9c107c4-8087-4049-9630-095c81fa337a	2015-02-24 20:43:07.247902
378	199	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 47\ndisease_id: 24\nsource_id: 133\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	41321ffc-1554-4671-9dd4-9a37172abd99	2015-02-24 20:43:07.304677
379	200	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic\n  for better survival compared to patients who harbor wild type mutation.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 25\nsource_id: 134\nvariant_id: 57\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c6487ce0-816d-4ada-9c9e-4c86ab1dc57d	2015-02-24 20:43:07.397998
380	201	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with glioblastoma, those harboring IDH1 R132 mutation has higher\n  overall survival compared to those who do not have IDH1 mutation\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 26\nsource_id: 135\nvariant_id: 57\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e060fdb2-bea2-494c-8dc2-1f97c4223d8b	2015-02-24 20:43:07.471584
381	202	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 136\nvariant_id: 57\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	079f3ba2-2a25-4d6e-8b84-1ca8bd0e3dfc	2015-02-24 20:43:07.540042
382	203	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The complete remission and overall survival rates in patients with IDH1 R132\n  mutation is not significantly different from those who do not have this mutation\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 137\nvariant_id: 57\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a7c50cf0-9064-4cee-92bf-c87a31277e19	2015-02-24 20:43:07.606012
383	204	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 138\nvariant_id: 58\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2180c714-029a-4da7-a124-d6d6b18cf9bc	2015-02-24 20:43:07.665117
384	205	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 138\nvariant_id: 59\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	478dc1ce-4e39-4015-acc4-9163df019302	2015-02-24 20:43:07.733162
385	206	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 138\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b783d36c-18e5-4ab4-b3e9-655ec4cb4b80	2015-02-24 20:43:07.790678
386	207	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated\n  with worse overall survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 27\nsource_id: 139\nvariant_id: 57\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fe63c390-05be-4aae-a663-090495e86ca0	2015-02-24 20:43:07.942383
387	208	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In leukemia patients, IDH1 R132 mutation is associated with adults and normal\n  karyotype.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 140\nvariant_id: 57\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c4292aae-071e-47e9-bde1-ddb74f557acf	2015-02-24 20:43:08.016318
388	209	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation is associated with patients of older age, high platelet count\n  during diagnosis, cytogenic normalcy and NPM1 mutation\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 138\nvariant_id: 58\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4b04704a-47a0-4031-b85d-c2cda6c4c65a	2015-02-24 20:43:08.076789
389	210	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with IDH2 R140K mutation have improved overall survival compared\n  to those with wild-type IDH2\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 141\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e66ddd9d-a18d-4b97-86f4-9dc982446cc5	2015-02-24 20:43:08.14409
390	211	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact\n  overall survival or disease free survival.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 142\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8ad084e6-ed05-42a5-b3a5-c34878cb1453	2015-02-24 20:43:08.208261
391	212	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R140Q/L have event free survival and\n  overall survival similar to those with wild-type IDH2\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 2\nsource_id: 143\nvariant_id: 61\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	368cfe02-8d1d-4fe0-b1fd-5b8513ef31c7	2015-02-24 20:43:08.273931
392	213	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: R140Q mutation in IDH2 does not have prognostic value in patients with MDS.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 28\nsource_id: 144\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	41056ebd-aa51-4cef-ada3-1eb827695cb6	2015-02-24 20:43:08.342898
393	214	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L\n  do not confer prognostic value (overall survival)\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 28\nsource_id: 145\nvariant_id: 61\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	02e641a2-9d9b-4341-ad6c-76526e170a51	2015-02-24 20:43:08.412777
394	215	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R172K have event free survival and\n  overall survival similar to those with wild-type IDH2.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 2\nsource_id: 143\nvariant_id: 62\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9bd81106-1210-4c25-b0d9-661d079003cf	2015-02-24 20:43:08.467651
395	216	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutations such as R172K\n  do not confer prognostic value (overall survival)\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 28\nsource_id: 145\nvariant_id: 62\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	78e00c87-cc69-4920-bf23-4b0950f1453b	2015-02-24 20:43:08.520899
396	217	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with an IDH2 R172K mutation have worse overall survival compared\n  to those with wild-type IDH2\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 141\nvariant_id: 62\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	db621e3b-136b-4f7b-8e41-f21157605166	2015-02-24 20:43:08.582164
397	218	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or\n  CLL).\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 29\nsource_id: 146\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	98dd1de3-6f4a-474c-97a4-442421adda30	2015-02-24 20:43:08.650923
398	219	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is associated with myeloid neoplasms (AML, MDS, CMML/aCML)\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 1\nsource_id: 146\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	de467ccf-1b27-48c4-8431-b0e59e6754e8	2015-02-24 20:43:08.709861
399	220	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical\n  CML).\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 1\nsource_id: 146\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d27f4800-3f0f-4a9b-a182-526e55de9d3d	2015-02-24 20:43:08.769757
400	221	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The SNP rs10974944 in the Jak2 locus is associated with increased predisposition\n  for JAK2 V617F mutation and its associated cancer\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 30\nsource_id: 147\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3fdb27d6-15fe-48e0-a498-7747f7080814	2015-02-24 20:43:08.826942
401	222	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 48\ndisease_id: 31\nsource_id: 148\nvariant_id: 63\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ff8c9765-0d6a-495c-a65d-151841bce40c	2015-02-24 20:43:08.893095
402	223	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction\n  in the percentage of cells harboring JAK2 V617F.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 49\ndisease_id: 32\nsource_id: 149\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	225b12aa-a69f-4c77-b951-fcbf92738ad4	2015-02-24 20:43:08.96336
403	224	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 7\nsource_id: 150\nvariant_id: 65\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c811a693-a75d-4cc2-9d00-b6be1d75cb37	2015-02-24 20:43:09.021784
404	225	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 7\nsource_id: 150\nvariant_id: 67\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e9283b05-6e55-432a-99b0-45d7a683327b	2015-02-24 20:43:09.091203
405	226	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 7\nsource_id: 150\nvariant_id: 69\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0150196a-d435-420c-b2ea-10603394374a	2015-02-24 20:43:09.147751
406	227	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between wildtype KIT or KIT mutations in\n  exon 9 or 11 in survival among GIST patients.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 33\nsource_id: 151\nvariant_id: 65\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	865a6746-f00a-4815-9bcd-1fb78b1d4990	2015-02-24 20:43:09.216332
407	228	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations were identified in morphologically benign, incidentally discovered\n  GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating\n  is not a prognostic marker for GIST\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 33\nsource_id: 152\nvariant_id: 65\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	142c73b4-ab11-4b9c-a7a4-498895321d22	2015-02-24 20:43:09.289384
408	229	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 17\nsource_id: 153\nvariant_id: 65\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9ada2fd4-c3f9-467e-a04d-818d60b0a47f	2015-02-24 20:43:09.359437
409	230	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between wildtype KIT or KIT mutations in\n  exon 9 or 11 in survival among GIST patients.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 33\nsource_id: 151\nvariant_id: 70\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0cc8f917-2f8f-4220-8874-1a34620de2b6	2015-02-24 20:43:09.459585
410	231	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 17\nsource_id: 153\nvariant_id: 70\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	281ed3b9-c2d4-4d40-88bb-7201a8608abd	2015-02-24 20:43:09.524367
411	232	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In acute myloid leukemia patients, D816 mutation is associated with earlier\n  relapse and poorer prognosis than wildtype KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 34\nsource_id: 154\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f5ddd089-8e4a-4e35-b831-ea0ae646c5ec	2015-02-24 20:43:09.592254
412	233	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with worse overall and cause-specific prognosis in\n  patients with GIST compared to patients with wildtype KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 33\nsource_id: 155\nvariant_id: 65\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6e442aea-f455-498a-bd35-3afd67072969	2015-02-24 20:43:09.664348
413	234	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST patients with exon 14 KIT mutations had reduced overall survival compared\n  to patients wiltype for KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 33\nsource_id: 151\nvariant_id: 68\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1571209d-24d0-4a97-a2ee-919113358ae1	2015-02-24 20:43:09.73232
414	235	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 33\nsource_id: 155\nvariant_id: 65\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ad66608e-86fc-4a93-8b28-fff6c51ab4b7	2015-02-24 20:43:09.80734
415	236	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 33\nsource_id: 155\nvariant_id: 69\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c126f5c0-81a4-4fb5-b326-235a5fc14393	2015-02-24 20:43:09.877871
416	237	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 7\nsource_id: 156\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0a4402fc-47cf-4e28-9a1f-3c0f2cf74be2	2015-02-24 20:43:09.953647
417	238	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 4\ndisease_id: 7\nsource_id: 156\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	555ec314-7590-4327-b66c-18e40476905a	2015-02-24 20:43:10.006909
418	239	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 7\nsource_id: 156\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bf579127-855a-4568-99a7-6e33c8ddb413	2015-02-24 20:43:10.065781
419	240	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT V654A results in imatinib resistance in GIST patient-derived cell lines.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 33\nsource_id: 157\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f71678b5-5041-4a68-ae38-567ee2038e7d	2015-02-24 20:43:10.143162
420	241	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a case study, a patient with anal melanoma harboring a 7 codon duplication\n  in the juxtamambrane region (exon 11) of KIT showed marked response 4 months after\n  imatinib treatment.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 35\nsource_id: 158\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9a6ee50b-f6ad-42cb-9228-e1fb0039f9d8	2015-02-24 20:43:10.213618
421	242	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive\n  to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation\n  of KIT but did not alter the binding affinity of dasatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 5\ndisease_id: 7\nsource_id: 156\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	55b28c19-1e3e-4909-9618-12b2987ba710	2015-02-24 20:43:10.282839
422	243	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 5\ndisease_id: 5\nsource_id: 159\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e26186c2-291b-406c-b51d-2a3bfad483cd	2015-02-24 20:43:10.353317
423	244	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 5\nsource_id: 159\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	59ee87f8-2e16-45ba-bcc0-4704fcda0059	2015-02-24 20:43:10.40854
424	245	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 4\ndisease_id: 5\nsource_id: 159\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7363741a-cc66-4efe-9d01-3d3f79bac05b	2015-02-24 20:43:10.461888
425	246	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective\n  for those that are refractory to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 50\ndisease_id: 33\nsource_id: 160\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2f85b30b-f678-4592-9e1a-f568d8622a4f	2015-02-24 20:43:10.533476
426	247	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 15\ndisease_id: 10\nsource_id: 161\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	dbfc12a5-fc30-4d5d-be97-98a081909381	2015-02-24 20:43:10.5986
427	248	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12\n  mutation did not impact overall or disease free survival.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 10\nsource_id: 162\nvariant_id: 75\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	35f4a2ce-57ee-4714-afa6-29208558c387	2015-02-24 20:43:10.658709
428	249	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no strong association between KRAS mutation status and overall or progression\n  free survival in patients with NSCLC\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 5\nsource_id: 163\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5da60921-d307-43c7-ba8f-e0ea11fcc400	2015-02-24 20:43:10.724911
429	250	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 36\nsource_id: 164\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b595ccaa-8fe5-45cd-a09e-6fc4d7ef3db0	2015-02-24 20:43:10.786211
430	251	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A meta-analysis showed KRAS mutation is associated with worse outcome in patients\n  with NSCLC\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 5\nsource_id: 165\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7bcfcf4b-4eee-402b-92bb-0c9d8ffe7033	2015-02-24 20:43:10.843265
431	252	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with reduced progression-free and\n  overall survival in patients with advanced colorectal cancer\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 9\nsource_id: 166\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	cc1a5509-3296-4427-ae4d-95986380aa8a	2015-02-24 20:43:10.907887
432	253	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 37\nsource_id: 164\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	571e52d9-4148-4dda-96f3-77b1ad0a8d51	2015-02-24 20:43:10.962362
433	254	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung cancer patients are not associated with smoking history,\n  age or gender\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 5\nsource_id: 167\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9e26c6ea-d7d1-4771-a1ae-c73320a9a3fe	2015-02-24 20:43:11.072173
434	255	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung cancer patients are not associated with smoking history,\n  age or gender\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 5\nsource_id: 167\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4fd7044e-bd19-43e3-9e93-da028198744c	2015-02-24 20:43:11.129268
435	256	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 38\nsource_id: 168\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	138edcc8-61e4-48b8-8062-e858ecdbdf97	2015-02-24 20:43:11.19551
436	257	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12C occur more frequently in women than men\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 17\nsource_id: 169\nvariant_id: 77\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0a8fd16e-9cf7-4db0-ba4b-28cde9f4961a	2015-02-24 20:43:11.264997
437	258	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12D mutation occurs in never smokers significantly more often than in\n  smokers\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 17\nsource_id: 169\nvariant_id: 78\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	30f69966-3073-460d-8164-608c9d659215	2015-02-24 20:43:11.334714
438	259	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 38\nsource_id: 168\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d5b089a4-6e17-4883-85df-53d2feedaa7e	2015-02-24 20:43:11.391234
439	260	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer who harbor KRAS mutation have low response rate\n  to cetuximab\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 15\ndisease_id: 9\nsource_id: 170\nvariant_id: 73\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3b08b483-cf94-47d4-8622-4810d5cd0ef8	2015-02-24 20:43:11.453112
440	261	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 30\ndisease_id: 5\nsource_id: 171\nvariant_id: 74\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fe0c2436-34c3-411f-a190-73e04ade7268	2015-02-24 20:43:11.528767
441	262	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 31\ndisease_id: 5\nsource_id: 171\nvariant_id: 74\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9d53d71e-9ae1-46b3-a176-92abf3ab2081	2015-02-24 20:43:11.583465
442	263	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with lack of response to cetuximab\n  in patients with advanced colorectal cancer\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 15\ndisease_id: 9\nsource_id: 166\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	536d89b6-9013-43f6-9a5f-c802571a9f93	2015-02-24 20:43:11.64065
443	264	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations\n  (primarily G12/G13) had lower response and disease control rates and shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype KRAS\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 15\ndisease_id: 9\nsource_id: 72\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	831f10b6-ee26-48f4-b0b6-a78ceee58bda	2015-02-24 20:43:11.702689
444	265	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 5\nsource_id: 172\nvariant_id: 78\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a522a5db-140a-46ad-a888-9479005ca5c8	2015-02-24 20:43:11.767111
445	266	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft model\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 15\ndisease_id: 10\nsource_id: 161\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4a960926-06db-4b53-82c5-a1bad6954e57	2015-02-24 20:43:11.827192
446	267	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: Sensitivity\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 51\ndisease_id: 9\nsource_id: 173\nvariant_id: 74\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c5c52c69-42d1-440f-b2c9-6533181fe172	2015-02-24 20:43:11.901302
447	268	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 52\ndisease_id: 5\nsource_id: 174\nvariant_id: 78\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d2c8c049-515f-47a4-9d31-ab2f0b5aaea3	2015-02-24 20:43:11.975486
448	269	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 53\ndisease_id: 5\nsource_id: 174\nvariant_id: 78\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	550897f6-165e-4290-b9cc-6990f54968ed	2015-02-24 20:43:12.032336
449	270	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft model\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 15\ndisease_id: 10\nsource_id: 161\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	14558256-6ae9-45f1-8e4e-0c9a010debc7	2015-02-24 20:43:12.097944
450	271	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 15\ndisease_id: 10\nsource_id: 161\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	528895a3-1a1d-4012-a3f8-aabc30facf75	2015-02-24 20:43:12.169252
451	272	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition\n  by about 5 fold.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 17\ndisease_id: 7\nsource_id: 175\nvariant_id: 81\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	78c26853-5a46-45fe-826d-fce60ca3c5d8	2015-02-24 20:43:12.237338
452	273	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 17\ndisease_id: 7\nsource_id: 176\nvariant_id: 82\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a17fccf2-d287-449e-8d35-1dfef1ba4861	2015-02-24 20:43:12.299239
453	274	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 19\nsource_id: 97\nvariant_id: 83\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	888d9aa4-f9d8-4ed2-a9f7-554eb6bf7617	2015-02-24 20:43:12.361916
454	275	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 27\ndisease_id: 19\nsource_id: 97\nvariant_id: 83\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	17a8864f-af13-4a69-ac6e-c75f557479eb	2015-02-24 20:43:12.416397
455	276	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation was associated with increased overall and disease-free\n  survival as well as tumor regression in patients with gliomas.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 39\nsource_id: 177\nvariant_id: 84\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	79ce28dc-44fb-4d85-9a20-f027b4c1c2a0	2015-02-24 20:43:12.479392
456	277	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted\n  from temozolomide. This benefit was also methylation status dependent, as those\n  without methylation did not see increased survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 39\nsource_id: 178\nvariant_id: 84\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f88f0d8c-2ef3-4811-8b8e-a8d16e9202dc	2015-02-24 20:43:12.587203
457	278	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor,\n  such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may\n  be useful in treating patients with initial resistance to alkylating agents.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 54\ndisease_id: 39\nsource_id: 177\nvariant_id: 84\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fc55ec45-d748-477a-b80b-2f9370d5c56e	2015-02-24 20:43:12.653369
458	279	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation has been shown to confer sensitivity to alkylating\n  agents, such as carmustine, in glioblastoma cells.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 55\ndisease_id: 39\nsource_id: 177\nvariant_id: 84\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	35ba20a6-b40f-40aa-8c29-433dc9d7c11e	2015-02-24 20:43:12.721138
459	280	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and event-free survival was longer for\n  normal karyotype AML patients with Exon 12 NPM1 mutations.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 179\nvariant_id: 85\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	46bead65-6f64-44a7-aaaa-16d9bd71a9a3	2015-02-24 20:43:12.78697
460	281	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and\n  without FLT3-ITD have increased overall survival, relapse-free survival and response\n  to induction chemotherapy\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 180\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9636d4b7-3645-4419-ac95-d0fdcfbd9fad	2015-02-24 20:43:12.848788
461	282	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with increased complete remission rates as well\n  as longer overall disease free survival in normal karyotype AML patients >59 years\n  old, particularly in those >69 years old\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 181\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	16aa9e4c-8423-4f70-881a-d1aa883b4209	2015-02-24 20:43:12.915228
491	312	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment improved overall and relapse-free survival in older (>60) patients\n  with mutant NPM1 and without FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 43\ndisease_id: 2\nsource_id: 189\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	59eabfd4-708c-4ae9-91b1-00c8a66a4b3a	2015-02-24 20:43:14.929769
462	283	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with higher complete remission rates, lower cumulative\n  incidence of relapse and higher overall survival in intermediate risk AML patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 182\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e2ee60a9-f66b-48eb-9601-a05b6dcfea63	2015-02-24 20:43:12.978491
463	284	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and both disease-free and overall survival\n  were longer for patients with Exon 12 NPM1 mutations.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 183\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e27faac7-6613-498a-8195-9616b2fd6375	2015-02-24 20:43:13.050057
464	285	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 120\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7bbcba92-dd95-47e9-9666-c055bdecc058	2015-02-24 20:43:13.111491
465	286	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved\n  overall and relapse-free survival, complete remission rates and remission duration\n  compared to all other combinations of mutants\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 184\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6e539239-0891-4d71-8d39-dd0d61238e6f	2015-02-24 20:43:13.181367
466	287	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with increased overall, event-free and relapse-free\n  survival and remission after induction therapy in the entire cohort and in the cytogenetically\n  normal subset of patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 185\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0a4e524e-ecdd-4cb2-8afd-30ee32f9340e	2015-02-24 20:43:13.251126
467	288	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Intermediate risk patients with mutant NPM1 had improved overall survival with\n  the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1\n  and IDH2\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 186\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	596a5bb4-78d5-4730-9f91-c50965c1f0e2	2015-02-24 20:43:13.317344
468	289	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was significantly associated with complete remission\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 94\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	98b75d7d-5f53-43f1-b460-504c77a16a0d	2015-02-24 20:43:13.37552
469	290	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 101\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5aa85ed8-a53f-4579-bf68-b9136c2a71e4	2015-02-24 20:43:13.447897
470	291	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 136\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1be29f4f-5399-4868-8f49-8fd18cd1deb2	2015-02-24 20:43:13.510055
471	292	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid\n  leukemia.\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 179\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a45925ce-c2f6-4f8e-abff-1f49eb7766ce	2015-02-24 20:43:13.576134
472	293	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse\n  in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction\n  in NPM1 levels following first line therapy (N=15)\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 187\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	577d2252-9d71-437c-b57d-f660b7cd8f09	2015-02-24 20:43:13.644103
473	294	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coocurrence of FLT3-ITD mutations significantly reduced overall survival of\n  normal karyotype AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 183\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4ad69423-9894-4128-a4f1-edf747b86f23	2015-02-24 20:43:13.711971
474	295	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 121\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	92a6d6e8-5788-4df9-ae78-af9dce4d41bf	2015-02-24 20:43:13.783891
475	296	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 121\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1dfc0285-0c91-453d-a883-e311e1c9cb76	2015-02-24 20:43:13.843694
476	297	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TET2 mutation reduces overall survival of normal karyotype patients with either\n  an NPM1 mutation or NPM1 mutation without FLT3-ITD\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 188\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3d99d551-2e8a-43e8-83f8-9ba40c199238	2015-02-24 20:43:13.911041
477	298	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival\n  and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate\n  risk AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 182\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9f0938c2-e4b7-4f6c-afd2-0013a9e2f19f	2015-02-24 20:43:13.969227
478	299	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype\n  (0/55).\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 183\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	aee64082-3aab-4267-aae8-37cef68a28e0	2015-02-24 20:43:14.032363
479	300	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No NPM1 mutations were identified in patients with favorable risk cytogenetics\n  (79/215 patients)\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 103\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c0726914-f198-4b2d-b7e2-fc716fcdab04	2015-02-24 20:43:14.13858
480	301	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid\n  leukemia.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 179\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8eafdd8f-2c66-41ff-affe-f061a10653fc	2015-02-24 20:43:14.216522
481	302	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid\n  leukemia\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 183\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2a329233-4e2f-46e5-8e2d-9dd0c3bf37d3	2015-02-24 20:43:14.279951
482	303	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with normal karyotype in older (>60) patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 189\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4eaba82a-e83c-47da-9c58-b5b3bb2962ef	2015-02-24 20:43:14.343912
483	304	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 122\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7a38bc52-e384-4625-8fbe-eaeb21284258	2015-02-24 20:43:14.416781
484	305	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: '"AML with mutated NPM1" is a provisional entity in WHO classification of acute\n  myeloid leukemia (AML). This mutation should be tested for in clinical trials and\n  is recommended for testing in patients with cytogentically normal AML.'\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 2\nsource_id: 190\nvariant_id: 85\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	91c7c4c8-972e-49b3-b69a-bdf7e147c233	2015-02-24 20:43:14.474742
485	306	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with intermediate risk cytogenetics (including\n  normal karyotype)\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 103\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8b541992-bcb3-4e03-82d7-d28dd397ea4b	2015-02-24 20:43:14.537848
486	307	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  NPM1 mutation regardless of FLT3-ITD status\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 43\ndisease_id: 2\nsource_id: 128\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7df6e8b2-7175-4317-b31d-35fd04cf6b73	2015-02-24 20:43:14.604215
487	308	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and\n  increased overall survival in young adult acute myeloid leukemia patients (median\n  age 43)\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 122\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4c6ab7a6-2c2e-4ad5-ae6b-ad378cfe3af9	2015-02-24 20:43:14.676138
488	309	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and\n  FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 56\ndisease_id: 2\nsource_id: 191\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	15d2e2bd-c8ab-4499-ba80-31dd708cd3df	2015-02-24 20:43:14.75039
489	310	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and\n  FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 43\ndisease_id: 2\nsource_id: 191\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f9f079f8-c86c-4db5-b020-cf482a9e5956	2015-02-24 20:43:14.804337
490	311	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations were associated with improved complete remission\n  rates as well as overall and disease free survival when all ages and karyotype were\n  analyzed; however, when only normal karyotype patients were analyzed, only complete\n  remission rates were improved\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 189\nvariant_id: 85\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ea20dc9f-8b72-46c2-b856-8aef18a1f678	2015-02-24 20:43:14.87288
492	313	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young patients (18-60) with NPM1 mutations had improved overall survival following\n  high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However,\n  the adjusted p-value for multiple testing is 0.11.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 2\nsource_id: 186\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7020f0ad-acb8-46a1-9b3a-c14e0710f8ed	2015-02-24 20:43:14.994627
493	314	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: For patients with NPM1 mutation that achieved CR following induction therapy,\n  relapse-free survival was improved following treatment with valproic acid\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 57\ndisease_id: 2\nsource_id: 192\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e3ccd171-6333-4666-b258-fea9fc76ff9b	2015-02-24 20:43:15.057237
494	315	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 expression was significantly increased in patients with NPM1 mutation with\n  or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 2\nsource_id: 193\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	76dbfbea-7140-4cd4-aee7-ebdd0b3867b1	2015-02-24 20:43:15.11474
495	316	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 and CD123 expression was significantly increased in patients with NPM1\n  mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33\n  and anti-CD123 therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 2\nsource_id: 193\nvariant_id: 85\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	75526cf7-5ce2-4460-b3a1-a9a8f074f01a	2015-02-24 20:43:15.180199
496	317	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 56\ndisease_id: 2\nsource_id: 191\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b4e66dcc-8d5d-4731-9432-024e81b638ec	2015-02-24 20:43:15.246729
497	318	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected\n  into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM\n  in AML patients was associated with good response to induction therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 2\nsource_id: 194\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a6b7e4e0-6999-45a6-ad65-9084f040c683	2015-02-24 20:43:15.317578
498	319	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of NRAS mutation in AML patients does not impact diease prognosis\n  (resistant disease, disease-free survival, complete remission rate, relapse rate,\n  induction death)\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 195\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3e7e478f-85aa-4245-b8d9-b827b9537f82	2015-02-24 20:43:15.394339
499	320	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NRAS mutation status does not impact overall, event-free or disease-free survival\n  in patients with AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 196\nvariant_id: 91\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c14e37ce-aa32-4955-ae31-bb6a04e93cbf	2015-02-24 20:43:15.471956
500	321	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 7\nsource_id: 197\nvariant_id: 87\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2e7c6cc3-1ad5-4cf7-8623-54e5b48c11c9	2015-02-24 20:43:15.533984
501	322	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 9\nsource_id: 198\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	41b926c9-7593-4c35-95b7-87e4e63faa6f	2015-02-24 20:43:15.595291
502	323	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 7\nsource_id: 197\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2362d9da-a5e8-43d8-870a-be3a9daea531	2015-02-24 20:43:15.706668
503	324	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 9\nsource_id: 198\nvariant_id: 89\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	78f6c84f-11ec-4395-9a45-a9d8832428d3	2015-02-24 20:43:15.768642
504	325	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 9\nsource_id: 198\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b738059d-9c06-49b9-bd75-088f5ab8a707	2015-02-24 20:43:15.83769
505	326	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma associated with NRAS Q61 mutation was more often associated with those\n  at the extremity than those at the trunk\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 7\nsource_id: 199\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	cfdcee49-0991-45fc-a021-99798041c01a	2015-02-24 20:43:15.905626
506	327	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status\n  (primarily Q61) was not informative for disease control rate, progression free survival\n  or overall survival following cetuximab plus chemotherapy.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 15\ndisease_id: 9\nsource_id: 72\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fbce2b23-a9f6-4510-8ed8-503270466316	2015-02-24 20:43:15.977556
507	328	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation\n  (primarily Q61) have a significantly lower response rate to cetuximab than patients\n  wildtype for NRAS.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 15\ndisease_id: 9\nsource_id: 72\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b101de5d-a82f-4e8f-b04c-a239b54c9837	2015-02-24 20:43:16.051314
508	329	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 22\ndisease_id: 7\nsource_id: 200\nvariant_id: 93\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d8bd1e25-6f02-4a57-9979-ea428748e3bc	2015-02-24 20:43:16.117787
509	330	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Likely due to increased reliance of mutant NRAS on HSP90 for stabilization,\n  inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic\n  malignant melanoma with an NRAS G13D mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 58\ndisease_id: 7\nsource_id: 201\nvariant_id: 92\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7addec09-da97-4b5a-b2c9-9770a4dceae0	2015-02-24 20:43:16.1879
510	331	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted\n  in disease free survival of 14 months\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 59\ndisease_id: 7\nsource_id: 202\nvariant_id: 94\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b8c083a3-8589-4361-bb27-45aec11d6ac0	2015-02-24 20:43:16.26255
511	332	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with Q61R mutation, treatment with temozolomide resulted\n  in overall survival of 16 months\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 59\ndisease_id: 7\nsource_id: 202\nvariant_id: 95\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b0296b63-cf0b-47ae-ba20-5293e392383e	2015-02-24 20:43:16.32274
512	333	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST tumors harboring PDGFRA D842V mutation are more likely to be benign than\n  malignant.\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 40\nsource_id: 203\nvariant_id: 97\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e29d2b89-e446-4193-a0ab-432500ba4ba2	2015-02-24 20:43:16.39071
513	334	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib,\n  double mutation of V561D and D842V mutants are resistant to imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 40\nsource_id: 157\nvariant_id: 97\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	92ad3877-4e38-4678-a4ae-df3c0c91e8a7	2015-02-24 20:43:16.462214
514	335	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST cancer with D842V mutation is resistant to imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 40\nsource_id: 204\nvariant_id: 97\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7abf374e-8d5b-4e4b-8ac8-4539e700d203	2015-02-24 20:43:16.530737
515	336	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 60\ndisease_id: 40\nsource_id: 205\nvariant_id: 96\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e9de3941-dd36-45d7-8dcd-5545f9a27abc	2015-02-24 20:43:16.600968
516	337	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 60\ndisease_id: 40\nsource_id: 205\nvariant_id: 97\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	32f6646c-a40c-4741-bc73-937771b27c06	2015-02-24 20:43:16.674747
517	338	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 60\ndisease_id: 40\nsource_id: 205\nvariant_id: 98\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ba17f23d-020a-454c-a996-388b91dbd83b	2015-02-24 20:43:16.746781
518	339	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib\n  resistance, crenolanib was significantly more potent at inhibiting kinase activity\n  than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 60\ndisease_id: 40\nsource_id: 205\nvariant_id: 99\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2c9ec6c9-d684-4b4c-9d9d-a1a08b24e535	2015-02-24 20:43:16.817549
519	340	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 60\ndisease_id: 40\nsource_id: 205\nvariant_id: 100\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	087854d4-9f92-402a-a7e0-a946d5e10fdb	2015-02-24 20:43:16.885178
520	341	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 51\ndisease_id: 9\nsource_id: 173\nvariant_id: 101\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4cdbebb3-f2db-468a-b357-13b82786a877	2015-02-24 20:43:16.957164
521	342	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 51\ndisease_id: 9\nsource_id: 173\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bf37c8ce-3771-46ea-b047-1903bfd0c47c	2015-02-24 20:43:17.016906
522	343	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations\n  (Exon 9) had no significant impact on response rate, disease control rate, progression\n  free surivival or overall survival following cetuximab plus chemotherapy than those\n  with wildtype PIK3CA\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 15\ndisease_id: 9\nsource_id: 72\nvariant_id: 104\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7c1ca477-6484-4c14-9d4d-2078775a1fbc	2015-02-24 20:43:17.086453
523	344	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 51\ndisease_id: 9\nsource_id: 173\nvariant_id: 105\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a5ed2940-81d9-4cd2-9516-dc0e359c56fa	2015-02-24 20:43:17.143912
524	345	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E542K) or exon 20 mutations were associated\n  with poorer survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 9\nsource_id: 206\nvariant_id: 101\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	be160789-7cbc-4603-81b5-a03fbb17e55b	2015-02-24 20:43:17.266762
525	346	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 9\nsource_id: 207\nvariant_id: 101\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	020dfd02-afbd-4ad4-8bfb-d71edeac90cf	2015-02-24 20:43:17.339772
526	347	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 9\nsource_id: 206\nvariant_id: 102\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	43bd35fd-4ea5-4bd6-84ac-4c10b2ff130b	2015-02-24 20:43:17.40341
527	348	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 9\nsource_id: 207\nvariant_id: 102\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8938b786-e41a-43eb-9f80-a0cbbfc3cb8a	2015-02-24 20:43:17.465761
528	349	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 9\nsource_id: 206\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a9c769ab-730e-43da-a348-d03116dd34e2	2015-02-24 20:43:17.534684
529	350	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 15\ndisease_id: 9\nsource_id: 206\nvariant_id: 101\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	09d35d13-c553-4e9a-89f0-ea7ec6b0c237	2015-02-24 20:43:17.612066
530	351	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 9\nsource_id: 206\nvariant_id: 101\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	afdcd2a4-af6c-4b3c-8f10-58e665e6683d	2015-02-24 20:43:17.66996
531	352	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 15\ndisease_id: 9\nsource_id: 206\nvariant_id: 102\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	06077efc-713c-4c73-8d72-ee1f6599ee46	2015-02-24 20:43:17.731517
532	353	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 9\nsource_id: 206\nvariant_id: 102\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e682b554-f424-47d9-99ba-cfaf16797d75	2015-02-24 20:43:17.789635
533	354	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA\n  mutations had lower response and disease control rates as well as shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype PIK3CA\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 15\ndisease_id: 9\nsource_id: 72\nvariant_id: 103\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6c0fa67c-9152-451a-824f-ef0444d61b5e	2015-02-24 20:43:17.853657
534	355	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 15\ndisease_id: 9\nsource_id: 206\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4fd485ac-587e-446f-98a4-142eef3adfdb	2015-02-24 20:43:17.916097
535	356	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 9\nsource_id: 206\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f2555606-10c5-414a-a1b6-33aad16cb908	2015-02-24 20:43:17.9755
536	357	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 61\ndisease_id: 11\nsource_id: 208\nvariant_id: 101\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ddbfc944-3d1d-4629-a77c-fe6a99c6a5e1	2015-02-24 20:43:18.040309
537	358	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 62\ndisease_id: 11\nsource_id: 209\nvariant_id: 101\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	e2c9b431-f0e0-4ddb-98be-2407d3f2ef8d	2015-02-24 20:43:18.112844
538	359	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 61\ndisease_id: 11\nsource_id: 208\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ca9a793c-5427-41d4-abae-7d1c08d4b996	2015-02-24 20:43:18.170214
539	360	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 62\ndisease_id: 11\nsource_id: 209\nvariant_id: 102\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3ccc2829-d2cd-48b2-8672-b28193909cb7	2015-02-24 20:43:18.228825
540	361	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with H1047R mutation showed increased sensitivity to\n  CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 61\ndisease_id: 11\nsource_id: 208\nvariant_id: 105\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0dd8b023-709d-4a72-adc3-d49c7a946021	2015-02-24 20:43:18.298766
541	362	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 62\ndisease_id: 11\nsource_id: 209\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3469131e-1eb0-472d-b3c6-3f72b0707790	2015-02-24 20:43:18.360176
542	363	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic\n  cells at pharmalogical concentrations.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 43\ndisease_id: 41\nsource_id: 210\nvariant_id: 106\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	da06fce0-12b5-4a53-8b66-14d7f25b92ab	2015-02-24 20:43:18.422706
543	364	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 20\nsource_id: 98\nvariant_id: 107\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	5ae5adcb-d634-4827-bff0-5c0d71610f1c	2015-02-24 20:43:18.490417
544	365	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PTEN nonsense mutations, including R233*, have been shown to be inactivating\n  and loss-of-function, but do not have prognostic value in glioblastoma multiforme\n  patients.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 39\nsource_id: 211\nvariant_id: 108\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	acd288d9-04d4-49c4-aee9-dc719873d36b	2015-02-24 20:43:18.557701
545	366	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse\n  to PI3K-mTOR inhibitors.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 3\nsource_id: 212\nvariant_id: 108\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f051799b-6f21-46d3-8f84-f9974695f722	2015-02-24 20:43:18.624934
546	367	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic\n  cells at pharmalogical concentrations.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 43\ndisease_id: 41\nsource_id: 210\nvariant_id: 109\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	ad82b20e-aae7-4c4e-ae1b-a33a554f101f	2015-02-24 20:43:18.744025
547	368	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Compared to those who harbor a wild type RET, patients with RET M918T mutation\n  develop larger and more aggressive medullary thyroid cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 42\nsource_id: 213\nvariant_id: 111\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d79d07c3-dc75-414f-9c43-c839f1928268	2015-02-24 20:43:18.817283
548	369	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with medullary carcinoma, the presence of RET M918T mutation is\n  associated with increased probability of lymph node metastases.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 42\nsource_id: 214\nvariant_id: 111\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	26900806-000a-4c3f-9ddc-644110cc0647	2015-02-24 20:43:18.881174
549	370	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 63\ndisease_id: 42\nsource_id: 215\nvariant_id: 110\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	345e35a1-d9f5-4551-b1ea-9443364adda6	2015-02-24 20:43:18.946281
550	371	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 63\ndisease_id: 42\nsource_id: 215\nvariant_id: 111\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	153c9f5a-c204-44ae-a200-c1f784020bb7	2015-02-24 20:43:19.018926
551	372	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong\n  repression of tyroid cancer cell growth.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 64\ndisease_id: 42\nsource_id: 216\nvariant_id: 111\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	831712b6-59e4-4ff4-bf0b-b8228e3972b0	2015-02-24 20:43:19.089213
552	373	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 28\nsource_id: 217\nvariant_id: 112\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	7ef605ba-df51-4652-940d-abe0a3a50a6e	2015-02-24 20:43:19.157055
553	374	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 28\nsource_id: 217\nvariant_id: 113\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	99825667-d334-4a1c-90a4-92e3cdf853ab	2015-02-24 20:43:19.216681
554	375	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R175H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 11\nsource_id: 218\nvariant_id: 114\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	9e9ed2c3-7664-4ff2-8008-58b4c2ada0b1	2015-02-24 20:43:19.281273
555	376	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R248 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 11\nsource_id: 219\nvariant_id: 115\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c7b034c3-424b-45f3-841b-9a4edce767aa	2015-02-24 20:43:19.340743
556	377	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R248Q mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 11\nsource_id: 218\nvariant_id: 115\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	32499f89-9748-4218-823b-5e3c34b78ab2	2015-02-24 20:43:19.403465
557	378	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring R248W mutation, the prognosis is worse than\n  any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 11\nsource_id: 218\nvariant_id: 116\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a8bb9b92-d5aa-44ad-9090-6c549919d692	2015-02-24 20:43:19.474703
558	379	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved regions\n  such as R249 are prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 11\nsource_id: 219\nvariant_id: 117\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	c2604808-6b0a-46e6-92e2-64d51fb6910f	2015-02-24 20:43:19.532286
559	380	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved regions\n  such as R249 are prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 11\nsource_id: 219\nvariant_id: 118\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	93af3197-fe50-4afc-8d9d-54b7777874ca	2015-02-24 20:43:19.599995
560	381	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 11\nsource_id: 219\nvariant_id: 119\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2d869bd9-69cd-4c16-bbf8-fa38e4c576c1	2015-02-24 20:43:19.659075
561	382	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273C mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 11\nsource_id: 218\nvariant_id: 119\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	231aca69-50e9-4bd1-8c77-31fed7a5e3c1	2015-02-24 20:43:19.725238
562	383	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 11\nsource_id: 219\nvariant_id: 120\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f00fefff-12b8-4463-89a6-521c03487888	2015-02-24 20:43:19.788068
563	384	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 11\nsource_id: 218\nvariant_id: 120\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	61ffbb3f-937b-4d89-97ba-c927f557f27f	2015-02-24 20:43:19.860972
564	385	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumor with R175H mutation are more responsive to doxorubicin than breast\n  tumors with wild type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 11\nsource_id: 219\nvariant_id: 121\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	83022010-035d-460d-8805-fa86dccfef42	2015-02-24 20:43:19.928204
565	386	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin\n  than breast tumors with wild type TP53.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 65\ndisease_id: 11\nsource_id: 220\nvariant_id: 114\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	2dc43843-421a-478c-a208-e9894609391d	2015-02-24 20:43:19.998046
566	387	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A patient with metastatic bladder cancer that responded well to the mTOR inhibitor\n  everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del,\n  p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations\n  also had mild responses to everolimus.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 66\ndisease_id: 43\nsource_id: 221\nvariant_id: 122\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	55f1153b-c606-4e0e-b81e-82e0edf655a2	2015-02-24 20:43:20.06711
567	388	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice\n  with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit\n  increased overall survival, increased apoptosis, reduced proliferation and reduced\n  tumor burden compared to Tsc1 wildtype mice.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 62\ndisease_id: 5\nsource_id: 222\nvariant_id: 122\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	10789296-0466-48e1-b4bb-d06e52d06ad8	2015-02-24 20:43:20.193129
568	389	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 66\ndisease_id: 23\nsource_id: 223\nvariant_id: 123\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	0ab999de-e00a-445f-b03d-d73a42684448	2015-02-24 20:43:20.268594
569	390	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 66\ndisease_id: 23\nsource_id: 221\nvariant_id: 124\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	3c3f5919-0940-445c-9ad3-ce998e45fb96	2015-02-24 20:43:20.337385
570	391	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 2\nsource_id: 224\nvariant_id: 125\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8e528dd2-6909-4622-9a1a-41c654b569a3	2015-02-24 20:43:20.412809
571	392	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 28\nsource_id: 224\nvariant_id: 125\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	d9239c68-92dd-4294-ba79-e866dcaf6221	2015-02-24 20:43:20.479057
572	393	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 2\nsource_id: 224\nvariant_id: 126\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	17f8f266-5523-4c29-a21f-2b0d00c61ba1	2015-02-24 20:43:20.544887
573	394	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 28\nsource_id: 224\nvariant_id: 126\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	6376ee68-be72-4040-918c-2ffcda502a47	2015-02-24 20:43:20.613876
574	395	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R\n  is prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 28\nsource_id: 225\nvariant_id: 125\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8d7353b8-b39f-4b26-bc0f-023d91300946	2015-02-24 20:43:20.679384
575	396	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, the presence of U2AF mutations such as\n  S34Y/F are prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 28\nsource_id: 225\nvariant_id: 126\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fe910b2e-e004-4721-a8d8-338d87e4e5d2	2015-02-24 20:43:20.74382
576	397	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 2\nsource_id: 224\nvariant_id: 125\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f14d9a39-2671-418d-bb12-274e478626d0	2015-02-24 20:43:20.801754
577	398	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 2\nsource_id: 224\nvariant_id: 126\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	8ecba64b-5191-45c6-ab61-005ac04ee801	2015-02-24 20:43:20.866127
578	399	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 226\nvariant_id: 127\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	f20ec715-a2d0-4282-a57b-9c6a9727a9a4	2015-02-24 20:43:20.926227
579	400	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No differences in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 226\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fb01b80f-7e4d-4636-9ad2-7a05dea7f8f0	2015-02-24 20:43:20.98476
580	401	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML.\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 226\nvariant_id: 128\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	21f303b6-58d6-424d-8f51-f4ccc32b8955	2015-02-24 20:43:21.051307
581	402	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No differences in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: N / A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 226\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b511f206-e761-46a7-81a6-4f504d412af3	2015-02-24 20:43:21.12071
582	403	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 227\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4d15e70a-bf8c-41ce-b601-7cc1f0537ccd	2015-02-24 20:43:21.178204
583	404	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 227\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	01e67079-d052-4483-a4b9-2bdf38e85dc4	2015-02-24 20:43:21.244003
584	405	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 228\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	b3226288-cde5-4b97-bb92-fbc5e5400a3f	2015-02-24 20:43:21.305702
585	406	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 226\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4d6621e9-0901-46e6-98ae-e4f4b3d3241d	2015-02-24 20:43:21.36887
586	407	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutations in WT1 were associated with a worse overall prognosis than patients\n  wildtype for WT1 in young patients (15-50), primarily because of increased risk\n  of disease recurrence\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 229\nvariant_id: 127\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	975ce6ed-8714-4039-b554-b65432ab75a3	2015-02-24 20:43:21.433249
587	408	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutations  in WT1 were associated with increased risk of recurrence in young\n  patients (15-50) with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 229\nvariant_id: 127\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	93f7fa58-8d1d-4be6-a7ee-c4d866c394f8	2015-02-24 20:43:21.499025
588	409	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 227\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	76335a2b-9e99-43f5-83d9-58f5763a0049	2015-02-24 20:43:21.609041
589	410	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 227\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	a1feb7d2-d8ad-4890-b621-03c5799b4de1	2015-02-24 20:43:21.678432
590	411	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 228\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	4cb5f95a-a335-41d5-9877-05506afa9692	2015-02-24 20:43:21.73773
591	412	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 226\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	bc4dac9a-ba59-41fc-852c-f3fbb19249dd	2015-02-24 20:43:21.803762
592	413	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 227\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	1c201f5b-155f-4b34-8732-029fd9f156d5	2015-02-24 20:43:21.876389
593	414	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 2\nsource_id: 227\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\nvariant_hgvs: N/A\n	1	\N	\N	fa9757a5-7424-4d41-80e8-e235d5ea1683	2015-02-24 20:43:21.937177
594	415	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The combination of PARP inhibition (Olaparib) with BRCA mutation results in\n  irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation positive\n  women with high grade serous or undifferentiated ovarian cancer.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 67\ndisease_id: 44\nsource_id: 230\nvariant_id: 129\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 2\nvariant_hgvs: N/A\n	1	\N	\N	6a292d60-b0cc-4eeb-9f19-2231eac08fd6	2015-02-24 20:43:22.012269
595	416	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The combination of PARP inhibition (Olaparib) with BRCA mutation results in\n  irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation positive\n  women with high grade serous or undifferentiated ovarian cancer.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 67\ndisease_id: 44\nsource_id: 230\nvariant_id: 130\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 2\nvariant_hgvs: N/A\n	1	\N	\N	4379bf1f-b651-4311-a9a9-6f24c7710644	2015-02-24 20:43:22.093374
\.


--
-- Name: audits_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('audits_id_seq', 595, true);


--
-- Data for Name: authorizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY authorizations (id, user_id, provider, uid, created_at, updated_at) FROM stdin;
\.


--
-- Name: authorizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('authorizations_id_seq', 1, false);


--
-- Data for Name: comments; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY comments (id, title, comment, commentable_id, commentable_type, user_id, role, created_at, updated_at) FROM stdin;
\.


--
-- Name: comments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('comments_id_seq', 1, false);


--
-- Data for Name: data_versions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY data_versions (id, version) FROM stdin;
\.


--
-- Name: data_versions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('data_versions_id_seq', 1, false);


--
-- Data for Name: definitions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY definitions (id, term, text, created_at, updated_at) FROM stdin;
\.


--
-- Name: definitions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('definitions_id_seq', 1, false);


--
-- Data for Name: delayed_jobs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY delayed_jobs (id, priority, attempts, handler, last_error, run_at, locked_at, failed_at, locked_by, queue, created_at, updated_at) FROM stdin;
\.


--
-- Name: delayed_jobs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('delayed_jobs_id_seq', 1, false);


--
-- Data for Name: diseases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY diseases (id, doid, name, created_at, updated_at) FROM stdin;
3	1612	Breast Cancer	2015-02-24 20:42:53.700244	2015-02-24 20:42:53.700244
4	769	Neuroblastoma	2015-02-24 20:42:53.769559	2015-02-24 20:42:53.769559
7	1909	Melanoma	2015-02-24 20:42:54.975279	2015-02-24 20:42:54.975279
22	4947	Cholangiocarcinoma	2015-02-24 20:43:04.915927	2015-02-24 20:43:04.915927
25	3069	Astrocytoma	2015-02-24 20:43:07.356689	2015-02-24 20:43:07.356689
38	1240	Leukemia	2015-02-24 20:43:11.176487	2015-02-24 20:43:11.176487
1	8552	Chronic Myeloid Leukemia	2015-02-24 20:42:52.783889	2015-02-24 20:45:43.499936
2	9119	Acute Myeloid Leukemia	2015-02-24 20:42:53.059618	2015-02-24 20:45:43.677668
5	3908	Non-small Cell Lung Carcinoma	2015-02-24 20:42:53.841859	2015-02-24 20:45:44.58034
6	1115	Sarcoma	2015-02-24 20:42:53.926078	2015-02-24 20:45:44.811624
8	1781	Thyroid Cancer	2015-02-24 20:42:55.119229	2015-02-24 20:45:45.334149
9	9256	Colorectal Cancer	2015-02-24 20:42:55.203171	2015-02-24 20:45:45.537552
10	9256	Colorectal Cancer	2015-02-24 20:42:56.096908	2015-02-24 20:45:45.731564
11	1612	Breast Cancer	2015-02-24 20:42:58.041692	2015-02-24 20:45:46.296193
12	8618	Oral Cavity Cancer	2015-02-24 20:42:58.113919	2015-02-24 20:45:46.609182
13	0050746	Mantle Cell Lymphoma	2015-02-24 20:42:58.190343	2015-02-24 20:45:46.765735
14	10534	Stomach Cancer	2015-02-24 20:42:58.358213	2015-02-24 20:45:47.189514
15	715	T-cell Leukemia	2015-02-24 20:42:58.693494	2015-02-24 20:45:47.365694
16	5635	Gastric Adenosquamous Carcinoma	2015-02-24 20:42:58.83852	2015-02-24 20:45:47.580349
17	1324	Lung Cancer	2015-02-24 20:42:58.977297	2015-02-24 20:45:47.861615
18	0060075	Estrogen-receptor Positive Breast Cancer	2015-02-24 20:42:59.048239	2015-02-24 20:45:48.010821
19	9952	Acute Lymphocytic Leukemia	2015-02-24 20:42:59.905867	2015-02-24 20:45:48.27589
20	5015	Hepatocellular Fibrolamellar Carcinoma	2015-02-24 20:43:00.028592	2015-02-24 20:45:48.514511
21	3908	Non-small Cell Lung Carcinoma	2015-02-24 20:43:02.475755	2015-02-24 20:45:48.739391
23	4007	Bladder Carcinoma	2015-02-24 20:43:04.98961	2015-02-24 20:45:49.131889
24	3909	Bronchogenic Lung Adenocarcinoma	2015-02-24 20:43:07.226972	2015-02-24 20:45:49.269021
26	3068	Glioblastoma Multiforme	2015-02-24 20:43:07.449937	2015-02-24 20:45:49.775438
27	3069	Astrocytoma	2015-02-24 20:43:07.923206	2015-02-24 20:45:50.114265
28	0050908	Myelodysplastic Syndrome	2015-02-24 20:43:08.323229	2015-02-24 20:45:50.281916
29	10747	Lymphoid Leukemia	2015-02-24 20:43:08.633648	2015-02-24 20:45:50.45176
30	4960	Bone Marrow Cancer	2015-02-24 20:43:08.812235	2015-02-24 20:45:50.688634
31	1240	Leukemia	2015-02-24 20:43:08.876944	2015-02-24 20:45:50.99355
32	8997	Polycythemia Vera	2015-02-24 20:43:08.943731	2015-02-24 20:45:51.147822
33	9253	Gastrointestinal Stromal Tumor	2015-02-24 20:43:09.200617	2015-02-24 20:45:51.307808
34	9119	Acute Myeloid Leukemia	2015-02-24 20:43:09.574401	2015-02-24 20:45:51.505421
35	14145	Malignant Anus Melanoma	2015-02-24 20:43:10.195856	2015-02-24 20:45:51.660139
36	9538	Multiple Myeloma	2015-02-24 20:43:10.768561	2015-02-24 20:45:51.884542
37	9119	Acute Myeloid Leukemia	2015-02-24 20:43:10.947514	2015-02-24 20:45:52.04954
39	3068	Glioblastoma Multiforme	2015-02-24 20:43:12.464521	2015-02-24 20:45:53.037934
40	9253	Gastrointestinal Stromal Tumor	2015-02-24 20:43:16.371184	2015-02-24 20:45:53.202657
41	9119	Acute Myeloid Leukemia	2015-02-24 20:43:18.403813	2015-02-24 20:45:53.370344
42	3973	Thyroid Medullary Carcinoma	2015-02-24 20:43:18.793345	2015-02-24 20:45:53.54523
43	6477	Invasive Bladder Transitional Cell Carcinoma	2015-02-24 20:43:20.050711	2015-02-24 20:45:53.852072
44	2394	Ovarian Cancer	2015-02-24 20:43:21.992634	2015-02-24 20:45:54.18785
\.


--
-- Name: diseases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('diseases_id_seq', 44, true);


--
-- Data for Name: drugs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs (id, name, pubchem_id, created_at, updated_at) FROM stdin;
1	Imatinib	\N	2015-02-24 20:42:52.776215	2015-02-24 20:42:52.776215
2	N/A	\N	2015-02-24 20:42:52.885401	2015-02-24 20:42:52.885401
3	Arsenic Trioxide	\N	2015-02-24 20:42:53.404139	2015-02-24 20:42:53.404139
4	Nilotinib	\N	2015-02-24 20:42:53.532498	2015-02-24 20:42:53.532498
5	Dasatinib	\N	2015-02-24 20:42:53.62391	2015-02-24 20:42:53.62391
6	MK-2206	\N	2015-02-24 20:42:53.696316	2015-02-24 20:42:53.696316
7	TAE684	\N	2015-02-24 20:42:53.765117	2015-02-24 20:42:53.765117
8	Crizotinib	\N	2015-02-24 20:42:53.837561	2015-02-24 20:42:53.837561
9	CH5424802	\N	2015-02-24 20:42:54.328427	2015-02-24 20:42:54.328427
10	Sorafenib	\N	2015-02-24 20:42:54.714682	2015-02-24 20:42:54.714682
11	Trametinib	\N	2015-02-24 20:42:54.790625	2015-02-24 20:42:54.790625
12	Dacarbazine	\N	2015-02-24 20:42:54.970383	2015-02-24 20:42:54.970383
13	Temozolomide	\N	2015-02-24 20:42:55.044684	2015-02-24 20:42:55.044684
14	Panitumumab	\N	2015-02-24 20:42:55.96976	2015-02-24 20:42:55.96976
15	Cetuximab	\N	2015-02-24 20:42:56.092461	2015-02-24 20:42:56.092461
16	Dabrafenib	\N	2015-02-24 20:42:56.445893	2015-02-24 20:42:56.445893
17	AZD6244	\N	2015-02-24 20:42:56.517631	2015-02-24 20:42:56.517631
18	PD0325901	\N	2015-02-24 20:42:56.845274	2015-02-24 20:42:56.845274
19	PLX4720	\N	2015-02-24 20:42:56.907152	2015-02-24 20:42:56.907152
20	Nutlin-3	\N	2015-02-24 20:42:57.042916	2015-02-24 20:42:57.042916
21	Capecitabine	\N	2015-02-24 20:42:57.138376	2015-02-24 20:42:57.138376
22	Vemurafenib	\N	2015-02-24 20:42:57.221081	2015-02-24 20:42:57.221081
23	Bevacizumab	\N	2015-02-24 20:42:57.316455	2015-02-24 20:42:57.316455
24	Palbociclib (PD-0332991)	\N	2015-02-24 20:42:58.688343	2015-02-24 20:42:58.688343
25	Palbociclib	\N	2015-02-24 20:42:59.044545	2015-02-24 20:42:59.044545
26	BYL719	\N	2015-02-24 20:42:59.118185	2015-02-24 20:42:59.118185
27	GW-2580	\N	2015-02-24 20:42:59.959308	2015-02-24 20:42:59.959308
28	Daunorubicin	\N	2015-02-24 20:43:01.996186	2015-02-24 20:43:01.996186
29	Idarubicin	\N	2015-02-24 20:43:02.062032	2015-02-24 20:43:02.062032
30	Gefitinib	\N	2015-02-24 20:43:02.204547	2015-02-24 20:43:02.204547
31	Erlotinib	\N	2015-02-24 20:43:02.259745	2015-02-24 20:43:02.259745
32	Premetrexed	\N	2015-02-24 20:43:02.81327	2015-02-24 20:43:02.81327
33	Stauroporine	\N	2015-02-24 20:43:02.906347	2015-02-24 20:43:02.906347
34	Lapatinib	\N	2015-02-24 20:43:02.980254	2015-02-24 20:43:02.980254
35	Neratinib	\N	2015-02-24 20:43:03.048409	2015-02-24 20:43:03.048409
36	Tamoxifen	\N	2015-02-24 20:43:03.856844	2015-02-24 20:43:03.856844
37	Fulvestrant	\N	2015-02-24 20:43:03.937633	2015-02-24 20:43:03.937633
38	Hormone Therapy	\N	2015-02-24 20:43:04.015879	2015-02-24 20:43:04.015879
39	Ponatinib	\N	2015-02-24 20:43:04.911925	2015-02-24 20:43:04.911925
40	PD173074	\N	2015-02-24 20:43:04.986036	2015-02-24 20:43:04.986036
41	Pazopanib	\N	2015-02-24 20:43:05.066387	2015-02-24 20:43:05.066387
42	SU5614	\N	2015-02-24 20:43:06.336	2015-02-24 20:43:06.336
43	ATRA	\N	2015-02-24 20:43:06.440247	2015-02-24 20:43:06.440247
44	CEP701	\N	2015-02-24 20:43:07.024202	2015-02-24 20:43:07.024202
45	AG1296	\N	2015-02-24 20:43:07.08775	2015-02-24 20:43:07.08775
46	Bortezomib	\N	2015-02-24 20:43:07.223177	2015-02-24 20:43:07.223177
47	Fasudil	\N	2015-02-24 20:43:07.286722	2015-02-24 20:43:07.286722
48	TG101348	\N	2015-02-24 20:43:08.872075	2015-02-24 20:43:08.872075
49	Pegylated IFN-α–2a	\N	2015-02-24 20:43:08.940252	2015-02-24 20:43:08.940252
50	SU11248	\N	2015-02-24 20:43:10.511509	2015-02-24 20:43:10.511509
51	Regorafenib	\N	2015-02-24 20:43:11.877629	2015-02-24 20:43:11.877629
52	NVP-BEZ235	\N	2015-02-24 20:43:11.948821	2015-02-24 20:43:11.948821
53	ARRY-142886	\N	2015-02-24 20:43:12.013489	2015-02-24 20:43:12.013489
54	O(6)-benzylguanine	\N	2015-02-24 20:43:12.633661	2015-02-24 20:43:12.633661
55	Carmustine	\N	2015-02-24 20:43:12.704238	2015-02-24 20:43:12.704238
56	NSC348884	\N	2015-02-24 20:43:14.726713	2015-02-24 20:43:14.726713
57	Valproic Acid	\N	2015-02-24 20:43:15.037583	2015-02-24 20:43:15.037583
58	17-AAG	\N	2015-02-24 20:43:16.164903	2015-02-24 20:43:16.164903
59	Temozolomide 	\N	2015-02-24 20:43:16.238089	2015-02-24 20:43:16.238089
60	Crenolanib	\N	2015-02-24 20:43:16.580537	2015-02-24 20:43:16.580537
61	CH5132799	\N	2015-02-24 20:43:18.020983	2015-02-24 20:43:18.020983
62	Rapamycin	\N	2015-02-24 20:43:18.090443	2015-02-24 20:43:18.090443
63	Motesanib	\N	2015-02-24 20:43:18.927815	2015-02-24 20:43:18.927815
64	AZD1480	\N	2015-02-24 20:43:19.067132	2015-02-24 20:43:19.067132
65	Doxorubicin	\N	2015-02-24 20:43:19.97353	2015-02-24 20:43:19.97353
66	Everolimus	\N	2015-02-24 20:43:20.046974	2015-02-24 20:43:20.046974
67	Olaparib	\N	2015-02-24 20:43:21.987916	2015-02-24 20:43:21.987916
\.


--
-- Name: drugs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('drugs_id_seq', 67, true);


--
-- Data for Name: variant_origins; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_origins (id, origin) FROM stdin;
1	Somatic
2	Germline
\.


--
-- Data for Name: evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_items (id, text, clinical_significance, evidence_direction, evidence_type_id, evidence_level_id, drug_id, disease_id, source_id, variant_id, created_at, updated_at, rating, status, remote_errors, remote_ids, variant_origin_id, variant_hgvs) FROM stdin;
1	BCR-ABL inhibitors such as imatinib have lead to "significantly improved prognosis, response rate, overall survival, and patient outcome in CML patients compared to previous therapeutic regimens."	Better Outcome	Supports	1	1	1	1	48	1	2015-02-24 20:42:52.793989	2015-02-24 20:42:52.793989	5	accepted	\N	\N	1	N/A
2	Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML.	Positive	Does Not Support	2	2	2	1	49	1	2015-02-24 20:42:52.893462	2015-02-24 20:42:52.893462	2	accepted	\N	\N	1	N/A
3	The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease.	Positive	Supports	2	1	2	1	48	1	2015-02-24 20:42:52.98361	2015-02-24 20:42:52.98361	4	accepted	\N	\N	1	N/A
4	BCR-ABL fusions have been found to contribute to imatinib resistance in AML cell lines.	Resistance or Non-Response	Supports	3	3	1	2	50	1	2015-02-24 20:42:53.065666	2015-02-24 20:42:53.065666	3	accepted	\N	\N	1	N/A
5	COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	Resistance or Non-Response	Supports	3	2	1	1	51	3	2015-02-24 20:42:53.13534	2015-02-24 20:42:53.13534	4	accepted	\N	\N	1	N/A
6	In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have shown to be resistant to imatinib treatment.	Resistance or Non-Response	Supports	3	3	1	1	48	2	2015-02-24 20:42:53.235666	2015-02-24 20:42:53.235666	4	accepted	\N	\N	1	N/A
7	COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	Resistance or Non-Response	Supports	3	2	1	1	51	2	2015-02-24 20:42:53.320901	2015-02-24 20:42:53.320901	4	accepted	\N	\N	1	N/A
8	Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide results in high rates of complete remission in patients.	Sensitivity	Supports	3	1	3	2	50	1	2015-02-24 20:42:53.407674	2015-02-24 20:42:53.407674	5	accepted	\N	\N	1	N/A
9	The clinical use of imatinib in patients with BCR-ABL fusion has resulted in drastic sensitivity to the drug.	Sensitivity	Supports	3	1	1	1	48	1	2015-02-24 20:42:53.470936	2015-02-24 20:42:53.470936	5	accepted	\N	\N	1	N/A
10	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.	Sensitivity	Supports	3	3	4	1	48	1	2015-02-24 20:42:53.535721	2015-02-24 20:42:53.535721	4	accepted	\N	\N	1	N/A
11	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.	Sensitivity	Supports	3	3	5	1	48	1	2015-02-24 20:42:53.628305	2015-02-24 20:42:53.628305	4	accepted	\N	\N	1	N/A
12	PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the AKT inhibitor MK-2206 in breast cancer cell lines.	Sensitivity	Does Not Support	3	3	6	3	52	4	2015-02-24 20:42:53.702542	2015-02-24 20:42:53.702542	3	accepted	\N	\N	1	N/A
13	The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.	Resistance or Non-Response	Supports	3	2	7	4	53	9	2015-02-24 20:42:53.771839	2015-02-24 20:42:53.771839	3	accepted	\N	\N	1	N/A
14	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y variant has been shown to confer resistance to crizotinib.	Resistance or Non-Response	Supports	3	3	8	5	54	6	2015-02-24 20:42:53.844307	2015-02-24 20:42:53.844307	4	accepted	\N	\N	1	N/A
15	Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without this mutation.	Resistance or Non-Response	Supports	3	2	8	6	55	8	2015-02-24 20:42:53.928794	2015-02-24 20:42:53.928794	3	accepted	\N	\N	1	N/A
16	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation.	Resistance or Non-Response	Supports	3	2	8	4	56	8	2015-02-24 20:42:53.997581	2015-02-24 20:42:53.997581	3	accepted	\N	\N	1	N/A
17	Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without this mutation.	Resistance or Non-Response	Supports	3	2	8	5	55	8	2015-02-24 20:42:54.065925	2015-02-24 20:42:54.065925	3	accepted	\N	\N	1	N/A
18	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M variant has been shown to confer resistance to crizotinib.	Resistance or Non-Response	Supports	3	3	8	5	54	7	2015-02-24 20:42:54.152661	2015-02-24 20:42:54.152661	4	accepted	\N	\N	1	N/A
19	In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment with crizotinib has shown to be effective in acheiving stable disease.	Sensitivity	Supports	3	3	8	5	54	5	2015-02-24 20:42:54.258381	2015-02-24 20:42:54.258381	4	accepted	\N	\N	1	N/A
20	CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.	Sensitivity	Supports	3	2	9	4	57	8	2015-02-24 20:42:54.333002	2015-02-24 20:42:54.333002	3	accepted	\N	\N	1	N/A
21	High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing the EML4-ALK fusion containing the F1174L mutation.	Sensitivity	Supports	3	2	8	4	58	8	2015-02-24 20:42:54.401618	2015-02-24 20:42:54.401618	3	accepted	\N	\N	1	N/A
22	SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition	Sensitivity	Supports	3	2	7	4	53	8	2015-02-24 20:42:54.46836	2015-02-24 20:42:54.46836	4	accepted	\N	\N	1	N/A
23	CH5424802 treatment resulted in significant tumor regression in xenograft models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation	Sensitivity	Supports	3	2	9	5	57	7	2015-02-24 20:42:54.5547	2015-02-24 20:42:54.5547	3	accepted	\N	\N	1	N/A
24	TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.	Sensitivity	Supports	3	2	7	4	53	9	2015-02-24 20:42:54.638256	2015-02-24 20:42:54.638256	3	accepted	\N	\N	1	N/A
25	In one patient with S214C mutation, the use of sorafenib has led to more than 5 years of survival and near remission.	Sensitivity	Supports	3	3	10	5	59	10	2015-02-24 20:42:54.720408	2015-02-24 20:42:54.720408	2	accepted	\N	\N	1	N/A
26	Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	Sensitivity	Supports	3	2	11	5	59	10	2015-02-24 20:42:54.794224	2015-02-24 20:42:54.794224	3	accepted	\N	\N	1	N/A
27	Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	Sensitivity	Supports	3	2	10	5	59	10	2015-02-24 20:42:54.894949	2015-02-24 20:42:54.894949	3	accepted	\N	\N	1	N/A
28	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	N / A	Does Not Support	3	3	12	7	60	12	2015-02-24 20:42:54.978672	2015-02-24 20:42:54.978672	2	accepted	\N	\N	1	N/A
29	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	N / A	Does Not Support	3	3	13	7	60	12	2015-02-24 20:42:55.047898	2015-02-24 20:42:55.047898	2	accepted	\N	\N	1	N/A
30	Unlike other studies that suggest a poorer outcome, BRAF mutation in this study was not correlated with poorer prognosis in papillary thyroid cancer.	Poor Outcome	Does Not Support	1	3	2	8	61	12	2015-02-24 20:42:55.121885	2015-02-24 20:42:55.121885	5	accepted	\N	\N	1	N/A
31	V600E is associated with adverse pathological features of colorectal cancer. This can be concluded as a marker of poor prognosis.	Poor Outcome	Supports	1	1	2	9	62	12	2015-02-24 20:42:55.20658	2015-02-24 20:42:55.20658	5	accepted	\N	\N	1	N/A
32	BRAF V600E is correlated with shorter disease-free and overall Survival in a Spanish cohort of melanoma patients.	Poor Outcome	Supports	1	3	2	7	63	12	2015-02-24 20:42:55.279542	2015-02-24 20:42:55.279542	3	accepted	\N	\N	1	N/A
33	V600E is correlated with disease recurrence in both age cohorts (>65 and <65 yo).	Poor Outcome	Supports	1	3	2	8	64	12	2015-02-24 20:42:55.351874	2015-02-24 20:42:55.351874	3	accepted	\N	\N	1	N/A
34	BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in a study of 187 patients with PTC and other thyroid diseases.	Poor Outcome	Supports	1	3	2	8	65	12	2015-02-24 20:42:55.422343	2015-02-24 20:42:55.422343	3	accepted	\N	\N	1	N/A
35	BRAF mutation correlated with poor prognosis in papillary thyroid cancer in both older (>65 yo) and younger (<65 yo) cohorts.	Poor Outcome	Supports	1	3	2	8	64	12	2015-02-24 20:42:55.524192	2015-02-24 20:42:55.524192	3	accepted	\N	\N	1	N/A
36	BRAF mutations are associated with melanoma arising in non-chronic sun damaged skin and with superficial spreading melanoma	Positive	Supports	2	3	2	7	66	17	2015-02-24 20:42:55.606478	2015-02-24 20:42:55.606478	4	accepted	\N	\N	1	N/A
37	Thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	Positive	Supports	2	3	2	8	67	12	2015-02-24 20:42:55.684808	2015-02-24 20:42:55.684808	5	accepted	\N	\N	1	N/A
38	Thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	Positive	Supports	2	3	2	8	68	12	2015-02-24 20:42:55.767775	2015-02-24 20:42:55.767775	5	accepted	\N	\N	1	N/A
39	BRAF V600E is shown to be associated with the tall-cell variant of papillary thyroid cancer	Positive	Supports	2	3	2	8	64	12	2015-02-24 20:42:55.835553	2015-02-24 20:42:55.835553	3	accepted	\N	\N	1	N/A
40	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP but does not show resistance to MEK inhibitors.	Resistance or Non-Response	Does Not Support	3	2	2	7	69	12	2015-02-24 20:42:55.909266	2015-02-24 20:42:55.909266	3	accepted	\N	\N	1	N/A
41	In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations were associated with poor progression free survival regardless of treatment (panitumumab with best supportive care or best supportive care alone)	Resistance or Non-Response	Supports	3	3	14	9	70	17	2015-02-24 20:42:55.977464	2015-02-24 20:42:55.977464	3	accepted	\N	\N	1	N/A
42	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	3	2	15	10	71	12	2015-02-24 20:42:56.099476	2015-02-24 20:42:56.099476	3	accepted	\N	\N	1	N/A
43	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	3	2	14	10	71	12	2015-02-24 20:42:56.162526	2015-02-24 20:42:56.162526	3	accepted	\N	\N	1	N/A
44	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	3	2	10	10	71	12	2015-02-24 20:42:56.220822	2015-02-24 20:42:56.220822	3	accepted	\N	\N	1	N/A
45	Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype BRAF	Resistance or Non-Response	Supports	3	3	15	9	72	12	2015-02-24 20:42:56.292177	2015-02-24 20:42:56.292177	3	accepted	\N	\N	1	N/A
46	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP and resistance to RAF inhibitors.	Resistance or Non-Response	Supports	3	2	2	7	69	12	2015-02-24 20:42:56.358742	2015-02-24 20:42:56.358742	3	accepted	\N	\N	1	N/A
47	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	Resistance or Non-Response	Supports	3	3	16	5	73	12	2015-02-24 20:42:56.451248	2015-02-24 20:42:56.451248	2	accepted	\N	\N	1	N/A
48	COLO201 and COLO206F cell harboring BRAF V600E mutations were cloned to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification of the BRAF V600E gene.	Resistance or Non-Response	Supports	3	4	17	9	74	14	2015-02-24 20:42:56.527186	2015-02-24 20:42:56.527186	4	accepted	\N	\N	1	N/A
49	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	Sensitivity	Supports	3	3	16	7	75	17	2015-02-24 20:42:56.601809	2015-02-24 20:42:56.601809	3	accepted	\N	\N	1	N/A
50	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	Sensitivity	Supports	3	3	11	7	75	17	2015-02-24 20:42:56.690422	2015-02-24 20:42:56.690422	3	accepted	\N	\N	1	N/A
51	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	3	3	16	7	76	11	2015-02-24 20:42:56.767827	2015-02-24 20:42:56.767827	5	accepted	\N	\N	1	N/A
52	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	Sensitivity	Supports	3	2	18	10	77	12	2015-02-24 20:42:56.849841	2015-02-24 20:42:56.849841	3	accepted	\N	\N	1	N/A
53	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	Sensitivity	Supports	3	2	19	10	77	12	2015-02-24 20:42:56.910223	2015-02-24 20:42:56.910223	3	accepted	\N	\N	1	N/A
54	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	Sensitivity	Supports	3	2	19	10	78	12	2015-02-24 20:42:56.979994	2015-02-24 20:42:56.979994	2	accepted	\N	\N	1	N/A
55	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	Sensitivity	Supports	3	2	20	10	78	12	2015-02-24 20:42:57.046907	2015-02-24 20:42:57.046907	2	accepted	\N	\N	1	N/A
56	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	3	2	21	9	79	12	2015-02-24 20:42:57.146923	2015-02-24 20:42:57.146923	2	accepted	\N	\N	1	N/A
88	CEBPA mutation was significantly associated with complete remission	Better Outcome	Supports	1	3	2	2	94	29	2015-02-24 20:42:59.479058	2015-02-24 20:42:59.479058	4	accepted	\N	\N	1	N/A
57	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	3	2	22	9	79	12	2015-02-24 20:42:57.225659	2015-02-24 20:42:57.225659	2	accepted	\N	\N	1	N/A
58	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	3	2	23	9	79	12	2015-02-24 20:42:57.319816	2015-02-24 20:42:57.319816	2	accepted	\N	\N	1	N/A
59	Cell lines expressing the BRAF V600E mutation responded better to vemurafenib treatment than cells wildtype for BRAF as measured by reduced cellular proliferation and inhibition of MET and ERK phosphorylation	Sensitivity	Supports	3	2	22	9	79	12	2015-02-24 20:42:57.3907	2015-02-24 20:42:57.3907	2	accepted	\N	\N	1	N/A
60	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	Sensitivity	Supports	3	3	16	7	80	12	2015-02-24 20:42:57.457136	2015-02-24 20:42:57.457136	5	accepted	\N	\N	1	N/A
61	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	Sensitivity	Supports	3	3	11	7	80	12	2015-02-24 20:42:57.517928	2015-02-24 20:42:57.517928	5	accepted	\N	\N	1	N/A
62	A single patient with BRAF V600E/V600M bi-allelic mutation responded to the V600E drug dabrafenib.	Sensitivity	Supports	3	3	16	7	81	13	2015-02-24 20:42:57.585335	2015-02-24 20:42:57.585335	3	accepted	\N	\N	1	N/A
63	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	3	3	16	7	76	15	2015-02-24 20:42:57.655423	2015-02-24 20:42:57.655423	5	accepted	\N	\N	1	N/A
64	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	3	3	16	7	76	16	2015-02-24 20:42:57.733139	2015-02-24 20:42:57.733139	5	accepted	\N	\N	1	N/A
65	Three studies have found no significant survival impact for CCND1 in lung cancer.	Poor Outcome	Does Not Support	1	3	2	5	82	19	2015-02-24 20:42:57.832282	2015-02-24 20:42:57.832282	3	accepted	\N	\N	1	N/A
66	Inceased copy number of CCND1 is associated with poorer overall survival.	Poor Outcome	Supports	1	3	2	5	82	18	2015-02-24 20:42:57.898162	2015-02-24 20:42:57.898162	3	accepted	\N	\N	1	N/A
67	Three studies have associated CCND1 expression with poorer survival.	Poor Outcome	Supports	1	3	2	5	82	19	2015-02-24 20:42:57.965974	2015-02-24 20:42:57.965974	3	accepted	\N	\N	1	N/A
68	In ER-positive breast cancer patients, Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis.	Poor Outcome	Supports	1	3	2	11	83	20	2015-02-24 20:42:58.045451	2015-02-24 20:42:58.045451	4	accepted	\N	\N	1	N/A
69	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in head and neck squamous cell carcinoma.	Poor Outcome	Supports	1	3	2	12	84	20	2015-02-24 20:42:58.116938	2015-02-24 20:42:58.116938	4	accepted	\N	\N	1	N/A
70	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in mantle cell lymphoma.	Poor Outcome	Supports	1	3	2	13	85	20	2015-02-24 20:42:58.193384	2015-02-24 20:42:58.193384	4	accepted	\N	\N	1	N/A
71	Cyclin D1 overexpression is associated with the ER-positive subtype of breast cancer.	Positive	Supports	2	3	2	11	83	20	2015-02-24 20:42:58.262312	2015-02-24 20:42:58.262312	4	accepted	\N	\N	1	N/A
72	CCND2 hypomethylation is seen to be more common in stage III and IV gastric tumors than stage I and II.	Negative	Supports	2	2	2	14	86	22	2015-02-24 20:42:58.360609	2015-02-24 20:42:58.360609	3	accepted	\N	\N	1	N/A
73	Cyclin D2 overexpression is associated with the presence pf lymph node metastasis in gastric cancers.	Poor Outcome	Supports	1	2	2	14	87	21	2015-02-24 20:42:58.431624	2015-02-24 20:42:58.431624	4	accepted	\N	\N	1	N/A
74	Cyclin D2 overexpression is associated with the increased depth of cancer invasion in gastric cancers.	Poor Outcome	Supports	1	2	2	14	87	21	2015-02-24 20:42:58.495599	2015-02-24 20:42:58.495599	4	accepted	\N	\N	1	N/A
75	Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.	Poor Outcome	Supports	1	2	2	14	87	21	2015-02-24 20:42:58.559156	2015-02-24 20:42:58.559156	4	accepted	\N	\N	1	N/A
76	Cyclin D2 overexpression is associated with vascular invasion by cancer cells in gastric cancers.	Poor Outcome	Supports	1	2	2	14	87	21	2015-02-24 20:42:58.624049	2015-02-24 20:42:58.624049	4	accepted	\N	\N	1	N/A
77	Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib (PD-0332991) significantly increased median survival.	Sensitivity	Supports	3	2	24	15	88	23	2015-02-24 20:42:58.695766	2015-02-24 20:42:58.695766	3	accepted	\N	\N	1	N/A
78	Patients with high expression of total cyclin E and low-molecular weight cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard ratio for these patients in 13.3 times higher than those with normal cyclin E levels.	Poor Outcome	Supports	1	3	2	11	89	24	2015-02-24 20:42:58.753844	2015-02-24 20:42:58.753844	4	accepted	\N	\N	1	N/A
79	Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced stage gastric carcinoma.	Poor Outcome	Supports	1	3	2	16	90	24	2015-02-24 20:42:58.840805	2015-02-24 20:42:58.840805	4	accepted	\N	\N	1	N/A
80	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	Poor Outcome	Supports	1	3	2	16	90	24	2015-02-24 20:42:58.91073	2015-02-24 20:42:58.91073	4	accepted	\N	\N	1	N/A
81	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	Poor Outcome	Supports	1	3	2	17	90	24	2015-02-24 20:42:58.980223	2015-02-24 20:42:58.980223	4	accepted	\N	\N	1	N/A
82	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	Better Outcome	Supports	1	3	25	18	91	25	2015-02-24 20:42:59.050954	2015-02-24 20:42:59.050954	3	accepted	\N	\N	1	N/A
83	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	Sensitivity	Supports	3	2	26	18	92	25	2015-02-24 20:42:59.12445	2015-02-24 20:42:59.12445	4	accepted	\N	\N	1	N/A
84	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	Better Outcome	Supports	1	3	25	18	91	26	2015-02-24 20:42:59.187375	2015-02-24 20:42:59.187375	3	accepted	\N	\N	1	N/A
85	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	Sensitivity	Supports	3	2	26	18	92	26	2015-02-24 20:42:59.250154	2015-02-24 20:42:59.250154	4	accepted	\N	\N	1	N/A
86	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter time to recurrence.	Poor Outcome	Supports	1	3	2	5	93	27	2015-02-24 20:42:59.342484	2015-02-24 20:42:59.342484	3	accepted	\N	\N	1	N/A
87	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter overall survival.	Poor Outcome	Supports	1	3	2	5	93	27	2015-02-24 20:42:59.409385	2015-02-24 20:42:59.409385	3	accepted	\N	\N	1	N/A
89	CEBPA mutation was associated with improved overall survival in older (>60) patients with AML	Better Outcome	Supports	1	3	2	2	95	29	2015-02-24 20:42:59.547629	2015-02-24 20:42:59.547629	4	accepted	\N	\N	1	N/A
90	Overall survival was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA mutations	Better Outcome	Supports	1	3	2	2	96	28	2015-02-24 20:42:59.617036	2015-02-24 20:42:59.617036	3	accepted	\N	\N	1	N/A
91	Remission duration was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA	Better Outcome	Supports	1	3	2	2	96	28	2015-02-24 20:42:59.681324	2015-02-24 20:42:59.681324	3	accepted	\N	\N	1	N/A
92	CEBPA mutation status had no impact on ATRA treatment response in older (>60) patients with AML	N / A	Does Not Support	3	3	2	2	95	29	2015-02-24 20:42:59.745412	2015-02-24 20:42:59.745412	4	accepted	\N	\N	1	N/A
93	FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically normal AML patients with CEBPA	N / A	Does Not Support	1	3	2	2	96	28	2015-02-24 20:42:59.835702	2015-02-24 20:42:59.835702	2	accepted	\N	\N	1	N/A
94	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	3	2	1	19	97	30	2015-02-24 20:42:59.909383	2015-02-24 20:42:59.909383	3	accepted	\N	\N	1	N/A
95	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	3	2	27	19	97	30	2015-02-24 20:42:59.9622	2015-02-24 20:42:59.9622	3	accepted	\N	\N	1	N/A
96	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	Positive	Supports	2	3	2	20	98	31	2015-02-24 20:43:00.031114	2015-02-24 20:43:00.031114	4	accepted	\N	\N	1	N/A
97	DNMT3A mutations were associated with achievement of complete remission in a large cohort of younger (18-60) AML patients	Better Outcome	Supports	1	3	2	2	99	32	2015-02-24 20:43:00.103982	2015-02-24 20:43:00.103982	3	accepted	\N	\N	1	N/A
98	There is no difference in the complete remission rate of de novo AML patients with DNMT3A mutation compared to those who are wild type for DNMT3A.	N / A	Does Not Support	1	3	2	2	100	32	2015-02-24 20:43:00.16922	2015-02-24 20:43:00.16922	4	accepted	\N	\N	1	N/A
99	In young AML patients (<60 years old), DNMT3A mutation status (60% of which were R882) was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N / A	Does Not Support	1	3	2	2	101	32	2015-02-24 20:43:00.238603	2015-02-24 20:43:00.238603	4	accepted	\N	\N	1	N/A
100	In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5% of which were R882) had no prognostic value on overall, relapse free and event free survival.	N / A	Does Not Support	1	3	2	2	99	32	2015-02-24 20:43:00.324803	2015-02-24 20:43:00.324803	3	accepted	\N	\N	1	N/A
101	In a large cohort of cytogenetically normal AML patients (18-60 years old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall, relapse free and event free survival	N / A	Does Not Support	1	3	2	2	99	32	2015-02-24 20:43:00.387215	2015-02-24 20:43:00.387215	3	accepted	\N	\N	1	N/A
102	Complete remission rates did not differ between patients with wildtype or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML	N / A	Does Not Support	1	3	2	2	102	32	2015-02-24 20:43:00.451629	2015-02-24 20:43:00.451629	4	accepted	\N	\N	1	N/A
103	Therapy-related AML was less common in patients with DNMT3A mutations (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of younger (18-60) AML patients	Negative	Supports	2	3	2	2	99	32	2015-02-24 20:43:00.511769	2015-02-24 20:43:00.511769	3	accepted	\N	\N	1	N/A
104	De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	Poor Outcome	Supports	1	3	2	2	103	32	2015-02-24 20:43:00.576836	2015-02-24 20:43:00.576836	4	accepted	\N	\N	1	N/A
105	AML patients with DNMT3A mutations (59% of which were R882) showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	Poor Outcome	Supports	1	3	2	2	103	32	2015-02-24 20:43:00.635247	2015-02-24 20:43:00.635247	5	accepted	\N	\N	1	N/A
106	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including R882) were associated with worse overall survival compared to those without DNMT3A mutation.	Poor Outcome	Supports	1	3	2	2	103	32	2015-02-24 20:43:00.72191	2015-02-24 20:43:00.72191	5	accepted	\N	\N	1	N/A
107	In cytogenetically normal AML patients, DNMT3A R882 mutations are associated with lower overall and disease free survival as compared to patients with wild type DNMT3A.	Poor Outcome	Supports	1	3	2	2	104	32	2015-02-24 20:43:00.781874	2015-02-24 20:43:00.781874	5	accepted	\N	\N	1	N/A
108	In young AML patients (<60 years old), DNMT3A mutations were associated with significantly reduced overall survival and relapse free survival in patients wildtype for NPM1 and FLT3	Poor Outcome	Supports	1	3	2	2	101	32	2015-02-24 20:43:00.842243	2015-02-24 20:43:00.842243	4	accepted	\N	\N	1	N/A
109	In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation is prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	Poor Outcome	Supports	1	3	2	2	102	32	2015-02-24 20:43:00.911575	2015-02-24 20:43:00.911575	4	accepted	\N	\N	1	N/A
110	In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations other than R882 are prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	Poor Outcome	Supports	1	3	2	2	102	32	2015-02-24 20:43:00.974443	2015-02-24 20:43:00.974443	4	accepted	\N	\N	1	N/A
111	DNMT3A R882 mutation was associated with reduced relapse free and overall survival in ELN-unfavorable, cytogenetically normal AMLs	Poor Outcome	Supports	1	3	2	2	99	32	2015-02-24 20:43:01.035742	2015-02-24 20:43:01.035742	3	accepted	\N	\N	1	N/A
112	In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations were associated with reduced relapse free survival in the entire cohort as well as the subset of patients with cytogenetically normal AML	Poor Outcome	Supports	1	1	2	2	99	32	2015-02-24 20:43:01.096786	2015-02-24 20:43:01.096786	3	accepted	\N	\N	1	N/A
113	DNMT3A mutations (62% of which were R882) were associated with reduced disease-free survival in patients with cytogenetically normal AML.	Poor Outcome	Supports	1	3	2	2	102	32	2015-02-24 20:43:01.239184	2015-02-24 20:43:01.239184	4	accepted	\N	\N	1	N/A
114	Young AML patients (<60 years old) with DNMT3A mutation have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	Poor Outcome	Supports	1	3	2	2	101	32	2015-02-24 20:43:01.301801	2015-02-24 20:43:01.301801	3	accepted	\N	\N	1	N/A
115	Complete remission rate was not different between young AML patients (<60 years old) with or without DNMT3A mutations (60% of which were R882)	Poor Outcome	Supports	1	3	2	2	101	32	2015-02-24 20:43:01.364344	2015-02-24 20:43:01.364344	3	accepted	\N	\N	1	N/A
116	Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	Poor Outcome	Supports	1	3	2	2	101	32	2015-02-24 20:43:01.42792	2015-02-24 20:43:01.42792	3	accepted	\N	\N	1	N/A
117	Young AML patients (<60 years old) with DNMT3A mutations (60% of which were R882) were older in age, had higher white blood cell counts and had higher platelet counts than patients wildtype for DNMT3A	Positive	Supports	2	3	2	2	101	32	2015-02-24 20:43:01.499923	2015-02-24 20:43:01.499923	3	accepted	\N	\N	1	N/A
118	DNMT3A R882 mutations occur most often in de novo AML patients with intermediate cytogenic risk.	Positive	Supports	2	3	2	2	100	32	2015-02-24 20:43:01.562566	2015-02-24 20:43:01.562566	3	accepted	\N	\N	1	N/A
119	DNMT3A R882 mutations were associated with older age, higher white blood cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically normal AML patients	Positive	Supports	2	3	2	2	104	32	2015-02-24 20:43:01.628773	2015-02-24 20:43:01.628773	3	accepted	\N	\N	1	N/A
120	DNMT3A mutations (59% of which were R882) were associated with an intermediate risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes	Positive	Supports	2	3	2	2	103	32	2015-02-24 20:43:01.702303	2015-02-24 20:43:01.702303	5	accepted	\N	\N	1	N/A
121	DNMT3A mutations (59% of which were R882) were associated with intermediate risk cytogenetics (including normal karyotype)	Positive	Supports	2	3	2	2	103	32	2015-02-24 20:43:01.790776	2015-02-24 20:43:01.790776	4	accepted	\N	\N	1	N/A
122	DNMT3A mutations (64.5% R882) were associated with older age, higher white blood cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML patients	Positive	Supports	2	3	2	2	99	32	2015-02-24 20:43:01.868708	2015-02-24 20:43:01.868708	3	accepted	\N	\N	1	N/A
123	DNMT3A R882 mutations were associated with cytogenetically normal AML in a large cohort of younger (18-60) AML patients	Positive	Supports	2	3	2	2	99	32	2015-02-24 20:43:01.936166	2015-02-24 20:43:01.936166	3	accepted	\N	\N	1	N/A
124	Daunorubicin treatment resulted in similar overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	Resistance or Non-Response	Does Not Support	3	3	28	2	100	32	2015-02-24 20:43:01.99909	2015-02-24 20:43:01.99909	4	accepted	\N	\N	1	N/A
125	Idarubicin increases the overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	Sensitivity	Supports	3	3	29	2	100	32	2015-02-24 20:43:02.064855	2015-02-24 20:43:02.064855	4	accepted	\N	\N	1	N/A
126	Median survival of patients with EGFR L858R mutation is better than those with wild type EGFR.	Better Outcome	Supports	1	3	2	5	105	33	2015-02-24 20:43:02.13847	2015-02-24 20:43:02.13847	3	accepted	\N	\N	1	N/A
127	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.	N / A	Does Not Support	3	3	30	5	106	33	2015-02-24 20:43:02.208919	2015-02-24 20:43:02.208919	3	accepted	\N	\N	1	N/A
128	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.	N / A	Does Not Support	3	3	31	5	106	33	2015-02-24 20:43:02.262798	2015-02-24 20:43:02.262798	3	accepted	\N	\N	1	N/A
129	Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall survival compared to patients with L858R or other exon 19 activating mutations.	Poor Outcome	Supports	1	3	2	5	107	34	2015-02-24 20:43:02.347651	2015-02-24 20:43:02.347651	4	accepted	\N	\N	1	N/A
130	In NSCLC patients with T790M and another activating mutations, their progression free survival is shorter compared to those who do not possess T790M mutation.	Poor Outcome	Supports	1	3	2	5	108	34	2015-02-24 20:43:02.411349	2015-02-24 20:43:02.411349	4	accepted	\N	\N	1	N/A
131	The T790M mutation in EGFR has been shown to confer resistance to the tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are placed on the drug are likely to relapse.	Resistance or Non-Response	Supports	3	1	31	21	109	34	2015-02-24 20:43:02.47836	2015-02-24 20:43:02.47836	5	accepted	\N	\N	1	N/A
132	In non-small cell lung cancer, the appearance of T790M mutation leads to resistance to gefitinib.	Resistance or Non-Response	Supports	3	3	30	5	110	34	2015-02-24 20:43:02.551034	2015-02-24 20:43:02.551034	3	accepted	\N	\N	1	N/A
133	In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence of L858R mutation is prognostic for better progression free survival.	Sensitivity	Supports	3	3	2	5	111	33	2015-02-24 20:43:02.619934	2015-02-24 20:43:02.619934	4	accepted	\N	\N	1	N/A
134	Geftinib added to pemetrexed and carboplatin showed greater effectiveness than premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.	Sensitivity	Supports	3	3	30	5	112	33	2015-02-24 20:43:02.68372	2015-02-24 20:43:02.68372	3	accepted	\N	\N	1	N/A
135	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.	Sensitivity	Supports	3	3	31	5	113	34	2015-02-24 20:43:02.751577	2015-02-24 20:43:02.751577	3	accepted	\N	\N	1	N/A
136	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.	Sensitivity	Supports	3	3	32	5	113	34	2015-02-24 20:43:02.816374	2015-02-24 20:43:02.816374	3	accepted	\N	\N	1	N/A
137	In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR, especially when an L858R mutation is also present.	Sensitivity	Supports	3	4	33	5	114	34	2015-02-24 20:43:02.913474	2015-02-24 20:43:02.913474	1	accepted	\N	\N	1	N/A
138	The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	Resistance or Non-Response	Supports	3	2	34	11	115	39	2015-02-24 20:43:02.98548	2015-02-24 20:43:02.98548	5	accepted	\N	\N	1	N/A
139	In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	3	2	35	11	115	35	2015-02-24 20:43:03.051423	2015-02-24 20:43:03.051423	4	accepted	\N	\N	1	N/A
140	In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	3	2	35	11	115	36	2015-02-24 20:43:03.118254	2015-02-24 20:43:03.118254	4	accepted	\N	\N	1	N/A
141	In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2, cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive to neratinib.	Sensitivity	Supports	3	2	35	11	115	37	2015-02-24 20:43:03.186035	2015-02-24 20:43:03.186035	5	accepted	\N	\N	1	N/A
142	In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	3	2	35	11	115	38	2015-02-24 20:43:03.253316	2015-02-24 20:43:03.253316	5	accepted	\N	\N	1	N/A
143	The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	Sensitivity	Supports	3	2	35	11	115	39	2015-02-24 20:43:03.317206	2015-02-24 20:43:03.317206	5	accepted	\N	\N	1	N/A
144	MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	Sensitivity	Supports	3	2	35	11	115	40	2015-02-24 20:43:03.406022	2015-02-24 20:43:03.406022	5	accepted	\N	\N	1	N/A
145	In MCF10A cell lines, the in-frame insertion of a single amino acid at position 780 was shown to be sensitive to neratinib.	Sensitivity	Supports	3	2	35	11	115	41	2015-02-24 20:43:03.473058	2015-02-24 20:43:03.473058	5	accepted	\N	\N	1	N/A
146	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	Sensitivity	Supports	3	2	35	11	115	42	2015-02-24 20:43:03.539064	2015-02-24 20:43:03.539064	5	accepted	\N	\N	1	N/A
147	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	Sensitivity	Supports	3	2	34	11	115	42	2015-02-24 20:43:03.593928	2015-02-24 20:43:03.593928	5	accepted	\N	\N	1	N/A
148	In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	3	2	35	11	115	43	2015-02-24 20:43:03.652743	2015-02-24 20:43:03.652743	5	accepted	\N	\N	1	N/A
149	In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	3	2	35	11	115	44	2015-02-24 20:43:03.716101	2015-02-24 20:43:03.716101	5	accepted	\N	\N	1	N/A
150	In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	3	2	35	11	115	45	2015-02-24 20:43:03.790789	2015-02-24 20:43:03.790789	5	accepted	\N	\N	1	N/A
151	The L536Q ligand binding domain mutation have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.	Sensitivity	Supports	3	2	36	11	116	46	2015-02-24 20:43:03.863541	2015-02-24 20:43:03.863541	5	accepted	\N	\N	1	N/A
152	The L536Q ligand binding domain mutation have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.	Sensitivity	Supports	3	2	37	11	116	46	2015-02-24 20:43:03.940478	2015-02-24 20:43:03.940478	5	accepted	\N	\N	1	N/A
153	MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	3	2	38	11	117	46	2015-02-24 20:43:04.020503	2015-02-24 20:43:04.020503	3	accepted	\N	\N	1	N/A
154	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	3	2	36	11	116	47	2015-02-24 20:43:04.096033	2015-02-24 20:43:04.096033	5	accepted	\N	\N	1	N/A
155	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	3	2	37	11	116	47	2015-02-24 20:43:04.15377	2015-02-24 20:43:04.15377	5	accepted	\N	\N	1	N/A
156	MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	3	2	38	11	117	47	2015-02-24 20:43:04.227165	2015-02-24 20:43:04.227165	3	accepted	\N	\N	1	N/A
157	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	3	2	36	11	116	48	2015-02-24 20:43:04.295952	2015-02-24 20:43:04.295952	5	accepted	\N	\N	1	N/A
158	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	3	2	37	11	116	48	2015-02-24 20:43:04.351437	2015-02-24 20:43:04.351437	5	accepted	\N	\N	1	N/A
159	MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	3	2	38	11	117	48	2015-02-24 20:43:04.435789	2015-02-24 20:43:04.435789	3	accepted	\N	\N	1	N/A
160	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	3	2	36	11	116	49	2015-02-24 20:43:04.501481	2015-02-24 20:43:04.501481	5	accepted	\N	\N	1	N/A
161	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	3	2	37	11	116	49	2015-02-24 20:43:04.56009	2015-02-24 20:43:04.56009	5	accepted	\N	\N	1	N/A
162	MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	3	2	38	11	117	49	2015-02-24 20:43:04.62547	2015-02-24 20:43:04.62547	3	accepted	\N	\N	1	N/A
163	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	3	2	36	11	116	50	2015-02-24 20:43:04.696186	2015-02-24 20:43:04.696186	5	accepted	\N	\N	1	N/A
164	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	3	2	37	11	116	50	2015-02-24 20:43:04.750582	2015-02-24 20:43:04.750582	5	accepted	\N	\N	1	N/A
165	MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Resistance or Non-Response	Supports	3	2	38	11	117	50	2015-02-24 20:43:04.821718	2015-02-24 20:43:04.821718	3	accepted	\N	\N	1	N/A
166	In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic sites and overall stable disease. However, this response did not meet criteria for RECIST partial response.	Sensitivity	Supports	3	3	39	22	118	51	2015-02-24 20:43:04.918235	2015-02-24 20:43:04.918235	2	accepted	\N	\N	1	N/A
167	Following identification of this fusion in a patient, this fusion was expressed in HEK 293T cells leading to oligomerization in the absence of ligand, induced morphologic changes in the cell and increased cellular proliferation consistent with FGFR activation. Mouse xenografts using the bladder cell line SW780 which harbors an FGFR3-BAIAP2L1 fusion showed reduced tumor burden when treated with the FGFR inhibitor PD173074.	Sensitivity	Supports	3	2	40	23	119	52	2015-02-24 20:43:04.99174	2015-02-24 20:43:04.99174	3	accepted	\N	\N	1	N/A
168	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be recurrent in multiple other cancer types.	Sensitivity	Supports	3	3	41	22	118	53	2015-02-24 20:43:05.069414	2015-02-24 20:43:05.069414	3	accepted	\N	\N	1	N/A
169	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be recurrent in multiple other cancer types.	Sensitivity	Supports	3	3	39	22	118	53	2015-02-24 20:43:05.121963	2015-02-24 20:43:05.121963	3	accepted	\N	\N	1	N/A
170	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	Better Outcome	Supports	1	3	2	2	120	55	2015-02-24 20:43:05.205545	2015-02-24 20:43:05.205545	3	accepted	\N	\N	1	N/A
171	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N / A	Does Not Support	1	3	2	2	101	54	2015-02-24 20:43:05.269966	2015-02-24 20:43:05.269966	4	accepted	\N	\N	1	N/A
172	FLT3-ITD duplication length did not effect prognosis in patients with an NPM1 mutation	N / A	Does Not Support	1	3	2	2	121	54	2015-02-24 20:43:05.345784	2015-02-24 20:43:05.345784	4	accepted	\N	\N	1	N/A
173	Size of the FLT3-ITD mutant duplication had no impact on overall survival or relapse rate	N / A	Does Not Support	1	3	2	2	122	54	2015-02-24 20:43:05.437189	2015-02-24 20:43:05.437189	2	accepted	\N	\N	1	N/A
174	AML patients with D835 mutation of FLT3 did not have worse overall survival than patients with wild type FLT3	N / A	Does Not Support	1	3	2	2	123	55	2015-02-24 20:43:05.510606	2015-02-24 20:43:05.510606	3	accepted	\N	\N	1	N/A
175	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated with worse overall survival compared to those without DNMT3A mutation.	Poor Outcome	Supports	1	3	2	2	103	54	2015-02-24 20:43:05.570295	2015-02-24 20:43:05.570295	5	accepted	\N	\N	1	N/A
176	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	Poor Outcome	Supports	1	3	2	2	121	54	2015-02-24 20:43:05.640722	2015-02-24 20:43:05.640722	4	accepted	\N	\N	1	N/A
177	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	Poor Outcome	Supports	1	3	2	2	121	54	2015-02-24 20:43:05.703611	2015-02-24 20:43:05.703611	4	accepted	\N	\N	1	N/A
178	A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation	Poor Outcome	Supports	1	3	2	2	121	54	2015-02-24 20:43:05.771096	2015-02-24 20:43:05.771096	4	accepted	\N	\N	1	N/A
179	Meta-analysis of studies involving cytogentically normal younger (<60) patients showed reduced overall and relapse-free survival for patients with FLT3-ITD	Poor Outcome	Supports	1	1	2	2	124	54	2015-02-24 20:43:05.839306	2015-02-24 20:43:05.839306	2	accepted	\N	\N	1	N/A
180	Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor survival and inferior disease free survival in patients with AML.	Poor Outcome	Supports	1	3	2	2	125	54	2015-02-24 20:43:05.925021	2015-02-24 20:43:05.925021	4	accepted	\N	\N	1	N/A
181	AML patients with internal tandem repeat of FLT3 has worse overall survival than patients with wild type FLT3	Poor Outcome	Supports	1	3	2	2	123	54	2015-02-24 20:43:05.991446	2015-02-24 20:43:05.991446	3	accepted	\N	\N	1	N/A
182	In AML patients, FLT3-TKD mutation was associated with poorer disease free survival compared to patients with wild type FLT3	Poor Outcome	Supports	1	3	2	2	126	55	2015-02-24 20:43:06.068029	2015-02-24 20:43:06.068029	4	accepted	\N	\N	1	N/A
183	WBC counts were not significantly different in AML patients with FLT3 D835 mutations compared to wild type	Positive	Does Not Support	2	3	2	2	123	55	2015-02-24 20:43:06.131311	2015-02-24 20:43:06.131311	3	accepted	\N	\N	1	N/A
184	WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared to wild type	Positive	Supports	2	3	2	2	123	54	2015-02-24 20:43:06.201195	2015-02-24 20:43:06.201195	3	accepted	\N	\N	1	N/A
185	FLT3-ITD mutation without NPM1 was associated with increased relapse risk and reduced overall survival in young adult AML patients (median age 43)	Resistance or Non-Response	Supports	3	3	2	2	122	54	2015-02-24 20:43:06.270615	2015-02-24 20:43:06.270615	4	accepted	\N	\N	1	N/A
186	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	Resistance or Non-Response	Supports	3	2	42	2	127	54	2015-02-24 20:43:06.344062	2015-02-24 20:43:06.344062	3	accepted	\N	\N	1	N/A
187	ATRA treatment did not effect overall survival in patients <60 years old with FLT3-ITD mutation	Resistance or Non-Response	Supports	3	3	43	2	128	54	2015-02-24 20:43:06.446331	2015-02-24 20:43:06.446331	3	accepted	\N	\N	1	N/A
188	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	Resistance or Non-Response	Supports	3	2	28	2	127	54	2015-02-24 20:43:06.519688	2015-02-24 20:43:06.519688	3	accepted	\N	\N	1	N/A
189	Increasing level of FLT3-ITD mutant correlated with reduced disease-free and overall survival as well as increased relapse rate	Resistance or Non-Response	Supports	3	3	2	2	122	54	2015-02-24 20:43:06.586657	2015-02-24 20:43:06.586657	3	accepted	\N	\N	1	N/A
190	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	Resistance or Non-Response	Supports	3	2	42	2	127	55	2015-02-24 20:43:06.658125	2015-02-24 20:43:06.658125	3	accepted	\N	\N	1	N/A
191	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	Resistance or Non-Response	Supports	3	2	28	2	127	55	2015-02-24 20:43:06.731226	2015-02-24 20:43:06.731226	3	accepted	\N	\N	1	N/A
192	Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated with sorafenib resistance	Resistance or Non-Response	Supports	3	3	10	2	129	55	2015-02-24 20:43:06.805537	2015-02-24 20:43:06.805537	4	accepted	\N	\N	1	N/A
193	Sorafenib is effective in patients with FLT3 internal tandem repeat mutations, but not in patients with wild type or D835 mutation.	Resistance or Non-Response	Supports	3	2	10	2	130	55	2015-02-24 20:43:06.867059	2015-02-24 20:43:06.867059	4	accepted	\N	\N	1	N/A
194	Sorafenib is effective in patients with FLT3 internal tandem repeats, but not in wild type or D835 mutation.	Sensitivity	Supports	3	2	10	2	130	54	2015-02-24 20:43:06.955797	2015-02-24 20:43:06.955797	4	accepted	\N	\N	1	N/A
195	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	Sensitivity	Supports	3	3	44	2	131	54	2015-02-24 20:43:07.030168	2015-02-24 20:43:07.030168	4	accepted	\N	\N	1	N/A
196	AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with FLT3 internal tandem repeat	Sensitivity	Supports	3	2	45	2	132	54	2015-02-24 20:43:07.092815	2015-02-24 20:43:07.092815	4	accepted	\N	\N	1	N/A
197	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	Sensitivity	Supports	3	3	44	2	131	55	2015-02-24 20:43:07.152795	2015-02-24 20:43:07.152795	4	accepted	\N	\N	1	N/A
198	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	Sensitivity	Supports	3	3	46	24	133	56	2015-02-24 20:43:07.230229	2015-02-24 20:43:07.230229	4	accepted	\N	\N	1	N/A
199	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	Sensitivity	Supports	3	3	47	24	133	56	2015-02-24 20:43:07.290074	2015-02-24 20:43:07.290074	4	accepted	\N	\N	1	N/A
200	In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic for better survival compared to patients who harbor wild type mutation.	Better Outcome	Supports	1	3	2	25	134	57	2015-02-24 20:43:07.359147	2015-02-24 20:43:07.359147	3	accepted	\N	\N	1	N/A
201	In patients with glioblastoma, those harboring IDH1 R132 mutation has higher overall survival compared to those who do not have IDH1 mutation	Better Outcome	Supports	1	3	2	26	135	57	2015-02-24 20:43:07.452365	2015-02-24 20:43:07.452365	3	accepted	\N	\N	1	N/A
202	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	N / A	Does Not Support	1	3	2	2	136	57	2015-02-24 20:43:07.520319	2015-02-24 20:43:07.520319	3	accepted	\N	\N	1	N/A
203	The complete remission and overall survival rates in patients with IDH1 R132 mutation is not significantly different from those who do not have this mutation	N / A	Does Not Support	1	3	2	2	137	57	2015-02-24 20:43:07.582541	2015-02-24 20:43:07.582541	2	accepted	\N	\N	1	N/A
204	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N / A	Does Not Support	1	3	2	2	138	58	2015-02-24 20:43:07.650666	2015-02-24 20:43:07.650666	2	accepted	\N	\N	1	N/A
205	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N / A	Does Not Support	1	3	2	2	138	59	2015-02-24 20:43:07.716147	2015-02-24 20:43:07.716147	3	accepted	\N	\N	1	N/A
206	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N / A	Does Not Support	1	3	2	2	138	60	2015-02-24 20:43:07.776034	2015-02-24 20:43:07.776034	3	accepted	\N	\N	1	N/A
207	IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated with worse overall survival	Poor Outcome	Supports	1	3	2	27	139	57	2015-02-24 20:43:07.925727	2015-02-24 20:43:07.925727	4	accepted	\N	\N	1	N/A
208	In leukemia patients, IDH1 R132 mutation is associated with adults and normal karyotype.	Positive	Supports	2	3	2	2	140	57	2015-02-24 20:43:07.999206	2015-02-24 20:43:07.999206	2	accepted	\N	\N	1	N/A
209	IDH1 R132 mutation is associated with patients of older age, high platelet count during diagnosis, cytogenic normalcy and NPM1 mutation	Positive	Supports	2	3	2	2	138	58	2015-02-24 20:43:08.061551	2015-02-24 20:43:08.061551	3	accepted	\N	\N	1	N/A
210	In AML, patients with IDH2 R140K mutation have improved overall survival compared to those with wild-type IDH2	Better Outcome	Supports	1	3	2	2	141	61	2015-02-24 20:43:08.130354	2015-02-24 20:43:08.130354	3	accepted	\N	\N	1	N/A
211	In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact overall survival or disease free survival.	N / A	Does Not Support	1	3	2	2	142	61	2015-02-24 20:43:08.191827	2015-02-24 20:43:08.191827	3	accepted	\N	\N	1	N/A
212	AML patients with IDH2 mutations such as R140Q/L have event free survival and overall survival similar to those with wild-type IDH2	N / A	Does Not Support	1	4	2	2	143	61	2015-02-24 20:43:08.259188	2015-02-24 20:43:08.259188	2	accepted	\N	\N	1	N/A
213	R140Q mutation in IDH2 does not have prognostic value in patients with MDS.	N / A	Does Not Support	1	3	2	28	144	61	2015-02-24 20:43:08.326318	2015-02-24 20:43:08.326318	3	accepted	\N	\N	1	N/A
214	In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L do not confer prognostic value (overall survival)	N / A	Does Not Support	1	4	2	28	145	61	2015-02-24 20:43:08.390487	2015-02-24 20:43:08.390487	2	accepted	\N	\N	1	N/A
215	AML patients with IDH2 mutations such as R172K have event free survival and overall survival similar to those with wild-type IDH2.	N / A	Does Not Support	1	4	2	2	143	62	2015-02-24 20:43:08.454719	2015-02-24 20:43:08.454719	2	accepted	\N	\N	1	N/A
216	In patients suffering from MDS, the presence of IDH2 mutations such as R172K do not confer prognostic value (overall survival)	N / A	Does Not Support	1	4	2	28	145	62	2015-02-24 20:43:08.504938	2015-02-24 20:43:08.504938	2	accepted	\N	\N	1	N/A
217	In AML, patients with an IDH2 R172K mutation have worse overall survival compared to those with wild-type IDH2	Poor Outcome	Supports	1	3	2	2	141	62	2015-02-24 20:43:08.566149	2015-02-24 20:43:08.566149	3	accepted	\N	\N	1	N/A
218	JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).	Negative	Supports	2	3	2	29	146	63	2015-02-24 20:43:08.635716	2015-02-24 20:43:08.635716	4	accepted	\N	\N	1	N/A
219	JAK2 V617F is associated with myeloid neoplasms (AML, MDS, CMML/aCML)	Positive	Supports	2	3	2	1	146	63	2015-02-24 20:43:08.696676	2015-02-24 20:43:08.696676	4	accepted	\N	\N	1	N/A
220	JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical CML).	Positive	Supports	2	3	2	1	146	63	2015-02-24 20:43:08.752369	2015-02-24 20:43:08.752369	4	accepted	\N	\N	1	N/A
221	The SNP rs10974944 in the Jak2 locus is associated with increased predisposition for JAK2 V617F mutation and its associated cancer	Positive	Supports	2	3	2	30	147	63	2015-02-24 20:43:08.814321	2015-02-24 20:43:08.814321	4	accepted	\N	\N	1	N/A
222	TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells	Sensitivity	Supports	3	2	48	31	148	63	2015-02-24 20:43:08.879877	2015-02-24 20:43:08.879877	3	accepted	\N	\N	1	N/A
223	In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction in the percentage of cells harboring JAK2 V617F.	Sensitivity	Supports	3	3	49	32	149	63	2015-02-24 20:43:08.945887	2015-02-24 20:43:08.945887	4	accepted	\N	\N	1	N/A
224	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	1	3	1	7	150	65	2015-02-24 20:43:09.0071	2015-02-24 20:43:09.0071	3	accepted	\N	\N	1	N/A
225	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	1	3	1	7	150	67	2015-02-24 20:43:09.076067	2015-02-24 20:43:09.076067	3	accepted	\N	\N	1	N/A
226	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	1	3	1	7	150	69	2015-02-24 20:43:09.131208	2015-02-24 20:43:09.131208	3	accepted	\N	\N	1	N/A
227	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	N / A	Does Not Support	1	3	2	33	151	65	2015-02-24 20:43:09.202773	2015-02-24 20:43:09.202773	3	accepted	\N	\N	1	N/A
228	KIT mutations were identified in morphologically benign, incidentally discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating is not a prognostic marker for GIST	N / A	Does Not Support	1	3	2	33	152	65	2015-02-24 20:43:09.275593	2015-02-24 20:43:09.275593	3	accepted	\N	\N	1	N/A
229	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	N / A	Does Not Support	1	3	2	17	153	65	2015-02-24 20:43:09.343068	2015-02-24 20:43:09.343068	2	accepted	\N	\N	1	N/A
230	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	N / A	Does Not Support	1	3	2	33	151	70	2015-02-24 20:43:09.412714	2015-02-24 20:43:09.412714	3	accepted	\N	\N	1	N/A
231	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	N / A	Does Not Support	1	3	2	17	153	70	2015-02-24 20:43:09.510978	2015-02-24 20:43:09.510978	2	accepted	\N	\N	1	N/A
232	In acute myloid leukemia patients, D816 mutation is associated with earlier relapse and poorer prognosis than wildtype KIT	Poor Outcome	Supports	1	3	2	34	154	64	2015-02-24 20:43:09.577608	2015-02-24 20:43:09.577608	4	accepted	\N	\N	1	N/A
233	KIT mutation is associated with worse overall and cause-specific prognosis in patients with GIST compared to patients with wildtype KIT	Poor Outcome	Supports	1	3	2	33	155	65	2015-02-24 20:43:09.647546	2015-02-24 20:43:09.647546	3	accepted	\N	\N	1	N/A
234	GIST patients with exon 14 KIT mutations had reduced overall survival compared to patients wiltype for KIT	Poor Outcome	Supports	1	3	2	33	151	68	2015-02-24 20:43:09.716868	2015-02-24 20:43:09.716868	3	accepted	\N	\N	1	N/A
235	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	Positive	Supports	2	3	2	33	155	65	2015-02-24 20:43:09.788063	2015-02-24 20:43:09.788063	2	accepted	\N	\N	1	N/A
236	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	Positive	Supports	2	3	2	33	155	69	2015-02-24 20:43:09.861115	2015-02-24 20:43:09.861115	3	accepted	\N	\N	1	N/A
237	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	3	2	1	7	156	71	2015-02-24 20:43:09.931389	2015-02-24 20:43:09.931389	3	accepted	\N	\N	1	N/A
238	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	3	2	4	7	156	71	2015-02-24 20:43:09.991077	2015-02-24 20:43:09.991077	3	accepted	\N	\N	1	N/A
239	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	3	2	10	7	156	71	2015-02-24 20:43:10.046679	2015-02-24 20:43:10.046679	3	accepted	\N	\N	1	N/A
240	KIT V654A results in imatinib resistance in GIST patient-derived cell lines.	Resistance or Non-Response	Supports	3	2	1	33	157	72	2015-02-24 20:43:10.123544	2015-02-24 20:43:10.123544	3	accepted	\N	\N	1	N/A
241	In a case study, a patient with anal melanoma harboring a 7 codon duplication in the juxtamambrane region (exon 11) of KIT showed marked response 4 months after imatinib treatment.	Sensitivity	Supports	3	3	1	35	158	66	2015-02-24 20:43:10.198256	2015-02-24 20:43:10.198256	3	accepted	\N	\N	1	N/A
242	The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation of KIT but did not alter the binding affinity of dasatinib.	Sensitivity	Supports	3	2	5	7	156	71	2015-02-24 20:43:10.267017	2015-02-24 20:43:10.267017	3	accepted	\N	\N	1	N/A
243	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	3	2	5	5	159	71	2015-02-24 20:43:10.337195	2015-02-24 20:43:10.337195	3	accepted	\N	\N	1	N/A
244	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	3	2	1	5	159	71	2015-02-24 20:43:10.392008	2015-02-24 20:43:10.392008	3	accepted	\N	\N	1	N/A
245	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	3	2	4	5	159	71	2015-02-24 20:43:10.442146	2015-02-24 20:43:10.442146	3	accepted	\N	\N	1	N/A
246	In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective for those that are refractory to imatinib.	Sensitivity	Supports	3	3	50	33	160	72	2015-02-24 20:43:10.517735	2015-02-24 20:43:10.517735	3	accepted	\N	\N	1	N/A
247	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	Better Outcome	Supports	1	3	15	10	161	80	2015-02-24 20:43:10.584559	2015-02-24 20:43:10.584559	4	accepted	\N	\N	1	N/A
248	In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12 mutation did not impact overall or disease free survival.	N / A	Does Not Support	1	3	2	10	162	75	2015-02-24 20:43:10.644189	2015-02-24 20:43:10.644189	4	accepted	\N	\N	1	N/A
249	There is no strong association between KRAS mutation status and overall or progression free survival in patients with NSCLC	N / A	Does Not Support	1	3	2	5	163	75	2015-02-24 20:43:10.705126	2015-02-24 20:43:10.705126	3	accepted	\N	\N	1	N/A
250	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	Poor Outcome	Supports	1	3	2	36	164	75	2015-02-24 20:43:10.771433	2015-02-24 20:43:10.771433	3	accepted	\N	\N	1	N/A
251	A meta-analysis showed KRAS mutation is associated with worse outcome in patients with NSCLC	Poor Outcome	Supports	1	3	2	5	165	75	2015-02-24 20:43:10.829455	2015-02-24 20:43:10.829455	3	accepted	\N	\N	1	N/A
252	KRAS mutations were significantly associated with reduced progression-free and overall survival in patients with advanced colorectal cancer	Poor Outcome	Supports	1	2	2	9	166	76	2015-02-24 20:43:10.889307	2015-02-24 20:43:10.889307	3	accepted	\N	\N	1	N/A
253	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	Poor Outcome	Supports	1	3	2	37	164	79	2015-02-24 20:43:10.949787	2015-02-24 20:43:10.949787	3	accepted	\N	\N	1	N/A
254	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	Positive	Does Not Support	2	3	2	5	167	79	2015-02-24 20:43:11.057827	2015-02-24 20:43:11.057827	3	accepted	\N	\N	1	N/A
255	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	Positive	Does Not Support	2	3	2	5	167	75	2015-02-24 20:43:11.115206	2015-02-24 20:43:11.115206	3	accepted	\N	\N	1	N/A
256	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	Positive	Supports	2	3	2	38	168	75	2015-02-24 20:43:11.178858	2015-02-24 20:43:11.178858	3	accepted	\N	\N	1	N/A
257	KRAS G12C occur more frequently in women than men	Positive	Supports	2	3	2	17	169	77	2015-02-24 20:43:11.248372	2015-02-24 20:43:11.248372	2	accepted	\N	\N	1	N/A
258	KRAS G12D mutation occurs in never smokers significantly more often than in smokers	Positive	Supports	2	3	2	17	169	78	2015-02-24 20:43:11.318182	2015-02-24 20:43:11.318182	3	accepted	\N	\N	1	N/A
259	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	Positive	Supports	2	3	2	38	168	79	2015-02-24 20:43:11.377234	2015-02-24 20:43:11.377234	3	accepted	\N	\N	1	N/A
260	Patients with colorectal cancer who harbor KRAS mutation have low response rate to cetuximab	Resistance or Non-Response	Supports	3	3	15	9	170	73	2015-02-24 20:43:11.439055	2015-02-24 20:43:11.439055	4	accepted	\N	\N	1	N/A
261	In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	Resistance or Non-Response	Supports	3	3	30	5	171	74	2015-02-24 20:43:11.506215	2015-02-24 20:43:11.506215	3	accepted	\N	\N	1	N/A
262	In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	Resistance or Non-Response	Supports	3	3	31	5	171	74	2015-02-24 20:43:11.566591	2015-02-24 20:43:11.566591	3	accepted	\N	\N	1	N/A
263	KRAS mutations were significantly associated with lack of response to cetuximab in patients with advanced colorectal cancer	Resistance or Non-Response	Supports	3	2	15	9	166	76	2015-02-24 20:43:11.624967	2015-02-24 20:43:11.624967	3	accepted	\N	\N	1	N/A
264	Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations (primarily G12/G13) had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype KRAS	Resistance or Non-Response	Supports	3	3	15	9	72	76	2015-02-24 20:43:11.687423	2015-02-24 20:43:11.687423	3	accepted	\N	\N	1	N/A
265	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	Resistance or Non-Response	Supports	3	3	16	5	172	78	2015-02-24 20:43:11.753254	2015-02-24 20:43:11.753254	2	accepted	\N	\N	1	N/A
266	Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	Resistance or Non-Response	Supports	3	2	15	10	161	80	2015-02-24 20:43:11.810799	2015-02-24 20:43:11.810799	4	accepted	\N	\N	1	N/A
267	KRAS mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	Sensitivity	Does Not Support	3	3	51	9	173	74	2015-02-24 20:43:11.883747	2015-02-24 20:43:11.883747	4	accepted	\N	\N	1	N/A
268	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.	Sensitivity	Supports	3	2	52	5	174	78	2015-02-24 20:43:11.953307	2015-02-24 20:43:11.953307	4	accepted	\N	\N	1	N/A
269	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.	Sensitivity	Supports	3	2	53	5	174	78	2015-02-24 20:43:12.016423	2015-02-24 20:43:12.016423	4	accepted	\N	\N	1	N/A
270	Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	Sensitivity	Supports	3	2	15	10	161	80	2015-02-24 20:43:12.081846	2015-02-24 20:43:12.081846	4	accepted	\N	\N	1	N/A
271	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	Sensitivity	Supports	3	3	15	10	161	80	2015-02-24 20:43:12.152051	2015-02-24 20:43:12.152051	4	accepted	\N	\N	1	N/A
272	The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition by about 5 fold.	Resistance or Non-Response	Supports	3	2	17	7	175	81	2015-02-24 20:43:12.220488	2015-02-24 20:43:12.220488	3	accepted	\N	\N	1	N/A
273	MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.	Resistance or Non-Response	Supports	3	2	17	7	176	82	2015-02-24 20:43:12.284113	2015-02-24 20:43:12.284113	3	accepted	\N	\N	1	N/A
274	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	3	2	1	19	97	83	2015-02-24 20:43:12.343571	2015-02-24 20:43:12.343571	3	accepted	\N	\N	1	N/A
275	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	3	2	27	19	97	83	2015-02-24 20:43:12.402539	2015-02-24 20:43:12.402539	3	accepted	\N	\N	1	N/A
276	MGMT promoter methylation was associated with increased overall and disease-free survival as well as tumor regression in patients with gliomas.	Better Outcome	Supports	1	3	2	39	177	84	2015-02-24 20:43:12.466741	2015-02-24 20:43:12.466741	4	accepted	\N	\N	1	N/A
277	In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted from temozolomide. This benefit was also methylation status dependent, as those without methylation did not see increased survival.	Sensitivity	Supports	3	1	13	39	178	84	2015-02-24 20:43:12.527293	2015-02-24 20:43:12.527293	5	accepted	\N	\N	1	N/A
278	In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor, such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may be useful in treating patients with initial resistance to alkylating agents.	Sensitivity	Supports	3	4	54	39	177	84	2015-02-24 20:43:12.637939	2015-02-24 20:43:12.637939	4	accepted	\N	\N	1	N/A
279	MGMT promoter methylation has been shown to confer sensitivity to alkylating agents, such as carmustine, in glioblastoma cells.	Sensitivity	Supports	3	3	55	39	177	84	2015-02-24 20:43:12.70725	2015-02-24 20:43:12.70725	4	accepted	\N	\N	1	N/A
280	Complete remission rates were higher and event-free survival was longer for normal karyotype AML patients with Exon 12 NPM1 mutations.	Better Outcome	Supports	1	3	2	2	179	85	2015-02-24 20:43:12.773686	2015-02-24 20:43:12.773686	5	accepted	\N	\N	1	N/A
281	Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and without FLT3-ITD have increased overall survival, relapse-free survival and response to induction chemotherapy	Better Outcome	Supports	1	3	2	2	180	85	2015-02-24 20:43:12.834244	2015-02-24 20:43:12.834244	4	accepted	\N	\N	1	N/A
282	NPM1 mutations were associated with increased complete remission rates as well as longer overall disease free survival in normal karyotype AML patients >59 years old, particularly in those >69 years old	Better Outcome	Supports	1	3	2	2	181	85	2015-02-24 20:43:12.901195	2015-02-24 20:43:12.901195	4	accepted	\N	\N	1	N/A
283	NPM1 mutation was associated with higher complete remission rates, lower cumulative incidence of relapse and higher overall survival in intermediate risk AML patients	Better Outcome	Supports	1	3	2	2	182	85	2015-02-24 20:43:12.964923	2015-02-24 20:43:12.964923	4	accepted	\N	\N	1	N/A
284	Complete remission rates were higher and both disease-free and overall survival were longer for patients with Exon 12 NPM1 mutations.	Better Outcome	Supports	1	3	2	2	183	85	2015-02-24 20:43:13.034329	2015-02-24 20:43:13.034329	3	accepted	\N	\N	1	N/A
285	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	Better Outcome	Supports	1	3	2	2	120	85	2015-02-24 20:43:13.097644	2015-02-24 20:43:13.097644	3	accepted	\N	\N	1	N/A
286	Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved overall and relapse-free survival, complete remission rates and remission duration compared to all other combinations of mutants	Better Outcome	Supports	1	3	2	2	184	85	2015-02-24 20:43:13.165144	2015-02-24 20:43:13.165144	3	accepted	\N	\N	1	N/A
287	NPM1 mutation was associated with increased overall, event-free and relapse-free survival and remission after induction therapy in the entire cohort and in the cytogenetically normal subset of patients	Better Outcome	Supports	1	3	2	2	185	85	2015-02-24 20:43:13.234052	2015-02-24 20:43:13.234052	3	accepted	\N	\N	1	N/A
288	Intermediate risk patients with mutant NPM1 had improved overall survival with the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1 and IDH2	Better Outcome	Supports	1	3	2	2	186	85	2015-02-24 20:43:13.303126	2015-02-24 20:43:13.303126	2	accepted	\N	\N	1	N/A
289	NPM1 mutation without FLT3-ITD was significantly associated with complete remission	Better Outcome	Supports	1	3	2	2	94	85	2015-02-24 20:43:13.360136	2015-02-24 20:43:13.360136	3	accepted	\N	\N	1	N/A
290	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N / A	Does Not Support	1	3	2	2	101	85	2015-02-24 20:43:13.421448	2015-02-24 20:43:13.421448	4	accepted	\N	\N	1	N/A
291	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	N / A	Does Not Support	1	3	2	2	136	85	2015-02-24 20:43:13.494802	2015-02-24 20:43:13.494802	3	accepted	\N	\N	1	N/A
292	NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid leukemia.	Negative	Supports	2	3	2	2	179	85	2015-02-24 20:43:13.554168	2015-02-24 20:43:13.554168	4	accepted	\N	\N	1	N/A
293	Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction in NPM1 levels following first line therapy (N=15)	Poor Outcome	Supports	1	3	2	2	187	85	2015-02-24 20:43:13.627758	2015-02-24 20:43:13.627758	2	accepted	\N	\N	1	N/A
294	Coocurrence of FLT3-ITD mutations significantly reduced overall survival of normal karyotype AML patients	Poor Outcome	Supports	1	3	2	2	183	85	2015-02-24 20:43:13.691263	2015-02-24 20:43:13.691263	4	accepted	\N	\N	1	N/A
295	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	Poor Outcome	Supports	1	3	2	2	121	85	2015-02-24 20:43:13.765533	2015-02-24 20:43:13.765533	4	accepted	\N	\N	1	N/A
296	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	Poor Outcome	Supports	1	3	2	2	121	85	2015-02-24 20:43:13.826715	2015-02-24 20:43:13.826715	4	accepted	\N	\N	1	N/A
297	TET2 mutation reduces overall survival of normal karyotype patients with either an NPM1 mutation or NPM1 mutation without FLT3-ITD	Poor Outcome	Supports	1	3	2	2	188	85	2015-02-24 20:43:13.890265	2015-02-24 20:43:13.890265	3	accepted	\N	\N	1	N/A
298	Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate risk AML patients	Poor Outcome	Supports	1	3	2	2	182	85	2015-02-24 20:43:13.954318	2015-02-24 20:43:13.954318	3	accepted	\N	\N	1	N/A
299	NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype (0/55).	Positive	Does Not Support	2	3	2	2	183	85	2015-02-24 20:43:14.019642	2015-02-24 20:43:14.019642	3	accepted	\N	\N	1	N/A
300	No NPM1 mutations were identified in patients with favorable risk cytogenetics (79/215 patients)	Positive	Does Not Support	2	3	2	2	103	86	2015-02-24 20:43:14.07724	2015-02-24 20:43:14.07724	4	accepted	\N	\N	1	N/A
301	NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid leukemia.	Positive	Supports	2	3	2	2	179	85	2015-02-24 20:43:14.196374	2015-02-24 20:43:14.196374	4	accepted	\N	\N	1	N/A
302	NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid leukemia	Positive	Supports	2	3	2	2	183	85	2015-02-24 20:43:14.261177	2015-02-24 20:43:14.261177	3	accepted	\N	\N	1	N/A
303	NPM1 mutations were associated with normal karyotype in older (>60) patients	Positive	Supports	2	3	2	2	189	85	2015-02-24 20:43:14.326634	2015-02-24 20:43:14.326634	3	accepted	\N	\N	1	N/A
304	NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes	Positive	Supports	2	3	2	2	122	85	2015-02-24 20:43:14.395507	2015-02-24 20:43:14.395507	2	accepted	\N	\N	1	N/A
305	"AML with mutated NPM1" is a provisional entity in WHO classification of acute myeloid leukemia (AML). This mutation should be tested for in clinical trials and is recommended for testing in patients with cytogentically normal AML.	Positive	Supports	2	1	2	2	190	85	2015-02-24 20:43:14.460977	2015-02-24 20:43:14.460977	5	accepted	\N	\N	1	N/A
306	NPM1 mutations were associated with intermediate risk cytogenetics (including normal karyotype)	Positive	Supports	2	3	2	2	103	86	2015-02-24 20:43:14.524125	2015-02-24 20:43:14.524125	4	accepted	\N	\N	1	N/A
307	ATRA treatment did not effect overall survival in patients <60 years old with NPM1 mutation regardless of FLT3-ITD status	Resistance or Non-Response	Supports	3	3	43	2	128	85	2015-02-24 20:43:14.588104	2015-02-24 20:43:14.588104	3	accepted	\N	\N	1	N/A
308	NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and increased overall survival in young adult acute myeloid leukemia patients (median age 43)	Sensitivity	Supports	3	3	2	2	122	85	2015-02-24 20:43:14.659858	2015-02-24 20:43:14.659858	4	accepted	\N	\N	1	N/A
309	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	Sensitivity	Supports	3	2	56	2	191	85	2015-02-24 20:43:14.731424	2015-02-24 20:43:14.731424	3	accepted	\N	\N	1	N/A
310	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	Sensitivity	Supports	3	2	43	2	191	85	2015-02-24 20:43:14.790266	2015-02-24 20:43:14.790266	3	accepted	\N	\N	1	N/A
311	Patients with NPM1 mutations were associated with improved complete remission rates as well as overall and disease free survival when all ages and karyotype were analyzed; however, when only normal karyotype patients were analyzed, only complete remission rates were improved	Sensitivity	Supports	3	3	2	2	189	85	2015-02-24 20:43:14.852712	2015-02-24 20:43:14.852712	3	accepted	\N	\N	1	N/A
312	ATRA treatment improved overall and relapse-free survival in older (>60) patients with mutant NPM1 and without FLT3-ITD	Sensitivity	Supports	3	3	43	2	189	85	2015-02-24 20:43:14.916029	2015-02-24 20:43:14.916029	2	accepted	\N	\N	1	N/A
313	Young patients (18-60) with NPM1 mutations had improved overall survival following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However, the adjusted p-value for multiple testing is 0.11.	Sensitivity	Supports	3	3	28	2	186	85	2015-02-24 20:43:14.974768	2015-02-24 20:43:14.974768	2	accepted	\N	\N	1	N/A
314	For patients with NPM1 mutation that achieved CR following induction therapy, relapse-free survival was improved following treatment with valproic acid	Sensitivity	Supports	3	3	57	2	192	85	2015-02-24 20:43:15.041999	2015-02-24 20:43:15.041999	2	accepted	\N	\N	1	N/A
315	CD33 expression was significantly increased in patients with NPM1 mutation with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy	Sensitivity	Supports	3	4	2	2	193	85	2015-02-24 20:43:15.100783	2015-02-24 20:43:15.100783	2	accepted	\N	\N	1	N/A
316	CD33 and CD123 expression was significantly increased in patients with NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33 and anti-CD123 therapy	Sensitivity	Supports	3	4	2	2	193	85	2015-02-24 20:43:15.164897	2015-02-24 20:43:15.164897	2	accepted	\N	\N	1	N/A
317	NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.	Sensitivity	Supports	3	2	56	2	191	86	2015-02-24 20:43:15.231436	2015-02-24 20:43:15.231436	3	accepted	\N	\N	1	N/A
318	NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM in AML patients was associated with good response to induction therapy.	Sensitivity	Supports	3	4	2	2	194	86	2015-02-24 20:43:15.301947	2015-02-24 20:43:15.301947	3	accepted	\N	\N	1	N/A
319	The presence of NRAS mutation in AML patients does not impact diease prognosis (resistant disease, disease-free survival, complete remission rate, relapse rate, induction death)	N / A	Does Not Support	1	3	2	2	195	88	2015-02-24 20:43:15.373907	2015-02-24 20:43:15.373907	3	accepted	\N	\N	1	N/A
320	NRAS mutation status does not impact overall, event-free or disease-free survival in patients with AML	N / A	Does Not Support	1	3	2	2	196	91	2015-02-24 20:43:15.451298	2015-02-24 20:43:15.451298	3	accepted	\N	\N	1	N/A
321	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	Poor Outcome	Supports	1	3	2	7	197	87	2015-02-24 20:43:15.519593	2015-02-24 20:43:15.519593	3	accepted	\N	\N	1	N/A
322	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis.	Poor Outcome	Supports	1	3	2	9	198	88	2015-02-24 20:43:15.581974	2015-02-24 20:43:15.581974	3	accepted	\N	\N	1	N/A
323	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	Poor Outcome	Supports	1	3	2	7	197	88	2015-02-24 20:43:15.688819	2015-02-24 20:43:15.688819	3	accepted	\N	\N	1	N/A
324	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis.	Poor Outcome	Supports	1	1	2	9	198	89	2015-02-24 20:43:15.75378	2015-02-24 20:43:15.75378	3	accepted	\N	\N	1	N/A
325	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis.	Poor Outcome	Supports	1	1	2	9	198	90	2015-02-24 20:43:15.821083	2015-02-24 20:43:15.821083	3	accepted	\N	\N	1	N/A
326	Melanoma associated with NRAS Q61 mutation was more often associated with those at the extremity than those at the trunk	Positive	Supports	2	3	2	7	199	93	2015-02-24 20:43:15.886224	2015-02-24 20:43:15.886224	3	accepted	\N	\N	1	N/A
327	In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status (primarily Q61) was not informative for disease control rate, progression free survival or overall survival following cetuximab plus chemotherapy.	Resistance or Non-Response	Does Not Support	3	3	15	9	72	93	2015-02-24 20:43:15.957929	2015-02-24 20:43:15.957929	3	accepted	\N	\N	1	N/A
328	Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation (primarily Q61) have a significantly lower response rate to cetuximab than patients wildtype for NRAS.	Resistance or Non-Response	Supports	3	3	15	9	72	93	2015-02-24 20:43:16.032753	2015-02-24 20:43:16.032753	3	accepted	\N	\N	1	N/A
329	Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.	Resistance or Non-Response	Supports	3	3	22	7	200	93	2015-02-24 20:43:16.099293	2015-02-24 20:43:16.099293	4	accepted	\N	\N	1	N/A
330	Likely due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic malignant melanoma with an NRAS G13D mutation	Sensitivity	Supports	3	3	58	7	201	92	2015-02-24 20:43:16.169542	2015-02-24 20:43:16.169542	2	accepted	\N	\N	1	N/A
331	In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted in disease free survival of 14 months	Sensitivity	Supports	3	3	59	7	202	94	2015-02-24 20:43:16.242643	2015-02-24 20:43:16.242643	2	accepted	\N	\N	1	N/A
332	In a melanoma patient with Q61R mutation, treatment with temozolomide resulted in overall survival of 16 months	Sensitivity	Supports	3	3	59	7	202	95	2015-02-24 20:43:16.306388	2015-02-24 20:43:16.306388	2	accepted	\N	\N	1	N/A
333	GIST tumors harboring PDGFRA D842V mutation are more likely to be benign than malignant.	Negative	Supports	2	3	2	40	203	97	2015-02-24 20:43:16.374	2015-02-24 20:43:16.374	3	accepted	\N	\N	1	N/A
334	While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib, double mutation of V561D and D842V mutants are resistant to imatinib.	Resistance or Non-Response	Supports	3	3	1	40	157	97	2015-02-24 20:43:16.447502	2015-02-24 20:43:16.447502	4	accepted	\N	\N	1	N/A
335	GIST cancer with D842V mutation is resistant to imatinib.	Resistance or Non-Response	Supports	3	3	1	40	204	97	2015-02-24 20:43:16.514732	2015-02-24 20:43:16.514732	4	accepted	\N	\N	1	N/A
336	In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	3	2	60	40	205	96	2015-02-24 20:43:16.585232	2015-02-24 20:43:16.585232	4	accepted	\N	\N	1	N/A
337	In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	3	2	60	40	205	97	2015-02-24 20:43:16.658008	2015-02-24 20:43:16.658008	4	accepted	\N	\N	1	N/A
338	In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	3	2	60	40	205	98	2015-02-24 20:43:16.732533	2015-02-24 20:43:16.732533	4	accepted	\N	\N	1	N/A
339	In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	3	2	60	40	205	99	2015-02-24 20:43:16.799197	2015-02-24 20:43:16.799197	4	accepted	\N	\N	1	N/A
340	In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	3	2	60	40	205	100	2015-02-24 20:43:16.870462	2015-02-24 20:43:16.870462	4	accepted	\N	\N	1	N/A
341	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N / A	Does Not Support	3	3	51	9	173	101	2015-02-24 20:43:16.935141	2015-02-24 20:43:16.935141	4	accepted	\N	\N	1	N/A
342	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N / A	Does Not Support	3	3	51	9	173	102	2015-02-24 20:43:16.999338	2015-02-24 20:43:16.999338	4	accepted	\N	\N	1	N/A
343	In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations (Exon 9) had no significant impact on response rate, disease control rate, progression free surivival or overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	N / A	Does Not Support	3	3	15	9	72	104	2015-02-24 20:43:17.065478	2015-02-24 20:43:17.065478	3	accepted	\N	\N	1	N/A
344	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N / A	Does Not Support	3	3	51	9	173	105	2015-02-24 20:43:17.129921	2015-02-24 20:43:17.129921	4	accepted	\N	\N	1	N/A
345	The presence of exon 9 (such as E542K) or exon 20 mutations were associated with poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	1	4	2	9	206	101	2015-02-24 20:43:17.243634	2015-02-24 20:43:17.243634	3	accepted	\N	\N	1	N/A
346	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	Poor Outcome	Supports	1	4	2	9	207	101	2015-02-24 20:43:17.317095	2015-02-24 20:43:17.317095	2	accepted	\N	\N	1	N/A
347	The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	1	4	2	9	206	102	2015-02-24 20:43:17.388789	2015-02-24 20:43:17.388789	3	accepted	\N	\N	1	N/A
348	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	Poor Outcome	Supports	1	4	2	9	207	102	2015-02-24 20:43:17.44953	2015-02-24 20:43:17.44953	2	accepted	\N	\N	1	N/A
349	The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	1	4	2	9	206	105	2015-02-24 20:43:17.520364	2015-02-24 20:43:17.520364	3	accepted	\N	\N	1	N/A
350	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	3	4	15	9	206	101	2015-02-24 20:43:17.591412	2015-02-24 20:43:17.591412	3	accepted	\N	\N	1	N/A
351	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	3	4	14	9	206	101	2015-02-24 20:43:17.652009	2015-02-24 20:43:17.652009	3	accepted	\N	\N	1	N/A
352	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	3	4	15	9	206	102	2015-02-24 20:43:17.713633	2015-02-24 20:43:17.713633	3	accepted	\N	\N	1	N/A
353	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	3	4	14	9	206	102	2015-02-24 20:43:17.774592	2015-02-24 20:43:17.774592	3	accepted	\N	\N	1	N/A
354	Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	Resistance or Non-Response	Supports	3	3	15	9	72	103	2015-02-24 20:43:17.837097	2015-02-24 20:43:17.837097	3	accepted	\N	\N	1	N/A
355	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	3	4	15	9	206	105	2015-02-24 20:43:17.901778	2015-02-24 20:43:17.901778	3	accepted	\N	\N	1	N/A
356	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	3	4	14	9	206	105	2015-02-24 20:43:17.958735	2015-02-24 20:43:17.958735	3	accepted	\N	\N	1	N/A
357	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	3	2	61	11	208	101	2015-02-24 20:43:18.026065	2015-02-24 20:43:18.026065	4	accepted	\N	\N	1	N/A
358	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	3	2	62	11	209	101	2015-02-24 20:43:18.096369	2015-02-24 20:43:18.096369	3	accepted	\N	\N	1	N/A
359	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	3	2	61	11	208	102	2015-02-24 20:43:18.156552	2015-02-24 20:43:18.156552	4	accepted	\N	\N	1	N/A
360	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	3	2	62	11	209	102	2015-02-24 20:43:18.214787	2015-02-24 20:43:18.214787	3	accepted	\N	\N	1	N/A
361	Breast cancer cell lines with H1047R mutation showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	3	2	61	11	208	105	2015-02-24 20:43:18.279901	2015-02-24 20:43:18.279901	4	accepted	\N	\N	1	N/A
362	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	3	2	62	11	209	105	2015-02-24 20:43:18.345367	2015-02-24 20:43:18.345367	3	accepted	\N	\N	1	N/A
363	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	Sensitivity	Supports	3	3	43	41	210	106	2015-02-24 20:43:18.406147	2015-02-24 20:43:18.406147	5	accepted	\N	\N	1	N/A
364	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	Positive	Supports	2	3	2	20	98	107	2015-02-24 20:43:18.466996	2015-02-24 20:43:18.466996	4	accepted	\N	\N	1	N/A
365	PTEN nonsense mutations, including R233*, have been shown to be inactivating and loss-of-function, but do not have prognostic value in glioblastoma multiforme patients.	Poor Outcome	Does Not Support	1	3	2	39	211	108	2015-02-24 20:43:18.543357	2015-02-24 20:43:18.543357	3	accepted	\N	\N	1	N/A
366	Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse to PI3K-mTOR inhibitors.	Sensitivity	Supports	3	2	2	3	212	108	2015-02-24 20:43:18.610815	2015-02-24 20:43:18.610815	4	accepted	\N	\N	1	N/A
367	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	Sensitivity	Supports	3	3	43	41	210	109	2015-02-24 20:43:18.725996	2015-02-24 20:43:18.725996	5	accepted	\N	\N	1	N/A
368	Compared to those who harbor a wild type RET, patients with RET M918T mutation develop larger and more aggressive medullary thyroid cancer.	Poor Outcome	Supports	1	3	2	42	213	111	2015-02-24 20:43:18.797153	2015-02-24 20:43:18.797153	4	accepted	\N	\N	1	N/A
369	In patients with medullary carcinoma, the presence of RET M918T mutation is associated with increased probability of lymph node metastases.	Positive	Supports	2	3	2	42	214	111	2015-02-24 20:43:18.866742	2015-02-24 20:43:18.866742	5	accepted	\N	\N	1	N/A
370	Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib, compared to wild-type RET.	Resistance or Non-Response	Supports	3	2	63	42	215	110	2015-02-24 20:43:18.932351	2015-02-24 20:43:18.932351	3	accepted	\N	\N	1	N/A
371	Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib, compared to wild-type RET.	Resistance or Non-Response	Supports	3	2	63	42	215	111	2015-02-24 20:43:18.99952	2015-02-24 20:43:18.99952	3	accepted	\N	\N	1	N/A
372	The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong repression of tyroid cancer cell growth.	Sensitivity	Supports	3	2	64	42	216	111	2015-02-24 20:43:19.074554	2015-02-24 20:43:19.074554	3	accepted	\N	\N	1	N/A
373	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	Better Outcome	Supports	1	4	2	28	217	112	2015-02-24 20:43:19.136904	2015-02-24 20:43:19.136904	3	accepted	\N	\N	1	N/A
374	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	Better Outcome	Supports	1	4	2	28	217	113	2015-02-24 20:43:19.200133	2015-02-24 20:43:19.200133	3	accepted	\N	\N	1	N/A
375	Breast cancer patients who harbor R175H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	1	3	2	11	218	114	2015-02-24 20:43:19.267079	2015-02-24 20:43:19.267079	3	accepted	\N	\N	1	N/A
376	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R248 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	1	3	2	11	219	115	2015-02-24 20:43:19.324354	2015-02-24 20:43:19.324354	3	accepted	\N	\N	1	N/A
377	Breast cancer patients who harbor R248Q mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	1	3	2	11	218	115	2015-02-24 20:43:19.386235	2015-02-24 20:43:19.386235	3	accepted	\N	\N	1	N/A
378	In breast cancer patients harboring R248W mutation, the prognosis is worse than any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.	Poor Outcome	Supports	1	3	2	11	218	116	2015-02-24 20:43:19.4545	2015-02-24 20:43:19.4545	3	accepted	\N	\N	1	N/A
379	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	Poor Outcome	Supports	1	3	2	11	219	117	2015-02-24 20:43:19.516339	2015-02-24 20:43:19.516339	3	accepted	\N	\N	1	N/A
380	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	Poor Outcome	Supports	1	3	2	11	219	118	2015-02-24 20:43:19.583245	2015-02-24 20:43:19.583245	3	accepted	\N	\N	1	N/A
381	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	1	3	2	11	219	119	2015-02-24 20:43:19.644566	2015-02-24 20:43:19.644566	3	accepted	\N	\N	1	N/A
382	Breast cancer patients who harbor R273C mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	1	3	2	11	218	119	2015-02-24 20:43:19.711194	2015-02-24 20:43:19.711194	3	accepted	\N	\N	1	N/A
383	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	1	3	2	11	219	120	2015-02-24 20:43:19.773093	2015-02-24 20:43:19.773093	3	accepted	\N	\N	1	N/A
384	Breast cancer patients who harbor R273H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	1	3	2	11	218	120	2015-02-24 20:43:19.842376	2015-02-24 20:43:19.842376	3	accepted	\N	\N	1	N/A
385	Breast tumor with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.	Poor Outcome	Supports	1	3	2	11	219	121	2015-02-24 20:43:19.909364	2015-02-24 20:43:19.909364	3	accepted	\N	\N	1	N/A
386	Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.	Sensitivity	Supports	3	2	65	11	220	114	2015-02-24 20:43:19.979737	2015-02-24 20:43:19.979737	3	accepted	\N	\N	1	N/A
387	A patient with metastatic bladder cancer that responded well to the mTOR inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del, p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations also had mild responses to everolimus.	Sensitivity	Supports	3	3	66	43	221	122	2015-02-24 20:43:20.052954	2015-02-24 20:43:20.052954	3	accepted	\N	\N	1	N/A
388	Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit increased overall survival, increased apoptosis, reduced proliferation and reduced tumor burden compared to Tsc1 wildtype mice.	Sensitivity	Supports	3	2	62	5	222	122	2015-02-24 20:43:20.174742	2015-02-24 20:43:20.174742	3	accepted	\N	\N	1	N/A
389	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	Sensitivity	Supports	3	3	66	23	223	123	2015-02-24 20:43:20.246811	2015-02-24 20:43:20.246811	3	accepted	\N	\N	1	N/A
390	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	Sensitivity	Supports	3	3	66	23	221	124	2015-02-24 20:43:20.317901	2015-02-24 20:43:20.317901	3	accepted	\N	\N	1	N/A
391	In patients with AML, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	N / A	Does Not Support	1	4	2	2	224	125	2015-02-24 20:43:20.388657	2015-02-24 20:43:20.388657	2	accepted	\N	\N	1	N/A
392	In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	N / A	Does Not Support	1	4	2	28	224	125	2015-02-24 20:43:20.464814	2015-02-24 20:43:20.464814	2	accepted	\N	\N	1	N/A
393	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	N / A	Does Not Support	1	4	2	2	224	126	2015-02-24 20:43:20.527015	2015-02-24 20:43:20.527015	2	accepted	\N	\N	1	N/A
394	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	N / A	Does Not Support	1	4	2	28	224	126	2015-02-24 20:43:20.59007	2015-02-24 20:43:20.59007	2	accepted	\N	\N	1	N/A
395	After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.	Poor Outcome	Supports	1	4	2	28	225	125	2015-02-24 20:43:20.665916	2015-02-24 20:43:20.665916	3	accepted	\N	\N	1	N/A
396	After adjust for age and cancer stage, the presence of U2AF mutations such as S34Y/F are prognostic for poorer survival outcomes in patients with MDS.	Poor Outcome	Supports	1	4	2	28	225	126	2015-02-24 20:43:20.727754	2015-02-24 20:43:20.727754	3	accepted	\N	\N	1	N/A
397	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.	Positive	Does Not Support	2	4	2	2	224	125	2015-02-24 20:43:20.787688	2015-02-24 20:43:20.787688	3	accepted	\N	\N	1	N/A
398	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.	Positive	Does Not Support	2	4	2	2	224	126	2015-02-24 20:43:20.851757	2015-02-24 20:43:20.851757	3	accepted	\N	\N	1	N/A
399	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML	N / A	Does Not Support	1	3	2	2	226	127	2015-02-24 20:43:20.911069	2015-02-24 20:43:20.911069	4	accepted	\N	\N	1	N/A
400	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	N / A	Does Not Support	1	3	2	2	226	127	2015-02-24 20:43:20.971027	2015-02-24 20:43:20.971027	3	accepted	\N	\N	1	N/A
401	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML.	N / A	Does Not Support	1	3	2	2	226	128	2015-02-24 20:43:21.033449	2015-02-24 20:43:21.033449	4	accepted	\N	\N	1	N/A
402	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	N / A	Does Not Support	1	3	2	2	226	128	2015-02-24 20:43:21.097806	2015-02-24 20:43:21.097806	3	accepted	\N	\N	1	N/A
403	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	1	3	2	2	227	127	2015-02-24 20:43:21.1637	2015-02-24 20:43:21.1637	3	accepted	\N	\N	1	N/A
404	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	1	3	2	2	227	127	2015-02-24 20:43:21.229598	2015-02-24 20:43:21.229598	3	accepted	\N	\N	1	N/A
405	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	Poor Outcome	Supports	1	3	2	2	228	127	2015-02-24 20:43:21.291762	2015-02-24 20:43:21.291762	3	accepted	\N	\N	1	N/A
406	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	Poor Outcome	Supports	1	3	2	2	226	127	2015-02-24 20:43:21.349853	2015-02-24 20:43:21.349853	3	accepted	\N	\N	1	N/A
407	Mutations in WT1 were associated with a worse overall prognosis than patients wildtype for WT1 in young patients (15-50), primarily because of increased risk of disease recurrence	Poor Outcome	Supports	1	3	2	2	229	127	2015-02-24 20:43:21.414565	2015-02-24 20:43:21.414565	2	accepted	\N	\N	1	N/A
408	Mutations  in WT1 were associated with increased risk of recurrence in young patients (15-50) with cytogenetically normal AML	Poor Outcome	Supports	1	3	2	2	229	127	2015-02-24 20:43:21.484385	2015-02-24 20:43:21.484385	2	accepted	\N	\N	1	N/A
409	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	1	3	2	2	227	128	2015-02-24 20:43:21.590262	2015-02-24 20:43:21.590262	3	accepted	\N	\N	1	N/A
410	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	1	3	2	2	227	128	2015-02-24 20:43:21.660399	2015-02-24 20:43:21.660399	3	accepted	\N	\N	1	N/A
411	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	Poor Outcome	Supports	1	3	2	2	228	128	2015-02-24 20:43:21.723734	2015-02-24 20:43:21.723734	3	accepted	\N	\N	1	N/A
412	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	Poor Outcome	Supports	1	3	2	2	226	128	2015-02-24 20:43:21.789303	2015-02-24 20:43:21.789303	3	accepted	\N	\N	1	N/A
413	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	Resistance or Non-Response	Supports	3	3	2	2	227	127	2015-02-24 20:43:21.858536	2015-02-24 20:43:21.858536	3	accepted	\N	\N	1	N/A
414	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	Resistance or Non-Response	Supports	3	3	2	2	227	128	2015-02-24 20:43:21.922405	2015-02-24 20:43:21.922405	3	accepted	\N	\N	1	N/A
415	The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation positive women with high grade serous or undifferentiated ovarian cancer.	Sensitivity	Supports	3	3	67	44	230	129	2015-02-24 20:43:21.996612	2015-02-24 20:43:21.996612	4	accepted	\N	\N	2	N/A
416	The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation positive women with high grade serous or undifferentiated ovarian cancer.	Sensitivity	Supports	3	3	67	44	230	130	2015-02-24 20:43:22.074088	2015-02-24 20:43:22.074088	4	accepted	\N	\N	2	N/A
\.


--
-- Name: evidence_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_items_id_seq', 416, true);


--
-- Data for Name: evidence_levels; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_levels (id, level, description, created_at, updated_at) FROM stdin;
1	A	\N	2015-02-24 20:42:52.789496	2015-02-24 20:42:52.789496
2	C	\N	2015-02-24 20:42:52.891591	2015-02-24 20:42:52.891591
3	B	\N	2015-02-24 20:42:53.063839	2015-02-24 20:42:53.063839
4	D	\N	2015-02-24 20:42:56.524493	2015-02-24 20:42:56.524493
\.


--
-- Name: evidence_levels_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_levels_id_seq', 4, true);


--
-- Data for Name: evidence_types; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_types (id, evidence_type, created_at, updated_at) FROM stdin;
1	Prognostic	2015-02-24 20:42:52.786612	2015-02-24 20:42:52.786612
2	Diagnostic	2015-02-24 20:42:52.889742	2015-02-24 20:42:52.889742
3	Predictive	2015-02-24 20:42:53.062123	2015-02-24 20:42:53.062123
\.


--
-- Name: evidence_types_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_types_id_seq', 3, true);


--
-- Data for Name: genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes (id, entrez_id, name, description, official_name, created_at, updated_at, clinical_description) FROM stdin;
1	238	ALK	ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines.	anaplastic lymphoma receptor tyrosine kinase	2015-02-24 20:42:38.628909	2015-02-24 20:42:38.628909	\N
2	207	AKT1	AKT1, also referred to as protein kinase B, is a known oncogene. AKT activation relies on the PI3K pathway, and is recognized as a critical node in the pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been shown to result in activation of the protein. Mutations in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors in vitro.	v-akt murine thymoma viral oncogene homolog 1	2015-02-24 20:42:38.864475	2015-02-24 20:42:38.864475	\N
3	369	ARAF	ARAF has recently become increasingly considered for its oncogenic potential. Its potential as a target for informing clinical action was demonstrated by a single case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated with sorafenib, acheived near-complete clinical remission. This finding has brought new focus on ARAF as a marker that should be assayed for in cancer treatment.	A-Raf proto-oncogene, serine/threonine kinase	2015-02-24 20:42:39.066505	2015-02-24 20:42:39.066505	\N
4	25	ABL1	ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein that has become a signature of chronic myeloid leukemia (CML). Cells harboring this fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook of the disease. However, additional mutations in ABL1 have been shown to confer resistance to imatinib. In these resistance cases, second-generation tyrosine kinase inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are currently undergoing clinical trials for treating acquired resistance in CML.	ABL proto-oncogene 1, non-receptor tyrosine kinase	2015-02-24 20:42:39.276906	2015-02-24 20:42:39.276906	\N
5	673	BRAF	BRAF mutations are found to be recurrent in many cancer types. Of these, the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E has been determined to be an activating mutation, and cells that harbor it, along with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations have also been correlated with poor prognosis in many cancer types, although there is at least one study that questions this conclusion in papillary thyroid cancer.	B-Raf proto-oncogene, serine/threonine kinase	2015-02-24 20:42:39.486626	2015-02-24 20:42:39.486626	\N
6	672	BRCA1	N/A	breast cancer 1, early onset	2015-02-24 20:42:39.712912	2015-02-24 20:42:39.712912	\N
7	675	BRCA2	N/A	breast cancer 2, early onset	2015-02-24 20:42:39.904054	2015-02-24 20:42:39.904054	\N
8	595	CCND1	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D1	2015-02-24 20:42:40.101841	2015-02-24 20:42:40.101841	\N
9	894	CCND2	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D2	2015-02-24 20:42:40.306848	2015-02-24 20:42:40.306848	\N
10	896	CCND3	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. In a mouse study, the targeted therapeutic palbociclib significantly increased the median survival of the cyclin D3 knockouts.	cyclin D3	2015-02-24 20:42:40.543737	2015-02-24 20:42:40.543737	\N
11	898	CCNE1	Cyclin E, while currenly not as widely implicated as its cyclin D counterparts, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements are correlated with later stage disease. In lung cancer, neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly expressed counterparts.	cyclin E1	2015-02-24 20:42:40.770978	2015-02-24 20:42:40.770978	\N
12	1021	CDK6	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 6	2015-02-24 20:42:40.97138	2015-02-24 20:42:40.97138	\N
13	1019	CDK4	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 4	2015-02-24 20:42:41.174747	2015-02-24 20:42:41.174747	\N
14	1029	CDKN2A	CDKN2A loss has been shown to be a significant event in a number of cancer types. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. One mechanism by which loss of CDKN2A can occur is by hypermethylation of the promoter region for the gene. However, the prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have suggesting poorer prognostic outcome for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	cyclin-dependent kinase inhibitor 2A	2015-02-24 20:42:41.374939	2015-02-24 20:42:41.374939	\N
15	1050	CEBPA	AML with mutated CEBPA' is a provisional entity in the WHO classification of acute myeloid leukemia (AML) and is recommended to be tested for in patients with AML. CEBPA mutations are particularly associated with cytogenetically normal AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha protein while C-terminal mutations reduce the DNA-binding potential of this transcription factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	CCAAT/enhancer binding protein (C/EBP), alpha	2015-02-24 20:42:41.592475	2015-02-24 20:42:41.592475	\N
16	3337	DNAJB1	DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	DnaJ (Hsp40) homolog, subfamily B, member 1	2015-02-24 20:42:41.801725	2015-02-24 20:42:41.801725	\N
17	5566	PRKACA	PRKACA has been studied in breast cancer and has been found to mediate resistance to HER2 targeted therapies. It has also been found to contain a mutation hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	protein kinase, cAMP-dependent, catalytic, alpha	2015-02-24 20:42:42.000643	2015-02-24 20:42:42.000643	\N
18	1788	DNMT3A	DNMT3A is one of several epigenetic modifiers identified as recurrently mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers. AML patient bone marrow harboring R882 mutations were similarly demonstrated to be hypomethylated compared to patients with wildtype DNMT3A. These studies also indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes and treatment protocols may need to be considered for these two classes of DNMT3A mutations.	DNA (cytosine-5-)-methyltransferase 3 alpha	2015-02-24 20:42:42.209102	2015-02-24 20:42:42.209102	\N
19	1956	EGFR	EGFR is widely recognized for its importance in cancer. Amplification and mutations have been shown to be driving events in many cancer types. Its role in non-small cell lung cancer has spurred many research and drug development efforts. Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance to these drugs, particularly the variant T790M, which has been functionally characterized as a resistance marker for both of these drugs. The later generation TKI's have seen some success in treating these resistant cases, and targeted sequencing of the EGFR locus has become a common practice in non-small cell lung cancer.	epidermal growth factor receptor	2015-02-24 20:42:42.421507	2015-02-24 20:42:42.421507	\N
20	2064	ERBB2	ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast cancer, and is treated in a separate manner than the other subtypes of breast cancer. Apart from being amplified, ERBB2 activating mutations have been shown to have clinical importance in HER2-negative breast cancer. These mutations have shown sensitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical sequencing efforts in treating breast cancer.	erb-b2 receptor tyrosine kinase 2	2015-02-24 20:42:42.624803	2015-02-24 20:42:42.624803	\N
21	2099	ESR1	ESR1 has been a focus in breast cancer for quite some time, but has also shown significance in endometrial, ovarian and other cancer types. ER-positive breast cancer that is resistant to hormone therapy has instigated clinical sequencing efforts to shed light on the mechanisms of this resistance. A number of mutations in the ligand binding domain of the protein have been implicated in hormone resistance and anti-estrogen therapies. This has spurred efforts to develop therapeutics that act to degrade the protein, rather than act as an antagonist. These agents are currently in clinical trials and have seen some success, highlighting the importance of sequencing efforts in treating breast cancer.	estrogen receptor 1	2015-02-24 20:42:42.855132	2015-02-24 20:42:42.855132	\N
22	2263	FGFR2	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions that lead to pathway actiation have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 2	2015-02-24 20:42:43.064683	2015-02-24 20:42:43.064683	\N
23	2261	FGFR3	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions that lead to pathway actiation have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 3	2015-02-24 20:42:43.279563	2015-02-24 20:42:43.279563	\N
24	2322	FLT3	FLT3 is an important cytokine receptor involved in normal hematopoiesis. Mutations in this gene are common in acute myeloid leukemia (AML) and screening for mutations in this gene has been recommended by the World Health Organization in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML). FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD mutations are still up for debate.	fms-related tyrosine kinase 3	2015-02-24 20:42:43.495836	2015-02-24 20:42:43.495836	\N
25	2624	GATA2	GATA2 is a transcription factor involved in stem cell maintenance with key roles in hematopoietic development. GATA2 mutations are associated with a variety of inherited and acquired immune disorders including myelodysplastic syndrome and acute myeloid leukemia. In addition to a role in hematopoiesis, the maintenance GATA2 expression has been implicated as a requirement in KRAS-driven non-small cell lung cancer. Preclinical models have indicated therapeutic benefit from targeting GATA2-mediated pathways in the context of KRAS-driven NSCLC.	GATA binding protein 2	2015-02-24 20:42:43.693647	2015-02-24 20:42:43.693647	\N
26	3417	IDH1	IDH1 mutations have been observed in a number of cancer types, including sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described as mutually exclusive in many of these cancer types. The most frequent mutations involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic enzyme activity. The implications of mutations in this gene vary greatly by cancer type. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations have been associated with worse outcome, shorter overall survival, and normal karyotype. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Unlike the association with cytogenetically normal AML, in glioblastoma, IDH1 mutations have been associated with specific cytogenetic abnormalities, 1p and 19q deletions.	isocitrate dehydrogenase 1 (NADP+), soluble	2015-02-24 20:42:43.905147	2015-02-24 20:42:43.905147	\N
27	3418	IDH2	IDH2 mutations have been observed in a number of cancer types, including sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described as mutually exclusive in many of these cancer types. The most frequent mutations involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic enzyme activity. Although IDH2 (R172) mutations are associated with poorer overall prognosis in AML patients, its utility as a prognostic marker in MDS is still under debate. Additionally, IDH2 (R140) has been associated with improved overall survival in AML. IDH2 mutations have been associated with improved prognosis in gliomas.	isocitrate dehydrogenase 2 (NADP+), mitochondrial	2015-02-24 20:42:44.143356	2015-02-24 20:42:44.143356	\N
28	3717	JAK2	JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative diseases and cancers. The mutation V617F is the most clinically relevant variant, and is seen in around half of myeloproliferative disorders. The variant is a known activating mutation, and activated JAK2 is sufficient to drive myeloproliferative disorders in mouse models. V617F, while most recurrent, is not the only mechanism by which JAK2 can be activated in patients. JAK2 is now one of the first diagnostic markers tested upon diagnosis with a myeloproliferative disorder.	Janus kinase 2	2015-02-24 20:42:44.363693	2015-02-24 20:42:44.363693	\N
29	3815	KIT	N/A	v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog	2015-02-24 20:42:44.559984	2015-02-24 20:42:44.559984	\N
30	3845	KRAS	N/A	Kirsten rat sarcoma viral oncogene homolog	2015-02-24 20:42:44.749195	2015-02-24 20:42:44.749195	\N
31	5604	MAP2K1	N/A	mitogen-activated protein kinase kinase 1	2015-02-24 20:42:44.973978	2015-02-24 20:42:44.973978	\N
32	4209	MEF2D	N/A	myocyte enhancer factor 2D	2015-02-24 20:42:45.159023	2015-02-24 20:42:45.159023	\N
33	1436	CSF1R	N/A	colony stimulating factor 1 receptor	2015-02-24 20:42:45.348682	2015-02-24 20:42:45.348682	\N
34	4255	MGMT	N/A	O-6-methylguanine-DNA methyltransferase	2015-02-24 20:42:45.543107	2015-02-24 20:42:45.543107	\N
35	4869	NPM1	AML with mutated NPM1 is provisional entity in the WHO classification of AML and is recommended to be tested for in patients with cytogenetically normal AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia, indicating it may require additional mutations to promote leukemic development.	nucleophosmin (nucleolar phosphoprotein B23, numatrin)	2015-02-24 20:42:45.739152	2015-02-24 20:42:45.739152	\N
36	4893	NRAS	N/A	neuroblastoma RAS viral (v-ras) oncogene homolog	2015-02-24 20:42:45.944393	2015-02-24 20:42:45.944393	\N
37	5290	PIK3CA	PIK3CA is the most recurrently mutated gene in breast cancer, and has been found to important in a number of cancer types. An integral part of the PI3K pathway, PIK3CA has long been described as an oncogene, with two main hotspots for activating mutations, the 542/545 region of the helical domain, and the 1047 region of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the subject of an immense amount of research and development, and PI3K inhibition has seen some limited success in recent clinical trials. While monotherapies seem to be limited in their potential, there is a recent interest in pursuing PI3K inhibition as part of a combination therapy regiment with inhibition partners including TKI's, MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.	phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha	2015-02-24 20:42:46.143487	2015-02-24 20:42:46.143487	\N
38	5156	PDGFRA	N/A	platelet-derived growth factor receptor, alpha polypeptide	2015-02-24 20:42:46.362602	2015-02-24 20:42:46.362602	\N
39	5371	PML	The PML-RARA fusion is the result of a recurrent, balanced translocation between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	promyelocytic leukemia	2015-02-24 20:42:46.565265	2015-02-24 20:42:46.565265	\N
40	5914	RARA	The PML-RARA fusion is the result of a recurrent, balanced translocation between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	retinoic acid receptor, alpha	2015-02-24 20:42:46.759141	2015-02-24 20:42:46.759141	\N
41	5728	PTEN	N/A	phosphatase and tensin homolog	2015-02-24 20:42:46.95155	2015-02-24 20:42:46.95155	\N
42	5979	RET	RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine kinase receptor and are associated with thyroid cancers. RET point mutations are the most common mutations identified in medullary thyroid cancer (MTC) with germline and somatic mutations in RET associated with hereditary and sporadic forms, respectively. The most common somatic form mutation is M918T (exon 16) and a variety of other mutations effecting exons 10, 11 and 15 have been described. The prognostic significance of these mutations have been hotly debated in the field, however, data suggests that some RET mutation may confer drug resistence. No RET-specific agents are currently clinically available but several promiscuous kinase inhibitors that target RET, among others, have been approved for MTC treatment.	ret proto-oncogene	2015-02-24 20:42:47.147294	2015-02-24 20:42:47.147294	\N
43	861	RUNX1	N/A	runt-related transcription factor 1	2015-02-24 20:42:47.349241	2015-02-24 20:42:47.349241	\N
44	23451	SF3B1	SF3B1 mutations have been described in several myeloid malignancies, predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has been identified as recurrently mutated in MDS and other malignanices. The mutations affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally deleterious with K700E described as a major hotspot mutation. MDS patients with SF3B1 mutations have been reported to have better overall and event-free survival than their wildtype counterparts. Additionally, these mutations are highly associated with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring sideroblasts).	splicing factor 3b, subunit 1, 155kDa	2015-02-24 20:42:47.543864	2015-02-24 20:42:47.543864	\N
45	7157	TP53	TP53 mutations are universal across cancer types. Loss of tumor suppressors is most recognized by large deleterious events, such as frameshift mutations, or premature stop codons. In TP53 however, many of the observed mutations in cancer are found to be single nucleotide variants, or missense mutations. These variants are also very broadly distributed throughout the gene, not localizing in any particular hotspot. While a large proportion of cancer genomics research is focused on somatic variants, TP53 is also of note in the germline. Germline TP53 mutations are the hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been found to have prognostic impact on patient outcomes.	tumor protein p53	2015-02-24 20:42:47.791928	2015-02-24 20:42:47.791928	\N
46	7248	TSC1	N/A	tuberous sclerosis 1	2015-02-24 20:42:47.990457	2015-02-24 20:42:47.990457	\N
47	7249	TSC2	N/A	tuberous sclerosis 2	2015-02-24 20:42:48.193168	2015-02-24 20:42:48.193168	\N
48	7307	U2AF1	U2AF1 is one of several spliceosome complex genes frequently mutated in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein. These mutations have been associated with altered splicing patterns in vitro and in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation to secondary acute myeloid leukemia, however, the impact of these mutations on overall survival has been an area of debate.	U2 small nuclear RNA auxiliary factor 1	2015-02-24 20:42:48.391012	2015-02-24 20:42:48.391012	\N
49	7490	WT1	WT1 is a tumor suppressor gene associated with the development of Wilms' Tumor, from which it was named. Mutations in exon 7 and 9 of WT1 have been recurrently identified in acute myeloid leukemia and associated with poorer prognosis and chemotherapy resistance.	Wilms tumor 1	2015-02-24 20:42:48.632696	2015-02-24 20:42:48.632696	\N
\.


--
-- Name: genes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('genes_id_seq', 49, true);


--
-- Data for Name: genes_sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes_sources (gene_id, source_id, created_at, updated_at) FROM stdin;
1	1	2015-02-24 20:42:38.635266	2015-02-24 20:42:38.635266
1	2	2015-02-24 20:42:38.640179	2015-02-24 20:42:38.640179
2	3	2015-02-24 20:42:38.870336	2015-02-24 20:42:38.870336
2	4	2015-02-24 20:42:38.875365	2015-02-24 20:42:38.875365
3	5	2015-02-24 20:42:39.072978	2015-02-24 20:42:39.072978
3	6	2015-02-24 20:42:39.078281	2015-02-24 20:42:39.078281
4	7	2015-02-24 20:42:39.282305	2015-02-24 20:42:39.282305
4	8	2015-02-24 20:42:39.286811	2015-02-24 20:42:39.286811
5	9	2015-02-24 20:42:39.49277	2015-02-24 20:42:39.49277
5	10	2015-02-24 20:42:39.499128	2015-02-24 20:42:39.499128
8	11	2015-02-24 20:42:40.106252	2015-02-24 20:42:40.106252
8	12	2015-02-24 20:42:40.111172	2015-02-24 20:42:40.111172
9	11	2015-02-24 20:42:40.311006	2015-02-24 20:42:40.311006
9	12	2015-02-24 20:42:40.31482	2015-02-24 20:42:40.31482
10	11	2015-02-24 20:42:40.547218	2015-02-24 20:42:40.547218
10	12	2015-02-24 20:42:40.551453	2015-02-24 20:42:40.551453
11	13	2015-02-24 20:42:40.776703	2015-02-24 20:42:40.776703
12	14	2015-02-24 20:42:40.977051	2015-02-24 20:42:40.977051
12	15	2015-02-24 20:42:40.981861	2015-02-24 20:42:40.981861
13	14	2015-02-24 20:42:41.177973	2015-02-24 20:42:41.177973
13	15	2015-02-24 20:42:41.181189	2015-02-24 20:42:41.181189
14	16	2015-02-24 20:42:41.380155	2015-02-24 20:42:41.380155
14	17	2015-02-24 20:42:41.385231	2015-02-24 20:42:41.385231
15	18	2015-02-24 20:42:41.59893	2015-02-24 20:42:41.59893
15	19	2015-02-24 20:42:41.603601	2015-02-24 20:42:41.603601
15	20	2015-02-24 20:42:41.608689	2015-02-24 20:42:41.608689
16	21	2015-02-24 20:42:41.806941	2015-02-24 20:42:41.806941
17	21	2015-02-24 20:42:42.004101	2015-02-24 20:42:42.004101
17	22	2015-02-24 20:42:42.009814	2015-02-24 20:42:42.009814
18	23	2015-02-24 20:42:42.214321	2015-02-24 20:42:42.214321
18	24	2015-02-24 20:42:42.21927	2015-02-24 20:42:42.21927
19	25	2015-02-24 20:42:42.425526	2015-02-24 20:42:42.425526
19	26	2015-02-24 20:42:42.429637	2015-02-24 20:42:42.429637
20	27	2015-02-24 20:42:42.631636	2015-02-24 20:42:42.631636
20	28	2015-02-24 20:42:42.636988	2015-02-24 20:42:42.636988
21	29	2015-02-24 20:42:42.859852	2015-02-24 20:42:42.859852
21	30	2015-02-24 20:42:42.864884	2015-02-24 20:42:42.864884
22	31	2015-02-24 20:42:43.068608	2015-02-24 20:42:43.068608
22	32	2015-02-24 20:42:43.072706	2015-02-24 20:42:43.072706
23	31	2015-02-24 20:42:43.281813	2015-02-24 20:42:43.281813
23	32	2015-02-24 20:42:43.284604	2015-02-24 20:42:43.284604
24	33	2015-02-24 20:42:43.499638	2015-02-24 20:42:43.499638
24	18	2015-02-24 20:42:43.502284	2015-02-24 20:42:43.502284
25	34	2015-02-24 20:42:43.698206	2015-02-24 20:42:43.698206
25	35	2015-02-24 20:42:43.703052	2015-02-24 20:42:43.703052
26	36	2015-02-24 20:42:43.914907	2015-02-24 20:42:43.914907
26	37	2015-02-24 20:42:43.919693	2015-02-24 20:42:43.919693
27	36	2015-02-24 20:42:44.146571	2015-02-24 20:42:44.146571
27	37	2015-02-24 20:42:44.150829	2015-02-24 20:42:44.150829
27	38	2015-02-24 20:42:44.155503	2015-02-24 20:42:44.155503
28	39	2015-02-24 20:42:44.368955	2015-02-24 20:42:44.368955
35	40	2015-02-24 20:42:45.743419	2015-02-24 20:42:45.743419
35	18	2015-02-24 20:42:45.745966	2015-02-24 20:42:45.745966
42	41	2015-02-24 20:42:47.153054	2015-02-24 20:42:47.153054
42	42	2015-02-24 20:42:47.158053	2015-02-24 20:42:47.158053
44	43	2015-02-24 20:42:47.548671	2015-02-24 20:42:47.548671
44	44	2015-02-24 20:42:47.553521	2015-02-24 20:42:47.553521
45	45	2015-02-24 20:42:47.797564	2015-02-24 20:42:47.797564
48	46	2015-02-24 20:42:48.396721	2015-02-24 20:42:48.396721
48	47	2015-02-24 20:42:48.401851	2015-02-24 20:42:48.401851
\.


--
-- Data for Name: notifications; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY notifications (id, subscription_id, user_id, subscribable_id, subscribable_type, content, url, acknowledged, delivered, created_at, updated_at) FROM stdin;
\.


--
-- Name: notifications_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('notifications_id_seq', 1, false);


--
-- Data for Name: ratings; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY ratings (id, value, evidence_item_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Name: ratings_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('ratings_id_seq', 1, false);


--
-- Data for Name: roles; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles (id, name, description, created_at, updated_at) FROM stdin;
\.


--
-- Name: roles_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('roles_id_seq', 1, false);


--
-- Data for Name: roles_users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles_users (role_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY schema_migrations (version) FROM stdin;
20150223170106
20140805184308
20141021192035
20141022180523
20141022195329
20141108234203
20141108234935
20141109000136
20141114222425
20150107205621
20150115231117
20150116220720
20150116230539
20150116230632
20150204192653
20150205000010
20150205205734
20150205210725
20150210191015
20150210204445
20150219202227
20150220160805
20150220182109
\.


--
-- Data for Name: sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY sources (id, pubmed_id, study_type, description, created_at, updated_at) FROM stdin;
1	24889366	\N	Rossi et al., 2014, Int. J. Oncol.	2015-02-24 20:42:38.632872	2015-02-24 20:43:22.732744
2	23401436	\N	Shaw et al., 2013, J. Clin. Oncol.	2015-02-24 20:42:38.63834	2015-02-24 20:43:23.162899
3	16095999	\N	Bellacosa et al., 2005, Adv. Cancer Res.	2015-02-24 20:42:38.867967	2015-02-24 20:43:23.574438
4	15023437	\N	Fresno Vara et al., 2004, Cancer Treat. Rev.	2015-02-24 20:42:38.873498	2015-02-24 20:43:24.000417
5	15676015	\N	Lee et al., 2005, APMIS	2015-02-24 20:42:39.069826	2015-02-24 20:43:24.409702
6	24569458	\N	Imielinski et al., 2014, J. Clin. Invest.	2015-02-24 20:42:39.076141	2015-02-24 20:43:24.933522
7	15719031	\N	Ren, 2005, Nat. Rev. Cancer	2015-02-24 20:42:39.280122	2015-02-24 20:43:25.349667
8	17457302	\N	Weisberg et al., 2007, Nat. Rev. Cancer	2015-02-24 20:42:39.285141	2015-02-24 20:43:25.773513
9	19724843	\N	Li et al., 2009, Oncol. Rep.	2015-02-24 20:42:39.490482	2015-02-24 20:43:26.149783
10	23594689	\N	Pakneshan et al., 2013, Pathology	2015-02-24 20:42:39.49641	2015-02-24 20:43:26.527203
11	12432268	\N	Diehl, Cancer Biol. Ther.	2015-02-24 20:42:40.104335	2015-02-24 20:43:26.91429
12	24387133	\N	Casimiro et al., 2014, Expert Opin Investig Drugs	2015-02-24 20:42:40.109281	2015-02-24 20:43:27.27041
13	14965268	\N	Mazumder et al., 2004, Curr Cancer Drug Targets	2015-02-24 20:42:40.774338	2015-02-24 20:43:27.758212
14	20370706	\N	Graf et al., 2010, Mini Rev Med Chem	2015-02-24 20:42:40.974795	2015-02-24 20:43:28.17277
15	24089445	\N	Sheppard et al., 2013, Clin. Cancer Res.	2015-02-24 20:42:40.980002	2015-02-24 20:43:29.110641
16	20473920	\N	Shima et al., 2011, Int. J. Cancer	2015-02-24 20:42:41.377817	2015-02-24 20:43:29.888366
17	23111194	\N	Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-24 20:42:41.383494	2015-02-24 20:43:30.276374
18	19357394	\N	Vardiman et al., 2009, Blood	2015-02-24 20:42:41.596416	2015-02-24 20:43:31.201642
19	11242107	\N	Pabst et al., 2001, Nat. Genet.	2015-02-24 20:42:41.601752	2015-02-24 20:43:31.618583
20	9012825	\N	Zhang et al., 1997, Proc. Natl. Acad. Sci. U.S.A.	2015-02-24 20:42:41.606589	2015-02-24 20:43:31.98325
21	24578576	\N	Honeyman et al., 2014, Science	2015-02-24 20:42:41.804734	2015-02-24 20:43:33.271075
22	24909179	\N	Moody et al., 2014, Oncogene	2015-02-24 20:42:42.007755	2015-02-24 20:43:33.707426
23	24167195	\N	Kim et al., 2013, Blood	2015-02-24 20:42:42.21213	2015-02-24 20:43:34.232284
24	24656771	\N	Russler-Germain et al., 2014, Cancer Cell	2015-02-24 20:42:42.217374	2015-02-24 20:43:34.647885
25	23953842	\N	Yewale et al., 2013, Biomaterials	2015-02-24 20:42:42.42377	2015-02-24 20:43:35.064892
26	18712184	\N	Charpidou et al., In Vivo	2015-02-24 20:42:42.428039	2015-02-24 20:43:36.115253
27	16947083	\N	Badache et al., 2006, J Mammary Gland Biol Neoplasia	2015-02-24 20:42:42.628295	2015-02-24 20:43:36.630894
28	10878580	\N	Yu et al., 2000, Bioessays	2015-02-24 20:42:42.635136	2015-02-24 20:43:38.280297
29	24185510	\N	Robinson et al., 2013, Nat. Genet.	2015-02-24 20:42:42.857668	2015-02-24 20:43:38.723869
30	22279420	\N	Mohibi et al., 2011, J Carcinog	2015-02-24 20:42:42.862642	2015-02-24 20:43:39.141406
31	18636142	\N	Katoh, 2008, Int. J. Oncol.	2015-02-24 20:42:43.066969	2015-02-24 20:43:39.555896
32	23558953	\N	Wu et al., 2013, Cancer Discov	2015-02-24 20:42:43.071099	2015-02-24 20:43:41.064836
33	12951584	\N	Stirewalt et al., 2003, Nat. Rev. Cancer	2015-02-24 20:42:43.497989	2015-02-24 20:43:41.474106
34	22541434	\N	Kumar et al., 2012, Cell	2015-02-24 20:42:43.696428	2015-02-24 20:43:41.904843
35	25619630	\N	Mir et al., 2015, Cancer Med	2015-02-24 20:42:43.701146	2015-02-24 20:43:42.38193
36	24065766	\N	Lu et al., 2013, Genes Dev.	2015-02-24 20:42:43.91216	2015-02-24 20:43:42.8268
37	19228619	\N	Yan et al., 2009, N. Engl. J. Med.	2015-02-24 20:42:43.917558	2015-02-24 20:43:43.277
38	22417203	\N	Patel et al., 2012, N. Engl. J. Med.	2015-02-24 20:42:44.153799	2015-02-24 20:43:43.635862
39	17133099	\N	Levine et al., 2007, Curr. Opin. Hematol.	2015-02-24 20:42:44.366814	2015-02-24 20:43:44.046845
40	23226219	\N	Chou et al., 2012, PLoS ONE	2015-02-24 20:42:45.741369	2015-02-24 20:43:44.400371
41	18073307	\N	Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-24 20:42:47.150348	2015-02-24 20:43:44.777937
42	25465739	\N	Perri et al., 2014, Crit. Rev. Oncol. Hematol.	2015-02-24 20:42:47.156177	2015-02-24 20:43:45.198799
43	21995386	\N	Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-24 20:42:47.546676	2015-02-24 20:43:45.721036
44	24136165	\N	Cazzola et al., 2013, Blood	2015-02-24 20:42:47.551782	2015-02-24 20:43:46.14503
45	20182602	\N	Olivier et al., 2010, Cold Spring Harb Perspect Biol	2015-02-24 20:42:47.79545	2015-02-24 20:43:47.709305
46	25311244	\N	Okeyo-Owuor et al., 2014, Leukemia	2015-02-24 20:42:48.394435	2015-02-24 20:43:48.104707
47	22158538	\N	Graubert et al., 2012, Nat. Genet.	2015-02-24 20:42:48.39991	2015-02-24 20:43:48.53977
48	20537386		An et al., 2010, Leuk. Res.	2015-02-24 20:42:52.780368	2015-02-24 20:43:48.961906
49	25212276		Schmidt et al., 2014, Leukemia	2015-02-24 20:42:52.887584	2015-02-24 20:43:49.376754
50	12476305		Nimmanapalli et al., 2002, Oncogene	2015-02-24 20:42:53.056949	2015-02-24 20:43:49.960887
51	15194504	Cell Line	Yamamoto et al., 2004, Biochem. Biophys. Res. Commun.	2015-02-24 20:42:53.132649	2015-02-24 20:43:50.418011
52	23888070		Beaver et al., 2013, Clin. Cancer Res.	2015-02-24 20:42:53.69826	2015-02-24 20:43:50.839661
53	18923525		George et al., 2008, Nature	2015-02-24 20:42:53.767585	2015-02-24 20:43:51.26522
54	20979473	Clinical Trial	Choi et al., 2010, N. Engl. J. Med.	2015-02-24 20:42:53.839587	2015-02-24 20:43:51.685536
55	21030459		Sasaki et al., 2010, Cancer Res.	2015-02-24 20:42:53.92337	2015-02-24 20:43:52.195354
56	22072639		Bresler et al., 2011, Sci Transl Med	2015-02-24 20:42:53.994137	2015-02-24 20:43:52.72053
57	21575866		Sakamoto et al., 2011, Cancer Cell	2015-02-24 20:42:54.330553	2015-02-24 20:43:54.222396
58	21948233		Heuckmann et al., 2011, Clin. Cancer Res.	2015-02-24 20:42:54.398813	2015-02-24 20:43:54.701896
59	24569458		Imielinski et al., 2014, J. Clin. Invest.	2015-02-24 20:42:54.717131	2015-02-24 20:43:55.120112
60	24586605	Retrospective study	Meckbach et al., 2014, PLoS ONE	2015-02-24 20:42:54.972845	2015-02-24 20:43:55.539339
61	24354346	Retrospective study	Walczyk et al., 2014, Clin. Endocrinol. (Oxf)	2015-02-24 20:42:55.117031	2015-02-24 20:43:56.161515
62	24594804	Metaanalysis	Chen et al., 2014, PLoS ONE	2015-02-24 20:42:55.200823	2015-02-24 20:43:56.580941
63	24388723	Retrospective study	Nagore et al., 2014, J. Am. Acad. Dermatol.	2015-02-24 20:42:55.276283	2015-02-24 20:43:56.998857
64	21594703	Retrospective study	Howell et al., 2011, Ann. Surg. Oncol.	2015-02-24 20:42:55.34855	2015-02-24 20:43:58.774589
65	24396464	Retrospective study	He et al., 2014, Oncol Lett	2015-02-24 20:42:55.419075	2015-02-24 20:43:59.192304
66	21166657	Meta analysis	Lee et al., 2011, Br. J. Dermatol.	2015-02-24 20:42:55.60308	2015-02-24 20:43:59.610046
67	24588959	Perspective study	Zhang et al., 2014, Diagn Pathol	2015-02-24 20:42:55.680246	2015-02-24 20:43:59.999059
68	24570209	Retrospective study	Crescenzi et al., 2014, Horm. Metab. Res.	2015-02-24 20:42:55.764729	2015-02-24 20:44:00.968523
69	24576830	In vitro study	Nissan et al., 2014, Cancer Res.	2015-02-24 20:42:55.906177	2015-02-24 20:44:01.386317
70	23325582	Clinical trial	Peeters et al., 2013, Clin. Cancer Res.	2015-02-24 20:42:55.972598	2015-02-24 20:44:01.807654
71	19001320	In vitro study	Di Nicolantonio et al., 2008, J. Clin. Oncol.	2015-02-24 20:42:56.094653	2015-02-24 20:44:02.222685
72	20619739	Retrospective study	De Roock et al., 2010, Lancet Oncol.	2015-02-24 20:42:56.288837	2015-02-24 20:44:03.809097
73	23524406	Case report	Rudin et al., 2013, J Thorac Oncol	2015-02-24 20:42:56.448106	2015-02-24 20:44:04.206245
74	21098728	In vitro study	Corcoran et al., 2010, Sci Signal	2015-02-24 20:42:56.520634	2015-02-24 20:44:04.625725
75	23020132	Clinical trial	Flaherty et al., 2012, N. Engl. J. Med.	2015-02-24 20:42:56.598842	2015-02-24 20:44:05.045424
76	23463675	Clinical trial	Ponti et al., 2013, J. Clin. Pathol.	2015-02-24 20:42:56.763708	2015-02-24 20:44:06.397182
77	23845441	In vitro study	Rad et al., 2013, Cancer Cell	2015-02-24 20:42:56.847348	2015-02-24 20:44:06.816155
78	23812671	In vitro study	Ji et al., 2013, Clin. Cancer Res.	2015-02-24 20:42:56.97725	2015-02-24 20:44:07.237543
79	22180495	In vivo study	Yang et al., 2012, Cancer Res.	2015-02-24 20:42:57.141616	2015-02-24 20:44:07.759987
80	24583796	Clinical trial	Menzies et al., 2014, Clin. Cancer Res.	2015-02-24 20:42:57.453151	2015-02-24 20:44:08.175013
81	23031422	Case report	Ponti et al., 2012, J Hematol Oncol	2015-02-24 20:42:57.582496	2015-02-24 20:44:08.609982
82	17070615		Gautschi et al., 2007, Lung Cancer	2015-02-24 20:42:57.828295	2015-02-24 20:44:10.517481
83	15961768		Arnold et al., 2005, J. Clin. Oncol.	2015-02-24 20:42:58.038555	2015-02-24 20:44:10.99378
84	16309541		Thomas et al., 2005, Int J Exp Pathol	2015-02-24 20:42:58.111645	2015-02-24 20:44:11.464166
85	17891190		Jares et al., 2007, Nat. Rev. Cancer	2015-02-24 20:42:58.186904	2015-02-24 20:44:11.874677
86	14612939		Oshimo et al., 2003, Int. J. Oncol.	2015-02-24 20:42:58.356117	2015-02-24 20:44:12.357047
87	10547574		Takano et al., 1999, J. Pathol.	2015-02-24 20:42:58.428427	2015-02-24 20:44:14.127561
88	23079656		Sawai et al., 2012, Cancer Cell	2015-02-24 20:42:58.691308	2015-02-24 20:44:14.54882
89	12432043		Keyomarsi et al., 2002, N. Engl. J. Med.	2015-02-24 20:42:58.751242	2015-02-24 20:44:14.965395
90	10224221		Donnellan et al., 1999, FASEB J.	2015-02-24 20:42:58.83657	2015-02-24 20:44:15.381681
91	23898052		Logan et al., 2013, Anticancer Res.	2015-02-24 20:42:59.046493	2015-02-24 20:44:15.8002
92	25002028		Vora et al., 2014, Cancer Cell	2015-02-24 20:42:59.121404	2015-02-24 20:44:16.21667
93	23111194		Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-24 20:42:59.33963	2015-02-24 20:44:16.633668
94	18450602	Retrospective study	Schlenk et al., 2008, N. Engl. J. Med.	2015-02-24 20:42:59.475516	2015-02-24 20:44:17.112469
95	19965647	Retrospective study	Burnett et al., 2010, Blood	2015-02-24 20:42:59.542609	2015-02-24 20:44:17.575683
96	14726504	Retrospective study	Fröhling et al., 2004, J. Clin. Oncol.	2015-02-24 20:42:59.613871	2015-02-24 20:44:18.099843
97	24186003		Lilljebjörn et al., 2014, Leukemia	2015-02-24 20:42:59.902868	2015-02-24 20:44:18.572716
98	24578576	Retrospective study	Honeyman et al., 2014, Science	2015-02-24 20:43:00.026534	2015-02-24 20:44:19.038663
99	23632886	Retrospective study	Gaidzik et al., 2013, Blood	2015-02-24 20:43:00.101177	2015-02-24 20:44:19.455594
100	22081665	Retrospective study	LaRochelle et al., 2011, Oncotarget	2015-02-24 20:43:00.166456	2015-02-24 20:44:20.92114
101	22490330	Retrospective study	Ribeiro et al., 2012, Blood	2015-02-24 20:43:00.234542	2015-02-24 20:44:22.38264
102	22291079	Retrospective study	Marcucci et al., 2012, J. Clin. Oncol.	2015-02-24 20:43:00.449003	2015-02-24 20:44:22.798547
103	21067377	Retrospective study	Ley et al., 2010, N. Engl. J. Med.	2015-02-24 20:43:00.574189	2015-02-24 20:44:23.219492
104	24512939	Retrospective study	El Ghannam et al., Blood Cells Mol. Dis.	2015-02-24 20:43:00.779107	2015-02-24 20:44:23.632231
105	24662454		Douillard et al., 2014, J Thorac Oncol	2015-02-24 20:43:02.135758	2015-02-24 20:44:24.05133
106	24736073		Lim et al., 2014, J Thorac Oncol	2015-02-24 20:43:02.206302	2015-02-24 20:44:25.484766
107	24729716		Li et al., 2014, Onco Targets Ther	2015-02-24 20:43:02.344511	2015-02-24 20:44:25.930908
108	24623981		Ding et al., 2014, Onco Targets Ther	2015-02-24 20:43:02.408096	2015-02-24 20:44:26.346976
109	25668228	Review	Denis et al., 2015, Clin. Chim. Acta	2015-02-24 20:43:02.473471	2015-02-24 20:44:26.765528
110	15728811		Kobayashi et al., 2005, N. Engl. J. Med.	2015-02-24 20:43:02.546589	2015-02-24 20:44:27.811666
111	24457318		Fukihara et al., 2014, Oncology	2015-02-24 20:43:02.616688	2015-02-24 20:44:28.281489
112	24585406		Li et al., 2014, Ann. Surg. Oncol.	2015-02-24 20:43:02.68074	2015-02-24 20:44:28.661736
113	24636847		Li et al., 2014, Lung Cancer	2015-02-24 20:43:02.748135	2015-02-24 20:44:29.063788
114	24658966		Ai et al., 2014, Amino Acids	2015-02-24 20:43:02.909245	2015-02-24 20:44:29.794056
115	23220880		Bose et al., 2013, Cancer Discov	2015-02-24 20:43:02.982092	2015-02-24 20:44:30.218517
116	24185510		Robinson et al., 2013, Nat. Genet.	2015-02-24 20:43:03.859598	2015-02-24 20:44:30.73965
117	24185512		Toy et al., 2013, Nat. Genet.	2015-02-24 20:43:04.017709	2015-02-24 20:44:32.765441
118	24550739	Case study	Borad et al., 2014, PLoS Genet.	2015-02-24 20:43:04.913894	2015-02-24 20:44:33.142716
119	23558953		Wu et al., 2013, Cancer Discov	2015-02-24 20:43:04.987873	2015-02-24 20:44:33.516387
120	17965322	Retrospective study	Bacher et al., 2008, Blood	2015-02-24 20:43:05.202619	2015-02-24 20:44:33.974671
121	21537333		Schnittger et al., 2011, Leukemia	2015-02-24 20:43:05.343092	2015-02-24 20:44:34.390008
122	17957027	Retrospective study	Gale et al., 2008, Blood	2015-02-24 20:43:05.434529	2015-02-24 20:44:34.913164
123	11290608	Retrospective study	Yamamoto et al., 2001, Blood	2015-02-24 20:43:05.50782	2015-02-24 20:44:36.733322
124	24801015		Port et al., 2014, Ann. Hematol.	2015-02-24 20:43:05.835476	2015-02-24 20:44:37.186145
125	11585760	Retrospective study	Whitman et al., 2001, Cancer Res.	2015-02-24 20:43:05.922339	2015-02-24 20:44:37.577006
126	17940205	Retrospective study	Whitman et al., 2008, Blood	2015-02-24 20:43:06.065282	2015-02-24 20:44:39.658015
127	15626738	In vitro study	Bagrintseva et al., 2005, Blood	2015-02-24 20:43:06.338862	2015-02-24 20:44:40.143211
128	19965647		Burnett et al., 2010, Blood	2015-02-24 20:43:06.442869	2015-02-24 20:44:40.552647
129	22368270	Clinical trial	Man et al., 2012, Blood	2015-02-24 20:43:06.802739	2015-02-24 20:44:40.939748
130	18230792	In vitro study	Zhang et al., 2008, J. Natl. Cancer Inst.	2015-02-24 20:43:06.864196	2015-02-24 20:44:41.391105
131	14726387	Clinical trial	Smith et al., 2004, Blood	2015-02-24 20:43:07.027049	2015-02-24 20:44:41.787088
132	12357354	In vitro study	Tse et al., 2002, Leukemia	2015-02-24 20:43:07.090049	2015-02-24 20:44:42.201644
133	22624710		Barbacid, 2012, Cancer Cell	2015-02-24 20:43:07.225062	2015-02-24 20:44:42.750448
134	19933982	Retrospective study	Dubbink et al., 2009, Neurology	2015-02-24 20:43:07.354724	2015-02-24 20:44:43.168205
135	20127344	Retrospective study	Bleeker et al., 2010, Acta Neuropathol.	2015-02-24 20:43:07.44801	2015-02-24 20:44:43.688991
136	20805365	Retrospective study	Schnittger et al., 2010, Blood	2015-02-24 20:43:07.517608	2015-02-24 20:44:44.39063
137	20368538	Retrospective study	Wagner et al., 2010, J. Clin. Oncol.	2015-02-24 20:43:07.579904	2015-02-24 20:44:44.941175
138	20538800	Retrospective study	Abbas et al., 2010, Blood	2015-02-24 20:43:07.64778	2015-02-24 20:44:45.3619
139	20494930	Retrospective study	Thol et al., 2010, Haematologica	2015-02-24 20:43:07.921251	2015-02-24 20:44:45.881956
140	20376086	Retrospective study	Ho et al., 2010, Leukemia	2015-02-24 20:43:07.996572	2015-02-24 20:44:46.299809
141	21596855		Green et al., 2011, Blood	2015-02-24 20:43:08.127365	2015-02-24 20:44:46.715686
142	20421455	Retrospective study	Thol et al., 2010, Blood	2015-02-24 20:43:08.189048	2015-02-24 20:44:47.133042
143	22616558		Zhou et al., 2012, Leuk. Lymphoma	2015-02-24 20:43:08.255975	2015-02-24 20:44:47.551131
144	22033490		Patnaik et al., 2012, Leukemia	2015-02-24 20:43:08.320275	2015-02-24 20:44:47.969753
145	21997850		Lin et al., 2012, Ann. Hematol.	2015-02-24 20:43:08.386202	2015-02-24 20:44:48.388872
146	16081687	Retrospective study	Levine et al., 2005, Blood	2015-02-24 20:43:08.631817	2015-02-24 20:44:48.805227
147	19287384	Retrospective study	Kilpivaara et al., 2009, Nat. Genet.	2015-02-24 20:43:08.810416	2015-02-24 20:44:49.224178
148	18394554	In vitro study	Wernig et al., 2008, Cancer Cell	2015-02-24 20:43:08.874532	2015-02-24 20:44:50.058795
149	16709929	Clinical trial	Kiladjian et al., 2006, Blood	2015-02-24 20:43:08.942076	2015-02-24 20:44:50.489325
150	23775962	Clinical trial	Hodi et al., 2013, J. Clin. Oncol.	2015-02-24 20:43:09.003803	2015-02-24 20:44:50.893896
151	16551858	Retrospective study	Wardelmann et al., 2006, Clin. Cancer Res.	2015-02-24 20:43:09.198734	2015-02-24 20:44:51.31162
152	12000708	Retrospective study	Corless et al., 2002, Am. J. Pathol.	2015-02-24 20:43:09.271316	2015-02-24 20:44:51.733404
153	15217946	Retrospective study	Boldrini et al., 2004, Clin. Cancer Res.	2015-02-24 20:43:09.33932	2015-02-24 20:44:52.14866
154	16384925	Retrospective study	Cairoli et al., 2006, Blood	2015-02-24 20:43:09.572273	2015-02-24 20:44:52.564195
155	10485475	Retrospective study	Taniguchi et al., 1999, Cancer Res.	2015-02-24 20:43:09.64481	2015-02-24 20:44:52.942813
156	19671763	In vitro study	Woodman et al., 2009, Mol. Cancer Ther.	2015-02-24 20:43:09.928595	2015-02-24 20:44:53.302835
157	16954519		Heinrich et al., 2006, J. Clin. Oncol.	2015-02-24 20:43:10.119732	2015-02-24 20:44:54.23595
158	18421059	Observational study	Hodi et al., 2008, J. Clin. Oncol.	2015-02-24 20:43:10.193855	2015-02-24 20:44:54.62012
159	17372901	In vitro study	Antonescu et al., 2007, Int. J. Cancer	2015-02-24 20:43:10.333888	2015-02-24 20:44:55.180183
160	16638875		Prenen et al., 2006, Clin. Cancer Res.	2015-02-24 20:43:10.51446	2015-02-24 20:44:55.595041
161	20978259	Clinical trial	De Roock et al., 2010, JAMA	2015-02-24 20:43:10.582028	2015-02-24 20:44:57.033573
162	19934290	Retrospective study	Ogino et al., 2009, Clin. Cancer Res.	2015-02-24 20:43:10.641281	2015-02-24 20:44:57.431355
163	11208838	Retrospective study	Schiller et al., 2001, J. Clin. Oncol.	2015-02-24 20:43:10.700579	2015-02-24 20:44:57.894334
164	18528420	Retrospective study	Chng et al., 2008, Leukemia	2015-02-24 20:43:10.76582	2015-02-24 20:44:58.308636
165	15597105	Metaanalysis	Mascaux et al., 2005, Br. J. Cancer	2015-02-24 20:43:10.826933	2015-02-24 20:44:58.728597
166	18202412	In vitro study	Lièvre et al., 2008, J. Clin. Oncol.	2015-02-24 20:43:10.885338	2015-02-24 20:44:59.088395
167	18794081	Retrospective study	Riely et al., 2008, Clin. Cancer Res.	2015-02-24 20:43:11.054898	2015-02-24 20:44:59.528523
168	24571676	Retrospective study	Andrade et al., 2014, BMC Cancer	2015-02-24 20:43:11.174249	2015-02-24 20:44:59.934194
169	23014527	Retrospective study	Dogan et al., 2012, Clin. Cancer Res.	2015-02-24 20:43:11.244974	2015-02-24 20:45:00.34221
170	16618717	Retrospective study	Lièvre et al., 2006, Cancer Res.	2015-02-24 20:43:11.436396	2015-02-24 20:45:00.802994
171	15696205	Retrospective study	Pao et al., 2005, PLoS Med.	2015-02-24 20:43:11.503414	2015-02-24 20:45:01.2358
172	23524406	case report	Rudin et al., 2013, J Thorac Oncol	2015-02-24 20:43:11.750008	2015-02-24 20:45:01.648942
173	24559322	Clinical trial	Sartore-Bianchi et al., 2014, Expert Rev Anticancer Ther	2015-02-24 20:43:11.880314	2015-02-24 20:45:02.029127
174	19029981	In vivo study	Engelman et al., 2008, Nat. Med.	2015-02-24 20:43:11.950772	2015-02-24 20:45:02.38368
175	19915144		Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-02-24 20:43:12.215991	2015-02-24 20:45:02.804639
176	19915144	In vitro study	Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-02-24 20:43:12.279815	2015-02-24 20:45:03.325477
177	11070098		Esteller et al., 2000, N. Engl. J. Med.	2015-02-24 20:43:12.462289	2015-02-24 20:45:03.741463
178	15758010	Clinical Trial	Hegi et al., 2005, N. Engl. J. Med.	2015-02-24 20:43:12.52479	2015-02-24 20:45:04.115733
179	16076867	Retrospective study	Schnittger et al., 2005, Blood	2015-02-24 20:43:12.771016	2015-02-24 20:45:04.459455
180	16051734	Retrospective study	Döhner et al., 2005, Blood	2015-02-24 20:43:12.831074	2015-02-24 20:45:04.890876
181	20026798	Retrospective study	Becker et al., 2010, J. Clin. Oncol.	2015-02-24 20:43:12.898691	2015-02-24 20:45:06.595122
182	24855211		Linch et al., 2014, Blood	2015-02-24 20:43:12.962346	2015-02-24 20:45:06.970774
183	16455956	Retrospective study	Thiede et al., 2006, Blood	2015-02-24 20:43:13.031712	2015-02-24 20:45:07.294373
184	19047294	Retrospective study	Büchner et al., 2009, J. Clin. Oncol.	2015-02-24 20:43:13.162432	2015-02-24 20:45:07.685925
185	22430270	Retrospective study	Gaidzik et al., 2012, J. Clin. Oncol.	2015-02-24 20:43:13.230858	2015-02-24 20:45:09.133935
186	22417203	Retrospective study	Patel et al., 2012, N. Engl. J. Med.	2015-02-24 20:43:13.300326	2015-02-24 20:45:09.545384
187	19587375	Retrospective study	Schnittger et al., 2009, Blood	2015-02-24 20:43:13.624513	2015-02-24 20:45:09.912066
188	24859829		Tian et al., 2014, Int. J. Hematol.	2015-02-24 20:43:13.887601	2015-02-24 20:45:11.15595
189	19059939	Retrospective study	Schlenk et al., 2009, Haematologica	2015-02-24 20:43:14.322039	2015-02-24 20:45:11.577193
190	19357394		Vardiman et al., 2009, Blood	2015-02-24 20:43:14.458284	2015-02-24 20:45:11.940703
191	21719597	In vitro study	Balusu et al., 2011, Blood	2015-02-24 20:43:14.72876	2015-02-24 20:45:12.412666
192	24797300		Tassara et al., 2014, Blood	2015-02-24 20:43:15.039506	2015-02-24 20:45:12.793033
193	24927407		Ehninger et al., 2014, Blood Cancer J	2015-02-24 20:43:15.098009	2015-02-24 20:45:13.538133
194	15659725		Falini et al., 2005, N. Engl. J. Med.	2015-02-24 20:43:15.299235	2015-02-24 20:45:14.739543
195	15951308	Retrospective study	Bowen et al., 2005, Blood	2015-02-24 20:43:15.369491	2015-02-24 20:45:15.336243
196	16434492	Clinical trial	Bacher et al., 2006, Blood	2015-02-24 20:43:15.446846	2015-02-24 20:45:16.38415
197	22180178	Clinical trial	Jakob et al., 2012, Cancer	2015-02-24 20:43:15.516975	2015-02-24 20:45:16.798454
198	24666267	Meta analysis	Therkildsen et al., 2014, Acta Oncol	2015-02-24 20:43:15.577857	2015-02-24 20:45:17.216928
199	23861977	Retrospective study	Tschandl et al., 2013, PLoS ONE	2015-02-24 20:43:15.883547	2015-02-24 20:45:18.801338
200	23569304	Clinical trial	Trunzer et al., 2013, J. Clin. Oncol.	2015-02-24 20:43:16.096287	2015-02-24 20:45:19.179115
201	18375819	Case report	Banerji et al., 2008, Mol. Cancer Ther.	2015-02-24 20:43:16.166908	2015-02-24 20:45:19.931027
202	21576590	Case report	Soon et al., 2011, Arch Dermatol	2015-02-24 20:43:16.240154	2015-02-24 20:45:21.820327
203	15146165		Lasota et al., 2004, Lab. Invest.	2015-02-24 20:43:16.369073	2015-02-24 20:45:22.229155
204	15928335		Corless et al., 2005, J. Clin. Oncol.	2015-02-24 20:43:16.51173	2015-02-24 20:45:22.647293
205	22745105		Heinrich et al., 2012, Clin. Cancer Res.	2015-02-24 20:43:16.582572	2015-02-24 20:45:24.735215
206	19223544		Sartore-Bianchi et al., 2009, Cancer Res.	2015-02-24 20:43:17.240732	2015-02-24 20:45:25.153928
207	22357840		Liao et al., 2012, Clin. Cancer Res.	2015-02-24 20:43:17.314415	2015-02-24 20:45:25.677121
208	21558396		Tanaka et al., 2011, Clin. Cancer Res.	2015-02-24 20:43:18.023034	2015-02-24 20:45:27.65983
209	15647370		Kang et al., 2005, Proc. Natl. Acad. Sci. U.S.A.	2015-02-24 20:43:18.093114	2015-02-24 20:45:28.186352
210	8674046		Yoshida et al., 1996, Cancer Res.	2015-02-24 20:43:18.401703	2015-02-24 20:45:29.855547
211	22479427		Carico et al., 2012, PLoS ONE	2015-02-24 20:43:18.540536	2015-02-24 20:45:30.273264
212	20085938		Courtney et al., 2010, J. Clin. Oncol.	2015-02-24 20:43:18.608107	2015-02-24 20:45:30.845609
213	9839497		Egawa et al., 1998, Jpn. J. Clin. Oncol.	2015-02-24 20:43:18.791506	2015-02-24 20:45:31.316809
214	18073307		Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-24 20:43:18.863906	2015-02-24 20:45:31.736167
215	21422803		Coxon et al., 2012, J. Endocrinol. Invest.	2015-02-24 20:43:18.929889	2015-02-24 20:45:32.164211
216	23056499		Couto et al., 2012, PLoS ONE	2015-02-24 20:43:19.069849	2015-02-24 20:45:33.658466
217	21995386		Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-24 20:43:19.134245	2015-02-24 20:45:34.049782
218	16489069		Olivier et al., 2006, Clin. Cancer Res.	2015-02-24 20:43:19.2641	2015-02-24 20:45:34.451448
219	9569050		Berns et al., 1998, Br. J. Cancer	2015-02-24 20:43:19.321653	2015-02-24 20:45:36.581109
220	22698404		Jackson et al., 2012, Cancer Cell	2015-02-24 20:43:19.976418	2015-02-24 20:45:36.940128
221	22923433		Iyer et al., 2012, Science	2015-02-24 20:43:20.048986	2015-02-24 20:45:37.375705
222	19966866	Mouse model	Liang et al., 2010, Oncogene	2015-02-24 20:43:20.170288	2015-02-24 20:45:37.79474
223	22923433	Retrospective study	Iyer et al., 2012, Science	2015-02-24 20:43:20.244128	2015-02-24 20:45:38.151249
224	23029227		Qian et al., 2012, PLoS ONE	2015-02-24 20:43:20.384089	2015-02-24 20:45:38.52252
225	23861105		Wu et al., 2013, Am. J. Hematol.	2015-02-24 20:43:20.663307	2015-02-24 20:45:38.939834
226	19221039	Retrospective study	Gaidzik et al., 2009, Blood	2015-02-24 20:43:20.908478	2015-02-24 20:45:40.922554
227	18591546	Retrospective study	Virappane et al., 2008, J. Clin. Oncol.	2015-02-24 20:43:21.161075	2015-02-24 20:45:41.343478
228	18559874	Retrospective study	Paschka et al., 2008, J. Clin. Oncol.	2015-02-24 20:43:21.289128	2015-02-24 20:45:41.755892
229	19536888	Retrospective study	Renneville et al., 2009, Cancer	2015-02-24 20:43:21.409988	2015-02-24 20:45:42.281378
230	23346317	Review	Goff, 2013, J Gynecol Oncol	2015-02-24 20:43:21.989773	2015-02-24 20:45:43.267925
\.


--
-- Name: sources_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('sources_id_seq', 230, true);


--
-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY users (id, email, name, url, nickname, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: subscriptions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY subscriptions (id, user_id, subscribable_id, subscribable_type, type, created_at, updated_at) FROM stdin;
\.


--
-- Name: subscriptions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('subscriptions_id_seq', 1, false);


--
-- Data for Name: suggested_changes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY suggested_changes (id, suggested_changes, moderated_id, moderated_type, user_id, status, created_at, updated_at) FROM stdin;
\.


--
-- Name: suggested_changes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('suggested_changes_id_seq', 1, false);


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('users_id_seq', 1, false);


--
-- Data for Name: variant_group_variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_group_variants (variant_id, variant_group_id, created_at, updated_at) FROM stdin;
3	1	2015-02-24 20:42:53.215855	2015-02-24 20:42:53.215855
2	1	2015-02-24 20:42:53.298383	2015-02-24 20:42:53.298383
2	1	2015-02-24 20:42:53.38126	2015-02-24 20:42:53.38126
6	2	2015-02-24 20:42:53.899627	2015-02-24 20:42:53.899627
6	3	2015-02-24 20:42:53.905112	2015-02-24 20:42:53.905112
8	3	2015-02-24 20:42:54.132194	2015-02-24 20:42:54.132194
7	2	2015-02-24 20:42:54.232491	2015-02-24 20:42:54.232491
7	3	2015-02-24 20:42:54.236407	2015-02-24 20:42:54.236407
5	2	2015-02-24 20:42:54.312257	2015-02-24 20:42:54.312257
11	4	2015-02-24 20:42:56.828588	2015-02-24 20:42:56.828588
13	4	2015-02-24 20:42:57.636521	2015-02-24 20:42:57.636521
15	4	2015-02-24 20:42:57.712166	2015-02-24 20:42:57.712166
16	4	2015-02-24 20:42:57.810081	2015-02-24 20:42:57.810081
34	5	2015-02-24 20:43:02.529286	2015-02-24 20:43:02.529286
35	6	2015-02-24 20:43:03.101622	2015-02-24 20:43:03.101622
36	6	2015-02-24 20:43:03.168259	2015-02-24 20:43:03.168259
38	6	2015-02-24 20:43:03.300725	2015-02-24 20:43:03.300725
41	6	2015-02-24 20:43:03.519234	2015-02-24 20:43:03.519234
43	6	2015-02-24 20:43:03.70023	2015-02-24 20:43:03.70023
44	6	2015-02-24 20:43:03.771	2015-02-24 20:43:03.771
45	6	2015-02-24 20:43:03.839549	2015-02-24 20:43:03.839549
46	7	2015-02-24 20:43:03.99781	2015-02-24 20:43:03.99781
46	7	2015-02-24 20:43:04.076141	2015-02-24 20:43:04.076141
47	7	2015-02-24 20:43:04.208647	2015-02-24 20:43:04.208647
47	7	2015-02-24 20:43:04.276168	2015-02-24 20:43:04.276168
48	7	2015-02-24 20:43:04.419354	2015-02-24 20:43:04.419354
48	7	2015-02-24 20:43:04.485104	2015-02-24 20:43:04.485104
49	7	2015-02-24 20:43:04.609654	2015-02-24 20:43:04.609654
49	7	2015-02-24 20:43:04.678567	2015-02-24 20:43:04.678567
50	7	2015-02-24 20:43:04.805126	2015-02-24 20:43:04.805126
50	7	2015-02-24 20:43:04.889408	2015-02-24 20:43:04.889408
51	8	2015-02-24 20:43:04.969745	2015-02-24 20:43:04.969745
52	8	2015-02-24 20:43:05.05005	2015-02-24 20:43:05.05005
53	8	2015-02-24 20:43:05.185148	2015-02-24 20:43:05.185148
65	9	2015-02-24 20:43:09.06116	2015-02-24 20:43:09.06116
69	10	2015-02-24 20:43:09.1824	2015-02-24 20:43:09.1824
65	9	2015-02-24 20:43:09.256019	2015-02-24 20:43:09.256019
65	9	2015-02-24 20:43:09.323428	2015-02-24 20:43:09.323428
65	9	2015-02-24 20:43:09.394814	2015-02-24 20:43:09.394814
64	10	2015-02-24 20:43:09.627234	2015-02-24 20:43:09.627234
65	9	2015-02-24 20:43:09.698142	2015-02-24 20:43:09.698142
65	9	2015-02-24 20:43:09.842642	2015-02-24 20:43:09.842642
69	10	2015-02-24 20:43:09.911355	2015-02-24 20:43:09.911355
71	9	2015-02-24 20:43:10.104035	2015-02-24 20:43:10.104035
72	9	2015-02-24 20:43:10.176834	2015-02-24 20:43:10.176834
66	9	2015-02-24 20:43:10.248185	2015-02-24 20:43:10.248185
71	9	2015-02-24 20:43:10.314133	2015-02-24 20:43:10.314133
71	9	2015-02-24 20:43:10.495175	2015-02-24 20:43:10.495175
72	9	2015-02-24 20:43:10.567159	2015-02-24 20:43:10.567159
86	11	2015-02-24 20:43:14.180786	2015-02-24 20:43:14.180786
86	11	2015-02-24 20:43:14.572197	2015-02-24 20:43:14.572197
86	11	2015-02-24 20:43:15.283632	2015-02-24 20:43:15.283632
86	11	2015-02-24 20:43:15.350033	2015-02-24 20:43:15.350033
97	1	2015-02-24 20:43:16.425851	2015-02-24 20:43:16.425851
97	1	2015-02-24 20:43:16.495326	2015-02-24 20:43:16.495326
97	1	2015-02-24 20:43:16.563251	2015-02-24 20:43:16.563251
96	1	2015-02-24 20:43:16.641925	2015-02-24 20:43:16.641925
97	1	2015-02-24 20:43:16.713493	2015-02-24 20:43:16.713493
98	1	2015-02-24 20:43:16.782148	2015-02-24 20:43:16.782148
99	1	2015-02-24 20:43:16.850731	2015-02-24 20:43:16.850731
100	1	2015-02-24 20:43:16.919088	2015-02-24 20:43:16.919088
108	12	2015-02-24 20:43:18.592376	2015-02-24 20:43:18.592376
108	12	2015-02-24 20:43:18.709413	2015-02-24 20:43:18.709413
110	13	2015-02-24 20:43:18.983227	2015-02-24 20:43:18.983227
111	13	2015-02-24 20:43:19.052216	2015-02-24 20:43:19.052216
122	14	2015-02-24 20:43:20.154774	2015-02-24 20:43:20.154774
122	14	2015-02-24 20:43:20.228056	2015-02-24 20:43:20.228056
123	14	2015-02-24 20:43:20.301216	2015-02-24 20:43:20.301216
124	14	2015-02-24 20:43:20.367686	2015-02-24 20:43:20.367686
129	15	2015-02-24 20:43:22.055088	2015-02-24 20:43:22.055088
130	15	2015-02-24 20:43:22.124372	2015-02-24 20:43:22.124372
\.


--
-- Data for Name: variant_groups; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_groups (id, name, description, created_at, updated_at) FROM stdin;
9	KIT Exon 11	\N	2015-02-24 20:43:09.058716	2015-02-24 20:43:09.058716
10	KIT Exon 17	\N	2015-02-24 20:43:09.179793	2015-02-24 20:43:09.179793
1	Imatinib Resistance	While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. 	2015-02-24 20:42:53.206175	2015-02-24 20:43:22.181391
2	ALK Fusions	ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase the activity of the ALK oncogene relative to normal cells. While EML4 is the most common fusion partner, other 5' partners have been observed. The EML4-ALK fusion has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. 	2015-02-24 20:42:53.896421	2015-02-24 20:43:22.185836
5	Erlotinib Resistance	EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine kinase inhibitors have been developed to target EGFR pathway activity. One such inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting. However, the T790M missense mutation has shown to confer resistance to this inhibitor in cell lines and case studies.  	2015-02-24 20:43:02.526558	2015-02-24 20:43:22.188807
3	Crizotinib Resistance	The ALK oncogene has long been considered a driving factor in non-small cell lung cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive this resistance. 	2015-02-24 20:42:53.902115	2015-02-24 20:43:22.191559
4	Other V600's	While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants have also been observed and studied to a lesser degree. At first approximation, many of these variants seem to behave similarly to V600E, and treatment with dabrafenib has been shown to be effective. 	2015-02-24 20:42:56.825856	2015-02-24 20:43:22.193764
6	HER2 Activating	HER2-positive breast cancer is widely recognized as a molecular subtype of breast cancer. While amplification of ERBB2 is a common mechanism for activation of the pathway, missense mutations have also demonstrated activation potential. These activating missense mutations also confer senstivity to the targeted therapeutic neratinib. 	2015-02-24 20:43:03.09863	2015-02-24 20:43:22.195908
7	ESR1 Ligand-Binding Domain	ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone therapy has been widely effective in treating the disease, however sequencing of resistant patients has uncovered a number of missense mutations in mediating this resistance. Many of these mutations lie within the ligand-binding domain, and contribute to constitutive activity of the receptor. This has lead to the development of estrogen receptor degrading agents such as fulvestrant, which have shown early efficacy in clinical trials.  	2015-02-24 20:43:03.993579	2015-02-24 20:43:22.198055
8	FGFR fusions	FGFR fusions have been demonstrated across many cancer types to have oncogenic potential. FGFR2 and FGFR3 have been shown to be fused to many 3' partners, which may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2 fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib and pazopanib have shown efficacy in case studies.  	2015-02-24 20:43:04.966883	2015-02-24 20:43:22.20118
11	NPM1 exon 12	To be filled in by Dr. KK, resident blood expert and knower of all things hematology. :) :) :) 	2015-02-24 20:43:14.177544	2015-02-24 20:43:22.203563
12	PTEN Loss-of-Function	PTEN loss is a common event in breast cancer, as well as others. The most common single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell lines harboring this mutation behave similarly to cell lines harboring larger insertion or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded growth. 	2015-02-24 20:43:18.589454	2015-02-24 20:43:22.206492
13	Motesanib Resistance	RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment of these patients with the targeted therapeutic motesanib has shown to be effective. However, the missense mutations C634W and M918T have shown to confer motesanib resistance in cell lines. 	2015-02-24 20:43:18.979693	2015-02-24 20:43:22.209721
14	TSC Loss	Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types. Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor everolimus has seen significant anti-tumorigenic activity. 	2015-02-24 20:43:20.151543	2015-02-24 20:43:22.213815
15	BRCA Germline Variants	BRCA germline variants have long been recognized as important potential predictors of breast and ovarian cancer risk. 	2015-02-24 20:43:22.051719	2015-02-24 20:43:22.216743
\.


--
-- Name: variant_groups_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_groups_id_seq', 15, true);


--
-- Name: variant_origins_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_origins_id_seq', 2, true);


--
-- Data for Name: variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variants (id, gene_id, name, description, created_at, updated_at) FROM stdin;
1	4	BCR-ABL	The BCR-ABL fusion protein, commonly referred to as the philadelphia chromosome, is the most well-studied fusion gene in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML), but despite its ability initiate disease in mice, its status an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and nilotinib) have seen some success in delivering a tumor response.	2015-02-24 20:42:48.686187	2015-02-24 20:42:48.686187
2	4	BCR-ABL T315I	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-02-24 20:42:48.725048	2015-02-24 20:42:48.725048
3	4	BCR-ABL E255K	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-02-24 20:42:48.753681	2015-02-24 20:42:48.753681
4	2	E17K	AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal, lung, and ovarian cancer. It has been convincingly shown to be an activating mutation resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and other AKT1 mutations, are the subject of much research and development for therapeutics.	2015-02-24 20:42:48.781192	2015-02-24 20:42:48.781192
5	1	EML4-ALK	The EML4-ALK fusion has been seen in non-small cell lung cancer, and appears to be an alternative mechanism for ALK activation. Cells with this fusion have been shown to be sensitive to the ALK inhibitor crizotinib.	2015-02-24 20:42:48.805397	2015-02-24 20:42:48.805397
6	1	EML4-ALK C1156Y	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-24 20:42:48.829034	2015-02-24 20:42:48.829034
7	1	EML4-ALK L1196M	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-24 20:42:48.851959	2015-02-24 20:42:48.851959
8	1	F1174L	ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells.	2015-02-24 20:42:48.881322	2015-02-24 20:42:48.881322
9	1	R1275Q	ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma, non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This, and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC cell lines.	2015-02-24 20:42:48.904803	2015-02-24 20:42:48.904803
10	3	S214C	ARAF S214C has been found to be a recurrent oncogenic mutation in non-small cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this mutation, sorafenib also acheived near-complete clinical remission. This case has brought more interest to the variant from a research and clinical perspective.	2015-02-24 20:42:48.934705	2015-02-24 20:42:48.934705
11	5	V600D	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-24 20:42:48.958765	2015-02-24 20:42:48.958765
12	5	V600E	BRAF V600E has been shown to be recurrent in many cancer types. It is one of the most widely studied variants in cancer. This variant is correlated with poor prognosis in certain cancer types, including colorectal cancer and papillary thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib has also shown to be effective when combined with the MEK inhibitor trametinib in colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib, and a number of combination therapies have been successful in treating V600E mutations. While the drugs cetuximab and panitumumab have been largely shown to be ineffective without supplementary treatment.	2015-02-24 20:42:48.986628	2015-02-24 20:42:48.986628
13	5	V600E/V600M	A case study of a single patient harboring both a V600E and a V600M mutation, dabrafenib was shown to acheive clinical response.	2015-02-24 20:42:49.020405	2015-02-24 20:42:49.020405
14	5	V600E AMPLIFICATION	Amplification of BRAF V600E has been shown to confer resistance to MEK inhibitors. For more information on the V600 locus, see the V600E entry.	2015-02-24 20:42:49.047706	2015-02-24 20:42:49.047706
15	5	V600M	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-24 20:42:49.07699	2015-02-24 20:42:49.07699
16	5	V600R	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-24 20:42:49.103457	2015-02-24 20:42:49.103457
17	5	V600	BRAF mutations of the valine 600 residue have been shown to be recurrent in many cancer types. Of the V600 mutations, V600E is the most widely researched. V600 mutations as a whole have been correlated to poorer prognosis in colorectal and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual mutation pages on the left sidebar.	2015-02-24 20:42:49.128126	2015-02-24 20:42:49.128126
18	8	AMPLIFICATION	CCND1 amplification has been implicated in poorer prognosis in non-small cell lung cancer.	2015-02-24 20:42:49.162209	2015-02-24 20:42:49.162209
19	8	EXPRESSION	CCND1 expression, and its prognositc impact, is still in dispute. Three experiments in non-small cell lung cancer have shown it to have no impact on survival, but three additional studies have shown it results in poorer prognosis. There is also some ambiguity in how the boundaries between expression and overexpression are defined.	2015-02-24 20:42:49.185484	2015-02-24 20:42:49.185484
96	38	D842I	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-24 20:42:51.524434	2015-02-24 20:42:51.524434
20	8	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-24 20:42:49.210385	2015-02-24 20:42:49.210385
21	9	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-24 20:42:49.236499	2015-02-24 20:42:49.236499
22	9	PROMOTER DEMETHYLATION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-24 20:42:49.267265	2015-02-24 20:42:49.267265
23	10	LOSS	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL), a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic palbociclib significantly increased the median survival of a Notch-driven model of T-ALL.	2015-02-24 20:42:49.312965	2015-02-24 20:42:49.312965
24	11	OVEREXPRESSION	Cyclin E, while currenly not as widely implicated as its counterpart, cyclin D, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, are correlated with later stage disease and have been shown to lead to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	2015-02-24 20:42:49.341647	2015-02-24 20:42:49.341647
25	13	EXPRESSION	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-24 20:42:49.370166	2015-02-24 20:42:49.370166
26	12	EXPRESSION	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-24 20:42:49.402214	2015-02-24 20:42:49.402214
27	14	PROMOTER HYPERMETHYLATION	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the CDKN2A promoter region. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcomes for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	2015-02-24 20:42:49.426772	2015-02-24 20:42:49.426772
28	15	N-TERMINAL FRAME SHIFT	CEBPA N-terminal frame shift mutations that result in a premature stop codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML). CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-24 20:42:49.456669	2015-02-24 20:42:49.456669
29	15	MUTATION	CEBPA mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-24 20:42:49.487827	2015-02-24 20:42:49.487827
30	33	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-24 20:42:49.513337	2015-02-24 20:42:49.513337
31	16	DNAJB1-PRKACA	This fusion has been found to be very recurrent in the rare form of adolescent liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-24 20:42:49.536664	2015-02-24 20:42:49.536664
32	18	R882	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-02-24 20:42:49.56886	2015-02-24 20:42:49.56886
97	38	D842V	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-24 20:42:51.555276	2015-02-24 20:42:51.555276
33	19	L858R	EGFR L858R has long been recognized as a functionally significant mutation in cancer, and is one of the most prevalent single mutations in lung cancer. Best described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity to first and second generation TKI's like gefitinib and neratinib. NSCLC patients with this mutation treated with TKI's show increased overall and progression-free survival, as compared to chemotherapy alone. Third generation TKI's are currently in clinical trials that specifically focus on mutant forms of EGFR, a few of which have shown efficacy in treating patients that failed to respond to earlier generation TKI therapies.	2015-02-24 20:42:49.597231	2015-02-24 20:42:49.597231
34	19	T790M	EGFR T790M was one of the very first mutations recognized to confer resistance to targeted therapies. While successful in amplified EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients harboring this mutation before treatment is notably lower. This lack of efficacy can likely be to blame for the poorer prognosis for patients with this mutation as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired resistance to TKI inhibition have been shown to harbor this mutation, implicating it as a mechanism of acquired therapy resistence. The third generation TKI's are being developed with this resistance problem in mind, and early stage experiments have shown some efficacy of these drugs in previously resistant tumors with mutant EGFR.	2015-02-24 20:42:49.622941	2015-02-24 20:42:49.622941
35	20	D769H	ERBB2 D769H was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-24 20:42:49.656581	2015-02-24 20:42:49.656581
36	20	D769Y	ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-24 20:42:49.681136	2015-02-24 20:42:49.681136
37	20	DEL 755-759	ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-24 20:42:49.708238	2015-02-24 20:42:49.708238
38	20	G309A	ERBB2 G309A was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-24 20:42:49.734253	2015-02-24 20:42:49.734253
39	20	L755S	ERBB2 L755S was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike many of the other variants queried. This mutation was also shown to confer resistance to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.	2015-02-24 20:42:49.766465	2015-02-24 20:42:49.766465
40	20	L755W	ERBB2 L755W was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-24 20:42:49.795654	2015-02-24 20:42:49.795654
41	20	P780INS	ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-24 20:42:49.825303	2015-02-24 20:42:49.825303
42	20	R678Q	ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-24 20:42:49.85515	2015-02-24 20:42:49.85515
43	20	R896C	ERBB2 R896C was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-24 20:42:49.882916	2015-02-24 20:42:49.882916
44	20	V777L	ERBB2 V777L was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-24 20:42:49.90664	2015-02-24 20:42:49.90664
45	20	V842I	ERBB2 V842I was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-24 20:42:49.932529	2015-02-24 20:42:49.932529
46	21	L536Q	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of these ligand binding domain mutations, and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-24 20:42:49.963732	2015-02-24 20:42:49.963732
47	21	N538G	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-24 20:42:49.994639	2015-02-24 20:42:49.994639
48	21	Y537C	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-24 20:42:50.040392	2015-02-24 20:42:50.040392
49	21	Y537N	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-24 20:42:50.066472	2015-02-24 20:42:50.066472
50	21	Y537S	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-24 20:42:50.096254	2015-02-24 20:42:50.096254
51	22	FGFR2-MGEA5	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. Additionally, tumor size reduction was achieved by both ponatinib and pazopanib treatments administered separately in a single patient with intrahepatic cholangiocarcinoma and this fusion. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-24 20:42:50.130852	2015-02-24 20:42:50.130852
52	23	FGFR3-BAIAP2L1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-24 20:42:50.16385	2015-02-24 20:42:50.16385
53	23	FGFR3-TACC3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-24 20:42:50.196381	2015-02-24 20:42:50.196381
54	24	ITD MUTATIONS	FLT3-ITD (internal tandem duplications) frequently occur in patients with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated sequence resulting in an in-frame duplication event. The length of these duplications can vary widely which may have prognostic consequences, but this has not been conclusively determined. FLT3-ITD mutations overall have generally been associated with poor prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the prognosis associated with this variant.	2015-02-24 20:42:50.22114	2015-02-24 20:42:50.22114
55	24	TKD MUTATIONS	FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common than FLT3-ITD (internal tandem duplication) mutations and may not confer the same prognostic impact. Although the majority of mutations are point mutations effecting D835 (most frequently D835Y), a small proportion involve an in-frame deletion of I836. These mutations are within the activation loop of the second tyrosine kinase domain of FLT3 and thought to result in constitutive activation of the receptor.	2015-02-24 20:42:50.248906	2015-02-24 20:42:50.248906
56	25	EXPRESSION	GATA2 misregulation has been observed in a number of hematologic malignancies, as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung cancer.	2015-02-24 20:42:50.276161	2015-02-24 20:42:50.276161
57	26	R132	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-24 20:42:50.305695	2015-02-24 20:42:50.305695
58	26	R132C	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-24 20:42:50.333946	2015-02-24 20:42:50.333946
59	26	R132H	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-24 20:42:50.367584	2015-02-24 20:42:50.367584
60	26	R132L	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-24 20:42:50.399508	2015-02-24 20:42:50.399508
61	27	R140Q/L	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival as compared to their wild-type counterparts. In myelodysplastic syndromes, however, no prognostic link was discovered between IDH2 mutation status and overall survival.	2015-02-24 20:42:50.427774	2015-02-24 20:42:50.427774
62	27	R172K	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic syndromes, studies did not find a prognostic association between this variant and patient outcomes.	2015-02-24 20:42:50.45654	2015-02-24 20:42:50.45654
63	28	V617F	JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases, occuring in around half of all MPD's. While less associated with cancer, when it is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The V617F mutation is an activating mutation, resulting in increased kinase activity. The mutation seems to be restricted to hematologic malignancies. Treatment of JAK mutant diseases with ruxolitinib has seen some clinical success.	2015-02-24 20:42:50.485237	2015-02-24 20:42:50.485237
64	29	D816V	KIT D816V is a mutation observed in acute myeloid leukemia (AML). This variant has been linked to poorer prognosis and worse outcome in AML patients.	2015-02-24 20:42:50.514558	2015-02-24 20:42:50.514558
65	29	EXON 11 MUTATIONS	c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-02-24 20:42:50.53944	2015-02-24 20:42:50.53944
66	29	INTERNAL DUPLICATION	c-KIT internal duplications have been observed in exon 11, within the juxtamembrane domain. In a case study of an anal melanoma patient harboring this event, imatinib confered marked response. Also, cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-02-24 20:42:50.56982	2015-02-24 20:42:50.56982
67	29	EXON 13 MUTATIONS	c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations. These mutations lie within the tyrosine kinase 1 domain, and are found primarily in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations show sensitivity to imatinib and sunitinib treatment. However, only imatinib has seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant cases.	2015-02-24 20:42:50.596828	2015-02-24 20:42:50.596828
68	29	EXON 14 MUTATIONS	c-KIT exon 14 mutations lie within the tyrosine kinase domain of the protein. While relatively rare in primary gastrointestinal tumors, they are notably more prevalent in refractory disease, suggesting a role in imatinib resistance. Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.	2015-02-24 20:42:50.627938	2015-02-24 20:42:50.627938
69	29	EXON 17 MUTATIONS	c-KIT exon 17 mutations lie within the TK2 domain, containing the activation loop of the protein. In cell lines, mutations within this domain have been shown to be sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.	2015-02-24 20:42:50.655941	2015-02-24 20:42:50.655941
70	29	EXON 9 MUTATIONS	c-KIT exon 9 mutations lie within the dimerization motif of the protein. Relative to other KIT mutations, exon 9 mutations have been associated with better overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and in vivo.	2015-02-24 20:42:50.689263	2015-02-24 20:42:50.689263
71	29	L576P	KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain. It is one of the most recurrent KIT mutations in melanoma, and both in vitro and in vivo studies have shown sensitivity to imatinib.	2015-02-24 20:42:50.721899	2015-02-24 20:42:50.721899
72	29	V654A	KIT V654A is an exon 13 mutation that lies within the tyrosine kinase 1 domain of the protein. It has been shown to be an activating mutation by in vitro studies. This mutation is associated with imatinib resistance in melanoma patients. However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have seen success in acheiving tumor response.	2015-02-24 20:42:50.77593	2015-02-24 20:42:50.77593
73	30	EXON 1 MUTATIONS	A study by Lièvre et al in 2006 showed that colorectal cancer patients with KRAS exon 1 mutations had low cetuximab response rates.	2015-02-24 20:42:50.807521	2015-02-24 20:42:50.807521
74	30	EXON 2 MUTATIONS	In a study by Pao et al in 2005, non-small cell lung cancer patients harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors gefinitib and erlotinib.	2015-02-24 20:42:50.843855	2015-02-24 20:42:50.843855
75	30	G12	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-24 20:42:50.874129	2015-02-24 20:42:50.874129
76	30	G12/G13	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-24 20:42:50.905387	2015-02-24 20:42:50.905387
77	30	G12C	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-24 20:42:50.93702	2015-02-24 20:42:50.93702
78	30	G12D	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-24 20:42:50.966845	2015-02-24 20:42:50.966845
79	30	G13	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-24 20:42:51.000381	2015-02-24 20:42:51.000381
80	30	G13D	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-24 20:42:51.035426	2015-02-24 20:42:51.035426
81	31	P124S	MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder and colon cancer to a lesser degree. The P124S mutation has been shown to contribute to AZD6244 resistance in melanoma cell lines, but considerably less so than its Q56P counterpart.	2015-02-24 20:42:51.067615	2015-02-24 20:42:51.067615
82	31	Q56P	MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This mutation has been shown to confer considerable resistance to AZD6244 treatment of melanoma cell lines.	2015-02-24 20:42:51.099602	2015-02-24 20:42:51.099602
83	32	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-24 20:42:51.132719	2015-02-24 20:42:51.132719
84	34	PROMOTER METHYLATION	MGMT promoter methylation has been observed to impact tumor progression in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating agent carmustine has shown efficacy. In patients lacking methylation, combining carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more experiments are required. Clinical trials have also shown selective sensitivity of promoter methylation-positive patients to temozolomide, making a case for wider methylation screening in GBM patients.	2015-02-24 20:42:51.163789	2015-02-24 20:42:51.163789
85	35	EXON 12 MUTATIONS	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-02-24 20:42:51.198941	2015-02-24 20:42:51.198941
86	35	W288FS	NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most common NPM1 mutation identified in acute myeloid leukemia. This mutation results in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation, it is generally grouped with other exon 12 mutations for patient analysis (see NPM1 Exon 12 variants for more information).	2015-02-24 20:42:51.235288	2015-02-24 20:42:51.235288
87	36	EXON 1 MUTATIONS	NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown to be correlated with poorer overall survival relative to wild-type NRAS in melanoma patients.	2015-02-24 20:42:51.263407	2015-02-24 20:42:51.263407
88	36	EXON 2 MUTATIONS	NRAS exon 2 mutations have been shown to be correlated with poorer overall survival in melanoma patients and colorectal cancer patients, however no prognostic impact was seen in acute myeloid leukemia patients.	2015-02-24 20:42:51.287475	2015-02-24 20:42:51.287475
89	36	EXON 3 MUTATIONS	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-24 20:42:51.318836	2015-02-24 20:42:51.318836
90	36	EXON 4 MUTATIONS	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-24 20:42:51.343677	2015-02-24 20:42:51.343677
91	36	G12	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-24 20:42:51.36922	2015-02-24 20:42:51.36922
92	36	G13D	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-24 20:42:51.398319	2015-02-24 20:42:51.398319
93	36	Q61	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-24 20:42:51.422676	2015-02-24 20:42:51.422676
94	36	Q61L	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-24 20:42:51.447906	2015-02-24 20:42:51.447906
95	36	Q61R	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-24 20:42:51.497958	2015-02-24 20:42:51.497958
98	38	D842Y	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-24 20:42:51.582868	2015-02-24 20:42:51.582868
99	38	DEL I843	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-24 20:42:51.612723	2015-02-24 20:42:51.612723
100	38	DI842-843IM	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-24 20:42:51.640469	2015-02-24 20:42:51.640469
101	37	E542K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-24 20:42:51.668863	2015-02-24 20:42:51.668863
102	37	E545K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-24 20:42:51.698294	2015-02-24 20:42:51.698294
103	37	EXON 20 MUTATIONS	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-24 20:42:51.728872	2015-02-24 20:42:51.728872
104	37	EXON 9 MUTATIONS	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-24 20:42:51.758396	2015-02-24 20:42:51.758396
105	37	H1047R	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-24 20:42:51.803616	2015-02-24 20:42:51.803616
106	39	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	2015-02-24 20:42:51.845126	2015-02-24 20:42:51.845126
107	17	DNAJB1-PRKACA	This fusion has been found to be very recurrent in a rare form of adolescent liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined, and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-24 20:42:51.896267	2015-02-24 20:42:51.896267
108	41	R233*	PTEN R233* has been shown to be a loss of function mutation, and PTEN loss has been the subject of considerable research in breast cancer. PTEN loss may sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data to support that PTEN loss is both associated with poorer prognosis, and no change in prognosis.	2015-02-24 20:42:51.952331	2015-02-24 20:42:51.952331
109	40	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and Arsenic Trioxide for treating these patients, and early results seem promising.	2015-02-24 20:42:52.002432	2015-02-24 20:42:52.002432
110	42	C634W	RET C639W has been implicated as an alternate mechanism of activating RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the C639W mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib.	2015-02-24 20:42:52.044861	2015-02-24 20:42:52.044861
111	42	M918T	RET M819T is the most common somatically acquired mutation in medullary thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the M918T mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T leads to more aggressive MTC with a poorer prognosis.	2015-02-24 20:42:52.085353	2015-02-24 20:42:52.085353
112	44	K666N	SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients.	2015-02-24 20:42:52.12109	2015-02-24 20:42:52.12109
113	44	K700E	SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients. Additionally, these mutations are the most common SF3B1 mutation observed in MDS and highly associated with subtypes of MDS that are defined by ringed sideroblasts.	2015-02-24 20:42:52.152796	2015-02-24 20:42:52.152796
129	6	LOSS-OF-FUNCTION	BRCA1 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.	2015-02-24 20:42:52.621731	2015-02-24 20:42:52.621731
114	45	R175H	While loss-of-function events in TP53 are very common in cancer, the R175H variant seems not only to result in loss of tumor-suppression, but also acts as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell lines harboring this mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been shown that the R175H mutation is correlated with worse overall survival than wild-type TP53, but is not as detrimental as the R248W variant.	2015-02-24 20:42:52.180226	2015-02-24 20:42:52.180226
115	45	R248Q	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival. The R248Q mutation has also shown an increased invasive behavior in cell lines. This is specific to the 248Q variant.	2015-02-24 20:42:52.207995	2015-02-24 20:42:52.207995
116	45	R248W	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival.	2015-02-24 20:42:52.241502	2015-02-24 20:42:52.241502
117	45	R249T	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-24 20:42:52.271104	2015-02-24 20:42:52.271104
118	45	R249W	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-24 20:42:52.319671	2015-02-24 20:42:52.319671
119	45	R273C	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-24 20:42:52.345186	2015-02-24 20:42:52.345186
120	45	R273H	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-24 20:42:52.37138	2015-02-24 20:42:52.37138
121	45	V173G/A	While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-24 20:42:52.3985	2015-02-24 20:42:52.3985
122	46	FRAMESHIFT TRUNCATION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-24 20:42:52.428574	2015-02-24 20:42:52.428574
123	46	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-24 20:42:52.457346	2015-02-24 20:42:52.457346
124	47	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-24 20:42:52.484735	2015-02-24 20:42:52.484735
125	48	Q157P/R	U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is less common than the S34F mutation, occurs in the second zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-02-24 20:42:52.512722	2015-02-24 20:42:52.512722
126	48	S34Y/F	U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is the most commonly identified variant in MDS, occurs in the first zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-02-24 20:42:52.54051	2015-02-24 20:42:52.54051
127	49	EXON 7 MUTATIONS	WT1 exon 7 mutations have been shown to be recurrent in acute myeloid leukemia. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 7 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-24 20:42:52.568198	2015-02-24 20:42:52.568198
128	49	EXON 9 MUTATIONS	WT1 exon 9 mutations have been shown to be recurrent in acute myeloid leukemia, although at a less frequent rate than their exon 7 counterparts. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 9 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-24 20:42:52.591778	2015-02-24 20:42:52.591778
130	7	LOSS-OF-FUNCTION	BRCA2 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.	2015-02-24 20:42:52.646878	2015-02-24 20:42:52.646878
\.


--
-- Name: variants_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variants_id_seq', 130, true);


--
-- PostgreSQL database dump complete
--

